{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import sys\n",
    "\n",
    "PACKAGE_DIR = \"/kaggle/src\"\n",
    "sys.path.append(PACKAGE_DIR)\n",
    "sys.path.append(os.path.join(PACKAGE_DIR, \"Penguin-ML-Library\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing /kaggle/input/whoosh-wheel-2-7-4/Whoosh-2.7.4-py2.py3-none-any.whl\n",
      "Requirement already satisfied: cached-property in /opt/conda/lib/python3.10/site-packages (from Whoosh==2.7.4) (1.5.2)\n",
      "Whoosh is already installed with the same version as the provided wheel. Use --force-reinstall to force an installation of the wheel.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "import whoosh_utils"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import gc\n",
    "import warnings\n",
    "\n",
    "import numpy as np\n",
    "import polars as pl\n",
    "import yaml\n",
    "from tqdm import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# train_idx = whoosh_utils.load_index(\"/kaggle/input/rich-index\")\n",
    "# searcher = whoosh_utils.get_searcher(train_idx)\n",
    "qp = whoosh_utils.get_query_parser()\n",
    "# train_idx.doc_count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\\'\"hoge\"\"fuga\"~2\\''\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "ti:\"hoge fuga\"\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge fuga\"~4'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:\"hoge ti\" AND (ti:fuga OR ab:fuga OR clm:fuga OR detd:fuga OR cpc::fuga))\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:hoge ADJ2 ti:fuga'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:\"hoge fuga\" OR ab:\"hoge fuga\" OR clm:\"hoge fuga\" OR detd:\"hoge fuga\" OR cpc:\"hoge fuga\")\n"
     ]
    }
   ],
   "source": [
    "query = \"hoge ADJ3 fuga\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:hoge-fuga'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND (ti:fuga OR ab:fuga OR clm:fuga OR detd:fuga OR cpc:fuga))\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:hoge&fuga'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "(ti:hoge AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:(hoge|fuga)'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(cpc:NOT& AND cpc:hoge)\n"
     ]
    }
   ],
   "source": [
    "query = 'NOT&cpc:hoge'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(cpc:NOT@ AND cpc:hoge)\n"
     ]
    }
   ],
   "source": [
    "query = 'NOT@cpc:hoge'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:hoge OR ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = '(ti:hoge) OR (ti:fuga)'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"ti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "ti:\"hoge fuga foo\"\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"\\'hoge\\'\\'fuga\\'\\'foo\\'\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:\"hoge fuga foo\" AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"ti:\"\\'hoge\\'\\'fuga\\'\\'foo\\'\"ti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:hoge OR ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"\\nOR\\nti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND (ti:or OR ab:or OR clm:or OR detd:or OR cpc:​OR​) AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"\\u200BOR\\u200Bti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:hoge AND cpc:| AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\" | ti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(cpc:! AND ti:hoge)\n"
     ]
    }
   ],
   "source": [
    "query = '!ti:\"hoge\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n",
      "((ti:mobile OR ab:mobile OR clm:mobile OR detd:mobile OR cpc:mobile) AND NOT (ti:mobile OR ab:mobile OR clm:mobile OR detd:mobile OR cpc:mobile OR ti:phone OR ab:phone OR clm:phone OR detd:phone OR cpc:phone))\n"
     ]
    }
   ],
   "source": [
    "query = \"mobile NOT (mobile OR phone)\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n",
      "((ti:mobile OR ab:mobile OR clm:mobile OR detd:mobile OR cpc:mobile) AND NOT (ti:\"mobile or phone\" OR ab:\"mobile or phone\" OR clm:\"mobile or phone\" OR detd:\"mobile or phone\" OR cpc:\"mobile OR phone\"))\n"
     ]
    }
   ],
   "source": [
    "query = 'mobile NOT \"mobile OR phone\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "(ti:\"mobile phone\" OR ab:\"mobile phone\" OR clm:\"mobile phone\" OR detd:\"mobile phone\" OR cpc:\"mobile phone\")\n"
     ]
    }
   ],
   "source": [
    "query = '\"mobile phone\"~2'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "(ti:\"mobile phone\" OR ab:\"mobile phone\" OR clm:\"mobile phone\" OR detd:\"mobile phone\" OR cpc:\"mobile phone\")\n"
     ]
    }
   ],
   "source": [
    "query = \"mobile ADJ2 phone\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "(ti:hoge AND (ti:\"mobile phone\" OR ab:\"mobile phone\" OR clm:\"mobile phone\" OR detd:\"mobile phone\" OR cpc:\"mobile phone\"))\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:hoge\"mobile phone\"~2'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND (ti:mobile OR ab:mobile OR clm:mobile OR detd:mobile OR cpc:mobile))\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:hoge\"mobile\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:hoge AND ti:mobile AND ti:fuga)\n"
     ]
    }
   ],
   "source": [
    "query = 'ti:\"hoge\"ti:\"mobile\"ti:\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "((ti:hoge OR ab:hoge OR clm:hoge OR detd:hoge OR cpc:hoge) AND (ti:mobile OR ab:mobile OR clm:mobile OR detd:mobile OR cpc:mobile) AND (ti:fuga OR ab:fuga OR clm:fuga OR detd:fuga OR cpc:fuga))\n"
     ]
    }
   ],
   "source": [
    "query = '\"hoge\"\"mobile\"\"fuga\"'\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7\n",
      "(ti:\"mobile phone\" OR ti:\"mobile tv\")\n"
     ]
    }
   ],
   "source": [
    "query = \"(ti:mobile ADJ2 phone) OR (ti:mobile ADJ2 tv)\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6\n",
      "(ti:mobile AND ti:adj2 AND (ti:phone OR ti:tv))\n"
     ]
    }
   ],
   "source": [
    "query = \"ti:(mobile ADJ2 (phone OR tv))\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:usa OR ab:usa OR clm:usa OR detd:usa OR cpc:USA OR id:USA)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[<Hit {'ab': 'The present invention provides polynucleotides encoding human and murine guanylate binding protein polypeptides, fragments and homologues thereof. Also provided are vectors, host cells, antibodies, and recombinant and synthetic methods for producing said polypeptides. The present invention further relates to diagnostic and therapeutic methods for applying the guanylate binding protein polypeptides to the diagnosis, treatment, and/or prevention of various diseases and/or disorders related to these polypeptides, such as rheumatoid arthritis and/or conditions related to aberrant NF-κB activity, guanylate binding activity and GTPase activity. The present invention further relates to screening methods for identifying agonists and antagonists of the polynucleotides and polypeptides of the present invention.', 'clm': '1. An isolated nucleic acid molecule comprising a polynucleotide sequence selected from the group consisting of:\\n (a) an isolated polynucleotide encoding a polypeptide comprising amino acids 1–668 of SEQ ID NO: 5; \\n (b) an isolated polynucleotide encoding a polypeptide comprising amino acids 2–668 of SEQ ID NO: 5; \\n (c) an isolated polynucleotide comprising nucleotides 88 to 2109 of SEQ ID NO: 4, wherein said nucleotides encode a polypeptide corresponding to amino acids 1 to 668 of SEQ ID NO: 5; \\n (d) an isolated polynucleotide comprising nucleotides 91 to 2091 of SEQ ID NO: 4, wherein said nucleotides encode a polypeptide corresponding to amino acids 2 to 668 of SEQ ID NO: 5; \\n (e) ATCC Deposit No: PTA-6007; and \\n (f) an isolated polynucleotide that hybridizes under stringent hybridization conditions to a polynucleotide sequence provided in (a)–(e), wherein said polynucleotide encodes a polypeptide that has guanylate binding activity wherein such stringent hybridization conditions are selected from the group consisting of (A) 7% SDS, 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 0.1×SSC, 0.1% SDS at 65° C. and (B) 4×SSC at 65° C., followed by washing in 0.1×SSC at 65° C. for one hour. \\n \\n     \\n     \\n       2. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (a). \\n     \\n     \\n       3. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (b). \\n     \\n     \\n       4. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (c). \\n     \\n     \\n       5. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (d). \\n     \\n     \\n       6. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (e). \\n     \\n     \\n       7. The isolated nucleic acid molecule of  claim 1 , wherein said polynucleotide is (f). \\n     \\n     \\n       8. An isolated nucleic acid molecule comprising a polynucleotide having a nucleotide sequence that is at least 95% identical to a polynucleotide sequence provided in  claim 2 , wherein percent identity is calculated using a CLUSTALW global sequence alignment using default parameters, wherein said polynucleotide encodes a polypeptide that has guanylate binding activity. \\n     \\n     \\n       9. An isolated nucleic acid molecule comprising a polynucleotide having a nucleotide sequence that is at least 95% identical to a polynucleotide sequence provided in  claim 3 , wherein percent identity is calculated using a CLUSTALW global sequence alignment using default parameters, wherein said polynucleotide encodes a polypeptide that has guanylate binding activity. \\n     \\n     \\n       10. An isolated nucleic acid molecule comprising a polynucleotide having a nucleotide sequence that is at least 95% identical to a polynucleotide sequence provided in  claim 4 , wherein percent identity is calculated using a CLUSTALW global sequence alignment using default parameters, wherein said polynucleotide encodes a polypeptide that has guanylate binding activity. \\n     \\n     \\n       11. An isolated nucleic acid molecule comprising a polynucleotide having a nucleotide sequence that is at least 95% identical to a polynucleotide sequence provided in  claim 5 , wherein percent identity is calculated using a CLUSTALW global sequence alignment using default parameters, wherein said polynucleotide encodes a polypeptide that has guanylate binding activity. \\n     \\n     \\n       12. An isolated nucleic acid molecule comprising a polynucleotide having a nucleotide sequence that is at least 95% identical to a polynucleotide sequence provided in  claim 6 , wherein percent identity is calculated using a CLUSTALW global sequence alignment using default parameters, wherein said polynucleotide encodes a polypeptide that has guanylate binding activity. \\n     \\n     \\n       13. A recombinant vector comprising the isolated nucleic acid molecule of  claim 1 . \\n     \\n     \\n       14. An isolated recombinant host cell comprising the vector sequence of  claim 13 . \\n     \\n     \\n       15. A method of making an isolated polypeptide comprising:\\n (a) culturing the isolated recombinant host cell of  claim 14  under conditions such that said polypeptide is expressed; and \\n (b) recovering said polypeptide.', 'cpc': 'C07K14/47', 'detd': 'This application claims benefit to U.S. provisional patent application Ser. No. 60/475,234 filed on Jun. 2, 2003. The entire teachings of the referenced application are incorporated herein by reference. \\n    \\n    \\n     FIELD OF THE INVENTION \\n     The present invention provides novel polynucleotides encoding guanylate binding proteins, fragments and homologues thereof. Also provided are vectors, host cells, antibodies, and recombinant and synthetic methods for producing said polypeptides. The present invention further relates to diagnostic and therapeutic methods for applying these novel guanylate binding proteins to the diagnosis, treatment, and/or prevention of various diseases and/or disorders related to these polypeptides. The present invention further relates to screening methods for identifying agonists and antagonists of the polynucleotides and polypeptides of the present invention. \\n     \\n       \\n         \\n           \\n               \\n            \\n               \\n                   \\n               \\n               \\n                 Amino Acid Abbreviations \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Single-Letter Code \\n                 Three-Letter Code \\n                 Name \\n               \\n               \\n                   \\n               \\n               \\n                 A \\n                 Ala \\n                 Alanine \\n               \\n               \\n                 V \\n                 Val \\n                 Valine \\n               \\n               \\n                 L \\n                 Leu \\n                 Leucine \\n               \\n               \\n                 I \\n                 Ile \\n                 Isoleucine \\n               \\n               \\n                 P \\n                 Pro \\n                 Proline \\n               \\n               \\n                 F \\n                 Phe \\n                 Phenylalanine \\n               \\n               \\n                 W \\n                 Trp \\n                 Tryptophan \\n               \\n               \\n                 M \\n                 Met \\n                 Methionine \\n               \\n               \\n                 G \\n                 Gly \\n                 Glycine \\n               \\n               \\n                 S \\n                 Ser \\n                 Serine \\n               \\n               \\n                 T \\n                 Thr \\n                 Threonine \\n               \\n               \\n                 C \\n                 Cys \\n                 Cysteine \\n               \\n               \\n                 Y \\n                 Tyr \\n                 Tyrosine \\n               \\n               \\n                 N \\n                 Asn \\n                 Asparagine \\n               \\n               \\n                 Q \\n                 Gln \\n                 Glutamine \\n               \\n               \\n                 D \\n                 Asp \\n                 Aspartic Acid \\n               \\n               \\n                 E \\n                 Glu \\n                 Glutamic Acid \\n               \\n               \\n                 K \\n                 Lys \\n                 Lysine \\n               \\n               \\n                 R \\n                 Arg \\n                 Arginine \\n               \\n               \\n                 H \\n                 His \\n                 Histidine \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                   \\n               \\n               \\n                 Functionally Equivalent Codons \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 Amino Acid \\n                 Codons \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Alanine \\n                 Ala \\n                 A \\n                 GCA GCC GCG GCU \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 Cysteine \\n                 Cys \\n                 C \\n                 UGC UGU \\n               \\n               \\n                   \\n               \\n               \\n                 Aspartic Acid \\n                 Asp \\n                 D \\n                 GAC GAU \\n               \\n               \\n                   \\n               \\n               \\n                 Glumatic Acid \\n                 Glu \\n                 E \\n                 GAA GAG \\n               \\n               \\n                   \\n               \\n               \\n                 Phenylalanine \\n                 Phe \\n                 F \\n                 UUC UUU \\n               \\n               \\n                   \\n               \\n               \\n                 Glycine \\n                 Gly \\n                 G \\n                 GGA GGC GGG GGU \\n               \\n               \\n                   \\n               \\n               \\n                 Histidine \\n                 His \\n                 H \\n                 CAC CAU \\n               \\n               \\n                   \\n               \\n               \\n                 Isoleucine \\n                 Ile \\n                 I \\n                 AUA AUC AUU \\n               \\n               \\n                   \\n               \\n               \\n                 Lysine \\n                 Lys \\n                 K \\n                 AAA AAG \\n               \\n               \\n                   \\n               \\n               \\n                 Methionine \\n                 Met \\n                 M \\n                 AUG \\n               \\n               \\n                   \\n               \\n               \\n                 Asparagine \\n                 Asn \\n                 N \\n                 AAC AAU \\n               \\n               \\n                   \\n               \\n               \\n                 Proline \\n                 Pro \\n                 P \\n                 CCA CCC CCG CCU \\n               \\n               \\n                   \\n               \\n               \\n                 Glutamine \\n                 Gln \\n                 Q \\n                 CAA CAG \\n               \\n               \\n                   \\n               \\n               \\n                 Threonine \\n                 Thr \\n                 T \\n                 ACA ACC ACG ACU \\n               \\n               \\n                   \\n               \\n               \\n                 Valine \\n                 Val \\n                 V \\n                 GUA GUC GUG GUU \\n               \\n               \\n                   \\n               \\n               \\n                 Tryptophan \\n                 Trp \\n                 W \\n                 UGG \\n               \\n               \\n                   \\n               \\n               \\n                 Tyrosine \\n                 Tyr \\n                 Y \\n                 UAC UAU \\n               \\n               \\n                   \\n               \\n               \\n                 Leucine \\n                 Leu \\n                 L \\n                 UUA UUG CUA CUC CUG CUU \\n               \\n               \\n                   \\n               \\n               \\n                 Arginine \\n                 Arg \\n                 R \\n                 AGA AGG CGA CGC CGG CGU \\n               \\n               \\n                   \\n               \\n               \\n                 Serine \\n                 Ser \\n                 S \\n                 ACG AGU UCA UCC UCG UCU \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     BACKGROUND OF THE INVENTION \\n     Guanylate binding proteins (GBP) were first identified as interferon-inducible proteins in fibroblasts (Cheng et al., (1983)  J. Biol. Chem.  258:7746–7750). They were identified by their ability to bind to guanylate-containing agaroses including GMP-agarose, GDP-agarose, and GTP-agarose. They failed to bind to ATP-agarose and all other nucleotide agaroses tested (Cheng et al., (1991)  Mol. Cell. Biol.  11:4717–4725). Purified GBP-1 protein was shown to hydrolyze GTP, but not ATP, UTP, or CTP with a K m  for substrate of 470 μM. The hydrolysis reaction occurred with a high turnover rate and yielded primarily GMP rather than GDP (Schwemmle et al., (1994)  J. Biol. Chem.  269:11299–11305). The hydrolysis reaction was thought to occur in two successive steps since pyrophosphate was not detected as a reaction product. GBP-1 was also shown to possess a functional CX 1 X 2 X 3  (SEQ ID NO:1) isoprenylation signal at the C terminus of the molecule (the signal comprises a cysteine residue (C) followed by two typically, but not necessarily, aliphatic amino acids (X 1  and X 2 ) and another amino acid (X 3 )). \\n     The crystal structure of human GBP-1 in nucleotide free (Prakash et al., (2000)  Nature  403:567–571) and bound states (Prakash et al., (2000)  EMBO J.  17:4555–4564) was solved to 1.8 Å resolution. The structure resembled that of other large GTP-binding proteins including Mx and dynamin. The structure consists of a compact amino-terminal globular α,β domain and an elongated C-terminal α-helical domain separated by a short intermediate region. The globular domain contains the conserved regions characteristic of GTP-binding proteins with some additional insertions. Significant differences in the glycosidic bond angle and guanine base interaction domains were observed in GBP-1 relative to the canonical GTP binding protein Ras. Unlike Ras, the phosphate binding region of GBP1 is closed off from solvent, and therefore unavailable for interactions with external regulators such as GTPase activating proteins or guanine nucleotide exchange proteins. Similar to dynamins, nucleotide binding to GBP-1 induced oligomerization. \\n     Expression of GBP-1 is regulated by interferon in fibroblasts and a number of other cell types. These include mouse 70Z/3 pre-B cells (Patrone et al., (2001)  Mol. Immunol.  38:597–606); murine macrophage lines and peritoneal macrophages (Wynn et al., (1991)  J. Immunol.  147:4384–4392); rat bone marrow-derived macrophages and microglia (Vestal et al., (1996)  Bioch. Biophys. Res. Commun.  224:528–534); cultured mammary epithelial tumor cell lines (Sun et al., (1999)  Int. J. Cancer  80:624–629); normal and transformed keratinocytes (Saunders et al., (1999)  J. Invest. Dermatol.  112:977–983); and human endothelial cells (Lubeseder-Martellato et al., (2002)  Am. J. Pathol.  161:1749–1759). Lipopolysaccharide (LPS) was also shown to induce GBP-1 in rat bone marrow derived macrophages and microglia (Vestal et al., (1996)  Bioch. Biophys. Res. Commun.  224:528–534), and in cultured mammary epithelial tumor lines (Sun et al., (1999)  Int. J. Cancer  80:624–629). TNFα stimulation of human endothelial cells also induced GBP-1 expression (Lubeseder-Martellato et al., (2002)  Am. J. Pathol.  161:1749–1759). In this study, GBP-1 expression was highly associated with vascular endothelial cells and was induced in vessels of skin diseases that have a high inflammatory component. \\n     Since the original characterization of GBP-1, a number of related proteins have been identified in both mouse and human. In human, known GBPs include GBP-1, GBP-2 (Cheng et al., (1991)  Mol. Cell. Biol.  11:4717–4725), GBP-4, and GBP-5. GBP-2 was shown to have similar nucleotide binding properties and GTPase activities as GBP-1 (Neun et al., (1996)  FEBS Lett.  390:69–72). Some differences were noted in the product specificity of the GTPase reaction. \\n     In mouse, the known GBPs include mGBP-1, mGBP-2 (Boehm et al., (1998)  J. Immunol.  161:6715–6723), mGBP-3 (Han et al., (1998)  Bioch. Biophys. Acta  1384:373–386), mGBP-4/mag-2 (Wynn et al., (1991)  J. Immunol.  12:4384–4392) and mGBP-5 (Nguyen et al., (2002)  J. Interferon Cytokine Res.  22:899–909). Recombinant mGBP-3 protein was shown to possess similar nucleotide binding properties and GTPase activity as compared to other GBPs (Han et al., (1998)  Bioch. Biophys. Acta  1384:373–386). The tissue expression and induction patterns of all the mouse GBPs were similar (Nguyen et al., (2002)  J. Interferon Cytokine Res.  22:899–909). All five were induced in multiple organs in response to endotoxemia. In RAW264 and Swiss 3T3 cells, all five GBPs were induced in response to LPS, interleukin-1β (IL-1β), and TNFα. \\n     Although GBP-1 represents one of the most abundant proteins induced in response to interferon treatment, its function, or the function of other family members is not clear. Stable transfection of human GBP-1 into HeLa cells increased the resistance of the cells to the cytopathic effects of both vesicular stomatitis virus (VSV) and encephalomyocarditis virus (EMCV, Anderson et al., (1999)  Virology  256:8–14). Conversely, cells transfected with GBP-1-specific antisense were more sensitive to viral infection, suggesting that GBP-1 is required for an antiviral response to VSV and EMCV viruses. \\n     Expression of GBP-1 was inversely correlated with the proliferation of human microvascular and macrovascular endothelial cells (Guenzi et al., (2001)  EMBO J.  20:5568–5577). GBP-1 expression is induced by pro-inflammatory, anti-proliferative cytokines including IL-1β, TNFα, and TFN-γ. Overexpression of full length GBP-1 in HUVECs significantly slowed the proliferative rate of these cells without increasing cell death. Conversely, expression of antisense to GBP-1 in HUVECs resulted in increased proliferation of the cells. Overexpression of GBP-1 also inhibited the proliferative response of the cells to VEGF and bFGF. Activation of the MAP kinase pathway in response to VEGF or bFGF was not affected by GBP-1 overexpression. Overexpression did not inhibit IL-1β-induced adhesion of the cells to monocytes. The anti-proliferative effect of GBP-1 was mapped to the C-terminal α-helical domain. Mutants lacking GTPase activity or the prenylation motif were still able to inhibit proliferation. Expression of the α-helical domain alone inhibited proliferation whereas expression of the globular nucleotide binding domain alone was inactive. \\n     In contrast to the anti-proliferative effect of GBP1 in HUVECs, overexpression of mGBP-2 in NIH3T3 stimulated proliferation (Gorbacheva et al., (2002)  J. Biol. Chem.  277:6080–6087). Cells constitutively expressing mGBP-2 formed foci when grown to post-confluence. They failed to grow in soft agar suggesting that they retained anchorage-dependent growth. These cells also grew as tumors in nude mice. Mutation of the nucleotide binding domain of mGBP-2 eliminated the effect on cell proliferation. This study suggests that mGBP-2 is able to alter the growth characteristics of fibroblasts. \\n     SUMMARY OF THE INVENTION \\n     The present invention provides guanylate binding proteins (e.g HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143)) encoded by polynucleotides that comprise, or alternatively consist of, the nucleotide sequences shown in  FIGS. 1–8  (SEQ ID NO:2, 4, 6, 8, 10, 12, 14 and 16, respectively) and fragments thereof, that comprise, or alternatively consist of the amino acid sequences shown in  FIGS. 1–8  (SEQ ID NO:3, 5, 7, 9, 11, 13, 15 and 17, respectively) and fragments thereof, and/or that are deposited as ATCC Deposit Number PTA-6007 on May 20, 2004. \\n     The present invention also relates to recombinant vectors comprising one or more isolated nucleic acid molecules of the present invention, and to host cells containing the recombinant vectors, as well as to methods of making such vectors and host cells. In addition, the present invention relates to the use of such vectors in the production of guanylate binding proteins (e.g., HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143)) and/or peptides using recombinant techniques. Synthetic methods for producing the polypeptides and polynucleotides of the present invention are also provided. \\n     The present invention also relates to diagnostic methods for detecting diseases, disorders, and/or conditions related to guanylate binding proteins (e.g., HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143)) and polynucleotides encoding guanylate binding proteins, and therapeutic methods for treating such diseases, disorders, and/or conditions. The present invention further relates to screening methods for identifying binding partners of the polypeptides. \\n     In another aspect, the present invention provides an isolated guanlyate binding protein (e.g., HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143)) comprising an amino acid sequence described herein (e.g., SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17) encoded by a polynucleotide described herein (e.g., SEQ ID NOs:2, 4, 6, 8, 10, 12 14, and/or 16). \\n     The present invention further relates to a nucleic acid molecule encoding a polypeptide fragment of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, or a polypeptide fragment encoded by a cDNA sequence included in a deposited clone that is hybridizable to SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16 or a fragment thereof. \\n     The present invention further relates to a nucleic acid molecule encoding a polypeptide domain of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 or a polypeptide domain encoded by a cDNA sequence included in a deposited clone that is hybridizable to SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule encoding a polypeptide epitope of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 or a polypeptide epitope encoded by a cDNA sequence included in a deposited clone that is hybridizable to SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule encoding a polypeptide comprising SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 or a cDNA sequence included in a deposited clone that is hybridizable SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule that is a variant comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule that is an allelic variant comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule that encodes a species homologue comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule comprising a nucleic acid sequence that is complimentary (antisense) to a nucleic acid sequence comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a nucleic acid molecule capable of hybridizing under stringent conditions to any one of the polynucleotides specified herein, wherein the polynucleotide does not hybridize under stringent conditions to a nucleic acid molecule having a nucleotide sequence of only A residues or of only T residues. \\n     The present invention further relates to an isolated nucleic acid molecule comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, wherein the polynucleotide comprises a nucleotide sequence encoding a guanylate binding protein. \\n     The present invention further relates to an isolated nucleic acid molecule SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, wherein the nucleic acid molecule comprises one of a nucleotide sequence encoding a polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, and a polypeptide encoded by a cDNA sequence included in a deposited clone that is hybridizable to SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to an isolated nucleic acid molecule comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, wherein the nucleic acid molecule comprises one of the entire nucleotide SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, and a cDNA sequence included in a deposited clone that is hybridizable SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to an isolated nucleic acid molecule comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, wherein the nucleic acid molecule comprises one or more sequential nucleotide deletions from either the 5′-terminus or the 3′-terminus of the nucleic acid molecule. \\n     The present invention further relates to an isolated polypeptide comprising a polypeptide fragment of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17; the present invention also relates to a nucleotide sequence encoding a polypeptide fragment of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 included in a deposited clone. \\n     The present invention further relates to a polypeptide domain of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, and to a nucleotide sequence encoding such a domain included in a deposited clone. \\n     The present invention further relates to a polypeptide epitope of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, and to a nucleotide sequence encoding such an epitope included in a deposited clone. \\n     The present invention further relates to a full length protein SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, and to a nucleotide sequence encoding such a full length protein included in a deposited clone. \\n     The present invention further relates to an isolated polypeptide comprising SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, wherein the full length protein comprises sequential amino acid deletions from either the C-terminus or the N-terminus. \\n     The present invention further relates to an isolated antibody that binds specifically to an isolated polypeptide of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17. \\n     The present invention further relates to a method for preventing, treating, or ameliorating a medical condition, comprising administering to a mammalian subject a therapeutically effective amount of a polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, or a fragment thereof, or a polynucleotide of SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16, or a fragment thereof. \\n     The present invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject, the method comprising the steps of: (a) determining the presence or absence of a mutation in a polynucleotide comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16; and (b) diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or absence of the mutation. \\n     The present invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject, the method comprising the steps of: (a) determining the presence and/or amount of expression of a polypeptide comprising SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 in a biological sample; and (b) diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence and/or amount of expression of the polypeptide. \\n     The present invention further relates to a method for identifying a binding partner to a polypeptide comprising SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, the method comprising the steps of: (a) contacting a polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 with a binding partner; and (b) determining whether the binding partner effects an activity of the polypeptide. \\n     The present invention further relates to a gene corresponding to a cDNA sequence comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16. \\n     The present invention further relates to a method of identifying an activity in a biological assay, the method comprising the steps of (a) expressing a polypeptide of polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 in a cell, (b) isolating the supernatant; (c) detecting an activity in a biological assay; and (d) identifying the protein in the supernatant having the activity. \\n     The present invention further relates to a process for making a polynucleotide sequence encoding a gene product having altered activity selected from the group consisting of polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, the method comprising the steps of: (a) shuffling a nucleotide sequence comprising SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16; (b) expressing the resulting shuffled nucleotide sequences; and, (c) selecting for altered activity of a polypeptide selected from the group consisting of polypeptide SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17 as compared to the activity of of the gene product of an unmodified nucleotide sequence encoding a polypeptide selected from the group consisting SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17. \\n     The present invention further relates to a shuffled polynucleotide sequence produced by a shuffling process, wherein the shuffled polynucleotide molecule encodes a gene product having enhanced tolerance to an inhibitor of any one of the activities of a polypeptide selected from the group consisting of SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17. \\n     The present invention further relates to a method for preventing, treating, or ameliorating a medical condition by treating a subject with a polypeptide comprising SEQ ID NOs:3, 5, 7, 9, 11, 13, 15 and/or 17, in addition to, its encoding nucleic acid (SEQ ID NOs:2, 4, 6, 8, 10, 12, and/or 16), wherein the medical condition is selected from the group consisting of a disorder related to aberrant NF-κB activity and an autoimmune condition. \\n     The present invention further relates to a method of identifying a compound that modulates a biological activity of a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide, the method comprising the steps of: (a) combining a candidate modulator compound with a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide having the sequence set forth in SEQ ID NO:7 and/or SEQ ID NO:17; and (b) measuring an effect of the candidate modulator compound on the biological activity of HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143). \\n     The present invention further relates to a method of identifying a compound that modulates a biological activity of a guanylate binding protein, the method comprising the steps of: (a) combining a candidate modulator compound with a host cell expressing a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide having the sequence as set forth in SEQ ID NO:7 and/or SEQ ID NO:17; and (b) measuring an effect of the candidate modulator compound on a biological activity of the expressed HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide. \\n     The present invention further relates to a method of identifying a compound that modulates a biological activity of a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide, the method comprising the steps of: (a) combining a candidate modulator compound with a host cell containing a vector described herein, whereby a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide is expressed by the cell; and (b) measuring an effect of the candidate modulator compound on a biological activity of the expressed HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide. \\n     The present invention further relates to a method of screening for a compound that is capable of modulating a biological activity of a HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) polypeptide, the method comprising the steps of: (a) providing a host cell as described herein; (b) determining a biological activity of HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) in the absence of a modulator compound; (c) contacting the cell with the modulator compound; and (d) determining the biological activity of HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) in the presence of the modulator compound, wherein a difference between the activity of HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or MGBPBMY4 (BC007143) in the presence of the modulator compound and in the absence of the modulator compound indicates a modulating effect of the compound. \\n     The present invention further relates to a compound that modulates a biological activity of human HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or mouse MGBPBMY4 (BC007143) as identified by the methods described herein. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with the polypeptide provided as SEQ ID NO:3, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is a pulmonary condition, ovarian cancer, testicular cancer or a related proliferative condition of the ovary or testicle. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of the polypeptide of of SEQ ID NO:3 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a member of the group consisting of ovarian cancer, testicular cancer and a pulmonary condition. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with the polypeptide provided as SEQ ID NO:5, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is a pulmonary condition. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of the polypeptide of of SEQ ID NO:5 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a pulmonary condition. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with the polypeptide provided as SEQ ID NO:7, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is hyperthyroidism, prostatic hypertrophy and a pulmonary condition. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of the polypeptide of SEQ ID NO:7 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a member of the group consisting of hyperthyroidism, prostatic hypertrophy and a pulmonary condition. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with the polypeptide provided as SEQ ID NO:9, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is multiple sclerosis, Parkinson&#39;s disease, an immune system disease, breast cancer, testicular cancer, and a pulmonary condition. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of the polypeptide of of SEQ ID NO:9 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a member of the group consisting of multiple sclerosis, Parkinson&#39;s disease, an immune system disease, breast cancer, testicular cancer and a pulmonary condition. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with human GBP1, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is a neurological condition, including Parkinson&#39;s, Alzheimer&#39;s and multiple sclerosis. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of GBP1 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a member of the group consisting of Parkinson&#39;s disease, Alzheimer&#39;s disease and multiple sclerosis. \\n     The invention further relates to a method for preventing, treating, or ameliorating a medical condition with human GBP5, in addition to, its encoding nucleic acid, or a modulator thereof, wherein the medical condition is a neurological condition, including Alzheimer&#39;s disease and multiple sclerosis, breast cancer, testicular cancer, Crohn&#39;s disease, or a pulmonary disease, including bronchitis. \\n     The invention further relates to a method of diagnosing a pathological condition or a susceptibility to a pathological condition in a subject comprising the steps of (a) determining the presence or amount of expression of GBP5 in a biological sample; (b) and diagnosing a pathological condition or a susceptibility to a pathological condition based on the presence or amount of expression of the polypeptide relative to a control, wherein said condition is a member of the group consisting of Alzheimer&#39;s disease, multiple sclerosis, breast cancer, testicular cancer, Crohn&#39;s disease and bronchitis. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE FIGURES/DRAWINGS \\n         FIGS. 1A–1C  show the polynucleotide sequence (SEQ ID NO:2) and the deduced amino acid sequence (SEQ ID NO:3) of the human guanylate binding protein HGBPBMY1 (AK096141) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 2454 nucleotides (SEQ ID NO:2) encoding a polypeptide of 638 amino acids (SEQ ID NO:3). \\n         FIGS. 2A–2C  show the polynucleotide sequence (SEQ ID NO:4) and the deduced amino acid sequence (SEQ ID NO:5) of the human guanylate binding protein HGBPBMY2 (4843 30 1 1; 4843 — 1) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 2367 nucleotides (SEQ ID NO:4), encoding a polypeptide of 788 amino acids (SEQ ID NO:5). \\n         FIGS. 3A–3B  show the polynucleotide sequence (SEQ ID NO:6) and deduced amino acid sequence (SEQ ID NO:7) of the novel human guanylated binding protein HGBPBMY3 (4843 30 2 1; 4843 — 2) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 1392 nucleotides (SEQ ID NO:6), encoding a polypeptide of 464 amino acids (SEQ ID NO:7). \\n         FIGS. 4A–4C  show the polynucleotide sequence (SEQ ID NO:8) and deduced amino acid sequence (SEQ ID NO:9) of the human guanylate binding protein HGBPBMY4 (FLJ10961) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 2952 nucleotides (SEQ ID NO:8), encoding a polypeptide of 563 amino acids (SEQ ID NO:9). \\n         FIGS. 5A–5C  show the polynucleotide sequence (SEQ ID NO:10) and deduced amino acid sequence (SEQ ID NO:11) of the mouse guanylate binding protein MGBPBMY1 (LOC229900) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 2484 nucleotides (SEQ ID NO:10), encoding a polypeptide of 632 amino acids (SEQ ID NO:11). \\n         FIGS. 6A–B  show the polynucleotide sequence (SEQ ID NO:12) and deduced amino acid sequence (SEQ ID NO:13) of the mouse guanylate binding protein MGBPBMY2 (LOC229902) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 1818 nucleotides (SEQ ID NO:12), encoding a polypeptide of 605 amino acids (SEQ ID NO:13). \\n         FIGS. 7A–7C  show the polynucleotide sequence (SEQ ID NO:14) and deduced amino acid sequence (SEQ ID NO:15) of the mouse guanylate biding protein MGBPBMY3 (BC031475) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 3387 nucleotides (SEQ ID NO:13), encoding a polypeptide of 385 amino acids (SEQ ID NO:14). \\n         FIGS. 8A–8D  show the polynucleotide sequence (SEQ ID NO:15) and deduced amino acid sequence (SEQ ID NO:16) of the novel mouse guanlate binding protein MGBPBMY4 (BC007143) of the present invention. The standard one-letter abbreviation for amino acids is used to illustrate the deduced amino acid sequence. The polynucleotide sequence contains a sequence of 4193 nucleotides (SEQ ID NO:15), encoding a polypeptide of 483 amino acids (SEQ ID NO:16). \\n         FIG. 9  is a table depicting the percent similarity and percent identity between human GBP1 and the polynucleotide sequences of the present invention. \\n         FIG. 10  is a schematic depicting the chromosome organization of the GBP family. Panel A of  FIG. 10  depicts human chromosome 1p22, Panel B of  FIG. 10  depicts mouse chromosome 3H1 and Panel C of  FIG. 10  depicts mouse chromosome 5E4. Arrows represent the direcion of transcription. The GBP genes of the present invention, with the exception of MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143), are indicated. \\n         FIG. 11A–11C  is an alignment showing the regions of identity and similarity between the encoded GBP proteins of the present invention (SEQ ID NOs:3, 5, 7, 9, 11, 13, 15) to GBP family members. The alignment was performed using the CLUSTALW algorithm using default parameters as described herein (Vector NTI suite of programs). The darkly shaded amino acids represent regions of matching identity. The lightly shaded amino acids represent regions of matching similarity. Spaces between residues indicate gapped regions of non-identity for the aligned polypeptides. \\n         FIG. 12  is a phylogenic tree of the mouse and human GBP families. \\n         FIG. 13  is tissue expression pattern of human GBP-2. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of GBP-2. \\n         FIG. 14  is bar graph depicting the tissue expression pattern of human GBP-4. Panels of cDNAs derived from normal and immune tissue were analyzed by real Time PCR for expression of human GBP-4. \\n         FIG. 15  is bar graph depicting the tissue expression pattern of human HGBPBMY4 (FLJ10961). Panels of cDNAs derived from normal and immune tissue were analyzed by real Time PCR for expression of HGBPBMY4 (FLJ10961). \\n         FIG. 16  is bar graph depicting the tissue expression pattern of human HGBPBMY2 (4843 30 1 1; 4843 — 1). Panels of cDNAs derived from normal and immune tissue were analyzed by real Time PCR for expression of HGBPBMY2 (4843 30 1 1; 4843 — 1). \\n         FIG. 17  is bar graph depicting the bar graph depicting the tissue expression pattern of human HGBPBMY3 (4843 30 2 1; 4843 — 2). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of HGBPBMY3 (4843 30 2 1; 4843 — 2). \\n         FIG. 18  is bar graph depicting the tissue expression pattern of human HGBPBMY1 (AK096141). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of HGBPBMY1 (AK096141). \\n         FIG. 19  is bar graph depicting the tissue expression pattern of mouse mGBP-1. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of mGBP-1. \\n         FIG. 20  is bar graph depicting the tissue expression pattern of mouse mGBP-2. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of mGBP-2. \\n         FIG. 21  is bar graph depicting the tissue expression pattern of mouse mGBP-3. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of mGBP-3. \\n         FIG. 22  is bar graph depicting the tissue expression pattern of mouse mGBP-4. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of mGBP-4. \\n         FIG. 23  is bar graph depicting the tissue expression pattern of mouse mGBP-5. Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of mGBP-5. \\n         FIG. 24  is bar graph depicting the tissue expression pattern of mouse MGBPBMY2 (LOC229902). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of MGBPBMY2 (LOC229902). \\n         FIG. 25  is bar graph depicting the tissue expression pattern of mouse MGBPBMY1 (LOC229900). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of MGBPBMY1 (LOC229900). \\n         FIG. 26  is bar graph depicting the tissue expression pattern of mouse MGBPBMY3 (BC031475). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of MGBPBMY3 (BC031475). \\n         FIG. 27  is bar graph depicting the tissue expression pattern of mouse MGBPBMY4 (BC007143). Panels of cDNAs derived from normal and immune tissue were analyzed by Real Time PCR for expression of MGBPBMY4 (BC007143). \\n         FIG. 28  is a series of bar graphs depicting induction of GBP expression by LPS and IFN-γ. RNA was isolated from THP-1 monocytes stimulated for 6 hours with either medium, LPS, or IFN-γ. Levels of HGBPBMY4 (FLJ10961) (Panel A), HGBPBMY2 (4843 30 1 1; 4843 — 1) (Panel B), and HGBPBMY3 (4843 30 2 1; 4843 — 2) (Panel C) were measured by Real Time PCR. \\n         FIG. 29  is a series of bar graphs depicting the effect of NF-kB inhibition on LPS-mediated induction of GBP-2. Panels A and B depict two independent groups of THP-1 monocytes that were stimulated with LPS in the presence and absence of the NF-κB inhibitor Compound 1 (denoted “peptide” in the figures). RNA was isolated and analyzed for GBP-2 expression by Real Time PCR. \\n         FIG. 30  is a series of bar graphs depicting the effect of NF-κB inhibition on LPS-mediated induction of GBP-4. Panels A and B depict two independent groups of THP-1 monocytes that were stimulated with LPS in the presence and absence of the NF-kB inhibitor Compound 1 (denoted “peptide” in the figure). RNA was isolated and analyzed for GBP-4 expression by Real Time PCR. \\n         FIG. 31  is a series of bar graphs depicting the effect of NF-kB inhibition on LPS-mediated induction of HGBPBMY4 (FLJ10961). Independent groups of THP-1 monocytes (Panels A and B) were stimulated with LPS in the presence and absence of the NF-kB inhibitor, Compound 1 (denoted peptide in the figure). RNA was isolated and analyzed for HGBPBMY4 (FLJ10961) expression by Real Time PCR. \\n         FIG. 32  is a series of bar graphs depicting the effect of NF-κB inhibition on LPS-mediated induction of HGBPBMY3 (4843 30 2 1; 4843 — 2). Independent groups of THP-1 monocytes (Panels A and B) were stimulated with LPS in the presence and absence of the NF-kB inhibitor, Compound 1 (denoted peptide in the figure). RNA was isolated and analyzed for HGBPBMY3 (4843 30 2 1; 4843 — 2) expression by Real Time PCR. \\n         FIG. 33  is a bar graph depicting mGBP-2 expression in mouse embryonic fibroblast lines derived from NF-kB and IkBα germline knockouts. Embryonic fibroblast lines derived form different knockout animals were stimulated for 2 or 8 hours with eiher TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of mGBP-2. \\n         FIG. 34  is a bar graph depicting mGBP-3 expression in mouse embryonic fibroblast lines derived from NF-κB and IκBα germline knockouts. Embryonic fibroblast lines derived form different knockout animals were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of mGBP-3. \\n         FIG. 35  is a bar graph depicting mGBP-4 expression in mouse embryonic fibroblast lines derived from NF-κB and IκBα germline knockouts. Embryonic fibroblast lines derived form different knockout animals were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of mGBP-4. \\n         FIG. 36  is a bar graph depicting MGBPBMY1 (LOC229900) expression in mouse embryonic fibroblast lines derived from NF-κB and IκBα (germline knockouts. Embryonic fibroblast lines derived from different knockout animals were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of MGBPBMY1 (LOC229900). \\n         FIG. 37  is a bar graph depicting MGBPBMY2 (LOC229902) expression in mouse embryonic fibroblast lines derived from NF-κB and IκBα germline knockouts. Embryonic fibroblast lines derived form different knockout animals were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of MGBPBMY2 (LOC229902). \\n         FIG. 38  is a bar graph depicting MGBPBMY4 (BC007143) expression in mouse embryonic fibroblast lines derived from NF-κB and IκBα germline knockouts. Embryonic fibroblast lines derived form different knockout animals were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and analyzed by Real Time PCR for expression of MGBPBMY4 (BC007143). \\n         FIG. 39  is a bar graph depicting expression of GBP-2 in synovial tissue. RNA was isolated from synovial tissue derived from three normal controls (denoted “Nor” in the figure), six osteoarthritis subjects (denoted “OA” in the figure), and six rheumatoid arthritis subjects, and analyzed for GBP-2 mRNA by Real Time PCR. \\n         FIG. 40  is a bar graph depicting expression of GBP-4 in synovial tissue. RNA was isolated from synovial tissue derived from three normal controls (denoted “Nor” in the figure), six osteoarthritis subjects (denoted “OA” in the figure), and six rheumatoid arthritis subjects, and analyzed for GBP-4 mRNA by Real Time PCR. \\n         FIG. 41  is a bar graph depicting expression of HGBPBMY2 (4843 30 1 1; 4843 — 1) in synovial tissue. RNA was isolated from synovial tissue derived from three normal controls (denoted “Nor” in the figure), six osteoarthritis subjects (denoted “OA” in the figure), and six rheumatoid arthritis subjects, and analyzed for HGBPBMY2 (4843 30 1 1; 4843 — 1) mRNA by Real Time PCR. \\n         FIG. 42  is a bar graph depicting expression of HGBPBMY4 (FLJ10961) in synovial tissue. RNA was isolated from synovial tissue derived from three normal controls (denoted “Nor” in the figure), six osteoarthritis subjects (denoted “OA” in the figure), and six rheumatoid arthritis subjects, and analyzed for HGBPBMY4 (FLJ10961) mRNA by Real Time PCR. \\n         FIG. 43  shows an expanded expression profile of the GBP HGBPBMY1 (AK096141). The figure illustrates the relative expression level of HGBPBMY1 amongst various mRNA tissue sources. As shown, the HGBPBMY1 polypeptide exhibited high expression in normal spleen; tonsil; blood mononuclear cells; liver parenchyma; lung parenchyma; digestive system including stomach, duodenum, jejunum, ileum; placenta; ovary; and was detectable in regions of the brain, adrenal gland, ureter, bladder.  FIG. 43  also illustrates the relative expression level of HGBPBMY1 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY1 polypeptide showed increased expression in breast and testicle tumors relative to controls; and high expression in normal and diseased lung parenchyma. Expression data was obtained by measuring the steady state HGBPBMY1 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:61 and 62, and a TAQMAN probe (SEQ ID NO:63) as described in Example 36 herein. \\n         FIG. 44  shows an expanded expression profile of the GBP HGBPBMY2 (4843 — 1). The figure illustrates the relative expression level of HGBPBMY2 amongst various mRNA tissue sources. As shown, the HGBPBMY2 polypeptide showed the highest expression in tonsil, foreskin, esophagus; detectable in uterus cervix, tertiary lung bronchus, trachea.  FIG. 44  also illustrates the relative expression level of HGBPBMY2 amongst various mRNA tissue sources isolated from normal and diseased tissues. Expression data was obtained by measuring the steady state HGBPBMY2 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:64 and 65, and TAQMAN probe (SEQ ID NO:66) as described in Example 37 herein. \\n         FIG. 45  shows an expanded expression profile of the GBP HGBPBMY3 (4843 — 2). The figure illustrates the relative expression level of HGBPBMY3 amongst various mRNA tissue sources.  FIG. 45  illustrates the relative expression level of HGBPBMY3 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY3 polypeptide showed increased expression in thyroids of hyperthyroidism patients compared to control thyroids.  FIG. 45  also illustrates increased expression in breast and testicle tumors relative to controls; increased expression in prostatic hypertrophy relative to normal prostate; expressed in normal and diseased lung parenchyma; increased in bronchitis. Expression data was obtained by measuring the steady state HGBPBMY3 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:38 and 39, and TAQMAN probe (SEQ ID NO:40) as described in Example 38 herein. \\n         FIG. 46  shows an expanded expression profile of the GBP HGBPBMY4 (FLJ10961). The figure illustrates the relative expression level of HGBPBMY4 amongst various mRNA tissue sources.  FIG. 46  also illustrates the relative expression level of HGBPBMY4 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY4 polypeptide showed increased expression in spinal cord from multiple sclerosis patients compared to controls; increased expression in putamen and caudate from Parkinson&#39;s patients compared to controls; high expression in normal and diseased thryroid. Increased expression in breast and testicle tumors relative to controls. Expression data was obtained by measuring the steady state HGBPBMY4 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:70 and 71, and TAQMAN probe (SEQ ID NO:72) as described in Example 39 herein. \\n         FIG. 47  shows an expanded expression profile of human GBP1 (GenBank Ref. No. NM — 002053). The figure illustrates the relative expression level of GBP1 amongst various mRNA tissue sources.  FIG. 47  also illustrates the relative expression level of GBP1 amongst various mRNA tissue sources isolated from normal and dieased tissues. As shown, the GBP1 polypeptide showed increased expression in spinal cord and brain cortex from multiple sclerosis patients relative to controls; increased expression in putamen from Parkinson&#39;s patients relative to controls; increased expression in hippocampus from Alzheimer&#39;s patients relative to controls. Expression data was obtained by measuring the steady state GBP1 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:73 and 74, and TAQMAN probe (SEQ ID NO:75) as described in Example 40 herein. \\n         FIG. 48  shows an expanded expression profile of human GBP5 (GenBank Ref. No. NM — 052942). The figure illustrates the relative expression level of GBP5 amongst various mRNA tissue sources.  FIG. 48  also illustrates the relative expression level of GBP5 amongst various mRNA tissue sources isolated from normal and tumor tissues. As shown, the GBP5 polypeptide showed increased expression in cortex and spinal cord of multiple sclerosis patients relative to controls; increased expression in hippocampus form Alzheimer&#39;s patients relative to controls. Expression data was obtained by measuring the steady state GBP5 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:76 and 77, and TAQMAN probe (SEQ ID NO:78) as described in Example 41 herein. \\n         FIG. 49  is a a western blot depicting guanine nucleotide binding by the GBP family members described herein. \\n         FIG. 50A  is a western blot depicting the knockdown in GBP-1 and GBP-2 expression observed when GBP-1 and GBP-2 were coexpressed with siRNAs targeting these genes (siRNA 1-3 corresponds to GBP-1 and siRNA 582 corresponds to GBP-2). \\n         FIG. 50B  is a bar graph depicting TNFα-induced MCP-1 expression in HUVECs transfected with siRNAs 1-3 (targeted to GBP-1) and 582 (targeted to GBP-2). \\n     \\n    \\n    \\n     DETAILED DESCRIPTION OF THE INVENTION \\n     The similar expression and induction patterns of the known GBP family members offered the possibility that the genes might share common regulatory elements. It was first determined whether the genes were localized to a similar region of the chromosome. In this process, four new members of the GBP family were identified in humans, and four new members were identified in the mouse. It was previously demonstrated that GBP-1 and GBP-5 expression is elevated in the synovium derived from rheumatoid arthritis subjects as compared to osteoarthritis subjects or normal control joints (U.S. patent application Ser. No. 10/308,279, incorporated herein by reference). It was also determined that expression of both GBP-1 and GBP-5 can be regulated by NF-κB. Aberrant activation of NF-κB has also been associated with rheumatoid arthritis (Marok et al., (1996)  Arthritis Rheum.  39:583–591). These data suggest that members of the GBP family might be involved in the development of autoimmune diseases such as rheumatoid arthritis. They therefore represent candidate therapeutic targets and biomarkers for these diseases and other diseases associated with aberrant NF-kB activity. Thus, the present invention provides novel human sequences that encode guanylate binding proteins. Such proteins have been implicated in a number of diseases and/or disorders, which are known in the art or described herein. \\n     Definitions \\n     Following long-standing patent law convention, the terms “a” and “an” mean “one or more” when used in this application, including the claims. \\n     As used herein, the term “about,” when referring to a value or to an amount of mass, weight, time, volume, concentration or percentage is meant to encompass variations of ±20% or less (e.g., ±15%, ±10%, ±7%, ±5%, ±4%, ±3%, ±2%, ±1%, or ±0.1%) from the specified amount, as such variations are appropriate. \\n     As used herein, the terms “amino acid” and “amino acid residue” are used interchangeably and mean any of the twenty naturally occurring amino acids. An amino acid is formed upon chemical digestion (hydrolysis) of a polypeptide at its peptide linkages. The amino acid residues described herein are preferably in the “L” isomeric form. However, residues in the “D” isomeric form can be substituted for any L-amino acid residue, as long as the desired functional property is retained by the polypeptide. NH 2  refers to the free amino group present at the amino terminus of a polypeptide. COOH refers to the free carboxy group present at the carboxy terminus of a polypeptide. In keeping with standard polypeptide nomenclature abbreviations for amino acid residues are shown in tabular form presented hereinabove. \\n     It is noted that all amino acid residue sequences represented herein by formulae have a left-to-right orientation in the conventional direction of amino terminus to carboxy terminus. In addition, the phrases “amino acid” and “amino acid residue” are broadly defined to include modified and unusual amino acids. \\n     Furthermore, it is noted that a dash at the beginning or end of an amino acid residue sequence indicates a peptide bond to a further sequence of one or more amino acid residues, or a covalent bond to an amino-terminal group, such as NH 2 , to an acetyl group or to a carboxy-terminal group, such as COOH. \\n     As used herein, the term “antibody” refers to immunoglobulin molecules and immunologically active portions of immunoglobulin molecules, i.e., molecules that contain an antigen binding site that immunospecifically binds an antigen. Immunoglobulin molecules of the present invention can be of any type (e.g., IgG, IgE, IgM, IgD, IgA and IgY), class (e.g., IgG1, IgG2, IgG3, IgG4, IgA1 and IgA2) or subclass of immunoglobulin molecule. Moreover, the term “antibody” (Ab) or “monoclonal antibody” (Mab) includes intact molecules, as well as antibody fragments (such as, for example, Fab and F(ab′) 2  fragments) that are capable of specifically binding to protein. Fab and F(ab′) 2  fragments lack the Fc fragment of intact antibody, clear more rapidly from the circulation of the animal or plant, and can have less non-specific tissue binding than an intact antibody (Wahl et al., (1983)  J. Nucl. Med.  24:316–325). Thus, these fragments are preferred, as well as the products of a FAB or other immunoglobulin expression library. Moreover, antibodies of the present invention include chimeric, single chain, and humanized antibodies. \\n     Most preferably the antibodies are human antigen-binding antibody fragments of the present invention and include, but are not limited to, Fab, Fab′ and F(ab′) 2 , Fd, single-chain Fvs (scFv), single-chain antibodies, disulfide-linked Fvs (sdFv) and fragments comprising either a VL or VH domain. Antigen-binding antibody fragments, including single-chain antibodies, can comprise the variable region(s) alone or in combination with the entirety or a portion of the following: hinge region, CH1, CH2, and CH3 domains. Also included in the present invention are antigen-binding fragments also comprising any combination of variable region(s) with a hinge region, CH1, CH2, and CH3 domains. The antibodies of the present invention can be from any animal origin including birds and mammals. Antibodies of the present invention can be, for example, human, murine (e.g., mouse and rat), donkey, sheep, rabbit, goat, guinea pig, camel, horse, or chicken. As used herein, “human” antibodies include antibodies having the amino acid sequence of a human immunoglobulin and include antibodies isolated from human immunoglobulin libraries or from transgenic animals for one or more human immunoglobulin and that do not express endogenous immunoglobulins, as described herein and, for example in, U.S. Pat. No. 5,939,598. \\n     As used herein, the term “biological activity” means any observable effect flowing from a guanylate binding protein. Representative, but non-limiting, examples of biological activity in the context of the present invention includes guanylate binding ability and GTPase activity. \\n     As used herein, the terms “cells,” “host cells” or “recombinant host cells” are used interchangeably and mean not only to the particular subject cell, but also to the progeny or potential progeny of such a cell. Because certain modifications can occur in succeeding generations due to either mutation or environmental influences, such progeny might not, in fact, be identical to the parent cell, but are still included within the scope of the term as used herein. \\n     As used herein, the terms “chimeric protein” and “fusion protein” are used interchangeably and mean a fusion of a first amino acid sequence encoding a GBP polypeptide with a second amino acid sequence defining a polypeptide domain foreign to, and not homologous with, a GBP polypeptide. A chimeric protein can present a foreign domain that is found in an organism that also expresses the first protein, or it can be an “interspecies” or “intergenic” fusion of protein structures expressed by different kinds of organisms. In general, a fusion protein can be represented by the general formula X-GBP-Y, wherein GBP represents a portion of the protein which is derived from a GBP polypeptide, and X and Y are independently absent or represent amino acid sequences which are not related to a GBP sequence in an organism, which includes naturally occurring mutants. Analogously, the term “chimeric gene” refers to a nucleic acid construct that encodes a “chimeric protein” or “fusion protein” as defined herein. \\n     As used herein the term “complementary” means a nucleic acid sequence that is capable of base-pairing according to the standard Watson-Crick complementarity rules. These rules generally hold that the larger purines will always base pair with the smaller pyrimidines to form only combinations of Guanine paired with Cytosine (G:C) and Adenine paired with either Thymine (A:T) in the case of DNA, or Adenine paired with Uracil (A:U) in the case of RNA. \\n     As used herein, the term “DNA segment” means a DNA molecule that has been isolated free of total genomic DNA of a particular species. In one embodiment, a DNA segment encoding a guanylate binding protein refers to a DNA segment that comprises a sequence disclosed herein (e.g., a sequence selected from the group consisting of SEQ ID NOs: 2, 4, 6, 8, 10, 12, 14, and 16), but can optionally comprise fewer or additional nucleic acids, yet is isolated away from, or purified free from, total genomic DNA of a source species. Included within the scope of the term “DNA segment” are DNA segments and smaller fragments of such segments, as well as recombinant vectors, including, for example, plasmids, cosmids, phages, viruses, and the like, and primers and probes, such as those represented in SEQ ID NOs:1–59. \\n     The term “epitope” as used herein, refers to a portion of a polypeptide having antigenic or immunogenic activity in an animal, preferably a mammal, and most preferably in a human. In a representative embodiment, the present invention encompasses a polypeptide comprising an epitope, as well as the polynucleotide encoding the polypeptide. An “immunogenic epitope” as used herein, is defined as a portion of a protein that elicits an antibody response in an animal, as determined by any method known in the art, for example, by the methods for generating antibodies described herein (see, for example, Geysen et al., (1983)  Proc. Natl. Acad. Sci. U.S.A.  81:3998–4002). The term “antigenic epitope” as used herein, is defined as a portion of a protein to which an antibody can immunospecifically bind its antigen as determined by any method well known in the art, for example, by the immunoassays described herein. Immunospecific binding excludes non-specific binding but does not necessarily exclude cross-reactivity with other antigens. Antigenic epitopes need not necessarily be immunogenic. \\n     As used herein, the term “expression” generally refers to the cellular processes by which a polypeptide is produced from RNA. \\n     As used herein, the term “GBP” means nucleic acids encoding a functional guanylate binding protein. The term “GBP” includes homologs. “GBP” further includes vertebrate homologs of GBP family members. \\n     As used herein, the terms “GBP gene” and “recombinant GBP gene” mean a nucleic acid molecule comprising an open reading frame encoding a GBP polypeptide of the present invention, including both exon and (optionally) intron sequences. \\n     As used herein, the terms “GBP gene product”, “GBP protein”, “GBP polypeptide”, and “GBP peptide” are used interchangeably and mean peptides having amino acid sequences that are substantially identical to native amino acid sequences from an organism of interest and which are biologically active in that they comprise all or a part of the amino acid sequence of a GBP polypeptide, or cross-react with antibodies raised against a GBP polypeptide, or retain all or some of the biological activity (e.g., guanylate binding ability and GTPase activity) of the native amino acid sequence or protein. Such biological activity can include immunogenicity. \\n     As used herein, the terms “GBP gene product”, “GBP protein”, “GBP polypeptide”, and “GBP peptide” also include analogs of a GBP polypeptide. By “analog” is intended that a DNA or amino acid sequence can contain alterations relative to the sequences disclosed herein, yet retain all or some of the biological activity of those sequences. Analogs can be derived from genomic nucleotide sequences as are disclosed herein or from other organisms, or can be created synthetically. Those of ordinary skill in the art will appreciate that other analogs as yet undisclosed or undiscovered can be used to design and/or construct GBP analogs. There is no need for a “GBP gene product”, “GBP protein”, “GBP polypeptide”, or “GBP peptide” to comprise all or substantially all of the amino acid sequence of a GBP polypeptide gene product. Shorter or longer sequences are anticipated to be of use in the present invention; shorter sequences are herein referred to as “segments”. Thus, the terms “GBP gene product”, “GBP protein”, “GBP polypeptide”, and “GBP peptide” also include fusion, chimeric or recombinant GBP polypeptides and proteins comprising sequences of the present invention. Methods of preparing such proteins are disclosed herein and/or are known in the art. \\n     As used herein, the term “gene” refers broadly to any segment of DNA associated with a biological function. A gene encompasses sequences including but not limited to a coding sequence, a promoter region, a cis-regulatory sequence, a non-expressed DNA segment that is a specific recognition sequence for regulatory proteins, a non-expressed DNA segment that contributes to gene expression, a DNA segment designed to have desired parameters, or combinations thereof. A gene can be obtained by a variety of methods, including cloning from a biological sample, synthesis based on known or predicted sequence information, and recombinant derivation of an existing sequence. \\n     As used herein, the term “hybridization” and grammatical derivations thereof means the binding of a molecule (e.g., a probe molecule, such as a molecule to which a detectable moiety has been bound), to a target sample (e.g., a target nucleic acid). The terms “hybridization” and “binding” are used interchangeably in the context of probes and denatured DNA. Probes that are hybridized or bound to denatured DNA are aggregated to complementary sequences in the polynucleotide. Whether or not a particular probe remains aggregated with the polynucleotide depends on the degree of complementarity, the length of the probe, and the stringency of the binding conditions. The higher the stringency, the higher must be the degree of complementarity and/or the longer the probe. \\n     As used herein, the term “hybridization techniques” refers to molecular biological techniques that involve the binding or hybridization of a probe to complementary sequences in a polynucleotide. Included among these techniques are northern blot analysis, Southern blot analysis, nuclease protection assay, etc. \\n     As used herein, the terms “isolated” and “purified” are used interchangeably and refers to material (e.g., a nucleic acid or a protein) removed from its original environment (e.g., the natural environment, if it is naturally occurring), and thus is altered “by the hand of man” from its natural state. For example, an isolated polynucleotide could be part of a vector or a composition of matter, or could be contained within a cell, and still be “isolated” because that vector, composition of matter, or particular cell is not the original environment of the polynucleotide. The term “isolated” does not refer to genomic or cDNA libraries, whole cell total or mRNA preparations, genomic DNA preparations (including those separated by electrophoresis and transferred onto blots), sheared whole cell genomic DNA preparations or other compositions where the art demonstrates no distinguishing features of the polynucleotide and/or protein sequences of the present invention. \\n     As used herein, the term “modified” means an alteration from an entity&#39;s normally occurring state. An entity can be modified by removing discrete chemical units or by adding discrete chemical units. The term “modified” encompasses detectable labels as well as those entities added as aids in purification. \\n     As used herein the terms “modulate” and grammatical derivations thereof refer to an increase or decrease in the amount, quality or effect of a particular activity, DNA, RNA, or protein. The definition of “modulate” as used herein is meant to encompass agonists and/or antagonists of a particular activity, DNA, RNA, or protein. \\n     Thus, as used herein, the term “modulate”, and grammatical derivations thereof, means an increase, decrease, or other alteration of any and/or all chemical and biological activities or properties mediated by a nucleic acid sequence or a peptide. The term “modulation” as used herein refers to both upregulation (i.e., activation or stimulation) and downregulation (i.e. inhibition or suppression) of a response by any mode of action. \\n     As used herein, the terms “nucleotide”, “base” and “nucleic acid” are used interchangeably and are equivalent. Additionally, the terms “nucleotide sequence”, “nucleic acid sequence”, “nucleic acid molecule” and “segment” are used interchangeably and are equivalent. The terms “nucleotide”, “base”, “nucleic acid”, “nucleotide sequence”, “nucleic acid sequence”, “nucleic acid molecule” and “segment” mean any of deoxyribonucleic acid (DNA), ribonucleic acid (RNA), oligonucleotides, fragments generated by the polymerase chain reaction (PCR), and fragments generated by any of ligation, scission, endonuclease action, and exonuclease action. Nucleic acids can be composed of monomers that are naturally-occurring nucleotides (such as deoxyribonucleotides and ribonucleotides), or analogs of naturally-occurring nucleotides (e.g., α-enantiomeric forms of naturally-occurring nucleotides), or a combination of both. Modified nucleotides can have modifications in sugar moieties and/or in pyrimidine or purine base moieties. Sugar modifications include, for example, replacement of one or more hydroxyl groups with halogens, allkyl groups, amines, and azido groups, or sugars can be functionalized as ethers or esters. Moreover, the entire sugar moiety can be replaced with sterically and electronically similar structures, such as aza-sugars and carbocyclic sugar analogs. Examples of modifications in a base moiety include alkylated purines and pyrimidines, acylated purines or pyrimidines, or other well-known heterocylcic substitutes. Nucleic acid monomers can be linked by phosphodiester bonds or analogs of such linkages. Analogs of phosphodiester linkages include phosphorothioate, phosphorodithioate, phosphoroselenoate, phosphorodiselenoate, phosphoroanilothioate, phosphoranilidate, phosphoramidate, and the like. The term “nucleic acid” also includes so-called “peptide nucleic acids,” which comprise naturally-occurring or modified nucleic acid bases attached to a polyamide backbone. Nucleic acids can be either single stranded or double stranded. \\n     As used herein, the terms “oligonucleotide” and “polynucleotide” are used interchangeably and mean a single- or double-stranded DNA or RNA sequence. Typically, an oligonucleotide is a short segment of about 50 or less nucleotides. An oligonucleotide or a polynucleotide can be naturally occurring or synthetic, but oligonucleotides are typically prepared by synthetic means. In the context of the present invention, an “oligonucleotide” and/or a “polynucleotide” includes segments of DNA, and/or their complements. The segments can be, for example, between 1 and 250 bases, and, in some embodiments, between 5–10, 5–20, 10–20, 10–50, 20–50, 10–100 bases, or 100 or more bases in length. \\n     The terms “oligonucleotide” and “polynucleotide” refer to a molecule comprising two or more nucleotides. For example, a polynucleotide can comprise a nucleotide sequence of a full length cDNA sequence, including the 5′ and 3′ untranslated sequences, the coding region, with or without a signal sequence, the secreted protein coding region, as well as fragments, epitopes, domains, and variants of the nucleic acid sequence. Moreover, as used herein, a “polypeptide”, defined further herein, refers to a molecule having the translated amino acid sequence generated directly or indirectly from a polynucleotide. A “polynucleotide” of the present invention also includes those polynucleotides capable of hybridizing, under stringent hybridization conditions (examples of which are provided herein), to sequences described herein, or the complement thereof. \\n     Thus, an oligonucleotide or a polynucleotide of the present invention can comprise any polyribonucleotide or polydeoxribonucleotide, and can comprise unmodified RNA or DNA or modified RNA or DNA. For example, a polynucleotide can comprise single- and double-stranded DNA, DNA that is a mixture of single- and double-stranded regions, single- and double-stranded RNA, RNA that is mixture of single- and double-stranded regions, hybrid molecules comprising DNA and RNA that can be single-stranded or, more typically, double-stranded or a mixture of single- and double-stranded regions. In addition, a polynucleotide can comprise triple-stranded regions comprising RNA or DNA or both RNA and DNA. A polynucleotide can also contain one or more modified bases or DNA or RNA backbones modified for stability or for other reasons. “Modified” bases include, for example, tritylated bases and unusual bases such as inosine. A variety of modifications can be made to DNA and RNA; thus, “polynucleotide” embraces chemically, enzymatically, or metabolically modified forms. \\n     By employing the disclosure presented herein, a nucleic acid molecule of the present invention encoding a polypeptide of the present invention can be obtained using standard cloning and screening procedures, such as those for cloning cDNAs using mRNA as starting material. \\n     Thus, the term “polynucleotide” encompasses a molecule having a nucleic acid sequence contained in SEQ ID NO:1, SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14 and/or SEQ ID NO:16, or the cDNA contained within a clone deposited with the ATCC. For example, the polynucleotide can contain the nucleotide sequence of the full length cDNA sequence, including the 5′ and 3′ untranslated sequences, the coding region, with or without a signal sequence, the secreted protein coding region, as well as fragments, epitopes, domains, and variants of the nucleic acid sequence. Moreover, as used herein, a “polypeptide” broadly refers to a molecule having a translated amino acid sequence generated from a polynucleotide. \\n     As used herein, the terms “organism”, “subject” and “subject” are used interchangeably and mean any organism referenced herein, including prokaryotes, though the terms preferably refer to eukaryotic organisms, more preferably to mammals, and most preferably to humans. \\n     As used herein, the terms “protein”, “polypeptide” and “peptide” are used interchangeably and mean any polymer comprising any of the 20 protein amino acids, regardless of its size. Although “protein” is often used in reference to relatively large polypeptides, and “peptide” is often used in reference to small polypeptides, usage of these terms in the art overlaps and varies. The term “polypeptide” as used herein refers to peptides, polypeptides and proteins, unless otherwise noted. The terms “protein”, “polypeptide” and “peptide” are used interchangeably herein. \\n     Thus, a polypeptide of the present invention can comprise amino acids joined to each other by peptide bonds or modified peptide bonds, i.e., peptide isosteres, and can contain amino acids other than the 20 gene-encoded amino acids. A polypeptide can be modified by either natural processes, such as by posttranslational processing, or by chemical modification techniques which are well known in the art. Such modifications are described in basic texts and in more detailed monographs, as well as in research literature known to those of ordinary skill in the art. Modifications can occur anywhere in a polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. The same type of modification can be present in the same or varying degrees at several sites in a given polypeptide. Also, a given polypeptide can contain many types of modifications. A polypeptide can be branched, for example, as a result of ubiquitination, or a polypeptide can be cyclic, with or without branching. Cyclic, branched, and branched cyclic polypeptides can result from posttranslation natural processes or can be made by synthetic methods. Representative modifications include acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of phosphotidylinositol, cross-linking, cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodination, methylation, myristoylation, oxidation, pegylation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, transfer-RNA mediated addition of amino acids to proteins such as arginylation, and ubiquitination (see, e.g., Creighton,  Proteins—Structure And Molecular Properties,  2nd ed., W. H. Freeman and Company, New York, N.Y., USA (1993);  Posttranslational Covalent Modification Of Proteins , (Johnson, ed.), Academic Press, New York, N.Y., USA, pp. 1–12 (1983); Seifter et al., (1990)  Method Enzymol.  182:626–646; Rattan et al., (1992)  Ann. N.Y. Acad. Sci.  663:48–62). \\n     As used herein, “a polypeptide having biological activity” refers to a polypeptide exhibiting activity similar, but not necessarily identical to, an activity of a polypeptide of the present invention, including mature forms, as measured in a particular biological assay, with or without dose dependency. In a case where dose dependency does exist, it need not be identical to that of the polypeptide, but rather substantially similar to the dose-dependence in a given activity as compared to a polypeptide of the present invention (i.e., a candidate polypeptide will exhibit greater activity of not more than about 25-fold less and, preferably, not more than about ten-fold less activity, and most preferably, not more than about three-fold less activity relative to a polypeptide of the present invention.) \\n     As used herein, the term “primer” means a single-stranded oligonucleotide sequence that acts as a point of initiation for template-directed DNA synthesis under appropriate conditions (e.g., in the presence of four different nucleoside triphosphates and an agent for polymerization, such as DNA or RNA polymerase or reverse transcriptase) in a suitable buffer and at a suitable temperature. The appropriate length of a primer can depend on the intended use of the primer, but typically ranges from to nucleotides. Short primer molecules generally require cooler temperatures to form sufficiently stable hybrid complexes with the template. A primer need not reflect the exact sequence of the template, but is preferably sufficiently complementary to hybridize with a template. The term primer site refers to the area of the target DNA to which a primer hybridizes. The term “primer pair” refers to a set of primers comprising a 5′ (upstream) primer that hybridizes with the 5′ end of the DNA sequence to be amplified and a 3′ (downstream) primer that hybridizes with the complement of the 3′ end of the sequence to be amplified. A primer can comprise, for example, two or more deoxyribonucleotides or ribonucleotides, more than three deoxyribonucleotides or ribonucleotides, more than eight deoxyribonucleotides or ribonucleotides or at least about 20 deoxyribonucleotides or ribonucleotides of an exonic or intronic region. Such oligonucleotides can be, for example, between ten and thirty bases in length. \\n     As used herein, the term “probe” refers to an oligonucleotide or short fragment of DNA designed, known or suspected to be sufficiently complementary to a sequence in a denatured nucleic acid to be probed and to be bound under selected stringency conditions. \\n     Continuing, in one embodiment a probe is a hybridization probe; such a probe can be an oligonucleotide that binds, in a base-specific manner, to a complementary strand of nucleic acid. Such probes include peptide nucleic acids, such as described for example in Nielsen et al., (1991)  Science  254:1497–1500. A probe can be of any length suitable for specific hybridization to the target nucleic acid sequence. The most appropriate length of the probe can vary, depending upon the hybridization method in which it is being used; for example, particular lengths might be more appropriate for use in microfabricated arrays, while other lengths might be more suitable for use in classical hybridization methods. Such optimizations will be known to the skilled artisan upon consideration of the present disclosure. Representative probes and primers can range from about 5 nucleotides to about 40 nucleotides in length. For example, probes and primers can be 5, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 25, 26, or 40 nucleotides in length. In some embodiments, the probe or primer overlaps at least one polymorphic site occupied by any of the possible variant nucleotides. The nucleotide sequence can correspond to the coding sequence of the allele or to the complement of the coding sequence of the allele. \\n     As used herein, the term “recombination” and grammatical derivations thereof, means a re-assortment of genes or characters in combinations different from what they were in the parents, in the case of linked genes by crossing over. \\n     As used herein, the term “related,” when used in connection with a particular amino acid sequence means a sequence comprising the described sequence, as well as fragments thereof, functional equivalents. Thus, for example, the term “MGBPBMY4 (BC007143)-related polypeptide” encompasses a polypeptide sequence comprising MGBPBMY4 (BC007143), fragments of MGBPBMY4 (BC007143) (examples of which are provided herein) and polypeptides that are functionally equivalent to a MGBPBMY4 (BC007143) polypeptide. \\n     As used herein, the term “sequencing” means determining the ordered linear sequence of nucleic acids or amino acids of a DNA, RNA or protein target sample, using manual or automated laboratory techniques. Unless otherwise indicated, the nucleotide sequence of all DNA sequences disclosed herein can be determined by employing an automated DNA sequencer (such as the Model 373 available from Applied Biosystems, Inc., Foster City, Calif., USA). All amino acid sequences of polypeptides encoded by DNA molecules disclosed herein can be predicted by translating a DNA sequence, or alternatively, by employing a known protein technique (e.g., Edman degradation). \\n     As used herein, the term “stringent hybridization conditions”, in the context of nucleic acid hybridization experiments such as Southern and northern blot analysis, means a set of conditions under which single stranded nucleic acid sequences are unlikely to hybridize to one another unless there is substantial complementarity between the sequences. Stringent hybridization conditions can be both sequence- and environment-dependent. Longer sequences hybridize specifically at higher temperatures. An extensive guide to the hybridization of nucleic acids is found in Tijssen,  Laboratory Techniques in Biochemistry and Molecular Biology - Hybridization with Nucleic Acid Probes , Elsevier, New York, N.Y., USA, (1993), part I, chapter 2. Generally, highly stringent hybridization and wash conditions are selected to be about 5° C. lower than the thermal melting point (T m ) for the specific sequence at a defined ionic strength and pH. Typically, under “stringent conditions” a probe will hybridize specifically to its target subsequence, but to no other sequences. \\n     Continuing, the T m  is the temperature (under defined ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectly matched probe. Very stringent conditions are selected to be equal to the T m  for a particular probe. An example of stringent hybridization conditions for Southern or northern blot analysis of complementary nucleic acids having more than about 100 complementary residues is overnight hybridization in 50% formamide with 1 mg of heparin at 42° C. An example of highly stringent wash conditions is 15 minutes in 0.15 M NaCl at 65° C. An example of stringent wash conditions is 15 minutes in 0.2×SSC buffer at 65° C. (see, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3 rd  ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001) for a description of SSC buffer). Often, a high stringency wash is preceded by a low stringency wash to remove background probe signal. An example of medium stringency wash conditions for a duplex of more than about 100 nucleotides, is 15 minutes in 1×SSC at 45° C. An example of low stringency wash for a duplex of more than about 100 nucleotides, is 15 minutes in 4–6×SSC at 40° C. For short probes (e.g., about 10 to about 50 nucleotides), stringent conditions typically involve salt concentrations of less than about 1.0 M Na +  ion, typically about 0.01 to 1.0 M Na +  ion concentration (or other salts) at pH 7.0–8.3, and the temperature is typically at least about 30° C. Stringent conditions can also be achieved with the addition of destabilizing agents such as formamide. In general, a signal to noise ratio of 2-fold (or higher) than that observed for an unrelated probe in a particular hybridization assay indicates the presence of a specific hybridization. \\n     The following are representative, but non-limiting, examples of stringent hybridization and wash conditions that can be employed in a hybridization-based operation (e.g., to clone homologous nucleotide sequences that are substantially identical to reference nucleotide sequences of the present invention): in one example, a probe nucleotide sequence hybridizes to a target nucleotide sequence in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 2×SSC, 0.1% SDS at 50° C.; in another example, a probe and target sequence hybridize in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 1×SSC, 0.1% SDS at 50° C.; in yet another example, a probe and target sequence hybridize in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 0.5×SSC, 0.1% SDS at 50° C.; in a further example, a probe and target sequence hybridize in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 0.1×SSC, 0.1% SDS at 50° C.; in yet a further example, a probe and target sequence hybridize in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPO 4 , 1 mM EDTA at 50° C. followed by washing in 0.1×SSC, 0.1% SDS at 65° C. \\n     In yet another example of stringent conditions, hybridization is carried out with 6×SSC, 0.2% polyvinylpyrrolidone, 0.2% Ficoll, 0.2% bovine serum albumin, 0.1% sodium dodecyl sulfate, 100 μg/ml salmon sperm DNA and 15% formamide at 68° C. For the purposes of specifying additional conditions of high stringency, conditions can comprise, for example, a salt concentration of about 200 mM and temperature of about 45° C. One example of such stringent conditions is hybridization at 4×SSC, at 65° C., followed by washing in 0.1×SSC at 65° C. for one hour. Another example stringent hybridization scheme uses 50% formamide, 4×SSC at 42° C. The above are only exemplary and additional high stringency conditions will be known to those of ordinary skill in the art upon consideration of the present disclosure. \\n     As used herein, the term “substantially pure” means that a polynucleotide or polypeptide of interest is substantially free of the sequences and molecules with which it is associated in its natural state, as well as those molecules used in a given isolation or synthesis procedure. The term “substantially free” means that the sample is at least 50%, preferably at least 70%, more preferably 80% and most preferably 90% free of the materials and compounds with which is it associated in nature or in a medium in which the polynucleotide or polypeptide of interest is synthesized. \\n     As used herein, the term “toxin” means a compound that binds and activates endogenous cytotoxic effector systems, radioisotopes, holotoxins, modified toxins, catalytic subunits of toxins, or any molecules or enzymes not normally present in or on the surface of a cell that under defined conditions cause the cell&#39;s death. Toxins that can be used according to the methods of the present invention include, but are not limited to, radioisotopes known in the art, compounds such as, for example, antibodies (or complement fixing containing portions thereof) that bind an inherent or induced endogenous cytotoxic effector system, thymidine kinase, endonuclease, RNAse, alpha toxin, ricin, abrin,  Pseudomonas  exotoxin A, diphtheria toxin, saporin, momordin, gelonin, pokeweed antiviral protein, alpha-sarcin and cholera toxin. \\n     As used herein, the term “transcription” means a cellular process involving the interaction of an RNA polymerase with a gene that directs the expression as RNA of the structural information present in the coding sequences of the gene. The process comprises, but is not limited to, the following steps: (a) the transcription initiation, (b) transcript elongation, (c) transcript splicing, (d) transcript capping, (e) transcript termination, (f) transcript polyadenylation, (g) nuclear export of the transcript, (h) transcript editing, and (i) stabilizing the transcript. \\n     As used herein, the term “vector” means a DNA molecule having sequences that enable its replication in a compatible host cell. A vector also includes nucleotide sequences to permit ligation of nucleotide sequences within the vector, wherein such nucleotide sequences are also replicated in a compatible host cell. A vector can also mediate recombinant production of an SREBP1 polypeptide, as described further herein. Some representative vectors include, but are not limited to, pCMV (Invitrogen, Carlsbad, Calif., USA) pBluescript (Stratagene, La Jolla, Calif., USA), pUC18, pBLCAT3 (Luckow &amp; Schutz, (1987)  Nucleic Acids Res  15: 5490), pLNTK (Gorman et al., (1996)  Immunity  5: 241–252), and pBAD/gIII (Stratagene, La Jolla, Calif.). \\n     Nucleic Acids of the Present Invention \\n     In specific embodiments, the polynucleotides of the present invention are at least 15, at least 30, at least 50, at least 100, at least 125, at least 500, or at least 1000 continuous nucleotides but are less than or equal to 300 kb, 200 kb, 100 kb, 50 kb, 15 kb, 10 kb, 7.5 kb, 5 kb, 2.5 kb, 2.0 kb, or 1 kb, in length. In a further embodiment, polynucleotides of the present invention comprise a portion of the coding sequences, as disclosed herein, but do not comprise all or a portion of any intron. In another embodiment, the polynucleotides comprising coding sequences do not contain coding sequences of a genomic flanking gene (i.e., 5′ or 3′ to the gene of interest in the genome). In other embodiments, the polynucleotides of the present invention do not contain the coding sequence of more than 1000, 500, 250, 100, 50, 25, 20, 15, 10, 5, 4, 3, 2, or 1 genomic flanking gene(s). \\n     In the present invention, the full length sequences identified as SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14 and/or SEQ ID NO:16 were generated by BLAST searches of genomic DNA. \\n     Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer (such as the Model 373, or a Model 3700, from Applied Biosystems, Inc., Foster City, Calif., USA), and all amino acid sequences of polypeptides encoded by DNA molecules determined herein were predicted by translation of a DNA sequence determined above. Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein might contain an error. Nucleotide sequences determined by automation are typically at least about 90% identical, more typically at least about 95% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence will be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion. \\n     Using the information provided herein, such as the nucleotide sequence in  FIGS. 1A–1C  (SEQ ID NO:2),  FIGS. 2A–2C  (SEQ ID NO:4),  FIGS. 3A–3B  (SEQ ID NO:6),  FIGS. 4A–4C  (SEQ ID NO:8),  FIGS. 5A–5C  (SEQ ID NO:10),  FIGS. 6A–6B  (SEQ ID NO:12),  FIGS. 7A–7C  (SEQ ID NO:14),  FIGS. 8A–8D  (SEQ ID NO:16), a nucleic acid molecule of the present invention encoding the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (SEQ ID Nos: 3, 5, 7, 9, 11, 13, 15, and 17, respectively) can be obtained using standard cloning and screening procedures, such as those for cloning cDNAs using mRNA as starting material. Illustrative of the present invention, the nucleic acid molecules described in  FIGS. 1A–1C  (SEQ ID NO:2),  FIGS. 2A–2C  (SEQ ID NO:4),  FIGS. 3A–3B  (SEQ ID NO:6),  FIGS. 4A–4C  (SEQ ID NO:8),  FIGS. 5A–5C  (SEQ ID NO:10),  FIGS. 6A–6B  (SEQ ID NO:12),  FIGS. 7A–7C  (SEQ ID NO:14),  FIGS. 8A–8D  (SEQ ID NO:16), were discovered in Human Multiple Tissue and Human Immune System MTC cDNA panels and a mouse−1 cDNA panel (Clontech, Palo Alto, Calif., USA). \\n     The present invention encompasses the identification of proteins, nucleic acids, or other molecules, that bind to polypeptides and polynucleotides of the present invention (for example, in a receptor-ligand interaction). The polynucleotides of the present invention can also be used in interaction trap assays (such as, for example, that described by Ozenberger and Young, (1995)  Mol. Endocrinol.  9(10):1321–9; and Ozenberger and Young, (1995)  Ann. N.Y. Acad. Sci.  766:279–81). \\n     The polynucleotide and polypeptides of the present invention are useful as probes for the identification and isolation of full-length cDNAs and/or genomic DNA which correspond to the polynucleotides of the present invention, as probes to hybridize and discover novel, related DNA sequences, as probes for positional cloning of this or a related sequence, as probe to “subtract-out” known sequences in the process of discovering other novel polynucleotides, as probes to quantify gene expression, and as probes for microarrays. \\n     Also, in preferred embodiments the present invention provides methods for further refining the biological function of the polynucleotides and/or polypeptides of the present invention. \\n     Specifically, the present invention provides methods for using the polynucleotides and polypeptides of the present invention to identify orthologs, homologs, paralogs, variants, and/or allelic variants of the present invention. Also provided are methods of using the polynucleotides and polypeptides of the present invention to identify the entire coding region of the present invention, non-coding regions of the present invention, regulatory sequences of the present invention, and secreted, mature, pro-, prepro-, forms of the present invention (as applicable). \\n     In some embodiments, the present invention provides methods for identifying glycosylation sites inherent in the polynucleotides and polypeptides of the present invention, and the subsequent alteration, deletion, and/or addition of said sites for a number of desirable characteristics which include, but are not limited to, augmentation of protein folding, inhibition of protein aggregation, regulation of intracellular trafficking to organelles, increasing resistance to proteolysis, modulation of protein antigenicity, and mediation of intercellular adhesion. \\n     In further embodiments, methods are provided for evolving the polynucleotides and polypeptides of the present invention using molecular evolution techniques in an effort to create and identify novel variants with desired structural, functional, and/or physical characteristics. \\n     The present invention further provides for other experimental methods and procedures currently available to derive functional assignments. These procedures include but are not limited to spotting of clones on arrays, micro-array technology, PCR based methods (e.g., quantitative PCR), anti-sense methodology, gene knockout experiments, and other procedures that could use sequence information from clones to build a primer or a hybrid partner. \\n     Polynucleotides and Polypeptides of the Present Invention \\n     The following paragraphs provide additional details regarding the various polynucleotide and polypeptide sequences that form aspects of the present invention. \\n     Features of the Polypeptide Encoded by Gene No. 1 \\n     The polypeptide encoded by this gene, HGBPBMY1 (AK096141), is provided as SEQ ID NO:3 ( FIGS. 1A–1C ) and is encoded by the polynucleotide sequence according to SEQ ID NO:3 ( FIGS. 1A–1C ) and/or by a polynucleotide contained within the deposited clone. HGBPBMY1 (AK096141) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, human GBP-1, human GBP-2, human GBP-3 and human GBP-4. \\n     The determined nucleotide sequence of the HGBPBMY1 (AK096141), (i.e. the cDNA shown in  FIGS. 1A–1C  and in SEQ ID NO:2) comprises an open reading frame encoding a protein of about 638 amino acid residues. The predicted amino acid sequence of the HGBPBMY1 (AK096141) polypeptide is shown in  FIGS. 1A–1C  (SEQ ID NO:3). The percent identity and similarity values between the HGBPBMY1 (AK096141) polypeptide to the known GBP family member hGBP1 is provided in  FIG. 9 . The HGBPBMY1 (AK096141) protein shown in  FIGS. 1A–1C  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding a HGBPBMY1 (AK096141) polypeptide showed expression predominantly in the liver (See  FIG. 18 ). \\n     Based upon the strong homology to members of the GBP family members, a HGBPBMY1 (AK096141) polypeptide is expected to share at least some biological activity with GBP family members, specifically hGBP-1, hGBP-2, hGBP-3 and hGBP-4. \\n     The HGBPBMY1 (AK096141) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the liver. \\n     The HGBPBMY1 (AK096141) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue, preferably human tissue. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with the predominately localized HGBPBMY1 (AK096141) expression in liver tissue suggests a potential utility for HGBPBMY1 (AK096141) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing liver diseases. In representative embodiments, HGBPBMY1 (AK096141) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing liver diseases. For example, the HGBPBMY1 (AK096141) protein can be used for the detection, treatment, amelioration, and/or prevention of hepatoblastoma, jaundice, hepatitis, liver metabolic diseases and conditions that are attributable to the differentiation of hepatocyte progenitor cells, cirrhosis, hepatic cysts, pyrogenic abscess, amebic abcess, hydatid cyst, cystadenocarcinoma, adenoma, focal nodular hyperplasia, hemangioma, hepatocellulae carcinoma, cholangiocarcinoma, and angiosarcoma, granulomatous liver disease, liver transplantation, hyperbilirubinemia, jaundice, parenchymal liver disease, portal hypertension, hepatobiliary disease, hepatic parenchyma, hepatic fibrosis, anemia, gallstones, cholestasis, carbon tetrachloride toxicity, beryllium toxicity, vinyl chloride toxicity, choledocholithiasis, hepatocellular necrosis, aberrant metabolism of amino acids, aberrant metabolism of carbohydrates, aberrant synthesis proteins, aberrant synthesis of glycoproteins, aberrant degradation of proteins, aberrant degradation of glycoproteins, aberrant metabolism of drugs, aberrant metabolism of hormones, aberrant degradation of drugs, aberrant degradation of drugs, aberrant regulation of lipid metabolism, aberrant regulation of cholesterol metabolism, aberrant glycogenesis, aberrant glycogenolysis, aberrant glycolysis, aberrant gluconeogenesis, hyperglycemia, glucose intolerance, hyperglycemia, decreased hepatic glucose uptake, decreased hepatic glycogen synthesis, hepatic resistance to insulin, portal-systemic glucose shunting, peripheral insulin resistance, hormonal abnormalities, increased levels of systemic glucagon, decreased levels of systemic cortisol, increased levels of systemic insulin, hypoglycemia, decreased gluconeogenesis, decreased hepatic glycogen content, hepatic resistance to glucagon, elevated levels of systemic aromatic amino acids, decreased levels of systemic branched-chain amino acids, hepatic encephalopathy, aberrant hepatic amino acid transamination, aberrant hepatic amino acid oxidative deamination, aberrant ammonia synthesis, aberant albumin secretion, hypoalbuminemia, aberrant cytochromes b5 function, aberrant P450 function, aberrant glutathione S-acyltransferase function, aberrant cholesterol synthesis, and aberrant bile acid synthesis. \\n     Moreover, HGBPBMY1 (AK096141) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, hepatic infections: liver disease caused by sepsis infection, liver disease caused by bacteremia, liver disease caused by Pneomococcal pneumonia infection, liver disease caused by Toxic shock syndrome, liver disease caused by Listeriosis, liver disease caused by Legionnaries&#39; disease, liver disease caused by Brucellosis infection, liver disease caused by  Neisseria gonorrhoeae  infection, liver disease caused by  Yersinia  infection, liver disease caused by Salmonellosis, liver disease caused by Nocardiosis, liver disease caused by Spirochete infection, liver disease caused by  Treponema pallidum  infection, liver disease caused by  Brrelia burgdorferi  infection, liver disease caused by Leptospirosis, liver disease caused by  Coxiella burnetii  infection, liver disease caused by  Rickettsia richettsii  infection, liver disease caused by  Chlamydia trachomatis  infection, liver disease caused by  Chlamydia psittaci  infection, liver disease caused by hepatitis virus infection, liver disease caused by Epstein-Barr virus infection in addition to any other hepatic disease and/or disorder implicated by the causative agents listed above or elsewhere herein. \\n     The protein can also be used to determine biological activity, raise antibodies, as tissuemarkers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a HGBPBMY1 (AK096141) protein, can show utility as a tumor marker and/or immunotherapy targets for liver tissue. \\n     The HGBPBMY1 (AK096141) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of HGBPBMY1 (AK096141) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the HGBPBMY1 (AK096141) protein could be used as diagnostic agents of conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     HGBPBMY1 (AK096141) polypeptides and polynucleotides have additional uses which include diagnosing diseases related to the over and/or under expression of HGBPBMY1 (AK096141) by identifying mutations in a HGBPBMY1 (AK096141) gene by using HGBPBMY1 (AK096141) sequences as probes or by determining HGBPBMY1 (AK096141) protein or mRNA expression levels. HGBPBMY1 (AK096141) polypeptides can be useful for screening compounds that affect the activity of the protein. HGBPBMY1 (AK096141) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with HGBPBMY1 (AK096141), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly hGBP-1, hGBP-2, hGBP-3, hGBP-4 and/or hGBP-5), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to a HGBPBMY1 (AK096141) polynucleotide, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of HGBPBMY1 (AK096141), liver tissue can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of a HGBPBMY1 (AK096141) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of HGBPBMY1 (AK096141), a disease correlation related to HGBPBMY1 (AK096141) can be made by comparing the mRNA expression level of HGBPBMY1 (AK096141) in normal tissue, as compared to diseased tissue (particularly diseased liver tissue). Significantly higher or lower levels of HGBPBMY1 (AK096141) expression in the diseased tissue can suggest HGBPBMY1 (AK096141) plays a role in disease progression, and antagonists against HGBPBMY1 (AK096141) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of HGBPBMY1 (AK096141) expression in the diseased tissue can suggest HGBPBMY1 (AK096141) plays a defensive role against disease progression, and agonists of HGBPBMY1 (AK096141) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:2 ( FIGS. 1A–1C ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of a HGBPBMY1 (AK096141), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence a HGBPBMY1 (AK096141) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in mice and/or rats and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of a polypeptide can be determined by the application of antisense and/or sense methodology and the resulting generation of transgenic mice and/or rats or other animal. Expressing a particular gene in either sense or antisense orientation in a transgenic mouse or rat, for example, could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a liver tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of HGBPBMY1 (AK096141) transgenic mice or rats, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic mice or rats to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In some embodiments, the following N-terminal HGBPBMY1 (AK096141) deletion polypeptides are encompassed by the present invention: M1-S638, A2-S638, S3-S638, E4-S638, I5-S638, H6-S638, M7-S638, P8-S638, G9-S638, P10-S638, V11-S638, C12-S638, L13-S638, I14-S638, E15-S638, N16-S638, T17-S638, K18-S638, G19-S638, H20-S638, L21-S638, V22-S638, V23-S638, N24-S638, S25-S638, E26-S638, A27-S638, L28-S638, E29-S638, I30-S638, L31-S638, S32-S638, A33-S638, I34-S638, T35-S638, Q36-S638, P37-S638, V38-S638, V39-S638, V40-S638, V41-S638, A42-S638, I43-S638, V44-S638, G45-S638, L46-S638, Y47-S638, R48-S638, T49-S638, G50-S638, K51-S638, S52-S638, Y53-S638, L54-S638, M55-S638, N56-S638, K57-S638, L58-S638, A59-S638, G60-S638, K61-S638, N62-S22638, K63-S638, G64-S638, F65-S638, P66-S638, L67-S638, G68-S638, C69-S638, T70-S638, V71-S638, K72-S638, S73-S638, E74-S638, T75-S638, K76-S638, G77-S638, I78-S638, W79-S638, M80-S638, W81-S638, C82-S638, V83-S638, P84-S638, H85-S638, P86-S638, S87-S638, K88-S638, P89-S638, N90-S638, H91-S638, T92-S638, L93-S638, I94-S638, L95-S638, L96-S638, D97-S638, T98-S638, E99-S638, G100-S638, L101-S638, G102-S638, D103-S638, M104-S638, E105-S638, K106-S638, S107-S638, D108-S638, P109-S638, K110-S638, S111-S638, D112-S638, S113-S638, W114-S638, I115-S638, F116-S638, A117-S638, L118-S638, A119-S638, V120-S638, L121-S638, L122-S638, S123-S638, S124-S638, S125-S638, F126-S638, V127-S638, Y128-S638, N129-S638, S130-S638, M131-S638, G132-S638, T133-S638, I134-S638, N135-S638, H136-S638, Q137-S638, A138-S638, L139-S638, E140-S638, Q141-S638, L142-S638, H143-S638, Y144-S638, V145-S638, T146-S638, E147-S638, L148-S638, T149-S638, E150-S638, L151-S638, I152-S638, R153-S638, A154-S638, K155-S638, S156-S638, C157-S638, P158-S638, R159-S638, P160-S638, D161-S638, E162-S638, V163-S638, E164-S638, D165-S638, S166-S638, S167-S638, E168-S638, F169-S638, V170-S638, S171-S638, F172-S638, F173-S638, P174-S638, D175-S638, F176-S638, I177-S638, W178-S638, T179-S638, V180-S638, R181-S638, D182-S638, F183-S638, T184-S638, L185-S638, E186-S638, L187-S638, K188-S638, L189-S638, D190-S638, G191-S638, H192-S638, P193-S638, I194-S638, T195-S638, E196-S638, D197-S638, E198-S638, Y199-S638, L200-S638, E201-S638, N202-S638, A203-S638, L204-S638, K205-S638, L206-S638, I207-S638, S208-S638, G209-S638, K210-S638, N211-S638, P212-S638, Q213-S638, I214-S638, Q215-S638, N216-S638, S217-S638, N218-S638, K219-S638, P220-S638, R221-S638, E222-S638, W223-S638, I224-S638, R225-S638, H226-S638, F227-S638, F228-S638, P229-S638, K230-S638, Q231-S638, K232-S638, C233-S638, F234-S638, V235-S638, F236-S638, D237-S638, R238-S638, P239-S638, I240-S638, N241-S638, D242-S638, K243-S638, K244-S638, L245-S638, L246-S638, L247-S638, H248-S638, V249-S638, E250-S638, E251-S638, V252-S638, R253-S638, E254-S638, D255-S638, Q256-S638, L257-S638, D258-S638, S259-S638, N260-S638, F261-S638, Q262-S638, M263-S638, Q264-S638, S265-S638, E266-S638, N267-S638, F268-S638, C269-S638, S270-S638, Y271-S638, I272-S638, F273-S638, T274-S638, H275-S638, A276-S638, K277-S638, T278-S638, K279-S638, T280-S638, L281-S638, R282-S638, E283-S638, G284-S638, I285-S638, L286-S638, V287-S638, T288-S638, G289-S638, N290-S638, R291-S638, L292-S638, G293-S638, M294-S638, L295-S638, V296-S638, E297-S638, T298-S638, Y299-S638, L300-S638, D301-S638, A302-S638, I303-S638, N304-S638, S305-S638, G306-S638, A307-S638, T308-S638, P309-S638, C310-S638, L311-S638, E312-S638, N313-S638, A314-S638, M315-S638, A316-S638, V317-S638, L318-S638, A319-S638, Q320-S638, C321-S638, E322-S638, N323-S638, S324-S638, A325-S638, A326-S638, V327-S638, Q328-S638, R329-S638, A330-S638, A331-S638, N332-S638, H333-S638, Y334-S638, S335-S638, Q336-S638, Q337-S638, M338-S638, A339-S638, Q340-S638, Q341-S638, V342-S638, R343-S638, F344-S638, P345-S638, T346-S638, D347-S638, T348-S638, L349-S638, Q350-S638, E351-S638, L352-S638, L353-S638, D354-S638, V355-S638, H356-S638, A357-S638, V358-S638, C359-S638, E360-S638, R361-S638, E362-S638, A363-S638, I364-S638, A365-S638, V366-S638, F367-S638, M368-S638, E369-S638, Y370-S638, S371-S638, F372-S638, K373-S638, D374-S638, K375-S638, S376-S638, Q377-S638, E378-S638, F379-S638, Q380-S638, K381-S638, K382-S638, L383-S638, V384-S638, D385-S638, T386-S638, M387-S638, E388-S638, K389-S638, K390-S638, K391-S638, E392-S638, D393-S638, F394-S638, V395-S638, L396-S638, Q397-S638, N398-S638, E399-S638, E400-S638, A401-S638, S402-S638, A403-S638, K404-S638, Y405-S638, C406-S638, Q407-S638, A408-S638, E409-S638, L410-S638, K411-S638, R412-S638, M413-S638, S414-S638, E415-S638, M416-S638, M417-S638, T418-S638, E419-S638, S420-S638, I421-S638, S422-S638, R423-S638, G424-S638, T425-S638, F426-S638, F427-S638, V428-S638, P429-S638, G430-S638, G431-S638, H432-S638, N433-S638, I434-S638, Y435-S638, L436-S638, E437-S638, A438-S638, K439-S638, K440-S638, K441-S638, I442-S638, E443-S638, Q444-S638, D445-S638, Y446-S638, T447-S638, L448-S638, V449-S638, P450-S638, R451-S638, K452-S638, G453-S638, V454-S638, K455-S638, A456-S638, D457-S638, E458-S638, V459-S638, L460-S638, Q461-S638, S462-S638, F463-S638, L-464-S638, Q465-S638, S466-S638, Q467-S638, V468-S638, V469-S638, I470-S638, E471-S638, E472-S638, S473-S638, L474-S638, L475-S638, Q476-S638, S477-S638, D478-S638, K479-S638, A480-S638, L481-S638, T482-S638, A483-S638, G484-S638, E485-S638, K486-S638, A487-S638, I488-S638, A489-S638, A490-S638, K491-S638, Q492-S638, A493-S638, K494-S638, K495-S638, E496-S638, A497-S638, A498-S638, E499-S638, K500-S638, E501-S638, Q502-S638, E503-S638, L504-S638, L505-S638, R506-S638, Q507-S638, K508-S638, Q509-S638, K510-S638, E511-S638, Q512-S638, Q513-S638, Q514-S638, M515-S638, M516-S638, E517-S638, A518-S638, Q519-S638, E520-S638, R521-S638, S522-S638, F523-S638, Q524-S638, E525-S638, N526-S638, I527-S638, A528-S638, Q529-S638, L530-S638, K531-S638, K532-S638, K533-S638, M534-S638, E535-S638, R536-S638, E537-S638, R538-S638, E539-S638, N540-S638, Y541-S638, M542-S638, R543-S638, E544-S638, L545-S638, R546-S638, K547-S638, M548-S638, L549-S638, S550-S638, H551-S638, K552-S638, M553-S638, K554-S638, V555-S638, L556-S638, E557-S638, E558-S638, L559-S638, L560-S638, T561-S638, E562-S638, G563-S638, F564-S638, K565-S638, E566-S638, I567-S638, F568-S638, E569-S638, S570-S638, L571-S638, N572-S638, E573-S638, E574-S638, I575-S638, N576-S638, R577-S638, L578-S638, K579-S638, E580-S638, Q581-S638, I582-S638, E583-S638, A584-S638, A585-S638, E586-S638, N587-S638, E588-S638, E589-S638, P590-S638, S591-S638, V592-S638, F593-S638, S594-S638, Q595-S638, I596-S638, L597-S638, D598-S638, V599-S638, A600-S638, G601-S638, S602-S638, I603-S638, F604-S638, I605-S638, A606-S638, A607-S638, L608-S638, P609-S638, G610-S638, A611-S638, A612-S638, K613-S638, L614-S638, V615-S638, D616-S638, L617-S638, G618-S638, M619-S638, K620-S638, I621-S638, L622-S638, S623-S638, S624-S638, L625-S638, C626-S638, N627-S638, R628-S638, L629-S638, R630-S638, N631-S638, P632-S638, G633-S638, K634-S638, K635-S638, I636-S638, I637-S638 of SEQ ID NO:3. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal HGBPBMY1 (AK096141) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal HGBPBMY1 (AK096141) deletion polypeptides are encompassed by the present invention: M1-A2, M1-S3, M1-E4, M1-I5, M1-H6, M1-M7, M1-P8, M1-G9, M1-P10, M1-V11, M1-C12, M1-L13, M1-I14, M1-E15, M1-N16, M1-T17, M1-K18, M1-G19, M1-H20, M1-L21, M1-V22, M1-V23, M1-N24, M1-S25, M1-E26, M1-A27, M1-L28, M1-E29, M1-I30, M1-L31, M1-S32, M1-A33, M1-I34, M1-T35, M1-Q36, M1-37P, M1-V38, M1-V39, M1-V40, M1-V41, M1-A42, M1-I43, M1-V44, M1-G45, M1-L46, M1-Y47, M1-R48, M1-T49, M1-G50, M1-K51, M1-S52, M1-Y53, M1-L54, M1-M55, M1-N56, M1-K57, M1-L58, M1-A59, M1-G60, M1-K61, M1-N62, M1-K63, M1-G64, M1-F65, M1-P66, M1-L67, M1-G68, M1-C69, M1-T70, M1-V71, M1-K72, M1-S73, M1-E74, M1-T75, M1-K76, M1-G77, M1-I78, M1-W79, M1-M80, M1-W81, M1-C82, M1-V83, M1-P84, M1-H85, M1-P86, M1-S87, M1-K88, M1-P89, M1-N90, M1-H91, M1-T92, M1-L93, M1-I94, M1-L95, M1-L96, M1-D97, M1-T98, M1-E99, M1-G100, M1-L101, M1-G102, M1-D103, M1-M104, M1-E105, M1-K106, M1-S107, M1-D108, M1-P109, M1-K110, M1-S111, M1-D112, M1-S113, M1-W114, M-I115, M1-F 116, M1-A117, M1-L118, M1-A119, M1-V120, M1-L121, M1-L122, M1-S123, M1-S124, M1-S125, M1-F126, M1-V127, M1-Y128, M1-N129, M1-S130, M1-M131, M1-G132, M1-T133, M1-I134, M1-N135, M1-H136, M1-Q137, M1-A138, M1-L139, M1-E140, M1-Q141, M1-L142, M1-H143, M1-Y144, M1-V145, M1-T146, M1-E147, M1-L148, M1-T149, M1-E150, M1-L151, M1-I152, M1-R153, M1-A154, M1-K155, M1-S156, M1-C157, M1-P158, M1-R159, M1-P160, M1-D161, M1-E162, M1-V163, M1-E164, M1-D165, M1-S166, M1-S167, M1-E168, M1-F169, M1-V170, M1-S171, M1-F172, M1-F173, M1-P174, M1-D175, M1-F176, M1-I177, M1-W178, M1-T179, M1-V180, M1-R181, M1-D182, M1-F183, M1-T184, M1-L185, M1-E186, M1-L187, M1-K188, M1-L189, M1-D190, M1-G191, M1-H192, M1-P193, M1-I194, M1-T195, M1-E196, M1-D197, M1-E198, M1-Y199, M1-L200, M1-E201, M1-N202, M1-A203, M1-L204, M1-K205, M1-L206, M1-I207, M1-S208, M1-G209, M1-K210, M1-N211, M1-P212, M1-Q213, M1-I214, M1-Q215, M1-N216, M1-S217, M1-N218, M1-K219, M1-P220, M1-R221, M1-E222, M1-W223, M1-I224, M1-R225, M1-H226, M1-F227, M1-F228, M1-P229, M1-K230, M1-Q231, M1-K232, M1-C233, M1-F234, M1-V235, M1-F236, M1-D237, M1-R238, M1-P239, M1-I240, M1-N241, M1-D242, M1-K243, M1-K244, M1-L245, M1-L246, M1-L247, M1-H248, M1-V249, M1-E250, M1-E251, M1-V252, M1-R253, M1-E254, M1-D255, M1-Q256, M1-L257, M1-D258, M1-S259, M1-N260, M1-F261, M1-Q262, M1-M263, M1-Q264, M1-S265, M1-E266, M1-N267, M1-F268, M1-C269, M1-S270, M1-Y271, M1-I272, M1-F273, M1-T274, M1-H275, M1-A276, M1-K277, M1-T278, M1-K279, M1-T280, M1-L281, M1-R282, M1-E283, M1-G284, M1-I285, M1-L286, M1-V287, M1-T288, M1-G289, M1-290, M1-R291, M1-L292, M1-G293, M1-M294, M1-L295, M1-V296, M1-E297, M1-T298, M1-Y299, M1-L300, M1-D301, M1-A302, M1-I303, M1-N304, M1-S305, M1-G306, M1-A307, M1-T308, M1-P309, M1-C310, M1-L311, M1-E312, M1-N313, M1-A314, M1-M315, M1-A316, M1-V317, M1-L318, M1-A319, M1-Q320, M1-C321, M1-E322, M1-N323, M1-S324, M1-A325, M1-A326, M1-V327, M1-Q328, M1-R329, M1-A330, M1-A331, M1-N332, M1-H333, M1-Y334, M1-S335, M1-Q336, M1-Q337, M1-M338, M1-A339, M1-Q340, M1-Q341, M1-V342, M1-R343, M1-F344, M1-P345, M1-T346, M1-D347, M1-T348, M1-L349, M1-Q350, M1-E351, M1-L352, M1-L353, M1-D354, M1-V355, M1-H356, M1-A357, M1-V358, M1-C359, M1-E360, M1-R361, M1-E362, M1-A363, M1-I364, M1-A365, M1-V366, M1-F367, M1-M368, M1-E369, M1-Y370, M1-S371, M1-F372, M1-K373, M1-D374, M1-K375, M1-S376, M1-Q377, M1-E378, M1-F379, M1-Q380, M1-K381, M1-K382, M1-L383, M1-V384, M1-D385, M1-T386, M1-M387, M1-E388, M1-K389, M1-K390, M1-K391, M1-E392, M1-D393, M1-F394, M1-V395, M1-L396, M1-Q397, M1-N398, M1-E399, M1-E400, M1-A401, M1-S402, M1-A403, M1-K404, M1-Y405, M1-C406, M1-Q407, M1-A408, M1-E409, M1-L410, M1-K411, M1-R412, M1-L413, M1-S414, M1-E415, M1-L416, M1-L417, M1-T418, M1-E419, M1-S420, M1-I421, M1-S422, M1-R423, M1-G424, M1-T425, M1-F426, M1-F427, M1-V428, M1-P429, M1-G430, M1-G431, M1-H432, M1-N433, M1-I434, M1-Y435, M1-L436, M1-E437, M1-A438, M1-K439, M1-K440, M1-K441, M1-I442, M1-E443, M1-Q444, M1-D445, M1-Y446, M1-T447, M1-L448, M1-V449, M1-P450, M1-R451, M1-K452, M1-G453, M1-V454, M1-K455, M1-A456, M1-D457, M1-E458, M1-V459, M1-L460, M1-Q461, M1-S462, M1-F463, M1-L464, M1-Q465, M1-S466, M1-Q467, M1-V468, M1-V469, M1-I470, M1-E471, M1-E472, M1-S473, M1-I474, M1-L475, M1-Q476, M1-S477, M1-D478, M1-K479, M1-A480, M1-L481, M1-T482, M1-A483, M1-G484, M1-E485, M1-K486, M1-A487, M1-I488, M1-A489, M1-A490, M1-K491, M1-Q492, M1-A493, M1-K494, M1-K495, M1-E496, M1-A497, M1-A498, M1-E499, M1-K500, M1-E501, M1-Q502, M1-E503, M1-L504, M1-L505, M1-R506, M1-Q507, M1-K508, M1-Q509, M1-K510, M1-E511, M1-Q512, M1-Q513, M1-Q514, M1-M515, M1-M516, M1-E517, M1-A518, M1-Q519, M1-E520, M1-R521, M1-S522, M1-F523, M1-Q524, M1-E525, M1-N526, M1-I527, M1-A528, M1-Q529, M1-L530, M1-K531, M1-K532, M1-K533, M1-M534, M1-E535, M1-R536, M1-E537, M1-R538, M1-E539, M1-N540, M1-Y541, M1-M542, M1-R543, M1-E544, M1-L545, M1-R546, M1-K547, M1-M548, M1-L549, M1-S550, M1-H551, M1-K552, M1-M553, M1-K554, M1-V555, M1-L556, M1-E557, M1-E558, M1-L559, M1-L560, M1-T561, M1-E562, M1-G563, M1-F564, M1-K565, M1-E566, M1-I567, M1-F568, M1-E569, M1-S570, M1-L571, M1-N572, M1-E573, M1-E574, M1-I575, M1-N576, M1-R577, M1-L578, M1-K579, M1-E580, M1-Q581, M1-I582, M1-E583, M1-A584, M1-A585, M1-E586, M1-N587, M1-E588, M1-E589, M1-P590, M1-S591, M1-V592, M1-F593, M1-S594, M1-Q595, M1-I596, M1-L597, M1-D598, M1-V599, M1-A600, M1-G601, M1-S602, M1-I603, M1-F604, M1-I605, M1-A606, M1-A607, M1-L608, M1-P609, M1-G610, M1-A611, M1-A612, M1-K613, M1-L614, M1-V615, M1-D616, M1-L617, M1-G618, M1-M619, M1-K620, M1-I621, M1-L622, M1-S623, M1-S624, M1-L625, M1-C626, M1-N627, M1-R628, M1-L629, M1-R630, M1-N631, M1-P632, M1-G633, M1-K634, M1-K635, M1-I636 and/or M1-I637 of SEQ ID NO:3. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal HGBPBMY1 (AK096141) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, polypeptides of the present invention may comprise polypeptide sequences corresponding to, for example, internal regions of a HGBPBMY1 (AK096141) polypeptide (e.g., any combination of both N- and C-terminal HGBPBMY1 (AK096141) polypeptide deletions) of SEQ ID NO:3. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of HGBPBMY1 (AK096141) (SEQ ID NO:3), and where CX refers to any C-terminal deletion polypeptide amino acid of HGBPBMY1 (AK096141) (SEQ ID NO:3). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the HGBPBMY1 (AK096141) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate HGBPBMY1 (AK096141) on the one hand (i.e., agonists) and which inhibit the function of HGBPBMY1 (AK096141) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express a HGBPBMY1 (AK096141) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n       FIG. 43  shows an expanded expression profile of the GBP HGBPBMY1 (AK096141). The figure illustrates the relative expression level of HGBPBMY1 amongst various mRNA tissue sources. As shown, the HGBPBMY1 polypeptide exhibited high expression in normal spleen; tonsil; blood mononuclear cells; liver parenchyma; lung parenchyma; digestive system including stomach, duodenum, jejunum, ileum; placenta; ovary; and was detectable in regions of the brain, adrenal gland, ureter, and bladder.  FIG. 43  also illustrates the relative expression level of HGBPBMY1 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY1 polypeptide showed increased expression in breast and testicle tumors relative to controls; and high expression in normal and diseased lung parenchyma. Expression data was obtained by measuring the steady state HGBPBMY1 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:61 and 62, and TAQMAN probe (SEQ ID NO:63) as described in Example 36 herein. These data support a role of HGBPBMY1 in regulating various functions. HGBPBMY1 may also be participating in the formation of testicle and breast tumors and thus small molecule modulators of HGBPBMY1 function may represent a novel therapeutic option in the treatment of breast and testicle cancers, as well as pulmonary, gastrointestinal and/or immune system diseases. \\n     Features of the Polypeptide Encoded by Gene No. 2 \\n     The polypeptide encoded by this gene, HGBPBMY2 (4843 30 1 1; 4843 — 1), is provided as SEQ ID NO:5 ( FIGS. 2A–2C ) and is encoded by the polynucleotide sequence according to SEQ ID NO:4 ( FIGS. 2A–2C ) and/or by a polynucleotide contained within the deposited clone. HGBPBMY2 (4843 30 1 1; 4843 — 1) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, human GBP-1, human GBP-2, human GBP-3, human GBP-4 and human GBP-5. \\n     The determined nucleotide sequence of a HGBPBMY2 (4843 30 1 1; 4843 — 1), (i.e., the cDNA shown in  FIGS. 2A–2C  and in SEQ ID NO:4) comprises an open reading frame encoding a protein of about 788 amino acid residues. The predicted amino acid sequence of a HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide is shown in  FIGS. 2A–2C  (SEQ ID NO:5). The percent identity and similarity values between the HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide to the known GBP family member hGBP1 is provided in  FIG. 9 . The HGBPBMY2 (4843 30 1 1; 4843 — 1) protein shown in  FIGS. 2A–2C  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide showed expression predominantly in tonsil (see  FIG. 16 ). \\n     Based upon the strong homology to members of the GBP family members, a HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide is expected to share at least some biological activity with GBP family members, specifically hGBP-1, hGBP-2, hGBP-3, and/or hGBP-4. \\n     The HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, diagnosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system related directly or indirectly to tonsil function. \\n     The HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue, preferably human tissue. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP3 and hGBP-4, combined with HGBPBMY2 (4843 30 1 1; 4843 — 1) expression in tonsil tissue suggests a potential utility for HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. In representative embodiments, HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. The HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. The HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide may be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide of the present invention can be useful in the treatment of include various autoimmune diseases such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide of the present invention can also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases. Such diseases and conditions include, for example, Eecombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimike immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against the HGBPBMY2 (4843 30 1 1; 4843 — 1) protein, may show utility as a tumor marker and/or immunotherapy targets. \\n     The HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides and polypeptides, including fragments and for antagonsists thereof, may have uses which include identification of modulators of HGBPBMY2 (4843 30 1 1; 4843 — 1) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the HGBPBMY2 (4843 30 1 1; 4843 — 1) protein could be used as diagnostic agents of conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptides and polynucleotides have additional uses which include diagnosing diseases related to the over and/or under expression of HGBPBMY2 (4843 30 1 1; 4843 — 1) by identifying mutations in the HGBPBMY2 (4843 30 1 1; 4843 — 1) gene by using HGBPBMY2 (4843 30 1 1; 4843 — 1) sequences as probes or by determining HGBPBMY2 (4843 30 1 1; 4843 — 1) protein or mRNA expression levels. HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptides can be useful for screening compounds that affect the activity of the protein. HGBPBMY2 (4843 30 1 1; 4843 — 1) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with HGBPBMY2 (4843 30 1 1; 4843 — 1), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the HGBPBMY2 (4843 30 1 1; 4843 — 1) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased immune system tissue, as compared to normal tissue might indicate a function in modulating immune system function, for example. In the case of HGBPBMY2 (4843 30 1 1; 4843 — 1), tonsil can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the HGBPBMY2 (4843 30 1 1; 4843 — 1) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of HGBPBMY2 (4843 30 1 1; 4843 — 1), a disease correlation related to HGBPBMY2 (4843 30 1 1; 4843 — 1) can be made by comparing the mRNA expression level of HGBPBMY2 (4843 30 1 1; 4843 — 1) in normal tissue, as compared to diseased tissue (particularly diseased immune system tissue, such as diseased tonsil tissue). Significantly higher or lower levels of HGBPBMY2 (4843 30 1 1; 4843 — 1) expression in the diseased tissue can suggest HGBPBMY2 (4843 30 1 1; 4843 — 1) plays a role in disease progression, and antagonists against HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of HGBPBMY2 (4843 30 1 1; 4843 — 1) expression in the diseased tissue can suggest HGBPBMY2 (4843 30 1 1; 4843 — 1) plays a defensive role against disease progression, and agonists of HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:4 ( FIGS. 2A–2C ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the HGBPBMY2 (4843 30 1 1; 4843 — 1), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. GTPase activity assays can also be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in mice and/or rats (e.g. by RNAi or homologous recombination) and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology and the resulting generation of transgenic mice and/or rats. Expressing a particular gene in either sense or antisense orientation in a transgenic mouse or rat could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a tonsil tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of HGBPBMY2 (4843 30 1 1; 4843 — 1) transgenic mice or rats, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic mice or rats to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal HGBPBMY2 (4843 30 1 1; 4843 — 1) deletion polypeptides are encompassed by the present invention: M1-K788, E2-K788, S3-K788, G4-K788, P5-K788, K6-K788, M7-K788, L8-K788, A9-K788, P10-K788, V11-K788, C12-K788, L13-K788, V14-K788, E15-K788, N16-K788, N17-K788, N18-K788, E19-K788, Q20-K788, L21-K788, L22-K788, V23-K788, N24-K788, Q25-K788, Q26-K788, A27-K788, I28-K788, Q29-K788, I30-K788, L31-K788, E32-K788, K33-K788, I34-K788, S35-K788, Q36-K788, P37-K788, V38-K788, V39-K788, V40-K788, V41-K788, A42-K788, I43-K788, V44-K788, G45-K788, L46-K788, Y47-K788, R48-K788, T49-K788, G50-K788, K51-K788, S52-K788, Y53-K788, L54-K788, M55-K788, N56-K788, H57-K788, L58-K788, A59-K788, G60-K788, Q61-K788, N62-K788, H63-K788, G64-K788, F65-K788, P66-K788, L67-K788, G68-K788, S69-K788, T70-K788, V71-K788, Q72-K788, S73-K788, E74-K788, T75-K788, K76-K788, G77-K788, I78-K788, W79-K788, M80-K788, W81-K788, C82-K788, V83-K788, P84-K788, H85-K788, P86-K788, S87-K788, K88-K788, P89-K788, N90-K788, H91-K788, T92-K788, L93-K788, V94-K788, L95-K788, L96-K788, D97-K788, T98-K788, E99-K788, G100-K788, L101-K788, G102-K788, D103-K788, V104-K788, E105-K788, K106-K788, G107-K788, D108-K788, P109-K788, K110 N111-K788, D112-K788, S113-K788, W114-K788, I115-K788, F116-K788, A117-K788, L118-K788, A119-K788, V120-K788, L121-K788, L122-K788, C123-K788, S124-K788, T125-K788, F126-K788, V127-K788, Y128-K788, N129-K788, S130-K788, M131-K788, S132-K788, T133-K788, I134-K788, N135-K788, H136-K788, Q137-K788, A138-K788, L139-K788, E140-K788, Q141-K788, L142-K788, Q143-K788, Y144-K788, P145-K788, S146-K788, R147-K788, N148-K788, R149-K788, T150-K788, E151-K788, P152-K788P153-K788, G154-K788, F155-K788, I156-K788, D157-K788, F158-K788, I159-K788, G160-K788, M161-K788, E162-K788, I163-K788, N164-K788, P165-K788, S166-K788, Y167-K788, V168-K788, T169-K788, E170-K788, L171-K788, T172-K788, E173-K788, L174-K788, I175-K788, K176-K788, A177-K788, K178-K788, S179-K788, S180-K788, P181-K788, R182-K788, P183-K788, D184-K788, G185-K788, V186-K788, E187-K788, D188-K788, S189-K788, T190-K788, E191-K788, F192-K788, V193-K788, S194-K788, F195-K788, F196-K788, P197-K788, D198-K788, F199-K788, L200-K788, W201-K788, T202-K788, V203-K788, R204-K788, D205-K788, F206-K788, T207-K788, L208-K788, E209-K788, L210-K788, K211-K788, L212-K788, N213-K788, G214-K788, H215-K788, P216-K788, I217-K788, T218-K788, E219-K788, D220-K788, E221-K788, Y222-K788, L223-K788, E224-K788, N225-K788, A226-K788, L227-K788, K228-K788, L229-K788, I230-K788, Q231-K788, G232-K788, N233-K788, N234-K788, P235-K788, R236-K788, V237-K788, Q238-K788, T239-K788, S240-K788, N241-K788, F242-K788, P243-K788, R244-K788, E245-K788, C246-K788, I247-K788, R248-K788, R249-K788, F250-K788, F251-K788, P252-K788, K253-K788, R254-K788, K255-K788, C256-K788, F257-K788, V258-K788, F259-K788, D260-K788, R261-K788, P262-K788, T263-K788, N264-K788, D265-K788, K266-K788, D267-K788, L268-K788, L269-K788, A270-K788, N271-K788, I272-K788, E273-K788, K274-K788, V275-K788, S276-K788, E277-K788, K278-K788, Q279-K788, L280-K788, D281-K788, P282-K788, K283-K788, F284-K788, Q285-K788, E286-K788, Q287-K788, T288-K788, N289-K788, I290-K788, F291-K788, C292-K788, S293-K788, Y294-K788, I295-K788, F296-K788, T296-K788, H298-K788, A299-K788, R300-K788, T301-K788, K302-K788, T303-K788, L304-K788, R305-K788, E306-K788, G307-K788, I308-K788, T309-K788, V310-K788, T311-K788, G312-K788, N313-K788, R314-K788, L315-K788, G316-K788, T317-K788, L318-K788, A319-K788, V320-K788, T321-K788, Y322-K788, V323-K788, E324-K788, A325-K788, I326-K788, N327-K788, S328-K788, G329-K788, A330-K788, V331-K788, P332-K788, C333-K788, L334-K788, E335-K788, N336-K788, A337-K788, V338-K788, I339-K788, T340-K788, L341-K788, A342-K788, Q343-K788, R344-K788, E345-K788, N346-K788, S347-K788, A348-K788, A349-K788, V350-K788, Q351-K788, R352-K788, A353-K788, A354-K788, D355-K788, Y356-K788, Y357-K788, S358-K788, Q359-K788, Q360-K788, M361-K788, A362-K788, Q363-K788, R364-K788, V365-K788, K366-K788, L367-K788, P368-K788, T369-K788, D370-K788, T371-K788, L372-K788, Q373-K788, E374-K788, L375-K788, L376-K788, D377-K788, M378-K788, H379-K788, A380-K788, A381-K788, C382-K788, E383-K788, R384-K788, E385-K788, A386-K788, I387-K788, A388-K788, I389-K788, F390-K788, M391-K788, E392-K788, H393-K788, S394-K788, F395-K788, K396-K788, D397-K788, E398-K788, N399-K788, Q400-K788, E401-K788, F402-K788, Q403-K788, K404-K788, K405-K788, F406-K788, M407-K788, E408-K788, T409-K788, T410-K788, M411-K788, N412-K788, K413-K788, K414-K788, G415-K788, D416-K788, F417-K788, L418-K788, L419-K788, Q420-K788, N421-K788, E422-K788, E423-K788, S424-K788, S425-K788, V426-K788, Q427-K788, Y428-K788, C429-K788, Q430-K788, A431-K788, K432-K788, L433-K788, N434-K788, E435-K788, L436-K788, S437-K788, K438-K788, G439-K788, L440-K788, M441-K788, E442-K788, S443-K788, I444-K788, S445-K788, A446-K788, G447-K788, S448-K788, F449-K788, S450-K788, V451-K788, P452-K788, G453-K788, G454-K788, H455-K788, K456-K788, L457-K788, Y458-K788, M459-K788, E460-K788, T461-K788, K462-K788, E463-K788, R464-K788, I465-K788, E466-K788, Q467-K788, D468-K788, Y469-K788, W470-K788, Q471-K788, V472-K788, P473-K788, R474-K788, K475-K788, G476-K788, V477-K788, K478-K788, A479-K788, K480-K788, E481-K788, V482-K788, F483-K788, Q484-K788, R485-K788, F486-K788, L487-K788, E488-K788, S489-K788, Q490-K788, M491-K788, V492-K788, I493-K788, E494-K788, E495-K788, S496-K788, I497-K788, L498-K788, Q499-K788, S500-K788, D501-K788, K502-K788, A503-K788, L504-K788, T505-K788, D506-K788, R507-K788, E508-K788, K509-K788, A510-K788, V511-K788, A512-K788, V513-K788, D514-K788, R515-K788, A516-K788, K517-K788, K518-K788, E519-K788, A520-K788, A521-K788, E522-K788, K523-K788, E524-K788, Q525-K788, E526-K788, L527-K788, L528-K788, K529-K788, Q530-K788, K531-K788, L532-K788, Q533-K788, E534-K788, Q535-K788, Q536-K788, Q537-K788, Q538-K788, M539-K788, E540-K788, A541-K788, Q542-K788, D543-K788, K544-K788, S545-K788, R546-K788, K547-K788, E548-K788, N549-K788, I550-K788, A551-K788, Q552-K788, L553-K788, K554-K788, E555-K788, K556-K788, L557-K788, Q558-K788, M559-K788, E560-K788, R561-K788, E562-K788, H563-K788, L564-K788, L565-K788, R566-K788, E567-K788, Q568-K788, I569-K788, M570-K788, M571-K788, L572-K788, E573-K788, H574-K788, T575-K788, Q576-K788, K577-K788, V578-K788, Q579-K788, N580-K788, D581-K788, W582-K788, L583-K788, H584-K788, E585-K788, G586-K788, F587-K788, K588-K788, K589-K788, K590-K788, Y591-K788, E592-K788, E593-K788, M594-K788, N595-K788, A596-K788, E597-K788, I598-K788, S599-K788, Q600-K788, F601-K788, K602-K788, R603-K788, M604-K788, I605-K788, D606-K788, T607-K788, T608-K788, K609-K788, N610-K788, D611-K788, D612-K788, T613-K788, P614-K788, W615-K788, I616-K788, A617-K788, R618-K788, T619-K788, L620-K788, D621-K788, N622-K788, L623-K788, A624-K788, D625-K788, E626-K788, L627-K788, T628-K788, A629-K788, I630-K788, L631-K788, S632-K788, A633-K788, P634-K788, A635-K788, K636-K788, L637-K788, I638-K788, G639-K788, H640-K788, G641-K788, V642-K788, K643-K788, G644-K788, N645-K788, E646-K788, L647-K788, T648-K788, R649-K788, N650-K788, M651-K788, S652-K788, P653-K788, H654-K788, I655-K788, Q656-K788, R657-K788, S658-K788, G659-K788, S660-K788, Q661-K788, F662-K788, F663-K788, S664-K788, R665-K788, G666-K788, R667-K788, R668-K788, G669-K788, G670-K788, R671-K788, V672-K788, A673-K788, V674-K788, C675-K788, F676-K788, L677-K788, G678-K788, G679-K788, I680-K788, I681-K788, S682-K788, S683-K788, I684-K788, M685-K788, E686-K788, V687-K788, A688-K788, T689-K788, Q690-K788, H691-K788, N692-K788, L693-K788, E694-K788, P695-K788, L696-K788, P697-K788, Q698-K788, H699-K788, T700-K788, H701-K788, C702-K788, S703-K788, N704-K788, T705-K788, E706-K788, A707-K788, N708-K788, K709-K788, S710-K788, E711-K788, E712-K788, V713-K788, H714-K788, H715-K788, F716-K788, W717-K788, R718-K788, L719-K788, F720-K788, A721-K788, R722-K788, L723-K788, A724-K788, G725-K788, D726-K788, D727-K788, I728-K788, E729-K788, V730-K788, S731-K788, G732-K788, T733-K788, E734-K788, L735-K788, M736-K788, N737-K788, V738-K788, V739-K788, N740-K788, K741-K788, V742-K788, V743-K788, I744-K788, S745-K788, R746-K788, T747-K788, E748-K788, L749-K788, K750-K788, T751-K788, H752-K788, G753-K788, F754-K788, G755-K788, I756-K788, D757-K788, T758-K788, C759-K788, Q760-K788, S761-K788, M762-K788, V763-K788, A764-K788, M765-K788, M766-K788, D767-K788, S768-K788, D769-K788, T770-K788, I771-K788, G772-K788, K773-K788, L774-K788, D775-K788, F776-K788, E777-K788, E778-K788, F779-K788, N780-K788, Y781-K788, L782-K788, W783-K788, N784-K788, N785-K788, I786-K788, and or K787-K788 of SEQ ID NO:5. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal HGBPBMY2 (4843 30 1 1; 4843 — 1) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal HGBPBMY2 (4843 30 1 1; 4843 — 1) deletion polypeptides are encompassed by the present invention: M1-E2, M1-S3, M1-G4, M1-P5, M1-K6, M1-M7, M1-L8, M1-A9, M1-P10, M1-V11, M1-C12, M1-L13 M1-V14, M1-E15, M1-N16, M1-N17, M1-N18, M1-E19, M1-Q20, M1-L21, M1-L22, M1-V23, M1-N24, M1-Q25, M1-Q26, M1-A27, M1-I28, M1-Q29, M1-I30, M1-L31, M1-E32, M1-K33, M1-I34, M1-S35, M1-Q36, M1-P37, M1-V38, M1-V39, M1-V40, M1-V41, M1-A42, M1-I43, M1-V44, M1-G45, M1-L46, M1-Y47, M1-R48, M1-T49, M1-G50, M1-K51, M1-S52, M1-Y53, M1-L54, M1-M55, M1-N56, M1-H57, M1-L58, M1-A59, M1-G60, M1-Q61, M1-N62, M1-H63, M1-G64, M1-F65, M1-P66, M1-L67, M1-G68, M1-S69, M1-T70, M1-V71, M1-Q72, M1-S73, M1-E74, M1-T75, M1-K76, M1-G77, M1-I78, M1-W79, M1-M80, M1-W81, M1-C82, M1-V83, M1-P84, M1-H85, M1-P86, M1-S87, M1-K88, M1-P89, M1-N90, M1-H91, M1-T92, M1-L93, M1-V94, M1-L95, M1-L96, M1-D97, M1-T98, M1-E99, M1-G100, M1-L101, M1-G102, M1-D103, M1-V104, M1-E105, M1-K106, M1-G107, M1-D108, M1-P109, M1-K110, M1-N111, M1-D112, M1-S113, M1-W114, M1-I115, M1-F116, M1-A117, M1-L118, M1-A119, M1-V120, M1-L121, M1-L122, M1-C123, M1-S124, M1-T125, M1-F126, M1-V127, M1-Y128, M1-N129, M1-S130, M1-M131, M1-S132, M1-T133, M1-I134, M1-N135, M1-H136, M1-Q137, M1-A138, M1-L139, M1-E140, M1-Q141, M1-L142, M1-Q143, M1-Y144, M1-P145, M1-S146, M1-R147, M1-N148, M1-R149, M1-T150, M1-E151, M1-P152, M1-P153, M1-G154, M1-F155, M1-I156, M1-D157, M1-F158, M1-I159, M1-G160, M1-M161, M1-E162, M1-I163, M1-N164, M1-P165, M1-S166, M1-Y167, M1-V168, M1-T169, M1-E170, M1-L171, M1-T172, M1-E173, M1-L174, M1-I175, M1-K176, M1-A177, M1-K178, M1-S179, M1-S180, M1-P181, M1-R182, M1-P183, M1-D184, M1-G185, M1-V186, M1-E187, M1-D188, M1-S189, M1-T190, M1-E191, M1-F192, M1-V193, M1-S194, M1-F195, M1-F196, M1-P197, M1-D198, M1-F199, M1-L200, M1-W201, M1-T202, M1-V203, M1-R204, M1-D205, M1-F206, M1-T207, M1-L208, M1-E209, M1-L210, M1-K211, M1-L212, M1-N213, M1-G214, M1-H215, M1-P216, M1-I217, M1-T218, M1-E219, M1-D220, M1-E221, M1-Y222, M1-L223, M1-E224, M1-N225, M1-A226, M1-L227, M1-K228, M1-L229, M1-I230, M1-Q231, M1-G232, M1-N233, M1-N234, M1-P235, M1-R236, M1-V237, M1-Q238, M1-T239, M1-S40, M1-N241, M1-F242, M1-P243, M1-R244, M1-E245, M1-C246, M1-I247, M1-R248, M1-R249, M1-F250, M1-F251, M1-P252, M1-K253, M1-R254, M1-K255, M1-C256, M1-F257, M1-V258, M1-F259, M1-D260, M1-R261, M1-P262, M1-T263, M1-N264, M1-D265, M1-K266, M1-D267, M1-L268, M1-L269, M1-A270, M1-N271, M1-I272, M1-E273, M1-K274, M1-V275, M1-S276, M1-E277, M1-K278, M1-Q279, M1-L280, M1-D281, M1-P282, M1-K283, M1-F284, M1-Q285, M1-E286, M1-Q287, M1-T288, M1-N289, M1-I290, M1-F291, M1-C292, M1-S293, M1-Y294, M1-I295, M1-F296, M1-T297, M1-H298, M1-A299, M1-R300, M1-T301, M1-K302, M1-T303, M1-L304, M1-R305, M1-E306, M1-G307, M1-I308, M1-T309, M1-V310, M1-T311, M1-G312, M1-N313, M1-R314, M1-L315, M1-G316, M1-T317, M1-L318, M1-A319, M1-V320, M1-T321, M1-Y322, M1-V323, M1-E324, M1-A325, M1-I326, M1-N327, M1-S328, M1-G329, M1-A330, M1-V331, M1-P332, M1-C333, M1-L334, M1-E335, M1-N336, M1-A337, M1-V338, M1-I339, M1-T340, M1-L341, M1-A342, M1-Q343, M1-R344, M1-E345, M1-N346, M1-S347, M1-A348, M1-A349, M1-V350, M1-Q351, M1-R352, M1-A353, M1-A354, M1-D355, M1-Y356, M1-Y357, M1-S358, M1-Q359, M1-Q360, M1-M361, M1-A362, M1-Q363, M1-R364, M1-V365, M1-K366, M1-L367, M1-P368, M1-T369, M1-D370, M1-T371, M1-L372, M1-Q373, M1-E374, M1-L375, M1-L376, M1-D377, M1-M378, M1-H379, M1-A380, M1-A381, M1-C382, M1-E383, M1-R384, M1-E385, M1-A386, M1-I387, M1-A388, M1-I389, M1-F390, M1-M391, M1-E392, M1-H393, M1-S394, M1-F395, M1-K396, M1-D397, M1-E398, M1-N399, M1-Q400, M1-E401, M1-F402, M1-Q403, M1-K404, M1-K405, M1-F406, M1-M407, M1-E408, M1-T409, M1-T410, M1-M411, M1-N412, M1-K413, M1-K414, M1-G415, M1-D416, M1-F417, M1-L418, M1-L419, M1-Q420, M1-N421, M1-E422, M1-E423, M1-S424, M1-S425, M1-V426, M1-Q427, M1-Y428, M1-C429, M1-Q430, M1-A431, M1-K432, M1-L433, M1-N434, M1-E435, M1-L436, M1-S437, M1-K438, M1-G439, M1-L440, M1-M441, M1-E442, M1-S443, M1-I444, M1-S445, M1-A446, M1-G447, M1-S448, M1-F449, M1-S450, M1-V451, M1-P452, M1-G453, M1-G454, M1-H455, M1-K456, M1-L457, M1-Y458, M1-M459, M1-E460, M1-T461, M1-K462, M1-E463, M1-R464, M1-I465, M1-E466, M1-Q467, M1-D468, M1-Y469, M1-W470, M1-Q471, M1-V472, M1-P473, M1-R474, M1-K475, M1-G476, M1-V477, M1-K478, M1-A479, M1-K480, M1-E481, M1-V482, M1-F483, M1-Q484, M1-R485, M1-F486, M1-L487, M1-E488, M1-S489, M1-Q490, M1-M491, M1-V492, M1-I493, M1-E494, M1-E495, M1-S496, M1-I497, M1-L498, M1-Q499, M1-S500, M1-D501, M1-K502, M1-A503, M1-L504, M1-T508, M1-D506, M1-R507, M1-E508, M1-K509, M1-A510, M1-V511, M1-A512, M1-V513, M1-D514, M1-R515, M1-A516, M1-K517, M1-K518, M1-E519, M1-A520, M1-A521, M1-E522, M1-K523, M1-E524, M1-Q525, M1-E526, M1-L527, M1-L528, M1-K529, M1-Q530, M1-K531, M1-L532, M1-Q533, M1-E534, M1-Q535, M1-Q536, M1-Q537, M1-Q538, M1-M539, M1-E540, M1-A541, M1-Q542, M1-D543, M1-K544, M1-S545, M1-R546, M1-K547, M1-E548, M1-N549, M1-I550, M1-A551, M1-Q552, M1-L553, M1-K554, M1-E555, M1-K556, M1-L557, M1-Q558, M1-M559, M1-E560, M1-R561, M1-E562, M1-H563, M1-L564, M1-L565, M1-R566, M1-E567, M1-Q568, M1-I569, M1-M570, M1-M571, M1-L572, M1-E573, M1-H574, M1-T575, M1-Q576, M1-K577, M1-V578, M1-Q579, M1-N580, M1-D581, M1-W582, M1-L583, M1-H584, M1-E585, M1-G586, M1-F587, M1-K588, M1-K589, M1-K590, M1-Y591, M1-E592, M1-E593, M1-M594, M1-N595, M1-A596, M1-E597, M1-I598, M1-S599, M1-Q600, M1-F601, M1-K602, M1-R603, M1-M604, M1-I605, M1-D606, M1-T607, M1-T608, M1-K609, M1-N610, M1-D611, M1-D612, M1-T613, M1-P614, M1-W615, M1-I616, M1-A617, M1-R618, M1-T619, M1-L620, M1-D621, M1-N622, M1-L623, M1-A624, M1-D625, M1-E626, M1-L627, M1-T628, M1-A629, M1-I630, M1-L631, M1-S632, M1-A633, M1-P634, M1-A635, M1-K636, M1-L637, M1-I638, M1-G639, M1-H640, M1-G641, M1-V642, M1-K643, M1-G644, M1-N645, M1-E646, M1-L647, M1-T648, M1-R649, M1-N650, M1-M651, M1-S652, M1-P653, M1-H654, M1-I655, M1-Q656, M1-R657, M1-S658, M1-G659, M1-S660, M1-Q661, M1-F662, M1-F663, M1-S664, M1-R665, M1-G666, M1-R667, M1-R668, M1-G669, M1-G670, M1-R671, M1-V672, M1-A672, M1-V674, M1-C675, M1-F676, M1-L677, M1-G678, M1-G679, M1-I680, M1-I681, M1-S682, M1-S683, M1-I684, M1-M685, M1-E686, M1-V687, M1-A688, M1-T689, M1-Q690, M1-H691, M1-N692, M1-L693, M1-E694, M1-P695, M1-L696, M1-P697, M1-Q698, M1-H699, M1-T700, M1-H701, M1-C702, M1-S703, M1-N704, M1-T705, M1-E706, M1-A707, M1-N708, M1-K709, M1-S710, M1-E711, M1-E712, M1-V713, M1-H714, M1-H715, M1-F716, M1-W717, M1-R718, M1-L719, M1-F720, M1-A721, M1-R722, M1-L723, M1-A724, M1-G725, M1-D726, M1-D727, M1-I728, M1-E729, M1-V730, M1-S731, M1-G732, M1-T733, M1-E734, M1-L735, M1-M736, M1-N737, M1-V738, M1-V739, M1-N740, M1-K741, M1-V742, M1-V743, M1-I744, M1-S745, M1-R746, M1-T747, M1-E748, M1-L749, M1-K750, M1-T751, M1-H752, M1-G753, M1-F754, M1-G755, M1-I756, M1-D757, M1-T758, M1-C759, M1-Q760, M1-S761, M1-M762, M1-V763, M1-A764, M1-M765, M1-M766, M1-D767, M1-S768, M1-D769, M1-T770, M1-I771, M1-G772, M1-K773, M1-L774, M1-D775, M1-F776, M1-E777, M1-E778, M1-F779, M1-N780, M1-Y781, M1-L782, M1-W783, M1-N784, M1-N785, M1-I786 and/or M1-K787 of SEQ ID NO:5. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal HGBPBMY2 (4843 30 1 1; 4843 — 1) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide (e.g., any combination of both N- and C-terminal HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide deletions) of SEQ ID NO:5. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of HGBPBMY2 (4843 30 1 1; 4843 — 1) (SEQ ID NO:5), and where CX refers to any C-terminal deletion polypeptide amino acid of HGBPBMY2 (4843 30 1 1; 4843 — 1) (SEQ ID NO:5). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate HGBPBMY2 (4843 30 1 1; 4843 — 1) on the one hand (i.e., agonists) and which inhibit the function of HGBPBMY2 (4843 30 1 1; 4843 — 1) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express a HGBPBMY2 (4843 30 1 1; 4843 — 1) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n       FIG. 44  shows an expanded expression profile of the GBP HGBPBMY2 (4843 — 1). The figure illustrates the relative expression level of HGBPBMY2 amongst various mRNA tissue sources. As shown, the HGBPBMY2 polypeptide showed the highest expression in tonsil, foreskin, esophagus, and was detectable in uterus cervix, tertiary lung bronchus and trachea.  FIG. 44  also illustrates the relative expression level of HGBPBMY2 amongst various mRNA tissue sources isolated from normal and diseased tissues. Expression data was obtained by measuring the steady state HGBPBMY2 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:64 and 65, and TAQMAN probe (SEQ ID NO:66) as described in Example 37 herein. These data support a role of HGBPBMY2 in regulating various functions, including immune system and pulmonary functions, and thus small molecule modulators of HGBPBMY2 function may represent a novel therapeutic option in the treatment of lung cancers, as well as other pulmonary diseases, and in the treatment of immune system diseases. \\n     Features of the Polypeptide Encoded by Gene No. 3 \\n     A polypeptide encoded by this gene, HGBPBMY3 (4843 30 2 1; 4843 — 2), is provided as SEQ ID NO:7 ( FIGS. 3A–3B ) and is encoded by the polynucleotide sequence according to SEQ ID NO:6 ( FIGS. 3A–3B ) and/or by a polynucleotide contained within the deposited clone. HGBPBMY3 (4843 30 2 1; 4843 — 2) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, human GBP-1, human GBP-2, human GBP-3 and human GBP-4. \\n     The determined nucleotide sequence of the HGBPBMY3 (4843 30 2 1; 4843 — 2), (i.e., the cDNA shown in  FIGS. 3A–3B  and in SEQ ID NO:6) comprises an open reading frame encoding a protein of about 464 amino acid residues. The predicted amino acid sequence of the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide is shown in  FIGS. 3A–3B  (SEQ ID NO:7). The percent identity and similarity values between the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide to the known GBP family member hGBP1 is provided in  FIG. 9 . The HGBPBMY3 (4843 30 2 1; 4843 — 2) protein shown in  FIGS. 3A–3B  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide showed expression predominately in spleen and lymph node; expression was also observed in heart, placenta, lung, liver, thymus and tonsil (see  FIG. 17 ). \\n     Based upon the strong homology to members of the GBP family members, the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide is expected to share at least some biological activity with GBP family members, specifically hGBP-1, hGBP-2, hGBP3 and hGBP-4. \\n     The HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system, the reproductive system, heart, lung, and liver. \\n     The HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue, preferably human tissue. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with significant HGBPBMY3 (4843 30 2 1; 4843 — 2) expression in liver tissue suggests a potential utility for HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing liver diseases. In representative embodiments, HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing liver diseases. For example, the HGBPBMY3 (4843 30 2 1; 4843 — 2) protein may be useful for the detection, treatment, amelioration, and/or prevention of hepatoblastoma, jaundice, hepatitis, liver metabolic diseases and conditions that are attributable to the differentiation of hepatocyte progenitor cells, cirrhosis, hepatic cysts, pyrogenic abscess, amebic abcess, hydatid cyst, cystadenocarcinoma, adenoma, focal nodular hyperplasia, hemangioma, hepatocellulae carcinoma, cholangiocarcinoma, and angiosarcoma, granulomatous liver disease, liver transplantation, hyperbilirubinemia, jaundice, parenchymal liver disease, portal hypertension, hepatobiliary disease, hepatic parenchyma, hepatic fibrosis, anemia, gallstones, cholestasis, carbon tetrachloride toxicity, beryllium toxicity, vinyl chloride toxicity, choledocholithiasis, hepatocellular necrosis, aberrant metabolism of amino acids, aberrant metabolism of carbohydrates, aberrant synthesis proteins, aberrant synthesis of glycoproteins, aberrant degradation of proteins, aberrant degradation of glycoproteins, aberrant metabolism of drugs, aberrant metabolism of hormones, aberrant degradation of drugs, aberrant degradation of drugs, aberrant regulation of lipid metabolism, aberrant regulation of cholesterol metabolism, aberrant glycogenesis, aberrant glycogenolysis, aberrant glycolysis, aberrant gluconeogenesis, hyperglycemia, glucose intolerance, hyperglycemia, decreased hepatic glucose uptake, decreased hepatic glycogen synthesis, hepatic resistance to insulin, portal-systemic glucose shunting, peripheral insulin resistance, hormonal abnormalities, increased levels of systemic glucagon, decreased levels of systemic cortisol, increased levels of systemic insulin, hypoglycemia, decreased gluconeogenesis, decreased hepatic glycogen content, hepatic resistance to glucagon, elevated levels of systemic aromatic amino acids, decreased levels of systemic branched-chain amino acids, hepatic encephalopathy, aberrant hepatic amino acid transamination, aberrant hepatic amino acid oxidative deamination, aberrant ammonia synthesis, aberant albumin secretion, hypoalbuminemia, aberrant cytochromes b5 function, aberrant P450 function, aberrant glutathione S-acyltransferase function, aberrant cholesterol synthesis, and aberrant bile acid synthesis. \\n     Moreover, HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, hepatic infections: liver disease caused by sepsis infection, liver disease caused by bacteremia, liver disease caused by Pneomococcal pneumonia infection, liver disease caused by Toxic shock syndrome, liver disease caused by Listeriosis, liver disease caused by Legionnaries&#39; disease, liver disease caused by Brucellosis infection, liver disease caused by  Neisseria gonorrhoeae  infection, liver disease caused by  Yersinia  infection, liver disease caused by Salmonellosis, liver disease caused by Nocardiosis, liver disease caused by Spirochete infection, liver disease caused by  Treponema pallidum  infection, liver disease caused by  Brrelia burgdorferi  infection, liver disease caused by Leptospirosis, liver disease caused by  Coxiella burnetii  infection, liver disease caused by  Rickettsia richettsii  infection, liver disease caused by  Chlamydia trachomatis  infection, liver disease caused by  Chlamydia psittaci  infection, liver disease caused by hepatitis virus infection, liver disease caused by Epstein-Barr virus infection in addition to any other hepatic disease and/or disorder implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with the high level of HGBPBMY3 (4843 30 2 1; 4843 — 2) expression in some immune system tissues (i.e., spleen, lymph node, thymus and/or tonsil) suggests that HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. In representative embodiments, HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. The HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. The HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide of the present invention can be useful in the treatment of include various autoimmune diseases such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide of the present invention can also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases. Such diseases and conditions include Recombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to human GBP family members, particularly GBP-1, GBP-2, GBP-3 and GBP-4, combined with HGBPBMY3 (4843 30 2 1; 4843 — 2) expression in heart tissue suggests the HGBPMY3 polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, smoking, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus viridans  infection,  Staphylococcus aureus  infection, coagulase-negative staphylococci infection, gram-negative  Bacilli  infection, Enterobacteriaceae infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative coccobacilli infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous Mycobacteria infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     Likewise, the expression in lung tissue also emphasizes a potential utility for HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing pulmonary diseases and disorders which include the following, not limiting examples: ARDS, emphysema, cystic fibrosis, interstitial lung disease, chronic obstructive pulmonary disease, bronchitis, lymphangioleiomyomatosis, pneumonitis, eosinophilic pneumonias, granulomatosis, pulmonary infarction, pulmonary fibrosis, pneumoconiosis, alveolar hemorrhage, neoplasms, lung abscesses, empyema, and increased susceptibility to lung infections (e.g., immumocompromised, HIV, etc.), for example. \\n     Moreover, HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, pulmonary infections: pnemonia, bacterial pnemonia, viral pnemonia (for example, as caused by Influenza virus, Respiratory syncytial virus, Parainfluenza virus, Adenovirus, Coxsackievirus, Cytomegalovirus, Herpes simplex virus, Hantavirus, etc.), mycobacteria pnemonia (for example, as caused by  Mycobacterium tuberculosis , etc.)  mycoplasma  pnemonia, fungal pnemonia (for example, as caused by  Pneumocystis carinii, Histoplasma capsulatum, Coccidioides immitis, Blastomyces dermatitidis, Candida  sp.,  Cryptococcus neoformans, Aspergillus  sp., Zygomycetes, etc.), Legionnaires&#39; Disease,  Chlamydia  pnemonia, aspiration pnemonia,  Nocordia  sp. Infections, parasitic pnemonia (for example, as caused by  Strongyloides, Toxoplasma gondii , etc.) necrotizing pnemonia, in addition to any other pulmonary disease and/or disorder (e.g., non-pneumonia) implicated by the causative agents listed above or elsewhere herein. \\n     Additionally, the expression in placenta tissue also emphasizes a potential utility for HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing placenta disorders, in addition to reproductive disorders. Such diseases and conditions include, but are not limited to, dysfunctional uterine bleeding, amenorrhea, primary dysmenorrhea, sexual dysfunction, infertility, pelvic inflammatory disease, endometriosis, placental aromatase deficiency, premature menopause, and placental dysfunction. \\n     The HGBPBMY3 (4843 30 2 1; 4843 — 2) protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against the HGBPBMY3 (4843 30 2 1; 4843 — 2) protein, can show utility as a tumor marker and/or immunotherapy targets for heart, placenta, lung, imune system and liver tissue. \\n     The HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of HGBPBMY3 (4843 30 2 1; 4843 — 2) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the HGBPBMY3 (4843 30 2 1; 4843 — 2) protein could be used as diagnostic agents of conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptides and polynucleotides have additional uses which include diagnosing diseases related to the over and/or under expression of HGBPBMY3 (4843 30 2 1; 4843 — 2) by identifying mutations in the HGBPBMY3 (4843 30 2 1; 4843 — 2) gene by using HGBPBMY3 (4843 30 2 1; 4843 — 2) sequences as probes or by determining HGBPBMY3 (4843 30 2 1; 4843 — 2) protein or mRNA expression levels. HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptides can be useful for screening compounds that affect the activity of the protein. HGBPBMY3 (4843 30 2 1; 4843 — 2) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with HGBPBMY3 (4843 30 2 1; 4843 — 2), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the HGBPBMY3 (4843 30 2 1; 4843 — 2) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of HGBPBMY3 (4843 30 2 1; 4843 — 2), spleen, lymph node, heart, placenta, lung, liver thymus and tonsil tissue can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the HGBPBMY3 (4843 30 2 1; 4843 — 2) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of HGBPBMY3 (4843 30 2 1; 4843 — 2), a disease correlation related to HGBPBMY3 (4843 30 2 1; 4843 — 2) can be made by comparing the mRNA expression level of HGBPBMY3 (4843 30 2 1; 4843 — 2) in normal tissue, as compared to diseased tissue (for example, diseased immune system tissue). Significantly higher or lower levels of HGBPBMY3 (4843 30 2 1; 4843 — 2) expression in the diseased tissue can suggest HGBPBMY3 (4843 30 2 1; 4843 — 2) plays a role in disease progression, and antagonists against HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of HGBPBMY3 (4843 30 2 1; 4843 — 2) expression in the diseased tissue can suggest HGBPBMY3 (4843 30 2 1; 4843 — 2) plays a defensive role against disease progression, and agonists of HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:6 ( FIGS. 3A–3B ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the HGBPBMY3 (4843 30 2 1; 4843 — 2), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay could be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in mice and/or rats and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and/or homolgous recombination) and the resulting generation of transgenic mice and/or rats. Expressing a particular gene in either sense or antisense orientation in a transgenic mouse or rat could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of HGBPBMY3 (4843 30 2 1; 4843 — 2) transgenic mice or rats, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic mice or rats to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal HGBPBMY3 (4843 30 2 1; 4843 — 2) deletion polypeptides are encompassed by the present invention: M1-R464, V2-R464, S3-R464, E4-R464, I5-R464, H6-R464, M7-R464, T8-R464, G9-R464, P10-R464, M11-R464, C12-R464, L13-R464, I14-R464, E15-R464, N16-R464, T17-R464, N18-R464, G19-R464, R20-R464, L21-R464, M22-R464, A23-R464, N24-R464, P25-R464, E26-R464, A27-R464, L28-R464, K29-R464, I30-R464, L31-R464, S32-R464, A33-R464, I34-R464, T35-R464, Q36-R464, P37-R464, V38-R464, V39-R464, V40-R464, V41-R464, A42-R464, T43-R464, R44-R464, T45-R464, G46-R464, K47-R464, S48-R464, Y49-R464, L50-R464, I51-R464, N52-R464, K53-R464, L54-R464, A55-R464, Q56-R464, K57-R464, K58-R464, K59-R464, G60-R464, F61-R464, S62-R464, L63-R464, G64-R464, S65-R464, T66-R464, V67-R464, Q68-R464, S69-R464, H70-R464, T71-R464, K72-R464, G73-R464, I74-R464, W75-R464, M76-R464, W77-R464, C78-R464, M79-R464, P80-R464, H81-R464, P82-R464, K83-R464, K84-R464, P85-R464, G86-R464, H87-R464, I88-R464, L89-R464, V90-R464, L91-R464, L92-R464, D93-R464, T94-R464, E95-R464, G96-R464, L97-R464, G98-R464, D99-R464, V100-R464, E101-R464, K102-R464, G103-R464, D104-R464, N105-R464, Q106-R464, N107-R464, D108-R464, S109-R464, W110-R464, I111-R464, F112-R464, A113-R464, L114-R464, A115-R464, V116-R464, L117-R464, L118-R464, N119-R464, S120-R464, T121-R464, S122-R464, M123-R464, Y124-R464, N125-R464, S126-R464, I127-R464, G128-R464, T129-R464, I130-R464, N131-R464, Q132-R464, Q133-R464, A134-R464, M135-R464, D136-R464, Q137-R464, L138-R464, H139-R464, Y140-R464, V141-R464, T142-R464, E143-R464, L144-R464, T145-R464, H146-R464, R147-R464, V148-R464, Q149-R464, P150-R464, K151-R464, S152-R464, S153-R464, P154-R464, D155-R464, E156-R464, N157-R464, E158-R464, N159-R464, E160-R464, D161-R464, S162-R464, A163-R464, D164-R464, F165-R464, E166-R464, S167-R464, F168-R464, F169-R464, P170-R464, D171-R464, F172-R464, A173-R464, G174-R464, L175-R464, E176-R464, S177-R464, L178-R464, V179-R464, L180-R464, T181-R464, Y182-R464, V183-R464, N184-R464, A185-R464, I186-R464, S187-R464, S188-R464, G189-R464, D190-R464, L191-R464, P192-R464, C193-R464, M194-R464, E195-R464, N196-R464, A197-R464, V198-R464, L199-R464, A200-R464, L201-R464, A202-R464, Q203-R464, I204-R464, E205-R464, N206-R464, S207-R464, A208-R464, A209-R464, V210-R464, Q211-R464, K212-R464, A213-R464, I214-R464, A215-R464, H216-R464, Y217-R464, E218-R464, K219-R464, Q220-R464, M221-R464, G222-R464, Q223-R464, K224-R464, V225-R464, Q226-R464, L227-R464, P228-R464, T229-R464, E230-R464, T231-R464, L232-R464, Q233-R464, E234-R464, L235-R464, L236-R464, D237-R464, 238-R464, H239-R464, R240-R464, D214-R464, S242-R464, E243-R464, S244-R464, K245-R464, A246-R464, T247-R464, E248-R464, V249-R464, F250-R464, I251-R464, R252-R464, S253-R464, S254-R464, F255-R464, K256-R464, D257-R464, V258-R464, D259-R464, H260-R464, L261-R464, F262-R464, Q263-R464, K264-R464, E265-R464, L266-R464, A267-R464, A268-R464, Q269-R464, L270-R464, D271-R464, K272-R464, K273-R464, R274-R464, D275-R464, D276-R464, F277-R464, C278-R464, K279-R464, Q280-R464, N281-R464, Q282-R464, E283-R464, A284-R464, S285-R464, S286-R464, D287-R464, R288-R464, C289-R464, S290-R464, A291-R464, L292-R464, L293-R464, Q294-R464, V295-R464, I296-R464, F297-R464, S298-R464, P299-R464, L300-R464, E301-R464, E302-R464, E303-R464, V304-R464, K305-R464, A306-R464, G307-R464, I308-R464, Y309-R464, S310-R464, K311-R464, P312-R464, G313-R464, G314-R464, Y315-R464, R316-R464, L317-R464, F318-R464, I319-R464, Q320-R464, K321-R464, L322-R464, Q323-R464, D324-R464, L325-R464, E326-R464, K327-R464, K328-R464, Y329-R464, Y330-R464, E 331-R464, E332-R464, P333-R464, R334-R464, K335-R464, G336-R464, I337-R464, Q338-R464, G339-R464, I340-R464, S341-R464, P342-R464, P343-R464, R344-R464, T345-R464, T346-R464, G347-R464, Q348-R464, R349-R464, K350-R464, E351-R464, F352-R464, P353-R464, E354-R464, E355-R464, R356-R464, M357-R464, A358-R464, G359-R464, R360-R464, Q361-R464, T362-R464, G363-R464, T364-R464, P365-R464, A366-R464, Y367-R464, S368-R464, R369-R464, L370-R464, L371-R464, L372-R464, L373-R464, T374-R464, L375-R464, C376-R464, S377-R464, L378-R464, G379-R464, P380-R464, K381-R464, A382-R464, E383-R464, E384-R464, I385-R464, L386-R464, Q387-R464, T388-R464, Y389-R464, L390-R464, K391-R464, S392-R464, K393-R464, E394-R464, S395-R464, M396-R464, T397-R464, D398-R464, A399-R464, I400-R464, L401-R464, Q402-R464, T403-R464, D404-R464, Q405-R464, T406-R464, L407-R464, T408-R464, E409-R464, K410-R464, E411-R464, K412-R464, E413-R464, I414-R464, E415-R464, V416-R464, E417-R464, R418-R464, V419-R464, K420-R464, A421-R464, E422-R464, S423-R464, A424-R464, Q425-R464, A426-R464, S427-R464, A428-R464, K429-R464, M430-R464, L431-R464, Q432-R464, Q433-R464, M434-R464, Q435-R464, R436-R464, K437-R464, N438-R464, E439-R464, Q440-R464, M441-R464, M442-R464, E443-R464, Q444-R464, K445-R464, E446-R464, R447-R464, S448-R464, Y449-R464, Q450-R464, E451-R464, H452-R464, L453-R464, K454-R464, Q455-R464, L456-R464, T457-R464, E458-R464, K459-R464, M460-R464, E461-R464, S462-R464 and/or D463-R464 of SEQ ID NO:7. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal HGBPBMY3 (4843 30 2 1; 4843 — 2) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal HGBPBMY3 (4843 30 2 1; 4843 — 2) deletion polypeptides are encompassed by the present invention: M1-V2, M1-S3, M1-E4, M1-I5, M1-H6, M1-M7, M1-T8, M1-G9, M1-P10, M1-M11, M1-C12, M1-L13, M1-I14, M1-E15, M1-N16, M1-T17, M1-N18, M1-G19, M1-R20, M1-L21, M1-M22, M1-A23, M1-N24, M1-P25, M1-E26, M1-A27, M1-L28, M1-K29, M1-I30, M1-L31, M1-S32, M1-A33, M1-I34, M1-T35, M1-Q36, M1-P37, M1-V38, M1-V39, M1-V40, M1-V41, M1-A42, M1-T43, M1-R44, M1-T45, M1-G46, M1-K47, M1-S48, M1-Y49, M1-L50, M1-151, M1-N52, M1-K53, M1-L54, M1-A55, M1-Q56, M1-K57, M1-K58, M1-K59, M1-G60, M1-F61, M1-S62, M1-L63, M1-G64, M1-S65, M1-T66, M1-V67, M1-Q68, M1-S69, M1-H70, M1-T71, M1-K72, M1-G73, M1-I74, M1-W75, M1-M76, M1-W77, M1-C78, M1-M79, M1-P80, M1-H81, M1-P82, M1-K83, M1-K84, M1-P85, M1-G86, M1-H87, M1-188, M1-L89, M1-V90, M1-L91, M1-L92, M1-D93, M1-T94, M1-E95, M1-G96, M1-L97, M1-G98, M1-D99, M1-V100, M1-E101, M1-K102, M1-G103, M1-D104, M1-N105, M1-Q106, M1-N107, M1-D108, M1-S109, M1-W110, M1-I111, M1-F112, M1-A113, M1-L114, M1-A115, M1-V116, M1-L117, M1-L118, M1-N119, M1-S120, M1-T121, M1-S122, M1-M123, M1-Y124, M1-N125, M1-S126, M1-1127, M1-G128, M1-T129, M1-I130, M1-N131, M1-Q132, M1-Q133, M1-A134, M1-M135, M1-D136, M1-Q137, M1-L138, M1-H139, M1-Y140, M1-V141, M1-T142, M1-E143, M1-L144, M1-T145, M1-H146, M1-R147, M1-V148, M1-Q149, M1-P150, M1-K151, M1-S152, M1-S153, M1-P154, M1-D155, M1-E156, M1-N157, M1-E158, M1-N159, M1-E160, M1-D161, M1-S162, M1-A163, M1-D164, M1-F165, M1-E166, M1-S167, M1-F168, M1-F169, M1-P170, M1-D171, M1-F172, M1-A173, M1-G174, M1-L175, M1-E176, M1-S177, M1-L178, M1-V179, M1-L180, M1-T181, M1-Y182, M1-V183, M1-N184, M1-A185, M1-I186, M1-S187, M1-S188, M1-G189, M1-D190, M1-L191, M1-P192, M1-C193, M1-M194, M1-E195, M1-N196, M1-A197, M1-V198, M1-L199, M1-A200, M1-L201, M1-A202, M1-Q203, M1-1204, M1-E205, M1-N206, M1-S207, M1-A208, M1-A209, M1-V210, M1-Q211, M1-K212, M1-A213, M1-I214, M1-A215, M1-H216, M1-Y217, M1-E218, M1-K219, M1-Q220, M1-M221, M1-G222, M1-Q223, M1-224, M1-V225, M1-Q226, M1-L227, M1-P228, M1-T229, M1-E230, M1-T231, M1-L232, M1-Q233, M1-E234, M1-L235, M1-L236, M1-D237, M1-L238, M1-H239, M1-R240, M1-D241, M1-S242, M1-E243, M1-S244, M1-K245, M1-A246, M1-T247, M1-E248, M1-V249, M1-F250, M1-I251, M1-R252, M1-S253, M1-S254, M1-F255, M1-K256, M1-D257, M1-V258, M1-D259, M1-H260, M1-L261, M1-F262, M1-Q263, M1-K264, M1-E265, M1-L266, M1-A267, M1-A268, M1-Q269, M1-L270, M1-D271, M1-K272, M1-K273, M1-R274, M1-D275, M1-D276, M1-F277, M1-C278, M1-K279, M1-Q280, M1-N281, M1-Q282, M1-E283, M1-A284, M1-S285, M1-S286, M1-D287, M1-R288, M1-C289, M1-S290, M1-A291, M1-L292, M1-L293, M1-Q294, M1-V295, M1-I296, M1-F297, M1-S298, M1-P299, M1-L300, M1-E301, M1-E302, M1-E303, M1-V304, M1-K305, M1-A306, M1-G307, M1-I308, M1-Y309, M1-S310, M1-K311, M1-P312, M1-G313, M1-G314, M1-Y315, M1-R316, M1-L317, M1-F318, M1-I319, M1-Q320, M1-K321, M1-L322, M1-Q323, M1-D324, M1-L325, M1-E326, M1-K327, M1-K328, M1-Y329, M1-Y330, M1-E331, M1-E332, M1-P333, M1-R334, M1-K335, M1-G336, M1-I337, M1-Q338, M1-G339, M1-I340, M1-S341, M1-P342, M1-P343, M1-R344, M1-T345, M1-T346, M1-G347, M1-Q348, M1-R349, M1-K350, M1-E351, M1-F352, M1-P353, M1-E354, M1-E355, M1-R356, M1-M357, M1-A358, M1-G359, M1-R360, M1-Q361, M1-T362, M1-G363, M1-T364, M1-P365, M1-A366, M1-Y367, M1-S368, M1-R369, M1-L370, M1-L371, M1-L372, M1-L373, M1-T374, M1-L375, M1-C376, M1-S377, M1-L378, M1-G379, M1-P380, M1-K381, M1-A382, M1-E383, M1-E384, M1-I385, M1-L386, M1-Q387, M1-T388, M1-Y389, M1-L390, M1-K391, M1-S392, M1-K393, M1-E394, M1-S395, M1-M396, M1-T397, M1-D398, M1-A399, M1-I400, M1-L401, M1-Q402, M1-T403, M1-D404, M1-Q405, M1-T406, M1-L407, M1-T408, M1-409, M1-K410. M1-E411, M1-K412, M1-E413, M1-I414, M1-E415, M1-V416, M1-E417, M1-R418, M1-V419, M1-K420, M1-A421, M1-E422, M1-S423, M1-A424, M1-Q425, M1-A426, M1-S427, M1-A428, M1-K429, M1-M430, M1-L431, M1-Q432, M1-Q433, M1-M434, M1-Q435, M1-R436, M1-K437, M1-N438, M1-E439, M1-Q440, M1-M441, M1-M442, M1-E443, M1-Q444, M1-K445, M1-E446, M1-R447, M1-S448, M1-Y449, M1-Q450, M1-E451, M1-H452, M1-L453, M1-K454, M1-Q455, M1-L456, M1-T457, M1-E458, M1-K459, M1-M460, M1-E461, M1-S462, M1-D463 and/or M1-R464 of SEQ ID NO:7. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal HGBPBMY3 (4843 30 2 1; 4843 — 2) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide (e.g., any combination of both N- and C-terminal HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide deletions) of SEQ ID NO:7. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of HGBPBMY3 (4843 30 2 1; 4843 — 2) (SEQ ID NO:7), and where CX refers to any C-terminal deletion polypeptide amino acid of HGBPBMY3 (4843 30 2 1; 4843 — 2) (SEQ ID NO:7). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate HGBPBMY3 (4843 30 2 1; 4843 — 2) on the one hand (i.e., agonists) and which inhibit the function of HGBPBMY3 (4843 30 2 1; 4843 — 2) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the HGBPBMY3 (4843 30 2 1; 4843 — 2) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n       FIG. 45  shows an expanded expression profile of the GBP HGBPBMY3 (4843 — 2). The figure illustrates the relative expression level of HGBPBMY3 amongst various mRNA tissue sources.  FIG. 45  illustrates the relative expression level of HGBPBMY3 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY3 polypeptide showed increased expression in thyroids of hyperthyroidism patients compared to control thyroids.  FIG. 45  also illustrates increased expression in breast and testicle tumors relative to controls; increased expression in prostatic hypertrophy relative to normal prostate; expressed in normal and diseased lung parenchyma; increased in bronchitis. Expression data was obtained by measuring the steady state HGBPBMY3 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:67 and 68, and TAQMAN probe (SEQ ID NO:69) as described in Example 38 herein. These data support a role of HGBPBMY3 in regulating various functions. HGBPBMY3 may also be participating in the formation of testicle and breast turmors and thus small molecule modulators of HGBPBMY3 function may represent a novel therapeutic option in the treatment of breast and testicle cancers, as well as diseases of the prostate. \\n     Features of the Polypeptide Encoded by Gene No. 4 \\n     A polypeptide encoded by this gene, HGBPBMY4 (FLJ10961), is provided as SEQ ID NO:9 ( FIGS. 4A–4C ) and is encoded by the polynucleotide sequence according to SEQ ID NO:8 ( FIGS. 4A–4C ) and/or by a polynucleotide contained within the deposited clone. HGBPBMY4 (FLJ10961) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, human GBP-1, human GBP-2, human GBP-3 and human GBP-4. \\n     The determined nucleotide sequence of the HGBPBMY4 (FLJ10961), (i.e. the cDNA shown in  FIGS. 4A–4C  and in SEQ ID NO:8) comprises an open reading frame encoding a protein of about 563 amino acid residues. The predicted amino acid sequence of the HGBPBMY4 (FLJ10961) polypeptide is shown in  FIGS. 4A–4C  (SEQ ID NO:9). The percent identity and similarity values between the HGBPBMY4 (FLJ10961) polypeptide to the known GBP family member hGBP1 is provided in  FIG. 9 . The HGBPBMY4 (FLJ10961) protein shown in  FIGS. 4A–4C  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the HGBPBMY4 (FLJ10961) polypeptide showed expression in spleen, lymph node, tonsil, leukocyte, placenta and lung, as well as expression in heart, liver, skeletal muscle, kidney and thymus; expression in bone marrow and pancreas was also observed (see  FIG. 15 ). \\n     Based upon the strong homology to members of the GBP family members, the HGBPBMY4 (FLJ10961) polypeptide is expected to share at least some biological activity with GBP family members, specifically hGBP-1, hGBP-2, hGBP3 and hGBP-4. \\n     The HGBPBMY4 (FLJ10961) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, diagnosing, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system (e.g., spleen, lymph node, tonsil, thymus), the pulmonary system (e.g., lung), the reproductive system (e.g., placenta), renal system (e.g., kidney), skeletal muscle, liver, cardiovascular system (e.g., heart), pancreas and hematopoesis (e.g., bone marrow). \\n     The HGBPBMY4 (FLJ10961) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue, preferably human tissue. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with HGBPBMY4 (FLJ10961) expression in liver tissue suggests HGBPBMY4 (FLJ10961) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing liver diseases. In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing liver diseases. For example, the HGBPBMY4 (FLJ10961) protein can be used for the detection, treatment, amelioration, and/or prevention of hepatoblastoma, jaundice, hepatitis, liver metabolic diseases and conditions that are attributable to the differentiation of hepatocyte progenitor cells, cirrhosis, hepatic cysts, pyrogenic abscess, amebic abcess, hydatid cyst, cystadenocarcinoma, adenoma, focal nodular hyperplasia, hemangioma, hepatocellulae carcinoma, cholangiocarcinoma, and angiosarcoma, granulomatous liver disease, liver transplantation, hyperbilirubinemia, jaundice, parenchymal liver disease, portal hypertension, hepatobiliary disease, hepatic parenchyma, hepatic fibrosis, anemia, gallstones, cholestasis, carbon tetrachloride toxicity, beryllium toxicity, vinyl chloride toxicity, choledocholithiasis, hepatocellular necrosis, aberrant metabolism of amino acids, aberrant metabolism of carbohydrates, aberrant synthesis proteins, aberrant synthesis of glycoproteins, aberrant degradation of proteins, aberrant degradation of glycoproteins, aberrant metabolism of drugs, aberrant metabolism of hormones, aberrant degradation of drugs, aberrant degradation of drugs, aberrant regulation of lipid metabolism, aberrant regulation of cholesterol metabolism, aberrant glycogenesis, aberrant glycogenolysis, aberrant glycolysis, aberrant gluconeogenesis, hyperglycemia, glucose intolerance, hyperglycemia, decreased hepatic glucose uptake, decreased hepatic glycogen synthesis, hepatic resistance to insulin, portal-systemic glucose shunting, peripheral insulin resistance, hormonal abnormalities, increased levels of systemic glucagon, decreased levels of systemic cortisol, increased levels of systemic insulin, hypoglycemia, decreased gluconeogenesis, decreased hepatic glycogen content, hepatic resistance to glucagon, elevated levels of systemic aromatic amino acids, decreased levels of systemic branched-chain amino acids, hepatic encephalopathy, aberrant hepatic amino acid transamination, aberrant hepatic amino acid oxidative deamination, aberrant ammonia synthesis, aberant albumin secretion, hypoalbuminemia, aberrant cytochromes b5 function, aberrant P450 function, aberrant glutathione S-acyltransferase function, aberrant cholesterol synthesis, and aberrant bile acid synthesis. \\n     Moreover, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, hepatic infections: liver disease caused by sepsis infection, liver disease caused by bacteremia, liver disease caused by Pneomococcal pneumonia infection, liver disease caused by Toxic shock syndrome, liver disease caused by Listeriosis, liver disease caused by Legionnaries&#39; disease, liver disease caused by Brucellosis infection, liver disease caused by  Neisseria gonorrhoeae  infection, liver disease caused by  Yersinia  infection, liver disease caused by Salmonellosis, liver disease caused by Nocardiosis, liver disease caused by Spirochete infection, liver disease caused by  Treponema pallidum  infection, liver disease caused by  Brrelia burgdorferi  infection, liver disease caused by Leptospirosis, liver disease caused by  Coxiella burnetii  infection, liver disease caused by  Rickettsia richettsii  infection, liver disease caused by  Chlamydia trachomatis  infection, liver disease caused by  Chlamydia psittaci  infection, liver disease caused by hepatitis virus infection, liver disease caused by Epstein-Barr virus infection in addition to any other hepatic disease and/or disorder implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with HGBPBMY4 (FLJ10961) expression in some immune system tissues (i.e., spleen, lymph node, thymus and tonsil) suggests a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system. The HGBPBMY4 (FLJ10961) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. The HGBPBMY4 (FLJ10961) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunological disorders that a HGBPBMY4 (FLJ10961) polypeptide of the present invention may be useful in the treatment of include various autoimmune diseases such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A HGBPBMY4 (FLJ10961) polypeptide of the present invention may also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases. Such diseases and conditions include Recombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to human GBP family members, particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4, combined with HGBPBMY4 (FLJ10961) expression levels in heart tissue suggests the HGBPMY4 polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, smoking, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus viridans  infection,  Staphylococcus aureus  infection, coagulase-negative  staphylococci  infection, gram-negative  Bacilli  infection, Enterobacteriaceae infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative coccobacilli infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous Mycobacteria infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     Likewise, the expression in lung tissue also emphasizes a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing pulmonary diseases and disorders which include the following, not limiting examples: ARDS, emphysema, cystic fibrosis, interstitial lung disease, chronic obstructive pulmonary disease, bronchitis, lymphangioleiomyomatosis, pneumonitis, eosinophilic pneumonias, granulomatosis, pulmonary infarction, pulmonary fibrosis, pneumoconiosis, alveolar hemorrhage, neoplasms, lung abscesses, empyema, and increased susceptibility to lung infections (e.g., immumocompromised, HIV, etc.), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which may include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, pulmonary infections: pnemonia, bacterial pnemonia, viral pnemonia (for example, as caused by Influenza virus, Respiratory syncytial virus, Parainfluenza virus, Adenovirus, Coxsackievirus, Cytomegalovirus, Herpes simplex virus, Hantavirus, etc.), mycobacteria pnemonia (for example, as caused by  Mycobacterium tuberculosis , etc.)  mycoplasma  pnemonia, fungal pnemonia (for example, as caused by  Pneumocystis carinii, Histoplasma capsulatum, Coccidioides immitis, Blastomyces dermatitidis, Candida  sp.,  Cryptococcus neoformans, Aspergillus  sp., Zygomycetes, etc.), Legionnaires&#39; Disease,  Chlamydia  pnemonia, aspiration pnemonia,  Nocordia  sp. Infections, parasitic pnemonia (for example, as caused by  Strongyloides, Toxoplasma gondii , etc.) necrotizing pnemonia, in addition to any other pulmonary disease and/or disorder (e.g., non-pneumonia) implicated by the causative agents listed above or elsewhere herein. \\n     Additionally, the expression in placenta tissue also emphasizes a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing placenta disorders, in addition to reproductive disorders. Such diseases and conditions include, but are not limited to, dysfunctional uterine bleeding, amenorrhea, primary dysmenorrhea, sexual dysfunction, infertility, pelvic inflammatory disease, endometriosis, placental aromatase deficiency, premature menopause, and placental dysfunction. \\n     Further, the expression in skeletal muscle tissue emphasizes a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing skeletal muscle disorders. In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists and fragements thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of skeletal muscled: dystrophies, pseudohypertrophic muscular dystrophy, Duchenne dystrophy, Becker muscular dystrophy, limb-girdle muscular dystrophy, muscle weakness, Emery-Dreifuss muscular dystrophy, Congenital muscular dystrophy, endometriosis, placental aromatase deficiency, premature menopause, Fukuyama congenital muscular dystrophy, laminin alpha 2 chain deficiency, alpha 7 integrin deficiency, Walker-Warburg syndrome, myotonic dystrophy, congenital myotonic dystrophy, facioscapulohumeral muscular dystrophy, distal myopathies, central core disease, nemaline (rod) myopathy, centronuclear (myotubular) myopathy, central core disease, delay in motor milestones, delayed walking, nemaline myopathy, congenital nemaline myopathy, muscle hypotonia, centronuclear myopathies, skeletal muscle energy metabolism disorders, disorders associated with aberrant skeletal muscle-fatty acid metabolism, disorders associated with aberrant skeletal glucose metabolism, acid maltase deficiency, debranching enzyme deficiency, branching enzyme deficiency, exercise intolerance, myophosphorylase deficiency (type V glycogenosis), phosphofructokinase deficiency (type VII glycogenosis), phosphoglycerate kinase deficiency (type IX glycogenosis), phosphoglycerate mutase deficiency (type X glycogenosis), lactate dehydrogenase deficiency (glycogensosis type XI), glycogen storage disorders, skeletal muscle lipid metabolism, carnitine deficiency, myoglobinuria, muscle cramping, myoadenylate deaminase deficiency, mitochondrial myopathies, Kearns-Sayre syndrome, myoclonic epilepsy, disorders of muscle membrane excitability, calcium channel disorders of muscle, sodium channel disorders of muscle, hyperkalemic periodic paralysis, paramyotonia congenita, potassium-aggravated myotonia, myotonia congenita, chloride channel disorders of muscle, thyrotoxic periodic paralysis, and/or Andersen&#39;s syndrome. \\n     Continuing, the expression of HGBPBMY4 (FLJ10961) in pancreas tissue suggests a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing pancreatic, in addition to metabolic and gastrointestinal disorders. \\n     In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which may include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of the pancreas: diabetes mellitus, diabetes, type 1 diabetes, type 2 diabetes, adult onset diabetes, indications related to islet cell transplantation, indications related to pancreatic transplantation, pancreatitis, pancreatic cancer, pancreatic exocrine insufficiency, alcohol induced pancreatitis, maldigestion of fat, maldigestion of protein, hypertriglyceridemia, vitamin b12 malabsorption, hypercalcemia, hypocalcemia, hyperglycemia, ascites, pleural effusions, abdominal pain, pancreatic necrosis, pancreatic abscess, pancreatic pseudocyst, gastrinomas, pancreatic islet cell hyperplasia, multiple endocrine neoplasia type 1 (men 1) syndrome, insulitis, amputations, diabetic neuropathy, pancreatic auto-immune disease, genetic defects of -cell function, HNF-1 aberrations (formerly MODY3), glucokinase aberrations (formerly MODY2), HNF-4 aberrations (formerly MODY1), mitochondrial DNA aberrations, genetic defects in insulin action, type a insulin resistance, leprechaunism, Rabson-Mendenhall syndrome, lipoatrophic diabetes, pancreatectomy, cystic fibrosis, hemochromatosis, fibrocalculous pancreatopathy, endocrinopathies, acromegaly, Cushing&#39;s syndrome, glucagonoma, pheochromocytoma, hyperthyroidism, somatostatinoma, aldosteronoma, drug- or chemical-induced diabetes such as from the following drugs: Vacor, Pentamdine, Nicotinic acid, Glucocorticoids, Thyroid hormone, Diazoxide, Adrenergic agonists, Thiazides, Dilantin, and Interferon, pancreatic infections, congential rubella, cytomegalovirus, uncommon forms of immune-mediated diabetes, “stiff-man” syndrome, anti-insulin receptor antibodies, in addition to other genetic syndromes sometimes associated with diabetes which include, for example, Down&#39;s syndrome, Klinefelter&#39;s syndrome, Turner&#39;s syndrome, Wolfram&#39;s syndrome, Friedrich&#39;s ataxia, Huntington&#39;s chorea, Lawrence Moon Beidel syndrome, Myotonic dystrophy, Porphyria, and Prader Willi syndrome, and/or Gestational diabetes mellitus (GDM). \\n     Further, the expression of HGBPBMY4 (FLJ-10961) in kidney tissue suggests a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the kidney. \\n     In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of the kidney: Plasma cell infiltration, Hypercalcemia, Myeloma kidney, Amyloidosis, Light chain deposition disease, Type I/II cryoglobulinemia, Immunotactoid glomerulopathy, Reduced glomerular filtration rate, Fanconi syndrome, Hyperchloremic acidosisa, Tubular or small-molecular-weight proteinuria, Polyuria, isothenuria, Hyperkalemia, Salt wasting, Nephrocalcinosis, hyperoxaluria, Cystinosis, Fabry&#39;s disease, Sjögren&#39;s Syndrome \\n     Further, the expression of HGBPBMY4 (FLJ10961) in bone marrow and leukocyte suggests a potential utility for HGBPBMY4 (FLJ10961) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing bone marrow and leukocyte diseases and disorders. \\n     In representative embodiments, HGBPBMY4 (FLJ10961) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, bone marrow and leukocyte diseases or disorders: Acute lymphocytic leukemia, acute myeloid leukemia, acute promyelocytic leukemia, chronic lymphocytic leukemia, chronic myeloid leukemia, myelodysplastic syndrome, myeloproliferative disorders, multiple myeloma, prolymphocytic, leukemia, amyloidosis, aplastic anemia, myelodysplasia, Polycythemia Vera, Fanconi&#39;s anemia, myelodysplasia syndromes, Rare aleukemic leukemia (AML), Myelodysplasia syndromes, Paroxysmal nocturnal hemoglobinuria, Myelofibrosis, Myelophthisis, Bone marrow lymphoma, Hairy cell leukemia, Systemic lupus erythematosus, Hypersplenism, Brucellosis, Sarcoidosis, Tuberculosis, Leishmaniasis, Hypocellular bone marrow±cytopenia, Q fever, Legionnaire&#39;s disease \\n     A HGBPBMY4 (FLJ10961) protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a HGBPBMY4 (FLJ10961) protein, can show utility as a tumor marker and/or immunotherapy targets for heart, placenta, lung, imune system and liver tissue. \\n     The HGBPBMY4 (FLJ10961) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of HGBPBMY4 (FLJ10961) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the HGBPBMY4 (FLJ10961) protein could be used as diagnostic agents of conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     HGBPBMY4 (FLJ10961) polypeptides and polynucleotides may have additional uses which include diagnosing diseases related to the over and/or under expression of HGBPBMY4 (FLJ10961) by identifying mutations in the HGBPBMY4 (FLJ10961) gene by using HGBPBMY4 (FLJ10961) sequences as probes or by determining HGBPBMY4 (FLJ10961) protein or mRNA expression levels. HGBPBMY4 (FLJ10961) polypeptides can be useful for screening compounds that affect the activity of the protein. HGBPBMY4 (FLJ10961) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with HGBPBMY4 (FLJ10961), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly hGBP-1, hGBP-2, hGBP-3 and hGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to HGBPBMY4 (FLJ10961) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of HGBPBMY4 (FLJ10961), spleen, lymph node, tonsil, thymus, lung, placenta, kidney, skeletal muscle, liver, heart, pancreas and bone marrow can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the HGBPBMY4 (FLJ10961) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of HGBPBMY4 (FLJ10961), a disease correlation related to HGBPBMY4 (FLJ10961) can be made by comparing the mRNA expression level of HGBPBMY4 (FLJ10961) in normal tissue, as compared to diseased tissue (for example diseased immune system tissue, such as spleen and lymph nodes). Significantly higher or lower levels of HGBPBMY4 (FLJ10961) expression in the diseased tissue can suggest HGBPBMY4 (FLJ10961) plays a role in disease progression, and antagonists against HGBPBMY4 (FLJ10961) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of HGBPBMY4 (FLJ10961) expression in the diseased tissue can suggest HGBPBMY4 (FLJ10961) plays a defensive role against disease progression, and agonists of HGBPBMY4 (FLJ10961) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:8 ( FIGS. 4A–4C ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of HGBPBMY4 (FLJ10961), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence a HGBPBMY4 (FLJ10961) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay could be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in mice and/or rats and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and homologous recombination) and the resulting generation of transgenic mice and/or rats. Expressing a particular gene in either sense or antisense orientation in a transgenic mouse or rat could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of HGBPBMY4 (FLJ10961), transgenic mice or rats, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic mice or rats to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal HGBPBMY4 (FLJ10961) deletion polypeptides are encompassed by the present invention: M1-I563, A2-I563, P3-I563, E4-I563, I5-I563, H6-I563, M7-I563, T8-I563, G9-I563, P10-I563, M11-I563, C12-I563, L13-I563, I14-I563, E15-I563, N16-I563, T17-I563, N18-I563, G19-I563, E20-I563, L21-I563, V22-I563, A23-I563, N24-I563, P25-I563, E26-I563, A27-I563, L28-I563, K29-I563, I30-I563, L31-I563, S32-I563, A33-I563, I34-I563, T35-I563, Q36-I563, P37-I563, V38-I563, V39-I563, V40-I563, V41-I563, A42-I563, I43-I563, V44-I563, G45-I563, L46-I563, Y47-I563, R48-I563, T49-I563, G50-I563, K51-I563, S52-I563, Y53-I563, L54-I563, M55-I563, N56-I563, K57-I563, L58-I563, A59-I563, G60-I563, K61-I563, N62-I563, K63-I563, G64-I563, F65-I563, S66-I563, L67-I563, G68-I563, S69-I563, T70-I563, V71-I563, K72-I563, S73-I563, H74-I563, T75-I563, K76-I563, G77-I563, I78-I563, W79-I563, M80-I563, W81-I563, C82-I563, V83-I563, P84-I563, H85-I563, P86-I563, K87-I563, K88-I563, P89-I563, E90-I563, H91-I563, T92-I563, L93-I563, V94-I563, L95-I563, L96-I563, D97-I563, T98-I563, E99-I563, G100-I563, L101-I563, G102-I563, D103-I563, V104-I563, K105-I563, K106-I563, G107-I563, D108-I563, N109-I563, Q110-I563, N111-I563, D112-I563, S113-I563, W114-I563, I115-I563, F116-I563, T117-I563, L118-I563, A119-I563, V120-I563, L121-I563, L122-I563, S123-I563, S124-I563, T125-I563, L126-I563, V127-I563, Y128-I563, N129-I563, S130-I563, M131-I563, G132-I563, T133-I563, I134-I563, N135-I563, Q136-I563, Q137-I563, A138-I563, M139-I563, D140-I563, Q141-I563, L142-I563, Y143-I563, Y144-I563, V145-I563, T146-I563, E147-I563, L148-I563, T149-I563, H150-I563, R151-I563, I152-I563, R153-I563, S154-I563, K155-I563, S156-I563, S157-I563, P158-I563, D159-I563, E160-I563, N161-I563, E162-I563, N163-I563, E164-I563, D165-I563, S665-I563, A167-I563, D168-I563, F169-I563, V170-I563, S171-I563, F172-I563, F173-I563, P174-I563, D175-I563, F176-I563, V177-I563, W178-I563, T179-I563, L180-I563, R181-I563, D182-I563, F183-I563, S184-I563, L185-I563, D186-I563, L187-I563, E188-I563, A189-I563, D190-I563, G191-I563, Q192-I563, P193-I563, L194-I563, T195-I563, P196-I563, D197-I563, E198-I563, Y199-I563, L200-I563, E201-I563, Y202-I563, S203-I563, L204-I563, K205-I563, L206-I563, T207-I563, Q208-I563, G209-I563, N210-I563, R211-I563, K212-I563, L213-I563, A214-I563, Q215-I563, L216-I563, E217-I563, K218-I563, L219-I563, Q220-I563, D221-I563, E222-I563, E223-I563, L224-I563, D225-I563, P226-I563, E227-I563, F228-I563, V229-I563, Q230-I563, Q231-I563, V232-I563, A233-I563, D234-I563, F235-I563, C236-I563, S237-I563, Y238-I563, I239-I563, F240-I563, S241-I563, N242-I563, S243-I563, K244-I563, T245-I563, K246-I563, T247-I563, L248-I563, S249-I563, G250-I563, G251-I563, I252-I563, K253-I563, V254-I563, N255-I563, G256-I563, P257-I563, C258-I563, L259-I563, E260-I563, S261-I563, L262-I563, V263-I563, L264-I563, T265-I563, Y266-I563, I267-I563, N268-I563, A269-I563, I270-I563, S271-I563, R272-I563, G273-I563, D274-I563, L275-I563, P276-I563, C277-I563, M278-I563, E279-I563, N280-I563, A281-I563, V282-I563, L283-I563, A284-I563, L285-I563, A286-I563, Q287-I563, I288-I563, E289-I563, N290-I563, S291-I563, A292-I563, A293-I563, V294-I563, Q295-I563, K296-I563, A297-I563, I298-I563, A299-I563, H300-I563, Y301-I563, D302-I563, Q303-I563, Q304-I563, M305-I563, G306-I563, Q307-I563, K308-I563, V309-I563, Q310-I563, L311-I563, P312-I563, A313-I563, E314-I563, T315-I563, L316-I563, Q317-I563, E318-I563, L319-I563, L320-I563, D321-I563, L322-I563, H323-I563, R324-I563, V325-I563, S325-I563, E326-I563, R327-I563, E328-I563, A329-I563, T330-I563, E331-I563, V332-I563, Y334-I563, M335-I563, K336-I563, N337-I563, S338-I563, F339-I563, K340-I563, D341-I563, V342-I563, D343-I563, H344-I563, L345-I563, F346-I563, Q347-I563, K348-I563, K349-I563, L350-I563, A351-I563, A352-I563, Q353-I563, L354-I563, D355-I563, K356-I563, K357-I563, R358-I563, D359-I563, D360-I563, F361-I563, C362-I563, K363-I563, Q364-I563, N365-I563, Q366-I563, E367-I563, A368-I563, S369-I563, S370-I563, D371-I563, R372-I563, C373-I563, S374-I563, A375-I563, L376-I563, L377-I563, Q378-I563, V379-I563, I380-I563, F381-I563, S382-I563, P383-I563, L384-I563, E385-I563, E386-I563, E387-I563, V388-I563, K389-I563, A390-I563, G391-I563, I392-I563, Y393-I563, S394-I563, K395-I563, P396-I563, G397-I563, G398-I563, Y399-I563, C400-I563, L401-I563, F402-I563, I403-I563, Q404-I563, K405-I563, L406-I563, Q407-I563, D408-I563, L409-I563, E410-I563, K411-I563, K412-I563, Y413-I563, Y414-I563, E415-I563, E416-I563, P417-I563, R418-I563, K419-I563, G420-I563, I421-I563, Q422-I563, A423-I563, E424-I563, E425-I563, I426-I563, L427-I563, Q428-I563, T429-I563, Y430-I563, L431-I563, K432-I563, S433-I563, K434-I563, E435-I563, S436-I563, V437-I563, T438-I563, D439-I563, A440-I563, I441-I563, L442-I563, Q443-I563, T444-I563, D445-I563, Q446-I563, I447-I563, L448-I563, T449-I563, E450-I563, K451-I563, E452-I563, K453-I563, E454-I563, I455-I563, E456-I563, V457-I563, E458-I563, C459-I563, V460-I563, K461-I563, A462-I563, E463-I563, S464-I563, A465-I563, Q466-I563, A467-I563, S468-I563, A469-I563, K470-I563, M471-I563, V472-I563, E473-I563, E474-I563, M475-I563, Q476-I563, I477-I563, K478-I563, Y479-I563, Q480-I563, Q481-I563, M482-I563, M483-I563, E484-I563, E485-I563, K486-I563, E487-I563, K488-I563, S489-I563, Y490-I563, Q491-I563, E492-I563, H493-I563, V494-I563, K495-I563, Q496-I563, L497-I563, T498-I563, E499-I563, K500-I563, M501-I563, E502-I563, R503-I563, E504-I563, R505-I563, A506-I563, Q507-I563, L508-I563, L509-I563, E510-I563, E511-I563, Q512-I563, E513-I563, K514-I563, T515-I563, L516-I563, T517-I563, S518-I563, K519-I563, L520-I563, Q521-I563, E522-I563, Q523-I563, A524-I563, R525-I563, V526-I563, L527-I563, K528-I563, E529-I563, R530-I563, C531-I563, Q532-I563, G533-I563, E534-I563, S535-I563, T536-I563, Q537-I563, L538-I563, Q539-I563, N540-I563, E541-I563, I542-I563, Q543-I563, K544-I563, L545-I563, Q546-I563, K547-I563, T548-I563, L549-I563, K550-I563, K551-I563, K552-I563, T553-I563, K554-I563, R555-I563, Y556-I563, M557-I563, S558-I563, H559-I563, K560-I563, L561-I563 and/or K562-I563 of SEQ ID NO:9. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal HGBPBMY4 (FLJ10961) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal HGBPBMY4 (FLJ10961) deletion polypeptides are encompassed by the present invention: M1-A2, M1-P3, M1-E4, M1-I5, M1-H6, M1-M7, M1-T8, M1-G9, M1-P10, M1-M 1, M1-C12, M1-L13, M1-I14, M1-E15, M1-N16, M1-T17, M1-N18, M1-G19, M1-E20, M1-L21, M1-V22, M1-A23, M1-N24, M1-P25, M1-E26, M1-A27, M1-L28, M1-K29, M1-I30, M1-L31, M1-S32, M1-A33, M1-I34, M1-T35, M1-Q36, M1-P37, M1-V38, M1-V39 M1-V40, M1-V41, M1-A42, M1-I43, M1-V44, M1-G45, M1-L46, M1-Y47, M1-R48, M1-T49, M1-G50, M1-K51, M1-S52, M1-Y53, M1-L54, M1-M55, M1-N56, M1-K57, M1-L58, M1-A59, M1-G60, M1-K61, M1-N62, M1-K63, M1-G64, M1-F65, M1-S66, M1-L67, M1-G68, M1-S69, M1-T70, M1-V71, M1-K72, M1-S73, M1-H74, M1-T75, M1-K76, M1-G77, M1-I78, M1-W79, M1-M80, M1-W81, M1-C82, M1-V83, M1-P84, M1-H85, M1-P86, M1-K87, M1-K88, M1-P89, M1-E90, M1-H91, M1-T92, M1-L93, M1-V94, M1-L95, M1-L96, M1-D97, M1-T98, M1-E99, M1-G100, M1-L101, M1-G102, M1-D103, M1-V104, M1-K105, M1-K106, M1-G107, M1-D108, M1-N109, M1-Q110, M1-N111, M1-D112, M1-S113, M1-W114, M1-I115, M1-F116, M1-T117, M1-L118, M1-A119, M1-V120, M1-L121, M1-L122, M1-S123, M1-S124, M1-T125, M1-L126, M1-V127, M1-Y128, M1-N129, M1-S130, M1-M131, M1-G132, M1-T133, M1-I134, M1-N135, M1-Q136, M1-Q137, M1-A138, M1-M139, M1-D140, M1-Q141, M1-L142, M1-Y143, M1-Y144, M1-V145, M1-T146, M1-E147, M1-L148, M1-T149, M1-H150, M1-R151, M1-I152, M1-R153, M1-S154, M1-K155, M1-S156, M1-S157, M1-P158, M1-D159, M1-E160, M1-N161, M1-E162, M1-N163, M1-E164, M1-D165, M1-S166, M1-A167, M1-D168, M1-F169, M1-V170, M1-S171, M1-F172, M1-F173, M1-P174 M1-D175, M1-F176, M1-V177, M1-W178, M1-T179, M1-L180, M1-R181, M1-D182, M1-F183, M1-S184, M1-L185, M1-D186, M1-L187, M1-E188, M1-A189, M1-D190, M1-G191, M1-Q192, M1-P193, M1-L194, M1-T195, M1-P196, M1-D197, M1-E198, M1-Y199, M1-L200, M1-E201, M1-Y202, M1-S203, M1-L204, M1-K205, M1-L206, M1-T207, M1-Q208, M1-G209, M1-N210, M1-R211, M1-K212, M1-L213, M1-A214, M1-Q215, M1-L216, M1-E217, M1-K218, M1-L219, M1-Q220, M1-D221, M1-E222, M1-E223, M1-L224, M1-D225, M1-P226, M1-E227, M1-F228, M1-V229, M1-Q230, M1-Q231, M1-V232, M1-A233, M1-D234, M1-F235, M1-C236, M1-S237, M1-Y238, M1-I239, M1-F240, M1-S241, M1-N242, M1-S243, M1-K244, M1-T245, M1-K246, M1-T247, M1-L248, M1-S249, M1-G250, M1-G251, M1-I252, M1-K253, M1-V254, M1-N255, M1-G256, M1-P257, M1-C258, M1-L259, M1-E260, M1-S261, M1-L262, M1-V263, M1-L264, M1-T265, M1-Y266, M1-I267, M1-N268, M1-A269, M1-I270, M1-S271, M1-R272, M1-G273, M1-D274, M1-L275, M1-P276, M1-C277, M1-M278, M1-E279, M1-N280, M1-A281, M1-V282, M1-L283, M1-A284, M1-L285, M1-A286, M1-Q287, M1-I288, M1-E289, M1-N290, M1-S291, M1-A292, M1-A293, M1-V294, M1-Q295, M1-K296, M1-A297, M1-I298, M1-A299, M1-H300, M1-Y301, M1-D302, M1-Q303, M1-Q304, M1-M305, M1-G306, M1-Q307, M1-K308, M1-V309, M1-Q310, M1-L311, M1-P312, M1-A313, M1-E314, M1-T315, M1-L316, M1-Q317, M1-E318, M1-L319, M1-L320, M1-D321, M1-L322, M1-H323, M1-R324, M1-V325, M1-S326, M1-E327, M1-R328, M1-E329, M1-A330, M1-T331, M1-E332, M1-V333, M1-Y334, M1-M335, M1-K336, M1-N337, M1-S338, M1-F339, M1-K340, M1-D341, M1-V342, M1-D343, M1-H344, M1-L345, M1-F346, M1-Q347, M1-K348, M1-K349, M1-L350, M1-A351, M1-A352, M1-Q353, M1-L354, M1-D355, M1-K356, M1-K357, M1-R358, M1-D359, M1-D360, M1-F361, M1-C362, M1-K363, M1-Q364, M1-N365, M1-Q366, M1-E367, M1-A368, M1-S369, M1-S370, M1-D371, M1-R372, M1-C373, M1-S374, M1-A375, M1-L376, M1-L377, M1-Q378, M1-V379, M1-I380, M1-F381, M1-S382, M1-P383, M1-L384, M1-E385, M1-E386, M1-E387, M1-V388, M1-K389, M1-A390, M1-G391, M1-I392, M1-Y393, M1-S394, M1-K395, M1-P396, M1-G397, M1-G398, M1-Y399, M1-C400, M1-L401, M1-F402, M1-I403, M1-Q404, M1-K405, M1-L406, M1-Q407, M1-D408, M1-L409, M1-E410, M1-K411, M1-K412, M1-Y413, M1-Y414, M1-E415, M1-E416, M1-P417, M1-R418, M1-K419, M1-G420, M1-I421, M1-Q422, M1-A423, M1-E424, M1-E425, M1-I426, M1-L427, M1-Q428, M1-T429, M1-Y430, M1-L431, M1-K432, M1-S433, M1-K434, M1-E435, M1-S436, M1-V437, M1-T438, M1-D439, M1-A440, M1-I441, M1-L442, M1-Q443, M1-T444, M1-D445, M1-Q446, M1-I447, M1-L448, M1-T449, M1-E450, M1-K451, M1-E452, M1-K453, M1-E454, M1-I455, M1-E456, M1-V457, M1-E458, M1-C459, M1-V460, M1-K461, M1-A462, M1-E463, M1-S464, M1-A465, M1-Q466, M1-A467, M1-S468, M1-A469, M1-K470, M1-M471, M1-V472, M1-E473, M1-E474, M1-M475, M1-Q476, M1-I477, M1-K478, M1-Y479, M1-Q480, M1-Q481, M1-M482, M1-M483, M1-E484, M1-E485, M1-K486, M1-E487, M1-K488, M1-S489, M1-Y490, M1-Q491, M1-E492, M1-H493, M1-V494, M1-K495, M1-Q496, M1-L497, M1-T498, M1-E499, M1-K500, M1-M501, M1-E502, M1-R503, M1-E504, M1-R505, M1-A506, M1-Q507, M1-L508, M1-L509, M1-E510, M1-E511, M1-Q512, M1-E513, M1-K514, M1-T515, M1-L516, M1-T517, M1-S518, M1-K519, M1-L520, M1-Q521, M1-E522, M1-Q523, M1-A542, M1-R525, M1-V526, M1-L527, M1-K528, M1-E529, M1-R530, M1-C531, M1-Q532, M1-G533, M1-E534, M1-S535, M1-T536, M1-Q537, M1-L538, M1-Q539, M1-N540, M1-E541, M1-I542, M1-Q543, M1-K544, M1-L545, M1-Q546, M1-K547, M1-T548, M1-L549, M1-K550, M1-K551, M1-K552, M1-T553, M1-K554, M1-R555, M1-Y556, M1-M557, M1-S558, M1-H559, M1-K560, M1-L561 and/or M1-K562 of SEQ ID NO:9. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal HGBPBMY4 (FLJ10961) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the HGBPBMY4 (FLJ10961) polypeptide (e.g., any combination of both N- and C-terminal HGBPBMY4 (FLJ10961) polypeptide deletions) of SEQ ID NO:9. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of HGBPBMY4 (FLJ10961) (SEQ ID NO:9), and where CX refers to any C-terminal deletion polypeptide amino acid of HGBPBMY4 (FLJ10961) (SEQ ID NO:9). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the HGBPBMY4 (FLJ10961) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate HGBPBMY4 (FLJ10961) on the one hand (i.e., agonists) and which inhibit the function of HGBPBMY4 (FLJ10961) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the HGBPBMY4 (FLJ10961) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n       FIG. 46  shows an expanded expression profile of the GBP HGBPBMY4 (FLJ10961). The figure illustrates the relative expression level of HGBPBMY4 amongst various mRNA tissue sources.  FIG. 46  also illustrates the relative expression level of HGBPBMY4 amongst various mRNA tissue sources isolated from normal and diseased tissues. As shown, the HGBPBMY4 polypeptide showed increased expression in spinal cord from multiple sclerosis patients compared to controls; increased expression in putamen and caudate from Parkinson&#39;s patients compared to controls; high expression in normal and diseased thryroid. Increased expression in breast and testicle tumors relative to controls. Expression data was obtained by measuring the steady state HGBPBMY4 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:70 and 71, and TAQMAN probe (SEQ ID NO:72) as described in Example 39 herein. These data support a role of HGBPBMY4 in regulating various functions, including immune functions and neurological functions. HGBPBMY4 may also be participating in the formation of testicle and breast turmors and thus small molecule modulators of HGBPBMY4 function may represent a novel therapeutic option in the treatment of breast and testicle cancers, as well as multiple sclerosis, Parkinson&#39;s disease and various immune system diseases. \\n     Features of the Polypeptide Encoded by Gene No. 5 \\n     A polypeptide encoded by this gene, MGBPBMY1 (LOC229900), is provided as SEQ ID NO:11 ( FIGS. 5A–5C ) and is encoded by the polynucleotide sequence according to SEQ ID NO:10 ( FIGS. 5A–5C ) and/or by a polynucleotide contained within a deposited clone. MGBPBMY1 (LOC229900) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, mouse GBP-1, mouse GBP-2, mouse GBP-3 and mouse GBP-4. \\n     The determined nucleotide sequence of the MGBPBMY1 (LOC229900), (i.e. the cDNA shown in  FIGS. 5A–5C  and in SEQ ID NO:10) comprises an open reading frame encoding a protein of about 632 amino acid residues. The predicted amino acid sequence of the MGBPBMY1 (LOC229900) polypeptide is shown in  FIGS. 5A–5C  (SEQ ID NO:11). The percent identity and similarity values between the MGBPBMY1 (LOC229900) polypeptide to the known GBP family member mGBP1 is provided in  FIG. 9 . The MGBPBMY1 (LOC229900) protein shown in  FIGS. 5A–5C  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the MGBPBMY1 (LOC229900) polypeptide showed expression in spleen, lung, heart, kidney, skeletal muscle and brain (see  FIG. 25 ). \\n     Based upon the strong homology to members of the GBP family members, the MGBPBMY1 (LOC229900) polypeptide is expected to share at least some biological activity with GBP family members, specifically mGBP-1, mGBP-2, mGBP3 and mGBP-4. \\n     The MGBPBMY1 (LOC229900) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system, lung, heart kidney, skeletal muscle and brain/nervous system in a mouse model of a human condition. \\n     The MGBPBMY1 (LOC229900) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, particularly in mammalian tissue and more preferably in a mouse model of a human condition. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY1 (LOC229900) expression in some immune system tissues (i.e., spleen) suggests a potential utility for MGBPBMY1 (LOC229900) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. In representative embodiments, MGBPBMY1 (LOC229900) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. The MGBPBMY1 (LOC229900) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. The MGBPBMY1 (LOC229900) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a MGBPBMY1 (LOC229900) polypeptide of the present invention may be useful in the treatment of include various autoimmune diseases in a mouse model of a human condition, such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A MGBPBMY1 (LOC229900) polypeptide of the present invention may also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases in a mouse model of a human condition. Such diseases and conditions include Recombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with MGBPBMY1 (LOC229900) expression levels in heart tissue suggests the MGBPBMY1 (LOC229900) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders in a mouse model of a human condition, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, MGBPBMY1 (LOC229900) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms in a mouse model of a human condition which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections in a mouse model of a human condition: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus viridans  infection,  Staphylococcus aureus  infection, coagulase-negative  staphylococci  infection, gram-negative  Bacilli  infection, Enterobacteriaceae infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative  coccobacilli  infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous Mycobacteria infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY1 (LOC229900) expression in lung tissue suggests a potential utility for MGBPBMY1 (LOC229900) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing pulmonary diseases and disorders in a mouse model of a human condition which include the following, not limiting examples: ARDS, emphysema, cystic fibrosis, interstitial lung disease, chronic obstructive pulmonary disease, bronchitis, lymphangioleiomyomatosis, pneumonitis, eosinophilic pneumonias, granulomatosis, pulmonary infarction, pulmonary fibrosis, pneumoconiosis, alveolar hemorrhage, neoplasms, lung abscesses, empyema, and increased susceptibility to lung infections (e.g., immumocompromised, HIV, etc.), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, pulmonary infections in a mouse model of a human condition: pnemonia, bacterial pnemonia, viral pnemonia (for example, as caused by Influenza virus, Respiratory syncytial virus, Parainfluenza virus, Adenovirus, Coxsackievirus, Cytomegalovirus, Herpes simplex virus, Hantavirus, etc.), mycobacteria pnemonia (for example, as caused by Mycobacterium tuberculosis, etc.) mycoplasma pnemonia, fungal pnemonia (for example, as caused by  Pneumocystis carinii, Histoplasma capsulatum, Coccidioides immitis, Blastomyces dermatitidis, Candida  sp.,  Cryptococcus neoformans, Aspergillus  sp., Zygomycetes, etc.), Legionnaires&#39; Disease,  Chlamydia pnemonia , aspiration pnemonia,  Nocordia  sp. Infections, parasitic pnemonia (for example, as caused by Strongyloides,  Toxoplasma gondii , etc.) necrotizing pnemonia, in addition to any other pulmonary disease and/or disorder (e.g., non-pneumonia) implicated by the causative agents listed above or elsewhere herein. \\n     Further, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY 1 (LOC229900) expression in skeletal muscle suggests a potential utility for MGBPBMY1 (LOC229900) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing skeletal muscle disorders in a mouse model of a human condition. In representative embodiments, MGBPBMY1 (LOC229900) polynucleotides and polypeptides including agonists and fragements thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of skeletal muscled: dystrophies, pseudohypertrophic muscular dystrophy, Duchenne dystrophy, Becker muscular dystrophy, limb-girdle muscular dystrophy, muscle weakness, Emery-Dreifuss muscular dystrophy, Congenital muscular dystrophy, endometriosis, placental aromatase deficiency, premature menopause, Fukuyama congenital muscular dystrophy, laminin alpha 2 chain deficiency, alpha 7 integrin deficiency, Walker-Warburg syndrome, myotonic dystrophy, congenital myotonic dystrophy, facioscapulohumeral muscular dystrophy, distal myopathies, central core disease, nemaline (rod) myopathy, centronuclear (myotubular) myopathy, central core disease, delay in motor milestones, delayed walking, nemaline myopathy, congenital nemaline myopathy, muscle hypotonia, centronuclear myopathies, skeletal muscle energy metabolism disorders, disorders associated with aberrant skeletal muscle-fatty acid metabolism, disorders associated with aberrant skeletal glucose metabolism, acid maltase deficiency, debranching enzyme deficiency, branching enzyme deficiency, exercise intolerance, myophosphorylase deficiency, (type V glycogenosis), phosphofructokinase deficiency (type VII glycogenosis), phosphoglycerate kinase deficiency (type IX glycogenosis), phosphoglycerate mutase deficiency (type X glycogenosis), lactate dehydrogenase deficiency (glycogensosis type XI), glycogen storage disorders, skeletal muscle lipid metabolism, carnitine deficiency, myoglobinuria, muscle cramping, myoadenylate deaminase deficiency, mitochondrial myopathies, Kearns-Sayre syndrome, myoclonic epilepsy, disorders of muscle membrane excitability, calcium channel disorders of muscle, sodium channel disorders of muscle, hyperkalemic periodic paralysis, paramyotonia congenita, potassium-aggravated myotonia, myotonia congenita, chloride channel disorders of muscle, thyrotoxic periodic paralysis, and/or Andersen&#39;s syndrome. \\n     Additionally, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the high level of MGBPBMY1 (LOC229900) expression in kidney suggests a potential utility for MGBPBMY1 (LOC229900) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the kidney in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY1 (LOC229900) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of the kidney in a mouse model of a human condition: Plasma cell infiltration, Hypercalcemia, Myeloma kidney, Amyloidosis, Light chain deposition disease, Type I/II cryoglobulinemia, Immunotactoid glomerulopathy, Reduced glomerular filtration rate, Fanconi syndrome, Hyperchloremic acidosisa, Tubular or small-molecular-weight proteinuria, Polyuria, isothenuria, Hyperkalemia, Salt wasting, Nephrocalcinosis, hyperoxaluria, Cystinosis, Fabry&#39;s disease, Sjogren&#39;s Syndrome \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the high level of MGBPBMY1 (LOC229900) expression in brain suggests a potential utility for MGBPBMY1 (LOC229900) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the brain and neurological tissue in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY1 (LOC229900) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, brain and neurological diseases or disorders in a mouse model of a human condition: the following lesions of either the central (including spinal cord, brain) or peripheral nervous systems: (1) ischemic lesions, in which a lack of oxygen in a portion of the nervous system results in neuronal injury or death, including cerebral infarction or ischemia, or spinal cord infarction or ischemia; (2) traumatic lesions, including lesions caused by physical injury or associated with surgery, for example, lesions which sever a portion of the nervous system, or compression injuries; (3) malignant lesions, in which a portion of the nervous system is destroyed or injured by malignant tissue which is either a nervous system associated malignancy or a malignancy derived from non-nervous system tissue; (4) infectious lesions, in which a portion of the nervous system is destroyed or injured as a result of infection, for example, by an abscess or associated with infection by human immunodeficiency virus, herpes zoster, or herpes simplex virus or with Lyme disease, tuberculosis, syphilis; (5) degenerative lesions, in which a portion of the nervous system is destroyed or injured as a result of a degenerative process including but not limited to degeneration associated with Parkinson&#39;s disease, Alzheimer&#39;s disease, Huntington&#39;s chorea, or amyotrophic lateral sclerosis (ALS); (6) lesions associated with nutritional diseases, disorders, and/or conditions, in which a portion of the nervous system is destroyed or injured by a nutritional disorder or disorder of metabolism including but not limited to, vitamin B 12 deficiency, folic acid deficiency, Wernicke disease, tobacco-alcohol amblyopia, Marchiafava-Bignami disease (primary degeneration of the corpus callosum), and alcoholic cerebellar degeneration; (7) neurological lesions associated with systemic diseases including, but not limited to, diabetes (diabetic neuropathy, Bell&#39;s palsy), systemic lupus erythematosus, carcinoma, or sarcoidosis; (8) lesions caused by toxic substances including alcohol, lead, or particular neurotoxins; and (9) demyelinated lesions in which a portion of the nervous system is destroyed or injured by a demyelinating disease including, but not limited to, multiple sclerosis, human immunodeficiency virus-associated myelopathy, transverse myelopathy or various etiologies, progressive multifocal leukoencephalopathy, and central pontine myelinolysis. \\n     In a representative embodiment, the MGBPBMY1 (LOC229900) polypeptides, polynucleotides, or agonists or antagonists of the present invention may be used to protect neural cells from the damaging effects of cerebral hypoxia. According to this embodiment, the compositions of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral hypoxia in a mouse model of a human condition. In one aspect of this embodiment, the MGBPBMY1 (LOC229900) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral ischemia. In another aspect of this embodiment, the MGBPBMY1 (LOC229900) polypeptides, polynucleotides, or agonists or antagonists of the invention may be used to treat, prevent, and/or diagnose neural cell injury associated with cerebral infarction in a mouse model of a human condition. In another aspect of this embodiment, the polypeptides, polynucleotides, or agonists or antagonists of the invention may be used to treat, prevent, and/or diagnose or prevent neural cell injury associated with a stroke. In a further aspect of this embodiment, the MGBPBMY1 (LOC229900) polypeptides, polynucleotides, or agonists or antagonists of the invention may be used to treat, prevent, and/or diagnose neural cell injury associated with a heart attack. \\n     The MGBPBMY1 (LOC229900) polypeptides and/or polynucleotides of the present invention which may be useful for treating or preventing a nervous system disorder may be selected by testing for biological activity in promoting the survival or differentiation of neurons in a mouse model of a human condition. For example, and not by way of limitation, compositions of the invention which elicit any of the following effects may be useful according to the invention: (1) increased survival time of neurons in culture; (2) increased sprouting of neurons in culture or in vivo; (3) increased production of a neuron-associated molecule in culture or in vivo, e.g., choline acetyltransferase or acetylcholinesterase with respect to motor neurons; or (4) decreased symptoms of neuron dysfunction in vivo. Such effects may be measured by any method known in the art. In representative, non-limiting embodiments, increased survival of neurons may routinely be measured using a method set forth herein or otherwise known in the art, such as, for example, the method set forth in Arakawa et al. (Arakawa et al., (1990)  J. Neurosci.  10:3507–3515); increased sprouting of neurons may be detected by methods known in the art, such as, for example, the methods set forth in Pestronk et al. (Pestronk et al., (1980)  Exp. Neurol.  70:65–82) or Brown et al. (Brown et al., (1981)  Ann. Rev. Neurosci.  4:17–42); increased production of neuron-associated molecules may be measured by bioassay, enzymatic assay, antibody binding, northern blot assay, etc., using techniques known in the art and depending on the molecule to be measured; and motor neuron dysfunction may be measured by assessing the physical manifestation of motor neuron disorder, e.g., weakness, motor neuron conduction velocity, or functional disability. \\n     In specific embodiments, motor neuron diseases, disorders, and/or conditions that may be treated, prevented, and/or diagnosed in a mouse model of a human condition according to the present invention include, but are not limited to, diseases, disorders, and/or conditions such as infarction, infection, exposure to toxin, trauma, surgical damage, degenerative disease or malignancy that may affect motor neurons as well as other components of the nervous system, as well as diseases, disorders, and/or conditions that selectively affect neurons such as amyotrophic lateral sclerosis, and including, but not limited to, progressive spinal muscular atrophy, progressive bulbar palsy, primary lateral sclerosis, infantile and juvenile muscular atrophy, progressive bulbar paralysis of childhood (Fazio-Londe syndrome), poliomyelitis and the post polio syndrome, and Hereditary Motorsensory Neuropathy (Charcot-Marie-Tooth Disease). \\n     It is noted that the use of mouse models to understand, diagnose, predict, treat and/or ameliorate human conditions is well documented. Thus, the uses for the MGBPBMY1 (LOC229900) may often be extrapolated to human conditions, as well as to further research such conditions and their treatments. \\n     A MGBPBMY1 (LOC229900) protein may also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a MGBPBMY1 (LOC229900) protein, can show utility as a tumor marker and/or immunotherapy targets for spleen, lung, heart, kidney, skeletal muscle and/or brain tissue. \\n     The MGBPBMY1 (LOC229900) polynucleotides and polypeptides, including fragments and for antagonsists thereof, may have uses which include identification of modulators of MGBPBMY1 (LOC229900) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the MGBPBMY1 (LOC229900) protein could be used as diagnostic agents of conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     MGBPBMY1 (LOC229900) polypeptides and polynucleotides may have additional uses which include diagnosing diseases related to the over and/or under expression of MGBPBMY1 (LOC229900) by identifying mutations in the MGBPBMY1 (LOC229900) gene by using MGBPBMY1 (LOC229900) sequences as probes or by determining MGBPBMY1 (LOC229900) protein or mRNA expression levels. MGBPBMY1 (LOC229900) polypeptides can be useful for screening compounds that affect the activity of the protein. MGBPBMY1 (LOC229900) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with MGBPBMY1 (LOC229900), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the MGBPBMY1 (LOC229900) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of MGBPBMY1 (LOC229900), heart, brain, spleen, lung, liver, skeletal muscle, kidney and/or testis, as well as various embryonic tissues, can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein may be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the MGBPBMY1 (LOC229900) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of MGBPBMY1 (LOC229900), a disease correlation related to MGBPBMY1 (LOC229900) can be made by comparing the mRNA expression level of MGBPBMY1 (LOC229900) in normal tissue, as compared to diseased tissue. Significantly higher or lower levels of MGBPBMY1 (LOC229900) expression in the diseased tissue can suggest MGBPBMY1 (LOC229900) plays a role in disease progression, and antagonists against MGBPBMY1 (LOC229900) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of MGBPBMY1 (LOC229900) expression in the diseased tissue can suggest MGBPBMY1 (LOC229900) plays a defensive role against disease progression, and agonists of MGBPBMY1 (LOC229900) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:10 ( FIGS. 5A–5C ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the MGBPBMY1 (LOC229900), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the MGBPBMY1 (LOC229900) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay could be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in another species (e.g., a mammalian species) and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and homologous recombination) and the resulting generation of transgenic animals. Expressing a particular gene in either sense or antisense orientation in a transgenic animal could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of MGBPBMY1 (LOC229900) transgenic animals, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic animals to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal MGBPBMY1 (LOC229900) deletion polypeptides are encompassed by the present invention: M1-L632, E2-L632, A3-L632, P4-L632, V5-L632, C6-L632, L7-L632, V8-L632, E9-L632, N10-L632, E11-L632, N12-L632, E13-L632, E14-L632, L15-L632, R16-L632, V17-L632, N18-L632, S19-L632, K20-L632, A21-L632, I22-L632, N23-L632, I24-L632, L25-L632, E26-L632, R27-L632, I28-L632, T29-L632, Q30-L632, P31-L632, V32-L632, V33-L632, V34-L632, V35-L632, A36-L632, I37-L632, V38-L632, G39-L632, L40-L632, Y41-L632, R42-L632, T43-L632, G44-L632, K45-L632, S46-L632, Y47-L632, L48-L632, M49-L632, N50-L632, R51-L632, L52-L632, A53-L632, G54-L632, Q55-L632, N56-L632, H57-L632, G58-L632, F59-L632, N60-L632, L61-L632, G62-L632, T63-L632, T64-L632, V65-L632, R66-L632, S67-L632, E68-L632, T69-L632, K70-L632, G71-L632, I72-L632, W73-L632, M74-L632, W75-L632, C76-L632, V77-L632, P78-L632, H79-L632, P80-L632, S81-L632, K82-L632, P83-L632, K84-L632, F85-L632, T86-L632, L87-L632, V88-L632, L89-L632, L90-L632, D81-L632, T92-L632, E93-L632, G94-L632, L95-L632, G96-L632, D97-L632, V98-L632, E99-L632, K100-L632, G101-L632, D102-L632, P103-L632, K104-L632, N105-L632, D106-L632, S107-L632, W108-L632, I109-L632, F110-L632, A111-L632, L112-L632, A113-L632, V114-L632, L115-L632, L116-L632, S117-L632, S118-L632, T119-L632, F120-L632, V121-L632, Y122-L632, N123-L632, S124-L632, M125-L632, S126-L632, T127-L632, I128-L632, N129-L632, H130-L632, Q131-L632, A132-L632, L133-L632, E134-L632, Q135-L632, L136-L632, H137-L632, Y138-L632, V139-L632, T140-L632, E141-L632, L142-L632, T143-L632, E144-L632, R145-L632, I146-L632, R147-L632, A148-L632, K149-L632, S150-L632, T151-L632, S152-L632, R153-L632, S154-L632, E155-L632, E156-L632, V157-L632, D158-L632, D159-L632, S160-L632, D161-L632, E162-L632, F163-L632, V164-L632, S165-L632, F166-L632, F167-L632, P168-L632, D169-L632, F170-L632, I171-L632, W172-L632, T173-L632, V174-L632, R175-L632, D176-L632, F177-L632, V178-L632, L179-L632, E180-L632, L181-L632, K182-L632, L183-L632, E184-L632, G185-L632, R186-L632, V187-L632, I188-L632, T189-L632, A190-L632, D191-L632, E192-L632, Y193-L632, L194-L632, E195-L632, N196-L632, A197-L632, L198-L632, K199-L632, L200-L632, I201-L632, P202-L632, G203-L632, M204-L632, S205-L632, I206-L632, K207-L632, A208-L632, Q209-L632, K210-L632, A211-L632, N212-L632, L213-L632, P214-L632, R215-L632, E216-L632, C217-L632, I218-L632, R219-L632, H220-L632, F221-L632, F222-L632, P223-L632, R224-L632, R225-L632, K226-L632, C227-L632, F228-L632, V229-L632, F230-L632, D231-L632, R232-L632, P233-L632, T234-L632, K235-L632, D236-L632, K237-L632, E238-L632, L239-L632, L240-L632, V241-L632, H242-L632, V243-L632, E244-L632, E245-L632, M246-L632, P247-L632, E248-L632, D249-L632, Q250-L632, L251-L632, D252-L632, H253-L632, S254-L632, F255-L632, Q256-L632, V257-L632, Q258-L632, S259-L632, K260-L632, E261-L632, F262-L632, C263-L632, S264-L632, Y265-L632, I266-L632, F267-L632, S268-L632, N269-L632, S270-L632, K271-L632, A272-L632, K273-L632, T274-L632, L275-L632, K276-L632, E277-L632, G278-L632, I279-L632, V280-L632, V281-L632, N282-L632, G283-L632, N284-L632, R285-L632, L286-L632, A287-L632, T288-L632, L289-L632, V290-L632, T291-L632, T292-L632, Y293-L632, V294-L632, D295-L632, A296-L632, I297-L632, N298-L632, S299-L632, G300-L632, D301-L632, V302-L632, P303-L632, C304-L632, L305-L632, E306-L632, N307-L632, A308-L632, V309-L632, T310-L632, T311-L632, L312-L632, A313-L632, Q314-L632, R315-L632, E316-L632, N317-L632, S318-L632, I319-L632, A320-L632, V321-L632, Q322-L632, K323-L632, A324-L632, A325-L632, D326-L632, H327-L632, Y328-L632, S329-L632, E330-L632, Q331-L632, M332-L632, A333-L632, Q334-L632, R335-L632, M336-L632, R337-L632, L338-L632, P339-L632, T340-L632, D341-L632, T342-L632, L343-L632, Q344-L632, E345-L632, L346-L632, L347-L632, T348-L632, V349-L632, H350-L632, T351-L632, A352-L632, C353-L632, E354-L632, K355-L632, E356-L632, A357-L632, I358-L632, A359-L632, V360-L632, F361-L632, M362-L632, E363-L632, H364-L632, S365-L632, F366-L632, K367-L632, D368-L632, E369-L632, N370-L632, Q371-L632, Q372-L632, F373-L632, Q374-L632, K375-L632, N376-L632, L377-L632, V378-L632, V379-L632, T380-L632, I381-L632, E382-L632, E383-L632, K384-L632, K385-L632, E386-L632, D387-L632, F388-L632, L389-L632, R390-L632, Q391-L632, N392-L632, E393-L632, A394-L632, A395-L632, S396-L632, L397-L632, S398-L632, H399-L632, C400-L632, Q401-L632, A402-L632, E403-L632, L404-L632, D405-L632, K406-L632, L407-L632, S408-L632, E409-L632, S410-L632, L411-L632, R412-L632, E413-L632, S414-L632, I415-L632, S416-L632, R417-L632, G418-L632, V419-L632, F420-L632, S421-L632, V422-L632, P423-L632, G424-L632, G425-L632, H426-L632, R427-L632, L428-L632, Y429-L632, L430-L632, E431-L632, A432-L632, R433-L632, K434-L632, K435-L632, V436-L632, E437-L632, Q438-L632, D439-L632, Y440-L632, E441-L632, R442-L632, V443-L632, P444-L632, R445-L632, K446-L632, G447-L632, V448-L632, K449, A450-L632, N451-L632, H452-L632, V453-L632, L454-L632, Q455-L632, S456-L632, F457-L632, L458-L632, Q459-L632, S460-L632, Q461-L632, I462-L632, S463-L632, I464-L632, E465-L632, D466-L632, S467-L632, I468-L632, M469-L632, Q470-L632, S471-L632, D472-L632, K473-L632, A474-L632, L475-L632, T476-L632, D477-L632, G478-L632, Q479-L632, K480-L632, A481-L632, M482-L632, E483-L632, A484-L632, E485-L632, R486-L632, A487-L632, Q488-L632, K489-L632, E490-L632, A491-L632, A492-L632, E493-L632, K494-L632, E495-L632, Q496-L632, E497-L632, L498-L632, L499-L632, R500-L632, Q501-L632, K502-L632, Q503-L632, K504-L632, E505-L632, L506-L632, Q507-L632, Q508-L632, V509-L632, M510-L632, E511-L632, A512-L632, Q513-L632, E514-L632, R515-L632, S516-L632, Y517-L632, K518-L632, E519-L632, N520-L632, V521-L632, A522-L632, Q523-L632, L524-L632, H525-L632, E526-L632, K527-L632, M528-L632, E529-L632, T530-L632, E531-L632, R532-L632, K533-L632, N534-L632, I535-L632, L536-L632, R537-L632, E538-L632, Q539-L632, E540-L632, V541-L632, K542-L632, L543-L632, E544-L632, H545-L632, K546-L632, L547-L632, K548-L632, I549-L632, Q550-L632, K551-L632, D552-L632, M553-L632, L554-L632, N555-L632, E556-L632, G557-L632, F558-L632, K559-L632, R560-L632, K561-L632, C562-L632, E563-L632, A564-L632, M565-L632, D566-L632, L567-L632, E568-L632, I569-L632, S570-L632, Q571-L632, L572-L632, Q573-L632, K574-L632, E575-L632, I576-L632, Q577-L632, L578-L632, N579-L632, K580-L632, E581-L632, K582-L632, N583-L632, S584-L632, S585-L632, L586-L632, G587-L632, A588-L632, K589-L632, I590-L632, L591-L632, D592-L632, G593-L632, F594-L632, G595-L632, D596-L632, V597-L632, L598-L632, I599-L632, S600-L632, V601-L632, V602-L632, P603-L632, G604-L632, S605-L632, G606-L632, K607-L632, Y608-L632, F609-L632, G610-L632, L611-L632, G612-L632, L613-L632, K614-L632, I615-L632, L616-L632, S617-L632, S618-L632, Q619-L632, M620-L632, N621-L632, Q622-L632, T623-L632, Q624-L632, N625-L632, S262-L632, D627-L632, K628-L632, V629-L632, R630-L632 and/or K631-L632, of SEQ ID NO:11. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal MGBPBMY1 (LOC229900) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal MGBPBMY1 (LOC229900) deletion polypeptides are encompassed by the present invention: M1-E2, M1-A3, M1-P4, M1-V5, M1-C6, M1-L7, M1-V8, M1-E9, M1-N10, M1-ElI, M1-N12, M1-E13, M1-E14, M1-L15, M1-R16, M1-V17, M1-N18, M1-S19, M1-K20, M1-A21, M1-I22, M1-N23, M1-I24, M1-L25, M1-E26, M1-R27, M1-I28, M1-T29, M1-Q30, M1-P31, M1-V32, M1-V33, M1-V34, M1-V35, M1-A36, M1-I37, M1-V38, M1-G39, M1-L40, M1-Y41, M1-R42, M1-T43, M1-G44, M1-K45, M1-S46, M1-Y47, M1-L48, M1-M49, M1-N50, M1-R51, M1-L52, M1-A53, M1-G54, M1-Q55, M1-N56, M1-H57, M1-G58, M1-F59, M1-N60, M1-L61, M1-G62, M1-T63, M1-T64, M1-V65, M1-R66, M1-S67, M1-E68, M1-T69, M1-K70, M1-G71, M1-I72, M1-W73, M1-M74, M1-W75, M1-C76, M1-V77, M1-P78, M1-H79, M1-P80, M1-S81, M1-K82, M1-P83, M1-K84, M1-F85, M1-T86, M1-L87, M1-V88, M1-L89, M1-L90, M1-D91, M1-T92, M1-E93, M1-G94, M1-L95, M1-G96, M1-D97, M1-V98, M1-E99, M1-K100, M1-G10, M1-D102, M1-P103, M1-K104, M1-N105, M1-D106, M1-S107, M1-W108, M1-I109, M1-F110, M1-A111, M1-L112, M1-A113, M1-V114, M1-L115, M1-L116, M1-S117, M1-S118, M1-T119, M1-F120, M1-V121, M1-Y122, M1-N123, M1-S124, M1-M125, M1-S126, M1-T127, M1-I128, M1-N129, M1-H130, M1-Q131, M1-A132, M1-L133, M1-E134, M1-Q135, M1-L136, M1-H137, M1-Y138, M1-V139, M1-T140, M1-E141, M1-L142, M1-T143, M1-E144, M1-R145, M1-I146, M1-R147, M1-A148, M1-K149, M1-S150, M1-T151, M1-S152, M1-R153, M1-S154, M1-E155, M1-E156, M1-V157, M1-D158, M1-D159, M1-S160, M1-D161, M1-E162, M1-F163, M1-V164, M1-S165, M1-F166, M1-F167, M1-P168, M1-D169, M1-F170, M1-I171, M1-W172, M1-T173, M1-V174, M1-R175, M1-D176, M1-F177, M1-V178, M1-L179, M1-E180, M1-L181, M1-K182, M1-L183, M1-E184, M1-G185, M1-R186, M1-V187, M1-I188, M1-T189, M1-A190, M1-D191, M1-E192, M1-Y193, M1-L194, M1-E195, M1-N196, M1-A197, M1-L198, M1-K199, M1-L200, M1-I201, M1-P202, M1-G203, M1-M204, M1-S205, M1-I206, M1-K207, M1-A208, M1-Q209, M1-K210, M1-A211, M1-N212, M1-L213, M1-P214, M1-R215, M1-E216, M1-C217, M1-I218, M1-R219, M1-H220, M1-F221, M1-F222, M1-P223, M1-R224, M1-R225, M1-K226, M1-C227, M1-F228, M1-V229, M1-F230, M1-D231, M1-R232, M1-P233, M1-T234, M1-K235, M1-D236, M1-K237, M1-E238, M1-L239, M1-L240, M1-V241, M1-H242, M1-V243, M1-E244, M1-E245, M1-M246, M1-P247, M1-E248, M1-D249, M1-Q250, M1-L251, M1-D252, M1-H253, M1-S254, M1-F255, M1-Q256, M1-V257, M1-Q258, M1-S259, M1-K260, M1-E261, M1-F262, M1-C263, M1-S264, M1-Y265, M1-I266, M1-F267, M1-S268, M1-N269, M1-S270, M1-K271, M1-A272, M1-K273, M1-T374, M1-L275, M1-K276, M1-E277, M1-G278, M1-I279, M1-V280, M1-V281, M1-N282, M1-G283, M1-N284, M1-R285, M1-L286, M1-A287, M1-T288, M1-L289, M1-V290, M1-T291, M1-T292, M1-Y293, M1-V294, M1-D295, M1-A296, M1-I297, M1-N298, M1-S299, M1-G300, M1-D301, M1-V302, M1-P303, M1-C304, M1-L305, M1-E306, M1-N307, M1-A308, M1-V309, M1-T310, M1-T311, M1-L312, M1-A313, M1-Q314, M1-R315, M1-E316, M1-N317, M1-S318, M1-I319, M1-A320, M1-V321, M1-Q322, M1-K323, M1-A324, M1-A325, M1-D326, M1-H327, M1-Y328, M1-S329, M1-E330, M1-Q331, M1-M332, M1-A333, M1-Q334, M1-R335, M1-M336, M1-R337, M1-L338, M1-P339, M1-T340, M1-D341, M1-T342, M1-L343, M1-Q344, M1-E345, M1-L346, M1-L347, M1-T348, M1-V349, M1-H350, M1-T351, M1-A352, M1-C353, M1-E354, M1-K355, M1-E356, M1-A357, M1-I358, M1-A359, M1-V360, M1-F361, M1-M362, M1-E363, M1-H364, M1-S365, M1-F366, M1-K367, M1-D368, M1-E369, M1-N370, M1-Q371, M1-Q372, M1-F373, M1-Q374, M1-K375, M1-N376, M1-L377, M1-V378, M1-V379, M1-T380, M1-1381, M1-E382, M1-E383, M1-K384, M1-K385, M1-E386, M1-D387, M1-F388, M1-L389, M1-R390, M1-Q391, M1-N392, M1-E393, M1-A394, M1-A395, M1-S396, M1-L397, M1-S398, M1-H399, M1-C400, M1-Q401, M1-A402, M1-E403, M1-L404, M1-D405, M1-K406, M1-L407, M1-S408, M1-E409, M1-S410, M1-L411, M1-R412, M1-E413, M1-S414, M1-I415, M1-S416, M1-R417, M1-G418, M1-V419, M1-F420, M1-S421, M1-V422, M1-P423, M1-G424, M1-G425, M1-H426, M1-R427, M1-L428, M1-Y429, M1-L430, M1-E431, M1-A432, M1-R433, M1-K434, M1-K435, M1-V436, M1-E437, M1-Q438, M1-D439, M1-Y440, M1-E441, M1-R442, M1-V443, M1-P444, M1-R445, M1-K446, M1-G447, M1-V448, M1-K449, M1-A450, M1-N451, M1-H452, M1-V453, M1-L454, M1-Q455, M1-S456, M1-F457, M1-L458, M1-Q459, M1-S460, M1-Q461, M1-I462, M1-S463, M1-I464, M1-E465, M1-D466, M1-S467, M1-I468, M1-M469, M1-Q470, M1-S471, M1-D472, M1-K473, M1-A474, M1-L475, M1-T476, M1-D477, M1-G478, M1-Q479, M1-K480, M1-A481, M1-M482, M1-E483, M1-A484, M1-E485, M1-R486, M1-A487, M1-Q488, M1-K489, M1-E490, M1-A491, M1-A492, M1-E493, M1-K494, M1-E495, M1-Q496, M1-E497, M1-L498, M1-L499, M1-R500, M1-Q501, M1-K502, M1-Q503, M1-K504, M1-E505, M1-L506, M1-Q507, M1-Q508, M1-V509, M1-M510, M1-E511, M1-A512, M1-Q513, M1-E514, M1-R515, M1-S516, M1-Y517, M1-K518, M1-E519, M1-N520, M1-V521, M1-A522, M1-Q523, M1-L524, M1-H525, M1-E526, M1-K527, M1-M528, M1-E529, M1-T530, M1-E531, M1-R532, M1-K533, M1-N534, M1-I535, M1-L536, M1-R537, M1-E538, M1-Q539, M1-E540, M1-V541, M1-K542, M1-L543, M1-E544, M1-H545, M1-K546, M1-L547, M1-K548, M1-I549, M1-Q550, M1-K551, M1-D552, M1-M553, M1-L554, M1-N555, M1-E556, M1-G557, M1-F558, M1-K559, M1-R560, M1-K561, M1-C562, M1-E563, M1-A564, M1-M565, M1-D566, M1-L567, M1-E568, M1-I569, M1-S570, M1-Q571, M1-L572, M1-Q573, M1-K574, M1-E575, M1-I576, M1-Q577, M1-L578, M1-N579, M1-K580, M1-E581, M1-K582, M1-N583, M1-S584, M1-S585, M1-L586, M1-G587, M1-A588, M1-K589, M1-I590, M1-L591, M1-D592, M1-G593, M1-F594, M1-G595, M1-D596, M1-V597, M1-L598, M1-I599, M1-S600, M1-V601, M1-V602, M1-P603, M1-G604, M1-S605, M1-G606, M1-K607, M1-Y608, M1-F609, M1-G610, M1-L611, M1-G612, M1-L613, M1-K614, M1-I615, M1-L616, M1-S617, M1-S618, M1-Q619, M1-M620, M1-N621, M1-Q622, M1-T623, M1-Q624, M1-N625, M1-S626, M1-D627, M1-K628, M1-V629, M1-R630 and/or M1-K631 of SEQ ID NO:11. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal MGBPBMY1 (LOC229900) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the MGBPBMY1 (LOC229900) polypeptide (e.g., any combination of both N- and C-terminal MGBPBMY1 (LOC229900) polypeptide deletions) of SEQ ID NO:11. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of MGBPBMY1 (LOC229900) (SEQ ID NO:11), and where CX refers to any C-terminal deletion polypeptide amino acid of MGBPBMY1 (LOC229900) (SEQ ID NO:11). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the MGBPBMY1 (LOC229900) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate MGBPBMY1 (LOC229900) on the one hand (i.e., agonists) and which inhibit the function of MGBPBMY1 (LOC229900) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the MGBPBMY1 (LOC229900) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n     Features of the Polypeptide Encoded by Gene No. 6 \\n     A polypeptide encoded by this gene, MGBPBMY2 (LOC229902), is provided as SEQ ID NO:13 ( FIGS. 6A–6B ) and is encoded by the polynucleotide sequence according to SEQ ID NO:12 ( FIGS. 6A–6B ) and/or by a polynucleotide contained within a deposited clone. MGBPBMY2 (LOC229902) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, mouse GBP-1, mouse GBP-2, mouse GBP-3 and mouse GBP-4. \\n     The determined nucleotide sequence of the MGBPBMY2 (LOC229902), (i.e. the cDNA shown in  FIGS. 6A–6B  and in SEQ ID NO:12) comprises an open reading frame encoding a protein of about 605 amino acid residues. The predicted amino acid sequence of the MGBPBMY2 (LOC229902) polypeptide is shown in  FIGS. 6A–6B  (SEQ ID NO:13). The percent identity and similarity values between the MGBPBMY2 (LOC229902) polypeptide to the known GBP family member mGBP1 is provided in  FIG. 9 . The MGBPBMY2 (LOC229902) protein shown in FIGS.  6 A– 6 B was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the MGBPBMY2 (LOC2299002) polypeptide showed experession in embryonic tissue and in adult, in spleen, as well as in lung, testis, brain, liver, heart, kidney and skeletal muscle (see  FIG. 24 ). \\n     Based upon the strong homology to members of the GBP family members, the MGBPBMY2 (LOC229902) polypeptide is expected to share at least some biological activity with GBP family members, specifically mGBP-1, mGBP-2, mGBP3 and mGBP-4. \\n     The MGBPBMY2 (LOC229902) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system, lung, the reproductive system, brain and nervous system, liver, heart, kidney and skeletal muscle in a mouse model of a human condition. \\n     The MGBPBMY2 (LOC229902) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue and more preferably in a mouse model of a human condition. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with observed MGBPBMY2 (LOC229902) expression in some immune system tissues (i.e., spleen) suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. The MGBPBMY2 (LOC229902) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. The MGBPBMY2 (LOC229902) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a MGBPBMY2 (LOC229902) polypeptide of the present invention may be useful in the treatment of include various autoimmune diseases in a mouse model of a human condition, such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A MGBPBMY2 (LOC229902) polypeptide of the present invention may also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases in a mouse model of a human condition. Such diseases and conditions include Recombinase activating gene (RAG ½) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with observed MGBPBMY2 (LOC229902) expression in heart tissue, suggests the MGBPBMY2 (LOC229902) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders in a mouse model of a human condition, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, MGBPBMY2 (LOC229902) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms in a mouse model of a human condition which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections in a mouse model of a human condition: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus  viridans infection,  Staphylococcus aureus  infection, coagulase-negative  staphylococci  infection, gram-negative  Bacilli  infection, Enterobacteriaceae infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative  coccobacilli  infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous Mycobacteria infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY2 (LOC229902) expression in lung tissue suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in in treating, diagnosing, prognosing, and/or preventing pulmonary diseases and disorders in a mouse model of a human condition which include the following, not limiting examples: ARDS, emphysema, cystic fibrosis, interstitial lung disease, chronic obstructive pulmonary disease, bronchitis, lymphangioleiomyomatosis, pneumonitis, eosinophilic pneumonias, granulomatosis, pulmonary infarction, pulmonary fibrosis, pneumoconiosis, alveolar hemorrhage, neoplasms, lung abscesses, empyema, and increased susceptibility to lung infections (e.g., immumocompromised, HIV, etc.), for example. \\n     Moreover, MGBPBMY2 (LOC229902) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, pulmonary infections in a mouse model of a human condition: pnemonia, bacterial pnemonia, viral pnemonia (for example, as caused by Influenza virus, Respiratory syncytial virus, Parainfluenza virus, Adenovirus, Coxsackievirus, Cytomegalovirus, Herpes simplex virus, Hantavirus, etc.), mycobacteria pnemonia (for example, as caused by  Mycobacterium tuberculosis , etc.) mycoplasma pnemonia, fungal pnemonia (for example, as caused by  Pneumocystis carinii, Histoplasma capsulatum, Coccidioides immitis, Blastomyces dermatitidis, Candida  sp.,  Cryptococcus neoformans, Aspergillus  sp., Zygomycetes, etc.), Legionnaires&#39; Disease,  Chlamydia pnemonia , aspiration pnemonia,  Nocordia  sp. Infections, parasitic pnemonia (for example, as caused by  Strongyloides, Toxoplasma gondii , etc.) necrotizing pnemonia, in addition to any other pulmonary disease and/or disorder (e.g., non-pneumonia) implicated by the causative agents listed above or elsewhere herein. \\n     Further, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY2 (LOC229902) expression in skeletal muscle suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing skeletal muscle disorders in a mouse model of a human condition. In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists and fragements thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of skeletal muscled: dystrophies, pseudohypertrophic muscular dystrophy, Duchenne dystrophy, Becker muscular dystrophy, limb-girdle muscular dystrophy, muscle weakness, Emery-Dreifuss muscular dystrophy, Congenital muscular dystrophy, endometriosis, placental aromatase deficiency, premature menopause, Fukuyama congenital muscular dystrophy, laminin alpha 2 chain deficiency, alpha 7 integrin deficiency, Walker-Warburg syndrome, myotonic dystrophy, congenital myotonic dystrophy, facioscapulohumeral muscular dystrophy, distal myopathies, central core disease, nemaline (rod) myopathy, centronuclear (myotubular) myopathy, central core disease, delay in motor milestones, delayed walking, nemaline myopathy, congenital nemaline myopathy, muscle hypotonia, centronuclear myopathies, skeletal muscle energy metabolism disorders, disorders associated with aberrant skeletal muscle-fatty acid metabolism, disorders associated with aberrant skeletal glucose metabolism, acid maltase deficiency, debranching enzyme deficiency, branching enzyme deficiency, exercise intolerance, myophosphorylase deficiency (type V glycogenosis), phosphofructokinase deficiency (type VII glycogenosis), phosphoglycerate kinase deficiency (type IX glycogenosis), phosphoglycerate mutase deficiency (type X glycogenosis), lactate dehydrogenase deficiency (glycogensosis type XI), glycogen storage disorders, skeletal muscle lipid metabolism, carnitine deficiency, myoglobinuria, muscle cramping, myoadenylate deaminase deficiency, mitochondrial myopathies, Keams-Sayre syndrome, myoclonic epilepsy, disorders of muscle membrane excitability, calcium channel disorders of muscle, sodium channel disorders of muscle, hyperkalemic periodic paralysis, paramyotonia congenita, potassium-aggravated myotonia, myotonia congenita, chloride channel disorders of muscle, thyrotoxic periodic paralysis, and/or Andersen&#39;s syndrome. \\n     Further, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY2 (LOC229902) expression in kidney suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the kidney in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of the kidney in a mouse model of a human condition: Plasma cell infiltration, Hypercalcemia, Myeloma kidney, Amyloidosis, Light chain deposition disease, Type I/II cryoglobulinemia, Immunotactoid glomerulopathy, Reduced glomerular filtration rate, Fanconi syndrome, Hyperchloremic acidosisa, Tubular or small-molecular-weight proteinuria, Polyuria, isothenuria, Hyperkalemia, Salt wasting, Nephrocalcinosis, hyperoxaluria, Cystinosis, Fabry&#39;s disease, Sjogren&#39;s Syndrome \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY2 (LOC229902) expression in brain suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the brain and neurological tissue in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, brain and neurological diseases or disorders in a mouse model of a human condition: the following lesions of either the central (including spinal cord, brain) or peripheral nervous systems: (1) ischemic lesions, in which a lack of oxygen in a portion of the nervous system results in neuronal injury or death, including cerebral infarction or ischemia, or spinal cord infarction or ischemia; (2) traumatic lesions, including lesions caused by physical injury or associated with surgery, for example, lesions which sever a portion of the nervous system, or compression injuries; (3) malignant lesions, in which a portion of the nervous system is destroyed or injured by malignant tissue which is either a nervous system associated malignancy or a malignancy derived from non-nervous system tissue; (4) infectious lesions, in which a portion of the nervous system is destroyed or injured as a result of infection, for example, by an abscess or associated with infection by human immunodeficiency virus, herpes zoster, or herpes simplex virus or with Lyme disease, tuberculosis, syphilis; (5) degenerative lesions, in which a portion of the nervous system is destroyed or injured as a result of a degenerative process including but not limited to degeneration associated with Parkinson&#39;s disease, Alzheimer&#39;s disease, Huntington&#39;s chorea, or amyotrophic lateral sclerosis (ALS); (6) lesions associated with nutritional diseases, disorders, and/or conditions, in which a portion of the nervous system is destroyed or injured by a nutritional disorder or disorder of metabolism including but not limited to, vitamin B12 deficiency, folic acid deficiency, Wemicke disease, tobacco-alcohol amblyopia, Marchiafava-Bignami disease (primary degeneration of the corpus callosum), and alcoholic cerebellar degeneration; (7) neurological lesions associated with systemic diseases including, but not limited to, diabetes (diabetic neuropathy, Bell&#39;s palsy), systemic lupus erythematosus, carcinoma, or sarcoidosis; (8) lesions caused by toxic substances including alcohol, lead, or particular neurotoxins; and (9) demyelinated lesions in which a portion of the nervous system is destroyed or injured by a demyelinating disease including, but not limited to, multiple sclerosis, human immunodeficiency virus-associated myelopathy, transverse myelopathy or various etiologies, progressive multifocal leukoencephalopathy, and central pontine myelinolysis. \\n     In a representative embodiment, the MGBPBMY2 (LOC229902) polypeptides, polynucleotides, or agonists or antagonists of the present invention may used to protect neural cells from the damaging effects of cerebral hypoxia. According to this embodiment, the compositions of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral hypoxia in a mouse model of a human condition. In one aspect of this embodiment, the MGBPBMY2 (LOC229902) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral ischemia. In another aspect of this embodiment, the MGBPBMY2 (LOC229902) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral infarction in a mouse model of a human condition. In another aspect of this embodiment, the polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose or prevent neural cell injury associated with a stroke. In a further aspect of this embodiment, the MGBPBMY2 (LOC229902) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with a heart attack. \\n     MGBPBMY2 (LOC229902) polypeptides and/or polynucleotides of the present invention that are useful for treating or preventing a nervous system disorder may be selected by testing for biological activity in promoting the survival or differentiation of neurons in a mouse model of a human condition. For example, and not by way of limitation, compositions of the invention which elicit any of the following effects may be useful according to the invention: (1) increased survival time of neurons in culture; (2) increased sprouting of neurons in culture or in vivo; (3) increased production of a neuron-associated molecule in culture or in vivo, e.g., choline acetyltransferase or acetylcholinesterase with respect to motor neurons; or (4) decreased symptoms of neuron dysfunction in vivo. Such effects may be measured by any method known in the art. In representative, non-limiting embodiments, increased survival of neurons may routinely be measured using a method set forth herein or otherwise known in the art, such as, for example, the method set forth in Arakawa et al. (Arakawa et al., (1990)  J. Neurosci.  10:3507–3515); increased sprouting of neurons may be detected by methods known in the art, such as, for example, the methods set forth in Pestronk et al. (Pestronk et al., (1980)  Exp. Neurol.  70:65–82) or Brown et al. (Brown et al., (1981)  Ann. Rev. Neurosci.  4:17–42); increased production of neuron-associated molecules may be measured by bioassay, enzymatic assay, antibody binding, northern blot assay, etc., using techniques known in the art and depending on the molecule to be measured; and motor neuron dysfunction may be measured by assessing the physical manifestation of motor neuron disorder, e.g., weakness, motor neuron conduction velocity, or functional disability. \\n     In specific embodiments, motor neuron diseases, disorders, and/or conditions that may be treated, prevented, and/or diagnosed in a mouse model of a human condition according to the present invention include, but are not limited to, diseases, disorders, and/or conditions such as infarction, infection, exposure to toxin, trauma, surgical damage, degenerative disease or malignancy that may affect motor neurons as well as other components of the nervous system, as well as diseases, disorders, and/or conditions that selectively affect neurons such as amyotrophic lateral sclerosis, and including, but not limited to, progressive spinal muscular atrophy, progressive bulbar palsy, primary lateral sclerosis, infantile and juvenile muscular atrophy, progressive bulbar paralysis of childhood (Fazio-Londe syndrome), poliomyelitis and the post polio syndrome, and Hereditary Motorsensory Neuropathy (Charcot-Marie-Tooth Disease). \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with MGBPBMY2 (LOC229902) expression in liver tissue suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing liver diseases in a mouse model of a human condition. In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing liver diseases in a mouse model of a human condition. For example, an MGBPBMY2 (LOC229902) protein can be used for the detection, treatment, amelioration, and/or prevention of diseases and conditions in a mouse model of a human condition including, but not limited to: hepatoblastoma, jaundice, hepatitis, liver metabolic diseases and conditions that are attributable to the differentiation of hepatocyte progenitor cells, cirrhosis, hepatic cysts, pyrogenic abscess, amebic abcess, hydatid cyst, cystadenocarcinoma, adenoma, focal nodular hyperplasia, hemangioma, hepatocellulae carcinoma, cholangiocarcinoma, and angiosarcoma, granulomatous liver disease, liver transplantation, hyperbilirubinemia, jaundice, parenchymal liver disease, portal hypertension, hepatobiliary disease, hepatic parenchyma, hepatic fibrosis, anemia, gallstones, cholestasis, carbon tetrachloride toxicity, beryllium toxicity, vinyl chloride toxicity, choledocholithiasis, hepatocellular necrosis, aberrant metabolism of amino acids, aberrant metabolism of carbohydrates, aberrant synthesis proteins, aberrant synthesis of glycoproteins, aberrant degradation of proteins, aberrant degradation of glycoproteins, aberrant metabolism of drugs, aberrant metabolism of hormones, aberrant degradation of drugs, aberrant degradation of drugs, aberrant regulation of lipid metabolism, aberrant regulation of cholesterol metabolism, aberrant glycogenesis, aberrant glycogenolysis, aberrant glycolysis, aberrant gluconeogenesis, hyperglycemia, glucose intolerance, hyperglycemia, decreased hepatic glucose uptake, decreased hepatic glycogen synthesis, hepatic resistance to insulin, portal-systemic glucose shunting, peripheral insulin resistance, hormonal abnormalities, increased levels of systemic glucagon, decreased levels of systemic cortisol, increased levels of systemic insulin, hypoglycemia, decreased gluconeogenesis, decreased hepatic glycogen content, hepatic resistance to glucagon, elevated levels of systemic aromatic amino acids, decreased levels of systemic branched-chain amino acids, hepatic encephalopathy, aberrant hepatic amino acid transamination, aberrant hepatic amino acid oxidative deamination, aberrant ammonia synthesis, aberant albumin secretion, hypoalbuminemia, aberrant cytochromes b5 function, aberrant P450 function, aberrant glutathione S-acyltransferase function, aberrant cholesterol synthesis, and aberrant bile acid synthesis. \\n     Moreover, MGBPBMY2 (LOC229902) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, hepatic infections in a mouse model of a human condition: liver disease caused by sepsis infection, liver disease caused by bacteremia, liver disease caused by Pneomococcal pneumonia infection, liver disease caused by Toxic shock syndrome, liver disease caused by Listeriosis, liver disease caused by Legionnaries&#39; disease, liver disease caused by Brucellosis infection, liver disease caused by  Neisseria gonorrhoeae  infection, liver disease caused by  Yersinia  infection, liver disease caused by Salmonellosis, liver disease caused by Nocardiosis, liver disease caused by Spirochete infection, liver disease caused by  Treponema pallidum  infection, liver disease caused by  Brrelia burgdorferi  infection, liver disease caused by Leptospirosis, liver disease caused by  Coxiella burnetii  infection, liver disease caused by  Rickettsia richettsii  infection, liver disease caused by  Chlamydia trachomatis  infection, liver disease caused by  Chlamydia psittaci  infection, liver disease caused by hepatitis virus infection, liver disease caused by Epstein-Barr virus infection in addition to any other hepatic disease and/or disorder implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with the observed MGBPBMY2 (LOC229902) expression in testis tissue suggests a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing testis diseases in a mouse model of a human condition. In representative embodiments, MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing testis diseases in a mouse model of a human condition. For example, an MGBPBMY2 (LOC229902) protein may be used for the detection, treatment, amelioration, diagnosis and/or prevention of diseases and conditions in a mouse model of a human condition including, but not limited to the following, non-limiting, diseases or disorders of the testis: spermatogenesis, infertility, Klinefelter&#39;s syndrome, XX male, epididymitis, genital warts, germinal cell aplasia, cryptorchidism, varicocele, immotile cilia syndrome, and viral orchitis. The MGBPBMY2 (LOC229902) polynucleotides and polypeptides including agonists and fragments thereof, may also have uses related to modulating testicular development, embryogenesis, reproduction, and in ameliorating, treating, and/or preventing testicular proliferative disorders (e.g., cancers, which include, for example, choriocarcinoma, Nonseminoma, seminona, and testicular germ cell tumors). \\n     Likewise, the observed MGBPBMY2 (LOC229902) expression in testis tissue also emphasizes a potential utility for MGBPBMY2 (LOC229902) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing metabolic diseases and disorders in a mouse model of a human condition which include the following, not limiting examples: premature puberty, incomplete puberty, Kallman syndrome, Cushing&#39;s syndrome, hyperprolactinemia, hemochromatosis, congenital adrenal hyperplasia, FSH deficiency, and granulomatous disease, for example. \\n     MGBPBMY2 (LOC229902) polynucleotides and/or poplypeptides may also be useful in assays designed to identify binding agents, as such agents (antagonists) are useful as male contraceptive agents. The testes are also a site of active gene expression of transcripts that is expressed, particularly at low levels, in other tissues of the body. Therefore, this gene product may be expressed in other specific tissues or organs where it may play related functional roles in other processes, such as hematopoiesis, inflammation, bone formation, and kidney function, to name a few possible target indications. \\n     It is noted that the use of mouse models to understand, diagnose, predict, treat and/or ameliorate human conditions is well documented. Thus, the uses for the MGBPBMY2 (LOC229902) can often be extrapolated to human conditions, as well as to further research such conditions and their treatments. \\n     A MGBPBMY2 (LOC229902) protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a MGBPBMY2 (LOC229902) protein, can show utility as a tumor marker and/or immunotherapy targets for spleen, lung, heart, kidney, skeletal muscle and/or brain tissue. \\n     The MGBPBMY2 (LOC229902) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of MGBPBMY2 (LOC229902) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the MGBPBMY2 (LOC229902) protein could be used as diagnostic agents of certain conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     MGBPBMY2 (LOC229902) polypeptides and polynucleotides may have additional uses which include diagnosing diseases related to the over and/or under expression of MGBPBMY2 (LOC229902) by identifying mutations in the MGBPBMY2 (LOC229902) gene by using MGBPBMY2 (LOC229902) sequences as probes or by determining MGBPBMY2 (LOC229902) protein or mRNA expression levels. MGBPBMY2 (LOC229902) polypeptides may be useful for screening compounds that affect the activity of the protein. MGBPBMY2 (LOC229902) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with MGBPBMY2 (LOC229902), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the MGBPBMY2 (LOC229902) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of MGBPBMY2 (LOC229902), heart, brain, spleen, lung, liver, skeletal muscle, kidney and/or testis, as well as various embryonic tissues, can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the MGBPBMY2 (LOC229902) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of MGBPBMY2 (LOC229902), a disease correlation related to MGBPBMY2 (LOC229902) can be made by comparing the mRNA expression level of MGBPBMY2 (LOC229902) in normal tissue, as compared to diseased tissue. Significantly higher or lower levels of MGBPBMY2 (LOC229902) expression in the diseased tissue can suggest MGBPBMY2 (LOC229902) plays a role in disease progression, and antagonists against MGBPBMY2 (LOC229902) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of MGBPBMY2 (LOC229902) expression in the diseased tissue can suggest MGBPBMY2 (LOC229902) plays a defensive role against disease progression, and agonists of MGBPBMY2 (LOC229902) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:12 ( FIGS. 6A–6B ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the MGBPBMY2 (LOC229902), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the MGBPBMY2 (LOC229902) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay can be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in another species (e.g., a mammalian species) and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and homlogous recombination) and the resulting generation of transgenic animals. Expressing a particular gene in either sense or antisense orientation in a transgenic animal could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of MGBPBMY2 (LOC229902), transgenic animals, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic animals to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal MGBPBMY2 (LOC229902) deletion polypeptides are encompassed by the present invention: M1-S605, E2-S605, G3-S605, K4-S605, V5-S605, L6-S605, Q7-S605, W8-S605, N9-S605, S10-S605, Y11-S605, L12-S605, S13-S605, E14-S605, F15-S605, H16-S605, C17-S605, D18-S605, Q19-S605, D20-S605, V21-S605, V22-S605, T23-S605, K24-S605, V25-S605, S26-S605, V27-S605, P28-S605, S29-S605, G30-S605, T31-S605, K32-S605, E33-S605, I34-S605, A35-S605, Q36-S605, L37-S605, P38-S605, L39-S605, T40-S605, P41-S605, I42-S605, P43-S605, N44-S605, P45-S605, L46-S605, I47-S605, T48-S605, S49-S605, P50-S605, K51-S605, K52-S605, Y53-S605, P54-S605, L55-S605, N56-S605, C57-S605, G58-S605, D59-S605, Q60-S605, R61-S605, N62-S605, G63-S605, H64-S605, K65-S605, S66-S605, W67-S605, L68-S605, M69-S605, H70-S605, G71-S605, L72-S605, L73-S605, M74-S605, V75-S605, H76-S605, Q77-S605, P78-S605, L79-S605, M80-S605, E81-S605, A82-S605, K83-S605, L84-S605, M85-S605, E86-S605, S87-S605, Q88-S605, S89-S605, L90-S605, E91-S605, T92-S605, W93-S605, G94-S605, W95-S605, N96-S605, G97-S605, Q98-S605, H99-S605, Q100-S605, G101-S605, R102-S605, N103-S605, H104-S605, K105-S605, I106-S605, S107-S605, I108-S605, A109-S605, L110-S605, L111-S605, A112-S605, I113-S605, K114-S605, Q115-S605, T116-S605, T117-S605, R118-S605, K119-S605, T120-S605, K121-S605, S122-S605, F123-S605, S124-S605, M125-S605, G126-S605, S127-S605, T128-S605, V129-S605, Q130-S605, S131-S605, Q132-S605, T133-S605, R134-S605, G135-S605, I136-S605, W137-S605, M138-S605, W139-S605, C140-S605, V141-S605, P142-S605, H143-S605, P144-S605, E145-S605, K146-S605, P147-S605, D148-S605, H149-S605, T150-S605, L151-S605, V152-S605, L153-S605, D154-S605, D155-S605, T156-S605, E157-S605, G158-S605, L159-S605, G160-S605, D161-S605, V162-S605, E163-S605, K164-S605, G165-S605, D166-S605, N167-S605, Q168-S605, N169-S605, D170-S605, C171-S605, W172-S605, I173-S605, F174-S605, A175-S605, L176-S605, A177-S605, I178-S605, L179-S605, L180, S181-S605, S182-S605, T183-S605, F184-S605, V185-S605, Y186-S605, N187-S605, S188-S605, I189-S605, G190-S605, A191-S605, I192-S605, N193-S605, Q194-S605, Q195-S605, A196-S605, M197-S605, D198-S605, Q199-S605, L200-S605, H201-S605, F202-S605, F203-S605, L204-S605, M205-S605, Q206-S605, H207-S605, E208-S605, M209-S605, M210-S605, L211-S605, I212-S605, S213-S605, Y214-S605, V215-S605, T216-S605, E217-S605, L218-S605, T219-S605, D220-S605, R221-S605, I222-S605, R223-S605, T224-S605, R225-S605, R226-S605, S227-S605, P228-S605, D229-S605, H230-S605, Q231-S605, A232-S605, L233-S605, E234-S605, D235-S605, S236-S605, D237-S605, E238-S605, Y239-S605, V240-S605, S241-S605, F242-S605, F243-S605, P244-S605, D245-S605, F246-S605, V247-S605, W248-S605, T249-S605, P250-S605, R251-S605, D252-S605, F253-S605, C254-S605, L255-S605, E256-S605, L257-S605, K258-S605, T259-S605, N260-S605, G261-S605, Q262-S605, P263-S605, L264-S605, S265-S605, A266-S605, D267-S605, E268-S605, Y269-S605, L270-S605, G271-S605, N272-S605, S273-S605, L274-S605, K275-S605, L276-S605, L277-S605, Q278-S605, G279-S605, C280-S605, S281-S605, Q282-S605, K283-S605, E284-S605, K285-S605, E286-S605, L287-S605, N288-S605, L289-S605, S290-S605, Q291-S605, L292-S605, C293-S605, I294-S605, R295-S605, K296-S605, F297-S605, F298-S605, P299-S605, T300-S605, K301-S605, K302-S605, C303-S605, F304-S605, V305-S605, F306-S605, E307-S605, R308-S605, P309-S605, A310-S605, P311-S605, G312-S605, K313-S605, K314-S605, I315-S605, G316-S605, Q317-S605, L318-S605, E319-S605, S320-S605, L321-S605, Q322-S605, D323-S605, K324-S605, D325-S605, L326-S605, D327-S605, S328-S605, D329-S605, F330-S605, V331-S605, K332-S605, Q333-S605, V334-S605, A335-S605, E336-S605, F337-S605, S338-S605, S339-S605, Y340-S605, V341-S605, F342-S605, R343-S605, S344-S605, S345-S605, K346-S605, I347-S605, K348-S605, K349-S605, I350-S605, P351-S605, G352-S605, D353-S605, L354-S605, K355-S605, V356-S605, N357-S605, G358-S605, P359-S605, R360-S605, L361-S605, K362-S605, N363-S605, L364-S605, V365-S605, T366-S605, T367-S605, Y368-S605, V369-S605, N370-S605, T371-S605, I372-S605, S373-S605, N374-S605, G375-S605, S376-S605, L377-S605, P378-S605, C379-S605, M380-S605, E381-S605, S382-S605, A383-S605, V384-S605, L385-S605, A386-S605, L387-S605, S388-S605, E389-S605, T390-S605, E391-S605, N392-S605, S393-S605, A394-S605, A395-S605, V396-S605, R397-S605, K398-S605, A399-S605, I400-S605, A401-S605, H402-S605, Y403-S605, D404-S605, Q405-S605, Q406-S605, M407-S605, S408-S605, Q409-S605, S410-S605, L411-S605, K412-S605, L413-S605, P414-S605, T415-S605, E416-S605, T417-S605, L418-S605, Q419-S605, E420-S605, L421-S605, L422-S605, D423-S605, L424-S605, H425-S605, R426-S605, S427-S605, S428-S605, E429-S605, K430-S605, E431-S605, A432-S605, I433-S605, K434-S605, I435-S605, F436-S605, M437-S605, E438-S605, N439-S605, S440-S605, F441-S605, K442-S605, D443-S605, V444-S605, D445-S605, Q446-S605, V447-S605, F448-S605, L449-S605, T450-S605, K451-S605, L452-S605, E453-S605, K454-S605, E455-S605, G456-S605, K457-S605, Q458-S605, R459-S605, E460-S605, F461-S605, C462-S605, K463-S605, K464-S605, N465-S605, Q466-S605, E467-S605, A468-S605, S469-S605, S470-S605, D471-S605, R472-S605, C473-S605, S474-S605, V475-S605, L476-S605, L477-S605, R478-S605, D479-S605, I480-S605, F481-S605, G482-S605, P483-S605, L484-S605, E485-S605, E486-S605, D487-S605, L488-S605, K489-S605, Q490-S605, G491-S605, V492-S605, F493-S605, Y494-S605, K495-S605, P496-S605, T497-S605, G498-S605, C499-S605, C500-S605, L501-S605, F502-S605, S503-S605, Q504-S605, K505-S605, I506-S605, Q507-S605, G508-S605, L509-S605, K510-S605, R511-S605, K512-S605, Y513-S605, E514-S605, E515-S605, P516-S605, G517-S605, K518-S605, G519-S605, A520-S605, G521-S605, N522-S605, Q523-S605, G524-S605, N525-S605, Q526-S605, G527-S605, S528-S605, A529-S605, C530-S605, P531-S605, G532-S605, K533-S605, F534-S605, L535-S605, T536-S605, I537-S605, R538-S605, L539-S605, Q540-S605, C541-S605, P542-S605, Q543-S605, A544-S605, S545-S605, L546-S605, G547-S605, N548-S605, A549-S605, S550-S605, L551-S605, C552-S605, C553-S605, S554-S605, C555-S605, I556-S605, T557-S605, Y558-S605, L559-S605, K560-S605, V561-S605, F562-S605, I563-S605, L564-S605, D565-S605, I566-S605, S567-S605, C568-S605, S569-S605, P570-S605, I571-S605, R572-S605, D573-S605, S574-S605, H575-S605, S576-S605, L577-S605, N578-S605, S579-S605, Q580-S605, T581-S605, V582-S605, T583-S605, R584-S605, L585-S605, T586-S605, T587-S605, E588-S605, L589-S605, K590-S605, L591-S605, T592-S605, T593-S605, L594-S605, A595-S605, T596-S605, S597-S605, G598-S605, T599-S605, V600-S605, T601-S605, L602-S605, C603-S605 and/or F604-S605 of SEQ ID NO:13. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal MGBPBMY2 (LOC229902) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal MGBPBMY2 (LOC229902) deletion polypeptides are encompassed by the present invention: M1-E2, M1-G3, M1-K4, M1-V5, M1-L6, M1-Q7, M1-W8, M1-N9, M1-S10, M1-Y11, M1-L12, M1-S13, M1-E14, M1-F15, M1-H16, M1-C17, M1-D18, M1-Q19, M1-D20, M1-V21, M1-V22, M1-T23, M1-K24, M1-V25, M1-S26, M1-V27, M1-P28, M1-S29, M1-G30, M1-T31, M1-K32, M1-E33, M1-I34, M1-A35, M1-Q36, M1-L37, M1-P38, M1-L39, M1-T40, M1-P41, M1-I42, M1-P43, M1-N44, M1-P45, M1-L46, M1-I47, M1-T48, M1-S49, M1-P50, M1-K51, M1-K52, M1-Y53, M1-P54, M1-L55, M1-N56, M1-C57, M1-G58, M1-D59, M1-Q60, M1-R61, M1-N62, M1-G63, M1-H64, M1-K65, M1-S66, M1-W67, M1-L68, M1-M69, M1-H70, M1-G71, M1-L72, M1-L73, M1-M74, M1-V75, M1-H76, M1-Q77, M1-P78, M1-L79, M1-M80, M1-E81, M1-A82, M1-K83, M1-L84, M1-M85, M1-E86, M1-S87, M1-Q88, M1-S89, M1-L90, M1-E91, M1-T92, M1-W93, M1-G94, M1-W95, M1-N96, M1-G97, M1-Q98, M1-H99, M1-Q100 M1-G101, M1-R102, M1-N103, M1-H104, M1-K105, M1-I106, M1-S107, M1-I108, M1-A109, M1-L110, M1-L111, M1-A112, M1-I113, M1-K114, M1-Q115, M1-T116, M1-T117, M1-R118, M1-K119, M1-T120, M1-K121, M1-S122, M1-F123, M1-S124, M1-M125, M1-G126, M1-S127, M1-T128, M1-V129, M1-Q130, M1-S131, M1-Q132, M1-T133, M1-R134, M1-G135, M1-I136, M1-W137, M1-M138, M1-W139, M1-C140, M1-V141, M1-P142, M1-H143, M1-P144, M1-E145, M1-K146, M1-P147, M1-D148, M1-H149, M1-T150, M1-L151, M1-V152, M1-L153, M1-D154, M1-D155, M1-T156, M1-E157, M1-G158, M1-L159, M1-G160, M1-D161, M1-V162, M1-E163, M1-K164, M1-G165, M1-D166, M1-N167, M1-Q168, M1-N169, M1-D170, M1-C171, M1-W172, M1-I173, M1-F174, M1-A175, M1-L176, M1-A177, M1-I178, M1-L179, M1-L180, M1-S181, M1-S182, M1-T183, M1-F184, M1-V185, M1-Y186, M1-N187, M1-S188, M1-I189, M1-G190, M1-A191, M1-I192, M1-N193, M1-Q194, M1-Q195, M1-A196, M1-M197, M1-D198, M1-Q199, M1-L200, M1-H201, M1-F202, M1-F203, M1-L204, M1-M205, M1-Q206, M1-H207, M1-E208, M1-M209, M1-M210, M1-L211, M1-I212, M1-S213, M1-Y214, M1-V215, M1-T216, M1-E217, M1-L218, M1-T219, M1-D220, M1-R221, M1-I222, M1-R223, M1-T224, M1-R225, M1-R226, M1-S227, M1-P228, M1-D229, M1-H230, M1-Q231, M1-A232, M1-L233, M1-E234, M1-D235, M1-S236, M1-D237, M1-E238, M1-Y239, M1-V240, M1-S241, M1-F242, M1-F243, M1-P244, M1-D245, M1-F246, M1-V247, M1-W248, M1-T249, M1-P250, M1-R251, M1-D252, M1-F253, M1-C254, M1-L255, M1-E256, M1-L257, M1-K258, M1-T259, M1-N260, M1-G261, M1-Q262, M1-P263, M1-L264, M1-S265, M1-A266, M1-D267, M1-E268, M1-Y269, M1-L270, M1-G271, M1-N272, M1-S273, M1-L274, M1-K275, M1-L276, M1-L277, M1-Q278, M1-G279, M1-C280, M1-S281, M1-Q282, M1-K283, M1-E284, M1-K285, M1-E286, M1-L287, M1-N288, M1-L289, M1-S290, M1-Q291, M1-L292, M1-C293, M1-I294, M1-R295, M1-K296, M1-F297, M1-F298, M1-P299, M1-T300, M1-K301, M1-K302, M1-C303, M1-F304, M1-V305, M1-F306, M1-E307, M1-R308, M1-P309, M1-A310, M1-P311, M1-G312, M1-K313, M1-K314, M1-I315, M1-G316, M1-Q317, M1-L318, M1-E319, M1-S320, M1-L321, M1-Q322, M1-D323, M1-K324, M1-D325, M1-L326, M1-D327, M1-S328, M1-D329, M1-F330, M1-V331, M1-K332, M1-Q333, M1-V334, M1-A335, M1-E336, M1-F337, M1-S338, M1-S339, M1-Y340, M1-V341, M1-F342, M1-R343, M1-S344, M1-S345, M1-K346, M1-I347, M1-K348, M1-K349, M1-I350, M1-P351, M1-G352, M1-D353, M1-L354, M1-K355, M1-V356, M1-N357, M1-G358, M1-P359, M1-R360, M1-L361, M1-K362, M1-N363, M1-L364, M1-V365, M1-T366, M1-T367, M1-Y368, M1-V369, M1-N370, M1-T371, M1-I372, M1-S373, M1-N374, M1-G375, M1-S376, M1-L377, M1-P378, M1-C379, M1-M380, M1-E381, M1-S382, M1-A383, M1-V384, M1-L385, M1-A386, M1-L387, M1-S388, M1-E389, M1-T390, M1-E391, M1-N392, M1-S393, M1-A394, M1-A395, M1-V396, M1-R397, M1-K398, M1-A399, M1-I400, M1-A401, M1-H402, M1-Y403, M1-D404, M1-Q405, M1-Q406, M1-M407, M1-S408, M1-Q409, M1-S410, M1-L411, M1-K412, M1-L413, M1-P414, M1-T415, M1-E416, M1-T417, M1-L418, M1-Q419, M1-E420, M1-L421, M1-L422, M1-D423, M1-L424, M1-H425, M1-R426, M1-S427, M1-S428, M1-E429, M1-K430, M1-E431, M1-A432, M1-I433, M1-K434, M1-I435, M1-F436, M1-M437, M1-E438, M1-N439, M1-S440, M1-F441, M1-K442, M1-D443, M1-V444, M1-D445, M1-Q446, M1-V447, M1-F448, M1-L449, M1-T450, M1-K451, M1-L452, M1-E453, M1-K454, M1-E455, M1-G456, M1-K457, M1-Q458, M1-R459, M1-E460, M1-F461, M1-C462, M1-K463, M1-K464, M1-N465, M1-Q466, M1-E467, M1-A468, M1-S469, M1-S470, M1-D471, M1-R472, M1-C473, M1-S474, M1-V475, M1-L476, M1-L477, M1-R478, M1-D479, M1-I480, M1-F481, M1-G482, M1-P483, M1-L484, M1-E485, M1-E486, M1-D487, M1-L488, M1-K489, M1-Q490, M1-G491, M1-V492, M1-F493, M1-Y494, M1-K495, M1-P496, M1-T497, M1-G498, M1-C499, M1-C500, M1-L501, M1-F502, M1-S503, M1-Q504, M1-K505, M1-I506, M1-Q507, M1-G508, M1-L509, M1-K510, M1-R511, M1-K512, M1-Y513, M1-E514, M1-E515, M1-P516, M1-G517, M1-K518, M1-G519, M1-A520, M1-G521, M1-N522, M1-Q523, M1-G524, M1-N525, M1-Q526, M1-G527, M1-S528, M1-A529, M1-C530, M1-P531, M1-G532, M1-K533, M1-F534, M1-L535, M1-T536, M1-I537, M1-R538, M1-L539, M1-Q540, M1-C541, M1-P542, M1-Q543, M1-A544, M1-S545, M1-L546, M1-G547, M1-N548, M1-A549, M1-S550, M1-L551, M1-C552, M1-C553, M1-S554, M1-C555, M1-I556, M1-T557, M1-Y557, M1-L559, M1-K560, M1-V561, M1-F562, M1-I563, M1-L564, M1-D565, M1-I566, M1-S567, M1-C568, M1-S569, M1-P570, M1-I571, M1-R572, M1-D573, M1-S574, M1-H575, M1-S576, M1-L577, M1-N578, M1-S579, M1-Q580, M1-T581, M1-V582, M1-T583, M1-R584, M1-L585, M1-T586, M1-T587, M1-E588, M1-L589, M1-K590, M1-L591, M1-T592, M1-T593, M1-L594, M1-A595, M1-T596, M1-S597, M1-G598, M1-T599, M1-V600, M1-T601, M1-L602, M1-C603 and/or M1-F604 of SEQ ID NO:13. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal MGBPBMY2 (LOC229902) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the MGBPBMY2 (LOC229902) polypeptide (e.g., any combination of both N- and C-terminal MGBPBMY2 (LOC229902) polypeptide deletions) of SEQ ID NO:13. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of MGBPBMY2 (LOC229902) (SEQ ID NO:13), and where CX refers to any C-terminal deletion polypeptide amino acid of MGBPBMY2 (LOC229902) (SEQ ID NO:13). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the MGBPBMY2 (LOC229902) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate MGBPBMY2 (LOC229902) on the one hand (i.e., agonists) and which inhibit the function of MGBPBMY2 (LOC229902) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the MGBPBMY2 (LOC229902) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n     Features of the Polypeptide Encoded by Gene No. 7 \\n     A polypeptide encoded by this gene, MGBPBMY3 (BC031475), is provided as SEQ ID NO:15 ( FIGS. 7A–7C ) and is encoded by the polynucleotide sequence according to SEQ ID NO:14 ( FIGS. 7A–7C ) and/or by a polynucleotide contained within a deposited clone. MGBPBMY3 (BC031475) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, mouse GBP-1, mouse GBP-2, mouse GBP-3 and mouse GBP-4. \\n     The determined nucleotide sequence of the MGBPBMY3 (BC031475), (i.e. the cDNA shown in  FIGS. 7A–7C  and in SEQ ID NO:14) comprises an open reading frame encoding a protein of about 385 amino acid residues. The predicted amino acid sequence of the MGBPBMY3 (BC031475) polypeptide is shown in  FIGS. 7A–7C  (SEQ ID NO:15). The percent identity and similarity values between the MGBPBMY3 (BC031475) polypeptide to the known GBP family member mGBP1 is provided in  FIG. 9 . The MGBPBMY3 (BC031475) protein shown in  FIGS. 7A–7C  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the MGBPBMY3 (BC031475) polypeptide showed experession in spleen and lung (see  FIG. 26 ). \\n     Based upon the strong homology to members of the GBP family members, the MGBPBMY3 (BC031475) polypeptide is expected to share at least some biological activity with GBP family members, specifically mGBP-1, mGBP-2, mGBP3 and mGBP-4. \\n     The MGBPBMY3 (BC031475) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, diagnosing treating, preventing, and/or ameliorating at least diseases and conditions of the immune system and lung in a mouse model of a human condition. \\n     The MGBPBMY3 (BC031475) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue and more preferably in a mouse model of a human condition. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with MGBPBMY3 (BC031475) expression in some immune system tissues (i.e., spleen) suggests a potential utility for MGBPBMY3 (BC031475) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. In representative embodiments, MGBPBMY3 (BC031475) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. The MGBPBMY3 (BC031475) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. A MGBPBMY3 (BC031475) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a MGBPBMY3 (BC031475) polypeptide of the present invention may be useful in the treatment of include various autoimmune diseases in a mouse model of a human condition, such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A MGBPBMY3 (BC031475) polypeptide of the present invention amy also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases in a mouse model of a human condition. Such diseases and conditions include Recombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with observed MGBPBMY3 (BC031475) expression levels in heart tissue suggests the MGBPBMY3 (BC031475) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders in a mouse model of a human condition, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, MGBPBMY3 (BC031475) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms in a mouse model of a human condition which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections in a mouse model of a human condition: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus  viridans infection,  Staphylococcus aureus  infection, coagulase-negative  staphylococci  infection, gram-negative  Bacilli  infection,  Enterobacteriaceae  infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative  coccobacilli  infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous Mycobacteria infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY3 (BC031475) expression in lung tissue suggests a potential utility for MGBPBMY3 (BC031475) polynucleotides and polypeptides in in treating, diagnosing, prognosing, and/or preventing pulmonary diseases and disorders in a mouse model of a human condition which include the following, non-limiting examples: ARDS, emphysema, cystic fibrosis, interstitial lung disease, chronic obstructive pulmonary disease, bronchitis, lymphangioleiomyomatosis, pneumonitis, eosinophilic pneumonias, granulomatosis, pulmonary infarction, pulmonary fibrosis, pneumoconiosis, alveolar hemorrhage, neoplasms, lung abscesses, empyema, and increased susceptibility to lung infections (e.g., immumocompromised, HIV, etc.), for example. \\n     Moreover, MGBPBMY3 (BC031475) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, pulmonary infections in a mouse model of a human condition: pnemonia, bacterial pnemonia, viral pnemonia (for example, as caused by Influenza virus, Respiratory syncytial virus, Parainfluenza virus, Adenovirus, Coxsackievirus, Cytomegalovirus, Herpes simplex virus, Hantavirus, etc.), mycobacteria pnemonia (for example, as caused by  Mycobacterium tuberculosis , etc.) mycoplasma pnemonia, fungal pnemonia (for example, as caused by  Pneumocystis carinii, Histoplasma capsulatum, Coccidioides immitis, Blastomyces dermatitidis, Candida  sp.,  Cryptococcus neoformans, Aspergillus  sp., Zygomycetes, etc.), Legionnaires&#39; Disease,  Chlamydia pnemonia , aspiration pnemonia,  Nocordia  sp. Infections, parasitic pnemonia (for example, as caused by Strongyloides,  Toxoplasma gondii , etc.) necrotizing pnemonia, in addition to any other pulmonary disease and/or disorder (e.g., non-pneumonia) implicated by the causative agents listed above or elsewhere herein. \\n     It is noted that the use of mouse models to understand, diagnose, predict, treat and/or ameliorate human conditions is well documented. Thus, the uses for the MGBPBMY3 (BC031475) can often be extrapolated to human conditions, as well as to further research such conditions and their treatments. \\n     A MGBPBMY3 (BC031475) protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a MGBPBMY3 (BC031475) protein, can show utility as a tumor marker and/or immunotherapy targets for spleen and lung brain tissue. \\n     The MGBPBMY3 (BC031475) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of MGBPBMY3 (BC031475) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the MGBPBMY3 (BC031475) protein could be used as diagnostic agents of certain conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     MGBPBMY3 (BC031475) polypeptides and polynucleotides have additional uses which include diagnosing diseases related to the over and/or under expression of MGBPBMY3 (BC031475) by identifying mutations in the MGBPBMY3 (BC031475) gene by using MGBPBMY3 (BC031475) sequences as probes or by determining MGBPBMY3 (BC031475) protein or mRNA expression levels. MGBPBMY3 (BC031475) polypeptides can be useful for screening compounds that affect the activity of the protein. MGBPBMY3 (BC031475) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with MGBPBMY3 (BC031475), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the MGBPBMY3 (BC031475) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased spleen tissue, as compared to normal tissue might indicate a function in modulating spleen function, for example. In the case of MGBPBMY3 (BC031475), spleen and/or lung tissues, can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the MGBPBMY3 (BC031475) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of MGBPBMY3 (BC031475), a disease correlation related to MGBPBMY3 (BC031475) can be made by comparing the mRNA expression level of MGBPBMY3 (BC031475) in normal tissue, as compared to diseased tissue. Significantly higher or lower levels of MGBPBMY3 (BC031475) expression in the diseased tissue can suggest MGBPBMY3 (BC031475) plays a role in disease progression, and antagonists against MGBPBMY3 (BC031475) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of MGBPBMY3 (BC031475) expression in the diseased tissue can suggest MGBPBMY3 (BC031475) plays a defensive role against disease progression, and agonists of MGBPBMY3 (BC031475) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:14 ( FIGS. 7A–7C ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the MGBPBMY3 (BC031475), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the MGBPBMY3 (BC031475) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay can be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in another species (e.g., a mammalian species) and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and homologous recombination) and the resulting generation of transgenic animals. Expressing a particular gene in either sense or antisense orientation in a transgenic animal could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of MGBPBMY3 (BC031475), transgenic animals, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic animals to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal MGBPBMY3 (BC031475) deletion polypeptides are encompassed by the present invention: M1-C385, T2-C385, Q3-C385, P4-C385, Q5-C385, M6-C385, A7-C385, P8-C385, I9-C385, C10-C385, L11-C385, V12-C385, E13-C385, N14-C385, H15-C385, N16-C385, E17-C385, H18-C385, L19-C385, S20-C385, M21-C385, N22-C385, H23-C385, E24-C385, A25-C385, I26-C385, E27-C385, I28-C385, L29-C385, E30-C385, K31-C385, I32-C385, S33-C385, Q34-C385, P35-C385, V36-C385, V37-C385, V38-C385, V39-C385, A40-C385, I41-C385, V42-C385, G43-C385, L44-C385, Y45-C385, R46-C385, T47-C385, G48-C385, K49-C385, S50-C385, Y51-C385, L52-C385, M53-C385, N54-C385, R55-C385, L56-C385, A57-C385, G58-C385, Q59-C385, N60-C385, H61-C385, G62-C385, F63-C385, P64-C385, L65-C385, G66-C385, S67-C385, T68-C385, V69-C385, Q70-C385, S71-C385, Q72-C385, T73-C385, K74-C385, G75-C385, I76-C385, W77-C385, M78-C385, W79-C385, C80-C385, M81-C385, P82-C385, H83-C385, P84-C385, T85-C385, K86-C385, P87-C385, E88-C385, H89-C385, T90-C385, L91-C385, V92-C385, L93-C385, L94-C385, D95-C385, T96-C385, E97-C385, G98-C385, L99-C385, G100-C385, D101-C385, V102-C385, E103-C385, K104-C385, G105-C385, D106-C385, P107-C385, K108-C385, N109-C385, D110-C385, L111-C385, W112-C385, I113-C385, F114-C385, A115-C385, L116-C385, G117-C385, V118-C385, L119-C385, L120-C385, S121-C385, S122-C385, T123-C385, F124-C385, I125-C385, Y126-C385, N127-C385, S128-C385, M129-C385, N130-C385, T131-C385, I132-C385, S133-C385, H134-C385, D135-C385, S136-C385, L137-C385, E138-C385, K139-C385, L140-C385, H141-C385, Y142-C385, V143-C385, T144-C385, E145-C385, L146-C385, T147-C385, E148-C385, L149-C385, I150-C385, R151-C385, A152-C385, K153-C385, S154-C385, S155-C385, P156-C385, N157-C385, P158-C385, D159-C385, G160-C385, I161-C385, K162-C385, N163-C385, S164-C385, T165-C385, E166-C385, F167-C385, V168-C385, S169-C385, F170-C385, F171-C385, P172-C385, D173-C385, F174-C385, V175-C385, W176-C385, T177-C385, V178-C385, R179-C385, D180-C385, F181-C385, M182-C385, L183-C385, E184-C385, L185-C385, K186-C385, L187-C385, N188-C385, G189-C385, E190-C385, D191-C385, I192-C385, T193-C385, S194-C385, D195-C385, E196-C385, Y197-C385, L198-C385, E199-C385, N200-C385, A201-C385, L202-C385, K203-C385, L204-C385, I205-C385, P206-C385, G207-C385, Y208-C385, N209-C385, P210-C385, R211-C385, V212-C385, Q213-C385, A214-C385, S215-C385, N216-C385, S217-C385, A218-C385, R219-C385, E220-C385, C221-C385, I222-C385, R223-C385, C224-C385, F225-C385, F226-C385, P227-C385, N228-C385, R229-C385, K230-C385, C231-C385, F232-C385, V233-C385, F234-C385, D235-C385, R236-C385, P237-C385, T238-C385, H239-C385, D240-C385, R241-C385, E242-C385, L243-C385, L244-C385, Q245-C385, K246-C385, L247-C385, E248-C385, T249-C385, I250-C385, S251-C385, E252-C385, D253-C385, Q254-C385, L255-C385, D256-C385, L257-C385, K258-C385, F259-C385, R260-C385, E261-C385, E262-C385, T263-C385, N264-C385, A265-C385, F266-C385, V267-C385, S268-C385, Y269-C385, I270-C385, F271-C385, N272-C385, Y273-C385, A274-C385, K275-C385, I276-C385, K277-C385, T278-C385, L279-C385, K280-C385, E281-C385, G282-C385, I283-C385, K284-C385, V285-C385, T286-C385, G287-C385, N288-C385, G289-C385, L290-C385, G291-C385, I292-C385, L293-C385, V294-C385, T295-C385, T296-C385, Y297-C385, V298-C385, D299-C385, A300-C385, I301-C385, N302-C385, S303-C385, G304-C385, A305-C385, V306-C385, P307-C385, C308-C385, V309-C385, D310-C385, D311-C385, A312-C385, V313-C385, T314-C385, T315-C385, L316-C385, A317-C385, Q318-C385, H319-C385, E320-C385, N321-C385, S322-C385, V323-C385, A324-C385, V325-C385, Q326-C385, P327-C385, A328-C385, A329-C385, D330, H331-C385, Y332-C385, S333-C385, E334-C385, Q335-C385, M336-C385, V337-C385, Q338-C385, R339-C385, L340-C385, S341-C385, L342-C385, P343-C385, T344-C385, D345-C385, T346-C385, L347-C385, Q348-C385, E349-C385, L350-C385, L351-C385, D352-C385, V353-C385, H354-C385, A355-C385, A356-C385, C357-C385, E358-C385, K359-C385, E360-C385, A361-C385, M362-C385, A363-C385, V364-C385, F365-C385, M366-C385, E367-C385, H368-C385, S369-C385, F370-C385, K371-C385, D372-C385, E373-C385, N374-C385, Q375-C385, Q376-C385, F377-C385, L378-C385, K379-C385, K380-C385, L381-C385, V382-C385, V383-C385 and/or 1384-C385, of SEQ ID NO:15. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal MGBPBMY3 (BC031475) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal MGBPBMY3 (BC031475) deletion polypeptides are encompassed by the present invention: M1-T2, M1-Q3, M1-P4, M1-Q5, M1-M6, M1-A7, M1-P8, M1-I9, M1-C10, M1-L11, M1-V12, M1-E13, M1-N14, M1-H15, M1-N16, M1-E17, M1-H18, M1-L19, M1-S20, M1-M21 M1-N22, M1-H23 M1-E24, M1-A25, M1-I26, M1-E27, M1-I28, M1-L29, M1-E30, M1-K31, M1-I32, M1-S33, M1-Q34, M1-P35, M1-V36, M1-V37, M1-V38, M1-V39, M1-A40, M1-I41, M1-V42, M1-G43, M1-L44, M1-Y45, M1-R46, M1-T47, M1-G48, M1-K49, M1-S50, M1-Y51, M1-L52, M1-M53, M1-N54, M1-R55, M1-L56, M1-A57, M1-G58, M1-Q59, M1-N60, M1-H61, M1-G62, M1-F63, M1-P64, M1-L65, M1-G66, M1-S67, M1-T68, M1-V69, M1-Q70, M1-S71, M1-Q72, M1-T73, M1-K74, M1-G75, M1-I76, M1-W77, M1-M78, M1-W79, M1-C80, M1-M81, M1-P82, M1-H83, M1-P84, M1-T85, M1-K86, M1-P87, M1-E88, M1-H89, M1-T90, M1-L91, M1-V92, M1-L93, M1-L94, M1-D95, M1-T96, M1-E97, M1-G98, M100, M1-D10, M1-V102, M1-E103, M1-K104, M1-G105, M1-D106, M1-P107, M1-K108, M1-N109, M1-D110, M1-L111, M1-W112, M1-I113, M1-F114, M1-A115, M1-L116, M1-G117, M1-V118, M1-L119, M1-L120, M1-S121, M1-S122, M1-T123, M1-F124, M1-I125, M1-Y126, M1-N127, M1-S128, M1-M129, M1-N130, M1-T131, M1-I132, M1-S133, M1-H134, M1-D135, M1-S136, M1-L137, M1-E138, M1-K139, M1-L140, M1-H141, M1-Y142, M1-V143, M1-T144, M1-E145, M1-L146, M1-T147, M1-E148, M1-L149, M1-I150, M1-R151, M1-A152, M1-K153, M1-S154, M1-S155, M1-P156, M1-N157, M1-P158, M1-D159, M1-G160, M1-I161, M1-K162, M1-N163, M1-S164, M1-T165, M1-E166, M1-F167, M1-V168, M1-S169, M1-F170, M1-F171, M1-P172, M1-D173, M1-F174, M1-V175, M1-W176, M1-T177, M1-V178, M1-R179, M1-D180, M1-F181, M1-M182, M1-L183, M1-E184, M1-L185, M1-K186, M1-L187, M1-N188, M1-G189, M1-E190, M1-D191, M1-I192, M1-T193, M1-S194, M1-D195, M1-E196, M1-Y197, M1-L198, M1-E199, M1-N200, M1-A201, M1-L202, M1-K203, M1-L204, M1-I205, M1-P206, M1-G207, M1-Y208, M1-N209, M1-P210, M1-R211, M1-V212, M1-Q213, M1-A214, M1-S215, M1-N216, M1-S217, M1-A218, M1-R219, M1-E220, M1-C221, M1-I222, M1-R223, M1-C224, M1-F225, M1-F226, M1-P227, M1-N228, M1-R229, M1-K230, M1-C231, M1-F232, M1-V233, M1-F234, M1-D235, M1-R236, M1-P237, M1-T238, M1-H239, M1-D240, M1-R241, M1-E242, M1-L243, M1-L244, M1-Q245, M1-K246, M1-L247, M1-E248, M1-T249, M1-I250, M1-S251, M1-E252, M1-D253, M1-Q254, M1-L255, M1-D256, M1-L257, M1-K258, M1-F259, M1-R260, M1-E261, M1-E262, M1-T263, M1-N264, M1-A265, M1-F266, M1-V267, M1-S268, M1-Y269, M1-I270, M1-F271, M1-N272, M1-Y273, M1-A274, M1-K275, M1-I276, M1-K277, M1-T278, M1-L279, M1-K280, M1-E281, M1-G282, M1-I283, M1-K284, M1-V285, M1-T286, M1-G287, M1-N288, M1-G289, M1-L290, M1-G291, M1-I292, M1-L293, M1-V294, M1-T295, M1-T296, M1-Y297, M1-V298, M1-D299, M1-A300, M1-I301, M1-N302, M1-S303, M1-G304, M1-A305, M1-V306, M1-P307, M1-C308, M1-V309, M1-D310, M1-D311, M1-A312, M1-V313, M1-T314, M1-T315, M1-L316, M1-A317, M1-Q318, M1-H319, M1-E320, M1-N321, M1-S322, M1-V323, M1-A324, M1-V325, M1-Q326, M1-R327, M1-A328, M1-A329, M1-D330, M1-H331, M1-Y332, M1-S333, M1-E334, M1-Q335, M1-M336, M1-V337, M1-Q338, M1-R339, M1-L340, M1-S341, M1-L342, M1-P343, M1-T344, M1-D345, M1-T346, M1-L347, M1-Q348, M1-E349, M1-L350, M1-L351, M1-D352, M1-V353, M1-H354, M1-A355, M1-A356, M1-C357, M1-E358, M1-K359, M1-E360, M1-A361, M1-M362, M1-A363, M1-V364, M1-F365, M1-M366, M1-E367, M1-H368, M1-S369, M1-F370, M1-K371, M1-D372, M1-E373, M1-N374, M1-Q375, M1-Q376, M1-F377, M1-L378, M1-K379, M1-K380, M1-L381, M1-V382, M1-V383 and/or M1-I384 of SEQ ID NO:15. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal MGBPBMY3 (BC031475) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the MGBPBMY3 (BC031475) polypeptide (e.g., any combination of both N- and C-terminal MGBPBMY3 (BC031475) polypeptide deletions) of SEQ ID NO:15. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of MGBPBMY3 (BC031475) (SEQ ID NO:15), and where CX refers to any C-terminal deletion polypeptide amino acid of MGBPBMY3 (BC031475) (SEQ ID NO:15). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the MGBPBMY3 (BC031475) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate MGBPBMY3 (BC031475) on the one hand (i.e., agonists) and which inhibit the function of MGBPBMY3 (BC031475) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the MGBPBMY3 (BC031475) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n     Features of the Polypeptide Encoded by Gene No. 8 \\n     A polypeptide encoded by this gene, MGBPBMY4 (BC007143), is provided as SEQ ID NO:17 ( FIGS. 8A–8D ) and is encoded by the polynucleotide sequence according to SEQ ID NO:16 ( FIGS. 8A–8D ) and/or by a polynucleotide contained within a deposited clone. MGBPBMY4 (BC007143) has significant homology at the nucleotide and amino acid level to a number of guanylate binding proteins, which include, for example, mouse GBP-1, mouse GBP-2, mouse GBP-3 and mouse GBP-4. \\n     The determined nucleotide sequence of the MGBPBMY4 (BC007143), (i.e. the cDNA shown in  FIGS. 8A–8D  and in SEQ ID NO:16) comprises an open reading frame encoding a protein of about 483 amino acid residues. The predicted amino acid sequence of the MGBPBMY4 (BC007143) polypeptide is shown in  FIGS. 8A–8D  (SEQ ID NO:17). The percent identity and similarity values between the MGBPBMY4 (BC007143) polypeptide to the known GBP family member mGBP1 is provided in  FIG. 9 . The MGBPBMY4 (BC007143) protein shown in  FIGS. 8A–8D  was determined to share significant identity and similarity to several known GBP family members, as shown in  FIG. 11A–11G . \\n     Expression profiling designed to measure the steady state mRNA levels encoding the MGBPBMY4 (BC007143) polypeptide showed experession in heart, kidney, spleen, liver, brain and skeletal muscle (see  FIG. 27 ). \\n     Based upon the strong homology to members of the GBP family members, the MGBPBMY4 (BC007143) polypeptide is expected to share at least some biological activity with GBP family members, specifically mGBP-1, mGBP-2, mGBP3 and mGBP-4. \\n     The MGBPBMY4 (BC007143) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include detecting, prognosing, treating, preventing, and/or ameliorating at least diseases and conditions of the immune system (e.g. spleen), heart, kidney, liver, brain and/or skeletal muscle in a mouse model of a human condition. \\n     The MGBPBMY4 (BC007143) polynucleotides and polypeptides of the present invention, including agonists and/or fragments thereof, may have uses that include modulating signal transduction activity, in various cells, tissues, and organisms, and particularly in mammalian tissue and more preferably in a mouse model of a human condition. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with MGBPBMY4 (BC007143) expression in some immune system tissues (i.e., spleen) suggests a potential utility for MGBPBMY4 (BC007143) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. In representative embodiments, MGBPBMY4 (BC007143) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing diseases and conditions of the immune system in a mouse model of a human condition. A MGBPBMY4 (BC007143) polypeptide may also be useful as a preventative agent for immunological disorders including arthritis, asthma, immunodeficiency diseases such as AIDS, leukemia, rheumatoid arthritis, granulomatous disease, inflammatory bowel disease, sepsis, acne, neutropenia, neutrophilia, psoriasis, hypersensitivities, such as T-cell mediated cytotoxicity; immune reactions to transplanted organs and tissues, such as host-versus-graft and graft-versus-host diseases, or autoimmunity disorders, such as autoimmune infertility, lense tissue injury, demyelination, drug induced hemolytic anemia, and scleroderma. A MGBPBMY4 (BC007143) polypeptide may also be useful for modulating cytokine production, antigen presentation, or other processes, such as for boosting immune responses, etc. \\n     Additional immunolgical disorders that a MGBPBMY4 (BC007143) polypeptide of the present invention may be useful in the treatment of include various autoimmune diseases in a mouse model of a human condition, such as Myasthenia gravis, Antiphospholipid syndrome, Insulin-resistant diabetes mellitus, Pernicious anemia, Graves&#39; disease, Wegener&#39;s granulomatosis, Pemphigus vulgaris, Goodpastures&#39; syndrome, Systemic lupus erythematosus (SLE), Rheumatoid arthritis, Autoimmune thrombocytopenic purpura, Autoimmune hemolytic anemia, Hashimoto&#39;s thyroiditis, Multiple sclerosis, Insulin-dependent diabetes mellitus, Autoimmune polyglandular syndrome, Immune-mediated infertility, Autoimmune Addison&#39;s disease, Pemphigus foliaceus, Dermatitis herpetiformis, Autoimmune alopecia, Vitiligo, Guillain-Barré syndrome, Stiff-man syndrome, Acute rheumatic fever, Sympathetic ophthalmia, Systemic necrotizing vasculitis, Sjögren&#39;s syndrome. \\n     A MGBPBMY4 (BC007143) polypeptide of the present invention may also be useful in treating or ameliorating primary immune diseases, as well as immune diseases associated with or secondary to other diseases in a mouse model of a human condition. Such diseases and conditions include Recombinase activating gene (RAG 1/2) deficiency, Adenosine deaminase (ADA) deficiency, Interleukin receptor chain (c) deficiency, Janus-associated kinase 3 (JAK3) deficiency, Reticular dysgenesis, DiGeorge syndrome, Nude syndrome, T cell receptor deficiency, MHC class II deficiency, TAP-2 deficiency (MHC class I deficiency), ZAP70 tyrosine kinase deficiency, Purine nucleotide phosphorylase (PNP) deficiency, X-linked agammaglobulinemia (Bruton&#39;s tyrosine kinase deficiency), Autosomal recessive agammaglobulinemia: Mu heavy chain deficiency, Surrogate light chain (5/14.1) deficiency), Hyper-IgM syndrome: X-linked (CD40 ligand deficiency), Ig heavy chain gene deletions, IgA deficiency, Selective deficiency of IgG subclasses (with or without IgA deficiency), Common variable immunodeficiency (CVID), Antibody deficiency with normal immunoglobulins, Transient hypogammaglobulinemia of infancy, Interferon receptor (IFNGR1, IFNGR2) deficiency, Interleukin 12 and interleukin 12 receptor deficiency, Immunodeficiency with thymoma, Wiskott-Aldrich syndrome (WAS protein deficiency), Ataxia telangiectasia (ATM deficiency), X-linked lymphoproliferative syndrome (SH2D1A/SAP deficiency), Hyper IgE syndrome, Bloom syndrome, Xeroderma pigmentosum, Fanconi anemia, ICF syndrome, Nijmegen breakage syndrome, Seckel syndrome, Down syndrome (Trisomy 21), Turner syndrome, Deletions or rings of chromosome 18 (18p- and 18q-), Short-limbed skeletal dysplasia (short-limbed dwarfism), Cartilage-hair hypoplasia (metaphyseal chondroplasia), Schimke immuno-osseous dysplasia, Dubowitz syndrome, Kyphomelic dysplasia with SCID, Mulibrey&#39;s nannism, Growth retardation, facial anomalies and immunodeficiency, Progeria (Hutchinson-Gilford syndrome), Ectrodactyly-ectodermal dysplasia-clefting syndrome, Immunodeficiency with absent thumbs, anosmia and ichthyosis, Partial albinism, Dyskeratosis congenita, Netherton syndrome, Anhidrotic ectodermal dysplasia, Papillon-Lefevre syndrome, Congenital ichthyosis, Acrodermatitis enteropathica, Transcobalamin 2 deficiency, Type 1 hereditary orotic aciduria, Intractable diarrhea, abnormal facies, trichorrhexis and immunodeficiency, Methylmalonic acidemia, Biotin dependent carboxylase deficiency, Mannosidosis, Glycogen storage disease, type 1b, Chediak-Higashi syndrome, Familial hypercatabolism, Intestinal lymphangiectasia, Chronic muco-cutaneous candidiasis, Hereditary or congenital hyposplenia or asplenia, Ivermark syndrome. \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with observed MGBPBMY4 (BC007143) expression levels in heart tissue suggests the MGBPBMY4 (BC007143) polynucleotides and polypeptides may be useful in treating, diagnosing, prognosing, and/or preventing cardiovascular diseases and/or disorders in a mouse model of a human condition, which include, but are not limited to: myocardio infarction, congestive heart failure, arrthymias, cardiomyopathy, atherosclerosis, arterialsclerosis, microvascular disease, embolism, thromobosis, pulmonary edema, palpitation, dyspnea, angina, hypotension, syncope, heart murmer, aberrant ECG, hypertrophic cardiomyopathy, the Marfan syndrome, sudden death, prolonged QT syndrome, congenital defects, cardiac viral infections, valvular heart disease, and hypertension. \\n     Similarly, MGBPBMY4 (BC007143) polynucleotides and polypeptides may be useful for treating and/or ameliorating cardiovascular diseases and symptoms in a mouse model of a human condition which result indirectly from various non-cardiavascular effects, which include, but are not limited to, the following, obesity, Down syndrome (associated with endocardial cushion defect); bony abnormalities of the upper extremities (associated with atrial septal defect in the Holt-Oram syndrome); muscular dystrophies (associated with cardiomyopathy); hemochromatosis and glycogen storage disease (associated with myocardial infiltration and restrictive cardiomyopathy); congenital deafness (associated with prolonged QT interval and serious cardiac arrhythmias); Raynaud&#39;s disease (associated with primary pulmonary hypertension and coronary vasospasm); connective tissue disorders, i.e., the Marfan syndrome, Ehlers-Danlos and Hurler syndromes, and related disorders of mucopolysaccharide metabolism (aortic dilatation, prolapsed mitral valve, a variety of arterial abnormalities); acromegaly (hypertension, accelerated coronary atherosclerosis, conduction defects, cardiomyopathy); hyperthyroidism (heart failure, atrial fibrillation); hypothyroidism (pericardial effusion, coronary artery disease); rheumatoid arthritis (pericarditis, aortic valve disease); scleroderma (cor pulmonale, myocardial fibrosis, pericarditis); systemic lupus erythematosus (valvulitis, myocarditis, pericarditis); sarcoidosis (arrhythmias, cardiomyopathy); postmenopausal effects, Chlamydial infections, polycystic ovary disease, thyroid disease, alcoholism, diet, and exfoliative dermatitis (high-output heart failure), for example. \\n     Moreover, polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, cardiovascular infections in a mouse model of a human condition: blood stream invasion, bacteremia, sepsis,  Streptococcus pneumoniae  infection, group a  streptococci  infection, group b  streptococci  infection,  Enterococcus  infection, nonenterococcal group D  streptococci  infection, nonenterococcal group C  streptococci  infection, nonenterococcal group G  streptococci  infection,  Streptoccus viridans  infection,  Staphylococcus aureus  infection, coagulase-negative  staphylococci  infection, gram-negative  Bacilli  infection,  Enterobacteriaceae  infection,  Psudomonas  spp. Infection,  Acinobacter  spp. Infection,  Flavobacterium meningosepticum  infection,  Aeromonas  spp. Infection,  Stenotrophomonas maltophilia  infection, gram-negative coccobacilli infection,  Haemophilus influenza  infection,  Branhamella catarrhalis  infection, anaerobe infection,  Bacteriodes fragilis  infection,  Clostridium  infection, fungal infection,  Candida  spp. Infection, non-albicans  Candida  spp. Infection,  Hansenula anomala  infection,  Malassezia furfur  infection, nontuberculous  Mycobacteria  infection,  Mycobacterium avium  infection,  Mycobacterium chelonae  infection,  Mycobacterium fortuitum  infection, spirochetal infection,  Borrelia burgdorferi  infection, in addition to any other cardiovascular disease and/or disorder (e.g., non-sepsis) implicated by the causative agents listed above or elsewhere herein. \\n     Further, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY4 (BC007143) expression in skeletal muscle suggests a potential utility for MGBPBMY4 (BC007143) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing skeletal muscle disorders in a mouse model of a human condition. In representative embodiments, MGBPBMY4 (BC007143) polynucleotides and polypeptides including agonists and fragements thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of skeletal muscled: dystrophies, pseudohypertrophic muscular dystrophy, Duchenne dystrophy, Becker muscular dystrophy, limb-girdle muscular dystrophy, muscle weakness, Emery-Dreifuss muscular dystrophy, Congenital muscular dystrophy, endometriosis, placental aromatase deficiency, premature menopause, Fukuyama congenital muscular dystrophy, laminin alpha 2 chain deficiency, alpha 7 integrin deficiency, Walker-Warburg syndrome, myotonic dystrophy, congenital myotonic dystrophy, facioscapulohumeral muscular dystrophy, distal myopathies, central core disease, nemaline (rod) myopathy, centronuclear (myotubular) myopathy, central core disease, delay in motor milestones, delayed walking, nemaline myopathy, congenital nemaline myopathy, muscle hypotonia, centronuclear myopathies, skeletal muscle energy metabolism disorders, disorders associated with aberrant skeletal muscle-fatty acid metabolism, disorders associated with aberrant skeletal glucose metabolism, acid maltase deficiency, debranching enzyme deficiency, branching enzyme deficiency, exercise intolerance, myophosphorylase deficiency (type V glycogenosis), phosphofructokinase deficiency (type VII glycogenosis), phosphoglycerate kinase deficiency (type IX glycogenosis), phosphoglycerate mutase deficiency (type X glycogenosis), lactate dehydrogenase deficiency (glycogensosis type XI), glycogen storage disorders, skeletal muscle lipid metabolism, carnitine deficiency, myoglobinuria, muscle cramping, myoadenylate deaminase deficiency, mitochondrial myopathies, Kearns-Sayre syndrome, myoclonic epilepsy, disorders of muscle membrane excitability, calcium channel disorders of muscle, sodium channel disorders of muscle, hyperkalemic periodic paralysis, paramyotonia congenita, potassium-aggravated myotonia, myotonia congenita, chloride channel disorders of muscle, thyrotoxic periodic paralysis, and/or Andersen&#39;s syndrome. \\n     Further, the strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY4 (BC007143) expression in kidney suggests a potential utility for MGBPBMY4 (BC007143) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the kidney in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY4 (BC007143) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, diseases or disorders of the kidney in a mouse model of a human condition: Plasma cell infiltration, Hypercalcemia, Myeloma kidney, Amyloidosis, Light chain deposition disease, Type I/II cryoglobulinemia, Immunotactoid glomerulopathy, Reduced glomerular filtration rate, Fanconi syndrome, Hyperchloremic acidosisa, Tubular or small-molecular-weight proteinuria, Polyuria, isothenuria, Hyperkalemia, Salt wasting, Nephrocalcinosis, hyperoxaluria, Cystinosis, Fabry&#39;s disease, Sjogren&#39;s Syndrome \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP3 and mGBP-4, combined with the observed MGBPBMY4 (BC007143) expression in brain suggests a potential utility for MGBPBMY4 (BC007143) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing diseases and disorders of the brain and neurological tissue in a mouse model of a human condition. \\n     In representative embodiments, MGBPBMY4 (BC007143) polynucleotides and polypeptides including agonists, antagonists, and fragments thereof, may have uses which include treating, diagnosing, prognosing, and/or preventing the following, non-limiting, brain and neurological diseases or disorders in a mouse model of a human condition: the following lesions of either the central (including spinal cord, brain) or peripheral nervous systems: (1) ischemic lesions, in which a lack of oxygen in a portion of the nervous system results in neuronal injury or death, including cerebral infarction or ischemia, or spinal cord infarction or ischemia; (2) traumatic lesions, including lesions caused by physical injury or associated with surgery, for example, lesions which sever a portion of the nervous system, or compression injuries; (3) malignant lesions, in which a portion of the nervous system is destroyed or injured by malignant tissue which is either a nervous system associated malignancy or a malignancy derived from non-nervous system tissue; (4) infectious lesions, in which a portion of the nervous system is destroyed or injured as a result of infection, for example, by an abscess or associated with infection by human immunodeficiency virus, herpes zoster, or herpes simplex virus or with Lyme disease, tuberculosis, syphilis; (5) degenerative lesions, in which a portion of the nervous system is destroyed or injured as a result of a degenerative process including but not limited to degeneration associated with Parkinson&#39;s disease, Alzheimer&#39;s disease, Huntington&#39;s chorea, or amyotrophic lateral sclerosis (ALS); (6) lesions associated with nutritional diseases, disorders, and/or conditions, in which a portion of the nervous system is destroyed or injured by a nutritional disorder or disorder of metabolism including but not limited to, vitamin B12 deficiency, folic acid deficiency, Wemicke disease, tobacco-alcohol amblyopia, Marchiafava-Bignami disease (primary degeneration of the corpus callosum), and alcoholic cerebellar degeneration; (7) neurological lesions associated with systemic diseases including, but not limited to, diabetes (diabetic neuropathy, Bell&#39;s palsy), systemic lupus erythematosus, carcinoma, or sarcoidosis; (8) lesions caused by toxic substances including alcohol, lead, or particular neurotoxins; and (9) demyelinated lesions in which a portion of the nervous system is destroyed or injured by a demyelinating disease including, but not limited to, multiple sclerosis, human immunodeficiency virus-associated myelopathy, transverse myelopathy or various etiologies, progressive multifocal leukoencephalopathy, and central pontine myelinolysis. \\n     In a representative embodiment, the MGBPBMY4 (BC007143) polypeptides, polynucleotides, or agonists or antagonists of the present invention may be used to protect neural cells from the damaging effects of cerebral hypoxia. According to this embodiment, the compositions of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral hypoxia in a mouse model of a human condition. In one aspect of this embodiment, the MGBPBMY4 (BC007143) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral ischemia. In another aspect of this embodiment, the MGBPBMY4 (BC007143) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with cerebral infarction in a mouse model of a human condition. In another aspect of this embodiment, the polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose or prevent neural cell injury associated with a stroke. In a further aspect of this embodiment, the MGBPBMY4 (BC007143) polypeptides, polynucleotides, or agonists or antagonists of the invention are used to treat, prevent, and/or diagnose neural cell injury associated with a heart attack. \\n     The MGBPBMY4 (BC007143) polypeptides and/or polynucleotides of the present invention which are useful for treating or preventing a nervous system disorder may be selected by testing for biological activity in promoting the survival or differentiation of neurons in a mouse model of a human condition. For example, and not by way of limitation, compositions of the invention which elicit any of the following effects may be useful according to the invention: (1) increased survival time of neurons in culture; (2) increased sprouting of neurons in culture or in vivo; (3) increased production of a neuron-associated molecule in culture or in vivo, e.g., choline acetyltransferase or acetylcholinesterase with respect to motor neurons; or (4) decreased symptoms of neuron dysfunction in vivo. Such effects may be measured by any method known in the art. In representative, non-limiting embodiments, increased survival of neurons may routinely be measured using a method set forth herein or otherwise known in the art, such as, for example, the method set forth in Arakawa et al. (Arakawa et al., (1990)  J. Neurosci.  10:3507–3515); increased sprouting of neurons may be detected by methods known in the art, such as, for example, the methods set forth in Pestronk et al. (Pestronk et al., (1980)  Exp. Neurol.  70:65–82) or Brown et al. (Brown et al., (1981)  Ann. Rev. Neurosci.  4:17–42); increased production of neuron-associated molecules may be measured by bioassay, enzymatic assay, antibody binding, northern blot assay, etc., using techniques known in the art and depending on the molecule to be measured; and motor neuron dysfunction may be measured by assessing the physical manifestation of motor neuron disorder, e.g., weakness, motor neuron conduction velocity, or functional disability. \\n     In specific embodiments, motor neuron diseases, disorders, and/or conditions that may be treated, prevented, and/or diagnosed in a mouse model of a human condition according to the present invention include, but are not limited to, diseases, disorders, and/or conditions such as infarction, infection, exposure to toxin, trauma, surgical damage, degenerative disease or malignancy that may affect motor neurons as well as other components of the nervous system, as well as diseases, disorders, and/or conditions that selectively affect neurons such as amyotrophic lateral sclerosis, and including, but not limited to, progressive spinal muscular atrophy, progressive bulbar palsy, primary lateral sclerosis, infantile and juvenile muscular atrophy, progressive bulbar paralysis of childhood (Fazio-Londe syndrome), poliomyelitis and the post polio syndrome, and Hereditary Motorsensory Neuropathy (Charcot-Marie-Tooth Disease). \\n     The strong homology to mouse GBP family members, particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4, combined with MGBPBMY4 (BC007143) expression in liver tissue suggests a potential utility for MGBPBMY4 (BC007143) polynucleotides and polypeptides in treating, diagnosing, prognosing, and/or preventing liver diseases in a mouse model of a human condition. In representative embodiments, MGBPBMY4 (BC007143) polynucleotides and polypeptides including agonists and fragments thereof, may have uses that include treating, diagnosing, prognosing, and/or preventing liver diseases in a mouse model of a human condition. For example, an MGBPBMY4 (BC007143) protein can be used for the detection, treatment, amelioration, and/or prevention of diseases and conditions in a mouse model of a human condition including, but not limited to: hepatoblastoma, jaundice, hepatitis, liver metabolic diseases and conditions that are attributable to the differentiation of hepatocyte progenitor cells, cirrhosis, hepatic cysts, pyrogenic abscess, amebic abcess, hydatid cyst, cystadenocarcinoma, adenoma, focal nodular hyperplasia, hemangioma, hepatocellulae carcinoma, cholangiocarcinoma, and angiosarcoma, granulomatous liver disease, liver transplantation, hyperbilirubinemia, jaundice, parenchymal liver disease, portal hypertension, hepatobiliary disease, hepatic parenchyma, hepatic fibrosis, anemia, gallstones, cholestasis, carbon tetrachloride toxicity, beryllium toxicity, vinyl chloride toxicity, choledocholithiasis, hepatocellular necrosis, aberrant metabolism of amino acids, aberrant metabolism of carbohydrates, aberrant synthesis proteins, aberrant synthesis of glycoproteins, aberrant degradation of proteins, aberrant degradation of glycoproteins, aberrant metabolism of drugs, aberrant metabolism of hormones, aberrant degradation of drugs, aberrant degradation of drugs, aberrant regulation of lipid metabolism, aberrant regulation of cholesterol metabolism, aberrant glycogenesis, aberrant glycogenolysis, aberrant glycolysis, aberrant gluconeogenesis, hyperglycemia, glucose intolerance, hyperglycemia, decreased hepatic glucose uptake, decreased hepatic glycogen synthesis, hepatic resistance to insulin, portal-systemic glucose shunting, peripheral insulin resistance, hormonal abnormalities, increased levels of systemic glucagon, decreased levels of systemic cortisol, increased levels of systemic insulin, hypoglycemia, decreased gluconeogenesis, decreased hepatic glycogen content, hepatic resistance to glucagon, elevated levels of systemic aromatic amino acids, decreased levels of systemic branched-chain amino acids, hepatic encephalopathy, aberrant hepatic amino acid transamination, aberrant hepatic amino acid oxidative deamination, aberrant ammonia synthesis, aberant albumin secretion, hypoalbuminemia, aberrant cytochromes b5 function, aberrant P450 function, aberrant glutathione S-acyltransferase function, aberrant cholesterol synthesis, and aberrant bile acid synthesis. \\n     Moreover, MGBPBMY4 (BC007143) polynucleotides and polypeptides, including fragments and/or antagonists thereof, may have uses which include, directly or indirectly, treating, preventing, diagnosing, and/or prognosing the following, non-limiting, hepatic infections in a mouse model of a human condition: liver disease caused by sepsis infection, liver disease caused by bacteremia, liver disease caused by Pneomococcal pneumonia infection, liver disease caused by Toxic shock syndrome, liver disease caused by Listeriosis, liver disease caused by Legionnaries&#39; disease, liver disease caused by Brucellosis infection, liver disease caused by  Neisseria gonorrhoeae  infection, liver disease caused by  Yersinia  infection, liver disease caused by Salmonellosis, liver disease caused by Nocardiosis, liver disease caused by Spirochete infection, liver disease caused by  Treponema pallidum  infection, liver disease caused by  Brrelia burgdorferi  infection, liver disease caused by Leptospirosis, liver disease caused by  Coxiella burnetii  infection, liver disease caused by  Rickettsia richettsii  infection, liver disease caused by  Chlamydia trachomatis  infection, liver disease caused by  Chlamydia psittaci  infection, liver disease caused by hepatitis virus infection, liver disease caused by Epstein-Barr virus infection in addition to any other hepatic disease and/or disorder implicated by the causative agents listed above or elsewhere herein. \\n     It is noted that the use of mouse models to understand, diagnose, predict, treat and/or ameliorate human conditions is well documented. Thus, the uses for the MGBPBMY4 (BC007143) can often be extrapolated to human conditions, as well as to further research such conditions and their treatments. \\n     A MGBPBMY4 (BC007143) protein can also be used to determine biological activity, raise antibodies, as tissue markers, to isolate cognate ligands or receptors, to identify agents that modulate their interactions. Further, proteins, as well as antibodies directed against a MGBPBMY4 (BC007143) protein, can show utility as a tumor marker and/or immunotherapy targets for heart, kidney, spleen liver, skeletal muscle and/or brain tissue. \\n     The MGBPBMY4 (BC007143) polynucleotides and polypeptides, including fragments and for antagonsists thereof, can have uses which include identification of modulators of MGBPBMY4 (BC007143) function including antibodies (for detection or neutralization), naturally-occurring modulators and small molecule modulators. Antibodies to a particular domain of the MGBPBMY4 (BC007143) protein could be used as diagnostic agents of certain conditions in subjects, are useful in monitoring the activation of signal transduction pathways, and can be used as a biomarker for the involvement of GBP&#39;s in disease states, as well as in the evaluation of inhibitors of GBP&#39;s in vivo. \\n     MGBPBMY4 (BC007143) polypeptides and polynucleotides have additional uses which include diagnosing diseases related to the over and/or under expression of MGBPBMY4 (BC007143) by identifying mutations in the MGBPBMY4 (BC007143) gene by using MGBPBMY4 (BC007143) sequences as probes or by determining MGBPBMY4 (BC007143) protein or mRNA expression levels. MGBPBMY4 (BC007143) polypeptides can be useful for screening compounds that affect the activity of the protein. MGBPBMY4 (BC007143) peptides can also be used for the generation of specific antibodies and as bait in yeast two hybrid screens to find proteins the specifically interact with MGBPBMY4 (BC007143), as described herein. \\n     Although it is believed the encoded polypeptide could share at least some biological activities with human guanylate binding proteins (particularly mGBP-1, mGBP-2, mGBP-3 and mGBP-4), a number of methods of determining the exact biological function of this clone are either known in the art or are described elsewhere herein. For example, the function of this clone can be determined by applying microarray methodology. Nucleic acids corresponding to the MGBPBMY4 (BC007143) polynucleotides, in addition to, other clones of the present invention, can be arrayed on microchips for expression profiling. Depending on which polynucleotide probe is used to hybridize to the slides, a change in expression of a specific gene can provide additional insight into the function of this gene based upon the conditions being studied. For example, an observed increase or decrease in expression levels when the polynucleotide probe used comes from diseased liver tissue, as compared to normal tissue might indicate a function in modulating liver function, for example. In the case of MGBPBMY4 (BC007143), heart, kidney, spleen, liver, brain and/or skeletal muscle, can be used, for example, to extract RNA to prepare the probe. \\n     In addition, the function of the protein can be assessed, for example, by applying quantitative PCR methodology. Real time quantitative PCR would provide the capability of following the expression of the MGBPBMY4 (BC007143) gene throughout development, for example. Quantitative PCR methodology requires only a nominal amount of tissue from each developmentally important step is needed to perform such experiments. Therefore, the application of quantitative PCR methodology to refining the biological function of this polypeptide is encompassed by the present invention. In the case of MGBPBMY4 (BC007143), a disease correlation related to MGBPBMY4 (BC007143) can be made by comparing the mRNA expression level of MGBPBMY4 (BC007143) in normal tissue, as compared to diseased tissue. Significantly higher or lower levels of MGBPBMY4 (BC007143) expression in the diseased tissue can suggest MGBPBMY4 (BC007143) plays a role in disease progression, and antagonists against MGBPBMY4 (BC007143) polypeptides would be useful therapeutically in treating, preventing, and/or ameliorating the disease. Alternatively, significantly higher or lower levels of MGBPBMY4 (BC007143) expression in the diseased tissue can suggest MGBPBMY4 (BC007143) plays a defensive role against disease progression, and agonists of MGBPBMY4 (BC007143) polypeptides can be useful therapeutically in treating, preventing, and/or ameliorating the disease. Also encompassed by the present invention are quantitative PCR probes corresponding to the polynucleotide sequence provided as SEQ ID NO:16 ( FIGS. 8A–8D ). \\n     The function of the protein can also be assessed through complementation assays in yeast. For example, in the case of the MGBPBMY4 (BC007143), transforming yeast deficient in GBP activity, for example, and assessing their ability to grow would provide convincing evidence the MGBPBMY4 (BC007143) polypeptide has GBP activity. Additional assay conditions and methods that can be used in assessing the function of the polynucleotides and polypeptides of the present invention are known in the art, some of which are disclosed herein. For example, a GTPase activity assay can be employed. \\n     Alternatively, the biological function of the encoded polypeptide can be determined by disrupting a homologue of this polypeptide in another species (e.g., a mammalian species) and observing the resulting phenotype. Such knock-out experiments are known in the art, some of which are disclosed elsewhere herein. \\n     Moreover, the biological function of this polypeptide can be determined by the application of antisense and/or sense methodology (including RNAi and homlogous recombination) and the resulting generation of transgenic animals. Expressing a particular gene in either sense or antisense orientation in a transgenic mouse or rat could lead to respectively higher or lower expression levels of that particular gene. Altering the endogenous expression levels of a gene can lead to the observation of a particular phenotype that can then be used to derive indications on the function of the gene. The gene can be either over-expressed or under expressed in every cell of the organism at all times using a strong ubiquitous promoter, or it could be expressed in one or more discrete parts of the organism using a well characterized tissue-specific promoter (e.g., a spleen tissue-specific promoter), or it can be expressed at a specified time of development using an inducible and/or a developmentally regulated promoter. \\n     In the case of MGBPBMY4 (BC007143), transgenic animals, if no phenotype is apparent in normal growth conditions, observing the organism under diseased conditions (e.g., reproductive, cardiovascular, endocrine, immune, renal, gastrointestinal, pulmonary, and/or neural disorders, in addition to cancers, etc.) can lead to understanding the function of the gene. Therefore, the application of antisense and/or sense methodology to the creation of transgenic animals to refine the biological function of the polypeptide is encompassed by the present invention. \\n     In preferred embodiments, the following N-terminal MGBPBMY4 (BC007143) deletion polypeptides are encompassed by the present invention: M1-A209, I2-A209, T3-A209, I4-A209, N5-A209, H6-A209, Q7-A209, A8-A209, L9-A209, E10-A209, Q11-A209, L12-A209, H13-A209, Y14-A209, V15-A209, T16-A209, E17-A209, L18-A209, T19-A209, E20-A209, L21-A209, I22-A209, R23-A209, A24-A209, K25-A209, S26-A209, S27-A209, P28-A209, N29-A209, P30-A209, A31-A209, G32-A209, I33-A209, K34-A209, N35-A209, S36-A209, T37-A209, E38-A209, F39-A209, V40-A209, S41-A209, F42-A209, F43-A209, P44-A209, D45-A209, F46-A209, V47-A209, W48-A209, T49-A209, V50-A209, R51-A209, D52-A209, F53-A209, M54-A209, L55-A209, E56-A209, L57-A209, K58-A209, L59-A209, N60-A209, G61-A209, E62-A209, D63-A209, I64-A209, T65-A209, S66-A209, D67-A209, D68-A209, Y69-A209, L70-A209, E71-A209, N72-A209, A73-A209, L74-A209, K75-A209, L76-A209, I77-A209, P78-A209, G79-A209, D80-A209, K81-A209, P82-A209, R83-A209, M84-A209, Q85-A209, A86-A209, S87-A209, N88-A209, S89-A209, C90-A209, R91-A209, E92-A209, C93-A209, I94-A209, R95-A209, L96-A209, F97-A209, F98-A209, P99-A209, N100-A209, R101-A209, K102-A209, C103-A209, F104-A209, V105-A209, F106-A209, D107-A209, R108-A209, P109-A209, T110-A209, H111-A209, D112-A209, K113-A209, E114-A209, L115-A209, L116-A209, Q117-A209, K118-A209, L119-A209, D120-A209, S121-A209, I122-A209, T123-A209, E124-A209, D125-A209, Q126-A209, L127-A209, D128-A209, P129-A209, K130-A209, F131-A209, Q132-A209, E133-A209, V134-A209, T135-A209, K136-A209, A137-A209, F138-A209, V139-A209, S140-A209, Y141-A209, I142-A209, F143-A209, T144-A209, Y145-A209, A146-A209, K147-A209, I148-A209, K149-A209, T150-A209, L151-A209, K152-A209, E153-A209, G154-A209, I155-A209, K156-A209, V157-A209, T158-A209, G159-A209, N160-A209, R161-A209, L162-A209, G163-A209, I164-A209, L165-A209, V166-A209, T167-A209, T168-A209, Y169-A209, V170-A209, N171-A209, A172-A209, I173-A209, N174-A209, S175-A209, G176-A209, A177-A209, V178-A209, P179-A209, C180-A209, L181-A209, D182-A209, D183-A209, A184-A209, V185-A209, T186-A209, T187-A209, L188-A209, A189-A209, Q190-A209, R191-A209, E192-A209, N193-A209, S194-A209, V195-A209, A196-A209, V197-A209, Q198-A209, K199-A209, A200-A209, A201-A209, D202-A209, H203-A209, Y204-A209, S205-A209, E206-A209, Q207-A209 and/or M208-A209, of SEQ ID NO:17. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these N-terminal MGBPBMY4 (BC007143) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     In other embodiments, the following C-terminal MGBPBMY4 (BC007143) deletion polypeptides are encompassed by the present invention: M1-I2, M1-T3, M1-I4, M1-N5, M1-H6, M1-Q7, M1-A8, M1-L9, M1-E10, M1-Q11, M1-L12, M1-H13, M1-Y14, M1-V15, M1-T16, M1-E17, M1-L18, M1-T19, M1-E20, M1-L21, M1-I22, M1-R23, M1-A24, M1-K25, M1-S26, M1-S27, M1-P28, M1-N29, M1-P30, M1-A31, M1-G32, M1-I33, M1-K34, M1-N35, M1-S36, M1-T37, M1-E38, M1-F39, M1-V40, M1-S41, M1-F42, M1-F43, M1-P44, M1-D45, M1-F46, M1-V47, M1-W48, M1-T49, M1-V50, M1-R51, M1-D52, M1-F53, M1-M54, M1-L55, M1-E56, M1-L57, M1-K58, M1-L59, M1-N60, M1-G61, M1-E62, M1-D63, M1-I64, M1-T65, M1-S66, M1-D67, M1-D68, M1-Y69, M1-L70, M1-E71, M1-N72, M1-A73, M1-L74, M1-K75, M1-L76, M1-I77, M1-P78, M1-G79, M1-D80, M1-K81, M1-P82, M1-R83, M1-M84, M1-Q85, M1-A86, M1-S87, M1-N88, M1-S89, M1-C90, M1-R91, M1-E92, M1-C93, M1-I94, M1-R95, M1-L96, M1-F97, M1-F98, M1-P99, M1-N100, M1-R101, M1-K102, M1-C103, M1-F104, M1-VIO5, M1-F106, M1-D107, M1-R108, M1-P109, M1-T110, M1-H111, M1-D112, M1-K113, M1-E114, M1-L115, M1-L116, M1-Q117, M1-K118, M1-L119, M1-D120, M1-S121, M1-I22, M1-T123, M1-E124, M1-D125, M1-Q126, M1-L127, M1-D128, M1-P129, M1-K130, M1-F131, M1-Q132, M1-E133, M1-V134, M1-T135, M1-K136, M1-A137, M1-F138, M1-V139, M1-S140, M1-Y141, M1-I42, M1-F143, M1-T144, M1-Y145, M1-A146, M1-K147, M1-I148, M1-K149, M1-T150, M1-L151, M1-K152, M1-E153, M1-G154, M1-I155, M1-K156, M1-V157, M1-T158, M1-G159, M1-N160, M1-R161, M1-L162, M1-G163, M1-I164, M1-L165, M1-V166, M1-T167, M1-T168, M1-Y169, M1-V170, M1-N171, M1-A172, M1-I173, M1-N174, M1-S175, M1-G176, M1-A177, M1-V178, M1-P179, M1-C180, M1-L181, M1-D182, M1-D183, M1-A184, M1-V185, M1-T186, M1-T187, M1-L188, M1-A189, M1-Q190, M1-R191, M1-E192, M1-N193, M1-S194, M1-V195, M1-A196, M1-V197, M1-Q198, M1-K199, M1-A200, M1-A201, M1-D202, M1-H203, M1-Y204, M1-S205, M1-E206, M1-Q207 and/or M1-M208 of SEQ ID NO:17. Polynucleotide sequences encoding these polypeptides are also provided. The present invention also encompasses the use of these C-terminal MGBPBMY4 (BC007143) deletion polypeptides as immunogenic and/or antigenic epitopes as described elsewhere herein. \\n     Alternatively, preferred polypeptides of the present invention can comprise polypeptide sequences corresponding to, for example, internal regions of the MGBPBMY4 (BC007143) polypeptide (e.g., any combination of both N- and C-terminal MGBPBMY4 (BC007143) polypeptide deletions) of SEQ ID NO:17. For example, internal regions could be defined by the equation: amino acid NX to amino acid CX, wherein NX refers to any N-terminal deletion polypeptide amino acid of MGBPBMY4 (BC007143) (SEQ ID NO:17), and where CX refers to any C-terminal deletion polypeptide amino acid of MGBPBMY4 (BC007143) (SEQ ID NO:17). Polynucleotides encoding these polypeptides are also provided. The present invention also encompasses the use of these polypeptides as an immunogenic and/or antigenic epitope as described elsewhere herein. \\n     The present invention also encompasses immunogenic and/or antigenic epitopes of the MGBPBMY4 (BC007143) polypeptide. \\n     The present invention encompasses the identification of compounds and drugs which stimulate MGBPBMY4 (BC007143) on the one hand (i.e., agonists) and which inhibit the function of MGBPBMY4 (BC007143) on the other hand (i.e., antagonists). In general, such screening procedures involve providing appropriate cells which express a polypeptide of the present invention on the surface thereof. Such cells can include, for example, cells from mammals, yeast,  Drosophila  or  E. coli . In a representative embodiment, a polynucleotide encoding a polypeptide of the present invention can be employed to transfect cells to thereby express the MGBPBMY4 (BC007143) polypeptide. The expressed polypeptide can then be contacted with a test compound to observe binding, stimulation or inhibition of a functional response. \\n     Features of GBP1 \\n     The tissue expression profile of GBP1 was investigated.  FIG. 47  shows an expanded expression profile of human GBP1.  FIG. 47  illustrates the relative expression level of GBP1 amongst various mRNA tissue sources.  FIG. 47  also illustrates the relative expression level of GBP1 amongst various mRNA tissue sources isolated from normal and dieased tissues. As shown, the GBP1 polypeptide showed increased expression in spinal cord and brain cortex from multiple sclerosis patients relative to controls; increased expression in putamen from Parkinson&#39;s patients relative to controls; increased expression in hippocampus from Alzheimer&#39;s patients relative to controls. Expression data was obtained by measuring the steady state GBP1 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:73 and 74, and TAQMAN probe (SEQ ID NO:75) as described in Example 40 herein. These data support a role of GBP1 in regulating various functions, including neurological functions. Thus small molecule modulators of GBP1 function may represent a novel therapeutic option in the treatment of multiple sclerosis, Parkinson&#39;s disease and Alzheimer&#39;s disease. \\n     Features of GBP5 \\n     The tissue expression profile of GBP5 was investigated.  FIG. 48  shows an expanded expression profile of human GBP5.  FIG. 48  illustrates the relative expression level of GBP5 amongst various mRNA tissue sources.  FIG. 48  also illustrates the relative expression level of GBP5 amongst various mRNA tissue sources isolated from normal and tumor tissues. As shown, the GBP5 polypeptide showed increased expression in cortex and spinal cord of multiple sclerosis patients relative to controls; increased expression in hippocampus form Alzheimer&#39;s patients relative to controls. Expression data was obtained by measuring the steady state GBP5 mRNA levels by quantitative PCR using the PCR primer pair provided as SEQ ID NO:76 and 77, and TAQMAN probe (SEQ ID NO:78) as described in Example 41 herein. These data support a role of GBP5 in regulating various functions, including digestive functions, neurological functions reproductive functions and pulmonary functions. Thus small molecule modulators of GBP5 function may represent a novel therapeutic option in the treatment of breast and testicle cancers, as well as multiple sclerosis, Alzheimer&#39;s disease, Crohn&#39;s disease, and bronchitis. \\n     Table I, presented herein below, summarizes the information corresponding to each “Gene No.” described above. The nucleotide sequence identified as “NT SEQ ID NO:X” was assembled from partially homologous (“overlapping”) sequences obtained from the “cDNA clone ID” identified in Table I and, in some cases, from additional related DNA clones. The overlapping sequences were assembled into a single contiguous sequence of high redundancy (usually several overlapping sequences at each nucleotide position), resulting in a final sequence identified as SEQ ID NO:X. \\n     The cDNA Clone ID was deposited on the date and given the corresponding deposit number listed in “ATCC Deposit No:Z and Date.” \\n     “Total NT Seq. Of Clone” refers to the total number of nucleotides in the clone contig identified by “Gene No.” The deposited clone may contain all or most of the sequence of SEQ ID NO:X. The nucleotide position of SEQ ID NO:X of the putative start codon (methionine) is identified as “5′ NT of Start Codon of ORF.” \\n     The translated amino acid sequence, beginning with the methionine, is identified as “AA SEQ ID NO:Y” although other reading frames can also be easily translated using molecular biology techniques known to those of ordinary skill in the art. The polypeptides produced by these alternative open reading frames are specifically contemplated by the present invention. \\n     The total number of amino acids within the open reading frame of SEQ ID NO:Y is identified as “Total AA of ORF”. \\n     SEQ ID NO:X (where X s any of the polynucleotide sequences disclosed in the Sequence Listing) and the translated SEQ ID NO:Y (where Y is any of the polypeptide sequences disclosed in the Sequence Listing) are sufficiently accurate and otherwise suitable for a variety of uses well known in the art and described further herein. For instance, SEQ ID NO:X is useful for designing nucleic acid hybridization probes that will detect nucleic acid sequences contained in SEQ ID NO:X or the cDNA contained in the deposited clone. These probes will also hybridize to nucleic acid molecules in biological samples, thereby enabling a variety of forensic and diagnostic methods of the present invention. Similarly, polypeptides identified from SEQ ID NO:Y can be used, for example, to generate antibodies which bind specifically to proteins containing the polypeptides and the proteins encoded by the cDNA clones identified in Table I. \\n     Nevertheless, DNA sequences generated by sequencing reactions can contain sequencing errors. The errors exist as misidentified nucleotides, or as insertions or deletions of nucleotides in the generated DNA sequence. The erroneously inserted or deleted nucleotides might cause frame shifts in the reading frames of the predicted amino acid sequence. In these cases, the predicted amino acid sequence diverges from the actual amino acid sequence, even though the generated DNA sequence may be greater than 99.9% identical to the actual DNA sequence (for example, one base insertion or deletion in an open reading frame of over 1000 bases). \\n     Accordingly, for those applications requiring precision in the nucleotide sequence or the amino acid sequence, the present invention provides not only the generated nucleotide sequence identified as SEQ ID NO:X and the predicted translated amino acid sequence identified as SEQ ID NO:Y, but also a sample of plasmid DNA containing a cDNA of the present invention deposited with the ATCC, as set forth in Table I. The nucleotide sequence of each deposited clone can readily be determined by sequencing the deposited clone in accordance with known methods. The predicted amino acid sequence can then be verified from such deposits. Moreover, the amino acid sequence of the protein encoded by a particular clone can also be directly determined by peptide sequencing or by expressing the protein in a suitable host cell containing the deposited cDNA, collecting the protein, and determining its sequence. \\n     The present invention also relates to the genes corresponding to SEQ ID NO:X, SEQ ID NO:Y, or the corresponding deposited clone. The corresponding gene can be isolated in accordance with known methods using the sequence information disclosed herein. Such methods include preparing probes or primers from the disclosed sequence and identifying or amplifying the corresponding gene from appropriate sources of genomic material. \\n     Also provided in the present invention are species homologs, allelic variants, and/or orthologs. The skilled artisan could, using procedures well-known in the art, obtain the polynucleotide sequence corresponding to full-length genes (including, but not limited to the full-length coding region), allelic variants, splice variants, orthologs, and/or species homologues of genes corresponding to SEQ ID NO:X, SEQ ID NO:Y, or a deposited clone, relying on the sequence from the sequences disclosed herein or the clones deposited with the ATCC. For example, allelic variants and/or species homologues may be isolated and identified by making suitable probes or primers which correspond to the 5′, 3′, or internal regions of the sequences provided herein and screening a suitable nucleic acid source for allelic variants and/or the desired homologue. \\n     The polypeptides of the present invention can be prepared in any suitable manner. Such polypeptides include isolated naturally occurring polypeptides, recombinantly produced polypeptides, synthetically produced polypeptides, or polypeptides produced by a combination of these methods. Means for preparing such polypeptides are well understood in the art. \\n     The polypeptides can be in the form of a complete protein, can be fragments of the protein or can be a part of a larger protein, such as a fusion protein, as described herein. It can be desirable to include an additional amino acid sequence that contains secretory or leader sequences, pro-sequences, sequences that can aid in purification, such as multiple histidine residues, or an additional sequence for stability during recombinant production. \\n     The polypeptides of the present invention are preferably, but not necessarily, provided in an isolated form, and preferably, but not necessarily, are substantially purified. A recombinantly produced version of a polypeptide, can be substantially purified using techniques described herein or otherwise known in the art, such as, for example, by the one-step method described in Smith &amp; Johnson, (1988)  Gene  67:31–40. Polypeptides of the present invention also can be purified from natural, synthetic or recombinant sources using protocols described herein or otherwise known in the art, such as, for example, antibodies of the present invention raised against the full-length form of the protein. \\n     The present invention also provides a polynucleotide comprising, or alternatively consisting of, the sequence identified as SEQ ID NO:X, and/or a cDNA provided in corresponding ATCC Deposit No. Z. The present invention also provides a polypeptide comprising, or alternatively consisting of, the sequence identified as SEQ ID NO:Y, and/or a polypeptide encoded by the cDNA provided in corresponding ATCC Deposit No:Z. The present invention also provides polynucleotides encoding a polypeptide comprising, or alternatively consisting of the polypeptide sequence of SEQ ID NO:Y, and/or a polypeptide sequence encoded by the cDNA contained in ATCC Deposit No:Z. \\n     In one embodiment, the present invention is directed to a polynucleotide comprising, or alternatively consisting of, the sequence identified as SEQ ID NO:X, and/or a cDNA provided in corresponding ATCC Deposit No.:Z that is less than, or equal to, a polynucleotide sequence that is 5 mega basepairs, 1 mega basepairs, 0.5 mega basepairs, 0.1 mega basepairs, 50,000 basepairs, 20,000 basepairs, or 10,000 basepairs in length. \\n     The present invention encompasses polynucleotides with sequences complementary to those of the polynucleotides of the present invention disclosed herein. Such sequences can be complementary to the sequence disclosed as SEQ ID NO:X, the sequence contained in an ATCC deposit, and/or the nucleic acid sequence encoding the polypeptide sequence disclosed as SEQ ID NO:Y. \\n     The present invention also encompasses polynucleotides capable of hybridizing, for example under reduced stringency conditions, stringent conditions or highly stringent conditions, to polynucleotides described herein. Examples of stringency conditions are described herein above and in Table II). Refering to Table II, highly stringent conditions are those that are at least as stringent as, for example, conditions A–F; stringent conditions are at least as stringent as, for example, conditions G–L; and reduced stringency conditions are at least as stringent as, for example, conditions M–R. \\n                                     TABLE II                           Hybridization           Stringency   Polynucleotide   Hybrid Length   Temperature and   Wash Temperature       Condition   Hybrid±   (bp)‡   Buffer†   and Buffer†                  A   DNA:DNA   &gt; or equal to 50   65° C.; 1xSSC -   65° C.;                   or - 42° C.; 1xSSC,   0.3xSSC                   50% formamide       B   DNA:DNA   &lt;50   Tb*; 1xSSC   Tb*; 1xSSC       C   DNA:RNA   &gt; or equal to 50   67° C.; 1xSSC -   67° C.;                   or - 45° C.; 1xSSC,   0.3xSSC                   50% formamide       D   DNA:RNA   &lt;50   Td*; 1xSSC   Td*; 1xSSC       E   RNA:RNA   &gt; or equal to 50   70° C.; 1xSSC -   70° C.;                   or - 50° C.; 1xSSC,   0.3xSSC                   50% formamide       F   RNA:RNA   &lt;50   Tf*; 1xSSC   Tf*; 1xSSC       G   DNA:DNA   &gt; or equal to 50   65° C.; 4xSSC -   65° C.; 1xSSC                   or - 45° C.; 4xSSC,                   50% formamide       H   DNA:DNA   &lt;50   Th*; 4xSSC   Th*; 4xSSC       I   DNA:RNA   &gt; or equal to 50   67° C.; 4xSSC -   67° C.; 1xSSC                   or - 45° C.; 4xSSC,                   50% formamide       J   DNA:RNA   &lt;50   Tj*; 4xSSC   Tj*; 4xSSC       K   RNA:RNA   &gt; or equal to 50   70° C.; 4xSSC -   67° C.; 1xSSC                   or - 40° C.; 6xSSC,                   50% formamide       L   RNA:RNA   &lt;50   Tl*; 2xSSC   Tl*; 2xSSC       M   DNA:DNA   &gt; or equal to 50   50° C.; 4xSSC -   50° C.; 2xSSC                   or - 40° C. 6xSSC,                   50% formamide       N   DNA:DNA   &lt;50   Tn*; 6xSSC   Tn*; 6xSSC       O   DNA:RNA   &gt; or equal to 50   55° C.; 4xSSC -   55° C.; 2xSSC                   or - 42° C.; 6xSSC,                   50% formamide       P   DNA:RNA   &lt;50   Tp*; 6xSSC   Tp*; 6xSSC       Q   RNA:RNA   &gt; or equal to 50   60° C.; 4xSSC -   60° C.; 2xSSC                   or - 45° C.; 6xSSC,                   50% formamide       R   RNA:RNA   &lt;50   Tr*; 4xSSC   Tr*; 4xSSC                    \\nThe following annotations pertain to Table II.\\n \\n     ‡—The “hybrid length” is the anticipated length for the hybridized region(s) of the hybridizing polynucleotides. When hybridizing a polynucleotide of unknown sequence, the hybrid is assumed to be that of the hybridizing polynucleotide of the present invention. When polynucleotides of known sequence are hybridized, the hybrid length can be determined by aligning the sequences of the polynucleotides and identifying the region or regions of optimal sequence complementarity. Methods of aligning two or more polynucleotide sequences and/or determining the percent identity between two polynucleotide sequences are well known in the art (e.g., the MEGALIGN program of the suite of programs available from DNA*Star of Madison, Wis., USA, etc). \\n     †—SSPE (1×SSPE is 0.15M NaCl, 10 mM NaH2PO4, and 1.25 mM EDTA, pH 7.4) can be substituted for SSC (1×SSC is 0.15M NaCl and 15 mM sodium citrate) in the hybridization and wash buffers; washes are performed for 15 minutes after hybridization is complete. The hydridizations and washes can optionally include 5× Denhardt&#39;s reagent, 0.5–1.0% SDS, 100 μg/ml denatured, fragmented salmon sperm DNA, 0.5% sodium pyrophosphate, and up to 50% formamide. \\n     *Tb−Tr: The hybridization temperature for hybrids anticipated to be less than 50 base pairs in length should be 5–10° C. less than the melting temperature Tm of the hybrids there Tm is determined according to the following equations. For hybrids less than 18 base pairs in length, Tm(° C.)=2(# of A+T bases)+4(# of G+C bases). For hybrids between 18 and 49 base pairs in length, Tm(° C.)=81.5+16.6(log 10  [Na + ])+0.41(% G+C)— (600/N), where N is the number of bases in the hybrid, and [Na + ] is the concentration of sodium ions in the hybridization buffer ([NA + ] for 1×SSC=0.165 M). \\n     ±—The present invention encompasses the substitution of any one, or more DNA or RNA hybrid partners with either a PNA, or a modified polynucleotide. Such modified polynucleotides are known in the art and are more particularly described elsewhere herein. \\n     Additional examples of stringency conditions for polynucleotide hybridization are provided, for example, in Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3 rd  ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001), chapters 9 and 11, and  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002) sections 2.10 and 6.3–6.4, which are hereby incorporated by reference herein. \\n     In some cases it can be desirable that such hybridizing polynucleotides have at least 70% sequence identity (e.g., at least 80% identity; at least 90% or at least 95% identity) with a polynucleotide of the present invention to which they hybridize, where sequence identity is determined by comparing the sequences of the hybridizing polynucleotides when aligned so as to maximize overlap and identity while minimizing sequence gaps. The determination of identity is well known in the art, and discussed more specifically elsewhere herein. \\n     The present invention encompasses the application of PCR methodology to the polynucleotide sequences of the present invention, the clone deposited with the ATCC, and/or the cDNA encoding the polypeptides of the present invention. PCR techniques for the amplification of nucleic acids are known (see, e.g., U.S. Pat. Nos. 4,683,195 and 4,800,159 and Saiki et al., (1988)  Science  239:487–491). PCR, for example, can include the following steps, of denaturation of template nucleic acid (if double-stranded), annealing of primer to target, and polymerization. The nucleic acid probed or used as a template in the amplification reaction can be genomic DNA, cDNA, RNA, or a PNA. PCR can be used to amplify specific sequences from genomic DNA, specific RNA sequence, and/or cDNA transcribed from mRNA. References for the general use of PCR techniques, including specific method parameters, include Mullis et al., (1987)  Cold Spring Harbor Symp. Quant. Biol.  51:263 ; PCR Technology , (Ehrlich, ed.), Stockton Press, New York, N.Y., USA (1989); Ehrlich et al., (1991)  Science  252:1643–1650; and  PCR Protocols, A Guide to Methods and Applications , (Innis et al., eds.), Academic Press, New York, N.Y., USA (1990). See also U.S. Pat. No. 4,800,159. \\n     Polynucleotide and Polypeptide Variants \\n     Polynucleotide Variants \\n     The present invention also encompasses variants (e.g., allelic variants, orthologs, etc.) of the polynucleotide sequence disclosed herein in SEQ ID NO:X, the complementary strand thereto, and/or the cDNA sequence contained in the deposited clone. \\n     The present invention also encompasses variants of the polypeptide sequence, and/or fragments therein, disclosed in SEQ ID NO:Y, a polypeptide encoded by the polynucleotide sequence in SEQ ID NO:X, and/or a polypeptide encoded by a cDNA in the deposited clone. \\n     As used herein, the term “variant” means a polynucleotide or polypeptide differing from the polynucleotide or polypeptide of the present invention, but retaining essential properties thereof. Generally, variants are overall closely similar, and, in many regions, identical to the polynucleotide or polypeptide of the present invention. \\n     Thus, one aspect of the present invention provides an isolated nucleic acid molecule comprising, or alternatively consisting of, a polynucleotide having a nucleotide sequence selected from the group consisting of: (a) a nucleotide sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence as shown in the sequence listing and described in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (b) a nucleotide sequence encoding a mature HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having the amino acid sequence as shown in the sequence listing and described in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (c) a nucleotide sequence encoding a biologically active fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence shown in the sequence listing and described in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (d) a nucleotide sequence encoding an antigenic fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence shown in the sequence listing and described in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (e) a nucleotide sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide comprising the complete amino acid sequence encoded by a human cDNA plasmid contained in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (f) a nucleotide sequence encoding a mature HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA plasmid contained in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (g) a nucleotide sequence encoding a biologically active fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA plasmid contained in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; (h) a nucleotide sequence encoding an antigenic fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA plasmid contained in SEQ ID NO:X or the cDNA contained in ATCC Deposit No:Z; and (i) a nucleotide sequence complimentary to any of the nucleotide sequences in (a), (b), (c), (d), (e), (f), (g), or (h), above. \\n     The present invention is also directed to polynucleotide sequences which comprise, or alternatively consist of, a polynucleotide sequence which is at least about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% identical to, for example, any of the nucleotide sequences in (a), (b), (c), (d), (e), (f), (g), or (h), above. Polynucleotides encoded by these nucleic acid molecules are also encompassed by the present invention. In another embodiment, the present invention encompasses nucleic acid molecules which comprise, or alternatively, consist of a polynucleotide which hybridizes under stringent conditions, or alternatively, under lower stringency conditions, to a polynucleotide in (a), (b), (c), (d), (e), (f), (g), or (h), above. Polynucleotides that hybridize to the complement of these nucleic acid molecules under stringent hybridization conditions or alternatively, under lower stringency conditions, are also encompassed by the present invention, as are polypeptides encoded by these polypeptides. \\n     Another aspect of the present invention provides an isolated nucleic acid molecule comprising, or alternatively, consisting of, a polynucleotide having a nucleotide sequence selected from the group consisting of: (a) a nucleotide sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence as shown in the sequence listing and described in Table I; (b) a nucleotide sequence encoding a mature HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having the amino acid sequence as shown in the sequence listing and descried in Table I; (c) a nucleotide sequence encoding a biologically active fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence as shown in the sequence listing and described in Table I; (d) a nucleotide sequence encoding an antigenic fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence as shown in the sequence listing and described in Table I; (e) a nucleotide sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide comprising the complete amino acid sequence encoded by a human cDNA in a cDNA plasmid contained in the coresponding ATCC Deposit and described in Table I; (f) a nucleotide sequence encoding a mature HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA in a cDNA plasmid contained in the ATCC Deposit and described in Table I: (g) a nucleotide sequence encoding a biologically active fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA in a cDNA plasmid contained in the correspoding ATCC Deposit and described in Table I; (h) a nucleotide sequence encoding an antigenic fragment of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-related polypeptide having an amino acid sequence encoded by a human cDNA in a cDNA plasmid contained in the corresponding ATCC deposit and described in Table I; (i) a nucleotide sequence complimentary to any of the nucleotide sequences in (a), (b), (c), (d), (e), (f), (g), or (h) above. \\n     The present invention is also directed to nucleic acid molecules which comprise, or alternatively, consist of, a nucleotide sequence which is at least about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% identical to, for example, any of the nucleotide sequences in (a), (b), (c), (d), (e), (f), (g), or (h), above. \\n     The present invention encompasses polypeptide sequences which comprise, or alternatively consist of, an amino acid sequence which is at least about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% identical to the following non-limited examples: the polypeptide sequence identified as SEQ ID NO:Y, the polypeptide sequence encoded by a cDNA provided in the deposited clone, and/or polypeptide fragments of any of the polypeptides provided herein. Polynucleotides encoded by these nucleic acid molecules are also encompassed by the present invention. In another embodiment, the present invention encompasses nucleic acid molecules which comprise, or alternatively, consist of a polynucleotide which hybridizes under stringent conditions, or alternatively, under lower stringency conditions, to a polynucleotide in (a), (b), (c), (d), (e), (f), (g), or (h), above. Polynucleotides which hybridize to the complement of these nucleic acid molecules under stringent hybridization conditions or alternatively, under lower stringency conditions, are also encompassed by the present invention, as are polypeptides encoded by these polypeptides. \\n     The present invention is also directed to polypeptides which comprise, or alternatively consist of, an amino acid sequence which is at least about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% identical to, for example, the polypeptide sequence shown in SEQ ID NO:Y, a polypeptide sequence encoded by the nucleotide sequence in SEQ ID NO:X, a polypeptide sequence encoded by the cDNA in cDNA plasmid:Z, and/or polypeptide fragments of any of these polypeptides (e.g., those fragments described herein). Polynucleotides which hybridize to the complement of the nucleic acid molecules encoding these polypeptides under stringent hybridization conditions or alternatively, under lower stringency conditions, are also encompasses by the present invention, as are the polypeptides encoded by these polynucleotides. \\n     By a nucleic acid having a nucleotide sequence at least, for example, 95% “identical” to a reference nucleotide sequence of the present invention, it is intended that the nucleotide sequence of the nucleic acid is identical to the reference sequence except that the nucleotide sequence can include up to five point mutations per each 100 nucleotides of the reference nucleotide sequence encoding the polypeptide. In other words, to obtain a nucleic acid having a nucleotide sequence at least 95% identical to a reference nucleotide sequence, up to 5% of the nucleotides in the reference sequence may be deleted or substituted with another nucleotide, or a number of nucleotides up to 5% of the total nucleotides in the reference sequence may be inserted into the reference sequence. The query sequence can be an entire sequence referenced in Table I, the ORF (open reading frame), or any fragment specified as described herein. \\n     As a practical matter, whether any particular nucleic acid molecule or polypeptide is at least about 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or 99.9% identical to a nucleotide sequence of the present invention can be determined conventionally using known computer programs. \\n     A representative method for determining the best overall match between a query sequence (e.g., a sequence of the present invention) and a subject sequence, also referred to as a global sequence alignment, can be determined using the CLUSTALW computer program (Thompson et al., (1994)  Nucl. Acids Res.  2(22):4673–4680), which is based on the algorithm of Higgins et al., (1992)  Computer Applications in the Biosciences  (CABIOS) 8(2):189–191. In a sequence alignment, the query and subject sequences are both DNA sequences. An RNA sequence can be compared by converting U&#39;s to T&#39;s. However, the CLUSTALW algorithm automatically converts U&#39;s to T&#39;s when comparing RNA sequences to DNA sequences. The result of a global sequence alignment is expressed in percent identity. \\n     Representative parameters used in a CLUSTALW alignment of DNA sequences to calculate percent identity via pairwise alignments are: Matrix=IUB, k-tuple=1, Number of Top Diagonals=5, Gap Penalty=3, Gap Open Penalty 10, Gap Extension Penalty=0.1, Scoring Method=Percent, Window Size=5 or the length of the subject nucleotide sequence, whichever is shorter. For multiple alignments, the following CLUSTALW parameters are preferred: Gap Opening Penalty=10; Gap Extension Parameter=0.05; Gap Separation Penalty Range=8; End Gap Separation Penalty=Off; % Identity for Alignment Delay=40%; Residue Specific Gaps:Off; Hydrophilic Residue Gap=Off; and Transition Weighting=0. The pairwise and multple alignment parameters provided for CLUSTALW above represent the default parameters as provided with the ALIGNX® software program (Vector NTI suite of programs, version 6.0, Informax, Frederick, Md., USA). \\n     The present invention encompasses the application of an optional manual correction to the percent identity results, in the instance where the subject sequence is shorter than the query sequence because of 5′ or 3′ deletions, not because of internal deletions. If only the local pairwise percent identity is required, no manual correction is needed. However, a manual correction can be applied to determine the global percent identity from a global polynucleotide alignment. Percent identity calculations based upon global polynucleotide alignments are often preferred since they reflect the percent identity between the polynucleotide molecules as a whole (i.e., including any polynucleotide overhangs, not just overlapping regions), as opposed to, only local matching polynucleotides. Manual corrections for global percent identity determinations are required since the CLUSTALW program does not account for 5′ and 3′ truncations of the subject sequence when calculating percent identity. For subject sequences truncated at the 5′ or 3′ ends, relative to the query sequence, the percent identity is corrected by calculating the number of bases of the query sequence that are 5′ and 3′ of the subject sequence, which are not matched/aligned, as a percent of the total bases of the query sequence. Whether a nucleotide is matched/aligned is determined by results of the CLUSTALW sequence alignment. This percentage is then subtracted from the percent identity, calculated by the above CLUSTALW program using the specified parameters, to arrive at a final percent identity score. This corrected score can be used for the purposes of the present invention. Only bases outside the 5′ and 3′ bases of the subject sequence, as displayed by the CLUSTALW alignment, which are not matched/aligned with the query sequence, are calculated for the purposes of manually adjusting the percent identity score. \\n     For example, a 90 base subject sequence is aligned to a 100 base query sequence to determine percent identity. The deletions occur at the 5′ end of the subject sequence and therefore, the CLUSTALW alignment does not show a matched/alignment of the first 10 bases at 5′ end. The 10 unpaired bases represent 10% of the sequence (number of bases at the 5′ and 3′ ends not matched/total number of bases in the query sequence) so 10% is subtracted from the percent identity score calculated by the CLUSTALW program. If the remaining 90 bases were perfectly matched the final percent identity would be 90%. In another example, a 90 base subject sequence is compared with a 100 base query sequence. This time the deletions are internal deletions so that there are no bases on the 5′ or 3′ of the subject sequence that are not matched/aligned with the query. In this case the percent identity calculated by CLUSTALW is not manually corrected. Once again, only bases 5′ and 3′ of the subject sequence which are not matched/aligned with the query sequence are manually corrected for. No other manual corrections are required for the purposes of the present invention. \\n     In addition to the above method of aligning two or more polynucleotide or polypeptide sequences to arrive at a percent identity value for the aligned sequences, it can be desirable in some circumstances to use a modified version of the CLUSTALW algorithm which takes into account known structural features of the sequences to be aligned, such as for example, the SWISS-PROT designations for each sequence. The result of such a modifed CLUSTALW algorithm can provide a more accurate value of the percent identity for two polynucleotide or polypeptide sequences. Support for such a modified version of CLUSTALW is provided within the CLUSTALW algorithm and would be readily appreciated to one of skill in the art of bioinformatics. \\n     The variants may contain alterations in the coding regions, non-coding regions, or both. Especially preferred are polynucleotide variants containing alterations that produce silent substitutions, additions, or deletions, but do not alter the properties or activities of the encoded polypeptide. Nucleotide variants produced by silent substitutions due to the degeneracy of the genetic code are often desirable. Moreover, variants in which 5–10, 1–5, or 1–2 amino acids are substituted, deleted, or added in any combination are also often desirable. Polynucleotide variants can be produced for a variety of reasons, e.g., to optimize codon expression for a particular host (change codons in the mRNA to those preferred by a bacterial host such as  E. coli ). \\n     Naturally occurring variants are called “allelic variants” and refer to one of several alternate forms of a gene occupying a given locus on a chromosome of an organism. (see, e.g., Lewin,  Genes VII , Oxford University Press, New York, USA (2000), incorporated herein in its entirety). These allelic variants can vary at either the polynucleotide and/or polypeptide level and are included in the present invention. Alternatively, non-naturally occurring variants may be produced by mutagenesis techniques or by direct synthesis. \\n     Polypeptide Variants \\n     Using methods of protein engineering and recombinant DNA technology known to those of ordinary skill in the art, variants can be generated to improve or alter the characteristics of the polypeptides of the present invention. For instance, one or more amino acids can be deleted from the N-terminus or C-terminus of the protein without substantial loss of biological function. For example, it has been reported that variant KGF proteins having heparin binding activity even after deleting 3, 8, or 27 amino-terminal amino acid residues (Ron et al., (1993)  J. Biol. Chem.  268:2984–2988). Similarly, interferon gamma exhibited up to ten times higher activity after deleting 8–10 amino acid residues from the carboxy terminus of this protein (Dobeli et al., (1988)  J. Biotechnol.  7:199–216). \\n     Moreover, ample evidence demonstrates that variants often retain a biological activity similar to that of the naturally occurring protein. For example, an extensive mutational analysis of human cytokine IL-1a was conducted (Gayle et al., (1993)  J. Biol. Chem.  268:22105–22111). In this study, random mutagenesis as employed to generate over 3,500 individual IL-1a mutants that averaged 2.5 amino acid changes per variant over the entire length of the molecule. Multiple mutations were examined at every possible amino acid position. It was found that “[m]ost of the molecule could be altered with little effect on either [binding or biological activity].” Gayle et al. at 22109. In fact, only 23 unique amino acid sequences, out of more than 3,500 nucleotide sequences examined, produced a protein that significantly differed in activity from wild-type. \\n     Furthermore, even if deleting one or more amino acids from the N-terminus or C-terminus of a polypeptide results in modification or loss of one or more biological functions, other biological activities might still be retained. For example, the ability of a deletion variant to induce and/or to bind antibodies that recognize the protein will likely be retained when less than the majority of the residues of the protein are removed from the N-terminus or C-terminus. Whether a particular polypeptide lacking N- or C-terminal residues of a protein retains such immunogenic activities can readily be determined by routine methods described herein and otherwise known to those of ordinary skill in the art. \\n     Alternatively, such N-terminus or C-terminus deletions of a polypeptide of the present invention might, in fact, result in a significant increase in one or more of the biological activities of the polypeptide(s). For example, a given biological activity of many polypeptides is governed by the presence of regulatory domains at either one or both termini. Such regulatory domains effectively inhibit the biological activity of such polypeptides in lieu of an activation event (e.g., binding to a cognate ligand or receptor, phosphorylation, proteolytic processing, etc.). Thus, by eliminating the regulatory domain of a polypeptide, the polypeptide can effectively be rendered biologically active in the absence of an activation event. \\n     Thus, the present invention further includes polypeptide variants that show substantial biological activity. Such variants include deletions, insertions, inversions, repeats, and substitutions selected according to general rules known in the art so as have little effect on activity. For example, guidance concerning how to make phenotypically silent amino acid substitutions is provided in several references (e.g., Bowie et al., (1990)  Science  247:1306–1310), wherein it is indicated that there are two main strategies for studying the tolerance of an amino acid sequence to change. \\n     The first strategy exploits the tolerance of amino acid substitutions by natural selection during the process of evolution. By comparing amino acid sequences in different species, conserved amino acids can be identified. These conserved amino acids are likely important for protein function. In contrast, the amino acid positions where substitutions have been tolerated by natural selection indicates that these positions are not critical for protein function. Thus, positions tolerating amino acid substitution could be modified while still maintaining biological activity of the protein. \\n     The second strategy employs genetic engineering to introduce amino acid changes at specific positions of a cloned gene to identify regions critical for protein function. For example, site directed mutagenesis or alanine-scanning mutagenesis (introduction of single alanine mutations at every residue in the molecule) can be employed (Cunningham &amp; Wells, (1989)  Science  244:1081–1085). The resulting mutant molecules can then be tested for biological activity. \\n     These two strategies have revealed that proteins are surprisingly tolerant of amino acid substitutions. The referenced studies further indicate which amino acid changes are likely to be permissive at certain amino acid positions in the protein. For example, most amino acid residues buried within the tertiary structure of the protein require nonpolar side chains, whereas few features of surface side chains are generally conserved. \\n     In one aspect, the present invention encompasses polypeptides having a lower degree of identity but having sufficient similarity so as to perform one or more of the same functions performed by a polypeptide of the present invention. Similarity is determined by conserved amino acid substitution. Conservative substitutions are substitutions that replace a given amino acid in a polypeptide by another amino acid having like characteristics (e.g., chemical properties). Such conservative substitutions are likely to be phenotypically silent (Cunningham &amp; Wells, (1989)  Science  244:1081–1085). Other references supply additional guidance concerning which amino acid changes are likely to be phenotypically silent (see, e.g., Bowie et al., (1990)  Science  247:1306–1310). \\n     In one embodiment, tolerated conservative amino acid substitutions of the present invention involve replacement of the aliphatic or hydrophobic amino acids Ala, Val, Leu and Ile; replacement of the hydroxyl residues Ser and Thr; replacement of the acidic residues Asp and Glu; replacement of the amide residues Asn and Gln, replacement of the basic residues Lys, Arg, and His; replacement of the aromatic residues Phe, Tyr, and Trp, and replacement of the small-sized amino acids Ala, Ser, Thr, Met, and Gly. \\n     In addition, the present invention also encompasses the conservative substitutions provided in Table III: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                 TABLE III \\n               \\n               \\n                   \\n               \\n               \\n                 For Amino Acid \\n                 Code \\n                 Replace with any of: \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Alanine \\n                 A \\n                 D-Ala, Gly, beta-Ala, L-Cys, D-Cys \\n               \\n               \\n                 Arginine \\n                 R \\n                 D-Arg, Lys, D-Lys, homo-Arg, D-homo-Arg, Met, Ile, D-Met, \\n               \\n               \\n                   \\n                   \\n                 D-Ile, Orn, D-Orn \\n               \\n               \\n                 Asparagine \\n                 N \\n                 D-Asn, Asp, D-Asp, Glu, D-Glu, Gln, D-Gln \\n               \\n               \\n                 Aspartic Acid \\n                 D \\n                 D-Asp, D-Asn, Asn, Glu, D-Glu, Gln, D-Gln \\n               \\n               \\n                 Cysteine \\n                 C \\n                 D-Cys, S-Me-Cys, Met, D-Met, Thr, D-Thr \\n               \\n               \\n                 Glutamine \\n                 Q \\n                 D-Gln, Asn, D-Asn, Glu, D-Glu, Asp, D-Asp \\n               \\n               \\n                 Glutamic Acid \\n                 E \\n                 D-Glu, D-Asp, Asp, Asn, D-Asn, Gln, D-Gln \\n               \\n               \\n                 Glycine \\n                 G \\n                 Ala, D-Ala, Pro, D-Pro, β-Ala, Acp \\n               \\n               \\n                 Isoleucine \\n                 I \\n                 D-Ile, Val, D-Val, Leu, D-Leu, Met, D-Met \\n               \\n               \\n                 Leucine \\n                 L \\n                 D-Leu, Val, D-Val, Met, D-Met \\n               \\n               \\n                 Lysine \\n                 K \\n                 D-Lys, Arg, D-Arg, homo-Arg, D-homo-Arg, Met, D- \\n               \\n               \\n                   \\n                   \\n                 Met, Ile, D-Ile, Orn, D-Orn \\n               \\n               \\n                 Methionine \\n                 M \\n                 D-Met, S-Me-Cys, Ile, D-Ile, Leu, D-Leu, Val, D-Val \\n               \\n               \\n                 Phenylalanine \\n                 F \\n                 D-Phe, Tyr, D-Thr, L-Dopa, His, D-His, Trp, D-Trp, \\n               \\n               \\n                   \\n                   \\n                 Trans-3,4, or 5-phenylproline, cis-3,4, or 5-phenylproline \\n               \\n               \\n                 Proline \\n                 P \\n                 D-Pro, L-1-thioazolidine-4-carboxylic acid, D- or L-1- \\n               \\n               \\n                   \\n                   \\n                 oxazolidine-4-carboxylic acid \\n               \\n               \\n                 Serine \\n                 S \\n                 D-Ser, Thr, D-Thr, allo-Thr, Met, D-Met, Met(O), D- \\n               \\n               \\n                   \\n                   \\n                 Met(O), L-Cys, D-Cys \\n               \\n               \\n                 Threonine \\n                 T \\n                 D-Thr, Ser, D-Ser, allo-Thr, Met, D-Met, Met(O), D- \\n               \\n               \\n                   \\n                   \\n                 Met(O), Val, D-Val \\n               \\n               \\n                 Tyrosine \\n                 Y \\n                 D-Tyr, Phe, D-Phe, L-Dopa, His, D-His \\n               \\n               \\n                 Valine \\n                 V \\n                 D-Val, Leu, D-Leu, Ile, D-Ile, Met, D-Met \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Aside from the uses described above, such amino acid substitutions can also increase protein or peptide stability. The present invention encompasses amino acid substitutions that contain, for example, one or more non-peptide bonds (which replace the peptide bonds) in the protein or peptide sequence. Also included are substitutions that include amino acid residues other than naturally occurring L-amino acids, e.g., D-amino acids or non-naturally occurring or synthetic amino acids, e.g., β or γ amino acids. Both identity and similarity can be readily calculated by those of ordinary skill in the art (see, e.g.,  Computational Molecular Biology , (Lesk, ed.), Oxford University Press, New York, N.Y., USA (1988);  Biocomputing: Informatics and Genome Projects , (Smith, ed.), Academic Press, New York, N.Y., USA (1993);  Informatics Computer Analysis of Sequence Data, Part  1, (Griffin &amp; Griffin, eds.), Humana Press, Totowa, N.J., USA (1994); von Heinje,  Sequence Analysis in Molecular Biology , Academic Press, New York, N.Y., USA (1987); and Sequence Analysis Primer, (Gribskov &amp; Devereux, eds.), Stockton Press, New York, N.Y., USA (1991)). \\n     In addition, the present invention also encompasses substitution of amino acids based upon the probability of an amino acid substitution resulting in conservation of function. Such probabilities are determined by aligning multiple genes with related function and assessing the relative penalty of each substitution to proper gene function. Such probabilities are often described in a matrix and are used by some algorithms (e.g., BLAST, CLUSTALW, GAP, etc.) in calculating percent similarity wherein similarity refers to the degree by which one amino acid may substitute for another amino acid without lose of function. An example of such a matrix is the PAM250 or BLOSUM62 matrix. \\n     Aside from the canonical chemically conservative substitutions referenced above, the present invention also encompasses substitutions which are typically not classified as conservative, but that can be chemically conservative under certain circumstances. Analysis of enzymatic catalysis for proteases, for example, has shown that certain amino acids within the active site of some enzymes may have highly perturbed pKa&#39;s due to the unique microenvironment of the active site. Such perturbed pKa&#39;s could enable some amino acids to substitute for other amino acids while conserving enzymatic structure and function. Examples of amino acids that are known to have amino acids with perturbed pKa&#39;s are the Glu-35 residue of lysozyme, the Ile-16 residue of chymotrypsin, the His-159 residue of papain, etc. The conservation of function relates to either anomalous protonation or anomalous deprotonation of such amino acids, relative to their canonical, non-perturbed pKa. The pKa perturbation may enable these amino acids to actively participate in general acid-base catalysis due to the unique ionization environment within the enzyme active site. Thus, substituting an amino acid capable of serving as either a general acid or general base within the microenvironment of an enzyme active site or cavity, as may be the case, in the same or similar capacity as the wild-type amino acid, would effectively serve as a conservative amino substitution. \\n     Variants of the present invention are not limited to conservative substitutions. Besides conservative amino acid substitution, variants of the present invention include, but are not limited to, the following: (i) substitutions with one or more of the non-conserved amino acid residues, where the substituted amino acid residues may or may not be one encoded by the genetic code; (ii) substitution with one or more of amino acid residues having a substituent group; (iii) fusion of the mature polypeptide with another compound, such as a compound to increase the stability and/or solubility of the polypeptide (for example, polyethylene glycol); and (iv) fusion of the polypeptide with additional amino acids, such as, for example, an IgG Fc fusion region peptide, or leader or secretory sequence, or a sequence facilitating purification. Such variant polypeptides are deemed to be within the scope of those skilled in the art upon consideration of the present disclosure. \\n     For example, polypeptide variants containing amino acid substitutions of charged amino acids with other charged or neutral amino acids might produce proteins with improved characteristics, such as less aggregation. Aggregation of pharmaceutical formulations both reduces activity and increases clearance due to the aggregate&#39;s immunogenic activity (see Pinckard et al., (1967)  Clin. Exp. Immunol.  2:331–340; Robbins et al., (1987)  Diabetes  36: 838–845; Cleland et al., (1993)  Crit. Rev. Therapeutic Drug Carrier Systems  10:307–377). \\n     The present invention further includes polypeptide variants created through the application of molecular evolution (“DNA Shuffling”) methodology to the polynucleotide disclosed as SEQ ID NO:X, the sequence of the clone submitted in a deposit, and/or the cDNA encoding the polypeptide disclosed as SEQ ID NO:Y. Such DNA Shuffling technology is known in the art and more particularly described elsewhere herein (e.g., Stemmer, (1994)  Proc. Natl. Acad. Sci. U.S.A.  91:10747), and in the Examples provided herein). \\n     Thus, an embodiment of the present invention relates to a polypeptide comprising an amino acid sequence of the present invention containing at least one amino acid substitution, but not more than 50 amino acid substitutions (e.g., 40 amino acid substitutions, 30 amino acid substitutions, 20 amino acid substitutions, 10 amino acid substitutions or 5 amino acid substitutions). In one embodiment, an amino acid sequence of the present invention can comprise at least one, but not more than 10, 9, 8, 7, 6, 5, 4, 3, 2 or 1 amino acid substitutions. In specific embodiments, the number of additions, substitutions, and/or deletions in an amino acid sequence of the present invention or fragments thereof (e.g., the mature form and/or other fragments described herein), is 1–5,5–10, 5–25, 5–50, 10–50 or 50–150 and conservative amino acid substitutions can be desirable under some circumstances. \\n     Polynucleotide and Polypeptide Fragments \\n     In addition to the full-length polypeptides encoded by full-length polynucleotides, the present invention is directed to polynucleotide fragments of the polynucleotides of the present invention, as well as to polypeptides encoded by such polynucleotides and/or fragments. \\n     In the present invention, a “polynucleotide fragment” refers to a short polynucleotide having a nucleic acid sequence which: (i) is a portion of that contained in a deposited clone, or encodes a portion of a polypeptide encoded by the cDNA in a deposited clone; (ii) is a portion of that shown in SEQ ID NO:X or the complementary strand thereto; or (iii) is a portion of a polynucleotide sequence encoding a polypeptide of SEQ ID NO:Y. The nucleotide fragments of the present invention are, for example, at least about 15 nt, at least about 20 nt, at least about 30 nt, at least about 40 nt, at least about 50 nt, at least about 75 nt, or at least about 150 nt in length. A fragment “at least 20 nt in length” for example, includes 20 or more contiguous bases from the cDNA sequence contained in a deposited clone or a nucleotide sequence shown in SEQ ID NO:X. In this context “about” includes the particularly recited value, a value larger or smaller by several (e.g., 5, 4, 3, 2, or 1) nucleotides, at either terminus, or at both termini. These nucleotide fragments have uses that include, but are not limited to, as diagnostic probes and primers as discussed herein. Of course, larger fragments (e.g., 50, 150, 500, 600, 2000 nucleotides) are contemplated and can be desirable in certain circumstances. \\n     Moreover, representative examples of polynucleotide fragments of the present invention, include, for example, fragments comprising, or alternatively consisting of, a sequence from about nucleotide number 1–50, 51–100, 101–150, 151–200, 201–250, 251–300, 301–350, 351–400, 401–450, 451–500, 501–550, 551–600, 651–700, 701–750, 751–800, 800–850, 851–900, 901–950, 951–1000, 1001–1050, 1051–1100, 1101–1150, 1151–1200, 1201–1250, 1251–1300, 1301–1350, 1351–1400, 1401–1450, 1451–1500, 1501–1550, 1551–1600, 1601–1650, 1651–1700, 1701–1750, 1751–1800, 1801–1850, 1851–1900, 1901–1950, 1951–2000, or 2001 (as applicabale based on the length of SEQ ID NO:X) to the end of SEQ ID NO:X, or the complementary strand thereto, or the cDNA contained in a deposited clone. In this context “about” includes the particularly recited ranges, and ranges larger or smaller by several (e.g., 5, 4, 3, 2, or 1) nucleotides, at either terminus or at both termini. In one embodiment, these fragments encode a polypeptide that has biological activity. These polynucleotides can be used as probes or primers as discussed herein. Also encompassed by the present invention are polynucleotides that hybridize to these nucleic acid molecules under stringent hybridization conditions or lower stringency conditions, as are the polypeptides encoded by these polynucleotides. \\n     In the present invention, a “polypeptide fragment” refers to an amino acid sequence that is a portion of a sequence of SEQ ID NO:Y or encoded by the cDNA contained in a deposited clone. Protein (polypeptide) fragments can be “free-standing” or comprised within a larger polypeptide of which the fragment forms a part or region, most preferably as a single continuous region. Representative examples of polypeptide fragments of the present invention, include, for example, fragments comprising, or alternatively consisting of, from about amino acid number 1–20, 21–40, 41–60, 61–80, 81–100, 102–120, 121–140, 141–160, or 161 to the end of the coding region. Moreover, polypeptide fragments can be about 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, or 150 amino acids in length. In this context “about” includes the particularly recited ranges or values, and ranges or values larger or smaller by several (e.g., 5, 4, 3, 2, or 1) amino acids, at either extreme or at both extremes. Polynucleotides encoding these polypeptides are also encompassed by the present invention. \\n     In one embodiment, a polypeptide fragment includes the full-length protein. Further preferred polypeptide fragments include the full-length protein having a continuous series of deleted residues from either the amino or the carboxy terminus of the polypeptide, or both. For example, any number of amino acids, e.g. ranging from 1–60, can be deleted from the amino terminus of the full-length polypeptide. Similarly, any number of amino acids, ranging from 1–30, can be deleted from the carboxy terminus of the full-length protein. Furthermore, any combination of the above amino and carboxy terminus deletions can be employed. Similarly, polynucleotides encoding these polypeptide fragments are also an aspect of the present invention. Examples of N- and C-terminal deletions are provided hereinabove. \\n     In yet other embodiments, polypeptide and polynucleotide fragments can be characterized by structural or functional domains, such as fragments that comprise alpha-helix and alpha-helix-forming regions, beta-sheet and beta-sheet-forming regions, turn and turn-forming regions, coil and coil-forming regions, hydrophilic regions, hydrophobic regions, alpha amphipathic regions, beta amphipathic regions, flexible regions, surface-forming regions, substrate binding region, and high antigenic index regions. Polypeptide fragments of SEQ ID NO:Y falling within conserved domains are specifically contemplated by the present invention. Moreover, polynucleotides encoding these domains are also contemplated and are within the scope of the present invention. \\n     In another embodiment, a polypeptide fragment is a biologically active fragment. Biologically active fragments are those exhibiting activity similar, but not necessarily identical, to an activity of a polypeptide of the present invention. The biological activity of the fragments can include an improved desired activity, or a decreased undesirable activity. Polynucleotides encoding these polypeptide fragments are also encompassed by the present invention. \\n     In a representative embodiment, the functional activity displayed by a polypeptide encoded by a polynucleotide fragment of the present invention can comprise one or more biological activities typically associated with a full-length polypeptide of the present invention. Illustrative examples of these biological activities include a fragment&#39;s ability to bind to at least one of the same antibodies which bind to the full-length protein; a fragment&#39;s ability to interact with at least one of the same proteins that bind to the full-length protein; a fragment&#39;s ability to elicit at least one of the same immune responses as the full-length protein (i.e., to cause the immune system to create antibodies specific to the same epitope, etc.); a fragment&#39;s ability to bind to at least one of the same polynucleotides as the full-length protein; the fragment&#39;s ability to bind to a receptor of the full-length protein; a fragment&#39;s ability to bind to a ligand of the full-length protein; and a fragment&#39;s ability to multimerize with the full-length protein. However, the skilled artisan will appreciate that some fragments may have biological activities that are desirable and directly inapposite to the biological activity of the full-length protein. The functional activity of polypeptides of the present invention, including fragments, variants, derivatives, and analogs thereof can be determined by numerous methods available to the skilled artisan and will be apparent upon consideration of the present disclosure, some of which are described herein. \\n     Polypeptide and Polypeptide Fragment Epitopes \\n     The present invention encompasses polypeptides comprising, or alternatively consisting of, an epitope of the polypeptide having an amino acid sequence of SEQ ID NO:Y, or an epitope of the polypeptide sequence encoded by a polynucleotide sequence contained in ATCC Deposit No.:Z or encoded by a polynucleotide that hybridizes to the complement of the sequence of SEQ ID NO:X or contained in ATCC Deposit No.:Z under stringent hybridization conditions or lower stringency hybridization conditions as defined herein or known to those of ordinary skill in the art. Such epitopes can comprise a polypeptide fragment. The present invention further encompasses polynucleotide sequences encoding an epitope of a polypeptide sequence of the present invention (such as, for example, an epitope of a sequence disclosed in SEQ ID NO:Y), polynucleotide sequences of the complementary strand of a polynucleotide sequence encoding an epitope of the present invention, and polynucleotide sequences that hybridize to a complementary strand under stringent hybridization conditions or lower stringency hybridization conditions, as defined herein or known to those or ordinary skill in the art. \\n     Fragments that function as epitopes can be produced by any conventional means (see, e.g., Houghten, (1985)  Proc. Natl. Acad. Sci. USA  82:5131–5135 and U.S. Pat. No. 4,631,211). \\n     In embodments of the present invention, antigenic epitopes, as defined herein, contain a sequence of at least 4, at least 5, at least 6, at least 7, more preferably at least 8, at least 9, at least 10, at least 11, at least 12, at least 13, at least 14, at least 15, at least 20, at least 25, at least 30, at least 40, at least 50, and, most preferably, between about 15 to about 30 amino acids. Representative polypeptides comprising immunogenic or antigenic epitopes are at least 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, or 100 amino acid residues in length, or longer. Additional non-exclusive preferred antigenic epitopes include the antigenic epitopes disclosed herein, as well as portions thereof. Antigenic epitopes are useful, for example, to raise antibodies, including monoclonal antibodies, that specifically bind the epitope. Examples of antigenic epitopes include the antigenic epitopes disclosed herein, as well as any combination of two, three, four, five or more of these antigenic epitopes. Antigenic epitopes can be used as the target molecules in immunoassays (see, e.g., Wilson et al., (1984)  Cell  37:767–778; Sutcliffe et al., (1983)  Science  219:660–666). \\n     Similarly, immunogenic epitopes can be used, for example, to induce antibodies according to methods well known in the art (see, e.g., Sutcliffe et al., (1983)  Science  219:660–666; Wilson et al., (1984)  Cell  37:767–778; Chow et al.,  Proc. Natl. Acad. Sci. USA  82:910–914; and Bittle et al., (1985)  J. Gen. Virol.  66:2347–2354). Examples of immunogenic epitopes include the immunogenic epitopes disclosed herein, as well as any combination of two, three, four, five or more of these immunogenic epitopes. \\n     A polypeptide comprising one or more immunogenic epitopes can be presented for eliciting an antibody response together with a carrier protein, such as an albumin, to an animal system (such as rabbit or mouse), or, if the polypeptide is of sufficient length (e.g., at least about 25 amino acids), the polypeptide can be presented without a carrier. Immunogenic epitopes comprising as few as 8 to 10 amino acids have, however, been shown to be sufficient to raise antibodies capable of binding to, at the very least, linear epitopes in a denatured polypeptide (e.g., in western blotting). \\n     Epitope-bearing polypeptides of the present invention can be used to induce antibodies according to methods well known in the art including, but not limited to, in vivo immunization, in vitro immunization, and phage display methods (see, e.g., Sutcliffe et al., (1983)  Science  219:660–666; Wilson et al., (1984)  Cell  37:767–778; Chow et al.,  Proc. Natl. Acad. Sci. USA  82:910–914; and Bittle et al., (1985)  J. Gen. Virol.  66:2347–2354). If in vivo immunization is employed, animals can be immunized with free peptide; however, anti-peptide antibody titer can be boosted by coupling the peptide to a macromolecular carrier, such as keyhole limpet hemacyanin (KLH) or tetanus toxoid. For instance, peptides containing cysteine residues can be coupled to a carrier using a linker such as maleimidobenzoyl-N-hydroxysuccinimide ester (MBS), while other peptides can be coupled to carriers using a more general linking agent such as glutaraldehyde. Animals such as rabbits, rats and mice are immunized with either free or carrier-coupled peptides, for instance, by intraperitoneal and/or intradermal injection of emulsions containing about 100 μg of peptide or carrier protein and Freund&#39;s adjuvant or any other adjuvant known for stimulating an immune response. Several booster injections might be needed, for instance, at intervals of about two weeks, to provide a useful titer of anti-peptide antibody which can be detected, for example, by ELISA assay using free peptide adsorbed to a solid surface. The titer of anti-peptide antibodies in serum from an immunized animal can be increased by selection of anti-peptide antibodies, for instance, by adsorption to the peptide on a solid support and elution of the selected antibodies according to methods well known in the art. \\n     As those of ordinary skill in the art will appreciate, and as discussed herein, a polypeptide of the present invention comprising an immunogenic or antigenic epitope can be fused to other polypeptide sequences. For example, a polypeptide of the present invention can be fused with a constant domain of an immunoglobulin (IgA, IgE, IgG, IgM), or a portion thereof (e.g., CH1, CH2, CH3, or any combination thereof and portions thereof), resulting in a chimeric polypeptide. As described further herein, in the context of the present invention a chimeric polypeptide comprises a full-length or fragment of SEQ ID NO:X fused with a sequence not derived from the same SEQ ID NO:X. Such fusion proteins can facilitate purification and can increase half-life in vivo. This has been shown for chimeric proteins consisting of the first two domains of the human CD4-polypeptide and various domains of the constant regions of the heavy or light chains of mammalian immunoglobulins (see, e.g., EP 394,827; Traunecker et al., (1988)  Nature  331:84–86). Enhanced delivery of an antigen across the epithelial barrier to the immune system has been demonstrated for antigens (e.g., insulin) conjugated to an FcRn binding partner such as IgG or Fc fragments (see, e.g., PCT Publications WO 96/22024 and WO 99/04813). IgG fusion proteins that have a disulfide-linked dimeric structure due to the IgG portion disulfide bonds have also been found to be more efficient in binding and neutralizing other molecules than monomeric polypeptides or fragments thereof alone (see, e.g., Fountoulakis et al., (1995)  J. Biochem.  270:3958–3964). \\n     Nucleic acids encoding the above epitopes can also be recombined with a gene of interest as an epitope tag (e.g., the hemagglutinin (“HA”) tag or FLAG® tag (Sigma, St. Louis, Mo., USA; SEQ ID NO:18) to aid in detection and purification of the expressed polypeptide. For example, one system allows for the ready purification of non-denatured fusion proteins expressed in human cell lines (Janknecht et al., (1991)  Proc. Natl. Acad. Sci. USA  88:8972–897). In this system, the gene of interest is subcloned into a vaccinia recombination plasmid such that the open reading frame of the gene is translationally fused to an amino-terminal tag conssting of six histidine residues. The tag serves as a matrix-binding domain for the fusion protein. Extracts from cells infected with the recombinant vaccinia virus are loaded onto Ni 2+  nitriloacetic acid-agarose column and histidine-tagged proteins can be selectively eluted with imidazole-containing buffers. \\n     Additional fusion proteins of the present invention may be generated through the techniques of gene-shuffling, motif-shuffling, exon-shuffling, and/or codon-shuffling (collectively referred to as “DNA shuffling”). DNA shuffling can be employed to modulate the activities of a polypeptide of the present invention, and such methods can be used to generate polypeptides with altered activity, as well as agonists and antagonists of the polypeptides. See, generally, U.S. Pat. Nos. 5,605,793; 5,811,238; 5,830,721; 5,834,252; and 5,837,458; and Patten et al., (1997)  Curr. Opinion Biotechnol.  8:724–33; Harayama, (1998)  Trends Biotechnol.  16(2):76–82; Hansson et al., (1999)  J. Mol. Biol.  287:265–76; and Lorenzo &amp; Blasco, (1998)  Biotechniques  24(2):308–13. In one embodiment, alteration of a polynucleotide corresponding to SEQ ID NO:X and a polypeptide encoded by such polynucleotides can be achieved by DNA shuffling. DNA shuffling involves the assembly of two or more DNA segments by homologous or site-specific recombination to generate variation in the polynucleotide sequence. In another embodiment, a polynucleotide of the present invention, or an encoded polypeptide, can be altered by being subjected to random mutagenesis by error-prone PCR, random nucleotide insertion or other methods prior to recombination. In another embodiment, one or more components, motifs, sections, parts, domains, fragments, etc., of a polynucleotide encoding a polypeptide of the present invention can be recombined with one or more components, motifs, sections, parts, domains, fragments, etc. of one or more heterologous molecules. \\n     Antibodies \\n     Further polypeptides of the present invention relate to antibodies and T-cell antigen receptors (TCR) which immunospecifically bind a polypeptide, polypeptide fragment, or variant of SEQ ID NO:Y, and/or an epitope, of the present invention (as determined by immunoassays well known in the art for assaying specific antibody-antigen binding). Antibodies of the present invention include, but are not limited to, polyclonal, monoclonal, monovalent, bispecific, heteroconjugate, multispecific, human, humanized or chimeric antibodies, single chain antibodies, Fab fragments, F(ab′) fragments, fragments produced by a Fab expression library, anti-idiotypic (anti-Id) antibodies (including, e.g., anti-Id antibodies to antibodies of the present invention), and epitope-binding fragments of any of the polypeptides and peptides disclosed herein. \\n     The antibodies of the present invention can be monospecific, bispecific, trispecific or of greater multispecificity. Multispecific antibodies can be specific for different epitopes of a polypeptide of the present invention or can be specific for both a polypeptide of the present invention as well as for a heterologous epitope, such as a heterologous polypeptide or solid support material. See, e.g., PCT publications WO 93/17715; WO 92/08802; WO 91/00360; WO 92/05793; U.S. Pat. Nos. 4,474,893; 4,714,681; 4,925,648; 5,573,920; 5,601,819; Kostelny et al., (1992)  J. Immunol.  148:1547–1553; Tutt et al., (1991)  J. Immunol.  147:60–69. \\n     Antibodies of the present invention can be described or specified in terms of an epitope(s) or portion(s) of a polypeptide of the present invention that a given antibody recognizes or specifically binds. The epitope(s) or polypeptide portion(s) may be specified as described herein, e.g., by N-terminal and C-terminal positions, by size in contiguous amino acid residues, or listed in the Tables, Figures and/or Sequence Listing. Antibodies that specifically bind any epitope or polypeptide of the present invention can also be excluded. Therefore, the present invention includes antibodies that specifically bind polypeptides of the present invention, and allows for the exclusion of the same. \\n     Antibodies of the present invention can also be described or specified in terms of their cross-reactivity. Antibodies that do not bind any other analog, ortholog, or homologue of a polypeptide of the present invention are included in the scope of the present invention. Antibodies that bind polypeptides with at least 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 65%, at least 60%, at least 55%, and at least 50% identity (as calculated using methods known in the art and described herein) to a polypeptide of the present invention are also encompassed by the present invention. In specific embodiments, antibodies of the present invention cross-react with murine, rat and/or rabbit homologues of human proteins and the corresponding epitopes thereof. Antibodies that do not bind polypeptides with less than 95%, less than 90%, less than 85%, less than 80%, less than 75%, less than 70%, less than 65%, less than 60%, less than 55%, and less than 50% identity (as calculated using methods known in the art and described herein) to a polypeptide of the present invention are also encompassed by the present invention. \\n     In a specific embodiment, the above-described cross-reactivity is with respect to any single specific antigenic or immunogenic polypeptide, or combination(s) of 2, 3, 4, 5, or more of the specific antigenic and/or immunogenic polypeptides disclosed herein. Further included in the present invention are antibodies that bind polypeptides encoded by polynucleotides which hybridize to a polynucleotide of the present invention under stringent hybridization conditions (such as those described herein). Antibodies of the present invention can also be described or specified in terms of their binding affinity to a polypeptide of the present invention. Representative binding affinities include those with a dissociation constant or Kd less than 5×10 −2  M, 10 −2  M, 5×10 −3  M, 10 −3  M, 5×10 −4  M, 10 −4  M, 5×10 −5  M, 1×10 −5  M, 5×10 −6  M, 1×10 −6  M, 5×10 −7  M, 1×10 −7  M, 5×10 −8  M, 1×10 −8  M, 5×10 −9  M, 1×10 −9  M, 5×10 −10  M, 1×10 −10  M, 5×10 −11  M, 1×10 −11  M, 5×10 −12  M, 10 −12  M, 5×10 −13  M, 1×10 −13  M, 5×10 −14  M, 1×10 −14  M, 5×10 −15  M, or 1×10 −15  M. \\n     The present invention also provides antibodies that competitively inhibit binding of an antibody to an epitope of the present invention as determined by any method known in the art for determining competitive binding, for example, the immunoassays described herein. In representative embodiments, the antibody competitively inhibits binding to the epitope by at least 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 60%, or at least 50%. \\n     Antibodies as Agonists and Antagonists \\n     Antibodies of the present invention may act as agonists or antagonists of the polypeptides of the present invention. For example, the present invention includes antibodies that disrupt the receptor/ligand interactions with the polypeptides of the present invention either partially or fully. In some cases, antibodies of the present invention bind an antigenic epitope disclosed herein, or a portion thereof. The present invention features both receptor-specific antibodies and ligand-specific antibodies. The present invention also features receptor-specific antibodies that do not prevent ligand binding but prevent receptor activation. Receptor activation (i.e., signaling) can be determined by techniques described herein or otherwise known in the art. For example, receptor activation can be determined by detecting the phosphorylation state of the receptor or its substrate (e.g., whether a given site, such as a tyrosine or serine/threonine residue is phosphorylated) by immunoprecipitation followed by western blot analysis (for example, as described herein). In specific embodiments, antibodies are provided that inhibit ligand activity and/or receptor activity by at least 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 60%, or at least 50% of the activity in absence of the antibody. \\n     The present invention also features receptor-specific antibodies that both prevent ligand binding and receptor activation as well as antibodies that recognize the receptor-ligand complex, and, in some cases, do not specifically recognize the unbound receptor or the unbound ligand. Likewise, included in the present invention are neutralizing antibodies that bind the ligand and prevent binding of the ligand to the receptor, as well as antibodies that bind the ligand, thereby preventing receptor activation, but do not prevent the ligand from binding the receptor. Also included in the present invention are antibodies that activate the receptor. These antibodies can act as receptor agonists, i.e., potentiate or activate either all or a subset of the biological activities of the ligand-mediated receptor activation, for example, by inducing dimerization of the receptor. The antibodies can be specified as agonists, antagonists or inverse agonists for biological activities comprising the specific biological activities of the peptides of the present invention disclosed herein. The above antibody agonists can be made using methods known in the art (see, e.g., PCT publication WO 96/40281; U.S. Pat. No. 5,811,097; Deng et al., (1998)  Blood  92(6):1981–1988; Chen et al., (1998)  Cancer Res.  58(16):3668–3678; Harrop et al., (1998)  J. Immunol.  161(4):1786–1794; Zhu et al., (1998)  Cancer. Res.  58(15):3209–3214; Yoon et al., (1998)  J. Immunol.  160(7):3170–3179; Prat et al., (1998)  J. Cell. Sci.  111(Pt2):237–247; Pitard et al., (1997)  J. Immunol. Methods  205(2):177–190; Liautard et al., (1997)  Cytokine  9(4):233–241; Carlson et al., (1997)  J. Biol. Chem.  272(17):11295–11301; Taryman et al., (1995)  Neuron  14(4):755–762; Muller et al., (1998)  Structure  6(9): 1153–1167; Bartunek et al., (1996)  Cytokine  8(1): 14–20). \\n     Uses of Antibodies \\n     Antibodies of the present invention can be used in a variety of applications. A representative, but non-limiting, list of applications for antibodies of the present invention includes to purify, detect, and target the polypeptides of the present invention, including both in vitro and in vivo diagnostic and therapeutic methods. For example, the antibodies have use in immunoassays for qualitatively and quantitatively measuring expression levels and amounts of the polypeptides of the present invention in biological samples (see, e.g., Harlow et al.,  Antibodies: A Laboratory Manual,  2 nd  ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (1988)). \\n     As discussed in more detail below, the antibodies of the present invention can be used either alone or in combination with other compositions. The antibodies can further be recombinantly fused to a heterologous polypeptide at the N- or C-terminus or chemically conjugated (including covalently and non-covalently conjugations) to polypeptides or other compositions. For example, antibodies of the present invention can be recombinantly fused or conjugated to molecules useful as labels in detection assays and effector molecules such as heterologous polypeptides, drugs, radionucleotides, or toxins (see, e.g., PCT publications WO 92/08495; WO 91/14438; WO 89/12624; U.S. Pat. No. 5,314,995; and EP 396,387). \\n     The antibodies of the present invention include derivatives that are modified, i.e., by the covalent attachment of any type of molecule to the antibody such that covalent attachment does not prevent the antibody from generating an anti-idiotypic response. For example, but not by way of limitation, the antibody derivatives include antibodies that have been modified, e.g., by glycosylation, acetylation, pegylation, phosphorylation, amidation, derivatization by known protecting/blocking groups, proteolytic cleavage, linkage to a cellular ligand or other protein, etc. Any of numerous chemical modifications can be carried out by known techniques, including, but not limited to specific chemical cleavage, acetylation, formylation, metabolic synthesis of tunicamycin, etc. Additionally, the derivative can contain one or more non-classical amino acids. The antibodies of the present invention can be generated by any suitable method known in the art. \\n     Polyclonal Antibodies \\n     The antibodies of the present invention can comprise polyclonal antibodies. Methods of preparing polyclonal antibodies are known to the skilled artisan (Harlow et al.,  Antibodies: A Laboratory Manual,  2 nd  ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (1988)). In a preferred method, a preparation of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) protein is prepared and purified to render it substantially free of natural contaminants. Such a preparation is then introduced into an animal in order to produce polyclonal antisera of greater specific activity. For example, a polypeptide of the present invention can be administered to various host animals including, but not limited to, rabbits, mice, rats, etc. to induce the production of sera containing polyclonal antibodies specific for the antigen. The administration of the polypeptides, of the present invention can entail one or more injections of an immunizing agent and, if desired, an adjuvant. Various adjuvants can be used to increase the immunological response, depending on the host species, and include but are not limited to, Freund&#39;s (complete and incomplete), mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, pluronic polyols, polyanions, peptides, oil emulsions, keyhole limpet hemocyanins, dinitrophenol, and potentially useful human adjuvants such as BCG (bacille Calmette-Guerin) and  corynebacterium parvum . Such adjuvants are also well known to those or ordinary skill in the art. For the purposes of the present invention, “immunizing agent” is defined as a polypeptide of the present invention, including fragments, variants, and/or derivatives thereof, in addition to fusions with heterologous polypeptides and other forms of the polypeptides described herein. \\n     In some embodiments, the immunizing agent and/or adjuvant is injected in the mammal by multiple subcutaneous or intraperitoneal injections, though they may also be given intramuscularly, and/or through IV injection. The immunizing agent can include polypeptides of the present invention or a fusion protein or variants thereof. Depending upon the nature of the polypeptides (i.e., percent hydrophobicity, percent hydrophilicity, stability, net charge, isoelectric point, etc.), it can be useful to conjugate the immunizing agent to a protein known to be immunogenic in the mammal being immunized. Such conjugation includes either chemical conjugation by derivitizing active chemical functional groups to both a polypeptide of the present invention and the immunogenic protein such that a covalent bond is formed, or through fusion-protein based methodology, or other methods known to those of ordinary skill in the art. Examples of such immunogenic proteins include, but are not limited to keyhole limpet hemocyanin, serum albumin, bovine thyroglobulin, and soybean trypsin inhibitor. Various adjuvants may be used to increase the immunological response, depending on the host species, including but not limited to Freund&#39;s (complete and incomplete), mineral gels such as aluminum hydroxide, surface active substances such as lysolecithin, pluronic polyols, polyanions, peptides, oil emulsions, keyhole limpet hemocyanin, dinitrophenol, and potentially useful human adjuvants such as BCG (bacille Calmette-Guerin) and  Corynebacterium parvum . Additional examples of adjuvants that can be employed include the MPL-TDM adjuvant (monophosphoryl lipid A, synthetic trehalose dicorynomycolate). A suitable immunization protocol can be selected by one of ordinary skill in the art upon consideration of the present disclosure. \\n     Monoclonal Antibodies \\n     The antibodies of the present invention can comprise monoclonal antibodies. Monoclonal antibodies can be prepared using known hybridoma methods, (see, e.g., Köhler &amp; Milstein, (1975)  Nature  256:495; U.S. Pat. No. 4,376,110, Harlow et al.,  Antibodies: A Laboratory Manual,  2 nd  ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (1988); Hammerling et al.,  Monoclonal Antibodies and T - Cell Hybridomas , Elsevier, New York, N.Y., USA, (1981) pp. 563–681; Köhler et al., (1976)  Eur. J. Immunol.  6:511; Köhler et al., (1976)  Eur. J. Immunol.  6:292). Other examples of methods that can be employed for producing monoclonal antibodies include, but are not limited to, the human B-cell hybridoma technique (Kosbor et al., (1983)  Immunology Today  4:72; Cole et al., (1983)  Proc. Natl. Acad. Sci. USA  80:2026–2030), and the EBV-hybridoma technique (Cole et al.,  Monoclonal Antibodies And Cancer Therapy , Alan R. Liss, Inc., New York, N.Y., USA (1985) pp. 77–96). Such antibodies can be of any immunoglobulin class including IgG, IgM, IgE, IgA, IgD and any subclass thereof. The hybridoma producing the mAb of this invention can be cultivated in vitro or in vivo. Production of high titers of mAbs in vivo makes this presently a preferred method of production in some situations. \\n     In a hybridoma method, a mouse, a humanized mouse, a mouse with a human immune system, hamster, or other appropriate host animal, is typically immunized with an immunizing agent to elicit lymphocytes that produce or are capable of producing antibodies that will specifically bind to the immunizing agent. Alternatively, the lymphocytes may be immunized in vitro. \\n     The immunizing agent typically, but not necessarily, includes polypeptides of the present invention or a fusion protein thereof. In some examples, the immunizing agent consists of an HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475), and/or MGBPBMY4 (BC007143) polypeptide-expressing cell. Such cells can be cultured in any suitable tissue culture medium; however, it is sometimes desirable to culture cells in Earle&#39;s modified Eagle&#39;s medium supplemented with 10% fetal bovine serum (inactivated at about 56° C.), and supplemented with about 10 g/l of nonessential amino acids, about 1,000 U/ml of penicillin, and about 100 μg/ml of streptomycin. Generally, either peripheral blood lymphocytes (“PBLs”) are used if cells of human origin are desired, or spleen cells or lymph node cells are used if non-human mammalian sources are desired. The lymphocytes are then fused with an immortalized cell line-using a suitable fusing agent, such as polyethylene glycol, to form a hybridoma cell (Goding,  Monoclonal Antibodies: Principles and Practice , Academic Press, New York, N.Y., USA (1986), pp. 59–103). Immortalized cell lines are often transformed mammalian cells, particularly myeloma cells of rodent, bovine and human origin. Often, rat or mouse myeloma cell lines are employed. The hybridoma cells can be cultured in a suitable culture medium that can optionally comprise one or more substances that inhibit the growth or survival of the unfused, immortalized cells. For example, if the parental cells lack the enzyme hypoxanthine guanine phosphoribosyl transferase (HGPRT or HPRT), the culture medium for the hybridomas typically will include hypoxanthine, aminopterin, and thymidine (“HAT medium”), which substances prevent the growth of HGPRT-deficient cells. \\n     In some embodiments, preferred immortalized cell lines are those that fuse efficiently, support stable high level expression of antibody by the selected antibody-producing cells, and are sensitive to a medium such as HAT medium. Other useful immortalized cell lines are murine myeloma lines, which can be obtained, for instance, from the Salk Institute Cell Distribution Center, San Diego, Calif., USA and the American Type Culture Collection, Manassas, Va., USA. Yet other useful immortalized cell lines are the parent myeloma cell line (SP2O) as provided by the ATCC. As noted and implied throughout the specification, human myeloma and mouse-human heteromyeloma cell lines also have been described for the production of human monoclonal antibodies (see, e.g., Kozbor, (1984)  J. Immunol.  133:3001; Brodeur et al.,  Monoclonal Antibody Production Techniques and Applications , Marcel Dekker, Inc., New York, N.Y., USA (1987) pp. 51–63). \\n     The culture medium in which the hybridoma cells are cultured can then be assayed for the presence of monoclonal antibodies directed against the polypeptides of the present invention. The binding specificity of monoclonal antibodies produced by the hybridoma cells can be determined, for example by immunoprecipitation or by an in vitro binding assay, such as radioimmunoassay (RIA) or enzyme-linked immunoabsorbant assay (ELISA). Such techniques are known in the art. The binding affinity of the monoclonal antibody can, for example, be determined by a Scatchard analysis (see Munson &amp; Pollart, (1980)  Anal. Biochem.  107:220. \\n     After the desired hybridoma cells are identified, the clones can be subcloned by limiting dilution procedures and grown by standard methods (see, e.g., Goding,  Monoclonal Antibodies: Principles and Practice , Academic Press, New York, N.Y., USA (1986) and/or Wands et al., (1981)  Gastroenterology  80:225–232). Suitable culture media for this purpose include, for example, Dulbecco&#39;s Modified Eagle&#39;s Medium and RPMI-1640. Alternatively, the hybridoma cells may be grown in vivo as ascites in a mammal. \\n     The monoclonal antibodies secreted by the subclones can be isolated or purified from the culture medium or ascites fluid by conventional immunoglobulin purification procedures such as, for example, protein A-sepharose, hydroxyapatite chromatography, gel exclusion chromatography, gel electrophoresis, dialysis, or affinity chromatography, all of which techniques will be known to those of ordinary skill in the art. \\n     A variety of methods exist in the art for the production of monoclonal antibodies and thus, the present invention is not limited to their sole production in hydridomas. For example, the monoclonal antibodies can be made by recombinant DNA methods, such as those described in U.S. Pat. No. 4,816,567. In this context, the term “monoclonal antibody” refers to an antibody derived from a single eukaryotic, phage, or prokaryotic clone. The DNA encoding the monoclonal antibodies of the present invention can be readily isolated and sequenced using conventional procedures (e.g., by using oligonucleotide probes that are capable of binding specifically to genes encoding the heavy and light chains of murine antibodies, or such chains from human, humanized, or other sources). The hydridoma cells of the present invention serve as a preferred source of such DNA. Once isolated, the DNA can be placed into expression vectors, which are then transformed into host cells such as Simian COS cells, Chinese hamster ovary (CHO) cells, or myeloma cells that do not otherwise produce immunoglobulin protein, to obtain the synthesis of monoclonal antibodies in the recombinant host cells. The DNA also can be modified, for example, by substituting the coding sequence for human heavy and light chain constant domains in place of the homologous murine sequences (U.S. Pat. No. 4,816,567; Morrison, (1985)  Science  229:1202) or by covalently joining to the immunoglobulin coding sequence all or part of the coding sequence for a non-immunoglobulin polypeptide. Such a non-immunoglobulin polypeptide can be substituted for the constant domains of an antibody of the present invention, or can be substituted for the variable domains of one antigen-combining site of an antibody of the present invention to create a chimeric bivalent antibody. \\n     Monoclonal antibodies can be prepared using a wide variety of techniques known in the art including the use of hybridoma, recombinant, and phage display technologies, or a combination thereof. For example, monoclonal antibodies can be produced using hybridoma techniques including those known in the art and taught, for example, in Harlow et al.,  Antibodies: A Laboratory Manual,  2 nd  ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (1988); Hammerling et al., in:  Monoclonal Antibodies and T - Cell Hybridomas , Elsevier, New York, N.Y., USA, (1981) pp. 563–681. The term “monoclonal antibody” as used herein is not limited to antibodies produced through hybridoma technology. Rather, the term “monoclonal antibody” broadly refers to an antibody that is derived from a single clone, including any eukaryotic, prokaryotic, or phage clone, and not the method by which it is produced. \\n     Methods for producing and screening for specific antibodies using hybridoma technology are routine and well known in the art and are discussed herein. In a non-limiting example, mice can be immunized with a polypeptide of the present invention or a cell expressing such peptide. Once an immune response is detected, e.g., antibodies specific for the antigen are detected in the mouse serum, the mouse spleen is harvested and splenocytes isolated. The splenocytes are then fused by known techniques to any suitable myeloma cells, for example cells from cell line SP20 available from the ATCC. Hybridomas are selected and cloned by limited dilution. The hybridoma clones are then assayed by methods known in the art for cells that secrete antibodies capable of binding a polypeptide of the present invention. Ascites fluid, which generally contains high levels of antibodies, can be generated by immunizing mice with positive hybridoma clones. \\n     Accordingly, in one aspect the present invention provides methods of generating monoclonal antibodies, as well as antibodies produced by the method, comprising: (i) culturing a hybridoma cell secreting an antibody of the present invention, optionally wherein the hybridoma is generated by fusing splenocytes isolated from a mouse immunized with an antigen of the present invention with myeloma cells and (ii) screening the hybridomas resulting from the fusion for hybridoma clones that secrete an antibody able to bind a polypeptide of the present invention. \\n     Antibody fragments that recognize specific epitopes can be generated by known techniques. For example, Fab and F(ab′) 2  fragments of the present invention can be produced by proteolytic cleavage of immunoglobulin molecules, using enzymes such as papain (to produce Fab fragments) or pepsin (to produce F(ab′) 2  fragments). F(ab′) 2  fragments contain a variable region, a light chain constant region and a CH1 domain of the heavy chain. \\n     The antibodies of the present invention can also be generated using various phage display methods known in the art. In phage display methods, functional antibody domains are displayed on the surface of phage particles that carry the polynucleotide sequences encoding them. In a particular embodiment, such phage particles can be utilized to display antigen binding domains expressed from a repertoire or combinatorial antibody library (e.g., human or murine). Phage expressing an antigen binding domain that binds the antigen of interest can be selected or identified with antigen, e.g., using labeled antigen or antigen bound or captured to a solid surface or bead. Phage used in these methods are typically filamentous phage including fd and M13 binding domains expressed from phage with Fab, Fv or disulfide stabilized Fv antibody domains recombinantly fused to either the phage gene III or gene VIII protein. Examples of phage display methods that can be used to make the antibodies of the present invention include those disclosed in Brinkman et al., (1995)  J. Immunol. Methods  182:41–50; Ames et al., (1995)  J. Immunol. Methods  184:177–186; Kettleborough et al., (1994)  Eur. J. Immunol.  24:952–958; Persic et al., (1997)  Gene  187 9–18; Burton et al., (1994)  Adv. Immunology  57:191–280; PCT Publications PCT/GB91/01134; WO 90/02809; WO 91/10737; WO 92/01047; WO 92/18619; WO 93/11236; WO 95/15982; WO 95/20401; and in U.S. Pat. Nos. 5,698,426; 5,223,409; 5,403,484; 5,580,717; 5,427,908; 5,750,753; 5,821,047; 5,571,698; 5,427,908; 5,516,637; 5,780,225; 5,658,727; 5,733,743 and 5,969,108; each of which is incorporated herein by reference in its entirety. \\n     As described in the above references, after phage selection, the antibody coding regions from the phage can be isolated and used to generate whole antibodies, including human antibodies, or any other desired antigen binding fragment, and expressed in any desired host, including mammalian cells, insect cells, plant cells, yeast, and bacteria, e.g., as described herein. For example, techniques to recombinantly produce Fab, Fab′ and F(ab′) 2  fragments can also be employed using methods known in the art such as those disclosed in PCT Publication WO 92/22324; Mullinax et al., (1992)  BioTechniques  12(6):864–869; and Sawai et al., (1995)  AJRI  34:26–34; and Better et al., (1988)  Science  240:1041–1043. Examples of techniques that can be used to produce single-chain Fvs and antibodies include those described in U.S. Pat. Nos. 4,946,778 and 5,258,498; Huston et al., (1991)  Method Enzymol.  203:46–88; Shu et al., (1993)  Proc. Natl. Acad. Sci. USA  90:7995–7999; and Skerra et al., (1988)  Science  240:1038–1040. \\n     For some uses, including in vivo use of antibodies in humans and in vitro detection assays, it can be desirable to use chimeric, humanized, or human antibodies. \\n     A chimeric antibody is a molecule in which different portions of the antibody are derived from different animal species, such as antibodies having a variable region derived from a murine monoclonal antibody and a human immunoglobulin constant region. Methods for producing chimeric antibodies are known in the art (see, e.g., Morrison, (1985)  Science  229:1202; Oi et al., (1986)  BioTechniques  4:214; Gillies et al., (1989)  J. Immunol. Methods  125:191–202; EP 171496; EP 173494; PCT Publications WO 8601533; WO 8702671; Boulianne et al., (1984)  Nature  312:643; Neuberger et al., (1985)  Nature  314:268; U.S. Pat. Nos. 5,807,715; 4,816,567; and 4,816,397). \\n     Humanized Antibodies \\n     Humanized antibodies are antibody molecules from non-human species antibody that binds the desired antigen having one or more complementarity determining regions (CDRs) from the non-human species and a framework region from a human immunoglobulin molecule. Often, framework residues in the human framework regions can be substituted with the corresponding residue from the CDR donor antibody to alter, preferably improve, antigen binding. These framework substitutions are identified by methods well known in the art, e.g., by modeling of the interactions of the CDR and framework residues to identify framework residues important for antigen binding and sequence comparison to identify unusual framework residues at particular positions (see, e.g., U.S. Pat. No. 5,585,089; Riechmann et al., (1988)  Nature  332:323). Antibodies can be humanized using a variety of techniques known in the art including, for example, CDR-grafting (EP 239,400; PCT Publication WO 91/09967; U.S. Pat. Nos. 5,225,539; 5,530,101; and 5,585,089), veneering or resurfacing (EP 592,106; EP 519,596; Padlan, (1991)  Molecular Immunology  28(4/5):489–498; Studnicka et al., (1994)  Protein Engineering  7(6):805–814; Roguska et al., (1994)  Proc. Natl. Acad. Sci. U.S.A.  91:969–973), and chain shuffling (U.S. Pat. No. 5,565,332). \\n     Generally, a humanized antibody has one or more amino acid residues introduced into it from a source that is non-human. These non-human amino acid residues are often referred to as “import” residues, which are typically taken from an “import” variable domain. Humanization can be essentially performed following known methods (Jones et al., (1986)  Nature  321:522–525 (1986); Reichmann et al., (1988)  Nature  332:323–327; Verhoeyen et al., (1988)  Science  239:1534–1536) by substituting rodent CDRs or CDR sequences for the corresponding sequences of a human antibody. Accordingly, such “humanized” antibodies are chimeric antibodies (U.S. Pat. No. 4,816,567), wherein substantially less than an intact human variable domain has been substituted by the corresponding sequence from a non-human species. In practice, humanized antibodies are typically human antibodies in which some CDR residues and possible some FR residues are substituted from analogous sites in rodent antibodies. \\n     In general, a humanized antibody comprises substantially all of at least one, and typically two, variable domains, in which all or substantially all of the CDR regions correspond to those of a non-human immunoglobulin and all or substantially all of the FR regions are those of a human immunoglobulin consensus sequence. The humanized antibody can also comprise at least a portion of an immunoglobulin constant region (Fc), typically that of a human immunoglobulin (see Jones et al., (1986)  Nature  321:522–525; Riechmann et al., (1988)  Nature  332:323–329 and Presta, (1992)  Curr. Opin. Struct. Biol.  2:593–596). \\n     Completely human antibodies are particularly desirable for therapeutic treatment of human subjects. Human antibodies can be made by a variety of methods known in the art including phage display methods described herein using antibody libraries derived from human immunoglobulin sequences. See also, U.S. Pat. Nos. 4,444,887 and 4,716,111; and PCT Publications WO 98/46645, WO 98/50433, WO 98/24893, WO 98/16654, WO 96/34096, WO 96/33735, and WO 91/10741. Other techniques are also available for the preparation of human monoclonal antibodies (Cole et al.,  Monoclonal Antibodies and Cancer Therapy , Alan R. Liss, New York, N.Y., USA (1985); and Boemer et al., (1991)  J. Immunol.  147(1):86–95). \\n     Human antibodies can also be produced using transgenic mice that are incapable of expressing functional endogenous immunoglobulins, but can express human immunoglobulin genes. For example, the human heavy and light chain immunoglobulin gene complexes can be introduced randomly or by homologous recombination into mouse embryonic stem cells. Alternatively, the human variable region, constant region, and diversity region can be introduced into mouse embryonic stem cells in addition to the human heavy and light chain genes. The mouse heavy and light chain immunoglobulin genes can be rendered non-functional separately or simultaneously with the introduction of human immunoglobulin loci by homologous recombination. In particular, homozygous deletion of the JH region prevents endogenous antibody production. The modified embryonic stem cells are expanded and microinjected into blastocysts to produce chimeric mice. The chimeric mice are then bred to produce homozygous offspring, which express human antibodies. The transgenic mice are immunized in the normal fashion with a selected antigen, e.g., all or a portion of a polypeptide of the present invention. Monoclonal antibodies directed against the antigen can be obtained from the immunized, transgenic mice using conventional hybridoma technology. The human immunoglobulin transgenes harbored by the transgenic mice rearrange during B cell differentiation, and subsequently undergo class switching and somatic mutation. Thus, using such a technique, it is possible to produce therapeutically useful IgG, IgA, IgM and IgE antibodies. For an overview of this technology for producing human antibodies, see Lonberg &amp; Huszar, (1995)  Int. Rev. Immunol.  13:65–93. See also PCT Publications WO 98/24893; WO 92/01047; WO 96/34096; WO 96/33735; European Patent No. 0 598 877; U.S. Pat. Nos. 5,413,923; 5,625,126; 5,633,425; 5,569,825; 5,661,016; 5,545,806; 5,814,318; 5,885,793; 5,916,771; and 5,939,598. In addition, companies such as Abgenix, Inc. (Fremont, Calif., USA), Genpharm (San Jose, Calif., USA), and Medarex, Inc. (Princeton, N.J., USA) can be engaged to provide human antibodies directed against a selected antigen using technology similar to that described herein. \\n     Similarly, human antibodies can be made by introducing human immunoglobulin loci into transgenic animals, e.g., mice in which the endogenous immunoglobulin genes have been partially or completely inactivated. Upon challenge, human antibody production is observed, which closely resembles that seen in humans in all respects, including gene rearrangement, assembly, and creation of an antibody repertoire. This approach is described, for example, in U.S. Pat. Nos. 5,545,807; 5,545,806; 5,569,825; 5,625,126; 5,633,425; 5,661,106, and in the following scientific publications: Marks et al., (1992)  Biotechnol.  10:779–783; Lonberg et al., (1994)  Nature  368:856–859; Fishwild et al., (1996)  Nature Biotechnol.  14:845–51; Neuberger, (1996)  Nature Biotechnol.  14:826; Lonberg &amp; Huszer, (1995)  Intern. Rev. Immunol.  13:65–93. \\n     Completely human antibodies which recognize a selected epitope can be generated using a technique referred to as “guided selection.” In this approach a selected non-human monoclonal antibody, e.g., a mouse antibody, is used to guide the selection of a completely human antibody recognizing the same epitope. (Jespers et al., (1988)  Bio/technology  12:899–903). \\n     Anti-Idiotype Antibodies \\n     Further, antibodies to the polypeptides of the present invention can, in turn, be utilized to generate anti-idiotype antibodies that “mimic” polypeptides of the present invention using techniques known to those skilled in the art. (See, e.g., Greenspan &amp; Bona, (1989)  FASEB J.  7(5):437–444; and Nissinoff, (1991)  J. Immunol.  147(8):2429–2438). For example, antibodies that bind to and competitively inhibit polypeptide multimerization, and/or binding of a polypeptide of the present invention to a ligand, can be used to generate anti-idiotypes that “mimic” the polypeptide multimerization and/or binding domain and, as a consequence, bind to and neutralize the polypeptide and/or its ligand. Neutralizing anti-idiotypes or Fab fragments of such anti-idiotypes can be used in therapeutic regimens to neutralize polypeptide ligand. For example, such anti-idiotypic antibodies can be used to bind a polypeptide of the present invention and/or to bind its ligands/receptors, and thereby block the polypeptide/ligand/receptor&#39;s biological activity. \\n     Such anti-idiotypic antibodies capable of binding to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475), and/or MGBPBMY4 (BC007143) polypeptide can be produced in a two-step procedure. Such a method makes use of the fact that antibodies are themselves antigens, and therefore, it is possible to obtain an antibody that binds to a second antibody. In accordance with this method, protein specific antibodies are used to immunize an animal, for example a mouse. The splenocytes of such an animal are then used to produce hybridoma cells, and the hybridoma cells are screened to identify clones that produce an antibody whose ability to bind to the protein-specific antibody can be blocked by the polypeptide. Such antibodies comprise anti-idiotypic antibodies to the protein-specific antibody and can be used to immunize an animal to induce formation of further protein-specific antibodies. \\n     Monospecific/Monovalent Antibodies \\n     The antibodies of the present invention can be monovalent antibodies. Methods for preparing monovalent antibodies are well known in the art. For example, one method involves recombinant expression of immunoglobulin light chain and modified heavy chain. The heavy chain is truncated generally at any point in the Fc region so as to prevent heavy chain crosslinking. Alternatively, the relevant cysteine residues are substituted with another amino acid residue or are deleted so as to prevent crosslinking. \\n     In vitro methods are also suitable for preparing monovalent antibodies. Digestion of antibodies to produce fragments thereof, particularly, Fab fragments, can be accomplished using routine techniques known in the art. \\n     Bispecific/Bivalent Antibodies \\n     The antibodies of the present invention can be bispecific antibodies. Bispecific antibodies are monoclonal antibodies (e.g., human or humanized monoclonal antibodies) that have binding specificities for at least two different antigens. In a bispecific antibody of the present invention, one of the binding specificities can be directed towards a polypeptide of the present invention, the other can be for any other antigen (e.g., a cell-surface protein, receptor, receptor subunit, tissue-specific antigen, virally derived protein, virally encoded envelope protein, bacterially derived protein, or bacterial surface protein, etc.). \\n     Methods for making bispecific antibodies are known in the art. Traditionally, the recombinant production of bispecific antibodies is based on the co-expression of two immunoglobulin heavy-chain/light-chain pairs, where the two heavy chains have different specificities (Milstein &amp; Cuello, (1983)  Nature  305:537–539). Because of the random assortment of immunoglobulin heavy and light chains, these hybridomas (quadromas) produce a potential mixture of ten different antibody molecules, of which only one has the correct bispecific structure. The purification of the correct molecule can be, and typically is, accomplished by affinity chromatography steps. Similar procedures are disclosed in PCT Publication WO 93/08829 and in Traunecker et al., (1991)  EMBO J.  10:3655–3659. \\n     Antibody variable domains with the desired binding specificities (antibody-antigen combining sites) can be fused to immunoglobulin constant domain sequences. The fusion can be with an immunoglobulin heavy-chain constant domain, comprising at least part of the hinge, CH2, and CH3 regions. It is is often desirable to have the first heavy-chain constant region (CH1) containing the site necessary for light-chain binding present in at least one of the fusions. DNAs encoding the immunoglobulin heavy-chain fusions and, if desired, the immunoglobulin light chain, are inserted into separate expression vectors, and are co-transformed into a suitable host organism. For further details of generating bispecific antibodies see, e.g., Suresh et al., (1986)  Meth. Enzymol.  121:210. \\n     Heteroconjugate Antibodies \\n     Heteroconjugate antibodies are also contemplated by the present invention. Heteroconjugate antibodies are composed of two covalently joined antibodies. Such antibodies have, for example, been proposed for targeting immune system cells to unwanted cells (U.S. Pat. No. 4,676,980), and for the treatment of HIV infection (PCT Publications WO 91/00360; WO 92/20373; and EP03089). It is contemplated that the antibodies can be prepared in vitro using known methods in synthetic protein chemistry, including those involving crosslinking agents. For example, immunotoxins may be constructed using a disulfide exchange reaction or by forming a thioester bond. Examples of suitable reagents for this purpose include iminothiolate and methyl-4-mercaptobutyrimidate and those disclosed, for example, in U.S. Pat. No. 4,676,980. \\n     Polynucleotides Encoding Antibodies \\n     The present invention further provides polynucleotides comprising a nucleotide sequence encoding an antibody of the present invention and fragments thereof. The present invention also encompasses polynucleotides that hybridize under stringent or lower stringency hybridization conditions, e.g., as defined herein, to polynucleotides that encode an antibody, preferably, that specifically binds to a polypeptide of the present invention, preferably, an antibody that binds to a polypeptide having the amino acid sequence of SEQ ID NO:Y or a fragment thereof. \\n     The polynucleotides can be obtained, and the nucleotide sequence of the polynucleotides determined, by any method known in the art. For example, if the nucleotide sequence of the antibody is known, a polynucleotide encoding the antibody can be assembled from chemically synthesized oligonucleotides (e.g., as described in Kutmeier et al., (1994)  BioTechniques  17:242), which, briefly, involves the synthesis of overlapping oligonucleotides containing portions of the sequence encoding the antibody, annealing and ligating of those oligonucleotides, and then amplification of the ligated oligonucleotides by PCR. \\n     Alternatively, a polynucleotide encoding an antibody can be generated from a nucleic acid derived from a suitable source. If a clone containing a nucleic acid encoding a particular antibody is not available, but the sequence of the antibody molecule is known, a nucleic acid encoding the immunoglobulin can be chemically synthesized or obtained from a suitable source (e.g., an antibody cDNA library, or a cDNA library generated from a nucleic acid, preferably poly A+ RNA, isolated from, any tissue or cells expressing the antibody, such as hybridoma cells selected to express an antibody of the present invention) by PCR amplification using synthetic primers hybridizable to the 3′ and 5′ ends of the sequence or by cloning using an oligonucleotide probe specific for the particular gene sequence to identify, e.g., a cDNA clone from a cDNA library that encodes the antibody. Amplified nucleic acids generated by PCR can then be cloned into replicable cloning vectors using any method well known in the art. \\n     Once the nucleotide sequence and corresponding amino acid sequence of the antibody is determined, the nucleotide sequence of the antibody can be manipulated using methods well known in the art for the manipulation of nucleotide sequences, e.g., recombinant DNA techniques, site directed mutagenesis, PCR, etc. (see, for example, the techniques described in Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3 rd  ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001), and  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002)), to generate antibodies having a different amino acid sequence, for example to create amino acid substitutions, deletions, and/or insertions. \\n     In a specific embodiment, an amino acid sequence of the heavy and/or light chain variable domains can be inspected to identify the sequences of the complementarity determining regions (CDRs) by methods that are known in the art, e.g., by comparison to known amino acid sequences of other heavy and light chain variable regions to determine a region(s) of sequence hypervariability. Using routine recombinant DNA techniques, one or more of the CDRs can be inserted within framework regions, e.g., into human framework regions to humanize a non-human antibody, as described herein. The framework regions can be naturally occurring or consensus framework regions, (e.g., human framework regions (see, e.g., Chothia et al., (1998)  J. Mol. Biol.  278:457–479 for a listing of representative human framework regions). Preferably, the polynucleotide generated by the combination of the framework regions and CDRs encodes an antibody that specifically binds a polypeptide of the present invention. As discussed herein, one or more amino acid substitutions can be made within the framework regions, and the amino acid substitutions improve binding of the antibody to its antigen. Additionally, such methods can be used to make amino acid substitutions or deletions of one or more variable region cysteine residues participating in an intrachain disulfide bond to generate antibody molecules lacking one or more intrachain disulfide bonds. Other alterations to the polynucleotide are encompassed by the present invention and will be known to those of ordinary skill of the art upon consideration of the present disclosure. \\n     In addition, techniques developed for the production of “chimeric antibodies” (Morrison et al., (1984)  Proc. Natl. Acad. Sci. USA  81:851–855; Neuberger et al., (1984) Nature 312:604–608; Takeda et al., (1985)  Nature  314:452–454) by splicing genes from a mouse antibody molecule of appropriate antigen specificity together with genes from a human antibody molecule of appropriate biological activity can be used. As described herein, a chimeric antibody is a molecule in which different portions are derived from different animal species, such as those having a variable region derived from a murine mAb and a human immunoglobulin constant region, e.g., humanized antibodies. \\n     Alternatively, techniques for the production of single chain antibodies are known (see, e.g., U.S. Pat. No. 4,946,778; Bird, (1988)  Science  242:423–42; Huston et al., (1988)  Proc. Natl. Acad. Sci. USA  85:5879–5883; and Ward et al., (1989)  Nature  334:544–54) can be adapted to produce single chain antibodies. Single chain antibodies are formed by linking the heavy and light chain fragments of the Fv region via an amino acid bridge, resulting in a single chain polypeptide. Techniques for the assembly of functional Fv fragments in  E. coli  can also be used (Skerra et al., (1988)  Science  242:1038–1041). \\n     A clone encoding an antibody of the present invention can be obtained according to the methods described herein. \\n     Methods of Producing Antibodies \\n     The antibodies of the present invention can be produced by any method known in the art for the synthesis of antibodies, in particular, by chemical synthesis or preferably, by recombinant expression techniques. The following is presented in addition to the discussion above regarding antibody production. \\n     Recombinant expression of an antibody of the present invention, or a fragment, derivative or analog thereof, (e.g., a heavy or light chain of an antibody of the present invention or a single chain antibody of the present invention), requires the construction of an expression vector containing a polynucleotide that encodes the antibody. Once a polynucleotide encoding an antibody molecule or a heavy or light chain of an antibody, or portion thereof (preferably containing the heavy or light chain variable domain) of the present invention has been obtained, a vector for the production of the antibody molecule can be produced by recombinant DNA technology using techniques known in the art. Methods for preparing a protein by expressing a polynucleotide containing an antibody-encoding nucleotide sequence are described herein. \\n     Methods known to those of ordinary skill in the art can be used to construct expression vectors containing antibody coding sequences and appropriate transcriptional and translational control signals. These methods include, for example, in vitro recombinant DNA techniques, synthetic techniques, and in vivo genetic recombination. The present invention, thus, provides replicable vectors comprising a nucleotide sequence encoding an antibody molecule of the present invention, or a heavy or light chain thereof, or a heavy or light chain variable domain, operably linked to a promoter. Such vectors can include the nucleotide sequence encoding the constant region of an antibody molecule (see, e.g., PCT Publication WO 86/05807; PCT Publication WO 89/01036; and U.S. Pat. No. 5,122,464), and a variable domain of the antibody can be cloned into such a vector for expression of the entire heavy or light chain. \\n     The expression vector is transferred to a host cell by conventional techniques and the transfected cells are then cultured by conventional techniques to produce an antibody of the present invention. Thus, the present invention includes host cells containing a polynucleotide encoding an antibody of the present invention, or a heavy or light chain thereof, or a single chain antibody of the present invention, operably linked to a heterologous promoter. In representative embodiments for the expression of double-chained antibodies, vectors encoding both the heavy and light chains may be co-expressed in the host cell for expression of the entire immunoglobulin molecule, as detailed herein. \\n     A variety of host-expression vector systems can be employed to express the antibody molecules of the present invention. Such host-expression systems represent vehicles by which a coding sequence of interest can be produced and subsequently purified, but also represent cells that may, when transformed or transfected with the appropriate nucleotide coding sequences, express an antibody molecule of the present invention in situ. These include but are not limited to microorganisms such as bacteria (e.g.,  E. coli, B. subtilis ) transformed with recombinant bacteriophage DNA, plasmid DNA or cosmid DNA expression vectors containing antibody coding sequences; yeast (e.g.,  Saccharomyces, Pichia ) transformed with recombinant yeast expression vectors containing antibody coding sequences; insect cell systems infected with recombinant virus expression vectors (e.g., baculovirus) containing antibody coding sequences; plant cell systems infected with recombinant virus expression vectors (e.g., cauliflower mosaic virus, (CaMV); tobacco mosaic virus, (TMV)) or transformed with recombinant plasmid expression vectors (e.g., Ti plasmid) containing antibody coding sequences; or mammalian cell systems (e.g., COS, CHO, BHK, 293, 3T3 cells) harboring recombinant expression constructs containing promoters derived from the genome of mammalian cells (e.g., metallothionein promoter) or from mammalian viruses (e.g., the adenovirus late promoter; the vaccinia virus 7.5K promoter). Under some conditions it can be desirable that bacterial cells such as  Escherichia coli , or eukaryotic cells are used for the expression of a recombinant antibody molecule, especially for the expression of whole recombinant antibody molecule. For example, mammalian cells such as Chinese hamster ovary cells (CHO), in conjunction with a vector such as the major intermediate early gene promoter element from human cytomegalovirus is an effective expression system for antibodies (Foecking et al., (1986)  Gene  45:101; Cockett et al., (1990)  Bio/Technology  8:2). \\n     In bacterial systems, a number of expression vectors can be advantageously employed, depending upon the use intended for the antibody molecule being expressed. For example, when a large quantity of such a protein is to be produced, for the generation of pharmaceutical compositions of an antibody molecule, vectors which direct the expression of high levels of fusion protein products that are readily purified can be desirable. Such vectors include, but are not limited, to the  E. coli  expression vector pUR278 (Ruther et al., (1983)  EMBO J.  2:1791), in which the antibody coding sequence can be ligated individually into the vector in frame with the lac Z coding region so that a fusion protein is produced; pIN vectors (Inouve &amp; Inouye, (1985)  Nucleic Acids Res.  13:3101–3109; Van Heeke &amp; Schuster, (1989)  J. Biol. Chem.  24:5503–5509); and the like pGEX vectors can also be used to express foreign polypeptides as fusion proteins with glutathione S-transferase (GST). In general, such fusion proteins are soluble and can easily be purified from lysed cells by adsorption and binding to matrix glutathione-agarose beads followed by elution in the presence of free glutathione. The pGEX vectors are designed to include thrombin or factor Xa protease cleavage sites so that the cloned target gene product can be released from the GST moiety. \\n     In an insect system,  Autographa californica  nuclear polyhedrosis virus (AcNPV) is used as a vector to express foreign genes. The virus grows in  Spodoptera frugiperda  cells. The antibody coding sequence may be cloned individually into non-essential regions (for example the polyhedrin gene) of the virus and placed under control of an AcNPV promoter (for example the polyhedrin promoter). \\n     In mammalian host cells, a number of viral-based expression systems can be utilized. In cases where an adenovirus is used as an expression vector, the antibody coding sequence of interest can be ligated to an adenovirus transcription/translation control complex, e.g., the late promoter and tripartite leader sequence. This chimeric gene can then be inserted in the adenovirus genome by in vitro or in vivo recombination. Insertion in a non-essential region of the viral genome (e.g., region E1 or E3) will result in a recombinant virus that is viable and capable of expressing the antibody molecule in infected hosts. (see, e.g., Logan &amp; Shenk, (1984)  Proc. Natl. Acad. Sci. U.S.A.  81:355–359). Specific initiation signals may also be required for efficient translation of inserted antibody coding sequences. These signals include the ATG initiation codon and adjacent sequences. Furthermore, the initiation codon must be in phase with the reading frame of the desired coding sequence to ensure translation of the entire insert. These exogenous translational control signals and initiation codons can be of a variety of origins, both natural and synthetic. The efficiency of expression may be enhanced by the inclusion of appropriate transcription enhancer elements, transcription terminators, etc. (see Bittner et al., (1987)  Method Enzymol.  153:51–544). \\n     In addition, a host cell strain can be chosen that modulates the expression of the inserted sequences, or modifies and processes the gene product in the specific fashion desired. Such modifications (e.g., glycosylation) and processing (e.g., cleavage) of protein products can be important for the function of the protein. Different host cells have characteristic and specific mechanisms for the post-translational processing and modification of proteins and gene products. Appropriate cell lines or host systems can be chosen to ensure the correct modification and processing of the foreign protein expressed. To this end, eukaryotic host cells that possess the cellular machinery for proper processing of the primary transcript, glycosylation, and phosphorylation of the gene product can be used. Such mammalian host cells include but are not limited to CHO, VERY, BHK, Hela, COS, MDCK, 293, 3T3, WI38, and in particular, breast cancer cell lines such as, for example, BT483, Hs578T, HTB2, BT20 and T47D, and normal mammary gland cell line such as, for example, CRL7030 and Hs578Bst. \\n     For long-term, high-yield production of recombinant proteins, stable expression is preferred. For example, cell lines that stably express the antibody molecule can be engineered. Rather than using expression vectors that contain viral origins of replication, host cells can be transformed with DNA controlled by appropriate expression control elements (e.g., promoter, enhancer, sequences, transcription terminators, polyadenylation sites, etc.), and a selectable marker. Following the introduction of the foreign DNA, engineered cells can be allowed to grow for 1–2 days in an enriched media, and then are switched to a selective media. The selectable marker in the recombinant plasmid confers resistance to the selection and allows cells to stably integrate the plasmid into their chromosomes and grow to form foci which in turn can be cloned and expanded into cell lines. This method may advantageously be used to engineer cell lines that express the antibody molecule. Such engineered cell lines may be particularly useful in screening and evaluation of compounds that interact directly or indirectly with the antibody molecule. \\n     A number of selection systems can be used. For example, the herpes simplex virus thymidine kinase (Wigler et al., (1977)  Cell  11:223), hypoxanthine-guanine phosphoribosyltransferase (Szybalska &amp; Szybalski, (1992)  Proc. Natl. Acad. Sci. USA  48:202), and adenine phosphoribosyltransferase (Lowy et al., (1980)  Cell  22:817) genes can be employed in tk-, hgprt- or aprt-cells, respectively. Also, antimetabolite resistance can be used as the basis of selection for the following genes: dhfr, which confers resistance to methotrexate (Wigler et al., (1980)  Proc. Natl. Acad. Sci. USA  77:357; O&#39;Hare et al., (1981)  Proc. Natl. Acad. Sci. USA  78:1527); gpt, which confers resistance to mycophenolic acid (Mulligan &amp; Berg, (1981)  Proc. Natl. Acad. Sci. USA  78:2072); neo, which confers resistance to the aminoglycoside G-418 ( Clinical Pharmacy  12:488–505; Wu &amp; Wu, (1991)  Biotherapy  3:87–95; Tolstoshev, (1993)  Ann. Rev. Pharmacol. Toxicol.  32:573–596; Mulligan, (1993)  Science  260:926–932; and Morgan &amp; Anderson, (1993)  Ann. Rev. Biochem.  62:191–217;  TIB TECH  11(5): 155–215, May, 1993); and hygro, which confers resistance to hygromycin (Santerre et al., (1984)  Gene  30:147). Methods commonly known in the art of recombinant DNA technology can be routinely applied to select the desired recombinant clone, and such methods are described, for example, in  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002); Kriegler,  Gene Transfer and Expression, A Laboratory Manual , Stockton Press, New York, N.Y., USA (1990);  Current Protocols in Human Genetics , (Dracopoli et al., eds.), John Wiley &amp; Sons, New York, N.Y., USA (1994), Chapters 12 and 13; Colberre-Garapin et al., (1981)  J. Mol. Biol.  150:1). \\n     The expression levels of an antibody molecule can be increased by vector amplification (for a review, see Bebbington &amp; Hentschel, in  DNA Cloning , vol.3. Academic Press, New York, N.Y., USA (1987)). When a marker in the vector system expressing antibody is amplifiable, an increase in the level of inhibitor present in culture of host cell will increase the number of copies of the marker gene. Since the amplified region is associated with the antibody gene, production of the antibody will also increase (Crouse et al., (1983)  Mol. Cell. Biol.  3:257). \\n     The host cell can be co-transfected with two expression vectors of the present invention, the first vector encoding a heavy chain derived polypeptide and the second vector encoding a light chain derived polypeptide. The two vectors can contain identical selectable markers that enable equal expression of heavy and light chain polypeptides. Alternatively, a single vector can be used which encodes, and is capable of expressing, both heavy and light chain polypeptides. In such situations, the light chain should be placed before the heavy chain to avoid an excess of toxic free heavy chain (Proudfoot, (1986)  Nature  322:52; Kohler, (1980)  Proc. Natl. Acad. Sci. USA  77:2197). The coding sequences for the heavy and light chains can comprise cDNA or genomic DNA. \\n     Once an antibody molecule of the present invention has been produced by an animal, chemically synthesized, or recombinantly expressed, it can be purified by any method known in the art for purification of an immunoglobulin molecule, for example, by chromatography (e.g., ion exchange, affinity, particularly by affinity for the specific antigen after Protein A, and sizing column chromatography), centrifugation, differential solubility, or by any other standard technique for the purification of proteins. In addition, the antibodies of the present invention or fragments thereof can be fused to heterologous polypeptide sequences described herein or otherwise known in the art, to facilitate purification. \\n     Fusion Proteins Comprising an Antibody \\n     The present invention encompasses antibodies recombinantly fused or chemically conjugated (including both covalently and non-covalently conjugations) to a polypeptide (or portion thereof, preferably at least 10, 20, 30, 40, 50, 60, 70, 80, 90 or 100 amino acids of the polypeptide) of the present invention to generate fusion proteins. The fusion does not necessarily need to be direct, but can occur through linker sequences. The antibodies can be specific for antigens other than polypeptides (or portion thereof, preferably at least 10, 20, 30, 40, 50, 60, 70, 80, 90 or 100 amino acids of the polypeptide) of the present invention. For example, antibodies can be used to target the polypeptides of the present invention to particular cell types, either in vitro or in vivo, by fusing or conjugating the polypeptides of the present invention to antibodies specific for particular cell surface receptors. Antibodies fused or conjugated to the polypeptides of the present invention can also be used in in vitro immunoassays and purification methods using methods known in the art. See e.g., PCT Publication WO 93/21232; EP 439,095; Naramura et al., (1994)  Immunol. Lett.  39:91–99; U.S. Pat. No. 5,474,981; Gillies et al., (1992)  Proc. Natl. Acad. Sci. USA  89:1428–1432; Fell et al., (1991)  J. Immunol.  146:2446–2452. \\n     The present invention further includes compositions comprising the polypeptides of the present invention fused or conjugated to antibody domains other than the variable regions. For example, the polypeptides of the present invention can be fused or conjugated to an antibody Fc region, or portion thereof. The antibody portion fused to a polypeptide of the present invention can comprise the constant region, hinge region, CH1 domain, CH2 domain, and CH3 domain or any combination of whole domains or portions thereof. The polypeptides can also be fused or conjugated to the above antibody portions to form multimers. For example, Fc portions fused to the polypeptides of the present invention can form dimers through disulfide bonding between the Fc portions. Higher multimeric forms can be made by fusing the polypeptides to portions of IgA and IgM. Methods for fusing or conjugating the polypeptides of the present invention to antibody portions are known in the art. See, e.g., U.S. Pat. Nos. 5,336,603; 5,622,929; 5,359,046; 5,349,053; 5,447,851; 5,112,946; EP 307,434; EP 367,166; PCT publications WO 96/04388; WO 91/06570; Ashkenazi et al., (1991)  Proc. Natl. Acad. Sci. USA  88:10535–10539 (1991); Zheng et al., (1995)  J. Immunol.  154:5590–5600; and Vil et al., (1992)  Proc. Natl. Acad. Sci. USA  89:11337–11341. \\n     As discussed herein, a polypeptide comprising a full length polypeptide, polypeptide fragment, or a variant of SEQ ID NO:Y may be fused or conjugated to the above antibody portions to increase the in vivo half life of the polypeptides or for use in immunoassays using methods known in the art. Further, a polypeptide corresponding to SEQ ID NO:Y can be fused or conjugated to the above antibody portions to facilitate purification. One reported example describes chimeric proteins consisting of the first two domains of the human CD4-polypeptide and various domains of the constant regions of the heavy or light chains of mammalian immunoglobulins (see, e.g., EP 394,827; Traunecker et al., (1988)  Nature  331:84–86). The polypeptides of the present invention fused or conjugated to an antibody having disulfide-linked dimeric structures (due to the IgG) can also be more efficient in binding and neutralizing other molecules, than the monomeric secreted protein or protein fragment alone (see, e.g., Fountoulakis et al., (1995)  J. Biochem.  270:3958–3964). In some cases, the Fc part in a fusion protein is beneficial in therapy and diagnosis, and thus can result in, for example, improved pharmacokinetic properties (see, e.g., EP A 232,262). Alternatively, deleting the Fc part after the fusion protein has been expressed, detected, and purified, would be desired. For example, the Fc portion might hinder therapy and diagnosis if the fusion protein is used as an antigen for immunizations. In drug discovery, for example, human proteins, such as hIL-5, have been fused with Fc portions for the purpose of high-throughput screening assays to identify antagonists of hIL-5 (see, e.g., Bennett et al., (1995)  J. Molecular Recognition  8:52–58; Johanson et al., (1995)  J. Biol. Chem.  270:9459–9471. \\n     Moreover, the antibodies or fragments thereof of the present invention can be fused to marker sequences, such as a peptide to facilitate purification. In some embodiments, the marker amino acid sequence is a hexa-histidine peptide, such as the tag provided in a pQE vector (Qiagen, Inc., La Jolla, Calif.), among others, many of which are commercially available. As described in Gentz et al., (1989)  Proc. Natl. Acad. Sci. USA  86:821–824, for instance, hexa-histidine provides for convenient purification of the fusion protein. Other peptide tags useful for purification include, but are not limited to, the “HA” tag, which corresponds to an epitope derived from the influenza hemagglutinin protein (Wilson et al., (1984)  Cell  37:767) and the FLAG® tag (Sigma, St. Louis, Mo., USA; SEQ ID NO:18). \\n     The present invention further encompasses antibodies or fragments thereof conjugated to a diagnostic or therapeutic agent. The antibodies can be used diagnostically to, for example, monitor the development or progression of a tumor as part of a clinical testing procedure to, e.g., determine the efficacy of a given treatment regimen. Detection can be facilitated by coupling the antibody to a detectable substance. Examples of detectable substances include various enzymes, prosthetic groups, fluorescent materials, luminescent materials, bioluminescent materials, radioactive materials, positron emitting metals using various positron emission tomographies, and nonradioactive paramagnetic metal ions. The detectable substance can be coupled or conjugated either directly to the antibody (or fragment thereof) or indirectly, through an intermediate (such as, for example, a linker known in the art) using techniques known in the art. See, for example, U.S. Pat. No. 4,741,900, disclosing metal ions that can be conjugated to antibodies for use as diagnostics according to the present invention. Examples of suitable enzymes include horseradish peroxidase, alkaline phosphatase, beta-galactosidase, or acetylcholinesterase; examples of suitable prosthetic group complexes include streptavidin/biotin and avidin/biotin; examples of suitable fluorescent materials include umbelliferone, fluorescein, fluorescein isothiocyanate, rhodamine, dichlorotriazinylamine fluorescein, dansyl chloride or phycoerythrin; an example of a luminescent material includes luminol; examples of bioluminescent materials include luciferase, luciferin, and aequorin; and examples of suitable radioactive material include  125 I,  131 I,  111 In or  99 Tc. \\n     Further, an antibody or fragment thereof can be conjugated to a therapeutic moiety such as a cytotoxin, e.g., a cytostatic or cytocidal agent, a therapeutic agent or a radioactive metal ion, e.g., alpha-emitters such as, for example,  213 Bi. A cytotoxin or cytotoxic agent includes any agent that is detrimental to cells. Examples include paclitaxol, cytochalasin B, gramicidin D, ethidium bromide, emetine, mitomycin, etoposide, tenoposide, vincristine, vinblastine, colchicin, doxorubicin, daunorubicin, dihydroxy anthracin dione, mitoxantrone, mithramycin, actinomycin D, 1-dehydrotestosterone, glucocorticoids, procaine, tetracaine, lidocaine, propranolol, and puromycin and analogs or homologues thereof. Representative therapeutic agents include, but are not limited to, antimetabolites (e.g., methotrexate, 6-mercaptopurine, 6-thioguanine, cytarabine, 5-fluorouracil decarbazine), alkylating agents (e.g., mechlorethamine, thioepa chlorambucil, melphalan, carmustine (BSNU) and lomustine (CCNU), cyclothosphamide, busulfan, dibromomannitol, streptozotocin, mitomycin C, and cis-dichlorodiamine platinum (II) (DDP) cisplatin), anthracyclines (e.g., daunorubicin (formerly daunomycin) and doxorubicin), antibiotics (e.g., dactinomycin (formerly actinomycin), bleomycin, mithramycin, and anthramycin (AMC)), and anti-mitotic agents (e.g., vincristine and vinblastine). \\n     The conjugates of the present invention can be used to modify a given biological response, and the therapeutic agent or drug moiety is not to be construed as limited to classical chemical therapeutic agents. For example, a drug moiety can be a protein or polypeptide possessing a desired biological activity. Such proteins can include, for example, a toxin such as abrin, ricin A, pseudomonas exotoxin, or diphtheria toxin; a protein such as tumor necrosis factor, α-interferon, β-interferon, nerve growth factor, platelet derived growth factor, tissue plasminogen activator, an apoptotic agent, e.g., TNF-alpha, TNF-beta, AIM I (see, PCT Publication WO 97/33899), AIM II (see, PCT Publication WO 97/34911), Fas Ligand (Takahashi et al., (1994)  Int. Immunol.  6:1567–1574), VEGI (see, PCT Publication WO 99/23105), a thrombotic agent or an anti-angiogenic agent, e.g., angiostatin or endostatin; or, biological response modifiers such as, for example, lymphokines, interleukin-1 (“IL-1”), interleukin-2 (“IL-2”), interleukin-6 (“IL-6”), granulocyte macrophage colony stimulating factor (“GM-CSF”), granulocyte colony stimulating factor (“G-CSF”), or other growth factors. \\n     Antibodies can also be attached to solid supports, which are particularly useful for immunoassays or purification of the target antigen. Such solid supports include, but are not limited to, glass, cellulose, polyacrylamide, nylon, polystyrene, polyvinyl chloride or polypropylene. \\n     Techniques for conjugating such therapeutic moiety to antibodies are known, (see, e.g., Arnon et al., in  Monoclonal Antibodies And Cancer Therapy , (Reisfeld et al., eds.), Alan R. Liss, Inc., New York, N.Y., USA (1985) pp. 243–56; Hellstrom et al., in  Controlled Drug Delivery , (2 nd  ed.), (Robinson et al., eds.), Marcel Dekker, Inc., New York, N.Y., USA (1987) pp. 623–53; Thorpe, in  Monoclonal Antibodies &#39; 84 : Biological And Clinical Applications , (Pinchera et al., eds.), (1985) pp. 475–506 ; Monoclonal Antibodies For Cancer Detection And Therapy , (Baldwin et al., eds), Academic Press, New York, N.Y., USA (1985) pp. 303–16, and Thorpe et al., (1982)  Immunol. Rev.  62:119–58). \\n     Alternatively, an antibody can be conjugated to a second antibody to form an antibody heteroconjugate (see U.S. Pat. No. 4,676,980). \\n     An antibody, with or without a therapeutic moiety conjugated to it, administered alone or in combination with cytotoxic factor(s) and/or cytokine(s) can be used as a therapeutic. \\n     Synthetic Antibodies \\n     The present invention also encompasses the creation of synthetic antibodies directed against the polypeptides of the present invention (see, e.g., Radrizzani et al., (1999)  Medicina , (Aires) 59(6):753–8). Recently, a new class of synthetic antibodies has been described and are referred to as molecularly imprinted polymers (MIPs) (commercially available from Semorex, Inc., Jerusalem, Israel). Antibodies, peptides, and enzymes are often used as molecular recognition elements in chemical and biological sensors. However, their lack of stability and signal transduction mechanisms limits their use as sensing devices. Molecularly imprinted polymers (MIPs) are capable of mimicking the function of biological receptors but with less stability constraints. Such polymers provide high sensitivity and selectivity while maintaining excellent thermal and mechanical stability. MIPs have the ability to bind to small molecules and to target molecules such as organics and proteins with equal or greater potency than that of natural antibodies. These “super” MIPs have higher affinities for their target and thus require lower concentrations for efficacious binding. \\n     During synthesis, the MIPs are imprinted so as to have complementary size, shape, charge and functional groups of the selected target by using the target molecule itself (such as a polypeptide, antibody, etc.), or a substance having a very similar structure, as its “print” or “template.” MIPs can be derivatized with the same reagents afforded to antibodies. For example, fluorescent “super” MIPs can be coated onto beads or wells for use in highly sensitive separations or assays, or for use in high throughput screening of proteins. \\n     Moreover, MIPs based upon the structure of a polypeptide of the present invention can be useful in screening for compounds that bind to the polypeptide(s) of the present invention. Such a MIP could serve the role of a synthetic “receptor” by minimicking the native architecture of the polypeptide. In fact, the ability of a MIP to serve the role of a synthetic receptor has already been demonstrated for the estrogen receptor (Ye et al., (2001)  Analyst  126(6):760–5; Dickert et al., (2001)  Analyst  126(6):766–71). A synthetic receptor can either be mimicked in its entirety (e.g., as the entire protein), or mimicked as a series of short peptides corresponding to the protein (Rachkov &amp; Minoura, (2001)  Biochim. Biophys. Acta.  1544(1–2):255–66). Such synthetic receptor MIPs can be employed in any one or more of the screening methods described elsewhere herein. \\n     MIPs have also been shown to be useful in “sensing” the presence of its mimicked molecule (Cheng et al., (2001)  Biosens. Bioelectron.  16(3):179–85; Jenkins et al., (2001)  Analyst  126(6):798–802; Jenkins et al., (2001)  Analyst  126(6):798–802). For example, a MIP designed using a polypeptide of the present invention may be used in assays designed to identify, and potentially quantitate, the level of said polypeptide in a sample. Such a MIP may be used as a substitute for any component described in the assays, or kits, provided herein (e.g., ELISA, etc.). \\n     A number of methods may be employed to create MIPs to a specific receptor, ligand, polypeptide, peptide, organic molecule. Several representative methods are described by Esteban et al. in Esteban et al., (2001)  J. Anal. Chem.  370(7):795–802, which is hereby incorporated herein by reference in its entirety in addition to any references cited therein. Additional methods are known in the art and are encompassed by the present invention, (see, e.g., Hart &amp; Shea, (2001)  J. Am. Chem. Soc.  123(9):2072–3; and Quaglia et al., (2001)  J. Am. Chem. Soc.  123(10):2146–54). \\n     Uses for Antibodies Directed Against Polypeptides of the Present Invention \\n     The antibodies of the present invention have various utilities. For example, such antibodies can be used in diagnostic assays to detect the presence or quantification of the polypeptides of the present invention in a sample. Such a diagnostic assay can comprise at least two steps. The first step can comprise contacting a sample with the antibody, wherein the sample is a tissue (e.g., human, animal, etc.), biological fluid (e.g., blood, urine, sputum, semen, amniotic fluid, saliva, etc.), biological extract (e.g., tissue or cellular homogenate, etc.), a protein microchip (see, e.g., Arenkov et al., (2000)  Anal. Biochem.  278(2):123–131), or a chromatography column, etc. A second step can comprise quantifying an amount of antibody bound to the substrate. Alternatively, the method can optionally involve a step of attaching the antibody, either covalently, electrostatically, or reversibly, to a solid support, and a second step of subjecting the bound antibody to the sample, as defined above and elsewhere herein. \\n     Diagnostic Assays Employing an Antibody of the Present Invention \\n     Various diagnostic assay techniques are known in the art, such as competitive binding assays, direct or indirect sandwich assays and immunoprecipitation assays conducted in either heterogeneous or homogenous phases (Zola,  Monoclonal Antibodies: A Manual of Techniques , CRC Press, Inc., Boca Raton, Fla., USA (1987), pp. 147–158). The antibodies used in the diagnostic assays can be labeled with a detectable moiety. The detectable moiety can be capable of producing, either directly or indirectly, a detectable signal. For example, the detectable moiety can be an isotope, such as  2 H,  14 C,  32 p, or  125 I, a florescent or chemiluminescent compound, such as fluorescein isothiocyanate, rhodamine, or luciferin, or an enzyme, such as alkaline phosphatase, beta-galactosidase, green fluorescent protein, or horseradish peroxidase. Any method known in the art for conjugating the antibody to the detectable moiety can be employed, including those methods described in Hunter et al., (1962)  Nature  144:945; Dafvid et al., (1974)  Biochem.  13:1014; Pain et al., (1981)  J. Immunol. Method.  40:219; and Nygren, (1982)  J. Histochem. Cytochem.  30:407. \\n     Antibodies directed against the polypeptides of the present invention are useful for the affinity purification of such polypeptides from recombinant cell culture or natural sources. In this process, the antibodies against a particular polypeptide are immobilized on a suitable support, such as a SEPHADEX™ (Amersham, Piscatraway, N.J., USA) resin or filter paper, using methods known in the art. The immobilized antibody then is contacted with a sample containing the polypeptides to be purified, and thereafter the support is washed with a suitable solvent that will remove substantially all the material in the sample except for the desired polypeptides, which are bound to the immobilized antibody. Finally, the support is washed with another suitable solvent that will release the desired polypeptide from the antibody. \\n     Immunophenotyping Using an Antibody of the Present Invention \\n     The antibodies of the present invention can be utilized for immunophenotyping of cell lines and biological samples. The translation product of the gene of the present invention can be useful as a cell specific marker, or more specifically as a cellular marker that is differentially expressed at various stages of differentiation and/or maturation of particular cell types. Monoclonal antibodies directed against a specific epitope, or combination of epitopes, allow for the screening of cellular populations expressing the marker. Various techniques can be utilized using monoclonal antibodies to screen for cellular populations expressing the marker(s), and include magnetic separation using antibody-coated magnetic beads, “panning” with antibody attached to a solid matrix (i.e., plate), and flow cytometry (see, e.g., U.S. Pat. No. 5,985,660; and Morrison et al., (1999)  Cell  96:737–49). \\n     These techniques allow for the screening of particular populations of cells, such as might be found with hematological malignancies (i.e. minimal residual disease (MRD) in acute leukemic subjects) and “non-self” cells in transplantations to prevent Graft-versus-Host Disease (GVHD). Alternatively, these techniques allow for the screening of hematopoietic stem and progenitor cells capable of undergoing proliferation and/or differentiation, as might be found in human umbilical cord blood. \\n     Assays for Antibody Binding \\n     The antibodies of the present invention can be assayed for immunospecific binding by any method known in the art. The immunoassays which can be used include but are not limited to competitive and non-competitive assay systems using techniques such as western blots, radioimmunoassays, ELISA (enzyme linked immunosorbent assay), “sandwich” immunoassays, immunoprecipitation assays, precipitin reactions, gel diffusion precipitin reactions, immunodiffusion assays, agglutination assays, complement-fixation assays, immunoradiometric assays, fluorescent immunoassays, protein A immunoassays, to name but a few. Such assays are routine and well known in the art (see, e.g,  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002)). Exemplary immunoassays are described briefly below (but are not intended to be in any way limiting). \\n     Immunoprecipitation protocols generally comprise lysing a population of cells in a lysis buffer such as RIPA buffer (1% NP-40 or Triton X-100, 1% sodium deoxycholate, 0.1% SDS, 0.15 M NaCl, 0.01 M sodium phosphate at pH 7.2, 1% Trasylol) supplemented with protein phosphatase and/or protease inhibitors (e.g., EDTA, PMSF, aprotinin, sodium vanadate), adding the antibody of interest to the cell lysate, incubating for a period of time (e.g., 1–4 hours) at 4° C., adding protein A and/or protein G sepharose beads to the cell lysate, incubating for about an hour or more at 4° C., washing the beads in lysis buffer and resuspending the beads in SDS/sample buffer. The ability of the antibody of interest to immunoprecipitate a particular antigen can be assessed by, e.g., western blot analysis. Those of ordinary skill in the art will be knowledgeable as to the parameters that can be modified to increase the binding of the antibody to an antigen and decrease the background (e.g., pre-clearing the cell lysate with sepharose beads), upon consideration of the present disclosure. Additional immunoprecipitation protocols are presented  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002). \\n     Western blot analysis generally comprises preparing protein samples, electrophoresis of the protein samples in a polyacrylamide gel (e.g., about 8–20% SDS-PAGE depending on the molecular weight of the antigen), transferring the protein sample from the polyacrylamide gel to a membrane such as nitrocellulose, PVDF or nylon, blocking the membrane in blocking solution (e.g., PBS with about 3% BSA or non-fat milk), washing the membrane in washing buffer (e.g., PBS-TWEEN 20), blocking the membrane with primary antibody (the antibody of interest) diluted in blocking buffer, washing the membrane in washing buffer, blocking the membrane with a secondary antibody (which recognizes the primary antibody, e.g., an anti-human antibody) conjugated to an enzymatic substrate (e.g., horseradish peroxidase or alkaline phosphatase) or radioactive molecule (e.g.,  32 P or  125 I) diluted in blocking buffer, washing the membrane in wash buffer, and detecting the presence of the antigen. Those of ordinary skill in the art will be knowledgeable as to the parameters that can be modified to increase the signal detected and to reduce the background noise, upon consideration of the present disclosure. Additional western blot protocols are presented in  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002). \\n     ELISAs comprise preparing antigen, coating the well of a 96 well microtiter plate with the antigen, adding the antibody of interest conjugated to a detectable compound such as an enzymatic substrate (e.g., horseradish peroxidase or alkaline phosphatase) to the well and incubating for a period of time, and detecting the presence of the antigen. When performing an ELISA, the antibody of interest does not need to be conjugated to a detectable compound; instead, a second antibody (which recognizes the antibody of interest) conjugated to a detectable compound can be added to the well. Further, instead of coating the well with the antigen, the antibody may be coated to the well. In this case, a second antibody conjugated to a detectable compound can be added following the addition of the antigen of interest to the coated well. One of ordinary skill in the art will be knowledgeable as to the parameters that can be modified to increase the signal detected as well as other variations of ELISA protocols known in the art. For further discussion regarding ELISA protocols see, e.g.,  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002). \\n     The binding affinity of an antibody to an antigen and the off-rate of an antibody-antigen interaction can be determined by competitive binding assays. One example of a competitive binding assay is a radioimmunoassay comprising the incubation of labeled antigen (e.g.,  3 H or  125 I) with the antibody of interest in the presence of increasing amounts of unlabeled antigen, and the detection of the antibody bound to the labeled antigen. The affinity of the antibody of interest for a particular antigen and the binding off-rates can be determined from the data by Scatchard plot analysis. Competition with a second antibody can also be determined using radioimmunoassays. In this case, the antigen is incubated with antibody of interest conjugated to a labeled compound (e.g.,  3 H or  125 I) in the presence of increasing amounts of an unlabeled second antibody. \\n     Therapeutic Applications of Antibodies \\n     The present invention is further directed to antibody-based therapies which involve administering antibodies of the present invention to an animal subject, preferably a mammal (and more preferably a human), for treating one or more of the disclosed diseases, disorders, or conditions. Therapeutic compounds of the present invention include, but are not limited to, antibodies of the present invention (including fragments, analogs and derivatives thereof as described herein) and nucleic acids encoding antibodies of the present invention (including fragments, analogs and derivatives thereof and anti-idiotypic antibodies as described herein). The antibodies of the present invention can be used to treat, inhibit or prevent diseases, disorders or conditions associated with aberrant expression and/or activity of a polypeptide of the present invention, including, but not limited to, any one or more of the diseases, disorders, or conditions described herein. The treatment and/or prevention of diseases, disorders, or conditions associated with aberrant expression and/or activity of a polypeptide of the present invention includes, but is not limited to, alleviating symptoms associated with those diseases, disorders or conditions. Antibodies of the present invention can be provided in pharmaceutically acceptable compositions as known in the art or as described herein. \\n     A summary of the ways in which the antibodies of the present invention can be used therapeutically includes binding polynucleotides or polypeptides of the present invention locally or systemically in the body or by direct cytotoxicity of the antibody, e.g. as mediated by complement (CDC) or by effector cells (ADCC). Some of these approaches are described in more detail below. Upon consideration of the present disclosure, one of ordinary skill in the art will know how to use the antibodies of the present invention for diagnostic, monitoring or therapeutic purposes. \\n     The antibodies of this invention can be advantageously utilized in combination with other monoclonal or chimeric antibodies, or with lymphokines or hematopoietic growth factors (such as, e.g., IL-2, IL-3 and IL-7), for example, which serve to increase the number or activity of effector cells that interact with the antibodies. \\n     The antibodies of the present invention can be administered alone or in combination with other types of treatments (e.g., radiation therapy, chemotherapy, hormonal therapy, immunotherapy and anti-tumor agents). Generally, administration of products of a species origin or species reactivity (in the case of antibodies) that is the same species as that of the subject is preferred. Thus, in one embodiment, human antibodies, fragments derivatives, analogs, or nucleic acids, are administered to a human subject for therapy or prophylaxis. \\n     High affinity and/or potent in vivo inhibiting and/or neutralizing antibodies against polypeptides or polynucleotides of the present invention, fragments or regions thereof, can be used for both immunoassays directed to and therapy of disorders related to polynucleotides or polypeptides, including fragments thereof, of the present invention. Such antibodies, fragments, or regions, can have an affinity for polynucleotides or polypeptides of the present invention, including fragments thereof. Representative binding affinities include those with a dissociation constant or Kd less than 5×10 −2  M, 10 −2  M, 5×10 −3  M, 10 −3  M, 5×10 −4  M, 10 −4  M, 5×10 −5  M, 1×10 −5  M, 5×10 −6  M, 1×10 −6  M, 5×10 −7  M, 1×10 −7  M, 5×10 −8  M, 1×10 −8  M, 5×10 −9  M, 1×10 −9  M, 5×10 −10  M, 1×10 −10  M, 5×10 −11  M, 1×10 −11  M, 5×10 −12  M, 10 −12  M, 5×10 −13  M, 1×10 −13  M, 5×10 −14  M, 1×10 −14  M, 5×10 −15  M, or 1×10 −15  M. \\n     Antibodies directed against polypeptides of the present invention are useful for inhibiting allergic reactions in animals. For example, by administering a therapeutically acceptable dose of an antibody, or antibodies, of the present invention, or a cocktail of the present antibodies, or in combination with other antibodies of varying sources, the animal may not elicit an allergic response to antigens. \\n     Likewise, one of ordinary skill in the art could clone the gene encoding an antibody directed against a polypeptide of the present invention, the polypeptide having the potential to elicit an allergic and/or immune response in an organism, and transforming the organism with said antibody gene such that it is expressed (e.g., constitutively, inducibly, etc.) in the organism. Thus, the organism would effectively become resistant to an allergic response resulting from the ingestion or presence of such an immune/allergic reactive polypeptide. \\n     Moreover, such a use of the antibodies of the present invention may have particular utility in preventing and/or ameliorating autoimmune diseases and/or disorders, as such conditions are typically a result of antibodies being directed against endogenous proteins. For example, in the case in which a polypeptide of the present invention is responsible for modulating the immune response to auto-antigens, transforming the organism and/or individual with a construct comprising any of the promoters disclosed herein or otherwise known in the art, in addition, to a polynucleotide encoding the antibody directed against a polypeptide of the present invention could effective inhibit the organisms immune system from eliciting an immune response to the auto-antigen(s). Additional descriptions of therapeutic and/or gene therapy applications of the present invention are provided herein. \\n     Alternatively, antibodies of the present invention could be produced in a plant (e.g., cloning the gene of the antibody directed against a polypeptide of the present invention, and transforming a plant with a suitable vector comprising said gene for constitutive expression of the antibody within the plant), and the plant subsequently ingested by an animal, thereby conferring temporary immunity to the animal for the specific antigen the antibody is directed towards (see, for example, U.S. Pat. Nos. 5,914,123 and 6,034,298). \\n     In another embodiment, antibodies of the present invention, in one example polyclonal antibodies, an in another example monoclonal antibodies, and in yet another example single-chain antibodies, can be used as a means of inhibiting gene expression of a particular gene, or genes, in a human, mammal, and/or other organism. See, for example, PCT Publication WO 00/05391. The application of such methods for the antibodies of the present invention are known in the art, and are additionally described herein. \\n     In yet another embodiment, antibodies of the present invention can be useful for multimerizing a polypeptide (or a combination of different polypeptides) of the present invention. For example, certain proteins can confer enhanced biological activity when present in a multimeric state (i.e., such enhanced activity may be due to the increased effective concentration of such proteins whereby more protein is available in a localized location). \\n     Antibody-Based Gene Therapy \\n     In a specific embodiment, nucleic acids comprising sequences encoding antibodies or functional derivatives thereof, are administered to treat, inhibit or prevent a disease or disorder associated with aberrant expression and/or activity of a polypeptide of the present invention, by way of gene therapy. Gene therapy refers to therapy performed by the administration to a subject of an expressed or expressible nucleic acid. In this embodiment of the present invention, the nucleic acids produce their encoded protein that mediates a therapeutic effect. \\n     Any of the methods for gene therapy available in the art can be used according to the present invention. Exemplary methods are described below. \\n     For general reviews of the methods of gene therapy, see Goldspiel et al., (1993)  Clin. Pharm.  12:488–505; Wu &amp; Wu, (1991)  Biotherapy  3:87–95; U, (1993)  Ann. Rev. Pharmacol. Toxicol.  32:573–596; Mulligan, (1993)  Science  260:926–932; Morgan &amp; Anderson, (1993)  Ann. Rev. Biochem.  62:191–217; May, (1993)  TIBTECH  11(5): 155–215. Methods commonly known in the art of recombinant DNA technology which can be used are described in  Current Protocols in Molecular Biology , (Ausubel et al., eds.), Greene Publishing Associates and Wiley-Interscience, New York (2002); and Kriegler,  Gene Transfer and Expression, A Laboratory Manual , Stockton Press, New York, N.Y., USA (1990). \\n     In one aspect, the compound comprises nucleic acid sequences encoding an antibody, the nucleic acid sequences being part of expression vectors that express the antibody or fragments or chimeric proteins or heavy or light chains thereof in a suitable host. In particular, such nucleic acid sequences have promoters operably linked to the antibody coding region, the promoter being inducible or constitutive, and, optionally, tissue-specific. In another particular embodiment, nucleic acid molecules are used in which the antibody coding sequences and any other desired sequences are flanked by regions that promote homologous recombination at a desired site in the genome, thus providing for intrachromosomal expression of the antibody encoding nucleic acids (Koller &amp; Smithies, (1989)  Proc. Natl. Acad. Sci. USA  86:8932–8935; Zijlstra et al., (1989)  Nature  342:435–438. In specific embodiments, the expressed antibody molecule is a single chain antibody; alternatively, the nucleic acid sequences include sequences encoding both the heavy and light chains, or fragments thereof, of the antibody. \\n     Delivery of the nucleic acids into a subject can be either direct, in which case the subject is directly exposed to the nucleic acid or nucleic acid-carrying vectors, or indirect, in which case, cells are first transformed with the nucleic acids in vitro, then transplanted into the subject. These two approaches are known, respectively, as in vivo or ex vivo gene therapy. \\n     In a specific embodiment, the nucleic acid sequences are directly administered in vivo, where it is expressed to produce the encoded product. This can be accomplished by any of numerous methods known in the art, e.g., by constructing them as part of an appropriate nucleic acid expression vector and administering it so that they become intracellular, e.g., by infection using defective or attenuated retrovirals or other viral vectors (see U.S. Pat. No. 4,980,286), or by direct injection of naked DNA, or by use of microparticle bombardment (e.g., a gene gun; BIOLISTIC®, Dupont, Wilmington, Del., USA), or coating with lipids or cell-surface receptors or transfecting agents, encapsulation in liposomes, microparticles, or microcapsules, or by administering them in linkage to a peptide which is known to enter the nucleus, by administering it in linkage to a ligand subject to receptor-mediated endocytosis (see, e.g., Wu &amp; Wu, (1987)  J. Biol. Chem.  262:4429–4432) (which can be used to target cell types specifically expressing the receptors), etc. In another embodiment, nucleic acid-ligand complexes can be formed in which the ligand comprises a fusogenic viral peptide to disrupt endosomes, allowing the nucleic acid to avoid lysosomal degradation. In yet another embodiment, the nucleic acid can be targeted in vivo for cell specific uptake and expression, by targeting a specific receptor (see, e.g., PCT Publications WO 92/06180; WO 92/22635; WO92/20316; WO93/14188, WO 93/20221). Alternatively, the nucleic acid can be introduced intracellularly and incorporated within host cell DNA for expression, by homologous recombination (Koller &amp; Smithies, (1989)  Proc. Natl. Acad. Sci. USA  86:8932–8935; Zijlstra et al., (1989)  Nature  342:435–438). \\n     In a specific embodiment, a viral vector that contains nucleic acid sequences encoding an antibody of the present invention is used. For example, a retroviral vector can be used (see, e.g., Miller et al., (1993)  Meth. Enzymol.  217:581–599). These retroviral vectors contain the components necessary for the correct packaging of the viral genome and integration into the host cell DNA. The nucleic acid sequences encoding the antibody to be used in gene therapy are cloned into one or more vectors, which facilitates delivery of the gene into a subject. More detail about retroviral vectors can be found in Boesen et al., (1994)  Biotherapy  6:291–302, which describes the use of a retroviral vector to deliver the mdr1 gene to hematopoietic stem cells in order to make the stem cells more resistant to chemotherapy. Other references illustrating the use of retroviral vectors in gene therapy are: Clowes et al., (1994)  J. Clin. Invest.  93:644–651; Kiem et al., (1994)  Blood  83:1467–1473; Salmons &amp; Gunzberg, (1993)  Human Gene Therapy  4:129–141; and Grossman &amp; Wilson, (1993)  Curr. Opin. Genet. Devel.  3:110–114. \\n     Adenoviruses are other viral vectors that can be used in gene therapy. Adenoviruses are especially attractive vehicles for delivering genes to respiratory epithelia. Adenoviruses naturally infect respiratory epithelia where they cause a mild disease. Other targets for adenovirus-based delivery systems are liver, the central nervous system, endothelial cells, and muscle. Adenoviruses have the advantage of being capable of infecting non-dividing cells. Kozarsky &amp; Wilson (Kozarsky &amp; Wilson, (1993)  Current Opinion Genet. Devel.  3:499–503) present a review of adenovirus-based gene therapy. Bout et al. (Bout et al., (1994)  Human Gene Therapy  5:3–10) demonstrated the use of adenovirus vectors to transfer genes to the respiratory epithelia of rhesus monkeys. Other instances of the use of adenoviruses in gene therapy are known (see, e.g., Rosenfeld et al., (1991)  Science  252:431–434; Rosenfeld et al., (1992)  Cell  68:143–155; Mastrangeli et al., (1993)  J. Clin. Invest.  91:225–234; PCT Publication WO94/12649; and Wang et al., (1995)  Gene Therapy  2:775–783. In one embodiment, adenovirus vectors are used. \\n     Adeno-associated virus (AAV) has also been proposed for use in gene therapy (Walsh et al., (1993)  Proc. Soc. Exp. Biol. Med.  204:289–300; U.S. Pat. No. 5,436,146). \\n     Another approach to gene therapy involves transferring a gene to cells in tissue culture by such methods as electroporation, lipofection, calcium phosphate mediated transfection, or viral infection. Often, the method of transfer includes the transfer of a selectable marker to the cells. The cells are then placed under selection to isolate those cells that have taken up and are expressing the transferred gene. Those cells are then delivered to a subject. \\n     In this embodiment, the nucleic acid is introduced into a cell prior to administration in vivo of the resulting recombinant cell. Such introduction can be carried out by any method known in the art, including but not limited to transfection, electroporation, microinjection, infection with a viral or bacteriophage vector containing the nucleic acid sequences, cell fusion, chromosome-mediated gene transfer, microcell-mediated gene transfer, spheroplast fusion, etc. Numerous techniques are known in the art for the introduction of foreign genes into cells (see, e.g., Loeffler &amp; Behr, (1993)  Meth. Enzymol.  217:599–618; Cohen et al., (1993)  Meth. Enzymol.  217:618–644; Cline, (1985)  Pharmac. Ther.  29:69–92) and can be used in accordance with the present invention, provided that the necessary developmental and physiological functions of the recipient cells are not disrupted. The technique should provide for the stable transfer of the nucleic acid to the cell, so that the nucleic acid is expressible by the cell and preferably heritable and expressible by its cell progeny. \\n     The resulting recombinant cells can be delivered to a subject by various methods known in the art. For example, recombinant blood cells (e.g., hematopoietic stem or progenitor cells) can be administered intravenously. The amount of cells envisioned for use depends on the desired effect, subject state, etc., and such a determination can be made by one of ordinary skill in the art, upon consideration of the present disclosure. \\n     Cells into which a nucleic acid can be introduced for purposes of gene therapy encompass any desired, available cell type, and include but are not limited to epithelial cells, endothelial cells, keratinocytes, fibroblasts, muscle cells, hepatocytes; blood cells such as T-lymphocytes, B-lymphocytes, monocytes, macrophages, neutrophils, eosinophils, megakaryocytes, granulocytes; various stem or progenitor cells, in particular hematopoietic stem or progenitor cells, e.g., as obtained from bone marrow, umbilical cord blood, peripheral blood, fetal liver, etc. \\n     In one embodiment, the cell used for gene therapy is autologous to the subject. \\n     In an embodiment in which recombinant cells are used in gene therapy, nucleic acid sequences encoding an antibody are introduced into the cells such that they are expressible by the cells or their progeny, and the recombinant cells are then administered in vivo for therapeutic effect. In a specific embodiment, stem or progenitor cells are used. Any stem and/or progenitor cells which can be isolated and maintained in vitro can potentially be used in accordance with this embodiment of the present invention (see e.g. PCT Publication WO 94/08598; Stemple &amp; Anderson, (1992)  Cell  71:973–985; Rheinwald, (1980)  Meth. Cell Bio.  21A:229; and Pittelkow &amp; Scott, (1986)  Mayo Clinic Proc.  61:771). \\n     In a specific embodiment, the nucleic acid to be introduced for purposes of gene therapy comprises an inducible promoter operably linked to the coding region, such that expression of the nucleic acid is controllable by controlling the presence or absence of the appropriate inducer of transcription. \\n     Demonstration of Therapeutic or Prophylactic Activity \\n     The compounds and/or pharmaceutical compositions of the present invention are preferably tested in vitro, and then in vivo for the desired therapeutic or prophylactic activity, prior to use in humans. For example, in vitro assays to demonstrate the therapeutic or prophylactic utility of a compound or pharmaceutical composition include the effect of a compound on a cell line or a subject tissue sample. The effect of the compound or composition on the cell line and/or tissue sample can be determined utilizing techniques known to those of skill in the art including, but not limited to, rosette formation assays and cell lysis assays. In accordance with the present invention, in vitro assays that can be used to determine whether administration of a specific compound is indicated, include in vitro cell culture assays in which a subject tissue sample is grown in culture, and exposed to or otherwise administered a compound, and the effect of such compound upon the tissue sample is observed. \\n     Therapeutic/Prophylactic Administration and Compositions \\n     The present invention provides methods of treatment, inhibition and prophylaxis by administration to a subject of an effective amount of a compound or pharmaceutical composition of the present invention, for example an antibody of the present invention. In one aspect, the compound is substantially purified (e.g., substantially free from substances that limit its effect or produce undesired side-effects). The subject can be an animal, including, but not limited to, animals such as cows, pigs, horses, chickens, cats, dogs, rats mice, rabbits, monkeys, etc., and is preferably a mammal, and most preferably human. \\n     Formulations and methods of administration that can be employed when the compound comprises a nucleic acid or an immunoglobulin are described above; additional appropriate formulations and routes of administration can be selected from among those described herein below. \\n     Delivery Systems \\n     Various delivery systems are known and can be used to administer a compound of the present invention, e.g., encapsulation in liposomes, microparticles, microcapsules, recombinant cells capable of expressing the compound, receptor-mediated endocytosis (see, e.g., Wu &amp; Wu, (1987)  J. Biol. Chem.  262:4429–4432), construction of a nucleic acid as part of a retroviral or other vector, etc. Methods of introduction include but are not limited to intradermal, intramuscular, intraperitoneal, intravenous, subcutaneous, intranasal, epidural, and oral routes. The compounds and/or compositions of the present invention can be administered by any convenient route, for example by infusion or bolus injection, by absorption through epithelial or mucocutaneous linings (e.g., oral mucosa, rectal and intestinal mucosa, etc.) and may be administered together with other biologically active agents. Administration can be systemic or local. In addition, it might be desirable to introduce the pharmaceutical compounds or compositions of the present invention into the central nervous system by any suitable route, including intraventricular and intrathecal injection; intraventricular injection may be facilitated by an intraventricular catheter, for example, attached to a reservoir, such as an Ommaya reservoir. Pulmonary administration can also be employed, e.g., by use of an inhaler or nebulizer, and formulation with an aerosolizing agent. \\n     In a specific embodiment, it can be desirable to administer the pharmaceutical compounds and/or compositions of the present invention locally to the area in need of treatment. This can be achieved by, for example, and not by way of limitation, local infusion during surgery, topical application (e.g., in conjunction with a wound dressing after surgery), by injection, by means of a catheter, by means of a suppository, or by means of an implant, the implant being formed of a porous, non-porous, or gelatinous material, including membranes, such as sialastic membranes, or fibers. When administering a protein, including an antibody, of the present invention, care should be taken to use materials to which the protein does not absorb. \\n     In another embodiment, a compound and/or composition can be delivered in a vesicle, in particular a liposome (see, e.g., Langer, (1990)  Science  249:1527–1533; Treat et al., in  Liposomes in the Therapy of Infectious Disease and Cancer , (Lopez-Berestein and Fidler, eds.), Alfred R. Liss, New York, N.Y., USA (1989) pp. 353–365; Lopez-Berestein, in  Liposomes in the Therapy of Infectious Disease and Cancer , (Lopez-Berestein and Fidler, eds.), Alfred R. Liss, New York, N.Y., USA (1989) pp. 317–327; see generally  Liposomes in the Therapy of Infectious Disease and Cancer , (Lopez-Berestein and Fidler, eds.), Alfred R. Liss, New York, N.Y., USA (1989)). \\n     In yet another embodiment, the compound or composition can be delivered in a controlled release system. In one embodiment, a pump may be used (see, e.g., Langer, (1990)  Science  249:1527–1533; Sefton, (1987)  CRC Crit. Ref. Biomed. Eng.  14:201; Buchwald et al., (1980)  Surgery  88:507; Saudek et al., (1989)  N. Engl. J. Med.  321:574). In another embodiment, polymeric materials can be used (see, e.g.,  Medical Applications of Controlled Release , (Langer &amp; Wise, eds.), CRC Press, Boca Raton, Fla., USA (1974);  Controlled Drug Bioavailability, Drug Product Design and Performance , (Smolen and Ball, eds.), Wiley, New York, N.Y., USA (1984); Ranger &amp; Peppas, (1983)  J. Macromol. Sci. Rev. Macromol. Chem.  23:61; see also Levy et al., (1985)  Science  228:190; During et al., (1989)  Ann. Neurol.  25:351; Howard et al., (1989)  J. Neurosurg.  71:105). In yet another embodiment, a controlled release system can be placed in proximity of the therapeutic target, i.e., the brain, thus requiring only a fraction of the systemic dose (see, e.g., Goodson, in  Medical Applications of Controlled Release , (Langer &amp; Wise, eds.), CRC Press, Boca Raton, Fla., USA (1974), vol. 2, pp. 115–138 (1984)). Other controlled release systems are discussed in the review by Langer (Langer, (1990)  Science  249:1527–1533). \\n     In a specific embodiment where a compound of the present invention is a nucleic acid encoding a protein, the nucleic acid can be administered in vivo to promote expression of its encoded protein, by constructing it as part of an appropriate nucleic acid expression vector and administering it so that it becomes intracellular, e.g., by use of a retroviral vector (see U.S. Pat. No. 4,980,286), or by direct injection, or by use of microparticle bombardment (e.g., a gene gun; BIOLISTIC®, Dupont, Wilmington, Del., USA), or coating with lipids or cell-surface receptors or transfecting agents, or by administering it in linkage to a homeobox-like peptide which is known to enter the nucleus (see e.g., Joliot et al., (1991)  Proc. Natl. Acad. Sci. USA  88:1864–1868), etc. Alternatively, a nucleic acid can be introduced intracellularly and incorporated within host cell DNA for expression, by homologous recombination. \\n     Pharmaceutical Compositions \\n     The present invention also provides pharmaceutical compositions. Such compositions comprise a therapeutically effective amount of a compound, and a pharmaceutically acceptable carrier. In a specific embodiment, the term “pharmaceutically acceptable” means approved by a regulatory agency of the Federal or a state government or listed in the U.S. Pharmacopeia or other generally recognized pharmacopeia for use in animals, and more particularly in humans. The term “carrier” refers to a diluent, adjuvant, excipient, or vehicle with which the therapeutic is administered. Such pharmaceutical carriers can be sterile liquids, such as water and oils, including those of petroleum, animal, vegetable or synthetic origin, such as peanut oil, soybean oil, mineral oil, sesame oil and the like. Water and water-based formulations are desirable carriers when the pharmaceutical composition is administered intravenously. Saline solutions and aqueous dextrose and glycerol solutions can also be employed as liquid carriers, particularly for injectable solutions. Suitable pharmaceutical excipients include starch, glucose, lactose, sucrose, gelatin, malt, rice, flour, chalk, silica gel, sodium stearate, glycerol monostearate, talc, sodium chloride, dried skim milk, glycerol, propylene, glycol, water, ethanol and the like. The composition, if desired, can also contain minor amounts of wetting or emulsifying agents, or pH buffering agents. These compositions can take the form of solutions, suspensions, emulsion, tablets, pills, capsules, powders, sustained-release formulations and the like. The composition can be formulated as a suppository, with traditional binders and carriers such as triglycerides. Oral formulation can include standard carriers such as pharmaceutical grades of mannitol, lactose, starch, magnesium stearate, sodium saccharine, cellulose, magnesium carbonate, etc. Examples of suitable pharmaceutical carriers are described in  Remington&#39;s Pharmaceutical Sciences , (Gennaro, ed.) 20th ed., Mack Publishing, Easton, Pa., USA (2000). Such compositions will contain a therapeutically effective amount of the compound, for example in purified form, together with a suitable amount of carrier so as to provide the form for proper administration to the subject. The formulation should suit the mode of administration. \\n     In one embodiment, a composition is formulated in accordance with routine procedures as a pharmaceutical composition adapted for intravenous administration to human beings. Typically, compositions for intravenous administration are solutions in sterile isotonic aqueous buffer. Where necessary, a composition may also include a solubilizing agent and a local anesthetic such as lignocaine to ease pain at the site of the injection. Generally, the ingredients are supplied either separately or mixed together in unit dosage form, for example, as a dry lyophilized powder or water free concentrate in a hermetically sealed container such as an ampoule or sachette indicating the quantity of active agent. Where a composition is to be administered by infusion, it can be dispensed with an infusion bottle containing sterile pharmaceutical grade water or saline. Where a composition is administered by injection, an ampoule of sterile water for injection or saline can be provided so that the ingredients may be mixed prior to administration. \\n     The compounds of the present invention can be formulated as neutral or salt forms. Pharmaceutically acceptable salts include those formed with anions such as those derived from hydrochloric, phosphoric, acetic, oxalic, tartaric acids, etc., and those formed with cations such as those derived from sodium, potassium, ammonium, calcium, ferric hydroxides, isopropylamine, triethylamine, 2-ethylamino ethanol, histidine, procaine, etc. \\n     The amount of the compound of the present invention which will be effective in the treatment, inhibition and prevention of a disease or disorder associated with aberrant expression and/or activity of a polypeptide of the present invention can be determined by standard clinical techniques. In addition, in vitro assays may optionally be employed to help identify optimal dosage ranges. The precise dose to be employed in the formulation will also depend on the route of administration, and the seriousness of the disease or disorder, and should be decided according to the judgment of the practitioner and each subject&#39;s circumstances. Effective doses can be extrapolated from dose-response curves derived from in vitro or animal model test systems. \\n     For antibodies, the dosage administered to a subject is typically 0.1 mg/kg to 100 mg/kg of the subject&#39;s body weight. Preferably, the dosage administered to a subject is between 0.1 mg/kg and 20 mg/kg of the subject&#39;s body weight, more preferably 1 mg/kg to 10 mg/kg of the subject&#39;s body weight. Generally, human antibodies have a longer half-life within the human body than antibodies from other species due to the immune response to the foreign polypeptides. Thus, lower dosages of human antibodies and less frequent administration is often possible. Further, the dosage and frequency of administration of antibodies of the present invention might be reduced by enhancing uptake and tissue penetration (e.g., into the brain) of the antibodies by modifications such as, for example, lipidation. \\n     The present invention also provides a pharmaceutical pack or kit comprising one or more containers filled with one or more of the ingredients of the pharmaceutical compositions of the present invention. Optionally a notice can be associated with such container(s) in the form prescribed by a governmental agency regulating the manufacture, use or sale of pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human administration. Such a notice can also provide guidance on how to use the pack or kit. \\n     Diagnosis and Imaging with Antibodies \\n     Labeled antibodies, and derivatives and analogs thereof, that specifically bind to a polypeptide of interest can be used for diagnostic purposes to detect, diagnose, or monitor diseases, disorders, and/or conditions associated with the aberrant expression and/or activity of a polypeptide of the present invention. The present invention provides for the detection of aberrant expression of a polypeptide of interest (e.g., SEQ ID NO:Y, and fragments thereof), comprising (a) assaying the expression of the polypeptide of interest in cells or body fluid of an individual using one or more antibodies specific to the polypeptide interest and (b) comparing the level of gene expression with a standard gene expression level, whereby an increase or decrease in the assayed polypeptide gene expression level compared to the standard expression level is indicative of aberrant expression. \\n     The present invention provides a diagnostic assay for diagnosing a disorder, comprising: (a) assaying the expression of the polypeptide of interest in cells or body fluid of an individual using one or more antibodies specific to the polypeptide interest; and (b) comparing the level of gene expression with a standard gene expression level, whereby an increase or decrease in the assayed polypeptide gene expression level compared to the standard expression level is indicative of a particular disorder. With respect to cancer, the presence of a relatively high amount of transcript in biopsied tissue from an individual may indicate a predisposition for the development of the disease, or may provide a means for detecting the disease prior to the appearance of actual clinical symptoms. A more definitive diagnosis of this type can allow health professionals to employ preventative measures or aggressive treatment earlier thereby preventing the development or further progression of the cancer. \\n     Antibodies of the present invention can be used to assay protein levels in a biological sample using classical immunohistological methods known to those of ordinary skill in the art (see, e.g., Jalkanen et al., (1985)  J. Cell. Biol.  101:976–985; Jalkanen et al., (1987)  J. Cell. Biol.  105:3087–3096). Other antibody-based methods useful for detecting protein gene expression include immunoassays, such as the enzyme linked immunosorbent assay (ELISA) and the radioimmunoassay (RIA). Suitable antibody assay labels are known in the art and include enzyme labels, such as, glucose oxidase; radioisotopes, such as iodine ( 125 I,  121 I), carbon ( 14 C), sulfur ( 35 S), tritium ( 3 H), indium ( 112 In), and technetium ( 99 Tc); luminescent labels, such as luminol; and fluorescent labels, such as fluorescein and rhodamine, and biotin. \\n     Another aspect of the present invention is the detection and diagnosis of a disease or disorder associated with aberrant expression of a polypeptide of interest in an animal, preferably a mammal and most preferably a human. In one embodiment, diagnosis comprises: (a) administering (for example, parenterally, subcutaneously, or intraperitoneally) to a subject an effective amount of a labeled molecule which specifically binds to a polypeptide of interest; (b) waiting for a time interval following the administering for permitting the labeled molecule to preferentially concentrate at sites in the subject where the polypeptide is expressed (and for unbound labeled molecule to be cleared to background level); (c) determining background level; and (d) detecting the labeled molecule in the subject, such that detection of labeled molecule above the background level indicates that the subject has a particular disease or disorder associated with aberrant expression of the polypeptide of interest. Background level can be determined by various methods including, comparing the amount of labeled molecule detected to a standard value previously determined for a particular system. \\n     It will be understood in the art that the size of the subject and the imaging system used can determine, in part, the quantity of imaging moiety needed to produce diagnostic images. In the case of a radioisotope moiety, for a human subject, the quantity of radioactivity injected will normally range from about 5 to 20 millicuries of  99 Tc. The labeled antibody or antibody fragment will then preferentially accumulate at the location of cells which contain the specific protein. In vivo tumor imaging is described by Burchiel et al., in  Tumor Imaging: The Radiochemical Detection of Cancer , (Burchiel &amp; Rhodes, eds.), Masson Publishing Inc., New York, N.Y., USA (1982), Chapter 13. \\n     Depending on several variables, including the type of label used and the mode of administration, the time interval following the administration for permitting the labeled molecule to preferentially concentrate at sites in the subject and for unbound labeled molecule to be cleared to background level is 6 to 48 hours or 6 to 24 hours or 6 to 12 hours. In another embodiment the time interval following administration is 5 to 20 days or 5 to 10 days. \\n     In an embodiment, monitoring of a disease or disorder is carried out by repeating the method for diagnosing the disease or disease, for example, one month after initial diagnosis, six months after initial diagnosis, one year after initial diagnosis, etc. \\n     Presence of the labeled molecule can be detected in the subject using methods known in the art for in vivo scanning. These methods depend upon the type of label used. Upon consideration of the presnt disclosure, those of ordinary skill in the art will be able to determine the appropriate method for detecting a particular label. Methods and devices that can be used in the diagnostic methods of the present invention include, but are not limited to, computed tomography (CT), whole body scan such as position emission tomography (PET), magnetic resonance imaging (MRI), and sonography. \\n     In a specific embodiment, the molecule is labeled with a radioisotope and is detected in the subject using a radiation responsive surgical instrument (U.S. Pat. No. 5,441,050). In another embodiment, the molecule is labeled with a fluorescent compound and is detected in the subject using a fluorescence responsive scanning instrument. In another embodiment, the molecule is labeled with a positron emitting metal and is detected in the patent using positron emission-tomography. In yet another embodiment, the molecule is labeled with a paramagnetic label and is detected in a subject using magnetic resonance imaging (MRI). \\n     Kits of the Present Invention \\n     The present invention provides kits that can be used in the methods disclosed herein. In one embodiment, a kit comprises an antibody of the present invention, for example a purified antibody, in one or more containers. In a specific embodiment, a kit of the present invention contains a substantially isolated polypeptide comprising an epitope that is specifically immunoreactive with an antibody included in the kit. A kit of the present invention can further comprise a control antibody that does not react with the polypeptide of interest. In another specific embodiment, a kit of the present invention comprises a means for detecting the binding of an antibody to a polypeptide of interest. For example, an antibody can be conjugated to a detectable substrate such as a fluorescent compound, an enzymatic substrate, a radioactive compound or a luminescent compound, or a second antibody that recognizes the first antibody may be conjugated to a detectable substrate. \\n     In another specific embodiment of the present invention, a kit is a diagnostic kit for use in screening serum containing antibodies specific against proliferative and/or cancerous polynucleotides and polypeptides. Such a kit can comprise a control antibody that does not react with the polypeptide of interest. Such a kit can include a substantially isolated polypeptide antigen comprising an epitope that is specifically immunoreactive with at least one anti-polypeptide antigen antibody. Further, such a kit can include means for detecting the binding of said antibody to the antigen (e.g., the antibody can be conjugated to a fluorescent compound, such as fluorescein or rhodamine, which can be detected by flow cytometry). In specific embodiments, a kit can comprse a recombinantly produced or chemically synthesized polypeptide antigen. A polypeptide antigen of the kit can also be attached to a solid support. \\n     In a more specific embodiment the detecting means of the above-described kit includes a solid support to which said polypeptide antigen is attached. Such a kit can also include a non-attached reporter-labeled anti-human antibody. In this embodiment, binding of the antibody to the polypeptide antigen can be detected via the binding of the reporter-labeled antibody. \\n     In an additional embodiment, the present invention includes a diagnostic kit for use in screening serum containing antigens of a polypeptide of the present invention. The diagnostic kit includes a substantially isolated antibody specifically immunoreactive with polypeptide or polynucleotide antigens, and means for detecting the binding of the polynucleotide or polypeptide antigen to the antibody. In one embodiment, the antibody is attached to a solid support. In a specific embodiment, the antibody can be a monoclonal antibody. The detecting means of the kit can also include a second, labeled monoclonal antibody. Alternatively, or in addition, the detecting means can include a labeled, competing antigen. \\n     In one diagnostic configuration, test serum is reacted with a solid phase reagent having a surface-bound antigen obtained by the methods of the present invention. After binding with specific antigen antibody to the reagent and removing unbound serum components by washing, the reagent is reacted with reporter-labeled anti-human antibody to bind reporter to the reagent in proportion to the amount of bound anti-antigen antibody on the solid support. The reagent is again washed to remove unbound labeled antibody, and the amount of reporter associated with the reagent is determined. Often, the reporter is an enzyme that is detected by incubating the solid phase in the presence of a suitable fluorometric, luminescent or colorimetric substrate (Sigma, St. Louis, Mo., USA). \\n     The solid surface reagent in the above assay is prepared by known techniques for attaching protein material to solid support material, such as polymeric beads, dip sticks, 96-well plate or filter material. These attachment methods generally include non-specific adsorption of the protein to the support or covalent attachment of the protein, typically through a free amine group, to a chemically reactive group on the solid support, such as an activated carboxyl, hydroxyl, or aldehyde group. Alternatively, streptavidin coated plates can also be used in conjunction with biotinylated antigen(s). \\n     Thus, the present invention provides an assay system or kit for carrying out this diagnostic method. Such a kit generally comprises a support with surface-bound recombinant antigens, and a reporter-labeled anti-human antibody for detecting surface-bound anti-antigen antibody. \\n     Fusion Proteins \\n     Any polypeptide of the present invention can be used to generate fusion (e.g., chimeric) proteins. For example, a polypeptide of the present invention, when fused to a second protein, can be used as an antigenic tag. Antibodies raised against a polypeptide of the present invention can be used to indirectly detect the second protein by binding to the polypeptide. Moreover, because certain proteins target cellular locations based on trafficking signals, the polypeptides of the present invention can be used as targeting molecules once fused to other proteins. \\n     Examples of domains that can be fused to polypeptides of the present invention include not only heterologous signal sequences, but also other heterologous functional regions. The fusion does not necessarily need to be direct, but can occur through linker sequences. \\n     Moreover, fusion proteins can also be engineered to improve characteristics of a polypeptide of the present invention. For instance, a region of additional amino acids, particularly charged amino acids, can be added to the N-terminus of the polypeptide to improve stability and persistence during purification from the host cell or subsequent handling and storage. Peptide moieties can be added to the polypeptide to facilitate purification. Such regions can be removed prior to final preparation of the polypeptide. Similarly, peptide cleavage sites can be introduced between such peptide moieties, which could additionally be subjected to protease activity to remove said peptide(s) from a protein of the present invention. The addition of peptide moieties, including peptide cleavage sites, to facilitate handling of polypeptides are familiar and routine techniques will be known to those of ordinary skill in the art, upon consideration of the present disclosure. \\n     Moreover, polypeptides of the present invention, including fragments, and specifically epitopes, can be combined with parts of the constant domain of immunoglobulins (IgA, IgE, IgG, IgM) or portions thereof (CH1, CH2, CH3, and any combination thereof, including both entire domains and portions thereof), resulting in chimeric polypeptides. In some cases, these fusion proteins can facilitate purification and show an increased half-life in vivo. One reported example describes chimeric proteins consisting of the first two domains of the human CD4-polypeptide and various domains of the constant regions of the heavy or light chains of mammalian immunoglobulins. (EP A 394,827; Traunecker et al., (1988)  Nature  331:84–86). Fusion proteins having disulfide-linked dimeric structures (due to the IgG) can also be more efficient in binding and neutralizing other molecules, than the monomeric secreted protein or protein fragment alone (Fountoulakis et al., (1995)  J. Biochem.  270:3958–3964). \\n     Similarly, EP-A-O 464 533 (Canadian counterpart 2045869) discloses fusion proteins comprising various portions of the constant region of immunoglobulin molecules and another human protein or part thereof. In many cases, the Fc part of a fusion protein is beneficial in therapy and diagnosis, and thus can result in, for example, improved pharmacokinetic properties (see, e.g, EP-A 0232 262). Alternatively, deleting the Fc part after the fusion protein has been expressed, detected, and purified, would be desired. For example, the Fc portion can hinder therapy and diagnosis if the fusion protein is used as an antigen for immunizations. In drug discovery, for example, human proteins, such as hIL-5, have been fused with Fc portions for the purpose of high-throughput screening assays to identify antagonists of hIL-5 (see, e.g., Bennett et al., (1995)  J. Molecular Recognition  8:52–58; Johanson et al., (1995)  J. Biol. Chem.  270:9459–9471). \\n     Moreover, the polypeptides of the present invention can be fused to marker sequences (also referred to as “tags”). Due to the availability of antibodies specific to such “tags”, purification of the fused polypeptide of the present invention, and/or its identification is significantly facilitated since antibodies specific to the polypeptides of the present invention are not required. Such purification can be in the form of an affinity purification whereby an anti-tag antibody or another type of affinity matrix (e.g., anti-tag antibody attached to the matrix of a flow-thru column) that binds to the epitope tag is present. In one embodiment, the marker amino acid sequence is a hexa-histidine peptide, such as the tag provided in a pQE vector (Qiagen, Inc.), among others, many of which are commercially available. In one example, a hexa-histidine tag provides for convenient purification of the fusion protein (Gentz et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86:821–824). Another peptide tag useful for purification, the “HA” tag, corresponds to an epitope derived from the influenza hemagglutinin protein. (Wilson et al., (1984)  Cell  37:767). \\n     Those of ordinary skill in the art will be aware of other “tags” which could be readily substituted for the tags referred to herein for purification and/or identification of polypeptides of the present invention (see, e.g., Jones et al., (1995)  J Chromatogr A  707(1):3–22). For example, the c-myc tag and the 8F9, 3C7, 6E10, G4m B7 and 9E10 antibodies thereto (Evan et al., (1985)  Mol. Cell. Biol.  5:3610–3616); the Herpes Simplex virus glycoprotein D (gD) tag and its antibody (Paborsky et al., (1990)  Prot. Eng.  3(6):547–553), the FLAG® tag (Sigma, St. Louis, Mo., USA)—i.e., the octapeptide sequence DYKDDDDK (SEQ ID NO:18), (Hopp et al., (1988)  Biotech.  6:1204–1210); the KT3 epitope peptide (Martin et al., (1992)  Science  255:192–194); a-tubulin epitope peptide (Skinner et al., (1991)  J. Biol. Chem.  266:15136–15166); the T7 gene 10 protein peptide tag (Lutz-Freyermuth et al., (1990)  Proc. Natl. Sci. USA  87:6363–6397), the FITC epitope (Zymed, Inc., South San Francisco, Calif., USA), the GFP epitope (Zymed, Inc., South San Francisco, Calif., USA), and the rhodamine epitope (Zymed, Inc., South San Francisco, Calif., USA). \\n     The present invention also encompasses the attachment of up to nine codons encoding a repeating series of up to nine arginine amino acids to the coding region of a polynucleotide of the present invention. The present invention also encompasses chemically derivitizing a polypeptide of the present invention with a repeating series of up to nine arginine amino acids. Such a tag, when attached to a polypeptide, can serve as a “universal pass”, allowing compounds access to the interior of cells without additional derivitization or manipulation. \\n     Protein fusions involving polypeptides of the present invention, including fragments and/or variants thereof, can be used for the following, non-limiting examples, subcellular localization of proteins, determination of protein-protein interactions via immunoprecipitation, purification of proteins via affinity chromatography, functional and/or structural characterization of protein. The present invention also encompasses the application of hapten specific antibodies for any of the uses referenced above for epitope fusion proteins. For example, the polypeptides of the present invention could be chemically derivatized to attach hapten molecules (e.g., DNP, (Zymed, Inc., South San Francisco, Calif., USA)). Due to the availability of monoclonal antibodies specific to such haptens, the protein could be readily purified using immunoprecipation, for example. \\n     Polypeptides of the present invention, including fragments and/or variants thereof, in addition to antibodies directed against such polypeptides, fragments, and/or variants, can be fused to any of a number of known, and yet to be determined, toxins, such as ricin, saporin (Mashiba et al., (1999)  Ann. N.Y. Acad. Sci.  886:233–35), or HC toxin (Tonukari et al., (2000)  Plant Cell  12(2):237–248), for example. Such fusions could be used to deliver the toxins to desired tissues for which a ligand or a protein capable of binding to the polypeptides of the present invention exists. \\n     The present invention encompasses the fusion of antibodies directed against polypeptides of the present invention, including variants and fragments thereof, to a toxin for the purpose of delivering the toxin to specific locations in a cell, to specific tissues, and/or to specific species. Such bifunctional antibodies are known in the art, though a review describing additional advantageous fusions, including citations for methods of production, can be found in Hudson, (1999)  Curr. Opin. Immunol.  11:548–557. In this context, the term “toxin” can be expanded to include any heterologous protein, a small molecule, radionucleotides, cytotoxic drugs, liposomes, adhesion molecules, glycoproteins, ligands, cell or tissue-specific ligands, enzymes, of bioactive agents, biological response modifiers, anti-fungal agents, hormones, steroids, vitamins, peptides, peptide analogs, anti-allergenic agents, anti-tubercular agents, anti-viral agents, antibiotics, anti-protozoan agents, chelates, radioactive particles, radioactive ions, X-ray contrast agents, monoclonal antibodies, polyclonal antibodies and genetic material. Upon consideration of the present disclosure, one of ordinary skill in the art could determine whether any particular “toxin” could be used in the compounds of the present invention. Examples of suitable “toxins” listed above are exemplary only and are not intended to limit the “toxins” that can be used in the present invention. \\n     Thus, any of these above fusions can be engineered using the polynucleotides or the polypeptides of the present invention. \\n     Vectors, Host Cells, and Protein Production \\n     The present invention also relates to vectors comprising a polynucleotide of the present invention, host cells, and the production of polypeptides by recombinant techniques. A vector can be, for example, a phage, plasmid, viral, or retroviral vector. Retroviral vectors can be replication competent or replication defective. In the latter case, viral propagation generally will occur only in complementing host cells. \\n     A polynucleotide can be joined to a vector containing a selectable marker for propagation in a host. Generally, a plasmid vector is introduced in a precipitate, such as a calcium phosphate precipitate, or in a complex with a charged lipid. If the vector is a virus, it can be packaged in vitro using an appropriate packaging cell line and then transduced into host cells. \\n     The polynucleotide insert can be operatively linked to an appropriate promoter, such as the phage lambda PL promoter, the  E. coli  lac, trp, phoA and tac promoters, the SV40 early and late promoters and promoters of retroviral LTRs, to name a few. Other suitable promoters will be known to those of ordinary skill in the art upon consideration of the present disclosure. The expression constructs can further contain sites for transcription initiation, termination, and, in the transcribed region, a ribosome binding site for translation. The coding portion of the transcripts expressed by the constructs will preferably include a translation initiating codon at the beginning and a termination codon (UAA, UGA or UAG) appropriately positioned at the end of the polypeptide to be translated. \\n     As indicated, an expression vectors can comprise at least one selectable marker. Such markers include dihydrofolate reductase, G418 or neomycin resistance for eukaryotic cell culture and tetracycline, kanamycin or ampicillin resistance genes for culturing in  E. coli  and other bacteria. Representative examples of appropriate hosts include, but are not limited to, bacterial cells, such as  E. coli, Streptomyces  and  Salmonella typhimurium  cells; fungal cells, such as yeast cells (e.g.,  Saccharomyces cerevisiae  or  Pichia pastoris  (ATCC Accession No. 201178)); insect cells such as  Drosophila  S2 and  Spodoptera  Sf9 cells; animal cells such as CHO, COS, 293, and Bowes melanoma cells; and plant cells. Appropriate culture mediums and conditions for the above-described host cells will be known to those of ordinary skill in the art upon consideration of the present disclosure. \\n     Representative vectors that can be employed in bacterial systems include pQE70, pQE60 and pQE-9, available from QIAGEN, Inc.; pBluescript vectors, Phagescript vectors, pNH8A, pNH16a, pNH18A, pNH46A, available from Stratagene Cloning Systems, Inc. of La Jolla, Calif., USA; and ptrc99a, pKK223-3, pKK233-3, pDR540, pRIT5 available from Pharmacia Biotech, Inc of Peakpack, N.J., USA. Representative eukaryotic vectors are pWLNEO, pSV2CAT, pOG44, pXT1 and pSG available from Stratagene (La Jolla, Calif., USA); and pSVK3, pBPV, pMSG and pSVL available from Pharmacia (Peakpack, N.J., USA). Representative vectors for use in yeast systems include, but are not limited to pYES2, pYD1, pTEF1/Zeo, pYES2/GS, pPICZ, pGAPZ, pGAPZalph, pPIC9, pPIC3.5, pHIL-D2, pHIL-S1, pPIC3.5K, pPIC9K, and PAO815 (all available from Invitrogen, Carlsbad, Calif., USA). Other suitable vectors will be readily apparent to those of ordinary skill in the art, upon consideration of the present disclosure. \\n     Introduction of the construct into the host cell can be effected by calcium phosphate transfection, DEAE-dextran mediated transfection, cationic lipid-mediated transfection, electroporation, transduction, infection, or other methods. Such methods are described in many standard laboratory manuals, such as Davis et al.,  Basic Methods In Molecular Biology , Appleton &amp; Lange, (1995). It is specifically contemplated that the polypeptides of the present invention can in fact be expressed by a host cell lacking a recombinant vector. \\n     A polypeptide of the present invention can be recovered and purified from recombinant cell cultures by known methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Often, high performance liquid chromatography (“HPLC”) techniques can be employed for purification. \\n     Polypeptides of the present invention, and preferably the secreted form, can also be recovered from: products purified from natural sources, including bodily fluids, tissues and cells, whether directly isolated or cultured; products of chemical synthetic procedures; and products produced by recombinant techniques from a prokaryotic or eukaryotic host, including, for example, bacterial, yeast, higher plant, insect, and mammalian cells. Depending upon the host employed in a recombinant production procedure, the polypeptides of the present invention can be glycosylated or can be non-glycosylated. In addition, polypeptides of the present invention can also include an initial modified methionine residue, in some cases as a result of host-mediated processes. Thus, it is known in the art that the N-terminal methionine encoded by the translation initiation codon generally is removed with high efficiency from any protein after translation in all eukaryotic cells. While the N-terminal methionine on most proteins also is efficiently removed in most prokaryotes, for some proteins this prokaryotic removal process is inefficient, depending on the nature of the amino acid to which the N-terminal methionine is covalently linked. \\n     In one embodiment, the yeast  Pichia pastoris  is used to express a polypeptide of the present invention in a eukaryotic system.  Pichia pastoris  is a methylotrophic yeast which can metabolize methanol as its sole carbon source. A main step in the methanol metabolization pathway is the oxidation of methanol to formaldehyde using O 2 . This reaction is catalyzed by the enzyme alcohol oxidase. In order to metabolize methanol as its sole carbon source,  Pichia pastoris  must generate high levels of alcohol oxidase due, in part, to the relatively low affinity of alcohol oxidase for O 2 . Consequently, in a growth medium depending on methanol as a main carbon source, the promoter region of one of the two alcohol oxidase genes (AOX1) is highly active. In the presence of methanol, alcohol oxidase produced from the AOX1 gene comprises up to approximately 30% of the total soluble protein in  Pichia pastoris  (see, e.g., Ellis et al., (1985)  Mol. Cell. Biol.  5:1111–21; Koutz et al., Yeast 5:167–77 (1989); Tschopp et al., (1987)  Nucl. Acids Res.  15:3859–76). Thus, a heterologous coding sequence, such as, for example, a polynucleotide of the present invention, under the transcriptional regulation of all or part of the AOX1 regulatory sequence is expressed at exceptionally high levels in  Pichia  yeast grown in the presence of methanol. \\n     In one example, the plasmid vector pPIC9K is used to express DNA encoding a polypeptide of the present invention, as set forth herein, in a  Pichia  yeast system essentially as described in  Pichia Protocols: Methods in Molecular Biology  (Higgins &amp; Cregg, eds.) The Humana Press, Totowa, N.J., USA (1998). This expression vector allows expression and secretion of a protein of the present invention by virtue of the strong AOX1 promoter linked to the  Pichia pastoris  alkaline phosphatase (PHO) secretory signal peptide (i.e., leader) located upstream of a multiple cloning site. \\n     Many other yeast vectors could be used in place of pPIC9K, such as, pYES2, pYD1, pTEF1/Zeo, pYES2/GS, pPICZ, pGAPZ, pGAPZalpha, pPIC9, pPIC3.5, pHIL-D2, pHIL-S1, pPIC3.5K, and PAO815, as one of ordinary skill in the art would readily appreciate, as long as the proposed expression construct provides appropriately located signals for transcription, translation, secretion (if desired), and the like, including an in-frame AUG, as required. \\n     In another embodiment, high-level expression of a heterologous coding sequence, such as, for example, a polynucleotide of the present invention, may be achieved by cloning the heterologous polynucleotide of the present invention into an expression vector such as, for example, pGAPZ or pGAPZalpha, and growing the yeast culture in the absence of methanol. \\n     In addition to encompassing host cells containing the vector constructs discussed herein, the present invention also encompasses primary, secondary, and immortalized host cells of vertebrate origin, particularly mammalian origin, that have been engineered to delete or replace endogenous genetic material (e.g., coding sequence), and/or to include genetic material (e.g., heterologous polynucleotide sequences) that is operably associated with a polynucleotide of the present invention, and that activates, alters, and/or amplifies endogenous polynucleotides. For example, techniques known in the art can be used to operably associate heterologous control regions (e.g., promoter and/or enhancer) and endogenous polynucleotide sequences via homologous recombination, resulting in the formation of a new transcription unit (see, e.g., U.S. Pat. Nos. 5,641,670 and 5,733,761; PCT Publications WO 96/29411 and WO 94/12650; Koller et al., (1989)  Proc. Natl. Acad. Sci. USA  86:8932–8935; and Zijlstra et al., (1989)  Nature  342:435–438). \\n     In addition, a polypeptide of the present invention can be chemically synthesized using techniques known in the art (see, e.g., Creighton,  Proteins: Structures and Molecular Principles , W.H. Freeman &amp; Co., New York, N.Y., USA (1983), and Hunkapiller et al., (1984)  Nature  310:105–111). For example, a polypeptide corresponding to a fragment of a polypeptide sequence of the present invention can be synthesized by use of a peptide synthesizer. Furthermore, if desired, nonclassical amino acids or chemical amino acid analogs can be introduced as a substitution or addition into the polypeptide sequence. Non-classical amino acids include, but are not limited to, to the D-isomers of the common amino acids, 2,4-diaminobutyric acid, a-amino isobutyric acid, 4-aminobutyric acid, Abu, 2-amino butyric acid, g-Abu, e-Ahx, 6-amino hexanoic acid, Aib, 2-amino isobutyric acid, 3-amino propionic acid, ornithine, norleucine, norvaline, hydroxyproline, sarcosine, citrulline, homocitrulline, cysteic acid, t-butylglycine, t-butylalanine, phenylglycine, cyclohexylalanine, b-alanine, fluoro-amino acids, designer amino acids such as b-methyl amino acids, Ca-methyl amino acids, Na-methyl amino acids, and amino acid analogs in general. Furthermore, the amino acid can be D (dextrorotary) or L (levorotary). \\n     The present invention encompasses polypeptides which are differentially modified during or after translation, e.g., by glycosylation, acetylation, phosphorylation, amidation, derivatization by known protecting/blocking groups, proteolytic cleavage, linkage to an antibody molecule or other cellular ligand, etc. Any of numerous chemical modifications may be carried out by known techniques, including but not limited, to specific chemical cleavage by cyanogen bromide, trypsin, chymotrypsin, papain, V8 protease, NaBH 4 ; acetylation, formylation, oxidation, reduction; metabolic synthesis in the presence of tunicamycin; etc. \\n     Additional post-translational modifications encompassed by the present invention include, for example, e.g., N-linked or O-linked carbohydrate chains, processing of N-terminal or C-terminal ends), attachment of chemical moieties to the amino acid backbone, chemical modifications of N-linked or O-linked carbohydrate chains, and addition or deletion of an N-terminal methionine residue as a result of prokaryotic host cell expression. The polypeptides can also be modified with a detectable label, such as an enzymatic, fluorescent, isotopic or affinity label to allow for detection and isolation of the protein, the addition of epitope tagged peptide fragments (e.g., FLAG® tag (Sigma, St. Louis, Mo., USA; SEQ ID NO:18), HA, GST, thioredoxin, maltose binding protein, etc.), attachment of affinity tags such as biotin and/or streptavidin, the covalent attachment of chemical moieties to the amino acid backbone, N- or C-terminal processing of the polypeptides ends (e.g., proteolytic processing), deletion of the N-terminal methionine residue, etc. \\n     Chemically Modified Derivatives \\n     Also provided by the present invention are chemically modified derivatives of a polypeptide of the present invention that might provide additional advantages such as increased solubility, stability and circulating time of the polypeptide, or decreased immunogenicity (see U.S. Pat. No. 4,179,337). The chemical moieties for derivitization can be selected from water soluble polymers such as polyethylene glycol, ethylene glycol/propylene glycol copolymers, carboxymethylcellulose, dextran, polyvinyl alcohol and the like. A polypeptide can be modified at random positions within the molecule, or at predetermined positions within the molecule and can include one, two, three or more attached chemical moieties. \\n     The present invention further encompasses chemical derivitization of a polypeptide of the present invention, preferably where the chemical is a hydrophilic polymer residue. Exemplary hydrophilic polymers, including derivatives, can be those that include polymers in which the repeating units contain one or more hydroxy groups (polyhydroxy polymers), including, for example, poly(vinyl alcohol); polymers in which the repeating units contain one or more amino groups (polyamine polymers), including, for example, peptides, polypeptides, proteins and lipoproteins, such as albumin and natural lipoproteins; polymers in which the repeating units contain one or more carboxy groups (polycarboxy polymers), including, for example, carboxymethylcellulose, alginic acid and salts thereof, such as sodium and calcium alginate, glycosaminoglycans and salts thereof, including salts of hyaluronic acid, phosphorylated and sulfonated derivatives of carbohydrates, genetic material, such as interleukin-2 and interferon, and phosphorothioate oligomers; and polymers in which the repeating units contain one or more saccharide moieties (polysaccharide polymers), including, for example, carbohydrates. \\n     The molecular weight of the hydrophilic polymers can vary, and is generally about 50 to about 5,000,000, with polymers having a molecular weight of about 100 to about 50,000 being preferred in some situations. The polymers can be branched or unbranched. Polymers having a molecular weight of about 150 to about 10,000 are preferred in some situations, with molecular weights of 200 to about 8,000 being preferred in other situations. \\n     For polyethylene glycol, the preferred molecular weight is often, but not always, between about 1 kDa and about 100 kDa (the term “about” indicating that in preparations of polyethylene glycol, some molecules will weigh more, some less, than the stated molecular weight) for ease in handling and manufacturing. Other sizes can be used, depending on the desired therapeutic profile (e.g., the duration of sustained release desired, the effects, if any on biological activity, the ease in handling, the degree or lack of antigenicity and other known effects of the polyethylene glycol to a therapeutic protein or analog). \\n     Additional polymers that can be used to derivatize polypeptides of the present invention, include, for example, poly(ethylene glycol) (PEG), poly(vinylpyrrolidine), polyoxomers, polysorbate and poly(vinyl alcohol), with PEG polymers being particularly preferred in many situations. Among the PEG polymers, PEG polymers having a molecular weight of from about 100 to about 10,000, from about 200 to about 8,000 can be employed, with PEG 2,000, PEG 5,000 and PEG 8,000, which have molecular weights of 2,000, 5,000 and 8,000, respectively, being preferred under many circumstances. Other suitable hydrophilic polymers, in addition to those exemplified above, will be readily apparent to one of ordinary skill in the art upon consideratoin of the present disclosure. Generally, the polymers used may include polymers that can be attached to the polypeptides of the present invention via alkylation or acylation reactions. \\n     The polyethylene glycol molecules (or other chemical moieties) can be attached to the protein with consideration of effects on functional or antigenic domains of the protein. There are a number of attachment methods available to those skilled in the art, (see, e.g., EP 0 401 384 (coupling PEG to G-CSF), and Malik et al., (1992)  Exp. Hematol.  20:1028–1035 (reporting pegylation of GM-CSF using tresyl chloride)). For example, polyethylene glycol can be covalently bound through amino acid residues via a reactive group, such as, a free amino or carboxyl group. Reactive groups are those to which an activated polyethylene glycol molecule can be bound. The amino acid residues having a free amino group can include lysine residues and the N-terminal amino acid residues; those having a free carboxyl group can include aspartic acid residues glutamic acid residues and the C-terminal amino acid residue. Sulfhydryl groups can also be used as a reactive group for attaching the polyethylene glycol molecules. For therapeutic purposes, attachment at an amino group, such as attachment at the N-terminus or lysine group, can be desirable. \\n     One may specifically desire proteins chemically modified at the N-terminus. Using polyethylene glycol as an illustration of the present composition, one can select from a variety of polyethylene glycol molecules (by molecular weight, branching, etc.), the proportion of polyethylene glycol molecules to protein (polypeptide) molecules in the reaction mix, the type of pegylation reaction to be performed, and the method of obtaining the selected N-terminally pegylated protein. The method of obtaining the N-terminally pegylated preparation (i.e., separating this moiety from other monopegylated moieties if necessary) can be by purification of the N-terminally pegylated material from a population of pegylated protein molecules. Selective proteins chemically modified at the N-terminus modification can be formed by reductive alkylation, which exploits differential reactivity of different types of primary amino groups (lysine versus the N-terminus) available for derivatization in a particular protein. Under appropriate reaction conditions, substantially selective derivatization of the protein at the N-terminus with a carbonyl group containing polymer is achieved. \\n     As with the various polymers exemplified above, the polymeric residues can comprise functional groups in addition, for example, to those typically involved in linking the polymeric residues to the polypeptides of the present invention. Such functionalities include, for example, carboxyl, amine, hydroxy and thiol groups. These functional groups on the polymeric residues can be further reacted, if desired, with materials that are generally reactive with such functional groups and which can assist in targeting specific tissues in the body including, for example, diseased tissue. Non-limiting examples of materials that can be reacted with the additional functional groups include, for example, proteins, including antibodies, carbohydrates, peptides, glycopeptides, glycolipids, lectins, and nucleosides. \\n     In addition to residues of hydrophilic polymers, the chemical used to derivatize the polypeptides of the present invention can be a saccharide residue. Representative saccharides that can be derived include, for example, monosaccharides or sugar alcohols, such as erythrose, threose, ribose, arabinose, xylose, lyxose, fructose, sorbitol, mannitol and sedoheptulose, with preferred monosaccharides being fructose, mannose, xylose, arabinose, mannitol and sorbitol; and disaccharides, such as lactose, sucrose, maltose and cellobiose. Other saccharides include, for example, inositol and ganglioside head groups. Other suitable saccharides, in addition to those exemplified above, will be readily apparent to one skilled in the art upon consideration of the present disclosure. Generally, saccharides that can be used for derivitization include saccharides that can be attached to a polypeptide of the present invention via alkylation or acylation reactions. \\n     Moreover, the present invention also encompasses derivitization of a polypeptide of the present invention, for example, with lipids (including cationic, anionic, polymerized, charged, synthetic, saturated, unsaturated, and any combination of the above, etc.) and stabilizing agents. \\n     The present invention encompasses derivitization of the polypeptides of the present invention, for example, with compounds that can serve a stabilizing function (e.g., to increase the polypeptides half-life in solution, to make the polypeptides more water soluble, to increase the polypeptides hydrophilic or hydrophobic character, etc.). Representative polymers useful as stabilizing materials can be of natural, semi-synthetic (modified natural) or synthetic origin. Representative natural polymers include naturally occurring polysaccharides, such as, for example, arabinans, fructans, fucans, galactans, galacturonans, glucans, mannans, xylans (such as, for example, inulin), levan, fucoidan, carrageenan, galatocarolose, pectic acid, pectins, including amylose, pullulan, glycogen, amylopectin, cellulose, dextran, dextrin, dextrose, glucose, polyglucose, polydextrose, pustulan, chitin, agarose, keratin, chondroitin, dermatan, hyaluronic acid, alginic acid, xanthin gum, starch and various other natural homopolymer or heteropolymers, such as those containing one or more of the following aldoses, ketoses, acids or amines: erythose, threose, ribose, arabinose, xylose, lyxose, allose, altrose, glucose, dextrose, mannose, gulose, idose, galactose, talose, erythrulose, ribulose, xylulose, psicose, fructose, sorbose, tagatose, mannitol, sorbitol, lactose, sucrose, trehalose, maltose, cellobiose, glycine, serine, threonine, cysteine, tyrosine, asparagine, glutamine, aspartic acid, glutamic acid, lysine, arginine, histidine, glucuronic acid, gluconic acid, glucaric acid, galacturonic acid, mannuronic acid, glucosamine, galactosamine, and neuraminic acid, and naturally occurring derivatives thereof. Accordingly, suitable polymers include, for example, proteins, such as albumin, polyalginates, and polylactide-coglycolide polymers. \\n     Representative semi-synthetic polymers include carboxymethylcellulose, hydroxymethylcellulose, hydroxypropylmethylcellulose, methylcellulose, and methoxycellulose. Exemplary synthetic polymers include polyphosphazenes, hydroxyapatites, fluoroapatite polymers, polyethylenes (such as, for example, polyethylene glycol (including for example, the class of compounds referred to as PLURONICS®, commercially available from BASF, Parsippany, N.J., USA), polyoxyethylene, and polyethylene terephthlate), polypropylenes (such as, for example, polypropylene glycol), polyurethanes (such as, for example, polyvinyl alcohol (PVA), polyvinyl chloride and polyvinylpyrrolidone), polyamides including nylon, polystyrene, polylactic acids, fluorinated hydrocarbon polymers, fluorinated carbon polymers (such as, for example, polytetrafluoroethylene), acrylate, methacrylate, and polymethylmethacrylate, and derivatives thereof. Methods for the preparation of a derivatized polypeptide of the present invention that employ polymers as stabilizing compounds will be readily apparent to those of ordinary skill in the art, in view of the present disclosure, when coupled with information known in the art, such as that described and referred to in U.S. Pat. No. 5,205,290. \\n     Moreover, the present invention encompasses additional modifications of a polypeptide of the present invention. Such additional modifications are known in the art, and are specifically provided, in addition to methods of derivitization, etc., in U.S. Pat. No. 6,028,066. \\n     Multimers \\n     A polypeptide of the present invention can take the form of monomers or multimers (i.e., dimers, trimers, tetramers and higher multimers). Accordingly, the present invention relates to monomers and multimers of the polypeptides of the present invention, their preparation, and compositions (e.g., therapeutics) containing them. In specific embodiments, the polypeptides of the present invention are monomers, dimers, trimers or tetramers. In additional embodiments, the multimers of the present invention are at least dimers, at least trimers, or at least tetramers. \\n     Multimers encompassed by the present invention may be homomers or heteromers. As used herein, the term “homomer”, refers to a multimer containing only polypeptides corresponding to the amino acid sequence of SEQ ID NO:Y or encoded by the cDNA contained in a deposited clone (including fragments, variants, splice variants, and fusion proteins, corresponding to these polypeptides as described herein). These homomers can contain polypeptides having identical or different amino acid sequences. In a specific embodiment, a homomer of the present invention is a multimer containing only polypeptides having an identical amino acid sequence. In another specific embodiment, a homomer of the present invention is a multimer containing polypeptides having different amino acid sequences. In specific embodiments, a multimer of the present invention is a homodimer (e.g., containing polypeptides having identical or different amino acid sequences) or a homotrimer (e.g., containing polypeptides having identical and/or different amino acid sequences). In additional embodiments, the homomeric multimer of the present invention is at least a homodimer, at least a homotrimer, or at least a homotetramer. \\n     As used herein, the term heteromer refers to a multimer containing one or more heterologous polypeptides (i.e., polypeptides of different proteins) in addition to the polypeptides of the present invention. In a specific embodiment, the multimer of the present invention is a heterodimer, a heterotrimer, or a heterotetramer. In additional embodiments, the heteromeric multimer of the present invention is at least a heterodimer, at least a heterotrimer, or at least a heterotetramer. \\n     Multimers of the present invention can be the result of hydrophobic, hydrophilic, ionic and/or covalent associations and/or may be indirectly linked, by for example, liposome formation. Thus, in one embodiment, multimers of the present invention, such as, for example, homodimers or homotrimers, are formed when polypeptides of the present invention contact one another in solution. In another embodiment, heteromultimers of the present invention, such as, for example, heterotrimers or heterotetramers, are formed when polypeptides of the present invention contact antibodies to the polypeptides of the present invention (including antibodies to the heterologous polypeptide sequence in a fusion protein of the present invention) in solution. In other embodiments, multimers of the present invention are formed by covalent associations with and/or between polypeptides of the present invention. Such covalent associations can involve one or more amino acid residues contained in the polypeptide sequence (e.g., that recited in the sequence listing, or contained in the polypeptide encoded by a deposited clone). In one instance, the covalent associations are cross-linking between cysteine residues located within the polypeptide sequences which interact in the native (i.e., naturally occurring) polypeptide. In another instance, the covalent associations are the consequence of chemical or recombinant manipulation. Alternatively, such covalent associations may involve one or more amino acid residues contained in a heterologous polypeptide sequence in a fusion protein of the present invention. \\n     In one example, covalent associations are between the heterologous sequence contained in a fusion protein of the present invention (see, e.g., U.S. Pat. No. 5,478,925). In a specific example, the covalent associations are between the heterologous sequence contained in a Fc fusion protein of the present invention (as described herein). In another specific example, covalent associations of fusion proteins of the present invention are between heterologous polypeptide sequence from another protein that is capable of forming covalently associated multimers, such as for example, osteoprotegerin (see, e.g., PCT Publication WO 98/49305). In another embodiment, two or more polypeptides of the present invention are joined through peptide linkers. Examples include those peptide linkers described in U.S. Pat. No. 5,073,627. Proteins comprising multiple polypeptides of the present invention separated by peptide linkers can be produced using conventional recombinant DNA technology. \\n     Another method for preparing multimer polypeptides of the present invention involves the use of polypeptides of the present invention fused to a leucine zipper or isoleucine zipper polypeptide sequence. Leucine zipper and isoleucine zipper domains are polypeptides that promote multimerization of the proteins in which they are found. Leucine zippers were originally identified in several DNA-binding proteins (Landschulz et al., (1988)  Science  240:1759), and have since been found in a variety of different proteins. Among the known leucine zippers are naturally occurring peptides and derivatives thereof that dimerize or trimerize. Examples of leucine zipper domains suitable for producing soluble multimeric proteins of the present invention are those described in PCT Publication WO 94/10308. Recombinant fusion proteins comprising a polypeptide of the present invention fused to a polypeptide sequence that dimerizes or trimerizes in solution are expressed in suitable host cells, and the resulting soluble multimeric fusion protein is recovered from the culture supernatant using techniques known to those of ordinary skill in the art. \\n     Trimeric polypeptides of the present invention can offer the advantage of enhanced biological activity. Representative leucine zipper moieties and isoleucine moieties are those that preferentially form trimers. One example is a leucine zipper derived from lung surfactant protein D (SPD) (Hoppe et al., (1994)  FEBS Lett.  344:191). Other peptides derived from naturally occurring trimeric proteins can be employed in preparing trimeric polypeptides of the present invention. \\n     In another example, proteins of the present invention are associated by interactions between a FLAG® tag (Sigma, St. Louis, Mo., USA) contained in fusion proteins of the present invention containing a FLAG® tag sequence. In a further embodiment, associations proteins of the present invention are associated by interactions between heterologous polypeptide sequence contained in a FLAG® tag fusion proteins of the present invention and anti-FLAG® antibody. \\n     Multimers of the present invention can be generated using chemical techniques known in the art. For example, for polypeptides desired to be contained in the multimers of the present invention can be chemically cross-linked using linker molecules and linker molecule length optimization techniques known in the art (see, e.g., U.S. Pat. No. 5,478,925). Additionally, multimers of the present invention can be generated using techniques known in the art to form one or more inter-molecule cross-links between the cysteine residues located within the sequence of the polypeptides desired to be contained in the multimer (see, e.g., U.S. Pat. No. 5,478,925). Further, polypeptides of the present invention can be routinely modified by the addition of cysteine or biotin to the C terminus or N-terminus of the polypeptide, and techniques known in the art can be employed to generate multimers comprising one or more of these modified polypeptides (see, e.g., U.S. Pat. No. 5,478,925). Additionally, techniques known in the art can be applied to generate liposomes containing the polypeptide components desired to be contained in the multimer of the present invention (see, e.g., U.S. Pat. No. 5,478,925). \\n     Alternatively, multimers of the present invention can be generated using genetic engineering techniques known in the art. In one embodiment, polypeptides contained in multimers of the present invention are produced recombinantly using fusion protein technology described herein or otherwise known in the art (see, e.g., U.S. Pat. No. 5,478,925). In a specific embodiment, polynucleotides coding for a homodimer of the present invention are generated by ligating a polynucleotide sequence encoding a polypeptide of the present invention to a sequence encoding a linker polypeptide and then further to a synthetic polynucleotide encoding the translated product of the polypeptide in the reverse orientation from the original C-terminus to the N-terminus (lacking the leader sequence) (see, e.g., U.S. Pat. No. 5,478,925). In another embodiment, recombinant techniques described herein or otherwise known in the art are applied to generate recombinant polypeptides of the present invention that contain a transmembrane domain (or hydrophobic or signal peptide) and that can be incorporated by membrane reconstitution techniques into liposomes (see, e.g., U.S. Pat. No. 5,478,925). \\n     In addition, a polynucleotide insert of the present invention could be operatively linked to “artificial” or chimeric promoters and transcription factors. Specifically, the artificial promoter could comprise, or alternatively consist, of any combination of cis-acting DNA sequence elements that are recognized by trans-acting transcription factors. For example, the cis acting DNA sequence elements and trans-acting transcription factors are operable in mammals. Further, the trans-acting transcription factors of such “artificial” promoters could also be “artificial” or chimeric in design themselves and could act as activators or repressors to said “artificial” promoter. \\n     Representative Applications for the Polynucleotides of the Present Invention \\n     Each of the polynucleotides identified herein can be used in numerous ways as reagents. The following description highlights only a few of the possible applications for the polynucleotides of the present invention. The described applications all employ techniques known to those of ordinary skill in the art, in conjunction with the polynucleotides of the present invention. \\n     The polynucleotides of the present invention are useful for chromosome identification. There exists an ongoing need to identify new chromosome markers, since few chromosome-marking reagents, based on actual sequence data (repeat polymorphisms), are presently available. Each polynucleotide of the present invention can be used as a chromosome marker. \\n     Briefly, sequences can be mapped to chromosomes by preparing PCR primers (e.g., about 15 to about 25 bp) from the sequences shown in SEQ ID NO:X. Primers can be selected using computer analysis so that primers do not span more than one predicted exon in the genomic DNA. These primers can then be used for PCR screening of somatic cell hybrids containing individual human chromosomes. Only those hybrids containing the human gene corresponding to the SEQ ID NO:X will yield an amplified fragment. \\n     Similarly, somatic hybrids provide a rapid method of PCR mapping the polynucleotides to particular chromosomes. Three or more clones can be assigned per day using a single thermal cycler. Moreover, sublocalization of the polynucleotides can be achieved with panels of specific chromosome fragments. Other gene mapping strategies that can be used include in situ hybridization, prescreening with labeled flow-sorted chromosomes, and preselection by hybridization to construct chromosome specific-cDNA libraries. \\n     Precise chromosomal location of the polynucleotides can also be achieved using fluorescence in situ hybridization (FISH) of a metaphase chromosomal spread. This technique can use polynucleotides as short as about 500 or about 600 bases; however, polynucleotides 2,000–4,000 bp are typically employed. For a review of this technique, see Verma et al. (Verma et al.,  Human Chromosomes: a Manual of Basic Techniques  Pergamon Press, New York, N.Y., USA (1988). \\n     For chromosome mapping, the polynucleotides can be used individually (to mark a single chromosome or a single site on that chromosome) or in panels (for marking multiple sites and/or multiple chromosomes). Representative polynucleotides correspond to the noncoding regions of the cDNAs because the coding sequences are more likely conserved within gene families, thus increasing the chance of cross hybridization during chromosomal mapping. \\n     Once a polynucleotide has been mapped to a precise chromosomal location, the physical position of the polynucleotide can be used in linkage analysis. Linkage analysis establishes coinheritance between a chromosomal location and presentation of a particular disease. Disease mapping data are known in the art. For example, assuming 1 megabase mapping resolution and one gene per 20 kb, a cDNA precisely localized to a chromosomal region associated with the disease could be one of 50–500 potential causative genes. \\n     Thus, once coinheritance is established, differences in the polynucleotide and the corresponding gene between affected and unaffected organisms can be examined. First, visible structural alterations in the chromosomes, such as deletions or translocations, are examined in chromosome spreads or by PCR. If no structural alterations exist, the presence of point mutations are ascertained. Mutations observed in some or all affected organisms, but not in normal organisms, indicates that the mutation may cause the disease. However, complete sequencing of the polypeptide and the corresponding gene from several normal organisms is required to distinguish the mutation from a polymorphism. If a new polymorphism is identified, this polymorphic polypeptide can be used for further linkage analysis. \\n     Furthermore, increased or decreased expression of the gene in affected organisms as compared to unaffected organisms can be assessed using a polynucleotide of the present invention. Any of these alterations (altered expression, chromosomal rearrangement, or mutation) can be used as a diagnostic or prognostic marker. \\n     The present invention, therefore, also provides a diagnostic method useful during diagnosis of a disorder, involving measuring the expression level of polynucleotides of the present invention in cells or body fluid from an organism and comparing the measured gene expression level with a standard level of polynucleotide expression level, whereby an increase or decrease in the gene expression level compared to the standard is indicative of a disorder. \\n     The term “measuring the expression level of a polynucleotide of the present invention” means qualitatively or quantitatively measuring or estimating the level of a polypeptide of the present invention or the level of the mRNA encoding the polypeptide in a first biological sample either directly (e.g., by determining or estimating absolute protein level or mRNA level) or relatively (e.g., by comparing to the polypeptide level or mRNA level in a second biological sample). In one embodiment, the polypeptide level or mRNA level in the first biological sample is measured or estimated and compared to a standard polypeptide level or mRNA level, the standard being taken from a second biological sample obtained from an individual not having the disorder or being determined by averaging levels from a population of organisms not having a disorder. As will be appreciated in the art, once a standard polypeptide level or mRNA level is known, it can be used repeatedly as a standard for comparison. \\n     The term “biological sample” means any biological sample obtained from an organism, body fluids, cell line, tissue culture, or other source which contains a polypeptide of the present invention or mRNA. As indicated, biological samples include body fluids (such as the following non-limiting examples, sputum, amniotic fluid, urine, saliva, breast milk, secretions, interstitial fluid, blood, serum, spinal fluid, etc.) that contain a polypeptide of the present invention, and other tissue sources found to express a polypeptide of the present invention. Methods for obtaining tissue biopsies and body fluids from organisms are known in the art. Where the biological sample is to include mRNA, a tissue biopsy is a preferred source. \\n     The method(s) provided herein can be applied in a diagnostic method and/or kits in which polynucleotides and/or polypeptides are attached to a solid support. In one represenative method, the support is a “gene chip” or a “biological chip” as described in U.S. Pat. Nos. 5,837,832, 5,874,219, and 5,856,174. Further, such a gene chip, comprising polynucleotides of the present invention attached thereto, can be used to identify polymorphisms between the polynucleotide sequences, with polynucleotides isolated from a test subject. The knowledge of such polymorphisms (i.e. their location, as well as, their existence) can be beneficial in identifying disease loci for many disorders, including proliferative diseases and conditions. Such a method is described in U.S. Pat. Nos. 5,858,659 and 5,856,104. \\n     The present invention encompasses polynucleotides of the present invention that are chemically synthesized, or reproduced as peptide nucleic acids (PNA), or according to other methods known in the art. The use of PNAs can be desirable if the polynucleotides are incorporated onto a solid support, or gene chip. For the purposes of the present invention, a peptide nucleic acid (PNA) comprises a polyamide type of DNA analog and the monomeric units for adenine, guanine, thymine and cytosine are available commercially (Perseptive Biosystems, Foster City, Calif., USA). Certain components of DNA, such as phosphorus, phosphorus oxides, or deoxyribose derivatives, are not present in PNAs. PNAs bind specifically and tightly to complementary DNA strands and are not degraded by nucleases (Nielsen et al., (1991)  Science  254:1497; and Egholm et al., (1993)  Nature  365:666). In fact, PNA binds more strongly to DNA than DNA itself does. PNA/DNA duplexes also bind under a wider range of stringency conditions than DNA/DNA duplexes, making it easier to perform multiplex hybridization. Smaller probes can be used than with DNA due to the stronger binding characteristics of PNA:DNA hybrids. In addition, it is more likely that single base mismatches can be determined with PNA/DNA hybridization because a single mismatch in a PNA/DNA 15-mer lowers the melting point (Tm) by 8°–20° C., vs. 4°–16° C. for the DNA/DNA 15-mer duplex. Also, the absence of charged groups in PNA means that hybridization can be done at low ionic strengths and reduce possible interference by salt during the analysis. \\n     In addition to the foregoing, a polynucleotide can be used to control gene expression through triple helix formation or antisense DNA or RNA. Antisense techniques are known (Okano, (1991)  J. Neurochem.  56: 560 ; Oligodeoxynucleotides as Antisense Inhibitors of Gene Expression , CRC Press, Boca Raton, Fla., USA (1988)). Triple helix formation is discussed, for instance, in Lee et al., (1979)  Nucl. Acid Res.  6:3073; Cooney et al., (1988)  Science  241:456; and Dervan et al., (1991)  Science  251:1360. Both methods rely on binding of the polynucleotide to a complementary DNA or RNA. For these techniques, representative polynucleotides are usually oligonucleotides 20 to 40 bases in length and complementary to either the region of the gene involved in transcription (triple helix—Lee et al., (1979)  Nucl. Acid Res.  6:3073; Cooney et al., (1988)  Science  241: 456; and Dervan et al., (1991)  Science  251: 1360) or to the mRNA itself (antisense—Okano, (1991)  J. Neurochem.  56: 560 ; Oligodeoxynucleotides as Antisense Inhibitors of Gene Expression , CRC Press, Boca Raton, Fla., USA (1988)). Triple helix formation optimally results in a shut-off of RNA transcription from DNA, while antisense RNA hybridization blocks translation of a mRNA molecule into polypeptide. Both techniques are effective in model systems, and the information disclosed herein can be used to design antisense or triple helix polynucleotides in an effort to treat or prevent disease. \\n     The present invention encompasses the addition of a nuclear localization signal, operably linked to the 5′ end, 3′ end, or any location therein, to any of the oligonucleotides, antisense oligonucleotides, triple helix oligonucleotides, ribozymes, PNA oligonucleotides, and/or polynucleotides, of the present invention (see, for example, Cutrona et al., (2000)  Nat. Biotechnol.  18:300–303). \\n     Polynucleotides of the present invention are also useful in gene therapy. One goal of gene therapy is to insert a normal gene into an organism having a defective gene, in an effort to correct the genetic defect. The polynucleotides disclosed in the present invention offer a means of targeting such genetic defects in a highly accurate manner. Another goal is to insert a new gene that was not present in the host genome, thereby producing a new trait in the host cell. In one example, polynucleotide sequences of the present invention can be used to construct chimeric RNA/DNA oligonucleotides corresponding to said sequences, specifically designed to induce host cell mismatch repair mechanisms in an organism upon systemic injection, for example (Bartlett et al., (2000)  Nat. Biotechnol.  18:615–622). Such RNA/DNA oligonucleotides could be designed to correct genetic defects in certain host strains, and/or to introduce desired phenotypes in the host (e.g., introduction of a specific polymorphism within an endogenous gene corresponding to a polynucleotide of the present invention that may ameliorate and/or prevent a disease symptom and/or disorder, etc.). Alternatively, the polynucleotide sequence of the present invention can be used to construct duplex oligonucleotides corresponding to the sequence, specifically designed to correct genetic defects in certain host strains, and/or to introduce desired phenotypes into the host (e.g., introduction of a specific polymorphism within an endogenous gene corresponding to a polynucleotide of the present invention that may ameliorate and/or prevent a disease symptom and/or disorder, etc). Such methods of using duplex oligonucleotides are known in the art and are encompassed by the present invention (see, e.g., EP 1007712). \\n     The polynucleotides are also useful for identifying organisms from minute biological samples. The United States military, for example, is considering the use of restriction fragment length polymorphism (RFLP) for identification of its personnel. In this technique, an individual&#39;s genomic DNA is digested with one or more restriction enzymes, and probed on a Southern blot to yield unique bands for identifying personnel. This method does not suffer from the current limitations of “Dog Tags” which can be lost, switched, or stolen, making positive identification difficult. The polynucleotides of the present invention can be used as additional DNA markers for RFLP. \\n     The polynucleotides of the present invention can also be used as an alternative to RFLP, by determining the actual base-by-base DNA sequence of selected portions of an organism&#39;s genome. These sequences can be used to prepare PCR primers for amplifying and isolating such selected DNA, which can then be sequenced. Using this technique, organisms can be identified because each organism will have a unique set of DNA sequences. Once a unique ID database is established for an organism, positive identification of that organism, living or dead, can be made from extremely small tissue samples. Similarly, polynucleotides of the present invention can be used as polymorphic markers, in addition to, the identification of transformed or non-transformed cells and/or tissues. \\n     There is also a need for reagents capable of identifying the source of a particular tissue. Such need arises, for example, when presented with tissue of unknown origin. Appropriate reagents can comprise, for example, DNA probes or primers specific to particular tissue prepared from the sequences of the present invention. Panels of such reagents can identify tissue by species and/or by organ type. In a similar fashion, these reagents can be used to screen tissue cultures for contamination. Moreover, as mentioned above, such reagents can be used to screen and/or identify transformed and non-transformed cells and/or tissues. \\n     Further, the polynucleotides of the present invention can be used as molecular weight markers on Southern gels, as diagnostic probes for the presence of a specific mRNA in a particular cell type, as a probe to “subtract-out” known sequences in the process of discovering novel polynucleotides, for selecting and making oligomers for attachment to a “gene chip” or other support, to raise anti-DNA antibodies using DNA immunization techniques, and as an antigen to elicit an immune response. \\n     Representative Applications for the Polypeptides of the Present Invention \\n     Each of the polypeptides identified herein can be used in numerous ways. The following description highlights only a few of the possible applications for the polynucleotides of the present invention. The described applicaitons all employ known techniques in conjunction with the polynucleotides of the present invention. \\n     A polypeptide of the present invention can be used to assay protein levels in a biological sample using antibody-based techniques. For example, protein expression in tissues can be studied with classical immunohistological methods (Jalkanen et al., (1985)  J. Cell. Biol.  101:976–985; Jalkanen et al., (1987)  J. Cell. Biol.  105:3087–3096). Other antibody-based methods useful for detecting protein gene expression include immunoassays, such as the enzyme linked immunosorbent assay (ELISA) and the radioimmunoassay (RIA). Suitable antibody assay labels are known in the art and include enzyme labels, such as, glucose oxidase, and radioisotopes, such as iodine ( 125 I, 121I), carbon ( 14 C), sulfur ( 35 S), tritium ( 3 H), indium ( 112 In), and technetium ( 99 Tc), and fluorescent labels, such as fluorescein and rhodamine, and biotin. \\n     In addition to assaying protein levels in a biological sample, proteins can also be detected in vivo by imaging. Antibody labels or markers for in vivo imaging of protein include those detectable by X-radiography, NMR or ESR. For X-radiography, suitable labels include radioisotopes such as barium or cesium, which emit detectable radiation but are not overtly harmful to the subject. Suitable markers for NMR and ESR include those with a detectable characteristic spin, such as deuterium, which can be incorporated into the antibody by labeling of nutrients for the relevant hybridoma. \\n     A protein-specific antibody or antibody fragment which has been labeled with an appropriate detectable imaging moiety, such as a radioisotope (for example,  131 I,  112 In,  99 Tc), a radio-opaque substance, or a material detectable by nuclear magnetic resonance, is introduced (for example, parenterally, subcutaneously, or intraperitoneally) into the subject. It will be understood in the art that the size of the subject and the imaging system used will determine the quantity of imaging moiety needed to produce diagnostic images. In the case of a radioisotope moiety, for a human subject, the quantity of radioactivity injected will normally range from about 5 to 20 millicuries of  99 Tc. The labeled antibody or antibody fragment will then preferentially accumulate at the location of cells which contain the specific protein. In vivo tumor imaging is described in  Tumor Imaging: The Radiochemical Detection of Cancer , (Burchiel and Rhodes, eds.), Masson Publishing Inc., New York, N.Y., USA (1982), Chapter 13) \\n     Thus, the present invention provides a diagnostic method of a disorder, comprising: (a) assaying the expression of a polypeptide of the present invention in cells or body fluid of an individual; and (b) comparing the level of gene expression with a standard gene expression level, whereby an increase or decrease in the assayed polypeptide gene expression level compared to the standard expression level is indicative of a disorder. With respect to cancer, the presence of a relatively high amount of transcript in biopsied tissue from an individual can indicate a predisposition for the development of the disease, or may provide a means for detecting the disease prior to the appearance of actual clinical symptoms. A more definitive diagnosis of this type may allow health professionals to employ preventative measures or aggressive treatment earlier thereby preventing the development or further progression of the cancer. \\n     Moreover, polypeptides of the present invention can be used to treat, prevent, and/or diagnose disease. For example, subjects can be administered a polypeptide of the present invention in an effort to replace absent or decreased levels of the polypeptide (e.g., insulin), to supplement absent or decreased levels of a different polypeptide (e.g., hemoglobin S for hemoglobin B, SOD, catalase, DNA repair proteins), to inhibit the activity of a polypeptide (e.g., an oncogene or tumor suppressor), to activate the activity of a polypeptide (e.g., by binding to a receptor), to reduce the activity of a membrane bound receptor by competing with it for free ligand (e.g., soluble TNF receptors used in reducing inflammation), or to bring about a desired response (e.g., blood vessel growth inhibition, enhancement of the immune response to proliferative cells or tissues). \\n     Similarly, antibodies directed to a polypeptide of the present invention can also be used to treat, prevent, and/or diagnose disease. For example, administration of an antibody directed to a polypeptide of the present invention can bind and reduce overproduction of the polypeptide. Similarly, administration of an antibody can activate the polypeptide, such as by binding to a polypeptide bound to a membrane (receptor). \\n     Additionally, the polypeptides of the present invention can be used as molecular weight markers on SDS-PAGE gels or on molecular sieve gel filtration columns using methods well known to those of skill in the art. Polypeptides can also be used to raise antibodies, which in turn are used to measure protein expression from a recombinant cell, as a way of assessing transformation of the host cell. Moreover, the polypeptides of the present invention can be used to test the following biological activities. \\n     Gene Therapy Methods \\n     Another aspect of the present invention is to gene therapy methods for treating or preventing disorders, diseases and conditions. The gene therapy methods relate to the introduction of nucleic acid (DNA, RNA and antisense DNA or RNA) sequences into an animal to achieve expression of a polypeptide of the present invention. This method requires a polynucleotide which codes for a polypeptide of the present invention that operatively linked to a promoter and any other genetic elements necessary for the expression of the polypeptide by the target tissue. Such gene therapy and delivery techniques are known in the art (see, for example, PCT Publication WO 90/11092). \\n     Thus, for example, cells from a subject may be engineered with a polynucleotide (DNA or RNA) comprising a promoter operably linked to a polynucleotide of the present invention ex vivo, with the engineered cells then being provided to a subject to be treated with the polypeptide. Such methods are known in the art (see, e.g., Belldegrun et al., (1993)  J. Natl. Cancer Inst.  85:207–216; Ferrantini et al., (1993)  Cancer Res.  53:107–1112; Ferrantini et al., (1994)  J. Immunol.  153: 4604–4615; Kaido et al., (1995)  Int. J. Cancer  60: 221–229; Ogura et al., (1990)  Cancer Res.  50:5102–5106; Santodonato et al., (1996)  Human Gene Therapy  7:1–10; Santodonato et al., (1997)  Gene Therapy  4:1246–1255; and Zhang et al., (1996)  Cancer Gene Therapy  3:31–38). In one embodiment, the cells that are engineered are arterial cells. The arterial cells can be reintroduced into the subject through direct injection to the artery, the tissues surrounding the artery, or through catheter injection. \\n     As discussed in more detail herein, a polynucleotide construct can be delivered by any method that delivers injectable materials to the cells of an animal, such as, injection into the interstitial space of tissues (heart, muscle, skin, lung, liver, and the like). The polynucleotide constructs can be delivered in a pharmaceutically acceptable liquid or aqueous carrier. \\n     In one embodiment, a polynucleotide of the present invention is delivered as a naked polynucleotide. The term “naked” polynucleotide, DNA or RNA refers to sequences that are free from any delivery vehicle that acts to assist, promote or facilitate entry into the cell, including viral sequences, viral particles, liposome formulations, lipofectin or precipitating agents and the like. However, the polynucleotides of the present invention can also be delivered in liposome formulations and lipofectin formulations and the like can be prepared by methods well known to those skilled in the art. Such methods are described, for example, in U.S. Pat. Nos. 5,593,972, 5,589,466, and 5,580,859. \\n     The polynucleotide vector constructs of the present invention used in the gene therapy method can comprise constructs that will not integrate into the host genome nor will they contain sequences that allow for replication. Representative vectors include pWLNEO, pSV2CAT, pOG44, pXT1 and pSG available from Stratagene (La Jolla, Calif., USA); pSVK3, pBPV, pMSG and pSVL available from Pharmacia (Peapack, N.J., USA); and pEF1/V5, pcDNA3.1, and pRc/CMV2 available from Invitrogen (Carlsbad, Calif., USA). Other suitable vectors will be readily apparent to those of ordinary skill in the art upon consideration of the present disclosure. \\n     Any strong promoter known to those skilled in the art can be used for driving the expression of polynucleotide sequence of the present invention. Representative promoters include adenoviral promoters, such as the adenoviral major late promoter; or heterologous promoters, such as the cytomegalovirus (CMV) promoter; the respiratory syncytial virus (RSV) promoter; inducible promoters, such as the MMT promoter, the metallothionein promoter; heat shock promoters; the albumin promoter; the ApoAI promoter; human globin promoters; viral thymidine kinase promoters, such as the Herpes Simplex thymidine kinase promoter; retroviral LTRs; the b-actin promoter; and human growth hormone promoters. The promoter also can be the native promoter for the polynucleotides of the present invention. \\n     Unlike other gene therapy techniques, one major advantage of introducing naked nucleic acid sequences into target cells is the transitory nature of the polynucleotide synthesis in the cells. Studies have shown that non-replicating DNA sequences can be introduced into cells to provide production of the desired polypeptide for periods of up to six months. \\n     The polynucleotide construct of the present invention can be delivered to the interstitial space of tissues within the an animal, including of muscle, skin, brain, lung, liver, spleen, bone marrow, thymus, heart, lymph, blood, bone, cartilage, pancreas, kidney, gall bladder, stomach, intestine, testis, ovary, uterus, rectum, nervous system, eye, gland, and connective tissue. Interstitial space of the tissues comprises the intercellular, fluid, mucopolysaccharide matrix among the reticular fibers of organ tissues, elastic fibers in the walls of vessels or chambers, collagen fibers of fibrous tissues, or that same matrix within connective tissue ensheathing muscle cells or in the lacunae of bone. It is similarly the space occupied by the plasma of the circulation and the lymph fluid of the lymphatic channels. Delivery to the interstitial space of muscle tissue is often, but not always, preferred for the reasons discussed below. Polynucleotides can be conveniently delivered by injection into the tissues comprising these cells. They can be delivered to and expressed in persistent, non-dividing cells which are differentiated, although delivery and expression may be achieved in non-differentiated or less completely differentiated cells, such as, for example, stem cells of blood or skin fibroblasts. In vivo muscle cells are particularly competent in their ability to take up and express polynucleotides. \\n     For the naked nucleic acid sequence injection, an effective dosage amount of DNA or RNA will be in the range of from about 0.05 mg/kg body weight to about 50 mg/kg body weight. In some situations, the dosage will be from about 0.005 mg/kg to about 20 mg/kg and in other situatoins from about 0.05 mg/kg to about 5 mg/kg. Of course, as the artisan of ordinary skill will appreciate, this dosage will vary according to the tissue site of injection. The appropriate and effective dosage of nucleic acid sequence can readily be determined by those of ordinary skill in the art and can depend on the condition being treated and the route of administration. \\n     Often, a preferred route of administration is by the parenteral route of injection into the interstitial space of tissues. However, other parenteral routes can also be used, such as, inhalation of an aerosol formulation particularly for delivery to lungs or bronchial tissues, throat or mucous membranes of the nose. In addition, naked DNA constructs can be delivered to arteries during angioplasty by the catheter used in the procedure. \\n     The naked polynucleotides can be delivered by any method known in the art, including, but not limited to, direct needle injection at the delivery site, intravenous injection, topical administration, catheter infusion, and so-called “gene guns”. These delivery methods are known in the art. \\n     The constructs may also be delivered with delivery vehicles such as viral sequences, viral particles, liposome formulations, lipofectin, precipitating agents, etc. Such methods of delivery are known in the art. \\n     In certain embodiments, a polynucleotide construct of the present invention is complexed in a liposome preparation. Representative liposomal preparations for use in the instant invention include cationic (positively charged), anionic (negatively charged) and neutral preparations. However, cationic liposomes are often preferred because a tight charge complex can be formed between the cationic liposome and the polyanionic nucleic acid. Cationic liposomes have been shown to mediate intracellular delivery of plasmid DNA (Felgner et al., (1987)  Proc. Natl. Acad. Sci. U.S.A.  84:7413–7416); mRNA (Malone et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86:6077–6081); and purified transcription factors (Debs et al., (1990)  J. Biol. Chem.  265:10189–10192) in functional form. \\n     Cationic liposomes are readily available. For example, N[1-2,3-dioleyloxy)propyl]-N,N,N-triethylammonium (DOTMA) liposomes are particularly useful and are available under the trademark LIPOFECTIN®, from GIBCO BRL, Grand Island, N.Y., USA (see, also, Felgner et al., (1987)  Proc. Natl. Acad. Sci. USA  84:7413–7416). Other commercially available liposomes include transfectace (DDAB/DOPE) and DOTAP/DOPE (Boehringer). \\n     Other cationic liposomes can be prepared from readily available materials using techniques known in the art (see, e.g. PCT Publication WO 90/11092 for a description of the synthesis of DOTAP (1,2-bis(oleoyloxy)-3-(trimethylammonio)propane) liposomes). Preparation of DOTMA liposomes is also explained in the literature (see, e.g., Felgner et al., (1987)  Proc. Natl. Acad. Sci. USA  84:7413–7416). Similar methods can be used to prepare liposomes from other cationic lipid materials. \\n     Similarly, anionic and neutral liposomes are readily available, such as from Avanti Polar Lipids (Birmingham, Ala., USA), or can be easily prepared using readily available materials. Such materials include phosphatidyl, choline, cholesterol, phosphatidyl ethanolamine, dioleoylphosphatidyl choline (DOPC), dioleoylphosphatidyl glycerol (DOPG), dioleoylphoshatidyl ethanolamine (DOPE), among others. These materials can also be mixed with the DOTMA and DOTAP starting materials in appropriate ratios. Methods for making liposomes using these materials are known in the art. \\n     For example, commercially dioleoylphosphatidyl choline (DOPC), dioleoylphosphatidyl glycerol (DOPG), and dioleoylphosphatidyl ethanolamine (DOPE) can be used in various combinations to make conventional liposomes, with or without the addition of cholesterol. Thus, for example, DOPG/DOPC vesicles can be prepared by drying 50 mg each of DOPG and DOPC under a stream of nitrogen gas into a sonication vial. The sample is placed under a vacuum pump overnight and is hydrated the following day with deionized water. The sample is then sonicated for 2 hours in a capped vial, (e.g., using a Heat Systems (Plainsview, N.Y., USA) Model 350 sonicator equipped with an inverted cup (bath type) probe at the maximum setting) while the bath is circulated at 15° C. Alternatively, negatively charged vesicles can be prepared without sonication to produce multilamellar vesicles or by extrusion through nucleopore membranes to produce unilamellar vesicles of discrete size. Other methods are known and available to those of ordinary skill in the art. \\n     The liposomes can comprise multilamellar vesicles (MLVs), small unilamellar vesicles (SUVs), or large unilamellar vesicles (LUVs), with SUVs being preferred. The various liposome-nucleic acid complexes are prepared using methods known in the art (see, e.g., Straubinger et al., (1983)  Method. Immunol.  101:512–527). For example, MLVs containing nucleic acid can be prepared by depositing a thin film of phospholipid on the walls of a glass tube and subsequently hydrating with a solution of the material to be encapsulated. SUVs are prepared by extended sonication of MLVs to produce a homogeneous population of unilamellar liposomes. The material to be entrapped is added to a suspension of preformed MLVs and then sonicated. When using liposomes containing cationic lipids, the dried lipid film is resuspended in an appropriate solution such as sterile water or an isotonic buffer solution such as 10 mM Tris/NaCl, sonicated, and then the preformed liposomes are mixed directly with the DNA. The liposome and DNA form a very stable complex due to binding of the positively charged liposomes to the cationic DNA. SUVs find use with small nucleic acid fragments. LUVs are prepared by a number of methods, well known in the art. Commonly used methods include Ca 2+ -EDTA chelation (Papahadjopoulos et al., (1975)  Biochim. Biophys. Acta  394:483; Wilson et al., (1979)  Cell  17:77); ether injection (Deamer et al., (1976)  Biochim. Biophys. Acta  443:629; Ostro et al., (1977)  Biochem. Biophys. Res. Commun.  76:836; Fraley et al., (1979)  Proc. Natl. Acad. Sci. USA  76:3348); detergent dialysis (Enoch et al., (1979)  Proc. Natl. Acad. Sci. USA  76:145); and reverse-phase evaporation (REV) (Fraley et al., (1980)  J. Biol. Chem.  255:10431; Szoka et al., (1978)  Proc. Natl. Acad. Sci. USA  75:145; Schaefer-Ridder et al., (1982)  Science  215:166). \\n     In one embodiment, the ratio of DNA to liposomes can be from about 10:1 to about 1:10. In other embodiments, the ratio can be from about 5:1 to about 1:5. In yet other embodiments, the ratio can be from about 3:1 to about 1:3. In still further embodiments, the ratio can be about 1:1. \\n     U.S. Pat. No. 5,676,954 reports on the injection of genetic material, complexed with cationic liposomes carriers, into mice. U.S. Pat. Nos. 4,897,355, 4,946,787, 5,049,386, 5,459,127, 5,589,466, 5,693,622, 5,580,859, 5,703,055, and PCT Publication WO 94/9469 provide cationic lipids for use in transfecting DNA into cells and mammals. U.S. Pat. Nos. 5,589,466, 5,693,622, 5,580,859, 5,703,055, and PCT Publication WO 94/9469 provide methods for delivering DNA-cationic lipid complexes to mammals. \\n     In certain embodiments, cells are engineered, ex vivo or in vivo, using a retroviral particle containing RNA that comprises a sequence encoding polypeptides of the present invention. Retroviruses from which the retroviral plasmid vectors may be derived include, but are not limited to, Moloney Murine Leukemia Virus, spleen necrosis virus, Rous sarcoma Virus, Harvey Sarcoma Virus, avian leukosis virus, gibbon ape leukemia virus, human immunodeficiency virus, Myeloproliferative Sarcoma Virus, and mammary tumor virus. \\n     A retroviral plasmid vector can be employed to transduce packaging cell lines to form producer cell lines. Examples of packaging cells that can be transfected include, but are not limited to, the PE501, PA317, R-2, R-AM, PA12, T19-14X, VT-19-17-H2, RCRE, RCRIP, GP+E-86, GP+envAm12, and DAN cell lines as described in Miller, (1990)  Human Gene Therapy,  1:5–14. A vector can transduce the packaging cells through any means known in the art. Such means include, but are not limited to, electroporation, the use of liposomes, and CaPO 4  precipitation. In one alternative, the retroviral plasmid vector can be encapsulated into a liposome, or coupled to a lipid, and then administered to a host. \\n     The producer cell line generates infectious retroviral vector particles, including polynucleotides encoding polypeptides of the present invention. Such retroviral vector particles can then be employed, to transduce eukaryotic cells, either in vitro or in vivo. The transduced eukaryotic cells will then express the polypeptides of the present invention. \\n     In certain other embodiments, cells are engineered, ex vivo or in vivo, with polynucleotides of the present invention contained in an adenovirus vector. Adenovirus can be manipulated such that it encodes and expresses polypeptides of the present invention, and at the same time is inactivated in terms of its ability to replicate in a normal lytic viral life cycle. Adenovirus expression is achieved without integration of the viral DNA into the host cell chromosome, thereby alleviating concerns about insertional mutagenesis. Furthermore, adenoviruses have been used as live enteric vaccines for many years with an excellent safety profile (Schwartz et al., (1974)  Am. Rev. Respir. Dis.  109:233–238). Additionally, adenovirus mediated gene transfer has been demonstrated in a number of instances including transfer of alpha-1-antitrypsin and CFTR to the lungs of cotton rats (Rosenfeld et al., (1991)  Science  252:431–434; Rosenfeld et al., (1992)  Cell  68:143–155). Furthermore, extensive studies to attempt to establish adenovirus as a causative agent in human cancer were uniformly negative (Green et al., (1979)  Proc. Natl. Acad. Sci. U.S.A.  76:6606). \\n     Suitable adenoviral vectors useful in the present invention are known (see, e.g., Kozarsky &amp; Wilson, (1993)  Curr. Opin. Genet. Devel.  3:499–503; Rosenfeld et al., (1992)  Cell  68:143–155; Engelhardt et al., (1993)  Human Genet. Ther.  4:759–769; Yang et al., (1994)  Nature Genet.  7:362–369; Wilson et al., (1993)  Nature  365:691–692; and U.S. Pat. No. 5,652,224). For example, the adenovirus vector Ad2 is useful and can be grown in human embryonic kidney 293 (HEK293) cells. These cells contain the E1 region of adenovirus and constitutively express E1a and E1b, which complement the defective adenoviruses by providing the products of the genes deleted from the vector. In addition to Ad2, other varieties of adenovirus (e.g., Ad3, Ad5, and Ad7) are also useful in the present invention. \\n     Adenoviruses used in the present invention can be replication deficient. Replication deficient adenoviruses require the aid of a helper virus and/or packaging cell line to form infectious particles. The resulting virus is capable of infecting cells and can express a polynucleotide of interest that is operably linked to a promoter, but cannot replicate in most cells. Replication deficient adenoviruses can be deleted in one or more of all or a portion of the following genes: E1a, E1b, E3, E4, E2a, or L1 through L5. \\n     In certain other embodiments, the cells are engineered, ex vivo or in vivo, using an adeno-associated virus (AAV). AAVs are naturally occurring defective viruses that require helper viruses to produce infectious particles (Muzyczka, (1992)  Curr. Topics in Microbiol. Immunol.  158:97). It is also one of the few viruses that may integrate its DNA into non-dividing cells. Vectors containing as little as about 300 base pairs of AAV can be packaged and can integrate, but space for exogenous DNA is limited to about 4.5 kb. Methods for producing and using such AAVs are known in the art (see, for example, U.S. Pat. Nos. 5,139,941, 5,173,414, 5,354,678, 5,436,146, 5,474,935, 5,478,745, and 5,589,377). \\n     In one example, an appropriate AAV vector for use in the present invention can include all the sequences necessary for DNA replication, encapsidation, and host-cell integration. A polynucleotide construct containing polynucleotides of the present invention is inserted into the AAV vector using standard cloning methods (see, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3 rd  ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). The recombinant AAV vector is then transfected into packaging cells which are infected with a helper virus, using any standard technique, including lipofection, electroporation, calcium phosphate precipitation, etc. Appropriate helper viruses include adenoviruses, cytomegaloviruses, vaccinia viruses, or herpes viruses. Once the packaging cells are transfected and infected, they will produce infectious AAV viral particles which contain a polynucleotide construct of the present invention. These viral particles are then used to transduce eukaryotic cells, either ex vivo or in vivo. The transduced cells will contain the polynucleotide construct integrated into its genome, and will express the desired gene product. \\n     Another method of gene therapy involves operably associating heterologous control regions and endogenous polynucleotide sequences (e.g. encoding the polypeptide sequence of interest) via homologous recombination (see, e.g., U.S. Pat. No. 5,641,670; PCT Publication WO 96/29411; PCT Publication WO 94/12650; Koller et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86:8932–8935; and Zijlstra et al., (1989)  Nature  342:435–438). This method involves the activation of a gene that is present in the target cells, but is not normally expressed in the cells, or is expressed at a lower level than desired. \\n     Polynucleotide constructs are made, using standard techniques known in the art, that contain the promoter with targeting sequences flanking the promoter. Suitable promoters are described herein. The targeting sequence is sufficiently complementary to an endogenous sequence to permit homologous recombination of the promoter-targeting sequence with the endogenous sequence. The targeting sequence will be sufficiently near the 5′ end of the desired endogenous polynucleotide sequence so the promoter will be operably linked to the endogenous sequence upon homologous recombination. \\n     The promoter and the targeting sequences can be amplified using PCR. In one embodiment, the amplified promoter comprises distinct restriction enzyme sites on the 5′ and 3′ ends. For example, the 3′ end of the first targeting sequence contains the same restriction enzyme site as the 5′ end of the amplified promoter and the 5′ end of the second targeting sequence contains the same restriction site as the 3′ end of the amplified promoter. The amplified promoter and targeting sequences are digested and ligated together. \\n     The promoter-targeting sequence construct is delivered to the cells, either as naked polynucleotide, or in conjunction with transfection-facilitating agents, such as liposomes, viral sequences, viral particles, whole viruses, lipofection, precipitating agents, etc., described in more detail above. The P promoter-targeting sequence can be delivered by any method, included direct needle injection, intravenous injection, topical administration, catheter infusion, particle accelerators, etc. These methods are described herein. \\n     The promoter-targeting sequence construct is taken up by cells. Homologous recombination between the construct and the endogenous sequence takes place, such that an endogenous sequence is placed under the control of the promoter. The promoter then drives the expression of the endogenous sequence. \\n     Polynucleotides encoding polypeptides of the present invention can be administered along with other polynucleotides encoding angiogenic proteins. Representative angiogenic proteins include, but are not limited to, acidic and basic fibroblast growth factors, VEGF-1, VEGF-2 (VEGF-C), VEGF-3 (VEGF-B), epidermal growth factor alpha and beta, platelet-derived endothelial cell growth factor, platelet-derived growth factor, tumor necrosis factor alpha, hepatocyte growth factor, insulin like growth factor, colony stimulating factor, macrophage colony stimulating factor, granulocyte/macrophage colony stimulating factor, and nitric oxide synthase. \\n     A polynucleotide encoding a polypeptide of the present invention can contain a secretory signal sequence that facilitates secretion of the protein. Often, the signal sequence is positioned in the coding region of the polynucleotide to be expressed towards or at the 5′ end of the coding region. The signal sequence can be homologous or heterologous to the polynucleotide of interest and may be homologous or heterologous to the cells to be transfected. Additionally, the signal sequence can be chemically synthesized using methods known in the art. \\n     Any mode of administration of any of the above-described polynucleotides constructs can be used so long as the mode results in the expression of one or more molecules in an amount sufficient to provide a therapeutic effect. This includes direct needle injection, systemic injection, catheter infusion, biolistic injectors, particle accelerators (i.e., “gene guns”), gelfoam sponge depots, other commercially available depot materials, osmotic pumps (e.g., Alza minipumps), oral or suppositorial solid (tablet or pill) pharmaceutical formulations, and decanting or topical applications during surgery. For example, direct injection of naked calcium phosphate-precipitated plasmid into rat liver and rat spleen or a protein-coated plasmid into the portal vein has resulted in gene expression of the foreign gene in the rat livers. (Kaneda et al., (1989)  Science  243:375). \\n     A representative method of local administration is by direct injection. For example, a recombinant molecule of the present invention complexed with a delivery vehicle can be administered by direct injection into or locally within the area of arteries. Administration of a composition “locally within the area of arteries” refers to injecting the composition centimeters and preferably, millimeters within arteries. \\n     Another method of local administration is to contact a polynucleotide construct of the present invention in or around a surgical wound. For example, a subject can undergo surgery and the polynucleotide construct can be coated on the surface of tissue inside the wound or the construct can be injected into areas of tissue inside the wound. \\n     Therapeutic compositions useful in systemic administration, can comprise recombinant molecules of the present invention complexed to a targeted delivery vehicle of the present invention. Suitable delivery vehicles for use with systemic administration comprise liposomes comprising ligands for targeting the vehicle to a particular site. \\n     Representative methods of systemic administration, include intravenous injection, aerosol, oral and percutaneous (topical) delivery. Intravenous injections can be performed using methods standard in the art. Aerosol delivery can also be performed using methods standard in the art (see, for example, Stribling et al., (1992)  Proc. Natl. Acad. Sci. U.S.A.  189:11277–11281). Oral delivery can be performed by complexing a polynucleotide construct of the present invention to a carrier capable of withstanding degradation by digestive enzymes in the gut of an animal. Examples of such carriers, include plastic capsules or tablets, such as those known in the art. Topical delivery can be performed by mixing a polynucleotide construct of the present invention with a lipophilic reagent (e.g., DMSO) that is capable of passing into the skin. \\n     Determining an effective amount of substance to be delivered can depend upon a number of factors including, for example, the chemical structure and biological activity of the substance, the age and weight of the animal, the precise condition requiring treatment and its severity, and the route of administration. The frequency of treatments depends upon a number of factors, such as the amount of polynucleotide constructs administered per dose, as well as the health and history of the subject. The precise amount, number of doses, and timing of doses will be determined by the attending physician or veterinarian. Therapeutic compositions of the present invention can be administered to any animal, preferably to mammals and birds. Representative mammals include humans, dogs, cats, rats, mice, rats, rabbits sheep, cattle, horses and pigs, with humans being particularly preferred. \\n     Testing for a Biological Activity \\n     The polynucleotides or polypeptides, or agonists or antagonists of the present invention can be used in assays to test for one or more biological activities. If these polynucleotides and polypeptides do exhibit activity in a particular assay, it is likely that these molecules may be involved in the diseases associated with the biological activity. Thus, the polynucleotides or polypeptides, or agonists or antagonists could be used to treat the associated disease. \\n     Immune Activity \\n     The polynucleotides or polypeptides, or agonists or antagonists of the present invention can be useful in treating, preventing, and/or diagnosing diseases, disorders, and/or conditions of the immune system, by activating or inhibiting the proliferation, differentiation, or mobilization (chemotaxis) of immune cells. Immune cells develop through a process called hematopoiesis, producing myeloid (platelets, red blood cells, neutrophils, and macrophages) and lymphoid (B and T lymphocytes) cells from pluripotent stem cells. The etiology of these immune diseases, disorders, and/or conditions can be genetic, somatic, such as cancer or some autoimmune diseases, disorders, and/or conditions, acquired (e.g., by chemotherapy or toxins), or infectious. Moreover, a polynucleotides or polypeptides, or agonists or antagonists of the present invention can be used as a marker or detector of a particular immune system disease or disorder. \\n     A polynucleotides or polypeptides, or agonists or antagonists of the present invention may be useful in treating, preventing, and/or diagnosing diseases, disorders, and/or conditions associated with NF-kB. NF-kB is a central regulator of inflammation, cell growth, differentiation, and survival. Therefore, a polynucleotide, polypeptide, agonist or antagonist of the present invention could be used to treat a condition characterized by aberrant cell growth, differentiation or survival, and inflammation, such as rheumatoid arthritis, osteoarthritis, inflammatory bowel disease, lupus, atherosclerosis, stroke, cancer. \\n     A polynucleotide, polypeptide, agonist or antagonist of the present invention could also be useful in treating, preventing, and/or diagnosing autoimmune diseases, disorders, and/or conditions. Many autoimmune diseases, disorders, and/or conditions result from inappropriate recognition of self as foreign material by immune cells. This inappropriate recognition results in an immune response leading to the destruction of the host tissue. Therefore, the administration of a polynucleotide, polypeptide, agonist or antagonist of the present invention that inhibits an immune response, and could be an effective therapy in preventing autoimmune diseases, disorders, and/or conditions. \\n     Examples of autoimmune diseases, disorders, and/or conditions that can be treated, prevented, and/or diagnosed or detected by the present invention include, but are not limited to: Addison&#39;s Disease, hemolytic anemia, antiphospholipid syndrome, rheumatoid arthritis, dermatitis, allergic encephalomyelitis, glomerulonephritis, Goodpasture&#39;s Syndrome, Graves&#39; Disease, Multiple Sclerosis, Myasthenia Gravis, Neuritis, Ophthalmia, Bullous Pemphigoid, Pemphigus, Polyendocrinopathies, Purpura, Reiter&#39;s Disease, Stiff-Man Syndrome, Autoimmune Thyroiditis, Systemic Lupus Erythematosus, Autoimmune Pulmonary Inflammation, Guillain-Barre Syndrome, insulin dependent diabetes mellitis, and autoimmune inflammatory eye disease. \\n     Similarly, allergic reactions and conditions, such as asthma (particularly allergic asthma) or other respiratory problems, may also be treated, prevented, and/or diagnosed by a polynucleotide, polypeptide, agonist or antagonist of the present invention. Moreover, these molecules may be used to treat anaphylaxis, hypersensitivity to an antigenic molecule, or blood group incompatibility. \\n     A polynucleotide, polypeptide, agonist or antagonist of the present invention might also be used to treat, prevent, and/or diagnose organ rejection or graft-versus-host disease (GVHD). Organ rejection occurs by host immune cell destruction of the transplanted tissue through an immune response. Similarly, an immune response is also involved in GVHD, but, in this case, the foreign transplanted immune cells destroy the host tissues. The administration of a polynucleotide, polypeptide, agonist or antagonist of the present invention that inhibits an immune response, particularly the proliferation, differentiation, or chemotaxis of T-cells, can be an effective therapy in preventing organ rejection or GVHD. \\n     Similarly, a polynucleotide, polypeptide, agonist or antagonist of the present invention might also be used to modulate inflammation. For example, the polynucleotide, polypeptide, agonist or antagonist might inhibit the proliferation and differentiation of cells involved in an inflammatory response. These molecules can be used to treat, prevent, and/or diagnose inflammatory conditions, both chronic and acute conditions, including chronic prostatitis, granulomatous prostatitis and malacoplakia, inflammation associated with infection (e.g., septic shock, sepsis, or systemic inflammatory response syndrome (SIRS)), ischemia-reperfusion injury, endotoxin lethality, arthritis, complement-mediated hyperacute rejection, nephritis, cytokine or chemokine induced lung injury, inflammatory bowel disease, Crohn&#39;s disease, or resulting from over production of cytokines (e.g., TNF or IL-1.) \\n     Hyperproliferative Disorders \\n     A polynucleotide, polypeptide, agonist or antagonist of the present invention can be used to treat, prevent, and/or diagnose hyperproliferative diseases, disorders, and/or conditions, including neoplasms. A polynucleotide, polypeptide, agonist or antagonist of the present invention can inhibit the proliferation of the disorder through direct or indirect interactions. Alternatively, a polynucleotides or polypeptides, or agonists or antagonists of the present invention can proliferate other cells that can inhibit the hyperproliferative disorder. \\n     For example, by increasing an immune response, particularly increasing antigenic qualities of the hyperproliferative disorder or by proliferating, differentiating, or mobilizing T-cells, hyperproliferative diseases, disorders, and/or conditions can be treated, prevented, and/or diagnosed. This immune response can be increased by either enhancing an existing immune response, or by initiating a new immune response. Alternatively, decreasing an immune response can also be a method of treating, preventing, and/or diagnosing hyperproliferative diseases, disorders, and/or conditions, such as a chemotherapeutic agent. \\n     Examples of hyperproliferative diseases, disorders, and/or conditions that can be treated, prevented, and/or diagnosed by a polynucleotide, polypeptide, agonist or antagonist of the present invention include, but are not limited to neoplasms located in the: colon, abdomen, bone, breast, digestive system, liver, pancreas, peritoneum, endocrine glands (adrenal, parathyroid, pituitary, testicles, ovary, thymus, thyroid), eye, head and neck, nervous (central and peripheral), lymphatic system, pelvic, skin, soft tissue, spleen, thoracic, and urogenital. \\n     Similarly, other hyperproliferative diseases, disorders, and/or conditions can also be treated, prevented, and/or diagnosed by a polynucleotide, polypeptide, agonist or antagonist of the present invention. Examples of such hyperproliferative diseases, disorders, and/or conditions include, but are not limited to: hypergammaglobulinemia, lymphoproliferative diseases, disorders, and/or conditions, paraproteinemias, purpura, sarcoidosis, Sezary Syndrome, Waldenstron&#39;s Macroglobulinemia, Gaucher&#39;s Disease, histiocytosis, and any other hyperproliferative disease, besides neoplasia, located in an organ system listed above. \\n     One embodiment of the present invention utilizes polynucleotides of the present invention to inhibit aberrant cellular division, by gene therapy using the present invention, and/or protein fusions or fragments thereof. \\n     Thus, the present invention provides a method for treating or preventing cell proliferative diseases, disorders, and/or conditions by inserting into an abnormally proliferating cell a polynucleotide of the present invention, wherein said polynucleotide represses said expression. \\n     Another embodiment of the present invention provides a method of treating or preventing cell-proliferative diseases, disorders, and/or conditions in individuals comprising administration of one or more active gene copies of the present invention to an abnormally proliferating cell or cells. In a representative embodiment, a polynucleotide of the present invention is a DNA construct comprising a recombinant expression vector effective in expressing a DNA sequence encoding the polynucleotide. In another embodiment of the present invention, a DNA construct encoding the polynucleotides of the present invention is inserted into cells to be treated utilizing a retrovirus, or an adenoviral vector (see Nabel et. al., (1999)  Proc. Natl. Acad. Sci. U.S.A.  96: 324–326). In yet another embodiment, the viral vector is defective and will not transform non-proliferating cells, only proliferating cells. Moreover, a polynucleotide of the present invention inserted into proliferating cells either alone, or in combination with or fused to other polynucleotides, can then be modulated via an external stimulus (i.e. magnetic, specific small molecule, chemical, or drug administration, etc.), that acts upon the promoter upstream of said polynucleotides to induce expression of the encoded protein product. As such, a beneficial therapeutic affect of the present invention can be expressly modulated (i.e. to increase, decrease, or inhibit expression of the present invention) based upon an external stimulus. \\n     Polynucleotides of the present invention may be useful in repressing expression of oncogenic genes or antigens. The term “repressing expression of the oncogenic genes” encompasses the suppression of the transcription of the gene, the degradation of the gene transcript (pre-message RNA), the inhibition of splicing, the destruction of the messenger RNA, the prevention of the post-translational modifications of the protein, the destruction of the protein or the inhibition of the normal function of the protein. \\n     For local administration to abnormally proliferating cells, a polynucleotide of the present invention can be administered by any method known to those of skill in the art including, but not limited to transfection, electroporation, microinjection of cells, or in vehicles such as liposomes, lipofectin, or as naked polynucleotides, or any other method described herein. A polynucleotide of the present invention can be delivered by known gene delivery systems such as, but not limited to, retroviral vectors (Gilboa, (1982)  J. Virology  44:845; Hocke, (1986)  Nature  320:275; Wilson et al.,  Proc. Natl. Acad. Sci. U.S.A.  85:3014), vaccinia virus system (Chakrabarty et al., (1985)  Mol. Cell Biol.  5:3403) or other efficient DNA delivery systems (see, e.g., Yates et al., (1985)  Nature  313:812) known to those of ordinary skill in the art. In order to specifically deliver or transfect cells which are abnormally proliferating and spare non-dividing cells, it is often preferable to utilize a retrovirus, or adenoviral (as described in the art and herein) delivery system known to those of ordinary skill in the art. Since host DNA replication is required for retroviral DNA to integrate and the retrovirus will be unable to self-replicate due to the lack of the retrovirus genes needed for its life cycle. Utilizing such a retroviral delivery system for polynucleotides of the present invention will target the gene and constructs to abnormally proliferating cells, and will spare the non-dividing normal cells. \\n     A polynucleotide of the present invention can be delivered directly to cell proliferative disorder/disease sites in internal organs, body cavities and the like by use of imaging devices used to guide an injecting needle directly to the disease site. A polynucleotide of the present invention can also be administered to disease sites at the time of surgical intervention. \\n     By “cell proliferative disease” is meant any human or animal disease or disorder, affecting any one or any combination of organs, cavities, or body parts, that is characterized by single or multiple local abnormal proliferations of cells, groups of cells, or tissues, whether benign or malignant. \\n     Any amount of a polynucleotide of the present invention can be administered as long as it has a biologically inhibiting effect on the proliferation of the treated cells. Moreover, it is possible to administer more than one polynucleotide of the present invention simultaneously to the same site. By “biologically inhibiting” is meant partial or total growth inhibition as well as decreases in the rate of proliferation or growth of the cells. A biologically inhibitory dose can be determined by assessing the effects of a polynucleotide of the present invention on target malignant or abnormally proliferating cell growth in tissue culture, tumor growth in animals and cell cultures, or any other method known to one of ordinary skill in the art. \\n     The present invention is further directed to antibody-based therapies which involve administering of anti-polypeptides and anti-polynucleotide antibodies to a mammalian, preferably human, subject for treating, preventing, and/or diagnosing one or more of the described diseases, disorders, and/or conditions. Methods for producing anti-polypeptides and anti-polynucleotide antibodies polyclonal and monoclonal antibodies are described in detail elsewhere herein. Such antibodies can be provided in pharmaceutically acceptable compositions as known in the art or as described herein. \\n     A summary of some of the ways in which the antibodies of the present invention can be used therapeutically includes binding polynucleotides or polypeptides of the present invention locally or systemically in the body or by direct cytotoxicity of the antibody, e.g. as mediated by complement (CDC) or by effector cells (ADCC). Some of these approaches are described in more detail herein. Upon consideration of the present disclosure, one of ordinary skill in the art will know how to use the antibodies of the present invention for diagnostic, monitoring or therapeutic purposes. \\n     In particular, the antibodies, fragments and derivatives of the present invention are useful for treating, preventing, and/or diagnosing a subject having or developing cell proliferative and/or differentiation diseases, disorders, and/or conditions as described herein. Such treatment comprises administering a single or multiple doses of the antibody, or a fragment, derivative, or a conjugate thereof. \\n     The antibodies of the present invention can be advantageously employed in combination with other monoclonal or chimeric antibodies, or with lymphokines or hematopoietic growth factors, for example, which serve to increase the number or activity of effector cells that interact with the antibodies. \\n     It can be desirable to use high affinity and/or potent in vivo inhibiting and/or neutralizing antibodies against polypeptides or polynucleotides of the present invention, or fragments or regions thereof, for both immunoassays directed to, and therapy of, diseases, disorders, and/or conditions related to polynucleotides or polypeptides, including fragments thereof, of the present invention. Such antibodies, fragments, or regions, can have an affinity for polynucleotides or polypeptides, including fragments thereof. As noted hereinabove, representative binding affinities include those with a dissociation constant or Kd less than about 5×10 −2  M, 10 −2  M, 5×10 −3  M, 10 −3  M, 5×10 −4  M, 10 −4  M, 5×10 −5  M, 1×10 −5  M, 5×10 −6  M, 1×10 −6  M, 5×10 −7 M, 1×10 −7 M, 5×10 −8  M, 1×10 −8 M, 5×10 −9  M, 1×10 −9  M, 5×10 −10  M, 1×10 −10  M, 5×10 −11 M, 1×10 −11  M, 5×10 −12  M, 10 −12  M, 5×10 −13  M, 1×10 −14  M, 5×10 −14 M, 1×10 −14  M, 5×10 −15 M, or 1×10 −15  M. \\n     In another embodiment, the present invention provides a method of delivering compositions containing a polypeptide of the present invention (e.g., compositions containing polypeptides or polypeptide antibodies associated with heterologous polypeptides, heterologous nucleic acids, toxins, or prodrugs) to targeted cells expressing a polypeptide of the present invention. A polypeptide or polypeptide antibody of the present invention can be associated with heterologous polypeptides, heterologous nucleic acids, toxins, or prodrugs via hydrophobic, hydrophilic, ionic and/or covalent interactions. \\n     Polypeptides, protein fusions to, or fragments thereof, of the present invention are useful in enhancing the immunogenicity and/or antigenicity of proliferating cells or tissues, either directly, such as would occur if the polypeptides of the present invention “vaccinated” the immune response to respond to proliferative antigens and immunogens, or indirectly, such as in activating the expression of proteins known to enhance the immune response (e.g. chemokines), to said antigens and immunogens. \\n     Chemotaxis \\n     A polypeptide, polynucleotide, agonist and/or antagonist of the present invention may have chemotaxis activity. A chemotaxic molecule attracts or mobilizes cells (e.g., monocytes, fibroblasts, neutrophils, T-cells, mast cells, eosinophils, epithelial and/or endothelial cells) to a particular site in the body, such as inflammation, infection, or site of hyperproliferation. The mobilized cells can then fight off and/or heal the particular trauma or abnormality. \\n     A polypeptide, polynucleotide, agonist and/or antagonist of the present invention may increase chemotaxic activity of particular cells. These chemotactic molecules can then be used to treat, prevent, and/or diagnose inflammation, infection, hyperproliferative diseases, disorders, and/or conditions, or any immune system disorder by increasing the number of cells targeted to a particular location in the body. For example, chemotaxic molecules can be used to treat, prevent, and/or diagnose wounds and other trauma to tissues by attracting immune cells to the injured location. Chemotactic molecules of the present invention can also attract fibroblasts, which can be used to treat, prevent, and/or diagnose wounds. \\n     It is also contemplated that a polypeptide, polynucleotide, agonist and/or antagonist of the present invention may inhibit chemotactic activity. These molecules could also be used to treat, prevent, and/or diagnose diseases, disorders, and/or conditions. Thus, a polypeptide, polynucleotide, agonist and/or antagonist of the present invention could be used as an inhibitor of chemotaxis. \\n     Binding Activity \\n     A polypeptide of the present invention may be used to screen for molecules that bind to the polypeptide, or for molecules to which the polypeptide binds. The binding of the polypeptide and the molecule could activate (agonist), increase, inhibit (antagonist), or decrease activity of the polypeptide or the molecule bound. Examples of such molecules include antibodies, oligonucleotides, proteins (e.g., receptors), or small molecules. \\n     In one example, the molecule is closely related to the natural ligand of the polypeptide, e.g., a fragment of the ligand, or a natural substrate, a ligand, a structural or functional mimetic (see, e.g., Coligan et al.,  Current Protocols in Immunology  Greene Publishing Associates and Wiley Interscope, New York, New or, USA 1(2):Chapter 5 (2001)). Similarly, the molecule can be closely related to the natural receptor to which the polypeptide binds, or at least, a fragment of the receptor capable of being bound by the polypeptide (e.g., active site). In either case, the molecule can be rationally designed using techniques known to those of ordinary skill in the art. \\n     In one example, the screening for these molecules involves producing appropriate cells that express the polypeptide, either as a secreted protein or on the cell membrane. Representative cells include cells from mammals, yeast,  Drosophila , or  E. coli . Cells expressing the polypeptide (or cell membrane containing the expressed polypeptide) are then contacted with a test compound potentially containing the molecule to observe binding, stimulation, or inhibition of activity of either the polypeptide or the molecule. \\n     The assay can simply test binding of a candidate compound to the polypeptide, wherein binding is detected by a label, or in an assay involving competition with a labeled competitor. Further, the assay can test whether the candidate compound results in a signal generated by binding to the polypeptide. \\n     Alternatively, the assay can be carried out using cell-free preparations, polypeptide/molecule affixed to a solid support, chemical libraries, or natural product mixtures. The assay can also simply comprise the steps of mixing a candidate compound with a solution containing a polypeptide, measuring polypeptide/molecule activity or binding, and comparing the polypeptide/molecule activity or binding to a standard. \\n     For example, an ELISA assay can measure polypeptide level or activity in a sample (e.g., biological sample) using a monoclonal or polyclonal antibody. The antibody can measure polypeptide level or activity by either binding, directly or indirectly, to the polypeptide or by competing with the polypeptide for a substrate. \\n     Additionally, a receptor to which a polypeptide of the present invention binds can be identified by numerous methods known to those of skill in the art, for example, ligand panning and FACS sorting (Coligan et al.,  Current Protocols in Immunology  Greene Publishing Associates and Wiley Interscope, New York, New or, USA 1(2), Chapter 5, (1991)). For example, expression cloning is employed wherein polyadenylated RNA is prepared from a cell responsive to the polypeptides and a cDNA library created from this RNA is divided into pools and used to transfect COS cells or other cells that are not responsive to the polypeptides. Transfected cells which are grown on glass slides are exposed to a polypeptide of the present invention, after they have been labeled. The polypeptides can be labeled by a variety of means including iodination or inclusion of a recognition site for a site-specific protein kinase. \\n     Following fixation and incubation, the slides are subjected to auto-radiographic analysis. Positive pools are identified and sub-pools are prepared and re-transfected using an iterative sub-pooling and re-screening process, eventually yielding a single clone that encodes the putative receptor. \\n     As an alternative approach for receptor identification, a labeled polypeptide can be photoaffinity linked with cell membrane or extract preparations that express the receptor molecule. Cross-linked material is resolved by PAGE analysis and exposed to X-ray film. The labeled complex containing the receptors of the polypeptides can be excised, resolved into peptide fragments, and subjected to protein microsequencing. The amino acid sequence obtained from microsequencing would be used to design a set of degenerate oligonucleotide probes to screen a cDNA library to identify the genes encoding the putative receptors. \\n     Moreover, the techniques of gene-shuffling, motif-shuffling, exon-shuffling, and/or codon-shuffling (collectively referred to as “DNA shuffling”) may be employed to modulate the activities of polypeptides of the present invention thereby effectively generating agonists and antagonists of polypeptides of the present invention (see generally, U.S. Pat. Nos. 5,605,793, 5,811,238, 5,830,721, 5,834,252, and 5,837,458, and Patten et al., (1997)  Curr. Opinion Biotechnol.  8:724–33; Harayama, (1998)  Trends Biotechnol.  16(2):76–82; Hansson et al., (1999)  J. Mol. Biol.  287:265–76 (1999); and Lorenzo &amp; Blasco, (1998)  Biotechniques  24(2):308–13). In one embodiment, alteration of polynucleotides and corresponding polypeptides of the present invention can be achieved by DNA shuffling. DNA shuffling involves the assembly of two or more DNA segments into a desired polynucleotide sequence of the present invention molecule by homologous, or site-specific, recombination. In another embodiment, polynucleotides and corresponding polypeptides of the present invention can be altered by being subjected to random mutagenesis by error-prone PCR, random nucleotide insertion or other methods prior to recombination. In another embodiment, one or more components, motifs, sections, parts, domains, fragments, etc., of the polypeptides of the present invention can be recombined with one or more components, motifs, sections, parts, domains, fragments, etc. of one or more heterologous molecules. In representative embodiments, the heterologous molecules are family members. In further preferred embodiments, the heterologous molecule is a growth factor such as, for example, platelet-derived growth factor (PDGF), insulin-like growth factor (IGF-I), transforming growth factor (TGF)-alpha, epidermal growth factor (EGF), fibroblast growth factor (FGF), TGF-beta, bone morphogenetic protein (BMP)-2, BMP-4, BMP-5, BMP-6, BMP-7, activins A and B, decapentaplegic (dpp), 60A, OP-2, dorsalin, growth differentiation factors (GDFs), nodal, MIS, inhibin-alpha, TGF-beta1, TGF-beta2, TGF-beta3, TGF-beta5, and glial-derived neurotrophic factor (GDNF). \\n     Other representative fragments are biologically active fragments of a polypeptide of the present invention. Biologically active fragments are those exhibiting activity similar, but not necessarily identical, to an activity of the polypeptide. The biological activity of the fragments can include an improved desired activity, or a decreased undesirable activity. \\n     Additionally, this invention provides a method of screening compounds to identify those that modulate the action of a polypeptide of the present invention. An example of such an assay comprises combining a mammalian fibroblast cell, a polypeptide of the present invention, the compound to be screened and  3 H thymidine under cell culture conditions where the fibroblast cell would normally proliferate. A control assay can be performed in the absence of the compound to be screened and compared to the amount of fibroblast proliferation in the presence of the compound to determine if the compound stimulates proliferation by determining the uptake of  3 H thymidine in each case. The amount of fibroblast cell proliferation is measured by liquid scintillation chromatography, which measures the incorporation of  3 H thymidine. Both agonist and antagonist compounds can be identified by this procedure. \\n     In another method, a mammalian cell or membrane preparation expressing a receptor for a polypeptide of the present invention is incubated with a labeled polypeptide of the present invention in the presence of the compound. The ability of the compound to enhance or block this interaction could then be measured. Alternatively, the response of a known second messenger system following interaction of a compound to be screened and the receptor is measured and the ability of the compound to bind to the receptor and elicit a second messenger response is measured to determine if the compound is a potential agonist or antagonist. Such second messenger systems include but are not limited to, cAMP guanylate cyclase, ion channels or phosphoinositide hydrolysis. \\n     All of these above assays can be used as diagnostic or prognostic markers. The molecules discovered using these assays can be used to treat, prevent, and/or diagnose disease or to bring about a particular result in a subject (e.g., blood vessel growth) by activating or inhibiting the polypeptide/molecule. Moreover, the assays can discover agents that can inhibit or enhance the production of the polypeptides of the present invention from suitably manipulated cells or tissues. Therefore, the present invention includes a method of identifying compounds that bind to the polypeptides of the present invention comprising the steps of: (a) incubating a candidate binding compound with the polypeptide; and (b) determining if binding has occurred. Moreover, the present invention includes a method of identifying agonists/antagonists comprising the steps of: (a) incubating a candidate compound with the polypeptide, (b) assaying a biological activity, and (b) determining if a biological activity of the polypeptide has been altered. \\n     Also, one could identify molecules that bind a polypeptide of the present invention experimentally by using the beta-pleated sheet regions contained in the polypeptide sequence of the protein. Accordingly, specific embodiments of the present invention are directed to polynucleotides encoding polypeptides which comprise, or alternatively consist of, the amino acid sequence of each beta pleated sheet region in a disclosed polypeptide sequence. Additional embodiments of the present invention are directed to polynucleotides encoding polypeptides that comprise, or alternatively consist of, any combination or all of contained in the polypeptide sequences of the present invention. Additional preferred embodiments of the present invention are directed to polypeptides that comprise, or alternatively consist of, the amino acid sequence of each of the beta pleated sheet regions in one of the polypeptide sequences of the present invention. Additional embodiments of the present invention are directed to polypeptides that comprise, or alternatively consist of, any combination, or all, of the beta pleated sheet regions in one of the polypeptide sequences of the present invention. \\n     Targeted Delivery \\n     In another embodiment, the present invention provides a method of delivering compositions to targeted cells expressing a receptor for a polypeptide of the present invention, or cells expressing a cell bound form of a polypeptide of the present invention. \\n     As discussed herein, polypeptides or antibodies of the present invention can be associated with heterologous polypeptides, heterologous nucleic acids, toxins, or prodrugs via hydrophobic, hydrophilic, ionic and/or covalent interactions. In one embodiment, the present invention provides a method for the specific delivery of compositions of the present invention to cells by administering polypeptides of the present invention (including antibodies) that are associated with heterologous polypeptides or nucleic acids. In one example, the present invention provides a method for delivering a therapeutic protein into the targeted cell. In another example, the present invention provides a method for delivering a single stranded nucleic acid (e.g., antisense or ribozymes) or double stranded nucleic acid (e.g., DNA that can integrate into the cell&#39;s genome or replicate episomally and that can be transcribed) into the targeted cell. \\n     In another embodiment, the present invention provides a method for the specific destruction of cells (e.g., the destruction of tumor cells) by administering a polypeptide of the present invention (e.g., a polypeptide of the present invention or antibodies to a polypeptide of the present invention) in association with toxins or cytotoxic prodrugs. \\n     Drug Screening \\n     Another aspect of the present invention is the use of the polypeptides of the present invention, or the polynucleotides encoding these polypeptides, to screen for molecules that modify the activities of the polypeptides of the present invention. Such a method would include contacting a polypeptide of the present invention with a selected compound(s) suspected of having antagonist or agonist activity, and assaying the activity of these polypeptides following binding. \\n     This invention is particularly useful for screening therapeutic compounds by using the polypeptides of the present invention, or binding fragments thereof, in any of a variety of drug screening techniques. The polypeptide or fragment employed in such a test can be affixed to a solid support, expressed on a cell surface, free in solution, or located intracellularly. One method of drug screening utilizes eukaryotic or prokaryotic host cells which are stably transformed with recombinant nucleic acids expressing the polypeptide or fragment. Drugs are screened against such transformed cells in competitive binding assays. One can measure, for example, the formulation of complexes between the agent being tested and a polypeptide of the present invention. \\n     Thus, the present invention provides methods of screening for drugs or any other agents that affect activities mediated by the polypeptides of the present invention. These methods comprise contacting such an agent with a polypeptide of the present invention or a fragment thereof and assaying for the presence of a complex between the agent and the polypeptide or a fragment thereof, by methods well known in the art. In such a competitive binding assay, the agents to screen are typically labeled. Following incubation, free agent is separated from that present in bound form, and the amount of free or uncomplexed label is a measure of the ability of a particular agent to bind to a polypeptide of the present invention. \\n     Another technique for drug screening provides high throughput screening for compounds having suitable binding affinity to the polypeptides of the present invention, and is described in great detail in European Patent Application 84/03564. Briefly stated, large numbers of different small peptide test compounds are synthesized on a solid substrate, such as plastic pins or some other surface. The peptide test compounds are reacted with one or more polypeptides of the present invention and washed. Bound polypeptides are then detected by methods well known in the art. Purified polypeptides are coated directly onto plates for use in the aforementioned drug screening techniques. In addition, non-neutralizing antibodies may be used to capture the peptide and immobilize it on the solid support. \\n     This invention also contemplates the use of competitive drug screening assays in which neutralizing antibodies capable of binding polypeptides of the present invention specifically compete with a test compound for binding to the polypeptides or fragments thereof. In this manner, the antibodies are used to detect the presence of any peptide that shares one or more antigenic epitopes with a polypeptide of the present invention. \\n     The HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptides and/or peptides of the present invention, or immunogenic fragments or oligopeptides thereof, can be used for screening therapeutic drugs or compounds in a variety of drug screening techniques. The fragment employed in such a screening assay can be free in solution, affixed to a solid support, borne on a cell surface, or located intracellularly. The reduction or abolition of activity of the formation of binding complexes between the ion channel protein and the agent being tested can be measured. Thus, the present invention provides a method for screening or assessing a plurality of compounds for their specific binding affinity with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y), or a bindable peptide fragment, of this invention, comprising (a) providing a plurality of compounds, (b) combining the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, or a bindable peptide fragment, with each of a plurality of compounds for a time sufficient to allow binding under suitable conditions and (c) detecting binding of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or peptide to each of the plurality of test compounds, thereby identifying the compounds that specifically bind to the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or peptide. \\n     Methods of identifying compounds that modulate the activity of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptides and/or peptides are provided by the present invention and comprise (a) combining a potential or candidate compound or drug modulator of GBP biological activity with an HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or peptide, for example, the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) amino acid sequence as set forth in SEQ ID NO:Y, and (b) measuring an effect of the candidate compound or drug modulator on the biological activity of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) or fragment thereof. Such measurable effects include, for example, physical binding interaction; the ability to cleave a suitable GBP substrate; effects on native and cloned HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ 10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)-expressing cell line; and effects of modulators or other GBP-mediated physiological measures. \\n     The host cell can also be capable of being induced to express a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y), e.g., via inducible expression. Physiological effects of a given modulator candidate on the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) can also be measured. Thus, cellular assays for particular GBP modulators can be either direct measurement or quantification of the physical biological activity of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y), or they can be measurement or quantification of a physiological effect. Such methods preferably employ a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) as described herein, or an overexpressed recombinant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) in suitable host cells containing an expression vector as described herein, wherein the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) is expressed, overexpressed, or undergoes upregulated expression. \\n     Another aspect of the present invention embraces a method of screening for a compound that is capable of modulating the biological activity of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y), comprising (a) providing a host cell containing an expression vector harboring a nucleic acid sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y), or a functional peptide or portion thereof; (b) determining the biological activity of the expressed HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide in the absence of a modulator compound; and (c) contacting the cell with the modulator compound and determining the biological activity of the expressed HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide in the presence of the modulator compound. In such a method, a difference between the activity of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide in the presence of the modulator compound and in the absence of the modulator compound indicates a modulating effect of the compound. \\n     Essentially any chemical compound can be employed as a potential modulator or ligand in the assays according to the present invention. Compounds tested as GBP modulators can be any small chemical compound, or biological entity (e.g., protein, carbohydrate, nucleic acid, lipid, etc.). Test compounds will typically be small chemical molecules and peptides. Generally, the compounds used as potential modulators can be dissolved in aqueous or organic (e.g., DMSO-based) solutions. The assays are designed to screen large chemical libraries by automating the assay steps and providing compounds from any convenient source. Assays are typically run in parallel, for example, in microtiter formats on microtiter plates in robotic assays. There are many suppliers of chemical compounds, including Sigma (St. Louis, Mo., USA), Aldrich (St. Louis, Mo., USA), Sigma-Aldrich (St. Louis, Mo., USA), and Fluka Chemika-Biochemica Analytika (Buchs, Switzerland), for example. Also, compounds can be synthesized by methods known in the art. \\n     High throughput screening methodologies are particularly envisioned for the detection of modulators of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polynucleotides (e.g., SEQ ID NO:X) and polypeptides (e.g., SEQ ID NO:Y) described herein. Such high throughput screening methods typically involve providing a combinatorial chemical or peptide library containing a large number of potential therapeutic compounds (e.g., ligand or modulator compounds). Such combinatorial chemical libraries or ligand libraries are then screened in one or more assays to identify those library members (e.g., particular chemical species or subclasses) that display a desired characteristic activity. The compounds so identified can serve as conventional lead compounds, or can themselves be used as potential or actual therapeutics. \\n     A combinatorial chemical library is a collection of diverse chemical compounds generated either by chemical synthesis or biological synthesis, by combining a number of chemical building blocks (i.e., reagents such as amino acids). As an example, a linear combinatorial library, e.g., a polypeptide or peptide library, is formed by combining a set of chemical building blocks in every possible way for a given compound length (i.e., the number of amino acids in a polypeptide or peptide compound). Millions of chemical compounds can be synthesized through such combinatorial mixing of chemical building blocks. \\n     Methods of preparating and screening combinatorial chemical libraries are known to those of ordinary skill in the art. Combinatorial libraries include, but are not limited to, peptide libraries (see, e.g. U.S. Pat. No. 5,010,175; Furka, (1991)  Int. J. Pept. Prot. Res.  37:487–493; and Houghton et al., (1991)  Nature  354:84–88). Other chemistries for generating chemical diversity libraries can also be used. Nonlimiting examples of chemical diversity library chemistries include, peptides (PCT Publication WO 91/019735), encoded peptides (PCT Publication WO 93/20242), random bio-oligomers (PCT Publication WO 92/00091), benzodiazepines (U.S. Pat. No. 5,288,514), diversomers such as hydantoins, benzodiazepines and dipeptides (Hobbs et al., (1993)  Proc. Natl. Acad. Sci. USA  90:6909–6913), vinylogous polypeptides (Hagihara et al., (1992)  J. Amer. Chem. Soc.  114:6568), nonpeptidal peptidomimetics with glucose scaffolding (Hirschmann et al., (1992)  J. Amer. Chem. Soc.  114:9217–9218), analogous organic synthesis of small compound libraries (Chen et al., (1994)  J. Am. Chem. Soc.  116:2661), oligocarbamates (Cho et al., (1993)  Science,  261:1303), and/or peptidyl phosphonates (Campbell et al., (1994)  J. Org. Chem.,  59:658), nucleic acid libraries (see Ausubel, Berger and Sambrook, all referenced herein), peptide nucleic acid libraries (U.S. Pat. No. 5,539,083), antibody libraries (e.g., Vaughn et al., (1996)  Nature Biotechnol.  14(3):309–314) and PCT Publication US96/10287), carbohydrate libraries (e.g., Liang et al., (1996)  Science  274–1520–1522) and U.S. Pat. No. 5,593,853), small organic molecule libraries (e.g., benzodiazepines, Baum, C&amp;EN, Jan. 18, 1993, page 33; and U.S. Pat. No. 5,288,514); isoprenoids (U.S. Pat. No. 5,569,588); thiazolidinones and metathiazanones (U.S. Pat. No. 5,549,974); pyrrolidines (U.S. Pat. Nos. 5,525,735 and 5,519,134); and morpholino compounds (U.S. Pat. No. 5,506,337; and the like). \\n     Devices for the preparation of combinatorial libraries are commercially available (e.g., 357 MPS, 390 MPS, Advanced Chem Tech, Louisville Ky., USA; SYMPHONY®, Rainin, Woburn, Mass., USA; 433A Applied Biosystems, Foster City, Calif., USA; and the 9050 Plus, Millipore, Bedford, Mass., USA). In addition, a large number of combinatorial libraries are commercially available (e.g., ComGenex, Princeton, N.J., USA; Asinex, Moscow, Russia; Tripos, Inc., St. Louis, Mo., USA; ChemStar, Ltd., Moscow, Russia; 3D Pharmaceuticals, Exton, Pa.; Martek Biosciences, Columbia, Md., USA, and the like). \\n     In one embodiment, the present invention provides solid phase based in vitro assays in a high throughput format, where the cell or tissue expressing an ion channel is attached to a solid phase substrate. In such high throughput assays, it is possible to screen up to several thousand different modulators or ligands in a single day. In particular, each well of a microtiter plate can be used to perform a separate assay against a selected potential modulator, or, if concentration or incubation time effects are to be observed, every 5–10 wells can test a single modulator. Thus, a single standard microtiter plate can assay about 96 modulators. If 1536 well plates are used, then a single plate can easily assay from about 100 to about 1500 different compounds. It is possible to assay several different plates per day; thus, for example, assay screens for up to about 6,000–20,000 different compounds are possible using the described integrated systems. \\n     In another aspect, the present invention encompasses screening and small molecule (e.g., drug) detection assays which involve the detection or identification of small molecules that can bind to a given protein, i.e., a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or peptide (e.g., SEQ ID NO:Y). Assays suitable for high throughput screening methodologies, can be desirable in certain situations. \\n     In such binding-based detection, identification, or screening assays, a functional assay is not typically required. All that is needed is a target protein, preferably substantially purified, and a library or panel of compounds (e.g., ligands, drugs, small molecules) or biological entities to be screened or assayed for binding to the protein target. It can be desirable that most small molecules that bind to the target protein will modulate activity in some manner, due to preferential, higher affinity binding to functional areas or sites on the protein. \\n     An example of such an assay is the fluorescence based thermal shift assay (3D Pharmaceuticals, Inc., (3DP), Exton, Pa., USA) as described in U.S. Pat. Nos. 6,020,141 and 6,036,920; see also, Zimmerman, (2000)  Gen. Eng. News,  20(8)). The assay allows the detection of small molecules (e.g., drugs, ligands) that bind to expressed, and preferably purified, ion channel polypeptide based on affinity of binding determinations by analyzing thermal unfolding curves of protein-drug or ligand complexes. The drugs or binding molecules determined by this technique can be further assayed, if desired, by methods, such as those described herein, to determine if the molecules affect or modulate function or activity of the target protein. \\n     To purify a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) or peptide to measure a biological binding or ligand binding activity, the source may be a whole cell lysate that can be prepared by successive freeze-thaw cycles (e.g., one to three) in the presence of standard protease inhibitors. A HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) can be partially or completely purified by standard protein purification methods, e.g., affinity chromatography using specific antibody described herein, or by ligands specific for an epitope tag engineered into the recombinant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide molecule, also as described herein. Binding activity can then be measured using assays known to those of ordinary skill in the art and as described herein. \\n     Compounds which are identified according to the methods provided herein, and that modulate or regulate the biological activity or physiology of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of to the present invention (e.g., SEQ ID NO:Y) are a representative embodiment of the present invention. Such modulatory compounds can be employed in treatment and therapeutic methods for treating a condition that is mediated by a MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475), MGBPBMY4 (BC007143), HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or HGBPBMY4 (FLJ10961) polypeptide of the present invention by administering to an individual in need of such treatment a therapeutically effective amount of the compound identified by the methods described herein. \\n     In addition, the present invention provides methods for treating a subject in need of such treatment for a disease, disorder, or condition that is mediated by a MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475), MGBPBMY4 (BC007143), HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or HGBPBMY4 (FLJ10961) polypeptide of the present invention, comprising administering to the individual a therapeutically effective amount of a MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475), MGBPBMY4 (BC007143), HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2) and/or HGBPBMY4 (FLJ10961)-modulating compound identified by a method provided herein. \\n     Antisense and Ribozyme (Antagonists) \\n     In specific embodiments, antagonists of to the present invention are nucleic acids corresponding to the sequences contained in SEQ ID NO:X, or the complementary strand thereof, and/or to nucleotide sequences contained a deposited clone. In one embodiment, antisense sequence is generated internally by the organism, in another embodiment, the antisense sequence is separately administered (see, for example, O&#39;Connor, (1991)  Neurochem.  56:560 ; Oligodeoxynucleotides as Antisense Inhibitors of Gene Expression , CRC Press, Boca Raton, Fla., USA (1988)). Antisense technology can be used to control gene expression through antisense DNA or RNA, or through triple-helix formation. Antisense techniques are discussed for example, in Okano, (1991)  Neurochem.  56:560 and  Oligodeoxynucleotides as Antisense Inhibitors of Gene Expression , CRC Press, Boca Raton, Fla. (1988). Triple helix formation is discussed in, for instance, Lee et al., (1979)  Nucl. Acid Res.  6:3073; Cooney et al., (1988)  Science  241:456; and Dervan et al., (1991)  Science  251:1300. The methods are based on binding of a polynucleotide to a complementary DNA or RNA. \\n     For example, the use of c-myc and c-myb antisense RNA constructs to inhibit the growth of the non-lymphocytic leukemia cell line HL-60 and other cell lines has been described (Wickstrom et al., (1988)  Proc. Natl. Acad. Sci. U.S.A.  85(4):1028–32; Anfossi et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86(9):3379–83). These experiments were performed in vitro by incubating cells with the oligoribonucleotide. A similar procedure for in vivo use is described in PCT Publication WO 91/15580. Briefly, a pair of oligonucleotides for a given antisense RNA is produced as follows: A sequence complimentary to the first 15 bases of the open reading frame is flanked by an EcoR1 site on the 5′ end and a HindIII site on the 3′ end. Next, the pair of oligonucleotides is heated at 90° C. for one minute and then annealed in 2× ligation buffer (20 mM TRIS HCl pH 7.5, 101M MgCl 2 , 10 MM dithiothreitol (DTT) and 0.2 mM ATP) and then ligated to the EcoR1/Hind III site of the retroviral vector PMV7 (see PCT Publication WO 91/15580). \\n     For example, the 5′ coding portion of a polynucleotide that encodes the mature polypeptide of the present invention may be used to design an antisense RNA oligonucleotide of from about 10 to 40 base pairs in length. A DNA oligonucleotide is designed to be complementary to a region of the gene involved in transcription thereby preventing transcription and the production of the receptor. The antisense RNA oligonucleotide hybridizes to the mRNA in vivo and blocks translation of the mRNA molecule into receptor polypeptide. \\n     In one embodiment, the antisense nucleic acid of the present invention is produced intracellularly by transcription from an exogenous sequence. For example, a vector or a portion thereof, is transcribed, producing an antisense nucleic acid (RNA) of the present invention. Such a vector would contain a sequence encoding the antisense nucleic acid of the present invention. Such a vector can remain episomal or become chromosomally integrated, as long as it can be transcribed to produce the desired antisense RNA. Such vectors can be constructed by recombinant DNA technology methods standard in the art. Vectors can be plasmid, viral, or others known in the art, used for replication and expression in vertebrate cells. Expression of the sequence encoding a polypeptide of the present invention, or fragments thereof, can be by any promoter known in the art to act in vertebrate, for example human, cells. Such promoters can be inducible or constitutive. Such promoters include, but are not limited to, the SV40 early promoter region (Bemoist &amp; Chambon, (1981)  Nature  29:304–310), the promoter contained in the 3′ long terminal repeat of Rous sarcoma virus (Yamamoto et al., (1980)  Cell  22:787–797), the herpes thymidine promoter (Wagner et al., (1981)  Proc. Natl. Acad. Sci. USA  78:1441–1445), the regulatory sequences of the metallothionein gene (Brinster et al., (1982)  Nature  296:39–42), etc. \\n     The antisense nucleic acids of the present invention comprise a sequence complementary to at least a portion of an RNA transcript of a gene of interest. However, absolute complementarity, although preferred, is not required. A sequence “complementary to at least a portion of an RNA” referred to herein, means a sequence having sufficient complementarity to be able to hybridize with the RNA, forming a stable duplex; in the case of double stranded antisense nucleic acids of the present invention, a single strand of the duplex DNA can thus be tested, or triplex formation may be assayed. The ability to hybridize will depend on both the degree of complementarity and the length of the antisense nucleic acid. Generally, the larger the hybridizing nucleic acid, the more base mismatches with a RNA sequence of the present invention it can contain and still form a stable duplex (or triplex as the case may be). One of ordinary skill in the art can ascertain a tolerable degree of mismatch by use of standard procedures to determine the melting point of the hybridized complex, upon consideration of the present disclosure. \\n     Oligonucleotides that are complementary to the 5′ end of the message, e.g., the 5′ untranslated sequence up to and including the AUG initiation codon, should work most efficiently at inhibiting translation. However, sequences complementary to the 3′ untranslated sequences of mRNAs have been shown to be effective at inhibiting translation of mRNAs as well (see, e.g., Wagner, (1994)  Nature  372:333–335). Thus, oligonucleotides complementary to either the 5′- or 3′-non-translated, non-coding regions of a polynucleotide sequence of the present invention could be used in an antisense approach to inhibit translation of endogenous mRNA. Oligonucleotides complementary to the 5′ untranslated region of the mRNA should include the complement of the AUG start codon. Antisense oligonucleotides complementary to mRNA coding regions are less efficient inhibitors of translation but could be used in accordance with the present invention. Whether designed to hybridize to the 5′-, 3′- or coding region of mRNA, antisense nucleic acids should be at least six nucleotides in length, and are preferably oligonucleotides ranging from 6 to about 50 nucleotides in length. In specific aspects the oligonucleotide is at least 10 nucleotides, at least 17 nucleotides, at least 25 nucleotides or at least 50 nucleotides. \\n     The polynucleotides of the present invention can be DNA or RNA or chimeric mixtures or derivatives or modified versions thereof, single-stranded or double-stranded. The oligonucleotide can be modified at the base moiety, sugar moiety, or phosphate backbone, for example, to improve stability of the molecule, hybridization, etc. The oligonucleotide may include other appended groups such as peptides (e.g., for targeting host cell receptors in vivo), or agents facilitating transport across the cell membrane (see, e.g., Letsinger et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86:6553–6556; Lemaitre et al., (1987)  Proc. Natl. Acad. Sci. U.S.A.  84:648–652; PCT Publication WO 88/09810) or the blood-brain barrier (see, e.g., PCT Publication WO 89/10134), hybridization-triggered cleavage agents. (see, e.g., Krol et al., (1988)  BioTechniques  6:958–976) or intercalating agents (see, e.g., Zon, (1988)  Pharm. Res.  5:539–549). To this end, the oligonucleotide can be conjugated to another molecule, e.g., a peptide, hybridization triggered cross-linking agent, transport agent, hybridization-triggered cleavage agent, etc. \\n     An antisense oligonucleotide can comprise at least one modified base moiety which is selected from the group including, but not limited to, 5-fluorouracil, 5-bromouracil, 5-chlorouracil, 5-iodouracil, hypoxanthine, xanthine, 4-acetylcytosine, 5-(carboxyhydroxylmethyl) uracil, 5-carboxymethylaminomethyl-2-thiouridine, 5-carboxymethylaminomethyluracil, dihydrouracil, beta-D-galactosylqueosine, inosine, N6-isopentenyladenine, 1-methylguanine, 1-methylinosine, 2,2-dimethylguanine, 2-methyladenine, 2-methylguanine, 3-methylcytosine, 5-methylcytosine, N6-adenine, 7-methylguanine, 5-methylaminomethyluracil, 5-methoxyaminomethyl-2-thiouracil, beta-D-mannosylqueosine, 5′-methoxycarboxymethyluracil, 5-methoxyuracil, 2-methylthio-N-6-isopentenyladenine, uracil-5-oxyacetic acid (v), wybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-methyl-2-thiouracil, 2-thiouracil, 4-thiouracil, 5-methyluracil, uracil-5-oxyacetic acid methylester, uracil-5-oxyacetic acid (v), 5-methyl-2-thiouracil, 3-(3-amino-3-N-2-carboxypropyl) uracil, (acp3)w, and 2,6-diaminopurine. \\n     The antisense oligonucleotide can also comprise at least one modified sugar moiety selected from the group including, but not limited to, arabinose, 2-fluoroarabinose, xylulose, and hexose. \\n     In yet another embodiment, the antisense oligonucleotide comprises at least one modified phosphate backbone selected from the group including, but not limited to, a phosphorothioate, a phosphorodithioate, a phosphoramidothioate, a phosphoramidate, a phosphordiamidate, a methylphosphonate, an alkyl phosphotriester, and a formacetal or analog thereof. \\n     In yet another embodiment, an antisense oligonucleotide is an α-anomeric oligonucleotide. An a-anomeric oligonucleotide forms specific double-stranded hybrids with complementary RNA in which, contrary to the usual b-units, the strands run parallel to each other (Gautier et al., (1987)  Nucl. Acids Res.  15:6625–6641). The oligonucleotide is a 2-0-methylribonucleotide (Inoue et al., (1987)  Nucl. Acids Res.  15:6131–6148), or a chimeric RNA-DNA analogue (Inoue et al., (1987)  FEBS Lett.  215:327–330). \\n     Polynucleotides of the present invention can be synthesized by standard methods known in the art, e.g. by use of an automated DNA synthesizer (such as are commercially available, for example from Biosearch, Applied Biosystems, Foster City, Calif., USA). As examples, phosphorothioate oligonucleotides may be synthesized by the method of Stein et al. (Stein et al., (1988)  Nucl. Acids Res.  16:3209), methylphosphonate oligonucleotides can be prepared by use of controlled pore glass polymer supports (Sarin et al., (1988)  Proc. Natl. Acad. Sci. U.S.A.  85:7448–7451), etc. \\n     While antisense nucleotides complementary to the coding region sequence of the present invention could be used, those complementary to the transcribed untranslated region are often most desirable. \\n     Potential antagonists according to the present invention also include catalytic RNA, or a ribozyme (see, e.g., PCT Publication WO 90/11364; Sarver et al., (1990)  Science  247:1222–1225). While ribozymes that cleave mRNA at site specific recognition sequences can be used to destroy mRNAs corresponding to a polynucleotide of the present invention, the use of hammerhead ribozymes is desirable in certain situations. Hammerhead ribozymes cleave mRNAs at locations dictated by flanking regions that form complementary base pairs with the target mRNA. The sole requirement is that the target mRNA have the following sequence of two bases: 5′-UG-3′. The construction and production of hammerhead ribozymes is known in the art (see, e.g., Haseloff &amp; Gerlach, (1988)  Nature  334:585–591). There are numerous potential hammerhead ribozyme cleavage sites within each nucleotide sequence disclosed in the sequence listing. In one embodiment, the ribozyme is engineered so that the cleavage recognition site is located near the 5′ end of the mRNA corresponding to a polynucleotide of the present invention; i.e., to increase efficiency and minimize the intracellular accumulation of non-functional mRNA transcripts. \\n     As in the antisense approach, the ribozymes of the present invention can comprise modified oligonucleotides (e.g. for improved stability, targeting, etc.) and can be delivered to cells that express a polynucleotide of the present invention in vivo. DNA constructs encoding the ribozyme may be introduced into the cell in the same manner as described above for the introduction of antisense encoding DNA. A representative method of delivery involves using a DNA construct “encoding” the ribozyme under the control of a strong constitutive promoter, such as, for example, pol III or pol 11 promoter, so that transfected cells will produce sufficient quantities of the ribozyme to destroy endogenous messages and inhibit translation. Since ribozymes unlike antisense molecules, are catalytic, a lower intracellular concentration is required for efficiency. \\n     In another embodiment, small interfering RNAs (siRNAs; reviewed in Scherr et al., (2003)  Curr. Med. Chem.  10:245–256) can be employed in a gene silencing approach. SiRNAs are short double stranded pieces of RNA, generally about 21 nt in size, often with dTdT or UU overhangs on the ends of the siRNA. These pieces of RNA can induce RNA interference, dsRNA-mediated destruction of target mRNA and ultimately the silencing of a gene siRNAs can be employed to post-transcriptionally silence a given gene, such as a gene of the present invention. Thus, siRNA approaches can be employed in the present invention. \\n     Homologous recombination techniques can also be employed to silence a gene. Generally, a transgene (often the neo gene) can be inserted into the gene to be silenced, disrupting the coding sequence of the gene. Upon recombination, the allele carrying the transgene becomes a null allele for the silenced gene. \\n     Antagonist/agonist compounds may be employed to modulate an immune response, including autoimmune conditions. Such compounds can also be employed to modulate aberrant NF-κB-related activity. \\n     An antagonist/agonist may also be employed to treat, prevent, and/or diagnose the diseases described herein, notably rheumatoid arthritis and conditions related to aberrant NF-κB activity. \\n     An antagonist/agonist may also be employed in combination with additional therapies to treat or prevent the diseases described herein. For example, an antagonist or agonist of the present invention (including single, double and antisense nucleic acids) may be employed in combination with an anti-TNF therapy. Alternatively, an an antagonist or agonist of the present invention may be employed in combination with KINERET for the treatment of a condition such as rheumatoid arthritis. \\n     Thus, the present invention provides a method of treating or preventing diseases, disorders, and/or conditions, including but not limited to the diseases, disorders, and/or conditions listed throughout this application, associated with overexpression of a polynucleotide of the present invention by administering to a subject (a) an antisense molecule directed to a polynucleotide of the present invention, and/or (b) a ribozyme directed to a polynucleotide of the present invention. \\n     Other Activities \\n     A polypeptide and/or polynucleotide of the present invention may be employed in methods in which antiviral activity is desired. For example, such polypeptides and/or polynucleotides can be employed in the treatment prevention or diagnosis of viral infections. \\n     A polypeptide of the present invention may be employed for treating immune system conditions, including rheumatoid arthritis. \\n     A polypeptide and/or polynucleotide of the present invention may also be used for treating some NF-κB-related conditions. \\n     A polynucleotide, polypeptide, agonist and/or antagonist of the present invention can also increase or decrease the differentiation or proliferation of embryonic stem cells, besides, as discussed herein, hematopoietic lineage. \\n     A polynucleotide, polypeptide, agonist and/or antagonist of the present invention can also be used to increase the efficacy of a pharmaceutical composition, either directly or indirectly. Such a procedure can be administered in simultaneous conjunction with said pharmaceutical, or separately through either the same or different route of administration (e.g., intravenous for the polynucleotide or polypeptide of the present invention, and orally for the pharmaceutical, among others described herein.). \\n     In another aspect, the present invention comprises a method of treatment of an individual in need of an increased level of a protein activity. In one embodiment the method comprises administering to such an individual a pharmaceutical composition comprising an amount of an isolated polypeptide, polynucleotide, or antibody of the claimed invention effective to increase the level of said protein activity in the individual. \\n     Having generally described the present invention, the same will be more readily understood by reference to the following examples, which are provided by way of illustration. \\n     EXAMPLES \\n     The following Examples have been included to illustrate various exemplary modes of the present invention. Certain aspects of the following Examples are described in terms of techniques and procedures found or contemplated by the inventors to work well in the practice of the present invention. These Examples are exemplified through the use of standard laboratory practices of the inventors. In light of the present disclosure and the general level of skill in the art, those of skill will appreciate that the following Examples are intended to be exemplary only and that numerous changes, modifications and alterations can be employed without departing from the spirit and scope of the present invention. \\n     Example 1 \\n     Identification of Novel Human GBPs \\n     The loci of human GBP-1, GBP-2, GBP-4, and GBP-5 were analyzed on NCBI&#39;s Genome View map of human chromosome 1p22 from position 89,500K through position 90,750K. A number of annotated hypothetical genes flanking the GBP locus were BLAST searched against the non-redundant protein database. Hypothetical protein HGBPBMY4 (FLJ10961) had high homology to the other GBP family members and was immediately distal to GBP-1 ( FIG. 10A ). Modeled gene sequences predicted by Genome Scan in and around the GBP locus were BLAST searched against the non-redundant protein database. Sequence HGBPBMY1 (AK096141) located between GBP-2 and GBP-4 and sequence HGBPBMY2 (4843 30 1 1; 4843 — 1) and novel sequence HGBPBMY3 (4843 30 2 1; 4843 — 2) immediately proximal to GBP-5 ( FIG. 10A ) had high homology to the GBP family members ( FIGS. 11A–11G ). \\n     Example 2 \\n     Identification of Novel Mouse GBPs \\n     The loci of mouse GBP-1, GBP-2, GBP-3, and GBP-5 were analyzed on NCBI&#39;s Genome View map of mouse chromosome 3H1 from position 142,900K through position 143,300K. A number of annotated hypothetical genes flanking the GBP locus were BLAST searched against the non-redundant protein database. Hypothetical protein MGBPBMY1 (LOC229900) located between GBP-5 and GBP-3 and hypothetical protein MGBPBMY2 (LOC229902) immediately distal to GBP-2 ( FIG. 10B ) had high homology to the GBP family ( FIGS. 11A–11G ). \\n     The locus of GBP-4 was analyzed on NCBI&#39;s Genome View map of mouse chromosome 5E4 from position 102,100K through position 103,300K ( FIG. 10C ). A number of annotated hypothetical genes flanking the GBP locus were BLAST searched against the non-redundant protein database. Hypothetical proteins MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143) distal to GBP-4 ( FIG. 10B ) had high homology to the GBP family ( FIGS. 11A–11G ). \\n     Example 3 \\n     TAQMAN Analyses \\n     Reactions were performed in a total volume of 40 μl. The master mix contained SYBR Green I Dye, 50 mM Tris-HCl pH=8.3, 75 mM KCl, DMSO, Rox reference dye, SmNM MgCl 2 , 2 mM dNTP, Platinum Taq High Fidelity (IU/reaction), and 0.5 μM of each primer. The cDNA was diluted 1:36 from the synthesis reaction and eight microliters was used in each PCR reaction. For tissue distribution analyses, two microliters of cDNA from the Human Multiple Tissue and Human Immune System MTC cDNA panels (Clontech, Palo Alto, Calif., USA) were used as templates. For the mouse GBPs, the Clontech mouse-1 cDNA panel was used as the template. The amplification program consisted of a 10 minute incubation at 95° C. followed by forty cycles of incubations at 95° C. for 15 seconds and 60° C. for 1 minute. Amplification was followed by melting curve analysis at 60° C. to demonstrate that the amplification was specific to a single amplicon. A negative control without cDNA template was run to assess the overall specificity. \\n     For THP-1 experiments, THP-1 cells (5×10 6) were cultured in triplicate at 10 6 /ml in RPMI containing 10% heat inactivated fetal calf serum, 2 mM L-glutamine with either medium, LPS (100 ng/ml, Sigma Chemical Company, St. Louis, Mo., USA), or 100 U/ml interferon-γ (IFN-γ, Peprotech, Rocky Hill, N.J.) for 6 hours at 37° C. in 5% CO 2 . In some experiments, cells were pretreated with either Compound 1 (2 μM) or dexamethasone (100 nM) for 30 minutes before addition of 100 ng/ml LPS. Compound 1 has the formula: PKKKRKVAAVALLPAVLLALLAPKKKRKV (SEQ ID NO:60) Compound 1 preferably comprises D amino acids, but can also comprise L amino acids or a combination of D and L amino acids. \\n     The cells were stimulated for 0.5–8 hr as described. At each time point, cells were pelleted and washed with PBS. RNA was isolated using Qiagen RNease® mini columns (Qiagen, Inc.) according to manufacturer&#39;s instructions. The RNA was eluted with 50 μl of RNase-free water. \\n     For synovial tissue experiments, human knee biopsy samples were homogenized in 3 ml TRIZOL® Reagent (Life Technologies, Rockville, Md., USA) and frozen in liquid nitrogen. The samples were thawed, one-third (1 ml) of the sample removed, mixed with 1 ml TRIZOL®, homogenized, and snap frozen in liquid nitrogen. Following a thaw, the samples were spun at 14,000 rpm for 10 minutes at 4° C. The supernatants were transferred to new microfuge tubes, extracted with chloroform, and precipitated with isopropanol overnight at −20° C. The RNA was pelleted by centrifugation at 14,000 rpm for 30 minutes. The supernatant was aspirated, and the samples washed two times with 75% ethanol. Following the last spin, the pellets were air-dried, and resuspended in 20 μl of ultra-pure RNase-free water. The RNA samples were further purified using Qiagen RNease mini columns (Qiagen, Inc.) according to manufacturer&#39;s instructions. The RNA was eluted with 50 ul of RNase-free water. \\n     Example 4 \\n     Data Analysis \\n     A relative value for the initial target concentration in each reaction was determined using the TAQMAN® 5700 software (Applied Biosystems, Foster City, Calf., USA). The threshold value was set to 0.5 to obtain cycle threshold values that were used to assign relative message levels for each target. The message levels of GAPDH were determined for each cDNA sample and were used to normalize all other genes tested from the same cDNA sample. \\n     Primers \\n     Gene specific primers for were designed using the Primer Express software and synthesized by Sigma Genosys (The Woodlands, Tex., USA). Primer names and sequences are below: \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 GBP1 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F4 \\n                 CATTGGTCTGGCCAAGTCTACA \\n                 2181 \\n                 (SEQ ID NO: 19) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R4 \\n                 TTCACTAAGAAGCTAGGGTGGTTGT \\n                 2244 \\n                 (SEQ ID NO: 20) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 GBP2 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F4 \\n                 GGAGGAAGAGCTGAACCCTGAT \\n                 912 \\n                 (SEQ ID NO: 21) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R4 \\n                 GACTGCAATGCCACCTGAAAG \\n                 1014 \\n                 (SEQ ID NO: 22) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 GBP4 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GGCATTAGAGATTCTTGACAAGATTTC \\n                 123 \\n                 (SEQ ID NO: 23) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CCTGCAAGACGATTCATGAGATAG \\n                 224 \\n                 (SEQ ID NO: 24) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 GBP5 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F2 \\n                 CAGCACAACATTCCAAGCTCAA \\n                 1665 \\n                 (SEQ ID NO: 25) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R2 \\n                 GGATCATCGTTATTAACAGTCCTCTG \\n                 1745 \\n                 (SEQ ID NO: 26) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 HGBPBMY1 (AK096141) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F2 \\n                 GGGAGTGGATCAGGCATTTCT \\n                 801 \\n                 (SEQ ID NO: 27) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R2 \\n                 ACTATCCAGTTGGTCTTCTCGTACTTC \\n                 916 \\n                 (SEQ ID NO: 28) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 HGBPBMY2 (4843 30 1 1; 4843_1) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F2 \\n                 GGAGTGCATCAGGCGTTTCT \\n                 732 \\n                 (SEQ ID NO: 29) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R2 \\n                 CCAGTTGCTTTTCTGACACCTTCT \\n                 841 \\n                 (SEQ ID NO: 30) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 HGBPBMY3 (4843 30 2 1; 4843_2) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 AGGATGGCAGGAAGACAAACA \\n                 1066 \\n                 (SEQ ID NO: 31) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CTGGTCTGTCTGGAGAATTGCA \\n                 1215 \\n                 (SEQ ID NO: 32) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 HGBPBMY4 (FLJ10961) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GATCATGAGTTGCCACCACTCA \\n                 2110 \\n                 (SEQ ID NO: 33) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 GTGCCCAAATATGTCCCAAGA \\n                 2206 \\n                 (SEQ ID NO: 34) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGBP1 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GAGATGCTGATGGAACAGAAGGA \\n                 1691 \\n                 (SEQ ID NO: 35) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 TCCTGCTCCATCTTCTCAGTCA \\n                 1764 \\n                 (SEQ ID NO: 36) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGBP2 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 CAGGCTTTGAAACAACTGCTATGA \\n                 2006 \\n                 (SEQ ID NO: 37) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CAGTGCCCAGTGGTCAGACA \\n                 2104 \\n                 (SEQ ID NO: 38) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGBP3 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GGAAACCCTCACTGTTTGGTCA \\n                 1850 \\n                 (SEQ ID NO: 39) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CTTAGTGAGCCGAGGAATTTCAG \\n                 1964 \\n                 (SEQ ID NO: 40) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGBP4 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GTCCATGTGAGGCGAGGAA \\n                 3015 \\n                 (SEQ ID NO: 41) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 AACGACTCGGGCACTGTTGT \\n                 3088 \\n                 (SEQ ID NO: 42) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGBP5 \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 GCTGAAGCAAGGTAGCGATGA \\n                 666 \\n                 (SEQ ID NO: 43) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CCTCGTTGCTGAGTGTTGGA \\n                 814 \\n                 (SEQ ID NO: 44) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 MGBPBMY1 (LOC229900) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F3 \\n                 CTGAGGGTGAACTCCAAAGC \\n                 136 \\n                 (SEQ ID NO: 45) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R3 \\n                 CCCAGATTGAAGCCATGGTT \\n                 278 \\n                 (SEQ ID NO: 46) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 MGBPBMY2 (LOC229902) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F2 \\n                 CACCATCAGCAATGGGTCTCT \\n                 1110 \\n                 (SEQ ID NO: 47) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R2 \\n                 CTGGCTCATCTGCTGGTCATAGT \\n                 1227 \\n                 (SEQ ID NO: 48) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 MGBPBMY3 (BC031475) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F2 \\n                 CACAGCAAGAGGAAGTCACTGATATC \\n                 3171 \\n                 (SEQ ID NO: 49) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R2 \\n                 TGCCAATCTAACTCAGGGATGA \\n                 3248 \\n                 (SEQ ID NO: 50) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 MGBPBMY4 (BC007143) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 CACAATGCTTATCCAGGGTAGCT \\n                 3050 \\n                 (SEQ ID NO: 51) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 GTCCCTTGGAGTTAGATTTACAGGTAGT \\n                 3121 \\n                 (SEQ ID NO: 52) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 mGAPDH \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F \\n                 CATGGCCTTCCGTGTTCCTA \\n                 730 \\n                 (SEQ ID NO: 53) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R \\n                 CCTGCTTCACCACCTTCTTGA \\n                 833 \\n                 (SEQ ID NO: 54) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 hGAPDH \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 F3 \\n                 AGCCGAGCCACATCGCT \\n                   \\n                 (SEQ ID NO: 55) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 R1 \\n                 GTGACCAGGCGCCCAATAC \\n                   \\n                 (SEQ ID NO: 56) \\n               \\n            \\n           \\n         \\n       \\n     \\n     RESULTS \\n     In the course of analyzing the chromosomal localization of guanylate binding proteins (GBPs) four GBPs in the human (HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2) and HGBPBMY4 (FLJ10961)) and four mouse GBPs (MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and MGBPBMY4 (BC007143)) were identified. A combined alignment of the human and mouse GBPs is shown in  FIGS. 11A–11G ; and the phylogenetic tree is shown in  FIG. 12 . \\n     It was previously determined that members of the GBP family are highly expressed in hematopoietic tissue (U.S. patent application Ser. No. 10/308,279, incorporated herein by reference). The highest levels of human GBP-1 were detected in spleen and lymph node, followed by leukocytes, lung, placenta, tonsil, liver, thymus and fetal liver. The highest levels of human GBP-5 were detected in spleen, leukocytes, followed by lymph node and lung. High levels of human GBP-2 were detected in leukocytes, followed by spleen, lymph node, lung, placenta, skeletal muscle, tonsil, fetal liver and thymus. The highest levels of human GBP-4 were detected in lung, followed by spleen, lymph node, leukocytes, skeletal muscle, placenta, and heart ( FIG. 14 ). \\n     Of the identified GBPs, HGBPBMY4 (FLJ10961) had the broadest expression pattern. High levels were detected in spleen, followed by lymph node, lung, placenta, leukocytes, tonsil, thymus, kidney, skeletal muscle, liver, heart, pancreas, fetal liver, and bone marrow ( FIG. 15 ). Expression of HGBPBMY2 (4843 30 1 1; 4843 — 1) was restricted to tonsil ( FIG. 16 ). Expression of the novel HGBPBMY3 (4843 30 2 1; 4843 — 2) was highest in spleen and lymph node, followed by placenta, heart, lung, liver, thymus, and tonsil ( FIG. 17 ). Expression of HGBPBMY1 (AK096141) was restricted to liver ( FIG. 18 ). \\n     For the mouse family members, the highest levels of mouse GBP-1 were detected in kidney, with much lower levels in spleen and lung ( FIG. 19 ). Mouse GBP-2 was most abundant in lung and spleen, followed by kidney, heart, and brain ( FIG. 20 ). Mouse GBP-3 was most abundant in spleen, followed by kidney, lung, heart, brain, and testis ( FIG. 21 ). Mouse GBP-4 was abundantly expressed in embryonic tissue. In adult tissue, the highest levels were detected in lung and spleen, followed by testis, heart, brain, liver and kidney ( FIG. 22 ). The highest levels of mouse GBP-5 were detected in spleen, followed by lung, heart, brain, skeletal muscle, kidney, and testis ( FIG. 23 ). \\n     Of the identified mouse GBP family members, MGBPBMY2 (LOC229902) had the broadest expression pattern. High expression was detected in embryonic tissue. In adult tissue, the highest expression was detected in spleen, followed by lung, testis, brain, liver, heart, kidney, and skeletal muscle ( FIG. 24 ). Expression of MGBPBMY1 (LOC229900) was highest in adult spleen, followed by lung, heart, kidney, skeletal muscle, and brain ( FIG. 25 ). Expression of MGBPBMY3 (BC031475) was restricted to spleen and lung ( FIG. 26 ). Expression of the novel MGBPBMY4 (BC007143) was highest in adult heart and kidney, followed by spleen, lung, liver, brain, and skeletal muscle ( FIG. 27 ). \\n     Message levels of all the known GBPs are upregulated in response to IFN-γ. Therefore an experiment was performed to determine whether the identified GBPs are similarly induced by treatment with IFN-γ. Levels of HGBPBMY4 (FLJ10961) ( FIG. 28A ), HGBPBMY2-(4843 — 1) ( FIG. 28B ), and the novel HGBPBMY3 (4843 — 2) ( FIG. 28C ) were all strongly upregulated by treatment with IFN-γ. Much lower levels of expression were induced in response to LPS. Expression of HGBPBMY1 (AK096141) was not detected in resting or stimulated THP-1 cells. \\n     It was previously determined that LPS-mediated induction of GBP-1 and GBP-5 is dependent on NF-κB activity (U.S. patent Ser. No. 10/308,279, incorporated by reference). An experiment was then performed to determine whether inductions of other known GBPs as well as the identified GBPs were similarly dependent on NF-κB activity. Two independent sets of THP-1 monocytes were stimulated with LPS in the presence and absence of a specific inhibitor of NF-κB nuclear localization, Compound 1 (Fujihara et al., (2000)  J. Immunol.  165:1004–1012). Expression of GBP-2 was induced in response to LPS stimulation. This induction was significantly inhibited by Compound 1 ( FIG. 29 ). In contrast to GBP-2, induction of GBP-4 in response to LPS was variable ( FIG. 30 ). The expression of GBP-4 was not affected by the NF-κB inhibitor. A similar pattern of expression was observed for HGBPBMY4 (FLJ10961). The response to LPS was variable, and unaffected by Compound 1 ( FIG. 31 ). Expression of the novel HGBPBMY3 (4843 30 2 1; 4843 — 2) was significantly induced by LPS. The induction was significantly inhibited by Compound 1 ( FIG. 32 ). Expression of HGBPBMY2 (4843 30 1 1; 4843 — 1) and HGBPBMY1 (AK096141) was not detected in resting and LPS-stimulated THP-1 cells. \\n     To determine whether expression of the mouse GBPs was dependent on NF-κB activity, their expression in mouse embryonic fibroblasts derived from germline knockouts of different NF-κB family members as profiled. Wild type 3T3 cells, embryonic fibroblasts derived from germline knockouts of p65, RelB, p50, and IκBα were stimulated for 2 or 8 hours with either TNFα or PMA. At each time point, mRNA was isolated and real time PCR was performed mGBP-2 expression was induced in wild type fibroblasts in response to TNFα ( FIG. 33 ). Significantly less induction was seen in fibroblasts derived from the p65 knockouts, suggesting that p65 is required for mGBP-2 expression. Induction was maintained in fibroblasts from the RelB and p50 knockouts mGBP-2 expression was superinduced in the IκBα-deficient fibroblasts, suggesting that IκBα negatively regulates mGBP-2 expression. IκBα is a known inhibitor of NF-κB activity (Baeuerle et al., (1988)  Science  242:540–545). \\n     Lower levels of mGBP-3 were detected in wild type fibroblasts. The message was induced with TNFα treatment ( FIG. 34 ). This induction was absent in fibroblasts derived from p65-deficient mice, suggesting that p65 regulates mGBP-3 expression. The induction was normal in fibroblasts derived from the RelB knockouts. Expression was superinduced in fibroblasts derived from the p50 and IkBα germline knockouts, suggesting that both proteins negatively regulate mGBP-3 expression. Homodimers of p50 have also been shown to repress certain genes (Plaksin et al., (1993)  J. Exp. Med.  177:1651–1662). \\n     In contrast to mGBP-2 and mGBP-3, mGBP-4 was constitutively expressed in wild type fibroblasts ( FIG. 35 ). Expression was maintained in fibroblast lines derived from the different NF-κB knockout lines. These data suggest that mGBP-4 is not regulated by NF-κB. This is also consistent with the NF-κB independent expression of human GBP-4 described above. \\n     Of the identified mouse GBPs, very low levels of MGBPBMY1 (LOC229900) were detected in wild type fibroblasts ( FIG. 36 ). The message was superinduced in fibroblasts derived from the p50 and IκBα knockouts, suggesting that some NF-κB family members are able to regulate MGBPBMY1 (LOC229900) expression. Expression of MGBPBMY2 (LOC229902) was low and variable in the different fibroblast lines ( FIG. 37 ). No consistent differences were observed between the fibroblast lines, suggesting that MGBPBMY2 (LOC229902) is not regulated by NF-κB. Expression of the novel MGBPBMY4 (BC007143) was inducible by TNFα at two hours, and by PMA at 8 hours ( FIG. 38 ). No induction was observed in fibroblasts derived from the p65 knockouts. Reduced induction was observed in the RelB knockout line. Superinduction was observed at 8 hours in the TNFα-stimulated p50 and IκBα knockout lines. These data are consistent with NF-κB-dependent regulation of the novel MGBPBMY4 (BC007143). Expression of MGBPBMY3 (BC031475) was not detected in any of the fibroblast lines. \\n     High expression of GBP-1 and GBP-5 in synovial tissue derived from rheumatoid arthritis subjects as compared to osteoarthritis subjects or normal controls (U.S. patent application Ser. No. 10/308,279) was previously detected. Therefore an experiment was performed to determine whether other members of the GBP family were overexpressed in synovia derived from rheumatoid arthritis subjects. High expression of GBP-2 was detected in six out of six rheumatoid arthritis synovial samples, and in two out of six osteoarthritis synovial samples ( FIG. 39 ). High levels of GBP-4 expression were detected in five out of six rheumatoid arthritis synovial samples, and in two out of six osteoarthritis samples ( FIG. 40 ). High levels of HGBPBMY2 (4843 30 1 1; 4843 — 1) expression were detected in three out of six rheumatoid arthritis synovial samples, and in two out of six osteoarthritis samples ( FIG. 41 ). High levels of HGBPBMY4 (FLJ10961) expression were detected in six out of six rheumatoid arthritis samples, and in two out of six osteoarthritis samples ( FIG. 42 ). Expression of HGBPBMY1 (AK096141) was not detected in the synovial samples. \\n     Overexpression of several members of the GBP family in rheumatoid arthritis-derived and osteoarthritis-derived synovial tissue suggests that this family might play an important role in inflammatory disease pathology. Members of this family therefore have utility as biomarkers of inflammatory disease and as therapeutic targets for the treatment of inflammatory diseases. \\n     Example 5 \\n     Method of Assessing the Physiological Function of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 1), HGBPBMY3 (4843 30 2 1; 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide at the Cellular Level \\n     The physiological function of the a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (e.g., SEQ ID NO:Y) can be assessed by expressing a sequence encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJI10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide at physiologically elevated levels in mammalian cell culture systems. cDNA is subcloned into a mammalian expression vector containing a strong promoter that drives high levels of cDNA expression (examples are provided elsewhere herein). Vectors of choice include pCMV SPORT (Life Technologies, Rockville, Md., USA) and pCR3.1 (Invitrogen, Carlsbad Calif., USA), both of which contain the cytomegalovirus promoter. 5–10, μg of recombinant vector are transiently transfected into a human cell line, for example of endothelial or hematopoietic origin, using either liposome formulations or electroporation. 1–2 μg of an additional plasmid containing sequences encoding a marker protein are cotransfected. Expression of a marker protein provides a means to distinguish transfected cells from nontransfected cells and is a reliable predictor of cDNA expression from the recombinant vector. Marker proteins of choice include, e.g., Green Fluorescent Protein (GFP; Clontech, Palo Alto, Calif., USA), CD64, or a CD64-GFP fusion protein. \\n     Flow cytometry (FCM), an automated, laser optics-based technique, can be used to identify transfected cells expressing GFP or CD64-GFP and to evaluate the apoptotic state of the cells and other cellular properties. FCM detects and quantifies the uptake of fluorescent molecules that diagnose events preceding or coincident with cell death. These events can include changes in nuclear DNA content as measured by staining of DNA with propidium iodide; changes in cell size and granularity as measured by forward light scatter and 90 degree side light scatter; down-regulation of DNA synthesis as measured by decrease in bromodeoxyuridine uptake; alterations in expression of cell surface and intracellular proteins as measured by reactivity with specific antibodies; and alterations in plasma membrane composition as measured by the binding of fluorescein-conjugated Annexin V protein to the cell surface. Methods in flow cytometry are discussed in Ormerod,  Flow Cyometry , Oxford, New York, N.Y., USA (1994). \\n     The influence of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide on gene expression can be assessed using highly purified populations of cells transfected with sequences encoding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ-10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) and either CD64 or CD64-GFP. CD64 and CD64-GFP are expressed on the surface of transfected cells and bind to conserved regions of human immunoglobulin G (IgG). Transfected cells are efficiently separated from nontransfected cells using magnetic beads coated with either human IgG or antibody against CD64 (DYNAL, Lake Success N.Y., USA). mRNA can be purified from the cells using methods well known by those of ordinary skill in the art. Expression of mRNA encoding HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptides and other genes of interest can be analyzed by northern analysis or microarray techniques. \\n     Example 6 \\n     Method of Assessing the Physiological Function of a HGBPBMY1 (AK096141). HGBPBMY2 (4843 30 1 1; 4843 1), HGBPBMY3 (4843 30 2 1, 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptides Using Microphysiometric Assays \\n     Activation of a wide variety of secondary messenger systems results in extrusion of small amounts of acid from a cell. The acid formed is largely as a result of the increased metabolic activity required to fuel the intracellular signaling process. The pH changes in the media surrounding the cell are very small but are detectable by the CYTOSENSOR® microphysiometer (Molecular Devices Ltd., Menlo Park, Calif., USA). The CYTOSENSOR® is thus capable of detecting the activation of a protein, such as a receptor, that is coupled to an energy utilizing intracellular signaling pathway. \\n     Example 7 \\n     Method of Screening for Compounds that Interact with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 1), HGBPBMY3 (4843 30 2 1: 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide \\n     The following assays are designed to identify compounds that bind to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, bind to other cellular proteins that interact with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, and to compounds that interfere with the interaction of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide with other cellular proteins. \\n     Such compounds can include, but are not limited to, other cellular proteins. Specifically, such compounds can include, but are not limited to, peptides, such as, for example, soluble peptides, including, but not limited to Ig-tailed fusion peptides, comprising a portion of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, and members of random peptide libraries (see, e.g., Lam et al., (1991)  Nature  354:82–84; Houghton et al., (1991)  Nature  354:84–86), made of D- and/or L-configuration amino acids, phosphopeptides (including, but not limited to, members of random or partially degenerate phosphopeptide libraries (see, e.g., Songyang et al., (1993)  Cell  72:767–778); antibodies (including, but not limited to, polyclonal, monoclonal, humanized, anti-idiotypic, chimeric or single chain antibodies, and FAb, F(ab′) 2  and FAb expression libary fragments, and epitope-binding fragments thereof), and small organic or inorganic molecules. \\n     Compounds identified via assays such as those described herein can be useful, for example, in elaborating the biological function of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, and for ameliorating symptoms of tumor progression, for example. In instances, for example, whereby a tumor progression state or disorder results from a lower overall level of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) expression, a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, and/or a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide activity in a cell involved in the tumor progression state or disorder, compounds that interact with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide can include ones which accentuate or amplify the activity of the bound a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143). Such compounds would bring about an effective increase in the level of HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide activity, thus ameliorating symptoms of the tumor progression disorder or state. \\n     In instances whereby mutations within a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide cause aberrant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptides to be made which have a deleterious effect that leads to tumor progression, compounds that bind a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide can be identified that inhibit the activity of the bound HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide. Assays for testing the effectiveness of such compounds are known in the art and discussed, elsewhere herein. \\n     Example 8 \\n     Method of Screening, In Vitro, Compounds that Bind to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1, 4843 1), HGBPBMY3 (4843 30 2 1, 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide \\n     In vitro systems can be designed to identify compounds capable of binding a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of the present invention. Compounds identified can be useful, for example, in modulating the activity of wild type and/or mutant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, preferably mutant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, can be useful in elaborating the biological function of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, can be utilized in screens for identifying compounds that disrupt normal HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide interactions, or can in themselves disrupt such interactions. \\n     One assay that can be used to identify compounds that bind to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide involves preparing a reaction mixture of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and the test compound under conditions and for a time sufficient to allow the two components to interact and bind, thus forming a complex which can be removed and/or detected in the reaction mixture. These assays can be conducted in a variety of ways. For example, one method to conduct such an assay would involve anchoring a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or the test substance onto a solid phase and detecting a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide/test compound complex anchored on the solid phase at the end of the reaction. In one embodiment of such a method, a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide can be anchored onto a solid surface, and the test compound, which is not anchored, can be labeled, either directly or indirectly. \\n     In practice, a microtiter plate can conveniently be utilized as the solid phase. The anchored component can be immobilized by non-covalent or covalent attachments. Non-covalent attachment can be accomplished by simply coating the solid surface with a solution of the protein and drying. Alternatively, an immobilized antibody, preferably a monoclonal antibody, specific for the protein to be immobilized can be used to anchor the protein to the solid surface. The surfaces can be prepared in advance and stored. \\n     In order to conduct the assay, the nonimmobilized component is added to the coated surface containing the anchored component. After the reaction is complete, unreacted components are removed (e.g., by washing) under conditions such that any complexes formed will remain immobilized on the solid surface. The detection of complexes anchored on the solid surface can be accomplished in a number of ways. Where the previously immobilized component is pre-labeled, the detection of label immobilized on the surface indicates that complexes were formed. Where the previously nonimmobilized component is not pre-labeled, an indirect label can be used to detect complexes anchored on the surface; e.g., using a labeled antibody specific for the immobilized component (the antibody, in turn, can be directly labeled or indirectly labeled with a labeled anti-Ig antibody). \\n     Alternatively, a reaction can be conducted in a liquid phase, the reaction products separated from unreacted components, and complexes detected; e.g., using an immobilized antibody specific for a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or the test compound to anchor any complexes formed in solution, and a labeled antibody specific for the other component of the possible complex to detect anchored complexes. \\n     Another example of a screening assay to identify compounds that bind to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, relates to the application of a cell membrane-based scintillation proximity assay (“SPA”). Such an assay would require the idenification of a ligand for a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide. Once identified, unlabeled ligand is added to assay-ready plates that would serve as a positive control. The SPA beads and membranes are added next, and then  125 I-labeled ligand is added. After an equilibration period of 2–4 hours at room temperature, the plates can be counted in a scintillation counting machine, and the percent inhibition or stimulation calculated. \\n     Such an SPA assay can be based upon a manual, automated, or semi-automated platform, and encompass 96, 384, 1536-well plates or more. Any number of SPA beads can be used as applicable to each assay. The utilized membranes can also be derived from a number of cell line and tissue sources depending upon the expression profile of the respective polypeptide and the adaptability of such a cell line or tissue source to the development of a SPA-based assay. Examples of membrane preparations include, for example, cell lines transformed to express the receptor to be assayed in CHO cells or HEK cells, for example. SPA-based assays are well known in the art and are encompassed by the present invention. One such assay is described in U.S. Pat. No. 4,568,649, which is incorporated herein by reference The skilled artisan would acknowledge that certain modifications of known SPA assays may be required to adapt such assays to each respective polypeptide. \\n     One such screening procedure involves the use of melanophores which are transfected to express a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of the present invention. Such a screening technique is described in PCT WO 92/01810. Such an assay can be employed to screen for a compound which inhibits activation of the receptor polypeptide of the present invention by contacting the melanophore cells which encode the receptor with both the receptor ligand, such as LPA, and a compound to be screened. Inhibition of the signal generated by the ligand indicates that a compound is a potential antagonist for the receptor, i.e., inhibits activation of the receptor. \\n     The technique can also be employed for screening of compounds thta activate the receptor by contacting such cells with compounds to be screened and determining whether such compound generates a signal, i.e., activates the receptor. Other screening techniques include the use of cells which express a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide (for example, transfected CHO cells) in a system which measures extracellular pH changes caused by receptor activation. In this technique, compounds can be contacted with cells expressing the receptor polypeptide of the present invention. A second messenger response, e.g., signal transduction or pH changes, is then measured to determine whether the potential compound activates or inhibits the receptor. \\n     Another method involves screening for HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide inhibitors by determining inhibition or stimulation of HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide-mediated cAMP and/or adenylate cyclase accumulation or dimunition. Such a method involves transiently or stably transfecting a eukaryotic cell with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide to express the polypeptide on the cell surface. \\n     The cell is then exposed to potential antagonists or agonists in the presence of HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide ligand. The changes in levels of cAMP is then measured over a defined period of time, for example, by radio-immuno or protein binding assays (for example using Flashplates or a scintillation proximity assay). Changes in cAMP levels can also be determined by directly measuring the activity of the enzyme, adenylyl cyclase, in broken cell preparations. If the potential antagonist or agonist binds the receptor, and thus inhibits HGPRBMY30 — 1, HGPRBMY30 — 2, HGPRBMY30 — 3, HGPRBMY41 — 1, HGPRBMY41 — 2, HGPRBMY41 — 3, HGPRBMY42, HGPRBMY42 — 1, HGPRBMY43, and/or HGPRBMY44 polypeptide-ligand binding, the levels of HGPRBMY30 — 1, HGPRBMY30 — 2, HGPRBMY30 — 3, HGPRBMY41 — 1, HGPRBMY41 — 2, HGPRBMY41 — 3, HGPRBMY42, HGPRBMY42 — 1, HGPRBMY43, and/or HGPRBMY44 polypeptide-mediated cAMP, or adenylate cyclase activity, will be reduced or increased. \\n     Alternatively, a GTPase assay can be employed to identify a binding event. For example, the GBP&#39;s of the present invention possess GTPase activity. Thus, a screening method can employ this property to identify compounds that bind to a GBP of the present invention. In one embodiment, a test compound is a GTP analog and liberates one or more phosphate groups when acted upon by a GBP of the present invention. Various GTPase assays are known in the art and can be employed in the present invention. For example, a test compound can comprise a detectable label. When a GBP acts on the test compound, the label is liberated and can be detected and/or quantitated. \\n     Further, a labeled test compound can be employed and the formation of a GBP-test compound complex can be detected after removing unbound test compound. \\n     Examples of labels that can be employed in a screening process include  32 P and other radiolabels, as well as various fluorescent moieties. \\n     Example 9 \\n     Method for Identifying a Putative Ligand for a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 1), HGBPBMY3 (4843 30 2 1; 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide \\n     Ligand binding assays provide a direct method for ascertaining protein pharmacology and are adaptable to a high throughput format. A panel of known GBP purified ligands can be radiolabeled to high specific activity (50–2000 Ci/mmol) for binding studies. A determination is then made that the process of radiolabeling does not diminish the activity of the ligand towards its cognate protein. Assay conditions for buffers, ions, pH and other modulators such as nucleotides are optimized to establish a workable signal to noise ratio. For these assays, specific protein binding is defined as total associated radioactivity minus the radioactivity measured in the presence of an excess of unlabeled competing ligand. Where possible, more than one competing ligand is used to define residual nonspecific binding. \\n     A number of GBP ligands are known in the art and are encompassed by the present invention. \\n     Alternatively, a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of the present invention can also be functionally screened (using calcium, cAMP, microphysiometer, oocyte electrophysiology, etc., functional screens) against tissue extracts to identify natural ligands. Extracts that produce positive functional responses can be sequentially subfractionated until an activating ligand is isolated identified using methods well known in the art, some of which are described herein. \\n     Example 10 \\n     Method of Identifying Compounds that Interfere with HGBPBMY1 (AK096141). HGBPBMY2 (4843 30 1 1.4843 1), HGBPBMY3 (4843 30 2; 14843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900). MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide/Cellular Product Interaction \\n     A HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of the present invention can, in vivo, interact with one or more cellular or extracellular macromolecules, such as proteins. Such macromolecules include, but are not limited to, polypeptides, particularly GBP ligands, and those products identified via screening methods described, elsewhere herein. For the purposes of this discussion, such cellular and extracellular macromolecules are referred to herein as “binding partner(s)”. For the purpose of the present invention, the term “binding partner” can also encompass polypeptides, small molecule compounds, polysaccarides, lipids, and any other molecule or molecule type referenced herein. Compounds that disrupt such interactions can be useful in regulating the activity of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide, especially a mutant HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide. Such compounds can include, but are not limited to molecules such as antibodies, peptides, and the like described in elsewhere herein. \\n     The basic principle of the assay systems used to identify compounds that interfere with the interaction between a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and its cellular or extracellular binding partner or partners involves preparing a reaction mixture containing a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and the binding partner under conditions and for a time sufficient to allow the two products to interact and bind, thus forming a complex. \\n     In order to test a compound for inhibitory activity, the reaction mixture is prepared in the presence and absence of the test compound. The test compound can be initially included in the reaction mixture, or can be added at a time subsequent to the addition of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and its cellular or extracellular binding partner. Control reaction mixtures are incubated without the test compound or with a placebo. The formation of any complexes between a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and the cellular or extracellular binding partner is then detected. The formation of a complex in the control reaction, but not in the reaction mixture containing the test compound, indicates that the compound interferes with the interaction of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and the interactive binding partner. Additionally, complex formation within reaction mixtures containing the test compound and a normal HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide can also be compared to complex formation within reaction mixtures containing the test compound and a mutant a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2. (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide. This comparison can be important in those cases in which it is desirable to identify compounds that disrupt interactions of a mutant but not a normal HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide. \\n     The assay for compounds that interfere with the interaction of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and binding partners can be conducted in a heterogeneous or homogeneous format. Heterogeneous assays involve anchoring either a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or the binding partner onto a solid phase and detecting complexes anchored on the solid phase at the end of the reaction. \\n     In homogeneous assays, the entire reaction is carried out in a liquid phase. In either approach, the order of addition of reactants can be varied to obtain different information about the compounds being tested. For example, test compounds that interfere with the interaction between a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and its binding partners, e.g., by competition, can be identified by conducting the reaction in the presence of the test substance; i.e., by adding the test substance to the reaction mixture prior to or simultaneously with a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and interactive cellular or extracellular binding partner. Alternatively, test compounds that disrupt preformed complexes, e.g. compounds with higher binding constants that displace one of the components from the complex, can be tested by adding the test compound to the reaction mixture after complexes have been formed. The various formats are described briefly below. \\n     In a heterogeneous assay system, either a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or the interactive cellular or extracellular binding partner, is anchored onto a solid surface, while the non-anchored species is labeled, either directly or indirectly. In practice, microtiter plates are conveniently utilized. The anchored species can be immobilized by non-covalent or covalent attachments. Non-covalent attachment can be accomplished simply by coating the solid surface with a solution of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or binding partner and drying. Alternatively, an immobilized antibody specific for the species to be anchored can be used to anchor the species to the solid surface. The surfaces can be prepared in advance and stored. \\n     In order to conduct the assay, the partner of the immobilized species is exposed to the coated surface with or without the test compound. After the reaction is complete, unreacted components are removed (e.g., by washing) and any complexes formed will remain immobilized on the solid surface. The detection of complexes anchored on the solid surface can be accomplished in a number of ways. Where the non-immobilized species is pre-labeled, the detection of label immobilized on the surface indicates that complexes were formed. Where the non-immobilized species is not pre-labeled, an indirect label can be used to detect complexes anchored on the surface; e.g., using a labeled antibody specific for the initially non-immobilized species (the antibody, in turn, can be directly labeled or indirectly labeled with a labeled anti-Ig antibody). Depending upon the order of addition of reaction components, test compounds which inhibit complex formation or which disrupt preformed complexes can be detected. \\n     Alternatively, the reaction can be conducted in a liquid phase in the presence or absence of the test compound, the reaction products separated from unreacted components, and complexes detected; e.g., using an immobilized antibody specific for one of the binding components to anchor any complexes formed in solution, and a labeled antibody specific for the other partner to detect anchored complexes. Again, depending upon the order of addition of reactants to the liquid phase, test compounds which inhibit complex or which disrupt preformed complexes can be identified. \\n     In an alternate embodiment of the present invention, a homogeneous assay can be used. In this approach, a preformed complex comprising a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and an interactive cellular or extracellular binding partner product is prepared in which either the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide or its binding partners are labeled, but the signal generated by the label is quenched due to complex formation (see, e.g., U.S. Pat. No. 4,109,496 which describes this approach for immunoassays). The addition of a test substance that competes with and displaces one of the species from the preformed complex will result in the generation of a signal above background. In this way, test substances that disrupt HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide/cellular or extracellular binding partner interaction can be identified. \\n     In a particular embodiment, a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide can be prepared for immobilization using recombinant DNA techniques known in the art. For example, a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide coding region can be fused to a glutathione-S-transferase (GST) gene using a fusion vector such as pGEX-5X-1, in such a manner that its binding activity is maintained in the resulting fusion product. The interactive cellular or extracellular product can be purified and used to raise a monoclonal antibody, using methods routinely practiced in the art and described above. This antibody can be labeled with the radioactive isotope,  125 I, for example, by methods known to those of ordinary skill in the art. In a heterogeneous assay, a GST-HGBPBMY1 (AK096141), GST-HGBPBMY2 (4843 30 1 1; 4843 — 1), GST-HGBPBMY3 (4843 30 2 1; 4843 — 2), GST-HGBPBMY4 (FLJ10961), GST-MGBPBMY1 (LOC229900), GST-MGBPBMY2 (LOC229902), GST-MGBPBMY3 (BC031475) and/or GST-MGBPBMY4 (BC007143) polypeptide fusion product can be anchored to glutathione-agarose beads. The interactive cellular or extracellular binding partner product can then be added in the presence or absence of the test compound in a manner that allows interaction and binding to occur. At the end of the reaction period, unbound material can be washed away, and the labeled monoclonal antibody can be added to the system and allowed to bind to the complexed components. The interaction between a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide and the interactive cellular or extracellular binding partner can be detected by measuring the amount of radioactivity that remains associated with the glutathione-agarose beads. A successful inhibition of the interaction by the test compound will result in a decrease in measured radioactivity. \\n     Alternatively, the GST-HGBPBMY1 (AK096141), GST-HGBPBMY2 (4843 30 1 1; 4843 — 1), GST-HGBPBMY3 (4843 30 2 1; 4843 — 2), GST-HGBPBMY4 (FLJ10961), GST-MGBPBMY1 (LOC229900), GST-MGBPBMY2 (LOC229902), GST-MGBPBMY3 (BC031475) and/or GST-MGBPBMY4 (BC007143) polypeptide fusion product and the interactive cellular or extracellular binding partner product can be mixed together in liquid in the absence of the solid glutathione-agarose beads. The test compound can be added either during or after the binding partners are allowed to interact. This mixture can then be added to the glutathione-agarose beads and unbound material is washed away. Again the extent of inhibition of the binding partner interaction can be detected by adding the labeled antibody and measuring the radioactivity associated with the beads. \\n     In another embodiment of the present invention, these same techniques can be employed using peptide fragments that correspond to the binding domains of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ-10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide product and the interactive cellular or extracellular binding partner (in case where the binding partner is a product), in place of one or both of the full length products. \\n     Any number of methods known to those of ordinary skill in the art can be used to identify and isolate the protein&#39;s binding site. These methods include, but are not limited to, mutagenesis of one of the genes encoding one of the products and screening for disruption of binding in a co-immunoprecipitation assay. Compensating mutations in the gene encoding the second species in the complex can be selected. Sequence analysis of the genes encoding the respective products will reveal the mutations that correspond to the region of the product involved in interactive binding. Alternatively, one product can be anchored to a solid surface using methods described herein, and allowed to interact with and bind to its labeled binding partner, which has been treated with a proteolytic enzyme, such as trypsin. After washing, a short, labeled peptide comprising the binding domain can remain associated with the solid material, which can be isolated and identified by amino acid sequencing. Also, once the gene coding for the cellular or extracellular binding partner product is obtained, short gene segments can be engineered to express peptide fragments of the product, which can then be tested for binding activity and purified or synthesized. \\n     Example 11 \\n     Isolation of a Specific Clone from a Deposited Sample \\n     The deposited material in a sample assigned a ATCC Deposit Number cited in Table I for any given cDNA clone also can contain one or more additional plasmids, each comprising a cDNA clone different from that given clone. Thus, deposits sharing the same ATCC Deposit Number contain at least a plasmid for each cDNA clone identified in Table I. Typically, each ATCC deposit sample cited in Table I comprises a mixture of approximately equal amounts (by weight) of about 1–10 plasmid DNAs, each containing a different cDNA clone and/or partial cDNA clone; but such a deposit sample can include plasmids for more or less than 2 cDNA clones. \\n     Two approaches can be used to isolate a particular clone from the deposited sample of plasmid DNA(s) cited for that clone in Table I. First, a plasmid is directly isolated by screening the clones using a polynucleotide probe corresponding to SEQ ID NO:X. \\n     Particularly, a specific polynucleotide with 30–40 nucleotides is synthesized using an DNA synthesizer (e.g., such as those avaiable from Applied Biosystems, Foster City, Calif., USA) according to the sequence reported. The oligonucleotide is labeled, for instance, with  32 P-labled ATP using T4 polynucleotide kinase and purified according to routine methods. (see, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3rd ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). The plasmid mixture is transformed into a suitable host, as indicated above (such as XL-1 Blue (Stratagene, La Jolla, Calif., USA)) using techniques known to those of skill in the art, such as those provided by the vector supplier or in related publications or patents cited herein. The transformants are plated on 1.5% agar plates (containing the appropriate selection agent, e.g., ampicillin) to a density of about 150 transformants (colonies) per plate. These plates are screened using nylon membranes according to routine methods for bacterial colony screening (e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3d ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)), or other techniques known to those of ordinary skill in the art. \\n     Alternatively, two primers of 17–20 nucleotides derived from both ends of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16 (i.e., within the region of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16 bounded by the 5′ NT and the 3′ NT of the clone defined in Table I; SEQ ID NO:X) are synthesized and used to amplify the desired cDNA using the deposited cDNA plasmid as a template. The polymerase chain reaction is carried out under routine conditions, for instance, in 25 μl of reaction mixture with 0.5 μg of the above cDNA template. A convenient reaction mixture is 1.5–5 mM MgCl 2 , 0.01% (w/v) gelatin, 20 μM each of dATP, dCTP, dGTP, dTTP, 25 pmol of each primer and 0.25 Unit of Taq polymerase. Thirty five cycles of PCR (e.g., denaturation at 94 degree C. for 1 min; annealing at 55 degree C. for 1 min; elongation at 72 degree C. for 1 min) are performed with an automated thermal cycler (e.g., a Perkin-Elmer Cetus 9600 Cycler, Wellesley, Mass., USA). The amplified product is analyzed by agarose gel electrophoresis and the DNA band with expected molecular weight is excised and purified. The PCR product is verified to be the selected sequence by subcloning and sequencing the DNA product. \\n     The polynucleotide(s) of the present invention, the polynucleotide encoding a polypeptide of the present invention, or the polypeptide encoded by the deposited clone may represent partial, or incomplete versions of the complete coding region (i.e., full-length gene). Several methods are known in the art for the identification of the 5′ or 3′ non-coding and/or coding portions of a gene that may not be present in the deposited clone. The methods that follow are exemplary and should not be construed as limiting the scope of the present invention. These methods include but are not limited to, filter probing, clone enrichment using specific probes, and protocols similar or identical to 5′ and 3′ “RACE” protocols that are known in the art. For instance, a method similar to 5′ RACE is available for generating the missing 5′ end of a desired full-length transcript. (Fromont-Racine et al., (1993)  Nucl. Acid Res.  21(7): 1683–1684). \\n     Briefly, a specific RNA oligonucleotide is ligated to the 5′ ends of a population of RNA presumably containing full-length gene RNA transcripts. A primer set containing a primer specific to the ligated RNA oligonucleotide and a primer specific to a known sequence of the gene of interest is used to PCR amplify the 5′ portion of the desired full-length gene. This amplified product may then be sequenced and used to generate the full-length gene. \\n     This method starts with total RNA isolated from the desired source, although poly-A+ RNA can also be used. The RNA preparation can then be treated with phosphatase if necessary to eliminate 5′ phosphate groups on degraded or damaged RNA that may interfere with the later RNA ligase step. The phosphatase should then be inactivated and the RNA treated with tobacco acid pyrophosphatase in order to remove the cap structure present at the 5′ ends of messenger RNAs. This reaction leaves a 5′ phosphate group at the 5′ end of the cap cleaved RNA which can then be ligated to an RNA oligonucleotide using T4 RNA ligase. \\n     This modified RNA preparation is used as a template for first strand cDNA synthesis using a gene specific oligonucleotide. The first strand synthesis reaction is used as a template for PCR amplification of the desired 5′ end using a primer specific to the ligated RNA oligonucleotide and a primer specific to the known sequence of the gene of interest. The resultant product is then sequenced and analyzed to confirm that the 5′ end sequence belongs to the desired gene. Moreover, it may be advantageous to optimize the RACE protocol to increase the probability of isolating additional 5′ or 3′ coding or non-coding sequences. Various methods of optimizing a RACE protocol are known in the art, though a detailed description summarizing these methods can be found in Schaefer, (1995)  Anal. Biochem.  227:255–273. \\n     An alternative method for carrying out 5′ or 3′ RACE for the identification of coding or non-coding sequences is provided by Frohman et al., (1988)  Proc. Natl. Acad. Sci. U.S.A.  85:8998–9002. Briefly, a cDNA clone missing either the 5′ or 3′ end can be reconstructed to include the absent base pairs extending to the translational start or stop codon, respectively. In some cases, cDNAs are missing the start translation codon. \\n     The following briefly describes a modification of this original 5′ RACE procedure. Poly A+ or total RNAs reverse transcribed with Superscript II (Gibco BRL) and an antisense or I complementary primer specific to the cDNA sequence. The primer is removed from the reaction with an Amicon Microcon Concentrator (Millipore, Bedford, Mass., USA). The first-strand cDNA is then tailed with dATP and terminal deoxynucleotide transferase (Gibco BRL). Thus, an anchor sequence is produced which is needed for PCR amplification. The second strand is synthesized from the dA-tail in PCR buffer, Taq DNA polymerase (Perkin-Elmer Cetus, Wellesley, Mass., USA), an oligo-dT primer containing three adjacent restriction sites (XhoI, SalI and ClaI) at the 5′ end and a primer containing just these restriction sites. This double-stranded cDNA is PCR amplified for 40 cycles with the same primers as well as a nested cDNA-specific antisense primer. The PCR products are size-separated on an ethidium bromide-agarose gel and the region of gel containing cDNA products the predicted size of missing protein-coding DNA is removed cDNA is purified from the agarose with the Magic PCR Prep kit (Promega, Madison, Wis., USA), restriction digested with XhoI or SalI, and ligated to a plasmid such as pBluescript SKII (Stratagene, La Jolla, Calif., USA) at XhoI and EcoRV sites. This DNA is transformed into bacteria and the plasmid clones sequenced to identify the correct protein-coding inserts. Correct 5′ ends are confirmed by comparing this sequence with the putatively identified homologue and overlap with the partial cDNA clone. Similar methods known in the art and/or commercial kits are used to amplify and recover 3′ ends. \\n     Several quality-controlled kits are commercially available for purchase. Similar reagents and methods to those above are supplied in kit form from Gibco BRL for both 5′ and 3′ RACE for recovery of full length genes. A second kit is available from Clontech (Palo Alto, Calif., USA) which is a modification of a related technique, SLIC (single-stranded ligation to single-stranded cDNA), developed by Dumas et al., (Dumas et al., (1991)  Nucl. Acid Res.  19:5227–32). The differences in procedure are that the RNA is alkaline hydrolyzed after reverse transcription and RNA ligase is used to join a restriction site-containing anchor primer to the first-strand cDNA. This obviates the necessity for the dA-tailing reaction which results in a polyT stretch that is difficult to sequence past. \\n     An alternative to generating 5′ or 3′ cDNA from RNA is to use cDNA library double-stranded DNA. An asymmetric PCR-amplified antisense cDNA strand is synthesized with an antisense cDNA-specific primer and a plasmid-anchored primer. These primers are removed and a symmetric PCR reaction is performed with a nested cDNA-specific antisense primer and the plasmid-anchored primer. \\n     RNA Ligase Protocol for Generating 5′ or 3′ End Sequences to Obtain Full Length Genes \\n     Once a gene of interest is identified, several methods are available for the identification of the 5′ or 3′ portions of the gene which may not be present in the original cDNA plasmid. These methods include, but are not limited to, filter probing, clone enrichment using specific probes and protocols similar and identical to 5′ and 3′ RACE. While the full-length gene may be present in the library and can be identified by probing, a useful method for generating the 5′ or 3′ end is to use the existing sequence information from the original cDNA to generate the missing information. A method similar to 5′ RACE is available for generating the missing 5′ end of a desired full-length gene. (This method was published by Fromont-Racine et al., (1993)  Nucl. Acid Res.  21(7):1683–1684). Briefly, a specific RNA oligonucleotide is ligated to the 5′ ends of a population of RNA presumably 30 containing full-length gene RNA transcript and a primer set containing a primer specific to the ligated RNA oligonucleotide and a primer specific to a known sequence of the gene of interest, is used to PCR amplify the 5′ portion of the desired full length gene which may then be sequenced and used to generate the full length gene. This method starts with total RNA isolated from the desired source, poly A RNA may be used, but is not a prerequisite for this procedure. The RNA preparation may then be treated with phosphatase if necessary to eliminate 5′ phosphate groups on degraded or damaged RNA that may interfere with the later RNA ligase step. The phosphatase if used is then inactivated and the RNA is treated with tobacco acid pyrophosphatase in order to remove the cap structure present at the 5′ ends of messenger RNAs. This reaction leaves a 5′ phosphate group at the 5′ end of the cap cleaved RNA which can then be ligated to an RNA oligonucleotide using T4 RNA ligase. This modified RNA preparation can then be used as a template for first strand cDNA synthesis using a gene specific oligonucleotide. The first strand synthesis reaction can then be used as a template for PCR amplification of the desired 5′ end using a primer specific to the ligated RNA oligonucleotide and a primer specific to the known sequence of the apoptosis related of interest. The resultant product is then sequenced and analyzed to confirm that the 5′ end sequence belongs to the relevant apoptosis related. \\n     Example 12 \\n     Tissue Distribution of a Polypeptide of the Present Invention \\n     Tissue distribution of mRNA expression of polynucleotides of the present invention is determined using protocols for Northern blot analysis, described by, among others, Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3 rd  ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001). For example, a cDNA probe produced by a method described herein is labeled with  32 P using the REDIPRIME® DNA labeling system (Amersham Life Science, Arlington Heights, Ill., USA), according to manufacturer&#39;s instructions. After labeling, the probe is purified using a CHROMA SPIN0–100 column (Clontech Laboratories, Inc., Palo Alto, Calif., USA) according to the manufacturer&#39;s protocol. The purified, labeled probe is then used to examine various tissues for mRNA expression. \\n     Tissue northern blots containing the bound mRNA of various tissues are examined with the labeled probe using EXPRESSHYB® hybridization solution (Clonetech, Palo Alto, Calif., USA) according to the manufacturer&#39;s protocol. Northern blots can be produced using various protocols known in the art (e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3rd ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). Following hybridization and washing, the blots are mounted and exposed to film at −70° C. overnight, and the films developed according to standard procedures. \\n     Example 13 \\n     Chromosomal Mapping of a Polynucleotide of the Present Invention \\n     An oligonucleotide primer set is designed according to the sequence at the 5′ end of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16. In one embodiment, this primer spans about 100 nucleotides. This primer set is then used in a polymerase chain reaction under the following set of conditions: 30 seconds, 95 degree C.; 1 minute, 56 degree C.; 1 minute, 70 degree C. This cycle is repeated 32 times followed by one 5 minute cycle at 70 degree C. Mammalian DNA, preferably human DNA, is used as template in addition to a somatic cell hybrid panel containing individual chromosomes or chromosome fragments (Bios Laboratories, Inc., New Haven, Conn., USA). The reactions are analyzed on either 8% polyacrylamide gels or 3.5% agarose gels. Chromosome mapping is determined by the presence of an approximately 100 bp PCR fragment in the particular somatic cell hybrid. \\n     Example 14 \\n     Bacterial Expression of a Polypeptide of the Present Invention \\n     A polynucleotide encoding a polypeptide of the present invention is amplified using PCR oligonucleotide primers corresponding to the 5′ and 3′ ends of the DNA sequence, as outlined herein, to synthesize insertion fragments. The primers used to amplify the cDNA insert should preferably contain restriction sites, such as BamHI and XbaI, at the 5′ end of the primers in order to clone the amplified product into the expression vector. For example, BamHI and XbaI correspond to the restriction enzyme sites on the bacterial expression vector pQE-9. (Qiagen, Inc.). This plasmid vector encodes antibiotic resistance (Ampr), a bacterial origin of replication (ori), an IPTG-regulatable promoter/operator (P/O), a ribosome binding site (RBS), a 6-histidine tag (6-His), and restriction enzyme cloning sites. \\n     The pQE-9 vector is digested with BamHI and XbaI and the amplified fragment is ligated into the pQE-9 vector maintaining the reading frame initiated at the bacterial RBS. The ligation mixture is then used to transform  E. coli  strain M15/rep4 (Qiagen, Inc.) which contains multiple copies of the plasmid pREP4, that expresses the lacI repressor and also confers kanamycin resistance (Kan r ). Transformants are identified by their ability to grow on LB plates and ampicillin/kanamycin resistant colonies are selected. Plasmid DNA is isolated and confirmed by restriction analysis. \\n     Clones containing the desired constructs are grown overnight (O/N) in liquid culture in LB media supplemented with both Amp (100 μg/mil) and Kan (25 μg/ml). The O/N culture is used to inoculate a large culture at a ratio of 1:100 to 1:250. The cells are grown to an optical density 600 (O.D.600) of between 0.4 and 0.6. IPTG (isopropyl-B-D-thiogalacto pyranoside) is then added to a final concentration of 1 mM. IPTG induces by inactivating the lacI repressor, clearing the P/O leading to increased gene expression. \\n     Cells are grown for another 3 to 4 hours. Cells are then harvested by centrifugation (20 mins at 6000×g). The cell pellet is solubilized in the chaotropic agent 6 molar Guanidine HCl by stirring for 3–4 hours at 4 degree C. The cell debris is removed by centrifugation, and the supernatant containing the polypeptide is loaded onto a nickel-nitrilo-tri-acetic acid (“Ni-NTA”) affinity resin column (available from QIAGEN, Inc.). Proteins with a 6×His tag bind to the Ni-NTA resin with high affinity and can be purified in a simple one-step procedure (for details see:  The QIAexpressionist  (1995) published by QIAGEN, Inc.). \\n     Briefly, the supernatant is loaded onto the column in 6 M guanidine-HCl, pH 8, the column is first washed with 10 volumes of 6 M guanidine-HCl, pH 8, then washed with 10 volumes of 6 M guanidine-HCl pH 6, and finally the polypeptide is eluted with 6 M guanidine-HCl, pH 5. \\n     The purified protein is then renatured by dialyzing it against phosphate-buffered saline (PBS) or 50 mM Na-acetate, pH 6 buffer plus 200 mM NaCl. Alternatively, the protein can be successfully refolded while immobilized on the Ni-NTA column. The recommended conditions are as follows: renature using a linear 6M-1M urea gradient in 500 mM NaCl, 20% glycerol, 20 mM Tris/HCl pH 7.4, containing protease inhibitors. The renaturation should be performed over a period of 1.5 hours or more. After renaturation the proteins are eluted by the addition of 250 mM imidazole. Imidazole is removed by a final dialyzing step against PBS or 50 mM sodium acetate pH 6 buffer plus 200 mM NaCl. The purified protein is stored at 4 degree C. or frozen at −80 degree C. \\n     Example 15 \\n     Purification of a Polypeptide of the Present Invention from an Inclusion Body \\n     The following alternative method can be used to purify a polypeptide expressed in  E coli  when it is present in the form of inclusion bodies. Unless otherwise specified, all of the following steps are conducted at 4–10 degree C. \\n     Upon completion of the production phase of the  E. coli  fermentation, the cell culture is cooled to 4–10° C. and the cells harvested by continuous centrifugation at 15,000 rpm (Heraeus Sepatech, Osterode, Germany). On the basis of the expected yield of protein per unit weight of cell paste and the amount of purified protein required, an appropriate amount of cell paste, by weight, is suspended in a buffer solution containing 100 mM Tris, 50 mM EDTA, pH 7.4. The cells are dispersed to a homogeneous suspension using a high shear mixer. \\n     The cells are then lysed by passing the solution through a microfluidizer (e.g., such as those available from Microfluidics Corp., Newton, Mass., USA or APV Systems, Wilmington, Mass., USA) twice at 4000–6000 psi. The homogenate is then mixed with NaCl solution to a final concentration of 0.5 M NaCl, followed by centrifugation at 7000×g for 15 min. The resultant pellet is washed again using 0.5M NaCl, 100 mM Tris, 50 mM EDTA, pH 7.4. \\n     The resulting washed inclusion bodies are solubilized with 1.5 M guanidine hydrochloride (GuHCl) for 2–4 hours. After 7000×g centrifugation for 15 min., the pellet is discarded and the polypeptide containing supernatant is incubated at 4 degree C. overnight to allow further GuHCl extraction. \\n     Following high speed centrifugation (30,000×g) to remove insoluble particles, the GuHCl solubilized protein is refolded by quickly mixing the GuHCl extract with 20 volumes of buffer containing 50 mM sodium, pH 4.5, 150 mM NaCl, 2 mM EDTA by vigorous stirring. The refolded diluted protein solution is kept at 4° C. without mixing for 12 hours prior to further purification steps. \\n     To clarify the refolded polypeptide solution, a previously prepared tangential filtration unit equipped with 0.16 μm membrane filter with appropriate surface area (e.g., Filtron), equilibrated with 40 mM sodium acetate, pH 6.0 is employed. The filtered sample is loaded onto a cation exchange resin (e.g., Poros HS-50, Perseptive Biosystems, Foster City Calif., USA). The column is washed with 40 mM sodium acetate, pH 6.0 and eluted with 250 mM, 500 mM, 1000 mM, and 1500 mM NaCl in the same buffer, in a stepwise manner. The absorbance at 280 nm of the effluent is continuously monitored. Fractions are collected and further analyzed by SDS-PAGE. \\n     Fractions containing the polypeptide are then pooled and mixed with 4 volumes of water. The diluted sample is then loaded onto a previously prepared set of tandem columns of strong anion (Poros HQ-500, Perceptive Biosystems, Foster City, Calif., USA) and weak anion (Poros CM-200, Perceptive Biosystems, Foster City, Calif., USA) exchange resins. The columns are equilibrated with 40 mM sodium acetate, pH 6.0. Both columns are washed with 40 mM sodium acetate, pH 6.0, 200 mM NaCl. The CM-20® column is then eluted using a 10 column volume linear gradient ranging from 0.2 M NaCl, 50 mM sodium acetate, pH 6.0 to 1.0 M NaCl, 50 mM sodium acetate, pH 6.5. Fractions are collected under constant A280 monitoring of the effluent. Fractions containing the polypeptide (determined, for instance, by 16% SDS-PAGE) are then pooled. \\n     The resultant polypeptide should exhibit greater than 95% purity after the above refolding and purification steps. No major contaminant bands should be observed from Coomassie blue stained 16% SDS-PAGE gel when 51g of purified protein is loaded. The purified protein can also be tested for endotoxin/LPS contamination, and typically the LPS content is less than 0.1 ng/ml according to LAL assays. \\n     Example 16 \\n     Cloning and Expression of a Polypeptide of the Present Invention in a Baculovirus Expression System \\n     In this example, the plasmid shuttle vector pAc373 is used to insert a polynucleotide into a baculovirus to express a polypeptide. A typical baculovirus expression vector contains the strong polyhedrin promoter of the  Autographa californica  nuclear polyhedrosis virus (AcMNPV) followed by convenient restriction sites, which may include, for example BamHI, Xba I and Asp718. The polyadenylation site of the simian virus 40 (“SV40”) is often used for efficient polyadenylation. For easy selection of recombinant virus, the plasmid contains the beta-galactosidase gene from  E. coli  under control of a weak  Drosophila  promoter in the same orientation, followed by the polyadenylation signal of the polyhedrin gene. The inserted genes are flanked on both sides by viral sequences for cell-mediated homologous recombination with wild-type viral DNA to generate a viable virus that express the cloned polynucleotide. \\n     Many other baculovirus vectors can be used in place of the vector above, such as pVL941 and pAcIMI, as one skilled in the art would readily appreciate, as long as the construct provides appropriately located signals for transcription, translation, secretion and the like, including a signal peptide and an in-frame AUG as required. Such vectors are described, for instance, in Luckow et al., (1989)  Virology  170:31–39. \\n     A polynucleotide encoding a polypeptide of the present invention is amplified using PCR oligonucleotide primers corresponding to the 5′ and 3′ ends of the DNA sequence, as outlined herein, to synthesize insertion fragments. The primers used to amplify the cDNA insert should preferably contain restriction sites at the 5′ end of the primers in order to clone the amplified product into the expression vector. Specifically, the cDNA sequence contained in the deposited clone, including the AUG initiation codon and the naturally associated leader sequence identified elsewhere herein (if applicable), is amplified using the PCR protocol described herein. If the naturally occurring signal sequence is used to produce the protein, the vector used does not need a second signal peptide. Alternatively, the vector can be modified to include a baculovirus leader sequence, using the standard methods described by Summers et al. (Summers et al., (1987) “A Manual of Methods for Baculovirus Vectors and Insect Cell Culture Procedures,” Texas Agricultural Experimental Station Bulletin No. 1555). \\n     The amplified fragment is isolated from a 1% agarose gel using a commercially available kit (GENECLEAN®, BIO 101 Inc., La Jolla, Calif., USA). The fragment then is digested with appropriate restriction enzymes and again purified on a 1% agarose gel. \\n     The plasmid is digested with the corresponding restriction enzymes and optionally, can be dephosphorylated using calf intestinal phosphatase, using routine procedures known in the art. The DNA is then isolated from a 1% agarose gel using a commercially available kit (GENECLEAN®, BIO 101 Inc., La Jolla, Calif., USA). \\n     The fragment and the dephosphorylated plasmid are ligated together with T4 DNA ligase.  E. coli  HB101 or other suitable  E. coli  hosts such as XL-1 Blue (Stratagene Cloning Systems, La Jolla, Calif., USA) cells are transformed with the ligation mixture and spread on culture plates. Bacteria containing the plasmid are identified by digesting DNA from individual colonies and analyzing the digestion product by gel electrophoresis. The sequence of the cloned fragment is confirmed by DNA sequencing. \\n     Five μg of a plasmid containing the polynucleotide is co-transformed with 1.0 μg of a commercially available linearized baculovirus DNA (e.g., BACULOGOLD® baculovirus DNA, Pharmingen, San Diego, Calif., USA), using the lipofection method described by Felgner et al. (Felgner et al., (1987)  Proc. Natl. Acad. Sci. U.S.A.  84:7413–7417). One μg of BACULOGOLD® virus DNA and 5 μg of the plasmid are mixed in a sterile well of a microtiter plate containing 50 μl of serum-free Grace&#39;s medium (Life Technologies Inc., Rockville, Md., USA). Afterwards, 10 μl Lipofectin plus 90 μl Grace&#39;s medium are added, mixed and incubated for 15 minutes at room temperature. Then the transfection mixture is added drop-wise to Sf9 insect cells (ATCC CRL 1711) seeded in a 35 mm tissue culture plate with 1 ml Grace&#39;s medium without serum. The plate is then incubated for 5 hours at 27° C. The transfection solution is then removed from the plate and 1 ml of Grace&#39;s insect medium supplemented with 10% fetal calf serum is added. Cultivation is then continued at 27° C. for four days. \\n     After four days the supernatant is collected and a plaque assay is performed, as described by Summers &amp; Smith, (Summers et al., (1987) “A Manual of Methods for Baculovirus Vectors and Insect Cell Culture Procedures,” Texas Agricultural Experimental Station Bulletin No. 1555). An agarose gel with BLUE GAL (Life Technologies Inc., Rockville, Md., USA) is used to allow easy identification and isolation of gal-expressing clones, which produce blue-stained plaques. (A detailed description of a “plaque assay” of this type can also be found in the user&#39;s guide for insect cell culture and baculovirology distributed by Life Technologies Inc., Rockville, Md., USA, pp. 9–10.) After appropriate incubation, blue stained plaques are picked with the tip of a micropipettor (e.g., Eppendorf). The agar containing the recombinant viruses is then resuspended in a microcentrifuge tube containing 200 μl of Grace&#39;s medium and the suspension containing the recombinant baculovirus is used to infect Sf9 cells seeded in 35 mm dishes. Four days later the supernatants of these culture dishes are harvested and then they are stored at 4 degree C. \\n     To verify the expression of the polypeptide, Sf9 cells are grown in Grace&#39;s medium supplemented with 10% heat-inactivated FBS. The cells are infected with the recombinant baculovirus containing the polynucleotide at a multiplicity of infection (“MOI”) of about 2. If radiolabeled proteins are desired, 6 hours later the medium is removed and is replaced with SF900 II medium minus methionine and cysteine (available from Life Technologies Inc., Rockville, Md., USA). After 42 hours, 5 μCi of  35 S-methionine and 5 RCi  35 S-cysteine (available from Amersham) are added. The cells are further incubated for 16 hours and then are harvested by centrifugation. The proteins in the supernatant as well as the intracellular proteins are analyzed by SDS-PAGE followed by autoradiography (if radiolabeled). \\n     Microsequencing of the amino acid sequence of the amino terminus of purified protein may be used to determine the amino terminal sequence of the produced protein. \\n     Example 17 \\n     Expression of a Polypeptide of the Present Invention in Mammalian Cells \\n     A polypeptide of the present invention can be expressed in a mammalian cell. A typical mammalian expression vector contains a promoter element, which mediates the initiation of transcription of mRNA, a protein coding sequence, and signals required for the termination of transcription and polyadenylation of the transcript. Additional elements include enhancers, Kozak sequences and intervening sequences flanked by donor and acceptor sites for RNA splicing. Highly efficient transcription is achieved with the early and late promoters from SV40, the long terminal repeats (LTRs) from retroviruses, e.g., RSV, HTLVI, HIVI and the early promoter of the cytomegalovirus (CMV). However, cellular elements can also be used (e.g., the human actin promoter). \\n     Suitable expression vectors for use in practicing the present invention include, for example, vectors such as pSVL and pMSG (Pharmacia, Uppsala, Sweden), pRSVcat (ATCC 37152), pSV2dhfr (ATCC 37146), pBC12MI (ATCC 67109), pCMVSport 2.0, and pCMVSport 3.0. Mammalian host cells that could be used include, human Hela, 293, H9 and Jurkat cells, mouse NIH3T3 and C127 cells, Cos 1, Cos 7 and CV1, quail QC 1-3  cells, mouse L cells and Chinese hamster ovary (CHO) cells. \\n     Alternatively, the polypeptide can be expressed in stable cell lines containing the polynucleotide integrated into a chromosome. The co-transformation with a selectable marker such as dhfr, gpt, neomycin, hygromycin allows the identification and isolation of the transformed cells. \\n     The transformed gene can also be amplified to express large amounts of the encoded protein. The DHFR (dihydrofolate reductase) marker is useful in developing cell lines that carry several hundred or even several thousand copies of the gene of interest (see, e.g., Alt et al., (1978)  J. Biol. Chem.  253:1357–1370; Hamlin &amp; Ma, (1990)  Biochim. Biophys. Acta  1097:107–143; Page &amp; Sydenham, (1991)  Biotechnol.  9:64–68). Another useful selection marker is the enzyme glutamine synthase (GS) (Murphy et al., (1991)  Biochem J.  227:277–279; Bebbington et al., (1992)  Bio/Technology  10:169–175). Using these markers, the mammalian cells are grown in selective medium and the cells with the highest resistance are selected. These cell lines contain the amplified gene(s) integrated into a chromosome. Chinese hamster ovary (CHO) and NSO cells are often used for the production of proteins. \\n     A polynucleotide of the present invention is amplified according to the protocol outlined in herein. If the naturally occurring signal sequence is used to produce the protein, the vector does not need a second signal peptide. Alternatively, if the naturally occurring signal sequence is not used, the vector can be modified to include a heterologous signal sequence (see, e.g., PCT Publication WO 96/34891). The amplified fragment is isolated from a 1% agarose gel using a commercially available kit (GENECLEAN®, BIO 101 Inc., La Jolla, Calif., USA). The fragment then is digested with appropriate restriction enzymes and again purified on a 1% agarose gel. \\n     The amplified fragment is then digested with the same restriction enzyme and purified on a 1% agarose gel. The isolated fragment and the dephosphorylated vector are then ligated with T4 DNA ligase.  E. coli  HB11 or XL-1 Blue cells are then transformed and bacteria are identified that contain the fragment inserted into plasmid pC6 using, for instance, restriction enzyme analysis. \\n     Chinese hamster ovary cells lacking an active DHFR gene is used for transformation. Five μg of an expression plasmid is cotransformed with 0.5 ug of the plasmid pSVneo using lipofectin (Felgner et al., (1987)  Proc. Natl. Acad. Sci. USA  84:7413–7417). The plasmid pSV2-neo contains a dominant selectable marker, the neo gene from Tn5 encoding an enzyme that confers resistance to a group of antibiotics including G418. The cells are seeded in alpha minus MEM supplemented with 1 mg/ml G418. After 2 days, the cells are trypsinized and seeded in hybridoma cloning plates (available from Greiner, Germany) in alpha minus MEM supplemented with 10, 25, or 50 ng/ml of methotrexate plus 1 mg/ml G418. After about 10–14 days single clones are trypsinized and then seeded in 6-well petri dishes or 10 ml flasks using different concentrations of methotrexate (50 nM, 100 nM, 200 nM, 400 nM, 800 nM). Clones growing at the highest concentrations of methotrexate are then transferred to new 6-well plates containing even higher concentrations of methotrexate (1 μM, 2 μM, 5 μM, 10 mM, 20 mM). The same procedure is repeated until clones are obtained which grow at a concentration of 100–200 μM. Expression of the desired gene product is analyzed, for instance, by SDS-PAGE and western blot or by reversed phase HPLC analysis. \\n     Example 18 \\n     Method of Creating N- and C-terminal Deletion Mutants Corresponding to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 1), HGBPBMY3 (4843 30 2 1:4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptide of the Present Invention \\n     As described elsewhere herein, the present invention encompasses the creation of N- and C-terminal deletion mutants, in addition to any combination of N- and C-terminal deletions thereof, corresponding to a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide of the present invention. A number of methods are available to one of ordinary skill in the art for creating such mutants. Such methods may include a combination of PCR amplification and gene cloning methodology. Although one of skill in the art of molecular biology, through the use of the teachings provided or referenced herein, and/or otherwise known in the art as standard methods, could readily create each deletion mutant of the present invention, exemplary methods are described below. \\n     Briefly, using the isolated cDNA clone encoding a full-length HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) polypeptide sequence (as described in herein, for example, in SEQ ID NO:Y), appropriate primers of about 15–25 nucleotides derived from the desired 5′ and 3′ positions of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16 (SEQ ID NO:X) may be designed to PCR amplify, and subsequently clone, the intended N- and/or C-terminal deletion mutant. Such primers could comprise, for example, an inititation and stop codon for the 5′ and 3′ primer, respectively. Such primers may also comprise restriction sites to facilitate cloning of the deletion mutant post amplification. Moreover, the primers may comprise additional sequences, such as, for example, FLAG® (Sigma, St. Louis, Mo., USA) tag sequences, kozac sequences, or other sequences discussed and/or referenced herein. \\n     Representative PCR amplification conditions are provided below, although the skilled artisan would appreciate that other conditions may be required for efficient amplification. A 100 μl PCR reaction mixture may be prepared using 10 ng of the template DNA (cDNA clone of HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143)), 200 μM 4dNTPs, 1 μM primers, 0.25U Taq DNA polymerase (PE), and standard Taq DNA polymerase buffer. Representative PCR cycling condition are as follows: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n                 20–25 cycles: \\n                 45 sec, 93 degrees \\n               \\n               \\n                   \\n                   \\n                 \\u20022 min, 50 degrees \\n               \\n               \\n                   \\n                   \\n                 \\u20022 min, 72 degrees \\n               \\n               \\n                   \\n                 1 cycle: \\n                 10 min, 72 degrees \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     After the final extension step of PCR, 5U Klenow Fragment may be added and incubated for 15 min at 30 degrees. \\n     Upon digestion of the fragment with the NotI and SailI restriction enzymes, the fragment could be cloned into an appropriate expression and/or cloning vector which has been similarly digested (e.g., pSport1, among others). The skilled artisan would appreciate that other plasmids could be equally substituted, and may be desirable in certain circumstances. The digested fragment and vector are then ligated using a DNA ligase, and then used to transform competent  E. coli  cells using methods provided herein and/or otherwise known in the art. \\n     The 5′ primer sequence for amplifying any additional N-terminal deletion mutants may be determined by reference to the following formula: (S+(X*3)) to ((S+(X*3))+25), wherein ‘S’ is equal to the nucleotide position of the initiating start codon of a HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBM3 and/or MGBPBMY4 (BC007143) gene (i.e., SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16), and ‘X’ is equal to the most N-terminal amino acid of the intended N-terminal deletion mutant. The first term will provide the start 5′ nucleotide position of the 5′ primer, while the second term will provide the end 3′ nucleotide position of the 5′ primer corresponding to sense strand of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16. Once the corresponding nucleotide positions of the primer are determined, the final nucleotide sequence may be created by the addition of applicable restriction site sequences to the 5′ end of the sequence, for example. As referenced herein, the addition of other sequences to the 5′ primer may be desired in certain circumstances (e.g., kozac sequences, etc.). \\n     The 3′ primer sequence for amplifying any additional N-terminal deletion mutants may be determined by reference to the following formula: (S+(X*3)) to ((S+(X*3))-25), wherein ‘S’ is equal to the nucleotide position of the initiating start codon of the HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBM3 and/or MGBPBMY4 (BC007143) gene (SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16), and ‘X’ is equal to the most C-terminal amino acid of the intended N-terminal deletion mutant. The first term will provide the start 5′ nucleotide position of the 3′ primer, while the second term will provide the end 3′ nucleotide position of the 3′ primer corresponding to the anti-sense strand of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16. Once the corresponding nucleotide positions of the primer are determined, the final nucleotide sequence may be created by the addition of applicable restriction site sequences to the 5′ end of the sequence, for example. As referenced herein, the addition of other sequences to the 3′ primer may be desired in certain circumstances (e.g., stop codon sequences, etc.). The skilled artisan would appreciate that modifications of the above nucleotide positions may be necessary for optimizing PCR amplification. \\n     The same general formulas provided above may be used in identifying the 5′ and 3′ primer sequences for amplifying any C-terminal deletion mutant of the present invention. Moreover, the same general formulas provided above may be used in identifying the 5′ and 3′ primer sequences for amplifying any combination of N-terminal and C-terminal deletion mutant of the present invention. The skilled artisan would appreciate that modifications of the above nucleotide positions may be necessary for optimizing PCR amplification. \\n     Example 19 \\n     Protein Fusions Comprising a Polypeptide of the Present Invention \\n     The polypeptides of the present invention are preferably fused to other proteins. These fusion proteins can be used for a variety of applications. For example, fusion of the present polypeptides to His-tag, HA-tag, protein A, IgG domains, and maltose binding protein facilitates purification (see the Examples described herein; see also EP A 394,827; Traunecker et al., (1988)  Nature  331:84–86). Similarly, fusion to IgG-1, IgG-3, and albumin increases the half-life time in vivo. Nuclear localization signals fused to the polypeptides of the present invention can target the protein to a specific subcellular localization, while covalent heterodimer or homodimers can increase or decrease the activity of a fusion protein. Fusion proteins can also create chimeric molecules having more than one function. Finally, fusion proteins can increase solubility and/or stability of the fused protein compared to the non-fused protein. All of the types of fusion proteins described above can be made by modifying the following protocol, which outlines the fusion of a polypeptide to an IgG molecule. \\n     Briefly, the human Fc portion of the IgG molecule can be PCR amplified, using primers that span the 5′ and 3′ ends of the sequence described below. These primers also should have convenient restriction enzyme sites that will facilitate cloning into an expression vector, preferably a mammalian expression vector. Note that the polynucleotide is cloned without a stop codon, otherwise a fusion protein will not be produced. \\n     The naturally occurring signal sequence may be used to produce the protein (if applicable). Alternatively, if the naturally occurring signal sequence is not used, the vector can be modified to include a heterologous signal sequence (see, e.g., PCT Publication WO 96/34891 and/or U.S. Pat. No. 6,066,781). \\n     Human IgG Fc region: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 gggatccggagcccaaatcttctgacaaaactcacacatgcccaccgtgcccagcacctgaattcgagggtgcaccgtca \\n                 (SEQ ID NO: 59) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 gtcttcctcttccccccaaaacccaaggacaccctcatgatctcccggactcctgaggtcacatgcgtggtggtggacgtaa \\n               \\n               \\n                   \\n               \\n               \\n                 gccacgaagaccctgaggtcaagttcaactggtacgtggacggcgtggaggtgcataatgccaagacaaagccgcggg \\n               \\n               \\n                   \\n               \\n               \\n                 aggagcagtacaacagcacgtaccgtgtggtcagcgtcctcaccgtcctgcaccaggactggctgaatggcaaggagta \\n               \\n               \\n                   \\n               \\n               \\n                 caagtgcaaggtctccaacaaagccctcccaacccccatcgagaaaaccatctccaaagccaaagggcagccccgaga \\n               \\n               \\n                   \\n               \\n               \\n                 accacaggtgtacaccctgcccccatcccgggatgagctgaccaagaaccaggtcagcctgacctgcctggtcaaaggc \\n               \\n               \\n                   \\n               \\n               \\n                 ttctatccaagcgacatcgccgtggagtgggagagcaatgggcagccggagaacaactacaagaccacgcctcccgtgc \\n               \\n               \\n                   \\n               \\n               \\n                 tggactccgacggctccttcttcctctacagcaagctcaccgtggacaagagcaggtggcagcaggggaacgtcttctcat \\n               \\n               \\n                   \\n               \\n               \\n                 gctccgtgatgcatgaggctctgcacaaccactacacgcagaagagcctctccctgtctccgggtaaatgagtgcgacgg \\n               \\n               \\n                   \\n               \\n               \\n                 ccgcgactctagaggat \\n               \\n            \\n           \\n         \\n       \\n     \\n     Example 20 \\n     Production of an Antibody from a Polypeptide of the Present Invention \\n     Antibodies of the present invention can be prepared by a variety of methods. As one example of such methods, cells expressing a polypeptide of the present invention are administered to an animal to induce the production of sera containing polyclonal antibodies. In a representative method, a preparation of the polypeptide is prepared and purified to render it substantially free of natural contaminants. Such a preparation is then introduced into an animal in order to produce polyclonal antisera of greater specific activity. \\n     In another representative method, the antibodies of the present invention are monoclonal antibodies (or protein binding fragments thereof). Such monoclonal antibodies can be prepared using hybridoma technology (Köhler et al., (1975)  Nature  256:495; Köhler et al., (1976)  Eur. J. Immunol.  6:511; Köhler et al., (1976)  Eur. J. Immunol.  6:292; Hammerling et al., in:  Monoclonal Antibodies and T - Cell Hybridomas , Elsevier, New York, N.Y., USA, pp. 563–681 (1981)). In general, such procedures involve immunizing an animal (e.g., a mouse or rabbit) with polypeptide or, alternatively, with a polypeptide-expressing cell. Such cells may be cultured in any suitable tissue culture medium; however, it is preferable to culture cells in Earle&#39;s modified Eagle&#39;s medium supplemented with 10% fetal bovine serum (inactivated at about 56° C.), and supplemented with about 10 g/l of nonessential amino acids, about 1,000 U/ml of penicillin, and about 100 ug/ml of streptomycin. \\n     The splenocytes of such animals are extracted and fused with a suitable myeloma cell line. Any suitable myeloma cell line may be employed in accordance with the present invention; however, it is preferable to employ the parent myeloma cell line (SP20), available from the ATCC. After fusion, the resulting hybridoma cells are selectively maintained in HAT medium, and then cloned by limiting dilution as described by Wands et al. (Wands et al., (1981)  Gastroenterology  80:225–232). The hybridoma cells obtained through such a selection are then assayed to identify clones that secrete antibodies capable of binding the polypeptide. \\n     Alternatively, additional antibodies capable of binding to the polypeptide can be produced in a two-step procedure using anti-idiotypic antibodies. Such a method makes use of the fact that antibodies are themselves antigens, and therefore, it is possible to obtain an antibody that binds to a second antibody. In accordance with this method, protein specific antibodies are used to immunize an animal, for example a mouse or rabbit. The splenocytes of such an animal are then used to produce hybridoma cells, and the hybridoma cells are screened to identify clones that produce an antibody whose ability to bind to the protein-specific antibody can be blocked by the polypeptide. Such antibodies comprise anti-idiotypic antibodies to the protein-specific antibody and can be used to immunize an animal to induce formation of further protein-specific antibodies. \\n     It will be appreciated that Fab and F(ab′) 2  and other fragments of the antibodies of the present invention may be used according to the methods disclosed herein. Such fragments are typically produced by proteolytic cleavage, using enzymes such as papain (to produce Fab fragments) or pepsin (to produce F(ab′) 2  fragments). Alternatively, protein-binding fragments can be produced through the application of recombinant DNA technology or through synthetic chemistry. \\n     For in vivo use of antibodies in humans, it may be preferable to use “humanized” chimeric monoclonal antibodies. Such antibodies can be produced using genetic constructs derived from hybridoma cells producing the monoclonal antibodies described herein. Methods for producing chimeric antibodies are known in the art (see, for a review, Morrison, (1985)  Science  229:1202; Oi et al., (1986)  BioTechniques  4:214; U.S. Pat. No. 4,816,567; EP 171496; EP 173494; PCT Publications WO 8601533 and WO 8702671; Boulianne et al., (1984)  Nature  312:643; Neuberger et al., (1985)  Nature  314:268). \\n     Moreover, in another representative method, antibodies directed against the polypeptides of the present invention may be produced in plants. Specific methods are disclosed in U.S. Pat. Nos. 5,959,177, and 6,080,560, which are hereby incorporated in their entirety herein. The methods not only describe methods of expressing antibodies, but also the means of assembling foreign multimeric proteins in plants (i.e., antibodies, etc,), and the subsequent secretion of such antibodies from the plant. \\n     Example 21 \\n     Regulation of Protein Expression via Controlled Aggregation in the Endoplasmic Reticulum \\n     As described more particularly herein, proteins regulate diverse cellular processes in higher organisms, ranging from rapid metabolic changes to growth and differentiation. Increased production of specific proteins could be used to prevent certain diseases and/or disease states. Thus, the ability to modulate the expression of specific proteins in an organism would provide significant benefits. \\n     Numerous methods have been developed to date for introducing foreign genes, either under the control of an inducible, constitutively active, or endogenous promoter, into organisms. Of particular interest are the inducible promoters (see, e.g., Gossen et al., (1992)  Proc. Natl. Acad. Sci. USA.  89:5547; Wang et al., (1994)  Proc. Natl. Acad. Sci. USA  91:8180; No. et al., (1996)  Proc. Natl. Acad. Sci. U.S.A.  93:3346; and Rivera et al., (1996)  Nature Med.  2:1028; in addition to additional examples disclosed elsewhere herein). In one example, the gene for erthropoietin (epo) was transferred into mice and primates under the control of a small molecule inducer for expression (e.g., tetracycline or rapamycin) (see, Bohl et al., (1998)  Blood  92:1512; Rendahl et al., (1998)  Nat. Biotech.  16:757; Rivera et al., (1999)  Proc. Natl. Acad. Sci. USA  96:8657; and Ye et al., (1999)  Science  283:88). Although such systems enable efficient induction of the gene of interest in the organism upon addition of the inducing agent (i.e., tetracycline, rapamycin, etc.), the levels of expression tend to peak at 24 hours and trail off to background levels after 4 to 14 days. Thus, controlled transient expression is difficult to attain using these systems, though such control would be desirable. \\n     An alternative method of controlling gene expression levels of a protein from a transgene (i.e., including stable and transient transformants) has recently been elucidated (Rivera et al., (2000)  Science  287:826–830). This method does not control gene expression at the level of the mRNA like the aforementioned systems. Rather, the system controls the level of protein in an active secreted form. In the absence of the inducing agent, the protein aggregates in the ER and is not secreted. However, addition of the inducing agent results in de-aggregation of the protein and the subsequent secretion from the ER. Such a system affords low basal secretion, rapid, high level secretion in the presence of the inducing agent, and rapid cessation of secretion upon removal of the inducing agent. In fact, protein secretion reached a maximum level within 30 minutes of induction, and a rapid cessation of secretion within 1 hour of removing the inducing agent. The method is also applicable for controlling the level of production for membrane proteins. \\n     Detailed methods are presented in Rivera et al., (2000)  Science  287:826–830). Briefly, fusion protein constructs are created using polynucleotide sequences of the present invention with one or more copies (for example, at least 2, 3, 4, or more) of a conditional aggregation domain (CAD) a domain that interacts with itself in a ligand-reversible manner (i.e., in the presence of an inducing agent) using molecular biology methods known in the art and discussed elsewhere herein. The CAD domain can be, for example, the mutant domain isolated from the human FKBP12 (Phe 36  to Met) protein (as disclosed in Rivera et al., (2000)  Science  287:826–830), or alternatively other proteins having domains with similar ligand-reversible, self-aggregation properties. As a principle of design the fusion protein vector would contain a furin cleavage sequence operably linked between the polynucleotides of the present invention and the CAD domains. Such a cleavage site would enable the proteolytic cleavage of the CAD domains from a polypeptide of the present invention subsequent to secretion from the ER and upon entry into the trans-Golgi (Denault et al., (1996)  FEBS Lett.  379:113). Alternatively, those of ordinary skill in the art would recognize that any proteolytic cleavage sequence could be substituted for the furin sequence provided the substituted sequence is cleavable either endogenously (e.g., the furin sequence) or exogenously (e.g., post secretion, post purification, post production, etc.). The preferred sequence of each feature of the fusion protein construct, from the 5′ to 3′ direction with each feature being operably linked to the other, would be a promoter, signal sequence, “X” number of (CAD) x  domains, the furin sequence (or other proteolytic sequence), and the coding sequence of a polypeptide of the present invention. Those of ordinary skill in the art will appreciate that the promotor and signal sequence, independent from the other, could be either the endogenous promotor or signal sequence of a polypeptide of the present invention, or alternatively, could be a heterologous signal sequence and promotor. \\n     The specific methods described herein for controlling protein secretion levels through controlled ER aggregation are not meant to be limiting are would be generally applicable to any of the polynucleotides and polypeptides of the present invention, including variants, homologues, orthologs, and fragments therein. \\n     Example 22 \\n     Alteration of Protein Glycosylation Sites to Enhance Characteristics of a Polypeptide of the Present Invention \\n     Many eukaryotic cell surface proteins are post-translationally processed to incorporate N-linked and O-linked carbohydrates (Kornfeld &amp; Kornfeld, (1985)  Ann. Rev. Biochem.  54:631–64; Rademacher et al., (1988)  Ann. Rev. Biochem.  57:785–838). Protein glycosylation is thought to serve a variety of functions including: augmentation of protein folding, inhibition of protein aggregation, regulation of intracellular trafficking to organelles, increasing resistance to proteolysis, modulation of protein antigenicity, and mediation of intercellular adhesion (Fieldler &amp; Simons, (1995)  Cell  81:309–312; Helenius, (1994)  Mol. Biol. Cell  5:253–265; Olden et al., (1978)  Cell  13:461–473; Caton et al., (1982)  Cell  37:417–427; Alexander &amp; Elder, (1984)  Science  226:1328–1330; and Flack et al., (1994)  J. Biol. Chem.  269:14015–14020). In higher organisms, the nature and extent of glycosylation can markedly affect the circulating half-life and bio-availability of proteins by mechanisms involving receptor mediated uptake and clearance (Ashwell &amp; Morrell, (1974)  Adv. Enzymol.  41:99–128; Ashwell &amp; Harford, (1982)  Ann. Rev. Biochem.  51:531–54). Receptor systems have been identified that are thought to play a major role in the clearance of serum proteins through recognition of various carbohydrate structures on the glycoproteins (Stockert, (1995)  Physiol. Rev.  75:591–609; Kery et al., (1992)  Arch. Biochem. Biophys.  298:49–55). Thus, production strategies resulting in incomplete attachment of terminal sialic acid residues might provide a means of shortening the bioavailability and half-life of glycoproteins. Conversely, expression strategies resulting in saturation of terminal sialic acid attachment sites might lengthen protein bioavailability and half-life. \\n     In the development of recombinant glycoproteins for use as pharmaceutical products, for example, it has been speculated that the pharmacodynamics of recombinant proteins can be modulated by the addition or deletion of glycosylation sites from a glycoprotein&#39;s primary structure (Berman &amp; Lasky, (1985)  Trends Biotechnol.  3:51–53). However, studies have reported that the deletion of N-linked glycosylation sites often impairs intracellular transport and results in the intracellular accumulation of glycosylation site variants (Machamer &amp; Rose, (1988)  J. Biol. Chem.  263:5955–5960; Gallagher et al., (1992)  J. Virology.  66:7136–7145; Collier et al., (1993)  Biochem.  32:7818–7823; Claffey et al., (1995)  Biochim. Biophys. Acta  1246:1–9; Dube et al., (1988)  J. Biol. Chem.  263:17516–17521). While glycosylation site variants of proteins can be expressed intracellularly, it has proved difficult to recover useful quantities from growth conditioned cell culture medium. \\n     Moreover, it is unclear to what extent a glycosylation site in one species will be recognized by another species glycosylation machinery. Due to the importance of glycosylation in protein metabolism, particularly the secretion and/or expression of the protein, whether a glycosylation signal is recognized may profoundly determine a protein&#39;s ability to be expressed, either endogenously or recombinately, in another organism (i.e., expressing a human protein in  E. coli , yeast, or viral organisms; or an  E. coli , yeast, or viral protein in human, etc.). Thus, it may be desirable to add, delete, or modify a glycosylation site, and possibly add a glycosylation site of one species to a protein of another species to improve the proteins functional, bioprocess purification, and/or structural characteristics (e.g., a polypeptide of the present invention). \\n     A number of methods may be employed to identify the location of glycosylation sites within a protein. One preferred method is to run the translated protein sequence through the PROSITE computer program (Swiss Institute of Bioinformatics). Once identified, the sites could be systematically deleted, or impaired, at the level of the DNA using mutagenesis methodology known in the art and available to those of ordinary skill in the art, for example using PCR-directed mutagenesis (see, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3rd ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). Similarly, glycosylation sites could be added, or modified at the level of the DNA using similar methods, for example PCR (see Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3d ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). The results of modifying the glycosylation sites for a particular protein (e.g., solubility, secretion potential, activity, aggregation, proteolytic resistance, etc.) could then be analyzed using methods know in the art. \\n     Those of ordinary skill in the art will acknowledge the existence of other computer algorithms capable of predicting the location of glycosylation sites within a protein. For example, the MOTIF computer program (Genetics Computer Group suite of programs) provides this function as well. \\n     Example 23 \\n     Method of Enhancing the Biological Activity/Functional Characteristics of a Polypeptide of the Present Invention through Molecular Evolution \\n     Although many of the most biologically active proteins known are highly effective for their specified function in an organism, they often possess characteristics that make them undesirable for transgenic, therapeutic, and/or industrial applications. Among these traits, a short physiological half-life is the most prominent problem, and is present either at the level of the protein, or the level of the proteins mRNA. The ability to extend the half-life, for example, would be particularly important for proteins used in gene therapy, transgenic animal production, the bioprocess production and purification of the protein, and use of the protein as a chemical modulator among others. Therefore, there is a need to identify novel variants of isolated proteins possessing characteristics that enhance their application as a therapeutic for treating diseases of animal origin, in addition to the protein&#39;s applicability to common industrial and pharmaceutical applications. \\n     Thus, one aspect of the present invention relates to the ability to enhance specific characteristics of a polypeptide of the present invention through directed molecular evolution. Such an enhancement may, in a non-limiting example, (a) provide an additional utility for a polypeptide and/or polynucleotide of the present invention, for example as a component in a kit, (b) impart a desirable physical property to a polypeptide of the present invention, such as an enhancement in solubility, structure, or codon optimization, (c) impart an enhanced biological activity, including any associated enzymatic activity, which encompasses a polypeptide&#39;s enzyme kinetics, (e.g., Ki, Kcat, Km, Vmax, Kd), protein-protein activity, protein-DNA binding activity, antagonist/inhibitory activity (including direct or indirect interaction), agonist activity (including direct or indirect interaction), (d) a polypeptide&#39;s antigenicity (e.g., where it would be desirable to either increase or decrease the antigenic potential of the protein), (e) the immunogenicity of the protein, (f) the ability of a polypeptide to form dimers, trimers, or multimers with either itself or other proteins, (g) the antigenic efficacy of the present invention, including its subsequent use a preventative treatment for disease or disease states, or (h) as an effector for targeting diseased genes, to name just a few examples. Moreover, the ability to enhance specific characteristics of a protein may also be applicable to changing the characterized activity of an enzyme to an activity completely unrelated to its initially characterized activity. Other desirable enhancements of the present invention would be specific to each individual protein, and would thus be well known in the art and contemplated by the present invention. \\n     Directed evolution is comprised of several steps. In one embodiment, a first step is to establish a library of variants for the gene or protein of interest. The next step is to then select for those variants that entail the activity it is desired to identify. The design of the screen is essential since the screen should be selective enough to eliminate non-useful variants, but not so stringent as to eliminate all variants. The last step is then to repeat the above steps using the best variant from the previous screen. Each successive cycle, can then be tailored as necessary, such as by increasing the stringency of the screen, for example. \\n     Over the years, there have been a number of methods developed to introduce mutations into macromolecules. Some of these methods include, random mutagenesis, “error-prone” PCR, chemical mutagenesis, site-directed mutagenesis, and other methods well known in the art (for a comprehensive listing of current mutagenesis methods, see Sambrook et al.,  Molecular Cloning: A Laboratory Manual , (3rd ed.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., USA (2001)). Typically, such methods have been used, for example, as tools for identifying the core functional region(s) of a protein or the function of specific domains of a protein (if a multi-domain protein). However, such methods have more recently been applied to the identification of macromolecule variants with specific or enhanced characteristics. \\n     Random mutagenesis has been the most widely recognized method to date. Typically, this has been carried out either through the use of “error-prone” PCR (as described in Moore et al., (1996)  Nature Biotechnol.  14:458), or through the application of randomized synthetic oligonucleotides corresponding to specific regions of interest (as described by Derbyshire et al., (1986)  Gene  46:145–152; and Hill et al., (1987)  Method Enzymol.  55:559–568). Both approaches have limits to the level of mutagenesis that can be obtained. However, either approach enables the investigator to effectively control the rate of mutagenesis. This is particularly important considering the fact that mutations beneficial to the activity of the enzyme are fairly rare. In fact, using too high a level of mutagenesis may counter or inhibit the desired benefit of a useful mutation. \\n     While both of the aforementioned methods are effective for creating randomized pools of macromolecule variants, a third method, termed “DNA Shuffling”, or “sexual PCR” (Stemmer, (1994)  Proc. Natl. Acad. Sci. USA  91:10747) has recently been elucidated. DNA shuffling has also been referred to as “directed molecular evolution”, “exon-shuffling”, “directed enzyme evolution”, “in vitro evolution”, and “artificial evolution”. Such reference terms are known in the art and are encompassed by the present invention. This new, and often preferred, method apparently overcomes the limitations of the previous methods in that it not only propagates positive traits, but simultaneously eliminates negative traits in the resulting progeny. \\n     DNA shuffling accomplishes this task by combining the principal of in vitro recombination, along with the method of “error-prone” PCR. In effect, the process begins with a randomly digested pool of small fragments of a target gene, created by DNase I digestion, and then introduce said random fragments into an “error-prone” PCR assembly reaction. During the PCR reaction, the randomly sized DNA fragments not only hybridize to their cognate strand, but also may hybridize to other DNA fragments corresponding to different regions of the polynucleotide of interest—regions not typically accessible via hybridization of the entire polynucleotide. Moreover, since the PCR assembly reaction utilizes “error-prone” PCR reaction conditions, random mutations are introduced during the DNA synthesis step of the PCR reaction for all of the fragments, further diversifying the potential hybridization sites during the annealing step of the reaction. \\n     A variety of reaction conditions could be utilized to carry-out the DNA shuffling reaction. However, specific reaction conditions for DNA shuffling are provided, for example, in Stemmer, (1994)  Proc. Natl. Acad. Sci. U.S.A.  91:10747. \\n     Briefly: \\n     In one embodiment, the DNA substrate to be subjected to the DNA shuffling reaction is prepared. Preparation may be in the form of simply purifying the DNA from contaminating cellular material, chemicals, buffers, oligonucleotide primers, deoxynucleotides, RNAs, etc., and may entail the use of DNA purification kits as those commerically available (e.g., from Qiagen, Inc., or from Promega, Corp., Madison, Wis., USA for example). \\n     Once the DNA substrate has been purified, it would be subjected to Dnase I digestion. About 2–4 μg of the DNA substrate(s) would be digested with 0.0015 units of Dnase I (Sigma, St. Louis, Mo., USA) per μl in 100 μl of 50 mM Tris-HCL, pH 7.4/lniM MgCl 2  for 10–20 min. at room temperature. The resulting fragments of 10–50 bp could then be purified by running them through a 2% low-melting point agarose gel by electrophoresis onto DE81 ion-exchange paper (Whatman) or could be purified using Microcon concentrators (Amicon, Millipore, Wellesley, Mass., USA) of the appropriate molecular weight cutoff, or could use oligonucleotide purification columns (Qiagen, Inc.), in addition to other methods known in the art. If using DE81 ion-exchange paper, the 10–50 bp fragments could be eluted from said paper using 1M NaCl, followed by ethanol precipitation. \\n     The resulting purified fragments would then be subjected to a PCR assembly reaction by re-suspension in a PCR mixture containing: 2 mM of each dNTP, 2.2 mM MgCl 2 , 50 mM KCl, 10 mM Tris.HCL, pH 9.0, and 0.1% Triton X-100, at a final fragment concentration of 10–30ng/μl. No primers are added at this point. Taq DNA polymerase (Promega, Madison, Wis., USA) would be used at 2.5 units per 100 μl of reaction mixture. A PCR program of 94 C for 60 seconds; 94 C for 30 seconds, 50–55 C for 30 seconds, and 72 C for 30 seconds using 30–45 cycles, followed by 72 C for 5 min using an MJ Research (Cambridge, Mass., USA) PTC-150 thermocycler. After the assembly reaction is completed, a 1:40 dilution of the resulting primeness product would then be introduced into a PCR mixture (using the same buffer mixture used for the assembly reaction) containing 0.8 μm of each primer and subjecting this mixture to 15 cycles of PCR (using 94 C for 30 seconds, 50 C for 30 seconds, and 72 C for 30 seconds). Representative primers would be primers corresponding to the nucleic acid sequences of the polynucleotide(s) utilized in the shuffling reaction. Said primers could consist of modified nucleic acid base pairs using methods known in the art and referred to else where herein, or could contain additional sequences (i.e., for adding restriction sites, mutating specific base-pairs, etc.). \\n     The resulting shuffled, assembled, and amplified product can be purified using methods well known in the art (e.g., Qiagen PCR purification kits) and then subsequently cloned using appropriate restriction enzymes. \\n     Although a number of variations of DNA shuffling have been published to date, such variations would be obvious to the skilled artisan and are encompassed by the present invention. The DNA shuffling method can also be tailored to the desired level of mutagenesis using the methods described by Zhao et al. (Zhao et al., (1997)  Nucl Acid Res.  25(6): 1307–1308). \\n     As described above, once the randomized pool has been created, it can then be subjected to a specific screen to identify the variant possessing the desired characteristic(s). Once the variant has been identified, DNA corresponding to the variant could then be used as the DNA substrate for initiating another round of DNA shuffling. This cycle of shuffling, selecting the optimized variant of interest, and then re-shuffling, can be repeated until the ultimate variant is obtained. Examples of model screens applied to identify variants created using DNA shuffling technology may be found in the following publications: Moore et al., (1997)  J. Mol. Biol.  272:336–347; Cross et al., (1998)  Mol. Cell. Biol.  18:2923–2931; and Crameri et al., (1997)  Nat. Biotech.  15:436–438). \\n     DNA shuffling has several advantages. First, it makes use of beneficial mutations. When combined with screening, DNA shuffling allows the discovery of the best mutational combinations and does not assume that the best combination contains all the mutations in a population. Secondly, recombination occurs simultaneously with point mutagenesis. An effect of forcing DNA polymerase to synthesize full-length genes from the small fragment DNA pool is a background mutagenesis rate. In combination with a stringent selection method, enzymatic activity has been evolved up to 16000-fold increase over the wild-type form of the enzyme. In essence, the background mutagenesis yielded the genetic variability on which recombination acted to enhance the activity. \\n     A third feature of recombination is that it can be used to remove deleterious mutations. As discussed herein, during the process of the randomization, for every one beneficial mutation, there may be at least one or more neutral or inhibitory mutations. Such mutations can be removed by including in the assembly reaction an excess of the wild-type random-size fragments, in addition to the random-size fragments of the selected mutant from the previous selection. During the next selection, some of the most active variants of the polynucleotide/polypeptide/enzyme, should have lost the inhibitory mutations. \\n     Finally, recombination enables parallel processing. This represents a significant advantage since there are likely multiple characteristics that would make a protein more desirable (e.g. solubility, activity, etc.). Since it is increasingly difficult to screen for more than one desirable trait at a time, other methods of molecular evolution tend to be inhibitory. However, using recombination, it would be possible to combine the randomized fragments of the best representative variants for the various traits, and then select for multiple properties at once. \\n     DNA shuffling can also be applied to the polynucleotides and polypeptides of the present invention to decrease their immunogenicity in a specified host. For example, a particular variant of the present invention may be created and isolated using DNA shuffling technology. Such a variant may have all of the desired characteristics, though may be highly immunogenic in a host due to its novel intrinsic structure. Specifically, the desired characteristic may cause the polypeptide to have a non-native structure which could no longer be recognized as a “self” molecule, but rather as a “foreign”, and thus activate a host immune response directed against the novel variant. Such a limitation can be overcome, for example, by including a copy of the gene sequence for a xenobiotic ortholog of the native protein in with the gene sequence of the novel variant gene in one or more cycles of DNA shuffling. The molar ratio of the ortholog and novel variant DNAs could be varied accordingly. Ideally, the resulting hybrid variant identified would contain at least some of the coding sequence which enabled the xenobiotic protein to evade the host immune system, and additionally, the coding sequence of the original novel variant that provided the desired characteristics. \\n     Likewise, the present invention encompasses the application of DNA shuffling technology to the evolution of polynucleotides and polypeptides of the present invention, wherein one or more cycles of DNA shuffling include, in addition to the gene template DNA, oligonucleotides coding for known allelic sequences, optimized codon sequences, known variant sequences, known polynucleotide polymorphism sequences, known ortholog sequences, known homologue sequences, additional homologous sequences, additional non-homologous sequences, sequences from another species, and any number and combination of the above. \\n     In addition to the described methods above, there are a number of related methods that may also be applicable, or desirable in certain cases. Representative among these are the methods discussed in PCT Publications WO 98/31700, and WO 98/32845, which are hereby incorporated by reference. Furthermore, related methods can also be applied to the polynucleotide sequences of the present invention in order to evolve invention for creating ideal variants for use in gene therapy, protein engineering, evolution of whole cells containing the variant, or in the evolution of entire enzyme pathways containing polynucleotides of the present invention as described in PCT Publications WO 98/13485, WO 98/13487, WO 98/27230, WO 98/31837, and Crameri et al., (1997)  Nat. Biotech.  15:436–438, respectively. \\n     Additional methods of applying “DNA Shuffling” technology to the polynucleotides and polypeptides of the present invention, including their proposed applications, can be found in U.S. Pat. No. 5,605,793; PCT Publications WO 95/22625; WO 97/20078; WO 97/35966; and WO 98/42832; WO 00/09727 specifically provides methods for applying DNA shuffling to the identification of herbicide selective crops which could be applied to the polynucleotides and polypeptides of the present invention; additionally, PCT Publication WO 00/12680 provides methods and compositions for generating, modifying, adapting, and optimizing polynucleotide sequences that confer detectable phenotypic properties on plant species; each of the above are hereby incorporated in their entirety herein for all purposes. \\n     Example 24 \\n     Method of Determining Alterations in a Gene Corresponding to a Polynucleotide of the Present Invention \\n     RNA isolated from entire families or individual subjects presenting with a phenotype of interest (such as a disease) is be isolated cDNA is then generated from these RNA samples using protocols known in the art. The cDNA is then used as a template for PCR, employing primers surrounding regions of interest in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID NO:12, SEQ ID NO:14, and/or SEQ ID NO:16. Representative PCR conditions consist of 35 cycles at 95 degrees C. for 30 seconds; 60–120 seconds at 52–58° C.; and 60–120 seconds at 70 degrees C., using buffer solutions described in Sidransky et al. (Sidransky et al., (1991)  Science  252:706). \\n     PCR products are then sequenced using primers labeled at their 5′ end with T4 polynucleotide kinase, employing SEQUITHERM Polymerase. (Epicentre Technologies, Madison, Wis., USA). The intron-exon borders of selected exons is also determined and genomic PCR products analyzed to confirm the results. PCR products harboring suspected mutations is then cloned and sequenced to validate the results of the direct sequencing. \\n     PCR products are cloned into T-tailed vectors as described in Holton et al., (Holton et al., (1991)  Nucl. Acid Research  19:1156 (1991) and sequenced with T7 polymerase (United States Biochemical). Affected individuals are identified by mutations not present in unaffected individuals. \\n     Genomic rearrangements are also observed as a method of determining alterations in a gene corresponding to a polynucleotide. Genomic clones isolated according to the methods described herein are nick-translated with digoxigenindeoxy-uridine 5′-triphosphate (Boehringer Manheim), and FISH performed as described in Johnson et al. (Johnson et al., (1991)  Method Cell Biol.  35:73–99). Hybridization with the labeled probe is carried out using a vast excess of human cot-i DNA for specific hybridization to the corresponding genomic locus. \\n     Chromosomes are counterstained with 4,6-diamino-2-phenylidole and propidium iodide, producing a combination of C- and R-bands. Aligned images for precise mapping are obtained using a triple-band filter set (Chroma Technology, Brattleboro, Vt., USA) in combination with a cooled charge-coupled device camera (Photometrics, Tucson, Ariz., USA) and variable excitation wavelength filters (Johnson et al., (1991)  Genet. Anal. Tech. Appl.  8:75). Image collection, analysis and chromosomal fractional length measurements are performed using the ISee Graphical Program System. (Inovision Corporation, Durham, N.C., USA). Chromosome alterations of the genomic region hybridized by the probe are identified as insertions, deletions, and translocations. These alterations are used as a diagnostic marker for an associated disease. \\n     Example 25 \\n     Method of Detecting Abnormal Levels of a Polypeptide of the Present Invention in a Biological Sample \\n     A polypeptide of the present invention can be detected in a biological sample, and if an increased or decreased level of the polypeptide is detected, this polypeptide is a marker for a particular phenotype. Methods of detection are numerous, and thus, it is understood that one skilled in the art can modify the following assay to fit their particular needs. \\n     For example, antibody-sandwich ELISAs are used to detect polypeptides in a sample, preferably a biological sample. The wells of a microtiter plate are coated with specific antibodies, at a final concentration of 0.2 to 10 μg/ml. The antibodies are either monoclonal or polyclonal and are produced by the method described elsewhere herein. The wells are blocked so that non-specific binding of the polypeptide to the well is reduced. \\n     The coated wells are then incubated for greater than about 2 hours at room temperature with a sample containing the polypeptide. Preferably, serial dilutions of the sample should be used to validate results. The plates are then washed three times with deionized or distilled water to remove unbounded polypeptide. \\n     Next, 50 μl of specific antibody-alkaline phosphatase conjugate, at a concentration of 25–400 ng, is added and incubated for 2 hours at room temperature. The plates are again washed three times with deionized or distilled water to remove unbounded conjugate. \\n     Add 75 μl of 4-methylumbelliferyl phosphate (MUP) or p-nitrophenyl phosphate (NPP) substrate solution to each well and incubate 1 hour at room temperature. Measure the reaction by a microtiter plate reader. Prepare a standard curve, using serial dilutions of a control sample, and plot polypeptide concentration on the X-axis (log scale) and fluorescence or absorbance of the Y-axis (linear scale). Interpolate the concentration of the polypeptide in the sample using the standard curve. \\n     Since the polypeptides of the present invention exhibit GTPase activity, a GTPase activity assay can also be employed to determine the concentration of polypeptide in the sample. \\n     Example 26 \\n     Formulations \\n     The present invention also provides methods of treatment and/or prevention diseases, disorders, and/or conditions (such as, for example, any one or more of the diseases or disorders disclosed herein) by administration to a subject of an effective amount of a therapeutic. The term “therapeutic” includes polynucleotides and/or polypeptides of the present invention (including fragments and variants), agonists or antagonists thereof, and/or antibodies thereto, in combination with a pharmaceutically acceptable carrier type (e.g., a sterile carrier). \\n     A therapeutic may be formulated and dosed in a fashion consistent with good medical practice, taking into account the clinical condition of the individual subject (especially the side effects of treatment with the therapeutic alone), the site of delivery, the method of administration, the scheduling of administration, and other factors known to practitioners. The “effective amount” for purposes herein is thus determined by such considerations. \\n     As a general proposition, the total pharmaceutically effective amount of the therapeutic administered parenterally per dose will be in the range of about 1 μg/kg/day to 10 mg/kg/day of subject body weight, although, as noted above, this will be subject to therapeutic discretion. More preferably, this dose is at least 0.01 mg/kg/day, and most preferably for humans between about 0.01 and 1 mg/kg/day for the hormone. If given continuously, the therapeutic is typically administered at a dose rate of about 1 μg/kg/hour to about 50 μg/kg/hour, either by 1–4 injections per day or by continuous subcutaneous infusions, for example, using a mini-pump. An intravenous bag solution may also be employed. The length of treatment needed to observe changes and the interval following treatment for responses to occur appears to vary depending on the desired effect. \\n     Therapeutics can be administered orally, rectally, parenterally, intracisternally, intravaginally, intraperitoneally, topically (as by powders, ointments, gels, drops or transdermal patch), bucally, or as an oral or nasal spray. “Pharmaceutically acceptable carrier” refers to a non-toxic solid, semisolid or liquid filler, diluent, encapsulating material or formulation auxiliary of any. The term “parenteral” as used herein refers to modes of administration, which include intravenous, intramuscular, intraperitoneal, intrasternal, subcutaneous and intraarticular injection and infusion. \\n     Therapeutics of the present invention are also suitably administered by sustained-release systems. Suitable examples of sustained-release therapeutics are administered orally, rectally, parenterally, intracisternally, intravaginally, intraperitoneally, topically (as by powders, ointments, gels, drops or transdermal patch), bucally, or as an oral or nasal spray. “Pharmaceutically acceptable carrier” refers to a non-toxic solid, semisolid or liquid filler, diluent, encapsulating material or formulation auxiliary of any type. \\n     A therapeutic of the present invention may also be suitably administered by sustained-release systems. Suitable examples of sustained-release therapeutics include suitable polymeric materials (such as, for example, semi-permeable polymer matrices in the form of shaped articles, e.g., films, or microcapsules), suitable hydrophobic materials (for example as an emulsion in an acceptable oil) or ion exchange resins, and sparingly soluble derivatives (such as, for example, a sparingly soluble salt). \\n     Sustained-release matrices include polylactides (U.S. Pat. No. 3,773,919, EP 58,481), copolymers of L-glutamic acid and gamma-ethyl-L-glutamate (Sidman et al., (1983)  Biopolymers  22:547–556), poly (2-hydroxyethyl methacrylate) (Langer et al., (1981)  J. Biomed. Mater. Res.  15:167–277, and Langer, (1982)  Chem. Tech.  12:98–10), ethylene vinyl acetate (Langer et al., (1981)  J. Biomed. Mater. Res.  15:167–277) or poly-D-(−)-3-hydroxybutyric acid (EP 133,988). \\n     Sustained-release therapeutics also include liposomally entrapped therapeutics of the present invention (see, generally, Langer, (1990)  Science  249:1527–1533; Treat et al., in  Liposomes in the Therapy of Infectious Disease and Cancer , (Lopez-Berestein and Fidler, eds.), Albert R. Liss, New York, N.Y., USA, pp. 317–327 and 353–365 (1989)). Liposomes containing the therapeutic are prepared by methods known per se: DE 3,218,121; Epstein et al., (1985)  Proc. Natl. Acad. Sci. U.S.A.  82:3688–3692; Hwang et al., (1980)  Proc. Natl. Acad. Sci. U.S.A.  77:4030–4034; EP 52,322; EP 36,676; EP 88,046; EP 143,949; EP 142,641; Japanese Patent Application 83-118008; U.S. Pat. Nos. 4,485,045 and 4,544,545; and EP 102,324). Ordinarily, the liposomes are of the small (about 200–800 Angstroms) unilamellar type in which the lipid content is greater than about 30 mol. percent cholesterol, the selected proportion being adjusted for the optimal therapeutic. \\n     In yet an additional embodiment, a therapeutics of the present invention are delivered by way of a pump (see Langer, (1990)  Science  249:1527–1533; Sefton, (1987)  CRC Crit. Ref. Biomed. Eng.  14:201; Buchwald et al., (1980)  Surgery  88:507; Saudek et al., (1989)  N. Engl. J. Med.  321:574). \\n     Other controlled release systems are discussed in the review by Langer (Langer, (1990)  Science  249:1527–1533). \\n     For parenteral administration, in one embodiment, the therapeutic is formulated generally by mixing it at the desired degree of purity, in a unit dosage injectable form (solution, suspension, or emulsion), with a pharmaceutically acceptable carrier, i.e., one that is non-toxic to recipients at the dosages and concentrations employed and is compatible with other ingredients of the formulation. For example, the formulation preferably does not include oxidizing agents and other compounds that are known to be deleterious to the therapeutic. \\n     Generally, the formulations are prepared by contacting the therapeutic uniformly and intimately with liquid carriers or finely divided solid carriers or both. Then, if necessary, the product is shaped into the desired formulation. For example, the carrier can be a parenteral carrier, or can be a solution that is isotonic with the blood of the recipient. Examples of such carrier vehicles include water, saline, Ringer&#39;s solution, and dextrose solution. Non-aqueous vehicles such as fixed oils and ethyl oleate are also useful herein, as well as liposomes. \\n     The carrier suitably contains minor amounts of additives such as substances that enhance isotonicity and chemical stability. Such materials are non-toxic to recipients at the dosages and concentrations employed, and include buffers such as phosphate, citrate, succinate, acetic acid, and other organic acids or their salts; antioxidants such as ascorbic acid; low molecular weight (less than about ten residues) polypeptides, e.g., polyarginine or tripeptides; proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidone; amino acids, such as glycine, glutamic acid, aspartic acid, or arginine; monosaccharides, disaccharides, and other carbohydrates including cellulose or its derivatives, glucose, mannose, or dextrins; chelating agents such as EDTA; sugar alcohols such as mannitol or sorbitol; counterions such as sodium; and/or nonionic surfactants such as polysorbates, poloxamers, or PEG. \\n     A therapeutic will typically be formulated in such vehicles at a concentration of about 0.1 mg/ml to 100 mg/ml, or 1–10 mg/ml, at a pH of about 3 to 8. It will be understood that the use of certain of the foregoing excipients, carriers, or stabilizers will result in the formation of polypeptide salts. \\n     Any pharmaceutical used for therapeutic administration can be sterile. Sterility is readily accomplished by filtration through sterile filtration membranes (e.g., 0.2 micron membranes). Therapeutics generally are placed into a container having a sterile access port, for example, an intravenous solution bag or vial having a stopper pierceable by a hypodermic injection needle. \\n     Therapeutics ordinarily will be stored in unit or multi-dose containers, for example, sealed ampoules or vials, as an aqueous solution or as a lyophilized formulation for reconstitution. As an example of a lyophilized formulation, 10-mil vials are filled with 5 mil of sterile-filtered 1% (w/v) aqueous therapeutic solution, and the resulting mixture is lyophilized. The infusion solution is prepared by reconstituting the lyophilized therapeutic using bacteriostatic “Water-for-Injection.” \\n     The present invention also provides a pharmaceutical pack or kit comprising one or more containers filled with one or more of the ingredients of the therapeutics of the present invention. Associated with such container(s) can be a notice in the form prescribed by a governmental agency regulating the manufacture, use or sale of pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human administration. In addition, the therapeutics may be employed in conjunction with other therapeutic compounds. \\n     The therapeutics of the present invention may be administered alone or in combination with adjuvants. Adjuvants that may be administered with the therapeutics of the present invention include, but are not limited to, alum, alum plus deoxycholate (ImmunoAg, Vienna, Austria), MTP-PE (Biocine Corp., Emeryville, Calif., USA), QS21 (Genentech, Inc., South San Francisco, Calif., USA), BCG, and MPL. In a specific embodiment, therapeutics of the present invention are administered in combination with alum. In another specific embodiment, a therapeutic of the present invention is administered in combination with QS-21. Further adjuvants that may be administered with the therapeutics of the present invention include, but are not limited to, Monophosphoryl lipid immunomodulator, ADJUVAX 100a, QS-21, QS-18, CRL1005, Aluminum salts, MF-59, and Virosomal adjuvant technology. Vaccines that may be administered with the therapeutics of the present invention include, but are not limited to, vaccines directed toward protection against MMR (measles, mumps, rubella), polio, varicella, tetanus/diptheria, hepatitis A, hepatitis B,  haemophilus  influenzae B, whooping cough, pneumonia, influenza, Lyme&#39;s Disease, rotavirus, cholera, yellow fever, Japanese encephalitis, poliomyelitis, rabies, typhoid fever, and pertussis. Combinations may be administered either concomitantly, e.g., as an admixture, separately but simultaneously or concurrently; or sequentially. This includes presentations in which the combined agents are administered together as a therapeutic mixture, and also procedures in which the combined agents are administered separately but simultaneously, e.g., as through separate intravenous lines into the same individual. Administration “in combination” further includes the separate administration of one of the compounds or agents given first, followed by the second. \\n     A therapeutic of the present invention may be administered alone or in combination with other therapeutic agents. Therapeutic agents that may be administered in combination with the therapeutics of the present invention, include but not limited to, other members of the TNF family, chemotherapeutic agents, antibiotics, steroidal and non-steroidal anti-inflammatories, conventional immunotherapeutic agents, cytokines and/or growth factors. Combinations may be administered either concomitantly, e.g., as an admixture, separately but simultaneously or concurrently; or sequentially. This includes presentations in which the combined agents are administered together as a therapeutic mixture, and also procedures in which the combined agents are administered separately but simultaneously, e.g., as through separate intravenous lines into the same individual. Administration “in combination” further includes the separate administration of one of the compounds or agents given first, followed by the second. \\n     In one embodiment, the therapeutics (which include the polypeptides, polynucleotides of the present invention, and fragments thereof) of the present invention are administered in combination with members of the TNF family. TNF, TNF-related or TNF-like molecules that may be administered with the therapeutics of the present invention include, but are not limited to, soluble forms of TNF-alpha, lymphotoxin-alpha (LT-alpha, also known as TNF-beta), LT-beta (found in complex heterotrimer LT-alpha2-beta), OPGL, FasL, CD27L, CD30L, CD40L, 4-1BBL, DcR3, OX40L, TNF-gamma (PCT Publication WO 96/14328), AIM-I (PCT Publication WO 97/33899), endokine-alpha (PCT Publication WO 98/07880), TR6 (PCT Publication WO 98/30694), OPG, and neutrokine-alpha (PCT Publication WO 98/18921, OX40, and nerve growth factor (NGF), and soluble forms of Fas, CD30, CD27, CD40 and 4-IBB, TR2 (PCT Publication WO 96/34095), DR3 (PCT Publication WO 97/33904), DR4 (PCT Publication WO 98/32856), TR5 (PCT Publication WO 98/30693), TR6 (PCT Publication WO 98/30694), TR7 (PCT Publication WO 98/41629), TRANK, TR9 (PCT Publication WO 98/56892),TR10 (PCT Publication WO 98/54202), 312C2 (PCT Publication WO 98/06842), and TR12, and soluble forms CD154, CD70, and CD153. \\n     In certain embodiments, a therapeutic of the present invention is administered in combination with antiretroviral agents, nucleoside reverse transcriptase inhibitors, non-nucleoside reverse transcriptase inhibitors, and/or protease inhibitors. Nucleoside reverse transcriptase inhibitors that may be administered in combination with a therapeutic of the present invention, include, but are not limited to, RETROVIR® (zidovudine/AZT), VIDEX® (didanosine/ddI), HIVID® (zalcitabine/ddC), ZERIT® (stavudine/d4T), EPIVIR® (lamivudine/3TC), and COMBIVIR® (zidovudine/lamivudine). Non-nucleoside reverse transcriptase inhibitors that may be administered in combination with a therapeutics of the present invention, include, but are not limited to, VIRAMUNE® (nevirapine), RESCRIPTOR® (delavirdine), and SUSTIVA® (efavirenz). Protease inhibitors that may be administered in combination with a therapeutic of the present invention, include, but are not limited to, CRIXIVAN® (indinavir), NORVIR® (ritonavir), INVIRASE® (saquinavir), and VIRACEPT® (nelfinavir). In a specific embodiment, antiretroviral agents, nucleoside reverse transcriptase inhibitors, non-nucleoside reverse transcriptase inhibitors, and/or protease inhibitors may be used in any combination with a therapeutic of the present invention to treat AIDS and/or to prevent or treat HIV infection. \\n     In other embodiments, a therapeutic of the present invention may be administered in combination with anti-opportunistic infection agents. Anti-opportunistic agents that may be administered in combination with a therapeutic of the present invention, include, but are not limited to, TRIMETHOPRIM-SULFAMETHOXAZOLE®, DAPSONE®, PENTAMIDINE®, ATOVAQUONE®, ISONIAZID®, RIFAMPIN®, PYRAZINAMIDE®, ETHAMBUTOL®, RIFABUTIN®, CLARITHROMYCIN®, AZITHROMYCIN®, GANCICLOVIR®, FOSCARNET®, CIDOFOVIR®, FLUCONAZOLE®, ITRACONAZOLE®, KETOCONAZOLE®, ACYCLOVIR®, FAMCICOLVIR®, PYRIMETHAMINE®, LEUCOVORIN®, NEUPOGEN® (filgrastim/G-CSF), and LEUKINE® (sargramostim/GM-CSF). In a specific embodiment, a therapeutic of the present invention is used in any combination with TRIMETHOPRIM-SULFAMETHOXAZOLE®, DAPSONE®, PENTAMIDINE®, and/or ATOVAQUONE® to prophylactically treat or prevent an opportunistic  Pneumocystis carinii  pneumonia infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with ISONIAZID®, RIFAMPIN®, PYRAZINAMIDE®, and/or ETHAMBUTOL® to prophylactically treat or prevent an opportunistic  Mycobacterium avium  complex infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with RIFABUTIN®, CLARITHROMYCIN®, and/or AZITHROMYCIN® to prophylactically treat or prevent an opportunistic  Mycobacterium tuberculosis  infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with GANCICLOVIR®, FOSCARNET®, and/or CIDOFOVIR® to prophylactically treat or prevent an opportunistic cytomegalovirus infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with FLUCONAZOLE®, ITRACONAZOLE®, and/or KETOCONAZOLE® to prophylactically treat or prevent an opportunistic fungal infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with ACYCLOVIR® and/or FAMCICOLVIR® to prophylactically treat or prevent an opportunistic herpes simplex virus type I and/or type II infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with PYRIMETHAMINE® and/or LEUCOVORIN® to prophylactically treat or prevent an opportunistic  Toxoplasma gondii  infection. In another specific embodiment, a therapeutic of the present invention is used in any combination with LEUCOVORIN® and/or NEUPOGEN® to prophylactically treat or prevent an opportunistic bacterial infection. \\n     In a further embodiment, a therapeutic of the present invention is administered in combination with an antiviral agent. Antiviral agents that may be administered with the Therapeutics of the present invention include, but are not limited to, acyclovir, ribavirin, amantadine, and remantidine. \\n     In a further embodiment, a therapeutic of the present invention is administered in combination with an antibiotic agent. Antibiotic agents that may be administered with a therapeutic of the present invention include, but are not limited to, amoxicillin, beta-lactamases, aminoglycosides, beta-lactam (glycopeptide), beta-lactamases, clindamycin, chloramphenicol, cephalosporins, ciprofloxacin, ciprofloxacin, erythromycin, fluoroquinolones, macrolides, metronidazole, penicillins, quinolones, rifampin, streptomycin, sulfonamide, tetracyclines, trimethoprim, trimethoprim-sulfamthoxazole, and vancomycin. \\n     Conventional nonspecific immunosuppressive agents, that may be administered in combination with a therapeutic of the present invention include, but are not limited to, steroids, cyclosporine, cyclosporine analogs, cyclophosphamide methylprednisone, prednisone, azathioprine, FK-506, I5-deoxyspergualin, and other immunosuppressive agents that act by suppressing the function of responding T cells. \\n     In specific embodiments, a therapeutic of the present invention is administered in combination with immunosuppressants. Immunosuppressants preparations that may be administered with a therapeutic of the present invention include, but are not limited to, ORTHOCLONE® (OKT3), SANDIMMUNE/NEORAL/SANGDYA® (cyclosporin), PROGRAF® (tacrolimus), CELLCEPT® (mycophenolate), Azathioprine, glucorticosteroids, and RAPAMUNE® (sirolimus). In a specific embodiment, immunosuppressants may be used to prevent rejection of organ or bone marrow transplantation. \\n     In an additional embodiment, a therapeutic of the present invention is administered alone or in combination with one or more intravenous immune globulin preparations. Intravenous immune globulin preparations that may be administered with a therapeutic of the present invention include, but are not limited to, GAMMAR®, IVEEGAM®, SANDOGLOBULIN®, GAMMAGARD S/D®, and GAMIMUNE®. In a specific embodiment, a therapeutic of the present invention is administered in combination with intravenous immune globulin preparations in transplantation therapy (e.g., bone marrow transplant). \\n     In an additional embodiment, a therapeutic of the present invention is administered alone or in combination with an anti-inflammatory agent. Anti-inflammatory agents that may be administered with a therapeutic of the present invention include, but are not limited to, glucocorticoids and the nonsteroidal anti-inflammatories, aminoarylcarboxylic acid derivatives, arylacetic acid derivatives, arylbutyric acid derivatives, arylcarboxylic acids, arylpropionic acid derivatives, pyrazoles, pyrazolones, salicylic acid derivatives, thiazinecarboxamides, e-acetamidocaproic acid, S-adenosylmethionine, 3-amino-4-hydroxybutyric acid, amixetrine, bendazac, benzydamine, bucolome, difenpiramide, ditazol, emorfazone, guaiazulene, nabumetone, nimesulide, orgotein, oxaceprol, paranyline, perisoxal, pifoxime, proquazone, proxazole, and tenidap. \\n     In another embodiment, compositions of the present invention are administered in combination with a chemotherapeutic agent. Chemotherapeutic agents that may be administered with a therapeutic of the present invention include, but are not limited to, antibiotic derivatives (e.g., doxorubicin, bleomycin, daunorubicin, and dactinomycin); antiestrogens (e.g., tamoxifen); antimetabolites (e.g., fluorouracil, 5-FU, methotrexate, floxuridine, interferon alpha-2b, glutamic acid, plicamycin, mercaptopurine, and 6-thioguanine); cytotoxic agents (e.g., carmustine, BCNU, lomustine, CCNU, cytosine arabinoside, cyclophosphamide, estramustine, hydroxyurea, procarbazine, mitomycin, busulfan, cis-platin, and vincristine sulfate); hormones (e.g., medroxyprogesterone, estramustine phosphate sodium, ethinyl estradiol, estradiol, megestrol acetate, methyltestosterone, diethylstilbestrol diphosphate, chlorotrianisene, and testolactone); nitrogen mustard derivatives (e.g., mephalen, chorambucil, mechlorethamine (nitrogen mustard) and thiotepa); steroids and combinations (e.g., bethamethasone sodium phosphate); and others (e.g., dicarbazine, asparaginase, mitotane, vincristine sulfate, vinblastine sulfate, and etoposide). \\n     In a specific embodiment, a therapeutic of the present invention is administered in combination with CHOP (cyclophosphamide, doxorubicin, vincristine, and prednisone) or any combination of the components of CHOP. In another embodiment, a therapeutic of the present invention is administered in combination with RITUXIMAB®. In a further embodiment, a therapeutic of the present invention is administered with RITUXIMAB® and CHOP, or RITUXIMAB® and any combination of the components of CHOP. \\n     In an additional embodiment, a therapeutic of the present invention is administered in combination with cytokines. Cytokines that may be administered with a therapeutic of the present invention include, but are not limited to, IL2, IL3, IL-4, IL5, IL6, IL7, IL10, IL12, IL13, IL15, anti-CD40, CD40L, IFN-gamma and TNF-alpha. In another embodiment, a therapeutic of the present invention may be administered with any interleukin, including, but not limited to, IL-1alpha, IL-1beta, IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, IL-8, IL-9, IL-10, IL-11, IL-12, IL-13, IL-14, IL-15, IL-16, IL-17, IL-18, IL-19, IL-20, and IL-21. \\n     In an additional embodiment, a therapeutic of the present invention is administered in combination with angiogenic proteins. Angiogenic proteins that may be administered with a therapeutic of the present invention includes, but are not limited to, Glioma Derived Growth Factor (GDGF), as disclosed in EP 399816; Platelet Derived Growth Factor-A (PDGF-A), as disclosed in EP 682110; Platelet Derived Growth Factor-B (PDGF-B), as disclosed in EP 282317; Placental Growth Factor (PlGF), as disclosed in PCT Publication WO 92/06194; Placental Growth Factor-2 (PlGF-2), as disclosed in Hauser et al., (1993)  Growth Factors  4:259–268; Vascular Endothelial Growth Factor (VEGF), as disclosed in PCT Publication WO 90/13649; Vascular Endothelial Growth Factor-A (VEGF-A), as disclosed in EP 506477; Vascular Endothelial Growth Factor-2 (VEGF-2), as disclosed in PCT Publication WO 96/39515; Vascular Endothelial Growth Factor B (VEGF-3); Vascular Endothelial Growth Factor B-186 (VEGF-B186), as disclosed in PCT Publication WO 96/26736; Vascular Endothelial Growth Factor-D (VEGF-D), as disclosed in PCT Publication WO 98/02543; Vascular Endothelial Growth Factor-D (VEGF-D), as disclosed in PCT Publication WO 98/07832; and Vascular Endothelial Growth Factor-E (VEGF-E), as disclosed in German Patent Number DE19639601. The above mentioned references are incorporated herein by reference. \\n     In an additional embodiment, a therapeutic of the present invention is administered in combination with hematopoietic growth factors. Hematopoietic growth factors that may be administered with a therapeutic of the present invention include, but are not limited to, LEUKINE® (SARGRAMOSTIM) and NEUPOGEN® (FILGRASTIM). \\n     In an additional embodiment, a therapeutic of the present invention is administered in combination with Fibroblast Growth Factors. Fibroblast Growth Factors that may be administered with a therapeutic of the present invention include, but are not limited to, FGF-1, FGF-2, FGF-3, FGF-4, FGF-5, FGF-6, FGF-7, FGF-8, FGF-9, FGF-10, FGF-11, FGF-12, FGF-13, FGF-14, and FGF-15. \\n     In a specific embodiment, formulations of the present invention may further comprise antagonists of P-glycoprotein (also referred to as the multiresistance protein, or PGP), including antagonists of its encoding polynucleotides (e.g., antisense oligonucleotides, ribozymes, zinc-finger proteins, etc.). P-glycoprotein is well known for decreasing the efficacy of various drug administrations due to its ability to export intracellular levels of absorbed drug to the cell exterior. While this activity has been particularly pronounced in cancer cells in response to the administration of chemotherapy regimens, a variety of other cell types and the administration of other drug classes have been noted (e.g., T-cells and anti-HIV drugs). In fact, certain mutations in the PGP gene significantly reduces PGP function, making it less able to force drugs out of cells. People who have two versions of the mutated gene—one inherited from each parent—have more than four times less PGP than those with two normal versions of the gene. People may also have one normal gene and one mutated one. Certain ethnic populations have increased incidence of such PGP mutations. Among individuals from Ghana, Kenya, the Sudan, as well as African Americans, frequency of the normal gene ranged from 73% to 84%. In contrast, the frequency was 34% to 59% among British whites, Portuguese, Southwest Asian, Chinese, Filipino and Saudi populations. As a result, certain ethnic populations may require increased administration of PGP antagonist in the formulation of the present invention to arrive at the an efficacious dose of therapeutic (e.g., those from african descent). Conversly, certain ethnic populations, particularly those having increased frequency of the mutated PGP (e.g., of caucasian descent, or non-african descent) may require less pharmaceutical compositions in the formulation due to an effective increase in efficacy of such compositions as a result of the increased effective absorption (e.g., less PGP activity) of said composition. \\n     In additional embodiments, a therapeutic of the present invention is administered in combination with other therapeutic or prophylactic regimens, such as, for example, radiation therapy, KINERET therapy or anti-TNF therapy. \\n     Example 27 \\n     Method of Treating Decreased Levels of a Polypeptide of the Present Invention \\n     The present invention relates to a method for treating an individual in need of an increased level of a polypeptide of the present invention comprising administering to such an individual a composition comprising a therapeutically effective amount of an agonist of the present invention (including polypeptides of the present invention). Moreover, it will be appreciated that conditions caused by a decrease in the standard or normal expression level of a secreted protein in an individual can be treated by administering a polypeptide of the present invention, preferably in the secreted form. \\n     Thus, the present invention also provides a method of treatment of an individual in need of an increased level of a polypeptide of the present invention comprising administering to such an individual a therapeutic comprising an amount of the polypeptide to increase the activity level of the polypeptide in such an individual. This therapy can be employed alone or in combination with another therapy. \\n     For example, a subject with decreased levels of a polypeptide of the present invention receives a daily dose 0.1–100 μg/kg of the polypeptide for six consecutive days. The polypeptide can be, for example, in the secreted form. A representative dosing scheme, based on administration and formulation, is provided herein. \\n     In another method, the in vivo level of a polypeptide of the present invention is elevated by increasing the transcription of the polynucleotide encoding the polypeptide. As described furthere herein, this method can include introducing a nucleic acid sequence encoding the polypeptide having decreased or undesirably low levels. \\n     Example 28 \\n     Method of Treating Increased Levels of a Polypeptide of the Present Invention \\n     The present invention also relates to a method of treating an individual in need of a decreased level of a polypeptide of the present invention in the body comprising administering to such an individual a composition comprising a therapeutically effective amount of an antagonist of the present invention (including polypeptides and antibodies of the present invention). This therapy can be employed alone or in combination with another therapy. \\n     In one example, antisense technology is used to inhibit production of a polypeptide of the present invention. This technology is one example of a method of decreasing levels of a polypeptide, preferably a secreted form, due to a variety of etiologies, such as cancer. For example, a subject diagnosed with abnormally increased levels of a polypeptide is administered intravenously antisense polynucleotides at 0.5, 1.0, 1.5, 2.0 and 3.0 mg/kg day for 21 days. This treatment is repeated after a 7-day rest period if the treatment was well tolerated. The formulation of the antisense polynucleotide is provided herein. \\n     Example 29 \\n     Method of Treatment Using Gene Therapy-Ex Vivo \\n     In one embodiment of the present invention, ex vivo gene therapy is employed to treat a condition. One method of gene therapy transplants fibroblasts, which are capable of expressing a polypeptide, onto a subject. Generally, fibroblasts are obtained from a subject by skin biopsy. The resulting tissue is placed in tissue-culture medium and separated into small pieces. Small chunks of the tissue are placed on a wet surface of a tissue culture flask, approximately ten pieces are placed in each flask. The flask is turned upside down, closed tight and left at room temperature over night. After 24 hours at room temperature, the flask is inverted and the chunks of tissue remain fixed to the bottom of the flask and fresh media (e.g., Ham&#39;s F12 media, with 10% FBS, penicillin and streptomycin) is added. The flasks are then incubated at 37° C. for approximately one week. \\n     At this time, fresh media is added and subsequently changed every several days. After an additional two weeks in culture, a monolayer of fibroblasts emerges. The monolayer is trypsinized and scaled into larger flasks. \\n     pMV-7 (Kirschmeier et al., (1988)  DNA  7:219–25), flanked by the long terminal repeats of the Moloney murine sarcoma virus, is digested with EcoRI and HindIII and subsequently treated with calf intestinal phosphatase. The linear vector is fractionated on agarose gel and purified, using glass beads. \\n     The cDNA encoding a polypeptide of the present invention can be amplified using PCR primers that correspond to the 5′ and 3′ end sequences respectively as set forth herein using primers and having appropriate restriction sites and initiation/stop codons, if necessary. In one embodiment, the 5′ primer contains an EcoRI site and the 3′ primer includes a HindIII site. Equal quantities of the Moloney murine sarcoma virus linear backbone and the amplified EcoRI and HindIII fragment are added together, in the presence of T4 DNA ligase. The resulting mixture is maintained under conditions appropriate for ligation of the two fragments. The ligation mixture is then used to transform bacteria HB101, which are then plated onto agar containing kanamycin for the purpose of confirming that the vector has the gene of interest properly inserted. \\n     The amphotropic pA317 or GP+am12 packaging cells are grown in tissue culture to confluent density in Dulbecco&#39;s Modified Eagles Medium (DMEM) with 10% calf serum (CS), penicillin and streptomycin. The MSV vector containing the gene is then added to the media and the packaging cells transduced with the vector. The packaging cells now produce infectious viral particles containing the gene (the packaging cells are now referred to as producer cells). \\n     Fresh media is added to the transduced producer cells, and subsequently, the media is harvested from a 10 cm plate of confluent producer cells. The spent media, containing the infectious viral particles, is filtered through a millipore filter to remove detached producer cells and this media is then used to infect fibroblast cells. Media is removed from a sub-confluent plate of fibroblasts and quickly replaced with the media from the producer cells. This media is removed and replaced with fresh media. If the titer of virus is high, then virtually all fibroblasts will be infected and no selection is required. If the titer is very low, then it is necessary to use a retroviral vector that has a selectable marker, such as neo or his. Once the fibroblasts have been efficiently infected, the fibroblasts are analyzed to determine whether protein is produced (e.g., by employing a GTPase activity assay). \\n     The engineered fibroblasts are then transplanted onto the host, either alone or after having been grown to confluence on CYTODEX 3 microcarrier beads. \\n     Example 30 \\n     Gene Therapy Using an Endogenous Polynucleotide of the Present Invention \\n     Another method of gene therapy according to the present invention involves operably associating an endogenous polynucleotide sequence of the present invention with a promoter via homologous recombination as described, for example, in U.S. Pat. No. 5,641,670; PCT Publication WO 96/29411; PCT Publication WO 94/12650; Koller et al., (1989)  Proc. Natl. Acad. Sci. U.S.A.  86:8932–8935; and Zijlstra et al., (1989)  Nature  342:435–438. This method involves the activation of a gene which is present in the target cells, but which is not expressed in the cells, or is expressed at a lower level than desired. This therapy can be employed alone or in combination with another therapy. \\n     Polynucleotide constructs are made which contain a promoter and targeting sequences, which are homologous to the 5′ non-coding sequence of endogenous polynucleotide sequence, flanking the promoter. The targeting sequence will be sufficiently near the 5′ end of the polynucleotide sequence so the promoter will be operably linked to the endogenous sequence upon homologous recombination. The promoter and the targeting sequences can be amplified using PCR. The amplified promoter can contain distinct restriction enzyme sites on the 5′ and 3′ ends. For example, the 3′ end of the first targeting sequence can contain the same restriction enzyme site as the 5′ end of the amplified promoter and the 5′ end of the second targeting sequence contains the same restriction site as the 3′ end of the amplified promoter. \\n     The amplified promoter and the amplified targeting sequences are digested with the appropriate restriction enzymes and subsequently treated with calf intestinal phosphatase. The digested promoter and digested targeting sequences are added together in the presence of T4 DNA ligase. The resulting mixture is maintained under conditions appropriate for ligation of the two fragments. The construct is size fractionated on an agarose gel then purified by phenol extraction and ethanol precipitation. \\n     In this Example, the polynucleotide constructs are administered as naked polynucleotides via electroporation. However, the polynucleotide constructs may also be administered with transfection-facilitating agents, such as liposomes, viral sequences, viral particles, precipitating agents, etc. Such methods of delivery are known in the art. \\n     Once the cells are transfected, homologous recombination will take place that results in the promoter being operably linked to the endogenous polynucleotide sequence. This results in the expression of polynucleotide corresponding to the polynucleotide in the cell. Expression may be detected by immunological staining, or any other method known in the art. \\n     Fibroblasts are obtained from a subject by skin biopsy. The resulting tissue is placed in DMEM+10% fetal calf serum. Exponentially growing or early stationary phase fibroblasts are trypsinized and rinsed from the plastic surface with nutrient medium. An aliquot of the cell suspension is removed for counting, and the remaining cells are subjected to centrifugation. The supernatant is aspirated and the pellet is resuspended in 5 ml of electroporation buffer (20 mM HEPES pH 7.3, 137 mM NaCl, 5 mM KCl, 0.7 mM Na 2 HPO 4 , 6 mM dextrose). The cells are recentrifuged, the supernatant aspirated, and the cells resuspended in electroporation buffer containing 1 mg/ml acetylated bovine serum albumin. The final cell suspension contains approximately 3×10 6  cells/ml. Electroporation should be performed immediately following resuspension. \\n     Plasmid DNA is prepared according to standard techniques. For example, to construct a plasmid for targeting to the locus corresponding to a polynucleotide of the present invention, plasmid pUC18 (MBI Fermentas, Amherst, N.Y., USA) is digested with HindIII. The CMV promoter is amplified by PCR with an XbaI site on the 5′ end and a BamHI site on the 3′ end. Two non-coding sequences are amplified via PCR: one non-coding sequence (fragment 1) is amplified with a HindIII site at the 5′ end and an Xba site at the 3′ end; the other non-coding sequence (fragment 2) is amplified with a BamHI site at the 5′ end and a HindIII site at the 3′ end. The CMV promoter and the fragments (1 and 2) are digested with the appropriate enzymes (CMV promoter-XbaI and BamHI; fragment 1-XbaI; fragment 2-BamHI) and ligated together. The resulting ligation product is digested with HindIII, and ligated with the HindIII-digested pUC18 plasmid. \\n     Plasmid DNA is added to a sterile cuvette with about a 0.4 cm electrode gap (Bio-Rad, Hercules, Calif., USA). The final DNA concentration is generally at least about 120μg/ml. 0.5 ml of the cell suspension (containing approximately 1.5×10 6  cells) is then added to the cuvette, and the cell suspension and DNA solutions are gently mixed. Electroporation is performed with a Gene-Pulser apparatus (Bio-Rad, Hercules, Calif., USA). Capacitance and voltage are set at about 960 μF and about 250–300 V, respectively. As voltage increases, cell survival decreases, but the percentage of surviving cells that stably incorporate the introduced DNA into their genome increases dramatically. Given these parameters, a pulse time of approximately 14–20 mSec should be observed. \\n     Electroporated cells are maintained at room temperature for approximately 5 min, and the contents of the cuvette are then gently removed with a sterile transfer pipette. The cells are added directly to about 10 ml of prewarmed nutrient media (DMEM with 15% calf serum) in a 10 cm dish and incubated at 37° C. The following day, the media is aspirated and replaced with 10 ml of fresh media and incubated for a further 16–24 hours. \\n     The engineered fibroblasts are then injected into the host, either alone or after having been grown to confluence on CYTODEX 3 microcarrier beads. The fibroblasts now produce the protein product. The fibroblasts can then be introduced into a subject as described above. \\n     Example 31 \\n     Method of Treatment Using Gene Therapy—In Vivo \\n     Another aspect of the present invention is using in vivo gene therapy methods to treat disorders, diseases and conditions. The gene therapy method relates to the introduction of naked nucleic acid (DNA, RNA, and antisense DNA or RNA) sequences into an animal to increase or decrease the expression of a polypeptide of the present invention. A polynucleotide of the present invention may be operatively linked to a promoter or any other genetic elements necessary for the expression of the polypeptide by the target tissue. Such gene therapy and delivery techniques and methods are known in the art, see, for example, PCT Publications WO 90/11092 and WO 98/11779; U.S. Pat. Nos. 5,693,622, 5,705,151, 5,580,859; Tabata et al., (1997)  Cardiovasc. Res.  35(3):470–479; Chao et al., (1997)  Pharmacol. Res.  35(6):517–522; Wolff, (1997)  Neuromuscul. Disord.  7(5):314–318; Schwartz et al., (1996)  Gene Ther.  3(5):405–411; Tsurumi et al., (1996)  Circulation  94(12):3281–3290 (1996) (all incorporated herein by reference). This therapy can be employed alone or in combination with another therapy, such as a KINERET or anti-TNF therapy. \\n     A polynucleotide construct may be delivered by any method that delivers injectable materials to the cells of an animal, such as, injection into the interstitial space of tissues (heart, muscle, skin, lung, liver, intestine and the like). A polynucleotide construct can be delivered in a pharmaceutically acceptable liquid or aqueous carrier. \\n     The term “naked” polynucleotide, DNA or RNA, refers to sequences that are free from any delivery vehicle that acts to assist, promote, or facilitate entry into the cell, including viral sequences, viral particles, liposome formulations, lipofectin or precipitating agents and the like. However, a polynucleotide of the present invention may also be delivered in liposome formulations (such as those taught in Felgner et al., (1995)  Ann. N.Y. Acad. Sci.  772:126–139 and Abdallah et al., (1995)  Biol. Cell  85(1):1–7), which can be prepared by methods known to those of ordinary skill in the art. \\n     The polynucleotide vector constructs used in the gene therapy method are preferably constructs that will not integrate into the host genome nor will they contain sequences that allow for replication. Any strong promoter known to those skilled in the art can be used for driving the expression of DNA. Unlike other gene therapies techniques, one major advantage of introducing naked nucleic acid sequences into target cells is the transitory nature of the polynucleotide synthesis in the cells. Studies have shown that non-replicating DNA sequences can be introduced into cells to provide production of the desired polypeptide for periods of up to six months. \\n     The polynucleotide construct can be delivered to the interstitial space of tissues within the an animal, including of muscle, skin, brain, lung, liver, spleen, bone marrow, thymus, heart, lymph, blood, bone, cartilage, pancreas, kidney, gall bladder, stomach, intestine, testis, ovary, uterus, rectum, nervous system, eye, gland, and connective tissue. Interstitial space of the tissues comprises the intercellular fluid, mucopolysaccharide matrix among the reticular fibers of organ tissues, elastic fibers in the walls of vessels or chambers, collagen fibers of fibrous tissues, or that same matrix within connective tissue ensheathing muscle cells or in the lacunae of bone. It is similarly the space occupied by the plasma of the circulation and the lymph fluid of the lymphatic channels. Delivery to the interstitial space of muscle tissue is often desirable for the reasons discussed herein. The construct may be conveniently delivered by injection into the tissues comprising these cells. In one embodiment, the construct is delivered to, and expressed in, persistent, non-dividing cells which are differentiated, although delivery and expression may be achieved in non-differentiated or less completely differentiated cells, such as, for example, stem cells of blood or skin fibroblasts. In vivo muscle cells are particularly competent in their ability to take up and express polynucleotides. \\n     For a naked polynucleotide injection, an effective dosage amount of DNA or RNA will be in the range of from about 0.05 g/kg body weight to about 50 mg/kg body weight. In one embodiment, the dosage will be from about 0.005 mg/kg to about 20 mg/kg and in another embodiment from about 0.05 mg/kg to about 5 mg/kg. Of course, as those of ordinary skill in the art will appreciate, this dosage will vary according to the tissue site of injection. The appropriate and effective dosage of nucleic acid sequence can readily be determined by those of ordinary skill in the art and may depend on the condition being treated and the route of administration. The representative route of administration is by the parenteral route of injection into the interstitial space of tissues. However, other parenteral routes may also be used, such as inhalation of an aerosol formulation particularly for delivery to lungs or bronchial tissues, throat or mucous membranes of the nose. In addition, naked polynucleotide constructs can be delivered to arteries during angioplasty by the catheter used in the procedure. \\n     The dose response effects of injected polynucleotide in muscle in vivo is determined as follows. Suitable template DNA for production of mRNA coding for polypeptide of the present invention is prepared in accordance with a standard recombinant DNA methodology. The template DNA, which may be either circular or linear, is either used as naked DNA or complexed with liposomes. The quadriceps muscles of mice are then injected with various amounts of the template DNA. \\n     Five to six week old female and male Balb/C mice are anesthetized by intraperitoneal injection with 0.3 ml of 2.5% Avertin. A 1.5 cm incision is made on the anterior thigh, and the quadriceps muscle is directly visualized. The template DNA is injected in 0.1 ml of carrier in a 1 cc syringe through a 27 gauge needle over one minute, approximately 0.5 cm from the distal insertion site of the muscle into the knee and about 0.2 cm deep. A suture is placed over the injection site for future localization, and the skin is closed with stainless steel clips. \\n     After an appropriate incubation time (e.g., 7 days) muscle extracts are prepared by excising the entire quadriceps. Every fifth 15 μm cross-section of the individual quadriceps muscles is histochemically stained for protein expression. A time course for protein expression may be done in a similar fashion except that quadriceps from different mice are harvested at different times. Persistence of DNA in muscle following injection may be determined by Southern blot analysis after preparing total cellular DNA and HIRT supernatants from injected and control mice. The results of the above experimentation in mice can be use to extrapolate proper dosages and other treatment parameters in humans and other animals using naked DNA. \\n     Example 32 \\n     Transgenic Animals \\n     The polypeptides of the present invention can also be expressed in transgenic animals. Animals of any species, including, but not limited to, mice, rats, rabbits, hamsters, guinea pigs, pigs, goats, sheep, cows and non-human primates, e.g., baboons, monkeys, and chimpanzees may be used to generate transgenic animals. In a specific embodiment, techniques described herein or otherwise known in the art, are used to express polypeptides of the present invention in humans, as part of a gene therapy protocol. \\n     Any technique known in the art may be used to introduce the transgene (i.e., polynucleotides of the present invention) into animals to produce the founder lines of transgenic animals. Such techniques include, but are not limited to, pronuclear microinjection (Paterson et al., (1994)  Appl. Microbiol. Biotechnol.  40:691–698; Carver et al., (1993)  Biotechnology  11:1263–1270; Wright et al., (1991)  Biotechnology  9:830–834; and U.S. Pat. No. 4,873,191); retrovirus mediated gene transfer into germ lines (Van der Putten et al., (1985)  Proc. Natl. Acad. Sci. U.S.A.  82:6148–6152), blastocysts or embryos; gene targeting in embryonic stem cells (Thompson et al., (1989)  Cell  56:313–321); electroporation of cells or embryos (Lo, (1983)  Mol Cell. Biol.  3:1803–1814); introduction of the polynucleotides of the present invention using a gene gun (see, e.g., Ulmer et al., (1993)  Science  259:1745; introducing nucleic acid constructs into embryonic pleuripotent stem cells and transferring the stem cells back into the blastocyst; and sperm-mediated gene transfer (Lavitrano et al., (1989)  Cell  57:717–723; etc. For a review of such techniques, see Gordon, (1989)  Intl. Rev. Cytol.  115:171–229, which is incorporated by reference herein in its entirety. \\n     Any technique known in the art may be used to produce transgenic clones containing polynucleotides of the present invention, for example, nuclear transfer into enucleated oocytes of nuclei from cultured embryonic, fetal, or adult cells induced to quiescence (Campell et al., (1996)  Nature  380:64–66; Wilmut et al., (1997)  Nature  385:810–813). \\n     The present invention provides for transgenic animals that carry the transgene in all their cells, as well as animals which carry the transgene in some, but not all their cells, i.e., mosaic animals or chimeric. The transgene may be integrated as a single transgene or as multiple copies such as in concatamers, e.g., head-to-head tandems or head-to-tail tandems. The transgene may also be selectively introduced into and activated in a particular cell type by following, for example, the teaching of Lasko et al. (Lasko et al., (1992)  Proc. Natl. Acad. Sci. U.S.A.  89:6232–6236). The regulatory sequences required for such a cell-type specific activation will depend upon the particular cell type of interest, and will be apparent to those of skill in the art. \\n     When it is desired that the polynucleotide transgene be integrated into the chromosomal site of the endogenous gene, gene targeting is sometimes desirable. Briefly, when such a technique is to be utilized, vectors containing some nucleotide sequences homologous to the endogenous gene are designed for the purpose of integrating, via homologous recombination with chromosomal sequences, into and disrupting the function of the nucleotide sequence of the endogenous gene. The transgene may also be selectively introduced into a particular cell type, thus inactivating the endogenous gene in only that cell type, by following, for example, the teaching of Gu et al. (Gu et al., (1994)  Science  265:103–106). The regulatory sequences required for such a cell-type specific inactivation will depend upon the particular cell type of interest, and will be apparent to those of ordinary skill in the art. \\n     Once transgenic animals have been generated, the expression of the recombinant gene may be assayed utilizing standard techniques. Initial screening may be accomplished by Southern blot analysis or PCR techniques to analyze animal tissues to verify that integration of the transgene has taken place. The level of mRNA expression of the transgene in the tissues of the transgenic animals may also be assessed using techniques which include, but are not limited to, Northern blot analysis of tissue samples obtained from the animal, in situ hybridization analysis, and reverse transcriptase-PCR (RT-PCR). Samples of transgenic gene-expressing tissue may also be evaluated immunocytochemically or immunohistochemically using antibodies specific for the transgene product. \\n     Once the founder animals are produced, they may be bred, inbred, outbred, or crossbred to produce colonies of the particular animal. Examples of such breeding strategies include, but are not limited to: outbreeding of founder animals with more than one integration site in order to establish separate lines; inbreeding of separate lines in order to produce compound transgenics that express the transgene at higher levels because of the effects of additive expression of each transgene; crossing of heterozygous transgenic animals to produce animals homozygous for a given integration site in order to both augment expression and eliminate the need for screening of animals by DNA analysis; crossing of separate homozygous lines to produce compound heterozygous or homozygous lines; and breeding to place the transgene on a distinct background that is appropriate for an experimental model of interest. \\n     Transgenic animals of the present invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying diseases, disorders, and/or conditions associated with aberrant expression, and in screening for compounds effective in ameliorating such diseases, disorders, and/or conditions. \\n     Example 33 \\n     Knock-Out Animals \\n     Endogenous gene expression can also be reduced by inactivating or “knocking out” the gene and/or its promoter using targeted homologous recombination. (see, e.g., Smithies et al., (1985)  Nature  317:230–234; Thomas &amp; Capecchi, (1987)  Cell  51:503–512; Thompson et al., (1989)  Cell  5:313–321; each of which is incorporated by reference herein in its entirety) or another method. For example, a mutant, non-functional polynucleotide of the present invention (or a completely unrelated DNA sequence) flanked by DNA homologous to the endogenous polynucleotide sequence (either the coding regions or regulatory regions of the gene) can be used, with or without a selectable marker and/or a negative selectable marker, to transfect cells that express polypeptides of the present invention in vivo. \\n     In another embodiment, techniques known in the art are used to generate knockouts in cells that contain, but do not express the gene of interest. Insertion of the DNA construct, via targeted homologous recombination, results in inactivation of the targeted gene. Such approaches are particularly suited in research and agricultural fields where modifications to embryonic stem cells can be used to generate animal offspring with an inactive targeted gene (see, e.g., Thomas &amp; Capecchi, (1987)  Cell  51:503–512; Thompson et al., (1989)  Cell  5:313–321). However, this approach can be routinely adapted for use in humans provided the recombinant DNA constructs are directly administered or targeted to the required site in vivo using appropriate viral vectors that will be apparent to those of ordinary skill in the art. \\n     In further embodiments of the present invention, cells that are genetically engineered to express the polypeptides of the present invention, or alternatively, that are genetically engineered not to express the polypeptides of the present invention (e.g., knockouts) are administered to a subject in vivo. Such cells may be obtained from the subject (i.e., animal, including human) or an MHC compatible donor and can include, but are not limited to fibroblasts, bone marrow cells, blood cells (e.g., lymphocytes), adipocytes, muscle cells, endothelial cells etc. The cells are genetically engineered in vitro using recombinant DNA techniques to introduce the coding sequence of polypeptides of the present invention into the cells, or alternatively, to disrupt the coding sequence and/or endogenous regulatory sequence associated with the polypeptides of the present invention, e.g., by transduction (using viral vectors, and preferably vectors that integrate a transgene, such as neo, into the cell genome) or transfection procedures, including, but not limited to, the use of plasmids, cosmids, YACs, naked DNA, electroporation, liposomes, etc. The coding sequence of the polypeptides of the present invention can be placed under the control of a strong constitutive or inducible promoter or promoter/enhancer to achieve expression, and preferably secretion, of the polypeptides of the present invention. The engineered cells that express and, in some cases, secrete the polypeptides of the present invention can be introduced into the subject systemically, e.g., in the circulation, or intraperitoneally. \\n     Alternatively, the cells can be incorporated into a matrix and implanted in the body, e.g., genetically engineered fibroblasts can be implanted as part of a skin graft; genetically engineered endothelial cells can be implanted as part of a lymphatic or vascular graft (see, for example, U.S. Pat. No. 5,399,349; and U.S. Pat. No. 5,460,959 each of which is incorporated by reference herein in its entirety). \\n     When the cells to be administered are non-autologous or non-MHC compatible cells, they can be administered using well known techniques which prevent the development of a host immune response against the introduced cells. For example, the cells may be introduced in an encapsulated form, which, while allowing for an exchange of components with the immediate extracellular environment, does not allow the introduced cells to be recognized by the host immune system. \\n     Transgenic and “knock-out” animals of the present invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying diseases, disorders, and/or conditions associated with aberrant expression, and in screening for compounds effective in ameliorating such diseases, disorders, and/or conditions. \\n     Example 34 \\n     Method of Isolating Antibody Fragments Directed against HGBPBMY1 (AK096141). HGPBMY2, HGBPBMY3 (4843 30 2 1; 4843 2). HGBPBMY4 (FLJ10961). MGPBMY1, MGBPBMY2 (LOC229902). MGBPBMY and/or MGBPBMY4 (BC007143) from a Library of scFvs \\n     Naturally occurring V-genes isolated from human PBLs can be constructed into a library of antibody fragments that contain reactivities against HGPBMY1, HGBPBMY2 (4843 30 1 1; 4843 — 1), HGBPBMY3 (4843 30 2 1; 4843 — 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBPBMY2 (LOC229902), MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143), to which the donor may or may not have been exposed (see e.g., U.S. Pat. No. 5,885,793 incorporated herein by reference in its entirety). \\n     Rescue of the Library \\n     A library of scFvs is constructed from the RNA of human PBLs as described in PCT Publication WO 92/01047. To rescue phage displaying antibody fragments, approximately 10 9    E. coli  harboring the phagemid are used to inoculate 50 ml of 2×TY containing 1% glucose and 100 μg/ml of ampicillin (2×TY-AMP-GLU) and grown to an O.D. of 0.8 with shaking. Five ml of this culture is used to inoculate 50 ml of 2×TY-AMP-GLU, 2×10 8  TU of delta gene 3 helper (M13 delta gene III, see PCT Publication WO 92/01047) are added and the culture incubated at 37° C. for 45 minutes without shaking and then at 37° C. for 45 minutes with shaking. The culture is centrifuged at 4000 r.p.m. for 10 min. and the pellet resuspended in 2 liters of 2×TY containing 100 μg/ml ampicillin and 50 μg/ml kanamycin and grown overnight. Phage are prepared using known techniques, such as those described in PCT Publication WO 92/01047. \\n     In one embodiment, M13 delta gene II is emplpoyed and is prepared as follows: M13 delta gene III helper phage does not encode gene III protein, hence the phage(mid) displaying antibody fragments have a greater avidity of binding to antigen. Infectious M13 delta gene III particles are made by growing the helper phage in cells harboring a pUC19 derivative supplying the wild type gene III protein during phage morphogenesis. The culture is incubated for 1 hour at 37° C. without shaking and then for a further hour at 37° C. with shaking. Cells are spun down (IEC-Centra 8,400 r.p.m. for 10 min), resuspended in 300 ml 2×TY broth containing 100 μg ampicillin/ml and 25 ug kanamycin/ml (2×TY-AMP-KAN) and grown overnight, shaking at 37° C. Phage particles are purified and concentrated from the culture medium by two PEG-precipitations, resuspended in 2 ml PBS and passed through a 0.45 μm filter (Minisart NML; Sartorius Corp., Edgewood, N.Y., USA) to give a final concentration of approximately 1013 transducing units/ml (ampicillin-resistant clones). \\n     Panning of the Library \\n     Immunotubes (Nunc, Roskilde, Denmark) are coated overnight in PBS with 4 ml of either 100 μg/ml or 10 μg/ml of a polypeptide of the present invention. Tubes are blocked with 2% Marvel-PBS for 2 hours at 37° C. and then washed 3 times in PBS. Approximately 1013 TU of phage is applied to the tube and incubated for 30 minutes at room temperature tumbling on an over and under turntable and then left to stand for another 1.5 hours. Tubes are washed 10 times with PBS, 0.1% Tween-20 and 10 times with PBS. Phage are eluted by adding 1 ml of 100 mM triethylamine and rotating 15 minutes on an under and over turntable after which the solution is immediately neutralized with 0.5 ml of 1.0M Tris-HCl, pH 7.4. Phage are then used to infect 10 ml of mid-log  E. coli  TG1 by incubating eluted phage with bacteria for 30 minutes at 37° C. The  E. coli  are then plated on TYE plates containing 1% glucose and 100 μg/ml ampicillin. The resulting bacterial library is then rescued with delta gene 3 helper phage as described above to prepare phage for a subsequent round of selection. This process is then repeated for a total of 4 rounds of affinity purification with tube-washing increased to 20 times with PBS, 0.1% Tween-20 and 20 times with PBS for rounds 3 and 4. \\n     Characterization of Binders \\n     Eluted phage from the 3rd and 4th rounds of selection are used to infect  E. coli  HB 2151 and soluble scFv is produced from single colonies for assay. ELISAs are performed with microtitre plates coated with either 10 pg/ml of a polypeptide of the present invention in 50 mM bicarbonate pH 9.6. Clones positive in ELISA are further characterized by PCR fingerprinting (see, e.g., PCT Publication WO 92/01047) and then by sequencing. These ELISA positive clones may also be further characterized by techniques known in the art, such as, for example, epitope mapping, binding affinity, receptor signal transduction, ability to block or competitively inhibit antibody/antigen binding, and competitive agonistic or antagonistic activity. \\n     Moreover, in another preferred method, the antibodies directed against the polypeptides of the present invention may be produced in plants. Specific methods are disclosed in U.S. Pat. Nos. 5,959,177, and 6,080,560, which are hereby incorporated in their entirety herein. The methods not only describe methods of expressing antibodies, but also the means of assembling foreign multimeric proteins in plants (i.e., antibodies, etc,), and the subsequent secretion of such antibodies from the plant. \\n     Example 35 \\n     Biological Effects of HGBPBMY1 (AK096141), HGBPBMY2 (4843 30 1 1, 4843 1), HGBPBMY3 (4843 30 2 1; 4843 2), HGBPBMY4 (FLJ10961), MGBPBMY1 (LOC229900), MGBBMY2, MGBPBMY3 (BC031475) and/or MGBPBMY4 (BC007143) Polypeptides of the Present Invention \\n     Astrocyte and Neuronal Assays \\n     Recombinant polypeptides of the present invention, expressed in  E. coli  and purified as described herein, can be tested for activity in promoting the survival, neurite outgrowth, or phenotypic differentiation of cortical neuronal cells and for inducing the proliferation of glial fibrillary acidic protein immunopositive cells, astrocytes. The selection of cortical cells for the bioassay is based on the prevalent expression of FGF-1 and FGF-2 in cortical structures and on the previously reported enhancement of cortical neuronal survival resulting from FGF-2 treatment. A thymidine incorporation assay, for example, can be used to elucidate a polypeptide of the present invention&#39;s activity on these cells. \\n     Moreover, previous reports describing the biological effects of FGF-2 (basic FGF) on cortical or hippocampal neurons in vitro have demonstrated increases in both neuron survival and neurite outgrowth (Walicke et al., (1986)  Proc. Natl. Acad. Sci. U.S.A.  83:3012–3016, assay herein incorporated by reference in its entirety). However, reports from experiments done on PC-12 cells suggest that these two responses are not necessarily synonymous and may depend on not only which FGF is being tested but also on which receptor(s) are expressed on the target cells. Using the primary cortical neuronal culture paradigm, the ability of a polypeptide of the present invention to induce neurite outgrowth can be compared to the response achieved with FGF-2 using, for example, a thymidine incorporation assay. \\n     Fibroblast and Endothelial Cell Assays \\n     In one emobidment, human lung fibroblasts are obtained from Clonetics (San Diego, Calif., USA) and maintained in growth media from Clonetics. Dermal microvascular endothelial cells are obtained from Cell Applications (San Diego, Calif., USA). For proliferation assays, the human lung fibroblasts and dermal microvascular endothelial cells can be cultured at 5,000 cells/well in a 96-well plate for one day in growth medium. The cells are then incubated for one day in 0.1% BSA basal medium. After replacing the medium with fresh 0.1% BSA medium, the cells are incubated with the test proteins for 3 days. Alamar Blue (Alamar Biosciences, Sacramento, Calif.) is added to each well to a final concentration of 10%. The cells are incubated for 4 hr. Cell viability is measured by reading in a CYTOFLUOR fluorescence reader (Applied Biosystems, Foster City, Calif., USA). For the PGE2 assays, the human lung fibroblasts are cultured at 5,000 cells/well in a 96-well plate for one day. After a medium change to 0.1% BSA basal medium, the cells are incubated with FGF-2 or polypeptides of the present invention with or without IL-1 for 24 hours. The supernatants are collected and assayed for PGE2 by EIA kit (Cayman, Ann Arbor, Mich., USA). For the 1L-6 assays, the human lung fibroblasts are cultured at 5,000 cells/well in a 96-well plate for one day. After a medium change to 0.1% BSA basal medium, the cells are incubated with FGF-2 or with or without polypeptides of the present invention for 24 hours. The supernatants are collected and assayed for 1L-6 by ELISA kit (Endogen, Cambridge, Mass., USA). \\n     Human lung fibroblasts are cultured with FGF-2 or polypeptides of the present invention for 3 days in basal medium before the addition of Alamar Blue to assess effects on growth of the fibroblasts. FGF-2 should show a stimulation at 10–2500 ng/ml which can be used to compare stimulation with polypeptides of the present invention. \\n     Example 36 \\n     Method of Assessing the Expression Profile of a HGBPBMY1 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For HGBPBMY1, the primer probe sequences were as follows: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                   \\n                   \\n               \\n               \\n                 5′- AGTGCCCAGAAAAGGAGTTAAGG -3′ \\n                 (SEQ ID NO: 61) \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n               \\n               \\n                 5′- GCAGGATGGATTCCTCTATAACCA -3′ \\n                 (SEQ ID NO: 62) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n               \\n               \\n                 5′- AGACGAGGTCCTCCAGAGCTTCCTGC -3′ \\n                 (SEQ ID NO: 63) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TAQMAN assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/μl of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (ΔCt)    \\n     A tissue expression profile of the HGBPBMY1 polypeptide is provided in  FIG. 43  and described herein. \\n     Example 37 \\n     Method of Assessing the Expression Profile of a HGBPBMY2 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For HGBPBMY2, the primer probe sequences were as follows: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                   \\n                   \\n               \\n               \\n                 5′- TGGAGCCCCTGCCTCAA -3′ \\n                 (SEQ ID NO: 64) \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n               \\n               \\n                 5′- CAGAAATGATGGACCTCCTCACT -3′ \\n                 (SEQ ID NO: 65) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n               \\n               \\n                 5′- ACACACATTGCTCCAACACTGAGGCC -3′ \\n                 (SEQ ID NO: 66) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TAQMAN assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/μl of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (ΔCt)    \\n     A tissue expression profile of the HGBPBMY2 polypeptide is provided in  FIG. 44  and described elsewhre herein. \\n     Example 38 \\n     Method of Assessing the Expression Profile of a HGBPBMY3 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For HGBPBMY3, the primer probe sequences were as follows: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                 5′- GGATGGCAGGAAGACAAACAG -3′ \\n                 (SEQ ID NO: 67) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n                 5′- GCCTTAGGACCCAGAGAACACA -3′ \\n                 (SEQ ID NO: 68) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n                 5′- CACCTGCTTACAGCCGTCTCCTACTTCTCACT -3′ \\n                 (SEQ ID NO: 69) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TaqMan assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/μl of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (Δct)    \\n     The tissue expression profile of the HGBPBMY3 polypeptide is provided in  FIG. 45  and described herein. \\n     Example 39 \\n     Method of Assessing the Expression Profile of a HGBPBMY4 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For HGBPBMY4, the primer probe sequences were as follows \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                 5′- AAGATGCATTTACCTCTGTACCAACA -3′ \\n                 (SEQ ID NO: 70) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n                 5′- CCACTGGTCGTCTGGAAGAATAA -3′ \\n                 (SEQ ID NO: 71) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n                 5′- AGGAGGGATCATGAGTTGCCACCACTC -3′ \\n                 (SEQ ID NO: 72) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TAQMAN assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/μl of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (ΔCt)    \\n     The tissue expression profile of the HGBPBMY4 polypeptide is provided in  FIG. 46  and described herein. \\n     Example 40 \\n     Method of Assessing the Expression Profile of a GBP1 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For GBP1, the primer probe sequences were as follows \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                 5′- AATGTTGCAGGAAATGCAAAGA -3′ \\n                 (SEQ ID NO: 73) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n                 5′- AGCAACTGGACCCTGTCGTT -3′ \\n                 (SEQ ID NO: 74) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n                 5′- CAGGAACACTTGAAACAACTGACTGAGAAGATGG -3′ \\n                 (SEQ ID NO: 75) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TAQMAN assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/μl of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (ΔCt)    \\n     The tissue expression profile of the GBP1 polypeptide is provided in  FIG. 47  and described herein. \\n     Example 41 \\n     Method of Assessing the Expression Profile of a GBP5 Polypeptide of the Present Invention Using mRNA Tissue and Cell Sources \\n     Total RNA from tissues was isolated using the TriZol protocol (Invitrogen) and quantified by determining its absorbance at 260 nm. An assessment of the 18s and 28s ribosomal RNA bands was made by denaturing gel electrophoresis to determine RNA integrity. \\n     The specific sequence to be measured was aligned with related genes found in GenBank to identity regions of significant sequence divergence to maximize primer and probe specificity. Gene-specific primers and probes were designed using the ABI primer express software to amplify small amplicons (150 base pairs or less) to maximize the likelihood that the primers function at 100% efficiency. All primer/probe sequences were searched against Public Genbank databases to ensure target specificity. Primers and probes were obtained from ABI. \\n     For GBP5, the primer probe sequences were as follows \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Forward Primer \\n                   \\n                   \\n               \\n               \\n                 5′- GGCACAAGTGAAAGCAGAAGCT -3′ \\n                 (SEQ ID NO: 76) \\n               \\n               \\n                   \\n               \\n               \\n                 Reverse Primer \\n               \\n               \\n                 5′- TTGCTCGTTCTGCCTTTGAA -3′ \\n                 (SEQ ID NO: 77) \\n               \\n               \\n                   \\n               \\n               \\n                 TAQMAN Probe \\n               \\n               \\n                 5′- TGAAGCGCAAAGGTTGGCGGC -3′ \\n                 (SEQ ID NO: 78) \\n               \\n            \\n           \\n         \\n       \\n     \\n     DNA Contamination \\n     To access the level of contaminating genomic DNA in the RNA, the RNA was divided into 2 aliquots and one half was treated with Rnase-free Dnase (Invitrogen). Samples from both the Dnase-treated and non-treated were then subjected to reverse transcription reactions with (RT+) and without (RT−) the presence of reverse transcriptase. TAQMAN assays were carried out with gene-specific primers (see above) and the contribution of genomic DNA to the signal detected was evaluated by comparing the threshold cycles obtained with the RT+/RT− non-Dnase treated RNA to that on the RT+/RT− Dnase treated RNA. The amount of signal contributed by genomic DNA in the Dnased RT− RNA must be less that 10% of that obtained with Dnased RT+RNA. If not the RNA was not used in actual experiments. \\n     Reverse Transcription Reaction and Sequence Detection \\n     100 ng of Dnase-treated total RNA was annealed to 2.5 μM of the respective gene-specific reverse primer in the presence of 5.5 mM magnesium chloride by heating the sample to 72° C. for 2 min and then cooling to 55° C. for 30 min. 1.25 U/l of MuLv reverse transcriptase and 500 μM of each dNTP was added to the reaction and the tube was incubated at 37° C. for 30 min. The sample was then heated to 90° C. for 5 min to denature enzyme. \\n     Quantitative sequence detection was carried out on an ABI PRISM 7700 by adding to the reverse transcribed reaction 2.5 μM forward and reverse primers, 2.0 μM of the TAQMAN probe, 500 μM of each dNTP, buffer and 5U AmpliTaq Gold™. The PCR reaction was then held at 94° C. for 12 min, followed by 40 cycles of 94° C. for 15 sec and 60° C. for 30 sec. \\n     Data Handling \\n     The threshold cycle (Ct) of the lowest expressing tissue (the highest Ct value) was used as the baseline of expression and all other tissues were expressed as the relative abundance to that tissue by calculating the difference in Ct value between the baseline and the other tissues and using it as the exponent in 2 (ΔCt)    \\n     The tissue expression profile of the GBP5 polypeptide is provided in  FIG. 48  and described herein. \\n     One of ordinary skill in the art could readily modify the exemplified studies to test the activity of polynucleotides of the present invention (e.g., gene therapy), agonists, and/or antagonists of polynucleotides or polypeptides of the present invention. \\n     Example 42 \\n     siRNA Studies \\n     Guanine Nucleotide Binding \\n     Genes encoding different GBP family members were cloned into the vector pcDNA3.1mychis (Invitrogen) for expression in mammalian cells. Cos-7 cells were transfected with 5 μg of DNA per group using lipofectamine 2000 (Invitrogen) according to manufacturer&#39;s instructions. Approximately 18 hours after transfection, the cells were harvested using trypsin and lysed in RIPA buffer (10 mM sodium phosphate pH 7.2, 0.25 M sodium chloride, 0.1% SDS, 1% NP40, 1% sodium deoxycholate, 2 mM EDTA, protease inhibitor cocktail). An aliquot of the lysate was reserved for the whole cell lysate sample. The remainder was divided in thirds, and incubated for three hours at 4° C. with agarose coated with either GTP, GDP, or GMP (Sigma Chemical Company). The beads were pelleted, washed with buffer (0.1% NP40, 150 mM NaCl, 50 mM Tris pH7.5), and resuspended in SDS sample buffer. Samples were electrophoresed through 4–20% tris-glycine gels, transferred to nitrocellulose, blocked overnight with 5% non-fat dry milk and 0.3% BSA in Tris-buffered saline, and probed with a mouse monoclonal antibody specific for the myc epitope tag (Invitrogen). \\n     siRNA Studies \\n     To test for knockdown of protein expression using siRNAs, Cos-7 cells were transfected as described herein with expression vectors encoding either GBP-1 or GBP-2 in the presence and absence of 40 nM siRNA duplex specific for either GBP-1 (“1–3”, GCUCGAGAAACUACAAGAUTT, (SEQ ID NO:79), Qiagen, Inc.) or GBP-2 (“582”, UUUGUAUUUCCUCCAACAUTT (SEQ ID NO:80), Sequitur Inc., Natick, Mass.). Approximately 18 hours following transfection, cells were harvested using trypsin, and lysed in RIPA buffer as described previously. Lysates were electrophoresed through 4–20% tris-glycine gels, transferred to nitrocellulose, blocked overnight with 5% non-fat dry milk and 0.3% BSA in Tris-buffered saline, and probed with a mouse monoclonal antibody specific for the myc epitope tag. \\n     In some experiments, human umbilical vein endothelial cells (HUVECs, Clonetics Corp., San Diego, Calif., USA) were transfected using lipofectamine 2000 with 100 nM siRNA duplexes. Cells were cultured overnight, and then stimulated for 6 hours with 10 ng/ml TNFα. Supernatants were analyzed for MCP-1 and IL-8 levels by ELISA (Pharmingen, San Diego, Calif., USA). Cytokine levels were corrected for cell number using the CELL TITER kit (Promega). \\n     RESULTS \\n     One of the hallmarks of the GBP family is the ability to bind guanine nucleotides. To test whether the novel GBP family members were also able to bind guanine nucleotides, the GBP family members were expressed in Cos-7 cells and the lysates batched onto agarose beads coated with either GTP, GDP, or GMP. Bound proteins were eluted from the beads and analyzed with whole cell lysates (WL) in a myc western blot (see  FIG. 49 ). As expected, GBP-1 and GBP-2 bound well to all three beads. GBP-5 and FLJ10961 bound well to GDP and GMP, and weakly to GTP. Family member AK096141 bound weakly to GDP and GMP, and not at all to GTP. Lastly, family member 4843 — 1 did not bind to any of the beads. \\n     To more fully elucidate the function of GBP family members, siRNA experiments were initiated. Co-expression of GBP-1 and GBP-2 expression vectors with siRNAs targeting these genes (indicated as 1–3 and 582 in  FIG. 50A ) resulted in specific knockdown of protein expression ( FIG. 50A ). When transfected into human umbilical vein endothelial cells, the GBP-1 specific siRNA (1–3) inhibited TNFα-induced MCP-1 expression by 70% ( FIG. 50B ). The GBP-2 specific siRNA (582) inhibited MCP-1 expression by approximately 40%. These siRNAs failed to inhibit IL-8 secretion, indicating that they were not toxic. The degree of inhibition observed was comparable to, or greater than, that observed with siRNAs targeting known components of the signal transduction pathway such as NF-κB p65 (40% inhibition) and Stat1 (20% inhibition). These data indicate that members of the GBP family may play roles in TNFα-induced inflammatory responses. \\n     It will be clear that the present invention may be practiced otherwise than as particularly described in the foregoing description and Examples. Numerous modifications and variations of the present invention are possible in light of the above teachings and, therefore, are within the scope of the appended claims. \\n     REFERENCES CITED HEREIN \\n     The entire disclosure of each document cited (including patents, patent applications, journal articles, abstracts, laboratory manuals, books, or other disclosures) in the Background of the present invention, Detailed Description, and Examples is hereby incorporated herein by reference. Further, the hard copy of the Sequence Listing submitted herewith and the corresponding computer readable form are both incorporated herein by reference in their entireties. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE I \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                 ATCC Deposit \\n                   \\n                 Total NT \\n                 5′ NT of \\n                   \\n                   \\n                 Total \\n               \\n               \\n                   \\n                   \\n                 No. Z \\n                 NT SEQ ID. \\n                 Seq of \\n                 Start Codon \\n                 3′ NT of \\n                 AA Seq \\n                 AA of \\n               \\n               \\n                 Gene No. \\n                 cDNA CloneID \\n                 and Date \\n                 No. X \\n                 Clone \\n                 of ORF \\n                 ORF \\n                 ID No. Y \\n                 ORF \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 1 \\n                 HGBPBMY1 \\n                   \\n                 2 \\n                 2454 \\n                 140 \\n                 2053 \\n                 3 \\n                 635 \\n               \\n               \\n                   \\n                 (AK096141) \\n               \\n               \\n                 2 \\n                 HGBPBMY2 \\n                 PTA-6007 \\n                 4 \\n                 2367 \\n                 1 \\n                 2364 \\n                 5 \\n                 788 \\n               \\n               \\n                   \\n                 (4843 301 \\n               \\n               \\n                   \\n                 1; 4843_1) \\n                 05/20/04 \\n               \\n               \\n                 3 \\n                 HGBPBMY3 \\n                   \\n                 6 \\n                 1392 \\n                 1 \\n                 1392 \\n                 7 \\n                 464 \\n               \\n               \\n                   \\n                 (4843 302 \\n               \\n               \\n                   \\n                 1; 4843_2) \\n               \\n               \\n                 4 \\n                 HGBPBMY4 \\n                   \\n                 8 \\n                 2952 \\n                 200 \\n                 1889 \\n                 9 \\n                 563 \\n               \\n               \\n                   \\n                 (FLJ10961) \\n               \\n               \\n                 5 \\n                 MGBPBMY1 \\n                   \\n                 10 \\n                 2484 \\n                 93 \\n                 1989 \\n                 11 \\n                 632 \\n               \\n               \\n                   \\n                 (LOC229900) \\n               \\n               \\n                 6 \\n                 MGBPBMY2 \\n                   \\n                 12 \\n                 1818 \\n                 1 \\n                 1815 \\n                 13 \\n                 605 \\n               \\n               \\n                   \\n                 (LOC229902) \\n               \\n               \\n                 7 \\n                 MGBPBMY3 \\n                   \\n                 14 \\n                 3387 \\n                 80 \\n                 1235 \\n                 15 \\n                 385 \\n               \\n               \\n                   \\n                 (BC031475) \\n               \\n               \\n                 8 \\n                 MGBPBMY4 \\n                   \\n                 16 \\n                 4193 \\n                 321 \\n                 1770 \\n                 17 \\n                 483 \\n               \\n               \\n                   \\n                 (BC007143)', 'id': 'US-7226791-B2', 'ti': 'Polynucleotides encoding novel guanylate binding proteins (GBP&#39;s)'}>,\n",
       " <Hit {'ab': 'A method for treating a hypophosphatemic bone disease, such as X-linked hypophosphatemia (XLH). The method entails administering to a subject a pharmaceutical composition, which contains an antibody that binds to an FGF-23 polypeptide and a pharmaceutically acceptable carrier.', 'clm': '1. A method for treating X-linked hypophosphatemia (XLH), comprising the step of administering to a subject a pharmaceutical composition comprising:\\n (i) an antibody that binds to a polypeptide having the amino acid sequence of SEQ ID NO: 2 or 4; and \\n (ii) a pharmaceutically acceptable carrier.', 'cpc': 'A61K38/00', 'detd': 'This application is a continuation of U.S. patent application Ser. No. 10/344,339, filed Jul. 21, 2003, which is a national phase entry of PCT Application No. PCT/JP01/06944, filed Aug. 10, 2001, which claims priority to Japanese Patent Application Nos. 2000-245144, filed Aug. 11, 2000, 2000-287684, filed Sep. 21, 2000, 2000-391077, filed Dec. 22, 2000, and 2001-121527, filed Apr. 19, 2001, the contents of which are incorporated by reference in their entireties. \\n    \\n    \\n     SEQUENCE LISTING \\n     The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS-Web and is hereby incorporated by reference in its entirety. Said ASCII copy, created on Feb. 22, 2011, is named 81356318.txt and is 72,592 bytes in size. \\n     TECHNICAL FIELD \\n     The present invention relates to a polypeptide that regulates phosphate metabolism, calcium metabolism, calcification and/or vitamin D metabolism, a DNA encoding the polypeptide, and a pharmaceutical composition containing the polypeptide as an active ingredient, and an antibody recognizing the polypeptide, a pharmaceutical composition containing the antibody as an active ingredient, a diagnostic method using the antibody, and a diagnostic composition. \\n     BACKGROUND ART \\n     Inorganic phosphates (hereinafter, may be referred to as “phosphate”) are essential in energy metabolism in vivo and maintenance of cellular functions, and play an important role in tissue calcification in cooperation with calcium. Supply of phosphate to an organism depends mainly on absorption in the intestinal tract, and phosphate excretion depends on urinary excretion in the kidney and fecal excretion in the intestinal tract. In living organisms, phosphate is distributed in body fluid, intracellular fractions and calcified tissues. The level of excretion of inorganic phosphate in an adult is maintained at almost the same level of absorption of inorganic phosphate, suggesting the presence of a regulatory mechanism which maintains homeostasis of the phosphate metabolism. It is known that the metabolism of calcium, which shares similarity with the phosphate metabolism in terms of a distribution and homeostatic control of blood level, is controlled in a co-operative manner in mammals by regulatory factors, such as, at least parathyroid hormone, calcitonin and 1α,25-dihydroxyvitamin D3. \\n     In the regulation of phosphate metabolism it is known that parathyroid hormone promotes phosphate excretion, and that 1α,25-dihydroxyvitamin D3 promotes phosphate absorption in the intestinal tract. This clearly suggests close association between phosphate metabolism and calcium metabolism. However, a substance primarily controlling phosphate has not yet been elucidated. \\n     Now, examples of a disease which is associated with the loss of the homeostasis of phosphate metabolism and lower inorganic phosphate levels in the blood include primary hyperparathyroidism, hereditary hypophosphatemic rickets, and tumor-induced osteomalacia. \\n     Primary hyperparathyroidism is a disease characterized by an overproduction of parathyroid hormone in the parathyroid glands, and is known to develop hypophosphatemia with increased phosphate excretion because overproduced parathyroid hormone suppresses reabsorption of inorganic phosphate in the kidney. \\n     Further, known examples of hypophosphatemia resulting from hereditary diseases include type I vitamin D-dependent rickets, type II vitamin D-dependent rickets and vitamin D-resistant rickets. Type I vitamin D-dependent rickets is a disease caused by hereditary dysfunction of the synthase to produce active vitamin D metabolites, and type II vitamin D-dependent rickets is a disease caused by hereditary dysfunction of vitamin D receptor. Both diseases develop hypophosphatemia together with hypocalcemia due to attenuated action of vitamin D3 metabolites. In contrast, for vitamin D-resistant rickets, at least 2 types of clinical conditions, X-linked chromosomal and autosomal hypophosphatemic rickets resulting from different causes are known to exist. \\n     Both of the above-mentioned clinical conditions of vitamin D-resistant rickets lead to hypophosphatemia characterized by renal phosphate. Recently, it has been shown in patients with X-linked hypophosphatemic rickets (hereinafter, also referred to as “XLH”) that the disease is induced by mutations in the gene encoding an endopeptidase-like protein, named PHEX, on X chromosome. However, a mechanism how dysfunction of PHEX protein induces hypophosphatemia has not been elucidated. Interestingly, gene analysis of a naturally occurring mutant mouse (Hyp) which developed hypophosphatemia has revealed the partial deletion of the gene encoding PHEX in this mouse. Experiments using these mice have revealed that PHEX deficient mice have normal renal function, and a humoral factor, which is different from parathyroid hormone, but induces hypophosphatemia, is present in the body fluid of Hyp mice. Concerning autosomal dominant hypophosphatemic rickets/osteomalacia (hereinafter also referred to as ADHR), a gene responsible for this disease has been pursued, and the presence of such a gene in 12p13 region has been indicated by linkage analysis. However, the region that has been narrowed down so far is still wide and contains many genes, so that no candidate gene has been specified yet. \\n     Tumor-induced osteomalacia is a disease which develops hypophosphatemia with increased renal phosphate in association with tumorigenesis, and is characterized in that the hypophosphatemia is eliminated by irradiation to tumor or removal of tumor. In this disease, it is thought that tumor produces a factor which induces hypophosphatemia due to suppressed reabsorption of phosphate in the kidneys. \\n     It has not been confirmed whether a putative causative molecule for vitamin D-resistant rickets is identical to that for tumor-induced osteomalacia. However, the two factors are identical in that they clearly are unknown phosphate metabolism factors which promote urinary phosphate excretion. The putative phosphate metabolism regulatory factor is often referred to as, the name Phosphatonin. The relationship of this unknown phosphate metabolism regulatory factor and vitamin D-resistant rickets or tumor-induced osteomalacia has been summarized as general remarks (Neison, A. E., Clinical Endocrinology, 47:635-642, 1997; Drezner, M. K., Kidney Int., 57:9-18, 2000). \\n     Another characteristic of vitamin D-resistant rickets or tumor-induced osteomalacia is impairment of bone calcification. This impaired bone calcification could be thought to be secondarily developed by hypophosphatemia. However, since abnormal bone calcification in experiments using Hyp mice, the model mice for vitamin D-resistant rickets is shown to develop independently from phosphate levels (Ecarot, B., J. Bone Miner. Res., 7:215-220, 1992; Xiao, Z. S., Am. J. Physiol., E700-E708, 1998), it is conceivable that the above unknown regulatory factor for phosphate metabolism can directly regulate calcification in bone tissue. \\n     As described above, research data have strongly been suggesting the presence of an unknown factor which regulates phosphate metabolism, but there has been no case that can elucidate at a molecular level, an entity which exhibits the putative activity. While WO99/60017 discloses a novel polypeptide sequence as a novel polypeptide hormone, Phosphatonin, however, it does not disclose the characteristic activity of phosphatonin which concerns induction of hypophosphatemia. Thus, it is conceivable that an unidentified intrinsic factor regulating phosphate metabolism may exist in organisms. \\n     Vitamin D2 and vitamin D3 ingested from foods, or vitamin D3 synthesized in the skin is hydrolyzed by vitamin D-25-hydroxylase existing mainly in the liver to produce 25-hydroxyvitamin D. Then, 25-hydroxyvitamin D is hydrolyzed by 25-hydroxyvitamin D-1α-hydroxylase existing in renal epithelial cells of proximal tubules in the kidney to produce 1α25-dihydroxyvitamin D. This 1α,25-dihydroxyvitamin D is a mineral regulatory hormone having physiological activities that increase serum calcium and phosphate levels, and is known to be responsible for inhibiting the secretion of parathyroid hormone and to be involved in the promotion of bone resorption. 1α,25-dihydroxyvitamin D is then converted into metabolites in vivo which has not the above physiological activities by 24-hydroxylase existing mainly in the kidney or small intestine. In this regard, 24-hydroxylase is thought to be an enzyme which is responsible for the inactivation of 1α,25-dihydroxyvitamin D. On the other hand, 24-hydroxylase is known to also act on 25-hydroxy vitamin D and convert it into 24,25-dihydroxyvitamin D. The 24,25-dihydroxyvitamin D has been reported to have physiological effects that increase bone mass or promote differentiation of cartilage, suggesting that this enzyme has an aspect for generating biological active vitamin D metabolites. \\n     Known factors that regulate the expression level of 1α-hydroxylase, which has an important role in the activation of vitamin D, include parathyroid hormone (PTH), calcitonin, 1α,25-dihydroxyvitamin D and the like. PTH whose secretion is promoted by decreases in blood calcium levels acts on PTH receptors existing in epithelial cells of the renal proximal tubules to promote transcription of 1α-hydroxylase gene through an elevated intracellular cAMP level, so as to increase blood 1α,25-dihydroxyvitamin D concentration. 1α,25-dihydroxyvitamin D promotes absorption of calcium from the intestinal tract and calcium reabsorption in the kidney, thereby increasing the blood calcium level. Further, it has been reported that the binding of 1α,25-dihydroxyvitamin D to vitamin D receptor (VDR) acts on a promoter region of 1α-hydroxylase gene or PTH gene to suppress the transcription of such genes. Specifically, 1α,25-dihydroxyvitamin D has a feedback control mechanism for its activation factor, PTH and 1α-hydroxylase. This mechanism plays an important role in maintaining homeostasis of calcium metabolism. \\n     Recently, it has been reported that a decrease in serum phosphate level enhances the expression of 1α-hydroxylase gene. In phosphate metabolism, the presence of a mechanism is also assumed that enhancement in the expression of 1α-hydroxylase gene association with decreased serum phosphate level elevates serum 1α,25-dihydroxyvitamin D level and, consequently, corrects the serum phosphate level by promoting absorption of phosphate from the small intestine. \\n     Examples of a factor responsible for regulating the expression of 24-hydroxylase gene include 1α,25-dihydroxyvitamin D and PTH. It has been shown that 1α,25-dihydroxyvitamin D interact with the vitamin D receptor (VDR) and the complex binds to a vitamin D receptor response sequence existing in the promoter region of 24-hydroxylase gene so as to promote transcription. 1α,25-dihydroxyvitamin D is thought to activate 24-hydroxylase, and then to induce a decrease in the 1α,25-dihydroxyvitamin D level due to the activated catabolic pathway. It is known that the expression of 24-hydroxylase gene is suppressed by PTH, but its detailed molecular mechanism is unknown. \\n     DISCLOSURE OF THE INVENTION \\n     The purpose of the present invention is to provide a novel tumor-derived factor which is capable of inducing decreases in blood phosphate levels. \\n     It is conceivable that a tumor identified in tumor-induced osteomalacia secretes a soluble factor having physiological activity, so that the blood phosphate levels decrease. The tumor-derived factor causes the homeostasis of phosphate metabolism to fail. Therefore, the factor may be characterized by any one of (1) the factor which is not originally produced in vivo is produced tumor-specifically, (2) the factor is overproduced in tumor, though it is also produced in normal tissue, and (3) the factor is produced in tumor without being physiologically controlled. \\n     Based on an assumption that a tumor-derived hypophosphatemia-inducing factor is characteristically produced in a tumor-induced osteomalacia-derived tumor as described above, we have anticipated enhanced transcription of a gene encoding the hypophosphatemia-inducing factor or enhanced stability of mRNA of the factor in a tumor. Hence, after extraction of RNA from a part of the tumor tissues that were excised from a patient with tumor-induced osteomalacia for therapeutic purposes, we prepared cDNA library using phage vectors and plasmid vectors, and then screened for gene fragments that were specifically expressed in the tumor. Methods performed for screening were a method which selects cDNA fragments determined to be specifically expressed in the tumor, and a method which selects cDNA fragments in a tumor-derived cDNA library which do not cross-react with cDNA probes derived from a cell line of epithelial cells of the renal proximal tubules. We further narrowed down the selected cDNA fragments by confirming for novelty in sequences and characteristic expression in the tumor, thereby obtaining a plurality of cDNA fragments expected to encode the hypophosphatemia-inducing factor. From the sequence information, we attempted to clone cDNAs containing ORF to which each fragment belongs and successfully obtained DNAs encoding novel polypeptides. We further thoroughly studied to find biological activities of the novel polypeptides, so that we have completed the present invention by elucidating that the novel polypeptide has activities to suppress phosphate transport, to induce hypophosphatemia and to suppress calcification of bone tissue in animals. \\n     Specifically, the present invention is as follows.\\n     (1) A DNA, which encodes the following polypeptide (a), (b), (c) or (d):\\n       (a) a polypeptide consisting of an amino acid sequence represented by SEQ ID NO: 2 or 4,   (b) a polypeptide consisting of an amino acid sequence derived from the amino acid sequence represented by SEQ ID NO: 2 or 4 by deletion, substitution or addition of one or several amino acids, and having hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or in vivo vitamin D metabolism-regulating activity,   (c) a polypeptide consisting of a partial sequence of the amino acid sequence represented by SEQ ID NO: 2, wherein the above partial sequence contains an amino acid sequence at least ranging from the 34 th  to 201 st  amino acids in the above amino acid sequence, or   (d) a polypeptide consisting of a partial sequence of the amino acid sequence represented by SEQ ID NO: 2, wherein the partial sequence:\\n           (i) contains an amino acid sequence at least ranging from the 34 th  to 201 st  amino acids in the above amino acid sequence,   (ii) consists of an amino acid sequence derived from the partial sequence by deletion, substitution or addition of one or several amino acids, and   (iii) has hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or in vivo vitamin D metabolism-regulating activity.   \\n           \\n       (2) A DNA, which contains the following DNA (e) or (f):\\n       (e) a DNA consisting of a nucleotide sequence ranging from the 133rd to 885th nucleotides in the nucleotide sequence represented by SEQ ID NO: 1 or a nucleotide sequence ranging from the 1 st  to 681 st  nucleotides in the nucleotide sequence represented by SEQ ID NO: 3, or   (f) a DNA hybridizing under stringent conditions to a probe prepared from a DNA consisting of the whole or a part of the nucleotide sequence represented by SEQ ID NO: 1 or 3, and encoding a polypeptide having hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or in vivo vitamin D metabolism-regulating activity.   \\n       \\n\\n     Here, the term “stringent conditions” satisfies conditions of a sodium concentration of 750 mM or more, preferably 900 mM or more, a temperature of 40° C. or more, preferably, 42° C. Specifically, stringent conditions consist of 6×SSC, 5× Denhardt, 0.5% SDS, 50% Formamide and 42° C.\\n     (3) A recombinant vector, which contains the above DNA.   (4) A transformant, which contains the above recombinant vector.   (5) A polypeptide, which is the following polypeptide (a), (b), (c) or (d):\\n       (a) a polypeptide consisting of an amino acid sequence represented by SEQ ID NO: 2 or 4,   (b) a polypeptide consisting of an amino acid sequence derived from the amino acid sequence represented by SEQ ID NO: 2 or 4 by deletion, substitution or addition of one or several amino acids, and having hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or in vivo vitamin D metabolism-regulating activity,   (c) a polypeptide consisting of a partial sequence of the amino acid sequence represented by SEQ ID NO: 2, wherein the above partial sequence contains at least an amino acid sequence ranging from the 34 th  to 201 st  amino acids in the above amino acid sequence, or   (d) a polypeptide consisting of a partial sequence of the amino acid sequence represented by SEQ ID NO: 2 wherein the partial sequence:\\n           (i) contains an amino acid sequence ranging from at least the 34 th  to 201 st  amino acids in the above amino acid sequence,   (ii) consists of an amino acid sequence derived from the partial sequence by deletion, substitution, or addition of one or several amino acids, and   (iii) has hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or in vivo vitamin D metabolism-regulating activity.   \\n           \\n       \\n\\n     The above polypeptide also includes a polypeptide modified by at least one substance selected from the group consisting of polyethylene glycol, dextran, poly (N-vinyl-pyrrolidone), polypropylene glycol homopolymer, copolymer of polypropylene oxide and polyethylene oxide, polyoxyethylated polyol and polyvinyl alcohol.\\n     (6) A pharmaceutical composition which contains the above polypeptide as an active ingredient.   \\n\\n     The above pharmaceutical composition can be used to enable in vivo regulation of calcium metabolism, phosphate metabolism, calcification or vitamin D metabolism. Further, the above pharmaceutical composition is effective against at least one condition selected from the group consisting of hyperphosphatemia, hyperparathyroidism, renal osteodystrophy, ectopic calcification, osteoporosis and hypervitaminosis D.\\n     (7) An antibody, which reacts with the above polypeptide or partial fragments thereof.   \\n\\n     The above antibody can be obtained by a method comprising the steps of immunizing an animal with the polypeptide of the present invention or partial fragments thereof, as an antigen.\\n     (8) A pharmaceutical composition, which contains the above antibody as an active ingredient.   \\n\\n     The above pharmaceutical composition can regulate in vivo calcium metabolism, phosphate metabolism, calcification or vitamin D metabolism, or be effective against bone diseases. Here, the bone disease is at least one disease selected from the group consisting of osteoporosis, vitamin D-resistant rickets, renal osteodystrophy, dialysis-associated bone diseases, osteopathy with hypocalcification, Paget&#39;s disease and tumor-induced osteomalacia.\\n     (9) A diagnostic agent, which contains the above antibody and is for a disease which develops at least one abnormality of abnormal calcium metabolism, abnormal phosphate metabolism, abnormal calcification and abnormal vitamin D metabolism (for example, a disease selected from the group consisting of renal failure, renal phosphate leak, renal tubular acidosis and Fanconi&#39;s syndrome).   (10) A diagnostic agent for a bone disease, which contains the above antibody, wherein the bone disease is at least a disease selected from the group consisting of osteoporosis, vitamin D-resistant rickets, renal osteodystrophy, dialysis-associated bone diseases, osteopathy with hypocalcification, Paget&#39;s disease and tumor-induced osteomalacia.   (11) A diagnostic agent, which contains a DNA having a nucleotide sequence represented by SEQ ID NO: 11 or partial fragments thereof, and is for a disease which develops at least one abnormality of abnormal calcium metabolism, abnormal phosphate metabolism, abnormal calcification and abnormal vitamin D metabolism.   \\n\\n     An example of the partial sequence has a sequence ranging from the 498 th  to 12966 th  nucleotides of the nucleotide sequence represented by SEQ ID NO: 11. An example of the disease is autosomal dominant hypophosphatemic rickets/osteomalacia. \\n     The present invention is explained in detail as follows. This specification includes part or all of the contents disclosed in the specification and/or drawings of Japanese Patent Application Nos. 2000-245144, 2000-287684, 2000-391077 and 2001-121527, which are priority documents of the present application. \\n     The terms used in the present specification are defined as follows. \\n     The term “activity to decrease blood 1,25-dihydroxyvitamin D3 levels” indicates an activity which acts to decrease blood levels of 1,25-dihydroxyvitamin D3. \\n     The term “hypophosphatemia-inducing activity” indicates an activity which acts to decrease blood phosphate levels. Blood phosphate level is defined by the balance between (i) absorption from the intestinal tract and excretion into urine and feces, and (ii) in vivo distribution of phosphate to cells or calcified tissues as represented by bone tissues. Therefore, the term “hypophosphatemia-inducing activity” used in the present specification means an activity to lower blood phosphate levels in a healthy living organism, and does not necessarily mean an activity to cause pathologic hypophosphatemia. The hypophosphatemia-inducing activity may be equivalent, on a tissue level, to phosphate absorption-suppressing activity in the intestinal tract, phosphate excretion-promoting activity in the kidney or the intestinal tract, or activity which promotes transfer of phosphate into cells. \\n     Further, the term “phosphate transport-suppressing activity” in the present invention means an activity which acts on a target cell so as to suppress activity of a phosphate transport carrier existing on the cell membrane. Possible target cells are mainly epithelial cells of the renal tubules, epithelial cells of the intestines or osteoblasts. \\n     Furthermore, the term “calcification-suppressing activity” in the present invention means an activity which suppresses the process to generate or accumulate crystal substances containing calcium and phosphate as compositions in bone tissues and soft tissues. \\n     Furthermore, the term “in vivo vitamin D metabolism-regulating activity” indicates a potency to regulate changes in the absolute amounts or in the abundance ratio of vitamin D existing in vivo or of the metabolites synthesized in vivo therefrom. In vivo regulation of the vitamin D and of the metabolite thereof is ruled by mainly (i) absorption or excretion in the intestinal tract and (ii) reabsorption or excretion in the kidney, followed by (iii) in vivo synthesis of vitamin D, and (iv) metabolic conversion mainly led by hydroxylation reaction. Known, main metabolites resulting from the metabolic conversion (iv) are as follows: 25-hydroxyvitamin D which is produced by hydroxylation at position 25 of vitamin D by vitamin D-25-hydroxylase; 1α,25-dihydroxyvitamin D which is produced by hydroxylation at position 1α of hydroxyvitamin D by 25-hydroxyvitamin D-1α-hydroxylase; or 24,25-dihydroxyvitamin D or 1α 24,25-trihydroxyvitamin D which is produced by introduction of a hydroxyl group at position 24 of the metabolite by 24-hydroxylase. Vitamin D metabolism-regulating activity can be represented as an activity to regulate an enzymatic activity, gene expression or changes in expressed protein levels of enzymes involved in the generation of such vitamin D metabolites. \\n     1. DNA Encoding Polypeptide which Regulates Phosphate Metabolism, Calcium Metabolism, Calcification and Vitamin D Metabolism \\n     (1) DNA Cloning \\n     A DNA represented by SEQ ID NO: 1, which is one of DNAs of the present invention, is obtained by screening a cDNA library prepared using a part of a tumor excised from a patient suspected of having tumor-induced osteomalacia. \\n     Tumor-induced osteomalacia is a disease which develops hypophosphatemia and osteomalacia due to insufficient calcification of bone tissues in association with the presence of tumors, and is characterized in that the removal of the tumor causes these symptoms to disappear. There have been reports that tumor extracts promote urinary phosphate excretion in rats (Popvtzer, M. M. et al., Clinical Research 29: 418A, 1981), and that hypophosphatemia was induced in an experiment of transplanting excised tumors into mice (Miyauchi, A. et al., J. Clin. Endocrinol. Metab. 67:46-53, 1988). Thus, it has been considered that tumors produce and secrete a systemic unknown factor. \\n     We used a case relating to the tumor described in Fukumoto, S. et al., Bone 25: 375-377, 1999. In this case, a significant recovery from hypophosphatemia was achieved by operative excision of the tumor. Further, the tumor size in this case was as small as about 1 cm in diameter. Based on an inference that such a small tissue produces and secretes an active substance which induces hypophosphatemia and systemic osteomalacia, we have anticipated that the tumor-derived cDNA library that we have prepared contains, at a higher frequency compared to another tissue-derived cDNA library, at least a partial fragment of the gene encoding the active substance involved in the induction of such clinical conditions. Accordingly, to identify a sequence of the fragment of the gene encoding the tumor-derived active substance, cDNA fragments that are specifically abundant in cDNA library of the tumor were extracted by a differential screening method. \\n     Next, the nucleotide sequences of the obtained cDNA fragments were identified, and compared to each other. Then based on overlap of the nucleotide sequences, contigs were prepared to classify each sequence thought to be derived from the same gene. Homology searches were performed for the thus obtained nucleotide sequences with the nucleotide sequences registered at Genbank which is the database provided by National Center for Biotechnology Information (USA) (hereinafter, may also referred to as “NCBI”). In this way, the nucleotide sequence that is specifically abundant in the tumor cDNA library, that is, a sequence ranging from nucleotide Nos. 1522 to 2770 of the nucleotide sequence represented by SEQ ID NO: 1 was obtained. This sequence was identical to a part of the human sequence, 12p13 BAC RPCI11-388F6 registered at Genbank under Accession No. AC008012. This registered sequence is thought to represent a partial sequence of 12p13 region of a human chromosome sequence. While the locations of estimated genes within the registered sequence were shown with the nucleotide sequence information, the sequence ranging from nucleotide Nos. 1522 to 2770 of the nucleotide sequence represented by SEQ ID NO: 1, and the nucleotide sequence represented by SEQ ID NO: 1 of the present invention were not included in the any of specified regions of estimated genes. \\n     Probes and PCR primers were then designed based on the nucleotide sequence ranging from nucleotide Nos. 1522 to 2770 of the nucleotide sequence represented by SEQ ID NO: 1, and then a continuous nucleotide sequence contained in the tumor cDNA library was isolated and identified, thereby obtaining the nucleotide sequence represented by SEQ ID NO: 1 of the present invention. The nucleotide sequence of SEQ ID NO: 1 had an open reading frame (hereinafter, may also be referred to as “ORF”) encoding a polypeptide consisting of an amino acid sequence represented by SEQ ID NO: 2 of the present invention that is inferred to have a secretion signal. We have considered that this is a polypeptide having a novel sequence, because the amino acid sequence of the polypeptide has not been registered at Genbank amino acid sequence database. After a search using the nucleotide sequence represented by SEQ ID NO: 1 and the amino acid sequence represented by SEQ ID NO: 2, we have identified a nucleotide sequence represented by SEQ ID NO: 9 and an amino acid sequence represented by SEQ ID NO: 10, which are thought to be murine orthologs of the molecule. As described later, a recombinant protein prepared according to a human amino acid sequence shows activity in mice. Thus, the amino acid sequence represented by SEQ ID NO: 2 was compared with the amino acid sequence represented by SEQ ID NO: 10 or murine full-length sequence (Biochem. Biophys. Res. Commun. 2000, 277(2), 494-498), so that the present invention makes it possible to easily assess whether proteins, wherein an amino acid, other than amino acids conserved between the two has been substituted, has biological activity equivalent to, or similar to that of the polypeptide of the present invention. \\n     (2) Determination of Nucleotide Sequence \\n     The nucleotide sequence of DNA obtained as described in (1) above is determined. The nucleotide sequence can be determined by known techniques, such as the Maxam-Gilbert&#39;s chemical modification method or a dideoxynucleotide chain termination method using M13 phage. Normally, sequencing is performed using an automatic sequencer (for example, 373A DNA sequencer manufactured by PERKIN-ELMER). \\n     The nucleotide sequence of the DNA of the present invention is exemplified in SEQ ID NO: 1, and the amino acid sequence of the polypeptide of the present invention is exemplified in SEQ ID NO: 2. As long as the polypeptide consisting of the amino acid sequence has hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or vitamin D metabolite-regulating activity, the amino acid sequence may contain a mutation, such as deletion, substitution or addition of one or several amino acids. \\n     For example, 1 or several, preferably 1 to 10, more preferably 1 to 5 amino acids may be deleted from the amino acid sequence represented by SEQ ID NO: 2; 1 or several, preferably 1 to 10, more preferably 1 to 5 amino acids may be added to the amino acid sequence represented by SEQ ID NO: 2; or 1 or several, preferably 1 to 10, more preferably 1 to 5 amino acids may be substituted with (an)other amino acids in the amino acid sequence represented by SEQ ID NO: 2. \\n     Further, as a method of substitution, conservative substitution may be performed within a family which retains the characteristics of the amino acids to some extent. Examples of families generally classified according to the characteristics of amino acid side chains are as follows.\\n     (i) Acidic amino acid family: aspartic acid, glutamic acid   (ii) Basic amino acid family: lysine, arginine, histidine   (iii) Nonpolar amino acid family: alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan   (iv) Non-charged polar amino acid family: glycin, asparagine, glutamine, cysteine, serine, threonine, tyrosine   (v) Aliphatic hydroxyamino acid family: serine, threonine   (vi) Amide-containing amino acid family: asparagine, glutamine   (vii) Aliphatic amino acid: alanine, valine, leucine, isoleucine   (viii) Aromatic amino acid family: phenylalanine, tryptophan, tyrosine   (ix) Hydrophobic amino acid family: leucine, isoleucine, valine   (x) Small amino acid family: alanine, serine, threonine, methionine, glycin   \\n\\n     Examples of substitution are sequences which are derived from the amino acid sequence represented by SEQ ID NO: 2 by substitution of the 176 th  Arg and/or 179 th  Arg with (an)other amino acids, preferably, Ala, Gln or Trp, so that cleavage is inhibited or suppressed therein. Further, the polypeptide of the present invention also encompasses a polypeptide consisting of an amino acid sequence derived from the amino acid sequence represented by SEQ ID NO: 2 by deletion of 10 or more amino acids on the N-terminal side, C-terminal side or both sides (terminal-deleted type). Examples of such terminus-deleted forms include a sequence derived from the amino acid sequence represented by SEQ ID NO: 2 by deletion of 20, 40, 45 or 50 amino acids on the C-terminal side, and/or 24 or 33 amino acids on the N-terminal side. Embodiments of the terminal-deleted types are as shown below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                 Position in the amino acid \\n               \\n               \\n                 Number of amino \\n                 Number of amino \\n                 sequence represented by \\n               \\n               \\n                 acids deleted on the \\n                 acids deleted on the \\n                 SEQ ID NO: 2 (Nucleotide \\n               \\n               \\n                 N-terminal side \\n                 C-terminal side \\n                 No. in SEQ ID NO: 1) \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 33 on the N- \\n                 No deletion \\n                 34-251(232-885) \\n               \\n               \\n                 terminal side \\n               \\n               \\n                 33 on the N- \\n                 20 on the C-terminal \\n                 34-231(232-825) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 33 on the N- \\n                 40 on the C-terminal \\n                 34-211(232-765) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 33 on the N- \\n                 45 on the C-terminal \\n                 34-206(232-750) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 33 on the N- \\n                 50 on the C-terminal \\n                 34-201(232-735) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 24 on the N- \\n                 No deletion \\n                 25-251(205-885) \\n               \\n               \\n                 terminal side \\n                   \\n                 (corresponding to the 1 st   \\n               \\n               \\n                   \\n                   \\n                 to the 681 st  in SEQ ID \\n               \\n               \\n                   \\n                   \\n                 NO: 3, and SEQ ID NO: 4) \\n               \\n               \\n                 24 on the N- \\n                 20 on the C-terminal \\n                 25-231(205-825) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 24 on the N- \\n                 40 on the C-terminal \\n                 25-211(205-765) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 24 on the N- \\n                 45 on the C-terminal \\n                 25-206(205-750) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 24 on the N- \\n                 50 on the C-terminal \\n                 25-201(205-735) \\n               \\n               \\n                 terminal side \\n                 side \\n               \\n               \\n                 No deletion \\n                 20 on the C-terminal \\n                 \\u20021-231(133-825) \\n               \\n               \\n                   \\n                 side \\n               \\n               \\n                 No deletion \\n                 40 on the C-terminal \\n                 \\u20021-211(133-765) \\n               \\n               \\n                   \\n                 side \\n               \\n               \\n                 No deletion \\n                 45 on the C-terminal \\n                 \\u20021-206(133-750) \\n               \\n               \\n                   \\n                 side \\n               \\n               \\n                 No deletion \\n                 50 on the C-terminal \\n                 \\u20021-201(133-735) \\n               \\n               \\n                   \\n                 side \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     In addition to partial fragments (terminal-deleted type partial fragments) of the amino acid sequence represented by SEQ ID NO: 2 above, the polypeptide of the present invention encompasses mutated fragments that are derived from these terminal-deleted type polypeptides by deletion, substitution or addition of one or several amino acids. Figures in parentheses following the position numbers of amino acids shown in SEQ ID NO: 2 in the above list indicate the position numbers of nucleotides in the nucleotide sequence represented by SEQ ID NO: 1. Hence, the present invention also encompasses DNAs consisting of the nucleotide sequences shown by these positions, or DNAs hybridizing under stringent conditions to these DNAs. \\n     In the present invention, to introduce a mutation into at least a part of the amino acid sequence of the polypeptide of the present invention, a technique which introduces a mutation into the nucleotide sequence of a DNA encoding the amino acid is employed. \\n     Mutations can be introduced into DNA by a known technique, such as the Kunkel method or the Gapped duplex method, or a method according thereto. For example, a mutation is introduced based on the site-directed mutagenesis method using a mutant oligonucleotide as a primer. Further, a mutation can also be introduced using a kit for introducing mutations, such as Mutan-K (TAKARA), Mutan-G (TAKARA), LA PCR in vitro Mutagenesis series kit (TAKARA) or the like. \\n     Furthermore, the DNA of the present invention also encompasses a DNA hybridizing under stringent conditions to a probe prepared from the above DNA of the present invention (SEQ ID NO: 1, 3, 5, 7 and 9) and encoding a polypeptide having hypophosphatemia-inducing activity, phosphate transport-suppressing activity, calcification-suppressing activity or vitamin D metabolism-regulating activity. The probe used herein has a sequence which is complementary to the entire sequence of or a sequence (partial sequence) of continuous 17 nucleotides or more of the sequence represented by SEQ ID NO: 1, 3, 5, 7 or 9. \\n     Here, the term “stringent conditions” satisfies conditions of sodium concentration of 750 mM or more, preferably, 900 mM or more, and temperature of 40° C. or more, preferably 42° C. Specifically, the stringent conditions used herein indicate the conditions consisting of 6×SSC, 5× Denhardt, 0.5% SDS, 50% Formamide and 42° C. In addition, 6×SSC means 900 mM NaCl and 90 mM sodium citrate. Denhardt&#39;s solution (Denhardt) contains BSA (bovine serum albumin), polyvinylpyrrolidone and Ficoll 400. 50× Denhardt consists of a composition of 1% BSA, 1% polyvinylpyrrolidone and 1% Ficoll 400 (5× Denhardt means a one-tenth concentration of 50× Denhardt). \\n     Once the nucleotide sequence of the DNA of the present invention is determined, the DNA of the present invention can be obtained by chemical synthesis or PCR using primers synthesized from the determined nucleotide sequence. \\n     2. Recombinant Vector Containing the DNA of the Present Invention and Preparation of Transformant \\n     (1) Preparation of Recombinant Vector \\n     The recombinant vector of the present invention can be obtained by ligating (inserting) the DNA of the present invention into an appropriate vector. A vector for inserting the DNA of the present invention is not specifically limited, as long as it can be replicated in a host. Examples of such a vector include plasmid DNA and phage DNA. \\n     Examples of plasmid DNAs include plasmids derived from  Escherichia coli  (for example, pBR322, pBR325, pUC118 and pUC119), plasmids derived from  Bacillus subtilis  (for example, pUB110 and pTP5), plasmids derived from yeast (for example, YEp13, YEp24 and YCp50). An example of a phage DNA is λ phage. Further, animal virus vectors such as a retrovirus, adenovirus or vaccinia virus, or insect virus vectors such as baculovirus can be used. Furthermore, a fusion plasmid to which GST, His-tag and the like are ligated can be used. \\n     In order to insert the DNA of the present invention into a vector, a method can be employed, which comprises cleaving the purified DNA using an appropriate restriction enzyme at first, and ligating to the obtained cleaved DNA to the vector by inserting the cleaved DNA into a restriction enzyme site or multi-cloning site of an appropriate vector DNA. \\n     The DNA of the present invention is required to be incorporated into a vector so that the DNA can exert its function. To the vector of the present invention, in addition to a promoter and the DNA of the present invention, cis element such as an enhancer, splicing signal, poly A addition signal, a selection marker and ribosome binding sequence (SD sequence) may be ligated, if necessary. In addition, examples of a selection marker include a dihydrofolate reductase gene, ampicillin resistance gene and neomycin-resistance gene. \\n     (2) Preparation of Transformant \\n     The transformant of the present invention can be obtained by introducing the recombinant vector of the present invention into a host, such that the target gene can be expressed. A host to be used herein is not specifically limited, as long as it can express the DNA of the present invention. Examples of such a host include bacteria of: the genus  Escherichia , such as  Escherichia coli ; the genus  Bacillus , such as  Bacillus subtilis ; the genus  Pseudomonas , such as  Pseudomonas putida ; or yeast such as  Saccharomyces cerevisiae  or  Schizosaccharomyces pombe . Further, animal cells, such as COS cells, CHO cells or HEK293 cells, and insect cells, such as Sf9 or Sf21 can also be used. \\n     When bacteria, such as  Escherichia coli  is used as a host, it is preferable that the recombinant vector of the present invention can autonomously replicate in the bacteria, and comprises a promoter, ribosome binding sequence, the DNA of the present invention and transcription termination sequence. In addition, a gene regulating a promoter may also be contained. Examples of  Escherichia coli  include JM109 and HB 101, and an example of  Bacillus subtilis  is  Bacillus subtilis . Any promoter may be used, as long as it can be expressed in a host, such as  Escherichia coli . For example, promoters derived from  Escherichia coli , such as a trp promoter, lac promoter, PL promoter or PR promoter or a phage-derived T7 promoter or the like may be used. An artificially designed and modified promoter, such as a tac promoter may be used. A method to be employed herein for introducing a recombinant vector into bacteria is not specifically limited, as long as it is a method for introducing DNA into bacteria. Examples of such a method include a method which uses calcium ion, and electroporation method. \\n     When yeast is used as a host, for example,  Saccharomyces cerevisiae, Schizosaccharomyces pombe  and  Pichia pastoris  are used. A promoter to be used in this case is not specifically limited, as long as it can be expressed in yeast. Examples of such a promoter include a gall promoter, gal10 promoter, heat-shock protein promoter, MFα 1 promoter, PHO5 promoter, PGK promoter, GAP promoter, ADH promoter and AOX1 promoter. A method for introducing a recombinant vector into yeast is not specifically limited, as long as it is a method for introducing DNA into yeast. Examples of such a method include an electroporation method, spheroplast method and lithium acetate method. \\n     When an animal cell is used as a host, monkey cells COS-7, Vero, Chinese hamster ovary cells (CHO cells), mouse L cells, rat GH3 cells, human FL, HEK293, HeLa, Jurkat cells or the like are used. As a promoter, SRα promoter, SV 40 promoter, LTR promoter, β-actin promoter or the like is used. In addition, an early gene promoter of human cytomegalovirus or the like may also be used. Examples of a method for introducing a recombinant vector into animal cells include an electroporation method, calcium phosphate method and lipofection method. \\n     When an insect cell is used as a host, Sf9 cells, Sf21 cells or the like are used. As a method for introducing a recombinant vector to an insect cell, a calcium phosphate method, lipofection method, electroporation method or the like is used. \\n     3. Polypeptide Having Hypophosphatemia-inducing Activity \\n     Several attempts have been made to isolate and identify a tumor-derived factor having hypophosphatemia-inducing activity in tumor-induced osteomalacia. Thus, the polypeptide of the present invention has been shown to have characteristics of being a novel secretion factor which is produced by tumor-induced osteomalacia tumors. The predicted biological activities of the hypophosphatemia-inducing factor have been reported as follows. \\n     Effect on the promotion of phosphate excretion into urine: \\n     Aschinberg, L. C. et al., Journal of Pediatrics 91:56-60, 1977, Lau, K. et al., Clinical Research 27:421A, 1979, Miyauchi, A. et al., J. Clin. Endocrinol. Metab. 67:46-53, 1988 \\n     Suppression of phosphate transport activity of epithelial cells of the renal tubules: \\n     Cai, Q. et al., N. Engl. J. Med. 330: 1645-1649, 1994, Wilkins, G. E. et al., J. Clin. Endocrinol. Metab. 80:1628-1634, 1995, Rowe, P. S. N. et al., Bone 18:159-169, 1996 \\n     Suppression of 25-hydroxyvitamin D-1α-hydroxylase activity: \\n     Miyauchi, A. et al., J. Clin. Endocrinol. Metab. 67:46-53, 1988 \\n     In particular, it has been proposed that an unknown molecule directly having activity to suppress reabsorption of phosphate in the kidney has been referred to as “Phosphatonin” (Econs, M. J. &amp; Drezner, M. K., N. Engl. J. Med 330: 1679-1681, 1994). It has also been suggested that an unknown molecule having such a biological activity is also present in XLH. Clinical findings for XLH patients are characterized by hypophosphatemia with enhanced urinary phosphate excretion, which is the same as tumor-induced osteomalacia patients, and XLH develops osteomalacia or rickets due to calcification insufficiency in bone tissues. A gene responsible for XLH has been shown to be a gene encoding an endopeptidase-like protein, called PHEX. Recently, Hyp mice, the natural mutant mice, known to express a phenotypic trait similar to that of XLH, have been shown to have a partial deletion in the gene encoding PHEX, thereby suggesting that determining Hyp mice to be XLH model mice is valid (Strom, T. M. et al. Human Molecular Genetics 6:165-171, 1997). That the hypophosphatemia-inducing factor in Hyp mice is a humoral factor has been shown by a parabiosis experiment using Hyp mice and normal mice (Meyer, R. A. et al., J. Bone Miner. Res. 4: 493-500, 1989). In this experiment, blood phosphate levels of normal mice decreased, and urinary phosphate excretion increased. Hence, it has been considered that the humoral hypophosphatemia-inducing factor existing in the Hyp mice acted on the normal mice. So far the relationship between PHEX expected to have a peptide cleavage activity and this unknown hypophosphatemia-inducing factor has not been clear. However, some hypotheses concerning a relationship that PHEX may regulate an activity of an unknown hypophosphatemia-inducing factor and a possibility that a hypophosphatemia-inducing factor found in tumor-induced osteomalacia may be identical to that found in XLH each other have been proposed (Drezner, M. K. Kidney Int 57:9-18, 2000). According to this hypothesis, PHEX and a hypophosphatemia-inducing factor are both normally expressed in the same cell, and PHEX acts suppressively on the hypophosphatemia-inducing factor. The functions of PHEX decrease or disappear in XLH patient, so that activity of hypophosphatemia-inducing factor is strongly expressed. It is presumed that in tumor-induced osteomalacia, both PHEX and hypophosphatemia-inducing factor are elevated, and finally active hypophosphatemia-inducing factor quantitatively exceeds the normal level. It is also presumed that this hypophosphatemia-inducing factor acts suppressively on the phosphate transport activity of NPT2 which is one of phosphate transporters in the kidney. Many attempts to search for such an unknown hypophosphatemia-inducing factor have been made, but none were able to identify the molecule. According to a study by Cai et al, it has been presumed that the molecular weight of a hypophosphatemia-inducing factor is between 8 kDa to 25 kDa (Cai, Q. et al., N. Engl. J. Med. 330: 1645-1649, 1994), while Rowe et al have proposed 56 kDa and 58 kDa proteins as candidate molecules. Recently, Rowe et al have filed a patent application (WO99/60017) for a polypeptide consisting of 430 amino acid residues as a tumor-derived phosphate metabolism-regulating factor for tumor-induced osteomalacia. However, the polypeptide disclosed in this application was a partial sequence of a protein which was originally present, and no biological activity relating to hypophosphatemia-inducing activity was disclosed. Recently, a polypeptide corresponding to the full-length molecule as disclosed by the name of MEPE in this patent has been reported, but no activity to induce hypophosphatemia was also disclosed (Rowe, P. S. N. et al, Genomics 67:54-68, 2000). In addition, no sequence or structural similarity between this molecule and the polypeptide of the present invention has been recognized. \\n     As described above, the presence of a physiologically active factor having an activity to induce hypophosphatemia is inferred, but the entity thereof has not been shown so far. In the present invention, we have clarified the entity of the polypeptide, and a gene sequence encoding the polypeptide. Further, as described later, we have produced the polypeptide of the present invention, showed that the product acts as a regulatory factor for phosphate metabolism, calcium metabolism and vitamin D metabolism or calcification and osteogenesis, and the product is useful as a pharmaceutical composition. Furthermore, we have shown that the antibody of the present invention is useful not only for therapy, but also for clinical examination and diagnosis. Moreover, we have shown that the DNA encoding the polypeptide of the present invention is useful for the diagnosis of hereditary diseases, and for polymorphic diagnosis of phosphate metabolism, calcium metabolism and bone metabolism. \\n     The polypeptide having a hypophosphatemia-inducing activity of the present invention can be produced by introducing, for example, a sequence containing nucleotide Nos. 133 rd  to 885 th  of the nucleotide sequence represented by SEQ ID NO: 1 into an appropriate host cell in a form capable of being expressed to prepare a transformant cell, and then allowing the DNA introduced into the transformant cell to be expressed. In addition, the polypeptide chain that is produced in this manner may be modified by a protein modification mechanism of the host, such as cleavage or addition of sugar chains. \\n     The polypeptide of the present invention can be obtained by culturing the above transformants, and then collecting from the culture product. The term “culture product” means, in addition to culture supernatant, any cultured cells or cultured microorganisms, or disrupted cells or disrupted microorganisms. \\n     The transformant of the present invention may be cultured by any ordinary method for culturing a host. \\n     Either natural or synthetic medium can be used for culturing the transformant that is obtained using a microorganism, such as  Escherichia coli  or yeast as a host, as long as it contains a carbon source, a nitrogen source, inorganic salts and the like that are assimilable by microorganisms and allows efficient culturing of transformants. Examples of a carbon source include carbohydrates such as glucose, fructose, sucrose or starch, organic acid such as acetic acid or propionic acid, and alcohols such as ethanol or propanol. Examples of a nitrogen source include ammonia, ammonium salts of organic or inorganic acid such as ammonium chloride, ammonium sulfate, ammonium acetate or ammonium phosphate, or other nitrogen-containing compounds, and peptone, meat extract, and corn steep liquor. Examples of minerals include potassium primary phosphate, potassium secondary phosphate, magnesium phosphate, magnesium sulfate, sodium chloride, ferrous sulfate, manganese sulfate, copper sulfate and calcium carbonate. \\n     Culturing is normally performed under aerobic conditions, such as shake culture or aeration-agitation culture, at 37° C. for 4 to 48 hours. During a culturing period, pH is maintained within 6.0 to 8.0. pH is adjusted using inorganic or organic acid, alkali solution or the like. While culturing, antibiotics, such as ampicillin or tetracycline, may be added to media, if necessary. \\n     When a microorganism transformed with an expression vector using an inducible promoter is cultured, an inducer may be added to a medium, if necessary. For example, when a microorganism transformed with an expression vector having T7 promoter which can be induced with isopropyl-β-D-thiogalactopyranoside (IPTG) is cultured, IPTG or the like may be added to the medium. In addition, when a microorganism transformed with an expression vector using trp promoter which can be induced with indoleacetic acid (IAA) is cultured, IAA or the like can be added to a medium. \\n     Examples of a medium for culturing a transformant that is obtained using an animal cell as a host include a generally employed RPMI1640 medium, DMEM medium or a medium supplemented with fetal calf serum or the like. Culturing is performed normally under 5% CO 2  at 37° C. for 1 to 10 days. While culturing, antibiotics, such as kanamycin or penicillin, may be added to media, if necessary. After culturing, when the polypeptide of the present invention is produced within microorganisms or cells, the target polypeptide is collected by ultrasonication, repetitive of freeze-thawing, homogenizing treatment or the like to disrupt the microorganisms or cells. Further, when the polypeptide of the present invention is produced outside bacteria or cells, the culture solution is used intact, or centrifugation or the like is performed to remove the bacteria or cells. Then, the polypeptide of the present invention can be isolated and purified from the above culture product by a singular or a combined use of general biochemical methods for isolation and purification of protein, such as ammonium sulfate precipitation, gel chromatography, ion exchange chromatography and affinity chromatography. \\n     As described above about XLH, PHEX which is thought to be an endopeptidase, has an important meaning in regulation of the hypophosphatemia-inducing factor. Hence, it is possible that the polypeptide having the amino acid sequence of SEQ ID NO: 2 of the present invention may have varied activities as a result of further modification and cleavage. In the present invention, using CHO ras-clone 1 cells as a host, a cloned cell line producing a recombinant polypeptide conferred with six continuous His (SEQ ID NO: 88) at the C-terminus of the polypeptide chain of the present invention was prepared. This cell line was deposited at the National Institute of Advanced Industrial Science and Technology, International Patent Organism Depositary (Chuo 6, 1-1-1, Higashi, Tsukuba-shi, Ibaraki, Japan) (Accession number FERM BP-7273, original deposit date: Aug. 11, 2000). \\n     When the polypeptide of the present invention produced by the cell line and secreted into the culture solution was examined, gene products having different sizes were detected by Western blotting using an antibody recognizing His-tag sequence, as shown in  FIG. 2 . Proteins corresponding to respective bands were isolated and the determination of N-terminal amino acid sequences was performed. Thus, the N-terminal amino acid sequences were identical to the N-terminal amino acid sequence represented by SEQ ID NO: 4 and that represented by SEQ ID NO: 8, respectively. It is considered that the protein having the former sequence corresponds to a protein of which signal sequence is removed, and the protein having the latter sequence corresponds to a protein cleaved with an enzyme, such as an endopeptidase. \\n     Now, furin is known as one of proteolytic enzymes which recognize RXXR. Actually, when the polypeptide of the present invention was expressed in a furin-deficient cell line, no fragment was detected. Further, when a recombinant protein α1-PDX having furin inhibition activity was co-expressed with the polypeptide of the present invention, cleaved products in the supernatant decreased significantly. \\n     Hence, the present invention also encompasses a method for producing the polypeptide of the present invention which comprises the step of using furin-deficient cells upon culturing, or allowing co-existence with a substance which suppresses furin activity. \\n     Cai et al have suggested that phosphatetransport-suppressing activity in the culture supernatant obtained by culturing tumor-derived cells of tumor-induced osteomalacia exhibited within the molecular weight range of 8 kDa to 25 kDa when measured by a fractionation method with dialysis membranes. It is also conceivable that the activity may be varied by converting the polypeptide having the amino acid sequence of SEQ ID NO: 4 of the present invention into polypeptides resulting from cleavage between residue No. 179, Arg, and residue No. 180, Ser, of SEQ ID NO: 2. \\n     The polypeptide of the present invention has an activity to suppress the phosphate transport activity of epithelial cells of the renal proximal tubules, which is a form of the effect of hypophosphatemia-inducing activity, as shown in Table 2 (Example 7). Most free inorganic phosphate existing in the blood is filtered in the glomeruli of the kidneys, wherein approximately 80 to 90% of the inorganic phosphate is reabsorbed in the renal proximal tubule. \\n     This reabsorption is performed by phosphate transport by type II Na-dependent phosphate transporter existing on the lumen side of the proximal tubule. The polypeptide of the present invention has an activity to suppress phosphate transport activity. This means that the polypeptide of the present invention promotes urinary excretion of phosphate in vivo. Thus, it can be considered that the polypeptide of the present invention induces hypophosphatemia by exerting its activity to suppress phosphate reabsorption in the kidney, in particular, phosphate transport in renal cells of proximal tubules, so that the polypeptide of the present invention is expected to be the same substance as the above Phosphatonin. \\n     Recently, Na-dependent phosphate transporter in the intestinal tract has been identified. The transporter is named type IIb, because it has a high homology with that of type II Na-dependent phosphate transporter existing in the kidney. It is conceivable that the polypeptide of the present invention may also be responsible for suppressing type IIb Na-dependent phosphate transporter existing on the lumen side of the intestinal tract, similarly for type II Na-dependent phosphate transporter in the kidney. This can be regarded as a form of the effect of the hypophosphatemia-inducing activity. \\n     The in vivo activity of the polypeptide of the present invention was evaluated by an experiment wherein the above recombinant cells expressing the polypeptide of the present invention had been transplanted subcutaneously to nude mice. \\n     The transplanted cells in this experiment were grown in subcutaneous space of nude mice, and then allowed to form tumors. The polypeptide of the present invention produced and secreted by the cell with the tumor formation is characterized by being released into the body fluid of the mice, so that releasing of tumor-derived humoral factor in tumor-induced osteomalacia can be reproduced in this animal model. In this model experiment, as shown in Table 4 (Example 11), mice transplanted with cells expressing the polypeptide of the present invention developed evident hypophosphatemia, compared to control individual mice allowed to generate tumors by transplantation with CHO cells into which no DNA of the present invention had been introduced, or individual mice that have generated no tumor. Accordingly, the polypeptide of the present invention was shown to have the hypophosphatemia-inducing activity. In addition, it was also shown that the phosphate reabsorption rate also decreased, and that phosphate reabsorption in the kidneys was suppressed. Therefore, it was concluded that the polypeptide of the present invention is the hypophosphatemia-inducing factor in tumor-induced osteomalacia. \\n     On the other hand, in the above model experiment, hypocalcemia was found in the mice transplanted with the recombinant cells producing the polypeptide of the present invention. Thus, it was also shown that the polypeptide of the present invention is also an hypocalcemia-inducing factor. In the experiment described in Example 16, wherein CHO cells expressing the polypeptide of the present invention were transplanted into nude mice, it was shown that serum 1α,25-dihydroxyvitamin D levels continuously decreased. As described in Examples 19 and 20, it was shown that when a mutation-introduced polypeptide of the present invention or a wild type full-length polypeptide of the present invention was administered three times to normal mice, serum 1α,25-dihydroxyvitamin D levels decreased in both cases. Further, as described in Example 24, after a single administration of the polypeptide of the present invention, decreased serum 1α,25-dihydroxyvitamin D levels were observed within several hours. Hence, it is conceivable that this activity which causes such a decrease of 1α,25-dihydroxyvitamin D level is a major biological or physiological effect of the polypeptide of the present invention. \\n     As described above, serum 1α,25-dihydroxyvitamin D levels are ruled by 1α-hydroxylase and 24-hydroxylase. As described in Example 16, it was shown that the effect of the polypeptide of the present invention to decrease serum 1α,25-dihydroxyvitamin D levels is accompanied by fluctuations in expression of these metabolic enzymes. Further, as described in Example 24, at 1 hour after administration of the polypeptide of the present invention, decreased gene transcription products of 1α-hydroxylase which is responsible for production of active vitamin D metabolites, and increased gene transcription products of 24-hydroxylase which catabolizes active vitamin D metabolites were observed. The serum 1α,25-dihydroxyvitamin D levels gradually decreased after these fluctuations in expression, suggesting that the effect of the polypeptide of the present invention to decrease serum 1α,25-dihydroxyvitamin D levels is at least due to suppressed expression of 1α-hydroxylase gene and enhanced expression of 24-hydroxylase gene. \\n     In contrast, in a long-term transplantation experiment (from 44 to 46 days after transplantation) wherein CHO cells expressing the polypeptide of the present invention was transplanted into nude mice as described in Example 11, expression of the 1α-hydroxylase gene was elevated. The mouse serum PTH levels at this time point was proved to be significantly elevated compared to a control group. Thus, it can be presumed that enhanced expression of 1α-hydroxylase gene was caused by a PTH effect at a high level. However, interestingly, even in the presence of high PTH level, expression of 24-hydroxylase gene was kept elevated, it can be understood that the high PTH level failed to interfere with the regulation by the polypeptide of the present invention of expression of 24-hydroxylase gene. As described in Example 11, serum 1α,25-dihydroxyvitamin D3 levels was not increased although the mice expressed severe hypophosphatemia- or rickets-like clinical findings. This suggests the affect of continuous enhancement of expression of 24-hydroxylase gene by the polypeptide of the present invention. \\n     A. Nykjaer et al have revealed that 25-hydroxyvitamin D is reabsorbed in the renal proximal tubule (Cell, Vol. 96, p507-515, 1999). While the present specification does not describe it, but in an experiment wherein CHO cells expressing the polypeptide of the present invention had been transplanted into nude mice, no significant change was found in the serum 25-hydroxyvitamin D level. In addition, it was not recognized that the action of the polypeptide of the present invention affected the fractional excretion (calculated with urinary level/serum level/GFR) of main electrolytes, such as sodium, potassium or chloride, main amino acids or glucose, supporting the fact that the reabsorption function of the renal tubule was undamaged (T. Shimada et al., Proc. Natl. Acad. Sci, in press). Therefore, it was suggested that the polypeptide of the present invention does not decrease serum 1α,25-dihydroxyvitamin D levels by inhibiting reabsorption of 25-hydroxyvitamin D in the renal tubule, but by specifically acting on the synthetic pathway of 1α,25-dihydroxyvitamin D. \\n     It is known that the serum 1α,25-dihydroxyvitamin D level is significantly decreased in tumor-induced osteomalacia. Further, in hypophosphatemic vitamin D-resistant rickets (XLH) or Hyp, the model mice expressing the clinical conditions of XLH, serum 1α,25-dihydroxyvitamin D levels are within the normal range or within a range somewhat below the lower limit of the normal range, regardless of severely decreased serum phosphate levels. It is also known that the expression of 24-hydroxylase gene is elevated in Hyp mice. In these clinical conditions developing hypophosphatemia, normally, as serum phosphate level decreases, 1α-hydroxylase gene expression rises, thereby increasing serum 1α,25-dihydroxyvitamin D levels. Thus, it is thought that a failure in any of regulatory systems disabling such normal physiological response is at least one of the causes of the clinical conditions. These phenomena are analogous to the physiological responses observed in the mice described in Example 11, 19 or 20, strongly suggesting that the polypeptide of the present invention functions to decrease serum 1α,25-dihydroxyvitamin D3 levels in the above clinical conditions. \\n     It is clear from the X-ray images shown in  FIG. 5  that the degree of calcification in bone tissues of the mice transplanted with the recombinant cells expressing the polypeptide of the present invention was significantly decreased when compared to that in the control group. Thorax deformation or the like was also observed, suggesting that the polypeptide of the present invention had an effect on skeletal formation. \\n     In other words, it is conceivable that the polypeptide of the present invention has an effect on suppressing calcification of bone tissues, or effect on promoting recruitment of calcium and phosphate from bone tissues. It is also conceivable that significant decreases in both blood phosphate and calcium levels caused secondary suppression of bone tissue calcification. \\n     In Hyp mice, it is thought that bone-derived cells produce a factor which suppresses phosphate transport activity in the renal proximal tubule (Lajeunesse, D. et al., Kidney Int. 50: 1531-1538, 1996). Further, it has been reported that osteoblasts of Hyp mice release calcification-suppressing factors (Xiao, Z. S., Am. J. Physiol., E700-E708, 1998). As described above, in XLH and tumor-induced osteomalacia, clinical findings such as hypophosphatemia and insufficient calcification in bone tissues closely resembles from each other, and these clinical findings would likely be induced by a single humoral factor. Taken together these facts suggest the possibility that the renal phosphate transport-suppressing activity and the bone calcification-suppressing activity reported in these studies of Hyp mice can be caused by the same factor. In addition, it has also been reported that the osteoblasts of Hyp mice exhibit abnormal osteogenesis even in the state wherein calcium and phosphate levels are in the normal range (Ecarot, B. et al., J. Bone Miner. Res. 7: 215-220, 1992). The polypeptide of the present invention has activities that is similar to those of a putative factor of the above Hyp mice. Thus, it is conceivable that the polypeptide of the present invention has, in addition to the hypophosphatemia-inducing activity, an effect of directly regulating calcification of bone tissues not mediated by the abnormalities in calcium or phosphate metabolism. \\n     In the course of completing the present invention, in addition to a gene encoding the polypeptide of the present invention, dentin matrix protein-1 (DMP-1) was also obtained as shown in Table 1 of Example 3. This gene is abundantly expressed in the dentin of teeth, and a protein encoded by this gene is thought to have an important role, as an extracellular matrix protein of dentin, in the formation of calcified matrix of dentin. Similarly, a gene encoding a matrix extracellular phosphorylation protein (MEPE) was obtained as OST190. Detailed functions of the molecule are unknown. Further similarly, a gene encoding osteopontin was also obtained. MEPE, DMP-1 and osteopontin have common characteristics in that they are phosphorylation proteins having RGD motif sequence, are rich in serine and threonine that can be phosphorylated, have high contents of glutamic acid and aspartic acid that are acidic amino acids, and show intense acidic protein characteristics. A characteristic acidic region, named ASARM sequence, is conserved between MEPE and DMP-1 (Rowe, P. S. N. et al, Genomics 67: 54-68, 2000), suggesting similarity in their physiological or functional significance. Interaction with inorganic calcium and/or phosphate upon the start of calcification is thought to be one of the functions of such a characteristic protein. Expression of osteopontin gene in a variety of cells has been reported, such as in macrophages, in addition to osteoblasts and osteoclasts. On the other hand, expression of DMP-1 in bone tissues, particularly in osteocytes has been reported recently. The gene expression of MEPE in the myeloid tissue or in osteosarcoma cells such as SaOS-2 are known. The fact that such an acidic matrix protein found in the calcified tissue has been found together with the polypeptide of the present invention in the course of the present invention represents an aspect of a type of effect of the polypeptide of the present invention. Specifically, there are possibilities that the polypeptide of the present invention induces expression of calcified tissue-specific molecules which are represented by the above molecules, so that the polypeptide regulates calcification, calcium metabolism and phosphate metabolism in a cooperative manner, or the induced molecule secondarily regulates calcification, calcium metabolism, and metabolism. It is also conceivable that the polypeptide of the present invention can regulate bone metabolism by directly acting on osteoblasts, osteocytes and osteoclasts. Hence, the polypeptide of the present invention may be effective in the therapy for metabolic bone diseases as represented by osteoporosis. \\n     Recently, cells having osteoblast-like phenotype have been shown to appear at ectopic calcification sites, suggesting that calcification may occur by the mechanism similar to that of the process of calcification in bone tissue. Therefore, it is also conceivable that the polypeptide of the present invention is effective in the therapy of ectopic calcification by suppressing the appearance or function of such cells in charge of calcification. \\n     In the present invention, the above polypeptide can be modified. For example, polyethylene glycol, dextran, poly (N-vinyl-pyrrolidone), polypropylene glycol homopolymer, copolymer of polypropylene oxide/ethylene oxide, polyoxyethylated polyol, polyvinyl alcohol and the like are appropriately selected for use. As a modification method, any known technique can be employed. For example, one such technique is disclosed in detail in JP Patent Publication (PCT translation) No. 10-510980. \\n     4. Antibody Against Polypeptide of the Present Invention \\n     An antibody of the present invention specifically reacts with the above polypeptide of the present invention. In the present invention, the term “antibody” means the entire antibody molecule or the fragment thereof (for example, Fab or F(ab′) 2  fragment) which are capable of binding to the antigenic polypeptide or fragments thereof, and may be a polyclonal or a monoclonal antibody. \\n     The antibody of the present invention can be prepared according to a standard method. For example, the antibody can be prepared by either an in vivo method, which involves immunization of animals once or several times (booster immunization) at an interval of several weeks using an antigen together with an adjuvant, or an in vitro method, which involves isolating immunocytes and allowing the immunocytes to be sensitized using an appropriate culture system. Examples of immunocytes capable of producing the antibody of the present invention include spleen cells, tonsil cells and lymphoid cells. \\n     A polypeptide to be used as an antigen does not have to be the above entire polypeptide of the present invention. A part of the polypeptide may be used as an antigen. To use a short peptide as an antigen, particularly, a peptide having as short as approximately 20 amino acid residues, such a peptide is bound by chemical modification or the like to a carrier protein with high antigenicity, such as keyhole limpet hemocyanin or bovine serum albumin, or covalently bound to a peptide having the branched skeleton, such as a lysine core MAP peptide, instead of a carrier protein (Posnett et al., J. Biol. Chem. 263, 1719-1725, 1988; Lu et al., Mol. Immunol. 28, 623-630, 1991; Briand et al., J. Immunol. Methods 156, 255-265, 1992). \\n     As an adjuvant, for example, Freund&#39;s complete or incomplete adjuvant, aluminum hydroxide gel or the like is used. As animals to be administered with an antigen, for example, a mouse, rat, rabbit, sheep, goat, chicken, cattle, horse, guinea pig and hamster are used. \\n     Polyclonal antibodies can be obtained by collecting blood from these immunized animals, separating the serum, and purifying immunoglobulins using one of or an appropriate combination of ammonium sulfate precipitation, anion exchange chromatography, and protein A or G chromatography. When the above animal is a chicken, antibodies can be purified from the eggs. \\n     Monoclonal antibodies can be prepared by purification from the culture supernatant of hybridomas prepared by allowing immunocytes, which have been sensitized in vitro or of the above animals, to fuse with parent cells that can be cultured, or from ascites obtained by intraperitoneal inoculation of the hybridomas into the animals. As parent cells, a generally available established cell line of an animal, such as a mouse, can be used. A preferred cell line to be used herein has drug selectivity, and has a characteristic such that it cannot survive in HAT selection medium (hypoxantine, aminopterin and thymidine are contained) when it is in unfused state, but can survive only in its fused state with antibody-producing cells. Examples of such a cell line include X63, NS-1, P3U1, X63.653, SP2/0, Y3, SKO-007, GM1500, UC729-6, HM2.0 and NP4-1 cells. \\n     Specific techniques to prepare monoclonal antibodies are as follows. \\n     The polypeptide or the fragment thereof prepared as described above is administered as an antigen to the above animal. Antigen dosage per animal is 1 to 100 μg when an adjuvant is used. Immunization is performed mainly by intravenous, subcutaneous or intraperitoneal injection. In addition, the immunization interval is not specifically limited. At an interval of several days to several weeks, preferably 1 to 3 weeks, immunization is performed 1 to 10 times, preferably 2 to 5 times. 1 to 10 days later, preferably 1 to 4 days later the final immunization date, antibody-producing cells are collected. \\n     To obtain hybridomas, cell fusion of antibody-producing cells and parent cells (myeloma cells) is performed. Cell fusion is performed in a serum-free medium for culturing animal cells, such as DMEM or RPMI-1640 medium, by mixing 5×10 6  to 1×10 8  cells/ml antibody-producing cells with 1×10 6  to 2×10 7  cells/ml myeloma cells (a preferred ratio of antibody-producing cells to myeloma cells is 5:1), and performing fusion reaction under the presence of a cell fusion-promoting agent. As a fusion-promoting agent, polyethylene glycol or the like having a mean molecular weight of 1000 to 6000 daltons can be used. In addition, antibody-producing cells and myeloma cells can also be fused with a commercial cell fusion device using electric stimulation (for example, electroporation). \\n     Following the treatment to accomplish cell fusion, hybridomas of interest are selected from the cells. The cell suspension is appropriately diluted in, for example, RPMI-1640 medium containing fetal calf serum, inoculated on a microtiter plate at a concentration of approximately 5×10 5  cells/well. The selection medium is added to each well, and then culturing is performed while properly exchanging the selection medium. As a result, cells that have proliferated at around 14 days after the start of culturing in the selection medium can be obtained as hybridomas. The culture supernatant of the hybridomas that have proliferated is screened for the presence of antibodies which react with the polypeptide of the present invention. Screening for hybridomas may be performed according to an ordinary method, and the screening method is not specifically limited. For example, a part of the culture supernatant contained in wells in which hybridomas are grown is collected, and then screened by enzyme immunoassay, radioimmunoassay or the like. \\n     Alternatively, monoclonal antibodies can be prepared by culturing immortalized antibody-producing cells which are obtained by allowing an appropriate virus, such as EB virus, to infect immunocytes sensitized in vitro or of the above immunized animal. \\n     Aside from these cell engineering techniques, monoclonal antibodies can also be obtained by gene engineering techniques. For example, such antibody genes can also be amplified and obtained by PCR (polymerase chain reaction) from immunocytes sensitized in vitro or of the above animal. The gene is introduced into a microorganism, such as  Escherichia coli , so as to allow it to produce antibodies, or used to allow a phage to express the antibody as a fusion protein on the surface. \\n     Quantitative determination of the amount of the polypeptide of the present invention in vivo using the antibody of the present invention makes it possible to elucidate the relationship between the polypeptide of the present invention and clinical conditions of various diseases. Moreover, the antibody can be applied to diagnosis or therapy and subjected to perform efficient affinity purification of the polypeptide of the present invention. \\n     It is assumed that there are some diseases of which cause resides in a decreased of serum 1α,25-dihydroxyvitamin D levels induced by the excessive action of the polypeptide of the present invention. For example, although hypophosphatemic vitamin D-resistant rickets (XLH) develops severe hypophosphatemia, no increase is found in serum 1α,25-dihydroxyvitamin D3 levels. The reason is thought to be abnormality in groups of vitamin D metabolizing enzyme genes. In this disease, excessive action of the polypeptide of the present invention may be involved. In Hyp, which is a mouse model of XLH, enhanced expression of 24-hydroxylase gene has been reported. This agrees with the effect of the polypeptide of the present invention to induce enhanced 24-hydroxylase gene expression. Therefore, it is expected that this disease can be treated by administering the antibody against the polypeptide of the present invention to normalize vitamin D metabolism and then to correct the serum 1α,25-dihydroxyvitamin D3 levels. An example of a disease that presents clinical findings similar to that of XLH is autosomal dominant hypophosphatemic rickets (ADHR). Upon cloning of the polypeptide of the present invention, we have inferred that a gene encoding the polypeptide is a gene responsible for ADHR, based on its location on the chromosome. Recently, a gene responsible for ADHR has been analyzed and reported that the disease is caused by a missense mutation in the gene encoding the polypeptide. We have further clarified that this mutation confers resistance against enzymatic cleavage, and showed that excessive effect of this molecule is a cause of the disease. It is conceivable that the antibody against the polypeptide of the present invention is effective in treating this disease through its suppressing effect. ADHR develops osteomalacia, disordered mineral metabolism and disordered vitamin D metabolism. Thus, among metabolic bone disease which closely relate to such metabolic pathway, there may be diseases on which the polypeptide of the present invention acts as a cause of the disease. It can be expected that the antibody against the polypeptide of the present invention is effective for such a disease. As one effect of 1α,25-dihydroxyvitamin D, suppression of differentiation into adipocytes is known. It is known that adipocytes in the bone marrow increases with age. In this case, there may be enhanced differentiation into adipocytes from common precursor cells of osteoblasts, adipocytes and stromal cells supporting hematopoiesis existing in the bone marrow. It is conceivable that in this process, the polypeptide of the present invention may excessively act to decrease local 1α,25-dihydroxyvitamin D levels. Therefore, it can be expected that the use of the antibody against the polypeptide of the present invention enables to increase blood or local 1α,25-dihydroxyvitamin D levels, suppress differentiation into adipocytes, and improve decreased ability of bone formation or hematopoiesis. In addition, the antibody is also expected to be effective against obesity. It is conceivable that there are other such diseases in which the polypeptide of the present invention is involved. Such a disease can be screened by an immunological quantitative determination method as represented by ELISA combined with the use of the antibody against the polypeptide of the present invention. Accordingly, the physiological normal range of the polypeptide of the present invention can be established, and disease groups deviated from the range can be clarified. It can be expected that the antibody against the polypeptide of the present invention is used for therapeutic purposes against diseases showing abnormally high blood levels of the polypeptide of the present invention as measured by the above method. \\n     5. Pharmaceutical Composition \\n     (1) Pharmaceutical Composition Comprising the Polypeptide of the Present Invention \\n     The polypeptide of the present invention can be used as a pharmaceutical composition for diseases with unfavorably elevated blood phosphate levels. In chronic renal failure, decreased levels of phosphate excretion from the kidney result in elevated blood phosphate levels. Hyperphosphatemia further aggravates renal functions, and promotes secretion of parathyroid hormone from the parathyroid gland, thereby inducing secondary hyperparathyroidism. This disease causes itching of the skin, as well as decreased Ca absorption from the intestinal tract due to disordered synthesis of 1α,25-dihydroxyvitamin D3 in the kidney. In addition, the state of oversecretion of parathyroid hormone due to the retention of blood phosphate promotes Ca mobilization from bone tissues. When this state continues, osteitis fibrosa or hyperplasia of the parathyroid glands that is one of the clinical conditions of renal osteodystrophy. One preferred method to evade this state is to improve the above-mentioned hyperphosphatemia, but current medical treatment cannot sufficiently control hyperphosphatemia. In chronic renal failure at a stage wherein the urination function is retained, the polypeptide of the present invention has an activity to correct blood phosphate levels by suppressing type II Na-dependent phosphate transporter existing in epithelial cells of the renal proximal tubules to promote phosphate excretion into urine (phosphate transport-suppressing activity). In addition, the polypeptide of the present invention can correct blood phosphate levels by acting on the intestinal tract in a way similar to that in the kidneys to suppress type IIb Na-dependent phosphate transporter and thus reduce phosphate absorption into the body. \\n     The polypeptide of the present invention can also be used as a pharmaceutical composition for diseases resulting from abnormal calcium metabolism and phosphate metabolism. The term “abnormal calcium metabolism” indicates a state at which serum calcium levels deviate from clinically defined normal range, or a state at which serum calcium levels are within the normal range, but the functions of the kidney, intestinal tract, bone tissue and parathyroid glands are abnormally increased or decreased to maintain serum calcium levels, or a state at which hormones regulating the serum calcium, such as parathyroid hormone, 1α,25-dihydroxyvitamin D3 or calcitonin exhibit abnormal values. In addition, the term “abnormal phosphate metabolism” indicates a state at which serum phosphate levels deviate from the clinically-defined normal range, or a state in which serum phosphate levels are within the normal range, but phosphate-balancing functions are abnormally increased or decreased in the kidney, intestinal tract and bone tissue. \\n     Renal osteodystrophy along with the above secondary hyperparathyroidism takes various clinical forms, such as adynamic bone disease, osteitis fibrosa or the mixed type thereof. Against secondary hyperparathyroidism, 1α,25-dihydroxyvitamin D3, 1α-hydroxyvitamin D3 or the like is generally used to suppress parathyroid hormone. When the value of parathyroid hormone is not sufficiently suppressed, a pulse therapy (hereinafter, may also be referred to as “vitamin D pulse therapy”) which involves administering over dose of 1α,25-dihydroxyvitamin D3 or 1α-hydroxyvitamin D3 may be performed. The normal serum parathyroid hormone level is 65 pg/ml or less. When parathyroid hormone levels are at the normal levels in this disease condition, adynamic bone disease, which is a form of renal osteodystrophy, is caused. Moreover, when parathyroid hormone levels increase, osteitis fibrosa, which is contrary to the above clinical condition, occurs. As a recent medical guideline for such diseases, it has been proposed that parathyroid hormone levels be maintained at approximately 130 to 260 pg/ml. However, fundamental causes of abnormal metabolism remain unknown. It is known that parathyroid hormone is induced by elevated blood phosphate levels, and is suppressed by elevated serum calcium levels. Since the polypeptide of the present invention lowers blood phosphate levels and blood calcium levels, it can be inferred that the polypeptide can modify the functions of parathyroid hormone. Further, there has been reported that in tumor-induced osteomalacia patients, 1α,25-dihydroxyvitamin D3 falls to the detection limit or below. Thus, it can also be inferred that the polypeptide of the present invention may be involved in regulating the activity of 1α,25-dihydroxyvitamin D3. It is conceivable that in renal osteodystrophy with the above abnormal regulation or impaired activity of parathyroid hormone, the polypeptide of the present invention can be used as a clinically useful pharmaceutical composition for either adynamic bone disease or osteitis fibrosa. Therefore, it is conceivable that administration of the polypeptide of the present invention provides a useful therapy for either osteitis fibrosa or adynamic bone disease, which are contrary to each other, in renal osteodystrophy. \\n     Furthermore, the polypeptide of the present invention can be used as a pharmaceutical composition for ectopic calcification. Calcification of tissues other than bone tissues damages biofunctions. In particular, dysfunction due to calcification of the heart or blood vessel threatens life. A risk factor of this ectopic calcification is an increase in the product of blood calcium ion and phosphate ion levels (hereinafter, may also be referred to as “the calcium-phosphate product”). In the above therapy for secondary hyperparathyroidism, when the vitamin D pulse therapy is performed against clinical conditions of hyperphosphatemia, the calcium-phosphate product can increase to cause ectopic calcification. Calcification of the cardiovascular system in hemodialysis patients is a serious problem. The polypeptide of the present invention has an activity to significantly decrease serum calcium levels and phosphate levels as shown in Table 4, so that it can be expected to be effective against ectopic calcification along with various diseases. \\n     Furthermore, the polypeptide of the present invention can be used as a pharmaceutical composition against metabolic bone diseases. The polypeptide of the present invention has a strong regulatory activity for calcium metabolism, phosphate metabolism, calcification or vitamin D metabolism. Examples of hormones involved in calcium metabolism and phosphate metabolism include calcitonin, parathyroid hormone and 1α,25-dihydroxyvitamin D3. Calcitonin has an activity to decrease serum calcium, and parathyroid hormone and 1α,25-dihydroxyvitamin D3 have an activity to increase serum calcium. Parathyroid hormone has been reported to have an effect on phosphate excretion to urine, and calcitonin has also been reported to have similar activity. 1α,25-dihydroxyvitamin D3 has an activity to promote phosphate absorption from the intestinal tract. As described above, each hormone has a different activity for maintaining the balance between blood calcium and phosphate levels, but calcitonin and 1α,25-dihydroxyvitamin D3 are used as therapeutic agents for osteoporosis. In addition, parathyroid hormone is being developed as a therapeutic drug for osteoporosis. \\n     Bone metabolism is characterized in that catabolism and anabolism of bone tissues, that is, bone resorption and bone formation, are coupling. Continuous administration with parathyroid hormone causes bone mass to decrease. However, when allowed to act intermittently, parathyroid hormone is known to promote bone formation. Since the polypeptide of the present invention has an effect of regulating calcium metabolism and phosphate metabolism, it can be expected that the polypeptide can be effective against metabolic bone diseases including osteoporosis, when an appropriate method for using the polypeptide is selected. \\n     Furthermore, the polypeptide of the present invention can be used as a pharmaceutical composition for diseases or clinical conditions with unfavorably elevated serum 1α,25-dihydroxyvitamin D levels, or for clinical conditions with unfavorable physiological responses induced by serum 1α,25-dihydroxyvitamin D. \\n     1α,25-dihydroxyvitamin D is known to act on the parathyroid glands to suppress secretion of parathyroid hormone (PTH). Therefore, a therapy, which has been clinically established for secondary hyperparathyroidism in chronic renal failure, particularly for cases with high serum PTH levels, involves intermittent administration of a high concentration of 1α,25-dihydroxyvitamin D. A disadvantage of this therapy is that it easily induces ectopic calcification. In chronic renal failure patients with high serum phosphate levels, unfavorable calcification due to administration of 1α,25-dihydroxyvitamin D is often observed in tissues other than bone tissues. Since the polypeptide of the present invention has an effect to promote a quick decrease in serum 1α,25-dihydroxyvitamin D within several hours after administration of the polypeptide, the polypeptide is useful in therapy and in preventing ectopic calcification that is caused by excessive 1α,25-dihydroxyvitamin D levels. \\n     Moreover, intermittent administration of high concentrations of 1α,25-dihydroxyvitamin D causes excessively suppressed PTH secretion, so that it can induce adynamic bone disease developing clinical conditions such as arrested bone metabolism. For such a case, it can be expected that the administration of the polypeptide of the present invention would cause 1α,25-dihydroxyvitamin D in serum to decrease, promote proper PTH secretion in the parathyroid glands, and provide recovery from adynamic bone disease. \\n     For calcification of blood vessels, involvement of 1α,25-dihydroxyvitamin D as a calcification-promoting factor has been reported. The polypeptide of the present invention can be used therapeutically or prophylactically against clinical conditions associated with calcification of blood vessels, such as arteriosclerosis due to aging, diabetic angiopathy, or calcification of the cardiovascular system in dialysis patients. \\n     Calcium absorption from the intestinal tract is known to be promoted by 1α,25-dihydroxyvitamin D in serum. The polypeptide of the present invention can be used to correct hypercalcemia by decreasing serum 1α,25-dihydroxyvitamin D levels. Examples of a cause of hypercalcemia include overproduction of PTH due to primary hyperparathyroidism, a high concentration of 1α,25-dihydroxyvitamin D along with chronic granuloma, such as sarcoidosis or tuberculosis, or accelerated bone resorption due to PTHrP produced by malignant tumors. In hypercalcemia which is mainly caused not only by excessive 1α,25-dihydroxyvitamin D, but also by excessive PTH or PTHrP, it can be expected that administration of the polypeptide of the present invention causes the serum 1α,25-dihydroxyvitamin D level to decrease, so as to improve hypercalcemia. In particular, activated macrophages in chronic granuloma, such as sarcoidosis or tuberculosis, has 1α-hydroxylase activity and excessively produces 1α,25-dihydroxyvitamin D3. It is expected that this 1α-hydroxylase activity is directly lowered by the polypeptide of the present invention. \\n     1α,25-dihydroxyvitamin D is known to promote differentiation of osteoclasts, and administration of the polypeptide of the present invention is expected to suppress bone resorption. In vitro, 1α,25-dihydroxyvitamin D is known to be a strong osteoclast differentiation-inducing factor. Excessive bone resorption by osteoclasts causes osteopenia as represented by osteoporosis. In such diseases for which promoted bone resorption is observed, the polypeptide of the present invention is expected to restore bone turnover to the normal condition by transiently lowering serum 1α,25-dihydroxyvitamin D levels. In addition to suppressing the differentiation of osteoblasts in vitro, 1α,25-dihydroxyvitamin D has also been suggested to be a factor of suppressing bone formation in vivo in a bone transplantation experiment using vitamin D receptor-deficient mice. From such view points, it can be expected that the polypeptide of the present invention, which is capable of lowering 1α,25-dihydroxyvitamin D, is effective against metabolic bone diseases with decreased bone mass. Further, there has been a report confirming that administration of 24,25-dihydroxyvitamin D, which is one of vitamin D3 metabolites, causes increased bone mass. 24,25-dihydroxyvitamin D is a product of hydroxylation of 25-hydroxyvitamin D by 24-hydroxylase. Since the polypeptide of the present invention has an effect to enhance the expression of 24-hydroxylase gene significantly, it can be expected that administration of this peptide causes blood 24,25-dihydroxyvitamin D levels to increase, and to increase bone mass under clinical conditions in bone diseases, such as osteoporosis or skeletal dysplasia. \\n     PTH is known to have a strong bone resorption-promoting effect. However, bone turnover can be stimulated by intermittent administration of PTH, and finally, an effect of increasing bone mass can be expressed. Physiological or biological activities of the polypeptide of the present invention that have been shown so far include: an effect of regulating 1α,25-dihydroxyvitamin D; effects of regulating serum calcium and serum phosphate levels; and effect of regulating calcification. Thus, it is conceivable that bone turnover can be regulated by allowing the polypeptide to effectively act on bone tissues. Therefore, the polypeptide can be expected to be effective against postmenopausal osteoporosis with enhanced bone turnover and senile osteoporosis with lowered bone turnover. There may be other diseases that the polypeptide of the present invention is involved in. Such a disease can be screened by an immunochemical assay as represented by ELISA combined with the use of one or more antibodies against the polypeptide of the present invention. \\n     Accordingly, the physiological normal range of the polypeptide of the present invention can be set, and disease groups with the polypeptide levels which deviate from the range can be clarified. It can be expected that the polypeptide of the present invention is therapeutically used for diseases showing abnormally low blood levels of the polypeptide of the present invention as measured by the above method. \\n     (2) Pharmaceutical Composition Containing the Antibody of the Present Invention \\n     The antibody of the present invention can be used as a pharmaceutical composition for vitamin D-resistant rickets and tumor-induced osteomalacia. The neutralizing antibody of this polypeptide can be obtained as a polyclonal antibody or a monoclonal antibody by the above-mentioned method for preparing antibodies. As a method for more properly using the antibody as a medicine, a human-type antibody or a humanized antibody can be prepared. Hypophosphatemia and osteomalacia in tumor-induced osteomalacia can be treated or improved by suppressing excessive activity of the polypeptide of the present invention. It can be expected that administration of the neutralizing antibody against the polypeptide of the present invention improves tumor-induced osteomalacia symptoms. Further, since hypophosphatemia-inducing factor and calcification-suppressing factor in XLH are thought to be equivalent to the polypeptide of the present invention, the neutralizing antibody can also be a therapeutic agent against vitamin D-resistant rickets including XLH. \\n     The antibody of the present invention can be used as a pharmaceutical composition for diseases with abnormal calcium or phosphate metabolism, or metabolic bone diseases associated with the presence of the polypeptide of the present invention in an excessive amount. As described above, in chronic renal failure or hemodialysis patients, abnormalities occur in the mechanism for maintaining the homeostasis of calcium metabolism or phosphate metabolism, and these abnormalities may be due to overproduction or accumulation of the polypeptides of the present invention. It has been shown that the polypeptide of the present invention regulates bone metabolism. Hence, it is conceivable that metabolic bone diseases which are caused by the presence of excessive amounts of the polypeptide of the present invention, may also exist. In this case, it can be expected that the clinical conditions can be treated or improved by the antibody against the polypeptide of the present invention. \\n     Examples of diseases to which the antibody of the present invention can be applied include at least one kind of bone disease, such as osteoporosis, vitamin D-resistant rickets, renal osteodystrophy, dialysis-associated bone diseases, osteopathy with hypocalcification, Paget&#39;s disease and tumor-induced osteomalacia. Here, the bone disease may be a single disease, a complication thereof, or the bone disease complicated with any disease other than the above diseases. \\n     (3) Administration Protocol \\n     A pharmaceutical composition which contains the polypeptide of the present invention or the antibody thereof, as an active ingredient, may contain a pharmaceutically acceptable carriers and additives. Examples of such carriers and additives include water, pharmaceutically acceptable organic solvent, collagen, polyvinyl alcohol, polyvinylpyrrolidone, carboxy vinyl polymer, sodium alginate, water-soluble dextran, sodium carboxymethyl starch, pectin, xanthan gum, gum arabic, casein, gelatine, agar, glycerin, propylene glycol, polyethylene glycol, Vaseline, paraffin, stearyl alcohol, stearic acid, human serum albumin, mannitol, sorbitol, lactose, and surfactant which is a pharmaceutically acceptable additives. The additives to be used herein are properly selected from the above items either singly or in combination depending on the employed dosage form of the present invention. \\n     When the polypeptide or the antibody of the present invention is used as a prophylactic or therapeutic agent for bone diseases, the subject for which the polypeptide or the antibody is used is not specifically limited. \\n     The polypeptide of the present invention can be used as a pharmaceutical composition which is capable of regulating calcium metabolism, phosphate metabolism, calcification or vitamin D metabolism in organisms. \\n     The antibody of the present invention can be used specifically to treat or prevent at least one kind of bone disease, such as osteoporosis, vitamin D-resistant rickets, renal osteodystrophy, dialysis-associated bone diseases, osteopathy with hypocalcification, Paget&#39;s disease, tumor-induced osteomalacia, as described above. The bone disease, for which the polypeptide or the antibody of the present invention can be used, may be a single disease, a complication thereof, or the bone disease complicated with any disease other than the above diseases. \\n     A prophylactic agent or therapeutic agent which contains the polypeptide or the antibody of the present invention may be administered orally or parenterally in the case of the polypeptide, and parenterally in the case of the antibody. \\n     When the polypeptide of the present invention is administered orally, the dosage form to be applied thereto may be a solid preparation, such as a tablet, granule, powder or pill, or a liquid preparation, such as a liquid drug or syrup, or the like. In particular, a granule and powder can be formulated into a unit dose form, that is, a capsule. In the case of a liquid preparation, it can be formulated into a dry product which is re-dissolved for use. \\n     Of these dosage forms, a solid preparation for oral administration normally contains in its composition additives, which are generally pharmaceutically employed, such as a binder, excipient, lubricant, disintegrating agent or wetting agent. In addition, a liquid preparation for oral administration normally contains in its composition an additives, which is pharmaceutically generally employed, such as a stabilizer, buffer, corrigent, preservative, flavoring agent or colorant. \\n     When the polypeptide or the antibody of the present invention is administered parenterally, it can be formulated into an injection, suppository or the like. \\n     In the case of an injection, it is normally provided in a unit dose ampule or a vessel for multiple dose, or may be in a powdery form which is re-dissolved, when used, in an appropriate carrier, such as sterilized water containing no pyrogenic substance. These dosage forms normally contain in their compositions, additives which are pharmaceutically generally employed, such as an emulsifier or suspension. Examples of injection procedures include drip intravenous infusion, intravenous injection, intramuscular injection, intraperitoneal injection, subcutaneous injection or intradermal injection. In addition, doses differ depending on the age of the subject, route for administration and dosage frequency, and can be varied widely. \\n     In this case, the effective dose to be administered is a combination of the effective dose of the peptide or the antibody of the present invention with an appropriate diluent, and a pharmacologically usable carrier, in the case of the polypeptide, is 0.01 to 100 μg, preferably, 0.5 to 20 μg/administration/kg of body weight. Further, in the case of the antibody, the effective dose is 0.1 μg to 2 mg, preferably, 1 to 500 μg/administration/kg of body weight. \\n     6. Diagnostic Agent of Disease \\n     (1) The Antibody or the Polypeptide of the Present Invention \\n     The antibody of the present invention is used for detection or quantitative determination of the polypeptide of the present invention or of the metabolites existing in blood or urine, so that the relationship between the polypeptide of the present invention and clinical conditions can be elucidated, and the antibody can be applied as a diagnostic agent for associated diseases. \\n     The term “associated disease” means a bone disease or a disease developing at least an abnormality from among: abnormal calcium metabolism, abnormal phosphate metabolism, abnormal calcification and abnormal vitamin D metabolism. Examples of such a disease include osteoporosis, vitamin D-resistant rickets, renal osteodystrophy, dialysis-associated bone diseases, osteopathy with hypocalcification, Paget&#39;s disease, renal failure, renal phosphate leak, renal tubular acidosis and Fanconi&#39;s syndrome. \\n     Methods for quantitatively determining bound molecules using antibodies have been generalized, such as radioimmunoassay or enzyme immunoassay. Levels of the polypeptide of the present invention in blood or urine measured by these methods can be indicators for new clinical judgement. For example, when a rickets patient shows high blood levels of the polypeptide of the present invention compared to a normal subject, XLH or ADHR can be strongly suspected. Further, based on changes in blood levels of the polypeptide of the present invention, prognosis of a progress into secondary hyperparathyroidism in a chronic renal failure patient can be made. \\n     For tumor-induced osteomalacia, generally, it is often difficult to find a tumor. However, the use of the antibody of the present invention enables to establish useful diagnostic measures. For example, when a patient with no family history of rickets or osteomalacia shows significantly higher blood levels of the polypeptide of the present invention compared to a normal individual, tumor-induced osteomalacia can be suspected. \\n     (2) DNA of the Present Invention \\n     In the present invention, detection of abnormal DNA of the present invention from a patient with abnormal phosphate metabolism or abnormal calcium metabolism, or a patient with a metabolic bone disease makes it possible to diagnose and prevent the disease. \\n     A search for the nucleotide sequence of the DNA of the present invention over the Genbank nucleotide sequence database reveals that the nucleotide sequence (in the form of three fragments) matches with a sequence of human 12p13 BAC RPCI11-388F6 (Accession No. AC008012). This fragmentation indicates that the nucleotide sequence of the DNA of the present invention is provided as a splicing product from a chromosome sequence. Thus it is clear that the DNA encoding the polypeptide of the present invention contains at least a sequence ranging from the 498 th  to 12966 th  nucleotides of a sequence represented by SEQ ID NO: 11 or partial fragments thereof. Substitution, insertion or deletion of nucleotides within the range causes increases, decreases or disappearance of the biological and physiological activities of the polypeptide of the present invention. The polypeptide of the present invention has a strong effect on phosphate metabolism, calcium metabolism, bone metabolism, and vitamin D metabolism. Therefore, when gene polymorphism or mutation due to substitution, insertion, deletion and the like of partial nucleotides of the nucleotide sequence represented by SEQ ID NO: 11 or partial regions thereof (for example, a sequence from the 498 th  to the 12966 th ) is shown, diagnosis and prevention of a disease containing abnormal phosphate metabolism and calcium metabolism, or a disease developing abnormal bone metabolism, or a disease developing abnormal vitamin D metabolism become possible. \\n     Now, it has been reported that the gene responsible for ADHR is present at 12p13 as a result of linkage analysis of families having ADHR (Econs, M. J. et al., J. Clin. Invest. 100: 2653-2657, 1997). In this report, it has also been shown that the responsible gene is present within a 18 cM range between micro satellite markers D12S100 and D12S397. We confirmed that the location at which the DNA of the present invention is present on the chromosome by comparing with the reported location. The region at which the DNA encoding the polypeptide of the present invention is present was identical to the region at which the gene responsible for ADHR is present. Based on the biological activities of the polypeptide of the present invention and the location of the gene on the chromosome, it is conceivable that the polypeptide of the present invention is encoded by the responsible gene of ADHR. This can be confirmed by separating cellular components from the blood of ADHR patients, isolating chromosomal DNAs from the cellular components, and then finding mutations in the nucleotide sequence within the region represented by SEQ ID NO: 11. Hence, the gene having the above nucleotide sequence region is used as a diagnostic agent for autosomal dominant hypophosphatemic rickets, X-linked hypophosphatemic rickets, hypophosphatemic bone disease, osteoporosis and the like. \\n     Between exon 1 and exon 2 of the DNA region encoding the polypeptide of the present invention that we have specified, STS sequence, which has been registered at NCBI Genbank under Accession No. G19259, is present. This marker is thought to be very important in studying the relationship between the DNA and hereditary characters. \\n     The present invention will bring a big change in understanding conventional calcium metabolism, phosphate metabolism, bone metabolism and vitamin D metabolism. According to the present invention, there is provided a delayed progress to the stage of hemodialysis in chronic renal failure, or a new therapeutic method and diagnostic method for phosphate metabolism-associated and calcium metabolism-associated diseases, and metabolic bone diseases. Further, the present invention is also useful to improve or support existing therapeutic methods. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF DRAWINGS \\n         FIG. 1  includes photographs showing the amplification products analyzed using agarose electrophoresis. To study the tumor specificity of OST311, PCR was performed using, as templates, first-strand cDNA extracted from tumor tissues and first-strand cDNA extracted from control bone tissues, and using OST311 specific primers represented by SEQ ID NOS: 22 and 23, and G3PDH specific primers represented by SEQ ID NOS: 26 and 27. \\n         FIG. 2  is a photograph showing that recombinant OST311 was detected by performing Western blotting for elution fractions which had been prepared by subjecting recombinant OST311 to affinity purification using nickel resin, and then isolated and purified using strong cation-exchange resin SP-5PW. \\n         FIG. 3A  shows the results of predicting sites of the polypeptide having the amino acid sequence represented by SEQ ID NO: 2, which are appropriate for preparation of a peptide antibody, using a computing function of MacVector version 6.5.1. \\n         FIG. 3B  shows the results of predicting the degree of hydrophobicity of the polypeptide having the amino acid sequence represented by SEQ ID NO: 2 using the computing function of MacVector version 6.5.1. \\n         FIG. 4  shows time-course changes in average body weight for 31 days after transplantation of CHO-OST311H cells between a non tumor-bearing group (a line in the graph indicated with avr.−) and a CHO-OST311H cells-transplanted tumor-bearing group (a line in the graph indicated with avr.+). \\n         FIG. 5  includes X-ray pictures showing the whole skeletal soft roentgenogram of a control CHO ras clone-1 cells-transplanted tumor-bearing individual, a CHO-OST190H cells-transplanted tumor-bearing individual, and a CHO-OST311H cells-transplanted tumor-bearing individual. \\n         FIG. 6  shows the results of alignment of amino acid sequence between human OST311 polypeptide and mouse OST311 polypeptide. \\n         FIG. 7  shows the results of measuring serum phosphate levels, serum calcium levels and serum alkaline phosphatase activities of a non tumor-bearing group (n=6), a CHO ras clone-1 tumor-bearing group (n=10), a CHO-OST190H tumor-bearing group (n=10), and a CHO-OST311H tumor-bearing group-1 (n=6), and a CHO-OST311H tumor-bearing group-2 (n=6). Blood was collected from the heart of each individual on days 44 to 46. \\n         FIG. 8  includes photographs showing comparison of expression levels of sodium-phosphate cotransporter (NaPi-7) as measured by the Western blotting method. Specifically, brush border membranes of proximal tubular epithelial cells were prepared from the kidneys excised from the CHO-OST311H tumor-bearing individuals, and non tumor-bearing individuals, and then the expression levels of NaPi-7 were compared by the Western blotting method. \\n         FIG. 9A  includes photographs showing changes in mRNA levels, as detected by Northern blotting, of renal sodium-phosphate cotransporters (NaPi-7, NPT-1). The kidneys were collected from the mice sacrificed on days 44 to 46 after tumor transplantation. \\n         FIG. 9B  includes photographs showing changes in mRNA levels, as detected by Northern blotting, of a sodium-phosphate cotransporter (NaPi-IIb) in the small intestines of the mice. The small intestines were collected from the mice sacrificed on days 44 to 46 after tumor transplantation. \\n         FIG. 9C  includes photographs showing changes in mRNA levels, as detected by Northern blotting, of vitamin D-metabolizing enzymes (1αOHase, 24OHase) in the kidneys of the mice. The kidneys were collected from the mice sacrificed on days 44 to 46 after tumor transplantation. \\n         FIG. 10  shows X-ray pictures showing the femora collected from the mice that were sacrificed on days 44 to 46 after tumor transplantation. \\n         FIG. 11A  includes graphs showing comparisons of serum phosphate levels and serum calcium levels, on day 2 after transplantation, among PBS, CHO ras clone-1 cells, and CHO-OST311H cells-transplanted nude mice (6-week-old, BALB/c, male). \\n       Each value is shown in terms of mean±standard deviation. \\n         FIG. 11B  includes graphs showing comparisons of serum phosphate levels and serum calcium levels, on day 6 after transplantation, among PBS, CHO ras clone-1 cells, and CHO-OST311H cells-transplanted nude mice (6-week-old, BALB/c, male). \\n       Each value is shown in terms of mean±standard deviation. \\n         FIG. 12  includes photographs showing the results obtained by purifying the culture supernatant of CHO-OST311H cells and subjecting the elution fractions to Western blotting using anti-His6 antibody(‘His6’ disclosed as SEQ ID NO: 88) and antiOST311 peptide antibody (311-114). The left panel shows the detection of 311:26-251, the center panel 311: 25-179, and the right panel 311: 180-251. Fractions shown with “*” (placed on the upper portion of the gel photographs) were used for a single dose examination conducted on normal mice. \\n         FIG. 13  includes photographs showing undemineralized slices stained with Villanueva-Goldner. The undemineralized sections were of the proximal metaphysis of tibia extracted from CHO-OST311H cells-transplanted mice and non tumor-bearing mice. \\n         FIG. 14  includes photographs showing the results of performing Northern blotting for vitamin D-metabolizing enzyme gene products in the kidneys excised from CHO-OST311H cells-transplanted mice and control mice. \\n         FIG. 15  A shows the time schedule of experiment 1 wherein CHO-producing recombinant OST311H full-length protein was administered to normal mice.  FIG. 15  B includes graphs showing serum phosphate levels at each time point of blood collection, and  FIG. 15  C includes graphs showing serum calcium levels at the same points in time. \\n         FIG. 16  A shows the time schedule of experiment 2 wherein CHO-producing recombinant OST311H full-length protein was administered to normal mice.  FIG. 16  B includes graphs showing serum phosphate levels at each time point of blood collection, and  FIG. 16  C includes graphs showing serum calcium levels at the same points in time. \\n         FIG. 17  is a photograph showing the recombinant protein detected in a culture supernatant when the culture supernatant of mutant recombinant OST311RQH-producing CHO-OST311RQH cells and OST311RQH/pEAK rapid cells were subjected to Western blotting. \\n         FIG. 18  A shows the time schedule of an experiment wherein mutant recombinant OST311RQH was administered to normal mice.  FIG. 18  B includes graphs showing serum phosphate levels at each time point of blood collection, and  FIG. 18  C includes graphs showing serum calcium levels at the same points in time. \\n         FIG. 19  A shows serum phosphate levels on day 2 after transplantation in a CHO-OST311RQH cells-transplanted experiment.  FIG. 19  B shows serum calcium levels in the same experiment. \\n         FIG. 20  includes photographs showing recombinant OST311H in serum-free culture supernatant of CHO-OST311H cells detected by Western blotting using anti-OST311 partial peptide rabbit anti-serum. ‘His 6’ disclosed as SEQ ID NO: 88. \\n         FIG. 21  A is a table showing recombinant OST311-detected levels in each combination of 6 types of anti-OST311 peptide polyclonal antibodies, when sandwich ELISA was constructed using these polyclonal antibodies. \\n         FIG. 21  B is a graph of plotting the relations between concentrations of purified recombinant OST311H and measured values corresponding thereto that were obtained using ELISA system combined with the use of 311-48 antibody or 311-180 antibody as an immobilized antibody, and 311-148 antibody as an antibody for detection. \\n         FIG. 22  A shows the expression of renal NaPi-7 analyzed by Western blotting at 1, 3 and 8 hours after administration of recombinant OST311 protein or vehicle to mice.  FIG. 22  B shows the expression of NaPi-7 analyzed by Northern blotting using total RNA of kidney following the similar treatment. \\n         FIG. 23  shows the changes in serum 1,25-dihydroxyvitamin D3 levels at 1, 3, and 8 hours after administration of recombinant OST311 protein or a vehicle to mice. \\n         FIG. 24  shows the expression of 25-hydroxyvitamin D-1-α-hydroxylase (1αOHase) or 25-hydroxyvitamin D-24-hydroxylase (24OHase) gene analyzed by Northern blotting using total RNA of kidney at 1, 3 and 8 hours after administration of recombinant OST311 protein or a vehicle to mice. \\n         FIG. 25  shows mean serum phosphate levels in each group, when the mean blood serum phosphate level on day 3 after cell transplantation in a CHO-ras clone-1 cells-transplanted group is considered as 100%. \\n         FIG. 26  shows the nucleotide sequence (SEQ ID NO: 90)and amino acid sequence (SEQ ID NO: 91)of recombinant His-OST311 encoded by plasmid OST311/pET28, and the DNA sequence (SEQ ID NO: 92)and amino acid sequence (SEQ ID NO: 93)of recombinant MK-OST311 encoded by plasmid pET22-MK-OST311. \\n         FIG. 27  shows an elution pattern when recombinant refolded His-OST311 was subjected to HPLC purification using cation-exchange column SP-5PW (TOSOH, Japan). \\n         FIG. 28  shows an elution pattern when recombinant refolded MK-OST311 was subjected to HPLC purification using cation-exchange column SP-5PW (TOSOH, Japan). \\n         FIG. 29  shows an elution pattern when PEGylated recombinant MK-OST311 was subjected to HPLC purification using cation-exchange column SP-5PW (TOSOH, Japan). \\n         FIG. 30  includes graphs showing serum phosphate levels at 8 or 9 hours after single administration of (A)  Escherichia coli -producing His-OST311 recombinant or (B) PEGylated MK-OST311 recombinant. \\n         FIG. 31  shows the results of analysis by the Northern blotting method on changes in the expression of vitamin D-metabolizing enzyme gene in the kidney at 1 and 4 hours after single administration of  Escherichia coli -producing His-OST311 recombinant. \\n         FIG. 32  shows changes in serum 1,25-dihydroxyvitamin D3 levels at 1, 4 and 9 hours after single administration of  Escherichia coli -producing His-OST311 recombinant. \\n         FIG. 33  shows a mutant OST311 recombinant detected by Western blotting, when mutation was introduced into amino acid Nos. 174 to 180 of OST311, and then the gene was expressed in pEAK cells so as to secrete the mutant OST311 recombinant in the cell supernatant. \\n     \\n    \\n    \\n     BEST MODE FOR CARRYING OUT THE INVENTION \\n     The present invention will be described more specifically by the following examples. These examples are not intended to limit the scope of the present invention. \\n     EXAMPLE 1 \\n     Construction of Human Tumor-induced Osteomalacia-Derived Tumor cDNA Library \\n     Tumor tissues frozen by liquid nitrogen were homogenized in 5 ml of ISOGEN (NIPPON GENE, Japan) solvent, and then approximately 0.13 mg of total RNA was prepared according to the attached manufacturer&#39;s manual. cDNA was synthesized from 1.5 μl of the total RNA using a SMART cDNA library preparation kit (CLONTECH, USA) according to the attached manufacturer&#39;s manual. Hereinafter, this cDNA is denoted as cDNA#2. EcoR I adapter was ligated to this cDNA#2, and then inserted into ?ZAPII phage vector (STRATAGENE, USA) that had been previously digested with a restriction enzyme EcoR I. Using a Gigapack III Gold phage packaging kit (STRATAGENE, USA), a tumor-induced osteomalacia tumor phage library was constructed according to the attached manufacturer&#39;s manual. The obtained library contained a total of approximately 600,000 independent clones. Further, the above phage library was allowed to infect  Escherichia coli  strain XLI-Blue MRF′, and then poured onto 20 petri dishes (15 cm). The petri dishes were incubated at 37° C. for 10 hours for plaque formation. All the plaques were extracted into an SM buffer, so that tumor-induced osteomalacia tumor cDNA phage library was constructed. \\n     EXAMPLE 2 \\n     Implementation for Positive Screening of Tumor cDNA Outline \\n     The fact that tumor-induced osteomalacia is curable by surgically excising a tumor suggests a possibility of high and specific expression of the causative gene in tumors. In addition, it has been reported so far that tumor-induced osteomalacia tumors are often constituted of mesoblastic, in particular mesenchymal cells. Therefore, it is necessary to identify a gene group that expressed lowly in normal mesoblast-derived tissues, but expressed specifically and highly only in tumor tissues. Hence, as described below, positive screening to which a cDNA subtraction technique was applied was performed. Subtraction of tumor tissue-derived cDNA and cDNA isolated from a bone tissue as a control was performed, so as to enrich a gene group that specifically and highly expressed only in tumor tissues, but not expressed in bone tissues. Hybridization was performed for tumor cDNA phage library using the subtracted cDNA group as a probe, thereby obtaining gene fragments specifically expressed in tumors. \\n     (1) Construction of Control Human Bone Tissue cDNA \\n     Human bone tissues frozen by liquid nitrogen were homogenized in 5 ml of ISOGEN(NIPPON GENE, Japan) solvent, and then approximately 0.011 mg of total RNA was prepared according to the attached manufacturer&#39;s manual. cDNA was synthesized from 3 μl of the total RNA using SMART cDNA library preparation kit (CLONTECH, USA) according to the attached manufacturer&#39;s manual. Hereinafter, the thus obtained cDNA is referred to as cDNA #4. \\n     (2) Subtraction of Tumor-Induced Osteomalacia Tumor cDNA and Control Bone Tissue cDNA \\n     To enrich a gene highly expressed in cDNA#2 described in Example 1, hybridization of cDNA#2 and cDNA#4 described in Example 2(1) was performed using a PCR-Select cDNA subtraction kit (CLONTECH, USA) according to the attached manufacturer&#39;s manual, thereby subtracting gene fragments contained in cDNA#4 from cDNA#2. Then, the subtracted cDNA#2 was amplified by PCR according to the attached manufacturer&#39;s manual, so that a subtracted cDNA group (A) was obtained. \\n     On the other hand, because of characteristics of the subtraction kit, such that its hybridization process is performed only twice, which is less than that of common techniques, it is difficult by this kit to completely subtract genes existing in many numbers in both subjects. Accordingly, when hybridization of tumor-induced osteomalacia tumor cDNA library with only subtracted cDNA group (A) as a probe, is performed, genes that cannot be subtracted completely are also obtained as positive clones. Thus, cDNA#2 was subtracted from cDNA#4 described in Example 2 (1) by the same method, and then the subtracted cDNA#4 was amplified by PCR, thereby preparing a subtracted cDNA group (B) as a control probe. Hybridization of tumor cDNA library respectively with the subtracted cDNA group (B) and the previously described subtracted cDNA group (A), and comparison of both signals make it possible to isolate gene fragments specifically contained in tumor-induced osteomalacia tumors. \\n     (3) Differential Hybridization of Tumor-Induced Osteomalacia Tumor cDNA Library \\n     After infecting  Escherichia coli  strain XLI-Blue with the tumor-induced osteomalacia tumor cDNA phage library described in Example 1, the infected  E. coli  was inoculated again so as to form 3,000 plaques per petri dish (15 cm), and then incubated at 37° C. for 8 hours. Then, plaques on each petri dish were transferred to two Hybond N+ (Amersham Pharmacia Biotech, USA) nylon filters. The nylon filters to which plaques are transferred were subjected to DNA immobilization treatment according to the attached manufacturer&#39;s manual, and then screening was performed using the subtracted cDNA(A) described in Example 2(2) and the subtracted cDNA(B) as probes, respectively. \\n     Probe labeling, hybridization and signal detection were performed using Alphos Direct system (Amersham Pharmacia Biotech, USA) according to the attached manufacturer&#39;s manual. The subtracted cDNA(A) and the subtracted cDNA(B) described in Example 2(2) were used as probes at 100 ng each, and then the probes were labeled with fluorescence according to protocols. The probes were respectively added to 50 ml of hybridization buffer supplied with Alphos Direct system, and at the same time, 2 sets of each of the 8 nylon filters which plaques had been transferred to were respectively hybridized and washed according to protocols. After washing, the nylon filters were subjected to luminescence reaction, exposed to ECL film (Amersham Pharmacia Biotech, USA) for 2 hours, developed with an automatic processor (FUJI FILM, Japan), and then the results were analyzed. \\n     As a result, independent plaques placed in a portion which were strongly burnt after exposure when the subtracted cDNA (A) was used as a probe, but not burnt when the subtracted cDNA (B) was used as a probe were visually selected, scraped off from the petri dish, and then suspended in 0.5 ml SM buffer. The suspensions were allowed to stand at 4° C. for 2 hours or more, thereby extracting phages. \\n     (4) Nucleotide Sequence Analysis of Positive Clone \\n     Using 0.5 μl of the phage solution obtained in Example 2(3) containing positive clones as a template, T7 primer (TAATACGACTCACTATAGGG) (SEQ ID NO: 24) and T3 primer (ATTAACCCTCACTAAAGGGA) (SEQ ID NO: 25) that were internal sequences of the phage vector, and LA-taq polymerase (TAKARA SHUZO, Japan), PCR was performed for 35 cycles, each cycle (process) consisting of 96° C. for 30 seconds, 55° C. for 30 seconds, and 72° C. for 30 seconds. The PCR products were subjected to 0.8% agarose gel electrophoresis. Clones for which a clear single band was observed were sequenced by ABI377 DNA sequencer (PE Applied systems, USA) using PCR amplification fragments as templates. \\n     When two clear bands were observed, PCR products were respectively extracted from each gel portion of the corresponding bands using QIAquick Gel Extraction kit (QIAGEN, Germany), and then sequenced by ABI377 DNA sequencer. \\n     As a result of differential hybridization for 341,000 plaques of tumor-induced osteomalacia phage library, 456 positive plaques were identified, and the nucleotide sequences of all of these plaques were determined. \\n     EXAMPLE 3 \\n     Narrowing Down the Candidate Genes of Human Hypophosphatemia Inducing Factor \\n     Homology search for the sequence information of 456 positive clones obtained in Example 2 against nucleotide sequences registered at Genbank, the nucleotide sequence database provided by NCBI was performed. As a result, a group of genes listed on Table 1 was obtained as existing genes that had appeared at high frequencies. Further, as a result of database search, there are 100 clones of unknown gene fragments of which biological activities were not known. As to nucleotide sequence information of these unknown gene fragments, overlaps among clones was further extracted as frequency information. Among them, the most frequently overlapping gene, OST311 sequence consisting of 7 clones sequentially forming one contig, was obtained. The nucleotide sequence obtained at this time ranged from nucleotide Nos. 1522 to 2770 of SEQ ID NO: 1. When a search for the gene fragment of OST311 was performed over the existing database, it was not registered as cDNA or EST, and corresponded only with a genomic sequence. The relevant genomic sequence is AC008012, and it has already been reported to be located at 12p13 on the chromosome. However, a region encoding a protein (ORF) was not found within the obtained sequence, so that the sequence was predicted to correspond to a 3′-untranslated region. Hence, cloning of full-length cDNA from tumor-induced osteomalacia tumor cDNA library was performed. In addition, the ORF-predicting function of DNASIS-Mac version 3.7 was used to predict ORF. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 1 \\n               \\n               \\n                   \\n               \\n               \\n                 Clone ID \\n                 Frequency \\n                 Description \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 OST 131 \\n                 236 \\n                 Dentin matrix protein-1(DMP1) \\n               \\n               \\n                 OST 1 \\n                 35 \\n                 Heat shock protein-90 (HSP90) \\n               \\n               \\n                 OST 2 \\n                 13 \\n                 Osteopontin \\n               \\n               \\n                 OST 311 \\n                 7 \\n                 Unknown/genomic DNA 12p13 \\n               \\n               \\n                 OST 1001 \\n                 4 \\n                 CD44 antigen \\n               \\n               \\n                 OST 584 \\n                 3 \\n                 Fibronectin \\n               \\n               \\n                 OST 666 \\n                 3 \\n                 Translational regulatory tumor protein \\n               \\n               \\n                 OST 133 \\n                 2 \\n                 Beta 2 microglobulin \\n               \\n               \\n                 OST 837 \\n                 2 \\n                 Fibroblast growth factor (FGF) \\n               \\n               \\n                 OST 562 \\n                 2 \\n                 Annexin H/lipocortin II \\n               \\n               \\n                 OST 1002 \\n                 2 \\n                 Cytochrome c oxygenase subunit 2 \\n               \\n               \\n                 OST 1003 \\n                 2 \\n                 Stathmin \\n               \\n               \\n                 OST 1004 \\n                 2 \\n                 Unknown \\n               \\n               \\n                 OST 903 \\n                 2 \\n                 Unknown \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 4 \\n     Cloning of Full-Length OST311 \\n     Based on the OST311 sequence obtained in Example 3, the following primers were synthesized. Then, PCR was performed, using a phage solution of tumor-induced osteomalacia tumor cDNA library as a template, for 35 cycles, each cycle (process) consisting of 96° C. for 30 seconds, 55° C. for 30 seconds, and 72° C. for 30 seconds. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 311-U65: \\n                 TTCTGTCTCGCTGTCTCCC \\n                 (SEQ\\u2003ID\\u2003NO:\\u200312) \\n               \\n               \\n                   \\n               \\n               \\n                 311-L344: \\n                 CCCCTTCCCAGTCACATTT \\n                 (SEQ\\u2003ID\\u2003NO:\\u200313) \\n               \\n            \\n           \\n         \\n       \\n     \\n     PCR products were subjected to 2% agarose gel electrophoresis, amplification of PCR products of predicted sizes was confirmed, and then the PCR products were purified using MicroSpin column S-300 HR (Amersham Pharmacia Biotech, USA). The resulting PCR products were fluorescence-labeled using Alphos Direct system (Amersham Pharmacia Biotech, USA) according to the attached manufacturer&#39;s manual. Then, plaque hybridization for 20,000 clones of tumor-induced osteomalacia tumor cDNA library was performed using these labeled products as probes. \\n     The thus obtained 40 positive clones were amplified by PCR in the manner as described in Example 2(4) using T7 and T3. Based on the nucleotide sequence of the resulting PCR product, a primer 311-L296 (SEQ ID NO: 14, GGGGCATCTAACATAAATGC) was synthesized. Again, plaque hybridization for 20,000 clones of tumor-induced osteomalacia tumor cDNA library was performed using as probes the PCR products amplified using 311-U65 (SEQ ID NO: 12) and 311-L344 (SEQ ID NO: 13) primers. For 62 positive clones, the nucleotide sequence of a PCR product amplified using T7 and 311-L296 (SEQ ID NO: 14) primers was determined. The determined sequence was linked to the nucleotide sequences that had been determined so far. Thus, the nucleotide sequence represented by SEQ ID NO: 1 was obtained. It became clear that ORF of OST311 starts from an initiation codon located at nucleotide No. 133 of SEQ ID NO: 1. Furthermore, the following primers were synthesized to finally determine the sequence of ORF. \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 311-F1: \\n                   \\n               \\n               \\n                 AGCCACTCAGAGCAGGGCAC \\n                 (SEQ\\u2003ID\\u2003NO:\\u200315, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003112\\u2003to\\u2003131) \\n               \\n               \\n                   \\n               \\n               \\n                 311-F2: \\n                   \\n               \\n               \\n                 GGTGGCGGCCGTCTAGAACTA \\n                 (SEQ\\u2003ID\\u2003NO:\\u200316, \\n               \\n               \\n                   \\n                 Vector\\u2003sequence) \\n               \\n               \\n                   \\n               \\n               \\n                 311-F3: \\n                   \\n               \\n               \\n                 TCAGTCTGGGCCGGGCGAAGA \\n                 (SEQ\\u2003ID\\u2003NO:\\u200317, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003539\\u2003to\\u2003559) \\n               \\n               \\n                   \\n               \\n               \\n                 311-L1: \\n                   \\n               \\n               \\n                 CACGTTCAAGGGGTCCCGCT \\n                 (SEQ\\u2003ID\\u2003NO:\\u200318, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003689\\u2003to\\u2003708) \\n               \\n               \\n                   \\n               \\n               \\n                 311-L3: \\n                   \\n               \\n               \\n                 TCTGAAATCCATGCAGAGGT \\n                 (SEQ\\u2003ID\\u2003NO:\\u200319, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003410\\u2003to\\u2003429) \\n               \\n               \\n                   \\n               \\n               \\n                 311-L5: \\n                   \\n               \\n               \\n                 GGGAGGCATTGGGATAGGCTC \\n                 (SEQ\\u2003ID\\u2003NO:\\u200320, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003200\\u2003to\\u2003220) \\n               \\n               \\n                   \\n               \\n               \\n                 311-L6: \\n                   \\n               \\n               \\n                 CTAGATGAACTTGGCGAAGGG \\n                 (SEQ\\u2003ID\\u2003NO:\\u200321, \\n               \\n               \\n                   \\n                 Nucleotide\\u2003Nos.\\u2003868\\u2003to\\u2003888) \\n               \\n            \\n           \\n         \\n       \\n     \\n     Using 311-F2 (SEQ ID NO: 16) and 311-L6 (SEQ ID NO: 21) primers, tumor-induced osteomalacia tumor cDNA library as a template, and Pyrobest DNA polymerase (TAKARA SHUZO, Japan), PCR was performed for 35 cycles, each cycle (process) consisting of 96° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 30 seconds. \\n     When the PCR products were subjected to 2% agarose gel electrophoresis, a single fragment of approximately 980 nucleotide pairs was confirmed. Then, the nucleotide sequence of the amplified fragment was determined using the above primers (SEQ ID NOS. 15 to 21). The thus determined ORF region (SEQ ID NO: 1) encoding the polypeptide represented by SEQ ID NO: 2 was placed between the initiation codon ATG located at nucleotide No. 133 and the termination codon TAG located at nucleotide No. 886 of SEQ ID NO: 1. \\n     EXAMPLE 5 \\n     Specificity of OST311 Against Tumor-induced Osteomalacia Tumor \\n     To study the tumor specificity of OST311, PCR was performed for 35 cycles using as templates first-strand cDNAs extracted from tumor tissues and from control bone tissues, and using OST311 specific primers shown below (SEQ ID NOS: 22 and 23). Each PCR cycle is a process consisting of 96° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 30 seconds. In addition, DMSO was added into both reaction solutions to have a final concentration of 2%, and LA-taq DNA polymerase (TAKARA SHUZO, Japan) was used as an enzyme. Further, as an internal standard, PCR was performed under similar conditions using primers specific to \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 G3PDH \\n                   \\n                   \\n                   \\n               \\n               \\n                 (FW: \\n                 ACCACAGTCCATGCCATCAC, \\n                 (SEQ\\u2003ID\\u2003NO:\\u200326) \\n               \\n               \\n                   \\n               \\n               \\n                 RV: \\n                 TCCACCACCCTGTTGCTGTA. \\n                 (SEQ\\u2003ID\\u2003NO:\\u200327)) \\n               \\n               \\n                   \\n               \\n               \\n                 311F1EcoRI: \\n                 CCGGAATTCAGCCACTCAGAGCAGGGCACG \\n                 (SEQ\\u2003ID\\u2003NO:\\u200322) \\n               \\n               \\n                   \\n               \\n               \\n                 311LHisNot: \\n                 ATAAGAATGCGGCCGCTCAATGGTGATGGTGATGATGGATGAACTTGGCGAA \\n                 (SEQ\\u2003ID\\u2003NO:\\u200323) \\n               \\n            \\n           \\n         \\n       \\n     \\n     As shown in  FIG. 1 , these PCR products were subjected to 2% agarose gel electrophoresis. When OST311 primers were used, PCR products with the predicted size were observed only when the tumor tissue was used as a template. In contrast, when G3PDH primers were used, PCR products with the predicted size were observed at the same level in both cases of the tumor tissues and control bone tissues. From these results, tumor tissue-specific expression of OST311 was confirmed. \\n     EXAMPLE 6 \\n     Isolation of the CHO Cells Stably Expressing OST311 \\n     (1) Construction of OST311 Expression Vector \\n     Using 311F1EcoRI (SEQ ID NO: 22) and 311LHisNot (SEQ ID NO: 23) primers shown in Example 5, and tumor-induced osteomalacia tumor cDNA library as a template, PCR was performed for 35 cycles, each cycle (process) consisting of 96° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 30 seconds. In addition, DMSO was added to the reaction solution to have a final concentration of 2%, and LA-taq DNA polymerase (TAKARA SHUZO, Japan) was used as an enzyme. 311F1EcoRI primer was annealed to nucleotide No. 111 of SEQ ID NO: 1 located upstream from Kozak sequence, and 311LHisNot primer was annealed to nucleotide No. 871 of SEQ ID NO: 1. A region encoding the full-length polypeptide represented by SEQ ID NO: 2 can be amplified by performing PCR using both primers. In addition, 311LHisNot primer contains a nucleotide sequence that adds six histidine residues (SEQ ID NO: 88)after the amino acid No. 251 of SEQ ID NO: 2 and also adds a termination codon after the last histidine codon. Thus, the translated recombinant protein has a His6 tag sequence (SEQ ID NO: 88) at the C-terminus, so that it is useful for recognition of a recombinant by an antibody, and purification of the recombinant using nickel resin. \\n     After being digested with restriction enzymes EcoR I and Not I, the PCR product was ligated to plasmid vector pcDNA3.1Zeo (INVITROGEN, USA) for expression in animal cells that had been digested with EcoR I and Not I similarly. The thus obtained recombinant vector was introduced into  Escherichia coli  strain DH5α.  E. coli  was cultured in 3 ml of LB medium containing 100 mg/ml ampicillin, and then the plasmid was purified using GFX plasmid purification kit (Amersham Pharmacia Biotech, USA). The sequence of the inserted gene was determined by a standard method. Thus, it was confirmed that the sequence was identical to an equivalent portion of SEQ ID NO: 1, and a nucleotide sequence encoding the His6 tag sequence (SEQ ID NO: 88) had been added immediately before the termination codon. \\n     (2) Isolation of the CHO Cells which Stably Express OST311 \\n     Approximately 20 pg of the plasmid, to which OST311 ORF portion prepared in Example 6 (1) had been inserted, was digested with a restriction enzyme Fsp I so as to cleave a site of the ampicillin resistance gene within the vector. Then the cleaved vector was subjected to ethanol precipitation, and then dissolved in 10 μl of ultrapure water. Subsequently, the total volume of the solution was introduced into host cells by an electroporation using Gene Pulser II (Bio Rad, USA). CHO Ras clone-1 cells (Shirahata, S., Biosci. Biotech. Biochem, 59(2): 345-347, 1995) were used as host cells. CHO Ras clone-1 was cultured in a 75 cm 2  culture flask (CORNING, USA) containing MEMα medium supplemented with 10% FCS at 37° C. under 5% CO 2  and 100% humidity until the cells grew to cover approximately 90% of culturing area. Then, adherent cells were removed by trypsin treatment, so that approximately 1×10 7  cells were obtained. The obtained cells were resuspended in 0.8 ml of PBS, mixed with the plasmid digested with Fsp I, and then cooled on ice for 10 minutes. The cells containing the plasmid were transferred into the cuvette four millimeter in width. After electric pulse was applied at set values (0.25 kV and 975 μF), the cuvette was cooled again for 10 minutes. The gene-transferred cells were cultured in 10% FCS-containing MEMα medium for 24 hours. Then, Zeosin (INVITROGEN, USA) was added to the culture medium to have a final concentration of 0.5 mg/ml, and then further cultured for 1 week. Subsequently, for cloning cells showing drug resistance ability, the cells were re-inoculated in a 96-well plate (CORNING, USA) to 0.2 cells/well by a limited dilution method, and then cultured in the presence of zeosin with a final concentration of 0.3 mg/ml for about 3 weeks, thereby obtaining 35 clones of the drug-resistant strain. \\n     (3) Confirmation of Production of Recombinant by CHO Cells which Stably Express OST311 \\n     For 35 clones showing drug resistance, the presence of recombinant OST311 in the conditioned medium was confirmed by the Western blotting method. \\n     0.2 ml of collected the conditioned medium was concentrated to about 40 to 50 μl using an ultra-free MC M.W. 5,000 cut membrane system (MILLIPORE, USA). 10 μl of a sample buffer containing 1 M Tris-Cl pH6.8, 5% SDS, 50% glycerol, and 100 mM DTT was added to the concentrate, and then it was heated at 95° C. for 5 minutes. Then the protein in the conditioned medium was separated by polyacrylamide electrophoresis with a 10 to 20% gradient. Thereafter, the protein in the gel was transferred to Immobilon PVDF membrane (MILLIPORE, USA) using a semi-dry blotting system (Owl Separation Systems, USA). This PVDF membrane was incubated with anti-His (C-terminus) antibody (INVITROGEN, USA) that had been diluted 1/5000 in TTBS buffer (Sigma, USA) at room temperature for 1 hour. Then, the membrane was exposed to film for 5 minutes using ECL system (Amersham Pharmacia Biotech, USA), and developed using an automatic processor (FUJIFILM, Japan). As a result, clone #20, for which the most intense signals had been observed at approximately 32 kDa and 10 kDa, was isolated. Hereinafter, #20 cell was named CHO-OST311H, and deposited with National Institute of Advanced Industrial Science and Technology, International Patent Organism Depositary (Higashi, Tsukuba-shi, Ibaraki 1-1-1) (Accession No. FERM BP-7273). \\n     EXAMPLE 7 \\n     Measurement of Inhibitory Activity of Phosphate Uptake by CHO-OST311H Conditioned Medium \\n     CHO-OST311H was cultured in a 225 cm 2  culture flask (CORNING, USA) containing MEM α medium supplemented with 10% FCS at 37° C. and under 5% CO 2 , and 100% humidity until the cells grew to cover about 80% of the flask area. Then, the medium was replaced with 30 ml of serum-free medium CHO-S-SFM II (LIFE TECHNOLOGY, USA). 48 hours later, the conditioned medium was collected. The conditioned medium was centrifuged at 1,200 g for 5 minutes to remove suspended cells and the like, and then filtered using a Minisart-plus 0.22 μm filter (Sartorius, Germany). \\n     Using this conditioned medium, the effect on phosphate uptake activity of a human renal proximal tubular cell line (CL-8 cells) was examined. The human renal proximal tubular cell line was cultured under 5% CO 2  and 100% humidity at 37° C. in DMEM medium containing 10% FCS (LIFE TECHNOLOGY). To measure phosphate uptake activity, the human renal proximal tubular cell line was first cultured in DMEM medium containing 10% FCS in a 48-well plate (CORNING, USA). When the cells grew to cover the entire bottom surface of the plate at 3 days after the start of culturing, the culture medium was replaced with 200 μl of serum-free medium CHO-S-SFM II (LIFE TECHNOLOGY, USA), followed by further culturing for 20 to 24 hours. Using the cells in this state, the following experiments for measuring phosphate uptake activity (Experiments 1 and 2) were conducted. \\n     (1) EXPERIMENT 1 \\n     CHO-S-SFM II medium was removed, and then 200 μl of the above conditioned medium of CHO-OST311H cells prepared in CHO-S-SFM II medium was added per well. At this time as control wells, 3 wells containing media which had not been replaced with CHO-S-SFM II, and 3 wells containing media which had been supplemented respectively with 200 μl of the conditioned medium of CHO-OST190H cells prepared in a manner similar to that for CHO-OST311H were prepared. The above-described CHO-OST190H cells are recombinant cells that have been prepared by introducing OST190H, which we have cloned in a manner similar to that for CHO-OST311H cells, into CHO ras clone-1 such that OST190H can be expressed. Similar to OST311H, expressed CHO-OST190H contains a His6 tag sequence (SEQ ID NO: 88)added to the C-terminus of a polypeptide, which is same as the polypeptide named MEPE as reported by Rowe, P. S. N. et al, Genomics 67:54-68, 2000. After each sample was added, incubation was further performed in a CO 2 -incubator for 26 hours, phosphate uptake activity of the cells in each well was measured by the following method for measuring phosphate uptake activity. \\n     (2) EXPERIMENT 2 \\n     100 μl of cultured medium was removed from 200 μl of the culturing CHO-S-SFM II medium. Into these culture, 100 μl of the conditioned medium of CHO ras clone-1 cells was added respectively to 3 wells, and 100 μl of the conditioned medium of CHO-OST311H cells was added respectively to 3 wells. Then, incubation was performed in a CO 2  incubator for 24 hours. Subsequently, phosphate uptake activity of the cells in each well was measured by the following method for measuring phosphate transport. \\n     Measurement method of phosphate uptake activity: \\n     After the addition of the conditioned medium and incubation, the cells were washed with a buffer containing no phosphoric acid (150 mM NaCl, 1 mM CaCl 2 , 1.8 mM MgSO 4 , 10 mM HEPES, pH7.4), and then incubated in the same solution at room temperature for 10 minutes. The solution was removed, and then an assay solution, which had been prepared by adding radioactive KH 2 PO 4  (NEN) to have 0.105 mM to the buffer, was added. The solution was subjected to incubation at room temperature for 10 minutes. After incubation, the cells were washed three times with a stop solution which had been ice-cooled immediately after the removal of the assay solution (150 mM choline chloride, 1 mM CaCl 2 , 1.8 mM MgSO 4 , 10 mM HEPES, pH7.4). This washing solution was removed, 80 μl of 0.2 N NaOH was added to the cells, and then the solution was incubated at room temperature for 10 minutes, so that the cells were lysed. To measure radioactivity in the cell lysis solution, the solution was transferred to ReadyCap (Beckman), dried at 50° C., and then placed in glass vial. Then, radioactivity was measured using a scintillation counter (Wallac1410, Pharmacia). Phosphate uptake activity in each experiment is shown in Table 2 wherein mean uptake in the control not supplemented with conditioned medium is considered as 100%. The conditioned medium of CHO-OST311H cells significantly suppressed phosphate uptake activity of the human epithelial cells of renal proximal convoluted tubules. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 2 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 OST311 activity against phosphate uptake \\n               \\n               \\n                 by renal tubular epithelial cells \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 Phosphate uptake \\n                   \\n               \\n               \\n                   \\n                 activity ± SEM \\n                 t-test \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Experiment 1 \\n                   \\n                   \\n               \\n               \\n                 Sample \\n               \\n               \\n                 Not supplemented \\n                 \\u2009100 ± 1.9 \\n                 — \\n               \\n               \\n                 CHO-OST190H \\n                 103.8 ± 0.9\\u2002 \\n                 Not significant \\n               \\n               \\n                 CHO-OST311H \\n                 87.4 ± 0.2 \\n                 p &lt; 0.01 \\n               \\n               \\n                 Experiment 2 \\n               \\n               \\n                 Conditioned media of CHO \\n                 \\u2009100 ± 1.5 \\n                 — \\n               \\n               \\n                 CHO-OST311H \\n                 87.2 ± 1.2 \\n                 p &lt; 0.01 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 8 \\n     Partial Purification Of Recombinant Ost311 From Cho-Ost311H Conditioned medium Recombinant OST was partially purified from the conditioned medium prepared in the manner described in Example 7 by the following method. Processes 1) to 4) were performed in a chromatochamber at 4° C. \\n     1) A disposable polypropylene column was filled with ProBond nickel resin (INVITROGEN, USA) to have a bed volume of 3 ml, and then the column was washed and equilibrated with 30 ml of buffer 1 (Table 3). \\n     2) 120 ml of the conditioned medium prepared in the manner described in Example 7 was applied to the above nickel column by free-fall, allowing recombinant OST311 to bind. \\n     3) 30 ml of buffer 2 shown in Table 3 was used to remove non-specifically adsorbed proteins. \\n     4) 3 ml of buffer 3 shown in Table 3 was respectively added at four separate times, so that recombinant OST311 was eluted. 20 μl each of these four fractions was subjected directly (as original concentration) to Western blotting in the manner described in Example 6(3). Thus, detection of OST311 was attempted with anti-His antibody. \\n     As a result, approximately 32 kDa and 10 kDa signals were intensively observed in the second fraction. \\n     5) The above second fraction was applied to NAP25 and NAP10 columns (Amersham Pharmacia Biotech, USA), and the solvent was replaced with buffer 4 shown in Table 3. \\n     6) Recombinant OST311 for which the solvent had been replaced with buffer 4 was applied to SP-5PW (strong cation-exchange resin, TOSOH, Japan) at a flow rate of 1 ml/minute using high performance liquid chromatography (Hitachi, Japan). Buffer 5 shown in Table 3 was added with a 1%/minute gradient for elution, thereby sampling 2 ml each of fractions. As shown in  FIG. 2 , Western blotting was performed for each elution fraction in the manner described in Example 8(5), so that detection of OST311 was attempted. Approximately 10 kDa signal was eluted with about 280 mM NaCl, and approximately 32 kDa signal was eluted with about 400 mM NaCl. The corresponding fractions were subjected to SDS-polyacrylamide gel electrophoresis, and then stained using a silver staining kit (Daiichi Chemicals, Japan). The purity of the fraction containing approximately 10 kDa and 32 kDa was 70% or more. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 3 \\n               \\n               \\n                   \\n               \\n               \\n                 Buffer 1 \\n                 Buffer 2 \\n                 Buffer 3 \\n                 Buffer 4 \\n                 Buffer 5 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 10 mM Na/Pi \\n                 10 mM Na/Pi \\n                 10 mM Na/Pi \\n                 10 mM Na/Pi \\n                 10 mM Na/Pi \\n               \\n               \\n                 pH 6.5 \\n                 pH 6.5 \\n                 pH 6.5 \\n                 pH 6.5 \\n                 pH 6.5 \\n               \\n               \\n                 0.5 M NaCl \\n                 10 mM \\n                 0.5 M \\n                 5 mM CHAPS \\n                 1 M NaCl \\n               \\n               \\n                   \\n                 imidazole \\n                 imidazole \\n                   \\n                 5 mM CHAPS \\n               \\n               \\n                   \\n                 0.5 M NaCl \\n                 0.5 M NaCl \\n               \\n               \\n                   \\n                   \\n                 5 mM CHAPS \\n               \\n               \\n                   \\n               \\n               \\n                 Na/Pi: sodium phosphate buffer \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 9 \\n     N-terminal Amino Acid Sequence Analysis of Partially Purified Recombinant \\n     Approximately 10 kDa and 32 kDa of partially purified fractions recognized by anti-His antibody that had been obtained by the method described in Example 8 were subjected to SDS-polyacrylamide gel electrophoresis. Next, using a semi-dry blotting system (Owl Separation Systems, USA), protein in the gel was transferred to an Immobilon PVDF membrane (MILLIPORE, USA). The PVDF membrane was stained with CBB, approximately 10 kDa and 32 kDa bands were excised, and then the N-terminal amino acid sequences were determined using a protein sequencer Model 492 (PE Applied Systems, USA). \\n     As a result, it became clear that the N-terminal amino acid sequence of approximately 35 kDa band was an OST311 sequence starting from residue No. 25, Tyr, of SEQ ID NO: 2. From this result, it was confirmed that a sequence ranging from residue No. 1, Met, to residue No. 24, Ala, of SEQ ID NO: 2 had been cleaved as a secretion signal sequence. On the other hand, it became clear that the N-terminal amino acid sequence of approximately 10 kDa band was an OST311 sequence starting from residue No. 180, Ser, of SEQ ID NO: 2. The presence of a motif consisting of RRXXR immediately before Ser at residue No. 180 revealed that recombinant OST311 had been cleaved by some protease derived from CHO cells. \\n     As described above, recombinant OST311 produced by CHO-OST311 cells was shown to be present as at least 3 types of polypeptides after secretion: a polypeptide (SEQ ID NO: 4) from residue No. 25, Tyr, to No. 251, Ile, a polypeptide (SEQ ID NO: 6) from residue No. 25, Tyr, to No. 179, Arg, and a polypeptide (SEQ ID NO: 8) from residue No. 180, Ser, to No. 251, Ile of SEQ ID NO: 2. \\n     EXAMPLE 10 \\n     Preparation of Anti-OST311 Partial Peptide Polyclonal Antibody \\n     The degree of hydrophobicity of the polypeptide of SEQ ID NO: 2 was predicted using a computing function of MacVector version 6.5.1., so that sites suitable for antigen preparing peptide antibodies were predicted ( FIGS. 3A  and B). Here, the suitable site was predicted from the perspective that the sites have high degree of hydrophilicity, and are less subject to sugar chain modification and phosphorylation. Hence, selected and synthesized as antigens were 311-48 (SEQ ID NO: 28) prepared by artificially adding a cysteine residue at the synthetic stage to the C-terminus of a peptide consisting of 20 amino acids starting from residue No. 48, Arg, of SEQ ID NO: 2, and 311-114 (SEQ ID NO: 29) prepared by similarly adding a cysteine residue to a peptide consisting of 20 amino acids starting from residue No. 114, Arg. Specifically, cysteine residues were artificially added to the C-termini of both peptides at the synthetic stage, so that the products could be coupled with carrier proteins (bovine thyroglobulin). Coupling with carrier proteins and immunization of rabbits were consigned to IBL, Co., Ltd. (1091-1, Fujioka-shi, Gunma, Japan) (Assignment number: 1515). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 311-48: \\n                 RNSYHLQIHKNGHVDGAPHQC \\n                 (SEQ\\u2003ID\\u2003NO:\\u200328) \\n               \\n               \\n                   \\n               \\n               \\n                 311-114: \\n                 RFQHQTLENGYDVYHSPQYHC \\n                 (SEQ\\u2003ID\\u2003NO:\\u200329) \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 11 \\n     Experiment of Transplanting CHO-OST311H Cells into Nude Mice \\n     To test whether OST311 is a causative factor for tumor-induced osteomalacia CHO-OST311H cells were transplanted into 6-week-old BALB/c nude mice (male) for tumor generation, thereby developing a murine tumor-induced osteomalacia model which constantly secrets recombinant OST311 from the tumors. As a control for the experiment, CHO ras clone-1 cells and CHO-OST190H described in Example 7 were used similarly for the transplantation experiment. \\n     (1) Transplantation of CHO Cells \\n     CHO-OST311H cells and CHO-OST190 cells were scattered by trypsin treatment from culture flasks, suspended to 1×10 8  cells/ml in PBS. The suspension was subcutaneously injected, 0.1 ml each, to both latera of the nude mice (2×10 7  cells/mouse). In addition, as a control group, subcutaneous injection of the same number of CHO ras clone-1 cells were performed in the same manner. For about 1 month after injection, five nude mice were housed in a plastic cage and allowed access to solid food CE-2 (CLEA JAPAN, Japan) and tap water, ad libitum. 2 weeks after transplantation, tumor generation was observed for 75% of the control group and 66.7% of OST311 group. \\n     (2) Comparison of Changes in Body Weight \\n     Time-course changes in average body weight for 31 days after transplantation of CHO-OST311H cells were compared between a non tumor-bearing group (a line indicated with avr.− in the graph) and a CHO-OST311H cell tumor-bearing group (a indicated with avr.+ line in the graph). As shown in  FIG. 4 , the CHO-OST311H cell tumor-bearing group showed suppressed increases in body weight compared to the non tumor-bearing group, and there were clear significant differences between the two groups (24.1±1.5 g vs. 26.7±1.0 g, p&lt;0.001, day 31). In contrast, similar differences were not observed in average body weight between the CHO ras clone-1 cell tumor-bearing control group and the non tumor-bearing group (27.0±1.8 g vs. 26.7±1.0 g, no significant difference, day 31). \\n     (3) Measurement of Serum Phosphate and Calcium, and Urine Phosphate and Calcium \\n     On days 30 to 40 after cell transplantation, nude mice were housed in metabolic cages for 24 hours. After urine was collected, blood was collected from the heart or orbital cavity of the mice under the anesthetized condition using diethylethel. The peripheral blood was subjected to preparation of serum using Microtainer (Beckton Dickinson, USA). After the volume was measured, the urine was centrifuged to collect supernatant. Serum and urine phosphate levels were measured using P-test Wako (Wako Pure Chemical Industries, Japan), and serum and urine calcium levels were measured using calcium-test Wako (Wako Pure Chemical Industries, Japan), and serum and urine creatinine levels were measured using CRE-EN KAINOS (KAINOS, Japan). \\n     EXPERIMENT 1 \\n     On day 34 after cell transplantation, serum phosphate levels of the non tumor-bearing group, the CHO ras clone-1 cell tumor-bearing group, the CHO-OST190H cell tumor-bearing group, and the CHO-OST311H cell tumor-bearing group were measured. \\n     EXPERIMENT 2 \\n     On days 44 to 46 after cell transplantation, serum and urine phosphate levels, calcium levels and creatinine levels of the non tumor-bearing group and the CHO-OST311H cell tumor-bearing group were measured. Renal fractional excretion of phosphate and calcium were determined by dividing phosphate or calcium clearance by creatinine clearance. \\n     Measurement results are shown in Table 4 below. \\n                     TABLE 4               Serum and urine phosphate and calcium in cells-transplanted mice                  Experiment 1                                 Number of   Serum phosphate           Group   mouse   level ± SEM (mg/dl)   t-test               Non tumor-bearing   7   8.17 ± 0.60   —       mice       CHO ras clone-1   4   8.50 ± 0.38   No significant                   difference       CHO-OST190H   4   9.49 ± 0.52   No significant                   difference       CHO-OST311H   9   4.39 ± 0.23   p &lt; 0.001                         Experiment 2                             Group   Non tumor   CHO-OST311H   t-test               Number of mouse   4   6   —       Serum phosphate   8.29 ± 0.59   4.25 ± 0.15   p &lt; 0.001       level ± SEM       (mg/dl)       Renal fractional   0.23 ± 0.02   0.44 ± 0.06   p &lt; 0.05       excretion of       phosphate       Serum Ca   6.72 ± 0.27   4.61 ± 0.19   p &lt; 0.001       level ± SEM       (mg/dl)       Renal fractional   0.0040 ± 0.0006   0.0059 ± 0.0010   No significant       excretion of           difference       calcium                    \\n(4) Soft Roentgenogram of Whole Skeleton\\n \\n     After transplantation of the CHO-OST311H cells, individuals in which formation of tumors is recognized exhibited significant abnormalities in their physical constitutions and walking functions, compared to non tumor-bearing individuals or the control CHO ras clone-1-transplanted individuals. Hence, the tumor-bearing individuals were predicted to have skeletal abnormalities. Then, individuals in which formation of tumors is recognized were selected at random from the control CHO ras clone-1 cells-transplanted group, the CHO-OST190H cells-transplanted group and CHO-OST311H cells-transplanted group, and then X-ray pictures were taken therefor using a radiography system μFX-100 (FUJI FILM, Japan) according to the attached manufacturer&#39;s manual. X-ray pictures were taken under conditions of X-ray tube voltage of 25 kV, X-ray tube current of 0.1 mA and exposure time of 10 seconds. The individuals were exposed to a imaging plate, and then image analysis was performed using BAS2000 (FUJI FILM, Japan). \\n     As a result, as shown in  FIG. 5 , reduced brightness of soft roentgenogram of the bone was recognized in the whole skeleton of CHO-OST311H cells-transplanted individual, so that defect of mineralization was recognized. In addition, skeletal deformity, such as distortion of rib cage was also recognized. \\n     (5) Measurement of Serum Phosphate and Calcium Levels and Alkaline Phosphatase Activities \\n     The serum obtained by collecting blood from the heart on days 44 and 46 after cell transplantation was stored at −20° C. once. The serum samples were thawed together, phosphate and calcium levels contained in each serum were measured again, and alkaline phosphatase activities were also measured. Phosphate levels were measured using P-test Wako (Wako Pure Chemical Industries, Japan), calcium levels were measured using calcium-test, Wako Pure Chemical Industries, Japan), and alkaline phosphatase activities were measured using calcium alkaline phosphor B-test Wako (Wako Pure Chemical Industries, Japan). The results were classified into a non tumor-bearing group (n=6), a CHO tumor-bearing group (n=10), a CHO-OST190H cell tumor-bearing group (n=10) and a CHO-OST311H tumor group (n=6×2). The CHO-OST311H group was classified into two groups: a group sacrificed on day 44 (CHO-OST311H-1: n=6) and a group sacrificed on day 46 (CHO-OST311-2: n=6). As shown in  FIG. 7 , significant changes including decreased serum phosphate level ( FIG. 7A ), decreased serum calcium level ( FIG. 7B ) and increased serum alkaline phosphatase activity ( FIG. 7C ) were recognized in the CHO-OST311H tumor group. \\n     (6) Expression of Sodium-Phosphate Cotransporter (NaPi-7) on Renal Proximal Tubule \\n     i) Preparation of Brush Border Membranes (Hereinafter, Referred to as BBM) of Proximal Tubular Epithelial Cell \\n     Kidneys were excised from the CHO-OST311H tumor-bearing individuals and non tumor-bearing individuals under the anesthetized condition using diethylether. Each kidney was cut in half to obtain coronal sections (Experiment 1: 6 numbers of CHO-OST311H tumor-bearing individuals and 4 numbers of non tumor-bearing individuals. Experiment 2: 6 numbers of CHO-OST311H tumor-bearing individuals, and 2 numbers of non tumor-bearing individuals). Using half of each of the respectively excised kidneys from the individuals, BBM was prepared according to the protocols as reported by Kessler et al (Biochem. Biophys. Acta. 506, pp. 136-154). \\n     The kidney was homogenized in 3 ml of a homogenizing buffer (50 mM mannitol, 2 mM Tris/HEPES pH 7.5) using a glass-made homogenizer at 1,300 rpm for 2 minutes so that homogenous kidney extracts were obtained. After CaCl 2  was added to the extract to have a final concentration of 10 mM, the solution was agitated at 4° C. for 15 minutes, and then centrifuged at 4,900 g for 15 minutes at 4° C. The thus obtained supernatant was filtered using a Kimwipe, and then centrifuged at 16,200 g for 60 minutes at 4° C., thereby allowing fractions containing many BBM to precipitate. The precipitate was resuspended in 5 ml of a suspension buffer (50 mM mannitol, 2 mM Tris/HEPES, pH 7.5), and then the solution was centrifuged again at 16,200 g for 60 minutes at 4° C. This procedure was repeated twice, and then the product was resuspended in 0.1 ml of a suspension buffer. The protein concentration of the thus obtained solution was 3 to 4 mg/ml as determined by standard methods. \\n     ii) Western Blotting of BBM Protein \\n     As described above, BBM proteins prepared from each mouse were diluted separately with a suspension buffer to 10 μg/μl. Then, 2.5 μl of a sample buffer containing 1 M Tris-Cl pH 6.8, 5% SDS, 50% glycerol and 100 mM DTT was added to the diluted solution. After the solution was heated at 95° C. for 5 minutes, protein in BBM solution was separated by polyacrylamide electrophoresis with a 10 to 20% gradient. Subsequently, the protein in the gel was transferred to Immobilon PVDF membrane (MILLIPORE, USA) using a semi-dry blotting system (Owl Separation Systems, USA). The PVDF membrane was incubated with anti-NaPi-7 polyclonal antibody diluted 1/2000 in TTBS buffer (Sigma, USA) at room temperature for 3 hours. The antibody is a polyclonal antibody which has been obtained by immunizing a rabbit with a synthetic peptide (LALPAHHNATRL) (SEQ ID NO: 89)corresponding to the C-terminal site of mouse NaPi-7 by standard methods in KIRIN BREWERY CO., LTD., Pharmaceutical Research Laboratories, Pharmaceutical Division. After reaction with this antibody, the reaction product was further incubated with anti-rabbit IgG secondary antibody (DAKO, Denmark) bound to horseradish peroxidase (HRP), and then bands were detected using ECL system (Amersham Pharmacia Biotech, USA). \\n     Under reduction conditions, approximately 80 kDa and 35 kDa of bands and 170 to 200 kDa of high molecular smears were detected with the antibody ( FIG. 8 ). These band patterns are the same as the cases reported by Tatsumi et al in J. Biol. Chem. Vol. 273, pp 28568-28575, 1998, and these bands had been confirmed to uniformly change depending on the amount of phosphate intake from food by mice or rats. From these facts, the bands proved to the polypeptides derived from NaPi-7. As shown in  FIG. 8 , for all the above fragments (bands indicated with arrows), the NaPi-7 signals contained in BBM proteins that had been prepared from CHO-OST311H tumor-bearing individuals were significantly reduced compared to those prepared from non tumor-bearing individuals. These results were reproduced in individually conducted Experiment 1 and 2. On the other hand, these BBM proteins were separated by polyacrylamide electrophoresis with a 10 to 20% gradient, and then stained with CBB. BBM proteins of individuals were equally stained, suggesting that reduced signals in Western blotting are specifically observed for NaPi-7 ( FIG. 8 ). Based on this fact, it is inferred that OST311 protein acts on renal proximal tubular cells, and downregulates the expression level of NaPi-7 at the protein level, so as to induce hypophosphatemia. \\n     (7) Analysis of Changes in mRNAs of Phosphate Transporters and Vitamin D-Metabolizing Enzymes in the Kidney and Small Intestine \\n     i) Preparation of Total RNA \\n     Small intestines and kidneys were excised from the mice sacrificed on days 44 to 46 after cell transplantation. The kidneys were rapidly frozen in dry ice. The frozen kidneys were cryopreserved in a deep freezer at −80° C. until use. One frozen kidney was homogenized in 5 ml of ISOGEN (Nippon Gene, Japan), and then total RNA was prepared according to the attached manufacturer&#39;s manual. 15 μg of the prepared total RNA was electrophoresed by formaldehyde-containing denatured gel with 1% agarose concentration according to standard methods, and then transferred to Hybond-N+ (Amersham Pharmacia, USA) overnight by a capillary transfer method. The filter transferred with RNA was irradiated with UV using a Stratalinker (STRATAGENE, USA) for immobilization of the transferred RNA, washed with 2×SSC, air-dried, and then stored at room temperature until use. The small intestines were washed with physiological saline to remove the content, and then reversed. Next, the epithelia of the small intestine were collected by scratching with a preparation, and were then rapidly frozen with liquid nitrogen. The frozen small intestinal epithelia were cryopreserved in a deep freezer at −80° C. until use. Frozen small intestinal epithelia were homogenized in 5 ml of ISOGEN (Nippon Gene, Japan), and then total RNA was prepared according to the attached manufacturer&#39;s manual. 20 μg of the prepared total RNA was electrophoresed by formaldehyde-containing denatured gel with 1% agarose concentration according to standard methods, and then transferred to Hybond-N+ (Amersham Pharmacia, USA) overnight by a capillary transfer method. The filter transferred with RNA was irradiated with UV using a Stratalinker (STRATAGENE, USA) for immobilization of the transferred RNA, washed with 2×SSC, air-dried, and then stored at room temperature until use. \\n     ii) Preparation of Template DNA for Probe \\n     5 μg of the total RNA prepared from a mouse (individual mouse No. 1) was used to synthesize cDNA in 20 μL of a reaction solution (50 mM Tris (pH8.3), 75 mM KCl, 3 mM MgCl 2 , 10 mM DTT, 25 g/mL (dT)18 (SEQ ID NO: 94), 2.5 mM dNTP, 200 units of MMLV reverse transcriptase (TOYOBO, Japan)) at 37° C. for 1 hour, and then the reaction solution was treated at 70° C. for 15 minutes to inactivate the enzyme. The synthesized cDNA was diluted 5 fold, and then used in the following reaction. \\n     The following primers were synthesized from the sequences registered at GenBank (NCBI, USA), and then used for PCR reaction: \\n     Synthetic primers for obtaining mouse GAPDH cDNA \\n                            mGAPDHFW   TGAAGGTCGGTGTGAACGGATTTGGC   (SEQ\\u2003ID\\u2003NO:\\u200330)               mGAPDHRV   CATGTAGGCCATGAGGTCCACCAC   (SEQ\\u2003ID\\u2003NO:\\u200331)            \\nSynthetic primers for obtaining mouse Npt-1 cDNA\\n \\n                                mNPt1FW   GTAAAGAACCCTGTGTATTCC   (SEQ ID NO: 32)                   mNpt1RV   CTGCCTTAAGAAATCCATAAT   (SEQ ID NO: 33)            \\nSynthetic primers for obtaining mouse NaPi-7 cDNA\\n \\n                                mNaPi7FW   GAGGAATCACAGTCTCATTC   (SEQ ID NO: 34)                   nNaPi7RV   CTTGGGGAGGTGCCCGGGAC   (SEQ ID NO: 35)            \\nSynthetic primers for obtaining mouse NaPi-2b cDNA\\n \\n                                mNaPi2bFW   TCCCTCTTAGAAGACAATACA   (SEQ ID NO: 36)                   mNaPi2bRV   GTGTTTAAAGGCAGTATTACA   (SEQ ID NO: 37)            \\nSynthetic primers for obtaining mouse vitamin D1α hydroxylase cDNA\\n \\n                                m1aOHaseFW   CAGACAGAGACATCCGTGTAG   (SEQ ID NO: 38)                   m1aOHaseRV   CCACATGGTCCAGGTTCAGTC   (SEQ ID NO: 39)            \\nSynthetic primers for obtaining mouse vitamin D 24 hydroxylase cDNA\\n \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 m24OhaseFW \\n                 GACGGTGAGACTCGGAACGT \\n                 (SEQ ID NO: 40) \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 m24OhaseRV \\n                 TCCGGAAAATCTGGCCATAC \\n                 (SEQ ID NO: 41) \\n               \\n            \\n           \\n         \\n       \\n     \\n     A reaction solution was prepared according to the attached manufacturer&#39;s manual of TakaLa LA-Taq (TAKARA SHUZO, Japan). 1 μL of cDNA and 10 pmol of each of the primer as described above were added to 50 mL of the reaction solution. The solution was maintained at 94° C. for 1 minute, and then amplification was performed for 40 cycles, each incubation cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. Then the amplified bands were separated by 0.8% agarose gel electrophoresis, and then target fragments were collected using Gene Clean II (Bio101, USA). Concerning GAPDH,  32 P-labeled probe was prepared using as a template the fragment obtained by the procedure and a Megaprimer Labeling kit (Amersham Pharmacia Biotech, USA), and then used for the following hybridization. Concerning other genes, the obtained PCR fragments were incorporated into pGEM-T vector (Promega, USA), and then introduced into  Escherichia coli  DH5α. T7 (SEQ ID NO: 42) and SP6 primers (SEQ ID NO: 43) were added, 10 pmol each, to a PCR reaction solution, and then transformed  Escherichia coli  was added to the same. After the solution was maintained at 94° C. for 10 minutes, amplification was performed for 40 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. The reaction solution was subjected to 0.8% agarose gel electrophoresis to separate amplified bands, and then a target fragment was extracted using Gene Clean II (Bio 101, USA). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 T7 \\n                 TAATACGACTCACTATAGGG \\n                 (SEQ ID NO: 42) \\n                   \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 SP6 \\n                 GATTTAGGTGACACTATAG \\n                 (SEQ ID NO: 43) \\n               \\n            \\n           \\n         \\n       \\n     \\n     The nucleotide sequences of the amplified fragments obtained by the above procedures were determined using ABI377 DNA sequencer (PE Applied System, USA), thereby confirming that each target fragment was obtained. The thus obtained DNA fragments were  32 P-labelled using a Megaprimer Labeling kit (Amersham Pharmacia, USA), and then used as probes in the following hybridization. \\n     iii) Hybridization \\n     Hybridization was performed according to the attached manufacturer&#39;s manual using ExpressHyb hybridization solution (CLONTECH, USA) or Perfecthyb hybridization solution (TOYOBO, Japan). After hybridization and washing, an imaging plate (FUJI FILM, Japan) was exposed for 30 minutes to overnight, and then analysis was made using BAS2000 image analyzer (FUJI FILM, Japan) ( FIG. 9A  to C). In addition, signal intensities of target bands were measured. After the signal intensities of each gene were corrected by the signal intensity of GAPDH, the ratio of the mean values of the non tumor-bearing group (Individual mice No. 1 to 4) to the tumor-bearing group (individual No. 5 to 10) was obtained. The following Table 5 shows the ratios. As tumors were developed in the group transplanted with CHO-OST311H, mRNA levels of NaPi-7, the type II phosphate transporter in the kidney, significantly decreased, while NPT-1, the type I phosphate transporter in the kidney, did not largely change. Further, a significant decrease was observed in mRNA of NaPi-IIb, the phosphate transporter in the small intestine. In contrast, for renal vitamin D metabolic enzyme, both mRNAs of 25-hydroxyvitamin D-1-α-hydroxylase (1αOHase) and 25-hydroxyvitamin D-24-hydroxylase (24OHase) increased. \\n                     TABLE 5               Each mRNA ratio of the tumor-bearing group to       the non tumor-bearing group       (Tumor-bearing group/non tumor-bearing group)                                                NPT-1   0.88           NaPi-7   0.50           NaPi-2b   0.23           Vitamin D1αhydroxylase   3.90           Vitamin D24 hydroxylase   1.94                        \\n(8) Measurement of Serum 1,25-Dihydroxyvitamin D Levels\\n \\n     The sera from mice of the control group and the sera of the mice from the OST311H group were collected in equivalent amounts from each individual on days 44 and 46 after tumor transplantation. The sera collected from each group (0.5 ml of each total amount) were submitted to Mitsubishi Kagaku Bio-Clinical Laboratories, Inc, and then the 1,25-dihydroxyvitamin D levels contained in the sera were measured in a manner similar to clinical examination. As a result, serum 1,25-dihydroxyvitamin D levels of the control group and OST311 group were 28.0 pg/ml and 23.9 pg/ml, respectively. As described above, 1,25-dihydroxyvitamin D levels did not increase even when hypophosphatemia and hypocalcemia were observed. This result clearly suggests that vitamin D metabolism was affected as a result of the effect of OST311. \\n     (9) Soft Roentgenogram of Femora \\n     Non tumor-bearing mice and CHO ras clone-1, CHO-OST190H or CHO-OST311H-transplanted mice were sacrificed on days 44 to 46 after transplantation, the femora were collected, and then the femora were fixed in 4% neutral formalin for 3 days. Next, the soft tissue surrounding the bone was removed. Two individuals were selected at random from each group, and then irradiated with X-ray using a radiography system μFX-100 (FUJI FILM, Japan) under the following conditions: X-ray tube voltage of 25 kV, X-ray tube current of 0.1 mA and exposure time of 5 seconds, and then exposed to an imaging plate. The results are shown in  FIG. 10 . Decreased bone trabecula of the cortical bone was observed in CHO-OST311H group. \\n     EXAMPLE 12 \\n     Analysis of Nucleotide Sequence Homology and Genomic Region of OST311 \\n     Using at least a part of the amino acid sequence represented by SEQ ID No: 2 and of the nucleotide sequence represented by SEQ ID NO: 1, molecules corresponding to OST311 can be searched from various species. The mouse genome sequence database was searched using a partial nucleotide sequence of SEQ ID NO: 1, thus, a sequence having high homology with OST311 was found in the sequence of mouse chromosome 6 registered at Genbank under Accession No. AC0 15538. An amino acid sequence of a partial polypeptide of mouse OST311 obtained from the sequence is shown in SEQ ID NO: 10, and a nucleotide sequence corresponding to a partial sequence of cDNA is shown in SEQ ID NO: 9.  FIG. 6  shows the results of comparing amino acid homology between human OST311 polypeptide and mouse OST311 polypeptide. As shown in Example 11, it became clear that OST311 polypeptide having the human amino acid sequence had clear biological activities in mice. These results suggest that the activity can be easily retained, even when these amino acids are substituted, deleted, or inserted within a region having low homology in the amino acid sequence as shown in  FIG. 6 . \\n     The nucleotide sequence shown in SEQ ID NO: 1 and human 12p13 BAC RPCI11-388F6 (Accession No. AC008012) which had been recognized using the database to have a region that matches with a part of SEQ ID NO: 1 were compared, so that the sequence of the region encoding OST311 was determined. The nucleotide sequence neighboring OST311 gene is shown in SEQ ID NO: 11. TATAA box is present between nucleotide No. 498 to 502 of SEQ ID NO: 11. The first sequence matching with cDNA sequence (SEQ ID NO: 1) which we have determined, starts from nucleotide No. 1713 of SEQ ID NO: 11 and continues to nucleotide No. 2057. The next section matching with the nucleotide sequence represented by SEQ ID NO: 1 ranges from nucleotide No. 8732 to 8833 of SEQ ID NO: 11. This section is considered to be exon 2. The last section matching with the nucleotide sequence represented by SEQ ID NO: 1 starts from nucleotide No. 10644 and ends at nucleotide No. 12966 of SEQ ID NO: 11. The sequence ranging from nucleotide No. 498 to 12966 of SEQ ID NO: 11 can be considered to be at least a part of the gene encoding OST311. Further, it became clear that the STS sequence registered at Genbank under Accession No. G19259 is present between exon 1 and exon 2. OST311 is present in 12p13 region. According to Econs, M. J. et al., J. Clin. Invest. 100:2653-2657, 1997, it is inferred that the gene responsible for autosomal vitamin D-resistant rickets (ADHR) is present within a 18 cM region between D12S100 and D12S397, the microsatellite markers of 12p13 (particularly, within approximately a 10 cM region between D12S314 and D12S397), from a result of chain analysis. We evaluated the physical locations of the OST311 gene and the above microsatellite marker on the chromosome 12. As a result, D12S100 and D12S314 were in a 4602 to 6129 kb region, OST311 was in a 8958 to 9129 kb region, and D12S397 was in a 16280 to 16537 kb region. Based on these results and the strong phosphate metabolism-regulating activity of OST311, it was found that OST311 is the gene responsible for ADHR. \\n     EXAMPLE 13 \\n     Short-term Experiment of Transplantation of CHO-OST311H Cells into Nude Mice \\n     CHO-OST311H cells were transplanted subcutaneously into the dorsa of nude mice (6-week-old, BALB/c, male). On days 2 and 6 after transplantation, serum phosphate and calcium levels were measured, and the effect of recombinant OST311 within the short term was examined. CHO ras clone-1 cells were used similarly, as a control for this transplantation experiment. \\n     (1) Transplantation of CHO Cells \\n     In a manner similar to the method described in Example 11 (1), 2×10 7  CHO-OST311H cells and 2×10 7  CHO ras clone-1 cells were subcutaneously transplanted, each, into nude mice (n=6 each). Further, equivalent amounts of PBS were similarly subcutaneously transplanted (n=6). Each group of 6 nude mice was housed in a plastic cage and allowed to access to solid food CE-2 (CLEA JAPAN, Japan) and tap water ad libitum. At 6 days after transplantation, no significant tumor development was observed. \\n     (2) Measurement of Serum Phosphate and Calcium Levels on Day 2 after Cell Transplantation \\n     On the next day after cell transplantation, blood was collected from the orbital cavity of the mice under the anesthetized condition using diethylethel. The peripheral blood was subjected to serum separation using Microtainer (Beckton Dickinson, USA). Serum Phosphate levels were measured using P-test Wako (Wako Pure Chemical Industries, Japan), and serum calcium levels were measured using calcium-test Wako (Wako Pure Chemical Industries, Japan). As shown in  FIG. 11A , compared to the PBS-administered group and the CHO ras clone-1 cells-transplanted group, significant decreases in serum phosphate levels were observed in all cases of the CHO-OST311H cells-transplanted group. In contrast, no change was observed in serum calcium levels. These results clearly showed that OST311 caused decreases only in serum phosphate levels on day 2 after administration. \\n     (3) Measurement of Serum Phosphate and Calcium Levels on Day 6 after Cell Transplantation \\n     On day 6 after cell transplantation, blood was collected from the hearts of the mice under the anesthetized condition using diethylethel. As described above, serum phosphate and calcium levels of each group were measured. As shown in  FIG. 11B , similar to the results on day 2 after transplantation, compared to the PBS-administered group and the CHO ras clone-1 cells-transplanted group, significant decreases in serum phosphate levels were observed in all cases of the CHO-OST311H cells-transplanted group. In contrast, slight decreases were observed in serum calcium levels of among the CHO-OST311H cells-transplanted group. \\n     EXAMPLE 14 \\n     Purification of Recombinant OST311 \\n     CHO-OST311H cells were allowed to grow in 10% FCS-containing MEMα medium within a 225 cm 2  culture flask (CORNING USA) at 37° C. under 5% CO 2  and 100% humidity. When the cells grew to cover approximately 80% of the area of the flask, the medium was replaced with 50 ml of a serum-free medium, CHO-S-SFM II (LIFE TECHNOLOGY, USA), and 48 hours later, conditioned medium was collected. Recombinant OST311 was purified by the following method using 1,000 ml in total of the conditioned medium obtained in this manner. \\n     1000 ml of the conditioned medium was centrifuged at 16,200 g for 15 minutes at 4° C. to remove the suspended cells, and then the supernatant was subjected to SP-sepharose FF (Amersham Pharmacia, USA) packed in a glass column (30 mm in internal diameter×200 mm in length). The fraction that had passed through the column was adsorbed to Talon Superflow (metal chelate resin, CLONTECH, USA). Non-specific adsorbate was removed using a washing buffer consisting of 50 mM sodium phosphate buffer (pH 6.6) and 0.3 M NaCl, and then elution was performed using 50 mM sodium phosphate buffer (pH 6.7) and 0.2 M Imidazole. The right panel in  FIG. 12  shows the elution fractions as detected by Western blotting using an anti-His6 antibody (‘His6’ disclosed as SEQ ID NO: 88) (INVITROGEN, USA). These fractions contained a partial polypeptide (SEQ ID NO: 8) consisting of the amino acid residue No. 180, Ser, to No. 251, Ile, described in Example 9. On the other hand, the above protein contained in the conditioned medium adsorbed to SP-sepharose FF was eluted in a 50 mM sodium phosphate buffer (pH 6.7) using a 0 to 0.7 M NaCl concentration gradient. The left panel in  FIG. 12  shows the elution fractions as detected by Western blotting using an anti-His6 antibody (‘His6’ disclosed as SEQ ID NO: 88) (INVITROGEN, USA). These fractions eluted at approximately 0.3 M NaCl contained a partial polypeptide (SEQ ID NO: 4) consisting of the amino acid residue No. 25, Tyr, to No. 251, Ile described in Example 9. Further the center panel in  FIG. 12  shows the elution fractions as detected by Western blotting using a polyclonal antibody (311-114) prepared using the OST311 partial peptide (SEQ ID NO: 29) described in Example 10. These fractions eluted at a concentration of approximately 0.4 M NaCl contained a partial peptide (SEQ ID NO: 6) which comprises amino acid residue No. 25, Tyr to No. 179, Arg, described in Example 9. Thus, the fractions containing three types of OST311 partial peptide, specifically, SEQ ID NO: 4 (hereinafter, referred to as 311: 25-251), SEQ ID NO: 6 (hereinafter, referred to as 311: 25-179) and SEQ ID NO: 8 (hereinafter, referred to as 311:180-251) were purified and separated, and then concentrated using a VIV ASPIN column with a molecular weight of 10,000 (Sartorius, USA) for ultrafiltration, followed by replacement with a solvent consisting of 1 ml of 5 mM HEPES (pH 6.9) and 0.1 M NaCl. \\n     EXAMPLE 15 \\n     Histological Analysis of Undemineralized Bone Section \\n     When CHO-OST311H cells-transplanted mice and non tumor-bearing mice prepared in Example 11 were sacrificed, the right femora and tibiae were excised, leaving the connection at the knee joint intact. Immediately after cutting the shaft of the tibiae and the shaft of femora, the femora and tibiae were stored in previously prepared ice-cooled neutral formalin. Thus, undemineralized specimens were prepared. The method for preparing the unmineralized specimens is described below. \\n     The bone tissues were pre-stained with Villanueva bone stain for 3 to 7 days. The tissues were dehydrated through a graded alcohol series, and then the solvent was replaced with acetone. After acetone monomer and then monomer were applied, the tissue samples were embedded in resin. Methyl methacrylate (MMA) resin was used for embedding the samples. The tissue samples were placed in an incubator at about 35° C. for complete polymerization. At this time, the tissues were kept embedded sufficiently within the resin by appropriate addition of MMA. MMA used herein for embedding samples was prepared by adding and completely dissolving 40 g of MMA polymer (Wako Pure Chemical Industries, Japan) in 100 ml of MMA monomer (Wako Pure Chemical Industries, Japan), and then adding and completely dissolving Benzoyl peroxide (Nacalai Tesque, Japan) at a rate of 1 g per solution. The specimens were prepared for tibia. To be able to observe the cancellous bone of the tibia, the frontal section was trimmed, and then 4 μm-thick frontal section samples were prepared using a microtome for hard tissues (type RM2065 super cut, Leica). Post-staining with Villanueva-Goldner was carried out. The thus obtained sections were cleared in xylene, and then sealed in using CLEAR SEAL (MARUTO, Japan) and ONE LIGHT (MARUTO, Japan). \\n     Microscopic images of the sections are shown in  FIG. 13 . Increased width of the growth plate was observed in CHO-OST311H cells-transplanted tumor-bearing mice, compared to the control group. Further, significantly increased osteoid and decreased mineralized area were observed in the metaphysis. There was no evidence of ostitis fibrosa, and the bone collected from CHO-OST311 cells-transplanted tumor-bearing mouse exhibited typical features of osteomalacia. \\n     EXAMPLE 16 \\n     Examination of Vitamin D Metabolism at an Early Stage after Transplantation with CHO-OST311H Cells \\n     To examine the effect of OST311 on vitamin D metabolism, an experiment of transplanting CHO-OST311H cells into nude mice (6-week-old, BALB/c, male) was conducted in a manner similar to the method described in Example 13. Two experimental control groups consisting of a group which was transplanted similarly with CHO ras clone-1 cells, and a group administered with PBS in an equivalent dose with the cell suspension solution were established and used for comparison. Each group consisting of 6 mice was housed in a plastic cage with free access to tap water and solid food CE2 containing 1.03% inorganic phosphate and 1.18% calcium (CLEA JAPAN, Japan). Fluctuations in serum 1,25-hydroxyvitamin D levels and changes in the expression of vitamin D-metabolizing enzyme groups on days 1, 2, 3 and 6 after transplantation were examined. \\n     (1) Measurement of Serum 1,25-Dihydroxyvitamin D Levels \\n     On days 1, 2, 3 and 6 after cell transplantation, blood was collected from the heart of the mice of the PBS-administered group, the CHO-ras clone-1 cells-transplanted group and the CHO-OST311H cells-transplanted group, respectively under the anesthetized condition with diethylethel, and then sera were separated using a Microtainer (Beckton Dickinson, USA). Equivalent volumes of the sera collected from each mouse were mixed together to have a total volume of 0.25 ml per group. 1,25-dihydroxyvitamin D levels contained therein were measured using 1,25(OH)2D RIA-Kit, “TFB” (TFB, Japan). As a result, as shown in Table 6, significant decreases in 1,25-dihydroxyvitamin D levels were already observed on day 1 after transplantation in the CHO-OST311H cells-transplanted group, compared to the PBS-administered group and the CHO-ras clone-1 cells-transplanted group. This decreasing effect was also observed on days 2, 3 and 6 after transplantation. These results were consistent with decreases in serum 1,25-dihydroxyvitamin D levels, which is a typical clinical finding for tumor-induced osteomalacia. \\n                     TABLE 6                  Serum 1,25-dihydroxyvitamin D levels in cells-transplanted mice                         Days after transplantation                                     1   2   3   6                                             PBS-administered group (pmol/L)   338   164.3   164.5   273.7       n = 5       CHO-ras clone-1 cells-transplanted   271.9   178.3   182.9   184.6       group (pmol/L) n = 6       CHO-OST311cells-transplanted   46.7   36.3   34.5   49.1       group (pmol/L) n = 6                    \\n(2) Expression Analysis of Vitamin D-Metabolizing Enzyme Genes in the Kidney\\n \\n     To study whether the above effect of decreasing 1,25-dihydroxyvitamin D3 was due to fluctuations in 25-hydroxyvitamin D-1-α-hydroxylase (1αOHase) gene or in 25-hydroxyvitamin D-24-hydroxylase (24OHase) gene, 3 or 4 mice were selected at random from each of the PBS-administered group, the CHO-ras clone-1 cells-transplanted group and the CHO-OST311H cells-transplanted group on day 3 after transplantation. The kidneys were excised, total RNAs were prepared according to the procedures described in Example 11 (7), and then the Northern blotting was performed using the probes described in the same.  FIG. 14  shows the results. mRNA expression levels of 1αOHase gene were observed to be significantly attenuated in the CHO-OST31H cells-transplanted group, compared to the PBS-administered group and the CHO-ras clone-1 cells-transplanted group. This result suggests a possibility that OST311 directly or indirectly suppresses the expression of this gene, so as to suppress biosynthesis of serum 1,25-dihydroxyvitamin D. On the other hand, mRNA expression levels of 24OHase gene were significantly enhanced in the CHO-OST311H cells-transplanted group, compared to the PBS-administered group and the CHO-ras clone-1 cells-transplanted group. This result suggests a possibility that OST311 directly or indirectly enhances the expression of this gene, so as to promote inactivation of serum 1,25-dihydroxyvitamin D. \\n     In example 11(8), no significant difference in serum 1,25-dihydroxyvitamin D levels on days 44 and 46 after transplantation was observed, compared to the control group. Another result was different from the result in this example in that mRNA expression levels of 1αOHase tended to increase. At least one possible explanation for the difference is due to the effect of serum parathyroid hormones described in Example 17. \\n     EXAMPLE 17 \\n     Examination of Serum Parathyroid Hormone Levels at an Early Stage after Transplantation with CHO-OST311H Cells \\n     Each mouse serum collected in equivalent volumes on days 1, 2, 3, 6 and 45 after transplantation with CHO cells described in Example 11, 13 and 16 was well mixed together to have a total volume of 0.15 ml. Then, serum parathyroid hormone levels were measured using a Rat PTH IRMA kit (Nihon Medi-Physics, Japan) according to the attached manufacturer&#39;s manual. As shown in Table 7, significantly increased levels of serum parathyroid hormone were observed in CHO-OST311-transplanted group, and the difference was significant on day 45 after transplantation. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 7 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Parathyroid hormone levels in cells-transplanted mice \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                   \\n                 Days after transplantation \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 1 \\n                 2 \\n                 3 \\n                 6 \\n                 45 \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 PBS-administered group (pg/ml) \\n                 45.2 \\n                 23.8 \\n                 28.2 \\n                 19.7 \\n                   1  41.9 \\n               \\n               \\n                 n = 5 \\n               \\n               \\n                 CHO-ras clone-1 cells-trans- \\n                 15.8 \\n                 26.6 \\n                 15.7 \\n                 13.8 \\n                   2  40.4 \\n               \\n               \\n                 planted group (pg/ml) n = 6 \\n               \\n               \\n                 CHO-OST311 cells-transplanted \\n                 13.8 \\n                 20.6 \\n                 44 \\n                 57.8 \\n                   3  211.7\\u2002 \\n               \\n               \\n                 group (pg/ml) n = 6 \\n               \\n               \\n                   \\n               \\n               \\n                   1  Measured value when non tumor-bearing mice (n = 6) were used. \\n               \\n               \\n                   2  n = 10. \\n               \\n               \\n                   3  n = 12. \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 18 \\n     Experiment of Administering Cho-producing Recombinant OST311H Full-Length Protein to Normal Mice \\n     To study the effect of CHO-producing recombinant OST311H full-length protein on normal mice (BALB/c, male, 6-week-old), a polypeptide having a histidine tag added to the C-terminus of the 25 th  amino acid residue, Tyr, to the 251 st  Ile (SEQ ID NO: 4) was partially purified by the purification method described in Example 14 (1). This purified fraction was intraperitoneally administered to normal mice at 0.1 ml per administration. It was assumed that this purified fraction contained approximately 0.15 to 0.75 μg of recombinant OST311, based on the fluorescent intensity obtained by Western blotting. Similar to Example 14, 0.1 ml each of a solvent (5 mM HEPES buffer/0.1 M NaCl, pH=7.0) was intraperitoneally administered to the control group. The OST311-administrated group and the control group consisted respectively of five mice. Each group of 5 mice was housed in a plastic cage and allowed to access to tap water and solid food CE2 (CLEA JAPAN, Japan) containing 1.03% inorganic phosphate and 1.18% calcium ad libitum. \\n     EXPERIMENT 1 \\n     The experimental outline is shown in  FIG. 15A . Additional administration was performed at 5, 10, 23, 28 and 33 hours after the 1st intraperitoneal administration. Thus, intraperitoneal administration was performed 6 times in total. Subsequently, blood was collected from the orbital cavity using glass-made capillaries at 36, 47 and 71 hours after the 1st intraperitoneal administration, and then sera were separated using a Microtainer (Beckton Dickinson, USA). \\n     Phosphate and calcium levels in the thus obtained sera were determined using P-test Wako or calcium-test Wako (Wako Pure Chemical Industries, Japan) according to the attached manufacturer&#39;s manual. Thus, as shown in  FIG. 15B , effects of significantly decreasing serum phosphate levels were observed (t-test **p&lt;0.001, *p&lt;0.01) in the OST311-administered group at 36 hours after the 1st administration. In addition, this effect was further maintained at 11 hours after this time point (47 hours after the 1st administration). On the other hand, this activity disappeared at 71 hours after the 1st administration (38 hours after the final administration). Moreover, no significant change was found in serum calcium levels at any time ( FIG. 15C ). \\n     EXPERIMENT 2 \\n     The experimental outline is shown in  FIG. 16A . Additional administration was performed at 5 and 11 hours after the 1st intraperitoneal administration. Thus, intraperitoneal administration was performed 3 times in total. Subsequently, blood was collected from the orbital cavity using glass-made capillaries at 13 and 24 hours after the 1st intraperitoneal administration, and then sera were separated using a Microtainer (Beckton Dickinson, USA). Phosphate and calcium levels in the thus obtained sera were measured respectively using P-test Wako and calcium-test Wako according to the attached manufacturer&#39;s manual. Thus, as shown in  FIG. 16B , effects of significantly decreasing serum phosphate levels were observed (t-test **p&lt;0.05, *p&lt;0.01) in the OST311-administered group at 13 hours after the 1st administration. Further, this effect was maintained at 11 hours following this time point. Furthermore, no significant change in serum calcium levels was observed at any time ( FIG. 16C ). \\n     The results of Experiments 1 and 2 revealed that intraperitoneal administration into normal mice with the full-length fraction of CHO-producing recombinant OST311 protein induces hypophosphatemia, the effect of decreasing serum phosphate levels was already observed at 13 hours after the first administration, and the activity was maintained at least for 11 hours after the administration was stopped. \\n     EXAMPLE 19 \\n     Introduction of Amino Acid Mutation into OST311 \\n     As described in Example 9, it was shown that a part of recombinant OST311 produced by CHO-OST311H cells is cleaved during its secreting process by a polypeptide (SEQ ID NO: 6) having a sequence from the 25 th  Tyr to the 179 th  Arg amino acid residues and a polypeptide (SEQ ID NO: 8) having a sequence from the 180 th  Ser to the 251 st  Ile amino acid residues. \\n     This cleavage may be due to a protease which recognizes a motif consisting of RXXR sequence or RRXXR sequence located immediately before the 180 th  Ser amino acid residue. When the full-length recombinant is administered into a living organism, it is considered that the recombinant has a possibility to undergo this cleavage or degradation similar to this cleavage. Hence, gene OST311RQ encoding a sequence which can substitute both the 176 th  Arg and the 179 th  Arg amino acid residues with Gln was prepared for introducing the mutation. \\n     (1) Preparation of OST311/pCAGGS Plasmid \\n     PCR was performed by LA Taq polymerase (TAKARA SHUZO, Japan) using OST311H/pcDNA3.1 plasmid as a template, and 311F1EcoRI (SEQ ID NO: 22) and 311Not (SEQ ID NO: 44) as primers. After maintaining the temperature at 94° C. for 1 minute, reaction was performed for 25 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. After reaction, the PCR products were blunt-ended using T4 DNA polymerase (Roche, Swiss), and then phenol-chloroform treatment was performed to inactivate the enzyme. DNA was precipitated using ethanol, and then the DNA ends were phosphorylated using polynucleotide kinase (Roche, Swiss). Target DNA fragments were separated by 0.8% agarose gel electrophoresis, and then collected using Gene clean II (BIO101, USA). Plasmid vector pCAGGS (Niwa H, et al., Gene. 1991 Dec. 15; 108(2): 193-9.) was digested with EcoR I, and then blunt-ended using Klenow fragments (Roche, Swiss). Subsequently, dephosphorylation of DNA ends was performed using bovine small intestine alkaline phosphatase (TAKARA SHUZO, Japan). The target DNA fragments were separated by 0.8% agarose gel electrophoresis, and then collected using Gene Clean II (BIO101, USA). The thus obtained OST311cDNA was ligated to pre-digested pCAGGS plasmid using a DNA ligation kit (version 2) (TAKARA SHUZO, Japan) according to the attached manufacturer&#39;s manual. The product was introduced into  Escherichia coli  DH5α for cloning, so that the relevant plasmid was obtained. This plasmid DNA was used for preparing OST311RQH gene. \\n                            311LNot:               ATAAGAATGCGGCCGCTCAGATGAACTTGGCGAA   (SEQ ID NO: 44)            \\n(2) Preparation of OST311RQH Gene\\n \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 45) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311ME1: \\n                   \\n               \\n               \\n                 ATGAATTCCACCATGTTGGGGGCCCGCCTCAGG \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 46) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311HNt: \\n                   \\n               \\n               \\n                 ATGCGGCCGCCTAATGATGATGATGATGATGGATGAACTTGGCGAAGGG \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 47) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311RQF: \\n                   \\n               \\n               \\n                 ATACCACGGCAGCACACCCAGAGCGCCGAG \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 48) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311RQR: \\n                   \\n               \\n               \\n                 CTCGGCGCTCTGGGTGTGCTGCCGTGGTAT \\n               \\n            \\n           \\n         \\n       \\n     \\n     OST311ME1 is a forward primer which is a section containing the initiation methionine of OST311, OST311HNt is a reverse primer which adds 6 histidines (SEQ ID NO: 88)to the 3′ terminus of OST311, OST311RQF and OST311RQR are a forward primer and a reverse primer for introducing mutations, respectively by substituting the 527 th  and 536 th  guanines (corresponding to the 659 th  and 668 th  guanines in SEQ ID NO: 1) in the coding region of OST311 cDNA with adenines, so as to substitute the arginines at amino acid No. 176 and 179 with glutamines. 2 types of reaction solutions were prepared 20 μL each according to the attached manufacturer&#39;s manual using pfu DNA polymerase (Promega, USA). On the one hand, OST311ME1 and OST311RQR were used as primers at a final concentration of 0.2 μM, and 10 ng of OST311 expression vector described in Example 6 (1) was used as a template. After the temperature was maintained at 94° C. for 1 minute, PCR reaction was performed for 25 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. On the other hand, OST311RQF and OST311HNt were used as primers at a final concentration of 0.2 μM, and 10 ng of OST311/pCAGGS plasmid was used as a template. After the temperature was maintained at 94° C. for 1 minute, PCR reaction was performed for 35 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. The above two types of reaction products were diluted 10-fold respectively, and then 1 μL of each solution was added to 50 μL of a reaction solution prepared using LA Taq polymerase (TAKARA SHUZO, Japan) according to the attached manufacturer&#39;s manual. After the temperature was maintained at 94° C. for 1 minute, PCR reaction was performed using LA Taq polymerase (TAKARA SHUZO, Japan) and OST311 ME1 and OST311HNt as primers at a final concentration of 0.2 μM for 25 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds, and 72° C. for 1 minute. After PCR reaction, the solution was maintained at 72° C. for 7 minutes. The thus obtained reaction product was subjected to phenol/chloroform treatment, deproteinization, ethanol precipitation, and then digestion with EcoR I and Not I. An approximately 800 bp DNA fragment was separated by 2% agarose gel electrophoresis, and then collected using Gene Clean II (BIO101, USA). The thus obtained DNA fragment was inserted to the EcoR I, Not I site of a vector, IRES-EGFP-pEAK8 that had been prepared by ligating an internal ribosomal entry site (IRES) and enhanced green fluorescent protein (EGFP) to a plasmid pEAK8 (EdgeBioSystems, USA), thereby obtaining OST311RQH/IRES-EGFP/pEAK8 plasmid. Plasmid DNA was prepared according to the standard methods, and then the nucleotide sequence was determined using ABI3700 fluorescence DNA sequencer (PE Applied Systems, USA), thereby confirming that the sequence encodes a polypeptide wherein relevant mutations R176Q and R179Q are introduced and the histidine tag is added to the C-terminus. The polypeptide encoded by this gene is hereinafter referred to as OST311RQH. \\n     (3) Isolation of the CHO Cells which Stably Express OST311RQH \\n     OST311RQH/IRES-EGFP/pEAK8 plasmid was introduced into CHO-ras clone-1 cell using Transfectam (Promega, USA) according to the attached manufacturer&#39;s manual. Drug resistant cells were selected in MEMα medium containing 5 μg/mL puromycin and 10% FCS. Then, cells with strong fluorescence intensity of GFP (Green Fluorescent Protein) were sorted using FACS vantage (Beckton Dickinson, USA), and then cloned. When the cloned cells reached confluent, the medium was replaced with serum-free DF (DMEM/F-12) medium, and then conditioned medium was collected 2 days after replacement. 50 μL of the collected conditioned medium was adsorbed to Immobilon P filter (Millipore, USA) using a 96 well convertible filter system (Lifetechoriental, USA). The prepared filter was washed with TBS and TTBS, and then subjected to blocking using Blockace (Daiichi Pharmaceutical, Japan) for 1 hour at room temperature. After blocking, the filter was allowed to react for 1 hour with HRP-labeled anti-His6 monoclonal antibody (‘His6’ disclosed as SEQ ID NO: 88) (Invitrogen, USA) diluted 5000 fold with Blockace. After reaction, the filter was washed with TTBS and TBS, and then signal detection was performed using ECL (Amersham Pharmacia, USA) according to the attached manufacturer&#39;s manual. Based on signal intensity, high expression clone CHO-OST311RQH was selected. \\n     (4) Preparation of Conditioned Medium of the OST311RQH Peak Rapid Cells \\n     pEAK rapid cells (EdgeBioSystems, USA) were inoculated in 20 flasks for tissue culture (225 cm 2 , CORNING, USA). 0.48 mg of OST311RQH/IRES-GFP/pEAK8 plasmid was transfected to the cells by a phosphate calcium method according to the attached manufacturer&#39;s manual of pEAK system (EdgeBioSystems, USA). The cells were allowed to stand for 4 hours. Next, the medium of each flask was replaced with 50 mL of serum-free MEMα medium, the cells were cultured for 2 days at 37° C., and then the conditioned medium was collected. \\n     (5) Confirmation of Expression of Recombinant OST311RQH \\n     The conditioned medium resulting from the transient expression of the 2 types of the above CHO-OST311RQH cell clones and pEAK rapid cells, 10 μL each, were subjected to the Western blotting in the manner as described in Example 6(3), so that the presence of recombinant OST311RQH in the culture supernatant was examined. Anti-His (C-terminus) antibody (Invitrogen, USA) was used as a detection antibody. Thus as shown in  FIG. 17 , a strong signal located at the same position with approximately 32 kDa band described in Example 6(3) was observed for all the culture supernatants. Moreover, for all the conditioned medium, an approximately 10 kDa signal which was present in the CHO-OST311H conditioned medium was not observed by Western blotting. It can be inferred from these results that introduction of mutations R176Q and R179Q caused inhibited or attenuated cleavage of the polypeptide that was predicted to occur at these positions, so that the ratio of the presence of the polypeptide (SEQ ID NO: 8) having a sequence from the 180 th  Ser to the 251 st  Ile amino acids decreased significantly. \\n     EXAMPLE 20 \\n     Administration Experiment of Recombinant OST311RQH to Normal Mice \\n     A purified fraction containing approximately 2.8 μg/ml recombinant OST311RQH protein was obtained from 500 ml of the culture supernatant prepared in Example 19 (5) according to the method described in Example 14 (1). The purified fraction was successively administered, 0.1 ml/administration, intraperitoneally to normal mice (BALB/c, male, 6-week-old), and then serum phosphate, calcium and 1,25-dihydroxyvitamin D levels were measured. To a control group, a vehicle (5 mM HEPES buffer/0.1 M NaCl pH=7.0) was similarly administered, 0.1 ml each, intraperitoneally. The OST311RQH-administrated group and the control group consisted respectively of 6 mice. Each group of 6 mice was housed in a plastic cage and allowed to access to tap water and solid food CE2 (CLEA JAPAN, Japan) containing 1.03% inorganic phosphate and 1.18% calcium ad libitum. \\n     Experimental protocols are shown in  FIG. 18A . Additional administration was performed at 5, 10, 24, 29 and 34 hours after the 1st administration. Thus, administration was successively performed 6 times in total. In the process of the above procedures, under the anesthetized condition with diethylethel, blood was collected from the orbital cavity using glass-made capillaries at 24 hours after the first administration (immediately before 4th administration), and blood was collected from the heart at 48 hours after the first administration. \\n     (1) Measurement of Serum Phosphate and Calcium Levels \\n     Serum phosphate levels of the serum collected at 24 and 48 hours after the first administration were measured by the method described in Example 14 (3). As a result, at any time of blood collection, the OST311RQH-administered group showed significant hypophosphatemia, as shown in  FIG. 18B  (t-test **p&lt;0.01, *p&lt;0.05). In contrast, no significant fluctuation was observed in serum calcium levels ( FIG. 18C ). \\n     (2) Measurement of Serum 1,25-Dihydroxyvitamin D Levels \\n     Equivalent volumes of sera collected from each mouse at 48 hours after the first administration were mixed together by each group. Then, serum 1,25-dihydroxyvitamin D levels were measured by the method described in Example 16(1). As a result, while the control group showed 244.7 pmol/L, the OST311RQH-administered group showed a significant decrease, 24.6 pmol/L. \\n     EXAMPLE 21 \\n     CHO-OST311RQH Cell Transplantation Experiment \\n     An experiment, wherein CHO-OST311RQH that stably expresses OST311RQH cells as established in Example 19 (3) were transplanted into nude mice (7-week-old, BALB/c-nude, male, n=8) was carried out similarly to the method described in Example 13. CHO ras clone-1 cells were similarly transplanted as a control group (n=6). Each group of the nude mice was housed in a plastic cage and allowed to access to tap water and solid food CE2 (CLEA JAPAN, Japan) 1.03% inorganic phosphate and 1.18% calcium ad libitum. \\n     On day 2 after cell transplantation, blood was collected from the orbital cavity using glass-made capillaries, and then serum phosphate and calcium levels were measured by a method similar to the method described in Example 14(3). As shown in  FIG. 19A , a significant decrease in serum phosphate levels was observed in the CHO-OST311RQH cells-transplanted group (t-test *p&lt;0.001), while no significant change was observed in serum calcium levels ( FIG. 19B ) \\n     EXAMPLE 22 \\n     Preparation of Anti-OST311 Partial Peptide Polyclonal Antibody (2) \\n     4 types of partial OST311 peptides were prepared (SEQ ID NO: 49 to 52) in the manner described in Example 10. Rabbits were immunized with these peptides as antigens, and then Western blotting was performed according to the method described in Example 6(3) using the resulting anti-sera, so that recombinant OST311H was detected in the serum-free conditioned medium of CHO-OST311H cells. Antibody reaction was performed in a solution, which had been prepared by diluting the anti-sera for each peptide 250-fold with TTBS, at 4° C. with agitation overnight. After washing, alkaline phosphatase-labeled goat anti-rabbit antibody (DAKO, Denmark) was added to the solution for binding, and then recombinant OST311 was detected using an alkaline phosphatase coloring kit (BIO-RAD, USA) ( FIG. 20 ). \\n     Partial Peptides \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 311-148: \\n                 GMNPPPYSQFLSRRNEC \\n                 (SEQ ID NO: 49) \\n                   \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 311-170: \\n                 CNTPI PRRHTR \\n                 (SEQ ID NO: 50) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 311-180: \\n                 SAEDDSERDPLNVLKC \\n                 (SEQ ID NO: 51) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 311-210: \\n                 LPSAEDNSPMASDC \\n                 (SEQ ID NO: 52) \\n               \\n            \\n           \\n         \\n       \\n     \\n     EXAMPLE 23 \\n     Construction of ELISA System for Detecting OST311 Protein \\n     (1) Purification of Antibody from Anti-OST311 Partial Peptide Rabbit Anti-Serum \\n     Econo-Pac disposable chromatography column (BIO-RAD, USA) was filled with 3 ml slurry of protein A sepharose 4FF (Amersham pharmacia, USA), and then washed with 10 ml of 0.1 M glycine hydrochlorate buffer (pH 3.3) and 20 ml of PBS. 2 types of rabbit anti-sera described in Example 10 and 4 types of the same described in Example 22 were added 800 to 900 μl each, so that antibody fractions were adsorbed to the resin. The column was washed with 9 ml of PBS to remove contaminants, and then 1 ml each of 0.1 M glycine hydrochlorate buffer (pH 3.3) was added, thereby obtaining IgG elution fractions. Upon elution, 10 μl of a neutralization buffer (1 M Tris) was added to each fraction whenever necessary to neutralize the solutions. The absorbance at 280 nm was measured so as to determine the concentration of antibody contained in the elution fraction (absorbance coefficient calculated as: 1.34 (mg/ml) −1 ·(cm) −1 ). Then, some fractions were together applied to a NAP25 column, and the solvent was replaced with 50 mM sodium hydrogen carbonate solution. As a result, 5 to 15 mg of antibodies were obtained (these polyclonal antibodies are hereinafter respectively referred to as 311-48 antibody, 311-114 antibody, 311-148 antibody, 311-170 antibody, 311-180 antibody and 311-210 antibody) from each peptide anti-serum. \\n     (2) Biotinylation of Anti-OST311 IgG \\n     All the above 6 types of anti-OST311 peptide polyclonal antibodies were diluted to 1 mg/ml in 50 mM sodium hydrogen carbonate solution. Then, 1 mg of each type of antibodies was mixed well with Biotin-AC5-Osu solution (1.82 μg/ml)(Japan, Dojindo) dissolved in 10 μl of dimethylformamide by inversion for 2 hours at 4° C. Subsequently, the mixed solution was subjected to NAP10 column to remove unreacted Biotin-AC5-Osu and the solvent was replaced with PBS, thereby obtaining 6 types of biotinylated anti-OST311 peptide polyclonal antibodies. \\n     (3) Detection of OST311 in the Conditioned Medium of OST311-Expressing Cells by the Sandwich ELISA Method Using Anti-OST Peptide Rabbit Polyclonal Antibody \\n     A sandwich ELISA system was constructed by combining the 6 types of anti-OST311 peptide polyclonal antibodies for immobilization and the above 6 types of biotinylated antibodies for detection. Thus, detection of OST311 protein in the conditioned medium of OST311-expressing cells was examined. \\n     The above 6 types of anti-OST311 peptide polyclonal antibodies for immobilization obtained by Protein A purification were diluted to 10 μg/ml in 50 mM sodium hydrogen carbonate solution. 50 μl of each diluted solution was added to each well of a 96-well ELISA plate Maxisorp (Nunc, USA), and then allowed to stand for 1 hour at 37° C., thereby immobilizing IgG Next, the reaction solution was removed, and then 50 μl of Superblock blocking buffer in TBS (PIERCE, USA) was added per well to carry out blocking at room temperature for 10 minutes. After the solution was removed, OST311RQH peak rapid culture supernatant described in Example 19(5) or MEMα medium as a control was added, 50 μl per well, and then allowed to stand at room temperature for 1 hour for binding with the immobilized antibodies. After antibody reaction, the solution was washed three times with TTBS, 50 μl each of the above 6 types of biotinylated anti-OST311 antibodies (311-48, 311-114, 311-148, 311-170, 311-180 and 311-210) diluted to 10 μg/ml with TTBS containing 10% Blockace (Dainippon Pharmaceutical, Japan) was added per well, and then allowed to stand at room temperature for 30 minutes, thereby performing secondary antibody reaction. Each well was washed three times with TTBS, and then 50 μl of HRP-labeled streptavidin (DAKO, Denmark) diluted 10,000 fold with TTBS containing 10% Blockace was added per well, and then allowed to stand at room temperature for 30 minutes for binding with the biotinylated antibodies. Each well was then washed three times with TTBS, and then 50 μl of tetramethylbenzidine, the peroxidase chromogenic substrate (DAKO, Denmark) was added per well, and then allowed to develop color at room temperature for 5 minutes. Subsequently, 50 μl of 0.5 M sulfuric acid solution was added per well to stop reaction. Measurement was performed using an absorbance measurement system MTP300 (CORONA ELECTRIC, Japan) for a 96-well plate, and the absorbance at 450 nm was divided by the absorbance at 570 nm. When only MEMα was added as a control, each of values obtained by 450 nm/570 nm was 0.02 or less in every cases. In contrast, as shown in  FIG. 21A , with a combination of immobilized 311-48 antibody and detection with 311-180 antibody, or a combination of immobilized 311-180 antibody and detection with 311-148 antibody, OST311RQH in the conditioned medium could be detected significantly more than the control. Moreover, with a combination of immobilized 311-48 antibody and detection with 311-148 antibody, it is inferred that not only the full-length polypeptide, but also the N-terminal partial polypeptide fragment can be detected, because the antigenic sites of both antibodies were contained in the N-terminal partial peptide (SEQ ID NO: 6) following the cleavage of OST311 protein described in Example 9. In contrast, with a combination of immobilized 311-210 antibody and detection with 311-180 antibody, it is inferred that not only the full-length, but also the C-terminal partial peptide (SEQ ID NO: 8) following cleavage described in Example 9 can be detected. Therefore, the multiple use of these combinations makes it possible to measure the absolute amount of and the ratio of the presence of OST311 full-length polypeptide and partial polypeptides in specimens, such as biological samples. \\n     (4) Quantitative Determination of Recombinant OST311 Protein Concentration by Sandwich ELISA Method Using Anti-OST311 Peptide Rabbit Polyclonal Antibody \\n     In the above ELISA system, detection of purified recombinant OST311H consisting of a serial dilution of 1, 0.67, 0.33, 0.1, 0.067, 0.033 and 0.01 μg/ml were examined with a combination of 311-48 antibody or 311-180 antibody as an immobilized antibody and 311-148 antibody as an antibody for detection. As shown in  FIG. 21B , fine linearity could thus be obtained within a range of 0.1 to 1 μg/ml (311-48: R 2 =0.990, 311-180: R 2 =0.996). This result revealed that recombinant OST311H at least within this concentration range can be detected. \\n     EXAMPLE 24 \\n     Examination of Effect by Single Administration of Recombinant OST311 Protein \\n     To examine the short-term effect of CHO-producing recombinant OST311H full-length protein on normal mice (BALB/c, male, 6-week-old), purified recombinant OST311H full-length protein was administered once, 5.0 μg/0.1 ml per mouse via the caudal vein. To a control group, 0.1 ml of a vehicle (PBS) was administered per mouse via the caudal vein. At 1, 3 and 8 hours after administration, blood collection from the heart and dissection were performed, serum phosphate, calcium and vitamin D levels were measured, and then the expression amount of the sodium-phosphate cotransporter on the renal proximal tubule was analyzed. The OST311-administrated group and the control group respectively consisted of 6 mice. 6 mice per group were housed and allowed to access to tap water and solid food CE2 (CLEA JAPAN, Japan) containing 1.03% inorganic phosphate and 1.18% calcium ad libitum. \\n     (1) Time-Course Changes in Serum Phosphate Levels \\n     As shown in Table. 8, while no significant change was observed in serum phosphate levels at 1 and 3 hours after single administration with OST311 protein, at 8 hours after administration, a significant decrease was observed. This result clarified that the effect of OST311 requires 3 to 8 hours to lower serum phosphate levels. On the other hand, no change was observed in serum calcium levels at all time. \\n                     TABLE 8                  Serum phosphate levels                         Time                                 1   3   8                                         Vehicle-administered group   9.82 ± 0.61   9.99 ± 0.20   9.55 ± 0.29       (mg/dL)       OST311-administred group   9.61 ± 0.51   9.96 ± 0.39   7.82 ± 0.27       (mg/dL)       t-test   p &gt; 0.5   p &gt; 0.5   p &lt; 0.005                    \\n(2) Expression of Sodium-Phosphate Cotransporter on Renal Proximal Tubule\\n \\n     According to the method described in Example 11(6), the kidneys collected at 1, 3 and 8 hours after administration were mixed together per group, and then brush border membranes (BBM) of proximal tubule were prepared. The ratio of the presence of sodium-phosphate cotransporter (NaPi7) protein in the obtained BBM was analyzed by the Western blotting method. As shown in  FIG. 22A , while the expression amounts of NaPi7 at 1 and 3 hours after administration were equivalent to that of the vehicle-administered group, NaPi7 in the OST311-administered group at 8 hours after administration was shown to be significantly decreased compared to that of the vehicle-administered group. Meanwhile, to examine if the decreased NaPi7 protein was associated with RNA transcription regulation, according to the procedure described in Example 11 (7), total RNA was prepared from the kidneys excised from each mouse and Northern blotting was performed using the probe described in the same. Thus, as shown in  FIG. 22B , while mRNA levels of NaPi7 at 1 and 3 hours after administration were equivalent to that of the vehicle-administered group, NaPi7 in the OST311-administered group at 8 hours after administration was shown to be significantly decreased compared to that of the vehicle-administered group. The above results clearly showed that decreases in serum phosphate levels due to direct or indirect effect of recombinant OST311 protein correlated with downregulation of sodium-phosphate cotransporter on the renal tubule in their times of fluctuations, and suppression of NaPi-7 at the mRNA transcription level occurs at least as a factor contributing to the downregulation at the protein level. \\n     (3) Time-course changes in serum 1,25-dihydroxy vitamin D3 levels Serum 1,25-dihydroxyvitamin D3 levels at 1, 3 and 8 hours after administration were measured by the method described in Example 16(1). As shown in  FIG. 23 , in the OST311-administered group, a significant decrease in serum 1,25-dihydroxyvitamin D3 level was already observed at 3 hours after administration, and a further decrease in the same was observed at 8 hours after administration.\\n \\n(4) Changes in Expression of Vitamin D-Metabolizing Enzyme Genes\\n \\n     To elucidate whether the decreased serum 1,25-dihydroxyvitamin D3 levels were due to fluctuations in 25-hydroxyvitamin D-1-α-hydroxylase (1αOHase) or 25-hydroxyvitamin D-24-hydroxylase (24OHase) gene expressed in the kidney, total RNAs were prepared from the kidneys at 1, 3 and 8 hours after administration according to the procedure described in Example 11(7), and then Northern blotting was performed using the probe described in the same. As shown in  FIG. 24 , already at 1 hour after administration, decreased mRNA levels of 1αOHase gene and increased mRNA levels of 24OHase gene were observed. This tendency was shown to be more significant at 8 hours after administration. In  FIG. 24 , “vehicle” indicates a solvent of recombinant OST311 protein comprising 20 mM phosphate buffer (pH 6.7) and 0.3 M NaCl. \\n     These results clearly showed that OST311 lowers serum 1,25-dihydroxyvitamin D3 levels by regulating the expression of 25-hydroxyvitamin D-1α-hydroxylase (1αOHase) or 25-hydroxyvitamin D-24-hydroxylase (24OHase) gene expressed in the kidney. \\n     EXAMPLE 25 \\n     Examination of Activity of C-terminus-Deleted OST 311 \\n     (1) Construction of Expression System for OST311 Lacking C-Terminal Portion \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 OST311R693 \\n                   \\n                   \\n               \\n               \\n                 ATGCGGCCGCTATCGACCGCCCCTGACCACCCC \\n                 (SEQ ID NO: 53) \\n               \\n               \\n                   \\n               \\n               \\n                 OST311R633 \\n               \\n               \\n                 ATGCGGCCGCTACGGGAGCTCCTGTGAACAGGA \\n                 (SEQ ID NO: 54) \\n               \\n               \\n                   \\n               \\n               \\n                 OST311R618 \\n               \\n               \\n                 ATGCGGCCGCTCAACAGGAGGCCGGGGCCGGGGT \\n                 (SEQ ID NO: 55) \\n               \\n               \\n                   \\n               \\n               \\n                 OST311R603 \\n               \\n               \\n                 ATGCGGCCGCTCACGGGGTCATCCGGGCCCGGGG \\n                 (SEQ ID NO: 56) \\n               \\n            \\n           \\n         \\n       \\n     \\n     OST311R693, OST311R633, OST311R618 and OST311R603 are reverse primers for deleting 20, 40, 45 and 50 amino acid residues from of the 3′ terminus of OST311, respectively and introducing a termination codon and Not I recognition sequence. Each of these reverse primers and a forward primer OST311ME1 (SEQ ID NO: 45) containing the initiation methionine of OST311 and EcoR I recognition sequence described in Example 19 were combined to have a final concentration of 0.2 μM. Using these primers, Pyrobest DNA polymerase (TAKARA SHUZO, Japan) and 100 ng of OST311RQH/IRES-EGFP/pEAK8 plasmid DNA described in Example 19(2) as a template, PCR reaction was performed for 25 cycles after maintaining the temperature at 94° C. for 1 minute. Each reaction cycle consisted of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. The obtained reaction product was subjected to phenol/chloroform treatment, deproteinization and then ethanol precipitation. The reaction product was then digested with EcoR I and Not L and then subjected to 2% agarose gel electrophoresis, so that each DNA fragment was separated and collected using Gene Clean II (BIO110, USA). The obtained DNA fragment was ligated to pEAK8 vector (EdgeBioSystems, USA) that had been digested with EcoR I and Not I, thereby obtaining pPKOST311-Δ C20, -Δ C40, -Δ C45, -ΔC50 plasmids. Plasmid DNAs were prepared by standard methods, and then the nucleotide sequences were determined using ABI3700 fluorescence DNA sequencer (PE Applied Biosystems, USA), thereby confirming that base pairs had been deleted as desired from each of the 3′ terminus of OST311RQH gene. \\n     (2) Isolation of the CHO Cells Stably Expressing Recombinant \\n     pPKOST311-ΔC20, -ΔC40, -ΔC45, -ΔC50 plasmid DNAs were respectively introduced into CHO ras clone-1 cells using Transfectam (Promega, USA) according to the attached manufacturer&#39;s manual. CHO-OST311RQ-ΔC20, -ΔC40, -ΔC45 and ΔC50 cells showing drug resistance in MEMα medium containing 5 μg/ml puromycin and 10% FCS were obtained. These cells were respectively inoculated into 24-well plates, and then cultured in MEMα medium containing 5 μg/ml puromycin and 10% FCS to reach confluent. Subsequently, the medium was replaced with a serum-free DF (DMEM/F-12) medium. 3 days later, conditioned medium was collected. The obtained conditioned medium was subjected to the Western blotting method using OST311-specific polyclonal antibody 311-148 or 311-180 described in Example 22, thereby confirming the expression of each relevant protein at a position corresponding to each predicted molecular weight. \\n     (3) Experiment of Transplanting CHO Cells Expressing C-Terminus-Deleted OST311- \\n     CHO cells expressing the above 20, 40, 45 and 50 residues-deleted OST311 were separately transplanted subcutaneously into nude mice (6-week-old, BALB/c-nude, male, 6 mice per group) in a manner similar to the method described in Example 13. As control groups, the full-length OST311RQH-expressing CHO cells and CHO ras clone-1 cells were separately transplanted subcutaneously (n=6). Each group of mice was housed in a plastic cage and allowed to access to tap water and solid food CE2 (CLEA JAPAN, Japan) ad libitum. \\n     On day 3 after cell transplantation, blood was collected from the heart, and then serum phosphate and calcium levels, and 1,25-dihydroxyvitamin D3 levels were measured by a method similar to that described in Example 20. As shown in  FIG. 25 , in all of CHO-OST311RQ-ΔC20, -ΔC40, -ΔC45 and -ΔA C50 cells-transplanted groups, significant decreases in serum phosphate levels, equivalent to that of the group transplanted with the cells expressing the full-length OST311RQH, were observed (t-test, **p&lt;0.001). Moreover, significant decreases in serum 1,25-dihydroxyvitamin D3 levels were also observed in CHO-OST311RQ-ΔC40, -ΔC45 and -ΔC50 cells-transplanted groups (when an average serum level of CHO-ras clone-1-transplanted group was defined as 100%, full-length: 3.1%, ΔC40: 9.4%, ΔC45: 10.0% and ΔC 50: 68.1%). These results clearly showed that even when at least 50 amino acids were deleted from the C-terminus of OST311 protein, serum phosphate-decreasing activity or serum 1,25-dihydroxyvitamin D3 level-decreasing activity was maintained. \\n     EXAMPLE 26 \\n     Examination of Activity of N-Terminus-Deleted OST311 \\n     (1) Construction of Expression System for OST311 Lacking 9 Amino Acid Residues of N-Terminus \\n     The following oligo DNAs were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 OST311SGFW: \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 57) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 aattccaccATGTTGGGGGCCCGCCTCAGGCTCTGGGTCTGTGCTTGTGC \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 AGCGTCTGCAGCATGAGCGTCCTgcatGC \\n               \\n               \\n                   \\n               \\n               \\n                 OST311SGRV: \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 58) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 aattGCatgcAGGACGCTCATGCTGCAGACGCTGCACAAGGCACAGACCC \\n                   \\n               \\n               \\n                   \\n               \\n               \\n                 AGAGCCTGAGGCGGGCCCCCAACATggtgg \\n               \\n            \\n           \\n         \\n       \\n     \\n     OST311SGFW is an oligo DNA which consists of a gene sequence encoding a signal peptide portion consisting of amino acid residues from the initiation methionine of OST311 to the 24 th  Ala of SEQ ID NO: 2, and contains an EcoR I recognition sequence on its 5′ terminus. OST311SGRV is a complementary strand of OST311SGFW, and contains an EcoR I recognition sequence on its 5′ terminus. In addition, the recognition site of a restriction enzyme Sph I has been introduced into the 3′ side of OST311SGFW and the 5′ side of OST311SGRV. The 23rd Arg within the signal peptide sequence is substituted with His by introduction of the Sph I recognition site. The above oligo DNAs were annealed according to standard methods, thereby obtaining double-stranded DNA fragments containing EcoR I recognition sequences on both ends and Sph I recognition sequence at a position corresponding to the 23 rd  amino acid residue in the signal peptide, and encoding the full-length signal peptide starting from the initiation methionine of OST311 and containing one modified residue. The obtained DNA fragments were inserted into EcoR I-digested pEAK8 vectors (EdgeBioSystems, USA). Plasmid DNAs wherein both EF1 promoters and the above DNA fragments existing within the vector in forward direction were then selected, thereby obtaining plasmid pPKFGSG. \\n     Next, the following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 59) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 OST311dN9: \\n                 ATATGCATGCCTCCAGCTGGGGTGGCCTGATCCAC. \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     OST311dN9 is a forward primer designed to contain a Sph I recognition site on its 5′ terminus, and the 24 th  Ala residue of SEQ ID NO: 2 followed by an amino acid sequence starting from the 34 th  Ser residue of the same. Using a combination of this primer and a reverse primer OST311HNt (Example 19, SEQ ID NO: 46) which has been designed to have an Not I recognition sequence and 6 histidine residues (SEQ ID NO: 88) added to the C-terminal portion followed by a termination codon, and OST311RQH/IRES-EGFP/pEAK8 plasmid DNA described in Example 19(2) as a template, PCR amplification was performed in the manner described in Example 25 (1). The obtained PCR product was digested with Sph I and Not I, and then inserted into the above-described, Sph I- and Not I-digested plasmid vector pPKFGSG according to standard methods. \\n     The nucleotide sequence of the obtained plasmid OST311ΔN9-pPKFGSG was determined using ABI3700 fluorescence DNA sequencer (PE Applied Biosystems, USA), so that the inserted gene sequence was confirmed to contain a signal peptide from the initiation methionine to the 24 th  Ala of OST311RQH gene (wherein the 23 rd  Arg had been substituted with His), contain deletion of only a gene sequence corresponding to 9 amino acid residues from the following 25 th  Tyr to the 33 rd  Gly, and encode the whole sequence from the 34 th  Ser to the termination codon containing the histidine tag. \\n     (2) Isolation of the CHO cells stably expressing recombinant OST311ΔN9-pPKFGSG plasmid DNA was introduced into CHO ras clone-1 cells using Transfectam (Promega, USA) according to the attached manufacturer&#39;s manual, and then CHO-OST311RQ-ΔN9 cells showing drug resistance in MEMα medium containing 5 μg/ml puromycin and 10% FSC were obtained. The conditioned medium was collected from the obtained cells in the manner described in Example 25. Western blotting was then performed using OST311 specific polyclonal antibody 311-148 described in Example 22 or a polyclonal antibody 311-237 newly obtained by immunizing a rabbit with a partial polypeptide from the 237 th  Gly to 251 st  Ile of SEQ ID NO: 2. Thus, expression of a relevant protein at the position corresponding to a predicted molecular weight was confirmed. These results revealed that OST311 signal peptide, wherein the 23 rd  Arg of SEQ ID NO: 2 had been substituted with His, functioned properly enough to secrete OST311 recombinant protein, and even when at least a portion from the 25 th  Tyr to 33 rd  Gly had been deleted, the recombinant protein could be present stably to some extent in the cultured medium after secretion. \\n     (3) Experiment of Transplanting CHO Cells Expressing N-Terminal 9 Amino Acids-Deleted OST311 \\n     The above CHO-OST311RQ-ΔN9 cells were subcutaneously transplanted to nude mice (8-week-old, BALB/c-nude, male, 6 mice per group) in a manner similar to the methods described in Example 13. As control groups, subcutaneous transplantation of full-length OST311RQH-expressing CHO cells and of CHO ras clone-1 cells were respectively performed similarly (n=6). Each group of the nude mice was housed in a plastic cage and allowed to access to tap water and solid food CE-2 (CLEA JAPAN, Japan) ad libitum. \\n     On day 4 after cell transplantation, blood was collected from the orbital cavity using glass-made capillaries, and then serum phosphate levels were measured in the manner described in Example 20. Thus, in CHO-OST311RQ-ΔN9 cells-transplanted group, a significant decrease in serum phosphate levels, which was equivalent to that of the full-length recombinant-expressing cells-transplanted group, was observed (CHO-ras clone-1 group: 6.85±0.12 mg/dL, CHO-OST311RQH group: 3.91±0.23 mg/dL (p&lt;0.001, to CHO-ras clone-1 group), CHO-OST311RQ-ΔN9 group: 4.33±0.15 mg/dL (p&lt;0.001, to CHO-ras clone-1 group). \\n     These results revealed that even when at least 9 amino acid residues consisting of the 25 th  Tyr to 33 rd  Gly of SEQ ID NO: 2 was deleted, the biological activity of OST311 remained undamaged. \\n     EXAMPLE 27 \\n     Examination of  Escherichia coli -pProducing OST311 Recombinant \\n     (1) Construction of OST311  Escherichia coli  Expression Vector OST311/pET3a \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 OST311N: \\n                   \\n                   \\n               \\n               \\n                 TGTATCCCAATGCCTCCCCACTG \\n                 (SEQ ID NO: 60) \\n               \\n               \\n                   \\n               \\n               \\n                 OST311Bm: \\n               \\n               \\n                 ATGGATCCCTAGATGAACTTGGCGAAGGG \\n                 (SEQ ID NO: 61) \\n               \\n            \\n           \\n         \\n       \\n     \\n     PCR was performed using as a template OST311/pCAGGS plasmid prepared in Example 19, OST311N (SEQ ID NO: 60) and OST311Bm (SEQ ID NO: 61) as primers, and pfu DNA polymerase (Promega, USA). After the temperature was maintained at 94° C. for 1 minute, reaction was performed for 35 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. After reaction, phenol/chloroform treatment was performed to inactivate the enzyme, and then DNA was collected by ethanol precipitation. The DNA was digested with BamH I, the target OST311 cDNA fragments were separated by 2% agarose gel electrophoresis, and then collected using Gene Clean II (BIO101, USA). Meanwhile, plasmid vector pET3a (Novagen, USA) was digested with Nde I, and then blunt-ended using Klenow fragments (Roche, Swiss). The vector was further digested with BamH I, the obtained plasmid DNA fragment was separated by 0.8% agarose gel electrophoresis, and then collected using Gene Clean II (BIO101, USA). The thus obtained OST311 cDNA fragment was ligated to the digested plasmid pET3a using a DNA ligation kit version 2 (TAKARA SHUZO, Japan). The product was then introduced into  Escherichia coli  DH5α for cloning, and then a plasmid was extracted. The nucleotide sequence of the plasmid was confirmed to make sure that OST311 cDNA had been inserted into pET3a as expected. The plasmid was named OST311/pET3a. \\n     (2) Construction of OST311/pET28 Vector for Expression of OST311 in  Escherichia coli    \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n            \\n               \\n                 OST311Nd: \\n                   \\n                   \\n               \\n               \\n                 ATCATATGTATCCCAATGCCTCCCCACTG \\n                 (SEQ ID NO: 62) \\n               \\n               \\n                   \\n               \\n               \\n                 OST311Not: \\n               \\n               \\n                 ATGCGGCCGCCTAGATGAACTTGGCGAAGGG \\n                 (SEQ ID NO: 63) \\n               \\n            \\n           \\n         \\n       \\n     \\n     PCR was performed using OST311/pET3a plasmid as a template, OST311Nd (SEQ ID NO: 62) and OST311Not (SEQ ID NO: 63) as primers, and LA Taq (TAKARA SHUZO, Japan). After the temperature was maintained at 94° C. for 1 minute, reaction was performed for 35 cycles, each cycle consisting of 94° C. for 30 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. After reaction, phenol/chloroform treatment was performed to inactivate the enzyme. The amplified DNA fragment was then collected by ethanol precipitation. The DNA fragment was digested with Nde I and Not I, the target OST311 cDNA fragment was separated by 2% agarose gel electrophoresis, and then collected using Gene Clean II (BIO101, USA). Meanwhile, plasmid vector pET28 (Novagen, USA) was digested with Nde I and Not I, and then dephosphorylated using bovine intestinal alkaline phosphatase (TAKARA SHUZO, Japan). The product was then separated by 0.8% agarose gel electrophoresis, and then the relevant digested plasmid was collected using Gene Clean II (BIO101, USA). The thus obtained OST311cDNA was ligated to the digested pET28 plasmid using a DNA ligation kit version 2 (TAKARA SHUZO, Japan), and then the ligated product was introduced into  Escherichia coli  DH5α for cloning, thereby extracting a plasmid. The nucleotide sequence of the plasmid was confirmed to make sure that OST311 cDNA which allowed expression of recombinant OST311 having a His-Tag sequence added to the N-terminal side had been inserted into pET28. This plasmid was named OST311/pET28. The amino acid sequence and the nucleotide sequence of the recombinant His-OST311 encoded by this vector are shown in  FIG. 26 . \\n     (3) Expression of Recombinant His-OST311 in  Escherichia coli  and Preparation of the Same \\n     Plasmid OST311/pET28 was introduced into  Escherichia coli  BL21 (DE3) Codon Plus RP (STRATAGENE, USA) for transformation, and then clones were obtained. The obtained  Escherichia coli  clones were inoculated in 100 ml of LB medium containing 10 mg of kanamycin (SIGMA, USA) and cultured at 37° C. overnight. The bacterial cell suspension was inoculated in 1 L of LB medium to A 600 =0.1, and then shake-cultured using a 3 L Sakaguchi flask at 37° C. Absorbance of the culture suspension was measured with time. When it reached A 600 =0.6 to 1.0, isopropyl-1-thio-β-galactoside (IPTG) (Wako Pure Chemical Industries, Japan) was added to 1 mM. 4 hours later, the cells were collected by centrifugation (7700 g×15 minutes). The collected cells were suspended in 20 ml of 0.1 M Tris hydrochloride buffer (pH 7.5) containing 1 mM DTT, and then disrupted using French Press. The solution containing the disrupted cells was centrifuged (7700 g×15 minutes), and then the precipitate fraction was suspended in 15 ml of 0.1 M Tris hydrochloride buffer (pH 7.5). DNase I (Roche, Swiss) was added to the suspension to 0.1 mg/mL, and then shaken at 4° C. for 1 hour. Next, centrifugation was performed (23400 g×15 minutes), and then the precipitate fraction was collected as inclusion body. The obtained inclusion body was washed by suspending in 10 ml of 20 mM Tris hydrochloride buffer (pH 8) containing 0.75 M urea and 1% Triton-X, and then centrifuging (23,400 g×15 minutes) to collect precipitate. This washing procedure was repeated twice. \\n     The washed inclusion body was suspended in 5 ml of a denaturing solution (50 mM phosphate buffer (pH 8) containing 1 mM DTT and 6 M guanidine hydrochloride), and then solubilized by shaking the suspension at 37° C. for 1 hour. Insoluble matters were removed as precipitate by centrifugation (23,400 g×15 minutes), and then the solution was equilibrated with 6 M guanidine hydrochloride-containing 50 mM phosphate buffer (pH 6). The solubilized sample was applied to a column filled with Ni-NTA Agarose (QIAGEN, Germany), and then washed with 6 M guanidine hydrochloride-containing 50 mM phosphate buffer (pH 6). Protein adsorbed to the column was eluted using 50 mM phosphate buffer (pH 4.5) containing 500 mM imidazole (Nacalai Tesque, Japan) and 6 M guanidine hydrochloride, thereby purifying denatured His-OST311. The concentration was obtained based on UV absorbance at 280 nm of the purified sample, and then 50 mM phosphate buffer (pH 6) containing 6 M guanidine hydrochloride was added to the sample to have a final concentration of 2 mg/ml, thereby preparing denatured His-OST311 solution. Cysteine as a reducing agent was added to the sample to have a final concentration of 1 mM, diluted 100 fold with 20 mM phosphate buffer (pH 6) containing 0.6 M guanidine hydrochloride and 0.1% Tween 20 to start refolding. Incubation was performed at 4° C. for 3 days or more. \\n     The refolding solution was dialyzed against 0.1 M acetate buffer (pH 4.8) at 4° C. The dialyzed refolding solution was concentrated approximately 10 fold using a ultrafiltration membrane, and then purified by HPLC using cation exchange column SP-5PW (TOSOH, Japan). Protein was eluted using 10% glycerol-containing 20 mM phosphate buffer (pH 6) with a linear NaCl gradient from 0.5 M to 2 M. This elution pattern is shown in  FIG. 27 . SDS-PAGE analysis and mass spectrometry measurement revealed that among eluted two types of protein peaks, His-OST311 was contained in a peak eluted at a lower salt concentration. As described above, approximately 0.6 mg of the final purified product, His-OST311, could be prepared from approximately 1 L of cultured cells. \\n     (4) Construction of pET22b-MK-OST311 Vector for Expression of MK-OST311 \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 64) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311MK1: \\n                   \\n               \\n               \\n                 gaattcatatgaaatacccgaacgcttccccgctgctgggctccagctg \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 (SEQ ID NO: 65) \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 OST311MK2: \\n                   \\n               \\n               \\n                 cccaagcttgcggccgcctagatgaacttggc \\n               \\n            \\n           \\n         \\n       \\n     \\n     A target sequence was amplified by PCR using the above His-OST311 expression plasmid OST311/pET28 as a template and OST311MK1 (SEQ ID NO: 64) and OST311MK2 (SEQ ID NO: 65) as primers. In the OST311 cDNA obtained by this procedure, 27 nucleotides following the initiation codon (ATG) had been converted to  Escherichia coli  type codons. The PCR product was purified using a QIAquick PCR purification Kit (QIAGEN, Germany), and then digested with restriction enzymes Nde I (TAKARA SHUZO, Japan) and Not I (TAKARA SHUZO, Japan) at 37° C. for 1 hour. The digested PCR product was separated by agarose electrophoresis, and then purified using a QIAquick PCR purification Kit (QIAGEN, Germany). The obtained DNA fragment was digested with restriction enzymes Nde I and Not I at 37° C. for 1 hour, and then ligated to plasmid vector pET22b (Novagen, USA), which had been separated and purified by agarose electrophoresis, using a DNA Ligation kit Ver 2 (TAKARA SHUZO, Japan) at 16° C. for 15 minutes. The ligated product was introduced into  Escherichia coli  JM109 (TAKARA SHUZO, Japan) for cloning, and then a plasmid was extracted by standard methods. The nucleotide sequence of the obtained plasmid was determined to make sure that the obtained OST311 cDNA had been inserted into pET22b vector as expected. This plasmid was named pET22-MK-OST311. The nucleotide sequence and amino acid sequence of recombinant MK-OST311 encoded by the vector are shown in  FIG. 26 . \\n     (5) Expression of MK-OST311 in  Escherichia coli  and Preparation of the Same \\n     Plasmid pET22-MK-OST311 was introduced into  Escherichia coli  BL21 (DE3) Codon Plus RP (STRATAGENE, USA), and then the transformed clones were obtained. The obtained  Escherichia coli  clones were inoculated in 100 ml of LB medium containing 10 mg of ampicillin and then cultured at 37° C. overnight. The bacterial cell suspension was inoculated in 1 L of LB medium to A 600 =0.1, and then cultured with shaking using a 3 L Sakaguchi flask at 37° C. IPTG was added to the bacterial cell culture to induce expression of the recombinant, and then inclusion bodies were prepared in a manner similar to the above preparation method of His-OST311. \\n     The washed inclusion bodies were suspended in 5 ml of denaturing solution (50 mM phosphate buffer (pH 8) containing 1 mM DTT and 6 M guanidine hydrochloride), and then solubilized by shaking the suspension at 37° C. for 1 hour. The solubilized product was diluted 2 fold using a denaturing solution, and then diluted 100 fold using 20 mM phosphate buffer (pH 6) containing 0.6 M guanidine hydrochloride and 0.1% Tween 20 to start refolding. Incubation was performed at 4° C. for 3 days or more. It was shown that under conditions of addition of oxidant and a pH of 7 or more, protein was precipitated so that refolding efficiency decreased significantly. The refolding solution was dialyzed against 0.1 M acetate buffer (pH 4.8) at 4° C. The dialyzed refolding solution was concentrated approximately 10 fold using an ultrafiltration membrane, and then purified by HPLC using cation exchange column SP-5PW (TOSOH, Japan). Protein was eluted using 10% glycerol-containing 20 mM phosphate buffer (pH 6) with a linear NaCl gradient from 0.5 M to 2 M. As shown in  FIG. 28 , there were 2 peaks of protein elution, and SDS-PAGE analysis and mass spectrometry measurement revealed that MK-OST311 was contained in a peak eluted at a lower salt concentration. Thus, approximately 0.6 mg of the final purified product, MK-OST311, could be purified from approximately 1 L of flask-cultured cells. \\n     (6) PEGylation of MK-OST311 \\n     10 ml of MK-OST311 (0.05 mg/ml) purified with an ion exchange column was adjusted to have pH 4.8 using 10% acetic acid. 25 mg of activated PEG (Sharewater, USA) with a molecular weight of 20,000 dissolved in 10 mM acetate buffer (pH 4.8) was added to this solution with agitation in ice. 15 minutes later, 1 M sodium cyano borohydride (Nacalai Tesque, Japan) dissolved in 10 mM acetate buffer (pH 4.8) was added to the solution to have a final concentration of 15 mM, and then the solution was agitated at 4° C. for 16 hours. OST311 PEGylated by this reaction was purified by HPLC using cation exchange column SP-5PW (TOSOH, Japan). Protein was eluted using 10% glycerol-containing 20 mM phosphate buffer (pH 6) with a linear NaCl gradient from 0.5 M to 2 M. As shown in  FIG. 29 , PEGylated MK-OST311 was eluted as a single peak on the side of lower ion strength compared to that of MK-OST311. \\n     (7) Examination of Activity of His-OST311 Recombinant \\n     To examine the biological activity of the purified His-OST311 recombinant, single administration of the recombinant protein, 4.5 μg/0.1 ml each, to normal mice (5-week-old, BALB/c, male, 6 mice per group) via the caudal vein in the manner described in Example 24 was performed. 9 hours later, serum phosphate and 1,25-dihydroxyvitamin D3 levels were measured by a method similar to that described in Example 20. Single administration of the same dose of CHO-OST311H cell-derived purified recombinant was performed to a positive control group, and single administration of a vehicle comprising 20 mM phosphate buffer (pH 6.9) and 0.3 M NaCl was performed, 0.1 ml each, to a vehicle-administered group, both via the caudal vein. \\n     As shown in  FIG. 30A , in His-OST311-administered group at 9 hours after administration, a significant effect of decreasing serum phosphate levels was observed compared to the vehicle-administered group. The degree of the decrease was equivalent to that of CHO-producing recombinant protein-administered group. In addition, the serum 1,25-dihydroxyvitamin D3 levels at 9 hours after administration in His-OST311-administered group also showed a significant decrease as shown in  FIG. 32 . \\n     As described in Example 24 (3) and (4), significantly decreased serum 1,25-dihydroxyvitamin D3 levels were already observed at 4 hours after single administration of CHO cell-producing OST311 protein. Before this time point, at 1 hour after administration, decreased expression of 25-hydroxyvitamin D-1-α-hydroxylase (1αOHase) and enhanced expression of 25hydroxyvitamin D-24-hydroxylase (24OHase) were observed in the kidneys. Hence, single administration of His-OST311, 4.5 μg/0.1 ml per mouse, to BALB/c mice (5-week-old, male) was performed via the caudal vein, and then the kidneys were excised 1 and 4 hours later. Changes in the expression of 1αOHase gene and 24OHase gene in the kidneys were analyzed by the Northern blotting method. Single administration of the same dose of CHO-OST311H cell-derived purified recombinant was performed to a positive control group, and single administration of a vehicle comprising 20 mM phosphate buffer (pH 7.0) and 0.3 M NaCl was performed, 0.1 ml each, to a vehicle-administered group, both via the caudal vein. As shown in  FIG. 31 , similar to CHO-producing recombinant, His-OST311 caused decreased expression of 1αOHase gene and enhanced expression of 24OHase gene already at 1 hour after administration. It was revealed that His-OST311 has activity regulating expression of vitamin D-metabolizing enzyme genes equivalent to that of the CHO-producing recombinant. Further, as shown in  FIG. 32 , changes in serum 1,25-dihydroxyvitamin D3 levels showed a moderate decrease at 4 hours after administration with recombinant, and showed a more significant decrease at 8 hours. It was revealed that this manner of changes was almost consistent with that of changes with time as observed in the experiment of administering CHO-producing recombinant described in Example 24 (3). \\n     From the above results, it was revealed that recombinant His-OST311 produced by  Escherichia coli  has biological activities of at least serum phosphate-decreasing activity and vitamin D metabolism-regulating activity equivalent to the activities of the secreted recombinant produced by CHO-OST311H cells. \\n     (8) Examination of Activity of PEGylated MK-OST311 \\n     The biological activity of PEGylated MK-OST311 was examined by performing single administration of PEGylated MK-OST311, 5.0 μg/0.1 ml per mouse, to normal mice (5-week-old, BALB/c, male, 8 mice per group) via the caudal vein in the manner described in Example 24, and then measuring serum phosphate levels at 9 hours after administration by a method similar to that of Example 20. Single administration of a vehicle comprising 20 mM PB (pH6.0), 10% glycerol, 1 M NaCl and 0.1% Tween 20 was performed, 0.1 ml/mouse, to a vehicle-administered group via the caudal vein. At 8 hours after administration, blood was collected from the orbital cavity using glass-made capillaries, and then inorganic phosphate levels of the obtained sera were measured. As shown in  FIG. 30B , a significant effect of decreasing serum phosphate levels was observed in PEGylated MK-OST311-administered group, compared to the vehicle-administered group. The result reveled that when  Escherichia coli -producing recombinant was PEGylated, the biological activity of it is not inhibited. \\n     EXAMPLE 28 \\n     Introduction of Amino Acid Mutation into Cleavage Site \\n     As described in Example 9, it was shown that OST311 was cleaved at a position between the 179 th  Arg and the 180 th  Ser amino acid residues of SEQ ID NO: 2. Meanwhile, it was confirmed as described in Example 19 that this cleavage was inhibited by simultaneous substitution of both 176 th  Arg and 179 th  Arg amino acid residues of OST311 with Gln. These facts suggest a possibility that this cleavage is due to some protease which recognizes a motif consisting of adjacent RXXR or RRXXR sequence. Moreover, when full-length recombinant consisting of the polypeptide represented by SEQ ID NO: 4 is administered in vivo, the same cleavage as described above or similar cleavage thereto may occur. Hence, the 175 th  to the 180 th  amino acid residues of SEQ ID NO: 2 were respectively substituted with Ala, Gln or Trp, and then how the substitution affected expression and secretion patterns in each mutant recombinant in CHO cells was examined. \\n     (1) Construction of OST311 Gene with Mutations at Cleavage Site \\n     The following primers were synthesized. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 pyh23PA1F \\n                   \\n                   \\n               \\n               \\n                   \\n                 AACACCCCCATAGCACGGCGGCACA \\n                 (SEQ ID NO: 66) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23PA1R \\n               \\n               \\n                   \\n                 TGTGCCGCCGTGCTATGGGGGTGTT \\n                 (SEQ ID NO: 67) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA1F \\n               \\n               \\n                   \\n                 ACCCCCATACCAGCGCGGCACACCCG \\n                 (SEQ ID NO: 68) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA1R \\n               \\n               \\n                   \\n                 CGGGTGTGCCGCGCTGGTATGGGGGT \\n                 (SEQ ID NO: 69) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA2F \\n               \\n               \\n                   \\n                 CCCATACCACGGGCGCACACCCGGAG \\n                 (SEQ ID NO: 70) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA2R \\n               \\n               \\n                   \\n                 CTCCGGGTGTGCGCCCGTGGTATGGG \\n                 (SEQ ID NO: 71) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23HA1F \\n               \\n               \\n                   \\n                 ATACCACGGCGGGCCACCCGGAGCGC \\n                 (SEQ ID NO: 72) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23HA1R \\n               \\n               \\n                   \\n                 GCGCTCCGGGTGGCCCGCCGTGGTAT \\n                 (SEQ ID NO: 73) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23TA1F \\n               \\n               \\n                   \\n                 CCACGGCGGCACGCCCGGAGCGCCG \\n                 (SEQ ID NO: 74) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23TA1R \\n               \\n               \\n                   \\n                 CGGCGCTCCGGGCGTGCCGCCGTGG \\n                 (SEQ ID NO: 75) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA3F \\n               \\n               \\n                   \\n                 CGGCGGCACACCGCGAGCGCCGAGGA \\n                 (SEQ ID NO: 76) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RA3R \\n               \\n               \\n                   \\n                 TCCTCGGCGCTCGCGGTGTGCCGCCG \\n                 (SEQ ID NO: 77) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23SA1F \\n               \\n               \\n                   \\n                 CGGCACACCCGGGCCGCCGAGGACGA \\n                 (SEQ ID NO: 78) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23SA1R \\n               \\n               \\n                   \\n                 TCGTCCTCGGCGGCCCGGGTGTGCCG \\n                 (SEQ ID NO: 79) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ1F \\n               \\n               \\n                   \\n                 ACCCCCATACCACAGCGGCACACCCG \\n                 (SEQ ID NO: 80) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ1R \\n               \\n               \\n                   \\n                 CGGGTGTGCCGCTGTGGTATGGGGGT \\n                 (SEQ ID NO: 81) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ2F \\n               \\n               \\n                   \\n                 CCCATACCACGGCAGCACACCCGGAG \\n                 (SEQ ID NO: 82) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ2R \\n               \\n               \\n                   \\n                 CTCCGGGTGTGCTGCCGTGGTATGGG \\n                 (SEQ ID NO: 83) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ3F \\n               \\n               \\n                   \\n                 CGGCGGCACACCCAGAGCGCCGAGGA \\n                 (SEQ ID NO: 84) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RKQ3R \\n               \\n               \\n                   \\n                 TCCTCGGCGCTCTGGGTGTGCCGCCG \\n                 (SEQ ID NO: 85) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RWF \\n               \\n               \\n                   \\n                 CGGCGGCACACCTGGAGCGCCGAGG \\n                 (SEQ ID NO: 86) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 pyh23RWR \\n               \\n               \\n                   \\n                 CCTCGGCGCTCCAGGTGTGCCGCCG \\n                 (SEQ ID NO: 87) \\n               \\n            \\n           \\n         \\n       \\n     \\n     pyh23PA1F and pyh23PA1R are forward and reverse primers for introducing a mutation, in which substitution of the 652 nd  cytosine of OST311 cDNA (SEQ ID NO: 1) with guanine causes substitution of the 174 th  Pro amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as P174A. \\n     pyh23RA1F and pyh23RA1R are forward and reverse primers for introducing a mutation, in which substitution of the 655 th  cytosine and the 656 th  guanine of OST311 cDNA (SEQ ID NO: 1) with guanine and cytosine, respectively, causes substitution of the 175 th  Arg amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as R175A. \\n     pyh23RA2F and pyh23RA2R are forward and reverse primers for introducing a mutation, in which substitution of the 658 th  cytosine and the 659 th  guanine of OST311 cDNA (SEQ ID NO: 1) with guanine and cytosine, respectively, causes substitution of the 176 th  Arg amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as R176A. \\n     pyh23HA1F and pyh23HA1R are forward and reverse primers for introducing a mutation, in which substitution of the 661 st  cytosine and the 662 nd  adenine of OST311 cDNA (SEQ ID NO: 1) with guanine and cytosine, respectively, causes substitution of the 177 th  His amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as H177A. \\n     pyh23TA1F and pyh23TA1R are forward and reverse primers for introducing a mutation, in which substitution of the 664 th  adenine of OST311 cDNA (SEQ ID NO: 1) with guanine causes substitution of the 178 th  Thr amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as T178A. \\n     pyh23RA3F and pyh23RA3R are forward and reverse primers for introducing a mutation, in which substitution of the 667 th  cytosine and the 668 th  guanine of OST311 cDNA (SEQ ID NO: 1) with guanine and cytosine, respectively, causes substitution of the 179 th  Arg amino acid residue, of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as R179A. \\n     pyh23SA1F and pyh23SA1R are forward and reverse primers for introducing a mutation, in which substitution of the 670 th  adenine and the 671 st  guanine of OST311 cDNA (SEQ ID NO: 1) with guanine and cytosine, respectively, causes substitution of the 180 th  Ser amino acid residue of SEQ ID NO: 2 with Ala. Hereinafter, this mutation is referred to as S180A. \\n     pyh23RKQ1F and pyh23RKQ1R are forward and reverse primers for introducing a mutation, in which substitution of the 656 th  guanine of OST311 cDNA (SEQ ID NO: 1) with adenine causes substitution of the 175 th  Arg amino acid residue of SEQ ID NO: 2 with Gln. Hereinafter, this mutation is referred to as R175Q. \\n     pyh23RKQ2F and pyh23RKQ2R are forward and reverse primers for introducing a mutation, in which substitution of the 659 th  guanine of OST311 cDNA (SEQ ID NO: 1) with adenine causes substitution of the 176 th  Arg amino acid residue of SEQ ID NO: 2 with Gln. Hereinafter, this mutation is referred to as R176Q. \\n     pyh23RKQ3F and pyh23RKQ3R are forward and reverse primers for introducing a mutation, in which substitution of the 668 th  guanine of OST311 cDNA (SEQ ID NO: 1) with adenine causes substitution of the 179 th  Arg amino acid residue of SEQ ID NO: 2 with Gln. Hereinafter, this mutation is referred to as R179Q. \\n     pyh23RWF and pyh23RWR are forward and reverse primers for introducing a mutation, in which substitution of the 667 th  cytosine of OST311 cDNA (SEQ ID NO: 1) with thymine causes substitution of the 179 th  Arg amino acid residue of SEQ ID NO: 2 with Trp. Hereinafter, this mutation is referred to as R179W. \\n     (1)-1 Construction of OST311P174AH Gene \\n     2 types of reaction solutions (100 μL each) were prepared using Pyrobest DNA polymerase (TAKARA SHUZO, Japan) according to the attached manufacturer&#39;s manual. For one reaction solution, OST311ME1 (SEQ ID NO: 45) and pyh23PA1F (SEQ ID NO: 66) were used as primers at a final concentration of 0.2 μM, and for the other reaction solution, pyh23PA1R (SEQ ID NO: 67) and OST311HNt (SEQ ID NO: 46) were used as primers at a final concentration of 0.2 μM. To each reaction solution, 10 ng of OST311/pCAGGS plasmid described in Example 19(1) was added as a template, and then the solution was maintained at 94° C. for 1 minute. Then, PCR reaction was performed for 40 cycles, each cycle consisting of 94° C. for 20 seconds, 55° C. for 30 seconds and 72° C. for 1 minute. The two types of reaction solutions were diluted respectively 10 fold, and then 1 μL of each solution was added to 100 μL of a reaction solution prepared according to the document attached to Pyrobest DNA polymerase (TAKARA SHUZO, Japan). OST311ME1 (SEQ ID NO: 45) and OST311HNt (SEQ ID NO: 46) were added as primers to the solution to have a final concentration of 0.2 μM, and then the solution was maintained at 94° C. for 1 minute. Then, PCR reaction was performed for 30 cycles, each cycle consisting of 94° C. for 20 seconds, 55° C. for 30 seconds and 72° C. for 1 minute and 30 seconds. After PCR reaction, the solution was further maintained at 72° C. for 7 minutes. The thus obtained reaction products were collected using Gene Clean II (BIO101, USA) according to the attached manufacturer&#39;s manual. The products were then digested with EcoR I and Not I, and then subjected to 2% agarose gel electrophoresis to separate approximately 800 bp DNA fragments. The fragment was collected using Gene Clean II (BIO101, USA). The thus obtained DNA fragments were inserted into EcoR I and Not I sites of pEAK8 vector (EdgeBio, USA), thereby obtaining plasmid OST311P174AH-pEAK8. Plasmid DNA was prepared according to standard methods, and the nucleotide sequence was determined using ABI3700 fluorescence DNA sequencer (PE Applied Systems, USA). Thus, it was confirmed that mutation P174H had been introduced as expected. Moreover, it was confirmed that a histidine tag had been added to the C-terminus. The polypeptide encoded by a mutant gene having mutation P174A introduced therein is referred to as OST311P174AH. \\n     (1)-2 Preparation of OST311R175AH Gene \\n     OST311R175AH gene was prepared using pyh23RA1F (SEQ ID NO: 68) and pyh23RA1R (SEQ ID NO: 69) primers by a method similar to that of (1)-1. \\n     (1)-3 Preparation of OST311R176AH Gene \\n     OST311R176AH gene was prepared using pyh23RA2F (SEQ ID NO: 70) and pyh23RA2R (SEQ ID NO: 71) primers by a method similar to that of (1)-1. \\n     (1)-4 Preparation of OST311H177AH Gene \\n     OST311H177AH gene was prepared using pyh23HA1F (SEQ ID NO: 72) and pyh23HA1R (SEQ ID NO: 73) primers by a method similar to that of (1)-1. \\n     (1)-5 Preparation of OST311T178AH Gene \\n     OST311T178AH gene was prepared using pyh23TA1F (SEQ ID NO: 74) and pyh23TA1R (SEQ ID NO: 75) primers by a method similar to that of (1)-1. \\n     (1)-6 Preparation of OST311R179AH Gene \\n     OST311R179AH gene was prepared using pyh23RA3F (SEQ ID NO: 76) and pyh23RA3R (SEQ ID NO: 77) primers by a method similar to that of (1)-1. \\n     (1)-7 Preparation of OST311S180AH Gene \\n     OST311S180AH gene was prepared using pyh23SA1F (SEQ ID NO: 78) and pyh23SA1R (SEQ ID NO: 79) primers by a method similar to that of (1)-1. \\n     (1)-8 Preparation of OST311R175QH Gene \\n     OST311R175QH gene was prepared using pyh23RKQ1F (SEQ ID NO: 80) and pyh23RKQ1R (SEQ ID NO: 81) primers by a method similar to that of (1)-1. \\n     (1)-9 Preparation of OST311R176QH Gene \\n     OST311R176QH gene was prepared using pyh23RKQ2F (SEQ ID NO: 82) and pyh23RKQ2R (SEQ ID NO: 83) primers by a method similar to that of (1)-1. \\n     (1)-10 Preparation of OST311R179QH Gene \\n     OST311R179QH gene was prepared using pyh23RKQ3F (SEQ ID NO: 84) and pyh23RKQ3R (SEQ ID NO: 85) primers by a method similar to that of (1)-1. \\n     (1)-11 Preparation of OST311R179WH Gene \\n     OST311R179WH gene was prepared using pyh23RWF (SEQ ID NO: 86) and pyh23RWR (SEQ ID NO: 87) primers by a method similar to that of (1)-1. \\n     (2) Transient Expression of OST311 Genes with Mutation at Cleavage Site and Preparation of Conditioned Media \\n     pEAK rapid cells (EdgeBiosystems, USA) were inoculated onto a 12-well plate. The above 11 types of expression plasmids for mutant OST311 were transfected into the cells by a phosphate calcium method according to the document attached to pEAK system (EdgeBiosystems, USA). The cells were allowed to stand for 4 hours, the medium was replaced with 1.5 ml of serum-free MEMα medium, the cells were cultured at 37° C. for 2 days, and then the conditioned medium was collected. \\n     (3) Evaluation of Expression of OST311 Genes with the Mutation at Cleavage Site \\n     The thus obtained conditioned medium was subjected to Western blotting in the manner described in Example 6(3), and then the presence of OST311 recombinant with the mutation at cleavage site in the conditioned medium was examined. OST311-specific polyclonal antibody, 311-148 described in Example 22 was used for detection. Thus, as shown in  FIG. 33 , a degradation product containing the polypeptide represented by SEQ ID NO: 6 was observed at around 16 kDa, similarly to the wild type, for mutation P174A, R175A, R175Q, H177A, T178A or S180A-introduced OST311 recombinant among those substituted at the 174 th  to 180 th  amino acid residues. However, no degradation product was observed for any of the mutations R176A, R179A, R176Q, R179Q or R179W-introduced OST311 recombinant. These results suggest that in particular the 176 th  Arg and the 179 th  Arg amino acid residues play an important role in the cleavage that occurs between the 179 th  Arg and the 180 th  Ser amino acid residues. That is, cleavage is inhibited or suppressed by substitution of both residues at least with any amino acid of Ala, Gln or Trp. Hence, it is expected that production of the full-length polypeptide is promoted using this finding. \\n     EXAMPLE 29 \\n     As shown in Example 6, a recombinant product obtained by expression of OST311 in CHO cells or COS cells is cleaved between the 179 th  arginine and 180 th  serine located immediately after RXXR motif as shown in Example 9. As clearly shown in Example 18, the hypophosphatemia-inducing activity of OST311 protein was retained in the full-length protein that had not been cleaved at this site. Further, involvement of RXXR motif in this cleavage was clear from the mutation introduction experiment shown in Examples 19 and 28. In order to produce and obtain recombinant OST311 efficiently, avoidance of this cleavage is an important problem, and introduction of the mutation into RXXR motif shown in Example 28 is one of the effective methods. Furin is known as one of proteinases which recognize RXXR motif. This enzyme is localized in the Trans-golgi region, and is thought to cleave post-translated protein by recognizing RXXR motif in the process of secretion. \\n     (1) Involvement of Furin in Cleavage of OST311 \\n     OST311/pCAGGS plasmid was introduced into furin-deficient LoVo cells using Transfectum (Promega, USA). Then, the cells were cultured for 48 hours. When OST311 protein transiently expressed and secreted in the culture solution was analyzed by Western blotting using 311-148 antibody, no cleaved product was detected. This result suggests that furin is involved in the observed cleavage when OST311 is produced. \\n     (2) Avoiding Cleavage of OST311 \\n     It was assumed from the above results that inhibition of furin activity was effective to improve the productivity of OST311, in which it was not cleaved between the 179 th  arginine and the 180 th  serine. Accordingly, it is conceivable that OST311 is expressed in a host, such as Lovo cells having no furin activity, or under a condition in which furin activity is suppressed by the addition of a furin inhibitor. In order to suppress furin activity of CHO-OST311H cells, α1-antitrypsin Portland (α1-PDX) was transiently expressed according to the method reported by Benjannet S et al (J Biol Chem 272: 26210-8, 1997), and then the conditioned medium was collected. The ratio of the full-length polypeptide in the recombinant products was increased, compared to that in the conditioned medium of the control CHO-OST311H cells without this gene. Accordingly, it was concluded that the production efficiency of the full-length OST311 protein can be elevated by introducing a substance suppressing furin activity extrinsically or intrinsically. \\n     All publications, patents and patent applications cited herein are incorporated herein by reference in their entirety. \\n     INDUSTRIAL APPLICABILITY \\n     According to the present invention, there are provided a polypeptide regulating phosphate metabolism, calcium metabolism and/or calcification, a DNA encoding the polypeptide, and a pharmaceutical composition containing the polypeptide as an active ingredient, and an antibody recognizing the polypeptide, a pharmaceutical composition containing the antibody as an active ingredient, a diagnostic method using the antibody, and a diagnostic composition. \\n     SEQUENCE LISTING FREE TEXT \\n     \\n         \\n         SEQ ID NO: 12: Synthetic DNA \\n         SEQ ID NO: 13: Synthetic DNA \\n         SEQ ID NO: 14: Synthetic DNA \\n         SEQ ID NO: 15: Synthetic DNA \\n         SEQ ID NO: 16: Synthetic DNA \\n         SEQ ID NO: 17: Synthetic DNA \\n         SEQ ID NO: 18: Synthetic DNA \\n         SEQ ID NO: 19: Synthetic DNA \\n         SEQ ID NO: 20: Synthetic DNA \\n         SEQ ID NO: 21: Synthetic DNA \\n         SEQ ID NO: 22: Synthetic DNA \\n         SEQ ID NO: 23: Synthetic DNA \\n         SEQ ID NO: 24: Synthetic DNA \\n         SEQ ID NO: 25: Synthetic DNA \\n         SEQ ID NO: 26: Synthetic DNA \\n         SEQ ID NO: 27: Synthetic peptide \\n         SEQ ID NO: 28: Synthetic peptide \\n         SEQ ID NO: 29: Synthetic DNA \\n         SEQ ID NO: 30: Synthetic DNA \\n         SEQ ID NO: 31: Synthetic DNA \\n         SEQ ID NO: 32: Synthetic DNA \\n         SEQ ID NO: 33: Synthetic DNA \\n         SEQ ID NO: 34: Synthetic DNA \\n         SEQ ID NO: 35: Synthetic DNA \\n         SEQ ID NO: 36: Synthetic DNA \\n         SEQ ID NO: 37: Synthetic DNA \\n         SEQ ID NO: 38: Synthetic DNA \\n         SEQ ID NO: 39: Synthetic DNA \\n         SEQ ID NO: 40: Synthetic DNA \\n         SEQ ID NO: 41: Synthetic DNA \\n         SEQ ID NO: 42: Synthetic DNA \\n         SEQ ID NO: 43: Synthetic DNA \\n         SEQ ID NO: 44: Synthetic DNA \\n         SEQ ID NO: 45: Synthetic DNA \\n         SEQ ID NO: 46: Synthetic DNA \\n         SEQ ID NO: 47: Synthetic DNA \\n         SEQ ID NO: 48: Synthetic DNA \\n         SEQ ID NO: 49: Synthetic peptide \\n         SEQ ID NO: 50: Synthetic peptide \\n         SEQ ID NO: 51: Synthetic peptide \\n         SEQ ID NO: 52: Synthetic peptide \\n         SEQ ID NO: 53: Synthetic DNA \\n         SEQ ID NO: 54: Synthetic DNA \\n         SEQ ID NO: 55: Synthetic DNA \\n         SEQ ID NO: 56: Synthetic DNA \\n         SEQ ID NO: 57: Synthetic DNA \\n         SEQ ID NO: 58: Synthetic DNA \\n         SEQ ID NO: 59: Synthetic DNA \\n         SEQ ID NO: 60: Synthetic DNA \\n         SEQ ID NO: 61: Synthetic DNA \\n         SEQ ID NO: 62: Synthetic DNA \\n         SEQ ID NO: 63: Synthetic DNA \\n         SEQ ID NO: 64: Synthetic DNA \\n         SEQ ID NO: 65: Synthetic DNA \\n         SEQ ID NO: 66: Synthetic DNA \\n         SEQ ID NO: 67: Synthetic DNA \\n         SEQ ID NO: 68: Synthetic DNA \\n         SEQ ID NO: 69: Synthetic DNA \\n         SEQ ID NO: 70: Synthetic DNA \\n         SEQ ID NO: 71: Synthetic DNA \\n         SEQ ID NO: 72: Synthetic DNA \\n         SEQ ID NO: 73: Synthetic DNA \\n         SEQ ID NO: 74: Synthetic DNA \\n         SEQ ID NO: 75: Synthetic DNA \\n         SEQ ID NO: 76: Synthetic DNA \\n         SEQ ID NO: 77: Synthetic DNA \\n         SEQ ID NO: 78: Synthetic DNA \\n         SEQ ID NO: 79: Synthetic DNA \\n         SEQ ID NO: 80: Synthetic DNA \\n         SEQ ID NO: 81: Synthetic DNA \\n         SEQ ID NO: 82: Synthetic DNA \\n         SEQ ID NO: 83: Synthetic DNA \\n         SEQ ID NO: 84: Synthetic DNA \\n         SEQ ID NO: 85: Synthetic DNA \\n         SEQ ID NO: 86: Synthetic DNA \\n         SEQ ID NO: 87: Synthetic DNA', 'id': 'US-7981419-B2', 'ti': 'Method for treating hypophosphatemic bone diseases using FGF-23 antibody'}>,\n",
       " <Hit {'ab': '(Modified) rice endosperm protein is used as novel protective hydrocolloid for active ingredients, especially fat-soluble active ingredients and/or colorants. Included are compositions comprising (modified) rice endosperm protein and at least one active ingredient and to their manufacture, as well as to the (modified) rice endosperm protein itself and its manufacture. These compositions are used for the enrichment, fortification and/or coloration of food, beverages, animal feed, personal care or pharmaceutical compositions, and to food, beverages, animal feed, personal care and pharmaceutical compositions containing such a (modified) rice endosperm protein and such a composition, respectively.', 'clm': 'The invention claimed is: \\n     \\n       1. A process for preparing a composition comprising a rice endosperm protein and an active ingredient which process comprises the following steps:\\n I) preparing an aqueous solution or colloidal solution of said rice endosperm protein, \\n II) preparing a solution or dispersion of an active ingredient, wherein said active ingredient is a fat-soluble active ingredient and/or colorant, and optionally comprises at least a fat-soluble adjuvant and/or excipient, \\n III) mixing the solutions prepared in step I) and step II), \\n IV) homogenizing the mixture resulted from step III, \\n V) submitting the mixture after having performed step IV) to enzymatic treatment in order to cross-link the rice endosperm protein. \\n \\n     \\n     \\n       2. The process according to  claim 1 , wherein the enzymatic treatment according to step V) uses a cross-linking enzyme that is transglutaminase.', 'cpc': 'A23J1/12', 'detd': 'This application is a divisional of application Ser. No. 12/095,927 filed Nov. 25, 2008 which in turn is the U.S. national phase of International Application No. PCT/EP2006/011873 filed 8 Dec. 2006 which designated the U.S. and claims priority to U.S. Provisional Application No. 60/748,192 filed 8 Dec. 2005, the entire contents of each of which are hereby incorporated by reference. \\n    \\n    \\n     The present invention is directed to the use of (modified) rice endosperm protein (REP as used herein) as novel protective hydrocolloid for active ingredients, especially fat-soluble active ingredients and/or colorants. Moreover, the present invention is directed to compositions comprising (modified) rice endosperm protein and at least one active ingredient and to their manufacture, as well as to the (modified) rice endosperm protein itself and its manufacture. The present invention is further directed to the use of such compositions for the enrichment, fortification and/or coloration of food, beverages, animal feed, personal care or pharmaceutical compositions, and to food, beverages, animal feed, personal care and pharmaceutical compositions containing such a (modified) rice endosperm protein and such a composition, respectively. \\n     Active ingredients, especially fat-soluble active ingredients or colorants, are often not added as such to food, beverages, animal feed, personal care and pharmaceutical compositions, but in form of formulations of the active ingredient in a hydroprotective colloid for reasons of enhancing properties such as chemical stability, (water-)solubility, free-flowing and controlled release etc. Known hydroprotective colloids are e.g. gelatine of different origin (poultry, bovine, pork, fish) and starch. Since hydroprotective colloids of animal origin are often not desired for religious or allergenic reasons and starch-based hydroprotective colloids might have low preference for consumers who are interested in gluten and corn-free products there is an on-going need for alternative hydroprotective colloids. \\n     Rice endosperm proteins are recognized as nutritional and hypoallergenic and can, thus, be a suitable alternative source of protective hydrocolloid for formulations of active ingredients. However, high insolubility and poor functionality of rice endosperm protein at neutral pH limits its industrial application as a functional ingredient in food and pharmaceuticals products. The present invention overcomes these limitations and incorporates the (modified) rice endosperm protein as a protective hydrocolloid for formulations of active ingredients, especially of fat-soluble active ingredients and/or colorants. \\n     Rice proteins rank high in nutritional quality in comparison to other cereals including corn and wheat, and are therefore perceived to have immense potential uses as food ingredients. Cereal grain proteins are rich in the essential amino acids cysteine and methionine. Lysine is the primary limiting amino acid in cereal proteins, but rice contains more lysine (3.8 g/16 g N) (grams per 16 grams of nitrogen) than other cereal proteins (wheat 2.3, corn 2.5 g/16 g N) (grams per 16 grams of nitrogen) (see reference 4 cited below). Although rice is generally regarded as having the lowest protein content (7.3%) among the common grains (wheat 10.6%, corn 9.8%, barley 11.0%, millet 11.5%), the net protein utilization of rice protein (73.8%) is the highest among the cereal grains (wheat 53.0%, corn 58.0%, barley 62.0%, millet 56.0%). \\n     Compared with other cereal proteins, isolation of rice protein is difficult and therefore costly. The predominant rice protein, glutelin, is hydrophobic and is cross-linked with disulfide bonds. The extracted proteins are highly insoluble in nature and the conditions used in protein isolation further decrease their solubility, and thus have limited application as a functional ingredient. High-protein rice products can be obtained from rice flour by alkali extraction followed by precipitation at the isoelectric pH of the protein. Starch-hydrolyzing enzymes such as alpha-amylase, glucoamylase, and pullulanase are often used to separate proteins in rice flour by solubilizing and removing starch. In addition to starch hydrolyzing enzymes, cellulase and hemicellulase enzymes have been used to further increase the protein content in rice protein concentrate. However, information on suitable extraction methods and functionalities of such isolates is limited. Efficient extraction methods using approved food grade enzymes and chemicals are essential for commercial production and application of rice protein. \\n     This need is fulfilled by the compositions of the present invention which comprise a rice endosperm protein and an active ingredient. \\n     BACKGROUND INFORMATION \\n     \\n         \\n         1. Fiocchi, A.; Travaini, M.; D&#39;Auria, E.; Banderali, G.; Bernardo, L.; Riva, E. Tolerance to a rice hydrolysate formula in children allergic to cow&#39;s milk and soy. Clin Exp Allergy 2003, 33(11), 1576-80. \\n         2. Eggum, B. O.; Cabrera, M. I. Z.; Juliano, B. O. Protein and lysine digestibility and protein quality of cooked Filipino rice diets and milled rice in growing rats. Plant Foods Hum. Nutr. 1992, 43 (2), 163-170 \\n         3. Bean, M. M.; Nishita, K. D. Rice flours for baking. In Juliano B. O., editor. Rice chemistry and technology, 2nd ed. St. Paul: Amer Assoc. of Cereal Chemists. 1985, pp 539-556. \\n         4. Juliano, B. O. Rice: chemistry and technology. St. Paul, Minn.: American Association of Cereal Chemists. 1985. \\n         5. Anderson, A.; Hettiarachchy, N. S.; Yu, Z. Y. Physicochemical properties of pronase treated rice glutelin. Journal of the American Oil Chemists&#39; Society 2000, 78 (1), 1-6. \\n         6. Yu, Z. Y.; Hettiarachchy, N. S.; Rath, N. Extraction, denaturation and hydrophobic properties of rice flour proteins. J. Food Sci. 2001, 66 (2), 229-232. \\n         7. Padhye, V. W.; Salunke, D. K. Extraction and characterization of rice proteins. Cereal chem. 1979, 106 (3), 389-393. \\n         8. Tecson, E. M. S.; Esmama, B. V.; Lontok, L. P.; Juliano, B. O. Studies on the extraction and composition of rice endosperm glutelin and prolamin. Cereal Chem. 1971, 168-181. \\n         9. Wen, T. N.; Luthe, D. S. Biochemical characterization of rice glutelin. Plant Physiol. 1985, 78, 172-177. \\n         10. Morita, T.; Kiriyama, S. Mass production method for rice protein isolate and nutritional evaluation. J. Food Sci. 1993, 58 (6), 1939-1406. \\n         11. Griffin, V. K.; Brooks, J. R. Production and size distribution of rice moltodextrin hydrolyzed from milled rice flour using heat stable alpha-amylase. J. Food Sci. 1989, 54, 190-193. \\n         12. Shih, F. F.; Daigle, K. Use of enzymes for the separation of protein from rice flour. Cereal Chem. 1997, 74 (4), 437-441. \\n         13. Shih, F. F.; Champagne, E. T.; Daigle, K.; Zarins, Z. Use of enzymes in the processing of protein products from rice bran and rice flour. Nahrung 1999, 43, 14-18. \\n         14. Shih, F. F.; Daigle, K. W. Preparation and characterization of rice protein isolates. Journal of the American Oil Chemists&#39; Society 2000, 77, 885-889. \\n         15. Tang, S.; Hettiarachchy, N. S.; Eswaranandam, S.; Crandall, P. Protein extraction from heat-stabilized defatted rice bran: II. The role of amylase, celluclast, and viscozyme. J. Food Sci. 2003, 68 (2), 471-470. \\n         16. AACC. Method 46-08. Approved Methods of the American Association of Cereal chemists, 8th ed., vol 2. AACC, St. Paul, Minn. 1990, P 1-2. \\n         17. Laemmli, U. K. Cleavage of structural proteins during the assembly of the head of the bacteriophage T4. Nature 1970, 227, 680-686. \\n         18. Hayakawa, S.; Nakai, S. Relationships of hydrophobicity and net charge to the solubility of milk and soy proteins. J. Food Sci. 1985, 50, 486-491. \\n         19. Bera, M. B.; Mukherjee, R. K. Solubility, emulsifying, and foaming properties of rice bran protein concentrates. J. Food Sci. 1989, 54 (1), 142-145. \\n         20. Pearce, K. N.; Kinsella, J. E. Emulsifying properties of proteins: evaluation of a turbidimetric technique. J. Agric. Food Chem. 1978, 26, 716-722. \\n         21. SAS. 2002. JMP® User&#39;s Guide, Version 5. SAS Institute Inc. Cary, N.C. \\n         22. Kolar, C. W.; Richert, S. H.; Decker, C. D.; Steinke, F. H.; Vander, R. J. Isolated soy protein. In New Protein Foods; Altschul, A M., Wilcke, H L., Eds.; Academic Press: New York, 1985; Vol. 5. \\n         23. Biliaderis, C. G. Differential scanning calorimetry in food research: A review. Food Chem. 1983, 10, 239-265. \\n         24. Damodarn, S. Protein-stabilized foams and emulsions. J. Food Sci. 2005, 70 (3), 54-66. \\n       \\n    \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE DRAWINGS \\n         FIG. 1  is a flow chart showing the procedures of methods to prepare rice protein isolates; \\n         FIG. 2  is a flow chart showing cellulase treatment following the treatment shown in  FIG. 1 ; \\n         FIG. 3  shows the electrophoresis profiles of the rice protein isolates of Table 6; \\n         FIG. 4  shows the hydrolysis profiles of the rice endosperm protein treatment with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or pepsin as a function of hydrolysis time; \\n         FIG. 5  shows the surface hydrophobicity of the rice endosperm protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or pepsin as a function of the degree of hydrolysis; \\n         FIG. 6  shows the solubility of the rice endosperm protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or pepsin as a function of the degree of hydrolysis; \\n         FIG. 7  shows the molecular size profiles of rice proteins and hydrolysate produced by treatment with pepsin, LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) and ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) as determined by SDS PAGE gel electrophoresis; \\n         FIG. 8  shows the molecular size profiles of rice protein hydrolysate determined by SDS PAGE gel electrophoresis; and \\n         FIG. 9  shows the emulsifying properties of rice protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANEOL and pepsin as a function of the degree of hydrolysis. \\n     \\n    \\n    \\n     DETAILED DESCRIPTION OF THE INVENTION \\n     Compositions of the Present Invention \\n     The compositions of the present invention may be solid compositions, i.e. stable, water-soluble or water-dispersible powders, or they may be liquid compositions, i.e. aqueous colloidal solutions or oil-in-water dispersions of the aforementioned powders. The stabilised oil-in-water dispersions, which may be oil-in-water emulsions or may feature a mixture of suspended, i.e. solid, particles and emulsified, i.e. liquid, droplets, may be prepared by the methods described below or by an analogous manner. \\n     More specifically, the present invention is concerned with stable compositions in powder form comprising one or more (fat-soluble) active ingredients and/or one or more colorants in a matrix of a (modified) rice endosperm protein. \\n     Preferably the amount of the (modified) rice endosperm protein is from 1 to 70 weight-%, more preferably from 5 to 50 weight-%, even more preferably from 10 to 40 weight-%, most preferably from 10 to 20 weight-% (with 20 weight-% being the most preferred one) and/or the amount of the (fat-soluble) active ingredient and/or colorant is from 0.1 to 90 weight-%, preferably from 1 to 80 weight-%, more preferably from 1 to 20 weight-%, based on the total amount of the composition. If additional adjuvants and/or excipients such as tocopherol and/or ascorbyl palmitate are present, they are present in an amount of from 0.01 to 50 weight-%, preferably in an amount of from 0.1 to 30 weight-%, more preferably in an amount of from 0.5 to 10 weight-%, based on the total amount of the composition. \\n     (Modified) Rice Endosperm Protein \\n     In preferred embodiments of the present invention the rice endosperm protein is a modified rice endosperm protein whose manufacture is described below (see page 13, line 26 till page 18, line 23 and examples 1 to 7). An especially preferred rice protein is one obtained by the following steps: alkaline extraction, (enzymatically modification, especially with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA)), centrifugation and ultra-filtration. If needed for the further use the thus obtained modified rice endosperm protein may also be dried. \\n     The invention is, however, not restricted to the use of such manufactured modified rice endosperm proteins. \\n     Even more preferred are (modified) rice endosperm proteins that have an emulsion capacity of ≧220, preferably of ≧350, more preferably of ≧500, even more preferably of from 500 to 1000. Additionally in preferred embodiments of the invention the used (modified) rice endosperm proteins have an emulsion activity of ≧0.2, preferably of ≧0.45, more preferably of ≧0.5, even more preferably of from 0.5 to 1.0. The determination of the emulsion capacity is described in example 8, the determination of the emulsion activity is described in example 9. The present invention refers also to these (modified) rice endosperm proteins themselves. \\n     In further preferred compositions of the present invention the (modified) rice endosperm protein is cross-linked with at least one compound selected from the group consisting of reducing sugars, glycoproteins or glycopeptides. \\n     Active Ingredient \\n     The active ingredients are those ingredients with a pharmacological effect or those providing health benefits to the human or animal body in general. Preferably the active ingredient is a fat-soluble active ingredient and/or a colorant. \\n     The fat-soluble active ingredient and/or the colorant is preferably selected from the group consisting of carotenes and structurally related polyene compounds, fat-soluble vitamins, coenzyme Q10, polyunsaturated fatty acids such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA) and esters thereof (such as the ethyl esters or the triglycerides (containing the same or different fatty acids)), mono-, di-, triglycerides rich in polyunsaturated fatty acids, fat-soluble UV-A filters, UV-B filters, as well as their physiologically acceptable derivatives such as their esters, especially with C 1-20  carbonic acids, and any mixtures of them. \\n     The most preferred fat-soluble vitamins are Vitamin A or E. \\n     Preferred examples of the carotenes and structurally related polyene compounds are carotenoids such as α-carotene, β-carotene, 8′-apo-β-carotenal, 8′-apo-β-carotenoic acid esters such as the ethyl ester, canthaxanthin, astaxanthin, lycopene, lutein, zeaxanthin, crocetin, α-zeacarotene, β-zeacarotene, as well as their physiologically acceptable derivatives such as their esters, especially with C 1-20  carbonic acids, and any mixtures of them. \\n     The most preferred carotenoid is β-carotene. \\n     The term “β-carotene” encompasses the all-cis as well as the all-trans isomers and all possible mixed cis-trans-isomers. The same applies for the other carotenoids. \\n     The term “zeaxanthin” encompasses the natural R,R-zeaxanthin, as well as S,S-zeaxanthin, meso-zeaxanthin and any mixture of them. The same applies for lutein. \\n     The (fat-soluble) active ingredients may be of natural origin, i.e. isolated/extracted from plants, purified and/or concentrated, as well as those synthesized by chemical and/or microbiological (fermentative) routes. \\n     Further Components \\n     Beside the active ingredient and the (modified) rice endosperm protein the compositions of the present invention may preferably additionally contain at least one water-soluble antioxidant and/or fat-soluble antioxidant. \\n     The water-soluble antioxidant may be for example ascorbic acid or a salt thereof, preferably sodium ascorbate, water soluble polyphenols such as hydroxytyrosol and oleuropein aglycon; epigallocatechingallate (EGCG) or extracts of rosemary or olives. \\n     The fat-soluble antioxidant may be for example a tocopherol, e.g. dl-α-tocopherol (i.e. synthetic tocopherol), d-α-tocopherol (i.e. natural tocopherol), β- or γ-tocopherol, or a mixture of two or more of these; butylated hydroxytoluene (BHT); butylated hydroxyanisole (BHA); ethoxyquin, propyl gallate; tert. butyl hydroxyquinoline; or 6-ethoxy-1,2-dihydroxy-2,2,4-trimethylquinoline (EMQ), or an ascorbic acid ester of a fatty acid, preferably ascorbyl palmitate or stearate. \\n     The compositions of the present invention may further contain a co-emulgator selected from the group consisting of mono- and diglycerides of fatty acids, polyglycerol esters of fatty acids, lecithins; N-acylated amino acids and derivatives thereof, N-acylated peptides with an alkyl or alkenyl radical, and salts thereof; alkyl or alkenyl ether or ester sulfates, and derivatives and salts thereof; polyoxyethylenated alkyl or alkenyl fatty ethers or esters; polyoxyethylenated alkyl or alkenyl carboxylic acids and salts thereof; N-alkyl or Nalkenyl betaines; alkyltrimethylammonium or alkenyltrimethylammonium and salts thereof; polyol alkyl or alkenyl ether or ester; and mixtures thereof. \\n     Preferred examples of polyol alkyl or alkenyl ethers or esters are sorbitan alkyl or alkenyl esters polyoxyethylenated with at least 20 units of ethylene oxide, such as sorbitan palmitate 20 EO or Polysorbate 40 marketed under the tradename Montanox 40 DF by the company Seppic, sorbitan laurate 20 EO or Polysorbate 20 marketed under the tradename Tween 20 by the company ICI, and sorbitan monostearate. \\n     The formulations according to the present invention may further be pressed into tablets, whereby one or more excipients and/or adjuvants selected from the group consisting of monosaccharides, disaccharides, oligosaccharides and polysaccharides, glycerol, and triglycerides, may be added. \\n     Preferred examples of mono- and disaccharides which may be present in the compositions of the present invention are sucrose, invert sugar, xylose, glucose, fructose, lactose, maltose, saccharose and sugar alcohols. \\n     Preferred examples of the oligo- and polysaccharides are starch, modified starch and starch hydrolysates. Preferred examples of starch hydrolysates are dextrins and maltodextrins, especially those having the range of 5 to 65 dextrose equivalents (DE), and glucose syrup, especially such having the range of 20 to 95 DE. The term “dextrose equivalent” (DE) denotes the degree of hydrolysis and is a measure of the amount of reducing sugar calculated as D-glucose based on dry weight; the scale is based on native starch having a DE close to 0 and glucose having a DE of 100. \\n     The triglyceride is suitably a vegetable oil or fat, preferably corn oil, sunflower oil, soybean oil, safflower oil, rapeseed oil, peanut oil, palm oil, palm kernel oil, cotton seed oil, olive oil or coconut oil. \\n     Solid compositions may in addition contain an anti-caking agent, such as silicic acid or tricalcium phosphate and the like, and up to 10 weight-%, as a rule 2 to 5 weight-%, of water. \\n     Manufacture of the Composition \\n     An object of the present invention is also a process for the manufacture of the composition of the present invention which comprises the following steps: \\n     I) preparing an aqueous solution or colloidal solution of a (modified) rice endosperm protein, \\n     II) optionally adding at least a water-soluble excipient and/or adjuvant to the solution prepared in step I), \\n     III) preparing a solution or dispersion of at least an active ingredient, preferably of at least a fat-soluble active ingredient and/or colorant, and optionally at least a fat-soluble adjuvant and/or excipient, \\n     IV) mixing the solutions prepared in step I) to III) with each other, \\n     V) homogenising the thus resulting mixture, \\n     VI) optionally adding a cross-linking agent for cross-linking the (modified) rice endosperm protein, \\n     VIa) optionally submitting the mixture resulting after having performed step VI) to enzymatic treatment or heat treatment to cross-link the (modified) rice endosperm protein \\n     VII) optionally converting the dispersion obtained in step V) and/or VI) into a powder, \\n     VIII) optionally drying the powder obtained in step VII), \\n     IX) optionally submitting the dry powder to heat treatment or to enzymatic treatment to cross-link the (modified) rice endosperm protein, \\n     with the proviso that only step VIa) or step IX) is carried out, but not both, when step VI) is carried out. \\n     Step I \\n     This step is simply performed by adding water to the (modified) rice endosperm protein or vice versa, optionally under stirring. Alternatively homogenization may be possible via ultrasonication. \\n     Preferably the (modified) rice endosperm protein with the preferences as described above is used. \\n     Step II \\n     Water-soluble excipients and/or adjuvants that may be added are e.g. monosaccharides, disaccharides, oligosaccharides and polysaccharides, glycerol and water-soluble antioxidants. Examples of them are given above. \\n     Step III \\n     Active ingredients are those as described above. \\n     The (fat-soluble) active ingredient and/or colorant and optional fat-soluble excipients and adjuvents are either used as such or dissolved or suspended in a triglyceride and/or an (organic) solvent. \\n     Suitable organic solvents are halogenated aliphatic hydrocarbons, aliphatic ethers, aliphatic and cyclic carbonates, aliphatic esters and cyclic esters (lactones), aliphatic and cyclic ketones, aliphatic alcohols and mixtures thereof. \\n     Examples of halogenated aliphatic hydrocarbons are mono- or polyhalogenated linear, branched or cyclic C1- to C15-alkanes. Especially preferred examples are mono- or polychlorinated or -brominated linear, branched or cyclic C1- to C15-alkanes. More preferred are mono- or polychlorinated linear, branched or cyclic C1- to C15-alkanes. Most preferred are methylene chloride and chloroform. \\n     Examples of aliphatic esters and cyclic esters (lactones) are ethyl acetate, isopropyl acetate and n-butyl acetate; and γ-butyrolactone. \\n     Examples of aliphatic and cyclic ketones are acetone, diethyl ketone and isobutyl methyl ketone; and cyclopentanone and isophorone. \\n     Examples of cyclic carbonates are especially ethylene carbonate and propylene carbonate and mixtures thereof. \\n     Examples of aliphatic ethers are dialkyl ethers, where the alkyl moiety has 1 to 4 carbon atoms. One preferred example is dimethyl ether. \\n     Examples of aliphatic alcohols are ethanol, iso-propanol, propanol and butanol. \\n     Furthermore any oil (triglycerides), orange oil, limonen or the like and water can be used as a solvent. \\n     Fat-soluble excipients and/or adjuvants that may be added are e.g. corn oil, mono- or diglycerides of fatty acids, polyglycerol fatty acids, and middle chain triglycerides (“MCT”). \\n     Step IV In an alternative process of the present invention step III) is not carried out, but the active ingredient and the optional fat-soluble excipient and/or adjuvant is directly added to the solution of step I) or II). \\n     Step V \\n     For the homogenisation conventional technologies, such as high-pressure homogenisation, high shear emulsification (rotor-stator systems), micronisation, wet milling, microchanel emulsification, membrane emulsification or ultrasonification can be applied. Other techniques used for the preparation of compositions containing (fat-soluble) active ingredients and/or colorant for enrichment fortification and/or coloration of food, beverages, animal feed, cosmetics or pharmaceutical compositions are disclosed in EP-A 0 937 412 (especially paragraphs [0008], [0014], [0015], [0022] to [0028]), EP-A 1 008 380 (especially paragraphs [0005], [0007], [0008], [0012], [0022], [0023] to [0039]) and in U.S. Pat. No. 6,093,348 (especially column 2, line 24 to column 3, line 32; column 3, line 48 to 65; column 4, line 53 to column 6, line 60), the contents of which are incorporated herein by reference. \\n     Step VI \\n     The cross-linking agent is preferably selected from the group consisting of reducing sugars, glycoproteins, and glycopeptides. Thus an intermolecular cross-linking between the (modified) rice endosperm protein and the sugar or sugar part of the glycoprotein/glycopeptide is formed. Preferred examples of the cross-linking agent are the reducing sugars such as glucose, fructose, saccharose and xylose. \\n     Step VIa \\n     The cross-linking can be achieved by submitting mixtures additionally containing a crosslinking agent as described above to heat-treatment to cause cross-linking of the sugar with the protein in a Maillard type reaction, i.e. by thermally treatment, preferably at temperatures from about 30 to about 160° C., more preferably at temperatures form about 70 to about 100° C., most preferably at temperatures from about 80 to about 90° C. \\n     Cross-linking of the (modified) rice endosperm protein with the cross-linking agent can also be achieved by treatment with cross-linking enzymes (acyltransferases, EC 2.3, e.g. transglutaminase, EC 2.3.2.13, protein-glutamine:γ-glutamyltransferase), i.e. by enzymatically treatment, conveniently carried out at temperatures from about 0 to about 70° C., preferably at temperatures from about 20 to about 40° C. Preferably the enzymatic treatment according to step VIa) is a treatment with a cross-linking enzyme, particularly with a transglutaminase. \\n     Enzymatic cross-linking results in stable protein-containing polysaccharide networks, in the case of a transglutaminase by the formation of ε-(γ-glutamyl)-lysine isopeptide bonds. The use of glycoproteins or glycopeptides is preferred for the enzymatic cross-linking. \\n     Both techniques, heat-treatment to cause cross-linking of the sugar with the protein in a Maillard type reaction and enzymatic cross-linking can be used for the incorporation of lipophilic moieties and can be carried out either in a dried form of the composition (step IX), or in an aqueous solution or suspension (step VIa). The enzymatic cross-linking is preferably carried out in an aqueous solution or suspension. \\n     Step VII \\n     The so-obtained dispersion, which is an oil-in-water dispersion, can be converted after removal of the organic solvent (if present) into a solid composition, e.g. a dry powder, using any conventional technology such as spray drying, spray drying in combination with fluidised bed granulation (the latter technique commonly known as fluidised spray drying or FSD), or by a powder-catch technique whereby sprayed emulsion droplets are caught in a bed of an absorbent, such as starch, calcium silicate and silicon dioxide, and subsequently dried. \\n     Spray-drying may be performed at an inlet-temperature of from about 100 to about 250° C., preferably of from about 150° C. to about 200° C., more preferably of from about 160 to about 190° C., and/or at an outlet-temperature (product temperature) of from about 45 to about 160° C., preferably of from about 55 to about 110° C., more preferably of from about 65 to about 95° C. \\n     Step VIII \\n     The drying of the powder obtained in step VII is preferably carried out at a temperature of ≦100° C., preferably at a temperature of from 20 to 100° C., more preferably at a temperature of from 60 to 70° C. If the drying is performed in vacuum the temperature is lower. \\n     Step IX \\n     The cross-linking via heat-treatment is carried out as already described above for step VIa. The same applies for the enzymatic treatment, which is, however, preferably carried out in solution/suspension. \\n     Manufacture of the (Modified) Rice Endosperm Protein \\n     The present invention is also directed to a process for the manufacture of a (modified) rice endosperm protein starting from milled rice, whereby the rice bran was removed before milling, comprising the following steps a) to e) with the proviso that at least one of the steps b), c) and d) is carried out: \\n     a) preparing an aqueous solution or suspension of milled rice, whereby the rice bran was removed before milling, whereby the solution or suspension preferably has a dry mass content of from 0.1 to 30 weight-%, preferably from 10 to 15 weight-%, based on the total amount of the aqueous solution or suspension;\\n \\nb) optionally removing the non-protein part or the protein part of the milled rice, whereby the rice bran was removed before milling, to obtain the rice endosperm protein;\\n \\nc) optionally modifying the protein part of the milled rice, whereby the rice bran was removed before milling, to obtain modified rice endosperm protein;\\n \\nd) optionally isolating the (modified) rice endosperm protein;\\n \\ne) optionally converting the (modified) rice endosperm protein into a solid form.\\n \\n     In the context of the present invention “rice endosperm protein” means especially the product obtained by performing either steps a) and b); or steps a) and d); or steps a), b) and e); or steps a), b) and d); or steps a), d) and e); or steps a), b), d) and e). Preferred are the embodiments where steps a) and b) (and d) and/or e)) are performed, especially preferred are the embodiments where steps a), b) and d) (and e)) are performed. \\n     In the context of the present invention “modified rice endosperm protein” means especially the product where step c) is carried out, i.e. the product obtained by performing either steps a) and c); or steps a), b) and c); or steps a), c) and d); or steps a), c) and e); or steps a), b) c) and d); or steps a), c), d) and e); or steps a), b), c) and e); or steps a), b), c), d) and e). \\n     The modified rice endosperm protein (and its preferences) are more preferred than the rice endosperm protein. Most preferred is the product obtained by performing steps a), b), c) and d) (and e)). \\n     Step a) \\n     Milled rice, where the rice bran was removed before milling, is also known under the expression “rice flour”. \\n     This step is simply performed by adding water to the rice flour or vice versa, optionally by stirring vigorously (with a mechanical stirrer) until the rice flour is completely dispersed, or by homogenizing the rice flour suspension with a homogenizer, e.g. for 5 minutes at room temperature. \\n     Step b) \\n     Removing of the Non Protein Part \\n     Step b) may preferably be achieved by treating the rice flour with non-protein degrading enzymes, e.g. with a 0.5% aqueous suspension of TERMAMYL® at a temperature of 90° C. for 2 hours and then with a 0.1% aqueous suspension of a cellulase at a temperature of 50° C. for 30 minutes—without any pH adjustment (pH 6-7), deactivating the enzymes, separating and removing the non-protein part from the protein part of the rice flour. \\n     Preferred examples of non-protein degrading enzymes are starch-degrading enzymes such as α-amylases and cellulases, i.e. cellulose-degrading enzymes, and mixtures thereof. A preferred example of an α-amylase is TERMAMYL 120, Type L, commercially available from Novo Nordisk Biochem, North America, Inc., USA. Other preferred examples are LIQUZYME® Supra, commercially available from Novo Nordisk Biochem, North America, Inc., USA, Amylase S “Amano” 35 G, commercially available from Amano Pharmaceutical Co. Ltd., Nagoya, Japan, Multifect Cellulase, commercially available from Genencor International, Inc., USA, and Cellulase T “Amano” 4, commercially available from Amano Pharmaceutical Co. Ltd., Nagoya, Japan. \\n     The reaction of the enzymes can be stopped by neutralising the solution or suspension if an inorganic acid (e.g. hydrochloric acid) or an organic acid (e.g. citric acid) or base is used or by heating to denature the enzymes. \\n     The denaturation may be achieved by heating the solution to a temperature of from 80 to 95° C., preferably to a temperature of from 80 to 85° C. (especially at a low pH of from 3.5 to 4.5) for 10 to 15 minutes. Afterwards the solution may be cooled to 50° C. \\n     The separation of the non-protein part may be achieved by centrifugation (5000 g for 15 minutes) (whereby the non-protein part is in the water phase), followed by washing with deionized water. The rice endosperm protein remains in pellets. \\n     Removing of the Protein Part \\n     Alternatively a so-called “alkaline extraction” or a so-called “salt-extraction” may be performed before the centrifugation or filtration. \\n     “Alkaline extraction” means that first the pH of the solution or suspension of the rice flour is adjusted to a value of from 7 to 12, preferably to a value of from 8 to 10, more preferably to a value of about 9, with an alkali solution (e.g. an aqueous NaOH solution) at 40 to 60° C. for 3 hours. \\n     In cases where the protein yield is more important than the protein functionality it may be advantageous to adjust the pH preferably to a value of from 8 to 12, more preferably of from 9 to 12, even more preferably from 10 to 12. \\n     Preferably such a base has a concentration of about 0.1 to 5 M, preferably of about 0.5 to about 2 M. The base may be an inorganic base. Examples of inorganic bases are (earth) alkali hydroxides such as sodium hydroxide (preferred), potassium hydroxide and calcium hydroxide. \\n     A “salt-extraction” is similar to an “alkali-extraction”, but in addition to the base a salt such as sodium chloride is used. In a preferred embodiment of the invention an aqueous 0.08 M sodium chloride solution (adjusted to pH 11 with NaOH) is used as the extracting solvent. \\n     In both cases (alkaline or salt extraction) the protein part is transferred to the water phase. The protein part may be separated then by centrifugation or filtration from the non-protein part. \\n     Step c) \\n     The modification of the rice flour may be achieved by treating it(s protein part) with (commercially available) food grade alkaline, neutral and/or acid proteases. For some proteases the enzyme specifications and the optimum conditions are given in the examples. \\n     The proteases may be from bacteria or fungi, as well as from fruit or may have animal origin. \\n     Examples of alkaline proteases are the commercially available ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA), ALKALINE PROTEASE® (Enzyme Development Corporation, New York, N.Y., USA), PROTEX 6L® (GENENCOR® Bacterial Alkaline Protease, Genencor International, Inc., Rochester, N.Y., USA) and GENENCOR® Protease 899 (Genencor International, Inc., Rochester, N.Y., USA). \\n     Examples of neutral proteases are the commercially available BROMELAIN® (Enzyme Development Corporation, New York, N.Y., USA), LIQUIPANOL® (Enzyme Development Corporation, New York, N.Y., USA) and bacterial neutral-protease (Genencor International, Inc., Rochester, N.Y., USA). A further example of a neutral protease is the commercially available COLLUPILIN® of DSM Food Beverages, Delft, Netherlands, produced from  Carica papaya , a plant, i.e. an enzyme of fruit origin. \\n     Examples of acid proteases are pepsin (Sigma, USA) and Acid protease (Amano Pharmaceutical Co. Ltd., Nagoya, Japan). \\n     In a preferred embodiment of the process of the present invention the protein part of the rice flour is treated subsequently by two different alkaline proteases at a pH range of from 7 to 10 for 10 to 80 minutes at 40 to 60° C. \\n     Preferably one of these proteases is a serine specific protease such as ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), PROTEX 6L or ALKALINE PROTEASE and the other is a cysteine specific protease such as LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or BROMELAIN. \\n     The step may also be modified by not adding the enzyme(s) at once but by adding them (subsequently or simultaneously) portion wise. \\n     Step c) may also be performed after step d), i.e. first the rice endosperm protein is isolated and then it is modified. \\n     Step d) \\n     Step d) is preferably carried out by centrifugation and/or filtration, preferably by ultrafiltration. The ultrafiltration may be carried out without prior centrifugation. \\n     After enzyme inactivation, the hydrolyate may be preferably centrifuged at low speed (1000 g for 10 minutes) to separate insoluble proteins and impurities. \\n     The soluble fractions of the protein hydrolysates may then be filtered through Whatman #0.4 filter paper and the filtrate be subjected to a sequential ultrafiltration (UF) and diafiltration (DF) with membranes with a molecular weight cut-off (MWCO) of ≧5 kDa, preferably with membranes with a molecular weight cut-off (MWCO) of ≧30 kDa, more preferably with membranes with a molecular weight cut-off (MWCO) of ≧50 kDa, most preferably with membranes with a molecular weight cut-off (MWCO) of from 50 to 750 kDa. \\n     Ultrafiltration and diafiltration may be performed at room temperature with an inlet pressure of from 20 to 25 psi and an outlet-pressure of 10 psi. The solution may then be ultrafiltrated to a concentration factor of 5 Immediately after ultrafiltration, the retenate may be diafiltrated two times with twice the volume of deionized water. During ultrafiltration and diafiltration the pH of the solution may be maintained between pH 8.0 and 9.0 in order to keep the protein soluble. \\n     Step e) \\n     The conversion into a solid form, e.g. a dry powder, can be achieved by any drying method known to the person skilled in the art. Preferred are spray drying or freeze-drying. Spray drying is preferably performed at an inlet temperature of about 200° C. to about 210° C. and at an outlet temperature of about 70° C. to about 75° C. The freeze-drying is preferably performed at a temperature of from about −20° C. to about −50° C. for 10 to 48 hours. \\n     An object of the present invention is also the (modified) rice endosperm protein obtainable by any process as described above. \\n     INDUSTRIAL APPLICABILITY \\n     The present invention is directed to the use of a composition as described above for the enrichment, fortification and/or coloration of food, beverages, animal feed, personal care or pharmaceutical compositions, as well as to the food, beverages, animal feed, personal care and pharmaceutical compositions containing such a composition as described above themselves. \\n     The present invention is also directed to food, beverages, animal feed, personal care and pharmaceutical compositions containing a (modified) rice endosperm protein as described above, as well as to the use of such a (modified) rice endosperm protein, preferably such as described above, as protective hydrocolloid for active ingredients, especially fat-soluble active ingredients and/or colorants. \\n     Animals including humans in the context of the present invention encompass besides humans especially farm animals such as sheep, cow, horses, poultry (broiler and egg pigmentation), shrimps and fish (especially salmon and rainbow trout) as well as pets such as cat, dogs, birds (e.g. flamingos) and fish. \\n     Beverages wherein the compositions of the present invention can be used, especially as a colorant or a functional ingredient, can be carbonated beverages e.g., flavoured seltzer waters, soft drinks or mineral drinks, as well as non-carbonated beverages e.g. flavoured waters, fruit juices, fruit punches and concentrated forms of these beverages. They may be based on natural fruit or vegetable juices or on artificial flavours. Also included are alcoholic beverages and instant beverage powders. Besides, sugar containing beverages, diet beverages with non-caloric and artificial sweeteners are also included. \\n     Further, dairy products, obtained from natural sources or synthetic, are within the scope of the food products wherein the compositions of the present invention can be used, especially as a colorant or as a functional ingredient. Typical examples of such products are milk drinks, ice cream, cheese, yoghurt and the like. Milk replacing products such as soymilk drinks and tofu products are also comprised within this range of application. \\n     Also included are sweets which contain the compositions of the present invention as a colorant or as a functional ingredient, such as confectionery products, candies, gums, desserts, e.g. ice cream, jellies, puddings, instant pudding powders and the like. \\n     Also included are cereals, snacks, cookies, pasta, soups and sauces, mayonnaise, salad dressings and the like which contain the compositions of the present invention as a colorant or a functional ingredient. Furthermore, fruit preparations used for dairy and cereals are also included. \\n     The final concentration of the (fat-soluble) active ingredient and/or the colorant which is added via the compositions of the present invention to the food products may be from 0.1 to 500 ppm, particularly from 1 to 50 ppm, based on the total weight of the food composition and depending on the particular food product to be coloured or fortified and the intended grade of coloration or fortification. \\n     The food compositions of this invention are preferably obtained by adding to a food product the (fat-soluble) active ingredient and/or the colorant in the form of a composition of this invention. For coloration or fortification of a food or a pharmaceutical product a composition of this invention can be used according to methods per se known for the application of water dispersible solid compositions of the present invention. \\n     In general the composition may be added either as an aqueous stock solution, a dry powder mix or a pre-blend with other suitable food ingredients according to the specific application. Mixing can be done e.g. using a dry powder blender, a low shear mixer, a high-pressure homogeniser or a high shear mixer depending on the formulation of the final application. As will be readily apparent such technicalities are within the skill of the expert. \\n     Pharmaceutical compositions such as tablets or capsules wherein the compositions are used as a colorant are also within the scope of the present invention. The coloration of tablets can be accomplished by adding the compositions of the present invention in form of a liquid or solid colorant composition separately to the tablet coating mixture or by adding a colorant composition to one of the components of the tablet coating mixture. Coloured hard or soft-shell capsules can be prepared by incorporating a colorant composition in the aqueous solution of the capsule mass. \\n     Pharmaceutical compositions such as tablets such as chewable tablets, effervescent tablets or filmcoated tablets or capsules such as hard shell capsules wherein the compositions are used as an active ingredient are also within the scope of the present invention. The compositions of the present invention are typically added as powders to the tableting mixture or filled into the capsules in a manner per se known for the production of capsules. \\n     Animal feed products such as premixes of nutritional ingredients, compound feeds, milk replacers, liquid diets or feed preparations wherein the compositions are either used as a colorant for pigmentation e.g. for egg yolks, table poultry, broilers or aquatic animals (especially shrimps, salmon, rainbow trout) or as an active ingredient are also within the scope of the present invention. \\n     Personal care compositions: Cosmetics, toiletries and derma products i.e. skin and hair care products such as creams, lotions, baths, lipsticks, shampoos, conditioners, sprays or gels wherein the compositions are used as a colorant or as an active ingredient are also within the scope of the present invention. \\n     The present invention is further illustrated by the following examples. \\n     EXAMPLES \\n     The following abbreviations are used: \\n     UF=ultrafiltration \\n     DF=diafiltration \\n     DH=degree of hydrolysis \\n     DI water=deionized water \\n     MWCO=molecular weight cut off \\n     SDS=sodium dodecyl sulfate \\n     Rice flour was obtained from Riceland Foods (Stuttgart, Ark.). The following food grade enzymes were used in isolating protein. (1) TERMAMYL—Heat stable alpha-amylase, Novo Nordisk Biochem, North America, Inc, USA, (3) Multifect Cellulase—Fungal Cellulase, 2,000 IU/g, Genencor International, Inc., USA. Food grade proteases with the following specifications were used (Table 1a &amp; 1b). \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 1a \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Enzyme specification I \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 Type of \\n                   \\n                   \\n               \\n               \\n                 Enzyme \\n                 protease \\n                 Source \\n                 Preferential specificity \\n               \\n               \\n                   \\n               \\n               \\n                 Protex 6L \\n                 Serine Protease \\n                 \\n                   Bacillus \\n                 \\n                 Hydrolysis of proteins with broad \\n               \\n               \\n                   \\n                   \\n                 \\n                   licheniformis \\n                 \\n                 specificity for peptide bonds \\n               \\n               \\n                 Bromelain \\n                 Cysteine \\n                 Pineapple \\n                 Broad specificity, but strong preference \\n               \\n               \\n                   \\n                 Protease \\n                 stem \\n                 for Arg-Arg in peptides \\n               \\n               \\n                 ALKALASE\\u2009® \\n                 Serine Protease \\n                 \\n                   Bacillus \\n                 \\n                 Broad specificity, and a preference for a \\n               \\n               \\n                 (Novo Nordisk \\n                   \\n                 \\n                   licheniformis \\n                 \\n                 large uncharged residue&#39;s carboxyl sites \\n               \\n               \\n                 Biochem,  \\n                   \\n                   \\n                   \\n               \\n               \\n                 Franklinton, NC, \\n                   \\n                   \\n                   \\n               \\n               \\n                 USA) \\n                   \\n                   \\n                   \\n               \\n               \\n                 LIQUIPANOL\\u2009® \\n                 Cysteine \\n                 Concentrated \\n                 Broad specificity \\n               \\n               \\n                 (Enzyme \\n                 Protease \\n                 papain \\n                   \\n               \\n               \\n                 Development \\n                   \\n                   \\n                   \\n               \\n               \\n                 Corporation, New \\n                   \\n                   \\n                   \\n               \\n               \\n                 York, NY, USA) \\n                   \\n                   \\n                   \\n               \\n               \\n                 Alkaline protease \\n                 Serine Protease \\n                 Bacterial protease \\n                 Hydrolysis of proteins with broad \\n               \\n               \\n                   \\n                   \\n                   \\n                 specificity for peptide bonds \\n               \\n               \\n                 Pepsin \\n                 Aspartic \\n                 Porcine stomach \\n                 The C-terminal side of tyrosine, \\n               \\n               \\n                   \\n                 Protease \\n                   \\n                 phenylalanine, and tryptophan residues \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 1b \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Enzyme specification II \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 pH- \\n                   \\n                   \\n               \\n               \\n                 Enzymes \\n                 range \\n                 Activity/g \\n                 Company \\n               \\n               \\n                   \\n               \\n               \\n                 Protex 6L \\n                 \\u20026-10 \\n                 580 000 \\n                 Genencor International, Inc., \\n               \\n               \\n                   \\n                   \\n                   \\n                 Rochester, NY 14618, USA \\n               \\n               \\n                 Bromelain \\n                 5-8 \\n                 150 000 \\n                 Enzyme Development \\n               \\n               \\n                   \\n                   \\n                   \\n                 Corporation, New \\n               \\n               \\n                   \\n                   \\n                   \\n                 York, NY 10001, USA. \\n               \\n               \\n                 ALKALASE\\u2009® \\n                 6-9 \\n                 2.4 AU \\n                 Novo Nordisk Biochem, \\n               \\n               \\n                 (Novo Nordisk \\n                   \\n                   \\n                 Franklinton, \\n               \\n               \\n                 Biochem,  \\n                   \\n                   \\n                 NC 27525, USA \\n               \\n               \\n                 Franklinton, NC, \\n                   \\n                   \\n                   \\n               \\n               \\n                 USA) \\n                   \\n                   \\n                   \\n               \\n               \\n                 LIQUIPANOL\\u2009® \\n                 5-8 \\n                 125 000 \\n                 Enzyme Development \\n               \\n               \\n                 (Enzyme \\n                   \\n                   \\n                 Corporation, New \\n               \\n               \\n                 Development \\n                   \\n                   \\n                 York, NY 10001, USA. \\n               \\n               \\n                 Corporation, New \\n                   \\n                   \\n                   \\n               \\n               \\n                 York, NY, USA) \\n                   \\n                   \\n                   \\n               \\n               \\n                 Alkaline \\n                 6-9 \\n                 175 000 \\n                 Enzyme Development \\n               \\n               \\n                 protease \\n                   \\n                   \\n                 Corporation, New \\n               \\n               \\n                   \\n                   \\n                   \\n                 York, NY 10001, USA. \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     SUMMARY \\n     Experiments were conducted with optimization of isolating rice endosperm proteins by alkali, salt, and enzymatic methods and evaluating them for their extractability and physicochemical and functional properties. The objectives were to optimize the extraction process, and evaluate the physicochemical properties of the extracts in order to identify the optimum extraction methods for improved rice protein functionality. \\n     Rice protein isolates were prepared by chemical and enzymatic methods. Preliminary trials were conducted to optimize the extraction conditions of protein with maximum yield and protein content. Examples of the procedures of chemical and enzymatic methods are given in  FIG. 1  of rice protein isolate preparation RPA and for rice protein isolate preparation (RP ET ) by TERMAMYL treatment at 90° C. followed by cellulase treatment in  FIG. 2 . \\n     Isolation of the (Modified) Rice Endosperm Protein by Filtration \\n     The ultrafiltration experiments were carried out with a Romicon ultrafiltration system (Koch membrane systems, USA) equipped with 1″ diameter Hollow-fiber Polysulfone membrane cartridges. The soluble fractions of (modified) rice endosperm protein were filtered through Whatman #0.4 filter paper and the filtrate was subjected to a sequential ultrafiltration with membrane-cartridges having nominal molecular weight cut off (MWCO) of 50, 30, 10, and 5 kDa. In each MWCO cartridges, the solution was ultrafiltrated at a concentration factor of 5 Immediately after ultrafiltration, the retenate was diafiltrated two times with twice the volume of deionized water. The permeates of the first (50 kDa) ultrafiltration (UF) and diafiltrations (DF) were pooled and subjected to next MWCO (30 kDa) UF &amp; DF. The resulted retenates from each of the MWCO cartridges were freeze-dried and evaluated for DH, solubility &amp; emulsifying properties. \\n     (A) Protein Extraction \\n     Example 1 \\n     Isolation of the Rice Endosperm Protein by Alkaline Extraction (“Chemical Extraction”) \\n     Example 1-1 \\n     REP 1-1 \\n     Five hundred grams of rice flour were homogenized with 4 L deionized water (1:8, w/v) in a blender for 5 minutes at setting 6 (Virtishear Tempest, The Virtis Co., Gardiner, N.Y., U.S.A.) at room temperature. The slurry was adjusted to pH 11.0 using 1 N NaOH, and the suspension was stirred for 3 hours at 40° C. The solubilized protein in the solution was separated by centrifugation (5,000 g, 20 minutes). This procedure was repeated once again to extract more from the residue. Proteins in combined supernatants of first and second extractions were isoelectrically precipitated at pH 4.5 and kept for 2 hours at 4° C. The precipitate was recovered by centrifugation at 10,000 g for 20 minutes, washed twice with deionized water (1:4, w/v, pH 4.5), adjusted to pH 7.0, freeze dried, and stored at 5° C. \\n     Alternative Example 1-2 \\n     REP 1-2 \\n     5 Kg rice flour was homogenized with 40 L of deionized water (1:8, w/v), and the pH of the slurry was adjusted to pH 11 by adding 3 N NaOH, and the suspension was stirred for 3 hours at 40° C. The solubilized protein in the solution was separated by centrifugation (5000 g, 15 min) This procedure was repeated one more time to extract more from the residue. Proteins in the combined supernatants of first and second extractions were isoelectrically precipitated at pH 4.5 and kept for 2 hours at 4° C. The precipitate was recovered by centrifugation at 5000 g for 20 min, washed twice with deionized water (1:4, w/v, pH 4.5), adjusted to pH 7.0, and stored at 5° C. \\n     Alternative Example 1-3 \\n     REP 1-3 \\n     One kilogram of rice flour was homogenized with 8 L deionized water (1:8, w/v) in a homogenizer (Virtishear Tempest, The Virtis Co., Gardiner, N.Y., U.S.A.) for 5 min. The pH of the slurry was adjusted to 11 using 1 N NaOH and the suspension was stirred for 3 h at 40° C. The solubilized protein in the solution was separated by centrifugation (2000 g, 15 min). This procedure was repeated to extract additional protein from the residue. Proteins in combined supernatants of first and second extractions were isoelectrically precipitated at pH 4.5 and kept at 4° C. for 1 h. The precipitate was recovered by centrifugation at 5000 g for 20 min, washed with deionized water (1:4, w/v, pH 4.5), adjusted to pH 7.0, freeze dried (RPA), and stored at 5° C. ( FIG. 1 ). \\n     Example 1-4 \\n     (REP 1-4): Isolation of the rice endosperm protein by salt extraction \\n     Salt extraction of rice protein (RP S ) was similar to the above alkali method (see REP 1-3) but a combined solution of 0.08 M sodium chloride (adjusted to pH 11 with NaOH) was used as the extracting solvent. \\n     Example 2 \\n     Isolation of the Rice Endosperm Protein by Enzymatic Degradation of the Non-Protein Part (“Enzymatic Extraction”) \\n     Example 2-1 \\n     REP 2-1 \\n     Five hundred grams of rice flour were dispersed in 3 L distilled water (1:6, w/v). The rice flour-water was stirred for 15 minutes at 40° C. to form a slurry. The slurry was treated with 0.5% alpha-amylase and the temperature was gradually raised to 90° C. and incubated for 2 hours at 90° C. Solubilized starch was removed by centrifugation at 5,000×g for 15 minutes at 20° C. The residue was again mixed with 1.5 L deionized water and treated with 0.1% cellulase and incubated at 50° C. for 30 minutes. The enzymes were inactivated by lowering the pH to 3.5 at 90° C. Solubilized starch and cellulose fractions were removed by centrifugation at 5000×g for 15 minutes. The precipitated protein was washed twice with warm deionized water to remove soluble sugar and remaining enzyme, adjusted to pH 7.0, freeze-dried, and stored at 5° C. \\n     Alternative Example 2-2 \\n     REP 2-2 \\n     5 Kg of rice flour was homogenized with 30 L deionized water and stirred for 15 min at 60° C. The slurry was treated with 0.2% TERMAMYL, and the temperature was gradually raised to 90° C. and incubated at that temperature for 2 hours. The solubilized starch was removed by filtering with cheese cloth. The pellet was again mixed with 15 L deionized water, and incubated with 0.1% pectinase at 50° C. for 30 min, and added another 0.1% enzyme TERMAMYL. The temperature was gradually raised to 90° C. and incubated at that temperature for 30 min. The enzymes were inactivated by lowering the pH to 5.0 at 90° C. The solubilized cell wall components and residual starch were removed by filtering with cheese cloth. The protein pellet was washed twice with warm deionized water to remove remaining soluble sugars and enzymes, adjusted to pH 7.0, stored at 5° C. \\n     Alternative Example 2-3 \\n     REP 2-3 \\n     One kilogram of rice flour was homogenized with 6 L deionized water and stirred for 15 min at 60° C. The slurry was treated with 0.5% TERMAMYL, and the temperature was gradually raised to 90° C. and incubated at that temperature for 2 h. The solubilized starch was removed by centrifugation (5000 g, 15 min) The pellet was again mixed with 3 L deionized water and incubated with 0.1% cellulase at 50° C. for 30 min. The enzymes were inactivated by lowering the pH to 4.5 at 90° C. Then a solubilized cellulose fraction was removed by centrifugation. The precipitated protein was washed twice with warm deionized water to remove soluble sugar and remaining enzyme, adjusted to pH 7.0, freeze-dried (RP ET ), and stored at 5° C. ( FIG. 2 ). \\n     Alternative Example 2-4 \\n     REP 2-4 \\n     The amylase S (RP EA ) extraction was essentially the same as the TERMAMYL method (see REP 2-3) except that amylase S was used instead of TERMAMYL. The amylase S has an optimum activity at 70° C., providing a milder extraction condition than the TERMAMYL method working at 90° C. The moisture, protein, starch, fiber, fat, and ash contents of the protein isolates were determined as disclosed in Approved Methods of the American Association of Cereal Chemists, 8th Ed., Vol. 2, AACC, St. Paul, Minn., 1990, p 1-2; Method 46-08. \\n     (B) Protein modification \\n     Example 3 \\n     Modification of Alkali-Extracted Rice Endosperm Protein by Protease \\n     REP 3-1 to 3-8 \\n     Detailed Description for Preparation of REP 3-1 to REP 3-5 \\n     Alkali extracted rice endosperm protein isolate REP 1-1 was subjected to protease treatment. The protein was mixed with deionized water (1:12.5, w/v), homogenized, adjusted to the optimum pH of each enzyme, and stirred at 50° C. for 10 minutes. The dispersion was treated with food grade proteases under optimized conditions of each enzyme. The optimized conditions for each enzyme are given in Table 2. Proteases of commercial food grade (BROMELAIN (REP 3-1), PROTEX 6 L (REP 3-2), pepsin (REP 3-3), ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) (REP 3-4), and LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) (REP 3-5)), were used to improve the solubility and functional properties. The degree of hydrolysis of the resulting hydrolysates has been optimized by OPA method (Nielsen et al, Journal of Food Science 2001, 66(5), 642-646) to maximize the functional properties. The enzymatic-reactions were terminated at required DH by the specific inactivation conditions of each enzyme. The hydrolyzed products were spray dried and stored at 5° C. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 2 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 The optimum enzyme conditions used in protein hydrolysis \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 Amount of Enzyme \\n                   \\n                   \\n                   \\n               \\n               \\n                   \\n                 [weight- \\n                   \\n                   \\n                   \\n               \\n               \\n                   \\n                 %, based on \\n                   \\n                 Temperature \\n                 Time \\n               \\n               \\n                 Enzyme \\n                 protein weight] \\n                 pH \\n                 [° C.] \\n                 [minutes] \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 LIQUIPANOL\\u2009® \\n                 1.0 \\n                 8.0 \\n                 50 \\n                 60 \\n               \\n               \\n                 (Enzyme \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Development \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Corporation, New \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 York, NY, USA) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Bromelain \\n                 1.0 \\n                 7.0 \\n                 50 \\n                 60 \\n               \\n               \\n                 ALKALASE\\u2009® \\n                 1.0 \\n                 9.0 \\n                 60 \\n                 60 \\n               \\n               \\n                 (Novo Nordisk \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Biochem,  \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Franklinton, NC, \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 USA) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Protex 6L \\n                 1.0 \\n                 10.0 \\n                 60 \\n                 60 \\n               \\n               \\n                 Pepsin \\n                 0.5 \\n                 3.0 \\n                 37 \\n                 30 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n       FIG. 4  shows the hydrolysis profiles of the rice endosperm protein treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or Pepsin as a function of the hydrolysis time. \\n       FIG. 7  shows the molecular size profiles of rice proteins and hydrolysate produced by treatment with pepsin, LIQUIPANOL® (Enzyme Development Corporation, New York, N.Y., USA), and ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) determined by SDS PAGE gel electrophoresis: \\n     Lane 1—Standard marker; \\n     Lane 2—Rice protein (control); \\n     Lane 3—Pepsin 2.2% DH; \\n     Lane 4—Pepsin 6.5% DH; \\n     Lane 5—LIQUIPANOL® (Enzyme Development Corporation, New York, N.Y., USA) 3.9% DH; \\n     Lane 6—LIQUIPANOL® (Enzyme Development Corporation, New York, N.Y., USA) 10.5% DH; \\n     Lane 7—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 7.7% DH; \\n     Lane 8—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 14.7% DH. \\n       FIG. 8  shows the molecular size profiles of ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) treated rice protein hydrolysate determined by SDS PAGE gel electrophoresis: \\n     Lane 1—Standard marker; \\n     Lane 2—Rice protein (control); \\n     Lane 3—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 5.2% DH; \\n     Lane 4—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 7.7%; \\n     Lane 5—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 11.2%; \\n     Lane 6—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 13.5%; \\n     Lane 7—ALKALASE® (Novo Nordisk Biochem, Franklinton, N.C., USA) 14.7% DH. \\n     Detailed Description for Preparation of REP 3-6, 3-7, and 3-8 \\n     REP 1-2 was used for the preparation of REP 3-6, REP 3-7 and REP 3-8 instead of REP 1-1 as in examples REP 3-1, REP 3-2, REP 3-3, REP 3-4 and REP 3-5. \\n     REP 3-6 \\n     Alkali extracted rice protein isolate REP 1-2 was homogenized with DI water (8% w/v) and, adjusted to pH 8.0. The protein solution was treated with 1% ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) at 50° C. for 3.5 min. The enzyme was inactivated at 70° C. at pH 5.0 for 15 minutes. The protein hydrolysate (REP 3-6) was spray dried and stored at 5° C. \\n     REP 3-7 \\n     Alkali extracted rice protein isolate REP 1-2 was homogenized with DI water (8% w/v) and, adjusted to pH 8.0. The protein solution was treated with 1% ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) at 50° C. for 7.5 min. The enzyme was inactivated at 70° C. at pH 5.0 for 15 minutes. The protein hydrolysate (REP 3-7) was spray dried and stored at 5° C. \\n     REP 3-8 \\n     Alkali extracted rice protein isolate REP 1-2 was homogenized with DI water (8% w/v) and, adjusted to pH 8.0. The protein solution was treated with 1% ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) at 50° C. for 11 minutes. The enzyme was inactivated at 70° C. at pH 6.0 for 15 minutes. The protein hydrolysate (REP 3-8) was spray dried and stored at 5° C. \\n     Example 4 \\n     Modification of alkali-extracted rice endosperm protein by two type proteases (REP 4) \\n     Alkali extracted rice endosperm protein isolate (REP 1) was subjected to a combined treatment with serine and cysteine protease. The protein was mixed with deionized water (1:12.5, w/v), homogenized, adjusted to pH 9.0 and stirred at 60° C. for 10 minutes. The dispersion was treated with food grade ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (0.5%) and incubated at 60° C. for 15 minutes. Then the LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) (0.5%) was added to the reaction mixture and incubated at 50° C. for 15 minutes. The enzymatic-reactions were terminated at 80° C. for 10 minutes and the modified rice endosperm protein was spray dried and stored at 5° C. \\n     Example 5 \\n     Modification of Alkali-Extracted Rice Endosperm Protein by Protease and Subsequent Ultra- and Diafiltration of Said Modified Rice Endosperm Protein (REP 5) \\n     Alkali extracted rice endosperm protein isolate (REP 1) was subjected to extended hydrolysis by a combined treatment with serine and cysteine protease: The protein was mixed with deionized water (1:12.5, w/v), homogenized, adjusted to pH 9.0 and stirred at 60° C. for 10 minutes. The dispersion was treated with food grade ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (0.5%) and incubated at 60° C. for 60 minutes. Then the LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) (0.5%) was added to the reaction mixture and incubated at 50° C. for 60 minutes. The enzymatic-reactions were terminated at 80° C. for 10 minutes. The thus modified rice endosperm protein was filtered through Whatman #0.5 filter paper. The filtrate was subjected to ultrafiltration with a MWCO of 50 kDa Hollow-fiber Polysulfone membrane-cartridge equipped in a Romicon ultrafiltration system (Koch membrane systems, USA) with a capacity of 2-10 L. The solution was ultrafiltrated at a concentration factor of 10 Immediately after ultrafiltration, the residue was diafiltrated two times with twice the volume of deionized water. The resulted residues were spray-dried. \\n     Example 6 \\n     Isolation of the Rice Endosperm Protein by Enzymatic Degradation of the Non-Protein Part, Modification of Said Rice Endosperm Protein by Treatment with a Protease and Subsequent Ultra- and Diafiltration of Said Modified Rice Endosperm Protein (REP 6-1 to REP 6-5) \\n     REP 6-1 \\n     Enzymatically extracted rice endosperm protein isolate obtained in example 2 (REP 2-1) was subjected to extended hydrolysis by a combined treatment with a serine and a cysteine protease: The protein was mixed with deionized water (1:12.5, w/v), homogenized, adjusted to pH 9.0 and stirred at 60° C. for 10 minutes. The dispersion was treated with food grade ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (0.5%) and incubated at 60° C. for 60 minutes. Then the LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) (0.5%) was added to the reaction mixture and incubated at 50° C. for 90 minutes. The enzymatic-reactions were terminated at 80° C. for 10 minutes. The hydrolyate was centrifuged at low speed (1000 g for 10 minutes) to separate insoluble proteins and impurities. The soluble portion of the modified rice endosperm protein were filtered through Whatman #0.4 filter paper and subjected to ultrafiltration with a MWCO of 50 kDa. The solution was ultrafiltrated at a concentration factor of 10 Immediately after ultrafiltration, the retenate was diafiltrated two times with twice the volume of deionized water. During the process of enzymatic hydrolysis, centrifugation, and in ultrafiltration the pH of the (modified) rice endosperm protein was maintained at 8.0 in order to keep the protein soluble. The resulting retentates were spray-dried (REP 6-1). \\n     Detailed Description for Preparation of REP 6-2 \\n     Enzymatically extracted rice endosperm protein isolate obtained in example 2 (REP 2-2) was subjected to ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) treatment. The protein isolate was mixed with DI water (4%), homogenized, adjusted to pH 9.0 and stirred at 50° C. for 10 min. The dispersion was treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (1%) and the mixture was incubated at 50° C. for 8 min. The enzymatic-reactions were terminated at 70° C. for at pH 5.0 for 15 min. The hydrolysate was centrifuged at low speed (1000 rpm, 1 min) to remove insoluble proteins and impurities. The soluble portion of the hydrolysates was subjected to ultrafiltration with 10 kDa MWCO membrane. The solution was ultrafiltrated to a concentration factor of 5. The resulted retentate (≧10 kDa) was spray-dried and stored at 5° C. (REP 6-2). \\n     Detailed Description for Preparation of REP 6-3 \\n     Enzymatically extracted rice endosperm protein isolate obtained in example 2 (REP 2-2) was subjected to ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) treatment. The protein isolate was mixed with DI water (4%), homogenized, adjusted to pH 9.0 and stirred at 50° C. for 10 min. The dispersion was treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (1%) and the mixture was incubated at 50° C. for 12 min. The enzymatic-reactions were terminated at 70° C. for at pH 5.0 for 15 min. The hydrolysate was centrifuged at low speed (1000 rpm, 1 min) to remove insoluble proteins and impurities. The soluble portion of the hydrolysates was subjected ultrafiltration with 10 kDa MWCO membrane. The solution was ultrafiltrated to a concentration factor of 5. The resulted retentate (&gt;10 kDa) was spray-dried and stored at 5° C. (REP 6-3). \\n     Detailed Description for Preparation of REP 6-4 \\n     Enzymatically extracted rice endosperm protein isolate obtained in example 2 (REP 2-2) was subjected to ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) treatment. The protein isolate was mixed with DI water (4%), added sodium meta-bisufite (20-mg/g protein), homogenized, adjusted to pH 9.0 and stirred at 50° C. for 10 min. The dispersion was treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (1%) and the mixture was incubated at 50° C. for 8 min. The enzymatic-reactions were terminated at 70° C. for at pH 5.0 for 15 min. The hydrolysate was centrifuged at low speed (1000 rpm, 1 min) to remove insoluble proteins and impurities. The soluble portion of the hydrolysates was subjected to ultrafiltration with 10 kDa MWCO membrane. The solution was ultrafiltrated to a concentration factor of 5. The resulted retentate (&gt;10 kDa) was spray-dried and stored at 5° C. (REP 6-4). \\n     Detailed Description for Preparation of REP 6-5 \\n     Enzymatically extracted rice endosperm protein isolate obtained in example 2 (REP 2-2) was subjected to ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) treatment. The protein isolate was mixed with DI water (4%), added sodium meta-bisufite (10-mg/g protein), homogenized, adjusted to pH 9.0 and stirred at 50° C. for 10 min. The dispersion was treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) 2.4 L (1%) and the mixture was incubated at 50° C. for 30 min. The enzymatic-reactions were terminated at 70° C. for at pH 5.0 for 15 min. The hydrolysate was centrifuged at low speed (1000 rpm, 1 min) to remove insoluble proteins and impurities. The soluble portion of the hydrolysates was subjected to ultrafiltration with 10 kDa MWCO membrane. The solution was ultrafiltrated to a concentration factor of 5. The resulted retentate (&gt;10 kDa) was spray-dried and stored at 5° C. (REP 6-5). (C) Pepsin assisted rice protein extraction followed by ultrafiltration \\n     Example 7 \\n     Modification of the Rice Endosperm Protein by an Acid Protease and Sub-Sequent Ultrafiltration of Said Modified Rice Endosperm Protein (REP 7) \\n     Five hundred grams of rice flour were homogenized with 3 L deionized water (1:6, w/v) in a blender for 5 minutes at room temperature. The slurry was adjusted to pH 2.4 using 3 N HCl, and the suspension was stirred for 30 minutes at 37° C. The slurry was treated with 0.44% pepsin and incubated at 37° C. for 130 minutes. The solubilized modified rice endosperm protein in the solution was separated by centrifugation (5,000 g, 20 minutes). The residual rice flour was mixed with 500 ml deionized water and the soluble portion was separated by centrifugation to extract more from the residue. Proteins in combined supernatants of first and second extractions were subjected to ultrafiltration using a membrane with a MWCO of 50 kDa as described in example 5. The combined UF &amp; DF excluded the small protein fragments and other impurities effectively. \\n     (D) Analytical Methods \\n     Example 8 \\n     Determination of the Emulsion Capacity \\n     The emulsion capacity of the (modified) rice endosperm protein was determined according to the method of Gbogouri et al, Journal of Food Science 2004, Vol. 69, Nr. 8, 615, based on oil titration. Dispersions (0.1% w/w, 50 mL, pH 7.0) of the (modified) rice endosperm protein were prepared in deionized water. The protein solution was homogenized with a homogenizer at setting 1 (Virtishear Tempest, The Virtis Co., Gardiner, N.Y., U.S.A.). Corn oil was added into the protein solution with a flow rate of about 17 g/min using a peristaltic pump. The conductivity of the emulsion was recorded continuously by a conductivity-Meter and used as a parameter for the determination of the inversion point of the emulsion. The amount of oil added to the inversion point was used to calculate the emulsifying capacity. The emulsifying capacity is expressed as the ratio of emulsified oil minus the blank over the amount of proteins in sample. The blank was the quantity of oil added before the phase inversion in 50 mL of deionized water. \\n     Alternatively, the emulsion capacity of protein hydrolysates REP 3-6, 3-7, 3-8, 6-2, 6-3, 6-4, 6-5) was determined according to the method of Vuillemard and others based on oil titration. Protein dispersions (0.5% w/w, 40 mL, pH 7.0) were prepared in distilled water. The protein solution was homogenized with homogenizer at setting 6 (Virtishear Tempest, The Virtis Co., Gardiner, N.Y., U.S.A.). Corn oil was added into the protein solution at about 12 g/min flow rate using a pump. The conductivity of the emulsion was recorded continuously by a conductivity-Meter and used as a parameter for the determination of the inversion point of the emulsion. The amount of oil that added up to the inversion point was used to calculate the emulsifying capacity. Emulsifying capacity is expressed as the ratio of emulsified oil minus the blank over the amount of proteins in sample. The blank was the quantity of oil added before the phase inversion in 40 mL distilled water. \\n     Example 9 \\n     Determination of the Emulsion Activity \\n     The emulsion activity was determined by the turbidimetric method of Pearce and Kinsella, Journal of Agric Food Chem. 1978, 26:716-722. A mixture of 6 mL of a 0.1% solution of the (modified) rice endosperm protein in 10 mM phosphate buffer of a pH of 7.0 and 2 mL of corn oil was homogenized for 1 minute with a sonicator at setting 6 (Virtishear Tempest, The Virtis Co., Gardiner, N.Y., U.S.A.). 50 microliters of the mixture were transferred into 5 mL of an 0.1% aqueous solution of SDS (w/v) 0 and 10 minutes after the homogenization. The absorbance of the solution at 500 nm was determined with a spectrometer (Shimadzu Model UV-1601, Kyoto, Japan). The absorbance at the time 0 after homogenization is the emulsion activity of the (modified) rice endosperm protein. \\n       FIG. 9  shows the emulsifying properties of rice protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) and Pepsin as a function of the degree of hydrolysis. \\n     Example 10 \\n     Determination of the Degree of Hydrolysis \\n     The DH was determined by the method of Nielsen and others (Nielsen, P. M., Petersen, D. &amp; Dambmann, C., 2001. Improved method for determining food protein degree of hydrolysis. Journal of Food Science, 66 (5), 642-646). The o-phthaldialdehyde (OPA) reagent was prepared as follow: 7.620 g of di-sodium tetraborate decahydrate (Na 2 B 4 O 7 .10H 2 O) and 200 mg sodium dodecyl sulfate (SDS) were dissolved in 150 mL of deionized water and then mixed with 160 mg of OPA (97% OPA pre-dissolved in 4 mL of ethanol) and 176 mg of 99% dithiothreitol (DTT). The final solution was made up to 200 mL with deionized water. Freeze dried protein sample of 0.1 g was solubilized in 10 mL deionized water. To measure the absorbance, 3 mL of OPA reagents was added to 10 mL tubes and then 400 μL of sample solution, serine standard (10 mg/100 mL) and deionized water was added in four tubes for each sample, standard and blank, respectively. This was followed by mixing for 5 s and held for exactly 2 min. Absorbance was read at 340 nm with a spectrophotometer (Shimadzu Model UV-1601, Kyoto, Japan). The DH was calculated as follows.\\n \\n DH=h/h   total *100%;\\n \\nwhere h is the number of hydrolyzed bonds and h total  is the total number of peptide bonds per protein equivalent; h=(Serine-NH2−β)/α equiv/g protein; where h is the number of hydrolyzed bonds and h total  is the total number of peptide bonds per protein equivalent; for cereal protein a is 1.00, (3 is 0.40, and h total  is 8.0.\\n \\nSerine-NH2=[( A   340sample   −A   340blank )/( A   340standard   −A   340blank )]*0.9516 meqv/L*0.01*100/( X*P );\\n \\nwhere serine-NH2=meqv serine NH2/g protein; X=g sample; P=% protein in sample; 0.01 is the sample volume in liter (L).\\n \\n     Example 11 \\n     Determination of the Protein and Total Solubility \\n     Protein solubility was determined by the method of Bera and Mukherjee (Bera, M. B., Mukherjee, R. K. 1989. Solubility, emulsifying, and foaming properties of rice bran protein concentrates. J Food Sci 54(1): 142-145) with some modifications. 200 mg of protein sample was dispersed in 10 mL of deionized water, the pH was adjusted to 7.0 by 1 N HCl or 1 N NaOH. The dispersion was stirred continuously for 30 min and centrifuged at 5000 rpm for 15 min. (model J2-21, Beckman, Fullerton, Calif., U.S.A.). The supernatant was recovered, and the protein content in the supernatant was determined by the Automatic Kjeldahl method (AACC 1990). The percentage of protein solubility was calculated by following equation: \\n     \\n       \\n         \\n           \\n             \\n               Protein \\n               \\u2062 \\n               \\n                   \\n               \\n               \\u2062 \\n               Solubility \\n               \\u2062 \\n               \\n                   \\n               \\n               \\u2062 \\n               \\n                 ( \\n                 % \\n                 ) \\n               \\n             \\n             = \\n             \\n               \\n                 \\n                   Protein \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   content \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   of \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   the \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   supernatant \\n                 \\n                 \\n                   Protein \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   in \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   200 \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   mg \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   protein \\n                   \\u2062 \\n                   \\n                     - \\n                   \\n                   \\u2062 \\n                   isolate \\n                 \\n               \\n               × \\n               100 \\n             \\n           \\n         \\n       \\n     \\n     The protein solubility was calculated as the percent ratio of protein in the supernatant to that of the total protein in the initial sample. \\n     The total solubility was determined by oven drying method, and expressed as the percent ratio of total soluble portion of the supernatant to that of the total weight of the protein isolate. \\n     \\n       \\n         \\n           \\n             \\n               Total \\n               \\u2062 \\n               \\n                   \\n               \\n               \\u2062 \\n               Solubility \\n               \\u2062 \\n               \\n                   \\n               \\n               \\u2062 \\n               \\n                 ( \\n                 % \\n                 ) \\n               \\n             \\n             = \\n             \\n               \\n                 \\n                   Soluble \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   portion \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   of \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   the \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   supernatant \\n                 \\n                 \\n                   200 \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   mg \\n                   \\u2062 \\n                   \\n                       \\n                   \\n                   \\u2062 \\n                   protein \\n                   \\u2062 \\n                   \\n                     - \\n                   \\n                   \\u2062 \\n                   isolate \\n                 \\n               \\n               × \\n               100 \\n             \\n           \\n         \\n       \\n     \\n       FIG. 6  shows the solubility of the rice endosperm protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or pepsin as a function of the degree of hydrolysis. \\n     Example 12 \\n     Determination of Molecular Weights \\n     The approximate molecular weights of the proteins were determined by SDS-PAGE according to the method of Laemmli, U.K. Cleavage of structural proteins during the assembly of the head of the bacteriophage T4. Nature 1970, 227, 680-686. The SDS-PAGE was carried out on a slab gel (4% stacking gel, 12% separating gel) in an SDS-Tris-Glycine discontinuous buffer system. Protein solutions (6-10 μg protein/μL) were prepared in non-reducing sample buffer (62.5 mM Tris-HCl pH 6.8, 2% SDS, 10% glycerol, and 0.05% bromophenol blue). Twelve microliters of the solutions were loaded onto the gel. Electrophoresis was performed at a constant voltage of 200 V for approximately 40 min. The gel was stained by 0.1% Coomassie brilliant blue in acetic acid/ethanol/water solution (10:40:50, v:v:v) and de-stained in the same solvent without Coomassie brilliant blue. The approximate molecular weights were determined by Bio-Rad broad range molecular weight standards ranging from 6.5 to 200 kDa. Molecular weights and density of bands were determined by using image analysis software, windows Mac &amp; Dos, Advanced American Biotechnology &amp; Biomedical Instruments Inc, CA, USA. \\n     Example 13 \\n     Determination of Surface Hydrophobicity \\n     Surface hydrophobicity of the protein isolates was determined by hydrophobic fluorescence probes of 1-anilino-8-naphthalene sulfonate (ANS) from the method outlined by Hayakawa and Nakai, Relationships of hydrophobicity and net charge to the solubility of ilk and soy proteins, J. Food Sci. 1985, 50, 486-491. Four-milliliters of protein solutions were made in 0.01 M phosphate buffer (pH 7) with concentrations ranging from 0.008 to 0.025% w/v. Ten-microliters of 8 mM ANS in 0.01 M phosphate buffer (pH 7) was added into each protein solution and fluorescence intensities of these solutions were measured at 390 nm of excitation and 470 nm of emission with a spectro-fluorophotometer (Shimadzu Model RF-1501, Kyoto, Japan). The surface hydrophobicity, expressed as the slope of fluorescence intensity vs. protein concentration, was calculated by linear regression. \\n       FIG. 5  shows the surface hydrophobicity of the rice endosperm protein hydrolysates treated with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA), LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) or pepsin as a function of the degree of hydrolysis. \\n     Example 14 \\n     Determination of Viscosity \\n     Viscosity of the protein isolates was determined by a rotational rheometer (Haake VT 550, Germany) equipped with a MVDIN measuring spindle (radius=19.36 mm, height=58.08 mm) at room temperature (26° C.). The protein isolates were mixed with deionized water to form slurry of 10%, and the slurry was left for 60 min for equilibrium before analysis. The samples (30 ml) were loaded into the cylindrical cup (radius=21.0 mm) and were subjected to a shear rate that changed from 0 to 400 l/s over 3 min using a computer-controlled program. The data were analyzed by Rheowin Pro Data manager version 2.84 (Haake Mess Tech, Germany). \\n     Example 15 \\n     Determination of Thermal Properties \\n     Thermal properties were determined using a differential scanning calorimeter model Pyris-1 (Perkins-Elmer Corp., Norwalk, Conn., U.S.A.) equipped with thermal analysis software (Version 4.00, Pyris-1-DSC, Perkin-Elmer Corp., Norwalk, Conn., U.S.A.). Two-hundred milligrams of protein were dispersed in an appropriate amount of water to form slurry with a protein content of 20%. The slurry was well mixed and left for 60 min for equilibrium before analysis. The slurry (about 50 μL) was weighed accurately into stainless steel pan (large volume capsule), hermetically sealed, and scanned during temperature increasing from 25 to 140° C. at a rate of 10° C./min. An empty pan was used as a reference. Peak temperature and enthalpy were computed from thermogram using the data processing software. \\n     Emulsifying activity and stability were determined by the turbidimetric method of Pearce and Kinsella using the procedure of Example 9. Absorbance at time 0 was expressed as the emulsifying activity of rice endosperm protein and emulsion stability (ES) was calculated as follows:\\n \\nEmulsion Stability= To×Δt/ΔT  \\n \\nwhere, ΔT is the decrease in turbidity (absorbance) of the initial absorbance (To) during the time interval of Δt (10 min)\\n \\n     All the experiments were done in triplicates. Data were analyzed for variance and multiple mean comparisons with JMP 5.1 software (SAS Inst) SAS, 2002, JMP® User&#39;s Guide, Version 5. SAS Institute Inc., Cary, N.C. The significance of difference between means was determined by the Tukey HSD procedure at the 5% significance level (P&lt;0.05). \\n     (E) Results \\n     In the following Table 3a the emulsion activity and capacity of the (modified) rice endosperm proteins (REP 1-1 to REP 7) obtained according to examples 1 to 7 measured by the methods as described in example 9 and 8, respectively, are given. \\n     For REP 3 eight values are given, because the example was carried out eight times, each time with another of the following enzymes: BROMELAIN, PROTEX 6L, pepsin, ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) or LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) (see REP 3-1 to REP 3-8), and for REP 6 five values are given, because the example was carried out five times (see REP 6-1 to REP 6-5), each time applying different process conditions as described in detail in example 6. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 3a \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Emulsion activity and capacity of the (modified) rice endosperm \\n               \\n               \\n                 proteins REP 1-1 to REP 7 according to examples 1 to 7 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 (Modified) rice endosperm \\n                   \\n                 Emulsion capacity \\n               \\n               \\n                 protein \\n                 Emulsion activity \\n                 [ml/g protein] \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 REP 1-1 \\n                 0.221 \\n                 228.1 \\n               \\n               \\n                 REP 2-1 \\n                 0.214 \\n                 177.6 \\n               \\n               \\n                 REP 3-1 (Bromelain) \\n                 0.469 \\n                 283.7 \\n               \\n               \\n                 REP 3-2 (Protex 6L) \\n                 0.507 \\n                 346.5 \\n               \\n               \\n                 REP 3-3 (Pepsin) \\n                 0.584 \\n                 363.8 \\n               \\n               \\n                 REP 3-4 \\n                 0.527 \\n                 357.1 \\n               \\n               \\n                 (ALKALASE\\u2009® \\n                   \\n                   \\n               \\n               \\n                 (Novo Nordisk Biochem,  \\n                   \\n                   \\n               \\n               \\n                 Franklinton, NC, USA)) \\n                   \\n                   \\n               \\n               \\n                 REP 3-5 \\n                 0.578 \\n                 386.8 \\n               \\n               \\n                 (LIQUIPANOL\\u2009® \\n                   \\n                   \\n               \\n               \\n                 (Enzyme Development \\n                   \\n                   \\n               \\n               \\n                 Corporation, New York, \\n                   \\n                   \\n               \\n               \\n                 NY, USA)) \\n                   \\n                   \\n               \\n               \\n                 REP 3-6 \\n                 0.409 \\n                 477.5 \\n               \\n               \\n                 REP 3-7 \\n                 0.361 \\n                 426.7 \\n               \\n               \\n                 REP 3-8 \\n                 0.370 \\n                 439.1 \\n               \\n               \\n                 REP 4 \\n                 0.624 \\n                 467.8 \\n               \\n               \\n                 REP 5 \\n                 0.643 \\n                 541.7 \\n               \\n               \\n                 REP 6-1 \\n                 0.604 \\n                 581.5 \\n               \\n               \\n                 REP 6-2 \\n                 0.341 \\n                 388.9 \\n               \\n               \\n                 REP 6-3 \\n                 0.342 \\n                 421.5 \\n               \\n               \\n                 REP 6-4 \\n                 0.365 \\n                 436.8 \\n               \\n               \\n                 REP 6-5 \\n                 0.277 \\n                 341.5 \\n               \\n               \\n                 REP 7 \\n                 0.473 \\n                 228.7 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     In the following tables 3b and 3c the protein and total solubility and the degree of hydrolysis of the modified rice endosperm proteins REP 3 and 6 obtained according to examples 3 and 6 measured by the methods as described in example 10 and 11, respectively, are given. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 3b \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Protein and total solubility of the modified rice endosperm \\n               \\n               \\n                 proteins REP 3 and REP 6 according to examples 3 and 6 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Modified rice endosperm \\n                   \\n                   \\n               \\n               \\n                 protein \\n                 Protein solubility (%) \\n                 Total solubility (%) \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 REP 3-6 \\n                 31.1 \\n                 45.0 \\n               \\n               \\n                 REP 3-7 \\n                 34.7 \\n                 49.5 \\n               \\n               \\n                 REP 3-8 \\n                 36.9 \\n                 53.0 \\n               \\n               \\n                 REP 6-2 \\n                 29.4 \\n                 47.2 \\n               \\n               \\n                 REP 6-3 \\n                 33.3 \\n                 56.4 \\n               \\n               \\n                 REP 6-4 \\n                 32.8 \\n                 49.7 \\n               \\n               \\n                 REP 6-5 \\n                 38.3 \\n                 58.2 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 3c \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Degree of hydrolysis of the modified rice endosperm proteins \\n               \\n               \\n                 REP 3 and REP 6 according to examples 3 and 6 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Modified rice \\n                   \\n                   \\n               \\n               \\n                 endosperm \\n                 *Degree of hydrolysis \\n                 **Degree of hydrolysis \\n               \\n               \\n                 protein \\n                 (DH) \\n                 (DH) \\n               \\n               \\n                   \\n               \\n               \\n                 REP 3-6 \\n                 3.7 \\n                 — \\n               \\n               \\n                 REP 3-7 \\n                 4.3 \\n                 — \\n               \\n               \\n                 REP 3-8 \\n                 5.9 \\n                 — \\n               \\n               \\n                 REP 6-2 \\n                 3.4 \\n                 5.6 \\n               \\n               \\n                 REP 6-3 \\n                 4.6 \\n                 6.7 \\n               \\n               \\n                 REP 6-4 \\n                 3.8 \\n                 5.1 \\n               \\n               \\n                 REP 6-5 \\n                 6.7 \\n                 8.8 \\n               \\n               \\n                   \\n               \\n               \\n                 *DH - enzymatic hydrolysis (before centrifugation and ultrafiltration) \\n               \\n               \\n                 **DH of the final products (enzymatic hydrolysis, after centrifugation and ultrafiltration) \\n               \\n            \\n           \\n         \\n       \\n     \\n     An overview of the chemical and enzymatic extraction procedures are presented in  FIGS. 1 and 2 , respectively. Of the total protein in rice flour, alkali- and salt-methods extracted protein isolates of 87.2 and 89.4% protein content, with 67.9 and 60.3% yield, respectively. In alkali and salt methods, the pH had strong influence on protein extraction. A positive correlation of pH and the extractability were found up to pH 12. However, high pH could lead to undesirable protein modification and increased the extraction of non-protein components that co-precipitate with protein and lower the isolate quality. However, the pH 10 or below was a poor solvent for rice protein extraction. Therefore, pH 11 was determined to be the optimum extraction pH concerning protein yield. Regarding protein functionality—such as emulsifying properties of the protein—a lower pH (preferably pH 8 to 10) might be of interest. Further, protein extraction at pH 11 combined with pretreatment of 3 min homogenization or sonication improved the extractability of the protein. These pretreatments might have dissociated the hydrophobically interacted protein-protein or protein-polysaccharide interactions and facilitated the protein extraction. \\n     In enzyme extraction, the thermo stable alpha amylase, TERMAMYL, followed by cellulase treatment isolated 85.8% protein content with 89.2% yield. The enzyme TERMAMYL has optimum activity at 90° C. In order to avoid extensive protein denaturation at 90° C. during TERMAMYL treatment, the procedure was adapted with another non-thermo stable amylase (amylase S), with optimum activity at 70° C. The amylase S combined with cellulase treatments isolated 81.7% protein with 90.5% yield as shown in the table below: \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 4 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Protein Content and Recovery of Rice Protein Isolates with \\n               \\n               \\n                 Optimized Conditions a   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Rice protein isolates \\n                 Protein (%) \\n                 Protein recovery (%) \\n               \\n               \\n                   \\n               \\n               \\n                 REP 1-3: RP A  (Alkali) \\n                 87.2 ± 1.08 \\n                 67.9 ± 3.37 \\n               \\n               \\n                 REP 1-4: RP S  (Salt) \\n                 89.4 ± 1.96 \\n                 60.3 ± 2.18 \\n               \\n               \\n                 REP 2-3: RP ET  (Termamyl) \\n                 85.8 ± 1.12 \\n                 89.2 ± 2.31 \\n               \\n               \\n                 REP 2-4: RP EA  (Amylase S) \\n                 81.7 ± 1.18 \\n                 90.5 ± 1.36 \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination and on dry weight basis. \\n               \\n            \\n           \\n         \\n       \\n     \\n     The compositions of the rice proteins, albumin, globulin, glutelin and prolamin are presented in Table 4. Rice flour contained 7.8% protein with 4.1%, albumin, 11.8% globulin, 77.4% glutelin, and 2.1% prolamin of the total protein. Alkali and salt extraction had similar profiles of proteins except globulin; salt-extracted protein contained higher globulin (6.2%) than alkali-extracted protein (3.1%). Since the major protein in rice is glutelin, the alkali- and salt-extracted proteins contained higher percentages of glutelin, 87.9 and 86.7, respectively, than the enzyme extracted proteins (80.2%). In contrast, prolamin content is higher in enzyme-extraction (2.8%) than in alkaline- (0.5%) and salt- (0.7%) extractions. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 5 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Albumin, Globulin, Glutelin, and Prolamin Content of Chemical- \\n               \\n               \\n                 and Enzyme-Extracted Protein Isolates a   \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                   \\n                 % of total protein \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                 REP 1-3: \\n                 REP 1-4: \\n                 REP 2-3: \\n                 REP 2-4: \\n               \\n               \\n                 Protein fraction \\n                   \\n                 RP A   \\n                 RP S   \\n                 RP ET   \\n                 RP EA   \\n               \\n               \\n                 components (%) \\n                 Rice flour \\n                 Alkali \\n                 Salt \\n                 Termamyl \\n                 Amylase S \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Total protein \\n                 7.92 \\n                 87.2 ± 1.1\\u2002 \\n                 89.4 ± 1.2\\u2002 \\n                 85.8 ± 1.1\\u2002 \\n                 81.7 ± 1.0\\u2002 \\n               \\n               \\n                 Albumin \\n                 4.13 \\n                 2.9 ± 0.1 \\n                 2.6 ± 0.2 \\n                 1.7 ± 0.1 \\n                 1.8 ± 0.1 \\n               \\n               \\n                 Globulin \\n                 11.88 \\n                 3.8 ± 0.2 \\n                 6.2 ± 0.3 \\n                 7.7 ± 0.0 \\n                 7.1 ± 0.6 \\n               \\n               \\n                 Glutelin \\n                 77.43 \\n                 87.2 ± 1.1\\u2002 \\n                 86.7 ± 0.9\\u2002 \\n                 80.6 ± 0.7\\u2002 \\n                 81.1 ± 1.05 \\n               \\n               \\n                 Prolamin \\n                 2.06 \\n                 1.5 ± 0.4 \\n                 1.1 ± 0.1 \\n                 2.8 ± 0.1 \\n                 2.9 ± 0.1 \\n               \\n               \\n                 Non-extractable \\n                 4.3 \\n                 3.1 ± 0.6 \\n                 2.9 ± 0.1 \\n                 6.4 ± 0.6 \\n                 5.7 ± 0.4 \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination. \\n               \\n            \\n           \\n         \\n       \\n     \\n     The composition of the four protein isolates are shown in Table 6. Both RPA and RP S  had similar composition and both of them had higher protein content, 87.2 and 89.4%, respectively, than those of enzymes-extracted proteins, RP ET  (84.8%), and RP EA  (81.7%). The non-protein components including starch, lipid, fiber, ash were lower in both alkali- and salt-extracted proteins than in enzyme-extracted proteins. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 6 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Proximate Composition of Rice Protein Isolates a  REP 1-3, 1-4, 2-3 and 2-4 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                   \\n                   \\n                 REP 2-4: \\n               \\n               \\n                 Proximate \\n                 REP 1-3: RP A   \\n                 REP 1-4: RP S   \\n                 REP 2-3: \\n                 RP EA   \\n               \\n               \\n                 composition (%) \\n                 Alkali \\n                 Salt \\n                 RP ET  Termamyl \\n                 Amylase S \\n               \\n               \\n                   \\n               \\n               \\n                 Moisture \\n                 \\u20022.4 ± 0.23 a   \\n                 2.5 ± 0.09 a   \\n                 2.0 ± 0.11 a   \\n                 2.3 ± 0.07 a   \\n               \\n               \\n                 Protein \\n                 87.2 ± 1.08 ab   \\n                 89.4 ± 1.96 a \\u2002 \\n                 85.8 ± 1.12 b \\u2002 \\n                 81.7 ± 1.08 c \\u2002 \\n               \\n               \\n                 Starch \\n                 \\u20022.1 ± 0.48 c   \\n                 1.7 ± 0.61 c   \\n                 5.1 ± 0.38 b   \\n                 6.7 ± 0.47 a   \\n               \\n               \\n                 Lipid \\n                 \\u20020.3 ± 0.04 b   \\n                 0.2 ± 0.06 b   \\n                 1.1 ± 0.05 a   \\n                 1.1 ± 0.13 a   \\n               \\n               \\n                 Fiber \\n                 \\u20020.6 ± 0.43 b   \\n                 0.5 ± 0.17 b   \\n                 4.8 ± 0.46 a   \\n                 5.3 ± 0.51 a   \\n               \\n               \\n                 Ash \\n                 \\u20020.4 ± 0.07 b   \\n                 0.7 ± 0.04 b   \\n                 3.7 ± 0.13 a   \\n                 3.4 ± 0.09 a   \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination and on dry weight basis. Mean values with different letters in the same row are significantly different (P &lt; 0.05). \\n               \\n            \\n           \\n         \\n       \\n     \\n     Electrophoresis profiles of rice proteins are shown in  FIG. 3 . \\n     Molecular size profiles of rice proteins determined by SDS PAGE gel electrophoresis \\n     Lane 1—Standard marker (6.5-200 kDa); \\n     Lane 2—RPA; \\n     Lane 3—RP S ; \\n     Lane 4—RP ET ; \\n     Lane 5—RP EA . \\n     Rice protein had a total of 6 bands and the molecular size ranged from 7 to 97 kDa. Major bands were observed at 97 kDa (2%), 45 kDa (3%), 33 kDa (72%), 21 kDa (7%), 14 kDa (6%), and 7 kDa (8%). The most predominant band was observed at the molecular size of 33 kDa with the density of 72% ( FIG. 3 , lanes 1 and 2). This is the major rice protein, glutelin. \\n     The thermal denaturation temperature and enthalpy values of denaturation of the rice protein isolates are presented in Table 7. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 7 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Thermal Properties of Rice Protein Isolates REP 1-3, 1-4, \\n               \\n               \\n                 2-3 and 2-4 and Rice Flour a   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                 Denaturation \\n                 Enthalpy value \\n               \\n               \\n                   \\n                   \\n                 temperature \\n                 of Denaturation \\n               \\n               \\n                   \\n                 Rice protein Isolates \\n                 (° C.) \\n                 (Jg −1 ) \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                 REP 1-3: RP A  (Alkali) \\n                 81.4 ± 1.1 a   \\n                 1.82 ± 0.14 b   \\n               \\n               \\n                   \\n                 REP 1-4: RP S  (Salt)) \\n                 78.9 ± 1.8 ab   \\n                 1.10 ± 0.09 c   \\n               \\n               \\n                   \\n                 REP 2-3: RP ET  (Termamyl) \\n                 nd \\n                 Nd \\n               \\n               \\n                   \\n                 REP 2-4: RP EA  (Amylase S) \\n                 79.9 ± 2.1 b   \\n                 0.18 ± 0.06 d   \\n               \\n               \\n                   \\n                 Rice flour \\n                 78.8 ± 2.7 ab   \\n                 7.31 ± 0.19 a   \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination. Mean values with different letters in the same column are significantly different (P &lt; 0.05). \\n               \\n               \\n                 *nd, non-detectable. \\n               \\n            \\n           \\n         \\n       \\n     \\n     The denaturation temperatures of RPA, RP S , RP ET , and RP EA  were 81.4, 78.9, 79.9, and 78.8° C., respectively. Enthalpy values of RPA (1.82 J/g), RP S  (1.10 J/g), RP ET  (non-detectable), and RP EA  (0.18 J/g) differed significantly, indicating that the extraction methods denatured the proteins at varying degrees depending on extraction conditions. The enthalpy changes could be used to predict the extent of protein denaturation. Biliaderis, C. G., Differential scanning calorimetry in food research: A review Food Chem. 1983, 10, 239-265. The enthalpy change is decreased when the protein is partially denatured, and the enthalpy change is zero when the protein is completely denatured. Yu, Z. Y., Hettiarachchy, N. S.; Rath, N., Extraction, denaturation and hydrophobic properties of rice flour proteins, J. Food Sci. 2001, 66 (2), 229-232, reported that the enthalpy values of 2.88, 3.14, and 3.79 J/g for albumin, globulin, and glutelin, respectively. In our study, alkali-extracted protein (RPA) had the highest enthalpy value (1.82 J/g), which was significantly lower than the values of Yu et al for albumin (2.88 J/g), globulin (3.14 J/g), and glutelin (3.79 J/g). The physical treatments applied in alkali and salt extraction may have denatured the protein to a degree. Among the enzyme-extracted proteins, RP ET  (TERMAMYL, at 90° C.) did not show any detectable enthalpy changes, which indicated complete protein denaturation. \\n     Surface Hydrophobicity and Solubility \\n     The surface hydrophobicity, solubility, and viscosity of the rice protein isolates are presented in Table 8. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 8 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Surface Hydrophobicity, Solubility, and Viscosity of Rice Protein Isolates a  REP \\n               \\n               \\n                 1-3, 1-4, 2-3 and 2-4 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 REP 1-3: RP A   \\n                 REP 1-4: RP S   \\n                 REP 2-3: RP ET   \\n                 REP 2-4: \\n               \\n               \\n                   \\n                 (Alkali) \\n                 (Salt) \\n                 (Termamyl) \\n                 RP EA  (Amylase S) \\n               \\n               \\n                   \\n               \\n               \\n                 Surface hydro- \\n                 563.8 ± 14.8 c   \\n                 544.2 ± 13.1 c   \\n                 931.3 ± 15.6 a   \\n                 771.6 ± 12.3 b   \\n               \\n               \\n                 phobicity \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Solubility % \\n                 \\u200213.6 ± 1.6 a   \\n                 \\u200211.6 ± 1.5 a   \\n                 \\u20038.1 ± 1.1 b   \\n                 \\u20039.3 ± 2.0 b   \\n               \\n               \\n                 (at pH 7.0) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Viscosity Pas \\n                 \\u200217.4 ± 0.6 a   \\n                 \\u200217.2 ± 0.9 a   \\n                 \\u200217.0 ± 1.1 a   \\n                 \\u200218.2 ± 1.4 a   \\n               \\n               \\n                 (10%, w/v) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 pH \\n                 \\u20037.0 ± 0.2 \\n                 \\u20036.8 ± 0.6 \\n                 \\u20036.8 ± 0.2 \\n                 \\u20036.9 ± 0.3 \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination. Mean values with different letters in the same row are significantly different (P &lt; 0.05). \\n               \\n            \\n           \\n         \\n       \\n     \\n     Hydrophobicity of rice protein isolates ranged from 544 to 931. The method of extraction contributed to the differences in hydrophobicities of these proteins. The hydrophobicity of enzyme-extracted proteins, RP ET  (931.3) and RP EA  (791.6), were higher than that of alkali- and salt-extracted proteins, which were 563.8 and 544.2, respectively. This might be due to protein denaturation, which increased surface hydrophobicity. The heat-induced unfolding of enzyme-extracted proteins increased surface hydrophobicity. \\n     The solubility of rice protein isolates were low at a wide pH range of 4 to 10 (Data not shown). Amino acid composition and hydrophobic interactions determine solubility. Sub-fractionation studies of rice glutelin by Tecson et al (8) showed that high molecular mass of rice glutelin had excessive intra- and inter-molecular disulfide and hydrophobic interactions, which reduced its solubility. Wen and Luthe (9) also reported that the most abundant amino acids in rice glutelin are glutamic acid/glutamine, aspartic acid/asparagine, arginine, glycine, and alanine. The amide groups in the glutamine and asparagine side chains promoted the aggregation of glutelin and reduced the rice protein solubility. Comparatively, the enzyme-extracted proteins had lower solubility than the alkali- and salt-extracted proteins, which might be due to the high level of denaturation, high surface hydrophobicity, and loss of water-soluble protein (albumin) in extraction. \\n     The emulsifying and foaming properties of rice protein isolates are presented in Table 9. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 9 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Emulsifying and Foaming Properties of Rice Protein Isolates a   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 REP 1-3: RP A   \\n                 REP 1-4: RP S   \\n                 REP 2-3: RP ET   \\n                 REP 2-4: RP EA   \\n               \\n               \\n                   \\n                 (Alkali) \\n                 (Salt) \\n                 (Termamyl) \\n                 (Amylase S) \\n               \\n               \\n                   \\n               \\n               \\n                 Emulsifying activity \\n                 0.244 ± 0.11 a   \\n                 0.215 ± 0.01 a   \\n                 0.146 ± 0.07 b \\u2002 \\n                 0.176 ± 0.21 b \\u2002 \\n               \\n               \\n                 Emulsifying \\n                 \\u200219.6 ± 0.91 a   \\n                 17.1 ± .81 b   \\n                 13.2 ± 2.11 c   \\n                 14.7 ± 0.91 c   \\n               \\n               \\n                 stability (min) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Foaming capacity \\n                 12.6 ± 1.2 a   \\n                 13.5 ± 1.0 a   \\n                 8.4 ± 0.8 b   \\n                 9.1 ± 0.7 b   \\n               \\n               \\n                 (ml) \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Foaming stability \\n                 \\u20028.6 ± 0.9 a   \\n                 \\u20028.2 ± 0.6 a   \\n                 5.3 ± 0.4 b   \\n                 6.8 ± 0.9 b   \\n               \\n               \\n                 (min) \\n               \\n               \\n                   \\n               \\n               \\n                   a Values are means of triplicate determination. Mean values with different letters in the same row are significantly different (P &lt; 0.05). \\n               \\n            \\n           \\n         \\n       \\n     \\n     The emulsifying and foaming properties of alkali- and salt-extracted proteins were higher than those of enzyme-extracted proteins. Emulsifying activities of alkali- and salt-extracted proteins were 0.224 and 0.216, respectively, and stabilities were 19.6 and 17.1 min, respectively, which were significantly higher than those of enzyme-extracted proteins. A similar pattern of results was observed in foaming capacity and stabilities. To have emulsifying and foaming properties, the protein should be able to solubilize in the aqueous phase and rapidly unfold to form a cohesive layer at the interface. The molecular and physical requirements of proteins to stabilize emulsion and foam are similar (Damodarn, S., Protein-stabilized foams and emulsions, J. Food Sci. 205, 70 (3), 54-66). \\n     The possible reasons for the differences in functional properties obtained by the different methods of extraction could be due to protein purity, composition, surface hydrophobicity, and solubility of the protein isolates. However, it is not meant, that the applicant is bound by these theories. \\n     Firstly, the thermal properties of the two enzyme-extracted proteins showed a higher degree of denaturation than those of RP A  and RP S . Excessive denaturation could be detrimental to emulsifying and foaming properties by either promoting protein aggregation or exposing a greater number of hydrophobic amino acids. The surface hydrophobicity data confirmed the increased hydrophobicity of enzyme-extracted proteins. The excessive surface hydrophobicity may be unfavorable for emulsifying or foaming properties of protein by facilitating to form extensive hydrophobic bonding and reduced solubility and flexibility of the protein. \\n     Secondly, the extreme extraction conditions such as high temperature and or pH might have promoted the formation of disulfide cross-linking. It has been reported that rice glutelin has excessive disulphide linkages. Extremely low solubility of the extracted protein indicated that heat- or pH-induced unfolding of the protein during extraction might have exposed thiol groups in the interior of the protein. These exposed thiol groups could readily form disulfide linkages with adjacent protein molecules. The newly formed disulfide bonds might have reduced the molecular flexibility, solubility, and surface activity of the proteins. The flexibility of a certain segments of a protein is one of the major properties determining the emulsifying and foaming properties of a protein. \\n     Thirdly, the composition of rice protein isolates showed that alkali- and salt-extracted protein isolates contained higher amounts of proteins and lower amounts of non-protein substances than those of enzyme-extracted proteins. Significant amounts of fiber, ash, lipid, and residual starch in the enzyme extracted protein isolates, RP ET  and RP EA , might have reduced their emulsifying and foaming properties. These non-protein components may interact with proteins and modify the net charge and hydrophobicity of proteins, affecting the protein functionalities. \\n     Finally, lower solubility of the enzyme extracted protein adversely affected the emulsifying and foaming properties. To have emulsion and foaming properties, the proteins should be able to solubilize and unfold to form cohesive layers at the interface. This requires protein solubility and flexibility. Lacking these properties, the enzyme-extracted proteins had diminished emulsifying and foaming properties than the alkali- and salt-extracted proteins. \\n     From these investigations it has been found that alkali and salt methods extracted 87.2 and 89.4% of proteins with 67.9 and 60.3% yield respectively. Enzymatic method with TERMAMYL and amylase S extracted 84.8 and 81.8% proteins with 89.2 and 90.5% yield, respectively. The alkali- and salt-extracted proteins had higher protein contents and lower non-protein components including starch, lipid, fiber, and ash than the enzyme-extracted proteins. Comparatively, more favorable protein composition, lower hydrophobicity, higher solubility, and a lower degree of thermal denaturation of alkali- and salt-extracted proteins contributed to higher emulsifying and foaming properties than those of enzyme-extracted proteins. Therefore, the alkali and salt method of extractions were milder extraction conditions and the proteins retained better functional properties. \\n     Rice proteins are nutritional and hypoallergenic, and healthy for human consumption. Efficient extraction with approved food grade enzymes and chemicals are essential for commercial production and application of rice protein as a functional ingredient. Rice endosperm proteins were isolated by alkali, salt, and enzymatic methods and evaluated for extractability and physicochemical properties. Alkali (RPA) and salt (RP S ) methods extracted 87.2 and 89.4% of proteins with 67.9 and 60.3% yield, respectively. The enzymatic methods with TERMAMYL (RP ET ) and amylase S (RP EA ) extracted 84.8 and 81.8% proteins with 89.2 and 90.5% yield, respectively. Enthalpy values of RPA (1.82 J/g), RP S  (1.10 J/g), RP ET  (non-detectable), and RP EA  (0.18 J/g) determined by Differential Scanning calorimeter, demonstrated that the varying level of denaturation of proteins depends on the method of extraction. Surface hydrophobicity data supported this observation. Alkali- and salt-extracted proteins had higher solubility and emulsifying properties than those of enzyme-extracted proteins; therefore, alkali- and salt-extracted proteins can have enhanced functional use and a potential starting material for preparing tailored rice protein isolates. \\n     Rice endosperm protein isolated with a thermo-stable alpha-amylase, TERMAMYL, was treated with neutral, acid, and/or alkaline proteases. The degree of hydrolysis of the hydrolysates was optimized by OPA method to maximize the solubility and emulsifying properties. After enzyme inactivation, the soluble fraction was separated by centrifugation (1000 g for 10 min), and the soluble hydrolysate was filtered through membrane of specific molecular weight cut-off (50 kDa). The resulted hydrolysate of greater than 50 kDa was spray dried and evaluated for degree of hydrolysis, solubility, emulsion capacity, activity, and stability. \\n     The combined treatments of two types of proteases; Cysteine (LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA)) and serine (ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA)) proteases followed by centrifugation and ultrafiltration significantly improved solubility and emulsifying properties. The resulted hydrolysate had 53.5% solubility; 586.3 ml/g emulsion-capacity, 0.604-activity, and 24.8 min-stability, which were significantly higher than that of unmodified rice protein isolate (solubility 11.8%, 177.6 ml/g emulsion-capacity, 0.214 activity, and, 14.7 min-stability). \\n     The combined enzymatic hydrolysis with ALKALASE® (NOVO NORDISK BIOCHEM, FRANKLINTON, N.C., USA) and LIQUIPANOL® (ENZYME DEVELOPMENT CORPORATION, NEW YORK, N.Y., USA) followed by ultrafiltration improved rice endosperm protein solubility and emulsifying properties. The ultrafiltrated rice endosperm protein hydrolysate with uniform molecular fractions of high-purity enhanced its solubility and emulsifying properties. \\n     Example 12 \\n     Manufacture of a formulation of β-carotene \\n     A formulation comprising a rice endosperm protein and β-carotene may be prepared as follows: \\n     a) Preparation of a(n Oil-Based) Solution 1: \\n     7.7 g of corn oil and 1.4 g of dl-α-tocopherol were mixed. 16.1 g of crystalline β-carotene were dispersed in 180 ml of chloroform (trichloromethane) and the resulting dispersion was added to the mixture of corn oil and tocopherol. By gently stirring and simultaneous heating the mixture to about 60° C. a solution was obtained. \\n     b) Preparation of a(n Aqueous) Solution 2: \\n     35 g of REP 1 (REP 1-1) according to example 1 was re-dissolved in 250 ml of water by stirring at 60° C. Alternatively REP 1-1 freshly prepared according to example 1 could be used, i.e. the steps of freeze drying and storing are not carried out, but the resulting REP 1-1 solution is used as such, and brought to a temperature of 60° C. Additionally 2.1 g of ascorbyl palmitate and 42.7 g of sucrose were added. 8 ml of aqueous 1 N NaOH were used to adjust the pH to a value of 7.9. Instead of REP 1-1, REP 1-2 can be used accordingly for the preparation of the aqueous solution. \\n     c) Preparation of an Emulsion from the Solutions 1 and 2: \\n     Under vigorous stirring solution 1 was added to solution 2 at 53° C. and the dispersion was vigorously stirred for another 30 minutes. The stirred dispersion was kept at 50 to 55° C. for 30 minutes. Residual trichloromethane was removed at 50 to 55° C. After removing entrapped air bubbles by centrifugation the emulsion was gently stirred at 50 to 55° C. for some minutes and then characterised with respect to the particle size of the inner phase. The mean particle size (Sauter diameter, D[3, 2]) of the inner phase of the emulsion was 380 nm as measured by laser diffraction (Malvern Masersizer). \\n     d) Preparation of a Solid Formulation from the Emulsion: \\n     The emulsion may be sprayed into a pre-cooled fluidised bed of cornstarch. Excess cornstarch can be removed by sieving and the powder obtained can be dried in an air stream at room temperature. The powder particle fraction in the range of 0.16 to 0.50 mm can be collected by sieving and characterised with respect to the carotenoid content, the colour intensity and the colour hue in an aqueous dispersion, the content of the corn starch and residual humidity. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 10 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Calculated composition of the dried formulation \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                 Amount [weight-%, based on \\n               \\n               \\n                   \\n                 Compound \\n                 the total dry weight] \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                   \\n                 REP 1-1: Rice endosperm \\n                 25.0 \\n               \\n               \\n                   \\n                 protein according to example 1 \\n                   \\n               \\n               \\n                   \\n                 sucrose \\n                 30.5 \\n               \\n               \\n                   \\n                 ascorbyl palmitate \\n                 1.5 \\n               \\n               \\n                   \\n                 β-carotene \\n                 11.5 \\n               \\n               \\n                   \\n                 corn oil \\n                 5.5 \\n               \\n               \\n                   \\n                 dl-α-tocopherol \\n                 1.0 \\n               \\n               \\n                   \\n                 Corn starch fluid \\n                 25.0', 'id': 'US-9133254-B2', 'ti': 'Protective, hydrocolloid for active ingredients'}>,\n",
       " <Hit {'ab': 'The present invention relates to newly identified nucleic acid molecules and polypeptides present in normal and neoplastic ovarian cells, including fragments, variants and derivatives of the nucleic acids and polypeptides. The present invention also relates to antibodies to the polypeptides of the invention, as well as agonists and antagonists of the polypeptides of the invention. The invention also relates to compositions containing the nucleic acid molecules, polypeptides, antibodies, agonists and antagonists of the invention and methods for the use of these compositions. These uses include identifying, diagnosing, monitoring, staging, imaging and treating ovarian cancer and non-cancerous disease states in ovarian, identifying ovarian tissue, monitoring and identifying and/or designing agonists and antagonists of polypeptides of the invention. The uses also include gene therapy, production of transgenic animals and cells, and production of engineered ovarian tissue for treatment and research.', 'clm': 'We claim: \\n     \\n       1. A method for determining the presence of an ovarian specific protein comprising a polypeptide comprising an amino acid sequence with at least 95% sequence identity to SEQ ID NO:254 in a sample, comprising the steps of:\\n (a) contacting the sample with a suitable reagent which selectively interacts with the ovarian specific protein under conditions in which the reagent will selectively interact with the ovarian specific protein; and \\n (b) detecting the interaction of the reagent with an ovarian specific protein in the sample, wherein the detection of binding indicates the presence of an ovarian specific protein in the sample. \\n \\n     \\n     \\n       2. The method of  claim 1  wherein the suitable reagent which selectively interacts with the ovarian specific protein is an antibody or fragment thereof that specifically binds to a polypeptide comprising an amino acid sequence with at least 95% sequence identity to SEQ ID NO:254 or a peptide thereof. \\n     \\n     \\n       3. The method of  claim 2  wherein the antibody or fragment thereof binds a peptide of SEQ ID NO: 254, wherein said peptide comprises amino acids 1-341, 191-198, 251-261, 4-23, 217-227, 321-331, 298-304, 341-366, 208-215, 267-275, 180-185, 285-294, 92-115, 121-146, 71-77, 212-214, 203-206, 154-157, 247-249, 365-367, 94-96, 151-156, 304-306, 48-51, 10-15, 188-193, 237-239, 150-153, 179-184, 227-233, 70-72, 66-68, 274-277, 203-205, 39-44, 81-84, 128-133, 276-278, 162-167, 187-190, 199-204, 210-213, 339-344, 279-282, 142-211, 172-215, 171-199 or 26-101 of SEQ ID NO: 254. \\n     \\n     \\n       4. The method of  claim 3  wherein the antibody or fragment thereof binds the peptide comprising amino acids 1-341 of SEQ ID NO: 254. \\n     \\n     \\n       5. The method of  claim 3  wherein the antibody or fragment thereof binds the peptide comprising amino acids 26-101 of SEQ ID NO: 254. \\n     \\n     \\n       6. The method of  claim 2  wherein the antibody or fragment thereof binds a peptide of SEQ ID NO: 254, wherein said peptide comprises a post translational modification, motif, or domain which is a thyroglobulin domain, TY domain, myristyl site, glycosylation site, pkc phospho site, ck2 phospho site, tyr phospho site or amidation site. \\n     \\n     \\n       7. The method of  claim 2  wherein the antibody or fragment is a monoclonal, polyclonal, human, humanized or chimeric antibody or fragment thereof. \\n     \\n     \\n       8. The method of  claim 2  wherein the antibody or fragment thereof is labeled. \\n     \\n     \\n       9. A method for diagnosing or monitoring the presence and metastases of ovarian cancer in a patient, comprising the steps of:\\n (a) determining an amount of: \\n a polypeptide comprising an amino acid sequence with at least 95% sequence identity to SEQ ID NO:254; \\n and \\n (b) comparing the amount of the determined polypeptide in the sample of the patient to the amount of the ovarian specific marker in a normal control; wherein a difference in the amount of the polypeptide in the sample compared to the amount of the polypeptide in the normal control is associated with the presence of ovarian cancer. \\n \\n     \\n     \\n       10. The method of  claim 9  wherein the amount of a polypeptide comprising an amino acid sequence with at least 95% sequence identity to SEQ ID NO:254 is determined with an antibody or fragment thereof that specifically binds to a polypeptide comprising an amino acid sequence with at least 95% sequence identity to SEQ ID NO:254 or a peptide thereof. \\n     \\n     \\n       11. The method of  claim 10  wherein the antibody or fragment thereof binds a peptide of SEQ ID NO: 254, wherein said peptide comprises amino acids 1-341, 191-198, 251-261, 4-23, 217-227, 321-331, 298-304, 341-366, 208-215, 267-275, 180-185, 285-294, 92-115, 121-146, 71-77, 212-214, 203-206, 154-157, 247-249, 365-367, 94-96, 151-156, 304-306, 48-51, 10-15, 188-193, 237-239, 150-153, 179-184, 227-233, 70-72, 66-68, 274-277, 203-205, 39-44, 81-84, 128-133, 276-278, 162-167, 187-190, 199-204, 210-213, 339-344, 279-282, 142-211, 172-215, 171-199 or 26-101 of SEQ ID NO: 254. \\n     \\n     \\n       12. The method of  claim 11  wherein the antibody or fragment thereof binds the peptide comprising amino acids 1-341 of SEQ ID NO: 254. \\n     \\n     \\n       13. The method of  claim 11  wherein the antibody or fragment thereof binds the peptide comprising amino acids 26-101 of SEQ ID NO: 254. \\n     \\n     \\n       14. The method of  claim 10  wherein the antibody or fragment thereof binds a peptide of SEQ ID NO: 254, wherein said peptide comprises a post translational modification, motif, or domain which is a thyroglobulin domain, TY domain, myristyl site, glycosylation site, pkc phospho site, ck2 phospho site, tyr phospho site or amidation site. \\n     \\n     \\n       15. The method of  claim 10  wherein the antibody or fragment thereof is a monoclonal, polyclonal, human, humanized or chimeric antibody or fragment thereof. \\n     \\n     \\n       16. The method of  claim 10  wherein the antibody or fragment thereof is labeled.', 'cpc': 'C07K14/47', 'detd': 'INTRODUCTION \\n     This patent application is a continuation of U.S. application Ser. No. 10/537,743, filed Nov. 9, 2006 now abandoned, which is the U.S. National Stage of International Application No. PCT/US2003/38855, filed Dec. 8, 2003, which claims the benefit of priority from U.S. Provisional Patent Application Ser. No. 60/431,321 filed Dec. 6, 2002, U.S. Provisional Patent Application Ser. No. 60/431,301 filed Dec. 6, 2002, U.S. Provisional Patent Application Ser. No. 60/484,584 filed Jun. 30, 2003 and U.S. Provisional Patent Application Ser. No. 60/518,607, filed Nov. 7, 2003, which are herein incorporated by reference in their entireties. \\n    \\n    \\n     FIELD OF THE INVENTION \\n     The present invention relates to newly identified nucleic acids and polypeptides present in normal and neoplastic ovarian cells, including fragments, variants and derivatives of the nucleic acids and polypeptides. The present invention also relates to antibodies to the polypeptides of the invention, as well as agonists and antagonists of the polypeptides of the invention. The invention also relates to compositions comprising the nucleic acids, polypeptides, antibodies, post translational modifications (PTMs), variants, derivatives, agonists and antagonists thereto and methods for the use of these compositions. These uses include identifying, diagnosing, monitoring, staging, imaging and treating ovarian cancer and/or non-cancerous disease states in ovarian, identifying ovarian tissue and monitoring and identifying and/or designing agonists and antagonists of polypeptides of the invention. The uses also include gene therapy, therapeutic molecules including but not limited to antibodies or antisense molecules, production of transgenic animals and cells, and production of engineered ovarian tissue for treatment and research. \\n     BACKGROUND OF THE INVENTION \\n     Cancer of the ovaries is the fourth-most common cause of cancer death in women in the United States, with more than 23,000 new cases and roughly 14,000 deaths predicted for the year 2001. Shridhar, V. et al.,  Cancer Res.  61(15):5895-904 (2001); Memarzadeh, S. &amp; Berek, J. S.,  J. Reprod. Med.  46(7):621-29 (2001). The incidence of ovarian cancer is of serious concern worldwide, with an estimated 191,000 new cases predicted annually. Runnebaum, I. B. &amp; Stickeler, E., J.  Cancer Res. Clin. Oncol.  127(2):73-79 (2001). These numbers continue to rise today. In the United States alone, it is estimated there will be 25,400 new cases of ovarian cancer, and 14,300 deaths due to ovarian cancer in 2003. (American Cancer Society Website: cancer.org on the world wide web). Unfortunately, women with ovarian cancer are typically asymptomatic until the disease has metastasized. Because effective screening for ovarian cancer is not available, roughly 70% of women diagnosed have an advanced stage of the cancer with a five-year survival rate of ˜25-30%. Memarzadeh, S. &amp; Berek, J. S., supra; Nunns, D. et al.,  Obstet. Gynecol. Surv.  55(12):746-51. Conversely, women diagnosed with early stage ovarian cancer enjoy considerably higher survival rates. Werness, B. A. &amp; Eltabbakh, G. H.,  Int&#39;l. J. Gynecol. Pathol.  20(1):48-63 (2001). Although our understanding of the etiology of ovarian cancer is incomplete, the results of extensive research in this area point to a combination of age, genetics, reproductive, and dietary/environmental factors. Age is a key risk factor in the development of ovarian cancer: while the risk for developing ovarian cancer before the age of 30 is slim, the incidence of ovarian cancer rises linearly between ages 30 to 50, increasing at a slower rate thereafter, with the highest incidence being among septagenarian women. Jeanne M. Schilder et al.,  Hereditary Ovarian Cancer: Clinical Syndromes and Management , in  Ovarian Cancer  182 (Stephen C. Rubin &amp; Gregory P. Sutton eds., 2d ed. 2001). \\n     With respect to genetic factors, a family history of ovarian cancer is the most significant risk factor in the development of the disease, with that risk depending on the number of affected family members, the degree of their relationship to the woman, and which particular first degree relatives are affected by the disease. Id. Mutations in several genes have been associated with ovarian cancer, including BRCA1 and BRCA2, both of which play a key role in the development of breast cancer, as well as hMSH2 and hMLH1, both of which are associated with hereditary non-polyposis colon cancer. Katherine Y. Look,  Epidemiology, Etiology, and Screening of Ovarian Cancer , in  Ovarian Cancer  169, 171-73 (Stephen C. Rubin &amp; Gregory P. Sutton eds., 2d ed. 2001). BRCA1, located on chromosome 17, and BRCA2, located on chromosome 13, are tumor suppressor genes implicated in DNA repair; mutations in these genes are linked to roughly 10% of ovarian cancers. Id. at 171-72; Schilder et al., supra at 185-86. hMSH2 and hMLH1 are associated with DNA mismatch repair, and are located on chromosomes 2 and 3, respectively; it has been reported that roughly 3% of hereditary ovarian carcinomas are due to mutations in these genes. Look, supra at 173; Schilder et al., supra at 184, 188-89. \\n     Reproductive factors have also been associated with an increased or reduced risk of ovarian cancer. Late menopause, nulliparity, and early age at menarche have all been linked with an elevated risk of ovarian cancer. Schilder et al., supra at 182. One theory hypothesizes that these factors increase the number of ovulatory cycles over the course of a woman&#39;s life, leading to “incessant ovulation,” which is thought to be the primary cause of mutations to the ovarian epithelium. Id; Laura J. Havrilesky &amp; Andrew Berchuck,  Molecular Alterations in Sporadic Ovarian Cancer , in  Ovarian Cancer  25 (Stephen C. Rubin &amp; Gregory P. Sutton eds., 2d ed. 2001). The mutations may be explained by the fact that ovulation results in the destruction and repair of that epithelium, necessitating increased cell division, thereby increasing the possibility that an undetected mutation will occur. Id. Support for this theory may be found in the fact that pregnancy, lactation, and the use of oral contraceptives, all of which suppress ovulation, confer a protective effect with respect to developing ovarian cancer. Id. \\n     Among dietary/environmental factors, there would appear to be an association between high intake of animal fat or red meat and ovarian cancer, while the antioxidant Vitamin A, which prevents free radical formation and also assists in maintaining normal cellular differentiation, may offer a protective effect. Look, supra at 169. Reports have also associated asbestos and hydrous magnesium trisilicate (talc), the latter of which may be present in diaphragms and sanitary napkins. Id. at 169-70. \\n     Current screening procedures for ovarian cancer, while of some utility, are quite limited in their diagnostic ability, a problem that is particularly acute at early stages of cancer progression when the disease is typically asymptomatic yet is most readily treatable. Walter J. Burdette,  Cancer: Etiology Diagnosis and Treatment  166 (1998); Memarzadeh &amp; Berek, supra; Runnebaum &amp; Stickeler, supra; Werness &amp; Eltabbakh, supra. Commonly used screening tests include biannual rectovaginal pelvic examination, radioimmunoassay to detect the CA-125 serum tumor marker, and transvaginal ultrasonography. Burdette, supra at 166. \\n     Pelvic examination has failed to yield adequate numbers of early diagnoses, and the other methods are not sufficiently accurate. Id. One study reported that only 15% of patients who suffered from ovarian cancer were diagnosed with the disease at the time of their pelvic examination. Look, supra at 174. Moreover, the CA-125 test is prone to giving false positives in pre-menopausal women and has been reported to be of low predictive value in post-menopausal women. Id. at 174-75. Although transvaginal ultrasonography is now the preferred procedure for screening for ovarian cancer, it is unable to distinguish reliably between benign and malignant tumors, and also cannot locate primary peritoneal malignancies or ovarian cancer if the ovary size is normal. Schilder et al., supra at 194-95. While genetic testing for mutations of the BRCA1, BRCA2, hMSH2, and hMLH1 genes is now available, these tests may be too costly for some patients and may also yield false negative or indeterminate results. Schilder et al., supra at 191-94. \\n     The staging of ovarian cancer, which is accomplished through surgical exploration, is crucial in determining the course of treatment and management of the disease.  AJCC Cancer Staging Handbook  187 (Irvin D. Fleming et al. eds., 5th ed. 1998); Burdette, supra at 170; Memarzadeh &amp; Berek, supra; Shridhar et al., supra. Staging is performed by reference to the classification system developed by the International Federation of Gynecology and Obstetrics. David H. Moore,  Primary Surgical Management of Early Epithelial Ovarian Carcinoma , in  Ovarian Cancer  203 (Stephen C. Rubin &amp; Gregory P. Sutton eds., 2d ed. 2001); Fleming et al. eds., supra at 188. Stage I ovarian cancer is characterized by tumor growth that is limited to the ovaries and is comprised of three substages. Id. In substage IA, tumor growth is limited to one ovary, there is no tumor on the external surface of the ovary, the ovarian capsule is intact, and no malignant cells are present in ascites or peritoneal washings. Id. Substage IB is identical to A1, except that tumor growth is limited to both ovaries. Id. Substage IC refers to the presence of tumor growth limited to one or both ovaries, and also includes one or more of the following characteristics: capsule rupture, tumor growth on the surface of one or both ovaries, and malignant cells present in ascites or peritoneal washings. Id. \\n     Stage II ovarian cancer refers to tumor growth involving one or both ovaries, along with pelvic extension. Id. Substage IIA involves extension and/or implants on the uterus and/or fallopian tubes, with no malignant cells in the ascites or peritoneal washings, while substage IIB involves extension into other pelvic organs and tissues, again with no malignant cells in the ascites or peritoneal washings. Id. Substage IIC involves pelvic extension as in IIA or IIB, but with malignant cells in the ascites or peritoneal washings. Id. \\n     Stage III ovarian cancer involves tumor growth in one or both ovaries, with peritoneal metastasis beyond the pelvis confirmed by microscope and/or metastasis in the regional lymph nodes. Id. Substage IIIA is characterized by microscopic peritoneal metastasis outside the pelvis, with substage IIIB involving macroscopic peritoneal metastasis outside the pelvis 2 cm or less in greatest dimension. Id. Substage IIIC is identical to IIIB, except that the metastasis is greater than 2 cm in greatest dimension and may include regional lymph node metastasis. Id. Lastly, Stage IV refers to the presence distant metastasis, excluding peritoneal metastasis. Id. \\n     While surgical staging is currently the benchmark for assessing the management and treatment of ovarian cancer, it suffers from considerable drawbacks, including the invasiveness of the procedure, the potential for complications, as well as the potential for inaccuracy. Moore, supra at 206-208, 213. In view of these limitations, attention has turned to developing alternative staging methodologies through understanding differential gene expression in various stages of ovarian cancer and by obtaining various biomarkers to help better assess the progression of the disease. Vartiainen, J. et al.,  Int&#39;l J. Cancer,  95(5):313-16 (2001); Shridhar et al. supra; Baekelandt, M. et al.,  J. Clin. Oncol.  18(22):3775-81. \\n     The treatment of ovarian cancer typically involves a multiprong attack, with surgical intervention serving as the foundation of treatment. Dennis S. Chi &amp; William J. Hoskins,  Primary Surgical Management of Advanced Epithelial Ovarian Cancer, _in  Ovarian Cancer  241 (Stephen C. Rubin &amp; Gregory P. Sutton eds., 2d ed. 2001). For example, in the case of epithelial ovarian cancer, which accounts for ˜90% of cases of ovarian cancer, treatment typically consists of: (1) cytoreductive surgery, including total abdominal hysterectomy, bilateral salpingo-oophorectomy, omentectomy, and lymphadenectomy, followed by (2) adjuvant chemotherapy with paclitaxel and either cisplatin or carboplatin. Eltabbakh, G. H. &amp; Awtrey, C. S.,  Expert Op. Pharmacother.  2(10):109-24. Despite a clinical response rate of 80% to the adjuvant therapy, most patients experience tumor recurrence within three years of treatment. Id. Certain patients may undergo a second cytoreductive surgery and/or second-line chemotherapy. Memarzadeh &amp; Berek, supra. \\n     From the foregoing, it is clear that procedures used for detecting, diagnosing, monitoring, staging, prognosticating, and preventing the recurrence of ovarian cancer are of critical importance to the outcome of the patient. Moreover, current procedures, while helpful in each of these analyses, are limited by their specificity, sensitivity, invasiveness, and/or their cost. As such, highly specific and sensitive procedures that would operate by way of detecting novel markers in cells, tissues, or bodily fluids, with minimal invasiveness and at a reasonable cost, would be highly desirable. \\n     Breast cancer, also referred to as mammary tumor cancer, is the second most common cancer among women, accounting for a third of the cancers diagnosed in the United States. One in nine women will develop breast cancer in her lifetime and about 192,000 new cases of breast cancer are diagnosed annually with about 42,000 deaths. Bevers,  Primary Prevention of Breast Cancer , in  Breast Cancer,  20-54 (Kelly K Hunt et al., ed., 2001); Kochanek et al., 49  Nat&#39;l. Vital Statistics Reports  1, 14 (2001). Breast cancer is extremely rare in women younger than 20 and is very rare in women under 30. The incidence of breast cancer rises with age and becomes significant by age 50. White Non-Hispanic women have the highest incidence rate for breast cancer and Korean women have the lowest. Increased prevalence of the genetic mutations BRCA1 and BRCA2 that promote breast and other cancers are found in Ashkenazi Jews. African American women have the highest mortality rate for breast cancer among these same groups (31 per 100,000), while Chinese women have the lowest at 11 per 100,000. Although men can get breast cancer, this is extremely rare. In the United States it is estimated there will be 212,600 new cases of breast cancer and 40,200 deaths due to breast cancer in 2003. (American Cancer Society Website: cancer.org on the world wide web). With the exception of those cases with associated genetic factors, precise causes of breast cancer are not known. \\n     In the treatment of breast cancer, there is considerable emphasis on detection and risk assessment because early and accurate staging of breast cancer has a significant impact on survival. For example, breast cancer detected at an early stage (stage T0, discussed below) has a five-year survival rate of 92%. Conversely, if the cancer is not detected until a late stage (i.e., stage T4 (IV)), the five-year survival rate is reduced to 13%.  AJCC Cancer Staging Handbook pp.  164-65 (Irvin D. Fleming et al. eds., 5 th  ed. 1998). Some detection techniques, such as mammography and biopsy, involve increased discomfort, expense, and/or radiation, and are only prescribed only to patients with an increased risk of breast cancer. \\n     Current methods for predicting or detecting breast cancer risk are not optimal. One method for predicting the relative risk of breast cancer is by examining a patient&#39;s risk factors and pursuing aggressive diagnostic and treatment regiments for high risk patients. A patient&#39;s risk of breast cancer has been positively associated with increasing age, nulliparity, family history of breast cancer, personal history of breast cancer, early menarche, late menopause, late age of first full term pregnancy, prior proliferative breast disease, irradiation of the breast at an early age and a personal history of malignancy. Lifestyle factors such as fat consumption, alcohol consumption, education, and socioeconomic status have also been associated with an increased incidence of breast cancer although a direct cause and effect relationship has not been established. While these risk factors are statistically significant, their weak association with breast cancer limited their usefulness. Most women who develop breast cancer have none of the risk factors listed above, other than the risk that comes with growing older. NIH Publication No. 00-1556 (2000). \\n     Current screening methods for detecting cancer, such as breast self exam, ultrasound, and mammography have drawbacks that reduce their effectiveness or prevent their widespread adoption. Breast self exams, while useful, are unreliable for the detection of breast cancer in the initial stages where the tumor is small and difficult to detect by palpation. Ultrasound measurements require skilled operators at an increased expense. Mammography, while sensitive, is subject to over diagnosis in the detection of lesions that have questionable malignant potential. There is also the fear of the radiation used in mammography because prior chest radiation is a factor associated with an increase incidence of breast cancer. \\n     At this time, there are no adequate methods of breast cancer prevention. The current methods of breast cancer prevention involve prophylactic mastectomy (mastectomy performed before cancer diagnosis) and chemoprevention (chemotherapy before cancer diagnosis) which are drastic measures that limit their adoption even among women with increased risk of breast cancer. Bevers, supra. \\n     A number of genetic markers have been associated with breast cancer. Examples of these markers include carcinoembryonic antigen (CEA) (Mughal et al.,  JAMA  249:1881 (1983)), MUC-1 (Frische and Liu,  J. Clin. Ligand  22:320 (2000)), HER-2/neu (Haris et al.,  Proc. Am. Soc. Clin. Oncology  15:A96 (1996)), uPA, PAI-1, LPA, LPC, RAK and BRCA (Esteva and Fritsche,  Serum and Tissue Markers for Breast Cancer , in  Breast Cancer,  286-308 (2001)). These markers have problems with limited sensitivity, low correlation, and false negatives which limit their use for initial diagnosis. For example, while the BRCA1 gene mutation is useful as an indicator of an increased risk for breast cancer, it has limited use in cancer diagnosis because only 6.2% of breast cancers are BRCA1 positive. Malone et al.,  JAMA  279:922 (1998). See also, Mewman et al.,  JAMA  279:915 (1998) (correlation of only 3.3%). \\n     There are four primary classifications of breast cancer varying by the site of origin and the extent of disease development.\\n         I. Ductal carcinoma in situ (DCIS): Malignant transformation of ductal epithelial cells that remain in their normal position. DCIS is a purely localized disease, incapable of metastasis.   II. Invasive ductal carcinoma (IDC): Malignancy of the ductal epithelial cells breaking through the basal membrane and into the supporting tissue of the breast. IDC may eventually spread elsewhere in the body.   III. Lobular carcinoma in situ (LCIS): Malignancy arising in a single lobule of the breast that fail to extend through the lobule wall, it generally remains localized.   IV. Infiltrating lobular carcinoma (ILC): Malignancy arising in a single lobule of the breast and invading directly through the lobule wall into adjacent tissues.   By virtue of its invasion beyond the lobule wall, ILC may penetrate lymphatics and blood vessels and spread to distant sites.       \\n\\n     For purpose of determining prognosis and treatment, these four breast cancer types have been staged according to the size of the primary tumor (T), the involvement of lymph nodes (N), and the presence of metastasis (M). Although DCIS by definition represents localized stage I disease, the other forms of breast cancer may range from stage II to stage IV. There are additional prognostic factors that further serve to guide surgical and medical intervention. The most common ones are total number of lymph nodes involved, ER (estrogen receptor) status, Her2/neu receptor status and histologic grades. \\n     Breast cancers are diagnosed into the appropriate stage categories recognizing that different treatments are more effective for different stages of cancer. Stage TX indicates that primary tumor cannot be assessed (i.e., tumor was removed or breast tissue was removed). Stage T0 is characterized by abnormalities such as hyperplasia but with no evidence of primary tumor. Stage Tis is characterized by carcinoma in situ, intraductal carcinoma, lobular carcinoma in situ, or Paget&#39;s disease of the nipple with no tumor. Stage T1 (I) is characterized as having a tumor of 2 cm or less in the greatest dimension. Within stage T1, Tmic indicates microinvasion of 0.1 cm or less, T1a indicates a tumor of between 0.1 to 0.5 cm, T1b indicates a tumor of between 0.5 to 1 cm, and T1c indicates tumors of between 1 cm to 2 cm. Stage T2 (II) is characterized by tumors from 2 cm to 5 cm in the greatest dimension. Tumors greater than 5 cm in size are classified as stage T3 (III). Stage T4 (IV) indicates a tumor of any size with extension to the chest wall or skin. Within stage T4, T4a indicates extension of the tumor to the chest wall, T4b indicates edema or ulceration of the skin of the breast or satellite skin nodules confined to the same breast, T4c indicates a combination of T4a and T4b, and T4d indicates inflammatory carcinoma.  AJCC Cancer Staging Handbook  pp. 159-70 (Irvin D. Fleming et al. eds., 5 th  ed. 1998). In addition to standard staging, breast tumors may be classified according to their estrogen receptor and progesterone receptor protein status. Fisher et al.,  Breast Cancer Research and Treatment  7:147 (1986). Additional pathological status, such as HER2/neu status may also be useful. Thor et al.,  J. Nat&#39;l. Cancer Inst.  90:1346 (1998); Paik et al.,  J. Nat&#39;l. Cancer Inst.  90:1361 (1998); Hutchins et al.,  Proc. Am. Soc. Clin. Oncology  17:A2 (1998); and Simpson et al.,  J. Clin. Oncology  18:2059 (2000). \\n     In addition to the staging of the primary tumor, breast cancer metastases to regional lymph nodes may be staged. Stage NX indicates that the lymph nodes cannot be assessed (e.g., previously removed). Stage N0 indicates no regional lymph node metastasis. Stage N1 indicates metastasis to movable ipsilateral axillary lymph nodes. Stage N2 indicates metastasis to ipsilateral axillary lymph nodes fixed to one another or to other structures. Stage N3 indicates metastasis to ipsilateral internal mammary lymph nodes. Id. \\n     Stage determination has potential prognostic value and provides criteria for designing optimal therapy. Simpson et al.,  J. Clin. Oncology  18:2059 (2000). Generally, pathological staging of breast cancer is preferable to clinical staging because the former gives a more accurate prognosis. However, clinical staging would be preferred if it were as accurate as pathological staging because it does not depend on an invasive procedure to obtain tissue for pathological evaluation. Staging of breast cancer would be improved by detecting new markers in cells, tissues, or bodily fluids which could differentiate between different stages of invasion. Progress in this field will allow more rapid and reliable method for treating breast cancer patients. \\n     Treatment of breast cancer is generally decided after an accurate staging of the primary tumor. Primary treatment options include breast conserving therapy (lumpectomy, breast irradiation, and surgical staging of the axilla), and modified radical mastectomy. Additional treatments include chemotherapy, regional irradiation, and, in extreme cases, terminating estrogen production by ovarian ablation. \\n     Until recently, the customary treatment for all breast cancer was mastectomy. Fonseca et al.,  Annals of Internal Medicine  127:1013 (1997). However, recent data indicate that less radical procedures may be equally effective, in terms of survival, for early stage breast cancer. Fisher et al.,  J. of Clinical Oncology  16:441 (1998). The treatment options for a patient with early stage breast cancer (i.e., stage Tis) may be breast-sparing surgery followed by localized radiation therapy at the breast. Alternatively, mastectomy optionally coupled with radiation or breast reconstruction may be employed. These treatment methods are equally effective in the early stages of breast cancer. \\n     Patients with stage I and stage II breast cancer require surgery with chemotherapy and/or hormonal therapy. Surgery is of limited use in stage III and stage IV patients. Thus, these patients are better candidates for chemotherapy and radiation therapy with surgery limited to biopsy to permit initial staging or subsequent restaging because cancer is rarely curative at this stage of the disease.  AJCC Cancer Staging Handbook  84, 164-65 (Irvin D. Fleming et al. eds., 5 th  ed. 1998). \\n     In an effort to provide more treatment options to patients, efforts are underway to define an earlier stage of breast cancer with low recurrence which could be treated with lumpectomy without postoperative radiation treatment. While a number of attempts have been made to classify early stage breast cancer, no consensus recommendation on postoperative radiation treatment has been obtained from these studies. Page et al.,  Cancer  75:1219 (1995); Fisher et al.,  Cancer  75:1223 (1995); Silverstein et al.,  Cancer  77:2267 (1996). \\n     As discussed above, each of the methods for diagnosing and staging ovarian, and breast cancer is limited by the technology employed. Accordingly, there is need for sensitive molecular and cellular markers for the detection of ovarian, and breast cancer as well as pancreatic cancer. There is a need for molecular markers for the accurate staging, including clinical and pathological staging, of ovarian, pancreatic or breast cancers to optimize treatment methods. Finally, there is a need for sensitive molecular and cellular markers to monitor the progress of cancer treatments, including markers that can detect recurrence of ovarian, pancreatic or breast cancers following remission. \\n     The present invention provides alternative methods of treating ovarian, pancreatic or breast cancer that overcome the limitations of conventional therapeutic methods as well as offer additional advantages that will be apparent from the detailed description below. \\n     Growth and metastasis of solid tumors are also dependent on angiogenesis. Folkman, J., 1986 , Cancer Research,  46, 467-473; Folkman, J., 1989 , Journal of the National Cancer Institute,  82, 4-6. It has been shown, for example, that tumors which enlarge to greater than 2 mm must obtain their own blood supply and do so by inducing the growth of new capillary blood vessels. Once these new blood vessels become embedded in the tumor, they provide a means for tumor cells to enter the circulation and metastasize to distant sites such as liver, lung or bone. Weidner, N., et al., 1991 , The New England Journal of Medicine,  324(1), 1-8. \\n     Angiogenesis, defined as the growth or sprouting of new blood vessels from existing vessels, is a complex process that primarily occurs during embryonic development. The process is distinct from vasculogenesis, in that the new endothelial cells lining the vessel arise from proliferation of existing cells, rather than differentiating from stem cells. The process is invasive and dependent upon proteolysis of the extracellular matrix (ECM), migration of new endothelial cells, and synthesis of new matrix components. Angiogenesis occurs during embryogenic development of the circulatory system; however, in adult humans, angiogenesis only occurs as a response to a pathological condition (except during the reproductive cycle in women). \\n     Under normal physiological conditions in adults, angiogenesis takes place only in very restricted situations such as hair growth and wounding healing. Auerbach, W. and Auerbach, R., 1994 , Pharmacol Ther.  63(3):265-3 11; Ribatti et al., 1991 , Haematologica  76(4):3 11-20; Risau, 1997 , Nature  386(6626):67 1-4. Angiogenesis progresses by a stimulus which results in the formation of a migrating column of endothelial cells. Proteolytic activity is focused at the advancing tip of this “vascular sprout”, which breaks down the ECM sufficiently to permit the column of cells to infiltrate and migrate. Behind the advancing front, the endothelial cells differentiate and begin to adhere to each other, thus forming a new basement membrane. The cells then cease proliferation and finally define a lumen for the new arteriole or capillary. \\n     Unregulated angiogenesis has gradually been recognized to be responsible for a wide range of disorders, including, but not limited to, cancer, cardiovascular disease, rheumatoid arthritis, psoriasis and diabetic retinopathy. Folkman, 1995 , Nat Med  1(1):27-31; Isner, 1999 , Circulation  99(13): 1653-5; Koch, 1998 , Arthritis Rheum  41(6):951-62; Walsh, 1999 , Rheumatology  (Oxford) 38(2):103-12; Ware and Simons, 1997 , Nat Med  3(2): 158-64. \\n     Of particular interest is the observation that angiogenesis is required by solid tumors for their growth and metastases. Folkman, 1986 supra; Folkman 1990 , J Natl. Cancer Inst.,  82(1) 4-6; Folkman, 1992 , Semin Cancer Biol  3(2):65-71; Zetter, 1998 , Annu Rev Med  49:407-24. A tumor usually begins as a single aberrant cell which can proliferate only to a size of a few cubic millimeters due to the distance from available capillary beds, and it can stay ‘dormant’ without further growth and dissemination for a long period of time. Some tumor cells then switch to the angiogenic phenotype to activate endothelial cells, which proliferate and mature into new capillary blood vessels. These newly formed blood vessels not only allow for continued growth of the primary tumor, but also for the dissemination and recolonization of metastatic tumor cells. The precise mechanisms that control the angiogenic switch is not well understood, but it is believed that neovascularization of tumor mass results from the net balance of a multitude of angiogenesis stimulators and inhibitors Folkman, 1995, supra. \\n     One of the most potent angiogenesis inhibitors is endostatin identified by O&#39;Reilly and Folkman. O&#39;Reilly et al., 1997 , Cell  88(2):277-85; O&#39;Reilly et al., 1994 , Cell  79(2):3 15-28. Its discovery was based on the phenomenon that certain primary tumors can inhibit the growth of distant metastases. O&#39;Reilly and Folkman hypothesized that a primary tumor initiates angiogenesis by generating angiogenic stimulators in excess of inhibitors. However, angiogenic inhibitors, by virtue of their longer half life in the circulation, reach the site of a secondary tumor in excess of the stimulators. The net result is the growth of primary tumor and inhibition of secondary tumor. Endostatin is one of a growing list of such angiogenesis inhibitors produced by primary tumors. It is a proteolytic fragment of a larger protein: endostatin is a 20 kDa fragment of collagen XVIII (amino acid H1132-K1315 in murine collagen XVIII). Endostatin has been shown to specifically inhibit endothelial cell proliferation in vitro and block angiogenesis in vivo. More importantly, administration of endostatin to tumor-bearing mice leads to significant tumor regression, and no toxicity or drug resistance has been observed even after multiple treatment cycles. Boehm et al., 1997, Nature 390(6658):404-407. The fact that endostatin targets genetically stable endothelial cells and inhibits a variety of solid tumors makes it a very attractive candidate for anticancer therapy. Fidler and Ellis, 1994, Cell 79(2):185-8; Gastl et al., 1997, Oncology 54(3):177-84; Hinsbergh et al., 1999, Ann Oncol 10 Suppl 4:60-3. In addition, angiogenesis inhibitors have been shown to be more effective when combined with radiation and chemotherapeutic agents. Klement, 2000, J. Clin Invest, 105(8) R15-24. Browder, 2000, Cancer Res. 6-(7) 1878-86, Arap et al., 1998, Science 279(5349):377-80; Mauceri et al., 1998, Nature 394(6690):287-91. \\n     SUMMARY OF THE INVENTION \\n     The present invention solves many needs in the art by providing nucleic acid molecules, polypeptides and antibodies thereto, variants and derivatives of the nucleic acids and polypeptides, and agonists and antagonists thereto that may be used to identify, diagnose, monitor, stage, image and treat ovarian cancer and/or non-cancerous disease states in ovarian; identify and monitor ovarian tissue; and identify and design agonists and antagonists of polypeptides of the invention. The invention also provides gene therapy, methods for producing transgenic animals and cells, and methods for producing engineered ovarian tissue for treatment and research. \\n     One aspect of the present invention relates to nucleic acid molecules that are specific to ovarian cells, ovarian tissue and/or the ovarian organ. These ovarian specific nucleic acids (OSNAs) may be a naturally occurring cDNA, genomic DNA, RNA, or a fragment of one of these nucleic acids, or may be a non-naturally occurring nucleic acid molecule. If the OSNA is genomic DNA, then the OSNA is an ovarian specific gene (OSG). If the OSNA is RNA, then it is an ovarian specific transcript encoded by an OSG. Due to alternative splicing and transcriptional modification one OSG may encode for multiple ovarian specific RNAs. In a preferred embodiment, the nucleic acid molecule encodes a polypeptide that is specific to ovarian. More preferred is a nucleic acid molecule that encodes a polypeptide comprising an amino acid sequence of SEQ ID NO: 129-295. In another preferred embodiment, the nucleic acid molecule comprises a nucleic acid sequence of SEQ ID NO: 1-128. For the OSNA sequences listed herein, DEX0455 — 001.nt.1 corresponds to SEQ ID NO: 1. For sequences with multiple splice variants, the parent sequence DEX0455 — 001.nt.1, will be followed by DEX0455 — 001.nt.2, etc. for each splice variant. The sequences off the corresponding peptides are listed as DEX0455 — 001.aa.1, etc. For the mapping of all of the nucleotides and peptides, see the table in the Example 1 section below. \\n     This aspect of the present invention also relates to nucleic acid molecules that selectively hybridize or exhibit substantial sequence similarity to nucleic acid molecules encoding an Ovarian Specific Protein (OSP), or that selectively hybridize or exhibit substantial sequence similarity to an OSNA. In one embodiment of the present invention the nucleic acid molecule comprises an allelic variant of a nucleic acid molecule encoding an OSP, or an allelic variant of an OSNA. In another embodiment, the nucleic acid molecule comprises a part of a nucleic acid sequence that encodes an OSP or a part of a nucleic acid sequence of an OSNA. \\n     In addition, this aspect of the present invention relates to a nucleic acid molecule further comprising one or more expression control sequences controlling the transcription and/or translation of all or a part of an OSNA or the transcription and/or translation of a nucleic acid molecule that encodes all or a fragment of an OSP. \\n     Another aspect of the present invention relates to vectors and/or host cells comprising a nucleic acid molecule of this invention. In a preferred embodiment, the nucleic acid molecule of the vector and/or host cell encodes all or a fragment of an OSP. In another preferred embodiment, the nucleic acid molecule of the vector and/or host cell comprises all or a part of an OSNA. Vectors and host cells of the present invention are useful in the recombinant production of polypeptides, particularly OSPs of the present invention. \\n     Another aspect of the present invention relates to polypeptides encoded by a nucleic acid molecule of this invention. The polypeptide may comprise either a fragment or a full-length protein. In a preferred embodiment, the polypeptide is an OSP. However, this aspect of the present invention also relates to mutant proteins (muteins) of OSPs, fusion proteins of which a portion is an OSP, and proteins and polypeptides encoded by allelic variants of an OSNA as provided herein. \\n     A further aspect of the present invention is a novel splice variant which encodes an amino acid sequence that provides a novel region to be targeted for the generation of reagents that can be used in the detection and/or treatment of cancer. The novel amino acid sequence may lead to a unique protein structure, protein subcellular localization, biochemical processing or function. This information can be used to directly or indirectly facilitate the generation of additional or novel therapeutics or diagnostics. The nucleotide sequence in this novel splice variant can be used as a nucleic acid probe for the diagnosis and/or treatment of cancer. \\n     Another aspect of the present invention relates to antibodies and other binders that specifically bind to a polypeptide of the instant invention. Accordingly antibodies or binders of the present invention specifically bind to OSPs, muteins, fusion proteins, and/or homologous proteins or polypeptides encoded by allelic variants of an OSNA as provided herein. \\n     Another aspect of the present invention relates to agonists and antagonists of the nucleic acid molecules and polypeptides of this invention. The agonists and antagonists of the instant invention may be used to treat ovarian cancer and non-cancerous disease states in ovarian and to produce engineered ovarian tissue. \\n     Another aspect of the present invention relates to methods for using the nucleic acid molecules to detect or amplify nucleic acid molecules that have similar or identical nucleic acid sequences compared to the nucleic acid molecules described herein. Such methods are useful in identifying, diagnosing, monitoring, staging, imaging and treating ovarian cancer and/or non-cancerous disease states in ovarian. Such methods are also useful in identifying and/or monitoring ovarian tissue. In addition, measurement of levels of one or more of the nucleic acid molecules of this invention may be useful as a diagnostic as part of a panel in combination with known other markers, particularly those described in the ovarian cancer background section above. \\n     Another aspect of the present invention relates to use of the nucleic acid molecules of this invention in gene therapy, for producing transgenic animals and cells, and for producing engineered ovarian tissue for treatment and research. \\n     Another aspect of the present invention relates to methods for detecting polypeptides of this invention, preferably using antibodies thereto. Such methods are useful to identify, diagnose, monitor, stage, image and treat ovarian cancer and non-cancerous disease states in ovarian. In addition, measurement of levels of one or more of the polypeptides of this invention may be useful to identify, diagnose, monitor, stage, and/or image ovarian cancer in combination with known other markers, particularly those described in the ovarian cancer background section above. The polypeptides of the present invention can also be used to identify and/or monitor ovarian tissue, and to produce engineered ovarian tissue. \\n     Yet another aspect of the present invention relates to a computer readable means of storing the nucleic acid and amino acid sequences of the invention. The records of the computer readable means can be accessed for reading and displaying of sequences for comparison, alignment and ordering of the sequences of the invention to other sequences. In addition, the computer records regarding the nucleic acid and/or amino acid sequences and/or measurements of their levels may be used alone or in combination with other markers to diagnose ovarian related diseases. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE FIGURES \\n         FIG. 1  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 049.nt.5 (SEQ ID NO:96; EpCAM) and DEX0455 — 049.nt.1 (SEQ ID NO:92; Ovr232); \\n         FIG. 2  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 049.aa.5 (SEQ ID NO:255; EpCAM) and DEX0455 — 049.aa.1 (SEQ ID NO:251; Ovr232); \\n         FIG. 3  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 049.nt.5 (SEQ ID NO:96; EpCAM) and DEX0455 — 049.nt.2 (SEQ ID NO:93; Ovr232v1); \\n         FIG. 4  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 049.aa.5 (SEQ ID NO:255; EpCAM) and DEX0455 — 049.aa.2 (SEQ ID NO:252; Ovr232v1); \\n         FIG. 5  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 049.nt.5 (SEQ ID NO:96; EpCAM) and DEX0455 — 049.nt.3 (SEQ ID NO:94; Ovr232v2); \\n         FIG. 6  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 049.aa.5 (SEQ ID NO:255; EpCAM) and DEX0455 — 049.aa.3 (SEQ ID NO:253; Ovr232v2); \\n         FIG. 7  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 049.nt.5 (SEQ ID NO:96; EpCAM) and DEX0455 — 049.nt.4 (SEQ ID NO:95; Ovr232v3); \\n         FIG. 8  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 049.aa.5 (SEQ ID NO:255; EpCAM) and DEX0455 — 049.aa.4 (SEQ ID NO:254; Ovr232v3); \\n         FIG. 9  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 051.nt.1 (SEQ ID NO:98; Ovr107) and DEX0455 — 051.nt.2 (SEQ ID NO:99); \\n         FIG. 10  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 051.aa.1 (SEQ ID NO:258; Ovr107) and DEX0455 — 051.aa.3 (SEQ ID NO:260); \\n         FIG. 11  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 051.aa.1 (SEQ ID NO:258; Ovr107) and DEX0455 — 051.aa.2 (SEQ ID NO:259); \\n         FIG. 12  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 051.nt.1 (SEQ ID NO:98; Ovr107) and DEX0455 — 051.nt.3 (SEQ ID NO:100); \\n         FIG. 13  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 051.nt.1 (SEQ ID NO:98; Ovr107) and DEX0455 — 051.nt.4 (SEQ ID NO:101); \\n         FIG. 14  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 051.nt.1 (SEQ ID NO:98; Ovr107) and DEX0455 — 051.nt.5 (SEQ ID NO:102); \\n         FIG. 15  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 051.nt.1 (SEQ ID NO:98; Ovr107) and DEX0455 — 051.nt.6 (SEQ ID NO:103; Ovr107v4); \\n         FIG. 16  is a nucleotide sequence alignment which shows regions of similarity and difference between DEX0455 — 053.nt.1 (SEQ ID NO:108; Ovr110) and DEX0455 — 053.nt.2 (SEQ ID NO:109; Ovr110v1); \\n         FIG. 17  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 053.aa.1 (SEQ ID NO:268; Ovr110) and DEX0455 — 053.aa.2 (SEQ ID NO:269); \\n         FIG. 18  is an amino acid sequence alignment which shows regions of similarity and difference between DEX0455 — 053.aa.1 (SEQ ID NO:268; Ovr110) and DEX0455 — 053.aa.3 (SEQ ID NO:270). \\n     \\n    \\n    \\n     DETAILED DESCRIPTION OF THE INVENTION \\n     Definitions and General Techniques \\n     Unless otherwise defined herein, scientific and technical terms used in connection with the present invention shall have the meanings that are commonly understood by those of ordinary skill in the art. Further, unless otherwise required by context, singular terms shall include pluralities and plural terms shall include the singular. Generally, nomenclatures used in connection with, and techniques of, cell and tissue culture, molecular biology, immunology, microbiology, genetics and protein and nucleic acid chemistry and hybridization described herein are those well known and commonly used in the art. The methods and techniques of the present invention are generally performed according to conventional methods well known in the art and as described in various general and more specific references that are cited and discussed throughout the present specification unless otherwise indicated. See, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual,  2d ed., Cold Spring Harbor Laboratory Press (1989) and Sambrook et al.,  Molecular Cloning: A Laboratory Manual,  3d ed., Cold Spring Harbor Press (2001); Ausubel et al.,  Current Protocols in Molecular Biology , Greene Publishing Associates (1992, and Supplements to 2000); Ausubel et al.,  Short Protocols in Molecular Biology: A Compendium of Methods from Current Protocols in Molecular Biology— 4 th    Ed ., Wiley &amp; Sons (1999); Harlow and Lane,  Antibodies: A Laboratory Manual , Cold Spring Harbor Laboratory Press (1990); and Harlow and Lane,  Using Antibodies: A Laboratory Manual , Cold Spring Harbor Laboratory Press (1999). \\n     Enzymatic reactions and purification techniques are performed according to manufacturer&#39;s specifications, as commonly accomplished in the art or as described herein. The nomenclatures used in connection with, and the laboratory procedures and techniques of, analytical chemistry, synthetic organic chemistry, and medicinal and pharmaceutical chemistry described herein are those well known and commonly used in the art. Standard techniques are used for chemical syntheses, chemical analyses, pharmaceutical preparation, formulation, and delivery, and treatment of patients. \\n     The following terms, unless otherwise indicated, shall be understood to have the following meanings: \\n     A “nucleic acid molecule” of this invention refers to a polymeric form of nucleotides and includes both sense and antisense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the above. A nucleotide refers to a ribonucleotide, deoxynucleotide or a modified form of either type of nucleotide. A “nucleic acid molecule” as used herein is synonymous with “nucleic acid” and “polynucleotide.” The term “nucleic acid molecule” usually refers to a molecule of at least 10 bases in length, unless otherwise specified. The term includes single- and double-stranded forms of DNA. In addition, a polynucleotide may include either or both naturally occurring and modified nucleotides linked together by naturally occurring and/or non-naturally occurring nucleotide linkages. \\n     Nucleotides are represented by single letter symbols in nucleic acid molecule sequences. The following table lists symbols identifying nucleotides or groups of nucleotides which may occupy the symbol position on a nucleic acid molecule. See Nomenclature Committee of the International Union of Biochemistry (NC-IUB), Nomenclature for incompletely specified bases in nucleic acid sequences, Recommendations 1984 ., Eur J Biochem.  150(1):1-5 (1985). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Complementary \\n               \\n               \\n                 Symbol \\n                 Meaning \\n                 Group/Origin of Designation \\n                 Symbol \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 a \\n                 a \\n                 Adenine \\n                 t/u \\n               \\n               \\n                 g \\n                 g \\n                 Guanine \\n                 c \\n               \\n               \\n                 c \\n                 c \\n                 Cytosine \\n                 g \\n               \\n               \\n                 t \\n                 t \\n                 Thymine \\n                 a \\n               \\n               \\n                 u \\n                 u \\n                 Uracil \\n                 a \\n               \\n               \\n                 r \\n                 g or a \\n                 puRine \\n                 y \\n               \\n               \\n                 y \\n                 t/u or c \\n                 pYrimidine \\n                 r \\n               \\n               \\n                 m \\n                 a or c \\n                 aMino \\n                 k \\n               \\n               \\n                 k \\n                 g or t/u \\n                 Keto \\n                 m \\n               \\n               \\n                 s \\n                 g or c \\n                 Strong interactions 3H-bonds \\n                 w \\n               \\n               \\n                 w \\n                 a or t/u \\n                 Weak interactions 2H-bonds \\n                 s \\n               \\n               \\n                 b \\n                 g or c or t/u \\n                 not a \\n                 v \\n               \\n               \\n                 d \\n                 a or g or t/u \\n                 not c \\n                 h \\n               \\n               \\n                 h \\n                 a or c or t/u \\n                 not g \\n                 d \\n               \\n               \\n                 v \\n                 a or g or c \\n                 not t, not u \\n                 b \\n               \\n               \\n                 n \\n                 a or g or c \\n                 aNy \\n                 n \\n               \\n               \\n                   \\n                 or t/u, \\n               \\n               \\n                   \\n                 unknown, or \\n               \\n               \\n                   \\n                 other \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     The nucleic acid molecules may be modified chemically or biochemically or may contain non-natural or derivatized nucleotide bases, as will be readily appreciated by those of skill in the art. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoramidates, carbamates, etc.), charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), pendent moieties (e.g., polypeptides), intercalators (e.g., acridine, psoralen, etc.), chelators, alkylators, and modified linkages (e.g., alpha anomeric nucleic acids, etc.) The term “nucleic acid molecule” also includes any topological conformation, including single-stranded, double-stranded, partially duplexed, triplexed, hairpinned, circular and padlocked conformations. Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known in the art and include, for example, those in which peptide linkages substitute for phosphate linkages in the backbone of the molecule. \\n     A “gene” is defined as a nucleic acid molecule that comprises a nucleic acid sequence that encodes a polypeptide and the expression control sequences that surround the nucleic acid sequence that encodes the polypeptide. For instance, a gene may comprise a promoter, one or more enhancers, a nucleic acid sequence that encodes a polypeptide, downstream regulatory sequences and, possibly, other nucleic acid sequences involved in regulation of the expression of an RNA. As is well known in the art, eukaryotic genes usually contain both exons and introns. The term “exon” refers to a nucleic acid sequence found in genomic DNA that is bioinformatically predicted and/or experimentally confirmed to contribute contiguous sequence to a mature mRNA transcript. The term “intron” refers to a nucleic acid sequence found in genomic DNA that is predicted and/or confirmed to not contribute to a mature mRNA transcript, but rather to be “spliced out” during processing of the transcript. \\n     A nucleic acid molecule or polypeptide is “derived” from a particular species if the nucleic acid molecule or polypeptide has been isolated from the particular species, or if the nucleic acid molecule or polypeptide is homologous to a nucleic acid molecule or polypeptide isolated from a particular species. \\n     An “isolated” or “substantially pure” nucleic acid or polynucleotide (e.g., an RNA, DNA or a mixed polymer) is one which is substantially separated from other cellular components that naturally accompany the native polynucleotide in its natural host cell, e.g., ribosomes, polymerases, or genomic sequences with which it is naturally associated. The term embraces a nucleic acid or polynucleotide that (1) has been removed from its naturally occurring environment, (2) is not associated with all or a portion of a polynucleotide in which the “isolated polynucleotide” is found in nature, (3) is operatively linked to a polynucleotide which it is not linked to in nature, (4) does not occur in nature as part of a larger sequence or (5) includes nucleotides or internucleoside bonds that are not found in nature. The term “isolated” or “substantially pure” also can be used in reference to recombinant or cloned DNA isolates, chemically synthesized polynucleotide analogs, or polynucleotide analogs that are biologically synthesized by heterologous systems. The term “isolated nucleic acid molecule” includes nucleic acid molecules that are integrated into a host cell chromosome at a heterologous site, recombinant fusions of a native fragment to a heterologous sequence, recombinant vectors present as episomes or as integrated into a host cell chromosome. \\n     A “part” of a nucleic acid molecule refers to a nucleic acid molecule that comprises a partial contiguous sequence of at least 10 bases of the reference nucleic acid molecule. Preferably, a part comprises at least 15 to 20 bases of a reference nucleic acid molecule. In theory, a nucleic acid sequence of 17 nucleotides is of sufficient length to occur at random less frequently than once in the three gigabase human genome, and thus provides a nucleic acid probe that can uniquely identify the reference sequence in a nucleic acid mixture of genomic complexity. A preferred part is one that comprises a nucleic acid sequence that can encode at least 6 contiguous amino acid sequences (fragments of at least 18 nucleotides) because they are useful in directing the expression or synthesis of peptides that are useful in mapping the epitopes of the polypeptide encoded by the reference nucleic acid. See, e.g., Geysen et al.,  Proc. Natl. Acad. Sci. USA  81:3998-4002 (1984); and U.S. Pat. Nos. 4,708,871 and 5,595,915, the disclosures of which are incorporated herein by reference in their entireties. A part may also comprise at least 25, 30, 35 or 40 nucleotides of a reference nucleic acid molecule, or at least 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400 or 500 nucleotides of a reference nucleic acid molecule. A part of a nucleic acid molecule may comprise no other nucleic acid sequences. Alternatively, a part of a nucleic acid may comprise other nucleic acid sequences from other nucleic acid molecules. \\n     The term “oligonucleotide” refers to a nucleic acid molecule generally comprising a length of 200 bases or fewer. The term often refers to single-stranded deoxyribonucleotides, but it can refer as well to single- or double-stranded ribonucleotides, RNA:DNA hybrids and double-stranded DNAs, among others. Preferably, oligonucleotides are 10 to 60 bases in length and most preferably 12, 13, 14, 15, 16, 17, 18, 19 or 20 bases in length. Other preferred oligonucleotides are 25, 30, 35, 40, 45, 50, 55 or 60 bases in length. Oligonucleotides may be single-stranded, e.g. for use as probes or primers, or may be double-stranded, e.g. for use in the construction of a mutant gene. Oligonucleotides of the invention can be either sense or antisense oligonucleotides. An oligonucleotide can be derivatized or modified as discussed above for nucleic acid molecules. \\n     Oligonucleotides, such as single-stranded DNA probe oligonucleotides, often are synthesized by chemical methods, such as those implemented on automated oligonucleotide synthesizers. However, oligonucleotides can be made by a variety of other methods, including in vitro recombinant DNA-mediated techniques and by expression of DNAs in cells and organisms. Initially, chemically synthesized DNAs typically are obtained without a 5′ phosphate. The 5′ ends of such oligonucleotides are not substrates for phosphodiester bond formation by ligation reactions that employ DNA ligases typically used to form recombinant DNA molecules. Where ligation of such oligonucleotides is desired, a phosphate can be added by standard techniques, such as those that employ a kinase and ATP. The 3′ end of a chemically synthesized oligonucleotide generally has a free hydroxyl group and, in the presence of a ligase, such as T4 DNA ligase, readily will form a phosphodiester bond with a 5′ phosphate of another polynucleotide, such as another oligonucleotide. As is well known, this reaction can be prevented selectively, where desired, by removing the 5′ phosphates of the other polynucleotide(s) prior to ligation. \\n     The term “naturally occurring nucleotide” referred to herein includes naturally occurring deoxyribonucleotides and ribonucleotides. The term “modified nucleotides” referred to herein includes nucleotides with modified or substituted sugar groups and the like. The term “nucleotide linkages” referred to herein includes nucleotide linkages such as phosphorothioate, phosphorodithioate, phosphoroselenoate, phosphorodiselenoate, phosphoroanilothioate, phoshoraniladate, phosphoroamidate, and the like. See e.g., LaPlanche et al.  Nucl. Acids Res.  14:9081-9093 (1986); Stein et al.  Nucl. Acids Res.  16:3209-3221 (1988); Zon et al.  Anti - Cancer Drug Design  6:539-568 (1991); Zon et al., in Eckstein (ed.)  Oligonucleotides and Analogues: A Practical Approach , pp. 87-108, Oxford University Press (1991); Uhlmann and Peyman  Chemical Reviews  90:543 (1990), and U.S. Pat. No. 5,151,510, the disclosure of which is hereby incorporated by reference in its entirety. \\n     Unless specified otherwise, the left hand end of a polynucleotide sequence in sense orientation is the 5′ end and the right hand end of the sequence is the 3′ end. In addition, the left hand direction of a polynucleotide sequence in sense orientation is referred to as the 5′ direction, while the right hand direction of the polynucleotide sequence is referred to as the 3′ direction. Further, unless otherwise indicated, each nucleotide sequence is set forth herein as a sequence of deoxyribonucleotides. It is intended, however, that the given sequence be interpreted as would be appropriate to the polynucleotide composition: for example, if the isolated nucleic acid is composed of RNA, the given sequence intends ribonucleotides, with uridine substituted for thymidine. \\n     The term “allelic variant” refers to one of two or more alternative naturally occurring forms of a gene, wherein each gene possesses a unique nucleotide sequence. In a preferred embodiment, different alleles of a given gene have similar or identical biological properties. \\n     The term “percent sequence identity” in the context of nucleic acid sequences refers to the residues in two sequences which are the same when aligned for maximum correspondence. The length of sequence identity comparison may be over a stretch of at least about nine nucleotides, usually at least about 20 nucleotides, more usually at least about 24 nucleotides, typically at least about 28 nucleotides, more typically at least about 32 nucleotides, and preferably at least about 36 or more nucleotides. There are a number of different algorithms known in the art which can be used to measure nucleotide sequence identity. For instance, polynucleotide sequences can be compared using FASTA, Gap or Bestfit, which are programs in Wisconsin Package Version 10.0, Genetics Computer Group (GCG), Madison, Wis. FASTA, which includes, e.g., the programs FASTA2 and FASTA3, provides alignments and percent sequence identity of the regions of the best overlap between the query and search sequences (Pearson,  Methods Enzymol.  183: 63-98 (1990); Pearson,  Methods Mol. Biol.  132: 185-219 (2000); Pearson,  Methods Enzymol.  266: 227-258 (1996); Pearson,  J. Mol. Biol.  276: 71-84 (1998)). Unless otherwise specified, default parameters for a particular program or algorithm are used. For instance, percent sequence identity between nucleic acid sequences can be determined using FASTA with its default parameters (a word size of 6 and the NOPAM factor for the scoring matrix) or using Gap with its default parameters as provided in GCG Version 6.1. \\n     A reference to a nucleic acid sequence encompasses its complement unless otherwise specified. Thus, a reference to a nucleic acid molecule having a particular sequence should be understood to encompass its complementary strand, with its complementary sequence. The complementary strand is also useful, e.g., for antisense therapy, double-stranded RNA (dsRNA) inhibition (RNAi), combination of triplex and antisense, hybridization probes and PCR primers. \\n     In the molecular biology art, researchers use the terms “percent sequence identity”, “percent sequence similarity” and “percent sequence homology” interchangeably. In this application, these terms shall have the same meaning with respect to nucleic acid sequences only. \\n     The term “substantial similarity” or “substantial sequence similarity,” when referring to a nucleic acid or fragment thereof, indicates that, when optimally aligned with appropriate nucleotide insertions or deletions with another nucleic acid (or its complementary strand), there is nucleotide sequence identity in at least about 50%, more preferably 60% of the nucleotide bases, usually at least about 70%, more usually at least about 80%, preferably at least about 90%, and more preferably at least about 95-98% of the nucleotide bases, as measured by any well known algorithm of sequence identity, such as FASTA, BLAST or Gap, as discussed above. \\n     Alternatively, substantial similarity exists between a first and second nucleic acid sequence when the first nucleic acid sequence or fragment thereof hybridizes to an antisense strand of the second nucleic acid, under selective hybridization conditions. Typically, selective hybridization will occur between the first nucleic acid sequence and an antisense strand of the second nucleic acid sequence when there is at least about 55% sequence identity between the first and second nucleic acid sequences—preferably at least about 65%, more preferably at least about 75%, and most preferably at least about 90%—over a stretch of at least about 14 nucleotides, more preferably at least 17 nucleotides, even more preferably at least 20, 25, 30, 35, 40, 50, 60, 70, 80, 90 or 100 nucleotides. \\n     Nucleic acid hybridization will be affected by such conditions as salt concentration, temperature, solvents, the base composition of the hybridizing species, length of the complementary regions, and the number of nucleotide base mismatches between the hybridizing nucleic acids, as will be readily appreciated by those skilled in the art. “Stringent hybridization conditions” and “stringent wash conditions” in the context of nucleic acid hybridization experiments depend upon a number of different physical parameters. The most important parameters include temperature of hybridization, base composition of the nucleic acids, salt concentration and length of the nucleic acid. One having ordinary skill in the art knows how to vary these parameters to achieve a particular stringency of hybridization. In general, “stringent hybridization” is performed at about 25° C. below the thermal melting point (T m ) for the specific DNA hybrid under a particular set of conditions. “Stringent washing” is performed at temperatures about 5° C. lower than the T m  for the specific DNA hybrid under a particular set of conditions. The T m  is the temperature at which 50% of the target sequence hybridizes to a perfectly matched probe. See Sambrook (1989), supra, p. 9.51. \\n     The T m  for a particular DNA-DNA hybrid can be estimated by the formula:\\n \\n T   m =81.5° C.+16.6(log 10 [Na + ])+0.41(fraction  G+C )−0.63(% formamide)−(600 /l )\\n \\nwhere l is the length of the hybrid in base pairs.\\n \\n     The T m  for a particular RNA-RNA hybrid can be estimated by the formula:\\n \\n T   m =79.8° C.+18.5(log 10 [Na + ])+0.58(fraction  G+C )+11.8(fraction  G+C ) 2 −0.35(% formamide)−(820/ l ).\\n \\n     The T m  for a particular RNA-DNA hybrid can be estimated by the formula:\\n \\n T   m =79.8° C.+18.5(log 10 [Na + ])+0.58(fraction  G+C )+11.8(fraction  G+C ) 2 −0.50(% formamide)−(820 /l ).\\n \\n     In general, the T m  decreases by 1-1.5° C. for each 1% of mismatch between two nucleic acid sequences. Thus, one having ordinary skill in the art can alter hybridization and/or washing conditions to obtain sequences that have higher or lower degrees of sequence identity to the target nucleic acid. For instance, to obtain hybridizing nucleic acids that contain up to 10% mismatch from the target nucleic acid sequence, 10-15° C. would be subtracted from the calculated T m  of a perfectly matched hybrid, and then the hybridization and washing temperatures adjusted accordingly. Probe sequences may also hybridize specifically to duplex DNA under certain conditions to form triplex or other higher order DNA complexes. The preparation of such probes and suitable hybridization conditions are well known in the art. \\n     An example of stringent hybridization conditions for hybridization of complementary nucleic acid sequences having more than 100 complementary residues on a filter in a Southern or Northern blot or for screening a library is 50% formamide/6×SSC at 42° C. for at least ten hours and preferably overnight (approximately 16 hours). Another example of stringent hybridization conditions is 6×SSC at 68° C. without formamide for at least ten hours and preferably overnight. An example of moderate stringency hybridization conditions is 6×SSC at 55° C. without formamide for at least ten hours and preferably overnight. An example of low stringency hybridization conditions for hybridization of complementary nucleic acid sequences having more than 100 complementary residues on a filter in a Southern or northern blot or for screening a library is 6×SSC at 42° C. for at least ten hours. Hybridization conditions to identify nucleic acid sequences that are similar but not identical can be identified by experimentally changing the hybridization temperature from 68° C. to 42° C. while keeping the salt concentration constant (6×SSC), or keeping the hybridization temperature and salt concentration constant (e.g. 42° C. and 6×SSC) and varying the formamide concentration from 50% to 0%. Hybridization buffers may also include blocking agents to lower background. These agents are well known in the art. See Sambrook et al. (1989), supra, pages 8.46 and 9.46-9.58. See also Ausubel (1992), supra, Ausubel (1999), supra, and Sambrook (2001), supra. \\n     Wash conditions also can be altered to change stringency conditions. An example of stringent wash conditions is a 0.2×SSC wash at 65° C. for 15 minutes (see Sambrook (1989), supra, for SSC buffer). Often the high stringency wash is preceded by a low stringency wash to remove excess probe. An exemplary medium stringency wash for duplex DNA of more than 100 base pairs is 1×SSC at 45° C. for 15 minutes. An exemplary low stringency wash for such a duplex is 4×SSC at 40° C. for 15 minutes. In general, signal-to-noise ratio of 2× or higher than that observed for an unrelated probe in the particular hybridization assay indicates detection of a specific hybridization. \\n     As defined herein, nucleic acids that do not hybridize to each other under stringent conditions are still substantially similar to one another if they encode polypeptides that are substantially identical to each other. This occurs, for example, when a nucleic acid is created synthetically or recombinantly using a high codon degeneracy as permitted by the redundancy of the genetic code. \\n     Hybridization conditions for nucleic acid molecules that are shorter than 100 nucleotides in length (e.g., for oligonucleotide probes) may be calculated by the formula:\\n \\n T   m =81.5° C.+16.6(log 10 [Na + ])+0.41(fraction  G+C )−(600 /N ),\\n \\nwherein N is change length and the [Na + ] is 1 M or less. See Sambrook (1989), supra, p. 11.46. For hybridization of probes shorter than 100 nucleotides, hybridization is usually performed under stringent conditions (5-10° C. below the T m ) using high concentrations (0.1-1.0 pmol/ml) of probe. Id. at p. 11.45. Determination of hybridization using mismatched probes, pools of degenerate probes or “guessmers,” as well as hybridization solutions and methods for empirically determining hybridization conditions are well known in the art. See, e.g., Ausubel (1999), supra; Sambrook (1989), supra, pp. 11.45-11.57.\\n \\n     The term “digestion” or “digestion of DNA” refers to catalytic cleavage of the DNA with a restriction enzyme that acts only at certain sequences in the DNA. The various restriction enzymes referred to herein are commercially available and their reaction conditions, cofactors and other requirements for use are known and routine to the skilled artisan. For analytical purposes, typically, 1 μg of plasmid or DNA fragment is digested with about 2 units of enzyme in about 20 μl of reaction buffer. For the purpose of isolating DNA fragments for plasmid construction, typically 5 to 50 μg of DNA are digested with 20 to 250 units of enzyme in proportionately larger volumes. Appropriate buffers and substrate amounts for particular restriction enzymes are described in standard laboratory manuals, such as those referenced below, and are specified by commercial suppliers. Incubation times of about 1 hour at 37° C. are ordinarily used, but conditions may vary in accordance with standard procedures, the supplier&#39;s instructions and the particulars of the reaction. After digestion, reactions may be analyzed, and fragments may be purified by electrophoresis through an agarose or polyacrylamide gel, using well known methods that are routine for those skilled in the art. \\n     The term “ligation” refers to the process of forming phosphodiester bonds between two or more polynucleotides, which most often are double-stranded DNAs. Techniques for ligation are well known to the art and protocols for ligation are described in standard laboratory manuals and references, such as, e.g., Sambrook (1989), supra. \\n     Genome-derived “single exon probes,” are probes that comprise at least part of an exon (“reference exon”) and can hybridize detectably under high stringency conditions to transcript-derived nucleic acids that include the reference exon but do not hybridize detectably under high stringency conditions to nucleic acids that lack the reference exon. Single exon probes typically further comprise, contiguous to a first end of the exon portion, a first intronic and/or intergenic sequence that is identically contiguous to the exon in the genome, and may contain a second intronic and/or intergenic sequence that is identically contiguous to the exon in the genome. The minimum length of genome-derived single exon probes is defined by the requirement that the exonic portion be of sufficient length to hybridize under high stringency conditions to transcript-derived nucleic acids, as discussed above. The maximum length of genome-derived single exon probes is defined by the requirement that the probes contain portions of no more than one exon. The single exon probes may contain priming sequences not found in contiguity with the rest of the probe sequence in the genome, which priming sequences are useful for PCR and other amplification-based technologies. In another aspect, the invention is directed to single exon probes based on the OSNAs disclosed herein. \\n     In one embodiment, the term “microarray” refers to a “nucleic acid microarray” having a substrate-bound plurality of nucleic acids, hybridization to each of the plurality of bound nucleic acids being separately detectable. The substrate can be solid or porous, planar or non-planar, unitary or distributed. Nucleic acid microarrays include all the devices so called in Schena (ed.),  DNA Microarrays: A Practical Approach  ( Practical Approach Series ), Oxford University Press (1999);  Nature Genet.  21(1) (suppl.): 1-60 (1999); Schena (ed.),  Microarray Biochip: Tools and Technology , Eaton Publishing Company/BioTechniques Books Division (2000). Additionally, these nucleic acid microarrays include a substrate-bound plurality of nucleic acids in which the plurality of nucleic acids are disposed on a plurality of beads, rather than on a unitary planar substrate, as is described, inter alia, in Brenner et al.,  Proc. Natl. Acad. Sci. USA  97(4): 1665-1670 (2000). Examples of nucleic acid microarrays may be found in U.S. Pat. Nos. 6,391,623, 6,383,754, 6,383,749, 6,380,377, 6,379,897, 6,376,191, 6,372,431, 6,351,712 6,344,316, 6,316,193, 6,312,906, 6,309,828, 6,309,824, 6,306,643, 6,300,063, 6,287,850, 6,284,497, 6,284,465, 6,280,954, 6,262,216, 6,251,601, 6,245,518, 6,263,287, 6,251,601, 6,238,866, 6,228,575, 6,214,587, 6,203,989, 6,171,797, 6,103,474, 6,083,726, 6,054,274, 6,040,138, 6,083,726, 6,004,755, 6,001,309, 5,958,342, 5,952,180, 5,936,731, 5,843,655, 5,814,454, 5,837,196, 5,436,327, 5,412,087, and 5,405,783, the disclosures of which are incorporated herein by reference in their entireties. \\n     In an alternative embodiment, a “microarray” may also refer to a “peptide microarray” or “protein microarray” having a substrate-bound collection or plurality of polypeptides, the binding to each of the plurality of bound polypeptides being separately detectable. Alternatively, the peptide microarray may have a plurality of binders, including but not limited to monoclonal antibodies, polyclonal antibodies, phage display binders, yeast 2 hybrid binders, and aptamers, which can specifically detect the binding of the polypeptides of this invention. The array may be based on autoantibody detection to the polypeptides of this invention, see Robinson et al.,  Nature Medicine  8(3):295-301 (2002). Examples of peptide arrays may be found in WO 02/31463, WO 02/25288, WO 01/94946, WO 01/88162, WO 01/68671, WO 01/57259, WO 00/61806, WO 00/54046, WO 00/47774, WO 99/40434, WO 99/39210, and WO 97/42507 and U.S. Pat. Nos. 6,268,210, 5,766,960, and 5,143,854, the disclosures of which are incorporated herein by reference in their entireties. \\n     In addition, determination of the levels of the OSNA or OSP may be made in a multiplex manner using techniques described in WO 02/29109, WO 02/24959, WO 01/83502, WO01/73113, WO 01/59432, WO 01/57269, and WO 99/67641, the disclosures of which are incorporated herein by reference in their entireties. \\n     The term “mutant”, “mutated”, or “mutation” when applied to nucleic acid sequences means that nucleotides in a nucleic acid sequence may be inserted, deleted or changed compared to a reference nucleic acid sequence. A single alteration may be made at a locus (a point mutation) or multiple nucleotides may be inserted, deleted or changed at a single locus. In addition, one or more alterations may be made at any number of loci within a nucleic acid sequence. In a preferred embodiment of the present invention, the nucleic acid sequence is the wild type nucleic acid sequence encoding an OSP or is an OSNA. The nucleic acid sequence may be mutated by any method known in the art including those mutagenesis techniques described infra. \\n     The term “error-prone PCR” refers to a process for performing PCR under conditions where the copying fidelity of the DNA polymerase is low, such that a high rate of point mutations is obtained along the entire length of the PCR product. See, e.g., Leung et al.,  Technique  1: 11-15 (1989) and Caldwell et al.,  PCR Methods Applic.  2: 28-33 (1992). \\n     The term “oligonucleotide-directed mutagenesis” refers to a process which enables the generation of site-specific mutations in any cloned DNA segment of interest. See, e.g., Reidhaar-Olson et al.,  Science  241: 53-57 (1988). \\n     The term “assembly PCR” refers to a process which involves the assembly of a PCR product from a mixture of small DNA fragments. A large number of different PCR reactions occur in parallel in the same vial, with the products of one reaction priming the products of another reaction. \\n     The term “sexual PCR mutagenesis” or “DNA shuffling” refers to a method of error-prone PCR coupled with forced homologous recombination between DNA molecules of different but highly related DNA sequence in vitro, caused by random fragmentation of the DNA molecule based on sequence similarity, followed by fixation of the crossover by primer extension in an error-prone PCR reaction. See, e.g., Stemmer,  Proc. Natl. Acad. Sci. U.S.A.  91: 10747-10751 (1994). DNA shuffling can be carried out between several related genes (“Family shuffling”). \\n     The term “in vivo mutagenesis” refers to a process of generating random mutations in any cloned DNA of interest which involves the propagation of the DNA in a strain of bacteria such as  E. coli  that carries mutations in one or more of the DNA repair pathways. These “mutator” strains have a higher random mutation rate than that of a wild-type parent. Propagating the DNA in a mutator strain will eventually generate random mutations within the DNA. \\n     The term “cassette mutagenesis” refers to any process for replacing a small region of a double-stranded DNA molecule with a synthetic oligonucleotide “cassette” that differs from the native sequence. The oligonucleotide often contains completely and/or partially randomized native sequence. \\n     The term “recursive ensemble mutagenesis” refers to an algorithm for protein engineering (protein mutagenesis) developed to produce diverse populations of phenotypically related mutants whose members differ in amino acid sequence. This method uses a feedback mechanism to control successive rounds of combinatorial cassette mutagenesis. See, e.g., Arkin et al.,  Proc. Natl. Acad. Sci. U.S.A.  89: 7811-7815 (1992). \\n     The term “exponential ensemble mutagenesis” refers to a process for generating combinatorial libraries with a high percentage of unique and functional mutants, wherein small groups of residues are randomized in parallel to identify, at each altered position, amino acids which lead to functional proteins. See, e.g., Delegrave et al.,  Biotechnology Research  11: 1548-1552 (1993); Arnold,  Current Opinion in Biotechnology  4: 450-455 (1993). \\n     “Operatively linked” expression control sequences refers to a linkage in which the expression control sequence is either contiguous with the gene of interest to control the gene of interest, or acts in trans or at a distance to control the gene of interest. \\n     The term “expression control sequence” as used herein refers to polynucleotide sequences which are necessary to affect the expression of coding sequences to which they are operatively linked. Expression control sequences are sequences which control the transcription, post-transcriptional events and translation of nucleic acid sequences. Expression control sequences include appropriate transcription initiation, termination, promoter and enhancer sequences; efficient RNA processing signals such as splicing and polyadenylation signals; sequences that stabilize cytoplasmic mRNA; sequences that enhance translation efficiency (e.g., ribosome binding sites); sequences that enhance protein stability; and when desired, sequences that enhance protein secretion. The nature of such control sequences differs depending upon the host organism; in prokaryotes, such control sequences generally include promoter, ribosomal binding site, and transcription termination sequence. The term “control sequences” is intended to include, at a minimum, all components whose presence is essential for expression, and can also include additional components whose presence is advantageous, for example, leader sequences and fusion partner sequences. \\n     The term “vector,” as used herein, is intended to refer to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a “plasmid”, which refers to a circular double-stranded DNA loop into which additional DNA segments may be ligated. Other vectors include cosmids, bacterial artificial chromosomes (BAC) and yeast artificial chromosomes (YAC). Another type of vector is a viral vector, wherein additional DNA segments may be ligated into the viral genome. Viral vectors that infect bacterial cells are referred to as bacteriophages. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication). Other vectors can be integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively linked. Such vectors are referred to herein as “recombinant expression vectors” (or simply, “expression vectors”). In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. In the present specification, “plasmid” and “vector” may be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include other forms of expression vectors that serve equivalent functions. \\n     The term “recombinant host cell” (or simply “host cell”), as used herein, is intended to refer to a cell into which a recombinant expression vector has been introduced. It should be understood that such terms are intended to refer not only to the particular subject cell but to the progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term “host cell” as used herein. \\n     As used herein, the phrase “open reading frame” and the equivalent acronym “ORF” refers to that portion of a transcript-derived nucleic acid that can be translated in its entirety into a sequence of contiguous amino acids. As so defined, an ORF has length, measured in nucleotides, exactly divisible by 3. As so defined, an ORF need not encode the entirety of a natural protein. \\n     As used herein, the phrase “ORF-encoded peptide” refers to the predicted or actual translation of an ORF. \\n     As used herein, the phrase “degenerate variant” of a reference nucleic acid sequence is meant to be inclusive of all nucleic acid sequences that can be directly translated, using the standard genetic code, to provide an amino acid sequence identical to that translated from the reference nucleic acid sequence. \\n     The term “polypeptide” encompasses both naturally occurring and non-naturally occurring proteins and polypeptides, as well as polypeptide fragments and polypeptide mutants, derivatives and analogs thereof. A polypeptide may be monomeric or polymeric. Further, a polypeptide may comprise a number of different modules within a single polypeptide each of which has one or more distinct activities. A preferred polypeptide in accordance with the invention comprises an OSP encoded by a nucleic acid molecule of the instant invention, or a fragment, mutant, analog or derivative thereof. \\n     The term “isolated protein” or “isolated polypeptide” is a protein or polypeptide that by virtue of its origin or source of derivation (1) is not associated with naturally associated components that accompany it in its native state, (2) is free of other proteins from the same species (3) is expressed by a cell from a different species, or (4) does not occur in nature. Thus, a polypeptide that is chemically synthesized or synthesized in a cellular system different from the cell from which it naturally originates will be “isolated” from its naturally associated components. A polypeptide or protein may also be rendered substantially free of naturally associated components by isolation, using protein purification techniques well known in the art. \\n     A protein or polypeptide is “substantially pure,” “substantially homogeneous” or “substantially purified” when at least about 60% to 75% of a sample exhibits a single species of polypeptide. The polypeptide or protein may be monomeric or multimeric. A substantially pure polypeptide or protein will typically comprise about 50%, 60%, 70%, 80% or 90% W/W of a protein sample, more usually about 95%, and preferably will be over 99% pure. Protein purity or homogeneity may be determined by a number of means well known in the art, such as polyacrylamide gel electrophoresis of a protein sample, followed by visualizing a single polypeptide band upon staining the gel with a stain well known in the art. For certain purposes, higher resolution may be provided by using HPLC or other means well known in the art for purification. \\n     The term “fragment” when used herein with respect to polypeptides of the present invention refers to a polypeptide that has an amino-terminal and/or carboxy-terminal deletion compared to a full-length OSP. In a preferred embodiment, the fragment is a contiguous sequence in which the amino acid sequence of the fragment is identical to the corresponding positions in the naturally occurring polypeptide. Fragments typically are at least 5, 6, 7, 8, 9 or 10 amino acids long, preferably at least 12, 14, 16 or 18 amino acids long, more preferably at least 20 amino acids long, more preferably at least 25, 30, 35, 40 or 45, amino acids, even more preferably at least 50 or 60 amino acids long, and even more preferably at least 70 amino acids long. \\n     A “derivative” when used herein with respect to polypeptides of the present invention refers to a polypeptide which is substantially similar in primary structural sequence to an OSP but which includes, e.g., in vivo or in vitro chemical and biochemical modifications that are not found in the OSP. Such modifications include, for example, acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of phosphotidylinositol, cross-linking, cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cystine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodination, methylation, myristoylation, oxidation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, transfer-RNA mediated addition of amino acids to proteins such as arginylation, and ubiquitination. Other modifications include, e.g., labeling with radionuclides, and various enzymatic modifications, as will be readily appreciated by those skilled in the art. A variety of methods for labeling polypeptides and of substituents or labels useful for such purposes are well known in the art, and include radioactive isotopes such as  125 I,  32 P,  35 S,  14 C and  3 H, ligands which bind to labeled antiligands (e.g., antibodies), fluorophores, chemiluminescent agents, enzymes, and antiligands which can serve as specific binding pair members for a labeled ligand. The choice of label depends on the sensitivity required, ease of conjugation with the primer, stability requirements, and available instrumentation. Methods for labeling polypeptides are well known in the art. See Ausubel (1992), supra; Ausubel (1999), supra. \\n     The term “fusion protein” refers to polypeptides of the present invention coupled to a heterologous amino acid sequence. Fusion proteins are useful because they can be constructed to contain two or more desired functional elements from two or more different proteins. A fusion protein comprises at least 10 contiguous amino acids from a polypeptide of interest, more preferably at least 20 or 30 amino acids, even more preferably at least 40, 50 or 60 amino acids, yet more preferably at least 75, 100 or 125 amino acids. Fusion proteins can be produced recombinantly by constructing a nucleic acid sequence that encodes the polypeptide or a fragment thereof in frame with a nucleic acid sequence encoding a different protein or peptide and then expressing the fusion protein. Alternatively, a fusion protein can be produced chemically by crosslinking the polypeptide or a fragment thereof to another protein. \\n     The term “analog” refers to both polypeptide analogs and non-peptide analogs. The term “polypeptide analog” as used herein refers to a polypeptide that is comprised of a segment of at least 25 amino acids that has substantial identity to a portion of an amino acid sequence but which contains non-natural amino acids or non-natural inter-residue bonds. In a preferred embodiment, the analog has the same or similar biological activity as the native polypeptide. Typically, polypeptide analogs comprise a conservative amino acid substitution (or insertion or deletion) with respect to the naturally occurring sequence. Analogs typically are at least 20 amino acids long, preferably at least 50 amino acids long or longer, and can often be as long as a full-length naturally occurring polypeptide. \\n     The term “non-peptide analog” refers to a compound with properties that are analogous to those of a reference polypeptide. A non-peptide compound may also be termed a “peptide mimetic” or a “peptidomimetic.” Such compounds are often developed with the aid of computerized molecular modeling. Peptide mimetics that are structurally similar to useful peptides may be used to produce an equivalent effect. Generally, peptidomimetics are structurally similar to a paradigm polypeptide (i.e., a polypeptide that has a desired biochemical property or pharmacological activity), but have one or more peptide linkages optionally replaced by a linkage selected from the group consisting of: —CH 2 NH—, —CH 2 S—, —CH 2 —CH 2 —, —CH═CH—(cis and trans), —COCH 2 —, —CH(OH)CH 2 —, and —CH 2 SO—, by methods well known in the art. Systematic substitution of one or more amino acids of a consensus sequence with a D-amino acid of the same type (e.g., D-lysine in place of L-lysine) may also be used to generate more stable peptides. In addition, constrained peptides comprising a consensus sequence or a substantially identical consensus sequence variation may be generated by methods known in the art (Rizo et al.,  Ann. Rev. Biochem.  61:387-418 (1992)). For example, one may add internal cysteine residues capable of forming intramolecular disulfide bridges which cyclize the peptide. \\n     The term “mutant” or “mutein” when referring to a polypeptide of the present invention relates to an amino acid sequence containing substitutions, insertions or deletions of one or more amino acids compared to the amino acid sequence of an OSP. A mutein may have one or more amino acid point substitutions, in which a single amino acid at a position has been changed to another amino acid, one or more insertions and/or deletions, in which one or more amino acids are inserted or deleted, respectively, in the sequence of the naturally occurring protein, and/or truncations of the amino acid sequence at either or both the amino or carboxy termini. Further, a mutein may have the same or different biological activity as the naturally occurring protein. For instance, a mutein may have an increased or decreased biological activity. A mutein has at least 50% sequence similarity to the wild type protein, preferred is 60% sequence similarity, more preferred is 70% sequence similarity. Even more preferred are muteins having 80%, 85% or 90% sequence similarity to an OSP. In an even more preferred embodiment, a mutein exhibits 95% sequence identity, even more preferably 97%, even more preferably 98% and even more preferably 99%. Sequence similarity may be measured by any common sequence analysis algorithm, such as GAP or BESTFIT or other variation Smith-Waterman alignment. See, T. F. Smith and M. S. Waterman, J. Mol. Biol. 147:195-197 (1981) and W. R. Pearson, Genomics 11:635-650 (1991). \\n     Preferred amino acid substitutions are those which: (1) reduce susceptibility to proteolysis, (2) reduce susceptibility to oxidation, (3) alter binding affinity for forming protein complexes, (4) alter binding affinity or enzymatic activity, and (5) confer or modify other physicochemical or functional properties of such analogs. For example, single or multiple amino acid substitutions (preferably conservative amino acid substitutions) may be made in the naturally occurring sequence (preferably in the portion of the polypeptide outside the domain(s) forming intermolecular contacts. In a preferred embodiment, the amino acid substitutions are moderately conservative substitutions or conservative substitutions. In a more preferred embodiment, the amino acid substitutions are conservative substitutions. A conservative amino acid substitution should not substantially change the structural characteristics of the parent sequence (e.g., a replacement amino acid should not tend to disrupt a helix that occurs in the parent sequence, or disrupt other types of secondary structure that characterize the parent sequence). Examples of art-recognized polypeptide secondary and tertiary structures are described in Creighton (ed.),  Proteins, Structures and Molecular Principles , W. H. Freeman and Company (1984); Branden et al. (ed.),  Introduction to Protein Structure , Garland Publishing (1991); Thornton et al.,  Nature  354:105-106 (1991). \\n     As used herein, the twenty conventional amino acids and their abbreviations follow conventional usage. See Golub et al. (eds.),  Immunology—A Synthesis  2 nd  Ed., Sinauer Associates (1991). Stereoisomers (e.g., D-amino acids) of the twenty conventional amino acids, unnatural amino acids such as α-, α-disubstituted amino acids, N-alkyl amino acids, and other unconventional amino acids may also be suitable components for polypeptides of the present invention. Examples of unconventional amino acids include: 4-hydroxyproline, γ-carboxyglutamate, εE-N,N,N-trimethyllysine, ε-N-acetyllysine, O-phosphoserine, N-acetylserine, N-formylmethionine, 3-methylhistidine, 5-hydroxylysine, s-N-methylarginine, and other similar amino acids and imino acids (e.g., 4-hydroxyproline). In the polypeptide notation used herein, the lefthand direction is the amino terminal direction and the right hand direction is the carboxy-terminal direction, in accordance with standard usage and convention. \\n     By “homology” or “homologous” when referring to a polypeptide of the present invention it is meant polypeptides from different organisms with a similar sequence to the encoded amino acid sequence of an OSP and a similar biological activity or function. Although two polypeptides are said to be “homologous,” this does not imply that there is necessarily an evolutionary relationship between the polypeptides. Instead, the term “homologous” is defined to mean that the two polypeptides have similar amino acid sequences and similar biological activities or functions. In a preferred embodiment, a homologous polypeptide is one that exhibits 50% sequence similarity to OSP, preferred is 60% sequence similarity, more preferred is 70% sequence similarity. Even more preferred are homologous polypeptides that exhibit 80%, 85% or 90% sequence similarity to an OSP. In yet a more preferred embodiment, a homologous polypeptide exhibits 95%, 97%, 98% or 99% sequence similarity. \\n     When “sequence similarity” is used in reference to polypeptides, it is recognized that residue positions that are not identical often differ by conservative amino acid substitutions. In a preferred embodiment, a polypeptide that has “sequence similarity” comprises conservative or moderately conservative amino acid substitutions. A “conservative amino acid substitution” is one in which an amino acid residue is substituted by another amino acid residue having a side chain (R group) with similar chemical properties (e.g., charge or hydrophobicity). In general, a conservative amino acid substitution will not substantially change the functional properties of a protein. In cases where two or more amino acid sequences differ from each other by conservative substitutions, the percent sequence identity or degree of similarity may be adjusted upwards to correct for the conservative nature of the substitution. Means for making this adjustment are well known to those of skill in the art. See, e.g., Pearson,  Methods Mol. Biol.  24: 307-31 (1994). \\n     For instance, the following six groups each contain amino acids that are conservative substitutions for one another: \\n     1) Serine (S), Threonine (T); \\n     2) Aspartic Acid (D), Glutamic Acid (E); \\n     3) Asparagine (N), Glutamine (Q); \\n     4) Arginine (R), Lysine (K); \\n     5) Isoleucine (I), Leucine (L), Methionine (M), Alanine (A), Valine (V), and \\n     6) Phenylalanine (F), Tyrosine (Y), Tryptophan (W). \\n     Alternatively, a conservative replacement is any change having a positive value in the PAM250 log-likelihood matrix disclosed in Gonnet et al.,  Science  256: 1443-45 (1992). A “moderately conservative” replacement is any change having a nonnegative value in the PAM250 log-likelihood matrix. \\n     Sequence similarity for polypeptides, which is also referred to as sequence identity, is typically measured using sequence analysis software. Protein analysis software matches similar sequences using measures of similarity assigned to various substitutions, deletions and other modifications, including conservative amino acid substitutions. For instance, GCG contains programs such as “Gap” and “Bestfit” which can be used with default parameters to determine sequence homology or sequence identity between closely related polypeptides, such as homologous polypeptides from different species of organisms or between a wild type protein and a mutein thereof. See, e.g., GCG Version 6.1. Other programs include FASTA, discussed supra. \\n     A preferred algorithm when comparing a sequence of the invention to a database containing a large number of sequences from different organisms is the computer program BLAST, especially blastp or tblastn. See, e.g., Altschul et al.,  J. Mol. Biol.  215: 403-410 (1990); Altschul et al.,  Nucleic Acids Res.  25:3389-402 (1997). Preferred parameters for blastp are: \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n                 Expectation value: \\n                 \\u200210 (default) \\n               \\n               \\n                   \\n                 Filter: \\n                 seg (default) \\n               \\n               \\n                   \\n                 Cost to open a gap: \\n                 \\u200211 (default) \\n               \\n               \\n                   \\n                 Cost to extend a gap: \\n                 \\u20031 (default \\n               \\n               \\n                   \\n                 Max. alignments: \\n                 100 (default) \\n               \\n               \\n                   \\n                 Word size: \\n                 \\u200211 (default) \\n               \\n               \\n                   \\n                 No. of descriptions: \\n                 100 (default) \\n               \\n               \\n                   \\n                 Penalty Matrix: \\n                 BLOSUM62 \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     The length of polypeptide sequences compared for homology will generally be at least about 16 amino acid residues, usually at least about 20 residues, more usually at least about 24 residues, typically at least about 28 residues, and preferably more than about 35 residues. When searching a database containing sequences from a large number of different organisms, it is preferable to compare amino acid sequences. \\n     Algorithms other than blastp for database searching using amino acid sequences are known in the art. For instance, polypeptide sequences can be compared using FASTA, a program in GCG Version 6.1. FASTA (e.g., FASTA2 and FASTA3) provides alignments and percent sequence identity of the regions of the best overlap between the query and search sequences (Pearson (1990), supra; Pearson (2000), supra. For example, percent sequence identity between amino acid sequences can be determined using FASTA with its default or recommended parameters (a word size of 2 and the PAM250 scoring matrix), as provided in GCG Version 6.1. \\n     An “antibody” refers to an intact immunoglobulin, or to an antigen-binding portion thereof that competes with the intact antibody for specific binding to a molecular species, e.g., a polypeptide of the instant invention. Antigen-binding portions may be produced by recombinant DNA techniques or by enzymatic or chemical cleavage of intact antibodies. Antigen-binding portions include, inter alia, Fab, Fab′, F(ab′) 2 , Fv, dAb, and complementarity determining region (CDR) fragments, single-chain antibodies (scFv), chimeric antibodies, diabodies and polypeptides that contain at least a portion of an immunoglobulin that is sufficient to confer specific antigen binding to the polypeptide. A Fab fragment is a monovalent fragment consisting of the VL, VH, CL and CH1 domains; a F(ab′) 2  fragment is a bivalent fragment comprising two Fab fragments linked by a disulfide bridge at the hinge region; a Fd fragment consists of the VH and CH1 domains; a Fv fragment consists of the VL and VH domains of a single arm of an antibody; and a dAb fragment consists of a VH domain. See, e.g., Ward et al.,  Nature  341: 544-546 (1989). \\n     By “bind specifically” and “specific binding” as used herein it is meant the ability of the antibody to bind to a first molecular species in preference to binding to other molecular species with which the antibody and first molecular species are admixed. An antibody is said to “recognize” a first molecular species when it can bind specifically to that first molecular species. \\n     A single-chain antibody (scFv) is an antibody in which VL and VH regions are paired to form a monovalent molecule via a synthetic linker that enables them to be made as a single protein chain. See, e.g., Bird et al.,  Science  242: 423-426 (1988); Huston et al.,  Proc. Natl. Acad. Sci. USA  85: 5879-5883 (1988). Diabodies are bivalent, bispecific antibodies in which VH and VL domains are expressed on a single polypeptide chain, but using a linker that is too short to allow for pairing between the two domains on the same chain, thereby forcing the domains to pair with complementary domains of another chain and creating two antigen binding sites. See e.g., Holliger et al.,  Proc. Natl. Acad. Sci. USA  90: 6444-6448 (1993); Poljak et al.,  Structure  2: 1121-1123 (1994). One or more CDRs may be incorporated into a molecule either covalently or noncovalently to make it an immunoadhesin. An immunoadhesin may incorporate the CDR(s) as part of a larger polypeptide chain, may covalently link the CDR(s) to another polypeptide chain, or may incorporate the CDR(s) noncovalently. The CDRs permit the immunoadhesin to specifically bind to a particular antigen of interest. A chimeric antibody is an antibody that contains one or more regions from one antibody and one or more regions from one or more other antibodies. \\n     An antibody may have one or more binding sites. If there is more than one binding site, the binding sites may be identical to one another or may be different. For instance, a naturally occurring immunoglobulin has two identical binding sites, a single-chain antibody or Fab fragment has one binding site, while a “bispecific” or “bifunctional” antibody has two different binding sites. \\n     An “isolated antibody” is an antibody that (1) is not associated with naturally-associated components, including other naturally-associated antibodies, that accompany it in its native state, (2) is free of other proteins from the same species, (3) is expressed by a cell from a different species, or (4) does not occur in nature. It is known that purified proteins, including purified antibodies, may be stabilized with non-naturally-associated components. The non-naturally-associated component may be a protein, such as albumin (e.g., BSA) or a chemical such as polyethylene glycol (PEG). \\n     A “neutralizing antibody” or “an inhibitory antibody” is an antibody that inhibits the activity of a polypeptide or blocks the binding of a polypeptide to a ligand that normally binds to it. An “activating antibody” is an antibody that increases the activity of a polypeptide. \\n     The term “epitope” includes any protein determinant capable of specific binding to an immunoglobulin or T-cell receptor. Epitopic determinants usually consist of chemically active surface groupings of molecules such as amino acids or sugar side chains and usually have specific three-dimensional structural characteristics, as well as specific charge characteristics. An antibody is said to specifically bind an antigen when the dissociation constant is less than 1 μM, preferably less than 100 nM and most preferably less than 10 nM. \\n     The term “patient” includes human and veterinary subjects. \\n     Throughout this specification and claims, the word “comprise,” or variations such as “comprises” or “comprising,” will be understood to imply the inclusion of a stated integer or group of integers but not the exclusion of any other integer or group of integers. \\n     The term “ovarian specific” refers to a nucleic acid molecule or polypeptide that is expressed predominantly in the ovarian as compared to other tissues in the body. In a preferred embodiment, a “ovarian specific” nucleic acid molecule or polypeptide is detected at a level that is 1.5-fold higher than any other tissue in the body. In a more preferred embodiment, the “ovarian specific” nucleic acid molecule or polypeptide is detected at a level that is 2-fold higher than any other tissue in the body, more preferably 5-fold higher, still more preferably at least 10-fold, 15-fold, 20-fold, 25-fold, 50-fold or 100-fold higher than any other tissue in the body. Nucleic acid molecule levels may be measured by nucleic acid hybridization, such as Northern blot hybridization, or quantitative PCR. Polypeptide levels may be measured by any method known to accurately quantitate protein levels, such as Western blot analysis. \\n     Nucleic Acid Molecules, Regulatory Sequences, Vectors, Host Cells and Recombinant Methods of Making Polypeptides \\n     Nucleic Acid Molecules \\n     One aspect of the invention provides isolated nucleic acid molecules that are specific to the ovarian or to ovarian cells or tissue or that are derived from such nucleic acid molecules. These isolated ovarian specific nucleic acids (OSNAs) may comprise cDNA genomic DNA, RNA, or a combination thereof, a fragment of one of these nucleic acids, or may be a non-naturally occurring nucleic acid molecule. An OSNA may be derived from an animal. In a preferred embodiment, the OSNA is derived from a human or other mammal. In a more preferred embodiment, the OSNA is derived from a human or other primate. In an even more preferred embodiment, the OSNA is derived from a human. \\n     In a preferred embodiment, the nucleic acid molecule encodes a polypeptide that is specific to ovarian, an ovarian-specific polypeptide (OSP). In a more preferred embodiment, the nucleic acid molecule encodes a polypeptide that comprises an amino acid sequence of SEQ ID NO: 129-295. In another highly preferred embodiment, the nucleic acid molecule comprises a nucleic acid sequence of SEQ ID NO: 1-128. Nucleotide sequences of the instantly-described nucleic acid molecules were determined by assembling several DNA molecules from either public or proprietary databases. Some of the underlying DNA sequences are the result, directly or indirectly, of at least one enzymatic polymerization reaction (e.g., reverse transcription and/or polymerase chain reaction) using an automated sequencer (such as the MegaBACE™ 1000, Amersham Biosciences, Sunnyvale, Calif., USA). \\n     Nucleic acid molecules of the present invention may also comprise sequences that selectively hybridize to a nucleic acid molecule encoding an OSNA or a complement or antisense thereof. The hybridizing nucleic acid molecule may or may not encode a polypeptide or may or may not encode an OSP. However, in a preferred embodiment, the hybridizing nucleic acid molecule encodes an OSP. In a more preferred embodiment, the invention provides a nucleic acid molecule that selectively hybridizes to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule that encodes a polypeptide comprising an amino acid sequence of SEQ ID NO: 129-295. In an even more preferred embodiment, the invention provides a nucleic acid molecule that selectively hybridizes to a nucleic acid molecule comprising the nucleic acid sequence of SEQ ID NO: 1-128 or the antisense sequence thereof. Preferably, the nucleic acid molecule selectively hybridizes to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule encoding an OSP under low stringency conditions. More preferably, the nucleic acid molecule selectively hybridizes to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule encoding an OSP under moderate stringency conditions. Most preferably, the nucleic acid molecule selectively hybridizes to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule encoding an OSP under high stringency conditions. In a preferred embodiment, the nucleic acid molecule hybridizes under low, moderate or high stringency conditions to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule encoding a polypeptide comprising an amino acid sequence of SEQ ID NO: 129-295. In a more preferred embodiment, the nucleic acid molecule hybridizes under low, moderate or high stringency conditions to a nucleic acid molecule or the antisense sequence of a nucleic acid molecule comprising a nucleic acid sequence selected from SEQ ID NO: 1-128. \\n     Nucleic acid molecules of the present invention may also comprise nucleic acid sequences that exhibit substantial sequence similarity to a nucleic acid encoding an OSP or a complement of the encoding nucleic acid molecule. In this embodiment, it is preferred that the nucleic acid molecule exhibit substantial sequence similarity to a nucleic acid molecule encoding human OSP. More preferred is a nucleic acid molecule exhibiting substantial sequence similarity to a nucleic acid molecule encoding a polypeptide having an amino acid sequence of SEQ ID NO: 129-295. By substantial sequence similarity it is meant a nucleic acid molecule having at least 60%, more preferably at least 70%, even more preferably at least 80% and even more preferably at least 85% sequence identity with a nucleic acid molecule encoding an OSP, such as a polypeptide having an amino acid sequence of SEQ ID NO: 129-295. In a more preferred embodiment, the similar nucleic acid molecule is one that has at least 90%, more preferably at least 95%, more preferably at least 97%, even more preferably at least 98%, and still more preferably at least 99% sequence identity with a nucleic acid molecule encoding an OSP. Most preferred in this embodiment is a nucleic acid molecule that has at least 99.5%, 99.6%, 99.7%, 99.8% or 99.9% sequence identity with a nucleic acid molecule encoding an OSP. \\n     The nucleic acid molecules of the present invention are also inclusive of those exhibiting substantial sequence similarity to an OSNA or its complement. In this embodiment, it is preferred that the nucleic acid molecule exhibit substantial sequence similarity to a nucleic acid molecule having a nucleic acid sequence of SEQ ID NO: 1-128. By substantial sequence similarity it is meant a nucleic acid molecule that has at least 60%, more preferably at least 70%, even more preferably at least 80% and even more preferably at least 85% sequence identity with an OSNA, such as one having a nucleic acid sequence of SEQ ID NO: 1-128. More preferred is a nucleic acid molecule that has at least 90%, more preferably at least 95%, more preferably at least 97%, even more preferably at least 98%, and still more preferably at least 99% sequence identity with an OSNA. Most preferred is a nucleic acid molecule that has at least 99.5%, 99.6%, 99.7%, 99.8% or 99.9% sequence identity with an OSNA. \\n     Nucleic acid molecules that exhibit substantial sequence similarity are inclusive of sequences that exhibit sequence identity over their entire length to an OSNA or to a nucleic acid molecule encoding an OSP, as well as sequences that are similar over only a part of its length. In this case, the part is at least 50 nucleotides of the OSNA or the nucleic acid molecule encoding an OSP, preferably at least 100 nucleotides, more preferably at least 150 or 200 nucleotides, even more preferably at least 250 or 300 nucleotides, still more preferably at least 400 or 500 nucleotides. \\n     The substantially similar nucleic acid molecule may be a naturally occurring one that is derived from another species, especially one derived from another primate, wherein the similar nucleic acid molecule encodes an amino acid sequence that exhibits significant sequence identity to that of SEQ ID NO: 129-295 or demonstrates significant sequence identity to the nucleotide sequence of SEQ ID NO: 1-128. The similar nucleic acid molecule may also be a naturally occurring nucleic acid molecule from a human, when the OSNA is a member of a gene family. The similar nucleic acid molecule may also be a naturally occurring nucleic acid molecule derived from a non-primate, mammalian species, including without limitation, domesticated species, e.g., dog, cat, mouse, rat, rabbit, hamster, cow, horse and pig; and wild animals, e.g., monkey, fox, lions, tigers, bears, giraffes, zebras, etc. The substantially similar nucleic acid molecule may also be a naturally occurring nucleic acid molecule derived from a non-mammalian species, such as birds or reptiles. The naturally occurring substantially similar nucleic acid molecule may be isolated directly from humans or other species. In another embodiment, the substantially similar nucleic acid molecule may be one that is experimentally produced by random mutation of a nucleic acid molecule. In another embodiment, the substantially similar nucleic acid molecule may be one that is experimentally produced by directed mutation of an OSNA. In a preferred embodiment, the substantially similar nucleic acid molecule is an OSNA. \\n     The nucleic acid molecules of the present invention are also inclusive of allelic variants of an OSNA or a nucleic acid encoding an OSP. For example, single nucleotide polymorphisms (SNPs) occur frequently in eukaryotic genomes and the sequence determined from one individual of a species may differ from other allelic forms present within the population. More than 1.4 million SNPs have already been identified in the human genome, International Human Genome Sequencing Consortium,  Nature  409: 860-921 (2001)—Variants with small deletions and insertions of more than a single nucleotide are also found in the general population, and often do not alter the function of the protein. In addition, amino acid substitutions occur frequently among natural allelic variants, and often do not substantially change protein function. \\n     In a preferred embodiment, the allelic variant is a variant of a gene, wherein the gene is transcribed into a mRNA that encodes an OSP. In a more preferred embodiment, the gene is transcribed into a mRNA that encodes an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. In another preferred embodiment, the allelic variant is a variant of a gene, wherein the gene is transcribed into a mRNA that is an OSNA. In a more preferred embodiment, the gene is transcribed into a mRNA that comprises the nucleic acid sequence of SEQ ID NO: 1-128. Also preferred is that the allelic variant be a naturally occurring allelic variant in the species of interest, particularly human. \\n     Nucleic acid molecules of the present invention are also inclusive of nucleic acid sequences comprising a part of a nucleic acid sequence of the instant invention. The part may or may not encode a polypeptide, and may or may not encode a polypeptide that is an OSP. In a preferred embodiment, the part encodes an OSP. In one embodiment, the nucleic acid molecule comprises a part of an OSNA. In another embodiment, the nucleic acid molecule comprises a part of a nucleic acid molecule that hybridizes or exhibits substantial sequence similarity to an OSNA. In another embodiment, the nucleic acid molecule comprises a part of a nucleic acid molecule that is an allelic variant of an OSNA. In yet another embodiment, the nucleic acid molecule comprises a part of a nucleic acid molecule that encodes an OSP. A part comprises at least 10 nucleotides, more preferably at least 15, 17, 18, 20, 25, 30, 35, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400 or 500 nucleotides. The maximum size of a nucleic acid part is one nucleotide shorter than the sequence of the nucleic acid molecule encoding the full-length protein. \\n     Nucleic acid molecules of the present invention are also inclusive of nucleic acid sequences that encode fusion proteins, homologous proteins, polypeptide fragments, muteins and polypeptide analogs, as described infra. \\n     Nucleic acid molecules of the present invention are also inclusive of nucleic acid sequences containing modifications of the native nucleic acid molecule. Examples of such modifications include, but are not limited to, normative internucleoside bonds, post-synthetic modifications or altered nucleotide analogues. One having ordinary skill in the art would recognize that the type of modification that may be made will depend upon the intended use of the nucleic acid molecule. For instance, when the nucleic acid molecule is used as a hybridization probe, the range of such modifications will be limited to those that permit sequence-discriminating base pairing of the resulting nucleic acid. When used to direct expression of RNA or protein in vitro or in vivo, the range of such modifications will be limited to those that permit the nucleic acid to function properly as a polymerization substrate. When the isolated nucleic acid is used as a therapeutic agent, the modifications will be limited to those that do not confer toxicity upon the isolated nucleic acid. \\n     Accordingly, in one embodiment, a nucleic acid molecule may include nucleotide analogues that incorporate labels that are directly detectable, such as radiolabels or fluorophores, or nucleotide analogues that incorporate labels that can be visualized in a subsequent reaction, such as biotin or various haptens. The labeled nucleic acid molecules are particularly useful as hybridization probes. \\n     Common radiolabeled analogues include those labeled with  33 P,  32 P, and  35 S, such as α- 32 P-dATP, α- 32 P-dCTP, α- 32 P-dGTP, α- 32 P-dTTP, α- 32 P-3′dATP, α- 32 P-ATP, α- 32 P-CTP, α- 32 P-GTP, α- 32 P-UTP, α- 35 S-dATP, γ- 35 S-GTP, γ- 33 P-dATP, and the like. \\n     Commercially available fluorescent nucleotide analogues readily incorporated into the nucleic acids of the present invention include Cy3-dCTP, Cy3-dUTP, Cy5-dCTP, Cy3-dUTP (Amersham Biosciences, Piscataway, N.J., USA), fluorescein-12-dUTP, tetramethylrhodamine-6-dUTP, Texas Red®-5-dUTP, Cascade Blue®-7-dUTP, BODIPY® FL-14-dUTP, BODIPY® TMR-14-dUTP, BODIPY® TR-14-dUTP, Rhodamine Green™-5-dUTP, Oregon Green® 488-5-dUTP, Texas Red®-12-dUTP, BODIPY® 630/650-14-dUTP, BODIPY® 650/665-14-dUTP, Alexa Fluor® 488-5-dUTP, Alexa Fluor® 532-5-dUTP, Alexa Fluor® 568-5-dUTP, Alexa Fluor® 594-5-dUTP, Alexa Fluor® 546-14-dUTP, fluorescein-12-UTP, tetramethylrhodamine-6-UTP, Texas Red®-5-UTP, Cascade Blue®-7-UTP, BODIPY® FL-14-UTP, BODIPY® TMR-14-UTP, BODIPY® TR-14-UTP, Rhodamine Green™-5-UTP, Alexa Fluor® 488-5-UTP, Alexa Fluor® 546-14-UTP (Molecular Probes, Inc. Eugene, Oreg., USA). One may also custom synthesize nucleotides having other fluorophores. See Henegariu et al.,  Nature Biotechnol.  18: 345-348 (2000). \\n     Haptens that are commonly conjugated to nucleotides for subsequent labeling include biotin (biotin-11-dUTP, Molecular Probes, Inc., Eugene, Oreg., USA; biotin-21-UTP, biotin-21-dUTP, Clontech Laboratories, Inc., Palo Alto, Calif., USA), digoxigenin (DIG-11-dUTP, alkali labile, DIG-11-UTP, Roche Diagnostics Corp., Indianapolis, Ind., USA), and dinitrophenyl (dinitrophenyl-11-dUTP, Molecular Probes, Inc., Eugene, Oreg., USA). \\n     Nucleic acid molecules of the present invention can be labeled by incorporation of labeled nucleotide analogues into the nucleic acid. Such analogues can be incorporated by enzymatic polymerization, such as by nick translation, random priming, polymerase chain reaction (PCR), terminal transferase tailing, and end-filling of overhangs, for DNA molecules, and in vitro transcription driven, e.g., from phage promoters, such as T7, T3, and SP6, for RNA molecules. Commercial kits are readily available for each such labeling approach. Analogues can also be incorporated during automated solid phase chemical synthesis. Labels can also be incorporated after nucleic acid synthesis, with the 5′ phosphate and 3′ hydroxyl providing convenient sites for post-synthetic covalent attachment of detectable labels. \\n     Other post-synthetic approaches also permit internal labeling of nucleic acids. For example, fluorophores can be attached using a cisplatin reagent that reacts with the N7 of guanine residues (and, to a lesser extent, adenine bases) in DNA, RNA, and Peptide Nucleic Acids (PNA) to provide a stable coordination complex between the nucleic acid and fluorophore label (Universal Linkage System) (available from Molecular Probes, Inc., Eugene, Oreg., USA and Amersham Pharmacia Biotech, Piscataway, N.J., USA); see Alers et al.,  Genes, Chromosomes  &amp;  Cancer  25: 301-305 (1999); Jelsma et al.,  J. NIH Res.  5: 82 (1994); Van Belkum et al.,  BioTechniques  16: 148-153 (1994). Alternatively, nucleic acids can be labeled using a disulfide-containing linker (FastTag™ Reagent, Vector Laboratories, Inc., Burlingame, Calif., USA) that is photo- or thermally coupled to the target nucleic acid using aryl azide chemistry; after reduction, a free thiol is available for coupling to a hapten, fluorophore, sugar, affinity ligand, or other marker. \\n     One or more independent or interacting labels can be incorporated into the nucleic acid molecules of the present invention. For example, both a fluorophore and a moiety that in proximity thereto acts to quench fluorescence can be included to report specific hybridization through release of fluorescence quenching or to report exonucleotidic excision. See, e.g., Tyagi et al.,  Nature Biotechnol.  14: 303-308 (1996); Tyagi et al.,  Nature Biotechnol.  16: 49-53 (1998); Sokol et al.,  Proc. Natl. Acad. Sci. USA  95: 11538-11543 (1998); Kostrikis et al.,  Science  279: 1228-1229 (1998); Marras et al.,  Genet. Anal.  14: 151-156 (1999); Holland et al.,  Proc. Natl. Acad. Sci. USA  88: 7276-7280 (1991); Heid et al.,  Genome Res.  6(10): 986-94 (1996); Kuimelis et al.,  Nucleic Acids Symp. Ser . (37): 255-6 (1997); and U.S. Pat. Nos. 5,846,726, 5,925,517, 5,925,517, 5,723,591 and 5,538,848, the disclosures of which are incorporated herein by reference in their entireties. \\n     Nucleic acid molecules of the present invention may also be modified by altering one or more native phosphodiester internucleoside bonds to more nuclease-resistant, internucleoside bonds. See Hartmann et al. (eds.),  Manual of Antisense Methodology: Perspectives in Antisense Science , Kluwer Law International (1999); Stein et al. (eds.),  Applied Antisense Oligonucleotide Technology , Wiley-Liss (1998); Chadwick et al. (eds.),  Oligonucleotides as Therapeutic Agents—Symposium No.  209, John Wiley &amp; Son Ltd (1997). Such altered internucleoside bonds are often desired for techniques or for targeted gene correction, Gamper et al.,  Nucl. Acids Res.  28(21): 4332-4339 (2000). For double-stranded RNA inhibition which may utilize either natural ds RNA or ds RNA modified in its, sugar, phosphate or base, see Hannon,  Nature  418(11): 244-251 (2002); Fire et al. in WO 99/32619; Tuschl et al. in US2002/0086356; Kruetzer et al. in WO 00/44895, the disclosures of which are incorporated herein by reference in their entirety. For circular antisense, see Kool in U.S. Pat. No. 5,426,180, the disclosure of which is incorporated herein by reference in its entirety. \\n     Modified oligonucleotide backbones include, without limitation, phosphorothioates, chiral phosphorothioates, phosphorodithioates, phosphotriesters, aminoalkylphosphotriesters, methyl and other alkyl phosphonates including 3′-alkylene phosphonates and chiral phosphonates, phosphinates, phosphoramidates including 3′-amino phosphoramidate and aminoalkylphosphoramidates, thionophosphoramidates, thionoalkylphosphonates, thionoalkylphosphotriesters, and boranophosphates having normal 3′-5′ linkages, 2′-5′ linked analogs of these, and those having inverted polarity wherein the adjacent pairs of nucleoside units are linked 3′-5′ to 5′-3′ or 2′-5′ to 5′-2′. Representative U.S. patents that teach the preparation of the above phosphorus-containing linkages include, but are not limited to, U.S. Pat. Nos. 3,687,808; 4,469,863; 4,476,301; 5,023,243; 5,177,196; 5,188,897; 5,264,423; 5,276,019; 5,278,302; 5,286,717; 5,321,131; 5,399,676; 5,405,939; 5,453,496; 5,455,233; 5,466,677; 5,476,925; 5,519,126; 5,536,821; 5,541,306; 5,550,111; 5,563,253; 5,571,799; 5,587,361; and 5,625,050, the disclosures of which are incorporated herein by reference in their entireties. In a preferred embodiment, the modified internucleoside linkages may be used for antisense techniques. \\n     Other modified oligonucleotide backbones do not include a phosphorus atom, but have backbones that are formed by short chain alkyl or cycloalkyl internucleoside linkages, mixed heteroatom and alkyl or cycloalkyl internucleoside linkages, or one or more short chain heteroatomic or heterocyclic internucleoside linkages. These include those having morpholino linkages (formed in part from the sugar portion of a nucleoside); siloxane backbones; sulfide, sulfoxide and sulfone backbones; formacetyl and thioformacetyl backbones; methylene formacetyl and thioformacetyl backbones; alkene containing backbones; sulfamate backbones; methyleneimino and methylenehydrazino backbones; sulfonate and sulfonamide backbones; amide backbones; and others having mixed N, O, S and CH 2  component parts. Representative U.S. patents that teach the preparation of the above backbones include, but are not limited to, U.S. Pat. Nos. 5,034,506; 5,166,315; 5,185,444; 5,214,134; 5,216,141; 5,235,033; 5,264,562; 5,264,564; 5,405,938; 5,434,257; 5,466,677; 5,470,967; 5,489,677; 5,541,307; 5,561,225; 5,596,086; 5,602,240; 5,610,289; 5,602,240; 5,608,046; 5,610,289; 5,618,704; 5,623,070; 5,663,312; 5,633,360; 5,677,437 and 5,677,439; the disclosures of which are incorporated herein by reference in their entireties. \\n     In other preferred nucleic acid molecules, both the sugar and the internucleoside linkage are replaced with novel groups, such as peptide nucleic acids (PNA). In PNA compounds, the phosphodiester backbone of the nucleic acid is replaced with an amide-containing backbone, in particular by repeating N-(2-aminoethyl) glycine units linked by amide bonds. Nucleobases are bound directly or indirectly to aza nitrogen atoms of the amide portion of the backbone, typically by methylene carbonyl linkages. PNA can be synthesized using a modified peptide synthesis protocol. PNA oligomers can be synthesized by both Fmoc and tBoc methods. Representative U.S. patents that teach the preparation of PNA compounds include, but are not limited to, U.S. Pat. Nos. 5,539,082; 5,714,331; and 5,719,262, each of which is herein incorporated by reference in its entirety. Automated PNA synthesis is readily achievable on commercial synthesizers (see, e.g., “PNA User&#39;s Guide,” Rev. 2, February 1998, Perseptive Biosystems Part No. 60138, Applied Biosystems, Inc., Foster City, Calif.). PNA molecules are advantageous for a number of reasons. First, because the PNA backbone is uncharged, PNA/DNA and PNA/RNA duplexes have a higher thermal stability than is found in DNA/DNA and DNA/RNA duplexes. The Tm of a PNA/DNA or PNA/RNA duplex is generally 1° C. higher per base pair than the Tm of the corresponding DNA/DNA or DNA/RNA duplex (in 100 mM NaCl). Second, PNA molecules can also form stable PNA/DNA complexes at low ionic strength, under conditions in which DNA/DNA duplex formation does not occur. Third, PNA also demonstrates greater specificity in binding to complementary DNA because a PNA/DNA mismatch is more destabilizing than DNA/DNA mismatch. A single mismatch in mixed a PNA/DNA 15-mer lowers the Tm by 8-20° C. (15° C. on average). In the corresponding DNA/DNA duplexes, a single mismatch lowers the Tm by 4-16° C. (11° C. on average). Because PNA probes can be significantly shorter than DNA probes, their specificity is greater. Fourth, PNA oligomers are resistant to degradation by enzymes, and the lifetime of these compounds is extended both in vivo and in vitro because nucleases and proteases do not recognize the PNA polyamide backbone with nucleobase sidechains. See, e.g., Ray et al.,  FASEB J.  14(9): 1041-60 (2000); Nielsen et al.,  Pharmacol Toxicol.  86(1): 3-7 (2000); Larsen et al.,  Biochim Biophys Acta.  1489(1): 159-66 (1999); Nielsen,  Curr. Opin. Struct. Biol.  9(3): 353-7 (1999), and Nielsen,  Curr. Opin. Biotechnol.  10(1): 71-5 (1999). \\n     Nucleic acid molecules may be modified compared to their native structure throughout the length of the nucleic acid molecule or can be localized to discrete portions thereof. As an example of the latter, chimeric nucleic acids can be synthesized that have discrete DNA and RNA domains and that can be used for targeted gene repair and modified PCR reactions, as further described in, Misra et al.,  Biochem.  37: 1917-1925 (1998); and Finn et al.,  Nucl. Acids Res.  24: 3357-3363 (1996), and U.S. Pat. Nos. 5,760,012 and 5,731,181, the disclosures of which are incorporated herein by reference in their entireties. \\n     Unless otherwise specified, nucleic acid molecules of the present invention can include any topological conformation appropriate to the desired use; the term thus explicitly comprehends, among others, single-stranded, double-stranded, triplexed, quadruplexed, partially double-stranded, partially-triplexed, partially-quadruplexed, branched, hairpinned, circular, and padlocked conformations. Padlocked conformations and their utilities are further described in Banér et al.,  Curr. Opin. Biotechnol.  12: 11-15 (2001); Escude et al.,  Proc. Natl. Acad. Sci. USA  14: 96(19):10603-7 (1999); and Nilsson et al.,  Science  265(5181): 2085-8 (1994). Triplexed and quadruplexed conformations, and their utilities, are reviewed in Praseuth et al.,  Biochim. Biophys. Acta.  1489(1): 181-206 (1999); Fox,  Curr. Med. Chem.  7(1): 17-37 (2000); Kochetkova et al.,  Methods Mol. Biol.  130: 189-201 (2000); Chan et al.,  J. Mol. Med.  75(4): 267-82 (1997); Rowley et al.,  Mol Med  5(10): 693-700 (1999); Kool,  Annu Rev Biophys Biomol Struct.  25: 1-28 (1996). \\n     SNP Polymorphisms \\n     Commonly, sequence differences between individuals involve differences in single nucleotide positions. SNPs may account for 90% of human DNA polymorphism. Collins et al., 8  Genome Res.  1229-31 (1998). SNPs include single base pair positions in genomic DNA at which different sequence alternatives (alleles) exist in a population. In addition, the least frequent allele generally must occur at a frequency of 1% or greater. DNA sequence variants with a reasonably high population frequency are observed approximately every 1,000 nucleotide across the genome, with estimates as high as 1 SNP per 350 base pairs. Wang et al., 280  Science  1077-82 (1998); Harding et al., 60 Am. J. Human Genet. 772-89 (1997); Taillon-Miller et al., 8  Genome Res.  748-54 (1998); Cargill et al., 22  Nat. Genet.  231-38 (1999); and Semple et al., 16  Bioinform. Disc. Note  735-38 (2000). The frequency of SNPs varies with the type and location of the change. In base substitutions, two-thirds of the substitutions involve the C-T and G-A type. This variation in frequency can be related to 5-methylcytosine deamination reactions that occur frequently, particularly at CpG dinucleotides. Regarding location, SNPs occur at a much higher frequency in non-coding regions than in coding regions. Information on over one million variable sequences is already publicly available via the Internet and more such markers are available from commercial providers of genetic information. Kwok and Gu, 5  Med. Today  538-53 (1999). \\n     Several definitions of SNPs exist. See, e.g., Brooks, 235  Gene  177-86 (1999). As used herein, the term “single nucleotide polymorphism” or “SNP” includes all single base variants, thus including nucleotide insertions and deletions in addition to single nucleotide substitutions. There are two types of nucleotide substitutions. A transition is the replacement of one purine by another purine or one pyrimidine by another pyrimidine. A transversion is the replacement of a purine for a pyrimidine, or vice versa. \\n     Numerous methods exist for detecting SNPs within a nucleotide sequence. A review of many of these methods can be found in Landegren et al., 8  Genome Res.  769-76 (1998). For example, a SNP in a genomic sample can be detected by preparing a Reduced Complexity Genome (RCG) from the genomic sample, then analyzing the RCG for the presence or absence of a SNP. See, e.g., WO 00/18960 which is herein incorporated by reference in its entirety. Multiple SNPs in a population of target polynucleotides in parallel can be detected using, for example, the methods of WO 00/50869 which is herein incorporated by reference in its entirety. Other SNP detection methods include the methods of U.S. Pat. Nos. 6,297,018 and 6,322,980 which are herein incorporated by reference in their entirety. Furthermore, SNPs can be detected by restriction fragment length polymorphism (RFLP) analysis. See, e.g., U.S. Pat. Nos. 5,324,631; 5,645,995 which are herein incorporated by reference in their entirety. RFLP analysis of SNPs, however, is limited to cases where the SNP either creates or destroys a restriction enzyme cleavage site. SNPs can also be detected by direct sequencing of the nucleotide sequence of interest. In addition, numerous assays based on hybridization have also been developed to detect SNPs and mismatch distinction by polymerases and ligases. Several web sites provide information about SNPs including Ensembl on the World Wide Web at ensemble.org, Sanger Institute on the World Wide Web at sanger.ac.uk/genetics/exon/, National Center for Biotechnology Information (NCBI) on the World Wide Web at ncbi.nlm.nih.gov/SNP/, The SNP Consortium Ltd. on the World Wide Web at snp.cshl.org. The chromosomal locations for the compositions disclosed herein are provided below. In addition, one of ordinary skill in the art could use a BLAST against the genome or any of the databases cited above to find the chromosomal location. Another a preferred method to find the genomic coordinates and associated SNPs would be to use the BLAT tool (genome.ucsc.edu, Kent et al. 2001, The Human Genome Browser at UCSC, Genome Research 996-1006 or Kent 2002 BLAT—The BLAST-Like Alignment Tool Genome Reseach, 1-9). All web sites above were accessed Dec. 3, 2003. \\n     RNA Interference \\n     RNA interference refers to the process of sequence-specific post transcriptional gene silencing in animals mediated by short interfering RNAs (siRNA). Fire et al., 1998 , Nature,  391, 806. The corresponding process in plants is commonly referred to as post transcriptional gene silencing or RNA silencing and is also referred to as quelling in fungi. The process of post transcriptional gene silencing is thought to be an evolutionarily conserved cellular defense mechanism used to prevent the expression of foreign genes which is commonly shared by diverse flora and phyla. Fire et al., 1999 , Trends Genet.,  15, 358. Such protection from foreign gene expression may have evolved in response to the production of double-stranded RNAs (dsRNA) derived from viral infection or the random integration of transposon elements into a host genome via a cellular response that specifically destroys homologous single-stranded RNA or viral genomic RNA. The presence of dsRNA in cells triggers the RNAi response though a mechanism that has yet to be fully characterized. This mechanism appears to be different from the interferon response that results from dsRNA mediated activation of protein kinase PKR and 2′,5′-oligoadenylate synthetase resulting in non-specific cleavage of mRNA by ribonuclease L. \\n     The presence of long dsRNAs in cells stimulates the activity of a ribonuclease III enzyme referred to as dicer. Dicer is involved in the processing of the dsRNA into short pieces of dsRNA known as short interfering RNAs (siRNA). Berstein et al., 2001 , Nature,  409, 363. Short interfering RNAs derived from dicer activity are typically about 21-23 nucleotides in length and comprise about 19 base pair duplexes. Dicer has also been implicated in the excision of 21 and 22 nucleotide small temporal RNAs (stRNA) from precursor RNA of conserved structure that are implicated in translational control. Hutvagner et al., 2001 , Science,  293, 834. The RNAi response also features an endonuclease complex containing a siRNA, commonly referred to as an RNA-induced silencing complex (RISC), which mediates cleavage of single-stranded RNA having sequence complementary to the antisense strand of the siRNA duplex. Cleavage of the target RNA takes place in the middle of the region complementary to the antisense strand of the siRNA duplex. Elbashir et al., 2001 , Genes Dev.,  15, 188. \\n     Short interfering RNA mediated RNAi has been studied in a variety of systems. Fire et al., 1998 , Nature,  391, 806, were the first to observe RNAi in  C. Elegans . Wianny and Goetz, 1999 , Nature Cell Biol.,  2, 70, describe RNAi mediated by dsRNA in mouse embryos. Hammond et al., 2000 , Nature,  404, 293, describe RNAi in  Drosophila  cells transfected with dsRNA. Elbashir et al., 2001 , Nature,  411, 494, describe RNAi induced by introduction of duplexes of synthetic 21-nucleotide RNAs in cultured mammalian cells including human embryonic kidney and HeLa cells. Recent work in  Drosophila  embryonic lysates (Elbashir et al., 2001 , EMBO J.,  20, 6877) has revealed certain requirements for siRNA length, structure, chemical composition, and sequence that are essential to mediate efficient RNAi activity. These studies have shown that 21 nucleotide siRNA duplexes are most active when containing two nucleotide 3′-overhangs. Furthermore, complete substitution of one or both siRNA strands with 2′-deoxy (2′-H) or 2′-O-methyl nucleotides abolishes RNAi activity, whereas substitution of the 3′-terminal siRNA overhang nucleotides with deoxy nucleotides (2′-H) was shown to be tolerated. Single mismatch sequences in the center of the siRNA duplex were also shown to abolish RNAi activity. In addition, these studies also indicate that the position of the cleavage site in the target RNA is defined by the 5′-end of the siRNA guide sequence rather than the 3′-end. Elbashir et al., 2001 , EMBO J.,  20, 6877. Other studies have indicated that a 5′-phosphate on the target-complementary strand of a siRNA duplex is required for siRNA activity and that ATP is utilized to maintain the 5′-phosphate moiety on the siRNA. Nykanen et al., 2001 , Cell,  107, 309. \\n     Studies have shown that replacing the 3′-overhanging segments of a 21-mer siRNA duplex having 2 nucleotide 3′ overhangs with deoxyribonucleotides does not have an adverse effect on RNAi activity. Replacing up to 4 nucleotides on each end of the siRNA with deoxyribonucleotides has been reported to be well tolerated whereas complete substitution with deoxyribonucleotides results in no RNAi activity. Elbashir et al., 2001 , EMBO J.,  20, 6877. In addition, Elbashir et al., supra, also report that substitution of siRNA with 2′-O-methyl nucleotides completely abolishes RNAi activity. Li et al., WO 00/44914, and Beach et al., WO 01/68836 both suggest that siRNA “may include modifications to either the phosphate-sugar back bone or the nucleoside to include at least one of a nitrogen or sulfur heteroatom”, however neither application teaches to what extent these modifications are tolerated in siRNA molecules nor provides any examples of such modified siRNA. Kreutzer and Limmer, Canadian Patent Application No. 2,359,180, also describe certain chemical modifications for use in dsRNA constructs in order to counteract activation of double-stranded RNA-dependent protein kinase PKR, specifically 2′-amino or 2′-O-methyl nucleotides, and nucleotides containing a 2′-O or 4′-C methylene bridge. However, Kreutzer and Limmer similarly fail to show to what extent these modifications are tolerated in siRNA molecules nor do they provide any examples of such modified siRNA. \\n     Parrish et al., 2000 , Molecular Cell,  6, 1977-1087, tested certain chemical modifications targeting the unc-22 gene in  C. elegans  using long (&gt;25 nt) siRNA transcripts. The authors describe the introduction of thiophosphate residues into these siRNA transcripts by incorporating thiophosphate nucleotide analogs with T7 and T3 RNA polymerase and observed that “RNAs with two [phosphorothioate] modified bases also had substantial decreases in effectiveness as RNAi triggers; [phosphorothioate] modification of more than two residues greatly destabilized the RNAs in vitro and we were not able to assay interference activities.” Parrish et al. at 1081. The authors also tested certain modifications at the 2′-position of the nucleotide sugar in the long siRNA transcripts and observed that substituting deoxynucleotides for ribonucleotides “produced a substantial decrease in interference activity”, especially in the case of Uridine to Thymidine and/or Cytidine to deoxy-Cytidine substitutions. Parrish et al. In addition, the authors tested certain base modifications, including substituting 4-thiouracil, 5-bromouracil, 5-iodouracil, 3-(aminoallyl)uracil for uracil, and inosine for guanosine in sense and antisense strands of the siRNA, and found that whereas 4-thiouracil and 5-bromouracil were all well tolerated, inosine “produced a substantial decrease in interference activity” when incorporated in either strand. Incorporation of 5-iodouracil and 3-(aminoallyl)uracil in the antisense strand resulted in substantial decrease in RNAi activity as well. \\n     Beach et al., WO 01/68836, describes specific methods for attenuating gene expression using endogenously derived dsRNA. Tuschl et al., WO 01/75164, describes a  Drosophila  in vitro RNAi system and the use of specific siRNA molecules for certain functional genomic and certain therapeutic applications; although Tuschl, 2001 , Chem. Biochem.,  2, 239-245, doubts that RNAi can be used to cure genetic diseases or viral infection due “to the danger of activating interferon response”. Li et al., WO 00/44914, describes the use of specific dsRNAs for use in attenuating the expression of certain target genes. Zernicka-Goetz et al., WO 01/36646, describes certain methods for inhibiting the expression of particular genes in mammalian cells using certain dsRNA molecules. Fire et al., WO 99/32619, U.S. Pat. No. 6,506,559, the contents of which are hereby incorporated by reference in their entirety, describes particular methods for introducing certain dsRNA molecules into cells for use in inhibiting gene expression. Plaetinck et al., WO 00/01846, describes certain methods for identifying specific genes responsible for conferring a particular phenotype in a cell using specific dsRNA molecules. Mello et al., WO 01/29058, describes the identification of specific genes involved in dsRNA mediated RNAi. Deschamps Depaillette et al., International PCT Publication No. WO 99/07409, describes specific compositions consisting of particular dsRNA molecules combined with certain anti-viral agents. Driscoll et al., International PCT Publication No. WO 01/49844, describes specific DNA constructs for use in facilitating gene silencing in targeted organisms. Parrish et al., 2000, Molecular Cell, 6, 1977-1087, describes specific chemically modified siRNA constructs targeting the unc-22 gene of  C. elegans . Tuschl et al., International PCT Publication No. WO 02/44321, describe certain synthetic siRNA constructs. \\n     Methods for Using Nucleic Acid Molecules as Probes and Primers \\n     The isolated nucleic acid molecules of the present invention can be used as hybridization probes to detect, characterize, and quantify hybridizing nucleic acids in, and isolate hybridizing nucleic acids from, both genomic and transcript-derived nucleic acid samples. When free in solution, such probes are typically, but not invariably, detectably labeled; bound to a substrate, as in a microarray, such probes are typically, but not invariably unlabeled. \\n     In one embodiment, the isolated nucleic acid molecules of the present invention can be used as probes to detect and characterize gross alterations in the gene of an OSNA, such as deletions, insertions, translocations, and duplications of the OSNA genomic locus through fluorescence in situ hybridization (FISH) to chromosome spreads. See, e.g., Andreeff et al. (eds.),  Introduction to Fluorescence In Situ Hybridization: Principles and Clinical Applications , John Wiley &amp; Sons (1999). The isolated nucleic acid molecules of the present invention can be used as probes to assess smaller genomic alterations using, e.g., Southern blot detection of restriction fragment length polymorphisms. The isolated nucleic acid molecules of the present invention can be used as probes to isolate genomic clones that include a nucleic acid molecule of the present invention, which thereafter can be restriction mapped and sequenced to identify deletions, insertions, translocations, and substitutions (single nucleotide polymorphisms, SNPs) at the sequence level. Alternatively, detection techniques such as molecular beacons may be used, see Kostrikis et al.  Science  279:1228-1229 (1998). \\n     The isolated nucleic acid molecules of the present invention can also be used as probes to detect, characterize, and quantify OSNA in, and isolate OSNA from, transcript-derived nucleic acid samples. In one embodiment, the isolated nucleic acid molecules of the present invention can be used as hybridization probes to detect, characterize by length, and quantify mRNA by Northern blot of total or poly-A + -selected RNA samples. In another embodiment, the isolated nucleic acid molecules of the present invention can be used as hybridization probes to detect, characterize by location, and quantify mRNA by in situ hybridization to tissue sections. See, e.g., Schwarchzacher et al.,  In Situ Hybridization , Springer-Verlag New York (2000). In another preferred embodiment, the isolated nucleic acid molecules of the present invention can be used as hybridization probes to measure the representation of clones in a cDNA library or to isolate hybridizing nucleic acid molecules acids from cDNA libraries, permitting sequence level characterization of mRNAs that hybridize to OSNAs, including, without limitations, identification of deletions, insertions, substitutions, truncations, alternatively spliced forms and single nucleotide polymorphisms. In yet another preferred embodiment, the nucleic acid molecules of the instant invention may be used in microarrays. \\n     All of the aforementioned probe techniques are well within the skill in the art, and are described at greater length in standard texts such as Sambrook (2001), supra; Ausubel (1999), supra; and Walker et al. (eds.),  The Nucleic Acids Protocols Handbook , Humana Press (2000). \\n     In another embodiment, a nucleic acid molecule of the invention may be used as a probe or primer to identify and/or amplify a second nucleic acid molecule that selectively hybridizes to the nucleic acid molecule of the invention. In this embodiment, it is preferred that the probe or primer be derived from a nucleic acid molecule encoding an OSP. More preferably, the probe or primer is derived from a nucleic acid molecule encoding a polypeptide having an amino acid sequence of SEQ ID NO: 129-295. Also preferred are probes or primers derived from an OSNA. More preferred are probes or primers derived from a nucleic acid molecule having a nucleotide sequence of SEQ ID NO: 1-128. \\n     In general, a probe or primer is at least 10 nucleotides in length, more preferably at least 12, more preferably at least 14 and even more preferably at least 16 or 17 nucleotides in length. In an even more preferred embodiment, the probe or primer is at least 18 nucleotides in length, even more preferably at least 20 nucleotides and even more preferably at least 22 nucleotides in length. Primers and probes may also be longer in length. For instance, a probe or primer may be 25 nucleotides in length, or may be 30, 40 or 50 nucleotides in length. Methods of performing nucleic acid hybridization using oligonucleotide probes are well known in the art. See, e.g., Sambrook et al., 1989, supra, Chapter 11 and pp. 11.31-11.32 and 11.40-11.44, which describes radiolabeling of short probes, and pp. 11.45-11.53, which describe hybridization conditions for oligonucleotide probes, including specific conditions for probe hybridization (pp. 11.50-11.51). \\n     Methods of performing primer-directed amplification are also well known in the art. Methods for performing the polymerase chain reaction (PCR) are compiled, inter alia, in McPherson,  PCR Basics: From Background to Bench , Springer Verlag (2000); Innis et al. (eds.),  PCR Applications: Protocols for Functional Genomics , Academic Press (1999); Gelfand et al. (eds.),  PCR Strategies , Academic Press (1998); Newton et al.,  PCR , Springer-Verlag New York (1997); Burke (ed.),  PCR: Essential Techniques , John Wiley &amp; Son Ltd (1996); White (ed.),  PCR Cloning Protocols: From Molecular Cloning to Genetic Engineering , Vol. 67, Humana Press (1996); and McPherson et al. (eds.),  PCR  2 : A Practical Approach , Oxford University Press, Inc. (1995). Methods for performing RT-PCR are collected, e.g., in Siebert et al. (eds.),  Gene Cloning and Analysis by RT - PCR, Eaton Publishing Company/Bio Techniques Books Division,  1998; and Siebert (ed.),  PCR Technique: RT - PCR , Eaton Publishing Company/BioTechniques Books (1995). \\n     PCR and hybridization methods may be used to identify and/or isolate nucleic acid molecules of the present invention including allelic variants, homologous nucleic acid molecules and fragments. PCR and hybridization methods may also be used to identify, amplify and/or isolate nucleic acid molecules of the present invention that encode homologous proteins, analogs, fusion proteins or muteins of the invention. Nucleic acid primers as described herein can be used to prime amplification of nucleic acid molecules of the invention, using transcript-derived or genomic DNA as the template. \\n     These nucleic acid primers can also be used, for example, to prime single base extension (SBE) for SNP detection (See, e.g., U.S. Pat. No. 6,004,744, the disclosure of which is incorporated herein by reference in its entirety). \\n     Isothermal amplification approaches, such as rolling circle amplification, are also now well-described. See, e.g., Schweitzer et al.,  Curr. Opin. Biotechnol.  12(1): 21-7 (2001); International Patent publications WO 97/19193 and WO 00/15779, and U.S. Pat. Nos. 5,854,033 and 5,714,320, the disclosures of which are incorporated herein by reference in their entireties. Rolling circle amplification can be combined with other techniques to facilitate SNP detection. See, e.g., Lizardi et al.,  Nature Genet.  19(3): 225-32 (1998). \\n     Nucleic acid molecules of the present invention may be bound to a substrate either covalently or noncovalently. The substrate can be porous or solid, planar or non-planar, unitary or distributed. The bound nucleic acid molecules may be used as hybridization probes, and may be labeled or unlabeled. In a preferred embodiment, the bound nucleic acid molecules are unlabeled. \\n     In one embodiment, the nucleic acid molecule of the present invention is bound to a porous substrate, e.g., a membrane, typically comprising nitrocellulose, nylon, or positively charged derivatized nylon. The nucleic acid molecule of the present invention can be used to detect a hybridizing nucleic acid molecule that is present within a labeled nucleic acid sample, e.g., a sample of transcript-derived nucleic acids. In another embodiment, the nucleic acid molecule is bound to a solid substrate, including, without limitation, glass, amorphous silicon, crystalline silicon or plastics. Examples of plastics include, without limitation, polymethylacrylic, polyethylene, polypropylene, polyacrylate, polymethylmethacrylate, polyvinylchloride, polytetrafluoroethylene, polystyrene, polycarbonate, polyacetal, polysulfone, celluloseacetate, cellulosenitrate, nitrocellulose, or mixtures thereof. The solid substrate may be any shape, including rectangular, disk-like and spherical. In a preferred embodiment, the solid substrate is a microscope slide or slide-shaped substrate. \\n     The nucleic acid molecule of the present invention can be attached covalently to a surface of the support substrate or applied to a derivatized surface in a chaotropic agent that facilitates denaturation and adherence by presumed noncovalent interactions, or some combination thereof. The nucleic acid molecule of the present invention can be bound to a substrate to which a plurality of other nucleic acids are concurrently bound, hybridization to each of the plurality of bound nucleic acids being separately detectable. At low density, e.g. on a porous membrane, these substrate-bound collections are typically denominated macroarrays; at higher density, typically on a solid support, such as glass, these substrate bound collections of plural nucleic acids are colloquially termed microarrays. As used herein, the term microarray includes arrays of all densities. It is, therefore, another aspect of the invention to provide microarrays that comprise one or more of the nucleic acid molecules of the present invention. \\n     In yet another embodiment, the invention is directed to single exon probes based on the OSNAs disclosed herein. \\n     Expression Vectors, Host Cells and Recombinant Methods of Producing Polypeptides \\n     Another aspect of the present invention provides vectors that comprise one or more of the isolated nucleic acid molecules of the present invention, and host cells in which such vectors have been introduced. \\n     The vectors can be used, inter alia, for propagating the nucleic acid molecules of the present invention in host cells (cloning vectors), for shuttling the nucleic acid molecules of the present invention between host cells derived from disparate organisms (shuttle vectors), for inserting the nucleic acid molecules of the present invention into host cell chromosomes (insertion vectors), for expressing sense or antisense RNA transcripts of the nucleic acid molecules of the present invention in vitro or within a host cell, and for expressing polypeptides encoded by the nucleic acid molecules of the present invention, alone or as fusion proteins with heterologous polypeptides (expression vectors). Vectors are by now well known in the art, and are described, inter alia, in Jones et al. (eds.),  Vectors: Cloning Applications: Essential Techniques  (Essential Techniques Series), John Wiley &amp; Son Ltd. (1998); Jones et al. (eds.),  Vectors: Expression Systems: Essential Techniques  (Essential Techniques Series), John Wiley &amp; Son Ltd. (1998); Gacesa et al.,  Vectors: Essential Data , John Wiley &amp; Sons Ltd. (1995); Cid-Arregui (eds.), Viral  Vectors: Basic Science and Gene Therapy , Eaton Publishing Co. (2000); Sambrook (2001), supra; Ausubel (1999), supra. Furthermore, a variety of vectors are available commercially. Use of existing vectors and modifications thereof are well within the skill in the art. Thus, only basic features need be described here. \\n     Nucleic acid sequences may be expressed by operatively linking them to an expression control sequence in an appropriate expression vector and employing that expression vector to transform an appropriate unicellular host. Expression control sequences are sequences that control the transcription, post-transcriptional events and translation of nucleic acid sequences. Such operative linking of a nucleic acid sequence of this invention to an expression control sequence, of course, includes, if not already part of the nucleic acid sequence, the provision of a translation initiation codon, ATG or GTG, in the correct reading frame upstream of the nucleic acid sequence. \\n     A wide variety of host/expression vector combinations may be employed in expressing the nucleic acid sequences of this invention. Useful expression vectors, for example, may consist of segments of chromosomal, non-chromosomal and synthetic nucleic acid sequences. \\n     In one embodiment, prokaryotic cells may be used with an appropriate vector. Prokaryotic host cells are often used for cloning and expression. In a preferred embodiment, prokaryotic host cells include  E. coli, Pseudomonas, Bacillus  and  Streptomyces . In a preferred embodiment, bacterial host cells are used to express the nucleic acid molecules of the instant invention. Useful expression vectors for bacterial hosts include bacterial plasmids, such as those from  E. coli, Bacillus  or  Streptomyces , including pBluescript, pGEX-2T, pUC vectors, col E1, pCR1, pBR322, pMB9 and their derivatives, wider host range plasmids, such as RP4, phage DNAs, e.g., the numerous derivatives of phage lambda, e.g., NM989, λGT10 and λGT11, and other phages, e.g., M13 and filamentous single-stranded phage DNA. Where  E. coli  is used as host, selectable markers are, analogously, chosen for selectivity in gram negative bacteria: e.g., typical markers confer resistance to antibiotics, such as ampicillin, tetracycline, chloramphenicol, kanamycin, streptomycin and zeocin; auxotrophic markers can also be used. \\n     In other embodiments, eukaryotic host cells, such as yeast, insect, mammalian or plant cells, may be used. Yeast cells, typically  S. cerevisiae , are useful for eukaryotic genetic studies, due to the ease of targeting genetic changes by homologous recombination and the ability to easily complement genetic defects using recombinantly expressed proteins. Yeast cells are useful for identifying interacting protein components, e.g. through use of a two-hybrid system. In a preferred embodiment, yeast cells are useful for protein expression. Vectors of the present invention for use in yeast will typically, but not invariably, contain an origin of replication suitable for use in yeast and a selectable marker that is functional in yeast. Yeast vectors include Yeast Integrating plasmids (e.g., YIp5) and Yeast Replicating plasmids (the YRp and YEp series plasmids), Yeast Centromere plasmids (the YCp series plasmids), Yeast Artificial Chromosomes (YACs) which are based on yeast linear plasmids, denoted YLp, pGPD-2, 2μ plasmids and derivatives thereof, and improved shuttle vectors such as those described in Gietz et al.,  Gene,  74: 527-34 (1988) (YIplac, YEplac and YCplac). Selectable markers in yeast vectors include a variety of auxotrophic markers, the most common of which are (in  Saccharomyces cerevisiae ) URA3, HIS3, LEU2, TRP1 and LYS2, which complement specific auxotrophic mutations, such as ura3-52, his3-D1, leu2-D1, trp1-D1 and lys2-201. \\n     Insect cells may be chosen for high efficiency protein expression. Where the host cells are from  Spodoptera frugiperda , e.g., Sf9 and Sf21 cell lines, and expresSF™ cells (Protein Sciences Corp., Meriden, Conn., USA), the vector replicative strategy is typically based upon the baculovirus life cycle. Typically, baculovirus transfer vectors are used to replace the wild-type AcMNPV polyhedrin gene with a heterologous gene of interest. Sequences that flank the polyhedrin gene in the wild-type genome are positioned 5′ and 3′ of the expression cassette on the transfer vectors. Following co-transfection with AcMNPV DNA, a homologous recombination event occurs between these sequences resulting in a recombinant virus carrying the gene of interest and the polyhedrin or p10 promoter. Selection can be based upon visual screening for lacZ fusion activity. \\n     The host cells may also be mammalian cells, which are particularly useful for expression of proteins intended as pharmaceutical agents, and for screening of potential agonists and antagonists of a protein or a physiological pathway. Mammalian vectors intended for autonomous extrachromosomal replication will typically include a viral origin, such as the SV40 origin (for replication in cell lines expressing the large T-antigen, such as COS1 and COS7 cells), the papillomavirus origin, or the EBV origin for long term episomal replication (for use, e.g., in 293-EBNA cells, which constitutively express the EBV EBNA-1 gene product and adenovirus E1A). Vectors intended for integration, and thus replication as part of the mammalian chromosome, can, but need not, include an origin of replication functional in mammalian cells, such as the SV40 origin. Vectors based upon viruses, such as adenovirus, adeno-associated virus, vaccinia virus, and various mammalian retroviruses, will typically replicate according to the viral replicative strategy. Selectable markers for use in mammalian cells include, but are not limited to, resistance to neomycin (G418), blasticidin, hygromycin and zeocin, and selection based upon the purine salvage pathway using HAT medium. \\n     Expression in mammalian cells can be achieved using a variety of plasmids, including pSV2, pBC12BI, and p91023, as well as lytic virus vectors (e.g., vaccinia virus, adeno virus, and baculovirus), episomal virus vectors (e.g., bovine papillomavirus), and retroviral vectors (e.g., murine retroviruses). Useful vectors for insect cells include baculoviral vectors and pVL 941. \\n     Plant cells can also be used for expression, with the vector replicon typically derived from a plant virus (e.g., cauliflower mosaic virus, CaMV; tobacco mosaic virus, TMV) and selectable markers chosen for suitability in plants. \\n     It is known that codon usage of different host cells may be different. For example, a plant cell and a human cell may exhibit a difference in codon preference for encoding a particular amino acid. As a result, human mRNA may not be efficiently translated in a plant, bacteria or insect host cell. Therefore, another embodiment of this invention is directed to codon optimization. The codons of the nucleic acid molecules of the invention may be modified to resemble, as much as possible, genes naturally contained within the host cell without altering the amino acid sequence encoded by the nucleic acid molecule. \\n     Any of a wide variety of expression control sequences may be used in these vectors to express the nucleic acid molecules of this invention. Such useful expression control sequences include the expression control sequences associated with structural genes of the foregoing expression vectors. Expression control sequences that control transcription include, e.g., promoters, enhancers and transcription termination sites. Expression control sequences in eukaryotic cells that control post-transcriptional events include splice donor and acceptor sites and sequences that modify the half-life of the transcribed RNA, e.g., sequences that direct poly(A) addition or binding sites for RNA-binding proteins. Expression control sequences that control translation include ribosome binding sites, sequences which direct targeted expression of the polypeptide to or within particular cellular compartments, and sequences in the 5′ and 3′ untranslated regions that modify the rate or efficiency of translation. \\n     Examples of useful expression control sequences for a prokaryote, e.g.,  E. coli , will include a promoter, often a phage promoter, such as phage lambda pL promoter, the trc promoter, a hybrid derived from the trp and lac promoters, the bacteriophage T7 promoter (in  E. coli  cells engineered to express the T7 polymerase), the TAC or TRC system, the major operator and promoter regions of phage lambda, the control regions of fd coat protein, and the araBAD operon. Prokaryotic expression vectors may further include transcription terminators, such as the aspA terminator, and elements that facilitate translation, such as a consensus ribosome binding site and translation termination codon, Schomer et al.,  Proc. Natl. Acad. Sci. USA  83: 8506-8510 (1986). \\n     Expression control sequences for yeast cells, typically  S. cerevisiae , will include a yeast promoter, such as the CYC1 promoter, the GAL1 promoter, the GAL10 promoter, ADH1 promoter, the promoters of the yeast  α -mating system, or the GPD promoter, and will typically have elements that facilitate transcription termination, such as the transcription termination signals from the CYC1 or ADH1 gene. \\n     Expression vectors useful for expressing proteins in mammalian cells will include a promoter active in mammalian cells. These promoters include, but are not limited to, those derived from mammalian viruses, such as the enhancer-promoter sequences from the immediate early gene of the human cytomegalovirus (CMV), the enhancer-promoter sequences from the Rous sarcoma virus long terminal repeat (RSV LTR), the enhancer-promoter from SV40 and the early and late promoters of adenovirus. Other expression control sequences include the promoter for 3-phosphoglycerate kinase or other glycolytic enzymes, the promoters of acid phosphatase. Other expression control sequences include those from the gene comprising the OSNA of interest. Often, expression is enhanced by incorporation of polyadenylation sites, such as the late SV40 polyadenylation site and the polyadenylation signal and transcription termination sequences from the bovine growth hormone (BGH) gene, and ribosome binding sites. Furthermore, vectors can include introns, such as intron II of rabbit β-globin gene and the SV40 splice elements. \\n     Preferred nucleic acid vectors also include a selectable or amplifiable marker gene and means for amplifying the copy number of the gene of interest. Such marker genes are well known in the art. Nucleic acid vectors may also comprise stabilizing sequences (e.g., ori- or ARS-like sequences and telomere-like sequences), or may alternatively be designed to favor directed or non-directed integration into the host cell genome. In a preferred embodiment, nucleic acid sequences of this invention are inserted in frame into an expression vector that allows a high level expression of an RNA which encodes a protein comprising the encoded nucleic acid sequence of interest. Nucleic acid cloning and sequencing methods are well known to those of skill in the art and are described in an assortment of laboratory manuals, including Sambrook (1989), supra, Sambrook (2000), supra; Ausubel (1992), supra; and Ausubel (1999), supra. Product information from manufacturers of biological, chemical and immunological reagents also provide useful information. \\n     Expression vectors may be either constitutive or inducible. Inducible vectors include either naturally inducible promoters, such as the trc promoter, which is regulated by the lac operon, and the pL promoter, which is regulated by tryptophan, the MMTV-LTR promoter, which is inducible by dexamethasone, or can contain synthetic promoters and/or additional elements that confer inducible control on adjacent promoters. Examples of inducible synthetic promoters are the hybrid Plac/ara-1 promoter and the PLtetO-1 promoter. The PLtetO-1 promoter takes advantage of the high expression levels from the PL promoter of phage lambda, but replaces the lambda repressor sites with two copies of operator 2 of the Tn10 tetracycline resistance operon, causing this promoter to be tightly repressed by the Tet repressor protein and induced in response to tetracycline (Tc) and Tc derivatives such as anhydrotetracycline. Vectors may also be inducible because they contain hormone response elements, such as the glucocorticoid response element (GRE) and the estrogen response element (ERE), which can confer hormone inducibility where vectors are used for expression in cells having the respective hormone receptors. To reduce background levels of expression, elements responsive to ecdysone, an insect hormone, can be used instead, with coexpression of the ecdysone receptor. \\n     In one embodiment of the invention, expression vectors can be designed to fuse the expressed polypeptide to small protein tags that facilitate purification and/or visualization. Such tags include a polyhistidine tag that facilitates purification of the fusion protein by immobilized metal affinity chromatography, for example using NiNTA resin (Qiagen Inc., Valencia, Calif., USA) or TALON™ resin (cobalt immobilized affinity chromatography medium, Clontech Labs, Palo Alto, Calif., USA). The fusion protein can include a chitin-binding tag and self-excising intein, permitting chitin-based purification with self-removal of the fused tag (IMPACT™ system, New England Biolabs, Inc., Beverley, Mass., USA). Alternatively, the fusion protein can include a calmodulin-binding peptide tag, permitting purification by calmodulin affinity resin (Stratagene, La Jolla, Calif., USA), or a specifically excisable fragment of the biotin carboxylase carrier protein, permitting purification of in vivo biotinylated protein using an avidin resin and subsequent tag removal (Promega, Madison, Wis., USA). As another useful alternative, the polypeptides of the present invention can be expressed as a fusion to glutathione-S-transferase, the affinity and specificity of binding to glutathione permitting purification using glutathione affinity resins, such as Glutathione-Superflow Resin (Clontech Laboratories, Palo Alto, Calif., USA), with subsequent elution with free glutathione. Other tags include, for example, the Xpress epitope, detectable by anti-Xpress antibody (Invitrogen, Carlsbad, Calif., USA), a myc tag, detectable by anti-myc tag antibody, the V5 epitope, detectable by anti-V5 antibody (Invitrogen, Carlsbad, Calif., USA), FLAG® epitope, detectable by anti-FLAG® antibody (Stratagene, La Jolla, Calif., USA), and the HA epitope, detectable by anti-HA antibody. \\n     For secretion of expressed polypeptides, vectors can include appropriate sequences that encode secretion signals, such as leader peptides. For example, the pSecTag2 vectors (Invitrogen, Carlsbad, Calif., USA) are 5.2 kb mammalian expression vectors that carry the secretion signal from the V-J2-C region of the mouse Ig kappa-chain for efficient secretion of recombinant proteins from a variety of mammalian cell lines. \\n     Expression vectors can also be designed to fuse proteins encoded by the heterologous nucleic acid insert to polypeptides that are larger than purification and/or identification tags. Useful protein fusions include those that permit display of the encoded protein on the surface of a phage or cell, fusions to intrinsically fluorescent proteins, such as those that have a green fluorescent protein (GFP)-like chromophore, fusions to the IgG Fc region, and fusions for use in two hybrid systems. \\n     Vectors for phage display fuse the encoded polypeptide to, e.g., the gene III protein (pIII) or gene VIII protein (pVIII) for display on the surface of filamentous phage, such as M13. See Barbas et al.,  Phage Display: A Laboratory Manual , Cold Spring Harbor Laboratory Press (2001); Kay et al. (eds.),  Phage Display of Peptides and Proteins: A Laboratory Manual , Academic Press, Inc., (1996); Abelson et al. (eds.),  Combinatorial Chemistry  (Methods in Enzymology, Vol. 267) Academic Press (1996). Vectors for yeast display, e.g. the pYD1 yeast display vector (Invitrogen, Carlsbad, Calif., USA), use the α-agglutinin yeast adhesion receptor to display recombinant protein on the surface of  S. cerevisiae . Vectors for mammalian display, e.g., the pDisplay™ vector (Invitrogen, Carlsbad, Calif., USA), target recombinant proteins using an N-terminal cell surface targeting signal and a C-terminal transmembrane anchoring domain of platelet derived growth factor receptor. \\n     A wide variety of vectors now exist that fuse proteins encoded by heterologous nucleic acids to the chromophore of the substrate-independent, intrinsically fluorescent green fluorescent protein from  Aequorea victoria  (“GFP”) and its variants. The GFP-like chromophore can be selected from GFP-like chromophores found in naturally occurring proteins, such as  A. victoria  GFP (GenBank accession number AAA27721),  Renilla reniformis  GFP, FP583 (GenBank accession no. AF168419) (DsRed), FP593 (AF272711), FP483 (AF168420), FP484 (AF168424), FP595 (AF246709), FP486 (AF168421), FP538 (AF168423), and FP506 (AF168422), and need include only so much of the native protein as is needed to retain the chromophore&#39;s intrinsic fluorescence. Methods for determining the minimal domain required for fluorescence are known in the art. See Li et al.,  J. Biol. Chem.  272: 28545-28549 (1997). Alternatively, the GFP-like chromophore can be selected from GFP-like chromophores modified from those found in nature. The methods for engineering such modified GFP-like chromophores and testing them for fluorescence activity, both alone and as part of protein fusions, are well known in the art. See Heim et al.,  Curr. Biol.  6: 178-182 (1996) and Palm et al.,  Methods Enzymol.  302: 378-394 (1999). A variety of such modified chromophores are now commercially available and can readily be used in the fusion proteins of the present invention. These include EGFP (“enhanced GFP”), EBFP (“enhanced blue fluorescent protein”), BFP2, EYFP (“enhanced yellow fluorescent protein”), ECFP (“enhanced cyan fluorescent protein”) or Citrine. EGFP (see, e.g, Cormack et al.,  Gene  173: 33-38 (1996); U.S. Pat. Nos. 6,090,919 and 5,804,387, the disclosures of which are incorporated herein by reference in their entireties) is found on a variety of vectors, both plasmid and viral, which are available commercially (Clontech Labs, Palo Alto, Calif., USA); EBFP is optimized for expression in mammalian cells whereas BFP2, which retains the original jellyfish codons, can be expressed in bacteria (see, e.g., Heim et al.,  Curr. Biol.  6: 178-182 (1996) and Cormack et al.,  Gene  173: 33-38 (1996)). Vectors containing these blue-shifted variants are available from Clontech Labs (Palo Alto, Calif., USA). Vectors containing EYFP, ECFP (see, e.g., Heim et al.,  Curr. Biol.  6: 178-182 (1996); Miyawaki et al.,  Nature  388: 882-887 (1997)) and Citrine (see, e.g., Heikal et al.,  Proc. Natl. Acad. Sci. USA  97: 11996-12001 (2000)) are also available from Clontech Labs. The GFP-like chromophore can also be drawn from other modified GFPs, including those described in U.S. Pat. Nos. 6,124,128; 6,096,865; 6,090,919; 6,066,476; 6,054,321; 6,027,881; 5,968,750; 5,874,304; 5,804,387; 5,777,079; 5,741,668; and 5,625,048, the disclosures of which are incorporated herein by reference in their entireties. See also Conn (ed.),  Green Fluorescent Protein  (Methods in Enzymology, Vol. 302), Academic Press, Inc. (1999); Yang, et al.,  J Biol Chem,  273: 8212-6 (1998); Bevis et al.,  Nature Biotechnology,  20:83-7 (2002). The GFP-like chromophore of each of these GFP variants can usefully be included in the fusion proteins of the present invention. \\n     Fusions to the IgG Fc region increase serum half-life of protein pharmaceutical products through interaction with the FcRn receptor (also denominated the FcRp receptor and the Brambell receptor, FcRb), further described in International Patent Application Nos. WO 97/43316, WO 97/34631, WO 96/32478, and WO 96/18412, the disclosures of which are incorporated herein by reference in their entireties. \\n     For long-term, high-yield recombinant production of the polypeptides of the present invention, stable expression is preferred. Stable expression is readily achieved by integration into the host cell genome of vectors having selectable markers, followed by selection of these integrants. Vectors such as pUB6/V5-His A, B, and C (Invitrogen, Carlsbad, Calif., USA) are designed for high-level stable expression of heterologous proteins in a wide range of mammalian tissue types and cell lines. pUB6/V5-His uses the promoter/enhancer sequence from the human ubiquitin C gene to drive expression of recombinant proteins: expression levels in 293, CHO, and NIH3T3 cells are comparable to levels from the CMV and human EF-1a promoters. The bsd gene permits rapid selection of stably transfected mammalian cells with the potent antibiotic blasticidin. \\n     Replication incompetent retroviral vectors, typically derived from Moloney murine leukemia virus, also are useful for creating stable transfectants having integrated provirus. The highly efficient transduction machinery of retroviruses, coupled with the availability of a variety of packaging cell lines such as RetroPack™ PT 67, EcoPack2™-293, AmphoPack-293, and GP2-293 cell lines (all available from Clontech Laboratories, Palo Alto, Calif., USA) allow a wide host range to be infected with high efficiency; varying the multiplicity of infection readily adjusts the copy number of the integrated provirus. \\n     Of course, not all vectors and expression control sequences will function equally well to express the nucleic acid molecules of this invention. Neither will all hosts function equally well with the same expression system. However, one of skill in the art may make a selection among these vectors, expression control sequences and hosts without undue experimentation and without departing from the scope of this invention. For example, in selecting a vector, the host must be considered because the vector must be replicated in it. The vector&#39;s copy number, the ability to control that copy number, the ability to control integration, if any, and the expression of any other proteins encoded by the vector, such as an antibiotic or other selection marker, should also be considered. The present invention further includes host cells comprising the vectors of the present invention, either present episomally within the cell or integrated, in whole or in part, into the host cell chromosome. Among other considerations, some of which are described above, a host cell strain may be chosen for its ability to process the expressed polypeptide in the desired fashion. Such post-translational modifications of the polypeptide include, but are not limited to, acetylation, carboxylation, glycosylation, phosphorylation, lipidation, and acylation, and it is an aspect of the present invention to provide OSPs with such post-translational modifications. \\n     In selecting an expression control sequence, a variety of factors should also be considered. These include, for example, the relative strength of the sequence, its controllability, and its compatibility with the nucleic acid molecules of this invention, particularly with regard to potential secondary structures. Unicellular hosts should be selected by consideration of their compatibility with the chosen vector, the toxicity of the product coded for by the nucleic acid sequences of this invention, their secretion characteristics, their ability to fold the polypeptide correctly, their fermentation or culture requirements, and the ease of purification from them of the products coded for by the nucleic acid molecules of this invention. \\n     The recombinant nucleic acid molecules and more particularly, the expression vectors of this invention may be used to express the polypeptides of this invention as recombinant polypeptides in a heterologous host cell. The polypeptides of this invention may be full-length or less than full-length polypeptide fragments recombinantly expressed from the nucleic acid molecules according to this invention. Such polypeptides include analogs, derivatives and muteins that may or may not have biological activity. \\n     Vectors of the present invention will also often include elements that permit in vitro transcription of RNA from the inserted heterologous nucleic acid. Such vectors typically include a phage promoter, such as that from T7, T3, or SP6, flanking the nucleic acid insert. Often two different such promoters flank the inserted nucleic acid, permitting separate in vitro production of both sense and antisense strands. \\n     Transformation and other methods of introducing nucleic acids into a host cell (e.g., conjugation, protoplast transformation or fusion, transfection, electroporation, liposome delivery, membrane fusion techniques, high velocity DNA-coated pellets, viral infection and protoplast fusion) can be accomplished by a variety of methods which are well known in the art (See, for instance, Ausubel, supra, and Sambrook et al., supra). Bacterial, yeast, plant or mammalian cells are transformed or transfected with an expression vector, such as a plasmid, a cosmid, or the like, wherein the expression vector comprises the nucleic acid of interest. Alternatively, the cells may be infected by a viral expression vector comprising the nucleic acid of interest. Depending upon the host cell, vector, and method of transformation used, transient or stable expression of the polypeptide will be constitutive or inducible. One having ordinary skill in the art will be able to decide whether to express a polypeptide transiently or stably, and whether to express the protein constitutively or inducibly. \\n     A wide variety of unicellular host cells are useful in expressing the DNA sequences of this invention. These hosts may include well known eukaryotic and prokaryotic hosts, such as strains of, fungi, yeast, insect cells such as  Spodoptera frugiperda  (SF9), animal cells such as CHO, as well as plant cells in tissue culture. Representative examples of appropriate host cells include, but are not limited to, bacterial cells, such as  E. coli, Caulobacter crescentus, Streptomyces  species, and  Salmonella typhimurium ; yeast cells, such as  Saccharomyces cerevisiae, Schizosaccharomyces pombe, Pichia pastoris, Pichia methanolica ; insect cell lines, such as those from  Spodoptera frugiperda , e.g., Sf9 and Sf21 cell lines, and expresSF™ cells (Protein Sciences Corp., Meriden, Conn., USA),  Drosophila  S2 cells, and  Trichoplusia ni  High Five® Cells (Invitrogen, Carlsbad, Calif., USA); and mammalian cells. Typical mammalian cells include BHK cells, BSC 1 cells, BSC 40 cells, BMT 10 cells, VERO cells, COS1 cells, COS7 cells, Chinese hamster ovary (CHO) cells, 3T3 cells, NIH 3T3 cells, 293 cells, HEPG2 cells, HeLa cells, L cells, MDCK cells, HEK293 cells, WI38 cells, murine ES cell lines (e.g., from strains 129/SV, C57/BL6, DBA-1, 129/SVJ), K562 cells, Jurkat cells, and BW5147 cells. Other mammalian cell lines are well known and readily available from the American Type Culture Collection (ATCC) (Manassas, Va., USA) and the National Institute of General Medical Sciences (NIGMS) Human Genetic Cell Repository at the Coriell Cell Repositories (Camden, N.J., USA). Cells or cell lines derived from ovarian are particularly preferred because they may provide a more native post-translational processing. Particularly preferred are human ovarian cells. \\n     Particular details of the transfection, expression and purification of recombinant proteins are well documented and are understood by those of skill in the art. Further details on the various technical aspects of each of the steps used in recombinant production of foreign genes in bacterial cell expression systems can be found in a number of texts and laboratory manuals in the art. See, e.g., Ausubel (1992), supra, Ausubel (1999), supra, Sambrook (1989), supra, and Sambrook (2001), supra. \\n     Methods for introducing the vectors and nucleic acid molecules of the present invention into the host cells are well known in the art; the choice of technique will depend primarily upon the specific vector to be introduced and the host cell chosen. \\n     Nucleic acid molecules and vectors may be introduced into prokaryotes, such as  E. coli , in a number of ways. For instance, phage lambda vectors will typically be packaged using a packaging extract (e.g., Gigapack® packaging extract, Stratagene, La Jolla, Calif., USA), and the packaged virus used to infect  E. coli.    \\n     Plasmid vectors will typically be introduced into chemically competent or electrocompetent bacterial cells.  E. coli  cells can be rendered chemically competent by treatment, e.g., with CaCl 2 , or a solution of Mg 2+ , Mn 2+ , Ca 2+ , R +  or K + , dimethyl sulfoxide, dithiothreitol, and hexamine cobalt (III), Hanahan,  J. Mol. Biol.  166(4):557-80 (1983), and vectors introduced by heat shock. A wide variety of chemically competent strains are also available commercially (e.g., Epicurian Coli® XL10-Gold® Ultracompetent Cells (Stratagene, La Jolla, Calif., USA); DH5α competent cells (Clontech Laboratories, Palo Alto, Calif., USA); and TOP10 Chemically Competent  E. coli  Kit (Invitrogen, Carlsbad, Calif., USA)). Bacterial cells can be rendered electrocompetent to take up exogenous DNA by electroporation by various pre-pulse treatments; vectors are introduced by electroporation followed by subsequent outgrowth in selected media. An extensive series of protocols is provided by BioRad (Richmond, Calif., USA). \\n     Vectors can be introduced into yeast cells by spheroplasting, treatment with lithium salts, electroporation, or protoplast fusion. Spheroplasts are prepared by the action of hydrolytic enzymes such as a snail-gut extract, usually denoted Glusulase or Zymolyase, or an enzyme from  Arthrobacter luteus  to remove portions of the cell wall in the presence of osmotic stabilizers, typically 1 M sorbitol. DNA is added to the spheroplasts, and the mixture is co-precipitated with a solution of polyethylene glycol (PEG) and Ca 2+ . Subsequently, the cells are resuspended in a solution of sorbitol, mixed with molten agar and then layered on the surface of a selective plate containing sorbitol. \\n     For lithium-mediated transformation, yeast cells are treated with lithium acetate to permeabilize the cell wall, DNA is added and the cells are co-precipitated with PEG. The cells are exposed to a brief heat shock, washed free of PEG and lithium acetate, and subsequently spread on plates containing ordinary selective medium. Increased frequencies of transformation are obtained by using specially-prepared single-stranded carrier DNA and certain organic solvents. Schiestl et al.,  Curr. Genet.  16(5-6): 339-46 (1989). \\n     For electroporation, freshly-grown yeast cultures are typically washed, suspended in an osmotic protectant, such as sorbitol, mixed with DNA, and the cell suspension pulsed in an electroporation device. Subsequently, the cells are spread on the surface of plates containing selective media. Becker et al.,  Methods Enzymol.  194: 182-187 (1991). The efficiency of transformation by electroporation can be increased over 100-fold by using PEG, single-stranded carrier DNA and cells that are in late log-phase of growth. Larger constructs, such as YACs, can be introduced by protoplast fusion. \\n     Mammalian and insect cells can be directly infected by packaged viral vectors, or transfected by chemical or electrical means. For chemical transfection, DNA can be coprecipitated with CaPO 4  or introduced using liposomal and nonliposomal lipid-based agents. Commercial kits are available for CaPO 4  transfection (CalPhos™ Mammalian Transfection Kit, Clontech Laboratories, Palo Alto, Calif., USA), and lipid-mediated transfection can be practiced using commercial reagents, such as LIPOFECTAMINE™ 2000, LIPOFECTAMINE™ Reagent, CELLFECTIN® Reagent, and LIPOFECTIN® Reagent (Invitrogen, Carlsbad, Calif., USA), DOTAP Liposomal Transfection Reagent, FuGENE 6, X-tremeGENE Q2, DOSPER, (Roche Molecular Biochemicals, Indianapolis, Ind. USA), Effectene™, PolyFect®, Superfect® (Qiagen, Inc., Valencia, Calif., USA). Protocols for electroporating mammalian cells can be found in, for example, Norton et al. (eds.),  Gene Transfer Methods: Introducing DNA into Living Cells and Organisms , BioTechniques Books, Eaton Publishing Co. (2000). Other transfection techniques include transfection by particle bombardment and microinjection. See, e.g., Cheng et al.,  Proc. Natl. Acad. Sci. USA  90(10): 4455-9 (1993); Yang et al.,  Proc. Natl. Acad. Sci. USA  87(24): 9568-72 (1990). \\n     Production of the recombinantly produced proteins of the present invention can optionally be followed by purification. \\n     Purification of recombinantly expressed proteins is now well within the skill in the art and thus need not be detailed here. See, e.g., Thorner et al. (eds.),  Applications of Chimeric Genes and Hybrid Proteins, Part A: Gene Expression and Protein Purification  (Methods in Enzymology, Vol. 326), Academic Press (2000); Harbin (ed.),  Cloning, Gene Expression and Protein Purification: Experimental Procedures and Process Rationale , Oxford Univ. Press (2001); Marshak et al.,  Strategies for Protein Purification and Characterization: A Laboratory Course Manual , Cold Spring Harbor Laboratory Press (1996); and Roe (ed.),  Protein Purification Applications , Oxford University Press (2001). \\n     Briefly, however, if purification tags have been fused through use of an expression vector that appends such tags, purification can be effected, at least in part, by means appropriate to the tag, such as use of immobilized metal affinity chromatography for polyhistidine tags. Other techniques common in the art include ammonium sulfate fractionation, immunoprecipitation, fast protein liquid chromatography (FPLC), high performance liquid chromatography (HPLC), and preparative gel electrophoresis. \\n     Polypeptides, Including Fragments Muteins, Homologous Proteins, Allelic Variants, Analogs and Derivatives \\n     Another aspect of the invention relates to polypeptides encoded by the nucleic acid molecules described herein. In a preferred embodiment, the polypeptide is an ovarian specific polypeptide (OSP). In an even more preferred embodiment, the polypeptide comprises an amino acid sequence of SEQ ID NO:129-295 or is derived from a polypeptide having the amino acid sequence of SEQ ID NO: 129-295. A polypeptide as defined herein may be produced recombinantly, as discussed supra, may be isolated from a cell that naturally expresses the protein, or may be chemically synthesized following the teachings of the specification and using methods well known to those having ordinary skill in the art. \\n     Polypeptides of the present invention may also comprise a part or fragment of an OSP. In a preferred embodiment, the fragment is derived from a polypeptide having an amino acid sequence selected from the group consisting of SEQ ID NO: 129-295. Polypeptides of the present invention comprising a part or fragment of an entire OSP may or may not be OSPs. For example, a full-length polypeptide may be ovarian-specific, while a fragment thereof may be found in other tissues as well as in ovarian. A polypeptide that is not an OSP, whether it is a fragment, analog, mutein, homologous protein or derivative, is nevertheless useful, especially for immunizing animals to prepare anti-OSP antibodies. In a preferred embodiment, the part or fragment is an OSP. Methods of determining whether a polypeptide of the present invention is an OSP are described infra. \\n     Polypeptides of the present invention comprising fragments of at least 6 contiguous amino acids are also useful in mapping B cell and T cell epitopes of the reference protein. See, e.g., Geysen et al.,  Proc. Natl. Acad. Sci. USA  81: 3998-4002 (1984) and U.S. Pat. Nos. 4,708,871 and 5,595,915, the disclosures of which are incorporated herein by reference in their entireties. Because the fragment need not itself be immunogenic, part of an immunodominant epitope, nor even recognized by native antibody, to be useful in such epitope mapping, all fragments of at least 6 amino acids of a polypeptide of the present invention have utility in such a study. \\n     Polypeptides of the present invention comprising fragments of at least 8 contiguous amino acids, often at least 15 contiguous amino acids, are useful as immunogens for raising antibodies that recognize polypeptides of the present invention. See, e.g., Lerner, Nature 299: 592-596 (1982); Shinnick et al.,  Annu. Rev. Microbiol.  37: 425-46 (1983); Sutcliffe et al.,  Science  219: 660-6 (1983). As further described in the above-cited references, virtually all 8-mers, conjugated to a carrier, such as a protein, prove immunogenic and are capable of eliciting antibody for the conjugated peptide; accordingly, all fragments of at least 8 amino acids of the polypeptides of the present invention have utility as immunogens. \\n     Polypeptides comprising fragments of at least 8, 9, 10 or 12 contiguous amino acids are also useful as competitive inhibitors of binding of the entire polypeptide, or a portion thereof, to antibodies (as in epitope mapping), and to natural binding partners, such as subunits in a multimeric complex or to receptors or ligands of the subject protein; this competitive inhibition permits identification and separation of molecules that bind specifically to the polypeptide of interest. See U.S. Pat. Nos. 5,539,084 and 5,783,674, incorporated herein by reference in their entireties. \\n     The polypeptide of the present invention thus preferably is at least 6 amino acids in length, typically at least 8, 9, 10 or 12 amino acids in length, and often at least 15 amino acids in length. Often, the polypeptide of the present invention is at least 20 amino acids in length, even 25 amino acids, 30 amino acids, 35 amino acids, or 50 amino acids or more in length. Of course, larger polypeptides having at least 75 amino acids, 100 amino acids, or even 150 amino acids are also useful, and at times preferred. \\n     One having ordinary skill in the art can produce fragments by truncating the nucleic acid molecule, e.g., an OSNA, encoding the polypeptide and then expressing it recombinantly. Alternatively, one can produce a fragment by chemically synthesizing a portion of the full-length polypeptide. One may also produce a fragment by enzymatically cleaving either a recombinant polypeptide or an isolated naturally occurring polypeptide. Methods of producing polypeptide fragments are well known in the art. See, e.g., Sambrook (1989), supra; Sambrook (2001), supra; Ausubel (1992), supra; and Ausubel (1999), supra. In one embodiment, a polypeptide comprising only a fragment, preferably a fragment of an OSP, may be produced by chemical or enzymatic cleavage of an OSP polypeptide. In a preferred embodiment, a polypeptide fragment is produced by expressing a nucleic acid molecule of the present invention encoding a fragment, preferably of an OSP, in a host cell. \\n     Polypeptides of the present invention are also inclusive of mutants, fusion proteins, homologous proteins and allelic variants. \\n     A mutant protein, or mutein, may have the same or different properties compared to a naturally occurring polypeptide and comprises at least one amino acid insertion, duplication, deletion, rearrangement or substitution compared to the amino acid sequence of a native polypeptide. Small deletions and insertions can often be found that do not alter the function of a protein. Muteins may or may not be ovarian-specific. Preferably, the mutein is ovarian-specific. More preferably the mutein is a polypeptide that comprises at least one amino acid insertion, duplication, deletion, rearrangement or substitution compared to the amino acid sequence of SEQ ID NO: 129-295. Accordingly, in a preferred embodiment, the mutein is one that exhibits at least 50% sequence identity, more preferably at least 60% sequence identity, even more preferably at least 70%, yet more preferably at least 80% sequence identity to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. In a yet more preferred embodiment, the mutein exhibits at least 85%, more preferably 90%, even more preferably 95% or 96%, and yet more preferably at least 97%, 98%, 99% or 99.5% sequence identity to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. \\n     A mutein may be produced by isolation from a naturally occurring mutant cell, tissue or organism. A mutein may be produced by isolation from a cell, tissue or organism that has been experimentally mutagenized. Alternatively, a mutein may be produced by chemical manipulation of a polypeptide, such as by altering the amino acid residue to another amino acid residue using synthetic or semi-synthetic chemical techniques. In a preferred embodiment, a mutein is produced from a host cell comprising a mutated nucleic acid molecule compared to the naturally occurring nucleic acid molecule. For instance, one may produce a mutein of a polypeptide by introducing one or more mutations into a nucleic acid molecule of the invention and then expressing it recombinantly. These mutations may be targeted, in which particular encoded amino acids are altered, or may be untargeted, in which random encoded amino acids within the polypeptide are altered. Muteins with random amino acid alterations can be screened for a particular biological activity or property, particularly whether the polypeptide is ovarian-specific, as described below. Multiple random mutations can be introduced into the gene by methods well known to the art, e.g., by error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis and site-specific mutagenesis. Methods of producing muteins with targeted or random amino acid alterations are well known in the art. See, e.g., Sambrook (1989), supra; Sambrook (2001), supra; Ausubel (1992), supra; and Ausubel (1999), as well as U.S. Pat. No. 5,223,408, which is herein incorporated by reference in its entirety. \\n     The invention also contemplates polypeptides that are homologous to a polypeptide of the invention. In a preferred embodiment, the polypeptide is homologous to an OSP. In an even more preferred embodiment, the polypeptide is homologous to an OSP selected from the group having an amino acid sequence of SEQ ID NO: 129-295. By homologous polypeptide it is meant one that exhibits significant sequence identity to an OSP, preferably an OSP having an amino acid sequence of SEQ ID NO: 129-295. By significant sequence identity it is meant that the homologous polypeptide exhibits at least 50% sequence identity, more preferably at least 60% sequence identity, even more preferably at least 70%, yet more preferably at least 80% sequence identity to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. More preferred are homologous polypeptides exhibiting at least 85%, more preferably 90%, even more preferably 95% or 96%, and yet more preferably at least 97% or 98% sequence identity to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. Most preferably, the homologous polypeptide exhibits at least 99%, more preferably 99.5%, even more preferably 99.6%, 99.7%, 99.8% or 99.9% sequence identity to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295. In a preferred embodiment, the amino acid substitutions of the homologous polypeptide are conservative amino acid substitutions as discussed supra. \\n     Homologous polypeptides of the present invention also comprise polypeptide encoded by a nucleic acid molecule that selectively hybridizes to an OSNA or an antisense sequence thereof. In this embodiment, it is preferred that the homologous polypeptide be encoded by a nucleic acid molecule that hybridizes to an OSNA under low stringency, moderate stringency or high stringency conditions, as defined herein. More preferred is a homologous polypeptide encoded by a nucleic acid sequence which hybridizes to a OSNA selected from the group consisting of SEQ ID NO: 1-128 or a homologous polypeptide encoded by a nucleic acid molecule that hybridizes to a nucleic acid molecule that encodes an OSP, preferably an OSP of SEQ ID NO:129-295 under low stringency, moderate stringency or high stringency conditions, as defined herein. \\n     Homologous polypeptides of the present invention may be naturally occurring and derived from another species, especially one derived from another primate, such as chimpanzee, gorilla, rhesus macaque, or baboon, wherein the homologous polypeptide comprises an amino acid sequence that exhibits significant sequence identity to that of SEQ ID NO: 129-295. The homologous polypeptide may also be a naturally occurring polypeptide from a human, when the OSP is a member of a family of polypeptides. The homologous polypeptide may also be a naturally occurring polypeptide derived from a non-primate, mammalian species, including without limitation, domesticated species, e.g., dog, cat, mouse, rat, rabbit, guinea pig, hamster, cow, horse, goat or pig. The homologous polypeptide may also be a naturally occurring polypeptide derived from a non-mammalian species, such as birds or reptiles. The naturally occurring homologous protein may be isolated directly from humans or other species. Alternatively, the nucleic acid molecule encoding the naturally occurring homologous polypeptide may be isolated and used to express the homologous polypeptide recombinantly. The homologous polypeptide may also be one that is experimentally produced by random mutation of a nucleic acid molecule and subsequent expression of the nucleic acid molecule. Alternatively, the homologous polypeptide may be one that is experimentally produced by directed mutation of one or more codons to alter the encoded amino acid of an OSP. In a preferred embodiment, the homologous polypeptide encodes a polypeptide that is an OSP. \\n     Relatedness of proteins can also be characterized using a second functional test, such as the ability of a first protein competitively to inhibit the binding of a second protein to an antibody. It is, therefore, another aspect of the present invention to provide isolated polypeptides not only identical in sequence to those described with particularity herein, but also to provide isolated polypeptides (“cross-reactive proteins”) that competitively inhibit the binding of antibodies to all or to a portion of the isolated polypeptides of the present invention. Such competitive inhibition can readily be determined using immunoassays well known in the art. \\n     As discussed above, single nucleotide polymorphisms (SNPs) occur frequently in eukaryotic genomes, and the sequence determined from one individual of a species may differ from other allelic forms present within the population. Thus, polypeptides of the present invention are also inclusive of those encoded by an allelic variant of a nucleic acid molecule encoding an OSP. In this embodiment, it is preferred that the polypeptide be encoded by an allelic variant of a gene that encodes a polypeptide having the amino acid sequence selected from the group consisting of SEQ ID NO: 129-295. More preferred is that the polypeptide be encoded by an allelic variant of a gene that has the nucleic acid sequence selected from the group consisting of SEQ ID NO: 1-128. \\n     Polypeptides of the present invention are also inclusive of derivative polypeptides encoded by a nucleic acid molecule according to the instant invention. In this embodiment, it is preferred that the polypeptide be an OSP. Also preferred are derivative polypeptides having an amino acid sequence selected from the group consisting of SEQ ID NO: 129-295 and which has been acetylated, carboxylated, phosphorylated, glycosylated, ubiquitinated or post-translationally modified in another manner. In another preferred embodiment, the derivative has been labeled with, e.g., radioactive isotopes such as  125 I,  32 P,  35 S, and  3 H. In another preferred embodiment, the derivative has been labeled with fluorophores, chemiluminescent agents, enzymes, and antiligands that can serve as specific binding pair members for a labeled ligand. \\n     Polypeptide modifications are well known to those of skill and have been described in great detail in the scientific literature. Several particularly common modifications, glycosylation, lipid attachment, sulfation, gamma-carboxylation of glutamic acid residues, hydroxylation and ADP-ribosylation, for instance, are described in most basic texts, such as, for instance Creighton,  Protein Structure and Molecular Properties,  2nd ed., W. H. Freeman and Company (1993). Many detailed reviews are available on this subject, such as, for example, those provided by Wold, in Johnson (ed.),  Posttranslational Covalent Modification of Proteins , pgs. 1-12, Academic Press (1983); Seifter et al.,  Meth. Enzymol.  182: 626-646 (1990) and Rattan et al.,  Ann. N.Y. Acad. Sci.  663: 48-62 (1992). \\n     One may determine whether a polypeptide of the invention is likely to be post-translationally modified by analyzing the sequence of the polypeptide to determine if there are peptide motifs indicative of sites for post-translational modification. There are a number of computer programs that permit prediction of post-translational modifications. See, e.g., expasy.org (accessed Nov. 11, 2002) of the world wide web, which includes PSORT, for prediction of protein sorting signals and localization sites, SignalP, for prediction of signal peptide cleavage sites, MITOPROT and Predotar, for prediction of mitochondrial targeting sequences, NetOGlyc, for prediction of type O-glycosylation sites in mammalian proteins, big-PI Predictor and DGPI, for prediction of prenylation-anchor and cleavage sites, and NetPhos, for prediction of Ser, Thr and Tyr phosphorylation sites in eukaryotic proteins. Other computer programs, such as those included in GCG, also may be used to determine post-translational modification peptide motifs. \\n     General examples of types of post-translational modifications include, but are not limited to: (Z)-dehydrobutyrine; 1-chondroitin sulfate-L-aspartic acid ester; 1′-glycosyl-L-tryptophan; 1′-phospho-L-histidine; 1-thioglycine; 2′-(S-L-cysteinyl)-L-histidine; 2′-[3-carboxamido (trimethylammonio)propyl]-L-histidine; 2′-alpha-mannosyl-L-tryptophan; 2-methyl-L-glutamine; 2-oxobutanoic acid; 2-pyrrolidone carboxylic acid; 3′-(1′-L-histidyl)-L-tyrosine; 3′-(8alpha-FAD)-L-histidine; 3′-(S-L-cysteinyl)-L-tyrosine; 3′,3″,5′-triiodo-L-thyronine; 3′-4′-phospho-L-tyrosine; 3-hydroxy-L-proline; 3′-methyl-L-histidine; 3-methyl-L-lanthionine; 3′-phospho-L-histidine; 4′-(L-tryptophan)-L-tryptophyl quinone; 42 N-cysteinyl-glycosylphosphatidylinositolethanolamine; 43-(T-L-histidyl)-L-tyrosine; 4-hydroxy-L-arginine; 4-hydroxy-L-lysine; 4-hydroxy-L-proline; 5′-(N6-L-lysine)-L-topaquinone; 5-hydroxy-L-lysine; 5-methyl-L-arginine; alpha-1-microglobulin-Ig alpha complex chromophore; bis-L-cysteinyl bis-L-histidino diiron disulfide; bis-L-cysteinyl-L-N3′-histidino-L-serinyl tetrairon’ tetrasulfide; chondroitin sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine; D-alanine; D-allo-isoleucine; D-asparagine; dehydroalanine; dehydrotyrosine; dermatan 4-sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine; D-glucuronyl-N-glycine; dipyrrolylmethanemethyl-L-cysteine; D-leucine; D-methionine; D-phenylalanine; D-serine; D-tryptophan; glycine amide; glycine oxazolecarboxylic acid; glycine thiazolecarboxylic acid; heme P450-bis-L-cysteine-L-tyrosine; heme-bis-L-cysteine; hemediol-L-aspartyl ester-L-glutamyl ester; hemediol-L-aspartyl ester-L-glutamyl ester-L-methionine sulfonium; heme-L-cysteine; heme-L-histidine; heparan sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine; heme P450-bis-L-cysteine-L-lysine; hexakis-L-cysteinyl hexairon hexasulfide; keratan sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-threonine; L oxoalanine-lactic acid; L phenyllacetic acid; 1′-(8alpha-FAD)-L-histidine; L-2′,4′,5′-topaquinone; L-3′,4′-dihydroxyphenylalanine; L-3′,4′,5′-trihydroxyphenylalanine; L-4′-bromophenylalanine; L-6′-bromotryptophan; L-alanine amide; L-alanyl imidazolinone glycine; L-allysine; L-arginine amide; L-asparagine amide; L-aspartic 4-phosphoric anhydride; L-aspartic acid 1-amide; L-beta-methylthioaspartic acid; L-bromohistidine; L-citrulline; L-cysteine amide; L-cysteine glutathione disulfide; L-cysteine methyl disulfide; L-cysteine methyl ester; L-cysteine oxazolecarboxylic acid; L-cysteine oxazolinecarboxylic acid; L-cysteine persulfide; L-cysteine sulfenic acid; L-cysteine sulfinic acid; L-cysteine thiazolecarboxylic acid; L-cysteinyl homocitryl molybdenum-heptairon-nonasulfide; L-cysteinyl imidazolinone glycine; L-cysteinyl molybdopterin; L-cysteinyl molybdopterin guanine dinucleotide; L-cystine; L-erythro-beta-hydroxyasparagine; L-erythro-beta-hydroxyaspartic acid; L-gamma-carboxyglutamic acid; L-glutamic acid 1-amide; L-glutamic acid 5-methyl ester; L-glutamine amide; L-glutamyl 5-glycerylphosphorylethanolamine; L-histidine amide; L-isoglutamyl-polyglutamic acid; L-isoglutamyl-polyglycine; L-isoleucine amide; L-lanthionine; L-leucine amide; L-lysine amide; L-lysine thiazolecarboxylic acid; L-lysinoalanine; L-methionine amide; L-methionine sulfone; L-phenyalanine thiazolecarboxylic acid; L-phenylalanine amide; L-proline amide; L-selenocysteine; L-selenocysteinyl molybdopterin guanine dinucleotide; L-serine amide; L-serine thiazolecarboxylic acid; L-seryl imidazolinone glycine; L-T-bromophenylalanine; L-T-bromophenylalanine; L-threonine amide; L-thyroxine; L-tryptophan amide; L-tryptophyl quinone; L-tyrosine amide; L-valine amide; meso-lanthionine; N-(L-glutamyl)-L-tyrosine; N-(L-isoaspartyl)-glycine; N-(L-isoaspartyl)-L-cysteine; N,N,N-trimethyl-L-alanine; N,N-dimethyl-L-proline; N2-acetyl-L-lysine; N2-succinyl-L-tryptophan; N4-(ADP-ribosyl)-L-asparagine; N4-glycosyl-L-asparagine; N4-hydroxymethyl-L-asparagine; N4-methyl-L-asparagine; N5-methyl-L-glutamine; N6-1-carboxyethyl-L-lysine; N6-(4-amino hydroxybutyl)-L-lysine; N6-(L-isoglutamyl)-L-lysine; N6-(phospho-5′-adenosine)-L-lysine; N6-(phospho-5′-guanosine)-L-tysine; N6,N6,N6-trimethyl-L-lysine; N6,N6-dimethyl-L-lysine; N6-acetyl-L-lysine; N6-biotinyl-L-lysine; N6-carboxy-L-lysine; N6-formyl-L-lysine; N6-glycyl-L-lysine; N6-lipoyl-L-lysine; N6-methyl-L-lysine; N6-methyl-N6-poly(N-methyl-propylamine)-L-lysine; N6-mureinyl-L-lysine; N6-myristoyl-L-lysine; N6-palmitoyl-L-lysine; N6-pyridoxal phosphate-L-lysine; N6-pyruvic acid 2-iminyl-L-lysine; N6-retinal-L-lysine; N-acetylglycine; N-acetyl-L-glutamine; N-acetyl-L-alanine; N-acetyl-L-aspartic acid; N-acetyl-L-cysteine; N-acetyl-L-glutamic acid; N-acetyl-L-isoleucine; N-acetyl-L-methionine; N-acetyl-L-proline; N-acetyl-L-serine; N-acetyl-L-threonine; N-acetyl-L-tyrosine; N-acetyl-L-valine; N-alanyl-glycosylphosphatidylinositolethanolamine; N-asparaginyl-glycosylphosphatidylinositolethanolamine; N-aspartyl-glycosylphosphatidylinositolethanolamine; N-formylglycine; N-formyl-L-methionine; N-glycyl-glycosylphosphatidylinositolethanolamine; N-L-glutamyl-poly-L-glutamic acid; N-methylglycine; N-methyl-L-alanine; N-methyl-L-methionine; N-methyl-L-phenylalanine; N-myristoyl-glycine; N-palmitoyl-L-cysteine; N-pyruvic acid 2-iminyl-L-cysteine; N-pyruvic acid 2-iminyl-L-valine; N-seryl-glycosylphosphatidylinositolethanolamine; N-seryl-glycosyOSPhingolipidinositolethanolamine; O-(ADP-ribosyl)-L-serine; O-(phospho-5′-adenosine)-L-threonine; O-(phospho-5′-DNA)-L-serine; O-(phospho-5′-DNA)-L-threonine; O-(phospho-5′rRNA)-L-serine; O-(phosphoribosyl dephospho-coenzyme A)-L-serine; O-(sn-1-glycerophosphoryl)-L-serine; O4′-(8alpha-FAD)-L-tyrosine; O4′-(phospho-5′-adenosine)-L-tyrosine; O4′-(phospho-5′-DNA)-L-tyrosine; O4′-(phospho-5′-RNA)-L-tyrosine; O4′-(phospho-5′-uridine)-L-tyrosine; 04-glycosyl-L-hydroxyproline; O4′-glycosyl-L-tyrosine; O4′-sulfo-L-tyrosine; O5-glycosyl-L-hydroxylysine; O-glycosyl-L-serine; O-glycosyl-L-threonine; omega-N-(ADP-ribosyl)-L-arginine; omega-N-omega-N′-dimethyl-L-arginine; omega-N-methyl-L-arginine; omega-N-omega-N-dimethyl-L-arginine; omega-N-phospho-L-arginine; O′octanoyl-L-serine; O-palmitoyl-L-serine; O-palmitoyl-L-threonine; O-phospho-L-serine; O-phospho-L-threonine; O-phosphopantetheine-L-serine; phycoerythrobilin-bis-L-cysteine; phycourobilin-bis-L-cysteine; pyrroloquinoline quinone; pyruvic acid; S hydroxycinnamyl-L-cysteine; S-(2-aminovinyl) methyl-D-eysteine; S-(2-aminovinyl)-D-cysteine; S-(6-FW-L-cysteine; S-(8alpha-FAD)-L-cysteine; S-(ADP-ribosyl)-L-cysteine; S-(L-isoglutamyl)-L-cysteine; S-12-hydroxyfarnesyl-L-cysteine; S-acetyl-L-cysteine; S-diacylglycerol-L-cysteine; S-diphytanylglycerot diether-L-cysteine; S-farnesyl-L-cysteine; S-geranylgeranyl-L-cysteine; S-glycosyl-L-cysteine; S-glycyl-L-cysteine; S-methyl-L-cysteine; S-nitrosyl-L-cysteine; S-palmitoyl-L-cysteine; S-phospho-L-cysteine; S-phycobiliviolin-L-cysteine; S-phycocyanobilin-L-cysteine; S-phycoerythrobilin-L-cysteine; S-phytochromobilin-L-cysteine; S-selenyl-L-cysteine; S-sulfo-L-cysteine; tetrakis-L-cysteinyl diiron disulfide; tetrakis-L-cysteinyl iron; tetrakis-L-cysteinyl tetrairon tetrasulfide; trans-2,3-cis 4-dihydroxy-L-proline; tris-L-cysteinyl triiron tetrasulfide; tris-L-cysteinyl triiron trisulfide; tris-L-cysteinyl-L-aspartato tetrairon tetrasulfide; tris-L-cysteinyl-L-cysteine persulfido-bis-L-glutamato-L-histidino tetrairon disulfide trioxide; tris-L-cysteinyl-L-N3′-histidino tetrairon tetrasulfide; tris-L-cysteinyl-L-N1′-histidino tetrairon tetrasulfide; and tris-L-cysteinyl-L-serinyl tetrairon tetrasulfide. \\n     Additional examples of PTMs may be found in web sites such as the Delta Mass database based on Krishna, R. G. and F. Wold (1998). Posttranslational Modifications. Proteins—Analysis and Design. R. H. Angeletti. San Diego, Academic Press. 1: 121-206; Methods in Enzymology, 193, J. A. McClosky (ed) (1990), pages 647-660; Methods in Protein Sequence Analysis edited by Kazutomo Imahori and Fumio Sakiyama, Plenum Press, (1993) “Post-translational modifications of proteins” R. G. Krishna and F. Wold pages 167-172; “GlycoSuiteDB: a new curated relational database of glycoprotein glycan structures and their biological sources” Cooper et al. Nucleic Acids Res. 29; 332-335 (2001) “O-GLYCBASE version 4.0: a revised database of O-glycosylated proteins” Gupta et al. Nucleic Acids Research, 27: 370-372 (1999); and “PhosphoBase, a database of phosphorylation sites: release 2.0.”, Kreegipuu et al. Nucleic Acids Res 27(1):237-239 (1999) see also, WO 02/21139A2, the disclosure of which is incorporated herein by reference in its entirety. \\n     Tumorigenesis is often accompanied by alterations in the post-translational modifications of proteins. Thus, in another embodiment, the invention provides polypeptides from cancerous cells or tissues that have altered post-translational modifications compared to the post-translational modifications of polypeptides from normal cells or tissues. A number of altered post-translational modifications are known. One common alteration is a change in phosphorylation state, wherein the polypeptide from the cancerous cell or tissue is hyperphosphorylated or hypophosphorylated compared to the polypeptide from a normal tissue, or wherein the polypeptide is phosphorylated on different residues than the polypeptide from a normal cell. Another common alteration is a change in glycosylation state, wherein the polypeptide from the cancerous cell or tissue has more or less glycosylation than the polypeptide from a normal tissue, and/or wherein the polypeptide from the cancerous cell or tissue has a different type of glycosylation than the polypeptide from a noncancerous cell or tissue. Changes in glycosylation may be critical because carbohydrate-protein and carbohydrate-carbohydrate interactions are important in cancer cell progression, dissemination and invasion. See, e.g., Barchi,  Curr. Pharm. Des.  6: 485-501 (2000), Verma,  Cancer Biochem. Biophys.  14: 151-162 (1994) and Dennis et al.,  Bioessays  5: 412-421 (1999). \\n     Another post-translational modification that may be altered in cancer cells is prenylation. Prenylation is the covalent attachment of a hydrophobic prenyl group (either farnesyl or geranylgeranyl) to a polypeptide. Prenylation is required for localizing a protein to a cell membrane and is often required for polypeptide function. For instance, the Ras superfamily of GTPase signalling proteins must be prenylated for function in a cell. See, e.g., Prendergast et al.,  Semin. Cancer Biol.  10: 443-452 (2000) and Khwaja et al.,  Lancet  355: 741-744 (2000). \\n     Other post-translation modifications that may be altered in cancer cells include, without limitation, polypeptide methylation, acetylation, arginylation or racemization of amino acid residues. In these cases, the polypeptide from the cancerous cell may exhibit either increased or decreased amounts of the post-translational modification compared to the corresponding polypeptides from noncancerous cells. \\n     Other polypeptide alterations in cancer cells include abnormal polypeptide cleavage of proteins and aberrant protein-protein interactions. Abnormal polypeptide cleavage may be cleavage of a polypeptide in a cancerous cell that does not usually occur in a normal cell, or a lack of cleavage in a cancerous cell, wherein the polypeptide is cleaved in a normal cell. Aberrant protein-protein interactions may be either covalent cross-linking or non-covalent binding between proteins that do not normally bind to each other. Alternatively, in a cancerous cell, a protein may fail to bind to another protein to which it is bound in a noncancerous cell. Alterations in cleavage or in protein-protein interactions may be due to over- or underproduction of a polypeptide in a cancerous cell compared to that in a normal cell, or may be due to alterations in post-translational modifications (see above) of one or more proteins in the cancerous cell. See, e.g., Henschen-Edman,  Ann. N.Y. Acad. Sci.  936: 580-593 (2001). \\n     Alterations in polypeptide post-translational modifications, as well as changes in polypeptide cleavage and protein-protein interactions, may be determined by any method known in the art. For instance, alterations in phosphorylation may be determined by using anti-phosphoserine, anti-phosphothreonine or anti-phosphotyrosine antibodies or by amino acid analysis. Glycosylation alterations may be determined using antibodies specific for different sugar residues, by carbohydrate sequencing, or by alterations in the size of the glycoprotein, which can be determined by, e.g., SDS polyacrylamide gel electrophoresis (PAGE). Other alterations of post-translational modifications, such as prenylation, racemization, methylation, acetylation and arginylation, may be determined by chemical analysis, protein sequencing, amino acid analysis, or by using antibodies specific for the particular post-translational modifications. Changes in protein-protein interactions and in polypeptide cleavage may be analyzed by any method known in the art including, without limitation, non-denaturing PAGE (for non-covalent protein-protein interactions), SDS PAGE (for covalent protein-protein interactions and protein cleavage), chemical cleavage, protein sequencing or immunoassays. \\n     In another embodiment, the invention provides polypeptides that have been post-translationally modified. In one embodiment, polypeptides may be modified enzymatically or chemically, by addition or removal of a post-translational modification. For example, a polypeptide may be glycosylated or deglycosylated enzymatically. Similarly, polypeptides may be phosphorylated using a purified kinase, such as a MAP kinase (e.g, p38, ERK, or JNK) or a tyrosine kinase (e.g., Src or erbB2). A polypeptide may also be modified through synthetic chemistry. Alternatively, one may isolate the polypeptide of interest from a cell or tissue that expresses the polypeptide with the desired post-translational modification. In another embodiment, a nucleic acid molecule encoding the polypeptide of interest is introduced into a host cell that is capable of post-translationally modifying the encoded polypeptide in the desired fashion. If the polypeptide does not contain a motif for a desired post-translational modification, one may alter the post-translational modification by mutating the nucleic acid sequence of a nucleic acid molecule encoding the polypeptide so that it contains a site for the desired post-translational modification. Amino acid sequences that may be post-translationally modified are known in the art. See, e.g., the programs described above on the website expasy.org of the world wide web. The nucleic acid molecule may also be introduced into a host cell that is capable of post-translationally modifying the encoded polypeptide. Similarly, one may delete sites that are post-translationally modified by either mutating the nucleic acid sequence so that the encoded polypeptide does not contain the post-translational modification motif, or by introducing the native nucleic acid molecule into a host cell that is not capable of post-translationally modifying the encoded polypeptide. \\n     It will be appreciated, as is well known and as noted above, that polypeptides are not always entirely linear. For instance, polypeptides may be branched as a result of ubiquitination, and they may be circular, with or without branching, generally as a result of posttranslation events, including natural processing events and events brought about by human manipulation which do not occur naturally. Circular, branched and branched circular polypeptides may be synthesized by non-translation natural processes and by entirely synthetic methods, as well. Modifications can occur anywhere in a polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. In fact, blockage of the amino or carboxyl group in a polypeptide, or both, by a covalent modification, is common in naturally occurring and synthetic polypeptides and such modifications may be present in polypeptides of the present invention, as well. For instance, the amino terminal residue of polypeptides made in  E. coli , prior to proteolytic processing, almost invariably will be N-formylmethionine. \\n     Useful post-synthetic (and post-translational) modifications include conjugation to detectable labels, such as fluorophores. A wide variety of amine-reactive and thiol-reactive fluorophore derivatives have been synthesized that react under nondenaturing conditions with N-terminal amino groups and epsilon amino groups of lysine residues, on the one hand, and with free thiol groups of cysteine residues, on the other. \\n     Kits are available commercially that permit conjugation of proteins to a variety of amine-reactive or thiol-reactive fluorophores: Molecular Probes, Inc. (Eugene, Oreg., USA), e.g., offers kits for conjugating proteins to Alexa Fluor 350, Alexa Fluor 430, Fluorescein-EX, Alexa Fluor 488, Oregon Green 488, Alexa Fluor 532, Alexa Fluor 546, Alexa Fluor 546, Alexa Fluor 568, Alexa Fluor 594, and Texas Red-X. \\n     A wide variety of other amine-reactive and thiol-reactive fluorophores are available commercially (Molecular Probes, Inc., Eugene, Oreg., USA), including Alexa Fluor® 350, Alexa Fluor® 488, Alexa Fluor® 532, Alexa Fluor® 546, Alexa Fluor® 568, Alexa Fluor® 594, Alexa Fluor® 647 (monoclonal antibody labeling kits available from Molecular Probes, Inc., Eugene, Oreg., USA), BODIPY dyes, such as BODIPY 493/503, BODIPY FL, BODIPY R6G, BODIPY 530/550, BODIPY TMR, BODIPY 558/568, BODIPY 558/568, BODIPY 564/570, BODIPY 576/589, BODIPY 581/591, BODIPY TR, BODIPY 630/650, BODIPY 650/665, Cascade Blue, Cascade Yellow, Dansyl, lissamine rhodamine B, Marina Blue, Oregon Green 488, Oregon Green 514, Pacific Blue, rhodamine 6G, rhodamine green, rhodamine red, tetramethylrhodamine, Texas Red (available from Molecular Probes, Inc., Eugene, Oreg., USA). \\n     The polypeptides of the present invention can also be conjugated to fluorophores, other proteins, and other macromolecules, using bifunctional linking reagents. Common homobifunctional reagents include, e.g., APG, AEDP, BASED, BMB, BMDB, BMH, BMOE, BM[PEO]3, BM[PEO]4, BS3, BSOCOES, DFDNB, DMA, DMP, DMS, DPDPB, DSG, DSP (Lomant&#39;s Reagent), DSS, DST, DTBP, DTME, DTSSP, EGS, HBVS, Sulfo-BSOCOES, Sulfo-DST, Sulfo-EGS (all available from Pierce, Rockford, Ill., USA); common heterobifunctional cross-linkers include ABH, AMAS, ANB-NOS, APDP, ASBA, BMPA, BMPH, BMPS, EDC, EMCA, EMCH, EMCS, KMUA, KMUH, GMBS, LC-SMCC, LC-SPDP, MBS, M2C2H, MPBH, MSA, NHS-ASA, PDPH, PMPI, SADP, SAED, SAND, SANPAH, SASD, SATP, SBAP, SFAD, SIA, SIAB, SMCC, SMPB, SMPH, SMPT, SPDP, Sulfo-EMCS, Sulfo-GMBS, Sulfo-HSAB, Sulfo-KMUS, Sulfo-LC-SPDP, Sulfo-MBS, Sulfo-NHS-LC-ASA, Sulfo-SADP, Sulfo-SANPAH, Sulfo-SIAB, Sulfo-SMCC, Sulfo-SMPB, Sulfo-LC-SMPT, SVSB, TFCS (all available Pierce, Rockford, Ill., USA). \\n     Polypeptides of the present invention, including full length polypeptides, fragments and fusion proteins, can be conjugated, using such cross-linking reagents, to fluorophores that are not amine- or thiol-reactive. Other labels that usefully can be conjugated to polypeptides of the present invention include radioactive labels, echosonographic contrast reagents, and MRI contrast agents. \\n     Polypeptides of the present invention, including full length polypeptides, fragments and fusion proteins, can also usefully be conjugated using cross-linking agents to carrier proteins, such as KLH, bovine thyroglobulin, and even bovine serum albumin (BSA), to increase immunogenicity for raising anti-OSP antibodies. \\n     Polypeptides of the present invention, including full length polypeptides, fragments and fusion proteins, can also usefully be conjugated to polyethylene glycol (PEG); PEGylation increases the serum half life of proteins administered intravenously for replacement therapy. Delgado et al.,  Crit. Rev. Ther. Drug Carrier Syst.  9(3-4): 249-304 (1992); Scott et al.,  Curr. Pharm. Des.  4(6): 423-38 (1998); DeSantis et al.,  Curr. Opin. Biotechnol.  10(4): 324-30 (1999). PEG monomers can be attached to the protein directly or through a linker, with PEGylation using PEG monomers activated with tresyl chloride (2,2,2-trifluoroethanesulphonyl chloride) permitting direct attachment under mild conditions. \\n     Polypeptides of the present invention are also inclusive of analogs of a polypeptide encoded by a nucleic acid molecule according to the instant invention. In a preferred embodiment, this polypeptide is an OSP. In a more preferred embodiment, this polypeptide is derived from a polypeptide having part or all of the amino acid sequence of SEQ ID NO: 129-295. Also preferred is an analog polypeptide comprising one or more substitutions of non-natural amino acids or non-native inter-residue bonds compared to the naturally occurring polypeptide. In one embodiment, the analog is structurally similar to an OSP, but one or more peptide linkages is replaced by a linkage selected from the group consisting of —CH 2 NH—, —CH 2 S—, —CH 2 —CH 2 —, —CH═CH—(cis and trans), —COCH 2 —, —CH(OH)CH 2 — and —CH 2 SO—. In another embodiment, the analog comprises substitution of one or more amino acids of an OSP with a D-amino acid of the same type or other non-natural amino acid in order to generate more stable peptides. D-amino acids can readily be incorporated during chemical peptide synthesis: peptides assembled from D-amino acids are more resistant to proteolytic attack; incorporation of D-amino acids can also be used to confer specific three-dimensional conformations on the peptide. Other amino acid analogues commonly added during chemical synthesis include ornithine, norleucine, phosphorylated amino acids (typically phosphoserine, phosphothreonine, phosphotyrosine), L-malonyltyrosine, a non-hydrolyzable analog of phosphotyrosine (see, e.g., Kole et al.,  Biochem. Biophys. Res. Com.  209: 817-821 (1995)), and various halogenated phenylalanine derivatives. \\n     Non-natural amino acids can be incorporated during solid phase chemical synthesis or by recombinant techniques, although the former is typically more common. Solid phase chemical synthesis of peptides is well established in the art. Procedures are described, inter alia, in Chan et al. (eds.),  Fmoc Solid Phase Peptide Synthesis: A Practical Approach  (Practical Approach Series), Oxford Univ. Press (March 2000); Jones,  Amino Acid and Peptide Synthesis  (Oxford Chemistry Primers, No 7), Oxford Univ. Press (1992); and Bodanszky,  Principles of Peptide Synthesis  (Springer Laboratory), Springer Verlag (1993). \\n     Amino acid analogues having detectable labels are also usefully incorporated during synthesis to provide derivatives and analogs. Biotin, for example can be added using biotinoyl-(9-fluorenylmethoxycarbonyl)-L-lysine (FMOC biocytin) (Molecular Probes, Eugene, Oreg., USA). Biotin can also be added enzymatically by incorporation into a fusion protein of an  E. coli  BirA substrate peptide. The FMOC and tBOC derivatives of dabcyl-L-lysine (Molecular Probes, Inc., Eugene, Oreg., USA) can be used to incorporate the dabcyl chromophore at selected sites in the peptide sequence during synthesis. The aminonaphthalene derivative EDANS, the most common fluorophore for pairing with the dabcyl quencher in fluorescence resonance energy transfer (FRET) systems, can be introduced during automated synthesis of peptides by using EDANS-FMOC-L-glutamic acid or the corresponding tBOC derivative (both from Molecular Probes, Inc., Eugene, Oreg., USA). Tetramethylrhodamine fluorophores can be incorporated during automated FMOC synthesis of peptides using (FMOC)-TMR-L-lysine (Molecular Probes, Inc. Eugene, Oreg., USA). \\n     Other useful amino acid analogues that can be incorporated during chemical synthesis include aspartic acid, glutamic acid, lysine, and tyrosine analogues having allyl side-chain protection (Applied Biosystems, Inc., Foster City, Calif., USA); the allyl side chain permits synthesis of cyclic, branched-chain, sulfonated, glycosylated, and phosphorylated peptides. \\n     A large number of other FMOC-protected non-natural amino acid analogues capable of incorporation during chemical synthesis are available commercially, including, e.g., Fmoc-2-aminobicyclo[2.2.1]heptane-2-carboxylic acid, Fmoc-3-endo-aminobicyclo[2.2.1]heptane-2-endo-carboxylic acid, Fmoc-3-exo-aminobicyclo[2.2.1]heptane-2-exo-carboxylic acid, Fmoc-3-endo-amino-bicyclo[2.2.1]hept-5-ene-2-endo-carboxylic acid, Fmoc-3-exo-amino-bicyclo[2.2.1]hept-5-ene-2-exo-carboxylic acid, Fmoc-cis-2-amino-1-cyclohexanecarboxylic acid, Fmoc-trans-2-amino-1-cyclohexanecarboxylic acid, Fmoc-1-amino-1-cyclopentanecarboxylic acid, Fmoc-cis-2-amino-1-cyclopentanecarboxylic acid, Fmoc-1-amino-1-cyclopropanecarboxylic acid, Fmoc-D-2-amino-4-(ethylthio)butyric acid, Fmoc-L-2-amino-4-(ethylthio)butyric acid, Fmoc-L-buthionine, Fmoc-5-methyl-L-Cysteine, Fmoc-2-aminobenzoic acid (anthranillic acid), Fmoc-3-aminobenzoic acid, Fmoc-4-aminobenzoic acid, Fmoc-2-aminobenzophenone-2′-carboxylic acid, Fmoc-N-(4-aminobenzoyl)-β-alanine, Fmoc-2-amino-4,5-dimethoxybenzoic acid, Fmoc-4-aminohippuric acid, Fmoc-2-amino-3-hydroxybenzoic acid, Fmoc-2-amino-5-hydroxybenzoic acid, Fmoc-3-amino-4-hydroxybenzoic acid, Fmoc-4-amino-3-hydroxybenzoic acid, Fmoc-4-amino-2-hydroxybenzoic acid, Fmoc-5-amino-2-hydroxybenzoic acid, Fmoc-2-amino-3-methoxybenzoic acid, Fmoc-4-amino-3-methoxybenzoic acid, Fmoc-2-amino-3-methylbenzoic acid, Fmoc-2-amino-5-methylbenzoic acid, Fmoc-2-amino-6-methylbenzoic acid, Fmoc-3-amino-2-methylbenzoic acid, Fmoc-3-amino-4-methylbenzoic acid, Fmoc-4-amino-3-methylbenzoic acid, Fmoc-3-amino-2-naphthoic acid, Fmoc-D,L-3-amino-3-phenylpropionic acid, Fmoc-L-Methyldopa, Fmoc-2-amino-4,6-dimethyl-3-pyridinecarboxylic acid, Fmoc-D,L-amino-2-thiophenacetic acid, Fmoc-4-(carboxymethyl)piperazine, Fmoc-4-carboxypiperazine, Fmoc-4-(carboxymethyl)homopiperazine, Fmoc-4-phenyl-4-piperidinecarboxylic acid, Fmoc-L-1,2,3,4-tetrahydronorharman-3-carboxylic acid, Fmoc-L-thiazolidine-4-carboxylic acid, all available from The Peptide Laboratory (Richmond, Calif., USA). \\n     Non-natural residues can also be added biosynthetically by engineering a suppressor tRNA, typically one that recognizes the UAG stop codon, by chemical aminoacylation with the desired unnatural amino acid. Conventional site-directed mutagenesis is used to introduce the chosen stop codon UAG at the site of interest in the protein gene. When the acylated suppressor tRNA and the mutant gene are combined in an in vitro transcription/translation system, the unnatural amino acid is incorporated in response to the UAG codon to give a protein containing that amino acid at the specified position. Liu et al.,  Proc. Natl. Acad. Sci. USA  96(9): 4780-5 (1999); Wang et al.,  Science  292(5516): 498-500 (2001). \\n     Fusion Proteins \\n     Another aspect of the present invention relates to the fusion of a polypeptide of the present invention to heterologous polypeptides. In a preferred embodiment, the polypeptide of the present invention is an OSP. In a more preferred embodiment, the polypeptide of the present invention that is fused to a heterologous polypeptide which comprises part or all of the amino acid sequence of SEQ ID NO: 129-295, or is a mutein, homologous polypeptide, analog or derivative thereof. In an even more preferred embodiment, the fusion protein is encoded by a nucleic acid molecule comprising all or part of the nucleic acid sequence of SEQ ID NO: 1-128, or comprises all or part of a nucleic acid sequence that selectively hybridizes or is homologous to a nucleic acid molecule comprising a nucleic acid sequence of SEQ ID NO: 1-128. \\n     The fusion proteins of the present invention will include at least one fragment of a polypeptide of the present invention, which fragment is at least 6, typically at least 8, often at least 15, and usefully at least 16, 17, 18, 19, or 20 amino acids long. The fragment of the polypeptide of the present to be included in the fusion can usefully be at least 25 amino acids long, at least 50 amino acids long, and can be at least 75, 100, or even 150 amino acids long. Fusions that include the entirety of a polypeptide of the present invention have particular utility. \\n     The heterologous polypeptide included within the fusion protein of the present invention is at least 6 amino acids in length, often at least 8 amino acids in length, and preferably at least 15, 20, or 25 amino acids in length. Fusions that include larger polypeptides, such as the IgG Fc region, and even entire proteins (such as GFP chromophore-containing proteins) are particularly useful. \\n     As described above in the description of vectors and expression vectors of the present invention, which discussion is incorporated here by reference in its entirety, heterologous polypeptides to be included in the fusion proteins of the present invention can usefully include those designed to facilitate purification and/or visualization of recombinantly-expressed proteins. See, e.g., Ausubel, Chapter 16, (1992), supra. Although purification tags can also be incorporated into fusions that are chemically synthesized, chemical synthesis typically provides sufficient purity that further purification by HPLC suffices; however, visualization tags as above described retain their utility even when the protein is produced by chemical synthesis, and when so included render the fusion proteins of the present invention useful as directly detectable markers of the presence of a polypeptide of the invention. \\n     As also discussed above, heterologous polypeptides to be included in the fusion proteins of the present invention can usefully include those that facilitate secretion of recombinantly expressed proteins into the periplasmic space or extracellular milieu for prokaryotic hosts or into the culture medium for eukaryotic cells through incorporation of secretion signals and/or leader sequences. For example, a His 6  tagged protein can be purified on a Ni affinity column and a GST fusion protein can be purified on a glutathione affinity column. Similarly, a fusion protein comprising the Fc domain of IgG can be purified on a Protein A or Protein G column and a fusion protein comprising an epitope tag such as myc can be purified using an immunoaffinity column containing an anti-c-myc antibody. It is preferable that the epitope tag be separated from the protein encoded by the essential gene by an enzymatic cleavage site that can be cleaved after purification. See also the discussion of nucleic acid molecules encoding fusion proteins that may be expressed on the surface of a cell. \\n     Other useful fusion proteins of the present invention include those that permit use of the polypeptide of the present invention as bait in a yeast two-hybrid system. See Bartel et al. (eds.),  The Yeast Two - Hybrid System , Oxford University Press (1997); Zhu et al.,  Yeast Hybrid Technologies , Eaton Publishing (2000); Fields et al.,  Trends Genet.  10(8): 286-92 (1994); Mendelsohn et al.,  Curr. Opin. Biotechnol.  5(5): 482-6 (1994); Luban et al.,  Curr. Opin. Biotechnol.  6(1): 59-64 (1995); Allen et al.,  Trends Biochem. Sci.  20(12): 511-6 (1995); Drees,  Curr. Opin. Chem. Biol.  3(1): 64-70 (1999); Topcu et al.,  Pharm. Res.  17(9): 1049-55 (2000); Fashena et al.,  Gene  250(1-2): 1-14 (2000); Colas et al.,  Nature  380, 548-550 (1996); Norman, T. et al.,  Science  285, 591-595 (1999); Fabbrizio et al.,  Oncogene  18, 4357-4363 (1999); Xu et al.,  Proc Natl Acad Sci USA.  94, 12473-12478 (1997); Yang, et al.,  Nuc. Acids Res.  23, 1152-1156 (1995); Kolonin et al.,  Proc Natl Acad Sci USA  95, 14266-14271 (1998); Cohen et al.,  Proc Natl Acad Sci USA  95, 14272-14277 (1998); Uetz, et al.  Nature  403, 623-627 (2000); Ito, et al.,  Proc Natl Acad Sci USA  98, 4569-4574 (2001). Typically, such fusion is to either  E. coli  LexA or yeast GAL4 DNA binding domains. Related bait plasmids are available that express the bait fused to a nuclear localization signal. \\n     Other useful fusion proteins include those that permit display of the encoded polypeptide on the surface of a phage or cell, fusions to intrinsically fluorescent proteins, such as green fluorescent protein (GFP), and fusions to the IgG Fc region, as described above. \\n     The polypeptides of the present invention can also usefully be fused to protein toxins, such as  Pseudomonas  exotoxin A, diphtheria toxin, shiga toxin A, anthrax toxin lethal factor, or ricin, in order to effect ablation of cells that bind or take up the proteins of the present invention. \\n     Fusion partners include, inter alia, myc, hemagglutinin (HA), GST, immunoglobulins, β-galactosidase, biotin trpE, protein A, β-lactamase, α-amylase, maltose binding protein, alcohol dehydrogenase, polyhistidine (for example, six histidine at the amino and/or carboxyl terminus of the polypeptide), lacZ, green fluorescent protein (GFP), yeast a mating factor, GAL4 transcription activation or DNA binding domain, luciferase, and serum proteins such as ovalbumin, albumin and the constant domain of IgG. See, e.g., Ausubel (1992), supra and Ausubel (1999), supra. Fusion proteins may also contain sites for specific enzymatic cleavage, such as a site that is recognized by enzymes such as Factor XIII, trypsin, pepsin, or any other enzyme known in the art. Fusion proteins will typically be made by either recombinant nucleic acid methods, as described above, chemically synthesized using techniques well known in the art (e.g., a Merrifield synthesis), or produced by chemical cross-linking. \\n     Another advantage of fusion proteins is that the epitope tag can be used to bind the fusion protein to a plate or column through an affinity linkage for screening binding proteins or other molecules that bind to the OSP. \\n     As further described below, the polypeptides of the present invention can readily be used as specific immunogens to raise antibodies that specifically recognize polypeptides of the present invention including OSPs and their allelic variants and homologues. The antibodies, in turn, can be used, inter alia, specifically to assay for the polypeptides of the present invention, particularly OSPs, e.g. by ELISA for detection of protein fluid samples, such as serum, by immunohistochemistry or laser scanning cytometry, for detection of protein in tissue samples, or by flow cytometry, for detection of intracellular protein in cell suspensions, for specific antibody-mediated isolation and/or purification of OSPs, as for example by immunoprecipitation, and for use as specific agonists or antagonists of OSPs. \\n     One may determine whether polypeptides of the present invention including OSPs, muteins, homologous proteins or allelic variants or fusion proteins of the present invention are functional by methods known in the art. For instance, residues that are tolerant of change while retaining function can be identified by altering the polypeptide at known residues using methods known in the art, such as alanine scanning mutagenesis, Cunningham et al.,  Science  244(4908): 1081-5 (1989); transposon linker scanning mutagenesis, Chen et al.,  Gene  263(1-2): 39-48 (2001); combinations of homolog- and alanine-scanning mutagenesis, Jin et al.,  J. Mol. Biol.  226(3): 851-65 (1992); and combinatorial alanine scanning, Weiss et al.,  Proc. Natl. Acad. Sci. USA  97(16): 8950-4 (2000), followed by functional assay. Transposon linker scanning kits are available commercially (New England Biolabs, Beverly, Mass., USA, catalog. no. E7-102S; EZ::TN™ In-Frame Linker Insertion Kit, catalogue no. EZI04KN, (Epicentre Technologies Corporation, Madison, Wis., USA). \\n     Purification of the polypeptides or fusion proteins of the present invention is well known and within the skill of one having ordinary skill in the art. See, e.g., Scopes,  Protein Purification,  2d ed. (1987). Purification of recombinantly expressed polypeptides is described above. Purification of chemically-synthesized peptides can readily be effected, e.g., by HPLC. \\n     Accordingly, it is an aspect of the present invention to provide the isolated polypeptides or fusion proteins of the present invention in pure or substantially pure form in the presence or absence of a stabilizing agent. Stabilizing agents include both proteinaceous and non-proteinaceous material and are well known in the art. Stabilizing agents, such as albumin and polyethylene glycol (PEG) are known and are commercially available. \\n     Although high levels of purity are preferred when the isolated polypeptide or fusion protein of the present invention are used as therapeutic agents, such as in vaccines and replacement therapy, the isolated polypeptides of the present invention are also useful at lower purity. For example, partially purified polypeptides of the present invention can be used as immunogens to raise antibodies in laboratory animals. \\n     In a preferred embodiment, the purified and substantially purified polypeptides of the present invention are in compositions that lack detectable ampholytes, acrylamide monomers, bis-acrylamide monomers, and polyacrylamide. \\n     The polypeptides or fusion proteins of the present invention can usefully be attached to a substrate. The substrate can be porous or solid, planar or non-planar; the bond can be covalent or noncovalent. For example, the peptides of the invention may be stabilized by covalent linkage to albumin. See, U.S. Pat. No. 5,876,969, the contents of which are hereby incorporated in its entirety. \\n     The polypeptides or fusion proteins of the present invention can also be usefully bound to a porous substrate, commonly a membrane, typically comprising nitrocellulose, polyvinylidene fluoride (PVDF), or cationically derivatized, hydrophilic PVDF; so bound, the polypeptides or fusion proteins of the present invention can be used to detect and quantify antibodies, e.g. in serum, that bind specifically to the immobilized polypeptide or fusion protein of the present invention. \\n     As another example, the polypeptides or fusion proteins of the present invention can usefully be bound to a substantially nonporous substrate, such as plastic, to detect and quantify antibodies, e.g. in serum, that bind specifically to the immobilized protein of the present invention. Such plastics include polymethylacrylic, polyethylene, polypropylene, polyacrylate, polymethylmethacrylate, polyvinylchloride, polytetrafluoroethylene, polystyrene, polycarbonate, polyacetal, polysulfone, celluloseacetate, cellulosenitrate, nitrocellulose, or mixtures thereof; when the assay is performed in a standard microtiter dish, the plastic is typically polystyrene. \\n     The polypeptides and fusion proteins of the present invention can also be attached to a substrate suitable for use as a surface enhanced laser desorption ionization source; so attached, the polypeptide or fusion protein of the present invention is useful for binding and then detecting secondary proteins that bind with sufficient affinity or avidity to the surface-bound polypeptide or fusion protein to indicate biologic interaction there between. The polypeptides or fusion proteins of the present invention can also be attached to a substrate suitable for use in surface plasmon resonance detection; so attached, the polypeptide or fusion protein of the present invention is useful for binding and then detecting secondary proteins that bind with sufficient affinity or avidity to the surface-bound polypeptide or fusion protein to indicate biological interaction there between. \\n     Alternative Transcripts \\n     In another aspect, the present invention provides splice variants of genes and proteins encoded thereby. The identification of a novel splice variant which encodes an amino acid sequence with a novel region can be targeted for the generation of reagents for use in detection and/or treatment of cancer. The novel amino acid sequence may lead to a unique protein structure, protein subcellular localization, biochemical processing or function of the splice variant. This information can be used to directly or indirectly facilitate the generation of additional or novel therapeutics or diagnostics. The nucleotide sequence in this novel splice variant can be used as a nucleic acid probe for the diagnosis and/or treatment of cancer. \\n     Specifically, the newly identified sequences may enable the production of new antibodies or compounds directed against the novel region for use as a therapeutic or diagnostic. Alternatively, the newly identified sequences may alter the biochemical or biological properties of the encoded protein in such a way as to enable the generation of improved or different therapeutics targeting this protein. \\n     Antibodies \\n     In another aspect, the invention provides antibodies, including fragments and derivatives thereof, that bind specifically to polypeptides encoded by the nucleic acid molecules of the invention. In a preferred embodiment, the antibodies are specific for a polypeptide that is an OSP, or a fragment, mutein, derivative, analog or fusion protein thereof. In a more preferred embodiment, the antibodies are specific for a polypeptide that comprises SEQ ID NO: 129-295, or a fragment, mutein, derivative, analog or fusion protein thereof. \\n     The antibodies of the present invention can be specific for linear epitopes, discontinuous epitopes, or conformational epitopes of such proteins or protein fragments, either as present on the protein in its native conformation or, in some cases, as present on the proteins as denatured, as, e.g., by solubilization in SDS. New epitopes may also be due to a difference in post translational modifications (PTMs) in disease versus normal tissue. For example, a particular site on an OSP may be glycosylated in cancerous cells, but not glycosylated in normal cells or vice versa. In addition, alternative splice forms of an OSP may be indicative of cancer. Differential degradation of the C or N-terminus of an OSP may also be a marker or target for anticancer therapy. For example, an OSP may be N-terminal degraded in cancer cells exposing new epitopes to antibodies which may selectively bind for diagnostic or therapeutic uses. \\n     As is well known in the art, the degree to which an antibody can discriminate among molecular species in a mixture will depend, in part, upon the conformational relatedness of the species in the mixture; typically, the antibodies of the present invention will discriminate over adventitious binding to non-OSP polypeptides by at least two-fold, more typically by at least 5-fold, typically by more than 10-fold, 25-fold, 50-fold, 75-fold, and often by more than 100-fold, and on occasion by more than 500-fold or 1000-fold. When used to detect the proteins or protein fragments of the present invention, the antibody of the present invention is sufficiently specific when it can be used to determine the presence of the polypeptide of the present invention in samples derived from human ovarian. \\n     Typically, the affinity or avidity of an antibody (or antibody multimer, as in the case of an IgM pentamer) of the present invention for a protein or protein fragment of the present invention will be at least about 1×10 −6  molar (M), typically at least about 5×10 −7  M, 1×10 −7  M, with affinities and avidities of at least 1×10 −8  M, 5×10 −9  M, 1×10 −10  M and up to 1×10 −13  M proving especially useful. \\n     The antibodies of the present invention can be naturally occurring forms, such as IgG, IgM, IgD, IgE, IgY, and IgA, from any avian, reptilian, or mammalian species. \\n     Human antibodies can, but will infrequently, be drawn directly from human donors or human cells. In such case, antibodies to the polypeptides of the present invention will typically have resulted from fortuitous immunization, such as autoimmune immunization, with the polypeptide of the present invention. Such antibodies will typically, but will not invariably, be polyclonal. In addition, individual polyclonal antibodies may be isolated and cloned to generate monoclonals. \\n     Human antibodies are more frequently obtained using transgenic animals that express human immunoglobulin genes, which transgenic animals can be affirmatively immunized with the protein immunogen of the present invention. Human Ig-transgenic mice capable of producing human antibodies and methods of producing human antibodies therefrom upon specific immunization are described, inter alia, in U.S. Pat. Nos. 6,162,963; 6,150,584; 6,114,598; 6,075,181; 5,939,598; 5,877,397; 5,874,299; 5,814,318; 5,789,650; 5,770,429; 5,661,016; 5,633,425; 5,625,126; 5,569,825; 5,545,807; 5,545,806, and 5,591,669, the disclosures of which are incorporated herein by reference in their entireties. Such antibodies are typically monoclonal, and are typically produced using techniques developed for production of murine antibodies. \\n     Human antibodies are particularly useful, and often preferred, when the antibodies of the present invention are to be administered to human beings as in vivo diagnostic or therapeutic agents, since recipient immune response to the administered antibody will often be substantially less than that occasioned by administration of an antibody derived from another species, such as mouse. \\n     IgG, IgM, IgD, IgE, IgY, and IgA antibodies of the present invention are also usefully obtained from other species, including mammals such as rodents (typically mouse, but also rat, guinea pig, and hamster), lagomorphs (typically rabbits), and also larger mammals, such as sheep, goats, cows, and horses; or egg laying birds or reptiles such as chickens or alligators. In such cases, as with the transgenic human-antibody-producing non-human mammals, fortuitous immunization is not required, and the non-human mammal is typically affirmatively immunized, according to standard immunization protocols, with the polypeptide of the present invention. One form of avian antibodies may be generated using techniques described in WO 00/29444, published 25 May 2000, which is herein incorporated by reference in its entirety. \\n     As discussed above, virtually all fragments of 8 or more contiguous amino acids of a polypeptide of the present invention can be used effectively as immunogens when conjugated to a carrier, typically a protein such as bovine thyroglobulin, keyhole limpet hemocyanin, or bovine serum albumin, conveniently using a bifunctional linker such as those described elsewhere above, which discussion is incorporated by reference here. \\n     Immunogenicity can also be conferred by fusion of the polypeptide of the present invention to other moieties. For example, polypeptides of the present invention can be produced by solid phase synthesis on a branched polylysine core matrix; these multiple antigenic peptides (MAPs) provide high purity, increased avidity, accurate chemical definition and improved safety in vaccine development. Tam et al.,  Proc. Natl. Acad. Sci. USA  85: 5409-5413 (1988); Posnett et al.,  J. Biol. Chem.  263: 1719-1725 (1988). \\n     Protocols for immunizing non-human mammals or avian species are well-established in the art. See Harlow et al. (eds.),  Using Antibodies: A Laboratory Manual , Cold Spring Harbor Laboratory (1998); Coligan et al. (eds.),  Current Protocols in Immunology , John Wiley &amp; Sons, Inc. (2001); Zola,  Monoclonal Antibodies: Preparation and Use of Monoclonal Antibodies and Engineered Antibody Derivatives  ( Basics: From Background to Bench ), Springer Verlag (2000); Gross M, Speck  J. Dtsch. Tierarztl. Wochenschr.  103: 417-422 (1996). Immunization protocols often include multiple immunizations, either with or without adjuvants such as Freund&#39;s complete adjuvant and Freund&#39;s incomplete adjuvant, and may include naked DNA immunization. Moss,  Semin. Immunol.  2: 317-327 (1990). \\n     Antibodies from non-human mammals and avian species can be polyclonal or monoclonal, with polyclonal antibodies having certain advantages in immunohistochemical detection of the polypeptides of the present invention and monoclonal antibodies having advantages in identifying and distinguishing particular epitopes of the polypeptides of the present invention. Antibodies from avian species may have particular advantage in detection of the polypeptides of the present invention, in human serum or tissues. Vikinge et al.,  Biosens. Bioelectron.  13: 1257-1262 (1998). Following immunization, the antibodies of the present invention can be obtained using any art-accepted technique. Such techniques are well known in the art and are described in detail in references such as Coligan, supra; Zola, supra; Howard et al. (eds.),  Basic Methods in Antibody Production and Characterization , CRC Press (2000); Harlow, supra; Davis (ed.),  Monoclonal Antibody Protocols , Vol. 45, Humana Press (1995); Delves (ed.),  Antibody Production Essential Techniques , John Wiley &amp; Son Ltd (1997); and Kenney,  Antibody Solution An Antibody Methods Manual , Chapman &amp; Hall (1997). \\n     Briefly, such techniques include, inter alia, production of monoclonal antibodies by hybridomas and expression of antibodies or fragments or derivatives thereof from host cells engineered to express immunoglobulin genes or fragments thereof. These two methods of production are not mutually exclusive: genes encoding antibodies specific for the polypeptides of the present invention can be cloned from hybridomas and thereafter expressed in other host cells. Nor need the two necessarily be performed together: e.g., genes encoding antibodies specific for the polypeptides of the present invention can be cloned directly from B cells known to be specific for the desired protein, as further described in U.S. Pat. No. 5,627,052, the disclosure of which is incorporated herein by reference in its entirety, or from antibody-displaying phage. \\n     Recombinant expression in host cells is particularly useful when fragments or derivatives of the antibodies of the present invention are desired. \\n     Host cells for recombinant antibody production of whole antibodies, antibody fragments, or antibody derivatives can be prokaryotic or eukaryotic. \\n     Prokaryotic hosts are particularly useful for producing phage displayed antibodies of the present invention. \\n     The technology of phage-displayed antibodies, in which antibody variable region fragments are fused, for example, to the gene III protein (pIII) or gene VIII protein (pVIII) for display on the surface of filamentous phage, such as M13, is by now well-established. See, e.g., Sidhu,  Curr. Opin. Biotechnol.  11(6): 610-6 (2000); Griffiths et al.,  Curr. Opin. Biotechnol.  9(1): 102-8 (1998); Hoogenboom et al.,  Immunotechnology,  4(1): 1-20 (1998); Rader et al.,  Current Opinion in Biotechnology  8: 503-508 (1997); Aujame et al.,  Human Antibodies  8: 155-168 (1997); Hoogenboom,  Trends in Biotechnol.  15: 62-70 (1997); de Kruif et al., 17: 453-455 (1996); Barbas et al.,  Trends in Biotechnol.  14: 230-234 (1996); Winter et al.,  Ann. Rev. Immunol.  433-455 (1994). Techniques and protocols required to generate, propagate, screen (pan), and use the antibody fragments from such libraries have recently been compiled. See, e.g., Barbas (2001), supra; Kay, supra; and Abelson, supra. \\n     Typically, phage-displayed antibody fragments are scFv fragments or Fab fragments; when desired, full length antibodies can be produced by cloning the variable regions from the displaying phage into a complete antibody and expressing the full length antibody in a further prokaryotic or a eukaryotic host cell. Eukaryotic cells are also useful for expression of the antibodies, antibody fragments, and antibody derivatives of the present invention. For example, antibody fragments of the present invention can be produced in  Pichia pastoris  and in  Saccharomyces cerevisiae . See, e.g., Takahashi et al.,  Biosci. Biotechnol. Biochem.  64(10): 2138-44 (2000); Freyre et al., J. Biotechnol. 76(2-3):1 57-63 (2000); Fischer et al.,  Biotechnol. Appl. Biochem.  30 (Pt 2): 117-20 (1999); Pennell et al.,  Res. Immunol.  149(6): 599-603 (1998); Eldin et al.,  J. Immunol. Methods.  201(1): 67-75 (1997); Frenken et al.,  Res. Immunol.  149(6): 589-99 (1998); and Shusta et al.,  Nature Biotechnol.  16(8): 773-7 (1998). \\n     Antibodies, including antibody fragments and derivatives, of the present invention can also be produced in insect cells. See, e.g., Li et al.,  Protein Expr. Purif.  21(1): 121-8 (2001); Ailor et al.,  Biotechnol. Bioeng.  58(2-3): 196-203 (1998); Hsu et al.,  Biotechnol. Prog.  13(1): 96-104 (1997); Edelman et al.,  Immunology  91(1): 13-9 (1997); and Nesbit et al.,  J. Immunol. Methods  151(1-2): 201-8 (1992). \\n     Antibodies and fragments and derivatives thereof of the present invention can also be produced in plant cells, particularly maize or tobacco, Giddings et al.,  Nature Biotechnol.  18(11): 1151-5 (2000); Gavilondo et al.,  Biotechniques  29(1): 128-38 (2000); Fischer et al.,  J. Biol. Regul. Homeost. Agents  14(2): 83-92 (2000); Fischer et al.,  Biotechnol. Appl. Biochem.  30 (Pt 2): 113-6 (1999); Fischer et al.,  Biol. Chem.  380(7-8): 825-39 (1999); Russell,  Curr. Top. Microbiol. Immunol.  240: 119-38 (1999); and Ma et al.,  Plant Physiol.  109(2): 341-6 (1995). \\n     Antibodies, including antibody fragments and derivatives, of the present invention can also be produced in transgenic, non-human, mammalian milk. See, e.g. Pollock et al.,  J. Immunol Methods.  231: 147-57 (1999); Young et al.,  Res. Immunol.  149: 609-10 (1998); and Limonta et al.,  Immunotechnology  1: 107-13 (1995). \\n     Mammalian cells useful for recombinant expression of antibodies, antibody fragments, and antibody derivatives of the present invention include CHO cells, COS cells, 293 cells, and myeloma cells. Verma et al.,  J. Immunol. Methods  216(1-2):165-81 (1998) review and compare bacterial, yeast, insect and mammalian expression systems for expression of antibodies. Antibodies of the present invention can also be prepared by cell free translation, as further described in Merk et al.,  J. Biochem . (Tokyo) 125(2): 328-33 (1999) and Ryabova et al.,  Nature Biotechnol.  15(1): 79-84 (1997), and in the milk of transgenic animals, as further described in Pollock et al.,  J. Immunol. Methods  231(1-2): 147-57 (1999). \\n     The invention further provides antibody fragments that bind specifically to one or more of the polypeptides of the present invention or to one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention, or the binding of which can be competitively inhibited by one or more of the polypeptides of the present invention or one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention. Among such useful fragments are Fab, Fab′, Fv, F(ab)′ 2 , and single-chain Fv (scFv) fragments. Other useful fragments are described in Hudson,  Curr. Opin. Biotechnol.  9(4): 395-402 (1998). \\n     The present invention also relates to antibody derivatives that bind specifically to one or more of the polypeptides of the present invention, to one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention, or the binding of which can be competitively inhibited by one or more of the polypeptides of the present invention or one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention. \\n     Among such useful derivatives are chimeric, primatized, and humanized antibodies; such derivatives are less immunogenic in human beings, and thus are more suitable for in vivo administration, than are unmodified antibodies from non-human mammalian species. Another useful method is PEGylation to increase the serum half life of the antibodies. \\n     Chimeric antibodies typically include heavy and/or light chain variable regions (including both CDR and framework residues) of immunoglobulins of one species, typically mouse, fused to constant regions of another species, typically human. See, e.g., Morrison et al.,  Proc. Natl. Acad. Sci. USA.  81(21): 6851-5 (1984); Sharon et al.,  Nature  309(5966): 364-7 (1984); Takeda et al.,  Nature  314(6010): 452-4 (1985); and U.S. Pat. No. 5,807,715 the disclosure of which is incorporated herein by reference in its entirety. Primatized and humanized antibodies typically include heavy and/or light chain CDRs from a murine antibody grafted into a non-human primate or human antibody V region framework, usually further comprising a human constant region, Riechmann et al.,  Nature  332(6162): 323-7 (1988); Co et al.,  Nature  351(6326): 501-2 (1991); and U.S. Pat. Nos. 6,054,297; 5,821,337; 5,770,196; 5,766,886; 5,821,123; 5,869,619; 6,180,377; 6,013,256; 5,693,761; and 6,180,370, the disclosures of which are incorporated herein by reference in their entireties. Other useful antibody derivatives of the invention include heteromeric antibody complexes and antibody fusions, such as diabodies (bispecific antibodies), single-chain diabodies, and intrabodies. \\n     It is contemplated that the nucleic acids encoding the antibodies of the present invention can be operably joined to other nucleic acids forming a recombinant vector for cloning or for expression of the antibodies of the invention. Accordingly, the present invention includes any recombinant vector containing the coding sequences, or part thereof, whether for eukaryotic transduction, transfection or gene therapy. Such vectors may be prepared using conventional molecular biology techniques, known to those with skill in the art, and would comprise DNA encoding sequences for the immunoglobulin V-regions including framework and CDRs or parts thereof, and a suitable promoter either with or without a signal sequence for intracellular transport. Such vectors may be transduced or transfected into eukaryotic cells or used for gene therapy (Marasco et al.,  Proc. Natl. Acad. Sci . ( USA ) 90: 7889-7893 (1993); Duan et al.,  Proc. Natl. Acad. Sci . ( USA ) 91: 5075-5079 (1994), by conventional techniques, known to those with skill in the art. \\n     The antibodies of the present invention, including fragments and derivatives thereof, can usefully be labeled. It is, therefore, another aspect of the present invention to provide labeled antibodies that bind specifically to one or more of the polypeptides of the present invention, to one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention, or the binding of which can be competitively inhibited by one or more of the polypeptides of the present invention or one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention. The choice of label depends, in part, upon the desired use. \\n     For example, when the antibodies of the present invention are used for immunohistochemical staining of tissue samples, the label can usefully be an enzyme that catalyzes production and local deposition of a detectable product. Enzymes typically conjugated to antibodies to permit their immunohistochemical visualization are well known, and include alkaline phosphatase, β-galactosidase, glucose oxidase, horseradish peroxidase (HRP), and urease. Typical substrates for production and deposition of visually detectable products include o-nitrophenyl-beta-D-galactopyranoside (ONPG); o-phenylenediamine dihydrochloride (OPD); p-nitrophenyl phosphate (PNPP); p-nitrophenyl-beta-D-galactopryanoside (PNPG); 3′,3′-diaminobenzidine (DAB); 3-amino-9-ethylcarbazole (AEC); 4-chloro-1-naphthol (CN); 5-bromo-4-chloro-3-indolyl-phosphate (BCIP); ABTS®; BluoGal; iodonitrotetrazolium (INT); nitroblue tetrazolium chloride (NBT); phenazine methosulfate (PMS); phenolphthalein monophosphate (PMP); tetramethyl benzidine (TMB); tetranitroblue tetrazolium (TNBT); X-Gal; X-Gluc; and X-Glucoside. \\n     Other substrates can be used to produce products for local deposition that are luminescent. For example, in the presence of hydrogen peroxide (H 2 O 2 ), horseradish peroxidase (HRP) can catalyze the oxidation of cyclic diacylhydrazides, such as luminol. Immediately following the oxidation, the luminol is in an excited state (intermediate reaction product), which decays to the ground state by emitting light. Strong enhancement of the light emission is produced by enhancers, such as phenolic compounds. Advantages include high sensitivity, high resolution, and rapid detection without radioactivity and requiring only small amounts of antibody. See, e.g., Thorpe et al.,  Methods Enzymol.  133: 331-53 (1986); Kricka et al.,  J. Immunoassay  17(1): 67-83 (1996); and Lundqvist et al.,  J. Biolumin. Chemilumin.  10(6): 353-9 (1995). Kits for such enhanced chemiluminescent detection (ECL) are available commercially. The antibodies can also be labeled using colloidal gold. \\n     As another example, when the antibodies of the present invention are used, e.g., for flow cytometric detection, for scanning laser cytometric detection, or for fluorescent immunoassay, they can usefully be labeled with fluorophores. There are a wide variety of fluorophore labels that can usefully be attached to the antibodies of the present invention. For flow cytometric applications, both for extracellular detection and for intracellular detection, common useful fluorophores can be fluorescein isothiocyanate (FITC), allophycocyanin (APC), R-phycoerythrin (PE), peridinin chlorophyll protein (PerCP), Texas Red, Cy3, Cy5, fluorescence resonance energy tandem fluorophores such as PerCP-Cy5.5, PE-Cy5, PE-Cy5.5, PE-Cy7, PE-Texas Red, and APC-Cy7. \\n     Other fluorophores include, inter alia, Alexa Fluor® 350, Alexa Fluor® 488, Alexa Fluor® 532, Alexa Fluor® 546, Alexa Fluor® 568, Alexa Fluor® 594, Alexa Fluor® 647 (monoclonal antibody labeling kits available from Molecular Probes, Inc., Eugene, Oreg., USA), BODIPY dyes, such as BODIPY 493/503, BODIPY FL, BODIPY R6G, BODIPY 530/550, BODIPY TMR, BODIPY 558/568, BODIPY 558/568, BODIPY 564/570, BODIPY 576/589, BODIPY 581/591, BODIPY TR, BODIPY 630/650, BODIPY 650/665, Cascade Blue, Cascade Yellow, Dansyl, lissamine rhodamine B, Marina Blue, Oregon Green 488, Oregon Green 514, Pacific Blue, rhodamine 6G, rhodamine green, rhodamine red, tetramethylrhodamine, Texas Red (available from Molecular Probes, Inc., Eugene, Oreg., USA), and Cy2, Cy3, Cy3.5, Cy5, Cy5.5, Cy7, all of which are also useful for fluorescently labeling the antibodies of the present invention. For secondary detection using labeled avidin, streptavidin, captavidin or neutravidin, the antibodies of the present invention can usefully be labeled with biotin. \\n     When the antibodies of the present invention are used, e.g., for western blotting applications, they can usefully be labeled with radioisotopes, such as  33 P,  32 P,  35 S,  3 H, and  125 I. As another example, when the antibodies of the present invention are used for radioimmunotherapy, the label can usefully be  228 Th,  227 Ac,  225 Ac,  223 Ra,  213 Bi,  212 Pb,  212 Bi,  211 At,  203 Pb,  194 Os,  188 Re,  186 Re,  153 Sm,  149 Tb,  131 I,  125 I,  111 In,  105 Tc,  99m Tc,  97 Ru,  90 Y,  90 Sr,  88 Y,  72 Se,  67 Cu, or  47 Sc. \\n     As another example, when the antibodies of the present invention are to be used for in vivo diagnostic use, they can be rendered detectable by conjugation to MRI contrast agents, such as gadolinium diethylenetriaminepentaacetic acid (DTPA), Lauffer et al.,  Radiology  207(2): 529-38 (1998), or by radioisotopic labeling. \\n     As would be understood, use of the labels described above is not restricted to the application as for which they were mentioned. \\n     The antibodies of the present invention, including fragments and derivatives thereof, can also be conjugated to toxins, in order to target the toxin&#39;s ablative action to cells that display and/or express the polypeptides of the present invention. Commonly, the antibody in such immunotoxins is conjugated to  Pseudomonas  exotoxin A, diphtheria toxin, shiga toxin A, anthrax toxin lethal factor, or ricin. See Hall (ed.),  Immunotoxin Methods and Protocols  (Methods in Molecular Biology, vol. 166), Humana Press (2000); and Frankel et al. (eds.),  Clinical Applications of Immunotoxins , Springer-Verlag (1998). \\n     The antibodies of the present invention can usefully be attached to a substrate, and it is, therefore, another aspect of the invention to provide antibodies that bind specifically to one or more of the polypeptides of the present invention, to one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention, or the binding of which can be competitively inhibited by one or more of the polypeptides of the present invention or one or more of the polypeptides encoded by the isolated nucleic acid molecules of the present invention, attached to a substrate. Substrates can be porous or nonporous, planar or nonplanar. For example, the antibodies of the present invention can usefully be conjugated to filtration media, such as NHS-activated Sepharose or CNBr-activated Sepharose for purposes of immunoaffinity chromatography. For example, the antibodies of the present invention can usefully be attached to paramagnetic microspheres, typically by biotin-streptavidin interaction, which microsphere can then be used for isolation of cells that express or display the polypeptides of the present invention. As another example, the antibodies of the present invention can usefully be attached to the surface of a microtiter plate for ELISA. \\n     As noted above, the antibodies of the present invention can be produced in prokaryotic and eukaryotic cells. It is, therefore, another aspect of the present invention to provide cells that express the antibodies of the present invention, including hybridoma cells, B cells, plasma cells, and host cells recombinantly modified to express the antibodies of the present invention. \\n     In yet a further aspect, the present invention provides aptamers evolved to bind specifically to one or more of the OSPs of the present invention or to polypeptides encoded by the OSNAs of the invention. \\n     In sum, one of skill in the art, provided with the teachings of this invention, has available a variety of methods which may be used to alter the biological properties of the antibodies of this invention including methods which would increase or decrease the stability or half-life, immunogenicity, toxicity, affinity or yield of a given antibody molecule, or to alter it in any other way that may render it more suitable for a particular application. \\n     Transgenic Animals and Cells \\n     In another aspect, the invention provides transgenic cells and non-human organisms comprising nucleic acid molecules of the invention. In a preferred embodiment, the transgenic cells and non-human organisms comprise a nucleic acid molecule encoding an OSP. In a preferred embodiment, the OSP comprises an amino acid sequence selected from SEQ ID NO: 129-295, or a fragment, mutein, homologous protein or allelic variant thereof. In another preferred embodiment, the transgenic cells and non-human organism comprise an OSNA of the invention, preferably an OSNA comprising a nucleotide sequence selected from the group consisting of SEQ ID NO: 1-128, or a part, substantially similar nucleic acid molecule, allelic variant or hybridizing nucleic acid molecule thereof. \\n     In another embodiment, the transgenic cells and non-human organisms have a targeted disruption or replacement of the endogenous orthologue of the human OSG. The transgenic cells can be embryonic stem cells or somatic cells. The transgenic non-human organisms can be chimeric, nonchimeric heterozygotes, and nonchimeric homozygotes. Methods of producing transgenic animals are well known in the art. See, e.g., Hogan et al.,  Manipulating the Mouse Embryo: A Laboratory Manual,  2d ed., Cold Spring Harbor Press (1999); Jackson et al.,  Mouse Genetics and Transgenics: A Practical Approach , Oxford University Press (2000); and Pinkert,  Transgenic Animal Technology: A Laboratory Handbook , Academic Press (1999). \\n     Any technique known in the art may be used to introduce a nucleic acid molecule of the invention into an animal to produce the founder lines of transgenic animals. Such techniques include, but are not limited to, pronuclear microinjection. (see, e.g., Paterson et al.,  Appl. Microbiol. Biotechnol.  40: 691-698 (1994); Carver et al.,  Biotechnology  11: 1263-1270 (1993); Wright et al.,  Biotechnology  9: 830-834 (1991); and U.S. Pat. No. 4,873,191, herein incorporated by reference in its entirety); retrovirus-mediated gene transfer into germ lines, blastocysts or embryos (see, e.g., Van der Putten et al.,  Proc. Natl. Acad. Sci., USA  82: 6148-6152 (1985)); gene targeting in embryonic stem cells (see, e.g., Thompson et al.,  Cell  56: 313-321 (1989)); electroporation of cells or embryos (see, e.g., Lo, 1983 , Mol. Cell. Biol.  3: 1803-1814 (1983)); introduction using a gene gun (see, e.g., Ulmer et al.,  Science  259: 1745-49 (1993); introducing nucleic acid constructs into embryonic pleuripotent stem cells and transferring the stem cells back into the blastocyst; and sperm-mediated gene transfer (see, e.g., Lavitrano et al.,  Cell  57: 717-723 (1989)). \\n     Other techniques include, for example, nuclear transfer into enucleated oocytes of nuclei from cultured embryonic, fetal, or adult cells induced to quiescence (see, e.g., Campell et al.,  Nature  380: 64-66 (1996); Wilmut et al.,  Nature  385: 810-813 (1997)). The present invention provides for transgenic animals that carry the transgene (i.e., a nucleic acid molecule of the invention) in all their cells, as well as animals which carry the transgene in some, but not all their cells, i.e. e., mosaic animals or chimeric animals. \\n     The transgene may be integrated as a single transgene or as multiple copies, such as in concatamers, e.g., head-to-head tandems or head-to-tail tandems. The transgene may also be selectively introduced into and activated in a particular cell type by following, e.g., the teaching of Lasko et al. et al.,  Proc. Natl. Acad. Sci. USA  89: 6232-6236 (1992). The regulatory sequences required for such a cell-type specific activation will depend upon the particular cell type of interest, and will be apparent to those of skill in the art. \\n     Once transgenic animals have been generated, the expression of the recombinant gene may be assayed utilizing standard techniques. Initial screening may be accomplished by Southern blot analysis or PCR techniques to analyze animal tissues to verify that integration of the transgene has taken place. The level of mRNA expression of the transgene in the tissues of the transgenic animals may also be assessed using techniques which include, but are not limited to, Northern blot analysis of tissue samples obtained from the animal, in situ hybridization analysis, and reverse transcriptase-PCR (RT-PCR). Samples of transgenic gene-expressing tissue may also be evaluated immunocytochemically or immunohistochemically using antibodies specific for the transgene product. \\n     Once the founder animals are produced, they may be bred, inbred, outbred, or crossbred to produce colonies of the particular animal. Examples of such breeding strategies include, but are not limited to: outbreeding of founder animals with more than one integration site in order to establish separate lines; inbreeding of separate lines in order to produce compound transgenics that express the transgene at higher levels because of the effects of additive expression of each transgene; crossing of heterozygous transgenic animals to produce animals homozygous for a given integration site in order to both augment expression and eliminate the need for screening of animals by DNA analysis; crossing of separate homozygous lines to produce compound heterozygous or homozygous lines; and breeding to place the transgene on a distinct background that is appropriate for an experimental model of interest. \\n     Transgenic animals of the invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying conditions and/or disorders associated with aberrant expression, and in screening for compounds effective in ameliorating such conditions and/or disorders. \\n     Methods for creating a transgenic animal with a disruption of a targeted gene are also well known in the art. In general, a vector is designed to comprise some nucleotide sequences homologous to the endogenous targeted gene. The vector is introduced into a cell so that it may integrate, via homologous recombination with chromosomal sequences, into the endogenous gene, thereby disrupting the function of the endogenous gene. The transgene may also be selectively introduced into a particular cell type, thus inactivating the endogenous gene in only that cell type. See, e.g., Gu et al.,  Science  265: 103-106 (1994). The regulatory sequences required for such a cell-type specific inactivation will depend upon the particular cell type of interest, and will be apparent to those of skill in the art. See, e.g., Smithies et al.,  Nature  317: 230-234 (1985); Thomas et al.,  Cell  51: 503-512 (1987); Thompson et al.,  Cell  5: 313-321 (1989). \\n     In one embodiment, a mutant, non-functional nucleic acid molecule of the invention (or a completely unrelated DNA sequence) flanked by DNA homologous to the endogenous nucleic acid sequence (either the coding regions or regulatory regions of the gene) can be used, with or without a selectable marker and/or a negative selectable marker, to transfect cells that express polypeptides of the invention in vivo. In another embodiment, techniques known in the art are used to generate knockouts in cells that contain, but do not express the gene of interest. Insertion of the DNA construct, via targeted homologous recombination, results in inactivation of the targeted gene. Such approaches are particularly suited in research and agricultural fields where modifications to embryonic stem cells can be used to generate animal offspring with an inactive targeted gene. See, e.g., Thomas, supra and Thompson, supra. However this approach can be routinely adapted for use in humans provided the recombinant DNA constructs are directly administered or targeted to the required site in vivo using appropriate viral vectors that will be apparent to those of skill in the art. \\n     In further embodiments of the invention, cells that are genetically engineered to express the polypeptides of the invention, or alternatively, that are genetically engineered not to express the polypeptides of the invention (e.g., knockouts) are administered to a patient in vivo. Such cells may be obtained from an animal or patient or an MHC compatible donor and can include, but are not limited to fibroblasts, bone marrow cells, blood cells (e.g., lymphocytes), adipocytes, muscle cells, endothelial cells etc. The cells are genetically engineered in vitro using recombinant DNA techniques to introduce the coding sequence of polypeptides of the invention into the cells, or alternatively, to disrupt the coding sequence and/or endogenous regulatory sequence associated with the polypeptides of the invention, e.g., by transduction (using viral vectors, and preferably vectors that integrate the transgene into the cell genome) or transfection procedures, including, but not limited to, the use of plasmids, cosmids, YACs, naked DNA, electroporation, liposomes, etc. \\n     The coding sequence of the polypeptides of the invention can be placed under the control of a strong constitutive or inducible promoter or promoter/enhancer to achieve expression, and preferably secretion, of the polypeptides of the invention. The engineered cells which express and preferably secrete the polypeptides of the invention can be introduced into the patient systemically, e.g., in the circulation, or intraperitoneally. \\n     Alternatively, the cells can be incorporated into a matrix and implanted in the body, e.g., genetically engineered fibroblasts can be implanted as part of a skin graft; genetically engineered endothelial cells can be implanted as part of a lymphatic or vascular graft. See, e.g., U.S. Pat. Nos. 5,399,349 and 5,460,959, each of which is incorporated by reference herein in its entirety. \\n     When the cells to be administered are non-autologous or non-MHC compatible cells, they can be administered using well known techniques which prevent the development of a host immune response against the introduced cells. For example, the cells may be introduced in an encapsulated form which, while allowing for an exchange of components with the immediate extracellular environment, does not allow the introduced cells to be recognized by the host immune system. \\n     Transgenic and “knock-out” animals of the invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying conditions and/or disorders associated with aberrant expression, and in screening for compounds effective in ameliorating such conditions and/or disorders. \\n     Computer Readable Means \\n     A further aspect of the invention is a computer readable means for storing the nucleic acid and amino acid sequences of the instant invention. In a preferred embodiment, the invention provides a computer readable means for storing SEQ ID NO: 129-295 and SEQ ID NO: 1-128 as described herein, as the complete set of sequences or in any combination. The records of the computer readable means can be accessed for reading and display and for interface with a computer system for the application of programs allowing for the location of data upon a query for data meeting certain criteria, the comparison of sequences, the alignment or ordering of sequences meeting a set of criteria, and the like. \\n     The nucleic acid and amino acid sequences of the invention are particularly useful as components in databases useful for search analyses as well as in sequence analysis algorithms. As used herein, the terms “nucleic acid sequences of the invention” and “amino acid sequences of the invention” mean any detectable chemical or physical characteristic of a polynucleotide or polypeptide of the invention that is or may be reduced to or stored in a computer readable form. These include, without limitation, chromatographic scan data or peak data, photographic data or scan data therefrom, and mass spectrographic data. \\n     This invention provides computer readable media having stored thereon sequences of the invention. A computer readable medium may comprise one or more of the following: a nucleic acid sequence comprising a sequence of a nucleic acid sequence of the invention; an amino acid sequence comprising an amino acid sequence of the invention; a set of nucleic acid sequences wherein at least one of said sequences comprises the sequence of a nucleic acid sequence of the invention; a set of amino acid sequences wherein at least one of said sequences comprises the sequence of an amino acid sequence of the invention; a data set representing a nucleic acid sequence comprising the sequence of one or more nucleic acid sequences of the invention; a data set representing a nucleic acid sequence encoding an amino acid sequence comprising the sequence of an amino acid sequence of the invention; a set of nucleic acid sequences wherein at least one of said sequences comprises the sequence of a nucleic acid sequence of the invention; a set of amino acid sequences wherein at least one of said sequences comprises the sequence of an amino acid sequence of the invention; a data set representing a nucleic acid sequence comprising the sequence of a nucleic acid sequence of the invention; a data set representing a nucleic acid sequence encoding an amino acid sequence comprising the sequence of an amino acid sequence of the invention. The computer readable medium can be any composition of matter used to store information or data, including, for example, commercially available floppy disks, tapes, hard drives, compact disks, and video disks. \\n     Also provided by the invention are methods for the analysis of character sequences, particularly genetic sequences. Preferred methods of sequence analysis include, for example, methods of sequence homology analysis, such as identity and similarity analysis, RNA structure analysis, sequence assembly, cladistic analysis, sequence motif analysis, open reading frame determination, nucleic acid base calling, and sequencing chromatogram peak analysis. \\n     A computer-based method is provided for performing nucleic acid sequence identity or similarity identification. This method comprises the steps of providing a nucleic acid sequence comprising the sequence of a nucleic acid of the invention in a computer readable medium; and comparing said nucleic acid sequence to at least one nucleic acid or amino acid sequence to identify sequence identity or similarity. \\n     A computer-based method is also provided for performing amino acid homology identification, said method comprising the steps of: providing an amino acid sequence comprising the sequence of an amino acid of the invention in a computer readable medium; and comparing said amino acid sequence to at least one nucleic acid or an amino acid sequence to identify homology. \\n     A computer-based method is still further provided for assembly of overlapping nucleic acid sequences into a single nucleic acid sequence, said method comprising the steps of: providing a first nucleic acid sequence comprising the sequence of a nucleic acid of the invention in a computer readable medium; and screening for at least one overlapping region between said first nucleic acid sequence and a second nucleic acid sequence. In addition, the invention includes a method of using patterns of expression associated with either the nucleic acids or proteins in a computer-based method to diagnose disease. \\n     Diagnostic Methods for Ovarian Cancer \\n     The present invention also relates to quantitative and qualitative diagnostic assays and methods for detecting, diagnosing, monitoring, staging and predicting cancers by comparing expression of an OSNA or an OSP in a human patient that has or may have ovarian cancer, or who is at risk of developing ovarian cancer, with the expression of an OSNA or an OSP in a normal human control. For purposes of the present invention, “expression of an OSNA” or “OSNA expression” means the quantity of OSNA mRNA that can be measured by any method known in the art or the level of transcription that can be measured by any method known in the art in a cell, tissue, organ or whole patient. Similarly, the term “expression of an OSP” or “OSP expression” means the amount of OSP that can be measured by any method known in the art or the level of translation of an OSNA that can be measured by any method known in the art. \\n     The present invention provides methods for diagnosing ovarian cancer in a patient, by analyzing for changes in levels of OSNA or OSP in cells, tissues, organs or bodily fluids compared with levels of OSNA or OSP in cells, tissues, organs or bodily fluids of preferably the same type from a normal human control, wherein an increase, or decrease in certain cases, in levels of an OSNA or OSP in the patient versus the normal human control is associated with the presence of ovarian cancer or with a predilection to the disease. In another preferred embodiment, the present invention provides methods for diagnosing ovarian cancer in a patient by analyzing changes in the structure of the mRNA of an OSG compared to the mRNA from a normal control. These changes include, without limitation, aberrant splicing, alterations in polyadenylation and/or alterations in 5′ nucleotide capping. In yet another preferred embodiment, the present invention provides methods for diagnosing ovarian cancer in a patient by analyzing changes in an OSP compared to an OSP from a normal patient. These changes include, e.g., alterations, including post translational modifications such as glycosylation and/or phosphorylation of the OSP or changes in the subcellular OSP localization. \\n     For purposes of the present invention, diagnosing means that OSNA or OSP levels are used to determine the presence or absence of disease in a patient. As will be understood by those of skill in the art, measurement of other diagnostic parameters may be required for definitive diagnosis or determination of the appropriate treatment for the disease. The determination may be made by a clinician, a doctor, a testing laboratory, or a patient using an over the counter test. The patient may have symptoms of disease or may be asymptomatic. In addition, the OSNA or OSP levels of the present invention may be used as screening marker to determine whether further tests or biopsies are warranted. In addition, the OSNA or OSP levels may be used to determine the vulnerability or susceptibility to disease. \\n     In a preferred embodiment, the expression of an OSNA is measured by determining the amount of a mRNA that encodes an amino acid sequence selected from SEQ ID NO: 129-295, a homolog, an allelic variant, or a fragment thereof. In a more preferred embodiment, the OSNA expression that is measured is the level of expression of an OSNA mRNA selected from SEQ ID NO: 1-128, or a hybridizing nucleic acid, homologous nucleic acid or allelic variant thereof, or a part of any of these nucleic acid molecules. OSNA expression may be measured by any method known in the art, such as those described supra, including measuring mRNA expression by Northern blot, quantitative or qualitative reverse transcriptase PCR (RT-PCR), microarray, dot or slot blots or in situ hybridization. See, e.g., Ausubel (1992), supra; Ausubel (1999), supra; Sambrook (1989), supra; and Sambrook (2001), supra. OSNA transcription may be measured by any method known in the art including using a reporter gene hooked up to the promoter of an OSG of interest or doing nuclear run-off assays. Alterations in mRNA structure, e.g., aberrant splicing variants, may be determined by any method known in the art, including, RT-PCR followed by sequencing or restriction analysis. As necessary, OSNA expression may be compared to a known control, such as normal ovarian nucleic acid, to detect a change in expression. \\n     In another preferred embodiment, the expression of an OSP is measured by determining the level of an OSP having an amino acid sequence selected from the group consisting of SEQ ID NO: 129-295, a homolog, an allelic variant, or a fragment thereof. Such levels are preferably determined in at least one of cells, tissues, organs and/or bodily fluids, including determination of normal and abnormal levels. Thus, for instance, a diagnostic assay in accordance with the invention for diagnosing over- or underexpression of an OSNA or OSP compared to normal control bodily fluids, cells, or tissue samples may be used to diagnose the presence of ovarian cancer. The expression level of an OSP may be determined by any method known in the art, such as those described supra. In a preferred embodiment, the OSP expression level may be determined by radioimmunoassays, competitive-binding assays, ELISA, Western blot, FACS, immunohistochemistry, immunoprecipitation, proteomic approaches: two-dimensional gel electrophoresis (2D electrophoresis) and non-gel-based approaches such as mass spectrometry or protein interaction profiling. See, e.g, Harlow (1999), supra; Ausubel (1992), supra; and Ausubel (1999), supra. Alterations in the OSP structure may be determined by any method known in the art, including, e.g., using antibodies that specifically recognize phosphoserine, phosphothreonine or phosphotyrosine residues, two-dimensional polyacrylamide gel electrophoresis (2D PAGE) and/or chemical analysis of amino acid residues of the protein. Id. \\n     In a preferred embodiment, a radioimmunoassay (RIA) or an ELISA is used. An antibody specific to an OSP is prepared if one is not already available. In a preferred embodiment, the antibody is a monoclonal antibody. The anti-OSP antibody is bound to a solid support and any free protein binding sites on the solid support are blocked with a protein such as bovine serum albumin. A sample of interest is incubated with the antibody on the solid support under conditions in which the OSP will bind to the anti-OSP antibody. The sample is removed, the solid support is washed to remove unbound material, and an anti-OSP antibody that is linked to a detectable reagent (a radioactive substance for RIA and an enzyme for ELISA) is added to the solid support and incubated under conditions in which binding of the OSP to the labeled antibody will occur. After binding, the unbound labeled antibody is removed by washing. For an ELISA, one or more substrates are added to produce a colored reaction product that is based upon the amount of an OSP in the sample. For an RIA, the solid support is counted for radioactive decay signals by any method known in the art. Quantitative results for both RIA and ELISA typically are obtained by reference to a standard curve. \\n     Other methods to measure OSP levels are known in the art. For instance, a competition assay may be employed wherein an anti-OSP antibody is attached to a solid support and an allocated amount of a labeled OSP and a sample of interest are incubated with the solid support. The amount of labeled OSP attached to the solid support can be correlated to the quantity of an OSP in the sample. \\n     Of the proteomic approaches, 2D PAGE is a well known technique. Isolation of individual proteins from a sample such as serum is accomplished using sequential separation of proteins by isoelectric point and molecular weight. Typically, polypeptides are first separated by isoelectric point (the first dimension) and then separated by size using an electric current (the second dimension). In general, the second dimension is perpendicular to the first dimension. Because no two proteins with different sequences are identical on the basis of both size and charge, the result of 2D PAGE is a roughly square gel in which each protein occupies a unique spot. Analysis of the spots with chemical or antibody probes, or subsequent protein microsequencing can reveal the relative abundance of a given protein and the identity of the proteins in the sample. \\n     Expression levels of an OSNA can be determined by any method known in the art, including PCR and other nucleic acid methods, such as ligase chain reaction (LCR) and nucleic acid sequence based amplification (NASBA), can be used to detect malignant cells for diagnosis and monitoring of various malignancies. For example, reverse-transcriptase PCR (RT-PCR) is a powerful technique which can be used to detect the presence of a specific mRNA population in a complex mixture of thousands of other mRNA species. In RT-PCR, an mRNA species is first reverse transcribed to complementary DNA (cDNA) with use of the enzyme reverse transcriptase; the cDNA is then amplified as in a standard PCR reaction. \\n     Hybridization to specific DNA molecules (e.g., oligonucleotides) arrayed on a solid support can be used to both detect the expression of and quantitate the level of expression of one or more OSNAs of interest. In this approach, all or a portion of one or more OSNAs is fixed to a substrate. A sample of interest, which may comprise RNA, e.g., total RNA or polyA-selected mRNA, or a complementary DNA (cDNA) copy of the RNA is incubated with the solid support under conditions in which hybridization will occur between the DNA on the solid support and the nucleic acid molecules in the sample of interest. Hybridization between the substrate-bound DNA and the nucleic acid molecules in the sample can be detected and quantitated by several means, including, without limitation, radioactive labeling or fluorescent labeling of the nucleic acid molecule or a secondary molecule designed to detect the hybrid. \\n     The above tests can be carried out on samples derived from a variety of cells, bodily fluids and/or tissue extracts such as homogenates or solubilized tissue obtained from a patient. Tissue extracts are obtained routinely from tissue biopsy and autopsy material. Bodily fluids useful in the present invention include blood, urine, saliva or any other bodily secretion or derivative thereof. As used herein “blood” includes whole blood, plasma, serum, circulating epithelial cells, constituents, or any derivative of blood. \\n     In addition to detection in bodily fluids, the proteins and nucleic acids of the invention are suitable to detection by cell capture technology. Whole cells may be captured by a variety methods for example magnetic separation, such as described in U.S. Pat. Nos. 5,200,084; 5,186,827; 5,108,933; and 4,925,788, the disclosures of which are incorporated herein by reference in their entireties. Epithelial cells may be captured using such products as Dynabeads® or CELLection™ (Dynal Biotech, Oslo, Norway). Alternatively, fractions of blood may be captured, e.g., the buffy coat fraction (50 mm cells isolated from 5 ml of blood) containing epithelial cells. In addition, cancer cells may be captured using the techniques described in WO 00/47998, the disclosure of which is incorporated herein by reference in its entirety. Once the cells are captured or concentrated, the proteins or nucleic acids are detected by the means described in the subject application. Alternatively, nucleic acids may be captured directly from blood samples, see U.S. Pat. Nos. 6,156,504, 5,501,963; or WO 01/42504, the disclosures of which are incorporated herein by reference in their entireties. \\n     In a preferred embodiment, the specimen tested for expression of OSNA or OSP includes without limitation ovarian tissue, ovarian cells grown in cell culture, blood, serum, lymph node tissue, and lymphatic fluid. In another preferred embodiment, especially when metastasis of a primary ovarian cancer is known or suspected, specimens include, without limitation, tissues from brain, bone, bone marrow, liver, lungs, colon, and adrenal glands. In general, the tissues may be sampled by biopsy, including, without limitation, needle biopsy, e.g., transthoracic needle aspiration, cervical mediatinoscopy, endoscopic lymph node biopsy, video-assisted thoracoscopy, exploratory thoracotomy, bone marrow biopsy and bone marrow aspiration. \\n     All the methods of the present invention may optionally include determining the expression levels of one or more other cancer markers in addition to determining the expression level of an OSNA or OSP. In many cases, the use of another cancer marker will decrease the likelihood of false positives or false negatives. In one embodiment, the one or more other cancer markers include other OSNAs or OSPs as disclosed herein. Other cancer markers useful in the present invention will depend on the cancer being tested and are known to those of skill in the art. In a preferred embodiment, at least one other cancer marker in addition to a particular OSNA or OSP is measured. In a more preferred embodiment, at least two other additional cancer markers are used. In an even more preferred embodiment, at least three, more preferably at least five, even more preferably at least ten additional cancer markers are used. \\n     Diagnosing \\n     In one aspect, the invention provides a method for determining the expression levels and/or structural alterations of one or more OSNA and/or OSP in a sample from a patient suspected of having ovarian cancer. In general, the method comprises the steps of obtaining the sample from the patient, determining the expression level or structural alterations of an OSNA and/or OSP and then ascertaining whether the patient has ovarian cancer from the expression level of the OSNA or OSP. In general, if high expression relative to a control of an OSNA or OSP is indicative of ovarian cancer, a diagnostic assay is considered positive if the level of expression of the OSNA or OSP is at least one and a half times higher, and more preferably are at least two times higher, still more preferably five times higher, even more preferably at least ten times higher, than in preferably the same cells, tissues or bodily fluid of a normal human control. In contrast, if low expression relative to a control of an OSNA or OSP is indicative of ovarian cancer, a diagnostic assay is considered positive if the level of expression of the OSNA or OSP is at least one and a half times lower, and more preferably are at least two times lower, still more preferably five times lower, even more preferably at least ten times lower than in preferably the same cells, tissues or bodily fluid of a normal human control. The normal human control may be from a different patient or from uninvolved tissue of the same patient. \\n     The present invention also provides a method of determining whether ovarian cancer has metastasized in a patient. One may identify whether the ovarian cancer has metastasized by measuring the expression levels and/or structural alterations of one or more OSNAs and/or OSPs in a variety of tissues. The presence of an OSNA or OSP in a tissue other than ovarian at levels higher than that of corresponding noncancerous tissue (e.g., the same tissue from another individual) is indicative of metastasis if high level expression of an OSNA or OSP is associated with ovarian cancer. Similarly, the presence of an OSNA or OSP in a tissue other than ovarian at levels lower than that of corresponding noncancerous tissue is indicative of metastasis if low level expression of an OSNA or OSP is associated with ovarian cancer. Further, the presence of a structurally altered OSNA or OSP that is associated with ovarian cancer is also indicative of metastasis. \\n     In general, if high expression relative to a control of an OSNA or OSP is indicative of metastasis, an assay for metastasis is considered positive if the level of expression of the OSNA or OSP is at least one and a half times higher, and more preferably are at least two times higher, still more preferably five times higher, even more preferably at least ten times higher, than in preferably the same cells, tissues or bodily fluid of a normal human control. In contrast, if low expression relative to a control of an OSNA or OSP is indicative of metastasis, an assay for metastasis is considered positive if the level of expression of the OSNA or OSP is at least one and a half times lower, and more preferably are at least two times lower, still more preferably five times lower, even more preferably at least ten times lower than in preferably the same cells, tissues or bodily fluid of a normal human control. \\n     Staging \\n     The invention also provides a method of staging ovarian cancer in a human patient. The method comprises identifying a human patient having ovarian cancer and analyzing cells, tissues or bodily fluids from such human patient for expression levels and/or structural alterations of one or more OSNAs or OSPs. First, one or more tumors from a variety of patients are staged according to procedures well known in the art, and the expression levels of one or more OSNAs or OSPs is determined for each stage to obtain a standard expression level for each OSNA and OSP. Then, the OSNA or OSP expression levels of the OSNA or OSP are determined in a biological sample from a patient whose stage of cancer is not known. The OSNA or OSP expression levels from the patient are then compared to the standard expression level. By comparing the expression level of the OSNAs and OSPs from the patient to the standard expression levels, one may determine the stage of the tumor. The same procedure may be followed using structural alterations of an OSNA or OSP to determine the stage of a ovarian cancer. \\n     Monitoring \\n     Further provided is a method of monitoring ovarian cancer in a human patient. One may monitor a human patient to determine whether there has been metastasis and, if there has been, when metastasis began to occur. One may also monitor a human patient to determine whether a preneoplastic lesion has become cancerous. One may also monitor a human patient to determine whether a therapy, e.g., chemotherapy, radiotherapy or surgery, has decreased or eliminated the ovarian cancer. The monitoring may determine if there has been a reoccurrence and, if so, determine its nature. The method comprises identifying a human patient that one wants to monitor for ovarian cancer, periodically analyzing cells, tissues or bodily fluids from such human patient for expression levels of one or more OSNAs or OSPs, and comparing the OSNA or OSP levels over time to those OSNA or OSP expression levels obtained previously. Patients may also be monitored by measuring one or more structural alterations in an OSNA or OSP that are associated with ovarian cancer. \\n     If increased expression of an OSNA or OSP is associated with metastasis, treatment failure, or conversion of a preneoplastic lesion to a cancerous lesion, then detecting an increase in the expression level of an OSNA or OSP indicates that the tumor is metastasizing, that treatment has failed or that the lesion is cancerous, respectively. One having ordinary skill in the art would recognize that if this were the case, then a decreased expression level would be indicative of no metastasis, effective therapy or failure to progress to a neoplastic lesion. If decreased expression of an OSNA or OSP is associated with metastasis, treatment failure, or conversion of a preneoplastic lesion to a cancerous lesion, then detecting a decrease in the expression level of an OSNA or OSP indicates that the tumor is metastasizing, that treatment has failed or that the lesion is cancerous, respectively. In a preferred embodiment, the levels of OSNAs or OSPs are determined from the same cell type, tissue or bodily fluid as prior patient samples. Monitoring a patient for onset of ovarian cancer metastasis is periodic and preferably is done on a quarterly basis, but may be done more or less frequently. \\n     The methods described herein can further be utilized as prognostic assays to identify subjects having or at risk of developing a disease or disorder associated with increased or decreased expression levels of an OSNA and/or OSP. The present invention provides a method in which a test sample is obtained from a human patient and one or more OSNAs and/or OSPs are detected. The presence of higher (or lower) OSNA or OSP levels as compared to normal human controls is diagnostic for the human patient being at risk for developing cancer, particularly ovarian cancer. The effectiveness of therapeutic agents to decrease (or increase) expression or activity of one or more OSNAs and/or OSPs of the invention can also be monitored by analyzing levels of expression of the OSNAs and/or OSPs in a human patient in clinical trials or in in vitro screening assays such as in human cells. In this way, the gene expression pattern can serve as a marker, indicative of the physiological response of the human patient or cells, as the case may be, to the agent being tested. \\n     Detection of Genetic Lesions or Mutations \\n     The methods of the present invention can also be used to detect genetic lesions or mutations in an OSG, thereby determining if a human with the genetic lesion is susceptible to developing ovarian cancer or to determine what genetic lesions are responsible, or are partly responsible, for a person&#39;s existing ovarian cancer. Genetic lesions can be detected, for example, by ascertaining the existence of a deletion, insertion and/or substitution of one or more nucleotides from the OSGs of this invention, a chromosomal rearrangement of an OSG, an aberrant modification of an OSG (such as of the methylation pattern of the genomic DNA), or allelic loss of an OSG. Methods to detect such lesions in the OSG of this invention are known to those having ordinary skill in the art following the teachings of the specification. \\n     Methods of Detecting Noncancerous Ovarian Diseases \\n     The present invention also provides methods for determining the expression levels and/or structural alterations of one or more OSNAs and/or OSPs in a sample from a patient suspected of having or known to have a noncancerous ovarian disease. In general, the method comprises the steps of obtaining a sample from the patient, determining the expression level or structural alterations of an OSNA and/or OSP, comparing the expression level or structural alteration of the OSNA or OSP to a normal ovarian control, and then ascertaining whether the patient has a noncancerous ovarian disease. In general, if high expression relative to a control of an OSNA or OSP is indicative of a particular noncancerous ovarian disease, a diagnostic assay is considered positive if the level of expression of the OSNA or OSP is at least two times higher, and more preferably are at least five times higher, even more preferably at least ten times higher, than in preferably the same cells, tissues or bodily fluid of a normal human control. In contrast, if low expression relative to a control of an OSNA or OSP is indicative of a noncancerous ovarian disease, a diagnostic assay is considered positive if the level of expression of the OSNA or OSP is at least two times lower, more preferably are at least five times lower, even more preferably at least ten times lower than in preferably the same cells, tissues or bodily fluid of a normal human control. The normal human control may be from a different patient or from uninvolved tissue of the same patient. \\n     One having ordinary skill in the art may determine whether an OSNA and/or OSP is associated with a particular noncancerous ovarian disease by obtaining ovarian tissue from a patient having a noncancerous ovarian disease of interest and determining which OSNAs and/or OSPs are expressed in the tissue at either a higher or a lower level than in normal ovarian tissue. In another embodiment, one may determine whether an OSNA or OSP exhibits structural alterations in a particular noncancerous ovarian disease state by obtaining ovarian tissue from a patient having a noncancerous ovarian disease of interest and determining the structural alterations in one or more OSNAs and/or OSPs relative to normal ovarian tissue. \\n     Methods for Identifying Ovarian Tissue \\n     In another aspect, the invention provides methods for identifying ovarian tissue. These methods are particularly useful in, e.g., forensic science, ovarian cell differentiation and development, and in tissue engineering. \\n     In one embodiment, the invention provides a method for determining whether a sample is ovarian tissue or has ovarian tissue-like characteristics. The method comprises the steps of providing a sample suspected of comprising ovarian tissue or having ovarian tissue-like characteristics, determining whether the sample expresses one or more OSNAs and/or OSPs, and, if the sample expresses one or more OSNAs and/or OSPs, concluding that the sample comprises ovarian tissue. In a preferred embodiment, the OSNA encodes a polypeptide having an amino acid sequence selected from SEQ ID NO: 129-295, or a homolog, allelic variant or fragment thereof. In a more preferred embodiment, the OSNA has a nucleotide sequence selected from SEQ ID NO: 1-128, or a hybridizing nucleic acid, an allelic variant or a part thereof. Determining whether a sample expresses an OSNA can be accomplished by any method known in the art. Preferred methods include hybridization to microarrays, Northern blot hybridization, and quantitative or qualitative RT-PCR. In another preferred embodiment, the method can be practiced by determining whether an OSP is expressed. Determining whether a sample expresses an OSP can be accomplished by any method known in the art. Preferred methods include Western blot, ELISA, RIA and 2D PAGE. In one embodiment, the OSP has an amino acid sequence selected from SEQ ID NO: 129-295, or a homolog, allelic variant or fragment thereof. In another preferred embodiment, the expression of at least two OSNAs and/or OSPs is determined. In a more preferred embodiment, the expression of at least three, more preferably four and even more preferably five OSNAs and/or OSPs are determined. \\n     In one embodiment, the method can be used to determine whether an unknown tissue is ovarian tissue. This is particularly useful in forensic science, in which small, damaged pieces of tissues that are not identifiable by microscopic or other means are recovered from a crime or accident scene. In another embodiment, the method can be used to determine whether a tissue is differentiating or developing into ovarian tissue. This is important in monitoring the effects of the addition of various agents to cell or tissue culture, e.g., in producing new ovarian tissue by tissue engineering. These agents include, e.g., growth and differentiation factors, extracellular matrix proteins and culture medium. Other factors that may be measured for effects on tissue development and differentiation include gene transfer into the cells or tissues, alterations in pH, aqueous:air interface and various other culture conditions. \\n     Methods for Producing and Modifying Ovarian Tissue \\n     In another aspect, the invention provides methods for producing engineered ovarian tissue or cells. In one embodiment, the method comprises the steps of providing cells, introducing an OSNA or an OSG into the cells, and growing the cells under conditions in which they exhibit one or more properties of ovarian tissue cells. In a preferred embodiment, the cells are pleuripotent. As is well known in the art, normal ovarian tissue comprises a large number of different cell types. Thus, in one embodiment, the engineered ovarian tissue or cells comprises one of these cell types. In another embodiment, the engineered ovarian tissue or cells comprises more than one ovarian cell type. Further, the culture conditions of the cells or tissue may require manipulation in order to achieve full differentiation and development of the ovarian cell tissue. Methods for manipulating culture conditions are well known in the art. \\n     Nucleic acid molecules encoding one or more OSPs are introduced into cells, preferably pleuripotent cells. In a preferred embodiment, the nucleic acid molecules encode OSPs having amino acid sequences selected from SEQ ID NO: 129-295, or homologous proteins, analogs, allelic variants or fragments thereof. In a more preferred embodiment, the nucleic acid molecules have a nucleotide sequence selected from SEQ ID NO: 1-128, or hybridizing nucleic acids, allelic variants or parts thereof. In another highly preferred embodiment, an OSG is introduced into the cells. Expression vectors and methods of introducing nucleic acid molecules into cells are well known in the art and are described in detail, supra. \\n     Artificial ovarian tissue may be used to treat patients who have lost some or all of their ovarian function. \\n     Pharmaceutical Compositions \\n     In another aspect, the invention provides pharmaceutical compositions comprising the nucleic acid molecules, polypeptides, fusion proteins, antibodies, antibody derivatives, antibody fragments, agonists, antagonists, or inhibitors of the present invention. In a preferred embodiment, the pharmaceutical composition comprises an OSNA or part thereof. In a more preferred embodiment, the OSNA has a nucleotide sequence selected from the group consisting of SEQ ID NO: 1-128, a nucleic acid that hybridizes thereto, an allelic variant thereof, or a nucleic acid that has substantial sequence identity thereto. In another preferred embodiment, the pharmaceutical composition comprises an OSP or fragment thereof. In a more preferred embodiment, the pharmaceutical composition comprises an OSP having an amino acid sequence that is selected from the group consisting of SEQ ID NO: 129-295, a polypeptide that is homologous thereto, a fusion protein comprising all or a portion of the polypeptide, or an analog or derivative thereof. In another preferred embodiment, the pharmaceutical composition comprises an anti-OSP antibody, preferably an antibody that specifically binds to an OSP having an amino acid that is selected from the group consisting of SEQ ID NO: 129-295, or an antibody that binds to a polypeptide that is homologous thereto, a fusion protein comprising all or a portion of the polypeptide, or an analog or derivative thereof. \\n     Due to the association of angiogenesis with cancer vascularization there is great need of new markers and methods for diagnosing angiogenesis activity to identify developing tumors and angiogenesis related diseases. Furthermore, great need is also present for new molecular targets useful in the treatment of angiogenesis and angiogenesis related diseases such as cancer. In addition known modulators of angiogenesis such as endostatin or vascular endothelial growth factor (VEGF). Use of the methods and compositions disclosed herein in combination with anti-angiogenesis drugs, drugs that block the matrix breakdown (such as BMS-275291, Dalteparin (Fragmin®), Suramin), drugs that inhibit endothelial cells (2-methoxyestradiol (2-ME), CC-5013 (Thalidomide Analog), Combretastatin A4 Phosphate, LY317615 (Protein Kinase C Beta Inhibitor), Soy Isoflavone (Genistein; Soy Protein Isolate), Thalidomide), drugs that block activators of angiogenesis (AE-941 (Neovastat™; GW786034), Anti-VEGF Antibody (Bevacizumab; Avastin™), Interferon-alpha, PTK787/ZK 222584, VEGF-Trap, ZD6474), Drugs that inhibit endothelial-specific integrin/survival signaling (EMD 121974, Anti-Anb3 Integrin Antibody (Medi-522; Vitaxin™)). \\n     Such a composition typically contains from about 0.1 to 90% by weight of a therapeutic agent of the invention formulated in and/or with a pharmaceutically acceptable carrier or excipient. \\n     Pharmaceutical formulation is a well-established art that is further described in Gennaro (ed.),  Remington: The Science and Practice of Pharmacy,  20 th  ed., Lippincott, Williams &amp; Wilkins (2000); Ansel et al.,  Pharmaceutical Dosage Forms and Drug Delivery Systems,  7 th  ed., Lippincott Williams &amp; Wilkins (1999); and Kibbe (ed.),  Handbook of Pharmaceutical Excipients  American Pharmaceutical Association, 3 rd  ed. (2000) and thus need not be described in detail herein. \\n     Briefly, formulation of the pharmaceutical compositions of the present invention will depend upon the route chosen for administration. The pharmaceutical compositions utilized in this invention can be administered by various routes including both enteral and parenteral routes, including oral, intravenous, intramuscular, subcutaneous, inhalation, topical, sublingual, rectal, intra-arterial, intramedullary, intrathecal, intraventricular, transmucosal, transdermal, intranasal, intraperitoneal, intrapulmonary, and intrauterine. \\n     Oral dosage forms can be formulated as tablets, pills, dragees, capsules, liquids, gels, syrups, slurries, suspensions, and the like, for ingestion by the patient. \\n     Solid formulations of the compositions for oral administration can contain suitable carriers or excipients, such as carbohydrate or protein fillers, such as sugars, including lactose, sucrose, mannitol, or sorbitol; starch from corn, wheat, rice, potato, or other plants; cellulose, such as methyl cellulose, hydroxypropylmethyl-cellulose, sodium carboxymethylcellulose, or microcrystalline cellulose; gums including arabic and tragacanth; proteins such as gelatin and collagen; inorganics, such as kaolin, calcium carbonate, dicalcium phosphate, sodium chloride; and other agents such as acacia and alginic acid. \\n     Agents that facilitate disintegration and/or solubilization can be added, such as the cross-linked polyvinyl pyrrolidone, agar, alginic acid, or a salt thereof, such as sodium alginate, microcrystalline cellulose, cornstarch, sodium starch glycolate, and alginic acid. \\n     Tablet binders that can be used include acacia, methylcellulose, sodium carboxymethylcellulose, polyvinylpyrrolidone (Povidone™), hydroxypropyl methylcellulose, sucrose, starch and ethylcellulose. \\n     Lubricants that can be used include magnesium stearates, stearic acid, silicone fluid, talc, waxes, oils, and colloidal silica. \\n     Fillers, agents that facilitate disintegration and/or solubilization, tablet binders and lubricants, including the aforementioned, can be used singly or in combination. \\n     Solid oral dosage forms need not be uniform throughout. For example, dragee cores can be used in conjunction with suitable coatings, such as concentrated sugar solutions, which can also contain gum arabic, talc, polyvinylpyrrolidone, carbopol gel, polyethylene glycol, and/or titanium dioxide, lacquer solutions, and suitable organic solvents or solvent mixtures. \\n     Oral dosage forms of the present invention include push-fit capsules made of gelatin, as well as soft, sealed capsules made of gelatin and a coating, such as glycerol or sorbitol. Push-fit capsules can contain active ingredients mixed with a filler or binders, such as lactose or starches, lubricants, such as talc or magnesium stearate, and, optionally, stabilizers. In soft capsules, the active compounds can be dissolved or suspended in suitable liquids, such as fatty oils, liquid, or liquid polyethylene glycol with or without stabilizers. \\n     Additionally, dyestuffs or pigments can be added to the tablets or dragee coatings for product identification or to characterize the quantity of active compound, i.e., dosage. \\n     Liquid formulations of the pharmaceutical compositions for oral (enteral) administration are prepared in water or other aqueous vehicles and can contain various suspending agents such as methylcellulose, alginates, tragacanth, pectin, kelgin, carrageenan, acacia, polyvinylpyrrolidone, and polyvinyl alcohol. The liquid formulations can also include solutions, emulsions, syrups and elixirs containing, together with the active compound(s), wetting agents, sweeteners, and coloring and flavoring agents. \\n     The pharmaceutical compositions of the present invention can also be formulated for parenteral administration. Formulations for parenteral administration can be in the form of aqueous or non-aqueous isotonic sterile injection solutions or suspensions. \\n     For intravenous injection, water soluble versions of the compounds of the present invention are formulated in, or if provided as a lyophilate, mixed with, a physiologically acceptable fluid vehicle, such as 5% dextrose (“D5”), physiologically buffered saline, 0.9% saline, Hanks&#39; solution, or Ringer&#39;s solution. Intravenous formulations may include carriers, excipients or stabilizers including, without limitation, calcium, human serum albumin, citrate, acetate, calcium chloride, carbonate, and other salts. \\n     Intramuscular preparations, e.g. a sterile formulation of a suitable soluble salt form of the compounds of the present invention, can be dissolved and administered in a pharmaceutical excipient such as Water-for-Injection, 0.9% saline, or 5% glucose solution. Alternatively, a suitable insoluble form of the compound can be prepared and administered as a suspension in an aqueous base or a pharmaceutically acceptable oil base, such as an ester of a long chain fatty acid (e.g., ethyl oleate), fatty oils such as sesame oil, triglycerides, or liposomes. \\n     Parenteral formulations of the compositions can contain various carriers such as vegetable oils, dimethylacetamide, dimethylformamide, ethyl lactate, ethyl carbonate, isopropyl myristate, ethanol, polyols (glycerol, propylene glycol, liquid polyethylene glycol, and the like). \\n     Aqueous injection suspensions can also contain substances that increase the viscosity of the suspension, such as sodium carboxymethyl cellulose, sorbitol, or dextran. Non-lipid polycationic amino polymers can also be used for delivery. Optionally, the suspension can also contain suitable stabilizers or agents that increase the solubility of the compounds to allow for the preparation of highly concentrated solutions. \\n     Pharmaceutical compositions of the present invention can also be formulated to permit injectable, long-term, deposition. Injectable depot forms may be made by forming microencapsulated matrices of the compound in biodegradable polymers such as polylactide-polyglycolide. Depending upon the ratio of drug to polymer and the nature of the particular polymer employed, the rate of drug release can be controlled. Examples of other biodegradable polymers include poly(orthoesters) and poly(anhydrides). Depot injectable formulations are also prepared by entrapping the drug in microemulsions that are compatible with body tissues. \\n     The pharmaceutical compositions of the present invention can be administered topically. For topical use the compounds of the present invention can also be prepared in suitable forms to be applied to the skin, or mucus membranes of the nose and throat, and can take the form of lotions, creams, ointments, liquid sprays or inhalants, drops, tinctures, lozenges, or throat paints. Such topical formulations further can include chemical compounds such as dimethylsulfoxide (DMSO) to facilitate surface penetration of the active ingredient. In other transdermal formulations, typically in patch-delivered formulations, the pharmaceutically active compound is formulated with one or more skin penetrants, such as 2-N-methyl-pyrrolidone (NMP) or Azone. A topical semi-solid ointment formulation typically contains a concentration of the active ingredient from about 1 to 20%, e.g., 5 to 10%, in a carrier such as a pharmaceutical cream base. \\n     For application to the eyes or ears, the compounds of the present invention can be presented in liquid or semi-liquid form formulated in hydrophobic or hydrophilic bases as ointments, creams, lotions, paints or powders. \\n     For rectal administration the compounds of the present invention can be administered in the form of suppositories admixed with conventional carriers such as cocoa butter, wax or other glyceride. \\n     Inhalation formulations can also readily be formulated. For inhalation, various powder and liquid formulations can be prepared. For aerosol preparations, a sterile formulation of the compound or salt form of the compound may be used in inhalers, such as metered dose inhalers, and nebulizers. Aerosolized forms may be especially useful for treating respiratory disorders. \\n     Alternatively, the compounds of the present invention can be in powder form for reconstitution in the appropriate pharmaceutically acceptable carrier at the time of delivery. \\n     The pharmaceutically active compound in the pharmaceutical compositions of the present invention can be provided as the salt of a variety of acids, including but not limited to hydrochloric, sulfuric, acetic, lactic, tartaric, malic, and succinic acid. Salts tend to be more soluble in aqueous or other protonic solvents than are the corresponding free base forms. \\n     After pharmaceutical compositions have been prepared, they are packaged in an appropriate container and labeled for treatment of an indicated condition. \\n     The active compound will be present in an amount effective to achieve the intended purpose. The determination of an effective dose is well within the capability of those skilled in the art. \\n     A “therapeutically effective dose” refers to that amount of active ingredient, for example OSP polypeptide, fusion protein, or fragments thereof, antibodies specific for OSP, agonists, antagonists or inhibitors of OSP, which ameliorates the signs or symptoms of the disease or prevent progression thereof; as would be understood in the medical arts, cure, although desired, is not required. \\n     The therapeutically effective dose of the pharmaceutical agents of the present invention can be estimated initially by in vitro tests, such as cell culture assays, followed by assay in model animals, usually mice, rats, rabbits, dogs, or pigs. The animal model can also be used to determine an initial preferred concentration range and route of administration. \\n     For example, the ED50 (the dose therapeutically effective in 50% of the population) and LD50 (the dose lethal to 50% of the population) can be determined in one or more cell culture of animal model systems. The dose ratio of toxic to therapeutic effects is the therapeutic index, which can be expressed as LD50/ED50. Pharmaceutical compositions that exhibit large therapeutic indices are preferred. \\n     The data obtained from cell culture assays and animal studies are used in formulating an initial dosage range for human use, and preferably provide a range of circulating concentrations that includes the ED50 with little or no toxicity. After administration, or between successive administrations, the circulating concentration of active agent varies within this range depending upon pharmacokinetic factors well known in the art, such as the dosage form employed, sensitivity of the patient, and the route of administration. \\n     The exact dosage will be determined by the practitioner, in light of factors specific to the subject requiring treatment. Factors that can be taken into account by the practitioner include the severity of the disease state, general health of the subject, age, weight, gender of the subject, diet, time and frequency of administration, drug combination(s), reaction sensitivities, and tolerance/response to therapy. Long-acting pharmaceutical compositions can be administered every 3 to 4 days, every week, or once every two weeks depending on half-life and clearance rate of the particular formulation. \\n     Normal dosage amounts may vary from 0.1 to 100,000 micrograms, up to a total dose of about 1 g, depending upon the route of administration. Where the therapeutic agent is a protein or antibody of the present invention, the therapeutic protein or antibody agent typically is administered at a daily dosage of 0.01 mg to 30 mg/kg of body weight of the patient (e.g., 1 mg/kg to 5 mg/kg). The pharmaceutical formulation can be administered in multiple doses per day, if desired, to achieve the total desired daily dose. \\n     Guidance as to particular dosages and methods of delivery is provided in the literature and generally available to practitioners in the art. Those skilled in the art will employ different formulations for nucleotides than for proteins or their inhibitors. Similarly, delivery of polynucleotides or polypeptides will be specific to particular cells, conditions, locations, etc. \\n     Conventional methods, known to those of ordinary skill in the art of medicine, can be used to administer the pharmaceutical formulation(s) of the present invention to the patient. The pharmaceutical compositions of the present invention can be administered alone, or in combination with other therapeutic agents or interventions. \\n     Therapeutic Methods \\n     The present invention further provides methods of treating subjects having defects in a gene of the invention, e.g., in expression, activity, distribution, localization, and/or solubility, which can manifest as a disorder of ovarian function. As used herein, “treating” includes all medically-acceptable types of therapeutic intervention, including palliation and prophylaxis (prevention) of disease. The term “treating” encompasses any improvement of a disease, including minor improvements. These methods are discussed below. \\n     Gene Therapy and Vaccines \\n     The isolated nucleic acids of the present invention can also be used to drive in vivo expression of the polypeptides of the present invention. In vivo expression can be driven from a vector, typically a viral vector, often a vector based upon a replication incompetent retrovirus, an adenovirus, or an adeno-associated virus (AAV), for the purpose of gene therapy. In vivo expression can also be driven from signals endogenous to the nucleic acid or from a vector, often a plasmid vector, such as pVAX1 (Invitrogen, Carlsbad, Calif., USA), for purpose of “naked” nucleic acid vaccination, as further described in U.S. Pat. Nos. 5,589,466; 5,679,647; 5,804,566; 5,830,877; 5,843,913; 5,880,104; 5,958,891; 5,985,847; 6,017,897; 6,110,898; 6,204,250, the disclosures of which are incorporated herein by reference in their entireties. For cancer therapy, it is preferred that the vector also be tumor-selective. See, e.g., Doronin et al.,  J. Virol.  75: 3314-24 (2001). \\n     In another embodiment of the therapeutic methods of the present invention, a therapeutically effective amount of a pharmaceutical composition comprising a nucleic acid molecule of the present invention is administered. The nucleic acid molecule can be delivered in a vector that drives expression of an OSP, fusion protein, or fragment thereof, or without such vector. Nucleic acid compositions that can drive expression of an OSP are administered, for example, to complement a deficiency in the native OSP, or as DNA vaccines. Expression vectors derived from virus, replication deficient retroviruses, adenovirus, adeno-associated (AAV) virus, herpes virus, or vaccinia virus can be used as can plasmids. See, e.g., Cid-Arregui, supra. In a preferred embodiment, the nucleic acid molecule encodes an OSP having the amino acid sequence of SEQ ID NO: 129-295, or a fragment, fusion protein, allelic variant or homolog thereof. \\n     In still other therapeutic methods of the present invention, pharmaceutical compositions comprising host cells that express an OSP, fusions, or fragments thereof can be administered. In such cases, the cells are typically autologous, so as to circumvent xenogeneic or allotypic rejection, and are administered to complement defects in OSP production or activity. In a preferred embodiment, the nucleic acid molecules in the cells encode an OSP having the amino acid sequence of SEQ ID NO: 129-295, or a fragment, fusion protein, allelic variant or homolog thereof. \\n     Antisense Administration \\n     Antisense nucleic acid compositions, or vectors that drive expression of an OSG antisense nucleic acid, are administered to downregulate transcription and/or translation of an OSG in circumstances in which excessive production, or production of aberrant protein, is the pathophysiologic basis of disease. \\n     Antisense compositions useful in therapy can have a sequence that is complementary to coding or to noncoding regions of an OSG. For example, oligonucleotides derived from the transcription initiation site, e.g., between positions −10 and +10 from the start site, are preferred. \\n     Catalytic antisense compositions, such as ribozymes, that are capable of sequence-specific hybridization to OSG transcripts, are also useful in therapy. See, e.g., Phylactou,  Adv. Drug Deliv. Rev.  44(2-3): 97-108 (2000); Phylactou et al.,  Hum. Mol. Genet.  7(10): 1649-53 (1998); Rossi,  Ciba Found. Symp.  209: 195-204 (1997); and Sigurdsson et al.,  Trends Biotechnol.  13(8): 286-9 (1995). \\n     Other nucleic acids useful in the therapeutic methods of the present invention are those that are capable of triplex helix formation in or near the OSG genomic locus. Such triplexing oligonucleotides are able to inhibit transcription. See, e.g., Intody et al.,  Nucleic Acids Res.  28(21): 4283-90 (2000); and McGuffie et al.,  Cancer Res.  60(14): 3790-9 (2000). Pharmaceutical compositions comprising such triplex forming oligos (TFOS) are administered in circumstances in which excessive production, or production of aberrant protein, is a pathophysiologic basis of disease. \\n     In a preferred embodiment, the antisense molecule is derived from a nucleic acid molecule encoding an OSP, preferably an OSP comprising an amino acid sequence of SEQ ID NO: 129-295, or a fragment, allelic variant or homolog thereof. In a more preferred embodiment, the antisense molecule is derived from a nucleic acid molecule having a nucleotide sequence of SEQ ID NO: 1-128, or a part, allelic variant, substantially similar or hybridizing nucleic acid thereof. \\n     Polypeptide Administration \\n     In one embodiment of the therapeutic methods of the present invention, a therapeutically effective amount of a pharmaceutical composition comprising an OSP, a fusion protein, fragment, analog or derivative thereof is administered to a subject with a clinically-significant OSP defect. \\n     Protein compositions are administered, for example, to complement a deficiency in native OSP. In other embodiments, protein compositions are administered as a vaccine to elicit a humoral and/or cellular immune response to OSP. The immune response can be used to modulate activity of OSP or, depending on the immunogen, to immunize against aberrant or aberrantly expressed forms, such as mutant or inappropriately expressed isoforms. In yet other embodiments, protein fusions having a toxic moiety are administered to ablate cells that aberrantly accumulate OSP. \\n     In a preferred embodiment, the polypeptide administered is an OSP comprising an amino acid sequence of SEQ ID NO: 129-295, or a fusion protein, allelic variant, homolog, analog or derivative thereof. In a more preferred embodiment, the polypeptide is encoded by a nucleic acid molecule having a nucleotide sequence of SEQ ID NO: 1-128, or a part, allelic variant, substantially similar or hybridizing nucleic acid thereof. \\n     Antibody, Agonist and Antagonist Administration \\n     In another embodiment of the therapeutic methods of the present invention, a therapeutically effective amount of a pharmaceutical composition comprising an antibody (including fragment or derivative thereof) of the present invention is administered. As is well known, antibody compositions are administered, for example, to antagonize activity of OSP, or to target therapeutic agents to sites of OSP presence and/or accumulation. In a preferred embodiment, the antibody specifically binds to an OSP comprising an amino acid sequence of SEQ ID NO: 129-295, or a fusion protein, allelic variant, homolog, analog or derivative thereof. In a more preferred embodiment, the antibody specifically binds to an OSP encoded by a nucleic acid molecule having a nucleotide sequence of SEQ ID NO: 1-128, or a part, allelic variant, substantially similar or hybridizing nucleic acid thereof. \\n     The present invention also provides methods for identifying modulators which bind to an OSP or have a modulatory effect on the expression or activity of an OSP. Modulators which decrease the expression or activity of OSP (antagonists) are believed to be useful in treating ovarian cancer. Such screening assays are known to those of skill in the art and include, without limitation, cell-based assays and cell-free assays. Small molecules predicted via computer imaging to specifically bind to regions of an OSP can also be designed, synthesized and tested for use in the imaging and treatment of ovarian cancer. Further, libraries of molecules can be screened for potential anticancer agents by assessing the ability of the molecule to bind to the OSPs identified herein. Molecules identified in the library as being capable of binding to an OSP are key candidates for further evaluation for use in the treatment of ovarian cancer. In a preferred embodiment, these molecules will downregulate expression and/or activity of an OSP in cells. \\n     In another embodiment of the therapeutic methods of the present invention, a pharmaceutical composition comprising a non-antibody antagonist of OSP is administered. Antagonists of OSP can be produced using methods generally known in the art. In particular, purified OSP can be used to screen libraries of pharmaceutical agents, often combinatorial libraries of small molecules, to identify those that specifically bind and antagonize at least one activity of an OSP. \\n     In other embodiments a pharmaceutical composition comprising an agonist of an OSP is administered. Agonists can be identified using methods analogous to those used to identify antagonists. \\n     In a preferred embodiment, the antagonist or agonist specifically binds to and antagonizes or agonizes, respectively, an OSP comprising an amino acid sequence of SEQ ID NO: 129-295, or a fusion protein, allelic variant, homolog, analog or derivative thereof. In a more preferred embodiment, the antagonist or agonist specifically binds to and antagonizes or agonizes, respectively, an OSP encoded by a nucleic acid molecule having a nucleotide sequence of SEQ ID NO: 1-128, or a part, allelic variant, substantially similar or hybridizing nucleic acid thereof. \\n     Targeting Ovarian Tissue \\n     The invention also provides a method in which a polypeptide of the invention, or an antibody thereto, is linked to a therapeutic agent such that it can be delivered to the ovarian or to specific cells in the ovarian. In a preferred embodiment, an anti-OSP antibody is linked to a therapeutic agent and is administered to a patient in need of such therapeutic agent. The therapeutic agent may be a toxin, if ovarian tissue needs to be selectively destroyed. This would be useful for targeting and killing ovarian cancer cells. In another embodiment, the therapeutic agent may be a growth or differentiation factor, which would be useful for promoting ovarian cell function. \\n     In another embodiment, an anti-OSP antibody may be linked to an imaging agent that can be detected using, e.g., magnetic resonance imaging, CT or PET. This would be useful for determining and monitoring ovarian function, identifying ovarian cancer tumors, and identifying noncancerous ovarian diseases. \\n     EXAMPLES \\n     Example 1a \\n     Alternative Splice Variants \\n     We identified gene transcripts using the Gencarta™ tools (Compugen Ltd., Tel Aviv, Israel) and a variety of public and proprietary databases. These splice variants are either sequences which differ from a previously defined sequence or new uses of known sequences. In general related variants are annotated as DEX0455_XXX.nt.1, DEX0455_XXX.nt.2, DEX0455_XXX.nt.3, etc. The variant DNA sequences encode proteins which differ from a previously defined protein sequence. In relation to the nucleotide sequence naming convention, protein variants are annotated as DEX0455_XXX.aa.1, DEX0455_XXX.aa.2, etc., wherein transcript DEX0455_XXX.nt.1 encodes protein DEX0455_XXX.aa.1. A single transcript may encode a protein from an alternate Open Reading Fram (ORF) which is designated DEX0455_XXX.orf.1. Additionally, multiple transcripts may encode for a single protein. In this case, DEX0455_XXX.nt.1 and DEX0455_XXX.nt.2 will both be associated with DEX0455_XXX.aa.1. \\n     The mapping of the nucleic acid (“NT”) SEQ ID NO; DEX ID; chromosomal location (if known); open reading frame (ORF) location; amino acid (“AA”) SEQ ID NO; AA DEX ID; are shown in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 SEQ \\n                   \\n                   \\n                   \\n                 SEQ \\n                   \\n               \\n               \\n                 ID \\n                   \\n                   \\n                   \\n                 ID \\n               \\n               \\n                 NO \\n                 DEX ID \\n                 Chromo Map \\n                 ORF Loc \\n                 NO \\n                 DEX ID \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 1 \\n                 DEX0455_001.nt.1 \\n                 X; 47722965-47733965 \\n                 1624-2937 \\n                 129 \\n                 DEX0455_001.orf.1 \\n               \\n               \\n                 1 \\n                 DEX0455_001.nt.1 \\n                 X; 47722965-47733965 \\n                 \\u2002322-1035 \\n                 130 \\n                 DEX0455_001.aa.1 \\n               \\n               \\n                 2 \\n                 DEX0455_002.nt.1 \\n                 17q11.1 \\n                 217-915 \\n                 131 \\n                 DEX0455_002.aa.1 \\n               \\n               \\n                 3 \\n                 DEX0455_003.nt.1 \\n                 10p11.23 \\n                 132-476 \\n                 132 \\n                 DEX0455_003.aa.1 \\n               \\n               \\n                 4 \\n                 DEX0455_004.nt.1 \\n                 3q29 \\n                 7357-7809 \\n                 133 \\n                 DEX0455_004.orf.1 \\n               \\n               \\n                 4 \\n                 DEX0455_004.nt.1 \\n                 3q29 \\n                 2974-5074 \\n                 134 \\n                 DEX0455_004.aa.1 \\n               \\n               \\n                 5 \\n                 DEX0455_004.nt.2 \\n                 3q29 \\n                 6201-6653 \\n                 135 \\n                 DEX0455_004.orf.2 \\n               \\n               \\n                 5 \\n                 DEX0455_004.nt.2 \\n                 3q29 \\n                 2968-5257 \\n                 136 \\n                 DEX0455_004.aa.2 \\n               \\n               \\n                 6 \\n                 DEX0455_005.nt.1 \\n                 2q13 \\n                 \\u2002854-1267 \\n                 137 \\n                 DEX0455_005.aa.1 \\n               \\n               \\n                 7 \\n                 DEX0455_005.nt.2 \\n                 2q13 \\n                 \\u2002853-1270 \\n                 137 \\n                 DEX0455_005.aa.1 \\n               \\n               \\n                 8 \\n                 DEX0455_006.nt.1 \\n                 9q22.1 \\n                 \\u2002730-1266 \\n                 138 \\n                 DEX0455_006.aa.1 \\n               \\n               \\n                 9 \\n                 DEX0455_007.nt.1 \\n                 19q13.41 \\n                 \\u20031-882 \\n                 139 \\n                 DEX0455_007.orf.1 \\n               \\n               \\n                 9 \\n                 DEX0455_007.nt.1 \\n                 19q13.41 \\n                 \\u20031-885 \\n                 140 \\n                 DEX0455_007.aa.1 \\n               \\n               \\n                 10 \\n                 DEX0455_008.nt.1 \\n                 4q27 \\n                 \\u2002869-1138 \\n                 141 \\n                 DEX0455_008.aa.1 \\n               \\n               \\n                 11 \\n                 DEX0455_009.nt.1 \\n                 20p12.2 \\n                 \\u2003\\u20021-1123 \\n                 142 \\n                 DEX0455_009.aa.1 \\n               \\n               \\n                 12 \\n                 DEX0455_010.nt.1 \\n                 Un_6; 1484154-1498876 \\n                 341-784 \\n                 143 \\n                 DEX0455_010.orf.1 \\n               \\n               \\n                 12 \\n                 DEX0455_010.nt.1 \\n                 Un_6; 1484154-1498876 \\n                 151-370 \\n                 144 \\n                 DEX0455_010.aa.1 \\n               \\n               \\n                 13 \\n                 DEX0455_010.nt.2 \\n                 Un_6; 1484154-1498876 \\n                 \\u200249-621 \\n                 145 \\n                 DEX0455_010.orf.2 \\n               \\n               \\n                 13 \\n                 DEX0455_010.nt.2 \\n                 Un_6; 1484154-1498876 \\n                 151-490 \\n                 146 \\n                 DEX0455_010.aa.2 \\n               \\n               \\n                 14 \\n                 DEX0455_011.nt.1 \\n                 19q13.31 \\n                 \\u20031-384 \\n                 147 \\n                 DEX0455_011.aa.1 \\n               \\n               \\n                 15 \\n                 DEX0455_012.nt.1 \\n                 1q32.1 \\n                 207-974 \\n                 148 \\n                 DEX0455_012.aa.1 \\n               \\n               \\n                 16 \\n                 DEX0455_012.nt.2 \\n                 1q32.1 \\n                 102-851 \\n                 149 \\n                 DEX0455_012.orf.2 \\n               \\n               \\n                 16 \\n                 DEX0455_012.nt.2 \\n                 1q32.1 \\n                 \\u2002206-1415 \\n                 150 \\n                 DEX0455_012.aa.2 \\n               \\n               \\n                 17 \\n                 DEX0455_013.nt.1 \\n                 12p12.3 \\n                 \\u200210-666 \\n                 151 \\n                 DEX0455_013.aa.1 \\n               \\n               \\n                 18 \\n                 DEX0455_013.nt.2 \\n                 12p12.3 \\n                 124-639 \\n                 152 \\n                 DEX0455_013.aa.2 \\n               \\n               \\n                 19 \\n                 DEX0455_014.nt.1 \\n                 1q42.2 \\n                 \\u2002880-1866 \\n                 153 \\n                 DEX0455_014.orf.1 \\n               \\n               \\n                 19 \\n                 DEX0455_014.nt.1 \\n                 1q42.2 \\n                 \\u200382-1870 \\n                 154 \\n                 DEX0455_014.aa.1 \\n               \\n               \\n                 20 \\n                 DEX0455_015.nt.1 \\n                 12q13.2 \\n                 \\u20031-255 \\n                 155 \\n                 DEX0455_015.aa.1 \\n               \\n               \\n                 21 \\n                 DEX0455_016.nt.1 \\n                 1p31.1 \\n                 104-868 \\n                 156 \\n                 DEX0455_016.aa.1 \\n               \\n               \\n                 22 \\n                 DEX0455_017.nt.1 \\n                 1p33 \\n                 102-623 \\n                 157 \\n                 DEX0455_017.aa.1 \\n               \\n               \\n                 23 \\n                 DEX0455_018.nt.1 \\n                 9q34.11 \\n                 \\u2002209-1270 \\n                 158 \\n                 DEX0455_018.aa.1 \\n               \\n               \\n                 24 \\n                 DEX0455_018.nt.2 \\n                 9q34.11 \\n                 \\u2002682-2148 \\n                 159 \\n                 DEX0455_018.aa.2 \\n               \\n               \\n                 25 \\n                 DEX0455_019.nt.1 \\n                 11q13.4 \\n                 \\u200266-926 \\n                 160 \\n                 DEX0455_019.aa.1 \\n               \\n               \\n                 26 \\n                 DEX0455_020.nt.1 \\n                 19p13.11 \\n                 365-793 \\n                 161 \\n                 DEX0455_020.aa.1 \\n               \\n               \\n                 27 \\n                 DEX0455_020.nt.2 \\n                 19p13.11 \\n                 \\u2002688-1035 \\n                 162 \\n                 DEX0455_020.orf.2 \\n               \\n               \\n                 27 \\n                 DEX0455_020.nt.2 \\n                 19p13.11 \\n                 474-678 \\n                 163 \\n                 DEX0455_020.aa.2 \\n               \\n               \\n                 28 \\n                 DEX0455_021.nt.1 \\n                 1p36.11 \\n                 175-486 \\n                 164 \\n                 DEX0455_021.orf.1 \\n               \\n               \\n                 28 \\n                 DEX0455_021.nt.1 \\n                 1p36.11 \\n                 \\u20031-250 \\n                 165 \\n                 DEX0455_021.aa.1 \\n               \\n               \\n                 29 \\n                 DEX0455_021.nt.2 \\n                 1p36.11 \\n                 \\u2002190-1269 \\n                 166 \\n                 DEX0455_021.aa.2 \\n               \\n               \\n                 30 \\n                 DEX0455_021.nt.3 \\n                 1p36.11 \\n                 \\u200346-1173 \\n                 167 \\n                 DEX0455_021.orf.3 \\n               \\n               \\n                 30 \\n                 DEX0455_021.nt.3 \\n                 1p36.11 \\n                 \\u2002189-1590 \\n                 168 \\n                 DEX0455_021.aa.3 \\n               \\n               \\n                 31 \\n                 DEX0455_021.nt.4 \\n                 1p36.11 \\n                 \\u2002190-1173 \\n                 169 \\n                 DEX0455_021.aa.4 \\n               \\n               \\n                 32 \\n                 DEX0455_022.nt.1 \\n                 19p13.12 \\n                 109-642 \\n                 170 \\n                 DEX0455_022.aa.1 \\n               \\n               \\n                 33 \\n                 DEX0455_022.nt.2 \\n                 19p13.12 \\n                 \\u200270-492 \\n                 171 \\n                 DEX0455_022.orf.2 \\n               \\n               \\n                 33 \\n                 DEX0455_022.nt.2 \\n                 19p13.12 \\n                 108-675 \\n                 172 \\n                 DEX0455_022.aa.2 \\n               \\n               \\n                 34 \\n                 DEX0455_022.nt.3 \\n                 19p13.12 \\n                 \\u200291-324 \\n                 173 \\n                 DEX0455_022.aa.3 \\n               \\n               \\n                 35 \\n                 DEX0455_023.nt.1 \\n                 7q11.21 \\n                 609-956 \\n                 174 \\n                 DEX0455_023.aa.1 \\n               \\n               \\n                 36 \\n                 DEX0455_024.nt.1 \\n                 2p13.3 \\n                 \\u2002486-1569 \\n                 175 \\n                 DEX0455_024.aa.1 \\n               \\n               \\n                 37 \\n                 DEX0455_024.nt.2 \\n                 2p13.3 \\n                 469-999 \\n                 176 \\n                 DEX0455_024.aa.2 \\n               \\n               \\n                 38 \\n                 DEX0455_025.nt.1 \\n                 17q24.3 \\n                 \\u2002475-1614 \\n                 177 \\n                 DEX0455_025.aa.1 \\n               \\n               \\n                 39 \\n                 DEX0455_025.nt.2 \\n                 17q24.3 \\n                 \\u2002328-1509 \\n                 178 \\n                 DEX0455_025.orf.2 \\n               \\n               \\n                 39 \\n                 DEX0455_025.nt.2 \\n                 17q24.3 \\n                 \\u2002474-2514 \\n                 179 \\n                 DEX0455_025.aa.2 \\n               \\n               \\n                 40 \\n                 DEX0455_025.nt.3 \\n                 17q24.3 \\n                 \\u2002474-1617 \\n                 177 \\n                 DEX0455_025.aa.1 \\n               \\n               \\n                 41 \\n                 DEX0455_025.nt.4 \\n                 17q24.3 \\n                 \\u2002474-1617 \\n                 177 \\n                 DEX0455_025.aa.1 \\n               \\n               \\n                 42 \\n                 DEX0455_026.nt.1 \\n                 2q32.2 \\n                 \\u20033-218 \\n                 180 \\n                 DEX0455_026.orf.1 \\n               \\n               \\n                 42 \\n                 DEX0455_026.nt.1 \\n                 2q32.2 \\n                 \\u20031-236 \\n                 181 \\n                 DEX0455_026.aa.1 \\n               \\n               \\n                 43 \\n                 DEX0455_027.nt.1 \\n                 2q24.3 \\n                 \\u2002986-1507 \\n                 182 \\n                 DEX0455_027.orf.1 \\n               \\n               \\n                 43 \\n                 DEX0455_027.nt.1 \\n                 2q24.3 \\n                 \\u200216-128 \\n                 183 \\n                 DEX0455_027.aa.1 \\n               \\n               \\n                 44 \\n                 DEX0455_028.nt.1 \\n                 9p24.3 \\n                 141-785 \\n                 184 \\n                 DEX0455_028.aa.1 \\n               \\n               \\n                 45 \\n                 DEX0455_029.nt.1 \\n                 9q21.11 \\n                 4134-4532 \\n                 185 \\n                 DEX0455_029.orf.1 \\n               \\n               \\n                 45 \\n                 DEX0455_029.nt.1 \\n                 9q21.11 \\n                 2985-5847 \\n                 186 \\n                 DEX0455_029.aa.1 \\n               \\n               \\n                 46 \\n                 DEX0455_029.nt.2 \\n                 9q21.11 \\n                 4562-5143 \\n                 187 \\n                 DEX0455_029.orf.2 \\n               \\n               \\n                 46 \\n                 DEX0455_029.nt.2 \\n                 9q21.11 \\n                 2962-5149 \\n                 188 \\n                 DEX0455_029.aa.2 \\n               \\n               \\n                 47 \\n                 DEX0455_030.nt.1 \\n                 16p11.2 \\n                 \\u2002188-1123 \\n                 189 \\n                 DEX0455_030.aa.1 \\n               \\n               \\n                 48 \\n                 DEX0455_030.nt.2 \\n                 16p11.2 \\n                 \\u200282-627 \\n                 190 \\n                 DEX0455_030.aa.2 \\n               \\n               \\n                 49 \\n                 DEX0455_031.nt.1 \\n                 12p13.31 \\n                 \\u2002135-1013 \\n                 191 \\n                 DEX0455_031.orf.1 \\n               \\n               \\n                 49 \\n                 DEX0455_031.nt.1 \\n                 12p13.31 \\n                 \\u2002248-2156 \\n                 192 \\n                 DEX0455_031.aa.1 \\n               \\n               \\n                 50 \\n                 DEX0455_031.nt.2 \\n                 12p13.31 \\n                 248-749 \\n                 193 \\n                 DEX0455_031.aa.2 \\n               \\n               \\n                 50 \\n                 DEX0455_031.nt.2 \\n                 12p13.31 \\n                 1325-2239 \\n                 194 \\n                 DEX0455_031.orf.2 \\n               \\n               \\n                 51 \\n                 DEX0455_031.nt.3 \\n                 12p13.31 \\n                 \\u20031-582 \\n                 195 \\n                 DEX0455_031.aa.3 \\n               \\n               \\n                 52 \\n                 DEX0455_032.nt.1 \\n                 7q31.1 \\n                 \\u200239-761 \\n                 196 \\n                 DEX0455_032.aa.1 \\n               \\n               \\n                 53 \\n                 DEX0455_033.nt.1 \\n                 1p34.1 \\n                 161-943 \\n                 197 \\n                 DEX0455_033.aa.1 \\n               \\n               \\n                 54 \\n                 DEX0455_034.nt.1 \\n                 15q21.1 \\n                 \\u2002197-1693 \\n                 198 \\n                 DEX0455_034.aa.1 \\n               \\n               \\n                 55 \\n                 DEX0455_034.nt.2 \\n                 15q21.1 \\n                 \\u2003\\u20021-1497 \\n                 198 \\n                 DEX0455_034.aa.1 \\n               \\n               \\n                 56 \\n                 DEX0455_034.nt.3 \\n                 15q21.1 \\n                 \\u2002197-1228 \\n                 199 \\n                 DEX0455_034.aa.3 \\n               \\n               \\n                 57 \\n                 DEX0455_034.nt.4 \\n                 15q21.1 \\n                 \\u2003\\u20022-1438 \\n                 200 \\n                 DEX0455_034.aa.4 \\n               \\n               \\n                 58 \\n                 DEX0455_035.nt.1 \\n                 10q22.1 \\n                 102-464 \\n                 201 \\n                 DEX0455_035.aa.1 \\n               \\n               \\n                 59 \\n                 DEX0455_035.nt.2 \\n                 10q22.1 \\n                 \\u2002755-1201 \\n                 202 \\n                 DEX0455_035.orf.2 \\n               \\n               \\n                 59 \\n                 DEX0455_035.nt.2 \\n                 10q22.1 \\n                 330-696 \\n                 203 \\n                 DEX0455_035.aa.2 \\n               \\n               \\n                 60 \\n                 DEX0455_035.nt.3 \\n                 10q22.1 \\n                 \\u2002634-1080 \\n                 204 \\n                 DEX0455_035.orf.3 \\n               \\n               \\n                 60 \\n                 DEX0455_035.nt.3 \\n                 10q22.1 \\n                 269-575 \\n                 205 \\n                 DEX0455_035.aa.3 \\n               \\n               \\n                 61 \\n                 DEX0455_036.nt.1 \\n                 19p13.2 \\n                 \\u200286-370 \\n                 206 \\n                 DEX0455_036.orf.1 \\n               \\n               \\n                 61 \\n                 DEX0455_036.nt.1 \\n                 19p13.2 \\n                 \\u200258-389 \\n                 207 \\n                 DEX0455_036.aa.1 \\n               \\n               \\n                 62 \\n                 DEX0455_036.nt.2 \\n                 19p13.2 \\n                 \\u2002295-4749 \\n                 208 \\n                 DEX0455_036.aa.2 \\n               \\n               \\n                 63 \\n                 DEX0455_036.nt.3 \\n                 19p13.2 \\n                 \\u20033-335 \\n                 209 \\n                 DEX0455_036.orf.3 \\n               \\n               \\n                 63 \\n                 DEX0455_036.nt.3 \\n                 19p13.2 \\n                 \\u200288-352 \\n                 210 \\n                 DEX0455_036.aa.3 \\n               \\n               \\n                 64 \\n                 DEX0455_036.nt.4 \\n                 19p13.2 \\n                 \\u200277-352 \\n                 211 \\n                 DEX0455_036.orf.4 \\n               \\n               \\n                 64 \\n                 DEX0455_036.nt.4 \\n                 19p13.2 \\n                 \\u20031-253 \\n                 212 \\n                 DEX0455_036.aa.4 \\n               \\n               \\n                 65 \\n                 DEX0455_037.nt.1 \\n                 9 \\n                 113-787 \\n                 213 \\n                 DEX0455_037.aa.1 \\n               \\n               \\n                 66 \\n                 DEX0455_037.nt.2 \\n                 9 \\n                 \\u2003\\u20022-1048 \\n                 214 \\n                 DEX0455_037.orf.2 \\n               \\n               \\n                 66 \\n                 DEX0455_037.nt.2 \\n                 9 \\n                 \\u2002112-1354 \\n                 215 \\n                 DEX0455_037.aa.2 \\n               \\n               \\n                 67 \\n                 DEX0455_037.nt.3 \\n                 9 \\n                 \\u2002113-1342 \\n                 216 \\n                 DEX0455_037.aa.3 \\n               \\n               \\n                 68 \\n                 DEX0455_037.nt.4 \\n                 9 \\n                 \\u20032-410 \\n                 217 \\n                 DEX0455_037.aa.4 \\n               \\n               \\n                 69 \\n                 DEX0455_037.nt.5 \\n                 9 \\n                 \\u20033-452 \\n                 218 \\n                 DEX0455_037.aa.5 \\n               \\n               \\n                 70 \\n                 DEX0455_037.nt.6 \\n                 9 \\n                 113-784 \\n                 219 \\n                 DEX0455_037.aa.6 \\n               \\n               \\n                 71 \\n                 DEX0455_037.nt.7 \\n                 9 \\n                 \\u2002113-1555 \\n                 220 \\n                 DEX0455_037.aa.7 \\n               \\n               \\n                 72 \\n                 DEX0455_038.nt.1 \\n                 20p12.1 \\n                 \\u2002298-3561 \\n                 221 \\n                 DEX0455_038.aa.1 \\n               \\n               \\n                 73 \\n                 DEX0455_038.nt.2 \\n                 20p12.1 \\n                 \\u2002298-3564 \\n                 221 \\n                 DEX0455_038.aa.1 \\n               \\n               \\n                 74 \\n                 DEX0455_038.nt.3 \\n                 20p12.1 \\n                 \\u2003\\u20021-1320 \\n                 222 \\n                 DEX0455_038.orf.3 \\n               \\n               \\n                 74 \\n                 DEX0455_038.nt.3 \\n                 20p12.1 \\n                 \\u2002298-1863 \\n                 223 \\n                 DEX0455_038.aa.3 \\n               \\n               \\n                 75 \\n                 DEX0455_039.nt.1 \\n                 19q13.2 \\n                 \\u20032-496 \\n                 224 \\n                 DEX0455_039.aa.1 \\n               \\n               \\n                 76 \\n                 DEX0455_039.nt.2 \\n                 19q13.2 \\n                 \\u20032-787 \\n                 225 \\n                 DEX0455_039.aa.2 \\n               \\n               \\n                 77 \\n                 DEX0455_040.nt.1 \\n                 19q13.2 \\n                 299-991 \\n                 226 \\n                 DEX0455_040.orf.1 \\n               \\n               \\n                 77 \\n                 DEX0455_040.nt.1 \\n                 19q13.2 \\n                 352-991 \\n                 227 \\n                 DEX0455_040.aa.1 \\n               \\n               \\n                 78 \\n                 DEX0455_040.nt.2 \\n                 19q13.2 \\n                 \\u2002770-1495 \\n                 228 \\n                 DEX0455_040.aa.2 \\n               \\n               \\n                 79 \\n                 DEX0455_041.nt.1 \\n                 20q11.23 \\n                 \\u200254-212 \\n                 229 \\n                 DEX0455_041.orf.1 \\n               \\n               \\n                 79 \\n                 DEX0455_041.nt.1 \\n                 20q11.23 \\n                 \\u20037-138 \\n                 230 \\n                 DEX0455_041.aa.1 \\n               \\n               \\n                 80 \\n                 DEX0455_041.nt.2 \\n                 20q11.23 \\n                 \\u200211-208 \\n                 231 \\n                 DEX0455_041.orf.2 \\n               \\n               \\n                 80 \\n                 DEX0455_041.nt.2 \\n                 20q11.23 \\n                 \\u20031-107 \\n                 232 \\n                 DEX0455_041.aa.2 \\n               \\n               \\n                 81 \\n                 DEX0455_042.nt.1 \\n                 4q22.1 \\n                 \\u200290-437 \\n                 233 \\n                 DEX0455_042.orf.1 \\n               \\n               \\n                 81 \\n                 DEX0455_042.nt.1 \\n                 4q22.1 \\n                 \\u200270-439 \\n                 234 \\n                 DEX0455_042.aa.1 \\n               \\n               \\n                 82 \\n                 DEX0455_043.nt.1 \\n                 1q42.12 \\n                 511-768 \\n                 235 \\n                 DEX0455_043.orf.1 \\n               \\n               \\n                 82 \\n                 DEX0455_043.nt.1 \\n                 1q42.12 \\n                 \\u20021-93 \\n                 236 \\n                 DEX0455_043.aa.1 \\n               \\n               \\n                 83 \\n                 DEX0455_043.nt.2 \\n                 1q42.12 \\n                 413-787 \\n                 237 \\n                 DEX0455_043.orf.2 \\n               \\n               \\n                 83 \\n                 DEX0455_043.nt.2 \\n                 1q42.12 \\n                 \\u20021-93 \\n                 236 \\n                 DEX0455_043.aa.1 \\n               \\n               \\n                 84 \\n                 DEX0455_043.nt.3 \\n                 1q42.12 \\n                 1220-1531 \\n                 238 \\n                 DEX0455_043.orf.3 \\n               \\n               \\n                 84 \\n                 DEX0455_043.nt.3 \\n                 1q42.12 \\n                 \\u20021-93 \\n                 236 \\n                 DEX0455_043.aa.1 \\n               \\n               \\n                 85 \\n                 DEX0455_044.nt.1 \\n                 17q25.3 \\n                 445-627 \\n                 239 \\n                 DEX0455_044.aa.1 \\n               \\n               \\n                 86 \\n                 DEX0455_045.nt.1 \\n                 16p12.3 \\n                 \\u20031-579 \\n                 240 \\n                 DEX0455_045.orf.1 \\n               \\n               \\n                 86 \\n                 DEX0455_045.nt.1 \\n                 16p12.3 \\n                 \\u20031-492 \\n                 241 \\n                 DEX0455_045.aa.1 \\n               \\n               \\n                 87 \\n                 DEX0455_046.nt.1 \\n                 17q21.32 \\n                 \\u2002709-1389 \\n                 242 \\n                 DEX0455_046.orf.1 \\n               \\n               \\n                 87 \\n                 DEX0455_046.nt.1 \\n                 17q21.32 \\n                 \\u2002802-1389 \\n                 243 \\n                 DEX0455_046.aa.1 \\n               \\n               \\n                 88 \\n                 DEX0455_047.nt.1 \\n                 8p23.1 \\n                 2887-3195 \\n                 244 \\n                 DEX0455_047.orf.1 \\n               \\n               \\n                 88 \\n                 DEX0455_047.nt.1 \\n                 8p23.1 \\n                 136-334 \\n                 245 \\n                 DEX0455_047.aa.1 \\n               \\n               \\n                 89 \\n                 DEX0455_047.nt.2 \\n                 8p23.1 \\n                 1091-1399 \\n                 246 \\n                 DEX0455_047.orf.2 \\n               \\n               \\n                 89 \\n                 DEX0455_047.nt.2 \\n                 8p23.1 \\n                 \\u200219-102 \\n                 247 \\n                 DEX0455_047.aa.2 \\n               \\n               \\n                 90 \\n                 DEX0455_048.nt.1 \\n                 X; 150645762-150649651 \\n                 \\u200284-545 \\n                 248 \\n                 DEX0455_048.aa.1 \\n               \\n               \\n                 91 \\n                 DEX0455_048.nt.2 \\n                 X; 150645762-150649651 \\n                 286-813 \\n                 249 \\n                 DEX0455_048.orf.2 \\n               \\n               \\n                 91 \\n                 DEX0455_048.nt.2 \\n                 X; 150645762-150649651 \\n                 \\u20031-817 \\n                 250 \\n                 DEX0455_048.aa.2 \\n               \\n               \\n                 92 \\n                 DEX0455_049.nt.1 \\n                 2p21 \\n                 1183-1986 \\n                 251 \\n                 DEX0455_049.aa.1 \\n               \\n               \\n                 93 \\n                 DEX0455_049.nt.2 \\n                 2p21 \\n                 \\u2002378-1403 \\n                 252 \\n                 DEX0455_049.aa.2 \\n               \\n               \\n                 94 \\n                 DEX0455_049.nt.3 \\n                 2p21 \\n                 \\u2002808-1527 \\n                 253 \\n                 DEX0455_049.aa.3 \\n               \\n               \\n                 95 \\n                 DEX0455_049.nt.4 \\n                 2p21 \\n                 \\u2003\\u20021-1170 \\n                 254 \\n                 DEX0455_049.aa.4 \\n               \\n               \\n                 96 \\n                 DEX0455_049.nt.5 \\n                 2p21 \\n                 \\u2002179-1120 \\n                 255 \\n                 DEX0455_049.aa.5 \\n               \\n               \\n                 97 \\n                 DEX0455_050.nt.1 \\n                 7p22.1 \\n                 186-551 \\n                 256 \\n                 DEX0455_050.orf.1 \\n               \\n               \\n                 97 \\n                 DEX0455_050.nt.1 \\n                 7p22.1 \\n                 \\u20031-149 \\n                 257 \\n                 DEX0455_050.aa.1 \\n               \\n               \\n                 98 \\n                 DEX0455_051.nt.1 \\n                 19 \\n                 \\u2003\\u20021-1788 \\n                 258 \\n                 DEX0455_051.aa.1 \\n               \\n               \\n                 99 \\n                 DEX0455_051.nt.2 \\n                 19 \\n                 \\u2003\\u20021-1224 \\n                 259 \\n                 DEX0455_051.aa.2 \\n               \\n               \\n                 100 \\n                 DEX0455_051.nt.3 \\n                 19 \\n                 \\u2003\\u20021-1410 \\n                 260 \\n                 DEX0455_051.aa.3 \\n               \\n               \\n                 101 \\n                 DEX0455_051.nt.4 \\n                 19 \\n                 \\u2003\\u20021-1224 \\n                 259 \\n                 DEX0455_051.aa.2 \\n               \\n               \\n                 101 \\n                 DEX0455_051.nt.4 \\n                 19 \\n                 \\u2003\\u20021-1422 \\n                 261 \\n                 DEX0455_051.orf.4 \\n               \\n               \\n                 102 \\n                 DEX0455_051.nt.5 \\n                 19 \\n                 \\u2003\\u20021-1224 \\n                 259 \\n                 DEX0455_051.aa.2 \\n               \\n               \\n                 102 \\n                 DEX0455_051.nt.5 \\n                 19 \\n                 \\u2003\\u20021-1422 \\n                 262 \\n                 DEX0455_051.orf.5 \\n               \\n               \\n                 103 \\n                 DEX0455_051.nt.6 \\n                 19 \\n                 \\u2003\\u20021-1224 \\n                 259 \\n                 DEX0455_051.aa.2 \\n               \\n               \\n                 103 \\n                 DEX0455_051.nt.6 \\n                 19 \\n                 \\u2003\\u20021-1422 \\n                 263 \\n                 DEX0455_051.orf.6 \\n               \\n               \\n                 104 \\n                 DEX0455_052.nt.1 \\n                 19 \\n                 \\u200313-1422 \\n                 264 \\n                 DEX0455_052.aa.1 \\n               \\n               \\n                 105 \\n                 DEX0455_052.nt.2 \\n                 19 \\n                 \\u200313-1518 \\n                 265 \\n                 DEX0455_052.aa.2 \\n               \\n               \\n                 106 \\n                 DEX0455_052.nt.3 \\n                 19 \\n                 \\u2002188-1831 \\n                 266 \\n                 DEX0455_052.aa.3 \\n               \\n               \\n                 107 \\n                 DEX0455_052.nt.4 \\n                 19 \\n                 100-930 \\n                 267 \\n                 DEX0455_052.aa.4 \\n               \\n               \\n                 108 \\n                 DEX0455_053.nt.1 \\n                 1p12 \\n                 \\u20031-846 \\n                 268 \\n                 DEX0455_053.aa.1 \\n               \\n               \\n                 109 \\n                 DEX0455_053.nt.2 \\n                 1p12 \\n                 \\u20031-177 \\n                 269 \\n                 DEX0455_053.aa.2 \\n               \\n               \\n                 109 \\n                 DEX0455_053.nt.2 \\n                 1p12 \\n                 \\u2002253-1008 \\n                 270 \\n                 DEX0455_053.aa.3 \\n               \\n               \\n                 110 \\n                 DEX0455_054.nt.1 \\n                 15q24.3 \\n                 1218-1682 \\n                 271 \\n                 DEX0455_054.orf.1 \\n               \\n               \\n                 110 \\n                 DEX0455_054.nt.1 \\n                 15q24.3 \\n                 1038-1362 \\n                 272 \\n                 DEX0455_054.aa.1 \\n               \\n               \\n                 111 \\n                 DEX0455_054.nt.2 \\n                 15q24.3 \\n                 410-874 \\n                 273 \\n                 DEX0455_054.orf.2 \\n               \\n               \\n                 111 \\n                 DEX0455_054.nt.2 \\n                 15q24.3 \\n                 122-554 \\n                 274 \\n                 DEX0455_054.aa.2 \\n               \\n               \\n                 112 \\n                 DEX0455_055.nt.1 \\n                 1 \\n                 \\u2002812-1570 \\n                 275 \\n                 DEX0455_055.aa.1 \\n               \\n               \\n                 113 \\n                 DEX0455_055.nt.2 \\n                 1 \\n                 \\u2002388-1470 \\n                 276 \\n                 DEX0455_055.aa.2 \\n               \\n               \\n                 114 \\n                 DEX0455_055.nt.3 \\n                 1p34.2 \\n                 402-902 \\n                 277 \\n                 DEX0455_055.aa.3 \\n               \\n               \\n                 115 \\n                 DEX0455_056.nt.1 \\n                 7q31.1 \\n                 \\u2002626-2533 \\n                 278 \\n                 DEX0455_056.orf.1 \\n               \\n               \\n                 115 \\n                 DEX0455_056.nt.1 \\n                 7q31.1 \\n                 \\u2002670-3283 \\n                 279 \\n                 DEX0455_056.aa.1 \\n               \\n               \\n                 116 \\n                 DEX0455_056.nt.2 \\n                 7q31.1 \\n                 \\u2002671-3043 \\n                 280 \\n                 DEX0455_056.aa.2 \\n               \\n               \\n                 117 \\n                 DEX0455_057.nt.1 \\n                 1q21.3 \\n                 146-511 \\n                 281 \\n                 DEX0455_057.orf.1 \\n               \\n               \\n                 117 \\n                 DEX0455_057.nt.1 \\n                 1q21.3 \\n                 \\u20031-513 \\n                 282 \\n                 DEX0455_057.aa.1 \\n               \\n               \\n                 118 \\n                 DEX0455_057.nt.2 \\n                 1q21.3 \\n                 405-681 \\n                 283 \\n                 DEX0455_057.aa.2 \\n               \\n               \\n                 119 \\n                 DEX0455_058.nt.1 \\n                 1q42.12 \\n                 1208-1405 \\n                 284 \\n                 DEX0455_058.orf.1 \\n               \\n               \\n                 119 \\n                 DEX0455_058.nt.1 \\n                 1q42.12 \\n                 315-513 \\n                 285 \\n                 DEX0455_058.aa.1 \\n               \\n               \\n                 120 \\n                 DEX0455_059.nt.1 \\n                 19p13.11 \\n                 \\u2002294-1382 \\n                 286 \\n                 DEX0455_059.orf.1 \\n               \\n               \\n                 120 \\n                 DEX0455_059.nt.1 \\n                 19p13.11 \\n                 \\u20031-352 \\n                 287 \\n                 DEX0455_059.aa.1 \\n               \\n               \\n                 121 \\n                 DEX0455_059.nt.2 \\n                 19p13.11 \\n                 \\u2002596-1093 \\n                 288 \\n                 DEX0455_059.orf.2 \\n               \\n               \\n                 121 \\n                 DEX0455_059.nt.2 \\n                 19p13.11 \\n                 \\u20031-352 \\n                 287 \\n                 DEX0455_059.aa.1 \\n               \\n               \\n                 122 \\n                 DEX0455_060.nt.1 \\n                 21q21.1 \\n                 \\u20033-623 \\n                 289 \\n                 DEX0455_060.aa.1 \\n               \\n               \\n                 123 \\n                 DEX0455_061.nt.1 \\n                 10q11.21 \\n                 1564-2619 \\n                 290 \\n                 DEX0455_061.aa.1 \\n               \\n               \\n                 124 \\n                 DEX0455_061.nt.2 \\n                 10q11.21 \\n                 2449-3231 \\n                 291 \\n                 DEX0455_061.aa.2 \\n               \\n               \\n                 125 \\n                 DEX0455_061.nt.3 \\n                 10q11.21 \\n                 2449-3255 \\n                 292 \\n                 DEX0455_061.aa.3 \\n               \\n               \\n                 126 \\n                 DEX0455_061.nt.4 \\n                 10q11.21 \\n                 1045-1443 \\n                 293 \\n                 DEX0455_061.aa.4 \\n               \\n               \\n                 127 \\n                 DEX0455_061.nt.5 \\n                 10q11.21 \\n                 \\u2002842-1330 \\n                 294 \\n                 DEX0455_061.orf.5 \\n               \\n               \\n                 127 \\n                 DEX0455_061.nt.5 \\n                 10q11.21 \\n                 1740-2142 \\n                 293 \\n                 DEX0455_061.aa.4 \\n               \\n               \\n                 128 \\n                 DEX0455_062.nt.1 \\n                 2p25.1 \\n                 \\u2002120-1592 \\n                 295 \\n                 DEX0455_062.aa.1 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     The polypeptides of the present invention were analyzed and the following attributes were identified; specifically, epitopes, post translational modifications, signal peptides and transmembrane domains. Antigenicity (Epitope) prediction was performed through the antigenic module in the EMBOSS package. Rice, P., EMBOSS: The European Molecular Biology Open Software Suite,  Trends in Genetics  16(6): 276-277 (2000). The antigenic module predicts potentially antigenic regions of a protein sequence, using the method of Kolaskar and Tongaonkar. Kolaskar, A S and Tongaonkar, P C., A semi-empirical method for prediction of antigenic determinants on protein antigens,  FEBS Letters  276: 172-174 (1990). Examples of post-translational modifications (PTMs) and other motifs of the OSPs of this invention are listed below. In addition, antibodies that specifically bind such post-translational modifications may be useful as a diagnostic or as therapeutic. The PTMs and other motifs were predicted by using the ProSite Dictionary of Proteins Sites and Patterns (Bairoch et al.,  Nucleic Acids Res.  25(1):217-221 (1997)), the following motifs, including PTMs, were predicted for the OSPs of the invention. The signal peptides were detected by using the SignalP 2.0, see Nielsen et al.,  Protein Engineering  12, 3-9 (1999). Prediction of transmembrane helices in proteins was performed by the application TMHMM 2.0, “currently the best performing transmembrane prediction program”, according to authors (Krogh et al.,  Journal of Molecular Biology,  305(3):567-580, (2001); Moller et al.,  Bioinformatics,  17(7):646-653, (2001); Sonnhammer, et al.,  A hidden Markov model for predicting transmembrane helices in protein sequences  in Glasgow, et al. Ed.  Proceedings of the Sixth International Conference on Intelligent Systems for Molecular Biology , pages 175-182, Menlo Park, Calif., 1998. AAAI Press. The PSORT II program may also be used to predict cellular localizations. Horton et al.,  Intelligent Systems for Molecular Biology  5: 147-152 (1997). The table below includes the following sequence annotations: Signal peptide presence; TM (number of membrane domain, topology in orientation and position); Amino acid location and antigenic index (location, AI score); PTM and other motifs (type, amino acid residue locations); and functional domains (type, amino acid residue locations). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 DEX ID \\n                 Sig P \\n                 TMHMM \\n                 Antigenicity \\n                 PTM \\n                 Domains \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 DEX0455_001.orf.1 \\n                 N \\n                 0 - \\n                 333-345, \\n                 CK2_PHOSPHO_SITE 104-107; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                 o1-438; \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 284-286; \\n                 111-128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 155-162, \\n                 CAMP_PHOSPHO_SITE 85-88; \\n                 PRO_RICH_2 185-351; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; \\n                 CK2_PHOSPHO_SITE 352-355; \\n                 PRICHEXTENSN 1-13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 184-208, \\n                 CK2_PHOSPHO_SITE 43-46; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.14; 231-239, \\n                 PKC_PHOSPHO_SITE 430-432; \\n                 63-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n                 MYRISTYL 138-143; MYRISTYL \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 56-63, \\n                 265-270; ASN_GLYCOSYLATION \\n                 206-231; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 167-170; PKC_PHOSPHO_SITE \\n                 PRO_RICH_1 1-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 170-176, \\n                 216-218; MYRISTYL 260-265; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.117; 38-47, \\n                 TYR_PHOSPHO_SITE 200-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.115; \\n                 MYRISTYL 179-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 75-81, \\n                 PKC_PHOSPHO_SITE 84-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 PKC_PHOSPHO_SITE 153-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 108-119, \\n                 PKC_PHOSPHO_SITE 163-165; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; 305-311, \\n                 CK2_PHOSPHO_SITE 284-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.041; \\n                 MYRISTYL 267-272; \\n               \\n               \\n                   \\n                   \\n                   \\n                 273-283, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; 241-256, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; 4-36, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 323-329, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; 290-302, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 383-435, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-131, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; 211-217, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.039; \\n               \\n               \\n                 DEX0455_001.aa.1 \\n                 N \\n                 0 - \\n                 164-217, \\n                 MYRISTYL 194-199; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 o1-237; \\n                 1.189; 14-30, \\n                 5-10; PKC_PHOSPHO_SITE 88-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n                 PKC_PHOSPHO_SITE 47-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                 79-85, \\n                 CK2_PHOSPHO_SITE 41-44; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.054; 64-74, \\n                 LEUCINE_ZIPPER 157-178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 MYRISTYL 198-203; \\n               \\n               \\n                   \\n                   \\n                   \\n                 148-161, \\n                 PKC_PHOSPHO_SITE 29-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; 89-114, \\n                 MYRISTYL 187-192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n                 PKC_PHOSPHO_SITE 235-237; \\n               \\n               \\n                   \\n                   \\n                   \\n                 116-138, \\n                 CK2_PHOSPHO_SITE 205-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.139; 43-61, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.159; \\n               \\n               \\n                   \\n                   \\n                   \\n                 33-39, 1.034; 4-11, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n               \\n               \\n                 DEX0455_002.aa.1 \\n                 N \\n                 0 - \\n                 174-212, \\n                 CK2_PHOSPHO_SITE 159-162; \\n                 TONB_DEPENDENT_REC_1 \\n               \\n               \\n                   \\n                   \\n                 o1-233; \\n                 1.218; 88-148, \\n                 MYRISTYL 139-144; MYRISTYL \\n                 1-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 112-117; MYRISTYL 83-88; \\n                 GALAPTIN 165-186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 153-158, \\n                 ASN_GLYCOSYLATION 47-50; \\n                 GLECT 103-233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 67-81, \\n                 MYRISTYL 181-186; MYRISTYL \\n                 Gal- \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n                 61-66; \\n                 bind_lectin \\n               \\n               \\n                   \\n                   \\n                   \\n                 46-65, \\n                   \\n                 104-233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; 9-44, 1.202; \\n               \\n               \\n                 DEX0455_003.aa.1 \\n                 N \\n                 0 - \\n                 9-18, \\n                 CAMP_PHOSPHO_SITE 21-24; \\n                 ARG_RICH 18-104; \\n               \\n               \\n                   \\n                   \\n                 o1-115; \\n                 1.083; 30-51, \\n                 CK2_PHOSPHO_SITE 24-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.14; \\n                 MYRISTYL 57-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 56-82, \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.155; \\n                 CK2_PHOSPHO_SITE 43-46; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-111, \\n                 MYRISTYL 25-30; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; \\n                 PKC_PHOSPHO_SITE 4-6; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 101-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 93-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 109-111; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 4-7; \\n               \\n               \\n                 DEX0455_004.orf.1 \\n                 N \\n                 1 - \\n                 112-143, \\n                 MYRISTYL 96-101; \\n               \\n               \\n                   \\n                   \\n                 o1-54; \\n                 1.199; 5-12, \\n                 PKC_PHOSPHO_SITE 145-147; \\n               \\n               \\n                   \\n                   \\n                 tm55-74; \\n                 1.074; \\n                 MYRISTYL 82-87; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 i75-151; \\n                 40-104, \\n                 105-110; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.27; 14-37, \\n                 25-27; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.097; \\n                 143-146; \\n               \\n               \\n                 DEX0455_004.aa.1 \\n                 N \\n                 2 - \\n                 245-261, \\n                 CK2_PHOSPHO_SITE 440-443; \\n                 RIBOSOMAL_S2_1 \\n               \\n               \\n                   \\n                   \\n                 o1-594; \\n                 1.17; 51-68, \\n                 CK2_PHOSPHO_SITE 36-39; \\n                 245-256; \\n               \\n               \\n                   \\n                   \\n                 tm595-617; \\n                 1.161; \\n                 PKC_PHOSPHO_SITE 641-643; \\n                 LYS_RICH 9-22; \\n               \\n               \\n                   \\n                   \\n                 i618-666; \\n                 429-437, \\n                 MYRISTYL 264-269; \\n               \\n               \\n                   \\n                   \\n                 tm667-689; \\n                 1.15; 202-222, \\n                 PKC_PHOSPHO_SITE 28-30; \\n               \\n               \\n                   \\n                   \\n                 o690-699; \\n                 1.139; \\n                 ASN_GLYCOSYLATION 471-474; \\n               \\n               \\n                   \\n                   \\n                   \\n                 533-553, \\n                 ASN_GLYCOSYLATION 628-631; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n                 MYRISTYL 591-596; \\n               \\n               \\n                   \\n                   \\n                   \\n                 102-115, \\n                 PKC_PHOSPHO_SITE 210-212; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 CAMP_PHOSPHO_SITE 15-18; \\n               \\n               \\n                   \\n                   \\n                   \\n                 186-193, \\n                 ASN_GLYCOSYLATION 569-572; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n                 CK2_PHOSPHO_SITE 140-143; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-140, \\n                 PKC_PHOSPHO_SITE 302-304; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.245; \\n                 MYRISTYL 557-562; \\n               \\n               \\n                   \\n                   \\n                   \\n                 333-341, \\n                 AMIDATION 19-22; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 ASN_GLYCOSYLATION 297-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 375-383, \\n                 AMIDATION 197-200; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.117; \\n                 CAMP_PHOSPHO_SITE 587-590; \\n               \\n               \\n                   \\n                   \\n                   \\n                 265-319, \\n                 CK2_PHOSPHO_SITE 542-545; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 420-422; \\n               \\n               \\n                   \\n                   \\n                   \\n                 142-174, \\n                 PKC_PHOSPHO_SITE 70-72; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.138; 70-98, \\n                 MYRISTYL 530-535; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n                 PKC_PHOSPHO_SITE 560-562; \\n               \\n               \\n                   \\n                   \\n                   \\n                 345-354, \\n                 MYRISTYL 256-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; \\n                 CAMP_PHOSPHO_SITE 16-19; \\n               \\n               \\n                   \\n                   \\n                   \\n                 579-588, \\n                 MYRISTYL 625-630; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.236; \\n                 PKC_PHOSPHO_SITE 527-529; \\n               \\n               \\n                   \\n                   \\n                   \\n                 445-458, \\n                 MYRISTYL 93-98; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.242; \\n                 PKC_PHOSPHO_SITE 19-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                 515-531, \\n                 PKC_PHOSPHO_SITE 514-516; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 TYR_PHOSPHO_SITE 149-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 398-405, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n               \\n               \\n                   \\n                   \\n                   \\n                 690-696, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n               \\n               \\n                   \\n                   \\n                   \\n                 593-617, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n               \\n               \\n                   \\n                   \\n                   \\n                 224-243, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 629-688, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.194; \\n               \\n               \\n                   \\n                   \\n                   \\n                 408-413, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.058; \\n               \\n               \\n                   \\n                   \\n                   \\n                 475-512, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n               \\n               \\n                   \\n                   \\n                   \\n                 462-473, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.227; 23-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.175; \\n               \\n               \\n                   \\n                   \\n                   \\n                 363-369, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.038; \\n               \\n               \\n                 DEX0455_004.orf.2 \\n                 N \\n                 1 - \\n                 14-37, \\n                 CK2_PHOSPHO_SITE 143-146; \\n               \\n               \\n                   \\n                   \\n                 o1-54; \\n                 1.097; 40-104, \\n                 MYRISTYL 96-101; \\n               \\n               \\n                   \\n                   \\n                 tm55-74; \\n                 1.27; \\n                 PKC_PHOSPHO_SITE 25-27; \\n               \\n               \\n                   \\n                   \\n                 i75-151; \\n                 112-143, \\n                 MYRISTYL 105-110; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.199; 5-12, \\n                 82-87; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.074; \\n                 145-147; \\n               \\n               \\n                 DEX0455_004.aa.2 \\n                 N \\n                 9 - \\n                 713-724, \\n                 PKC_PHOSPHO_SITE 257-259; \\n                 N4_MTASE 609-614; \\n               \\n               \\n                   \\n                   \\n                 o1-210; \\n                 1.168; 19-28, \\n                 MYRISTYL 14-19; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 tm211-233; \\n                 1.088; \\n                 173-178; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 i234-282; \\n                 131-153, \\n                 676-679; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 tm283-305; \\n                 1.129; \\n                 620-622; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 o306-324; \\n                 746-755, \\n                 143-145; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                 tm325-347; \\n                 1.027; \\n                 418-421; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 i348-367; \\n                 209-233, \\n                 176-178; MYRISTYL 207-212; \\n               \\n               \\n                   \\n                   \\n                 tm368-385; \\n                 1.232; \\n                 ASN_GLYCOSYLATION 87-90; \\n               \\n               \\n                   \\n                   \\n                 o386-543; \\n                 604-653, \\n                 CAMP_PHOSPHO_SITE 203-206; \\n               \\n               \\n                   \\n                   \\n                 tm544-566; \\n                 1.189; 78-89, \\n                 MYRISTYL 161-166; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 i567-578; \\n                 1.227; \\n                 13-18; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                 tm579-599; \\n                 484-491, \\n                 493-496; MYRISTYL 691-696; \\n               \\n               \\n                   \\n                   \\n                 o600-613; \\n                 1.123; \\n                 TYR_PHOSPHO_SITE 755-761; \\n               \\n               \\n                   \\n                   \\n                 tm614-633; \\n                 402-459, \\n                 PKC_PHOSPHO_SITE 676-678; \\n               \\n               \\n                   \\n                   \\n                 i634-679; \\n                 1.205; \\n                 MYRISTYL 241-246; \\n               \\n               \\n                   \\n                   \\n                 tm680-699; \\n                 497-519, \\n                 ASN_GLYCOSYLATION 363-366; \\n               \\n               \\n                   \\n                   \\n                 o700-703; \\n                 1.232; \\n                 CK2_PHOSPHO_SITE 466-469; \\n               \\n               \\n                   \\n                   \\n                 tm704-721; \\n                 367-394, \\n                 PKC_PHOSPHO_SITE 594-596; \\n               \\n               \\n                   \\n                   \\n                 i722-762; \\n                 1.253; \\n                 ASN_GLYCOSYLATION 312-315; \\n               \\n               \\n                   \\n                   \\n                   \\n                 195-204, \\n                 ASN_GLYCOSYLATION 244-247; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.236; 91-128, \\n                 CK2_PHOSPHO_SITE 56-59; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 320-323; \\n               \\n               \\n                   \\n                   \\n                   \\n                 657-705, \\n                 CK2_PHOSPHO_SITE 506-509; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.254; \\n                 PKC_PHOSPHO_SITE 425-427; \\n               \\n               \\n                   \\n                   \\n                   \\n                 245-313, \\n                 PKC_PHOSPHO_SITE 721-723; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.194; \\n                 MYRISTYL 712-717; \\n               \\n               \\n                   \\n                   \\n                   \\n                 577-601, \\n                 PKC_PHOSPHO_SITE 130-132; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.285; \\n                 ASN_GLYCOSYLATION 185-188; \\n               \\n               \\n                   \\n                   \\n                   \\n                 555-569, \\n                 PKC_PHOSPHO_SITE 506-508; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.252; \\n               \\n               \\n                   \\n                   \\n                   \\n                 523-553, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 737-744, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 61-74, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.242; \\n               \\n               \\n                   \\n                   \\n                   \\n                 156-169, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.143; 45-53, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 323-353, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.237; \\n               \\n               \\n                   \\n                   \\n                   \\n                 472-480, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n               \\n               \\n                 DEX0455_005.aa.1 \\n                 N \\n                 0 - \\n                 19-29, \\n                 PKC_PHOSPHO_SITE 87-89; \\n               \\n               \\n                   \\n                   \\n                 o1-138; \\n                 1.121; \\n                 MYRISTYL 21-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 125-132, \\n                 PKC_PHOSPHO_SITE 131-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.063; 4-9, \\n                 MYRISTYL 102-107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.045; \\n                 CK2_PHOSPHO_SITE 38-41; \\n               \\n               \\n                   \\n                   \\n                   \\n                 45-61, \\n                 CK2_PHOSPHO_SITE 29-32; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; 88-107, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.22; \\n               \\n               \\n                   \\n                   \\n                   \\n                 76-85, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n               \\n               \\n                 DEX0455_006.aa.1 \\n                 N \\n                 0 - \\n                 34-45, \\n                 CAMP_PHOSPHO_SITE 62-65; \\n               \\n               \\n                   \\n                   \\n                 o1-179; \\n                 1.149; 92-99, \\n                 CK2_PHOSPHO_SITE 110-113; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 AMIDATION 138-141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 8-14, \\n                 CK2_PHOSPHO_SITE 67-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n                 TYR_PHOSPHO_SITE 88-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-135, \\n                 CK2_PHOSPHO_SITE 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.172; 49-61, \\n                 PKC_PHOSPHO_SITE 112-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.175; \\n                 MYRISTYL 36-41; \\n               \\n               \\n                   \\n                   \\n                   \\n                 168-176, \\n                 CAMP_PHOSPHO_SITE 127-130; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.097; \\n                 CK2_PHOSPHO_SITE 106-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-110, \\n                 MYRISTYL 133-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; 77-85, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n               \\n               \\n                   \\n                   \\n                   \\n                 143-155, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n               \\n               \\n                 DEX0455_007.orf.1 \\n                 N \\n                 0 - \\n                 4-23, \\n                 PKC_PHOSPHO_SITE 24-26; \\n               \\n               \\n                   \\n                   \\n                 o1-294; \\n                 1.139; \\n                 CK2_PHOSPHO_SITE 71-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 194-201, \\n                 MYRISTYL 231-236; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n                 CK2_PHOSPHO_SITE 203-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 133-176, \\n                 PKC_PHOSPHO_SITE 270-272; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; 39-45, \\n                 PKC_PHOSPHO_SITE 131-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 CK2_PHOSPHO_SITE 189-192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 58-68, \\n                 MYRISTYL 278-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 CK2_PHOSPHO_SITE 24-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 244-261, \\n                 MYRISTYL 201-206; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; 78-120, \\n                 266-271; MYRISTYL 120-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 56-59; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 132-135; \\n               \\n               \\n                 DEX0455_007.aa.1 \\n                 N \\n                 0 - \\n                 4-23, \\n                 MYRISTYL 266-271; \\n               \\n               \\n                   \\n                   \\n                 o1-294; \\n                 1.139; \\n                 CAMP_PHOSPHO_SITE 132-135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 133-176, \\n                 PKC_PHOSPHO_SITE 131-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n                 MYRISTYL 278-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 194-201, \\n                 CK2_PHOSPHO_SITE 24-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; 39-45, \\n                 MYRISTYL 120-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 CK2_PHOSPHO_SITE 189-192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 244-261, \\n                 MYRISTYL 201-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; 58-68, \\n                 CK2_PHOSPHO_SITE 71-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 PKC_PHOSPHO_SITE 270-272; \\n               \\n               \\n                   \\n                   \\n                   \\n                 78-120, \\n                 CK2_PHOSPHO_SITE 203-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 MYRISTYL 231-236; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 24-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 56-59; \\n               \\n               \\n                 DEX0455_008.aa.1 \\n                 N \\n                 0 - \\n                 43-78, \\n                 CK2_PHOSPHO_SITE 57-60; \\n                 HORMA 14-72; \\n               \\n               \\n                   \\n                   \\n                 o1-90; \\n                 1.171; 16-37, \\n                 PKC_PHOSPHO_SITE 79-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 MYRISTYL 36-41; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 81-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 12-14; \\n               \\n               \\n                 DEX0455_009.aa.1 \\n                 N \\n                 1 - \\n                 189-217, \\n                 MYRISTYL 335-340; MYRISTYL \\n                 LAMP_1 149-163; \\n               \\n               \\n                   \\n                   \\n                 o1-328; \\n                 1.126; \\n                 83-88; PKC_PHOSPHO_SITE \\n                 LYSASSOCTDMP \\n               \\n               \\n                   \\n                   \\n                 tm329-351; \\n                 139-145, \\n                 241-243; ASN_GLYCOSYLATION \\n                 139-163; \\n               \\n               \\n                   \\n                   \\n                 i352-373; \\n                 1.097; \\n                 146-149; MYRISTYL 34-39; \\n                 LYSASSOCTDMP \\n               \\n               \\n                   \\n                   \\n                   \\n                 258-270, \\n                 MYRISTYL 32-37; \\n                 267-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; 38-61, \\n                 PKC_PHOSPHO_SITE 228-230; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 AMIDATION 51-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 272-286, \\n                 PKC_PHOSPHO_SITE 4-6; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 CK2_PHOSPHO_SITE 195-198; \\n               \\n               \\n                   \\n                   \\n                   \\n                 156-177, \\n                 ASN_GLYCOSYLATION 220-223; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.157; 73-122, \\n                 PKC_PHOSPHO_SITE 13-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n                 CK2_PHOSPHO_SITE 134-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 231-240, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 289-321, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; 328-354, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-32, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-253, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n               \\n               \\n                 DEX0455_010.orf.1 \\n                 N \\n                 1 - \\n                 51-57, \\n                 MYRISTYL 57-62; MYRISTYL \\n                 IG_LIKE 8-112; \\n               \\n               \\n                   \\n                   \\n                 o1-109; \\n                 1.138; 78-98, \\n                 52-57; MYRISTYL 140-145; \\n                 IGc1 23-94; ig \\n               \\n               \\n                   \\n                   \\n                 tm110-132; \\n                 1.159; \\n                   \\n                 21-86; IG_MHC \\n               \\n               \\n                   \\n                   \\n                 i133-148; \\n                 111-132, \\n                   \\n                 82-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; 4-41, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.243; \\n               \\n               \\n                   \\n                   \\n                   \\n                 67-75, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                 DEX0455_010.aa.1 \\n                 N \\n                 0 - \\n                 4-12, \\n                 TYR_PHOSPHO_SITE 50-58; \\n                 sp_P13761_HB2J_HUMAN \\n               \\n               \\n                   \\n                   \\n                 o1-72; \\n                 1.148; 34-44, \\n                 CK2_PHOSPHO_SITE 37-40; \\n                 43-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n                 PKC_PHOSPHO_SITE 37-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                 55-69, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; 22-28, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.112; \\n               \\n               \\n                 DEX0455_010.orf.2 \\n                 Y \\n                 2 - \\n                 94-100, \\n                 MYRISTYL 24-29; MYRISTYL \\n                 IGc1 66-137; ig \\n               \\n               \\n                   \\n                   \\n                 o1-14; \\n                 1.138; 4-84, \\n                 95-100; MYRISTYL 100-105; \\n                 64-129; IG_MHC \\n               \\n               \\n                   \\n                   \\n                 tm15-37; \\n                 1.243; \\n                 MYRISTYL 183-188; \\n                 125-131; \\n               \\n               \\n                   \\n                   \\n                 i38-152; \\n                 154-175, \\n                   \\n                 IG_LIKE 51-155; \\n               \\n               \\n                   \\n                   \\n                 tm153-175; \\n                 1.193; \\n               \\n               \\n                   \\n                   \\n                 o176-191; \\n                 121-141, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.159; \\n               \\n               \\n                   \\n                   \\n                   \\n                 110-118, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                 DEX0455_010.aa.2 \\n                 N \\n                 0 - \\n                 87-103, \\n                 MYRISTYL 107-112; MYRISTYL \\n                 IG_LIKE 17-112; \\n               \\n               \\n                   \\n                   \\n                 o1-112; \\n                 1.09; 60-66, \\n                 61-66; MYRISTYL 66-71; \\n                 IGc1 32-103; ig \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.138; \\n                 CK2_PHOSPHO_SITE 106-109; \\n                 30-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-50, \\n                 MYRISTYL 96-101; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.243; 76-84, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                 DEX0455_011.aa.1 \\n                 N \\n                 0 - \\n                 78-97, \\n                 PKC_PHOSPHO_SITE 82-84; \\n                 GALAPTIN 61-81; \\n               \\n               \\n                   \\n                   \\n                 o1-128; \\n                 1.118; 99-115, \\n                 ASN_GLYCOSYLATION 48-51; \\n                 SUI1_1 115-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n                 CAMP_PHOSPHO_SITE 66-69; \\n                 GLECT 4-128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 7-33, \\n                 MYRISTYL 59-64; \\n                 Gal-bind_lectin \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; 37-43, \\n                 ASN_GLYCOSYLATION 58-61; \\n                 1-128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n                 CK2_PHOSPHO_SITE 34-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                 118-125, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.159; \\n               \\n               \\n                 DEX0455_012.aa.1 \\n                 N \\n                 0 - \\n                 136-143, \\n                 CK2_PHOSPHO_SITE 20-23; \\n                 SAM_PNT 48-132; \\n               \\n               \\n                   \\n                   \\n                 o1-256; \\n                 1.08; 193-199, \\n                 CK2_PHOSPHO_SITE 215-218; \\n                 SAM_PNT 48-132; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 MYRISTYL 202-207; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-189, \\n                 CK2_PHOSPHO_SITE 132-135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; 69-82, \\n                 CK2_PHOSPHO_SITE 131-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                 MYRISTYL 200-205; \\n               \\n               \\n                   \\n                   \\n                   \\n                 38-45, \\n                 CK2_PHOSPHO_SITE 93-96; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; 87-93, \\n                 MYRISTYL 53-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 CK2_PHOSPHO_SITE 88-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-9, 1.066; \\n                 MYRISTYL 182-187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 100-128, \\n                 CAMP_PHOSPHO_SITE 237-240; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; 26-34, \\n                 MYRISTYL 162-167; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 249-254; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 236-245, \\n                 54-56; MYRISTYL 99-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n               \\n               \\n                 DEX0455_012.orf.2 \\n                 N \\n                 0 - \\n                 4-10, 1.07; \\n                 CK2_PHOSPHO_SITE 166-169; \\n                 SAM_PNT 83-167; \\n               \\n               \\n                   \\n                   \\n                 o1-250; \\n                 104-117, \\n                 MYRISTYL 1-6; MYRISTYL \\n                 SAM_PNT 83-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                 134-139; MYRISTYL 217-222; \\n               \\n               \\n                   \\n                   \\n                   \\n                 240-246, \\n                 MYRISTYL 197-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                 CK2_PHOSPHO_SITE 167-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                 135-163, \\n                 MYRISTYL 88-93; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; 16-32, \\n                 CK2_PHOSPHO_SITE 238-241; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n                 PKC_PHOSPHO_SITE 89-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 61-69, \\n                 CK2_PHOSPHO_SITE 128-131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 PKC_PHOSPHO_SITE 11-13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 228-234, \\n                 MYRISTYL 28-33; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; 73-80, \\n                 22-27; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 123-126; MYRISTYL 33-38; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-128, \\n                 MYRISTYL 235-240; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 CK2_PHOSPHO_SITE 55-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 171-178, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 37-44, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n               \\n               \\n                   \\n                   \\n                   \\n                 211-224, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n               \\n               \\n                 DEX0455_012.aa.2 \\n                 N \\n                 0 - \\n                 176-189, \\n                 AMIDATION 273-276; \\n                 ETSDOMAIN 330-348; \\n               \\n               \\n                   \\n                   \\n                 o1-402; \\n                 1.099; 38-45, \\n                 MYRISTYL 53-58; AMIDATION \\n                 ETSDOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 377-380; CK2_PHOSPHO_SITE \\n                 349-367; \\n               \\n               \\n                   \\n                   \\n                   \\n                 288-294, \\n                 246-249; MYRISTYL 99-104; \\n                 ETSDOMAIN 304-317; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; 26-34, \\n                 CK2_PHOSPHO_SITE 224-227; \\n                 SAM_PNT \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 MYRISTYL 162-167; \\n                 48-132; AT_hook \\n               \\n               \\n                   \\n                   \\n                   \\n                 69-82, \\n                 CK2_PHOSPHO_SITE 203-206; \\n                 275-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                 PKC_PHOSPHO_SITE 54-56; \\n                 ETS_DOMAIN_3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-218, \\n                 ASN_GLYCOSYLATION 388-391; \\n                 304-386; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 CK2_PHOSPHO_SITE 213-216; \\n                 SAM_PNT 48-132; \\n               \\n               \\n                   \\n                   \\n                   \\n                 100-128, \\n                 CK2_PHOSPHO_SITE 88-91; \\n                 HSF_ETS 314-376; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 MYRISTYL 182-187; \\n                 ETSDOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 136-143, \\n                 ASN_GLYCOSYLATION 354-357; \\n                 368-386; Ets \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 311-318, \\n                 AMIDATION 293-296; \\n                 303-388; ETS \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 CK2_PHOSPHO_SITE 20-23; \\n                 303-390; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-9, 1.066; \\n                 CAMP_PHOSPHO_SITE 282-285; \\n               \\n               \\n                   \\n                   \\n                   \\n                 87-93, \\n                 CK2_PHOSPHO_SITE 93-96; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 CAMP_PHOSPHO_SITE 350-353; \\n               \\n               \\n                   \\n                   \\n                   \\n                 379-385, \\n                 CK2_PHOSPHO_SITE 132-135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n                 MYRISTYL 233-238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 193-199, \\n                 CK2_PHOSPHO_SITE 131-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 MYRISTYL 200-205; \\n               \\n               \\n                   \\n                   \\n                   \\n                 332-346, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n               \\n               \\n                 DEX0455_013.aa.1 \\n                 N \\n                 0 - \\n                 38-45, \\n                 CK2_PHOSPHO_SITE 37-40; \\n               \\n               \\n                   \\n                   \\n                 o1-219; \\n                 1.105; \\n                 MYRISTYL 49-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 183-198, \\n                 ASN_GLYCOSYLATION 46-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 MYRISTYL 30-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-146, \\n                 CK2_PHOSPHO_SITE 173-176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.173; \\n                 MYRISTYL 147-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 154-173, \\n                 PKC_PHOSPHO_SITE 200-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; 93-115, \\n                 MYRISTYL 100-105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 PKC_PHOSPHO_SITE 214-216; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; 50-60, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.202; \\n               \\n               \\n                 DEX0455_013.aa.2 \\n                 N \\n                 0 - \\n                 82-99 \\n                 CK2_PHOSPHO_SITE 126-129; \\n               \\n               \\n                   \\n                   \\n                 o1-172; \\n                 1.173; 4-10, \\n                 MYRISTYL 100-105; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 53-58; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 46-68, \\n                 167-169; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 153-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 136-151, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n               \\n               \\n                   \\n                   \\n                   \\n                 107-126, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; \\n               \\n               \\n                 DEX0455_014.orf.1 \\n                 N \\n                 0 - \\n                 4-10, \\n                 CK2_PHOSPHO_SITE 110-113; \\n                 EFh 204-232; \\n               \\n               \\n                   \\n                   \\n                 o1-329; \\n                 1.104; \\n                 CK2_PHOSPHO_SITE 24-27; \\n                 EF_HAND_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 140-146, \\n                 CAMP_PHOSPHO_SITE 198-201; \\n                 153-229; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; 76-82, \\n                 PKC_PHOSPHO_SITE 95-97; \\n                 EF_HAND 213-225; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.057; \\n                 PKC_PHOSPHO_SITE 308-310; \\n                 Calpain_III 1-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-125, \\n                 CK2_PHOSPHO_SITE 156-159; \\n                 EF_HAND \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; \\n                 AMIDATION 12-15; \\n                 243-255; efhand \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-192, \\n                 CK2_PHOSPHO_SITE 177-180; \\n                 204-232; EFh \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 PKC_PHOSPHO_SITE 204-206; \\n                 234-262; \\n               \\n               \\n                   \\n                   \\n                   \\n                 300-307, \\n                 MYRISTYL 29-34; \\n                 calpain_III 5-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 CK2_PHOSPHO_SITE 210-213; \\n                 efhand \\n               \\n               \\n                   \\n                   \\n                   \\n                 162-169, \\n                 PKC_PHOSPHO_SITE 126-128; \\n                 234-262; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 CK2_PHOSPHO_SITE 150-153; \\n                 EF_HAND_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 314-320, \\n                 CK2_PHOSPHO_SITE 251-254; \\n                 237-294; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.054; 52-65, \\n                 PKC_PHOSPHO_SITE 24-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.124; \\n                 ASN_GLYCOSYLATION 299-302; \\n               \\n               \\n                   \\n                   \\n                   \\n                 197-212, \\n                 CK2_PHOSPHO_SITE 22-25; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n                 MYRISTYL 248-253; \\n               \\n               \\n                   \\n                   \\n                   \\n                 235-242, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; 32-40, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 96-113, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 224-230, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n               \\n               \\n                   \\n                   \\n                   \\n                 18-24, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.096; \\n               \\n               \\n                   \\n                   \\n                   \\n                 255-278, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 290-298, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                 DEX0455_014.aa.1 \\n                 N \\n                 0 - \\n                 21-44, \\n                 CK2_PHOSPHO_SITE 153-156; \\n                 Calpain_III \\n               \\n               \\n                   \\n                   \\n                 o1-595; \\n                 1.113; \\n                 ASN_GLYCOSYLATION 143-146; \\n                 261-399; \\n               \\n               \\n                   \\n                   \\n                   \\n                 556-564, \\n                 PKC_PHOSPHO_SITE 290-292; \\n                 THIOL_PROTEASE_CYS \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; 49-73, \\n                 CK2_PHOSPHO_SITE 200-203; \\n                 53-64; EFh \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.155; \\n                 MYRISTYL 189-194; \\n                 470-498; \\n               \\n               \\n                   \\n                   \\n                   \\n                 75-82, \\n                 CK2_PHOSPHO_SITE 288-291; \\n                 CALPAIN 89-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 CK2_PHOSPHO_SITE 290-293; \\n                 efhand 500-528; \\n               \\n               \\n                   \\n                   \\n                   \\n                 179-185, \\n                 CK2_PHOSPHO_SITE 517-520; \\n                 EF_HAND_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; \\n                 CK2_PHOSPHO_SITE 250-253; \\n                 419-495; \\n               \\n               \\n                   \\n                   \\n                   \\n                 284-290, \\n                 ASN_GLYCOSYLATION 17-20; \\n                 Peptidase_C2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.096; \\n                 MYRISTYL 295-300; \\n                 14-273; \\n               \\n               \\n                   \\n                   \\n                   \\n                 342-348, \\n                 PKC_PHOSPHO_SITE 392-394; \\n                 CYS_PROT_CALPAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.057; \\n                 ASN_GLYCOSYLATION 248-251; \\n                 32-273; CysPc \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-125, \\n                 ASN_GLYCOSYLATION 565-568; \\n                 2-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.165; \\n                 PKC_PHOSPHO_SITE 252-254; \\n                 EF_HAND_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 103-110, \\n                 MYRISTYL 211-216; \\n                 503-560; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 221-224; \\n                 CALPAIN 53-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 566-573, \\n                 CK2_PHOSPHO_SITE 422-425; \\n                 EFh 500-528; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 CAMP_PHOSPHO_SITE 464-467; \\n                 EF_HAND 509-521; \\n               \\n               \\n                   \\n                   \\n                   \\n                 259-275, \\n                 MYRISTYL 164-169; MYRISTYL \\n                 CALPAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.108; \\n                 144-149; CK2_PHOSPHO_SITE \\n                 29-51; EF_HAND \\n               \\n               \\n                   \\n                   \\n                   \\n                 428-435, \\n                 114-117; CK2_PHOSPHO_SITE \\n                 479-491; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 47-50; PKC_PHOSPHO_SITE \\n                 calpain_III \\n               \\n               \\n                   \\n                   \\n                   \\n                 188-197, \\n                 114-116; MYRISTYL 45-50; \\n                 241-399; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.149; 86-98, \\n                 PKC_PHOSPHO_SITE 470-472; \\n                 CALPAIN 365-393; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 PKC_PHOSPHO_SITE 227-229; \\n                 efhand \\n               \\n               \\n                   \\n                   \\n                   \\n                 318-331, \\n                 CK2_PHOSPHO_SITE 19-22; \\n                 470-498; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.124; \\n                 PKC_PHOSPHO_SITE 361-363; \\n                 CALPAIN 119-142; \\n               \\n               \\n                   \\n                   \\n                   \\n                 447-458, \\n                 CK2_PHOSPHO_SITE 376-379; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 CK2_PHOSPHO_SITE 476-479; \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-143, \\n                 MYRISTYL 514-519; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.081; 4-11, \\n                 PKC_PHOSPHO_SITE 574-576; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.247; \\n                 CK2_PHOSPHO_SITE 443-446; \\n               \\n               \\n                   \\n                   \\n                   \\n                 362-379, \\n                 CK2_PHOSPHO_SITE 416-419; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 209-229, \\n                 CAMP_PHOSPHO_SITE 187-190; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 TYR_PHOSPHO_SITE 172-179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 490-496, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n               \\n               \\n                   \\n                   \\n                   \\n                 298-306, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 521-544, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 580-586, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.054; \\n               \\n               \\n                   \\n                   \\n                   \\n                 463-478, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n               \\n               \\n                   \\n                   \\n                   \\n                 501-508, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n               \\n               \\n                   \\n                   \\n                   \\n                 383-391, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; \\n               \\n               \\n                   \\n                   \\n                   \\n                 406-412, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n               \\n               \\n                 DEX0455_015.aa.1 \\n                 N \\n                 0 - \\n                 63-82, \\n                 CK2_PHOSPHO_SITE 42-45; \\n                 ER_TARGET 82-85; \\n               \\n               \\n                   \\n                   \\n                 o1-85; \\n                 1.15; 20-26, \\n                 PKC_PHOSPHO_SITE 35-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; \\n                 CK2_PHOSPHO_SITE 81-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-14, \\n                 CK2_PHOSPHO_SITE 19-22; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.174; 35-46, \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.138; \\n               \\n               \\n                 DEX0455_016.aa.1 \\n                 N \\n                 0 - \\n                 105-110, \\n                 MYRISTYL 24-29; MYRISTYL \\n                 KH 189-240; \\n               \\n               \\n                   \\n                   \\n                 o1-255; \\n                 1.047; \\n                 22-27; PKC_PHOSPHO_SITE \\n                 GLY_RICH 13-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 223-241, \\n                 207-209; MYRISTYL 152-157; \\n                 KH 99-169; KH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.155; 75-81, \\n                 MYRISTYL 21-26; MYRISTYL \\n                 184-239; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n                 225-230; MYRISTYL 198-203; \\n                 KH_TYPE_1_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 244-252, \\n                 MYRISTYL 16-21; MYRISTYL \\n                 100-164; KH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; 86-94, \\n                 17-22; MYRISTYL 18-23; \\n                 104-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.048; \\n                 ASN_GLYCOSYLATION 82-85; \\n                 KH_TYPE_1_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 145-151, \\n                 MYRISTYL 13-18; MYRISTYL \\n                 185-224; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.048; \\n                 20-25; MYRISTYL 46-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                 154-171, \\n                 MYRISTYL 26-31; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 19-24; MYRISTYL 50-55; \\n               \\n               \\n                   \\n                   \\n                   \\n                 185-202, \\n                 PKC_PHOSPHO_SITE 159-161; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; 4-12, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n               \\n               \\n                   \\n                   \\n                   \\n                 32-44, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; 129-138, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n               \\n               \\n                 DEX0455_017.aa.1 \\n                 N \\n                 0 - \\n                 82-95, \\n                 ASN_GLYCOSYLATION 29-32; \\n                 TM4_2 16-124; \\n               \\n               \\n                   \\n                   \\n                 o1-174; \\n                 1.136; \\n                 MYRISTYL 104-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                 112-129, \\n                 PKC_PHOSPHO_SITE 166-168; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.149; 98-110, \\n                 ASN_GLYCOSYLATION 42-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.124; \\n                 PKC_PHOSPHO_SITE 162-164; \\n               \\n               \\n                   \\n                   \\n                   \\n                 58-65, \\n                 CK2_PHOSPHO_SITE 70-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 PKC_PHOSPHO_SITE 31-33; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-155, \\n                 ASN_GLYCOSYLATION 72-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; 4-15, \\n                 MYRISTYL 105-110; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.221; \\n                 CK2_PHOSPHO_SITE 44-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 34-41, \\n                 MYRISTYL 34-39; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n                 112-117; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 66-69; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 20-23; \\n               \\n               \\n                 DEX0455_018.aa.1 \\n                 N \\n                 0 - \\n                 152-174, \\n                 MYRISTYL 174-179; \\n                 PROTEIN_KINASE_DOM \\n               \\n               \\n                   \\n                   \\n                 o1-354; \\n                 1.101; \\n                 CK2_PHOSPHO_SITE 62-65; \\n                 10-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-240, \\n                 PKC_PHOSPHO_SITE 334-336; \\n                 TYRKINASE 104-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; 67-74, \\n                 PKC_PHOSPHO_SITE 348-350; \\n                 S_TK_X \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n                 CK2_PHOSPHO_SITE 11-14; \\n                 284-348; \\n               \\n               \\n                   \\n                   \\n                   \\n                 289-311, \\n                 MYRISTYL 319-324; \\n                 sp_Q9UM03_Q9UM03_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.168; 57-63, \\n                 PKC_PHOSPHO_SITE 46-48; \\n                 37-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 CK2_PHOSPHO_SITE 228-231; \\n                 S_TKc 37—283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 33-44, \\n                 MYRISTYL 19-24; MYRISTYL \\n                 TyrKc 38-277; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.198; \\n                 18-23; TYR_PHOSPHO_SITE \\n                 PROTEIN_KINASE_ST \\n               \\n               \\n                   \\n                   \\n                   \\n                 207-225, \\n                 63-70; \\n                 145-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.149; \\n                   \\n                 TYRKINASE 139-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 276-282, \\n                   \\n                 TYRKINASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; 22-28, \\n                   \\n                 205-227; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.058; \\n                   \\n                 pkinase_C 284-351; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-150, \\n                   \\n                 pkinase \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.291; \\n                   \\n                 31-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-265, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 319-338, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 182-199, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n               \\n               \\n                   \\n                   \\n                   \\n                 344-350, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; 49-55, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.112; \\n               \\n               \\n                   \\n                   \\n                   \\n                 76-110, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.213; \\n               \\n               \\n                 DEX0455_018.aa.2 \\n                 N \\n                 0 - \\n                 356-363, \\n                 CK2_PHOSPHO_SITE 135-138; \\n                 HR1 105-181; \\n               \\n               \\n                   \\n                   \\n                 o1-489; \\n                 1.123; \\n                 PKC_PHOSPHO_SITE 60-62; \\n                 HR1 182-255; \\n               \\n               \\n                   \\n                   \\n                   \\n                 215-226, \\n                 PKC_PHOSPHO_SITE 434-436; \\n                 HR1 18-90; HR1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.053; \\n                 PKC_PHOSPHO_SITE 206-208; \\n                 18-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                 151-169, \\n                 MYRISTYL 442-447; \\n                 REM_REPEAT_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 PKC_PHOSPHO_SITE 45-47; \\n                 15-74; HR1 182-255; \\n               \\n               \\n                   \\n                   \\n                   \\n                 175-195, \\n                 CK2_PHOSPHO_SITE 307-310; \\n                 REM_REPEAT_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.157; \\n                 PKC_PHOSPHO_SITE 453-455; \\n                 107-166; HR1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 378-390, \\n                 LEUCINE_ZIPPER 50-71; \\n                 105-181; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 PKC_PHOSPHO_SITE 205-207; \\n                 REM_REPEAT_3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 103-122, \\n                 CK2_PHOSPHO_SITE 164-167; \\n                 175-239; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.165; 50-59, \\n                 LEUCINE_ZIPPER 222-243; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 ASN_GLYCOSYLATION 126-129; \\n               \\n               \\n                   \\n                   \\n                   \\n                 258-276, \\n                 MYRISTYL 452-457; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.096; \\n                 CK2_PHOSPHO_SITE 414-417; \\n               \\n               \\n                   \\n                   \\n                   \\n                 228-252, \\n                 MYRISTYL 407-412; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 PKC_PHOSPHO_SITE 226-228; \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-134, \\n                 PKC_PHOSPHO_SITE 135-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.041; \\n                 LEUCINE_ZIPPER 215-236; \\n               \\n               \\n                   \\n                   \\n                   \\n                 393-422, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.174; \\n               \\n               \\n                   \\n                   \\n                   \\n                 440-457, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 459-483, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.218; \\n               \\n               \\n                   \\n                   \\n                   \\n                 327-343, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.235; \\n               \\n               \\n                   \\n                   \\n                   \\n                 139-147, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 278-307, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; 88-94, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.072; \\n               \\n               \\n                   \\n                   \\n                   \\n                 65-83, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 198-206, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.174; \\n               \\n               \\n                 DEX0455_019.aa.1 \\n                 Y \\n                 0 - \\n                 209-234, \\n                 MYRISTYL 82-87; AMIDATION \\n                 Folate_rec 7-287; \\n               \\n               \\n                   \\n                   \\n                 o1-287; \\n                 1.163; \\n                 102-105; MYRISTYL 214-219; \\n               \\n               \\n                   \\n                   \\n                   \\n                 268-274, \\n                 PKC_PHOSPHO_SITE 26-28; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.108; \\n                 ASN_GLYCOSYLATION 243-246; \\n               \\n               \\n                   \\n                   \\n                   \\n                 249-254, \\n                 ASN_GLYCOSYLATION 163-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 PKC_PHOSPHO_SITE 238-240; \\n               \\n               \\n                   \\n                   \\n                   \\n                 171-180, \\n                 PKC_PHOSPHO_SITE 193-195; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.184; 65-73, \\n                 MYRISTYL 247-252; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n                 CK2_PHOSPHO_SITE 218-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 189-197, \\n                 PKC_PHOSPHO_SITE 165-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; 8-21, \\n                 ASN_GLYCOSYLATION 203-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.242; \\n                 CK2_PHOSPHO_SITE 48-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                 91-102, \\n                 PKC_PHOSPHO_SITE 248-250; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; 54-60, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-136, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; 33-40, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n               \\n               \\n                   \\n                   \\n                   \\n                 236-244, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n               \\n               \\n                   \\n                   \\n                   \\n                 109-127, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.096; \\n               \\n               \\n                   \\n                   \\n                   \\n                 140-154, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; \\n               \\n               \\n                 DEX0455_020.aa.1 \\n                 N \\n                 0 - \\n                 73-79, \\n                 AMIDATION 25-28; \\n               \\n               \\n                   \\n                   \\n                 o1-143; \\n                 1.044; \\n                 CK2_PHOSPHO_SITE 99-102; \\n               \\n               \\n                   \\n                   \\n                   \\n                 101-120, \\n                 MYRISTYL 126-131; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.206; \\n                 79-84; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-140, \\n                 71-73; MYRISTYL 119-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; 27-32, \\n                 MYRISTYL 75-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.039; \\n                 PKC_PHOSPHO_SITE 72-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 83-93, \\n                 ASN_GLYCOSYLATION 97-100; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; 4-18, \\n                 PKC_PHOSPHO_SITE 136-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n                 MYRISTYL 68-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 46-67, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n               \\n               \\n                 DEX0455_020.orf.2 \\n                 N \\n                 0 - \\n                 32-64, \\n                 MYRISTYL 106-111; \\n               \\n               \\n                   \\n                   \\n                 o1-116; \\n                 1.153; 84-90, \\n                 CK2_PHOSPHO_SITE 34-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n                 PKC_PHOSPHO_SITE 12-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                 96-102, \\n                 MYRISTYL 6-11; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 13-21, \\n                 29-34; MYRISTYL 68-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.109; \\n                 AMIDATION 61-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 78-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 92-97; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 25-28; \\n               \\n               \\n                 DEX0455_020.aa.2 \\n                 N \\n                 0 - \\n                 29-46, \\n                 CK2_PHOSPHO_SITE 44-47; \\n               \\n               \\n                   \\n                   \\n                 o1-67; \\n                 1.092; 52-64, \\n                 ASN_GLYCOSYLATION 25-28; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 MYRISTYL 7-12; \\n               \\n               \\n                   \\n                   \\n                   \\n                 11-21, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                 DEX0455_021.orf.1 \\n                 N \\n                 0 - \\n                 86-97, \\n                 PKC_PHOSPHO_SITE 81-83; \\n               \\n               \\n                   \\n                   \\n                 o1-104; \\n                 1.124; 4-9, \\n                 PKC_PHOSPHO_SITE 100-102; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n                 CAMP_PHOSPHO_SITE 33-36; \\n               \\n               \\n                   \\n                   \\n                   \\n                 35-71, \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 MYRISTYL 10-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 14-22, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n               \\n               \\n                 DEX0455_021.aa.1 \\n                 N \\n                 0 - \\n                 45-51, \\n               \\n               \\n                   \\n                   \\n                 o1-82; \\n                 1.09; 4-11, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 15-23, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.096; \\n               \\n               \\n                 DEX0455_021.aa.2 \\n                 N \\n                 0 - \\n                 112-119, \\n                 ASN_GLYCOSYLATION 108-111; \\n                 Epimerase 5-311; \\n               \\n               \\n                   \\n                   \\n                 o1-360; \\n                 1.083; \\n                 MYRISTYL 203-208; \\n                 galE 4-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-218, \\n                 AMIDATION 217-220; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; \\n                 CK2_PHOSPHO_SITE 44-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-147, \\n                 AMIDATION 325-328; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 PKC_PHOSPHO_SITE 48-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 179-186, \\n                 CK2_PHOSPHO_SITE 18-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; 73-108, \\n                 CAMP_PHOSPHO_SITE 78-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 PKC_PHOSPHO_SITE 56-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 290-300, \\n                 MYRISTYL 263-268; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; 169-175, \\n                 190-195; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n               \\n               \\n                   \\n                   \\n                   \\n                 46-55, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n               \\n               \\n                   \\n                   \\n                   \\n                 309-318, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; 4-10, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n               \\n               \\n                   \\n                   \\n                   \\n                 341-347, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; 13-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n               \\n               \\n                   \\n                   \\n                   \\n                 193-199, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n               \\n               \\n                   \\n                   \\n                   \\n                 239-267, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n               \\n               \\n                 DEX0455_021.orf.3 \\n                 N \\n                 0 - \\n                 24-47, \\n                 CK2_PHOSPHO_SITE 92-95; \\n                 galE 52-344; \\n               \\n               \\n                   \\n                   \\n                 o1-376; \\n                 1.119; \\n                 MYRISTYL 4-9; \\n                 Epimerase 53-359; \\n               \\n               \\n                   \\n                   \\n                   \\n                 241-247, \\n                 PKC_PHOSPHO_SITE 96-98; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; 94-103, \\n                 MYRISTYL 8-13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 PKC_PHOSPHO_SITE 104-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                 61-82, \\n                 CAMP_PHOSPHO_SITE 126-129; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 MYRISTYL 238-243; \\n               \\n               \\n                   \\n                   \\n                   \\n                 357-366, \\n                 CK2_PHOSPHO_SITE 66-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; 51-58, \\n                 MYRISTYL 9-14; AMIDATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n                 373-376; MYRISTYL 6-11; \\n               \\n               \\n                   \\n                   \\n                   \\n                 257-266, \\n                 MYRISTYL 10-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; \\n                 ASN_GLYCOSYLATION 156-159; \\n               \\n               \\n                   \\n                   \\n                   \\n                 169-195, \\n                 AMIDATION 265-268; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 MYRISTYL 311-316; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-156, \\n                 251-256; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 160-167, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n               \\n               \\n                   \\n                   \\n                   \\n                 287-315, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 338-348, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; 217-223, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n               \\n               \\n                   \\n                   \\n                   \\n                 227-234, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n               \\n               \\n                 DEX0455_021.aa.3 \\n                 N \\n                 0 - \\n                 290-300, \\n                 PKC_PHOSPHO_SITE 48-50; \\n                 galE 4-462; \\n               \\n               \\n                   \\n                   \\n                 o1-466; \\n                 1.12; 239-267, \\n                 MYRISTYL 203-208; \\n                 Epimerase 5-459; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 AMIDATION 217-220; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-218, \\n                 PKC_PHOSPHO_SITE 56-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; \\n                 MYRISTYL 263-268; \\n               \\n               \\n                   \\n                   \\n                   \\n                 399-417, \\n                 AMIDATION 325-328; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 PKC_PHOSPHO_SITE 399-401; \\n               \\n               \\n                   \\n                   \\n                   \\n                 193-199, \\n                 PKC_PHOSPHO_SITE 372-374; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; 4-10, \\n                 MYRISTYL 359-364; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n                 190-195; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-147, \\n                 108-111; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; 46-55, \\n                 346-348; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 44-47; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 112-119, \\n                 78-81; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 18-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                 337-393, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.213; 13-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n               \\n               \\n                   \\n                   \\n                   \\n                 309-318, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; 73-108, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 169-175, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; 422-434, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n               \\n               \\n                   \\n                   \\n                   \\n                 179-186, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n               \\n               \\n                 DEX0455_021.aa.4 \\n                 N \\n                 0 - \\n                 112-119, \\n                 MYRISTYL 190-195; MYRISTYL \\n                 galE 4-296; \\n               \\n               \\n                   \\n                   \\n                 o1-328; \\n                 1.083; \\n                 263-268; PKC_PHOSPHO_SITE \\n                 Epimerase 5-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 239-267, \\n                 56-58; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 44-47; MYRISTYL 203-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 193-199, \\n                 CK2_PHOSPHO_SITE 18-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n                 AMIDATION 217-220; \\n               \\n               \\n                   \\n                   \\n                   \\n                 169-175, \\n                 ASN_GLYCOSYLATION 108-111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; 309-318, \\n                 PKC_PHOSPHO_SITE 48-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 AMIDATION 325-328; \\n               \\n               \\n                   \\n                   \\n                   \\n                 73-108, \\n                 CAMP_PHOSPHO_SITE 78-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-218, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 290-300, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; 121-147, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 13-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; 46-55, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n               \\n               \\n                   \\n                   \\n                   \\n                 179-186, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; 4-10, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n               \\n               \\n                 DEX0455_022.aa.1 \\n                 Y \\n                 1 - \\n                 54-73, \\n                 PKC_PHOSPHO_SITE 145-147; \\n               \\n               \\n                   \\n                   \\n                 i1-21; \\n                 1.132; 19-46, \\n                 TYR_PHOSPHO_SITE 147-154; \\n               \\n               \\n                   \\n                   \\n                 tm22-44; \\n                 1.26; \\n                 AMIDATION 173-176; \\n               \\n               \\n                   \\n                   \\n                 o45-178; \\n                 123-135, \\n                 CK2_PHOSPHO_SITE 4-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.041; 5-12, \\n                 ASN_GLYCOSYLATION 65-68; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 ASN_GLYCOSYLATION 92-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 81-101, \\n                 PKC_PHOSPHO_SITE 45-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 140-149, \\n                 CK2_PHOSPHO_SITE 156-159; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; \\n               \\n               \\n                   \\n                   \\n                   \\n                 151-157, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; \\n               \\n               \\n                   \\n                   \\n                   \\n                 161-171, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n               \\n               \\n                 DEX0455_022.orf.2 \\n                 Y \\n                 1 - \\n                 32-59, \\n                 ASN_GLYCOSYLATION 78-81; \\n               \\n               \\n                   \\n                   \\n                 o1-34; \\n                 1.26; 67-86, \\n                 PKC_PHOSPHO_SITE 58-60; \\n               \\n               \\n                   \\n                   \\n                 tm35-57; \\n                 1.132; \\n                 PKC_PHOSPHO_SITE 7-9; \\n               \\n               \\n                   \\n                   \\n                 i58-141; \\n                 4-9, 1.102; \\n                 CK2_PHOSPHO_SITE 17-20; \\n               \\n               \\n                   \\n                   \\n                   \\n                 126-138, \\n                 ASN_GLYCOSYLATION 105-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; 18-25, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 94-114, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n               \\n               \\n                 DEX0455_022.aa.2 \\n                 Y \\n                 1 - \\n                 113-128, \\n                 ASN_GLYCOSYLATION 92-95; \\n               \\n               \\n                   \\n                   \\n                 i1-21; \\n                 1.071; \\n                 ASN_GLYCOSYLATION 65-68; \\n               \\n               \\n                   \\n                   \\n                 tm22-44; \\n                 131-144, \\n                 TYR_PHOSPHO_SITE 155-162; \\n               \\n               \\n                   \\n                   \\n                 o45-188; \\n                 1.136; \\n                 PKC_PHOSPHO_SITE 153-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 148-157, \\n                 CK2_PHOSPHO_SITE 4-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; 19-46, \\n                 PKC_PHOSPHO_SITE 45-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.26; \\n                 CK2_PHOSPHO_SITE 164-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 81-101, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 5-12, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 169-185, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 159-165, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; 54-73, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n               \\n               \\n                 DEX0455_022.aa.3 \\n                 Y \\n                 0 - \\n                 6-14, 1.07; \\n                 MYRISTYL 48-53; MYRISTYL \\n                 GLY_RICH 5-48; \\n               \\n               \\n                   \\n                   \\n                 o1-78; \\n                 48-65, \\n                 34-39; MYRISTYL 14-19; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.202; 70-75, \\n                 MYRISTYL 27-32; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 6-11; PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 39-44; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 13-18; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 20-25; \\n               \\n               \\n                 DEX0455_023.aa.1 \\n                 N \\n                 0 - \\n                 69-80, \\n                 PKC_PHOSPHO_SITE 57-59; \\n               \\n               \\n                   \\n                   \\n                 o1-116; \\n                 1.147; 50-61, \\n                 CAMP_PHOSPHO_SITE 48-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n                 PKC_PHOSPHO_SITE 52-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 27-37, \\n                 PKC_PHOSPHO_SITE 47-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 82-89, \\n                 PKC_PHOSPHO_SITE 101-103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n                 PKC_PHOSPHO_SITE 43-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-13, 1.11; \\n                 CK2_PHOSPHO_SITE 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 39-42; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 97-99; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 54-57; \\n               \\n               \\n                 DEX0455_024.aa.1 \\n                 N \\n                 0 - \\n                 245-252, \\n                 PKC_PHOSPHO_SITE 34-36; \\n                 ANNEXINI 343-356; \\n               \\n               \\n                   \\n                   \\n                 o1-360; \\n                 1.15; 127-136, \\n                 PKC_PHOSPHO_SITE 257-259; \\n                 sp_P09525_ANX4_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; \\n                 PKC_PHOSPHO_SITE 33-35; \\n                 131-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-215, \\n                 PKC_PHOSPHO_SITE 298-300; \\n                 ANNEXINV \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; 309-317, \\n                 MYRISTYL 6-11; MYRISTYL \\n                 219-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; \\n                 144-149; CK2_PHOSPHO_SITE \\n                 annexin 212-280; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-124, \\n                 176-179; ASN_GLYCOSYLATION \\n                 ANNEXINV \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                 286-289; PKC_PHOSPHO_SITE \\n                 343-356; \\n               \\n               \\n                   \\n                   \\n                   \\n                 274-284, \\n                 22-24; CAMP_PHOSPHO_SITE \\n                 ANNEXINV 136-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.206; \\n                 254-257; MYRISTYL 29-34; \\n                 annexin \\n               \\n               \\n                   \\n                   \\n                   \\n                 189-197, \\n                 PKC_PHOSPHO_SITE 7-9; \\n                 288-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; \\n                 CK2_PHOSPHO_SITE 344-347; \\n                 ANNEXIN 109-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-357, \\n                 CK2_PHOSPHO_SITE 315-318; \\n                 sp_P08132_ANX4_PIG \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.265; \\n                 CK2_PHOSPHO_SITE 272-275; \\n                 214-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 324-340, \\n                 CAMP_PHOSPHO_SITE 163-166; \\n                 ANNEXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 CK2_PHOSPHO_SITE 71-74; \\n                 136-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 147-157, \\n                 MYRISTYL 303-308; \\n                 sp_Q9NFS4_Q9NFS4_GIALA \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; 78-85, \\n                 CK2_PHOSPHO_SITE 158-161; \\n                 89-348; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.168; \\n                 PKC_PHOSPHO_SITE 264-266; \\n                 ANNEXIN 219-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-240, \\n                 CK2_PHOSPHO_SITE 73-76; \\n                 ANNEXINV \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 CK2_PHOSPHO_SITE 257-260; \\n                 299-325; \\n               \\n               \\n                   \\n                   \\n                   \\n                 167-172, \\n                   \\n                 ANNEXIN 72-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.042; \\n                   \\n                 ANNEXINI 299-319; \\n               \\n               \\n                   \\n                   \\n                   \\n                 289-295, \\n                   \\n                 ANNEXINI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.073; 61-70, \\n                   \\n                 219-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                   \\n                 annexin 16-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 228-280; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINI \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 136-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 annexin 129-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANX 303-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINV \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 69-91; ANX 144-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINI \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 69-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINI 109-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 69-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 sp_P09525_ANX4_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 290-358; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANX 228-280; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINV 109-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANX 72-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 303-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINI 109-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 136-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINII 69-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 343-356; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 144-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 299-319; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 sp_P09525_ANX4_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 63-127; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 299-319; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 219-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 343-356; \\n               \\n               \\n                 DEX0455_024.aa.2 \\n                 N \\n                 0 - \\n                 4-14, \\n                 CK2_PHOSPHO_SITE 132-135; \\n                 ANNEXINIV 36-62; \\n               \\n               \\n                   \\n                   \\n                 o1-177; \\n                 1.157; \\n                 CK2_PHOSPHO_SITE 161-164; \\n                 sp_P08132_ANX4_PIG \\n               \\n               \\n                   \\n                   \\n                   \\n                 165-174, \\n                 ASN_GLYCOSYLATION 103-106; \\n                 31-100; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.265; \\n                 MYRISTYL 120-125; \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                 141-157, \\n                 CK2_PHOSPHO_SITE 89-92; \\n                 36-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; 91-101, \\n                 CAMP_PHOSPHO_SITE 71-74; \\n                 sp_P09525_ANX4_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.206; \\n                 PKC_PHOSPHO_SITE 115-117; \\n                 107-175; \\n               \\n               \\n                   \\n                   \\n                   \\n                 126-134, \\n                 PKC_PHOSPHO_SITE 81-83; \\n                 ANNEXINII 160-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; 49-57, \\n                 PKC_PHOSPHO_SITE 74-76; \\n                 ANNEXINII \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 CK2_PHOSPHO_SITE 74-77; \\n                 116-136; \\n               \\n               \\n                   \\n                   \\n                   \\n                 25-32, \\n                   \\n                 ANNEXIN 160-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; 106-112, \\n                   \\n                 ANNEXINIV \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.073; \\n                   \\n                 160-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 62-69, \\n                   \\n                 annexin 29-97; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n                   \\n                 ANNEXIN 120-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANX 45-97; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 36-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANX 120-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN 45-97; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXINIV 116-142; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 ANNEXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 116-136; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 annexin 105-172; \\n               \\n               \\n                 DEX0455_025.aa.1 \\n                 N \\n                 0 - \\n                 55-61, \\n                 PKC_PHOSPHO_SITE 296-298; \\n               \\n               \\n                   \\n                   \\n                 o1-380; \\n                 1.085; 95-102, \\n                 CK2_PHOSPHO_SITE 230-233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 MYRISTYL 357-362; \\n               \\n               \\n                   \\n                   \\n                   \\n                 124-165, \\n                 CK2_PHOSPHO_SITE 118-121; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.203; \\n                 TYR_PHOSPHO_SITE 298-305; \\n               \\n               \\n                   \\n                   \\n                   \\n                 253-265, \\n                 ASN_GLYCOSYLATION 51-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.112; \\n                 PKC_PHOSPHO_SITE 358-360; \\n               \\n               \\n                   \\n                   \\n                   \\n                 280-315, \\n                 CK2_PHOSPHO_SITE 318-321; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 ASN_GLYCOSYLATION 344-347; \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-211, \\n                 MYRISTYL 55-60; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 PKC_PHOSPHO_SITE 349-351; \\n               \\n               \\n                   \\n                   \\n                   \\n                 323-341, \\n                 MYRISTYL 224-229; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; 25-49, \\n                 CK2_PHOSPHO_SITE 169-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.169; \\n                 PKC_PHOSPHO_SITE 71-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 350-357, \\n                 CK2_PHOSPHO_SITE 352-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.074; \\n                 CK2_PHOSPHO_SITE 308-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 362-370, \\n                 CK2_PHOSPHO_SITE 228-231; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; 9-15, \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 269-278, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; 73-90, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 175-185, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.225; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-250, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; \\n               \\n               \\n                   \\n                   \\n                   \\n                 111-121, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                 DEX0455_025.orf.2 \\n                 Y \\n                 0 - \\n                 329-364, \\n                 PKC_PHOSPHO_SITE 45-47; \\n               \\n               \\n                   \\n                   \\n                 o1-394; \\n                 1.154; \\n                 CK2_PHOSPHO_SITE 35-38; \\n               \\n               \\n                   \\n                   \\n                   \\n                 318-327, \\n                 TYR_PHOSPHO_SITE 347-354; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; 18-31, \\n                 MYRISTYL 104-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.139; \\n                 PKC_PHOSPHO_SITE 117-119; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-15, 1.18; \\n                 PKC_PHOSPHO_SITE 120-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-110, \\n                 CK2_PHOSPHO_SITE 357-360; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 CK2_PHOSPHO_SITE 218-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-139, \\n                 PKC_PHOSPHO_SITE 15-17; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 CK2_PHOSPHO_SITE 279-282; \\n               \\n               \\n                   \\n                   \\n                   \\n                 302-314, \\n                 MYRISTYL 273-278; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.112; \\n                 CK2_PHOSPHO_SITE 277-280; \\n               \\n               \\n                   \\n                   \\n                   \\n                 250-260, \\n                 ASN_GLYCOSYLATION 100-103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; 74-98, \\n                 CK2_PHOSPHO_SITE 15-18; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.169; \\n                 PKC_PHOSPHO_SITE 345-347; \\n               \\n               \\n                   \\n                   \\n                   \\n                 372-391, \\n                 CK2_PHOSPHO_SITE 367-370; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 CK2_PHOSPHO_SITE 167-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-299, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; \\n               \\n               \\n                   \\n                   \\n                   \\n                 173-214, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.203; 36-43, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.073; \\n               \\n               \\n                   \\n                   \\n                   \\n                 160-170, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 144-151, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n               \\n               \\n                   \\n                   \\n                   \\n                 224-234, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.225; 58-64, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n               \\n               \\n                 DEX0455_025.aa.2 \\n                 N \\n                 0 - \\n                 95-102, \\n                 PKC_PHOSPHO_SITE 652-654; \\n               \\n               \\n                   \\n                   \\n                 o1-679; \\n                 1.121; \\n                 PKC_PHOSPHO_SITE 296-298; \\n               \\n               \\n                   \\n                   \\n                   \\n                 557-564, \\n                 TYR_PHOSPHO_SITE 298-305; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.066; \\n                 MYRISTYL 224-229; \\n               \\n               \\n                   \\n                   \\n                   \\n                 111-121, \\n                 PKC_PHOSPHO_SITE 632-634; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; 9-15, \\n                 CK2_PHOSPHO_SITE 653-656; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 CK2_PHOSPHO_SITE 228-231; \\n               \\n               \\n                   \\n                   \\n                   \\n                 655-665, \\n                 MYRISTYL 349-354; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 CK2_PHOSPHO_SITE 230-233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-250, \\n                 MYRISTYL 507-512; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; 73-90, \\n                 361-366; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 588-590; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 507-513, \\n                 640-643; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 526-528; MYRISTYL 55-60; \\n               \\n               \\n                   \\n                   \\n                   \\n                 124-165, \\n                 PKC_PHOSPHO_SITE 555-557; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.203; \\n                 CK2_PHOSPHO_SITE 118-121; \\n               \\n               \\n                   \\n                   \\n                   \\n                 539-548, \\n                 PKC_PHOSPHO_SITE 71-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                 LEUCINE_ZIPPER 453-474; \\n               \\n               \\n                   \\n                   \\n                   \\n                 623-630, \\n                 PKC_PHOSPHO_SITE 631-633; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 253-265, \\n                 CK2_PHOSPHO_SITE 540-543; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.112; \\n                 PKC_PHOSPHO_SITE 513-515; \\n               \\n               \\n                   \\n                   \\n                   \\n                 360-365, \\n                 ASN_GLYCOSYLATION 51-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.048; \\n                 MYRISTYL 674-679; \\n               \\n               \\n                   \\n                   \\n                   \\n                 280-315, \\n                 CK2_PHOSPHO_SITE 318-321; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 CK2_PHOSPHO_SITE 308-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 386-392, \\n                 CK2_PHOSPHO_SITE 169-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n               \\n               \\n                   \\n                   \\n                   \\n                 394-416, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 489-499, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.139; \\n               \\n               \\n                   \\n                   \\n                   \\n                 269-278, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; \\n               \\n               \\n                   \\n                   \\n                   \\n                 368-384, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.143; \\n               \\n               \\n                   \\n                   \\n                   \\n                 578-589, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 175-185, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.225; \\n               \\n               \\n                   \\n                   \\n                   \\n                 473-482, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-211, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n               \\n               \\n                   \\n                   \\n                   \\n                 323-355, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 25-49, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.169; \\n               \\n               \\n                   \\n                   \\n                   \\n                 424-465, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.143; 55-61, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n               \\n               \\n                   \\n                   \\n                   \\n                 633-639, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 601-618, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n               \\n               \\n                   \\n                   \\n                   \\n                 521-528, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n               \\n               \\n                 DEX0455_026.orf.1 \\n                 N \\n                 0 - \\n                   \\n                 ASN_GLYCOSYLATION 46-49; \\n               \\n               \\n                   \\n                   \\n                 i1-72; \\n                   \\n                 CK2_PHOSPHO_SITE 62-65; \\n               \\n               \\n                 DEX0455_026.aa.1 \\n                 N \\n                 0 - \\n                 69-74, \\n                 CK2_PHOSPHO_SITE 58-61; \\n               \\n               \\n                   \\n                   \\n                 i1-77; \\n                 1.113; 47-54, \\n                 CK2_PHOSPHO_SITE 53-56; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 7-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n               \\n               \\n                 DEX0455_027.orf.1 \\n                 N \\n                 0 - \\n                 54-76, \\n                 CK2_PHOSPHO_SITE 65-68; \\n               \\n               \\n                   \\n                   \\n                 o1-174; \\n                 1.16; 118-139, \\n                 MYRISTYL 53-58; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 40-45; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 40-52, \\n                 75-77; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; 4-20, \\n                 93-96; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.164; \\n                 171-174; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 155-171, \\n                 114-117; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n                 83-86; MYRISTYL 152-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 103-113, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.167; \\n               \\n               \\n                 DEX0455_027.aa.1 \\n                 N \\n                 0 - \\n                 5-28, \\n               \\n               \\n                   \\n                   \\n                 i1-36; \\n                 1.145; \\n               \\n               \\n                 DEX0455_028.aa.1 \\n                 N \\n                 0 - \\n                   \\n                 MYRISTYL 144-149; \\n                 cobW 41-214; ATP_GTP_A 49-56; \\n               \\n               \\n                   \\n                   \\n                 o1-215; \\n                   \\n                 CK2_PHOSPHO_SITE 30-33; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 111-113; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 111-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 52-57; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 167-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 28-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 GLYCOSAMINOGLYCAN 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 106-111; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 7-10; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 210-212; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 82-85; \\n               \\n               \\n                 DEX0455_029.orf.1 \\n                 Y \\n                 0 - \\n                 109-124, \\n                 CK2_PHOSPHO_SITE 120-123; \\n               \\n               \\n                   \\n                   \\n                 o1-133; \\n                 1.141; 4-18, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n               \\n               \\n                   \\n                   \\n                   \\n                 26-79, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; \\n               \\n               \\n                 DEX0455_029.aa.1 \\n                 N \\n                 5 - \\n                 589-601, \\n                 CK2_PHOSPHO_SITE 503-506; \\n               \\n               \\n                   \\n                   \\n                 i1-20; \\n                 1.168; \\n                 PKC_PHOSPHO_SITE 546-548; \\n               \\n               \\n                   \\n                   \\n                 tm21-43; \\n                 141-213, \\n                 CK2_PHOSPHO_SITE 714-717; \\n               \\n               \\n                   \\n                   \\n                 o44-55; \\n                 1.251; 4-19, \\n                 ASN_GLYCOSYLATION 747-750; \\n               \\n               \\n                   \\n                   \\n                 tm56-78; \\n                 1.116; \\n                 MYRISTYL 695-700; \\n               \\n               \\n                   \\n                   \\n                 i79-188; \\n                 84-92, \\n                 PKC_PHOSPHO_SITE 716-718; \\n               \\n               \\n                   \\n                   \\n                 tm189-211; \\n                 1.108; \\n                 ASN_GLYCOSYLATION 51-54; \\n               \\n               \\n                   \\n                   \\n                 o212-265; \\n                 810-816, \\n                 MYRISTYL 651-656; \\n               \\n               \\n                   \\n                   \\n                 tm266-288; \\n                 1.062; \\n                 PKC_PHOSPHO_SITE 351-353; \\n               \\n               \\n                   \\n                   \\n                 i289-294; \\n                 787-805, \\n                 ASN_GLYCOSYLATION 228-231; \\n               \\n               \\n                   \\n                   \\n                 tm295-317; \\n                 1.18; 292-319, \\n                 CK2_PHOSPHO_SITE 697-700; \\n               \\n               \\n                   \\n                   \\n                 o318-953; \\n                 1.22; \\n                 PKC_PHOSPHO_SITE 808-810; \\n               \\n               \\n                   \\n                   \\n                   \\n                 818-881, \\n                 ASN_GLYCOSYLATION 782-785; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.176; \\n                 CK2_PHOSPHO_SITE 904-907; \\n               \\n               \\n                   \\n                   \\n                   \\n                 914-920, \\n                 PKC_PHOSPHO_SITE 622-624; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n                 PKC_PHOSPHO_SITE 904-906; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-132, \\n                 TYR_PHOSPHO_SITE 297-303; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 PKC_PHOSPHO_SITE 755-757; \\n               \\n               \\n                   \\n                   \\n                   \\n                 721-739, \\n                 PKC_PHOSPHO_SITE 784-786; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 PKC_PHOSPHO_SITE 317-319; \\n               \\n               \\n                   \\n                   \\n                   \\n                 531-544, \\n                 PKC_PHOSPHO_SITE 2-4; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n                 PKC_PHOSPHO_SITE 879-881; \\n               \\n               \\n                   \\n                   \\n                   \\n                 655-680, \\n                 CK2_PHOSPHO_SITE 546-549; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.23; 52-80, \\n                 PKC_PHOSPHO_SITE 332-334; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.244; \\n                 PKC_PHOSPHO_SITE 835-837; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-380, \\n                 MYRISTYL 130-135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 CK2_PHOSPHO_SITE 936-939; \\n               \\n               \\n                   \\n                   \\n                   \\n                 388-401, \\n                 PKC_PHOSPHO_SITE 139-141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 700-708, \\n                 CK2_PHOSPHO_SITE 934-937; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.061; \\n                 ASN_GLYCOSYLATION 741-744; \\n               \\n               \\n                   \\n                   \\n                   \\n                 609-619, \\n                 PKC_PHOSPHO_SITE 259-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 PKC_PHOSPHO_SITE 938-940; \\n               \\n               \\n                   \\n                   \\n                   \\n                 753-782, \\n                 CK2_PHOSPHO_SITE 879-882; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.251; \\n                 PKC_PHOSPHO_SITE 230-232; \\n               \\n               \\n                   \\n                   \\n                   \\n                 321-340, \\n                 PKC_PHOSPHO_SITE 912-914; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 MYRISTYL 240-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 551-568, \\n                 PKC_PHOSPHO_SITE 909-911; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n                 PKC_PHOSPHO_SITE 161-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 884-898, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n               \\n               \\n                   \\n                   \\n                   \\n                 570-587, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 924-935, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 691-697, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.081; \\n               \\n               \\n                   \\n                   \\n                   \\n                 255-286, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n               \\n               \\n                   \\n                   \\n                   \\n                 409-462, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; 99-116, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n               \\n               \\n                   \\n                   \\n                   \\n                 492-507, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; 21-46, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.201; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-252, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 512-524, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n               \\n               \\n                   \\n                   \\n                   \\n                 624-652, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.188; \\n               \\n               \\n                 DEX0455_029.orf.2 \\n                 Y \\n                 0 - \\n                 26-79, \\n                 CK2_PHOSPHO_SITE 145-148; \\n                 cobW 1-176; \\n               \\n               \\n                   \\n                   \\n                 o1-194; \\n                 1.144; \\n                 PKC_PHOSPHO_SITE 153-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 165-176, \\n                 PKC_PHOSPHO_SITE 145-147; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n                 PKC_PHOSPHO_SITE 120-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 125-139, \\n                 PKC_PHOSPHO_SITE 179-181; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 CK2_PHOSPHO_SITE 177-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                 109-122, \\n                 PKC_PHOSPHO_SITE 150-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 4-18, \\n                 CK2_PHOSPHO_SITE 120-123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 CK2_PHOSPHO_SITE 175-178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 155-161, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n               \\n               \\n                 DEX0455_029.aa.2 \\n                 N \\n                 6 - \\n                 499-531, \\n                 PKC_PHOSPHO_SITE 312-314; \\n                 cobW 458-710; \\n               \\n               \\n                   \\n                   \\n                 o1-129; \\n                 1.166; \\n                 TYR_PHOSPHO_SITE 448-454; \\n               \\n               \\n                   \\n                   \\n                 tm130-149; \\n                 154-160, \\n                 PKC_PHOSPHO_SITE 290-292; \\n               \\n               \\n                   \\n                   \\n                 i150-168; \\n                 1.051; 9-27, \\n                 PKC_PHOSPHO_SITE 381-383; \\n               \\n               \\n                   \\n                   \\n                 tm169-191; \\n                 1.196; \\n                 PKC_PHOSPHO_SITE 410-412; \\n               \\n               \\n                   \\n                   \\n                 o192-205; \\n                 689-695, \\n                 CK2_PHOSPHO_SITE 32-35; \\n               \\n               \\n                   \\n                   \\n                 tm206-228; \\n                 1.094; 94-120, \\n                 CK2_PHOSPHO_SITE 709-712; \\n               \\n               \\n                   \\n                   \\n                 i229-339; \\n                 1.182; \\n                 PKC_PHOSPHO_SITE 654-656; \\n               \\n               \\n                   \\n                   \\n                 tm340-362; \\n                 560-613, \\n                 CK2_PHOSPHO_SITE 654-657; \\n               \\n               \\n                   \\n                   \\n                 o363-416; \\n                 1.144; \\n                 ASN_GLYCOSYLATION 379-382; \\n               \\n               \\n                   \\n                   \\n                 tm417-439; \\n                 250-283, \\n                 ASN_GLYCOSYLATION 202-205; \\n               \\n               \\n                   \\n                   \\n                 i440-445; \\n                 1.185; \\n                 PKC_PHOSPHO_SITE 4-6; \\n               \\n               \\n                   \\n                   \\n                 tm446-468; \\n                 406-437, \\n                 CK2_PHOSPHO_SITE 113-116; \\n               \\n               \\n                   \\n                   \\n                 o469-728; \\n                 1.151; \\n                 MYRISTYL 391-396; \\n               \\n               \\n                   \\n                   \\n                   \\n                 472-491, \\n                 PKC_PHOSPHO_SITE 153-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 713-715; \\n               \\n               \\n                   \\n                   \\n                   \\n                 699-710, \\n                 PKC_PHOSPHO_SITE 687-689; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n                 PKC_PHOSPHO_SITE 684-686; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-197, \\n                 CK2_PHOSPHO_SITE 711-714; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.201; \\n                 PKC_PHOSPHO_SITE 468-470; \\n               \\n               \\n                   \\n                   \\n                   \\n                 162-170, \\n                 PKC_PHOSPHO_SITE 90-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n                 ASN_GLYCOSYLATION 120-123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-364, \\n                 PKC_PHOSPHO_SITE 502-504; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.251; \\n                 MYRISTYL 281-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 538-552, \\n                 CK2_PHOSPHO_SITE 679-682; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 659-673, \\n                 PKC_PHOSPHO_SITE 483-485; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 PKC_PHOSPHO_SITE 679-681; \\n               \\n               \\n                   \\n                   \\n                   \\n                 394-403, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; 38-69, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.174; \\n               \\n               \\n                   \\n                   \\n                   \\n                 443-470, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.22; 126-152, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.243; \\n               \\n               \\n                   \\n                   \\n                   \\n                 74-83, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n               \\n               \\n                   \\n                   \\n                   \\n                 643-656, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 203-243, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.244; \\n               \\n               \\n                 DEX0455_030.aa.1 \\n                 N \\n                 0 - \\n                 250-278, \\n                 CK2_PHOSPHO_SITE 5-8; \\n                 Metallophos 47-242; \\n               \\n               \\n                   \\n                   \\n                 o1-312; \\n                 1.145; 42-55, \\n                 MYRISTYL 267-272; \\n                 STPHPHTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 209-211; \\n                 48-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 185-198, \\n                 ASN_GLYCOSYLATION 250-253; \\n                 STPHPHTASE 145-171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 22-40, \\n                 CK2_PHOSPHO_SITE 198-201; \\n                 STPHPHTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 MYRISTYL 166-171; MYRISTYL \\n                 230-250; \\n               \\n               \\n                   \\n                   \\n                   \\n                 213-225, \\n                 113-118; TYR_PHOSPHO_SITE \\n                 STPHPHTASE 252-268; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; 57-71, \\n                 141-149; \\n                 STPHPHTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n                   \\n                 77-104; PP2Ac \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-175, \\n                   \\n                 20-290; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.145; 10-20, \\n                   \\n                 STPHPHTASE 174-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n                   \\n                 PHOSPHO_ESTER \\n               \\n               \\n                   \\n                   \\n                   \\n                 299-309, \\n                   \\n                 47-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.17; 232-243, \\n                   \\n                 STPHPHTASE 110-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.157; \\n                   \\n                 sp_P33172_PPP4_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 280-289, \\n                   \\n                 7-290; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.066; 85-105, \\n                   \\n                 SER_THR_PHOSPHATASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.201; \\n                   \\n                 111-116; \\n               \\n               \\n                 DEX0455_030.aa.2 \\n                 N \\n                 0 - \\n                 108-118, \\n                 PKC_PHOSPHO_SITE 126-128; \\n                 STPHPHTASE 169-182; \\n               \\n               \\n                   \\n                   \\n                 o1-182; \\n                 1.128; 4-12, \\n                 PKC_PHOSPHO_SITE 95-97; \\n                 sp_P11084_PPP4_RABIT \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; \\n                 CK2_PHOSPHO_SITE 53-56; \\n                 104-182; \\n               \\n               \\n                   \\n                   \\n                   \\n                 47-67, \\n                 CK2_PHOSPHO_SITE 116-119; \\n                 STPHPHTASE 147-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.252; 71-81, \\n                 MYRISTYL 83-88; \\n                 PP2Ac 33-182; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 ASN_GLYCOSYLATION 167-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                 38-44, 1.1; \\n               \\n               \\n                   \\n                   \\n                   \\n                 149-160, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.157; 91-97, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n               \\n               \\n                   \\n                   \\n                   \\n                 167-179, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; 18-36, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.164; \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-142, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n               \\n               \\n                 DEX0455_031.orf.1 \\n                 Y \\n                 0 - \\n                 161-189, \\n                 ASN_GLYCOSYLATION 78-81; \\n                 TNFR_NGFR_1 \\n               \\n               \\n                   \\n                   \\n                 o1-293; \\n                 1.17; 150-156, \\n                 MYRISTYL 103-108; \\n                 121-164; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 CK2_PHOSPHO_SITE 80-83; \\n                 TNFR_c6 81-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 285-190, \\n                 MYRISTYL 233-238; \\n                 TNFR_NGFR_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; 12-45, \\n                 PKC_PHOSPHO_SITE 220-222; \\n                 120-162; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 MYRISTYL 167-172; MYRISTYL \\n                 CYS_RICH 96-186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 89-124, \\n                 201-206; CK2_PHOSPHO_SITE \\n                 TNFR 81-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n                 178-181; PKC_PHOSPHO_SITE \\n                 TNFR_c6 \\n               \\n               \\n                   \\n                   \\n                   \\n                 249-255, \\n                 10-12; MYRISTYL 281-286; \\n                 208-248; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 MYRISTYL 61-66; AMIDATION \\n                 TNFR_NGFR_2_3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 203-211, \\n                 16-19; PKC_PHOSPHO_SITE \\n                 207-248; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n                 247-249; PKC_PHOSPHO_SITE \\n                 TNFR_NGFR_1 81-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 231-238, \\n                 156-158; CK2_PHOSPHO_SITE \\n                 TNFR 121-162; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 47-74, \\n                 125-128; CAMP_PHOSPHO_SITE \\n                 TNFR 164-205; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 157-160; PKC_PHOSPHO_SITE \\n                 TNFR_c6 \\n               \\n               \\n                   \\n                   \\n                   \\n                 262-279, \\n                 80-82; PKC_PHOSPHO_SITE \\n                 121-162; TNFR \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; \\n                 107-109; PKC_PHOSPHO_SITE \\n                 208-248; \\n               \\n               \\n                   \\n                   \\n                   \\n                 220-227, \\n                 155-157; CK2_PHOSPHO_SITE \\n                 TNFR_c6 164-194; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 120-123; MYRISTYL 240-245; \\n                 TNFR_NGFR_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-148, \\n                 CAMP_PHOSPHO_SITE 18-21; \\n                 80-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.205; \\n                 TYR_PHOSPHO_SITE 82-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 227-230; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 190-193; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 215-218; \\n               \\n               \\n                 DEX0455_031.aa.1 \\n                 Y \\n                 1 - \\n                 193-200, \\n                 MYRISTYL 23-28; \\n                 TNFR_NGFR_1 43-80; \\n               \\n               \\n                   \\n                   \\n                 o1-425; \\n                 1.08; 51-86, \\n                 PKC_PHOSPHO_SITE 182-184; \\n                 TNFR_NGFR_2_3 \\n               \\n               \\n                   \\n                   \\n                 tm426-448; \\n                 1.191; \\n                 PKC_PHOSPHO_SITE 331-333; \\n                 169-210; \\n               \\n               \\n                   \\n                   \\n                 i449-635; \\n                 165-173, \\n                 TYR_PHOSPHO_SITE 44-51; \\n                 TNFR_c6 43-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n                 ASN_GLYCOSYLATION 177-180; \\n                 CYS_RICH 58-148; \\n               \\n               \\n                   \\n                   \\n                   \\n                 614-621, \\n                 CK2_PHOSPHO_SITE 523-526; \\n                 PRO_RICH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.063; \\n                 MYRISTYL 547-552; \\n                 567-602; \\n               \\n               \\n                   \\n                   \\n                   \\n                 182-189, \\n                 PKC_PHOSPHO_SITE 209-211; \\n                 TNFR_NGFR_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 MYRISTYL 65-70; MYRISTYL \\n                 82-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 427-464, \\n                 552-557; PKC_PHOSPHO_SITE \\n                 TNFR_NGFR_1 83-126; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.216; \\n                 118-120; AMIDATION 382-385; \\n                 TNFR 43-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 112-118, \\n                 MYRISTYL 195-200; \\n                 TNFR_NGFR_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 CK2_PHOSPHO_SITE 87-90; \\n                 42-80; TNFR 83-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 268-294, \\n                 PKC_PHOSPHO_SITE 623-625; \\n                 TNFR_c6 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.175; \\n                 MYRISTYL 202-207; MYRISTYL \\n                 83-124; TNFR \\n               \\n               \\n                   \\n                   \\n                   \\n                 211-217, \\n                 411-416; MYRISTYL 376-381; \\n                 126-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 MYRISTYL 286-291; \\n                 TNFR_c6 170-210; \\n               \\n               \\n                   \\n                   \\n                   \\n                 344-423, \\n                 CAMP_PHOSPHO_SITE 119-122; \\n                 TNFR 170-210; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.224; \\n                 MYRISTYL 396-401; \\n                 TNFR_c6 \\n               \\n               \\n                   \\n                   \\n                   \\n                 296-325, \\n                 CK2_PHOSPHO_SITE 140-143; \\n                 126-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; \\n                 ASN_GLYCOSYLATION 40-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-265, \\n                 CK2_PHOSPHO_SITE 42-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 PKC_PHOSPHO_SITE 321-323; \\n               \\n               \\n                   \\n                   \\n                   \\n                 472-477, \\n                 MYRISTYL 243-248; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n                 CK2_PHOSPHO_SITE 152-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 482-528, \\n                 PKC_PHOSPHO_SITE 42-44; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 CK2_PHOSPHO_SITE 82-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 596-602, \\n                 MYRISTYL 163-168; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; 224-241, \\n                 619-624; MYRISTYL 129-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; \\n                 PKC_PHOSPHO_SITE 117-119; \\n               \\n               \\n                   \\n                   \\n                   \\n                 123-151, \\n                 CK2_PHOSPHO_SITE 189-192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.17; 575-589, \\n                 PKC_PHOSPHO_SITE 69-71; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; \\n               \\n               \\n                   \\n                   \\n                   \\n                 538-543, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; 9-36, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 560-572, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; 94-110, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.205; \\n               \\n               \\n                 DEX0455_031.aa.2 \\n                 Y \\n                 0 - \\n                 112-118, \\n                 PKC_PHOSPHO_SITE 161-163; \\n                 TNFR_c6 43-80; \\n               \\n               \\n                   \\n                   \\n                 o1-166; \\n                 1.092; \\n                 MYRISTYL 65-70; \\n                 TNFR 43-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 123-151, \\n                 PKC_PHOSPHO_SITE 117-119; \\n                 TNFR_NGFR_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.17; 51-86, \\n                 CK2_PHOSPHO_SITE 87-90; \\n                 42-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n                 MYRISTYL 129-134; \\n                 TNFR_NGFR_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 94-110, \\n                 CK2_PHOSPHO_SITE 42-45; \\n                 82-124; TNFR \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.205; 9-36, \\n                 CK2_PHOSPHO_SITE 82-85; \\n                 83-124; TNFR \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 PKC_PHOSPHO_SITE 42-44; \\n                 126-160; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 40-43; \\n                 TNFR_NGFR_1 43-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 69-71; \\n                 TNFR_NGFR_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 140-143; \\n                 83-126; TNFR_c6 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 118-120; \\n                 126-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 23-28; \\n                 TNFR_c6 83-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 119-122; \\n                 CYS_RICH 58-148; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 44-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 152-155; \\n               \\n               \\n                 DEX0455_031.orf.2 \\n                 N \\n                 1 - \\n                 266-272, \\n                 PKC_PHOSPHO_SITE 293-295; \\n                 TNFR_NGFR_2 39-80; \\n               \\n               \\n                   \\n                   \\n                 o1-95; \\n                 1.04; 81-87, \\n                 PKC_PHOSPHO_SITE 5-7; \\n                 PRO_RICH \\n               \\n               \\n                   \\n                   \\n                 tm96-118; \\n                 1.087; \\n                 MYRISTYL 72-77; MYRISTYL \\n                 237-272; TNFR \\n               \\n               \\n                   \\n                   \\n                 i119-305; \\n                 11-29, \\n                 65-70; PKC_PHOSPHO_SITE \\n                 40-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n                 52-54; MYRISTYL 33-38; \\n               \\n               \\n                   \\n                   \\n                   \\n                 284-291, \\n                 MYRISTYL 217-222; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.063; 97-134, \\n                 PKC_PHOSPHO_SITE 79-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.216; \\n                 ASN_GLYCOSYLATION 47-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-213, \\n                 MYRISTYL 289-294; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; 35-43, \\n                 CK2_PHOSPHO_SITE 59-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.196; \\n                 CK2_PHOSPHO_SITE 193-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                 152-198, \\n                 MYRISTYL 222-227; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 1-6; \\n               \\n               \\n                   \\n                   \\n                   \\n                 245-259, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 63-70, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n               \\n               \\n                   \\n                   \\n                   \\n                 52-59, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n               \\n               \\n                   \\n                   \\n                   \\n                 230-242, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n               \\n               \\n                   \\n                   \\n                   \\n                 142-147, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n               \\n               \\n                 DEX0455_031.aa.3 \\n                 Y \\n                 0 - \\n                 13-25, \\n                 MYRISTYL 178-183; MYRISTYL \\n                 PRO_RICH 126-161; \\n               \\n               \\n                   \\n                   \\n                 o1-194; \\n                 1.182; 31-36, \\n                 106-111; MYRISTYL 111-116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n                 PKC_PHOSPHO_SITE 182-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 41-87, \\n                 CAMP_PHOSPHO_SITE 9-12; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 CK2_PHOSPHO_SITE 82-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 155-161, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; 119-131, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.094; \\n               \\n               \\n                   \\n                   \\n                   \\n                 97-102, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n               \\n               \\n                   \\n                   \\n                   \\n                 134-148, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 173-180, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.063; \\n               \\n               \\n                 DEX0455_032.aa.1 \\n                 N \\n                 0 - \\n                 42-50, \\n                 PKC_PHOSPHO_SITE 152-154; \\n               \\n               \\n                   \\n                   \\n                 o1-241; \\n                 1.149; 84-96, \\n                 PKC_PHOSPHO_SITE 168-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; \\n                 ASN_GLYCOSYLATION 39-42; \\n               \\n               \\n                   \\n                   \\n                   \\n                 216-238, \\n                 CK2_PHOSPHO_SITE 206-209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.152; \\n                 CK2_PHOSPHO_SITE 204-207; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-179, \\n                 PKC_PHOSPHO_SITE 112-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 22-32, \\n                 ASN_GLYCOSYLATION 142-145; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n                 CK2_PHOSPHO_SITE 59-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-141, \\n                 CK2_PHOSPHO_SITE 193-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; 144-168, \\n                 MYRISTYL 37-42; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 CK2_PHOSPHO_SITE 169-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 198-204, \\n                 CK2_PHOSPHO_SITE 112-115; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.108; 69-78, \\n                 ASN_GLYCOSYLATION 104-107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n                 CAMP_PHOSPHO_SITE 33-36; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 92-95; \\n               \\n               \\n                 DEX0455_033.aa.1 \\n                 Y \\n                 5 - \\n                 34-40, \\n                 PKC_PHOSPHO_SITE 248-250; \\n                 VACATPASE 149-175; \\n               \\n               \\n                   \\n                   \\n                 i1-6; \\n                 1.091; 4-32, \\n                 MYRISTYL 53-58; MYRISTYL \\n                 ATP-synt_C \\n               \\n               \\n                   \\n                   \\n                 tm7-29; \\n                 1.218; \\n                 161-166; MYRISTYL 104-109; \\n                 49-114; \\n               \\n               \\n                   \\n                   \\n                 o30-48; \\n                 89-113, \\n                 MYRISTYL 139-144; MYRISTYL \\n                 VACATPASE 176-199; \\n               \\n               \\n                   \\n                   \\n                 tm49-71; \\n                 1.207; \\n                 183-188; MYRISTYL 159-164; \\n                 VACATPASE \\n               \\n               \\n                   \\n                   \\n                 i72-90; \\n                 234-258, \\n                 PKC_PHOSPHO_SITE 233-235; \\n                 65-89; ATP- \\n               \\n               \\n                   \\n                   \\n                 tm91-113; \\n                 1.182; \\n                 MYRISTYL 141-146; MYRISTYL \\n                 synt_C 135-200; \\n               \\n               \\n                   \\n                   \\n                 o114-141; \\n                 203-232, \\n                 64-69; MYRISTYL 156-161; \\n               \\n               \\n                   \\n                   \\n                 tm142-164; \\n                 1.183; \\n                 CK2_PHOSPHO_SITE 27-30; \\n               \\n               \\n                   \\n                   \\n                 i165-175; \\n                 141-167, \\n                 MYRISTYL 10-15; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 tm176-198; \\n                 1.258; \\n                 190-195; MYRISTYL 23-28; \\n               \\n               \\n                   \\n                   \\n                 o199-261; \\n                 169-201, \\n                 CK2_PHOSPHO_SITE 119-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.2; 53-74, \\n                 MYRISTYL 68-73; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n                 55-60; MYRISTYL 152-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 78-84, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.057; \\n               \\n               \\n                 DEX0455_034.aa.1 \\n                 Y \\n                 1 - \\n                 315-320, \\n                 MYRISTYL 281-286; MYRISTYL \\n                 ldl_recept_a \\n               \\n               \\n                   \\n                   \\n                 i1-11; \\n                 1.032; \\n                 457-462; PKC_PHOSPHO_SITE \\n                 333-371; \\n               \\n               \\n                   \\n                   \\n                 tm12-34; \\n                 247-254, \\n                 450-452; ASN_GLYCOSYLATION \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                 o35-499; \\n                 1.118; \\n                 235-238; ASN_GLYCOSYLATION \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 434-444, \\n                 66-69; MYRISTYL 423-428; \\n                 sp_Q99J04_Q99J04_MOUSE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.138; \\n                 MYRISTYL 454-459; MYRISTYL \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 409-419, \\n                 282-287; PKC_PHOSPHO_SITE \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n                 80-82; MYRISTYL 310-315; \\n                 391-441; LDLa \\n               \\n               \\n                   \\n                   \\n                   \\n                 328-355, \\n                 CK2_PHOSPHO_SITE 225-228; \\n                 334-371; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 MYRISTYL 426-431; MYRISTYL \\n                 LDLRA_2 334-370; \\n               \\n               \\n                   \\n                   \\n                   \\n                 270-283, \\n                 422-427; MYRISTYL 161-166; \\n                 LDLRA_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 CK2_PHOSPHO_SITE 374-377; \\n                 347-369; \\n               \\n               \\n                   \\n                   \\n                   \\n                 379-385, \\n                 RGD 193-195; MYRISTYL 472-477; \\n                 BPTI_KUNITZ_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 MYRISTYL 212-217; \\n                 391-441; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-132, \\n                 MYRISTYL 29-34; MYRISTYL \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n                 313-318; MYRISTYL 482-487; \\n                 419-437; \\n               \\n               \\n                   \\n                   \\n                   \\n                 142-152, \\n                 MYRISTYL 43-48; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; \\n                 PKC_PHOSPHO_SITE 242-244; \\n                 278-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 366-372, \\n                 PKC_PHOSPHO_SITE 440-442; \\n                 BPTI_KUNITZ_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; 10-33, \\n                 PKC_PHOSPHO_SITE 490-492; \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.215; \\n                 PKC_PHOSPHO_SITE 148-150; \\n                 BASICPTASE 275-285; \\n               \\n               \\n                   \\n                   \\n                   \\n                 36-53, \\n                 CK2_PHOSPHO_SITE 350-353; \\n                 KU 389-442; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 84-105, \\n                 MYRISTYL 494-499; \\n                 sp_Q99J04_Q99J04_MOUSE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n                 PKC_PHOSPHO_SITE 253-255; \\n                 391-441; \\n               \\n               \\n                   \\n                   \\n                   \\n                 55-67, \\n                 CK2_PHOSPHO_SITE 74-77; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 MYRISTYL 285-290; \\n                 426-441; \\n               \\n               \\n                   \\n                   \\n                   \\n                 168-183, \\n                 PKC_PHOSPHO_SITE 386-388; \\n                 BASICPTASE 247-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; \\n                   \\n                 KU 248-301; \\n               \\n               \\n                   \\n                   \\n                   \\n                 388-403, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n               \\n               \\n                   \\n                   \\n                   \\n                 235-243, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 256-262, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.043; \\n               \\n               \\n                   \\n                   \\n                   \\n                 216-227, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n               \\n               \\n                   \\n                   \\n                   \\n                 293-303, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n               \\n               \\n                   \\n                   \\n                   \\n                 462-486, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n               \\n               \\n                 DEX0455_034.aa.3 \\n                 Y \\n                 1 - \\n                 315-320, \\n                 ASN_GLYCOSYLATION 235-238; \\n                 BPTI_KUNITZ_2 \\n               \\n               \\n                   \\n                   \\n                 i1-11; \\n                 1.032; \\n                 MYRISTYL 285-290; \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                 tm12-34; \\n                 270-283, \\n                 CK2_PHOSPHO_SITE 74-77; \\n                 BASICPTASE 247-261; \\n               \\n               \\n                   \\n                   \\n                 o35-344; \\n                 1.163; \\n                 PKC_PHOSPHO_SITE 242-244; \\n                 sp_Q99J04_Q99J04_MOUSE \\n               \\n               \\n                   \\n                   \\n                   \\n                 293-303, \\n                 MYRISTYL 282-287; \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 CK2_PHOSPHO_SITE 225-228; \\n                 KU 248-301; \\n               \\n               \\n                   \\n                   \\n                   \\n                 142-152, \\n                 MYRISTYL 281-286; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; 10-33, \\n                 PKC_PHOSPHO_SITE 253-255; \\n                 278-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.215; \\n                 MYRISTYL 43-48; RGD 193-195; \\n                 BASICPTASE 275-285; \\n               \\n               \\n                   \\n                   \\n                   \\n                 36-53, \\n                 MYRISTYL 161-166; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 ASN_GLYCOSYLATION 66-69; \\n                 285-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 168-183, \\n                 PKC_PHOSPHO_SITE 148-150; \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; \\n                 MYRISTYL 212-217; MYRISTYL \\n                 250-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 216-227, \\n                 313-318; MYRISTYL 29-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n                 PKC_PHOSPHO_SITE 80-82; \\n               \\n               \\n                   \\n                   \\n                   \\n                 330-341, \\n                 MYRISTYL 310-315; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 256-262, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.043; 84-105, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 55-67, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-132, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-254, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 235-243, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.176; \\n               \\n               \\n                 DEX0455_034.aa.4 \\n                 N \\n                 0 - \\n                 431-437, \\n                 CK2_PHOSPHO_SITE 439-442; \\n                 LDLRA_1 412-434; \\n               \\n               \\n                   \\n                   \\n                 o1-479; \\n                 1.056; \\n                 MYRISTYL 94-99; \\n                 KU 313-366; \\n               \\n               \\n                   \\n                   \\n                   \\n                 207-217, \\n                 CK2_PHOSPHO_SITE 290-293; \\n                 LDLa 399-436; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; \\n                 PKC_PHOSPHO_SITE 451-453; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 335-348, \\n                 MYRISTYL 226-231; MYRISTYL \\n                 340-350; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 277-282; PKC_PHOSPHO_SITE \\n                 BASICPTASE 312-326; \\n               \\n               \\n                   \\n                   \\n                   \\n                 444-450, \\n                 307-309; MYRISTYL 462-467; \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 MYRISTYL 378-383; \\n                 315-365; \\n               \\n               \\n                   \\n                   \\n                   \\n                 300-308, \\n                 CK2_PHOSPHO_SITE 415-418; \\n                 sp_Q99J04_Q99J04_MOUSE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.176; 9-16, \\n                 MYRISTYL 350-355; \\n                 315-365; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; \\n                 PKC_PHOSPHO_SITE 213-215; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 101-118, \\n                 ASN_GLYCOSYLATION 131-134; \\n                 350-365; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 MYRISTYL 347-352; MYRISTYL \\n                 ldl_recept_a \\n               \\n               \\n                   \\n                   \\n                   \\n                 380-385, \\n                 375-380; PKC_PHOSPHO_SITE \\n                 398-436; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.032; 54-61, \\n                 318-320; MYRISTYL 346-351; \\n                 BPTI_KUNITZ_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 MYRISTYL 464-469; \\n                 315-365; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-132, \\n                 CK2_PHOSPHO_SITE 139-142; \\n                 LDLRA_2 399-435; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 MYRISTYL 108-113; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 233-248, \\n                 PKC_PHOSPHO_SITE 145-147; \\n                 343-361; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; \\n                 ASN_GLYCOSYLATION 300-303; \\n               \\n               \\n                   \\n                   \\n                   \\n                 312-319, \\n                 RGD 258-260; AMIDATION 61-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; 75-98, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.215; \\n               \\n               \\n                   \\n                   \\n                   \\n                 178-197, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 466-476, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 393-420, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 281-292, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n               \\n               \\n                   \\n                   \\n                   \\n                 321-327, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.043; \\n               \\n               \\n                   \\n                   \\n                   \\n                 149-170, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 457-463, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n               \\n               \\n                   \\n                   \\n                   \\n                 358-368, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n               \\n               \\n                   \\n                   \\n                   \\n                 37-46, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n               \\n               \\n                 DEX0455_035.aa.1 \\n                 N \\n                 1 - \\n                 81-101, \\n                 MYRISTYL 54-59; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 i1-92; \\n                 1.213; \\n                 40-45; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 tm93-115; \\n                 103-118, \\n                 119-121; \\n               \\n               \\n                   \\n                   \\n                 o116-121; \\n                 1.241; 54-62, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n               \\n               \\n                 DEX0455_035.orf.2 \\n                 N \\n                 1 - \\n                 21-32, \\n                 PKC_PHOSPHO_SITE 30-32; \\n               \\n               \\n                   \\n                   \\n                 o1-125; \\n                 1.239; \\n                 MYRISTYL 46-51; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 tm126-148; \\n                 111-146, \\n                 45-50; MYRISTYL 7-12; \\n               \\n               \\n                   \\n                   \\n                 i149-149; \\n                 1.245; 57-64, \\n                 MYRISTYL 49-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 PKC_PHOSPHO_SITE 53-55; \\n               \\n               \\n                   \\n                   \\n                   \\n                 69-84, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; 4-11, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 86-95, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; 34-47, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.143; \\n               \\n               \\n                 DEX0455_035.aa.2 \\n                 N \\n                 1 - \\n                 54-62, \\n                 MYRISTYL 54-59; \\n               \\n               \\n                   \\n                   \\n                 i1-92; \\n                 1.049; \\n                 PKC_PHOSPHO_SITE 119-121; \\n               \\n               \\n                   \\n                   \\n                 tm93-115; \\n                 103-118, \\n                 MYRISTYL 40-45; \\n               \\n               \\n                   \\n                   \\n                 o116-121; \\n                 1.241; 81-101, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.213; \\n               \\n               \\n                 DEX0455_035.orf.3 \\n                 N \\n                 1 - \\n                 34-47, \\n                 MYRISTYL 7-12; \\n               \\n               \\n                   \\n                   \\n                 o1-125; \\n                 1.143; 57-64, \\n                 PKC_PHOSPHO_SITE 53-55; \\n               \\n               \\n                   \\n                   \\n                 tm126-148; \\n                 1.082; \\n                 MYRISTYL 45-50; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 i149-149; \\n                 21-32, \\n                 49-54; MYRISTYL 46-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.239; 86-95, \\n                 PKC_PHOSPHO_SITE 30-32; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-11, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 111-146, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.245; 69-84, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; \\n               \\n               \\n                 DEX0455_035.aa.3 \\n                 Y \\n                 1 - \\n                 34-42, \\n                 MYRISTYL 20-25; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 o1-71; \\n                 1.049; 61-81, \\n                 34-39; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 tm72-94; \\n                 1.213; \\n                 99-101; \\n               \\n               \\n                   \\n                   \\n                 i95-101; \\n                 83-98, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.241; 4-12, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n               \\n               \\n                 DEX0455_036.orf.1 \\n                 N \\n                 1 - \\n                 39-66, \\n                 TYR_PHOSPHO_SITE 68-75; \\n               \\n               \\n                   \\n                   \\n                 o1-40; \\n                 1.248; 76-92, \\n                 PKC_PHOSPHO_SITE 66-68; \\n               \\n               \\n                   \\n                   \\n                 tm41-63; \\n                 1.111; \\n                 MYRISTYL 61-66; \\n               \\n               \\n                   \\n                   \\n                 i64-95; \\n                 15-24, \\n                 CK2_PHOSPHO_SITE 87-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 PKC_PHOSPHO_SITE 65-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 11-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 53-58; \\n               \\n               \\n                 DEX0455_036.aa.1 \\n                 N \\n                 1 - \\n                 53-80, \\n                 TYR_PHOSPHO_SITE 82-89; \\n                 G_PROTEIN_RECEP_F1_1 \\n               \\n               \\n                   \\n                   \\n                 o1-54; \\n                 1.248; 4-16, \\n                 MYRISTYL 75-80; \\n                 18-34; \\n               \\n               \\n                   \\n                   \\n                 tm55-77; \\n                 1.142; \\n                 CK2_PHOSPHO_SITE 101-104; \\n               \\n               \\n                   \\n                   \\n                 i78-109; \\n                 29-38, \\n                 PKC_PHOSPHO_SITE 80-82; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; 90-106, \\n                 MYRISTYL 67-72; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 ASN_GLYCOSYLATION 25-28; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 79-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 10-15; \\n               \\n               \\n                 DEX0455_036.aa.2 \\n                 N \\n                 0 - \\n                 427-433, \\n                 PKC_PHOSPHO_SITE 1000-1002; \\n                 SEA_1 757-823; \\n               \\n               \\n                   \\n                   \\n                 o1-1485; \\n                 1.051; \\n                 MYRISTYL 191-196; \\n                 SEA_2 913-979; \\n               \\n               \\n                   \\n                   \\n                   \\n                 715-725, \\n                 CK2_PHOSPHO_SITE 440-443; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.093; \\n                 ASN_GLYCOSYLATION 338-341; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1161-1180, \\n                 TYR_PHOSPHO_SITE 237-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 986-989; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1009-1017, \\n                 ASN_GLYCOSYLATION 806-809; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n                 PKC_PHOSPHO_SITE 1415-1417; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1369-1374, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.054; \\n                 1385-1388; MYRISTYL 35-40; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1428-1480, \\n                 PKC_PHOSPHO_SITE 837-839; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.146; \\n                 ASN_GLYCOSYLATION 478-481; \\n               \\n               \\n                   \\n                   \\n                   \\n                 673-683, \\n                 PKC_PHOSPHO_SITE 1148-1150; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; 594-600, \\n                 CK2_PHOSPHO_SITE 82-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n                 MYRISTYL 124-129; \\n               \\n               \\n                   \\n                   \\n                   \\n                 170-199, \\n                 PKC_PHOSPHO_SITE 525-527; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 ASN_GLYCOSYLATION 457-460; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1388-1401, \\n                 MYRISTYL 436-441; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; \\n                 PKC_PHOSPHO_SITE 318-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 126-132, \\n                 CK2_PHOSPHO_SITE 550-553; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n                 CK2_PHOSPHO_SITE 1275-1278; \\n               \\n               \\n                   \\n                   \\n                   \\n                 809-823, \\n                 CK2_PHOSPHO_SITE 50-53; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n                 CK2_PHOSPHO_SITE 362-365; \\n               \\n               \\n                   \\n                   \\n                   \\n                 405-413, \\n                 PKC_PHOSPHO_SITE 102-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 CK2_PHOSPHO_SITE 674-677; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-110, \\n                 MYRISTYL 47-52; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 480-511, \\n                 ASN_GLYCOSYLATION 790-793; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 ASN_GLYCOSYLATION 1235-1238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1405-1417, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.159; \\n                 1365-1368; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1356-1366, \\n                 PKC_PHOSPHO_SITE 681-683; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; \\n                 CK2_PHOSPHO_SITE 862-865; \\n               \\n               \\n                   \\n                   \\n                   \\n                 895-901, \\n                 ASN_GLYCOSYLATION 1029-1032; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; 49-61, \\n                 PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 1038-1040; MYRISTYL 659-664; \\n               \\n               \\n                   \\n                   \\n                   \\n                 73-89, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 114-121, \\n                 634-637; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 636-638; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 14-43, \\n                 1081-1084; MYRISTYL 1126-1131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 MYRISTYL 954-959; \\n               \\n               \\n                   \\n                   \\n                   \\n                 416-422, \\n                 CK2_PHOSPHO_SITE 206-209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.032; \\n                 PKC_PHOSPHO_SITE 1103-1105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1195-1210, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 650-653; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 873-880, \\n                 518-521; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 258-260; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 341-355, \\n                 1097-1099; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.139; \\n                 ASN_GLYCOSYLATION 1117-1120; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1061-1081, \\n                 MYRISTYL 983-988; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.165; \\n                 ASN_GLYCOSYLATION 1101-1104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1051-1059, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.101; \\n                 769-772; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 135-145, \\n                 301-304; MYRISTYL 503-508; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 ASN_GLYCOSYLATION 1018-1021; \\n               \\n               \\n                   \\n                   \\n                   \\n                 572-578, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.043; \\n                 322-325; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 537-557, \\n                 925-928; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 746-748; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 603-620, \\n                 613-616; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 948-950; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1341-1351, \\n                 786-788; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 830-833; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 742-747, \\n                 162-164; MYRISTYL 1255-1260; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.017; \\n                 PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1276-1305, \\n                 347-349; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 882-884; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1022-1027, \\n                 942-944; MYRISTYL 203-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n                 MYRISTYL 971-976; \\n               \\n               \\n                   \\n                   \\n                   \\n                 853-869, \\n                 PKC_PHOSPHO_SITE 324-326; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 ASN_GLYCOSYLATION 946-949; \\n               \\n               \\n                   \\n                   \\n                   \\n                 987-998, \\n                 PKC_PHOSPHO_SITE 792-794; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 1173-1176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 750-756, \\n                 PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n                 570-572; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1241-1253, \\n                 733-735; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.146; \\n                 1268-1270; MYRISTYL 1227-1232; \\n               \\n               \\n                   \\n                   \\n                   \\n                 260-266, \\n                 MYRISTYL 904-909; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.074; \\n                 PKC_PHOSPHO_SITE 213-215; \\n               \\n               \\n                   \\n                   \\n                   \\n                 561-568, \\n                 PKC_PHOSPHO_SITE 1337-1339; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 517-527, \\n                 1231-1233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 TYR_PHOSPHO_SITE 1002-1009; \\n               \\n               \\n                   \\n                   \\n                   \\n                 583-589, \\n                 PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.035; \\n                 168-170; MYRISTYL 815-820; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1255-1269, \\n                 ASN_GLYCOSYLATION 166-169; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; 437-443, \\n                 PKC_PHOSPHO_SITE 474-476; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 CK2_PHOSPHO_SITE 1425-1428; \\n               \\n               \\n                   \\n                   \\n                   \\n                 361-374, \\n                 MYRISTYL 1426-1431; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 1141-1144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 649-667, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.12; \\n                 10-13; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1120-1134, \\n                 1214-1217; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n                 PKC_PHOSPHO_SITE 57-59; \\n               \\n               \\n                   \\n                   \\n                   \\n                 759-769, \\n                 PKC_PHOSPHO_SITE 630-632; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 PKC_PHOSPHO_SITE 12-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1142-1150, \\n                 MYRISTYL 1422-1427; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 1193-1195; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1311-1324, \\n                 ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 145-148; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 448-457, \\n                 6-8; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-221, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1029-1037, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.068; \\n               \\n               \\n                   \\n                   \\n                   \\n                 529-535, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.029; \\n               \\n               \\n                   \\n                   \\n                   \\n                 950-979, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 229-257, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n               \\n               \\n                   \\n                   \\n                   \\n                 904-925, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.198; \\n               \\n               \\n                   \\n                   \\n                   \\n                 271-277, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.063; \\n               \\n               \\n                   \\n                   \\n                   \\n                 697-708, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.073; \\n               \\n               \\n                   \\n                   \\n                   \\n                 326-335, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1105-1114, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 794-806, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; 93-101, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n               \\n               \\n                   \\n                   \\n                   \\n                 385-396, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n               \\n               \\n                   \\n                   \\n                   \\n                 829-839, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1040-1046, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n               \\n               \\n                   \\n                   \\n                   \\n                 884-890, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1184-1192, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n               \\n               \\n                   \\n                   \\n                   \\n                 638-647, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 282-301, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n               \\n               \\n                 DEX0455_036.orf.3 \\n                 N \\n                 1 - \\n                   \\n                 MYRISTYL 12-17; \\n               \\n               \\n                   \\n                   \\n                 o1-56; \\n                   \\n                 PKC_PHOSPHO_SITE 81-83; \\n               \\n               \\n                   \\n                   \\n                 tm57-79; \\n                   \\n                 CK2_PHOSPHO_SITE 103-106; \\n               \\n               \\n                   \\n                   \\n                 i80-111; \\n                   \\n                 TYR_PHOSPHO_SITE 84-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 11-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 82-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 69-74; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 11-16; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 10-13; TYR_PHOSPHO_SITE 6-12; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 77-82; \\n               \\n               \\n                 DEX0455_036.aa.3 \\n                 N \\n                 1 - \\n                 8-16, 1.19; \\n                 MYRISTYL 53-58; \\n               \\n               \\n                   \\n                   \\n                 o1-32; \\n                 68-84, \\n                 TYR_PHOSPHO_SITE 60-67; \\n               \\n               \\n                   \\n                   \\n                 tm33-55; \\n                 1.111; 31-58, \\n                 CK2_PHOSPHO_SITE 79-82; \\n               \\n               \\n                   \\n                   \\n                 i56-87; \\n                 1.248; \\n                 PKC_PHOSPHO_SITE 58-60; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 45-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 57-59; \\n               \\n               \\n                 DEX0455_036.orf.4 \\n                 N \\n                 0 - \\n                 20-29, \\n                 AMIDATION 34-37; \\n               \\n               \\n                   \\n                   \\n                 o1-92; \\n                 1.064; 41-53, \\n                 PKC_PHOSPHO_SITE 51-53; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 MYRISTYL 56-61; \\n               \\n               \\n                   \\n                   \\n                   \\n                 62-72, \\n                 PKC_PHOSPHO_SITE 88-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.212; 4-10, \\n                 PKC_PHOSPHO_SITE 62-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; \\n                 MYRISTYL 61-66; \\n               \\n               \\n                 DEX0455_036.aa.4 \\n                 N \\n                 1 - \\n                 64-80, \\n                 PKC_PHOSPHO_SITE 54-56; \\n               \\n               \\n                   \\n                   \\n                 o1-28; \\n                 1.111; 27-54, \\n                 MYRISTYL 49-54; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 tm29-51; \\n                 1.248; \\n                 41-46; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                 i52-83; \\n                 4-13, \\n                 75-78; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 53-55; TYR_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 56-63; \\n               \\n               \\n                 DEX0455_037.aa.1 \\n                 Y \\n                 0 - \\n                 199-205, \\n                 MYRISTYL 124-129; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                 o1-225; \\n                 1.08; 60-72, \\n                 CK2_PHOSPHO_SITE 154-157; \\n                 74-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 PKC_PHOSPHO_SITE 189-191; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 150-159, \\n                 PKC_PHOSPHO_SITE 106-108; \\n                 31-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 109-112; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 161-169, \\n                 MYRISTYL 129-134; \\n                 57-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 116-126, \\n                 ASN_GLYCOSYLATION 78-81; \\n                 LIPOCALIN 33-46; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; \\n                 PKC_PHOSPHO_SITE 109-111; \\n                 lipocalin \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-186, \\n                 MYRISTYL 144-149; \\n                 38-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.025; 45-52, \\n                 CK2_PHOSPHO_SITE 158-161; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 MYRISTYL 100-105; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 8-33, \\n                 133-138; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 51-54; MYRISTYL 76-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 210-219, \\n                 MYRISTYL 148-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n                 PKC_PHOSPHO_SITE 218-220; \\n               \\n               \\n                   \\n                   \\n                   \\n                 137-143, \\n                 MYRISTYL 47-52; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 94-108, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n               \\n               \\n                 DEX0455_037.orf.2 \\n                 N \\n                 0 - \\n                 294-307, \\n                 PKC_PHOSPHO_SITE 215-217; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                 o1-349; \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 173-175; \\n                 247-263; \\n               \\n               \\n                   \\n                   \\n                   \\n                 108-114, \\n                 MYRISTYL 185-190; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 AMIDATION 123-126; \\n                 172-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 200-209, \\n                 PKC_PHOSPHO_SITE 4-6; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 MYRISTYL 190-195; \\n                 339-349; \\n               \\n               \\n                   \\n                   \\n                   \\n                 320-335, \\n                 CAMP_PHOSPHO_SITE 228-231; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.295; \\n                 PKC_PHOSPHO_SITE 179-181; \\n                 148-164; \\n               \\n               \\n                   \\n                   \\n                   \\n                 263-275, \\n                 MYRISTYL 223-228; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.199; 15-38, \\n                 116-121; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.169; \\n                 83-85; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 246-252, \\n                 226-228; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.033; 6-12, \\n                 256-259; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n                 120-122; AMIDATION 226-229; \\n               \\n               \\n                   \\n                   \\n                   \\n                 87-93, \\n                 PKC_PHOSPHO_SITE 243-245; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 PKC_PHOSPHO_SITE 105-107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 237-242, \\n                 MYRISTYL 283-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; 45-70, \\n                 MYRISTYL 287-292; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 PKC_PHOSPHO_SITE 95-97; \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-149, \\n                 MYRISTYL 195-200; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; 97-102, \\n                 193-198; MYRISTYL 288-293; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 MYRISTYL 275-280; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 338-346, \\n                 285-290; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n                 1-4; MYRISTYL 99-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 159-172, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; 309-315, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n               \\n               \\n                   \\n                   \\n                   \\n                 219-225, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n               \\n               \\n                 DEX0455_037.aa.2 \\n                 Y \\n                 0 - \\n                 369-374, \\n                 PKC_PHOSPHO_SITE 83-85; \\n                 LIPOCALIN 256-269; \\n               \\n               \\n                   \\n                   \\n                 o1-413; \\n                 1.025; \\n                 MYRISTYL 270-275; \\n                 A1MCGLOBULIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 283-295, \\n                 PKC_PHOSPHO_SITE 329-331; \\n                 269-280; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 MYRISTYL 62-67; MYRISTYL \\n                 lipocalin 261-409; \\n               \\n               \\n                   \\n                   \\n                   \\n                 349-357, \\n                 236-241; MYRISTYL 171-176; \\n                 MAJORURINARY \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 338-347, \\n                 PKC_PHOSPHO_SITE 406-408; \\n                 263-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 AMIDATION 256-259; \\n                 VNEBNERGLAND \\n               \\n               \\n                   \\n                   \\n                   \\n                 50-56, \\n                 MYRISTYL 154-159; MYRISTYL \\n                 300-312; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 168-173; AMIDATION 86-89; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 268-275, \\n                 MYRISTYL 190-195; \\n                 368-382; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 CK2_PHOSPHO_SITE 342-345; \\n                 VNEBNERGLAND \\n               \\n               \\n                   \\n                   \\n                   \\n                 387-393, \\n                 PKC_PHOSPHO_SITE 216-218; \\n                 371-394; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 193-201, \\n                 MYRISTYL 336-341; MYRISTYL \\n                 LIPOCALIN 343-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 186-191; CK2_PHOSPHO_SITE \\n                 LIPOCALIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 317-332, \\n                 346-349; ASN_GLYCOSYLATION \\n                 260-272; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n                 301-304; PKC_PHOSPHO_SITE \\n                 VNEBNERGLAND \\n               \\n               \\n                   \\n                   \\n                   \\n                 218-224, \\n                 162-164; CK2_PHOSPHO_SITE \\n                 260-274; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.049; \\n                 177-180; MYRISTYL 256-261; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 231-240, \\n                 MYRISTYL 323-328; \\n                 385-403; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.065; 60-65, \\n                 ASN_GLYCOSYLATION 274-277; \\n                 LIPOCALIN 371-386; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 MYRISTYL 170-175; \\n                 A1MCGLOBULIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 71-77, \\n                 PKC_PHOSPHO_SITE 377-379; \\n                 366-387; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 PKC_PHOSPHO_SITE 58-60; \\n                 MAJORURINARY \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-135; \\n                 MYRISTYL 299-304; \\n                 393-410; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.033; 8-33 \\n                 ASN_GLYCOSLATION 226-229; \\n                 A1MCGLOBULIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 MYRISTYL 79-84; \\n                 394-413; \\n               \\n               \\n                   \\n                   \\n                   \\n                 141-148, \\n                 PKC_PHOSPHO_SITE 46-48; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; 95-113, \\n                 PKC_PHOSPHO_SITE 68-70; \\n                 280-290; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                   \\n                 MAJORURINARY \\n               \\n               \\n                   \\n                   \\n                   \\n                 398-407, \\n                   \\n                 365-386; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n                   \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 332-335; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 A1MCGLOBULIN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 334-353; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 254-277; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 297-315; \\n               \\n               \\n                 DEX0455_037.aa.3 \\n                 Y \\n                 0 - \\n                 95-112, \\n                 MYRISTYL 304-309; MYRISTYL \\n                 GLY_RICH 236-395; \\n               \\n               \\n                   \\n                   \\n                 o1-410; \\n                 1.075; \\n                 251-256; MYRISTYL 336-341; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-215, \\n                 MYRISTYL 264-269; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.033; \\n                 AMIDATION 285-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 163-172, \\n                 MYRISTYL 148-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 CAMP_PHOSPHO_SITE 191-194; \\n               \\n               \\n                   \\n                   \\n                   \\n                 200-205, \\n                 MYRISTYL 282-287; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; 60-65, \\n                 156-161; MYRISTYL 153-158; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 AMIDATION 189-192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-298, \\n                 PKC_PHOSPHO_SITE 189-191; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.068; \\n                 MYRISTYL 250-255; \\n               \\n               \\n                   \\n                   \\n                   \\n                 370-376, \\n                 PKC_PHOSPHO_SITE 136-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n                 PKC_PHOSPHO_SITE 142-144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 381-396, \\n                 MYRISTYL 299-304; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.295; 50-56, \\n                 AMIDATION 86-89; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 238-243; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 226-238, \\n                 178-180; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.199; \\n                 58-60; MYRISTYL 330-335; \\n               \\n               \\n                   \\n                   \\n                   \\n                 281-288, \\n                 AMIDATION 339-342; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 MYRISTYL 79-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-135, \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; 71-77, \\n                 MYRISTYL 186-191; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 318-323; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 345-351, \\n                 206-208; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; 8-33, \\n                 219-222; MYRISTYL 322-327; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 MYRISTYL 274-279; \\n               \\n               \\n                   \\n                   \\n                   \\n                 182-188, \\n                 PKC_PHOSPHO_SITE 83-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n                 MYRISTYL 248-253; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 308-317, \\n                 246-251; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.074; \\n                 46-48; MYRISTYL 158-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 399-407, \\n                 MYRISTYL 326-331; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n                 62-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                 360-368, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n               \\n               \\n                 DEX0455_037.aa.4 \\n                 Y \\n                 0 - \\n                 38-44, \\n                 MYRISTYL 71-76; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                 o1-135; \\n                 1.052; 8-36, \\n                 PKC_PHOSPHO_SITE 99-101; \\n                 90-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 MYRISTYL 54-59; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 109-115, \\n                 ASN_GLYCOSYLATION 93-96; \\n                 107-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; 46-89, \\n                 MYRISTYL 21-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.223; \\n                 PKC_PHOSPHO_SITE 128-130; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-129, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n               \\n               \\n                 DEX0455_037.aa.5 \\n                 N \\n                 0 - \\n                 110-116, \\n                 MYRISTYL 62-67; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 o1-150; \\n                 1.09; 11-27, \\n                 51-56; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n                 71-74; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 75-102, \\n                 33-35; MYRISTYL 74-79; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.154; \\n                 MYRISTYL 138-143; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 137-147, \\n                 65-70; MYRISTYL 69-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.065; 47-53, \\n                 PKC_PHOSPHO_SITE 116-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 MYRISTYL 115-120; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 38-45, \\n                 105-110; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.081; \\n                 46-49; \\n               \\n               \\n                 DEX0455_037.aa.6 \\n                 Y \\n                 0 - \\n                 149-158, \\n                 MYRISTYL 100-105; MYRISTYL \\n                 LIPOCALIN 33-46; \\n               \\n               \\n                   \\n                   \\n                 o1-224; \\n                 1.178; \\n                 147-152; PKC_PHOSPHO_SITE \\n                 lipocalin \\n               \\n               \\n                   \\n                   \\n                   \\n                 137-143, \\n                 188-190; MYRISTYL 47-52; \\n                 38-220; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 209-218, \\n                 ASN_GLYCOSYLATION 51-54; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.123; \\n                 MYRISTYL 124-129; \\n                 31-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 116-126, \\n                 PKC_PHOSPHO_SITE 106-108; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 198-204, \\n                 CK2_PHOSPHO_SITE 109-112; \\n                 57-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n                 MYRISTYL 129-134; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 45-52, \\n                 CK2_PHOSPHO_SITE 153-156; \\n                 74-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; 94-108, \\n                 ASN_GLYCOSYLATION 78-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n                 MYRISTYL 133-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 160-168, \\n                 PKC_PHOSPHO_SITE 109-111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 8-33, \\n                 CK2_PHOSPHO_SITE 157-160; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 MYRISTYL 76-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 60-72, \\n                 PKC_PHOSPHO_SITE 217-219; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 180-185, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.025; \\n               \\n               \\n                 DEX0455_037.aa.7 \\n                 Y \\n                 0 - \\n                 458-464, \\n                 MYRISTYL 156-161; MYRISTYL \\n                 LIPOCALIN 354-367; \\n               \\n               \\n                   \\n                   \\n                 o1-481; \\n                 1.07; 281-288, \\n                 336-341; PKC_PHOSPHO_SITE \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 58-60; PKC_PHOSPHO_SITE \\n                 395-413; \\n               \\n               \\n                   \\n                   \\n                   \\n                 381-393, \\n                 68-70; MYRISTYL 79-84; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 430-433; \\n                 352-375; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-135, \\n                 MYRISTYL 282-287; MYRISTYL \\n                 GLY_RICH 236-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; 358-364, \\n                 264-269; AMIDATION 285-288; \\n                 PGNDSYNTHASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.026; \\n                 MYRISTYL 445-450; \\n                 378-388; \\n               \\n               \\n                   \\n                   \\n                   \\n                 308-317, \\n                 MYRISTYL 330-335; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.074; \\n                 318-323; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 226-238, \\n                 46-48; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.199; 71-77, \\n                 136-138; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 83-85; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 200-205, \\n                 399-402; MYRISTYL 186-191; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 MYRISTYL 450-455; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 345-351, \\n                 299-304; MYRISTYL 421-426; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 CAMP_PHOSPHO_SITE 191-194; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-215, \\n                 MYRISTYL 454-459; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.033; \\n                 238-243; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 163-172, \\n                 427-429; MYRISTYL 62-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 PKC_PHOSPHO_SITE 206-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 182-188, \\n                 MYRISTYL 246-251; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.098; \\n                 AMIDATION 86-89; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-298, \\n                 274-279; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.068; 60-65, \\n                 189-191; MYRISTYL 397-402; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 MYRISTYL 368-373; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 415-429, \\n                 468-473; MYRISTYL 158-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; 50-56, \\n                 PKC_PHOSPHO_SITE 142-144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 MYRISTYL 148-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 95-112, \\n                 ASN_GLYCOSYLATION 372-375; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 CK2_PHOSPHO_SITE 219-222; \\n               \\n               \\n                   \\n                   \\n                   \\n                 366-373, \\n                 MYRISTYL 304-309; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; 8-33, \\n                 250-255; MYRISTYL 251-256; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                 PKC_PHOSPHO_SITE 472-474; \\n               \\n               \\n                   \\n                   \\n                   \\n                 437-447, \\n                 MYRISTYL 322-327; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; \\n                 153-158; AMIDATION 339-342; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 326-331; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 248-253; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 178-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 430-432; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 189-192; \\n               \\n               \\n                 DEX0455_038.aa.1 \\n                 N \\n                 1 - \\n                 605-611, \\n                 AMIDATION 268-271; \\n                 TROPOMYOSIN \\n               \\n               \\n                   \\n                   \\n                 i1-8; \\n                 1.066; \\n                 CK2_PHOSPHO_SITE 423-426; \\n                 622-639; \\n               \\n               \\n                   \\n                   \\n                 tm9-31; \\n                 944-955, \\n                 AMIDATION 278-281; \\n                 TROPOMYOSIN \\n               \\n               \\n                   \\n                   \\n                 o32-1088; \\n                 1.104; \\n                 AMIDATION 288-291; \\n                 695-715; \\n               \\n               \\n                   \\n                   \\n                   \\n                 810-816, \\n                 PKC_PHOSPHO_SITE 750-752; \\n                 Rib_recp_KP_reg \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 CAMP_PHOSPHO_SITE 476-479; \\n                 33-716; \\n               \\n               \\n                   \\n                   \\n                   \\n                 542-551, \\n                 PKC_PHOSPHO_SITE 722-724; \\n                 LYS_RICH 47-82; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                 CK2_PHOSPHO_SITE 632-635; \\n                 TROPOMYOSIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 674-692, \\n                 PKC_PHOSPHO_SITE 164-166; \\n                 764-787; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n                 AMIDATION 248-251; \\n                 TROPOMYOSIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 879-897, \\n                 PKC_PHOSPHO_SITE 209-211; \\n                 831-856; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; \\n                 AMIDATION 338-341; \\n               \\n               \\n                   \\n                   \\n                   \\n                 765-778, \\n                 PKC_PHOSPHO_SITE 525-527; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 PKC_PHOSPHO_SITE 481-483; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1006-1012, \\n                 CK2_PHOSPHO_SITE 781-784; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 518-531, \\n                 CK2_PHOSPHO_SITE 135-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 ASN_GLYCOSYLATION 931-934; \\n               \\n               \\n                   \\n                   \\n                   \\n                 856-863, \\n                 CK2_PHOSPHO_SITE 998-1001; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 AMIDATION 228-231; \\n               \\n               \\n                   \\n                   \\n                   \\n                 462-468, \\n                 AMIDATION 318-321; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.065; \\n                 CK2_PHOSPHO_SITE 1059-1062; \\n               \\n               \\n                   \\n                   \\n                   \\n                 174-186, \\n                 AMIDATION 238-241; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.219; \\n                 CK2_PHOSPHO_SITE 37-40; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1080-1085, \\n                 CK2_PHOSPHO_SITE 847-850; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n                 CK2_PHOSPHO_SITE 408-411; \\n               \\n               \\n                   \\n                   \\n                   \\n                 822-829, \\n                 PKC_PHOSPHO_SITE 135-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 144-172, \\n                 AMIDATION 308-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 AMIDATION 436-439; \\n               \\n               \\n                   \\n                   \\n                   \\n                 565-571, \\n                 AMIDATION 396-399; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; 841-849, \\n                 PKC_PHOSPHO_SITE 539-541; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.057; \\n                 AMIDATION 416-419; \\n               \\n               \\n                   \\n                   \\n                   \\n                 712-722, \\n                 CK2_PHOSPHO_SITE 615-618; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.142; \\n                 CK2_PHOSPHO_SITE 722-725; \\n               \\n               \\n                   \\n                   \\n                   \\n                 731-752, \\n                 AMIDATION 426-429; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 615-617; \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-135, \\n                 MYRISTYL 186-191; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 AMIDATION 446-449; \\n               \\n               \\n                   \\n                   \\n                   \\n                 661-667, \\n                 PKC_PHOSPHO_SITE 804-806; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.036; 89-127, \\n                 PKC_PHOSPHO_SITE 72-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.18; \\n                 AMIDATION 368-371; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1039-1050, \\n                 CK2_PHOSPHO_SITE 775-778; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.161; \\n                 CK2_PHOSPHO_SITE 561-564; \\n               \\n               \\n                   \\n                   \\n                   \\n                 904-911; \\n                 CK2_PHOSPHO_SITE 828-831; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.142; \\n                 AMIDATION 218-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 957-964, \\n                 AMIDATION 376-379; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n                 PKC_PHOSPHO_SITE 902-904; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1017-1027, \\n                 MYRISTYL 189-194; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; 973-986, \\n                 CK2_PHOSPHO_SITE 32-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 CK2_PHOSPHO_SITE 664-667; \\n               \\n               \\n                   \\n                   \\n                   \\n                 915-930, \\n                 ASN_GLYCOSYLATION 207-210; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.17; 780-793, \\n                 CK2_PHOSPHO_SITE 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 AMIDATION 386-389; \\n               \\n               \\n                   \\n                   \\n                   \\n                 493-510, \\n                 AMIDATION 198-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; 8-31, \\n                 AMIDATION 328-331; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 MYRISTYL 195-200; \\n               \\n               \\n                   \\n                   \\n                   \\n                 594-602, \\n                 PKC_PHOSPHO_SITE 1070-1072; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 AMIDATION 348-351; \\n               \\n               \\n                   \\n                   \\n                   \\n                 616-631, \\n                 AMIDATION 258-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 ASN_GLYCOSYLATION 91-94; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 1027-1029; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 1031-1036; \\n               \\n               \\n                 DEX0455_038.orf.3 \\n                 N \\n                 1 - \\n                 49-58, \\n                 PKC_PHOSPHO_SITE 263-265; \\n                 LYS_RICH 146-181; \\n               \\n               \\n                   \\n                   \\n                 i1-107; \\n                 1.131; \\n                 AMIDATION 357-360; \\n                 Rib_recp_KP_reg \\n               \\n               \\n                   \\n                   \\n                 tm108-130; \\n                 243-271, \\n                 PKC_PHOSPHO_SITE 234-236; \\n                 132-275; \\n               \\n               \\n                   \\n                   \\n                 o131-440; \\n                 1.186; \\n                 AMIDATION 297-300; \\n               \\n               \\n                   \\n                   \\n                   \\n                 188-226, \\n                 AMIDATION 377-380; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.18; 107-130, \\n                 AMIDATION 327-330; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 CK2_PHOSPHO_SITE 136-139; \\n               \\n               \\n                   \\n                   \\n                   \\n                 16-27, \\n                 CK2_PHOSPHO_SITE 234-237; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; 64-96, \\n                 PKC_PHOSPHO_SITE 308-310; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 AMIDATION 427-430; \\n               \\n               \\n                   \\n                   \\n                   \\n                 229-234, \\n                 CK2_PHOSPHO_SITE 135-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 AMIDATION 317-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 273-285, \\n                 AMIDATION 39-42; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.219; \\n                 70-75; AMIDATION 347-350; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 417-420; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 285-290; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 294-299; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 190-193; MYRISTYL 7-12; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 131-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 43-46; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 131-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 367-370; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 387-390; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 59-61; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 171-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 306-309; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 407-410; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 337-340; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 288-293; \\n               \\n               \\n                 DEX0455_038.aa.3 \\n                 Y \\n                 1 - \\n                 174-186, \\n                 ASN_GLYCOSYLATION 91-94; \\n                 Rib_recp_KP_reg \\n               \\n               \\n                   \\n                   \\n                 i1-8; \\n                 1.219; \\n                 MYRISTYL 417-422; MYRISTYL \\n                 278-426; \\n               \\n               \\n                   \\n                   \\n                 tm9-31; \\n                 415-423, \\n                 189-194; PKC_PHOSPHO_SITE \\n                 LYS_RICH 47-82; \\n               \\n               \\n                   \\n                   \\n                 o32-521; \\n                 1.129; \\n                 438-440; PKC_PHOSPHO_SITE \\n                 Rib_recp_KP_reg \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-135, \\n                 452-454; MYRISTYL 186-191; \\n                 33-176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n                 PKC_PHOSPHO_SITE 135-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 396-413, \\n                 AMIDATION 228-231; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; 8-31, \\n                 PKC_PHOSPHO_SITE 209-211; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.182; \\n                 AMIDATION 238-241; \\n               \\n               \\n                   \\n                   \\n                   \\n                 144-172, \\n                 PKC_PHOSPHO_SITE 519-521; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 AMIDATION 198-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                 339-377, \\n                 PKC_PHOSPHO_SITE 164-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.18; 455-464, \\n                 CK2_PHOSPHO_SITE 32-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                 AMIDATION 248-251; \\n               \\n               \\n                   \\n                   \\n                   \\n                 507-515, \\n                 PKC_PHOSPHO_SITE 385-387; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 AMIDATION 258-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 478-484, \\n                 MYRISTYL 195-200; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; 431-444, \\n                 CK2_PHOSPHO_SITE 135-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.078; \\n                 AMIDATION 278-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 380-385, \\n                 PKC_PHOSPHO_SITE 72-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; 89-127, \\n                 AMIDATION 308-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.18; \\n                 CK2_PHOSPHO_SITE 474-477; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 413-416; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 218-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 318-321; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 385-388; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 288-291; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 268-271; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 37-40; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 328-331; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 207-210; \\n               \\n               \\n                 DEX0455_039.aa.1 \\n                 Y \\n                 0 - \\n                 53-71, \\n                 PKC_PHOSPHO_SITE 72-74; \\n                 LYS_RICH 141-161; \\n               \\n               \\n                   \\n                   \\n                 o1-165; \\n                 1.107; 74-80, \\n                 MYRISTYL 41-46; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n                 78-83; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 86-93, \\n                 110-113; AMIDATION 157-160; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.108; 6-30, \\n                 PKC_PHOSPHO_SITE 3-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; \\n                 CK2_PHOSPHO_SITE 121-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 34-40, \\n                 MYRISTYL 76-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 CK2_PHOSPHO_SITE 117-120; \\n               \\n               \\n                   \\n                   \\n                   \\n                 125-132, \\n                 CK2_PHOSPHO_SITE 119-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.062; \\n                 CK2_PHOSPHO_SITE 32-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 152-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 66-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 45-50; \\n               \\n               \\n                 DEX0455_039.aa.2 \\n                 N \\n                 0 - \\n                 103-113, \\n                 MYRISTYL 76-81; AMIDATION \\n                 LYS_RICH 238-258; \\n               \\n               \\n                   \\n                   \\n                 o1-262; \\n                 1.095; 74-80, \\n                 254-257; MYRISTYL 112-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.034; \\n                 PKC_PHOSPHO_SITE 3-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                 210-229, \\n                 CK2_PHOSPHO_SITE 66-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; 6-30, \\n                 AMIDATION 249-252; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.126; \\n                 MYRISTYL 41-46; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 118-132, \\n                 173-178; MYRISTYL 45-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; 86-93, \\n                 PKC_PHOSPHO_SITE 72-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.108; \\n                 CK2_PHOSPHO_SITE 32-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 53-71, \\n                 MYRISTYL 78-83; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 139-168, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n               \\n               \\n                   \\n                   \\n                   \\n                 174-207, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.173; 34-40, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n               \\n               \\n                 DEX0455_040.orf.1 \\n                 Y \\n                 0 - \\n                   \\n                 CK2_PHOSPHO_SITE 51-54; \\n                 BPTI_KUNITZ_2_1 \\n               \\n               \\n                   \\n                   \\n                 o1-231; \\n                   \\n                 MYRISTYL 88-93; \\n                 56-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 98-101; \\n                 BASICPTASE 53-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 RGD 219-221; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 223-225; \\n                 179-197; KU 54-107; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 108-111; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 91-96; \\n                 91-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 190-192; \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 183-188; MYRISTYL \\n                 56-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 212-217; MYRISTYL 42-47; \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 204-209; \\n                 151-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 112-115; \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 186-191; MYRISTYL \\n                 151-201; KU \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 216-221; ASN_GLYCOSYLATION \\n                 149-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 75-78; MYRISTYL 24-29; \\n                 Kunitz_BPTI 56-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 182-187; \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 119-121; \\n                 81-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 87-92; \\n                 BPTI_KUNITZ_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 151-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 84-102; \\n               \\n               \\n                 DEX0455_040.aa.1 \\n                 Y \\n                 0 - \\n                 60-70, \\n                 MYRISTYL 69-74, \\n                 KU 131-184; \\n               \\n               \\n                   \\n                   \\n                 o1-213; \\n                 1.189; \\n                 CK2_PHOSPHO_SITE 80-83; \\n                 Kunitz_BPTI 38-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 107-113, \\n                 MYRISTYL 195-200; MYRISTYL \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 207-212; PKC_PHOSPHO_SITE \\n                 35-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                 147-153, \\n                 204-206; CK2_PHOSPHO_SITE \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.047; \\n                 90-93; MYRISTYL 70-75; \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 178-188, \\n                 MYRISTYL 73-78; MYRISTYL \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; 81-91, \\n                 203-208; MYRISTYL 168-173; \\n                 66-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 ASN_GLYCOSYLATION 94-97; \\n                 BPTI_KUNITZ_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-136, \\n                 ASN_GLYCOSYLATION 57-60; \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; 10-28, \\n                 PKC_PHOSPHO_SITE 172-174; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 MYRISTYL 6-11; MYRISTYL \\n                 161-179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-145, \\n                 165-170; PKC_PHOSPHO_SITE \\n                 BASICPTASE 73-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; 35-47, \\n                 101-103; MYRISTYL 198-203; \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.25; \\n                 MYRISTYL 164-169; \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 33-36; \\n                 BPTI_KUNITZ_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 24-29; \\n                 38-88; KU 36-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 38-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 BASICPTASE 63-73; \\n               \\n               \\n                 DEX0455_040.aa.2 \\n                 Y \\n                 0 - \\n                 35-47, \\n                 MYRISTYL 73-78; MYRISTYL \\n                 Kunitz_BPTI 38-88; \\n               \\n               \\n                   \\n                   \\n                 o1-242; \\n                 1.25; 107-113, \\n                 6-11; MYRISTYL 168-173; \\n                 KU 131-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 ASN_GLYCOSYLATION 94-97; \\n                 BPTI_KUNITZ_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-239, \\n                 CK2_PHOSPHO_SITE 33-36; \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.186; \\n                 PKC_PHOSPHO_SITE 172-174; \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 178-186, \\n                 MYRISTYL 70-75; MYRISTYL \\n                 38-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 69-74; PKC_PHOSPHO_SITE \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 130-136, \\n                 101-103; ASN_GLYCOSYLATION \\n                 66-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; 138-145, \\n                 57-60; CK2_PHOSPHO_SITE \\n                 sp_O43291_SPT2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 214-217; CK2_PHOSPHO_SITE \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 189-195, \\n                 90-93; MYRISTYL 164-169; \\n                 BPTI_KUNITZ_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; 10-28, \\n                 MYRISTYL 24-29; \\n                 38-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 CK2_PHOSPHO_SITE 80-83; \\n                 BPTI_KUNITZ_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 81-91, \\n                 MYRISTYL 165-170; \\n                 161-179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 60-70, \\n                   \\n                 Kunitz_BPTI \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.189; \\n                   \\n                 133-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 147-153, \\n                   \\n                 BASICPTASE 73-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.047; \\n                   \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 63-73; KU 36-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 BASICPTASE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 35-49; \\n               \\n               \\n                 DEX0455_041.orf.1 \\n                 N \\n                 0 - \\n                 16-21, \\n                 PKC_PHOSPHO_SITE 21-23; \\n               \\n               \\n                   \\n                   \\n                 o1-53; \\n                 1.006; 4-11, \\n                 PKC_PHOSPHO_SITE 42-44; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; \\n                 PKC_PHOSPHO_SITE 20-22; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 7-12; \\n               \\n               \\n                 DEX0455_041.aa.1 \\n                 N \\n                 0 - \\n                 33-38, \\n                 MYRISTYL 28-33; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 o1-43; \\n                 1.046; 5-20, \\n                 6-11; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129, \\n                 32-35; \\n               \\n               \\n                 DEX0455_041.orf.2 \\n                 N \\n                 0 - \\n                 4-17, \\n                 ASN_GLYCOSYLATION 29-32; \\n               \\n               \\n                   \\n                   \\n                 o1-66; \\n                 1.129; 30-35, \\n                 MYRISTYL 25-30; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n                 PKC_PHOSPHO_SITE 55-57; \\n               \\n               \\n                 DEX0455_041.aa.2 \\n                 Y \\n                 0 - \\n                 6-18, \\n               \\n               \\n                   \\n                   \\n                 i1-34; \\n                 1.141; 25-31, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                 DEX0455_042.orf.1 \\n                 N \\n                 0 - \\n                   \\n                 PKC_PHOSPHO_SITE 3-5; \\n               \\n               \\n                   \\n                   \\n                 o1-116; \\n                   \\n                 MYRISTYL 10-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 7-10; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 75-78; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 40-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 41-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 81-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 34-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 72-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 51-59; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 93-96; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 80-83; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 29-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 40-48; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 82-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 17-20; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 56-59; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 69-72; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 71-74; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 112-115; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 76-79; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 42-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 50-57; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 44-46; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 77-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 24-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 4-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 113-115; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 71-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 35-41; \\n               \\n               \\n                 DEX0455_042.aa.1 \\n                 N \\n                 0 - \\n                   \\n                 MYRISTYL 8-13; \\n               \\n               \\n                   \\n                   \\n                 o1-122; \\n                   \\n                 CK2_PHOSPHO_SITE 87-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 9-11; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 99-102; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 47-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 46-49; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 23-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 40-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 30-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 88-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 16-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 35-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 48-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 7-10; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 87-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 TYR_PHOSPHO_SITE 41-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 42-45; \\n               \\n               \\n                 DEX0455_043.orf.1 \\n                 Y \\n                 0 - \\n                 64-72, \\n                 MYRISTYL 3-8; \\n               \\n               \\n                   \\n                   \\n                 o1-86; \\n                 1.133; 76-83, \\n                 ASN_GLYCOSYLATION 60-63; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.14; \\n                 PKC_PHOSPHO_SITE 29-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                 27-38, \\n                 MYRISTYL 64-69; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; 4-20, \\n                 56-61; MYRISTYL 61-66; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n               \\n               \\n                   \\n                   \\n                   \\n                 41-50, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n               \\n               \\n                 DEX0455_043.aa.1 \\n                 N \\n                 0 - \\n                 11-26, \\n                 CK2_PHOSPHO_SITE 24-27; \\n               \\n               \\n                   \\n                   \\n                 o1-30; \\n                 1.155; \\n               \\n               \\n                 DEX0455_043.orf.2 \\n                 N \\n                 0 - \\n                   \\n                 CK2_PHOSPHO_SITE 94-97; \\n               \\n               \\n                   \\n                   \\n                 o1-125; \\n                   \\n                 MYRISTYL 47-52; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 84-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 8-13; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 77-79; \\n               \\n               \\n                 DEX0455_043.orf.3 \\n                 N \\n                 0 - \\n                 89-95, \\n                 PKC_PHOSPHO_SITE 42-44; \\n               \\n               \\n                   \\n                   \\n                 i1-104; \\n                 1.052; 59-67, \\n                 CK2_PHOSPHO_SITE 34-37; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.023; \\n                 MYRISTYL 9-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                 10-38, \\n                 PKC_PHOSPHO_SITE 45-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.08; \\n                 PKC_PHOSPHO_SITE 3-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 11-14; \\n               \\n               \\n                 DEX0455_044.aa.1 \\n                 N \\n                 1 - \\n                 14-24, \\n                 MYRISTYL 27-32; \\n               \\n               \\n                   \\n                   \\n                 i1-31; \\n                 1.177; 29-58, \\n               \\n               \\n                   \\n                   \\n                 tm32-54; \\n                 1.199; \\n               \\n               \\n                   \\n                   \\n                 o55-61; \\n               \\n               \\n                 DEX0455_045.orf.1 \\n                 N \\n                 0 - \\n                 178-190, \\n                 AMIDATION 65-68; \\n               \\n               \\n                   \\n                   \\n                 o1-193; \\n                 1.144; \\n                 PKC_PHOSPHO_SITE 129-131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 135-145, \\n                 MYRISTYL 10-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.124; \\n                 PKC_PHOSPHO_SITE 93-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-128, \\n                 MYRISTYL 81-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; 70-111, \\n                 CAMP_PHOSPHO_SITE 175-178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n                 AMIDATION 173-176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 25-46, \\n                 PKC_PHOSPHO_SITE 5-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.115; 51-66, \\n                 CAMP_PHOSPHO_SITE 110-113; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.216; \\n                 MYRISTYL 83-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 167-170; \\n               \\n               \\n                 DEX0455_045.aa.1 \\n                 N \\n                 0 - \\n                 25-46, \\n                 CK2_PHOSPHO_SITE 149-152; \\n               \\n               \\n                   \\n                   \\n                 o1-163; \\n                 1.115; \\n                 AMIDATION 65-68; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-128, \\n                 PKC_PHOSPHO_SITE 5-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; 51-66, \\n                 MYRISTYL 10-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.216; \\n                 CAMP_PHOSPHO_SITE 146-149; \\n               \\n               \\n                   \\n                   \\n                   \\n                 135-144, \\n                 MYRISTYL 81-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.081; 70-111, \\n                 PKC_PHOSPHO_SITE 93-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n                 CAMP_PHOSPHO_SITE 110-113; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 83-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 129-131; \\n               \\n               \\n                 DEX0455_046.orf.1 \\n                 N \\n                 0 - \\n                   \\n                 MYRISTYL 115-120; \\n               \\n               \\n                   \\n                   \\n                 o1-227; \\n                   \\n                 PKC_PHOSPHO_SITE 3-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 97-102; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 133-138; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 163-166; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 3-6; MYRISTYL 3-8; \\n               \\n               \\n                 DEX0455_046.aa.1 \\n                 N \\n                 0 - \\n                 4-11, \\n                 MYRISTYL 90-95; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                 o1-198; \\n                 1.144; 51-59, \\n                 71-76; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                 156-159; MYRISTYL 161-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 160-184, \\n                 PKC_PHOSPHO_SITE 63-65; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.151; 15-25, \\n                 MYRISTYL 153-158; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; \\n                 149-154; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 93-99, \\n                 119-121; MYRISTYL 86-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.11; 73-85, \\n                 MYRISTYL 157-162; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 68-73; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 32-40, \\n                 120-123; MYRISTYL 104-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.076; \\n               \\n               \\n                   \\n                   \\n                   \\n                 146-153, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n               \\n               \\n                 DEX0455_047.orf.1 \\n                 N \\n                 0 - \\n                 66-100, \\n                 AMIDATION 14-17; \\n               \\n               \\n                   \\n                   \\n                 o1-103; \\n                 1.232; 23-44, \\n                 PKC_PHOSPHO_SITE 73-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.101; \\n                 CK2_PHOSPHO_SITE 33-36; \\n               \\n               \\n                   \\n                   \\n                   \\n                 10-17, \\n                 PKC_PHOSPHO_SITE 59-61; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n                 MYRISTYL 96-101; \\n               \\n               \\n                 DEX0455_047.aa.1 \\n                 Y \\n                 0 - \\n                 42-48, \\n                 ASN_GLYCOSYLATION 32-35; \\n               \\n               \\n                   \\n                   \\n                 o1-65; \\n                 1.073; 55-62, \\n                 CK2_PHOSPHO_SITE 23-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n                 ASN_GLYCOSYLATION 39-42; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-30, \\n                 MYRISTYL 19-24; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.142; 32-40, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                 DEX0455_047.orf.2 \\n                 N \\n                 0 - \\n                 10-17, \\n                 AMIDATION 14-17; \\n               \\n               \\n                   \\n                   \\n                 o1-103; \\n                 1.088; 66-100, \\n                 CK2_PHOSPHO_SITE 33-36; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 MYRISTYL 96-101; \\n               \\n               \\n                   \\n                   \\n                   \\n                 23-44, \\n                 PKC_PHOSPHO_SITE 73-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.101; \\n                 PKC_PHOSPHO_SITE 59-61; \\n               \\n               \\n                 DEX0455_047.aa.2 \\n                 N \\n                 0 - \\n               \\n               \\n                   \\n                   \\n                 o1-27; \\n               \\n               \\n                 DEX0455_048.aa.1 \\n                 Y \\n                 0 - \\n                 111-117, \\n                 TYR_PHOSPHO_SITE 105-112; \\n               \\n               \\n                   \\n                   \\n                 o1-154; \\n                 1.077; 83-97, \\n                 MYRISTYL 80-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 PKC_PHOSPHO_SITE 56-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-60, \\n                 CK2_PHOSPHO_SITE 37-40; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.197; 99-107, \\n                 TYR_PHOSPHO_SITE 79-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; \\n               \\n               \\n                   \\n                   \\n                   \\n                 63-78, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n               \\n               \\n                   \\n                   \\n                   \\n                 126-151, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n               \\n               \\n                 DEX0455_048.orf.2 \\n                 N \\n                 0 - \\n                 151-173, \\n                 PKC_PHOSPHO_SITE 169-171; \\n               \\n               \\n                   \\n                   \\n                 o1-176; \\n                 1.203; 86-100, \\n                 MYRISTYL 83-88; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 PKC_PHOSPHO_SITE 59-61; \\n               \\n               \\n                   \\n                   \\n                   \\n                 114-120, \\n                 TYR_PHOSPHO_SITE 108-115; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 CK2_PHOSPHO_SITE 40-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                 129-141, \\n                 TYR_PHOSPHO_SITE 82-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 CK2_PHOSPHO_SITE 24-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 102-110, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; 66-81, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-63, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.197; \\n               \\n               \\n                 DEX0455_048.aa.2 \\n                 N \\n                 0 - \\n                 39-50, \\n                 TYR_PHOSPHO_SITE 203-210; \\n               \\n               \\n                   \\n                   \\n                 o1-271; \\n                 1.182; 4-21, \\n                 PKC_PHOSPHO_SITE 154-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 PKC_PHOSPHO_SITE 34-36; \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-195, \\n                 RGD 26-28; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; 98-158, \\n                 PKC_PHOSPHO_SITE 264-266; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 ASN_GLYCOSYLATION 95-98; \\n               \\n               \\n                   \\n                   \\n                   \\n                 246-268, \\n                 MYRISTYL 38-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.203; \\n                 PKC_PHOSPHO_SITE 21-23; \\n               \\n               \\n                   \\n                   \\n                   \\n                 161-176, \\n                 CK2_PHOSPHO_SITE 88-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n                 TYR_PHOSPHO_SITE 177-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 197-205, \\n                 MYRISTYL 178-183; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.051; 66-90, \\n                 CK2_PHOSPHO_SITE 135-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n                 PKC_PHOSPHO_SITE 96-98; \\n               \\n               \\n                   \\n                   \\n                   \\n                 224-236, \\n                 CK2_PHOSPHO_SITE 119-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-215, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; 57-62, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.053; \\n               \\n               \\n                 DEX0455_049.aa.1 \\n                 N \\n                 1 - \\n                 199-209, \\n                 PKC_PHOSPHO_SITE 243-245; \\n                 THYROGLOBULIN_1 \\n               \\n               \\n                   \\n                   \\n                 o1-219; \\n                 1.147; \\n                 MYRISTYL 66-71; \\n                 49-77; \\n               \\n               \\n                   \\n                   \\n                 tm220-242; \\n                 129-139, \\n                 TYR_PHOSPHO_SITE 105-111; \\n                 thyroglobulin_1 \\n               \\n               \\n                   \\n                   \\n                 i243-268; \\n                 1.104; 95-105, \\n                 MYRISTYL 77-82; \\n                 20-89; TY 50-93; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 PKC_PHOSPHO_SITE 154-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-24, \\n                 CK2_PHOSPHO_SITE 88-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.212; \\n                 MYRISTYL 217-222; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 219-244, \\n                 9-14; PKC_PHOSPHO_SITE 81-83; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.317; 58-63, \\n                 ASN_GLYCOSYLATION 152-155; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n                 PKC_PHOSPHO_SITE 115-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 145-153, \\n                 ASN_GLYCOSYLATION 28-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; 69-76, \\n                 AMIDATION 32-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 MYRISTYL 29-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 86-93, \\n                 PKC_PHOSPHO_SITE 125-127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 MYRISTYL 57-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 163-172, \\n                 CK2_PHOSPHO_SITE 81-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 CK2_PHOSPHO_SITE 157-160; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-182, \\n                 PKC_PHOSPHO_SITE 90-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 MYRISTYL 40-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 65-68; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 182-184; \\n               \\n               \\n                 DEX0455_049.aa.2 \\n                 N \\n                 1 - \\n                 237-246, \\n                 MYRISTYL 10-15; \\n                 TY 124-167; \\n               \\n               \\n                   \\n                   \\n                 o1-293; \\n                 1.122; 52-67, \\n                 ASN_GLYCOSYLATION 139-142; \\n                 thyroglobulin_1 \\n               \\n               \\n                   \\n                   \\n                 tm294-316; \\n                 1.194; \\n                 MYRISTYL 291-296; \\n                 94-163; \\n               \\n               \\n                   \\n                   \\n                 i317-342; \\n                 169-179, \\n                 TYR_PHOSPHO_SITE 179-185; \\n                 THYROGLOBULIN_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 MYRISTYL 29-34; \\n                 123-151; \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-137, \\n                 PKC_PHOSPHO_SITE 164-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n                 MYRISTYL 151-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 219-227, \\n                 PKC_PHOSPHO_SITE 228-230; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; 41-50, \\n                 ASN_GLYCOSYLATION 25-28; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.059; \\n                 CK2_PHOSPHO_SITE 231-234; \\n               \\n               \\n                   \\n                   \\n                   \\n                 203-213, \\n                 MYRISTYL 32-37; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 140-145; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 273-283, \\n                 155-158; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 317-319; MYRISTYL 103-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 250-256, \\n                 PKC_PHOSPHO_SITE 155-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; 24-29, \\n                 PKC_PHOSPHO_SITE 256-258; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.038; \\n                 MYRISTYL 114-119; \\n               \\n               \\n                   \\n                   \\n                   \\n                 293-318, \\n                 ASN_GLYCOSYLATION 226-229; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.317; 73-98, \\n                 PKC_PHOSPHO_SITE 189-191; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; \\n                 MYRISTYL 131-136; \\n               \\n               \\n                   \\n                   \\n                   \\n                 143-150, \\n                 PKC_PHOSPHO_SITE 199-201; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 CK2_PHOSPHO_SITE 162-165; \\n               \\n               \\n                   \\n                   \\n                   \\n                 160-167, \\n                 ASN_GLYCOSYLATION 102-105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 AMIDATION 106-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 80-85; \\n               \\n               \\n                 DEX0455_049.aa.3 \\n                 Y \\n                 0 - \\n                 45-70, \\n                 MYRISTYL 112-117; \\n                 thyroglobulin_1 \\n               \\n               \\n                   \\n                   \\n                 o1-240; \\n                 1.193; 4-22, \\n                 PKC_PHOSPHO_SITE 127-129; \\n                 66-135; TY 96-139; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 PKC_PHOSPHO_SITE 136-138; \\n                 THYROGLOBULIN_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-139, \\n                 TYR_PHOSPHO_SITE 151-157; \\n                 95-123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 MYRISTYL 10-15; AMIDATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 191-199, \\n                 78-81; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; 24-39, \\n                 198-201; MYRISTYL 86-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.194; \\n                 CK2_PHOSPHO_SITE 134-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-109, \\n                 MYRISTYL 75-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n                 PKC_PHOSPHO_SITE 171-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 115-122, \\n                 ASN_GLYCOSYLATION 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 PKC_PHOSPHO_SITE 200-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                 175-185, \\n                 MYRISTYL 123-128; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 52-57; MYRISTYL 103-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-216, \\n                 CK2_PHOSPHO_SITE 127-130; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 CK2_PHOSPHO_SITE 203-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 141-151, \\n                 PKC_PHOSPHO_SITE 161-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 ASN_GLYCOSYLATION 111-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 220-237, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.155; \\n               \\n               \\n                 DEX0455_049.aa.4 \\n                 Y \\n                 1 - \\n                 191-198, \\n                 PKC_PHOSPHO_SITE 212-214; \\n                 thyroglobulin_1 \\n               \\n               \\n                   \\n                   \\n                 o1-341; \\n                 1.121; \\n                 CK2_PHOSPHO_SITE 203-206; \\n                 142-211; TY \\n               \\n               \\n                   \\n                   \\n                 tm342-364; \\n                 251-261, \\n                 AMIDATION 154-157; \\n                 172-215; \\n               \\n               \\n                   \\n                   \\n                 i365-390; \\n                 1.104; 4-23, \\n                 PKC_PHOSPHO_SITE 247-249; \\n                 THYROGLOBULIN_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n                 PKC_PHOSPHO_SITE 365-367; \\n                 171-199; \\n               \\n               \\n                   \\n                   \\n                   \\n                 217-227, \\n                 PKC_PHOSPHO_SITE 94-96; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 MYRISTYL 151-156; \\n               \\n               \\n                   \\n                   \\n                   \\n                 321-331, \\n                 PKC_PHOSPHO_SITE 304-306; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 48-51; \\n               \\n               \\n                   \\n                   \\n                   \\n                 298-304, \\n                 MYRISTYL 10-15; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 188-193; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 341-366, \\n                 237-239; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.317; \\n                 150-153; MYRISTYL 179-184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-215, \\n                 TYR_PHOSPHO_SITE 227-233; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 PKC_PHOSPHO_SITE 70-72; \\n               \\n               \\n                   \\n                   \\n                   \\n                 267-275, \\n                 RGD 66-68; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 ASN_GLYCOSYLATION 274-277; \\n               \\n               \\n                   \\n                   \\n                   \\n                 180-185, \\n                 PKC_PHOSPHO_SITE 203-205; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n                 MYRISTYL 39-44; AMIDATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 285-294, \\n                 81-84; MYRISTYL 128-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 92-115, \\n                 PKC_PHOSPHO_SITE 276-278; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.194; \\n                 MYRISTYL 162-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-146, \\n                 ASN_GLYCOSYLATION 187-190; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; 71-77, \\n                 MYRISTYL 199-204; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.09; \\n                 CK2_PHOSPHO_SITE 210-213; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 339-344; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 279-282; \\n               \\n               \\n                 DEX0455_049.aa.5 \\n                 Y \\n                 1 - \\n                 4-22, \\n                 MYRISTYL 86-91; \\n                 thyroglobulin_1 \\n               \\n               \\n                   \\n                   \\n                 o1-265; \\n                 1.166; \\n                 ASN_GLYCOSYLATION 198-201; \\n                 66-135; TY 96-139; \\n               \\n               \\n                   \\n                   \\n                 tm266-288; \\n                 115-122, \\n                 PKC_PHOSPHO_SITE 127-129; \\n                 THYROGLOBULIN_1 \\n               \\n               \\n                   \\n                   \\n                 i289-314; \\n                 1.133; \\n                 PKC_PHOSPHO_SITE 228-230; \\n                 95-123; \\n               \\n               \\n                   \\n                   \\n                   \\n                 132-139, \\n                 MYRISTYL 75-80; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; \\n                 52-57; MYRISTYL 103-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 222-228, \\n                 CK2_PHOSPHO_SITE 127-130; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 PKC_PHOSPHO_SITE 289-291; \\n               \\n               \\n                   \\n                   \\n                   \\n                 191-199, \\n                 PKC_PHOSPHO_SITE 200-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.156; \\n                 TYR_PHOSPHO_SITE 151-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 245-255, \\n                 MYRISTYL 123-128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 203-206; \\n               \\n               \\n                   \\n                   \\n                   \\n                 141-151, \\n                 PKC_PHOSPHO_SITE 136-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; 45-70, \\n                 ASN_GLYCOSYLATION 111-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.193; \\n                 MYRISTYL 263-268; \\n               \\n               \\n                   \\n                   \\n                   \\n                 209-218, \\n                 AMIDATION 78-81; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 CK2_PHOSPHO_SITE 134-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 175-185, \\n                 ASN_GLYCOSYLATION 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; 24-39, \\n                 PKC_PHOSPHO_SITE 161-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.194; \\n                 PKC_PHOSPHO_SITE 171-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 104-109, \\n                 MYRISTYL 10-15; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.099; \\n                 112-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 265-290, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.317; \\n               \\n               \\n                 DEX0455_050.orf.1 \\n                 Y \\n                 1 - \\n                 60-71, \\n                 CK2_PHOSPHO_SITE 72-75; \\n                 PHE_RICH 18-29; \\n               \\n               \\n                   \\n                   \\n                 i1-11; \\n                 1.212; 79-93, \\n                 PKC_PHOSPHO_SITE 39-41; \\n               \\n               \\n                   \\n                   \\n                 tm12-31; \\n                 1.103; \\n                 PKC_PHOSPHO_SITE 113-115; \\n               \\n               \\n                   \\n                   \\n                 o32-122; \\n                 97-119, \\n                 ASN_GLYCOSYLATION 2-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.218; 37-45, \\n                 CK2_PHOSPHO_SITE 4-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 12-34, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n               \\n               \\n                 DEX0455_050.aa.1 \\n                 Y \\n                 0 - \\n                 9-25, \\n                 PKC_PHOSPHO_SITE 4-6; \\n                 ACTINS_2 29-37; \\n               \\n               \\n                   \\n                   \\n                 o1-48; \\n                 1.107; 40-45, \\n                 CK2_PHOSPHO_SITE 31-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 MYRISTYL 16-21; \\n               \\n               \\n                 DEX0455_051.aa.1 \\n                 N \\n                 0 - \\n                 130-149, \\n                 TRY_PHOSPHO_SITE 182-189; \\n                 PRO_RICH 407-449; \\n               \\n               \\n                   \\n                   \\n                 o1-596; \\n                 1.131; \\n                 PKC_PHOSPHO_SITE 463-465; \\n                 SH3 351-410; \\n               \\n               \\n                   \\n                   \\n                   \\n                 509-519, \\n                 PKC_PHOSPHO_SITE 167-169; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; 40-45, \\n                 MYRISTYL 459-464; MYRISTYL \\n                 368-383; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.041; \\n                 530-535; CK2_PHOSPHO_SITE \\n                 354-409; \\n               \\n               \\n                   \\n                   \\n                   \\n                 523-540, \\n                 46-49; CAMP_PHOSPHO_SITE \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 65-68; CK2_PHOSPHO_SITE \\n                 360-403; \\n               \\n               \\n                   \\n                   \\n                   \\n                 215-230, \\n                 295-298; CK2_PHOSPHO_SITE \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 99-105, \\n                 106-109; PKC_PHOSPHO_SITE \\n                 354-364; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.045; \\n                 241-243; MYRISTYL 524-529; \\n                 SH3DOMAIN 396-408; \\n               \\n               \\n                   \\n                   \\n                   \\n                 397-422, \\n                 PKC_PHOSPHO_SITE 371-373; \\n                 SH3 354-408; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 MYRISTYL 506-511; \\n               \\n               \\n                   \\n                   \\n                   \\n                 239-258, \\n                 CK2_PHOSPHO_SITE 549-552; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 12-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 573-579, \\n                 CK2_PHOSPHO_SITE 463-466; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 13-16; \\n               \\n               \\n                   \\n                   \\n                   \\n                 327-334, \\n                 ASN_GLYCOSYLATION 2-5; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n                 CK2_PHOSPHO_SITE 185-188; \\n               \\n               \\n                   \\n                   \\n                   \\n                 369-383, \\n                 MYRISTYL 26-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 PKC_PHOSPHO_SITE 384-386; \\n               \\n               \\n                   \\n                   \\n                   \\n                 428-443, \\n                 CK2_PHOSPHO_SITE 574-577; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 CK2_PHOSPHO_SITE 282-285; \\n               \\n               \\n                   \\n                   \\n                   \\n                 470-477, \\n                 PKC_PHOSPHO_SITE 260-262; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; 58-76, \\n                 PKC_PHOSPHO_SITE 178-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 CK2_PHOSPHO_SITE 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 490-496, \\n                 ASN_GLYCOSYLATION 231-234; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n                 MYRISTYL 79-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 554-568, \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 260-263; \\n               \\n               \\n                   \\n                   \\n                   \\n                 500-506, \\n                 PKC_PHOSPHO_SITE 33-35; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n                 ASN_GLYCOSYLATION 366-369; \\n               \\n               \\n                   \\n                   \\n                   \\n                 545-551, \\n                 CK2_PHOSPHO_SITE 214-217; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 MYRISTYL 103-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-128, \\n                 CK2_PHOSPHO_SITE 565-568; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 AMIDATION 163-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 354-361, \\n                 CK2_PHOSPHO_SITE 269-272; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 174-182, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n               \\n               \\n                   \\n                   \\n                   \\n                 298-304, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 154-160, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; \\n               \\n               \\n                   \\n                   \\n                   \\n                 189-205, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; 25-31, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.047; \\n               \\n               \\n                 DEX0455_051.aa.2 \\n                 N \\n                 0 - \\n                 4-41, \\n                 PKC_PHOSPHO_SITE 401-403; \\n                 SH3DOMAIN 274-286; \\n               \\n               \\n                   \\n                   \\n                 o1-408; \\n                 1.232; 67-83, \\n                 CK2_PHOSPHO_SITE 341-344; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 63-66; \\n                 246-261; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-212, \\n                 ASN_GLYCOSYLATION 109-112; \\n                 232-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n                 PKC_PHOSPHO_SITE 56-58; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 306-321, \\n                 PKC_PHOSPHO_SITE 138-140; \\n                 165-182; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 PKC_PHOSPHO_SITE 249-251; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-136, \\n                 AMIDATION 401-404; \\n                 50-62; PRO_RICH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 119-121; \\n                 285-327; \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-239, \\n                 PKC_PHOSPHO_SITE 262-264; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 CK2_PHOSPHO_SITE 138-141; \\n                 238-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 368-374, \\n                 ASN_GLYCOSYLATION 244-247; \\n                 SH3 229-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; 93-108, \\n                 CK2_PHOSPHO_SITE 147-150; \\n                 SH3 232-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 MYRISTYL 384-389; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-300, \\n                 CK2_PHOSPHO_SITE 92-95; \\n                 232-242; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 PKC_PHOSPHO_SITE 341-343; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-182, \\n                 MYRISTYL 337-342; \\n                 308-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 TYR_PHOSPHO_SITE 60-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-355, \\n                 CK2_PHOSPHO_SITE 160-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 CK2_PHOSPHO_SITE 173-176; \\n               \\n               \\n                   \\n                   \\n                   \\n                 387-397, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-261, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 45-60, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n               \\n               \\n                   \\n                   \\n                   \\n                 378-384, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n               \\n               \\n                 DEX0455_051.aa.3 \\n                 N \\n                 0 - \\n                 383-393, \\n                 CK2_PHOSPHO_SITE 88-91; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                 o1-470; \\n                 1.077; \\n                 CK2_PHOSPHO_SITE 423-426; \\n                 234-277; \\n               \\n               \\n                   \\n                   \\n                   \\n                 428-442, \\n                 PKC_PHOSPHO_SITE 258-260; \\n                 SH3 228-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 156-159; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 344-351, \\n                 CK2_PHOSPHO_SITE 169-172; \\n                 304-316; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 CK2_PHOSPHO_SITE 59-62; \\n                 SH3DOMAIN 228-238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 374-380, \\n                 CK2_PHOSPHO_SITE 143-146; \\n                 PRO_RICH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; 4-37, \\n                 MYRISTYL 380-385; \\n                 281-323; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 CK2_PHOSPHO_SITE 134-137; \\n                 225-284; \\n               \\n               \\n                   \\n                   \\n                   \\n                 41-56, \\n                 CK2_PHOSPHO_SITE 448-451; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 ASN_GLYCOSYLATION 105-108; \\n                 46-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 271-296, \\n                 PKC_PHOSPHO_SITE 134-136; \\n                 SH3DOMAIN 270-282; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 PKC_PHOSPHO_SITE 52-54; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 447-453, \\n                 CK2_PHOSPHO_SITE 439-442; \\n                 242-257; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 337-340; \\n                 228-282; \\n               \\n               \\n                   \\n                   \\n                   \\n                 419-425, \\n                 PKC_PHOSPHO_SITE 115-117; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 MYRISTYL 398-403; \\n                 161-178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-257, \\n                 PKC_PHOSPHO_SITE 337-339; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 MYRISTYL 404-409; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-132, \\n                 TYR_PHOSPHO_SITE 56-63; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 ASN_GLYCOSYLATION 240-243; \\n               \\n               \\n                   \\n                   \\n                   \\n                 228-235, \\n                 PKC_PHOSPHO_SITE 245-247; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 MYRISTYL 333-338; \\n               \\n               \\n                   \\n                   \\n                   \\n                 302-317, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-178, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-208, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; 89-104, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 63-79, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 397-414, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 364-370, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n               \\n               \\n                 DEX0455_051.orf.4 \\n                 N \\n                 0 - \\n                 378-384, \\n                 PKC_PHOSPHO_SITE 341-343; \\n                 PRO_RICH 285-327; \\n               \\n               \\n                   \\n                   \\n                 o1-474; \\n                 1.046; 45-60, \\n                 PKC_PHOSPHO_SITE 262-264; \\n                 SH3 229-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 CK2_PHOSPHO_SITE 173-176; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 306-321, \\n                 PKC_PHOSPHO_SITE 119-121; \\n                 165-182; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 CK2_PHOSPHO_SITE 63-66; \\n                 SH3DOMAIN 274-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-182, \\n                 CK2_PHOSPHO_SITE 138-141; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; 67-83, \\n                 CK2_PHOSPHO_SITE 160-163; \\n                 246-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 452-455; \\n                 SH3DOMAIN 232-242; \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-239, \\n                 MYRISTYL 384-389; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 PKC_PHOSPHO_SITE 138-140; \\n                 308-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-261, \\n                 ASN_GLYCOSYLATION 109-112; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 93-108, \\n                 CK2_PHOSPHO_SITE 427-430; \\n                 50-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 TYR_PHOSPHO_SITE 60-67; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 432-446, \\n                 PKC_PHOSPHO_SITE 249-251; \\n                 238-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 341-344; \\n                 SH3 232-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-136, \\n                 MYRISTYL 402-407; \\n                 SH3 232-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 147-150; \\n               \\n               \\n                   \\n                   \\n                   \\n                 451-457, \\n                 MYRISTYL 408-413; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 443-446; \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-212, \\n                 PKC_PHOSPHO_SITE 56-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; 4-41, \\n                 MYRISTYL 337-342; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 CK2_PHOSPHO_SITE 92-95; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-355, \\n                 ASN_GLYCOSYLATION 244-247; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-300, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 387-397, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n               \\n               \\n                   \\n                   \\n                   \\n                 368-374, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n               \\n               \\n                   \\n                   \\n                   \\n                 423-429, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n               \\n               \\n                   \\n                   \\n                   \\n                 401-418, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n               \\n               \\n                 DEX0455_051.orf.5 \\n                 N \\n                 0 - \\n                 306-321, \\n                 ASN_GLYCOSYLATION 244-247; \\n                 PRO_RICH 285-327; \\n               \\n               \\n                   \\n                   \\n                 o1-474; \\n                 1.091; 93-108, \\n                 PKC_PHOSPHO_SITE 249-251; \\n                 SH3 232-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 CK2_PHOSPHO_SITE 341-344; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-239, \\n                 CK2_PHOSPHO_SITE 443-446; \\n                 308-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 MYRISTYL 402-407; \\n                 SH3DOMAIN 246-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 423-429, \\n                 PKC_PHOSPHO_SITE 119-121; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 PKC_PHOSPHO_SITE 56-58; \\n                 232-242; \\n               \\n               \\n                   \\n                   \\n                   \\n                 451-457, \\n                 CK2_PHOSPHO_SITE 160-163; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 PKC_PHOSPHO_SITE 138-140; \\n                 165-182; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-261, \\n                 CK2_PHOSPHO_SITE 63-66; \\n                 232-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n                 MYRISTYL 337-342; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 432-446, \\n                 TYR_PHOSPHO_SITE 60-67; \\n                 50-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 CK2_PHOSPHO_SITE 92-95; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-355, \\n                 CK2_PHOSPHO_SITE 173-176; \\n                 238-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 MYRISTYL 408-413; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 401-418, \\n                 ASN_GLYCOSYLATION 109-112; \\n                 274-286; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 MYRISTYL 384-389; \\n                 229-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 368-374, \\n                 CK2_PHOSPHO_SITE 427-430; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; 45-60, \\n                 CK2_PHOSPHO_SITE 452-455; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 CK2_PHOSPHO_SITE 138-141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-300, \\n                 PKC_PHOSPHO_SITE 341-343; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 CK2_PHOSPHO_SITE 147-150; \\n               \\n               \\n                   \\n                   \\n                   \\n                 387-397, \\n                 PKC_PHOSPHO_SITE 262-264; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-212, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-182, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 378-384, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-136, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; 67-83, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-41, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n               \\n               \\n                 DEX0455_051.orf.6 \\n                 N \\n                 0 - \\n                 368-374, \\n                 CK2_PHOSPHO_SITE 443-446; \\n                 SH3 229-288; \\n               \\n               \\n                   \\n                   \\n                 o1-474; \\n                 1.055; \\n                 CK2_PHOSPHO_SITE 452-455; \\n                 SH3DOMAIN 232-242; \\n               \\n               \\n                   \\n                   \\n                   \\n                 205-212, \\n                 PKC_PHOSPHO_SITE 56-58; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n                 MYRISTYL 408-413; \\n                 246-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-355, \\n                 CK2_PHOSPHO_SITE 63-66; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; 93-108, \\n                 MYRISTYL 402-407; \\n                 308-320; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n                 CK2_PHOSPHO_SITE 147-150; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-300, \\n                 MYRISTYL 384-389; \\n                 238-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; 45-60, \\n                 TYR_PHOSPHO_SITE 60-67; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 ASN_GLYCOSYLATION 109-112; \\n                 50-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-182, \\n                 MYRISTYL 337-342; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 CK2_PHOSPHO_SITE 138-141; \\n                 165-182; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 401-418, \\n                 CK2_PHOSPHO_SITE 92-95; \\n                 232-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 PKC_PHOSPHO_SITE 119-121; \\n                 SH3DOMAIN 274-286; \\n               \\n               \\n                   \\n                   \\n                   \\n                 378-384, \\n                 CK2_PHOSPHO_SITE 173-176; \\n                 PRO_RICH \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; 67-83, \\n                 ASN_GLYCOSYLATION 244-247; \\n                 285-327; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 249-251; \\n                 232-287; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-136, \\n                 CK2_PHOSPHO_SITE 160-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 138-140; \\n               \\n               \\n                   \\n                   \\n                   \\n                 387-397, \\n                 PKC_PHOSPHO_SITE 341-343; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; 4-41, \\n                 PKC_PHOSPHO_SITE 262-264; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 CK2_PHOSPHO_SITE 341-344; \\n               \\n               \\n                   \\n                   \\n                   \\n                 232-239, \\n                 CK2_PHOSPHO_SITE 427-430; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 306-321, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n               \\n               \\n                   \\n                   \\n                   \\n                 423-429, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n               \\n               \\n                   \\n                   \\n                   \\n                 432-446, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n               \\n               \\n                   \\n                   \\n                   \\n                 451-457, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n               \\n               \\n                   \\n                   \\n                   \\n                 247-261, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                 DEX0455_052.aa.1 \\n                 N \\n                 0 - \\n                 63-79, \\n                 PKC_PHOSPHO_SITE 258-260; \\n                 PRO_RICH 281-323; \\n               \\n               \\n                   \\n                   \\n                 o1-470; \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 423-426; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 271-296, \\n                 CK2_PHOSPHO_SITE 59-62; \\n                 161-178; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 PKC_PHOSPHO_SITE 245-247; \\n                 228-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 383-393, \\n                 PKC_PHOSPHO_SITE 52-54; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 CK2_PHOSPHO_SITE 143-146; \\n                 304-316; \\n               \\n               \\n                   \\n                   \\n                   \\n                 243-257, \\n                 CK2_PHOSPHO_SITE 134-137; \\n                 SH3DOMAIN 270-282; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 4-37, \\n                 CK2_PHOSPHO_SITE 88-91; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 PKC_PHOSPHO_SITE 134-136; \\n                 242-257; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 89-104, \\n                 ASN_GLYCOSYLATION 105-108; \\n                 228-282; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 374-380, \\n                 PKC_PHOSPHO_SITE 337-339; \\n                 SH3DOMAIN 228-238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n                 MYRISTYL 398-403; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 41-56, \\n                 ASN_GLYCOSYLATION 240-243; \\n                 46-58; SH3 225-284; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n                 TYR_PHOSPHO_SITE 56-63; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 397-414, \\n                 MYRISTYL 333-338; \\n                 234-277; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 CK2_PHOSPHO_SITE 439-442; \\n               \\n               \\n                   \\n                   \\n                   \\n                 364-370, \\n                 MYRISTYL 380-385; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n                 PKC_PHOSPHO_SITE 115-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 419-425, \\n                 CK2_PHOSPHO_SITE 337-340; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 CK2_PHOSPHO_SITE 448-451; \\n               \\n               \\n                   \\n                   \\n                   \\n                 302-317, \\n                 MYRISTYL 404-409; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 CK2_PHOSPHO_SITE 169-172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-178, \\n                 CK2_PHOSPHO_SITE 156-159; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 447-453, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n               \\n               \\n                   \\n                   \\n                   \\n                 228-235, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 344-351, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104, \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-208, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-132, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 428-442, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n               \\n               \\n                 DEX0455_052.aa.2 \\n                 N \\n                 0 - \\n                 479-485, \\n                 CK2_PHOSPHO_SITE 156-159; \\n                 SH3DOMAIN 302-314; \\n               \\n               \\n                   \\n                   \\n                 o1-502; \\n                 1.113; \\n                 CK2_PHOSPHO_SITE 455-458; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 260-267, \\n                 CK2_PHOSPHO_SITE 59-62; \\n                 266-309; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 CK2_PHOSPHO_SITE 134-137; \\n                 SH3DOMAIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 223-240, \\n                 PKC_PHOSPHO_SITE 290-292; \\n                 274-289; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n                 CK2_PHOSPHO_SITE 369-372; \\n                 257-316; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 406-412, \\n                 MYRISTYL 365-370; \\n                 260-314; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n                 ASN_GLYCOSYLATION 105-108; \\n                 260-315; \\n               \\n               \\n                   \\n                   \\n                   \\n                 303-328, \\n                 ASN_GLYCOSYLATION 272-275; \\n                 SH3DOMAIN 260-270; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.135; \\n                 TYR_PHOSPHO_SITE 56-63; \\n                 PRO_RICH \\n               \\n               \\n                   \\n                   \\n                   \\n                 376-383, \\n                 PKC_PHOSPHO_SITE 277-279; \\n                 313-355; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 CK2_PHOSPHO_SITE 88-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 451-457, \\n                 PKC_PHOSPHO_SITE 52-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; \\n                 CK2_PHOSPHO_SITE 143-146; \\n               \\n               \\n                   \\n                   \\n                   \\n                 415-425, \\n                 PKC_PHOSPHO_SITE 115-117; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 MYRISTYL 412-417; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-289, \\n                 430-435; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 4-37, \\n                 134-136; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.232; \\n                 169-172; MYRISTYL 436-441; \\n               \\n               \\n                   \\n                   \\n                   \\n                 334-349, \\n                 CK2_PHOSPHO_SITE 471-474; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; 63-79, \\n                 CK2_PHOSPHO_SITE 480-483; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 PKC_PHOSPHO_SITE 369-371; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-132, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 396-402, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-178, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 429-446, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; 89-104, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 460-474, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; 41-56, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.079; \\n               \\n               \\n                 DEX0455_052.aa.3 \\n                 Y \\n                 0 - \\n                 4-29, \\n                 PKC_PHOSPHO_SITE 512-514; \\n               \\n               \\n                   \\n                   \\n                 o1-548; \\n                 1.211; 79-95, \\n                 MYRISTYL 135-140; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 ASN_GLYCOSYLATION 340-343; \\n               \\n               \\n                   \\n                   \\n                   \\n                 56-73, \\n                 MYRISTYL 188-193; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 PKC_PHOSPHO_SITE 41-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                 495-502, \\n                 CK2_PHOSPHO_SITE 36-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 CK2_PHOSPHO_SITE 391-394; \\n               \\n               \\n                   \\n                   \\n                   \\n                 510-519, \\n                 PKC_PHOSPHO_SITE 350-352; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 167-185, \\n                 AMIDATION 1-4; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.133; \\n                 CK2_PHOSPHO_SITE 294-297; \\n               \\n               \\n                   \\n                   \\n                   \\n                 239-258, \\n                 PKC_PHOSPHO_SITE 276-278; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n                 MYRISTYL 212-217; \\n               \\n               \\n                   \\n                   \\n                   \\n                 263-269, \\n                 CK2_PHOSPHO_SITE 183-186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; 99-109, \\n                 PKC_PHOSPHO_SITE 287-289; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.259; \\n                 ASN_GLYCOSYLATION 12-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-214, \\n                 PKC_PHOSPHO_SITE 369-371; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.045; \\n                 CK2_PHOSPHO_SITE 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 407-413, \\n                 CK2_PHOSPHO_SITE 323-326; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 CK2_PHOSPHO_SITE 369-372; \\n               \\n               \\n                   \\n                   \\n                   \\n                 298-314, \\n                 CAMP_PHOSPHO_SITE 174-177; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 68-71; \\n               \\n               \\n                   \\n                   \\n                   \\n                 134-140, \\n                 TYR_PHOSPHO_SITE 291-298; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.047; \\n                 AMIDATION 272-275; \\n               \\n               \\n                   \\n                   \\n                   \\n                 324-339, \\n                 CK2_PHOSPHO_SITE 404-407; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.13; 42-48, \\n                 PKC_PHOSPHO_SITE 177-179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 CK2_PHOSPHO_SITE 155-158; \\n               \\n               \\n                   \\n                   \\n                   \\n                 149-154, \\n                 CK2_PHOSPHO_SITE 215-218; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.041; \\n                 ASN_GLYCOSYLATION 507-510; \\n               \\n               \\n                   \\n                   \\n                   \\n                 348-367, \\n                 CK2_PHOSPHO_SITE 526-529; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 CK2_PHOSPHO_SITE 378-381; \\n               \\n               \\n                   \\n                   \\n                   \\n                 525-531, \\n                 PKC_PHOSPHO_SITE 142-144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 MYRISTYL 121-126; \\n               \\n               \\n                   \\n                   \\n                   \\n                 283-291, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n               \\n               \\n                   \\n                   \\n                   \\n                 113-127, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 458-475, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 229-237, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                 DEX0455_052.aa.4 \\n                 Y \\n                 0 - \\n                 78-103, \\n                 MYRISTYL 2-7; \\n                 SH3DOMAIN 77-89; \\n               \\n               \\n                   \\n                   \\n                 o1-277; \\n                 1.135; \\n                 PKC_PHOSPHO_SITE 16-18; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 109-124, \\n                 CK2_PHOSPHO_SITE 230-233; \\n                 170-195; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 MYRISTYL 211-216; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 235-249, \\n                 CK2_PHOSPHO_SITE 246-249; \\n                 112-128; SH3 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.178; \\n                 MYRISTYL 205-210; \\n                 35-89; \\n               \\n               \\n                   \\n                   \\n                   \\n                 254-260, \\n                 CK2_PHOSPHO_SITE 255-258; \\n                 PRICHEXTENSN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; 4-19, \\n                 CK2_PHOSPHO_SITE 144-147; \\n                 88-109; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.169; \\n                 PKC_PHOSPHO_SITE 65-67; \\n                 PRO_RICH 88-130; \\n               \\n               \\n                   \\n                   \\n                   \\n                 151-158, \\n                 MYRISTYL 140-145; \\n                 SH3 32-91; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.104; \\n                 PKC_PHOSPHO_SITE 144-146; \\n                 SH3DOMAIN 35-45; \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-187, \\n                 ASN_GLYCOSYLATION 47-50; \\n                 SH3 35-90; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.046; \\n                 CK2_PHOSPHO_SITE 16-19; \\n                 sp_Q9GZQ2_Q9GZQ2_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 204-221, \\n                 PKC_PHOSPHO_SITE 52-54; \\n                 41-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.153; \\n                 MYRISTYL 187-192; \\n                 SH3DOMAIN 49-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                 190-200, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; 50-64, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 226-232, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.071; 35-42, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n               \\n               \\n                   \\n                   \\n                   \\n                 171-177, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n               \\n               \\n                 DEX0455_053.aa.1 \\n                 Y \\n                 1 - \\n                 73-83, \\n                 ASN_GLYCOSYLATION 112-115; \\n                 IG_LIKE_2 153-241; \\n               \\n               \\n                   \\n                   \\n                 i1-6; \\n                 1.16; 36-45, \\n                 CK2_PHOSPHO_SITE 91-94; \\n                 IG_LIKE_1 \\n               \\n               \\n                   \\n                   \\n                 tm7-29; \\n                 1.103; \\n                 ASN_GLYCOSYLATION 216-219; \\n                 49-151; \\n               \\n               \\n                   \\n                   \\n                 o30-282; \\n                 152-160, \\n                 CK2_PHOSPHO_SITE 151-154; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 CAMP_PHOSPHO_SITE 246-249; \\n               \\n               \\n                   \\n                   \\n                   \\n                 176-186, \\n                 ASN_GLYCOSYLATION 196-199; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 PKC_PHOSPHO_SITE 32-34; \\n               \\n               \\n                   \\n                   \\n                   \\n                 127-134, \\n                 MYRISTYL 188-193; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n                 CK2_PHOSPHO_SITE 183-186; \\n               \\n               \\n                   \\n                   \\n                   \\n                 207-217, \\n                 PKC_PHOSPHO_SITE 207-209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.215; 53-59, \\n                 ASN_GLYCOSYLATION 190-193; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.066; \\n                 ASN_GLYCOSYLATION 160-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 61-71, \\n                 MYRISTYL 52-57; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n                 PKC_PHOSPHO_SITE 127-129; \\n               \\n               \\n                   \\n                   \\n                   \\n                 248-279, \\n                 ASN_GLYCOSYLATION 205-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n                 CK2_PHOSPHO_SITE 241-244; \\n               \\n               \\n                   \\n                   \\n                   \\n                 100-123, \\n                 PKC_PHOSPHO_SITE 134-136; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; \\n                 CK2_PHOSPHO_SITE 197-200; \\n               \\n               \\n                   \\n                   \\n                   \\n                 165-171, \\n                 PKC_PHOSPHO_SITE 165-167; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 4-30, \\n                 PKC_PHOSPHO_SITE 114-116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.146; \\n                 ASN_GLYCOSYLATION 220-223; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 126-131; \\n               \\n               \\n                 DEX0455_053.aa.2 \\n                 Y \\n                 1 - \\n                 4-39, \\n               \\n               \\n                   \\n                   \\n                 i1-6; \\n                 1.146; 41-56, \\n               \\n               \\n                   \\n                   \\n                 tm7-29; \\n                 1.17; \\n               \\n               \\n                   \\n                   \\n                 o30-59; \\n               \\n               \\n                 DEX0455_053.aa.3 \\n                 N \\n                 0 - \\n                 177-187, \\n                 ASN_GLYCOSYLATION 166-169; \\n                 IG_LIKE_1 19-121; \\n               \\n               \\n                   \\n                   \\n                 o1-252; \\n                 1.215; 43-53, \\n                 PKC_PHOSPHO_SITE 177-179; \\n                 ig 19-102; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; \\n                 PKC_PHOSPHO_SITE 97-99; \\n                 IG 11-116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 23-29, 1.1; \\n                 CK2_PHOSPHO_SITE 153-156; \\n                 IG_LIKE_2 123-211; \\n               \\n               \\n                   \\n                   \\n                   \\n                 122-130, \\n                 PKC_PHOSPHO_SITE 104-106; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.069; \\n                 ASN_GLYCOSYLATION 160-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 146-156, \\n                 CK2_PHOSPHO_SITE 167-170; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 ASN_GLYCOSYLATION 186-189; \\n               \\n               \\n                   \\n                   \\n                   \\n                 218-249, \\n                 CAMP_PHOSPHO_SITE 216-219; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; 6-15, \\n                 CK2_PHOSPHO_SITE 61-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n                 ASN_GLYCOSYLATION 82-85; \\n               \\n               \\n                   \\n                   \\n                   \\n                 70-93, \\n                 CK2_PHOSPHO_SITE 121-124; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.136; 97-104, \\n                 PKC_PHOSPHO_SITE 84-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n                 CK2_PHOSPHO_SITE 211-214; \\n               \\n               \\n                   \\n                   \\n                   \\n                 135-141, \\n                 ASN_GLYCOSYLATION 190-193; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.06; 31-41, \\n                 MYRISTYL 158-163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.119; \\n                 ASN_GLYCOSYLATION 130-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 96-101; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 22-27; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 135-137; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 175-178; \\n               \\n               \\n                 DEX0455_054.orf.1 \\n                 N \\n                 0 - \\n                 6-13, \\n                 CK2_PHOSPHO_SITE 72-75; \\n                 PRO_RICH 51-70; \\n               \\n               \\n                   \\n                   \\n                 o1-155; \\n                 1.134; \\n                 PKC_PHOSPHO_SITE 67-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 145-152, \\n                 AMIDATION 124-127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.23; 34-51, \\n                 MYRISTYL 109-114; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.251; \\n                 79-84; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 53-87, \\n                 89-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; 21-30, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n               \\n               \\n                   \\n                   \\n                   \\n                 99-123, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.205; \\n               \\n               \\n                 DEX0455_054.aa.1 \\n                 N \\n                 0 - \\n                 23-31, \\n                 PKC_PHOSPHO_SITE 100-102; \\n                 sp_P14786_KPY2_HUMAN \\n               \\n               \\n                   \\n                   \\n                 o1-107; \\n                 1.113; 4-20, \\n                 MYRISTYL 94-99; \\n                 12-105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.19; \\n                 PKC_PHOSPHO_SITE 35-37; \\n                 PK_C 3-105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 61-67, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; 36-57, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.184; \\n               \\n               \\n                   \\n                   \\n                   \\n                 81-90, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n               \\n               \\n                 DEX0455_054.orf.2 \\n                 N \\n                 0 - \\n                 99-123, \\n                 AMIDATION 124-127; \\n                 PRO_RICH 51-70; \\n               \\n               \\n                   \\n                   \\n                 o1-155; \\n                 1.205; 6-13, \\n                 CK2_PHOSPHO_SITE 89-92; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 MYRISTYL 79-84; \\n               \\n               \\n                   \\n                   \\n                   \\n                 53-87, \\n                 PKC_PHOSPHO_SITE 67-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 MYRISTYL 109-114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 145-152, \\n                 CK2_PHOSPHO_SITE 72-75; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.23; 34-51, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.251; \\n               \\n               \\n                   \\n                   \\n                   \\n                 21-30, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.088; \\n               \\n               \\n                 DEX0455_054.aa.2 \\n                 N \\n                 0 - \\n                 28-45, \\n                 MYRISTYL 27-32; \\n                 PK_C 21-141; \\n               \\n               \\n                   \\n                   \\n                 o1-143; \\n                 1.179; 72-93, \\n                 PKC_PHOSPHO_SITE 32-34; \\n                 sp_P11974_KPY1_RABIT \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.184; \\n                 PKC_PHOSPHO_SITE 136-138; \\n                 1-141; \\n               \\n               \\n                   \\n                   \\n                   \\n                 50-56, \\n                 PKC_PHOSPHO_SITE 71-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; \\n                 MYRISTYL 130-135; \\n               \\n               \\n                   \\n                   \\n                   \\n                 117-126, \\n                 PKC_PHOSPHO_SITE 46-48; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; 7-14, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; \\n               \\n               \\n                   \\n                   \\n                   \\n                 97-103, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.089; 59-67, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n               \\n               \\n                 DEX0455_055.aa.1 \\n                 N \\n                 0 - \\n                   \\n                 MYRISTYL 52-57; \\n               \\n               \\n                   \\n                   \\n                 o1-253; \\n                   \\n                 CK2_PHOSPHO_SITE 11-14; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 19-21; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 207-209; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 9-12; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 164-166; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 7-10; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 131-134; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 130-132; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 245-247; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 243-246; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 176-179; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 19-22; \\n               \\n               \\n                 DEX0455_055.aa.2 \\n                 N \\n                 0 - \\n                 107-114, \\n                 CAMP_PHOSPHO_SITE 18-21; \\n               \\n               \\n                   \\n                   \\n                 o1-361; \\n                 1.114; 42-52, \\n                 CK2_PHOSPHO_SITE 12-15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 AMIDATION 348-351; \\n               \\n               \\n                   \\n                   \\n                   \\n                 83-97, \\n                 MYRISTYL 107-112; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.14; 70-76, \\n                 PKC_PHOSPHO_SITE 344-346; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.055; \\n                 CAMP_PHOSPHO_SITE 186-189; \\n               \\n               \\n                   \\n                   \\n                   \\n                 236-242, \\n                 MYRISTYL 311-316; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.115; 4-11, \\n                 CAMP_PHOSPHO_SITE 309-312; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 325-327; \\n               \\n               \\n                   \\n                   \\n                   \\n                 320-326, \\n                 AMIDATION 231-234; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; \\n                 CK2_PHOSPHO_SITE 62-65; \\n               \\n               \\n                   \\n                   \\n                   \\n                 164-186, \\n                 MYRISTYL 22-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.208; \\n                 CK2_PHOSPHO_SITE 66-69; \\n               \\n               \\n                   \\n                   \\n                   \\n                 124-135, \\n                 ASN_GLYCOSYLATION 358-361; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n                 PKC_PHOSPHO_SITE 262-264; \\n               \\n               \\n                   \\n                   \\n                   \\n                 273-278, \\n                 AMIDATION 306-309; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.03; 99-105, \\n                 PKC_PHOSPHO_SITE 16-18; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.047; \\n                 CK2_PHOSPHO_SITE 74-77; \\n               \\n               \\n                   \\n                   \\n                   \\n                 224-230, \\n                 PKC_PHOSPHO_SITE 185-187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.064; \\n                 PKC_PHOSPHO_SITE 219-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 74-76; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 319-322; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 339-342; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 64-67; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 341-344; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 293-295; \\n               \\n               \\n                 DEX0455_055.aa.3 \\n                 N \\n                 0 - \\n                 28-49, \\n                 AMIDATION 112-115; \\n               \\n               \\n                   \\n                   \\n                 i1-167; \\n                 1.115; \\n                 MYRISTYL 117-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 126-137, \\n                 PKC_PHOSPHO_SITE 25-27; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.056; 68-84, \\n                 PKC_PHOSPHO_SITE 131-133; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.038; \\n                 PKC_PHOSPHO_SITE 150-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-10, \\n                 AMIDATION 145-148; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.009; 12-19, \\n                 CAMP_PHOSPHO_SITE 147-150; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.035; \\n                 CK2_PHOSPHO_SITE 125-128; \\n               \\n               \\n                   \\n                   \\n                   \\n                 60-66, \\n                 PKC_PHOSPHO_SITE 99-101; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.026; \\n                 AMIDATION 37-40; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CAMP_PHOSPHO_SITE 115-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 AMIDATION 154-157; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 68-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 ASN_GLYCOSYLATION 164-167; \\n               \\n               \\n                 DEX0455_056.orf.1 \\n                 N \\n                 0 - \\n                 53-59, \\n                 ASN_GLYCOSYLATION 199-202; \\n               \\n               \\n                   \\n                   \\n                 o1-636; \\n                 1.147; \\n                 MYRISTYL 65-70; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 401-411, \\n                 451-456; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; 4-10, \\n                 221-224; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; \\n                 454-457; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 592-633, \\n                 574-577; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n                 115-118; MYRISTYL 531-536; \\n               \\n               \\n                   \\n                   \\n                   \\n                 218-253, \\n                 CK2_PHOSPHO_SITE 347-350; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n                 CK2_PHOSPHO_SITE 137-140; \\n               \\n               \\n                   \\n                   \\n                   \\n                 474-480, \\n                 CK2_PHOSPHO_SITE 308-311; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.058; 95-136, \\n                 CK2_PHOSPHO_SITE 301-304; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.207; \\n                 MYRISTYL 336-341; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 440-448, \\n                 154-159; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 201-203; MYRISTYL 231-236; \\n               \\n               \\n                   \\n                   \\n                   \\n                 354-372, \\n                 CK2_PHOSPHO_SITE 525-528; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 ASN_GLYCOSYLATION 415-418; \\n               \\n               \\n                   \\n                   \\n                   \\n                 165-203, \\n                 MYRISTYL 206-211; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.177; 66-74, \\n                 PKC_PHOSPHO_SITE 565-567; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 PKC_PHOSPHO_SITE 286-288; \\n               \\n               \\n                   \\n                   \\n                   \\n                 429-437, \\n                 TYR_PHOSPHO_SITE 351-357; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 MYRISTYL 60-65; \\n               \\n               \\n                   \\n                   \\n                   \\n                 142-148, \\n                 TYR_PHOSPHO_SITE 421-428; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 AMIDATION 20-23; \\n               \\n               \\n                   \\n                   \\n                   \\n                 153-163, \\n                 CK2_PHOSPHO_SITE 70-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 MYRISTYL 63-68; \\n               \\n               \\n                   \\n                   \\n                   \\n                 274-285, \\n                 PKC_PHOSPHO_SITE 70-72; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n                 CK2_PHOSPHO_SITE 293-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 374-386, \\n                 MYRISTYL 588-593; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 51-56; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 578-584, \\n                 23-26; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n                 548-551; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 530-574, \\n                 35-38; MYRISTYL 4-9; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 MYRISTYL 11-16; \\n               \\n               \\n                   \\n                   \\n                   \\n                 460-466, \\n                 CK2_PHOSPHO_SITE 235-238; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; 76-83, \\n                 CK2_PHOSPHO_SITE 286-289; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 MYRISTYL 49-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 490-518, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 259-265, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n               \\n               \\n                   \\n                   \\n                   \\n                 290-302, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 309-325, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n               \\n               \\n                 DEX0455_056.aa.1 \\n                 N \\n                 0 - \\n                 38-44, \\n                 CK2_PHOSPHO_SITE 559-562; \\n               \\n               \\n                   \\n                   \\n                 o1-870; \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 122-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 386-396, \\n                 MYRISTYL 659-664; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 36-41; MYRISTYL 34-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                 690-699, \\n                 PKC_PHOSPHO_SITE 271-273; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; \\n                 CK2_PHOSPHO_SITE 510-513; \\n               \\n               \\n                   \\n                   \\n                   \\n                 679-688, \\n                 MYRISTYL 45-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.065; \\n                 TYR_PHOSPHO_SITE 406-413; \\n               \\n               \\n                   \\n                   \\n                   \\n                 339-357, \\n                 CK2_PHOSPHO_SITE 206-209; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 MYRISTYL 834-839; \\n               \\n               \\n                   \\n                   \\n                   \\n                 259-270, \\n                 PKC_PHOSPHO_SITE 664-666; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n                 CK2_PHOSPHO_SITE 332-335; \\n               \\n               \\n                   \\n                   \\n                   \\n                 634-652, \\n                 ASN_GLYCOSYLATION 533-536; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 MYRISTYL 436-441; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 515-559, \\n                 139-144; MYRISTYL 191-196; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; 51-59, \\n                 MYRISTYL 573-578; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 PKC_PHOSPHO_SITE 550-552; \\n               \\n               \\n                   \\n                   \\n                   \\n                 425-433, \\n                 CK2_PHOSPHO_SITE 220-223; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n                 PKC_PHOSPHO_SITE 762-764; \\n               \\n               \\n                   \\n                   \\n                   \\n                 294-310, \\n                 CK2_PHOSPHO_SITE 100-103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n                 CK2_PHOSPHO_SITE 824-827; \\n               \\n               \\n                   \\n                   \\n                   \\n                 445-451, \\n                 CK2_PHOSPHO_SITE 640-643; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 MYRISTYL 321-326; \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-287, \\n                 PKC_PHOSPHO_SITE 55-57; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 PKC_PHOSPHO_SITE 186-188; \\n               \\n               \\n                   \\n                   \\n                   \\n                 244-250, \\n                 CK2_PHOSPHO_SITE 20-23; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; \\n                 CK2_PHOSPHO_SITE 286-289; \\n               \\n               \\n                   \\n                   \\n                   \\n                 763-796, \\n                 MYRISTYL 216-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.144; \\n                 CK2_PHOSPHO_SITE 278-281; \\n               \\n               \\n                   \\n                   \\n                   \\n                 459-465, \\n                 PKC_PHOSPHO_SITE 842-844; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.058; \\n                 ASN_GLYCOSYLATION 184-187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 827-844, \\n                 CAMP_PHOSPHO_SITE 748-751; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n                 MYRISTYL 516-521; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 150-188, \\n                 50-55; TYR_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.177; \\n                 336-342; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 414-422, \\n                 756-758; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 439-442; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 739-754, \\n                 400-403; AMIDATION 5-8; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n                 PKC_PHOSPHO_SITE 824-826; \\n               \\n               \\n                   \\n                   \\n                   \\n                 659-664, \\n                 CAMP_PHOSPHO_SITE 8-11; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.038; \\n                 CK2_PHOSPHO_SITE 293-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 707-723, \\n                 CK2_PHOSPHO_SITE 55-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.101; \\n                 CK2_PHOSPHO_SITE 271-274; \\n               \\n               \\n                   \\n                   \\n                   \\n                 359-371, \\n                 MYRISTYL 48-53; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 798-805, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.037; \\n               \\n               \\n                   \\n                   \\n                   \\n                 203-238, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; 80-121, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.207; \\n               \\n               \\n                   \\n                   \\n                   \\n                 563-569, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 851-867, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n               \\n               \\n                   \\n                   \\n                   \\n                 577-629, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n               \\n               \\n                   \\n                   \\n                   \\n                 475-503, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 127-133, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-148, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; 61-68, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 666-676, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.164; \\n               \\n               \\n                 DEX0455_056.aa.2 \\n                 N \\n                 0 - \\n                 244-250, \\n                 CAMP_PHOSPHO_SITE 738-741; \\n               \\n               \\n                   \\n                   \\n                 o1-791; \\n                 1.052; 51-59, \\n                 PKC_PHOSPHO_SITE 55-57; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.083; \\n                 CK2_PHOSPHO_SITE 220-223; \\n               \\n               \\n                   \\n                   \\n                   \\n                 339-357, \\n                 CK2_PHOSPHO_SITE 100-103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.185; \\n                 CK2_PHOSPHO_SITE 122-125; \\n               \\n               \\n                   \\n                   \\n                   \\n                 577-619, \\n                 CK2_PHOSPHO_SITE 271-274; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.179; \\n                 CK2_PHOSPHO_SITE 286-289; \\n               \\n               \\n                   \\n                   \\n                   \\n                 772-788, \\n                 CAMP_PHOSPHO_SITE 8-11; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.148; \\n                 PKC_PHOSPHO_SITE 654-656; \\n               \\n               \\n                   \\n                   \\n                   \\n                 669-678, \\n                 MYRISTYL 191-196; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.065; \\n                 516-521; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-148, \\n                 510-513; MYRISTYL 36-41; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.075; \\n                 PKC_PHOSPHO_SITE 763-765; \\n               \\n               \\n                   \\n                   \\n                   \\n                 656-666, \\n                 CK2_PHOSPHO_SITE 630-633; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.164; \\n                 MYRISTYL 321-326; \\n               \\n               \\n                   \\n                   \\n                   \\n                 294-310, \\n                 CK2_PHOSPHO_SITE 332-335; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n                 PKC_PHOSPHO_SITE 271-273; \\n               \\n               \\n                   \\n                   \\n                   \\n                 475-503, \\n                 MYRISTYL 216-221; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 PKC_PHOSPHO_SITE 186-188; \\n               \\n               \\n                   \\n                   \\n                   \\n                 515-559, \\n                 CK2_PHOSPHO_SITE 293-296; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 MYRISTYL 139-144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 459-465, \\n                 ASN_GLYCOSYLATION 439-442; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.058; \\n                 MYRISTYL 573-578; \\n               \\n               \\n                   \\n                   \\n                   \\n                 697-713, \\n                 AMIDATION 5-8; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.101; \\n                 50-55; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 386-396, \\n                 752-754; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.092; \\n                 550-552; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 624-642, \\n                 533-536; TYR_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 406-413; MYRISTYL 48-53; \\n               \\n               \\n                   \\n                   \\n                   \\n                 445-451, \\n                 PKC_PHOSPHO_SITE 746-748; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 CK2_PHOSPHO_SITE 559-562; \\n               \\n               \\n                   \\n                   \\n                   \\n                 127-133, \\n                 ASN_GLYCOSYLATION 400-403; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.091; \\n                 MYRISTYL 34-39; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 680-689, \\n                 436-441; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.137; 38-44, \\n                 278-281; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.147; \\n                 206-209; MYRISTYL 649-654; \\n               \\n               \\n                   \\n                   \\n                   \\n                 359-371, \\n                 CK2_PHOSPHO_SITE 20-23; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 ASN_GLYCOSYLATION 184-187; \\n               \\n               \\n                   \\n                   \\n                   \\n                 259-270, \\n                 TYR_PHOSPHO_SITE 336-342; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.132; \\n                 MYRISTYL 755-760; \\n               \\n               \\n                   \\n                   \\n                   \\n                 414-422, \\n                 CK2_PHOSPHO_SITE 55-58; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 MYRISTYL 45-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 563-569, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.107; \\n               \\n               \\n                   \\n                   \\n                   \\n                 753-765, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.127; \\n               \\n               \\n                   \\n                   \\n                   \\n                 203-238, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.106; \\n               \\n               \\n                   \\n                   \\n                   \\n                 425-433, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.082; \\n               \\n               \\n                   \\n                   \\n                   \\n                 729-744, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-287, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.163; \\n               \\n               \\n                   \\n                   \\n                   \\n                 150-188, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.177; 80-121, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.207; \\n               \\n               \\n                   \\n                   \\n                   \\n                 61-68, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n               \\n               \\n                   \\n                   \\n                   \\n                 649-654, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.038; \\n               \\n               \\n                 DEX0455_057.orf.1 \\n                 N \\n                 0 - \\n                 97-119, \\n                 PKC_PHOSPHO_SITE 3-5; \\n                 EF_HAND_2 25-101; \\n               \\n               \\n                   \\n                   \\n                 o1-122; \\n                 1.114; 30-41, \\n                 ASN_GLYCOSYLATION 48-51; \\n                 S100_CABP \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.138; \\n                 CK2_PHOSPHO_SITE 25-28; \\n                 80-101; EFh 76-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-13, \\n                 PKC_PHOSPHO_SITE 27-29; \\n                 sp_P31949_S111_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.125; \\n                 CK2_PHOSPHO_SITE 23-26; \\n                 25-94; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 118-120; \\n                 S_100 27-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 103-108; \\n                 efhand 76-104; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 12-15; \\n                 sp_O93395_O93395_SALFO \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 86-89; \\n                 44-98; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 52-55; \\n                 EF_HAND 85-97; \\n               \\n               \\n                 DEX0455_057.aa.1 \\n                 N \\n                 0 - \\n                 78-89, \\n                 ASN_GLYCOSYLATION 96-99; \\n                 sp_P31949_S111_HUMAN \\n               \\n               \\n                   \\n                   \\n                 o1-170; \\n                 1.138; \\n                 CK2_PHOSPHO_SITE 71-74; \\n                 73-142; \\n               \\n               \\n                   \\n                   \\n                   \\n                 145-167, \\n                 PKC_PHOSPHO_SITE 55-57; \\n                 EF_HAND 133-145; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; 4-38, \\n                 CK2_PHOSPHO_SITE 43-46; \\n                 S100_CABP \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.16; \\n                 PKC_PHOSPHO_SITE 75-77; \\n                 128-149; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 134-137; \\n                 sp_O93395_O93395_SALFO \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 166-168; \\n                 92-146; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 100-103; \\n                 efhand 124-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 151-156; \\n                 EFh 124-152; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 73-76; \\n                 S_100 75-118; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 PKC_PHOSPHO_SITE 58-60; \\n                 EF_HAND_2 73-149; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 6-9; \\n               \\n               \\n                 DEX0455_057.aa.2 \\n                 N \\n                 0 - \\n                 66-88, \\n                 CK2_PHOSPHO_SITE 55-58; \\n                 EFh 45-73; \\n               \\n               \\n                   \\n                   \\n                 o1-91; \\n                 1.114; 4-11, \\n                 ASN_GLYCOSYLATION 17-20; \\n                 efhand 45-73; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 PKC_PHOSPHO_SITE 87-89; \\n                 S100_CABP 49-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 CK2_PHOSPHO_SITE 21-24; \\n                 S_100 3-39; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                 MYRISTYL 72-77; \\n                 EF_HAND 54-66; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 sp_P31949_S111_HUMAN \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 9-63; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 EF_HAND_2 19-70; \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 sp_O93395_O93395_SALFO \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 13-67; \\n               \\n               \\n                 DEX0455_058.orf.1 \\n                 N \\n                 1 —o1-14; \\n                 4-25, \\n                 CK2_PHOSPHO_SITE 23-26; \\n               \\n               \\n                   \\n                   \\n                 tm15-37; \\n                 1.178; 27-63, \\n                 MYRISTYL 27-32; \\n               \\n               \\n                   \\n                   \\n                 i38-66; \\n                 1.191; \\n               \\n               \\n                 DEX0455_058.aa.1 \\n                 N \\n                 0 - \\n                 24-32, \\n                 TYR_PHOSPHO_SITE 12-18; \\n               \\n               \\n                   \\n                   \\n                 o1-65; \\n                 1.155; 36-54, \\n                 ASN_GLYCOSYLATION 59-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.162; \\n                 CK2_PHOSPHO_SITE 6-9; \\n               \\n               \\n                   \\n                   \\n                   \\n                 14-22, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; \\n               \\n               \\n                 DEX0455_059.orf.1 \\n                 N \\n                 0 - \\n                 104-122, \\n                 PKC_PHOSPHO_SITE 80-82; \\n               \\n               \\n                   \\n                   \\n                 o1-363; \\n                 1.145; \\n                 CK2_PHOSPHO_SITE 28-31; \\n               \\n               \\n                   \\n                   \\n                   \\n                 261-268, \\n                 PKC_PHOSPHO_SITE 281-283; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 11-21, \\n                 CAMP_PHOSPHO_SITE 105-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.113; \\n                 MYRISTYL 47-52; \\n               \\n               \\n                   \\n                   \\n                   \\n                 341-351, \\n                 CAMP_PHOSPHO_SITE 77-80; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n                 PKC_PHOSPHO_SITE 103-105; \\n               \\n               \\n                   \\n                   \\n                   \\n                 125-159, \\n                 MYRISTYL 335-340; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 PKC_PHOSPHO_SITE 206-208; \\n               \\n               \\n                   \\n                   \\n                   \\n                 178-212, \\n                 PKC_PHOSPHO_SITE 76-78; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; 69-77, \\n                 MYRISTYL 304-309; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.128; \\n                 71-76; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 315-330, \\n                 28-30; PKC_PHOSPHO_SITE 8-10; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.192; \\n                 CK2_PHOSPHO_SITE 17-20; \\n               \\n               \\n                   \\n                   \\n                   \\n                 270-275, \\n                 PKC_PHOSPHO_SITE 62-64; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.039; \\n                 CK2_PHOSPHO_SITE 237-240; \\n               \\n               \\n                   \\n                   \\n                   \\n                 235-248, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 281-289, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 353-360, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; 85-102, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.157; \\n               \\n               \\n                   \\n                   \\n                   \\n                 295-305, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; \\n               \\n               \\n                   \\n                   \\n                   \\n                 225-232, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; \\n               \\n               \\n                 DEX0455_059.aa.1 \\n                 N \\n                 0 - \\n                 38-67, \\n                 PKC_PHOSPHO_SITE 25-27; \\n               \\n               \\n                   \\n                   \\n                 o1-116; \\n                 1.204; 28-34, \\n                 PKC_PHOSPHO_SITE 58-60; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 AMIDATION 16-19; \\n               \\n               \\n                   \\n                   \\n                   \\n                 12-23, \\n                 CK2_PHOSPHO_SITE 5-8; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.084; 71-113, \\n                 PKC_PHOSPHO_SITE 45-47; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.168; \\n               \\n               \\n                 DEX0455_059.orf.2 \\n                 N \\n                 0 - \\n                 118-133, \\n                 MYRISTYL 107-112; \\n               \\n               \\n                   \\n                   \\n                 o1-166; \\n                 1.192; 16-26, \\n                 CK2_PHOSPHO_SITE 40-43; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.124; \\n                 MYRISTYL 138-143; \\n               \\n               \\n                   \\n                   \\n                   \\n                 98-108, \\n                 PKC_PHOSPHO_SITE 84-86; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; 144-154, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n               \\n               \\n                   \\n                   \\n                   \\n                 64-71, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.085; 7-13, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.026; \\n               \\n               \\n                   \\n                   \\n                   \\n                 38-51, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; 84-92, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.116; \\n               \\n               \\n                   \\n                   \\n                   \\n                 28-35, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 156-163, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; 73-78, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.039; \\n               \\n               \\n                 DEX0455_060.aa.1 \\n                 Y \\n                 0 - \\n                 115-122, \\n                 MYRISTYL 76-81; AMIDATION \\n                 BTG_1 124-144; \\n               \\n               \\n                   \\n                   \\n                 o1-207; \\n                 1.108; \\n                 79-82; PKC_PHOSPHO_SITE \\n                 ANTIPRLFBTG1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 149-187, \\n                 42-44; MYRISTYL 34-39; \\n                 173-202; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.215; \\n                 MYRISTYL 41-46; \\n                 Anti_proliferat \\n               \\n               \\n                   \\n                   \\n                   \\n                 193-199, \\n                 PKC_PHOSPHO_SITE 24-26; \\n                 83-207; BTG_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.122; 4-26, \\n                 MYRISTYL 75-80; \\n                 170-189; btg1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.251; \\n                   \\n                 83-190; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-145, \\n                   \\n                 ANTIPRLFBTG1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.131; 43-56, \\n                   \\n                 88-112; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.102; \\n                   \\n                 ANTIPRLFBTG1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 87-101, \\n                   \\n                 113-142; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.175; \\n               \\n               \\n                 DEX0455_061.aa.1 \\n                 N \\n                 0 - \\n                 173-179, \\n                 MYRISTYL 57-62; \\n                 RA 203-293; \\n               \\n               \\n                   \\n                   \\n                 o1-352; \\n                 1.031; \\n                 CK2_PHOSPHO_SITE 22-25; \\n                 RA_DOMAIN 205-293; \\n               \\n               \\n                   \\n                   \\n                   \\n                 264-290, \\n                 MYRISTYL 133-138; \\n                 RA 203-293; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.129; \\n                 CK2_PHOSPHO_SITE 150-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-187, \\n                 PKC_PHOSPHO_SITE 178-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; 138-148, \\n                 ASN_GLYCOSYLATION 134-137; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; \\n                 MYRISTYL 43-48; AMIDATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 36-68, \\n                 10-13; CAMP_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.256; 4-10, \\n                 119-122; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n                 314-317; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 108-113, \\n                 225-228; TYR_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; \\n                 263-269; PKC_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 230-241, \\n                 331-333; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; \\n                 170-173; MYRISTYL 217-222; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-128, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; \\n               \\n               \\n                   \\n                   \\n                   \\n                 249-257, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 344-349, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.095; 14-20, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-225, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 333-340, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.068; 81-91, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-319, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n               \\n               \\n                 DEX0455_061.aa.2 \\n                 N \\n                 0 - \\n                 230-241, \\n                 ASN_GLYCOSYLATION 134-137; \\n                 RA 203-260; \\n               \\n               \\n                   \\n                   \\n                 o1-261; \\n                 1.187; \\n                 MYRISTYL 43-48; \\n                 RA_DOMAIN 205-261; \\n               \\n               \\n                   \\n                   \\n                   \\n                 108-113, \\n                 CAMP_PHOSPHO_SITE 119-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; \\n                 MYRISTYL 57-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 173-179, \\n                 CK2_PHOSPHO_SITE 150-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.031; \\n                 PKC_PHOSPHO_SITE 178-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-225, \\n                 AMIDATION 10-13; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; 36-68, \\n                 CK2_PHOSPHO_SITE 22-25; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.256; \\n                 MYRISTYL 217-222; \\n               \\n               \\n                   \\n                   \\n                   \\n                 81-91, \\n                 ASN_GLYCOSYLATION 225-228; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n                 MYRISTYL 133-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 249-258, \\n                 CK2_PHOSPHO_SITE 170-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.172; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-148, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-128, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; 4-10, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.15; \\n               \\n               \\n                   \\n                   \\n                   \\n                 14-20, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n               \\n               \\n                   \\n                   \\n                   \\n                 181-187, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.04; \\n               \\n               \\n                 DEX0455_061.aa.3 \\n                 N \\n                 0 - \\n                 181-187, \\n                 CK2_PHOSPHO_SITE 22-25; \\n                 RA 203-269; \\n               \\n               \\n                   \\n                   \\n                 o1-269; \\n                 1.04; 36-68, \\n                 CK2_PHOSPHO_SITE 258-261; \\n                 RA_DOMAIN 205-269; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.256; \\n                 MYRISTYL 133-138; \\n               \\n               \\n                   \\n                   \\n                   \\n                 120-128, \\n                 CK2_PHOSPHO_SITE 170-173; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.086; \\n                 CK2_PHOSPHO_SITE 150-153; \\n               \\n               \\n                   \\n                   \\n                   \\n                 230-241, \\n                 ASN_GLYCOSYLATION 225-228; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.187; 14-20, \\n                 AMIDATION 10-13; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.171; \\n                 217-222; ASN_GLYCOSYLATION \\n               \\n               \\n                   \\n                   \\n                   \\n                 4-10, 1.15; \\n                 134-137; MYRISTYL 57-62; \\n               \\n               \\n                   \\n                   \\n                   \\n                 173-179, \\n                 PKC_PHOSPHO_SITE 178-180; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.031; \\n                 MYRISTYL 43-48; \\n               \\n               \\n                   \\n                   \\n                   \\n                 108-113, \\n                 CAMP_PHOSPHO_SITE 119-122; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.105; 81-91, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.111; \\n               \\n               \\n                   \\n                   \\n                   \\n                 138-148, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.114; \\n               \\n               \\n                   \\n                   \\n                   \\n                 208-225, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.103; \\n               \\n               \\n                   \\n                   \\n                   \\n                 249-257, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.172; \\n               \\n               \\n                 DEX0455_061.aa.4 \\n                 Y \\n                 0 - \\n                 4-31, \\n                 ASN_GLYCOSYLATION 51-54; \\n               \\n               \\n                   \\n                   \\n                 o1-133; \\n                 1.159; 61-129, \\n                 PKC_PHOSPHO_SITE 77-79; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.207; \\n               \\n               \\n                   \\n                   \\n                   \\n                 41-58, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n               \\n               \\n                 DEX0455_061.orf.5 \\n                 N \\n                 0 - \\n                 92-130, \\n                 CK2_PHOSPHO_SITE 51-54; \\n               \\n               \\n                   \\n                   \\n                 o1-163; \\n                 1.205; 56-85, \\n                 MYRISTYL 17-22; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.181; \\n                 PKC_PHOSPHO_SITE 5-7; \\n               \\n               \\n                   \\n                   \\n                   \\n                 147-160, \\n                 CK2_PHOSPHO_SITE 47-50; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; 4-11, \\n                 MYRISTYL 143-148; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 144-149; MYRISTYL 139-144; \\n               \\n               \\n                   \\n                   \\n                   \\n                 32-52, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.22; \\n               \\n               \\n                 DEX0455_062.aa.1 \\n                 Y \\n                 0 - \\n                 395-401, \\n                 CK2_PHOSPHO_SITE 22-25; \\n                 thiored 24-132; \\n               \\n               \\n                   \\n                   \\n                 o1-491; \\n                 1.032; \\n                 MYRISTYL 140-145; \\n                 THIOREDOXIN 47-65; \\n               \\n               \\n                   \\n                   \\n                   \\n                 180-196, \\n                 CK2_PHOSPHO_SITE 290-293; \\n                 pdi_dom \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.118; \\n                 CK2_PHOSPHO_SITE 315-318; \\n                 165-269; \\n               \\n               \\n                   \\n                   \\n                   \\n                 403-409, \\n                 CK2_PHOSPHO_SITE 343-346; \\n                 THIOREDOXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.077; \\n                 PKC_PHOSPHO_SITE 157-159; \\n                 189-198; \\n               \\n               \\n                   \\n                   \\n                   \\n                 121-138, \\n                 MYRISTYL 422-427; \\n                 THIOREDOXIN 46-54; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.121; \\n                 CK2_PHOSPHO_SITE 257-260; \\n                 THIOREDOXIN_2_2 \\n               \\n               \\n                   \\n                   \\n                   \\n                 214-232, \\n                 MYRISTYL 454-459; \\n                 161-284; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.143; 4-31, \\n                 CK2_PHOSPHO_SITE 405-408; \\n                 THIOREDOXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.234; \\n                 PKC_PHOSPHO_SITE 100-102; \\n                 233-244; \\n               \\n               \\n                   \\n                   \\n                   \\n                 431-463, \\n                 CK2_PHOSPHO_SITE 158-161; \\n                 THIOREDOXIN_2_1 \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.191; \\n                 CK2_PHOSPHO_SITE 248-251; \\n                 26-137; thiored \\n               \\n               \\n                   \\n                   \\n                   \\n                 259-269, \\n                 MYRISTYL 401-406; \\n                 159-270; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.1; 86-103, \\n                 CK2_PHOSPHO_SITE 375-378; \\n                 pdi_dom 30-131; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.07; \\n                 MYRISTYL 90-95; \\n                 THIOREDOXIN \\n               \\n               \\n                   \\n                   \\n                   \\n                 292-308, \\n                 PKC_PHOSPHO_SITE 464-466; \\n                 182-200; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.304; \\n                 PKC_PHOSPHO_SITE 106-108; \\n               \\n               \\n                   \\n                   \\n                   \\n                 465-481, \\n                 MYRISTYL 116-121; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.149; 69-84, \\n                 PKC_PHOSPHO_SITE 158-160; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.2; \\n                 MYRISTYL 105-110; \\n               \\n               \\n                   \\n                   \\n                   \\n                 159-165, \\n                 PKC_PHOSPHO_SITE 148-150; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.052; 38-61, \\n                 MYRISTYL 79-84; MYRISTYL \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.134; \\n                 19-24; MYRISTYL 458-463; \\n               \\n               \\n                   \\n                   \\n                   \\n                 201-207, \\n                 MYRISTYL 144-149; \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.072; \\n                 PKC_PHOSPHO_SITE 239-241; \\n               \\n               \\n                   \\n                   \\n                   \\n                 275-281, \\n                 MYRISTYL 7-12; CK2_PHOSPHO_SITE \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.087; \\n                 23-26; \\n               \\n               \\n                   \\n                   \\n                   \\n                 369-375, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.067; 422-429, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.117; 316-326, \\n               \\n               \\n                   \\n                   \\n                   \\n                 1.166; \\n               \\n               \\n                   \\n                   \\n                   \\n                 172-178, 1.087; \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Example 1b \\n     Sequence Alignment Support \\n     Alignments between previously identified sequences and splice variant sequences are performed to confirm unique portions of splice variant nucleic acid and amino acid sequences. The alignments are done using the Needle program in the European Molecular Biology Open Software Suite (EMBOSS) version 2.2.0 available at www.emboss.org from EMBnet (http://www.embnet.org). Default settings are used unless otherwise noted. The Needle program in EMBOSS implements the Needleman-Wunsch algorithm. Needleman, S. B., Wunsch, C. D.,  J. Mol. Biol.  48:443-453 (1970). \\n     It is well know to those skilled in the art that implication of alignment algorithms by various programs may result in minor changes in the generated output. These changes include but are not limited to: alignment scores (percent identity, similarity, and gap), display of nonaligned flanking sequence regions, and number assignment to residues. These minor changes in the output of an alignment do not alter the physical characteristics of the sequences or the differences between the sequences, e.g. regions of homology, insertions, or deletions. \\n     Example 1c \\n     RT-PCR Analysis \\n     To detect the presence and tissue distribution of a particular splice variant Reverse Transcription-Polymerase Chain Reaction (RT-PCR) is performed using cDNA generated from a panel of tissue RNAs. See, e.g., Sambrook et al.,  Molecular Cloning: A Laboratory Manual,  2d ed., Cold Spring Harbor Laboratory Press (1989) and; Kawasaki E S et al.,  PNAS  85(15):5698 (1988). Total RNA is extracted from a variety of tissues and first strand cDNA is prepared with reverse transcriptase (RT). Each panel includes 23 cDNAs from five cancer types (lung, ovary, breast, colon, and prostate) and normal samples of testis, placenta and fetal brain. Each cancer set is composed of three cancer cDNAs from different donors and one normal pooled sample. Using a standard enzyme kit from BD Bioscience Clontech (Mountain View, Calif.), the target transcript is detected with sequence-specific primers designed to only amplify the particular splice variant. The PCR reaction is run on the GeneAmp PCR system 9700 (Applied Biosystem, Foster City, Calif.) thermocycler under optimal conditions. One of ordinary skill can design appropriate primers and determine optimal conditions. The amplified product is resolved on an agarose gel to detect a band of equivalent size to the predicted RT-PCR product. A band indicated the presence of the splice variant in a sample. The relation of the amplified product to the splice variant was subsequently confirmed by DNA sequencing. \\n     After subcloning, all positively screened clones are sequence verified. The DNA sequence verification results show the splice variant contains the predicted sequence differences in comparison with the reference sequence. \\n     Results for RT-PCR analysis in the table below include the sequence DEX ID, Lead Name, Cancer Tissue(s) the transcript was detected in, Normal Tissue(s) the transcript was detected in, the predicted length of the RT-PCR product, and the confirmed Length of the RT-PCR product. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 Lead \\n                 Cancer \\n                 Normal \\n                 Predicted \\n                 Confirmed \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 Tissue(s) \\n                 Tissue(s) \\n                 Length \\n                 Length \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 DEX0455_019.nt.1 \\n                 Ovr224 \\n                 Lung, \\n                 Placenta, \\n                 334 bp \\n                 334 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n                 Fetal brain \\n               \\n               \\n                   \\n                   \\n                 Colon, \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n               \\n               \\n                 DEX0455_034.nt.1, \\n                 Ovr223 \\n                 Lung, \\n                   \\n                 448 bp \\n                 894 bp \\n               \\n               \\n                 DEX0455_034.nt.2 \\n                   \\n                 Ovary, \\n                   \\n                   \\n                 (exon \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n                   \\n                   \\n                 insertion) \\n               \\n               \\n                   \\n                   \\n                 Colon \\n               \\n               \\n                 DEX0455_034.nt.3 \\n                 Ovr223v1 \\n                 Lung, \\n                 Lung, \\n                 385 bp \\n                 385 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n                 Colon, \\n               \\n               \\n                   \\n                   \\n                 Colon, \\n                 Prostate, \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n                 Placenta \\n               \\n               \\n                 DEX0455_034.nt.4 \\n                 Ovr223v2 \\n                 Lung, \\n                 Lung, \\n                 491 bp \\n                 491 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n                 Colon, \\n               \\n               \\n                   \\n                   \\n                 Colon, \\n                 Prostate, \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n                 Placenta \\n               \\n               \\n                 DEX0455_037.nt.6 \\n                 Ovr229 \\n                 Ovary, \\n                 Prostate \\n                 390 bp \\n                 387 bp \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n               \\n               \\n                 DEX0455_037.nt.7 \\n                 Ovr227 \\n                 Prostate \\n                 Placenta \\n                 257 bp \\n                 256 bp \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 Ovr232 \\n                 Lung, \\n                 Breast \\n                 134 bp \\n                 134 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Colon \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 Ovr232v1 \\n                 Lung, \\n                 Ovary, \\n                 345 bp \\n                 345 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n                 Breast \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Colon, \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 Ovr232v2 \\n                 Lung, \\n                 Lung, Ovary, \\n                 334 bp \\n                 334 bp \\n               \\n               \\n                   \\n                   \\n                 Ovary, \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n                 Colon, \\n               \\n               \\n                   \\n                   \\n                 Colon, \\n                 Prostate \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 Ovr232v3 \\n                 Colon \\n                 Breast \\n                 254 bp \\n                 254 bp \\n               \\n               \\n                 DEX0455_053.nt.2 \\n                 Ovr110V1 \\n                 Ovary, \\n                 Breast \\n                 383 bp \\n                 383 bp \\n               \\n               \\n                   \\n                   \\n                 Breast, \\n               \\n               \\n                   \\n                   \\n                 Prostate \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     RT-PCR results confirm the presence SEQ ID NO: 1-128 in biologic samples and distinguish between related transcripts. \\n     Example 1d \\n     Secretion Assay \\n     To determine if a protein encoded by a splice variant is secreted from cells a secretion assay is preformed. A pcDNA3.1 clone containing the gene transcript which encodes the variant protein is transfected into 293T cells using the Superfect transfection reagent (Qiagen, Valencia Calif.). Transfected cells are incubated for 28 hours before the media is collected and immediately spun down to remove any detached cells. The adherent cells are solubilized with lysis buffer (1% NP40, 10 mM sodium phosphate pH7.0, and 0.15M NaCl). The lysed cells are collected and spun down and the supernatant extracted as cell lysate. Western immunoblot is carried out in the following manner: 15 μl of the cell lysate and media are run on 4-12% NuPage Bis-Tris gel (Invitrogen, Carlsbad Calif.), and blotted onto a PVDF membrane (Invitrogen, Carlsbad Calif.). The blot is incubated with a polyclonal primary antibody which binds to the variant protein (Imgenex, San Diego Calif.) and polyclonal goat anti-rabbit-peroxidase secondary antibody (Sigma-Aldrich, St. Louis Mo.). The blot is developed with the ECL Plus chemiluminescent detection reagent (Amersham BioSciences, Piscataway N.J.). \\n     Secretion assay results are indicative of SEQ ID NO: 129-295 being a diagnostic marker and/or therapeutic target for cancer. \\n     Example 2a \\n     Gene Expression Analysis \\n     Custom Microarray Experiment—Cancer \\n     Custom oligonucleotide microarrays were provided by Agilent Technologies, Inc. (Palo Alto, Calif.). The microarrays were fabricated by Agilent using their technology for the in-situ synthesis of 60mer oligonucleotides (Hughes, et al. 2001, Nature Biotechnology 19:342-347). The 60mer microarray probes were designed by Agilent, from gene sequences provided by diaDexus, using Agilent proprietary algorithms. Whenever possible two different 60mers were designed for each gene of interest. \\n     All microarray experiments were two-color experiments and were preformed using Agilent-recommended protocols and reagents. Briefly, each microarray was hybridized with cRNAs synthesized from RNA (total RNA for ovarian and prostate, polyA+ RNA for lung, breast and colon samples), isolated from cancer and normal tissues, labeled with fluorescent dyes Cyanine-3 (Cy3) or Cyanine-5 (Cy5) (NEN Life Science Products, Inc., Boston, Mass.) using a linear amplification method (Agilent). In each experiment the experimental sample was RNA isolated from cancer tissue from a single individual and the reference sample was a pool of RNA isolated from normal tissues of the same organ as the cancerous tissue (i.e. normal ovarian tissue in experiments with ovarian cancer samples). Hybridizations were carried out at 60° C., overnight using Agilent in-situ hybridization buffer. Following washing, arrays were scanned with a GenePix 4000B Microarray Scanner (Axon Instruments, Inc., Union City, Calif.). The resulting images were analyzed with GenePix Pro 3.0 Microarray Acquisition and Analysis Software (Axon). \\n     Data normalization and expression profiling were done with Expressionist software from GeneData Inc. (Daly City, Calif./Basel, Switzerland). Gene expression analysis was performed using only experiments that met certain quality criteria. The quality criteria that experiments must meet are a combination of evaluations performed by the Expressionist software and evaluations performed manually using raw and normalized data. To evaluate raw data quality, detection limits (the mean signal for a replicated negative control+2 Standard Deviations (SD)) for each channel were calculated. The detection limit is a measure of non-specific hybridization. Acceptable detection limits were defined for each dye (&lt;80 for Cy5 and &lt;150 for Cy3). Arrays with poor detection limits in one or both channels were not analyzed and the experiments were repeated. To evaluate normalized data quality, positive control elements included in the array were utilized. These array features should have a mean ratio of 1 (no differential expression). If these features have a mean ratio of greater than 1.5-fold up or down, the experiments were not analyzed further and were repeated. In addition to traditional scatter plots demonstrating the distribution of signal in each experiment, the Expressionist software also has minimum thresholding criteria that employ user defined parameters to identify quality data. These thresholds include two distinct quality measurements: 1) minimum area percentage, which is a measure of the integrity of each spot and 2) signal to noise ratio, which ensures that the signal being measured is significantly above any background (nonspecific) signal present. Only those features that met the threshold criteria were included in the filtering and analyses carried out by Expressionist. The thresholding settings employed require a minimum area percentage of 60% [(% pixels&gt;background+2SD)−(% pixels saturated)], and a minimum signal to noise ratio of 2.0 in both channels. By these criteria, very low expressors, saturated features and spots with abnormally high local background were not included in analysis. \\n     Relative expression data was collected from Expressionist based on filtering and clustering analyses. Up-regulated genes were identified using criteria for the percentage of experiments in which the gene is up-regulated by at least 2-fold. In general, up-regulation in ˜30% of samples tested was used as a cutoff for filtering. \\n     Two microarray experiments were preformed for each normal and cancer tissue pair. The tissue specific Array Chip for each cancer tissue is a unique microarray specific to that tissue and cancer. The Multi-Cancer Array Chip is a universal microarray that was hybridized with samples from each of the cancers (ovarian, breast, colon, lung, and prostate). See the description below for the experiments specific to the different cancers. \\n     Microarray Experiments and Data Tables \\n     Ovarian Cancer Chips \\n     For ovarian cancer two different chip designs were evaluated with overlapping sets of a total of 19 samples, comparing the expression patterns of ovarian cancer derived total RNA to total RNA isolated from a pool of 9 normal ovarian tissues. For the Multi-Cancer Array Chip, all 19 samples (14 invasive carcinomas, 5 low malignant potential samples were analyzed and for the Ovarian Array Chip, a subset of 17 of these samples (13 invasive carcinomas, 4 low malignant potential samples) were assessed. \\n     The results for the statistically significant up-regulated genes on the Ovarian Array Chip are shown in Table 1. The results for the Multi-Cancer Array Chip are shown in Table 2. The first two columns of each table contain information about the sequence itself (DEX ID, Oligo Name), the next columns show the results obtained for all (“ALL”) ovarian cancer samples, invasive carcinomas (“INV”) and low malignant potential (“LMP”) samples. ‘% up’ indicates the percentage of all experiments in which up-regulation of at least 2-fold was observed (n=19 for the Multi-Cancer Array Chip, n=17 for the Ovarian Array Chip), ‘% valid up’ indicates the percentage of experiments with valid expression values in which up-regulation of at least 2-fold was observed. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 1 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                 Ovr \\n                   \\n                 Ovr \\n                   \\n                 Ovr \\n                   \\n               \\n               \\n                   \\n                   \\n                 ALL \\n                 Ovr ALL \\n                 INV \\n                 Ovr INV \\n                 LMP \\n                 Ovr LMP \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 % valid \\n                 % up \\n                 % valid \\n                 % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 17 \\n                 up n = 17 \\n                 n = 13 \\n                 up n = 13 \\n                 n = 4 \\n                 up n = 4 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_001.nt.1 \\n                 34930.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_001.nt.1 \\n                 34930.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21553.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21553.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21577.01 \\n                 17.6 \\n                 20.0 \\n                 15.4 \\n                 16.7 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21577.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_003.nt.1 \\n                 17466.01 \\n                 11.8 \\n                 11.8 \\n                 7.7 \\n                 7.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_003.nt.1 \\n                 17466.02 \\n                 11.8 \\n                 11.8 \\n                 7.7 \\n                 7.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 20619.01 \\n                 23.5 \\n                 25.0 \\n                 23.1 \\n                 23.1 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 20619.02 \\n                 17.6 \\n                 20.0 \\n                 15.4 \\n                 16.7 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 24874.01 \\n                 23.5 \\n                 25.0 \\n                 23.1 \\n                 25.0 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 24874.02 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_005.nt.2 \\n                 20619.01 \\n                 23.5 \\n                 25.0 \\n                 23.1 \\n                 23.1 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_005.nt.2 \\n                 20619.02 \\n                 17.6 \\n                 20.0 \\n                 15.4 \\n                 16.7 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_005.nt.2 \\n                 24874.01 \\n                 23.5 \\n                 25.0 \\n                 23.1 \\n                 25.0 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_005.nt.2 \\n                 24874.02 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_007.nt.1 \\n                 30109.01 \\n                 41.2 \\n                 46.7 \\n                 30.8 \\n                 33.3 \\n                 75.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_007.nt.1 \\n                 30109.02 \\n                 35.3 \\n                 40.0 \\n                 23.1 \\n                 27.3 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 18508.01 \\n                 23.5 \\n                 44.4 \\n                 30.8 \\n                 44.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 18508.02 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 30.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 22387.01 \\n                 35.3 \\n                 54.5 \\n                 46.2 \\n                 66.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 22387.02 \\n                 41.2 \\n                 43.8 \\n                 53.8 \\n                 58.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_009.nt.1 \\n                 9720.01 \\n                 47.1 \\n                 47.1 \\n                 38.5 \\n                 38.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_009.nt.1 \\n                 9720.02 \\n                 52.9 \\n                 52.9 \\n                 46.2 \\n                 46.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 20627.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 20627.02 \\n                 23.5 \\n                 25.0 \\n                 23.1 \\n                 25.0 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 21675.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 21675.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 21675.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 21675.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_013.nt.1 \\n                 9838.01 \\n                 35.3 \\n                 42.9 \\n                 38.5 \\n                 45.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_013.nt.1 \\n                 9838.02 \\n                 35.3 \\n                 37.5 \\n                 38.5 \\n                 38.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_013.nt.2 \\n                 9838.01 \\n                 35.3 \\n                 42.9 \\n                 38.5 \\n                 45.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_013.nt.2 \\n                 9838.02 \\n                 35.3 \\n                 37.5 \\n                 38.5 \\n                 38.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 10624.01 \\n                 17.6 \\n                 50.0 \\n                 15.4 \\n                 50.0 \\n                 25.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 10624.02 \\n                 17.6 \\n                 50.0 \\n                 7.7 \\n                 33.3 \\n                 50.0 \\n                 66.7 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 14604.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 14604.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_015.nt.1 \\n                 19518.01 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_015.nt.1 \\n                 19518.02 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_016.nt.1 \\n                 23734.01 \\n                 5.9 \\n                 6.2 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_016.nt.1 \\n                 23734.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21571.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21571.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21575.01 \\n                 41.2 \\n                 41.2 \\n                 46.2 \\n                 46.2 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21575.02 \\n                 41.2 \\n                 41.2 \\n                 46.2 \\n                 46.2 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21609.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21609.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_018.nt.2 \\n                 21575.01 \\n                 41.2 \\n                 41.2 \\n                 46.2 \\n                 46.2 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_018.nt.2 \\n                 21575.02 \\n                 41.2 \\n                 41.2 \\n                 46.2 \\n                 46.2 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_019.nt.1 \\n                 20669.01 \\n                 35.3 \\n                 42.9 \\n                 46.2 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_019.nt.1 \\n                 20669.02 \\n                 35.3 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21433.01 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21433.02 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21469.01 \\n                 70.6 \\n                 70.6 \\n                 61.5 \\n                 61.5 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21469.02 \\n                 82.4 \\n                 82.4 \\n                 76.9 \\n                 76.9 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21475.01 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21475.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 23780.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 23780.02 \\n                 41.2 \\n                 50.0 \\n                 46.2 \\n                 54.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21433.01 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21433.02 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21469.01 \\n                 70.6 \\n                 70.6 \\n                 61.5 \\n                 61.5 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21469.02 \\n                 82.4 \\n                 82.4 \\n                 76.9 \\n                 76.9 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21475.01 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21475.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 23780.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 23780.02 \\n                 41.2 \\n                 50.0 \\n                 46.2 \\n                 54.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21433.01 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21433.02 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21469.01 \\n                 70.6 \\n                 70.6 \\n                 61.5 \\n                 61.5 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21469.02 \\n                 82.4 \\n                 82.4 \\n                 76.9 \\n                 76.9 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21475.01 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21475.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 23780.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 23780.02 \\n                 41.2 \\n                 50.0 \\n                 46.2 \\n                 54.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21433.01 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21433.02 \\n                 64.7 \\n                 64.7 \\n                 61.5 \\n                 61.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21469.01 \\n                 70.6 \\n                 70.6 \\n                 61.5 \\n                 61.5 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21469.02 \\n                 82.4 \\n                 82.4 \\n                 76.9 \\n                 76.9 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21475.01 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21475.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 23780.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 23780.02 \\n                 41.2 \\n                 50.0 \\n                 46.2 \\n                 54.5 \\n                 25.0 \\n                 33.3 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 9920.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 9920.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20299.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20299.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20311.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20311.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20317.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20317.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 9920.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 9920.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20299.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20299.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20311.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20311.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20317.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20317.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 9920.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 9920.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20311.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20311.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20317.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20317.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16187.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16187.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16374.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16374.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16378.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16378.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 12149.01 \\n                 52.9 \\n                 52.9 \\n                 46.2 \\n                 46.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 12149.02 \\n                 47.1 \\n                 47.1 \\n                 38.5 \\n                 38.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21487.01 \\n                 5.9 \\n                 6.7 \\n                 7.7 \\n                 8.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21487.02 \\n                 17.6 \\n                 18.8 \\n                 15.4 \\n                 16.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21507.01 \\n                 29.4 \\n                 29.4 \\n                 23.1 \\n                 23.1 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21507.02 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21547.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21547.02 \\n                 41.2 \\n                 41.2 \\n                 38.5 \\n                 38.5 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 12149.01 \\n                 52.9 \\n                 52.9 \\n                 46.2 \\n                 46.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 12149.02 \\n                 47.1 \\n                 47.1 \\n                 38.5 \\n                 38.5 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21507.01 \\n                 29.4 \\n                 29.4 \\n                 23.1 \\n                 23.1 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21507.02 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21547.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21547.02 \\n                 41.2 \\n                 41.2 \\n                 38.5 \\n                 38.5 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 12167.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 12167.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16956.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16956.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16958.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16958.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16964.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16964.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 19010.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 19010.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 12167.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 12167.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16956.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16956.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16958.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16958.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16964.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16964.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 19010.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 19010.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 12167.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 12167.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 16958.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 16958.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 19010.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 19010.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 12167.01 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 12167.02 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16956.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16956.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16958.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16958.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16964.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16964.02 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 19010.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 19010.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_027.nt.1 \\n                 21549.01 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_027.nt.1 \\n                 21549.02 \\n                 29.4 \\n                 31.2 \\n                 23.1 \\n                 25.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17430.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17430.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17448.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17448.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 22113.01 \\n                 11.8 \\n                 25.0 \\n                 15.4 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 22113.02 \\n                 11.8 \\n                 20.0 \\n                 15.4 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23386.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23386.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23400.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23400.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17424.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17424.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17430.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17430.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17448.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17448.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 22113.01 \\n                 11.8 \\n                 25.0 \\n                 15.4 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 22113.02 \\n                 11.8 \\n                 20.0 \\n                 15.4 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23386.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23386.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23400.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23400.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 11613.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 11613.02 \\n                 11.8 \\n                 13.3 \\n                 15.4 \\n                 16.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17204.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17204.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17262.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17262.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17278.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17278.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 11613.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 11613.02 \\n                 11.8 \\n                 13.3 \\n                 15.4 \\n                 16.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17204.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17204.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17262.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17262.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17274.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17274.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17278.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17278.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.1 \\n                 20773.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.1 \\n                 20773.02 \\n                 23.5 \\n                 25.0 \\n                 30.8 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.2 \\n                 20773.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.2 \\n                 20773.02 \\n                 23.5 \\n                 25.0 \\n                 30.8 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.3 \\n                 20773.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_031.nt.3 \\n                 20773.02 \\n                 23.5 \\n                 25.0 \\n                 30.8 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 11585.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 11585.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 18556.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 18556.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 10722.01 \\n                 82.4 \\n                 82.4 \\n                 84.6 \\n                 84.6 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 10722.02 \\n                 76.5 \\n                 81.2 \\n                 84.6 \\n                 84.6 \\n                 50.0 \\n                 66.7 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21401.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21401.02 \\n                 5.9 \\n                 6.7 \\n                 7.7 \\n                 8.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21421.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21421.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 103385.01 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 103385.02 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 103385.01 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 103385.02 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 103385.01 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 103385.02 \\n                 58.8 \\n                 58.8 \\n                 76.9 \\n                 76.9 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_036.nt.1 \\n                 92327.01 \\n                 52.9 \\n                 56.2 \\n                 61.5 \\n                 66.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.1 \\n                 92327.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.2 \\n                 92327.01 \\n                 52.9 \\n                 56.2 \\n                 61.5 \\n                 66.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.2 \\n                 92327.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.3 \\n                 92327.01 \\n                 52.9 \\n                 56.2 \\n                 61.5 \\n                 66.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.3 \\n                 92327.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.4 \\n                 92327.01 \\n                 52.9 \\n                 56.2 \\n                 61.5 \\n                 66.7 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_036.nt.4 \\n                 92327.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 11575.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 11575.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17486.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17486.02 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17490.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17490.02 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 11575.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 11575.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 17486.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 17486.02 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 17490.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 17490.02 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 11575.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 11575.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17486.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17486.02 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17490.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17490.02 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 11575.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 11575.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17486.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17486.02 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17490.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17490.02 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 11575.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 11575.02 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17486.01 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17486.02 \\n                 47.1 \\n                 47.1 \\n                 46.2 \\n                 46.2 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17490.01 \\n                 52.9 \\n                 52.9 \\n                 53.8 \\n                 53.8 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17490.02 \\n                 58.8 \\n                 58.8 \\n                 53.8 \\n                 53.8 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_039.nt.1 \\n                 21505.01 \\n                 94.1 \\n                 94.1 \\n                 92.3 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_039.nt.1 \\n                 21505.02 \\n                 94.1 \\n                 94.1 \\n                 92.3 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_039.nt.2 \\n                 11527.01 \\n                 88.2 \\n                 88.2 \\n                 84.6 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_039.nt.2 \\n                 11527.02 \\n                 88.2 \\n                 88.2 \\n                 84.6 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21489.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21489.02 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21501.01 \\n                 47.1 \\n                 50.0 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21501.02 \\n                 41.2 \\n                 41.2 \\n                 53.8 \\n                 53.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21511.01 \\n                 47.1 \\n                 47.1 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21511.02 \\n                 47.1 \\n                 47.1 \\n                 53.8 \\n                 53.8 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21489.01 \\n                 11.8 \\n                 11.8 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21489.02 \\n                 17.6 \\n                 18.8 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21501.01 \\n                 47.1 \\n                 50.0 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21501.02 \\n                 41.2 \\n                 41.2 \\n                 53.8 \\n                 53.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21511.01 \\n                 47.1 \\n                 47.1 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21511.02 \\n                 47.1 \\n                 47.1 \\n                 53.8 \\n                 53.8 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 12155.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 12155.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16980.01 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16980.02 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.2 \\n                 12155.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.2 \\n                 12155.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_042.nt.1 \\n                 18214.01 \\n                 94.1 \\n                 94.1 \\n                 92.3 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_042.nt.1 \\n                 18214.02 \\n                 88.2 \\n                 93.8 \\n                 84.6 \\n                 91.7 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_043.nt.1 \\n                 14656.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_043.nt.1 \\n                 14656.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_043.nt.3 \\n                 14656.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_043.nt.3 \\n                 14656.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_045.nt.1 \\n                 36013.01 \\n                 23.5 \\n                 23.5 \\n                 7.7 \\n                 7.7 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_045.nt.1 \\n                 36013.02 \\n                 11.8 \\n                 11.8 \\n                 0.0 \\n                 0.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 17314.01 \\n                 23.5 \\n                 26.7 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 66.7 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 17314.02 \\n                 23.5 \\n                 26.7 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 66.7 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 11511.01 \\n                 94.1 \\n                 100.0 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 11511.02 \\n                 88.2 \\n                 100.0 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 11511.01 \\n                 94.1 \\n                 100.0 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 11511.02 \\n                 88.2 \\n                 100.0 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 11511.01 \\n                 94.1 \\n                 100.0 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 11511.02 \\n                 88.2 \\n                 100.0 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 11511.01 \\n                 94.1 \\n                 100.0 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 11511.02 \\n                 88.2 \\n                 100.0 \\n                 84.6 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.01 \\n                 11.8 \\n                 18.2 \\n                 15.4 \\n                 20.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.02 \\n                 17.6 \\n                 23.1 \\n                 7.7 \\n                 9.1 \\n                 50.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_052.nt.1 \\n                 91971.01 \\n                 94.1 \\n                 94.1 \\n                 92.3 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_052.nt.1 \\n                 91971.02 \\n                 94.1 \\n                 94.1 \\n                 92.3 \\n                 92.3 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 11273.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 11273.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 20541.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 20541.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 11273.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 11273.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 20541.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 20541.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 11273.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 11273.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 20541.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 20541.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 18520.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 18520.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 22734.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 22734.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 23444.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 23444.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 18520.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 18520.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 22734.01 \\n                 5.9 \\n                 5.9 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 22734.02 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 23444.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 23444.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 24524.01 \\n                 70.6 \\n                 70.6 \\n                 69.2 \\n                 69.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 24524.02 \\n                 70.6 \\n                 70.6 \\n                 69.2 \\n                 69.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 24524.01 \\n                 70.6 \\n                 70.6 \\n                 69.2 \\n                 69.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 24524.02 \\n                 70.6 \\n                 70.6 \\n                 69.2 \\n                 69.2 \\n                 75.0 \\n                 75.0 \\n               \\n               \\n                 DEX0455_058.nt.1 \\n                 14656.01 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_058.nt.1 \\n                 14656.02 \\n                 17.6 \\n                 17.6 \\n                 23.1 \\n                 23.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 11469.01 \\n                 47.1 \\n                 47.1 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 11469.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17370.01 \\n                 5.9 \\n                 25.0 \\n                 7.7 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17370.02 \\n                 5.9 \\n                 25.0 \\n                 7.7 \\n                 25.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17372.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17372.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 11469.01 \\n                 47.1 \\n                 47.1 \\n                 61.5 \\n                 61.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 11469.02 \\n                 52.9 \\n                 52.9 \\n                 61.5 \\n                 61.5 \\n                 25.0 \\n                 25.0 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 17372.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 17372.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 10372.01 \\n                 35.3 \\n                 35.3 \\n                 46.2 \\n                 46.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 10372.02 \\n                 35.3 \\n                 35.3 \\n                 46.2 \\n                 46.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 18582.01 \\n                 23.5 \\n                 23.5 \\n                 30.8 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 18582.02 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 96523.01 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 96523.02 \\n                 17.6 \\n                 17.6 \\n                 7.7 \\n                 7.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 103529.01 \\n                 23.5 \\n                 25.0 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 103529.02 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 96523.01 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 96523.02 \\n                 17.6 \\n                 17.6 \\n                 7.7 \\n                 7.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 103529.01 \\n                 23.5 \\n                 25.0 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 103529.02 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 96523.01 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 96523.02 \\n                 17.6 \\n                 17.6 \\n                 7.7 \\n                 7.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 103529.01 \\n                 23.5 \\n                 25.0 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 103529.02 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 96523.01 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 96523.02 \\n                 17.6 \\n                 17.6 \\n                 7.7 \\n                 7.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 103529.01 \\n                 23.5 \\n                 25.0 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 103529.02 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 96523.01 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 96523.02 \\n                 17.6 \\n                 17.6 \\n                 7.7 \\n                 7.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 103529.01 \\n                 23.5 \\n                 25.0 \\n                 15.4 \\n                 16.7 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 103529.02 \\n                 23.5 \\n                 23.5 \\n                 15.4 \\n                 15.4 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 17464.01 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 17464.02 \\n                 29.4 \\n                 29.4 \\n                 38.5 \\n                 38.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 18094.01 \\n                 52.9 \\n                 52.9 \\n                 69.2 \\n                 69.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 18094.02 \\n                 52.9 \\n                 52.9 \\n                 69.2 \\n                 69.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 2 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Ovr \\n                   \\n                 Ovr \\n                   \\n                 Ovr \\n               \\n               \\n                   \\n                   \\n                 Ovr \\n                 Multi- \\n                 Ovr \\n                 Multi- \\n                 Ovr \\n                 Multi- \\n               \\n               \\n                   \\n                   \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n               \\n               \\n                   \\n                   \\n                 Cancer \\n                 ALL \\n                 Cancer \\n                 INV \\n                 Cancer \\n                 LMP \\n               \\n               \\n                   \\n                 Oligo \\n                 ALL % up \\n                 % valid \\n                 INV % up \\n                 % valid \\n                 LMP % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 19 \\n                 up n = 19 \\n                 n = 14 \\n                 up n = 14 \\n                 n = 5 \\n                 up n = 5 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_002.nt.1 \\n                 79699.1 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.0 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.1 \\n                 26.3 \\n                 26.3 \\n                 21.4 \\n                 21.4 \\n                 40.0 \\n                 40.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.0 \\n                 21.1 \\n                 21.1 \\n                 28.6 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.1 \\n                 21.1 \\n                 21.1 \\n                 28.6 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.0 \\n                 21.1 \\n                 21.1 \\n                 28.6 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.1 \\n                 21.1 \\n                 21.1 \\n                 28.6 \\n                 28.6 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 31.6 \\n                 42.9 \\n                 14.3 \\n                 20.0 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.1 \\n                 31.6 \\n                 35.3 \\n                 14.3 \\n                 15.4 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 89.5 \\n                 89.5 \\n                 85.7 \\n                 85.7 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.1 \\n                 84.2 \\n                 88.9 \\n                 85.7 \\n                 85.7 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 94.7 \\n                 100.0 \\n                 92.9 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.1 \\n                 89.5 \\n                 100.0 \\n                 92.9 \\n                 100.0 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 89.5 \\n                 89.5 \\n                 85.7 \\n                 85.7 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.1 \\n                 84.2 \\n                 88.9 \\n                 85.7 \\n                 85.7 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 94.7 \\n                 100.0 \\n                 92.9 \\n                 100.0 \\n                 100.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.1 \\n                 89.5 \\n                 100.0 \\n                 92.9 \\n                 100.0 \\n                 80.0 \\n                 100.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.0 \\n                 31.6 \\n                 42.9 \\n                 28.6 \\n                 40.0 \\n                 40.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.1 \\n                 31.6 \\n                 50.0 \\n                 28.6 \\n                 44.4 \\n                 40.0 \\n                 66.7 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2023.0 \\n                 21.1 \\n                 23.5 \\n                 28.6 \\n                 30.8 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5327.0 \\n                 15.8 \\n                 16.7 \\n                 21.4 \\n                 21.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5328.0 \\n                 10.5 \\n                 11.1 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.0 \\n                 42.1 \\n                 47.1 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.1 \\n                 47.4 \\n                 52.9 \\n                 64.3 \\n                 69.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.0 \\n                 36.8 \\n                 38.9 \\n                 50.0 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.1 \\n                 42.1 \\n                 44.4 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.0 \\n                 42.1 \\n                 47.1 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.1 \\n                 47.4 \\n                 52.9 \\n                 64.3 \\n                 69.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.0 \\n                 36.8 \\n                 38.9 \\n                 50.0 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.1 \\n                 42.1 \\n                 44.4 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.0 \\n                 42.1 \\n                 47.1 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.1 \\n                 47.4 \\n                 52.9 \\n                 64.3 \\n                 69.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.0 \\n                 36.8 \\n                 38.9 \\n                 50.0 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.1 \\n                 42.1 \\n                 44.4 \\n                 57.1 \\n                 57.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.0 \\n                 5.3 \\n                 5.6 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.1 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.0 \\n                 15.8 \\n                 16.7 \\n                 14.3 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.0 \\n                 5.3 \\n                 5.6 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.1 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.0 \\n                 15.8 \\n                 16.7 \\n                 14.3 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.0 \\n                 5.3 \\n                 5.6 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.1 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.0 \\n                 15.8 \\n                 16.7 \\n                 14.3 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.0 \\n                 10.5 \\n                 11.8 \\n                 14.3 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.1 \\n                 5.3 \\n                 5.9 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.0 \\n                 10.5 \\n                 12.5 \\n                 14.3 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.1 \\n                 15.8 \\n                 16.7 \\n                 14.3 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.0 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.1 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.0 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.1 \\n                 21.1 \\n                 22.2 \\n                 21.4 \\n                 23.1 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.0 \\n                 10.5 \\n                 11.8 \\n                 14.3 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.1 \\n                 5.3 \\n                 5.9 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.0 \\n                 10.5 \\n                 12.5 \\n                 14.3 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.1 \\n                 15.8 \\n                 16.7 \\n                 14.3 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.0 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.1 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.0 \\n                 15.8 \\n                 15.8 \\n                 14.3 \\n                 14.3 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.1 \\n                 21.1 \\n                 22.2 \\n                 21.4 \\n                 23.1 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1168.0 \\n                 10.5 \\n                 10.5 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1175.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 5.3 \\n                 5.3 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 5.3 \\n                 5.3 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.0 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.0 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.0 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.0 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.0 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.1 \\n                 21.1 \\n                 21.1 \\n                 21.4 \\n                 21.4 \\n                 20.0 \\n                 20.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Breast Cancer Chips \\n     For breast cancer two different chip designs were evaluated with overlapping sets of a total of 36 samples, comparing the expression patterns of breast cancer derived polyA+ RNA to polyA+ RNA isolated from a pool of 10 normal breast tissues. For the Breast Array Chip, all 36 samples (9 stage I cancers, 23 stage II cancers, 4 stage III cancers) were analyzed. These samples also represented 10 Grade 1/2 and 26 Grade 3 cancers. The histopathologic grades for cancer are classified as follows: GX, cannot be assessed; G1, well differentiated; G2, moderately differentiated; G3, poorly differentiated; and G4, undifferentiated.  AJCC Cancer Staging Handbook , pp. 9, (5th Ed, 1998). Samples were further grouped based on the expression patterns of the known breast cancer associated genes Her2 and ERα (10 HER2 up, 26 HER2 not up, 20 ER up and 16 ER not up) and for the Multi-Cancer Array Chip, a subset of 20 of these samples (9 stage I cancers, 8 stage II cancers, 3 stage III cancers) were assessed. \\n     The results for the statistically significant up-regulated genes on the Breast Array Chip are shown in Tables 3 and 4. The results for the statistically significant up-regulated genes on the Multi-Cancer Array Chip are shown in Table 5. The first two columns of each table contain information about the sequence itself (Seq ID, Oligo Name), the next columns show the results obtained for all (“ALL”) breast cancer samples, cancers corresponding to stageI (“ST1”), stages II and III (“ST2,3”), grades 1 and 2 (“GR1,2”), grade 3 (“GR3”), cancers exhibiting up-regulation of Her2 (“HER2up”) or ERα (“ERup”) or those not exhibiting up-regulation of Her2 (“NOT HER2up”) or ERα (“NOT ERup”). ‘% up’ indicates the percentage of all experiments in which up-regulation of at least 2-fold was observed (n=36 for Colon Array Chip, n=20 for the Multi-Cancer Array Chip), ‘% valid up’ indicates the percentage of experiments with valid expression values in which up-regulation of at least 2-fold was observed. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 3 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Mam \\n                   \\n                 Mam \\n                   \\n                 Mam \\n                   \\n                 Mam \\n                   \\n                 Mam \\n               \\n               \\n                   \\n                   \\n                 Mam \\n                 ALL % \\n                 Mam \\n                 ST1 % \\n                 Mam \\n                 ST2,3 % \\n                 Mam \\n                 GR1,2 % \\n                 Mam \\n                 GR3 % \\n               \\n               \\n                   \\n                   \\n                 ALL \\n                 valid \\n                 ST1 \\n                 valid \\n                 ST2,3 \\n                 valid \\n                 GR1,2 \\n                 valid \\n                 GR3 \\n                 valid \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 36 \\n                 n = 36 \\n                 n = 9 \\n                 n = 9 \\n                 n = 27 \\n                 n = 27 \\n                 n = 10 \\n                 n = 10 \\n                 n = 26 \\n                 n = 26 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 32151.0 \\n                 22.2 \\n                 22.2 \\n                 44.4 \\n                 44.4 \\n                 14.8 \\n                 14.8 \\n                 10.0 \\n                 10.0 \\n                 26.9 \\n                 26.9 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 28221.0 \\n                 2.8 \\n                 3.1 \\n                 0.0 \\n                 0.0 \\n                 3.7 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 3.8 \\n                 4.5 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 23280.0 \\n                 11.1 \\n                 11.8 \\n                 11.1 \\n                 11.1 \\n                 11.1 \\n                 12.0 \\n                 10.0 \\n                 10.0 \\n                 11.5 \\n                 12.5 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21143.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21144.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16998.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 19072.0 \\n                 11.1 \\n                 11.4 \\n                 0.0 \\n                 0.0 \\n                 14.8 \\n                 15.4 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 8.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 22136.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 5.6 \\n                 5.7 \\n                 0.0 \\n                 0.0 \\n                 7.4 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 8.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.2 \\n                 5.6 \\n                 5.7 \\n                 0.0 \\n                 0.0 \\n                 7.4 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 8.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 29736.0 \\n                 2.8 \\n                 2.8 \\n                 0.0 \\n                 0.0 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 3.8 \\n                 3.8 \\n               \\n               \\n                 DEX0455_054.nt.1 \\n                 19799.0 \\n                 8.3 \\n                 8.3 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n                 0.0 \\n                 0.0 \\n                 11.5 \\n                 11.5 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 4 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 Mam \\n                   \\n                   \\n                   \\n                 Mam \\n               \\n               \\n                   \\n                   \\n                   \\n                 Mam \\n                 Mam \\n                 NOT \\n                   \\n                 Mam \\n                 Mam \\n                 NOT \\n               \\n               \\n                   \\n                   \\n                 Mam \\n                 HER2up \\n                 NOT \\n                 HER2up \\n                 Mam \\n                 ERup \\n                 NOT \\n                 ERup \\n               \\n               \\n                   \\n                   \\n                 HER2up \\n                 % valid \\n                 HER2up \\n                 % valid \\n                 ERup \\n                 % valid \\n                 ERup \\n                 % valid \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 10 \\n                 n = 10 \\n                 n = 26 \\n                 n = 26 \\n                 n = 20 \\n                 n = 20 \\n                 n = 16 \\n                 n = 16 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 32151.0 \\n                 20.0 \\n                 20.0 \\n                 23.1 \\n                 23.1 \\n                 10.0 \\n                 10.0 \\n                 37.5 \\n                 37.5 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 28221.0 \\n                 10.0 \\n                 11.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 6.2 \\n                 8.3 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 23280.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 8.3 \\n                 10.0 \\n                 11.1 \\n                 12.5 \\n                 12.5 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21143.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21144.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16998.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 19072.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 8.0 \\n                 15.0 \\n                 15.0 \\n                 6.2 \\n                 6.7 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 22136.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 8.0 \\n                 0.0 \\n                 0.0 \\n                 12.5 \\n                 13.3 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.2 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 8.0 \\n                 0.0 \\n                 0.0 \\n                 12.5 \\n                 13.3 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 29736.0 \\n                 0.0 \\n                 0.0 \\n                 3.8 \\n                 3.8 \\n                 0.0 \\n                 0.0 \\n                 6.2 \\n                 6.2 \\n               \\n               \\n                 DEX0455_054.nt.1 \\n                 19799.0 \\n                 10.0 \\n                 10.0 \\n                 7.7 \\n                 7.7 \\n                 10.0 \\n                 10.0 \\n                 6.2 \\n                 6.2 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12731.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12732.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 5 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Mam \\n                   \\n                 Mam \\n                 Mam \\n                 Mam \\n               \\n               \\n                   \\n                   \\n                 Mam \\n                 Multi- \\n                 Mam \\n                 Multi- \\n                 Multi- \\n                 Multi- \\n               \\n               \\n                   \\n                   \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n                 Cancer \\n                 Cancer \\n               \\n               \\n                   \\n                   \\n                 Cancer \\n                 ALL \\n                 Cancer \\n                 ST1 \\n                 ST2,3 \\n                 ST2,3 \\n               \\n               \\n                   \\n                 Oligo \\n                 ALL % up \\n                 % valid \\n                 ST1 % up \\n                 % valid \\n                 % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 20 \\n                 up n = 20 \\n                 n = 9 \\n                 up n = 9 \\n                 n = 11 \\n                 up n = 11 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_002.nt.1 \\n                 79699.1 \\n                 20.0 \\n                 20.0 \\n                 44.4 \\n                 44.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.0 \\n                 10.0 \\n                 10.0 \\n                 22.2 \\n                 22.2 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.1 \\n                 15.0 \\n                 15.0 \\n                 33.3 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.0 \\n                 15.0 \\n                 15.0 \\n                 11.1 \\n                 11.1 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.1 \\n                 15.0 \\n                 15.0 \\n                 11.1 \\n                 11.1 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.0 \\n                 15.0 \\n                 15.0 \\n                 11.1 \\n                 11.1 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.1 \\n                 15.0 \\n                 15.0 \\n                 11.1 \\n                 11.1 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 5.0 \\n                 7.1 \\n                 11.1 \\n                 20.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.1 \\n                 5.0 \\n                 7.1 \\n                 11.1 \\n                 20.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.1 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.1 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.0 \\n                 10.0 \\n                 10.0 \\n                 11.1 \\n                 11.1 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.1 \\n                 10.0 \\n                 10.0 \\n                 11.1 \\n                 11.1 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2023.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5327.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5328.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 66.7 \\n                 66.7 \\n                 36.4 \\n                 36.4 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.1 \\n                 40.0 \\n                 40.0 \\n                 66.7 \\n                 66.7 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.0 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.1 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 66.7 \\n                 66.7 \\n                 36.4 \\n                 36.4 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.1 \\n                 40.0 \\n                 40.0 \\n                 66.7 \\n                 66.7 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.0 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.1 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 66.7 \\n                 66.7 \\n                 36.4 \\n                 36.4 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.1 \\n                 40.0 \\n                 40.0 \\n                 66.7 \\n                 66.7 \\n                 18.2 \\n                 18.2 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.0 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.1 \\n                 20.0 \\n                 20.0 \\n                 33.3 \\n                 33.3 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1168.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1175.0 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n                 9.1 \\n                 9.1 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Colon Cancer Chips \\n     For colon cancer two different chip designs were evaluated with overlapping sets of a total of 38 samples, comparing the expression patterns of colon cancer derived polyA+ RNA to polyA+ RNA isolated from a pool of 7 normal colon tissues. For the Colon Array Chip all 38 samples (23 Ascending colon carcinomas and 15 Rectosigmoidal carcinomas including: 5 stage I cancers, 15 stage II cancers, 15 stage III and 2 stage IV cancers, as well as 28 Grade 1/2 and 10 Grade 3 cancers) were analyzed. The histopathologic grades for cancer are classified as follows: GX, cannot be assessed; G1, well differentiated; G2, Moderately differentiated; G3, poorly differentiated; and G4, undifferentiated.  AJCC Cancer Staging Handbook,  5 th  Edition, 1998, page 9. For the Colon Array Chip analysis, samples were further divided into groups based on the expression pattern of the known colon cancer associated gene Thymidilate Synthase (TS) (13 TS up 25 TS not up). The association of TS with advanced colorectal cancer is well documented. Paradiso et al.,  Br J Cancer  82(3):560-7 (2000); Etienne et al.,  J Clin Oncol.  20(12):2832-43 (2002); Aschele et al.  Clin Cancer Res.  6(12):4797-802 (2000). For the Multi-Cancer Array Chip a subset of 27 of these samples (14 Ascending colon carcinomas and 13 Rectosigmoidal carcinomas including: 3 stage I cancers, 9 stage II cancers, 13 stage III and 2 stage IV cancers) were assessed. \\n     The results for the statistically significant up-regulated genes on the Colon Array Chip are shown in Tables 6 and 7. The results for the statistically significant up-regulated genes on the Multi-Cancer Array Chip are shown in Table 8. \\n     The first two columns of each table contain information about the sequence itself (Seq ID, Oligo Name), the next columns show the results obtained for all (“ALL”) the colon samples, ascending colon carcinomas (“ASC”), Rectosigmoidal carcinomas (“RS”), cancers corresponding to stages I and II (“ST1,2”), stages III and IV (“ST3,4”), grades 1 and 2 (“GR1,2”), grade 3 (“GR3”), cancers exhibiting up-regulation of the TS gene (“TSup”) or those not exhibiting up-regulation of the TS gene (“NOT TSup”). ‘% up’ indicates the percentage of all experiments in which up-regulation of at least 2-fold was observed n=38 for the Colon Array Chip (n=27 for the Multi-Cancer Array Chip), ‘% valid up’ indicates the percentage of experiments with valid expression values in which up-regulation of at least 2-fold was observed. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 6 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Cln \\n                   \\n                 Cln \\n                   \\n                 Cln \\n                   \\n                 Cln \\n                   \\n                 Cln \\n               \\n               \\n                   \\n                   \\n                 Cln \\n                 ALL % \\n                 Cln \\n                 ASC % \\n                 Cln \\n                 RS % \\n                 Cln \\n                 ST1,2 % \\n                 Cln \\n                 ST3,4 % \\n               \\n               \\n                   \\n                   \\n                 ALL \\n                 valid \\n                 ASC \\n                 valid \\n                 RS \\n                 valid \\n                 ST1,2 \\n                 valid \\n                 ST3,4 \\n                 valid \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 38 \\n                 n = 38 \\n                 n = 23 \\n                 n = 23 \\n                 n = 15 \\n                 n = 15 \\n                 n = 20 \\n                 n = 20 \\n                 n = 18 \\n                 n = 18 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 37415.0 \\n                 52.6 \\n                 52.6 \\n                 69.6 \\n                 69.6 \\n                 26.7 \\n                 26.7 \\n                 50.0 \\n                 50.0 \\n                 55.6 \\n                 55.6 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34343.0 \\n                 5.3 \\n                 5.7 \\n                 4.3 \\n                 4.5 \\n                 6.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 12.5 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34368.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34369.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34343.0 \\n                 5.3 \\n                 5.7 \\n                 4.3 \\n                 4.5 \\n                 6.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 12.5 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34368.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34369.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 21032.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 17957.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 30821.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30820.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30821.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30824.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 10.0 \\n                 10.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30869.0 \\n                 18.4 \\n                 18.4 \\n                 17.4 \\n                 17.4 \\n                 20.0 \\n                 20.0 \\n                 15.0 \\n                 15.0 \\n                 22.2 \\n                 22.2 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41117.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41151.0 \\n                 10.5 \\n                 10.5 \\n                 13.0 \\n                 13.0 \\n                 6.7 \\n                 6.7 \\n                 15.0 \\n                 15.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41152.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30820.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30821.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30824.0 \\n                 7.9 \\n                 7.9 \\n                 8.7 \\n                 8.7 \\n                 6.7 \\n                 6.7 \\n                 10.0 \\n                 10.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30922.0 \\n                 10.5 \\n                 10.5 \\n                 13.0 \\n                 13.0 \\n                 6.7 \\n                 6.7 \\n                 15.0 \\n                 15.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41117.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41151.0 \\n                 10.5 \\n                 10.5 \\n                 13.0 \\n                 13.0 \\n                 6.7 \\n                 6.7 \\n                 15.0 \\n                 15.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41152.0 \\n                 2.6 \\n                 2.6 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n                 5.0 \\n                 5.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 16423.0 \\n                 2.6 \\n                 3.1 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 9.1 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 6.2 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 36902.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36901.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36902.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 36901.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 36902.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36901.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36902.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 36901.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 36902.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19803.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19804.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19803.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19804.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19803.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19804.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19803.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19804.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19803.0 \\n                 2.6 \\n                 2.6 \\n                 4.3 \\n                 4.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 5.6 \\n                 5.6 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19804.0 \\n                 5.3 \\n                 5.3 \\n                 4.3 \\n                 4.3 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 11.1 \\n                 11.1 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 7 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 Cln \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Cln \\n                   \\n                 Cln \\n                 Cln \\n                 Cln TS \\n                 NOT \\n               \\n               \\n                   \\n                   \\n                 Cln \\n                 GR1,2 \\n                 Cln \\n                 GR3 \\n                 TS \\n                 up \\n                 TS \\n                 Cln NOT \\n               \\n               \\n                   \\n                   \\n                 GR1,2 \\n                 % valid \\n                 GR3 \\n                 % valid \\n                 up \\n                 % valid \\n                 up \\n                 TS up \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 28 \\n                 n = 28 \\n                 n = 10 \\n                 n = 10 \\n                 n = 13 \\n                 n = 13 \\n                 n = 25 \\n                 up n = 25 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 37415.0 \\n                 46.4 \\n                 46.4 \\n                 70.0 \\n                 70.0 \\n                 46.2 \\n                 46.2 \\n                 56.0 \\n                 56.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34343.0 \\n                 3.6 \\n                 3.7 \\n                 10.0 \\n                 12.5 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34368.0 \\n                 7.1 \\n                 7.1 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34369.0 \\n                 7.1 \\n                 7.1 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34343.0 \\n                 3.6 \\n                 3.7 \\n                 10.0 \\n                 12.5 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34368.0 \\n                 7.1 \\n                 7.1 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34369.0 \\n                 7.1 \\n                 7.1 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 21032.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 17957.0 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 30821.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30820.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30821.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30824.0 \\n                 10.7 \\n                 10.7 \\n                 0.0 \\n                 0.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30869.0 \\n                 17.9 \\n                 17.9 \\n                 20.0 \\n                 20.0 \\n                 30.8 \\n                 30.8 \\n                 12.0 \\n                 12.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41117.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41151.0 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n                 15.4 \\n                 15.4 \\n                 8.0 \\n                 8.0 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41152.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30820.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30821.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30824.0 \\n                 10.7 \\n                 10.7 \\n                 0.0 \\n                 0.0 \\n                 15.4 \\n                 15.4 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30922.0 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n                 15.4 \\n                 15.4 \\n                 8.0 \\n                 8.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41117.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41120.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41151.0 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n                 15.4 \\n                 15.4 \\n                 8.0 \\n                 8.0 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41152.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 16423.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 12.5 \\n                 7.7 \\n                 8.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 36902.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36901.0 \\n                 3.6 \\n                 3.6 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36902.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 36901.0 \\n                 3.6 \\n                 3.6 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 36902.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36901.0 \\n                 3.6 \\n                 3.6 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36902.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 36901.0 \\n                 3.6 \\n                 3.6 \\n                 10.0 \\n                 10.0 \\n                 15.4 \\n                 15.4 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 36902.0 \\n                 3.6 \\n                 3.6 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19803.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19804.0 \\n                 0.0 \\n                 0.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19803.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19804.0 \\n                 0.0 \\n                 0.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19803.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19804.0 \\n                 0.0 \\n                 0.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19803.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19804.0 \\n                 0.0 \\n                 0.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19803.0 \\n                 0.0 \\n                 0.0 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19804.0 \\n                 0.0 \\n                 0.0 \\n                 20.0 \\n                 20.0 \\n                 7.7 \\n                 7.7 \\n                 4.0 \\n                 4.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 8 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Cln \\n                   \\n                 Cln \\n                   \\n                 Cln \\n               \\n               \\n                   \\n                   \\n                 Cln \\n                 Multi- \\n                 Cln \\n                 Multi- \\n                 Cln \\n                 Multi- \\n               \\n               \\n                   \\n                   \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n               \\n               \\n                   \\n                   \\n                 Cancer \\n                 ALL \\n                 Cancer \\n                 ASC \\n                 Cancer \\n                 RS \\n               \\n               \\n                   \\n                 Oligo \\n                 ALL % up \\n                 % valid \\n                 ASC % up \\n                 % valid \\n                 RS % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 27 \\n                 up n = 27 \\n                 n = 14 \\n                 up n = 14 \\n                 n = 13 \\n                 up n = 13 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_002.nt.1 \\n                 79699.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.1 \\n                 7.4 \\n                 33.3 \\n                 14.3 \\n                 66.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.0 \\n                 3.7 \\n                 20.0 \\n                 7.1 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.1 \\n                 3.7 \\n                 25.0 \\n                 7.1 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.0 \\n                 3.7 \\n                 3.7 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.1 \\n                 7.4 \\n                 33.3 \\n                 14.3 \\n                 66.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.0 \\n                 3.7 \\n                 20.0 \\n                 7.1 \\n                 50.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.1 \\n                 3.7 \\n                 25.0 \\n                 7.1 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.0 \\n                 3.7 \\n                 3.7 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2023.0 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5327.0 \\n                 3.7 \\n                 3.8 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5328.0 \\n                 3.7 \\n                 3.7 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.0 \\n                 51.9 \\n                 51.9 \\n                 50.0 \\n                 50.0 \\n                 53.8 \\n                 53.8 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.1 \\n                 44.4 \\n                 46.2 \\n                 42.9 \\n                 46.2 \\n                 46.2 \\n                 46.2 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.0 \\n                 33.3 \\n                 33.3 \\n                 42.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.1 \\n                 33.3 \\n                 33.3 \\n                 42.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.0 \\n                 51.9 \\n                 51.9 \\n                 50.0 \\n                 50.0 \\n                 53.8 \\n                 53.8 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.1 \\n                 44.4 \\n                 46.2 \\n                 42.9 \\n                 46.2 \\n                 46.2 \\n                 46.2 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.0 \\n                 33.3 \\n                 33.3 \\n                 42.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.1 \\n                 33.3 \\n                 33.3 \\n                 42.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.0 \\n                 51.9 \\n                 51.9 \\n                 50.0 \\n                 50.0 \\n                 53.8 \\n                 53.8 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.1 \\n                 44.4 \\n                 46.2 \\n                 42.9 \\n                 46.2 \\n                 46.2 \\n                 46.2 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.0 \\n                 33.3 \\n                 33.3 \\n                 2.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.1 \\n                 33.3 \\n                 33.3 \\n                 42.9 \\n                 42.9 \\n                 23.1 \\n                 23.1 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.0 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.1 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.0 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.1 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.0 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.1 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.0 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.1 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.0 \\n                 7.4 \\n                 8.0 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.1 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.0 \\n                 3.7 \\n                 3.7 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.0 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.1 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.0 \\n                 7.4 \\n                 8.0 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.1 \\n                 7.4 \\n                 7.4 \\n                 14.3 \\n                 14.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.0 \\n                 3.7 \\n                 3.7 \\n                 7.1 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1168.0 \\n                 3.7 \\n                 3.7 \\n                 0.0 \\n                 0.0 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1175.0 \\n                 3.7 \\n                 4.0 \\n                 7.1 \\n                 7.7 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.0 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.0 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.0 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.0 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.0 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.1 \\n                 7.4 \\n                 7.4 \\n                 7.1 \\n                 7.1 \\n                 7.7 \\n                 7.7 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Lung Cancer Chips \\n     For lung cancer two different chip designs were evaluated with overlapping sets of a total of 29 samples, comparing the expression patterns of lung cancer derived polyA+RNA to polyA+ RNA isolated from a pool of 12 normal lung tissues. For the Lung Array Chip all 29 samples (15 squamous cell carcinomas and 14 adenocarcinomas including 14 stage I and 15 stage II/III cancers) were analyzed and for the Multi-Cancer Array Chip a subset of 22 of these samples (10 squamous cell carcinomas, 12 adenocarcinomas) were assessed. \\n     The results for the statistically significant up-regulated genes on the Lung Array Chip are shown in Table 9. The results for the statistically significant up-regulated genes on the Multi-Cancer Array Chip are shown in Table 10. The first two columns of each table contain information about the sequence itself (DEX ID, Oligo Name), the next columns show the results obtained for all (“ALL”) lung cancer samples, squamous cell carcinomas (“SQ”), adenocarcinomas (“AD”), or cancers corresponding to stage I (“ST1”), or stages II and III (“ST2,3”). ‘% up’ indicates the percentage of all experiments in which up-regulation of at least 2-fold was observed (n=29 for Lung Array Chip, n=22 for Multi-Cancer Array Chip), ‘% valid up’ indicates the percentage of experiments with valid expression values in which up-regulation of at least 2-fold was observed. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 9 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Lng \\n                   \\n                 Lng \\n                   \\n                 Lng \\n                   \\n                 Lng \\n                   \\n                 Lng \\n               \\n               \\n                   \\n                   \\n                 Lng \\n                 ALL % \\n                 Lng \\n                 SQ % \\n                 Lng \\n                 AD % \\n                 Lng \\n                 ST1 % \\n                 Lng \\n                 ST2,3 % \\n               \\n               \\n                   \\n                   \\n                 ALL \\n                 valid \\n                 SQ \\n                 valid \\n                 AD \\n                 valid \\n                 ST1 \\n                 valid \\n                 ST2,3 \\n                 valid \\n               \\n               \\n                   \\n                 Oligo \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n                 % up \\n                 up \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 29 \\n                 n = 29 \\n                 n = 15 \\n                 n = 15 \\n                 n = 14 \\n                 n = 14 \\n                 n = 14 \\n                 n = 14 \\n                 n = 15 \\n                 n = 15 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 791.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 2720.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 2721.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 791.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 2720.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 2721.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 2688.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 2689.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 5313.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2006.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2022.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2032.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_042.nt.1 \\n                 889.0 \\n                 93.1 \\n                 93.1 \\n                 100.0 \\n                 100.0 \\n                 85.7 \\n                 85.7 \\n                 92.9 \\n                 92.9 \\n                 93.3 \\n                 93.3 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1009.0 \\n                 3.4 \\n                 3.4 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1010.0 \\n                 6.9 \\n                 6.9 \\n                 13.3 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 13.3 \\n                 13.3 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1011.0 \\n                 6.9 \\n                 6.9 \\n                 13.3 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 7.1 \\n                 7.1 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1169.0 \\n                 3.4 \\n                 3.4 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1009.0 \\n                 3.4 \\n                 3.4 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1010.0 \\n                 6.9 \\n                 6.9 \\n                 13.3 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 13.3 \\n                 13.3 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1011.0 \\n                 6.9 \\n                 6.9 \\n                 13.3 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 7.1 \\n                 7.1 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1169.0 \\n                 3.4 \\n                 3.4 \\n                 6.7 \\n                 6.7 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 6.7 \\n                 6.7 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1174.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 7815.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 1582.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 1583.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 2661.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3143.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3160.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3161.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3164.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3165.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 1582.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 1583.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 2661.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3160.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3161.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3164.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3165.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 7612.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 7613.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 7612.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 7613.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 10 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Lng \\n                   \\n                 Lng \\n                   \\n                 Lng \\n               \\n               \\n                   \\n                   \\n                 Lng \\n                 Multi- \\n                 Lng \\n                 Multi- \\n                 Lng \\n                 Multi- \\n               \\n               \\n                   \\n                   \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n                 Multi- \\n                 Cancer \\n               \\n               \\n                   \\n                   \\n                 Cancer \\n                 ALL \\n                 Cancer \\n                 SQ \\n                 cancer \\n                 AD \\n               \\n               \\n                   \\n                 Oligo \\n                 ALL % up \\n                 % valid \\n                 SQ % up \\n                 % valid \\n                 AD % up \\n                 % valid \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 n = 22 \\n                 up n = 22 \\n                 n = 10 \\n                 up n = 10 \\n                 n = 12 \\n                 up n = 12 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_002.nt.1 \\n                 79699.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.0 \\n                 9.1 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 16.7 \\n                 18.2 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317.1 \\n                 9.1 \\n                 13.3 \\n                 0.0 \\n                 0.0 \\n                 16.7 \\n                 18.2 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.0 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334.1 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.0 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335.1 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.0 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334.1 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.0 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335.1 \\n                 13.6 \\n                 13.6 \\n                 10.0 \\n                 10.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.0 \\n                 4.5 \\n                 4.5 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482.1 \\n                 4.5 \\n                 4.5 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2023.0 \\n                 4.5 \\n                 4.5 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5327.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5328.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519.1 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.0 \\n                 40.9 \\n                 40.9 \\n                 30.0 \\n                 30.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520.1 \\n                 45.5 \\n                 45.5 \\n                 40.0 \\n                 40.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519.1 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.0 \\n                 40.9 \\n                 40.9 \\n                 30.0 \\n                 30.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520.1 \\n                 45.5 \\n                 45.5 \\n                 40.0 \\n                 40.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.0 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519.1 \\n                 50.0 \\n                 50.0 \\n                 40.0 \\n                 40.0 \\n                 58.3 \\n                 58.3 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.0 \\n                 40.9 \\n                 40.9 \\n                 30.0 \\n                 30.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520.1 \\n                 45.5 \\n                 45.5 \\n                 40.0 \\n                 40.0 \\n                 50.0 \\n                 50.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.0 \\n                 4.5 \\n                 4.5 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.1 \\n                 9.1 \\n                 9.1 \\n                 0.0 \\n                 0.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.0 \\n                 4.5 \\n                 4.5 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.1 \\n                 9.1 \\n                 9.1 \\n                 0.0 \\n                 0.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.0 \\n                 4.5 \\n                 4.5 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.1 \\n                 9.1 \\n                 9.1 \\n                 0.0 \\n                 0.0 \\n                 16.7 \\n                 16.7 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.0 \\n                 4.5 \\n                 5.0 \\n                 10.0 \\n                 12.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764.1 \\n                 4.5 \\n                 5.0 \\n                 10.0 \\n                 11.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.0 \\n                 4.5 \\n                 5.0 \\n                 10.0 \\n                 12.5 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105764.1 \\n                 4.5 \\n                 5.0 \\n                 10.0 \\n                 11.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1168.0 \\n                 4.5 \\n                 4.5 \\n                 10.0 \\n                 10.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1175.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.1 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008.2 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.0 \\n                 31.8 \\n                 31.8 \\n                 30.0 \\n                 30.0 \\n                 33.3 \\n                 33.3 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508.1 \\n                 31.8 \\n                 31.8 \\n                 20.0 \\n                 20.0 \\n                 41.7 \\n                 41.7 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.0 \\n                 31.8 \\n                 31.8 \\n                 30.0 \\n                 30.0 \\n                 33.3 \\n                 33.3 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508.1 \\n                 31.8 \\n                 31.8 \\n                 20.0 \\n                 20.0 \\n                 41.7 \\n                 41.7 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.0 \\n                 31.8 \\n                 31.8 \\n                 30.0 \\n                 30.0 \\n                 33.3 \\n                 33.3 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508.1 \\n                 31.8 \\n                 31.8 \\n                 20.0 \\n                 20.0 \\n                 41.7 \\n                 41.7 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.0 \\n                 31.8 \\n                 31.8 \\n                 30.0 \\n                 30.0 \\n                 33.3 \\n                 33.3 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508.1 \\n                 31.8 \\n                 31.8 \\n                 20.0 \\n                 20.0 \\n                 41.7 \\n                 41.7 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.0 \\n                 31.8 \\n                 31.8 \\n                 30.0 \\n                 30.0 \\n                 33.3 \\n                 33.3 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508.1 \\n                 31.8 \\n                 31.8 \\n                 20.0 \\n                 20.0 \\n                 41.7 \\n                 41.7 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Prostate Cancer \\n     For prostate cancer three different chip designs were evaluated with overlapping sets of a total of 29 samples, comparing the expression patterns of prostate cancer or benign disease derived total RNA to total RNA isolated from a pool of 35 normal prostate tissues. For the Prostate 1 Array and Prostate 2 Array Chips all 29 samples (17 prostate cancer samples, 12 non-malignant disease samples) were analyzed. For the Multi-Cancer Array Chip a subset of 28 of these samples (16 prostate cancer samples, 12 non-malignant disease samples) were analyzed. \\n     The results for the statistically significant up-regulated genes on the Prostate 1 Array Chip and the Prostate 2 Array Chip are shown in Table 11. The results for the statistically significant up-regulated genes on the Multi-Cancer Array Chip are shown in Table 12. The first two columns of each table contain information about the sequence itself (DEX ID, Oligo Name), the next columns show the results obtained for prostate cancer samples (“CAN”) or non-malignant disease samples (“DIS”). ‘% up’ indicates the percentage of all experiments in which up-regulation of at least 2-fold was observed (n=29 for the Prostate 2 Array Chip and the Multi-Cancer Array Chip), ‘% valid up’ indicates the percentage of experiments with valid expression values in which up-regulation of at least 2-fold was observed. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 11 \\n               \\n               \\n                   \\n               \\n               \\n                   \\n                   \\n                   \\n                 Pro CAN \\n                   \\n                 Pro DIS \\n               \\n               \\n                   \\n                 Oligo \\n                 Pro CAN \\n                 % valid up \\n                 Pro DIS \\n                 % valid up \\n               \\n               \\n                 DEX ID \\n                 Name \\n                 % up n = 17 \\n                 n = 17 \\n                 % up n = 12 \\n                 n = 12 \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_010.nt.1 \\n                 28129.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 28129.02 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 28129.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 28129.02 \\n                 0.0 \\n                 0.0 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 8770.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 8770.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 8770.03 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 26867.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 26867.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32554.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32554.02 \\n                 5.9 \\n                 5.9 \\n                 8.3 \\n                 8.3 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32554.03 \\n                 5.9 \\n                 7.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32558.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32558.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32558.03 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23492.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23492.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23546.01 \\n                 5.9 \\n                 33.3 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23546.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24418.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24418.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24422.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24422.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 27965.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 27965.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 28535.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 28535.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23492.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23492.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23684.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23684.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 24418.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 24418.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 27965.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 27965.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 28535.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 28535.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23492.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23492.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 27965.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 27965.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 28535.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 28535.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378.02 \\n                 5.9 \\n                 11.1 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 33332.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 33332.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 33332.01 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 33332.02 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n                 0.0 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n                                         TABLE 12                           Pro Multi       Pro Multi-               Pro Multi-   Cancer CAN   Pro Multi-   Cancer DIS           Oligo   Cancer CAN   % valid up   Cancer DIS   % valid up       DEX ID   Name   % up n = 16   n = 16   % up n = 12   n = 12                                                        DEX0455_002.nt.1   79699.1   0.0   0.0   0.0   0.0       DEX0455_002.nt.1   79700.0   0.0   0.0   0.0   0.0       DEX0455_002.nt.1   79700.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   96339.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   96339.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   96340.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   96340.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105991.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105991.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105992.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105992.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105996.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.1   105996.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   96339.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   96339.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   96340.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   96340.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105991.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105991.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105992.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105992.1   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105996.0   0.0   0.0   0.0   0.0       DEX0455_004.nt.2   105996.1   0.0   0.0   0.0   0.0       DEX0455_011.nt.1   35317.0   0.0   0.0   0.0   0.0       DEX0455_011.nt.1   35317.1   0.0   0.0   0.0   0.0       DEX0455_012.nt.1   34334.0   17.6   18.8   0.0   0.0       DEX0455_012.nt.1   34334.1   23.5   25.0   0.0   0.0       DEX0455_012.nt.1   34335.0   23.5   26.7   8.3   8.3       DEX0455_012.nt.1   34335.1   17.6   18.8   0.0   0.0       DEX0455_012.nt.2   34334.0   17.6   18.8   0.0   0.0       DEX0455_012.nt.2   34334.1   23.5   25.0   0.0   0.0       DEX0455_012.nt.2   34335.0   23.5   26.7   8.3   8.3       DEX0455_012.nt.2   34335.1   17.6   18.8   0.0   0.0       DEX0455_017.nt.1   36482.0   5.9   6.7   0.0   0.0       DEX0455_017.nt.1   36482.1   5.9   6.2   0.0   0.0       DEX0455_033.nt.1   2023.0   0.0   0.0   0.0   0.0       DEX0455_033.nt.1   5327.0   0.0   0.0   0.0   0.0       DEX0455_033.nt.1   5328.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.1   78519.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.1   78519.1   0.0   0.0   0.0   0.0       DEX0455_035.nt.1   78520.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.1   78520.1   0.0   0.0   0.0   0.0       DEX0455_035.nt.2   78519.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.2   78519.1   0.0   0.0   0.0   0.0       DEX0455_035.nt.2   78520.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.2   78520.1   0.0   0.0   0.0   0.0       DEX0455_035.nt.3   78519.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.3   78519.1   0.0   0.0   0.0   0.0       DEX0455_035.nt.3   78520.0   0.0   0.0   0.0   0.0       DEX0455_035.nt.3   78520.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.1   23542.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.1   23542.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.1   23543.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.1   23543.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.2   23542.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.2   23542.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.2   23543.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.2   23543.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.3   23542.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.3   23542.1   0.0   0.0   0.0   0.0       DEX0455_038.nt.3   23543.0   0.0   0.0   0.0   0.0       DEX0455_038.nt.3   23543.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   96212.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   96212.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105764.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105764.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105767.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105767.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105768.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.1   105768.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   96212.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   96212.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105764.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105764.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105767.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105767.1   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105768.0   0.0   0.0   0.0   0.0       DEX0455_047.nt.2   105768.1   0.0   0.0   0.0   0.0       DEX0455_048.nt.1   1168.0   0.0   0.0   8.3   8.3       DEX0455_048.nt.2   1175.0   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   23378.0   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   23378.1   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   23379.0   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   23379.1   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42007.0   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42007.1   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42007.2   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42008.0   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42008.1   0.0   0.0   0.0   0.0       DEX0455_050.nt.1   42008.2   0.0   0.0   0.0   0.0       DEX0455_061.nt.1   78508.0   0.0   0.0   0.0   0.0       DEX0455_061.nt.1   78508.1   0.0   0.0   0.0   0.0       DEX0455_061.nt.2   78508.0   0.0   0.0   0.0   0.0       DEX0455_061.nt.2   78508.1   0.0   0.0   0.0   0.0       DEX0455_061.nt.3   78508.0   0.0   0.0   0.0   0.0       DEX0455_061.nt.3   78508.1   0.0   0.0   0.0   0.0       DEX0455_061.nt.4   78508.0   0.0   0.0   0.0   0.0       DEX0455_061.nt.4   78508.1   0.0   0.0   0.0   0.0       DEX0455_061.nt.5   78508.0   0.0   0.0   0.0   0.0       DEX0455_061.nt.5   78508.1   0.0   0.0   0.0   0.0                    \\nSEQ ID NO: 1-128 was up-regulated on various tissue microarrays. Accordingly, nucleotide SEQ ID NO: 1-128 or the encoded protein SEQ ID NO: 129-295 may be used as a cancer therapeutic and/or diagnostic target for the tissues in which expression is shown.\\n \\n     The following table lists the location (Oligo Location) where the microarray oligos (Oligo ID) map on the transcripts (DEX ID) of the present invention. Each Oligo ID may have been printed multiple times on a single chip as replicates. The Oligo Name is an exemplary replicate (e.g. 1000.01) for the Oligo ID (e.g. 1000), and data from other replicates (e.g. 1000.02, 1000.03) may be reported. Additionally, the Array (Chip Name) that each oligo and oligo replicates were printed on is included. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 DEX NT ID \\n                 Oligo ID \\n                 Oligo Name \\n                 Chip Name \\n                 Oligo Location \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DEX0455_001.nt.1 \\n                 34930 \\n                 34930.01 \\n                 Ovarian array \\n                 4736-4795 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21577 \\n                 21577.02 \\n                 Ovarian array \\n                 198-257 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79699 \\n                 79699.0 \\n                 Multi-Cancer array \\n                 1430-1489 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 21553 \\n                 21553.01 \\n                 Ovarian array \\n                 513-572 \\n               \\n               \\n                 DEX0455_002.nt.1 \\n                 79700 \\n                 79700.0 \\n                 Multi-Cancer array \\n                 1429-1488 \\n               \\n               \\n                 DEX0455_003.nt.1 \\n                 17466 \\n                 17466.02 \\n                 Ovarian array \\n                 1075-1134 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96340 \\n                 96340.0 \\n                 Multi-Cancer array \\n                 6807-6866 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 96339 \\n                 96339.0 \\n                 Multi-Cancer array \\n                 6906-6965 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105991 \\n                 105991.0 \\n                 Multi-Cancer array \\n                 6906-6965 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105992 \\n                 105992.0 \\n                 Multi-Cancer array \\n                 6807-6866 \\n               \\n               \\n                 DEX0455_004.nt.1 \\n                 105996 \\n                 105996.0 \\n                 Multi-Cancer array \\n                 8462-8521 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96340 \\n                 96340.0 \\n                 Multi-Cancer array \\n                 5651-5710 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105991 \\n                 105991.0 \\n                 Multi-Cancer array \\n                 5750-5809 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105992 \\n                 105992.0 \\n                 Multi-Cancer array \\n                 5651-5710 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 105996 \\n                 105996.0 \\n                 Multi-Cancer array \\n                 7306-7365 \\n               \\n               \\n                 DEX0455_004.nt.2 \\n                 96339 \\n                 96339.0 \\n                 Multi-Cancer array \\n                 5750-5809 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 24874 \\n                 24874.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_005.nt.1 \\n                 20619 \\n                 20619.02 \\n                 Ovarian array \\n                 472-531 \\n               \\n               \\n                 DEX0455_005.nt.2 \\n                 24874 \\n                 24874.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_007.nt.1 \\n                 30109 \\n                 30109.01 \\n                 Ovarian array \\n                 \\u2002982-1041 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 22387 \\n                 22387.01 \\n                 Ovarian array \\n                 1666-1725 \\n               \\n               \\n                 DEX0455_008.nt.1 \\n                 18508 \\n                 18508.02 \\n                 Ovarian array \\n                 1193-1252 \\n               \\n               \\n                 DEX0455_009.nt.1 \\n                 9720 \\n                 9720.02 \\n                 Ovarian array \\n                 1745-1804 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 2721 \\n                 2721.0 \\n                 Lung array \\n                 501-560 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 37415 \\n                 37415.0 \\n                 Colon array \\n                 1040-1099 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 32151 \\n                 32151.0 \\n                 Breast array \\n                 748-807 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 2720 \\n                 2720.0 \\n                 Lung array \\n                 542-601 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 21675 \\n                 21675.02 \\n                 Ovarian array \\n                 \\u2002965-1024 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 20627 \\n                 20627.02 \\n                 Ovarian array \\n                 250-309 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 28129 \\n                 28129.02 \\n                 Prostate1 array \\n                 \\u2002964-1023 \\n               \\n               \\n                 DEX0455_010.nt.1 \\n                 791 \\n                 791.0 \\n                 Lung array \\n                 1045-1104 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 2720 \\n                 2720.0 \\n                 Lung array \\n                 379-438 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 32151 \\n                 32151.0 \\n                 Breast array \\n                 585-644 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 28129 \\n                 28129.02 \\n                 Prostate1 array \\n                 801-860 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 37415 \\n                 37415.0 \\n                 Colon array \\n                 877-936 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 21675 \\n                 21675.02 \\n                 Ovarian array \\n                 802-861 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 791 \\n                 791.0 \\n                 Lung array \\n                 882-941 \\n               \\n               \\n                 DEX0455_010.nt.2 \\n                 2721 \\n                 2721.0 \\n                 Lung array \\n                 338-397 \\n               \\n               \\n                 DEX0455_011.nt.1 \\n                 35317 \\n                 35317.0 \\n                 Colon array \\n                 398-457 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34368 \\n                 34368.0 \\n                 Colon array \\n                 2484-2543 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34369 \\n                 34369.0 \\n                 Colon array \\n                 2441-2500 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34334 \\n                 34334.0 \\n                 Colon array \\n                 3108-3167 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34343 \\n                 34343.0 \\n                 Colon array \\n                 472-531 \\n               \\n               \\n                 DEX0455_012.nt.1 \\n                 34335 \\n                 34335.0 \\n                 Colon array \\n                 3022-3081 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34334 \\n                 34334.0 \\n                 Colon array \\n                 2527-2586 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34343 \\n                 34343.0 \\n                 Colon array \\n                 472-531 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34369 \\n                 34369.0 \\n                 Colon array \\n                 1860-1919 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34335 \\n                 34335.0 \\n                 Colon array \\n                 2441-2500 \\n               \\n               \\n                 DEX0455_012.nt.2 \\n                 34368 \\n                 34368.0 \\n                 Colon array \\n                 1903-1962 \\n               \\n               \\n                 DEX0455_013.nt.1 \\n                 9838 \\n                 9838.02 \\n                 Ovarian array \\n                 1304-1363 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 10624 \\n                 10624.02 \\n                 Ovarian array \\n                 1832-1891 \\n               \\n               \\n                 DEX0455_014.nt.1 \\n                 14604 \\n                 14604.01 \\n                 Ovarian array \\n                 925-984 \\n               \\n               \\n                 DEX0455_015.nt.1 \\n                 19518 \\n                 19518.01 \\n                 Ovarian array \\n                 277-336 \\n               \\n               \\n                 DEX0455_016.nt.1 \\n                 23734 \\n                 23734.02 \\n                 Ovarian array \\n                 531-590 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 28221 \\n                 28221.0 \\n                 Breast array \\n                 679-738 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 21032 \\n                 21032.0 \\n                 Colon array \\n                 314-373 \\n               \\n               \\n                 DEX0455_017.nt.1 \\n                 36482 \\n                 36482.0 \\n                 Multi-Cancer array \\n                 314-373 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21575 \\n                 21575.01 \\n                 Ovarian array \\n                 1516-1575 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21571 \\n                 21571.02 \\n                 Ovarian array \\n                 623-682 \\n               \\n               \\n                 DEX0455_018.nt.1 \\n                 21609 \\n                 21609.02 \\n                 Ovarian array \\n                 933-992 \\n               \\n               \\n                 DEX0455_018.nt.2 \\n                 21575 \\n                 21575.01 \\n                 Ovarian array \\n                 2287-2346 \\n               \\n               \\n                 DEX0455_019.nt.1 \\n                 20669 \\n                 20669.01 \\n                 Ovarian array \\n                 615-674 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 23780 \\n                 23780.01 \\n                 Ovarian array \\n                 517-576 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21469 \\n                 21469.02 \\n                 Ovarian array \\n                 430-489 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21433 \\n                 21433.01 \\n                 Ovarian array \\n                 518-577 \\n               \\n               \\n                 DEX0455_021.nt.1 \\n                 21475 \\n                 21475.01 \\n                 Ovarian array \\n                 517-576 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21469 \\n                 21469.02 \\n                 Ovarian array \\n                 1528-1587 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21475 \\n                 21475.01 \\n                 Ovarian array \\n                 1615-1674 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 21433 \\n                 21433.01 \\n                 Ovarian array \\n                 1616-1675 \\n               \\n               \\n                 DEX0455_021.nt.2 \\n                 23780 \\n                 23780.01 \\n                 Ovarian array \\n                 1615-1674 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21433 \\n                 21433.01 \\n                 Ovarian array \\n                 1859-1918 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21475 \\n                 21475.01 \\n                 Ovarian array \\n                 1858-1917 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 21469 \\n                 21469.02 \\n                 Ovarian array \\n                 1771-1830 \\n               \\n               \\n                 DEX0455_021.nt.3 \\n                 23780 \\n                 23780.01 \\n                 Ovarian array \\n                 1858-1917 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21469 \\n                 21469.02 \\n                 Ovarian array \\n                 1914-1973 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21475 \\n                 21475.01 \\n                 Ovarian array \\n                 2001-2060 \\n               \\n               \\n                 DEX0455_021.nt.4 \\n                 21433 \\n                 21433.01 \\n                 Ovarian array \\n                 2002-2061 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 9920 \\n                 9920.02 \\n                 Ovarian array \\n                 1022-1081 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20311 \\n                 20311.01 \\n                 Ovarian array \\n                 718-777 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20299 \\n                 20299.01 \\n                 Ovarian array \\n                 529-588 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 23280 \\n                 23280.0 \\n                 Breast array \\n                 427-486 \\n               \\n               \\n                 DEX0455_022.nt.1 \\n                 20317 \\n                 20317.02 \\n                 Ovarian array \\n                 718-777 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 9920 \\n                 9920.02 \\n                 Ovarian array \\n                 1016-1075 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20311 \\n                 20311.01 \\n                 Ovarian array \\n                 712-771 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20317 \\n                 20317.02 \\n                 Ovarian array \\n                 712-771 \\n               \\n               \\n                 DEX0455_022.nt.2 \\n                 20299 \\n                 20299.01 \\n                 Ovarian array \\n                 552-611 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 9920 \\n                 9920.02 \\n                 Ovarian array \\n                 613-672 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20317 \\n                 20317.02 \\n                 Ovarian array \\n                 309-368 \\n               \\n               \\n                 DEX0455_022.nt.3 \\n                 20311 \\n                 20311.01 \\n                 Ovarian array \\n                 309-368 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16374 \\n                 16374.02 \\n                 Ovarian array \\n                 2119-2178 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 8770 \\n                 8770.03 \\n                 Prostate2 array \\n                 1897-1956 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16378 \\n                 16378.01 \\n                 Ovarian array \\n                 937-996 \\n               \\n               \\n                 DEX0455_023.nt.1 \\n                 16187 \\n                 16187.01 \\n                 Ovarian array \\n                 666-725 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21507 \\n                 21507.01 \\n                 Ovarian array \\n                 2357-2416 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21487 \\n                 21487.01 \\n                 Ovarian array \\n                 796-855 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 12149 \\n                 12149.01 \\n                 Ovarian array \\n                 2439-2498 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 21547 \\n                 21547.02 \\n                 Ovarian array \\n                 1555-1614 \\n               \\n               \\n                 DEX0455_024.nt.1 \\n                 17957 \\n                 17957.0 \\n                 Colon array \\n                 2002-2061 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21507 \\n                 21507.01 \\n                 Ovarian array \\n                 1790-1849 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 12149 \\n                 12149.01 \\n                 Ovarian array \\n                 1872-1931 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 21547 \\n                 21547.02 \\n                 Ovarian array \\n                 \\u2002988-1047 \\n               \\n               \\n                 DEX0455_024.nt.2 \\n                 17957 \\n                 17957.0 \\n                 Colon array \\n                 1435-1494 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 12167 \\n                 12167.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16964 \\n                 16964.02 \\n                 Ovarian array \\n                 3509-3568 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16956 \\n                 16956.02 \\n                 Ovarian array \\n                 3533-3592 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 16958 \\n                 16958.01 \\n                 Ovarian array \\n                 808-867 \\n               \\n               \\n                 DEX0455_025.nt.1 \\n                 19010 \\n                 19010.01 \\n                 Ovarian array \\n                 1260-1319 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 12167 \\n                 12167.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16964 \\n                 16964.02 \\n                 Ovarian array \\n                 2465-2524 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16956 \\n                 16956.02 \\n                 Ovarian array \\n                 2489-2548 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 16958 \\n                 16958.01 \\n                 Ovarian array \\n                 808-867 \\n               \\n               \\n                 DEX0455_025.nt.2 \\n                 19010 \\n                 19010.01 \\n                 Ovarian array \\n                 1260-1319 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 12167 \\n                 12167.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 19010 \\n                 19010.01 \\n                 Ovarian array \\n                 1260-1319 \\n               \\n               \\n                 DEX0455_025.nt.3 \\n                 16958 \\n                 16958.01 \\n                 Ovarian array \\n                 808-867 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 19010 \\n                 19010.01 \\n                 Ovarian array \\n                 1260-1319 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16956 \\n                 16956.02 \\n                 Ovarian array \\n                 2167-2226 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 16964 \\n                 16964.02 \\n                 Ovarian array \\n                 2143-2202 \\n               \\n               \\n                 DEX0455_025.nt.4 \\n                 12167 \\n                 12167.01 \\n                 Ovarian array \\n                 475-534 \\n               \\n               \\n                 DEX0455_027.nt.1 \\n                 21549 \\n                 21549.01 \\n                 Ovarian array \\n                 1483-1542 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 41120 \\n                 41120.0 \\n                 Colon array \\n                 477-536 \\n               \\n               \\n                 DEX0455_028.nt.1 \\n                 30821 \\n                 30821.0 \\n                 Colon array \\n                 673-732 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41151 \\n                 41151.0 \\n                 Colon array \\n                 2429-2488 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 22113 \\n                 22113.01 \\n                 Ovarian array \\n                 3222-3281 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30869 \\n                 30869.0 \\n                 Colon array \\n                 5572-5631 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41120 \\n                 41120.0 \\n                 Colon array \\n                 1984-2043 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23386 \\n                 23386.01 \\n                 Ovarian array \\n                 2429-2488 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30820 \\n                 30820.0 \\n                 Colon array \\n                 2388-2447 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41117 \\n                 41117.0 \\n                 Colon array \\n                 2296-2355 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30821 \\n                 30821.0 \\n                 Colon array \\n                 2348-2407 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 23400 \\n                 23400.02 \\n                 Ovarian array \\n                 2296-2355 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 41152 \\n                 41152.0 \\n                 Colon array \\n                 2372-2431 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17430 \\n                 17430.02 \\n                 Ovarian array \\n                 2388-2447 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 30824 \\n                 30824.0 \\n                 Colon array \\n                 5798-5857 \\n               \\n               \\n                 DEX0455_029.nt.1 \\n                 17448 \\n                 17448.01 \\n                 Ovarian array \\n                 5798-5857 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41120 \\n                 41120.0 \\n                 Colon array \\n                 2412-2471 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17430 \\n                 17430.02 \\n                 Ovarian array \\n                 2816-2875 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23386 \\n                 23386.01 \\n                 Ovarian array \\n                 2857-2916 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30824 \\n                 30824.0 \\n                 Colon array \\n                 5101-5160 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17424 \\n                 17424.01 \\n                 Ovarian array \\n                 4880-4939 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30922 \\n                 30922.0 \\n                 Colon array \\n                 4880-4939 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 23400 \\n                 23400.02 \\n                 Ovarian array \\n                 2724-2783 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41152 \\n                 41152.0 \\n                 Colon array \\n                 2800-2859 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30820 \\n                 30820.0 \\n                 Colon array \\n                 2816-2875 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 22113 \\n                 22113.01 \\n                 Ovarian array \\n                 3650-3709 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41117 \\n                 41117.0 \\n                 Colon array \\n                 2724-2783 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 41151 \\n                 41151.0 \\n                 Colon array \\n                 2857-2916 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 30821 \\n                 30821.0 \\n                 Colon array \\n                 2776-2835 \\n               \\n               \\n                 DEX0455_029.nt.2 \\n                 17448 \\n                 17448.01 \\n                 Ovarian array \\n                 5101-5160 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17204 \\n                 17204.02 \\n                 Ovarian array \\n                 1225-1284 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17262 \\n                 17262.02 \\n                 Ovarian array \\n                 1011-1070 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 17278 \\n                 17278.02 \\n                 Ovarian array \\n                 \\u2002991-1050 \\n               \\n               \\n                 DEX0455_030.nt.1 \\n                 11613 \\n                 11613.01 \\n                 Ovarian array \\n                 1011-1070 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17274 \\n                 17274.02 \\n                 Ovarian array \\n                 696-755 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17204 \\n                 17204.02 \\n                 Ovarian array \\n                 \\u2002984-1043 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17278 \\n                 17278.02 \\n                 Ovarian array \\n                 713-772 \\n               \\n               \\n                 DEX0455_030.nt.2 \\n                 17262 \\n                 17262.02 \\n                 Ovarian array \\n                 733-792 \\n               \\n               \\n                 DEX0455_031.nt.1 \\n                 20773 \\n                 20773.02 \\n                 Ovarian array \\n                 2724-2783 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 2688 \\n                 2688.0 \\n                 Lung array \\n                 \\u2002952-1011 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 11585 \\n                 11585.01 \\n                 Ovarian array \\n                 1342-1401 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 2689 \\n                 2689.0 \\n                 Lung array \\n                 910-969 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 18556 \\n                 18556.02 \\n                 Ovarian array \\n                 \\u2002952-1011 \\n               \\n               \\n                 DEX0455_032.nt.1 \\n                 5313 \\n                 5313.0 \\n                 Lung array \\n                 1342-1401 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5328 \\n                 5328.0 \\n                 Multi-Cancer array \\n                 402-461 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2006 \\n                 2006.0 \\n                 Lung array \\n                 402-461 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2022 \\n                 2022.0 \\n                 Lung array \\n                 482-541 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2032 \\n                 2032.0 \\n                 Lung array \\n                 290-349 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2007 \\n                 2007.0 \\n                 Lung array \\n                 361-420 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 5327 \\n                 5327.0 \\n                 Multi-Cancer array \\n                 442-501 \\n               \\n               \\n                 DEX0455_033.nt.1 \\n                 2023 \\n                 2023.0 \\n                 Multi-Cancer array \\n                 442-501 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 10722 \\n                 10722.02 \\n                 Ovarian array \\n                 2454-2513 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32554 \\n                 32554.02 \\n                 Prostate2 array \\n                 1815-1874 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21421 \\n                 21421.02 \\n                 Ovarian array \\n                 1815-1874 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 32558 \\n                 32558.01 \\n                 Prostate2 array \\n                 1053-1112 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 16423 \\n                 16423.0 \\n                 Colon array \\n                 885-944 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 21401 \\n                 21401.02 \\n                 Ovarian array \\n                 1053-1112 \\n               \\n               \\n                 DEX0455_034.nt.1 \\n                 26867 \\n                 26867.01 \\n                 Prostate1 array \\n                 2454-2513 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78519 \\n                 78519.0 \\n                 Multi-Cancer array \\n                 923-982 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 78520 \\n                 78520.0 \\n                 Multi-Cancer array \\n                 857-916 \\n               \\n               \\n                 DEX0455_035.nt.1 \\n                 103385 \\n                 103385.01 \\n                 Ovarian array \\n                 926-985 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78519 \\n                 78519.0 \\n                 Multi-Cancer array \\n                 1152-1211 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 103385 \\n                 103385.01 \\n                 Ovarian array \\n                 1155-1214 \\n               \\n               \\n                 DEX0455_035.nt.2 \\n                 78520 \\n                 78520.0 \\n                 Multi-Cancer array \\n                 1086-1145 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 103385 \\n                 103385.01 \\n                 Ovarian array \\n                 1034-1093 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78519 \\n                 78519.0 \\n                 Multi-Cancer array \\n                 1031-1090 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 78520 \\n                 78520.0 \\n                 Multi-Cancer array \\n                 \\u2002965-1024 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21144 \\n                 21144.0 \\n                 Breast array \\n                 126-185 \\n               \\n               \\n                 DEX0455_035.nt.3 \\n                 21143 \\n                 21143.0 \\n                 Breast array \\n                 212-271 \\n               \\n               \\n                 DEX0455_036.nt.1 \\n                 92327 \\n                 92327.01 \\n                 Ovarian array \\n                 177-236 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17490 \\n                 17490.01 \\n                 Ovarian array \\n                 894-953 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 11575 \\n                 11575.01 \\n                 Ovarian array \\n                 892-951 \\n               \\n               \\n                 DEX0455_037.nt.1 \\n                 17486 \\n                 17486.01 \\n                 Ovarian array \\n                 887-946 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 17490 \\n                 17490.01 \\n                 Ovarian array \\n                 1459-1518 \\n               \\n               \\n                 DEX0455_037.nt.2 \\n                 11575 \\n                 11575.01 \\n                 Ovarian array \\n                 1457-1516 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17490 \\n                 17490.01 \\n                 Ovarian array \\n                 2399-2458 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 11575 \\n                 11575.01 \\n                 Ovarian array \\n                 2397-2456 \\n               \\n               \\n                 DEX0455_037.nt.3 \\n                 17486 \\n                 17486.01 \\n                 Ovarian array \\n                 2392-2451 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17490 \\n                 17490.01 \\n                 Ovarian array \\n                 515-574 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 17486 \\n                 17486.01 \\n                 Ovarian array \\n                 508-567 \\n               \\n               \\n                 DEX0455_037.nt.4 \\n                 11575 \\n                 11575.01 \\n                 Ovarian array \\n                 513-572 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17486 \\n                 17486.01 \\n                 Ovarian array \\n                 571-630 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 17490 \\n                 17490.01 \\n                 Ovarian array \\n                 578-637 \\n               \\n               \\n                 DEX0455_037.nt.5 \\n                 11575 \\n                 11575.01 \\n                 Ovarian array \\n                 576-635 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23543 \\n                 23543.0 \\n                 Multi-Cancer array \\n                 5011-5070 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23492 \\n                 23492.02 \\n                 Prostate1 array \\n                 5433-5492 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23546 \\n                 23546.01 \\n                 Prostate1 array \\n                 3874-3933 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24422 \\n                 24422.01 \\n                 Prostate1 array \\n                 3874-3933 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 23542 \\n                 23542.0 \\n                 Multi-Cancer array \\n                 5118-5177 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 24418 \\n                 24418.01 \\n                 Prostate1 array \\n                 2859-2918 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 27965 \\n                 27965.01 \\n                 Prostate1 array \\n                 1956-2015 \\n               \\n               \\n                 DEX0455_038.nt.1 \\n                 28535 \\n                 28535.01 \\n                 Prostate1 array \\n                 5154-5213 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 27965 \\n                 27965.01 \\n                 Prostate1 array \\n                 1956-2015 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23543 \\n                 23543.0 \\n                 Multi-Cancer array \\n                 4443-4502 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 28535 \\n                 28535.01 \\n                 Prostate1 array \\n                 4586-4645 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23492 \\n                 23492.02 \\n                 Prostate1 array \\n                 4865-4924 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 24418 \\n                 24418.01 \\n                 Prostate1 array \\n                 2859-2918 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23542 \\n                 23542.0 \\n                 Multi-Cancer array \\n                 4550-4609 \\n               \\n               \\n                 DEX0455_038.nt.2 \\n                 23684 \\n                 23684.02 \\n                 Prostate1 array \\n                 3719-3778 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23543 \\n                 23543.0 \\n                 Multi-Cancer array \\n                 2528-2587 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23492 \\n                 23492.02 \\n                 Prostate1 array \\n                 2950-3009 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 23542 \\n                 23542.0 \\n                 Multi-Cancer array \\n                 2635-2694 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 27965 \\n                 27965.01 \\n                 Prostate1 array \\n                 1693-1752 \\n               \\n               \\n                 DEX0455_038.nt.3 \\n                 28535 \\n                 28535.01 \\n                 Prostate1 array \\n                 2671-2730 \\n               \\n               \\n                 DEX0455_039.nt.1 \\n                 21505 \\n                 21505.02 \\n                 Ovarian array \\n                 355-414 \\n               \\n               \\n                 DEX0455_039.nt.2 \\n                 11527 \\n                 11527.01 \\n                 Ovarian array \\n                 467-526 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21489 \\n                 21489.02 \\n                 Ovarian array \\n                 281-340 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21501 \\n                 21501.02 \\n                 Ovarian array \\n                 772-831 \\n               \\n               \\n                 DEX0455_040.nt.1 \\n                 21511 \\n                 21511.01 \\n                 Ovarian array \\n                 586-645 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21489 \\n                 21489.02 \\n                 Ovarian array \\n                 698-757 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21511 \\n                 21511.01 \\n                 Ovarian array \\n                 1003-1062 \\n               \\n               \\n                 DEX0455_040.nt.2 \\n                 21501 \\n                 21501.02 \\n                 Ovarian array \\n                 1189-1248 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16980 \\n                 16980.01 \\n                 Ovarian array \\n                 125-184 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 16998 \\n                 16998.0 \\n                 Breast array \\n                 125-184 \\n               \\n               \\n                 DEX0455_041.nt.1 \\n                 12155 \\n                 12155.01 \\n                 Ovarian array \\n                 309-368 \\n               \\n               \\n                 DEX0455_042.nt.1 \\n                 889 \\n                 889.0 \\n                 Lung array \\n                 346-405 \\n               \\n               \\n                 DEX0455_042.nt.1 \\n                 18214 \\n                 18214.02 \\n                 Ovarian array \\n                 346-405 \\n               \\n               \\n                 DEX0455_043.nt.1 \\n                 14656 \\n                 14656.02 \\n                 Ovarian array \\n                 463-522 \\n               \\n               \\n                 DEX0455_045.nt.1 \\n                 36013 \\n                 36013.01 \\n                 Ovarian array \\n                 382-441 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 17314 \\n                 17314.01 \\n                 Ovarian array \\n                 614-673 \\n               \\n               \\n                 DEX0455_046.nt.1 \\n                 19072 \\n                 19072.0 \\n                 Breast array \\n                 614-673 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105768 \\n                 105768.0 \\n                 Multi-Cancer array \\n                 3274-3333 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 96212 \\n                 96212.0 \\n                 Multi-Cancer array \\n                 2703-2762 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105767 \\n                 105767.0 \\n                 Multi-Cancer array \\n                 3314-3373 \\n               \\n               \\n                 DEX0455_047.nt.1 \\n                 105764 \\n                 105764.0 \\n                 Multi-Cancer array \\n                 2703-2762 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105768 \\n                 105768.0 \\n                 Multi-Cancer array \\n                 1478-1537 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 105767 \\n                 105767.0 \\n                 Multi-Cancer array \\n                 1518-1577 \\n               \\n               \\n                 DEX0455_047.nt.2 \\n                 96212 \\n                 96212.0 \\n                 Multi-Cancer array \\n                 907-966 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1169 \\n                 1169.0 \\n                 Lung array \\n                 175-234 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1011 \\n                 1011.0 \\n                 Lung array \\n                 202-261 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1009 \\n                 1009.0 \\n                 Lung array \\n                 192-251 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1010 \\n                 1010.0 \\n                 Lung array \\n                 242-301 \\n               \\n               \\n                 DEX0455_048.nt.1 \\n                 1168 \\n                 1168.0 \\n                 Multi-Cancer array \\n                 180-239 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1169 \\n                 1169.0 \\n                 Lung array \\n                 386-445 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1011 \\n                 1011.0 \\n                 Lung array \\n                 413-472 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1009 \\n                 1009.0 \\n                 Lung array \\n                 403-462 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1175 \\n                 1175.0 \\n                 Multi-Cancer array \\n                 254-313 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1168 \\n                 1168.0 \\n                 Multi-Cancer array \\n                 391-450 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1174 \\n                 1174.0 \\n                 Lung array \\n                 259-318 \\n               \\n               \\n                 DEX0455_048.nt.2 \\n                 1010 \\n                 1010.0 \\n                 Lung array \\n                 453-512 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 11511 \\n                 11511.02 \\n                 Ovarian array \\n                 2111-2170 \\n               \\n               \\n                 DEX0455_049.nt.1 \\n                 36902 \\n                 36902.0 \\n                 Colon array \\n                 928-987 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36901 \\n                 36901.0 \\n                 Colon array \\n                 621-680 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 11511 \\n                 11511.02 \\n                 Ovarian array \\n                 1528-1587 \\n               \\n               \\n                 DEX0455_049.nt.2 \\n                 36902 \\n                 36902.0 \\n                 Colon array \\n                 582-641 \\n               \\n               \\n                 DEX0455_049.nt.3 \\n                 36901 \\n                 36901.0 \\n                 Colon array \\n                 \\u2002967-1026 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 11511 \\n                 11511.02 \\n                 Ovarian array \\n                 2102-2161 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36902 \\n                 36902.0 \\n                 Colon array \\n                 1156-1215 \\n               \\n               \\n                 DEX0455_049.nt.4 \\n                 36901 \\n                 36901.0 \\n                 Colon array \\n                 1195-1254 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 36902 \\n                 36902.0 \\n                 Colon array \\n                 299-358 \\n               \\n               \\n                 DEX0455_049.nt.5 \\n                 11511 \\n                 11511.02 \\n                 Ovarian array \\n                 1245-1304 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 29736 \\n                 29736.0 \\n                 Breast array \\n                 171-230 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 7815 \\n                 7815.0 \\n                 Lung array \\n                 385-444 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23378 \\n                 23378.0 \\n                 Breast array \\n                 684-743 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42008 \\n                 42008.0 \\n                 Multi-Cancer array \\n                 329-388 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 42007 \\n                 42007.0 \\n                 Multi-Cancer array \\n                 329-388 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 22136 \\n                 22136.0 \\n                 Breast array \\n                 636-695 \\n               \\n               \\n                 DEX0455_050.nt.1 \\n                 23379 \\n                 23379.0 \\n                 Breast array \\n                 385-444 \\n               \\n               \\n                 DEX0455_052.nt.1 \\n                 91971 \\n                 91971.01 \\n                 Ovarian array \\n                 1686-1745 \\n               \\n               \\n                 DEX0455_054.nt.1 \\n                 19799 \\n                 19799.0 \\n                 Breast array \\n                 1918-1977 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 20541 \\n                 20541.01 \\n                 Ovarian array \\n                 1705-1764 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12731 \\n                 12731.0 \\n                 Breast array \\n                 1601-1660 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 12732 \\n                 12732.0 \\n                 Breast array \\n                 1395-1454 \\n               \\n               \\n                 DEX0455_055.nt.1 \\n                 11273 \\n                 11273.02 \\n                 Ovarian array \\n                 1815-1874 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 20541 \\n                 20541.01 \\n                 Ovarian array \\n                 1403-1462 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12731 \\n                 12731.0 \\n                 Breast array \\n                 1299-1358 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 12732 \\n                 12732.0 \\n                 Breast array \\n                 1136-1195 \\n               \\n               \\n                 DEX0455_055.nt.2 \\n                 11273 \\n                 11273.02 \\n                 Ovarian array \\n                 1513-1572 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12732 \\n                 12732.0 \\n                 Breast array \\n                 568-627 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 12731 \\n                 12731.0 \\n                 Breast array \\n                 731-790 \\n               \\n               \\n                 DEX0455_055.nt.3 \\n                 20541 \\n                 20541.01 \\n                 Ovarian array \\n                 835-894 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 23444 \\n                 23444.01 \\n                 Ovarian array \\n                 2588-2647 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3161 \\n                 3161.0 \\n                 Lung array \\n                 2547-2606 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3164 \\n                 3164.0 \\n                 Lung array \\n                 3317-3376 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3160 \\n                 3160.0 \\n                 Lung array \\n                 2588-2647 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3165 \\n                 3165.0 \\n                 Lung array \\n                 3277-3336 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 1583 \\n                 1583.0 \\n                 Lung array \\n                 3277-3336 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 18520 \\n                 18520.02 \\n                 Ovarian array \\n                 3317-3376 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 3143 \\n                 3143.0 \\n                 Lung array \\n                 3107-3166 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 1582 \\n                 1582.0 \\n                 Lung array \\n                 3317-3376 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 22734 \\n                 22734.02 \\n                 Ovarian array \\n                 3317-3376 \\n               \\n               \\n                 DEX0455_056.nt.1 \\n                 2661 \\n                 2661.0 \\n                 Lung array \\n                 3523-3582 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 23444 \\n                 23444.01 \\n                 Ovarian array \\n                 2559-2618 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 2661 \\n                 2661.0 \\n                 Lung array \\n                 3287-3346 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3161 \\n                 3161.0 \\n                 Lung array \\n                 2518-2577 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 1582 \\n                 1582.0 \\n                 Lung array \\n                 3081-3140 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 22734 \\n                 22734.02 \\n                 Ovarian array \\n                 3081-3140 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3160 \\n                 3160.0 \\n                 Lung array \\n                 2559-2618 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 18520 \\n                 18520.02 \\n                 Ovarian array \\n                 3081-3140 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3165 \\n                 3165.0 \\n                 Lung array \\n                 3041-3100 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 1583 \\n                 1583.0 \\n                 Lung array \\n                 3041-3100 \\n               \\n               \\n                 DEX0455_056.nt.2 \\n                 3164 \\n                 3164.0 \\n                 Lung array \\n                 3081-3140 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 7613 \\n                 7613.0 \\n                 Lung array \\n                 292-351 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 33332 \\n                 33332.02 \\n                 Prostate1 array \\n                 600-659 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 7612 \\n                 7612.0 \\n                 Lung array \\n                 381-440 \\n               \\n               \\n                 DEX0455_057.nt.1 \\n                 24524 \\n                 24524.02 \\n                 Ovarian array \\n                 600-659 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 7613 \\n                 7613.0 \\n                 Lung array \\n                 458-517 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 7612 \\n                 7612.0 \\n                 Lung array \\n                 547-606 \\n               \\n               \\n                 DEX0455_057.nt.2 \\n                 33332 \\n                 33332.02 \\n                 Prostate1 array \\n                 766-825 \\n               \\n               \\n                 DEX0455_058.nt.1 \\n                 14656 \\n                 14656.02 \\n                 Ovarian array \\n                 555-614 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17372 \\n                 17372.01 \\n                 Ovarian array \\n                 1778-1837 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 11469 \\n                 11469.02 \\n                 Ovarian array \\n                 424-483 \\n               \\n               \\n                 DEX0455_059.nt.1 \\n                 17370 \\n                 17370.01 \\n                 Ovarian array \\n                 \\u2002957-1016 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 17372 \\n                 17372.01 \\n                 Ovarian array \\n                 1489-1548 \\n               \\n               \\n                 DEX0455_059.nt.2 \\n                 11469 \\n                 11469.02 \\n                 Ovarian array \\n                 424-483 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 10372 \\n                 10372.01 \\n                 Ovarian array \\n                 1201-1260 \\n               \\n               \\n                 DEX0455_060.nt.1 \\n                 18582 \\n                 18582.01 \\n                 Ovarian array \\n                 672-731 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 78508 \\n                 78508.0 \\n                 Multi-Cancer array \\n                 3736-3795 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 103529 \\n                 103529.01 \\n                 Ovarian array \\n                 3740-3799 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19803 \\n                 19803.0 \\n                 Colon array \\n                 3736-3795 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 96523 \\n                 96523.02 \\n                 Ovarian array \\n                 3740-3799 \\n               \\n               \\n                 DEX0455_061.nt.1 \\n                 19804 \\n                 19804.0 \\n                 Colon array \\n                 3684-3743 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19803 \\n                 19803.0 \\n                 Colon array \\n                 4690-4749 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 78508 \\n                 78508.0 \\n                 Multi-Cancer array \\n                 4690-4749 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 19804 \\n                 19804.0 \\n                 Colon array \\n                 4638-4697 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 103529 \\n                 103529.01 \\n                 Ovarian array \\n                 4694-4753 \\n               \\n               \\n                 DEX0455_061.nt.2 \\n                 96523 \\n                 96523.02 \\n                 Ovarian array \\n                 4694-4753 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19803 \\n                 19803.0 \\n                 Colon array \\n                 4556-4615 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 78508 \\n                 78508.0 \\n                 Multi-Cancer array \\n                 4556-4615 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 103529 \\n                 103529.01 \\n                 Ovarian array \\n                 4560-4619 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 19804 \\n                 19804.0 \\n                 Colon array \\n                 4504-4563 \\n               \\n               \\n                 DEX0455_061.nt.3 \\n                 96523 \\n                 96523.02 \\n                 Ovarian array \\n                 4560-4619 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 103529 \\n                 103529.01 \\n                 Ovarian array \\n                 1702-1761 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19804 \\n                 19804.0 \\n                 Colon array \\n                 1646-1705 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 78508 \\n                 78508.0 \\n                 Multi-Cancer array \\n                 1698-1757 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 19803 \\n                 19803.0 \\n                 Colon array \\n                 1698-1757 \\n               \\n               \\n                 DEX0455_061.nt.4 \\n                 96523 \\n                 96523.02 \\n                 Ovarian array \\n                 1702-1761 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 78508 \\n                 78508.0 \\n                 Multi-Cancer array \\n                 2394-2453 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 103529 \\n                 103529.01 \\n                 Ovarian array \\n                 2398-2457 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19803 \\n                 19803.0 \\n                 Colon array \\n                 2394-2453 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 19804 \\n                 19804.0 \\n                 Colon array \\n                 2342-2401 \\n               \\n               \\n                 DEX0455_061.nt.5 \\n                 96523 \\n                 96523.02 \\n                 Ovarian array \\n                 2398-2457 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 18094 \\n                 18094.01 \\n                 Ovarian array \\n                 914-973 \\n               \\n               \\n                 DEX0455_062.nt.1 \\n                 17464 \\n                 17464.02 \\n                 Ovarian array \\n                 1167-1226 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Example 2b \\n     Relative Quantitation of Gene Expression \\n     Real-Time quantitative PCR with fluorescent Taqman® probes is a quantitation detection system utilizing the 5′-3′ nuclease activity of Taq DNA polymerase. The method uses an internal fluorescent oligonucleotide probe (Taqman®) labeled with a 5′ reporter dye and a downstream, 3′ quencher dye. During PCR, the 5′-3′ nuclease activity of Taq DNA polymerase releases the reporter, whose fluorescence can then be detected by the laser detector of the Model 7700 Sequence Detection System (PE Applied Biosystems, Foster City, Calif., USA). Amplification of an endogenous control is used to standardize the amount of sample RNA added to the reaction and normalize for Reverse Transcriptase (RT) efficiency. Either cyclophilin, glyceraldehyde-3-phosphate dehydrogenase (GAPDH), ATPase, or 18S ribosomal RNA (rRNA) is used as this endogenous control. To calculate relative quantitation between all the samples studied, the target RNA levels for one sample were used as the basis for comparative results (calibrator). Quantitation relative to the “calibrator” can be obtained using the comparative method (User Bulletin #2: ABI PRISM 7700 Sequence Detection System). \\n     The tissue distribution and the level of the target gene are evaluated for every sample in normal and cancer tissues. Total RNA is extracted from normal tissues, cancer tissues, and from cancers and the corresponding matched adjacent tissues. Subsequently, first strand cDNA is prepared with reverse transcriptase and the polymerase chain reaction is done using primers and Taqman® probes specific to each target gene. The results are analyzed using the ABI PRISM 7700 Sequence Detector. The absolute numbers are relative levels of expression of the target gene in a particular tissue compared to the calibrator tissue. \\n     One of ordinary skill can design appropriate primers. The relative levels of expression of the OSNA versus normal tissues and other cancer tissues can then be determined. All the values are compared to the calibrator. Normal RNA samples are commercially available pools, originated by pooling samples of a particular tissue from different individuals. \\n     The relative levels of expression of the OSNA in pairs of matched samples may also be determined. A matched pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. All the values are compared to the calibrator. \\n     In the analysis of matching samples, the OSNAs show a high degree of tissue specificity for the tissue of interest. These results confirm the tissue specificity results obtained with normal pooled samples. Further, the level of mRNA expression in cancer samples and the isogenic normal adjacent tissue from the same individual are compared. This comparison provides an indication of specificity for the cancer state (e.g. higher levels of mRNA expression in the cancer sample compared to the normal adjacent). \\n     Information on the samples tested in the QPCR experiments below include the Sample ID (Smpl ID), Organ, Tissue Type (Tiss Type), Diagnosis (DIAG), Disease Detail, and Stage or Grade (STG or GRD) in following table. \\n                                                 Sample       TISS           STAGE OR       ID   ORGAN   TYPE   DIAGNOSIS   DISEASE DETAIL   GRADE                  A084   Ovary   CAN   Mucinous                           borderline tumor       A084   Ovary   NAT   NAT       G010   Ovary   CAN   Adenocarcinoma   Adenocarcinoma   Stage III       G010   Ovary   NAT       NAT       G021   Ovary   CAN   Carcinoma   St. IIIC, poorly   Stage-                       diff.   IIIC,                           poorly                           diff.       G021   Ovary   NAT       NAT       1157   Ovary   CAN       malignant tumor       773O   Ovary   CAN   Papillary   serous papillary   metastatic                   adenocarcinoma   adenocarcinoma       814O   Ovary   CAN       Papillary Serous   Stage IV                       Adenocarcinoma       C360   Ovary   CAN   Adenocarcinoma   endometrioid                       adenocarcinoma       1005O   Ovary   CAN       papillary serous   3                       and endometrioid                       ovarian                       carcinoma,                       concurrent                       metastatic                       breast cancer       1040O   Ovary   CAN       papillary serous                       adeno,                       metastatic       105O   Ovary   CAN       Papillary Serous   Stage IC                       Carcinoma with   G0;                       Focal Mucinous   T1cN0M0                       Differentiation       130X   Ovary   CAN       Ovarian cancer       718O   Ovary   CAN   Adenocarcinoma   malignant tumor   IIIC       A1B   Ovary   CAN   Adenocarcinoma   CA       988Z   Ovary   CAN       papillary serous   poorly                       adenocarcinoma   diff, FIGO                           IIIC       451O   Ovary   NRM       Normal Tissue       247A   Ovary   NRM       NL       35GA   Ovary   NRM       NL       C087   Ovary   NRM       NL       C109   Ovary   NRM       NL       206I   Ovary   NRM       NL       515O   Ovary   NRM       Normal       18GA   Ovary   NRM       NL       337O   Ovary   NRM       Normal       123O   Ovary   NRM       Normal       C177   Ovary   NRM       several fluid                       filled cysts       40G   Ovary   NRM       NL       C004   Ovary   NRM       NL       030B   Urinary   CAN   Carcinoma   invasive   Stage III,           Bladder           Carcinoma, poorly   Grade 3                       differentiated       030B   Urinary   NAT       NAT           Bladder       TR17   Urinary   CAN   Carcinoma   transitional   StageII/Grade           Bladder           cell carcinoma   III       TR17   Urinary   NAT       NAT           Bladder       520B   Urinary   CAN   Sarcomatoid   Sarcomatoid           Bladder       transitional   transitional                   cell carcinoma   cell carcinoma       520B   Urinary   NAT       NAT           Bladder       401C   Colon   CAN   Adenocarcinoma   Adenocarcinoma   Stage III                       of ascending                       colon and cecum       401C   Colon   NAT       NAT       AS43   Colon   CAN   Adenocarcinoma   malignant       AS43   Colon   NAT   Adenocarcinoma   NAT       AS98   Colon   CAN   Adenocarcinoma   Moderately to   Duke&#39;s C                       poorly                       differentiated                       adenocarcinoma       AS98   Colon   NAT       NAT       CM12   Colon   CAN       T   Stage D       CM12   Colon   NAT   Adenocarcinoma   Nat       DC19   Colon   CAN       T   Stage B       DC19   Colon   NAT       NL       RC01   Colon   CAN   Cancer       Stage IV       RC01   Colon   NAT       NAT       RS53   Colon   CAN   Adenocarcinoma   moderately                       differentiated                       adenocarcinoma       RS53   Colon   NAT   Adenocarcinoma   NAT       SG27   Colon   CAN       malig   Stage B       SG27   Colon   NAT       NAT       TX01   Colon   CAN   Adenocarcinoma   Moderately   Stage II;                       differentiated   T3NoMo                       adenocarcinoma                       of cecum       TX01   Colon   NAT       NAT       KS52   Cervix   CAN   Squamous cell   Keratinizing   IIIB, well                   carcinoma   Squamous Cell   diff. G1;                       Carcinoma   T3bNxM0       KS52   Cervix   NAT       NAT       NK23   Cervix   CAN       Nonkeratinizing   FIGO IIIB,                       Large Cell   undiff.                           G4;                           T3bNxM0       NK23   Cervix   NAT       NAT       NKS54   Cervix   CAN   Squamous cell   Nonkeratinizing   IIB, mod                   carcinoma   Squamous Cell   diff. G2;                       Carcinoma   T2bNxM0       NKS54   Cervix   NAT       NAT       NKS55   Cervix   CAN   Squamous cell   Nonkeratinizing   IIIB, Mod                   carcinoma   Squamous Cell   diff. G2;                       Carcinoma   T3bNxM0       NKS55   Cervix   NAT       NAT       NKS81   Cervix   CAN   Squamous cell   large cell   IIB                   carcinoma   nonkeratinizing                       sq carc, IIB,                       moderately diff       NKS81   Cervix   NAT       NAT       NKS25   Cervix   CAN       NKS25   Cervix   NAT       NAT       NKS18   Cervix   CAN   Squamous cell   Nonkeratinizing   GII                   carcinoma   squamous cell                       carcinoma       NKS18   Cervix   NAT       NAT       10479   Endometrium   CAN       malignant mixed   T?, Nx, M1                       mullerian tumor       10479   Endometrium   NAT       NAT       28XA   Endometrium   CAN   Endometrial   malignant   II/III                   adenocarcinoma       28XA   Endometrium   NAT       NAT   II/III       8XA   Endometrium   CAN   mod. diff,                   invasive,                   squamous                   differentiation,                   FIGO-II       8XA   Endometrium   NAT       NAT       106XD   Kidney   CAN   Renal cell   renal cell   3                   carcinoma   carcinoma, clear                       cell, localized       106XD   Kidney   NAT       NL       107XD   Kidney   CAN   Renal cell   renal cell   G III                   carcinoma   carcinoma, clear                       cell, with                       metastatic       107XD   Kidney   NAT       NL       109XD   Kidney   CAN       Malignant   G III       109XD   Kidney   NAT       NL       10XD   Kidney   CAN   Renal cell   renal cell   3                   carcinoma   carcinoma, clear                       cell, localized,                       grade 2-3       10XD   Kidney   NAT       NL       22K   Kidney   CAN   Renal cell   Renal cell   G2, Mod.                   carcinoma   carcinoma   Diff.       22K   Kidney   NAT       NAT       12XD   Kidney   CAN   Renal cell   Left renal cell                   carcinoma   carcinoma       12XD   Kidney   NAT       NAT       15XA   Liver   CAN       Sarcoma, Retroperitoneal   Grade-2                       Tumor       15XA   Liver   NAT       CA   St. I, G4       174L   Liver   CAN   Hepatocellular   Moderate to well                   carcinoma   differentiated                       hepatocellular                       carcinoma       174L   Liver   NAT   Hepatocellular   NAT                   carcinoma       187L   Liver   CAN   Adenocarcinoma   Metastatic   Liver                       Adenocarcinoma   (Gallbladder)       187L   Liver   NAT       NAT       205L   Lung   CAN   Adenocarcinoma   poorly   T2, N1, Mx                       differentiated                       adenocarcinoma       205L   Lung   NAT       NAT       315L   Lung   CAN   Squamous cell                   carcinoma       315L   Lung   NAT   Adenocarcinoma   NAT       507L   Lung   CAN   Bronchioloalveolar   bronchioalveolar   Stage IB,                   carcinoma   carcinoma   G1, well                           diff.       507L   Lung   NAT       NAT       528L   Lung   CAN   Adenocarcinoma   Adenocarcinoma   St.IV, T2N0                           M1,                           infiltrating                           poorly                           diff.       528L   Lung   NAT       NAT       8837L   Lung   CAN   Squamous cell   Squamous cell   T2, N0, M0                   carcinoma   carcinoma       8837L   Lung   NAT       NAT       AC11   Lung   CAN   Adenocarcinoma   poorly   T2, N2, M1                       differentiated                       adenocarcinoma       AC11   Lung   NAT       NAT       AC39   Lung   CAN   Adenocarcinoma   intermediate   T2, N2, Mx                       grade                       adnocarcinoma       AC39   Lung   NAT       NAT       SQ80   Lung   CAN   Squamous cell   poorly   T1, N1, M0                   carcinoma   differentiated                       squamous cell                       carcinoma       SQ80   Lung   NAT       NAT       SQ81   Lung   CAN   Squamous cell   poorly   T3, N1, Mx                   carcinoma   differentiated                       squamous                       carcinoma       SQ81   Lung   NAT       NAT       19DN   Mammary   CAN   Invasive ductal   Invasive ductal   G3, Stage                   carcinoma   carcinoma   IIA;                           T2N0M0       19DN   Mammary   NAT       NAT       42DN   Mammary   CAN   Invasive ductal   Invasive Ductal   T3aN1M0                   carcinoma   Carcinoma   IIIA, G3       42DN   Mammary   NAT       NAT       517   Mammary   CAN   Infiltrating   Infiltrating   St. IIA,                   ductal carcinoma   ductal carcinoma   G3       517   Mammary   NAT       NAT       781M   Mammary   CAN   Invasive ductal       Architectural                   carcinoma       grade-                           3/3, Nuclear                           grade-                           3/3       781M   Mammary   NAT       NAT       869M   Mammary   CAN   Invasive   Invasive   Stage IIA                   carcinoma   Carcinoma   G1; T2NoMo       869M   Mammary   NAT       NAT       976M   Mammary   CAN   Invasive ductal   Invasive Ductal   T2N1M0                   carcinoma   Carcinoma   (Stage 2B                           Grade 2-3)       976M   Mammary   NAT       NAT       S570   Mammary   CAN   Carcinoma   Carcinoma   Stage                           IIA; T1N1Mo       S570   Mammary   NAT       NAT       S699   Mammary   CAN   Invasive lobular   Invasive Lobular   Stage IIB                   carcinoma   Carcinoma   G1; T2N1Mo       S699   Mammary   NAT       NAT       S997   Mammary   CAN   Invasive ductal   Invasive Ductal   Stage IIB                   carcinoma   Carcinoma   G3; T2N1Mo       S997   Mammary   NAT       NAT       71XL   Pancreas   CAN       villous adenoma   localized                       with paneth cell                       metaplasia       71XL   Pancreas   NAT       NL       82XP   Pancreas   CAN       serious                       cystadenoma       82XP   Pancreas   NAT       NL       92X   Pancreas   CAN   Ductal   ductal   mod to                   adenocarcinoma   adenocarcinoma   focally                           poorly                           diff.       92X   Pancreas   NAT       NL       77X   Pancreas   CAN   Hepatic adenoma   Hepatic adenoma       77X   Pancreas   NAT       NL       23B   Prostate   CAN       Prostate tumor   Gleason&#39;s                           3 + 4       23B   Prostate   NAT       NAT       65XB   Prostate   CAN   Adenocarcinoma   adenocarcinom   3 + 4 = 7       65XB   Prostate   NAT       NL       675P   Prostate   CAN   Adenocarcinoma   adenocarcinoma       675P   Prostate   NAT       Normal       84XB   Prostate   CAN   Adenocarcinoma   adenocarcinom   2 + 3       84XB   Prostate   NAT       NL       958P   Prostate   CAN   Adenocarcinoma   Adenocarcinoma   T2C, NO,                           MX       958P   Prostate   NAT   NAT   Normal       263C   Prostate   BPH       BPH       276P   Prostate   BPH       BPH       767B   Prostate   BPH       prostate BPH       855P   Prostate   BPH       BPH       10R   Prostate   PROST       active chronic   T0, N0, M0                       prostatitis       20R   Prostate   PROST       PROSTATITIS       287S   Skin   CAN   Squamous cell   Invasive   Moderately                   carcinoma   Keratinizing   Differentiated                       Squamous Cell                       Carcinoma       287S   Skin   NAT       NAT       39A   Skin   CAN       CA   St. II       39A   Skin   NAT       CA   St. II       669S   Skin   CAN   Melanoma   Nodular                       malignant                       melanoma       669S   Skin   NAT       NAT       171S   Small   CAN   Adenocarcinoma   Moderately           Intestine           differentiated                       Adenocarcinoma,                       invasive       171S   Small   NAT       NAT           Intestine       20SM   Small   CAN   Adenocarcinoma   Adenocarcinoma,   St. IV,           Intestine           metastic to lung   poorly                       &amp; liver   diff.       20SM   Small   NAT       NAT           Intestine       H89   Small   CAN   Adenocarcinoma   Adenocarcinoma   80% tumor,           Intestine               50%                           necrosis,                           moderately                           differentiated,                           G2-                           3; T3N1MX       H89   Small   NAT   Adenocarcinoma   NAT           Intestine       261S   Stomach   CAN   Signet-ring cell   Signet-ring cell   Stage                   carcinoma   carcinoma   IIIA,                           T3N1M0       261S   Stomach   NAT       NAT       288S   Stomach   CAN   Adenocarcinoma   Infiltrating   Moderately                       Adneocarcinoma   Differentiated       288S   Stomach   NAT       NAT       AC93   Stomach   CAN   Adenocarcinoma   Adenocarcinoma   St. IV,       or                   G4,       509L                   T4N3M0,                           poorly                           diff.       AC93   Stomach   NAT       NAT       or       509L       88S   Stomach   CAN   Adenocarcinoma   Mucinous   T3N1M0,                       adenocarcinoma   St. IIIA       88S   Stomach   NAT       NAT       143N   Thyroid   CAN   Follicular   Follicular           Gland       carcinoma   Carcinoma       143N   Thyroid   NAT       NAT           Gland       270T   Thyroid   CAN       CA           Gland       270T   Thyroid   NAT       NAT           Gland       56T   Thyroid   CAN   Papillary   Papillary   St. III;           Gland       carcinoma   Carcinoma   T4N1M0       56T   Thyroid   NAT       NAT           Gland       39X   Testes   CAN       CA       39X   Testes   NAT       NAT       647T   Testes   CAN   Teratocarcinoma   Teratocarcinoma   Stage IA       647T   Testes   NAT   Teratocarcinoma   NAT       663T   Testes   CAN   Teratocarcinoma   Teratocarcinoma       663T   Testes   NAT       NAT       135XO   Uterus   CAN       Uterus normal       135XO   Uterus   NAT       Uterus tumor       85XU   Uterus   CAN       endometrial   I                       carcinoma       85XU   Uterus   NAT       NL       B1   Blood   NRM       Normal       B3   Blood   NRM       Normal       B5   Blood   NRM       Normal       B6   Blood   NRM       Normal       B11   Blood   NRM       Normal       982B   Blood   NRM       Normal       B69   Blood   NRM       Normal       B72   Blood   NRM       Normal       B73   Blood   NRM       Normal       B75   Blood   NRM       Normal       48AD   Adrenal   NRM       Normal           Gland       10BR   Brain   NRM       Normal       01CL   Colon   NRM       Normal       06CV   Cervix   NRM       Normal       01ES   Esophagus   NRM       Normal       46HR   Heart   NRM       Normal       00HR   Human   CAN   CAN   Cancer pool           Reference       55KD   Kidney   NRM       Normal       89LV   Liver   NRM       Normal       90LN   Lung   NRM       Normal       01MA   Mammary   NRM       Normal       84MU   Skeletal   NRM       Normal           Muscle       3APV   Ovary   NRM       Normal       04PA   Pancreas   NRM       Normal       59PL   Placenta   NRM       Normal       09PR   Prostate   NRM       Normal       21RC   Rectum   NRM       Normal       59SM   Small   NRM       Normal           Intestine       7GSP   Spleen   NRM       Normal       09ST   Stomach   NRM       Normal       4GTS   Testes   NRM       Normal       99TM   Thymus   NRM       Normal           Gland       16TR   Trachea   NRM       Normal       57UT   Uterus   NRM       Normal                    \\nDEX0455 — 019.nt.1 (Ovr224)\\n \\n     The relative expression level of Ovr224 in various tissue samples is included below. Tissue samples include 68 pairs of matching samples, 10 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 4 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to ovarian cancer sample OVR7730 (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.01 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.00 \\n                 0.06 \\n               \\n               \\n                 OVRG021 \\n                 0.03 \\n                 0.03 \\n               \\n               \\n                 OVR1157 \\n                 0.36 \\n               \\n               \\n                 OVR773O \\n                 1.00 \\n               \\n               \\n                 OVR814O \\n                 0.02 \\n               \\n               \\n                 OVRC360 \\n                 0.02 \\n               \\n               \\n                 OVR1005O \\n                 0.35 \\n               \\n               \\n                 OVR1040O \\n                 0.10 \\n               \\n               \\n                 OVR105O \\n                 0.00 \\n               \\n               \\n                 OVR130X \\n                 0.44 \\n               \\n               \\n                 OVR718O \\n                 0.02 \\n               \\n               \\n                 OVRA1B \\n                 0.04 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 BLDTR17 \\n                 0.00 \\n                 0.03 \\n               \\n               \\n                 CLN401C \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNAS98 \\n                 0.02 \\n                 0.00 \\n               \\n               \\n                 CLNCM12 \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 CLNDC19 \\n                 0.02 \\n                 0.00 \\n               \\n               \\n                 CLNRC01 \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 CLNRS53 \\n                 0.14 \\n                 0.00 \\n               \\n               \\n                 CLNSG27 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNTX01 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.03 \\n               \\n               \\n                 CVXNK23 \\n                 0.01 \\n                 0.00 \\n               \\n               \\n                 CVXNKS54 \\n                 0.00 \\n                 0.25 \\n               \\n               \\n                 CVXNKS55 \\n                 0.06 \\n                 0.17 \\n               \\n               \\n                 CVXNKS81 \\n                 0.87 \\n                 0.00 \\n               \\n               \\n                 ENDO10479 \\n                 0.03 \\n                 0.00 \\n               \\n               \\n                 ENDO28XA \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 ENDO8XA \\n                 0.02 \\n                 0.00 \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.08 \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.07 \\n               \\n               \\n                 KID109XD \\n                 0.06 \\n                 0.37 \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 KID22K \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 LNG205L \\n                 0.00 \\n                 0.33 \\n               \\n               \\n                 LNG315L \\n                 0.00 \\n                 0.53 \\n               \\n               \\n                 LNG507L \\n                 0.21 \\n                 0.43 \\n               \\n               \\n                 LNG528L \\n                 0.00 \\n                 2.39 \\n               \\n               \\n                 LNG8837L \\n                 0.02 \\n                 0.13 \\n               \\n               \\n                 LNGAC11 \\n                 0.32 \\n                 0.23 \\n               \\n               \\n                 LNGSQ80 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.04 \\n               \\n               \\n                 MAM19DN \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM42DN \\n                 0.13 \\n                 0.00 \\n               \\n               \\n                 MAM517 \\n                 0.62 \\n                 0.00 \\n               \\n               \\n                 MAM781M \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM869M \\n                 0.00 \\n                 0.42 \\n               \\n               \\n                 MAM976M \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS699 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS997 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 PAN71XL \\n                 0.01 \\n                 0.04 \\n               \\n               \\n                 PAN82XP \\n                 0.01 \\n                 0.00 \\n               \\n               \\n                 PAN92X \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 PRO23B \\n                 0.02 \\n                 0.03 \\n               \\n               \\n                 PRO65XB \\n                 0.01 \\n                 0.02 \\n               \\n               \\n                 PRO675P \\n                 0.07 \\n                 0.00 \\n               \\n               \\n                 PRO84XB \\n                 0.02 \\n                 0.09 \\n               \\n               \\n                 PRO958P \\n                 0.00 \\n                 0.04 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.04 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 SKN287S \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 SKN39A \\n                 0.62 \\n                 0.73 \\n               \\n               \\n                 SKN669S \\n                 0.02 \\n                 0.00 \\n               \\n               \\n                 SMINT171S \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 SMINT20SM \\n                 0.04 \\n                 0.00 \\n               \\n               \\n                 SMINTH89 \\n                 0.01 \\n                 0.00 \\n               \\n               \\n                 STO261S \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 STO288S \\n                 0.00 \\n                 0.03 \\n               \\n               \\n                 STO88S \\n                 0.04 \\n                 0.03 \\n               \\n               \\n                 THRD143N \\n                 0.00 \\n                 0.04 \\n               \\n               \\n                 THRD270T \\n                 0.05 \\n                 0.03 \\n               \\n               \\n                 THRD56T \\n                 0.44 \\n                 0.05 \\n               \\n               \\n                 TST39X \\n                 0.00 \\n                 0.33 \\n               \\n               \\n                 TST647T \\n                 0.02 \\n                 0.07 \\n               \\n               \\n                 TST663T \\n                 0.05 \\n                 0.01 \\n               \\n               \\n                 UTR135XO \\n                 0.05 \\n                 0.00 \\n               \\n               \\n                 UTR85XU \\n                 0.03 \\n                 0.00 \\n               \\n               \\n                 BLOB1 \\n                   \\n                   \\n                 9.03 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.71 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 5.37 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 3.85 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 0.93 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.22 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.63 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                   \\n               \\n               \\n                 Note: \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr224 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr224 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr224 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, Up \\n                 44% \\n                 \\u20020% \\n                 22% \\n                 0% \\n                 20% \\n               \\n               \\n                 vs. NAT \\n               \\n               \\n                 Sensitivity, Down \\n                 22% \\n                 56% \\n                 11% \\n                 0% \\n                 40% \\n               \\n               \\n                 vs. NAT \\n               \\n               \\n                 Sensitivity, Up \\n                 44% \\n                 33% \\n                 22% \\n                 92%\\u2002 \\n                 80% \\n               \\n               \\n                 vs. NRM \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 44% \\n                 \\u20020% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. NRM \\n               \\n               \\n                 Specificity \\n                 47.03%\\u2003\\u2009 \\n                 54.59%\\u2003\\u2009 \\n                 45.41%\\u2003\\u2009 \\n                 56%\\u2002 \\n                 52.41%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr224 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr224 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003296)                     (Ovr224_forward):   TCCTCAAGGGCCCTCCCCAG                         (SEQ\\u2003ID\\u2003NO:\\u2003297)                     (Ovr224_reverse):   CCACAGCCATCTCCTCCATATTCTG                         (SEQ\\u2003ID\\u2003NO:\\u2003298)                     (Ovr224_probe):   AAGTGTTCCTCTGGATGACCTACCTGG            \\nDEX0455 — 031.nt.2 (Cln257)\\n \\n     The relative expression level of Cln257 in various tissue samples is included below. Tissue samples include 78 pairs of matching samples, 6 non matched cancer samples, and 35 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal colon sample CLN01CL (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 CLNAS12 \\n                 5.55 \\n                 10.39 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS46 \\n                 6.28 \\n                 3.22 \\n               \\n               \\n                 CLNB34 \\n                 1.78 \\n                 3.88 \\n               \\n               \\n                 CLNC9XR \\n                 2.76 \\n                 3.35 \\n               \\n               \\n                 CLNCM67 \\n                 2.91 \\n                 2.44 \\n               \\n               \\n                 CLNTX89 \\n                 6.56 \\n                 5.08 \\n               \\n               \\n                 CLNAS43 \\n                 27.92 \\n                 6.39 \\n               \\n               \\n                 CLNAS98 \\n                 6.93 \\n                 5.42 \\n               \\n               \\n                 CLNRS53 \\n                 8.04 \\n                 6.77 \\n               \\n               \\n                 CLNRC01 \\n                 9.91 \\n                 2.51 \\n               \\n               \\n                 CLNSG27 \\n                 4.56 \\n                 7.39 \\n               \\n               \\n                 CLNDC19 \\n                 3.97 \\n                 3.84 \\n               \\n               \\n                 CLN401C \\n                 7.09 \\n                 4.98 \\n               \\n               \\n                 CLNCM12 \\n                 3.28 \\n                 6.25 \\n               \\n               \\n                 CLNTX01 \\n                 16.34 \\n                 8.61 \\n               \\n               \\n                 BLD030B \\n                 2.29 \\n                 2.59 \\n               \\n               \\n                 BLD520B \\n                 12.82 \\n                 14.74 \\n               \\n               \\n                 BLDTR17 \\n                 10.50 \\n                 5.28 \\n               \\n               \\n                 CVXKS52 \\n                 12.36 \\n                 17.89 \\n               \\n               \\n                 CVXNK23 \\n                 12.42 \\n                 62.77 \\n               \\n               \\n                 CVXNKS54 \\n                 24.16 \\n                 13.33 \\n               \\n               \\n                 CVXNKS55 \\n                 15.58 \\n                 17.45 \\n               \\n               \\n                 CVXNKS81 \\n                 84.82 \\n                 132.51 \\n               \\n               \\n                 ENDO10479 \\n                 15.86 \\n                 25.40 \\n               \\n               \\n                 ENDO28XA \\n                 12.96 \\n                 13.04 \\n               \\n               \\n                 ENDO8XA \\n                 12.25 \\n                 3.60 \\n               \\n               \\n                 KID106XD \\n                 0.32 \\n                 1.89 \\n               \\n               \\n                 KID107XD \\n                 29.14 \\n                 4.27 \\n               \\n               \\n                 KID109XD \\n                 8.21 \\n                 5.31 \\n               \\n               \\n                 KID10XD \\n                 5.61 \\n                 0.84 \\n               \\n               \\n                 KID22K \\n                 2.84 \\n                 1.47 \\n               \\n               \\n                 LNG205L \\n                 8.83 \\n                 9.05 \\n               \\n               \\n                 LNG315L \\n                 16.63 \\n                 28.85 \\n               \\n               \\n                 LNG507L \\n                 13.87 \\n                 27.96 \\n               \\n               \\n                 LNG528L \\n                 20.05 \\n                 27.89 \\n               \\n               \\n                 LNG8837L \\n                 16.21 \\n                 10.02 \\n               \\n               \\n                 LNGAC11 \\n                 15.21 \\n                 14.83 \\n               \\n               \\n                 LNGAC39 \\n                 49.00 \\n                 16.41 \\n               \\n               \\n                 LNGSQ80 \\n                 18.40 \\n                 11.35 \\n               \\n               \\n                 LNGSQ81 \\n                 7.80 \\n                 54.12 \\n               \\n               \\n                 LVR15XA \\n                 9.04 \\n                 2.93 \\n               \\n               \\n                 LVR174L \\n                 4.08 \\n                 6.13 \\n               \\n               \\n                 LVR187L \\n                 3.52 \\n                 3.60 \\n               \\n               \\n                 MAM19DN \\n                 14.68 \\n                 14.78 \\n               \\n               \\n                 MAM42DN \\n                 12.41 \\n                 26.01 \\n               \\n               \\n                 MAM517 \\n                 133.69 \\n                 12.41 \\n               \\n               \\n                 MAM781M \\n                 23.89 \\n                 12.22 \\n               \\n               \\n                 MAM869M \\n                 7.84 \\n                 17.28 \\n               \\n               \\n                 MAM976M \\n                 39.22 \\n                 32.92 \\n               \\n               \\n                 MAMS570 \\n                 21.06 \\n                 26.04 \\n               \\n               \\n                 MAMS699 \\n                 6.70 \\n                 0.00 \\n               \\n               \\n                 MAMS997 \\n                 11.37 \\n                 13.47 \\n               \\n               \\n                 OVRG021 \\n                 9.65 \\n                 18.53 \\n               \\n               \\n                 OVR1005O \\n                 36.75 \\n               \\n               \\n                 OVR1040O \\n                 14.88 \\n               \\n               \\n                 OVR105O \\n                 8.82 \\n               \\n               \\n                 OVR130X \\n                 32.30 \\n               \\n               \\n                 OVR718O \\n                 22.87 \\n               \\n               \\n                 OVRA1B \\n                 15.50 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 16.94 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 13.92 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 15.98 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 13.28 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 20.23 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 26.97 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 54.21 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 6.97 \\n               \\n               \\n                 PAN71XL \\n                 9.65 \\n                 8.64 \\n               \\n               \\n                 PAN82XP \\n                 7.20 \\n                 24.22 \\n               \\n               \\n                 PAN92X \\n                 8.74 \\n                 26.55 \\n               \\n               \\n                 PRO23B \\n                 13.10 \\n                 14.00 \\n               \\n               \\n                 PRO65XB \\n                 6.20 \\n                 10.57 \\n               \\n               \\n                 PRO675P \\n                 20.64 \\n                 27.15 \\n               \\n               \\n                 PRO84XB \\n                 10.46 \\n                 10.35 \\n               \\n               \\n                 PRO958P \\n                 11.48 \\n                 10.47 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 35.87 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 7.20 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 17.09 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 8.27 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 16.92 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 15.27 \\n               \\n               \\n                 SKN287S \\n                 8.51 \\n                 9.87 \\n               \\n               \\n                 SKN39A \\n                 12.75 \\n                 8.64 \\n               \\n               \\n                 SKN669S \\n                 8.95 \\n                 23.59 \\n               \\n               \\n                 SMINT171S \\n                 9.57 \\n                 15.19 \\n               \\n               \\n                 SMINT20SM \\n                 30.83 \\n                 12.12 \\n               \\n               \\n                 SMINTH89 \\n                 10.91 \\n                 10.48 \\n               \\n               \\n                 STO261S \\n                 16.09 \\n                 3.67 \\n               \\n               \\n                 STO288S \\n                 8.76 \\n                 3.43 \\n               \\n               \\n                 STO88S \\n                 14.77 \\n                 4.27 \\n               \\n               \\n                 THRD143N \\n                 6.43 \\n                 17.06 \\n               \\n               \\n                 THRD270T \\n                 25.28 \\n                 27.05 \\n               \\n               \\n                 THRD56T \\n                 12.28 \\n                 9.55 \\n               \\n               \\n                 TST39X \\n                 7.03 \\n                 1.37 \\n               \\n               \\n                 TST647T \\n                 4.87 \\n                 5.35 \\n               \\n               \\n                 TST663T \\n                 10.23 \\n                 3.49 \\n               \\n               \\n                 UTR135XO \\n                 10.47 \\n                 13.31 \\n               \\n               \\n                 UTR85XU \\n                 25.28 \\n                 27.08 \\n               \\n               \\n                 BLOB1 \\n                   \\n                   \\n                 82.99 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 15.84 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 81.31 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 12.68 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 3.82 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 1.96 \\n               \\n               \\n                 HUMREF00HR \\n                 0.94 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 4.70 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.59 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.58 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 1.93 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 3.14 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 6.01 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.21 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 5.62 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 3.59 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 5.14 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 3.40 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 8.88 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 3.09 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 3.91 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 2.19 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 4.39 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 6.32 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 1.10 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 14.36 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Cln257 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Cln257 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of colon tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Cln257 being useful as an colon cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, Up vs. \\n                 13% \\n                 11% \\n                 22% \\n                 0% \\n                 0% \\n               \\n               \\n                 NAT \\n               \\n               \\n                 Sensitivity, Down \\n                 \\u20027% \\n                 22% \\n                 22% \\n                 0% \\n                 0% \\n               \\n               \\n                 vs. NAT \\n               \\n               \\n                 Sensitivity, Up \\n                 93% \\n                 100%\\u2002 \\n                 67% \\n                 29%\\u2002 \\n                 80%\\u2002 \\n               \\n               \\n                 vs. NRM \\n               \\n               \\n                 Sensitivity, Down \\n                 \\u20020% \\n                 \\u20020% \\n                 \\u20020% \\n                 0% \\n                 0% \\n               \\n               \\n                 vs. NRM \\n               \\n               \\n                 Specificity \\n                 3.47%\\u2002\\u2009 \\n                 6.49%\\u2002\\u2009 \\n                 5.95%\\u2002\\u2009 \\n                 6.42%\\u2003\\u2009 \\n                 5.35%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Cln257 a good marker for diagnosing, monitoring, staging, imaging and/or treating colon cancer. \\n     Primers used for QPCR Expression Analysis of Cln257 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003299)                             (Cln257_forward):   CTGAAGCCGAGCTCAAAGGT                         (SEQ\\u2003ID\\u2003NO:\\u2003300)                             (Cln257_reverse):   CCCTGCTCCCACTTGAGATC                         (SEQ\\u2003ID\\u2003NO:\\u2003301)                             (Cln257_probe):   TGTGAAAAGGAGGCTGGGTGCCAG            \\nDEX0455 — 034.nt.1 and DEX0455 — 034.nt.2 (Ovr223)\\n \\n     The relative expression level of Ovr223 in various tissue samples is included below. Tissue samples include 75 pairs of matching samples, 11 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 4 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to ovarian cancer sample OVR7730 (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.12 \\n                 0.05 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.04 \\n                 0.24 \\n               \\n               \\n                 OVRG021 \\n                 0.16 \\n                 0.05 \\n               \\n               \\n                 OVR1157 \\n                 0.32 \\n               \\n               \\n                 OVR773O \\n                 1.00 \\n               \\n               \\n                 OVR814O \\n                 0.06 \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n               \\n               \\n                 OVR1005O \\n                 0.75 \\n               \\n               \\n                 OVR1040O \\n                 0.97 \\n               \\n               \\n                 OVR105O \\n                 0.80 \\n               \\n               \\n                 OVR130X \\n                 2.15 \\n               \\n               \\n                 OVR718O \\n                 0.80 \\n               \\n               \\n                 OVRA1B \\n                 1.90 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.06 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.04 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.12 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 BLD520B \\n                 0.74 \\n                 0.02 \\n               \\n               \\n                 BLDTR17 \\n                 0.00 \\n                 0.11 \\n               \\n               \\n                 CLN401C \\n                 0.40 \\n                 0.35 \\n               \\n               \\n                 CLNAS43 \\n                 1.05 \\n                 0.16 \\n               \\n               \\n                 CLNAS98 \\n                 0.16 \\n                 0.25 \\n               \\n               \\n                 CLNCM12 \\n                 0.21 \\n                 0.31 \\n               \\n               \\n                 CLNDC19 \\n                 0.47 \\n                 0.17 \\n               \\n               \\n                 CLNRC01 \\n                 0.31 \\n                 0.31 \\n               \\n               \\n                 CLNRS53 \\n                 0.18 \\n                 1.03 \\n               \\n               \\n                 CLNSG27 \\n                 0.00 \\n                 0.29 \\n               \\n               \\n                 CLNTX01 \\n                 0.36 \\n                 0.25 \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.74 \\n               \\n               \\n                 CVXNK23 \\n                 0.68 \\n                 2.29 \\n               \\n               \\n                 CVXNKS54 \\n                 1.18 \\n                 2.21 \\n               \\n               \\n                 CVXNKS55 \\n                 0.92 \\n                 0.82 \\n               \\n               \\n                 CVXNKS81 \\n                 1.72 \\n               \\n               \\n                 ENDO10479 \\n                 0.48 \\n                 1.16 \\n               \\n               \\n                 ENDO28XA \\n                 1.17 \\n                 0.25 \\n               \\n               \\n                 ENDO8XA \\n                 0.52 \\n                 0.13 \\n               \\n               \\n                 KID106XD \\n                 0.05 \\n                 0.05 \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.21 \\n               \\n               \\n                 KID109XD \\n                 0.14 \\n                 0.61 \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.06 \\n               \\n               \\n                 KID22K \\n                 0.21 \\n                 0.10 \\n               \\n               \\n                 LNG205L \\n                 0.23 \\n                 0.00 \\n               \\n               \\n                 LNG315L \\n                 0.15 \\n                 2.19 \\n               \\n               \\n                 LNG507L \\n                 0.37 \\n                 0.82 \\n               \\n               \\n                 LNG528L \\n                 2.95 \\n                 0.60 \\n               \\n               \\n                 LNG8837L \\n                 0.45 \\n                 0.70 \\n               \\n               \\n                 LNGAC11 \\n                 0.17 \\n                 0.54 \\n               \\n               \\n                 LNGAC39 \\n                 1.86 \\n                 0.23 \\n               \\n               \\n                 LNGSQ80 \\n                 0.82 \\n                 0.00 \\n               \\n               \\n                 LNGSQ81 \\n                 1.06 \\n                 0.69 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.29 \\n               \\n               \\n                 MAM19DN \\n                 1.16 \\n                 0.87 \\n               \\n               \\n                 MAM42DN \\n                 0.60 \\n                 0.00 \\n               \\n               \\n                 MAM517 \\n                 7.70 \\n                 0.00 \\n               \\n               \\n                 MAM781M \\n                 0.41 \\n                 0.74 \\n               \\n               \\n                 MAM869M \\n                 0.58 \\n                 0.00 \\n               \\n               \\n                 MAM976M \\n                 1.01 \\n                 0.42 \\n               \\n               \\n                 MAMS570 \\n                 2.29 \\n                 4.07 \\n               \\n               \\n                 MAMS699 \\n                 0.39 \\n                 0.00 \\n               \\n               \\n                 MAMS997 \\n                 1.33 \\n                 0.86 \\n               \\n               \\n                 PAN71XL \\n                 0.44 \\n                 0.77 \\n               \\n               \\n                 PAN82XP \\n                 0.10 \\n                 7.85 \\n               \\n               \\n                 PAN92X \\n                 0.49 \\n                 0.81 \\n               \\n               \\n                 PRO23B \\n                 0.15 \\n                 0.19 \\n               \\n               \\n                 PRO65XB \\n                 0.20 \\n                 0.52 \\n               \\n               \\n                 PRO675P \\n                 0.43 \\n                 0.32 \\n               \\n               \\n                 PRO84XB \\n                 0.43 \\n                 0.45 \\n               \\n               \\n                 PRO958P \\n                 0.46 \\n                 0.52 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.13 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.48 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.28 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.34 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.95 \\n               \\n               \\n                 SKN287S \\n                 0.49 \\n                 0.46 \\n               \\n               \\n                 SKN39A \\n                 0.00 \\n                 0.16 \\n               \\n               \\n                 SKN669S \\n                 0.38 \\n                 2.09 \\n               \\n               \\n                 SMINT171S \\n                 0.70 \\n                 0.51 \\n               \\n               \\n                 SMINT20SM \\n                 0.83 \\n                 0.31 \\n               \\n               \\n                 SMINTH89 \\n                 0.43 \\n                 1.27 \\n               \\n               \\n                 STO261S \\n                 1.61 \\n                 0.52 \\n               \\n               \\n                 STO288S \\n                 0.39 \\n                 0.16 \\n               \\n               \\n                 STO88S \\n                 0.00 \\n                 0.18 \\n               \\n               \\n                 THRD143N \\n                 0.25 \\n                 0.45 \\n               \\n               \\n                 THRD270T \\n                 0.95 \\n                 2.10 \\n               \\n               \\n                 THRD56T \\n                 2.62 \\n                 0.23 \\n               \\n               \\n                 TST39X \\n                 0.47 \\n                 0.90 \\n               \\n               \\n                 TST647T \\n                 0.38 \\n                 0.16 \\n               \\n               \\n                 TST663T \\n                 0.30 \\n                 0.02 \\n               \\n               \\n                 UTR135XO \\n                 0.09 \\n                 0.30 \\n               \\n               \\n                 UTR85XU \\n                 1.07 \\n                 0.59 \\n               \\n               \\n                 BLOB1 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.95 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.04 \\n               \\n               \\n                 CVX1ACV \\n                   \\n                   \\n                 7.20 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.56 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.26 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.11 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.33 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.27 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.18 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.09 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.06 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.21 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.69 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.14 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr223 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr223 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr223 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, Up vs. NAT \\n                 22% \\n                 44% \\n                 56%\\u2002 \\n                 0% \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NAT \\n                 22% \\n                 33% \\n                 0% \\n                 0% \\n                 20%\\u2002 \\n               \\n               \\n                 Sensitivity, Up vs. NRM \\n                 89% \\n                 44% \\n                 100%\\u2003 \\n                 85%\\u2002 \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NRM \\n                 11% \\n                 \\u20020% \\n                 0% \\n                 8% \\n                 0% \\n               \\n               \\n                 Specificity \\n                 24.32%\\u2003\\u2009 \\n                 25.41%\\u2003\\u2009 \\n                 30.81%\\u2003\\u2002\\u2009 \\n                 22.86%\\u2003\\u2002\\u2009 \\n                 25.67%\\u2003\\u2002\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr223 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Additionally, the tissue specificity, plus the mRNA differential expression in the samples tested may make Ovr223 a good marker for diagnosing, monitoring, staging, imaging and/or treating breast cancer. \\n     Primers used for QPCR Expression Analysis of Ovr223 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003302)                             (Ovr223_forward):   AGTGAGAGGGTGGGCATGTATG                         (SEQ\\u2003ID\\u2003NO:\\u2003303)                             (Ovr223_reverse):   TACTCCAGGCGCTCTGAGGAT                         (SEQ\\u2003ID\\u2003NO:\\u2003304)                             (Ovr223_probe):   TTAGCCAGTGGCCTCCACTCTGTCCC            \\nDEX0455 — 034.nt.4 (Ovr223v2)\\n \\n     The relative expression level of Ovr223v2 in various tissue samples is included below. Tissue samples include 74 pairs of matching samples, 11 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 4 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal pancreas sample PAN04PA (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.25 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 4.93 \\n               \\n               \\n                 OVRG021 \\n                 0.40 \\n                 0.06 \\n               \\n               \\n                 OVR1157 \\n                 3.69 \\n               \\n               \\n                 OVR773O \\n                 7.06 \\n               \\n               \\n                 OVR988Z \\n                 1.93 \\n               \\n               \\n                 OVRC360 \\n                 0.34 \\n               \\n               \\n                 OVR1005O \\n                 2.85 \\n               \\n               \\n                 OVR1040O \\n                 3.20 \\n               \\n               \\n                 OVR105O \\n                 3.02 \\n               \\n               \\n                 OVR130X \\n                 1.84 \\n               \\n               \\n                 OVR718O \\n                 2.15 \\n               \\n               \\n                 OVRA1B \\n                 7.99 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.44 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.21 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.23 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.20 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.11 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.12 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.07 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.20 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.93 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.05 \\n               \\n               \\n                 OVR451O \\n                   \\n                   \\n                 0.32 \\n               \\n               \\n                 BLD030B \\n                 0.20 \\n                 1.10 \\n               \\n               \\n                 BLD520B \\n                 2.00 \\n                 0.18 \\n               \\n               \\n                 BLDTR17 \\n                 0.39 \\n                 1.04 \\n               \\n               \\n                 CLN401C \\n                 0.85 \\n                 1.23 \\n               \\n               \\n                 CLNAS43 \\n                 2.68 \\n                 0.16 \\n               \\n               \\n                 CLNAS98 \\n                 0.61 \\n                 0.35 \\n               \\n               \\n                 CLNCM12 \\n                 0.61 \\n                 0.80 \\n               \\n               \\n                 CLNDC19 \\n                 1.94 \\n                 1.18 \\n               \\n               \\n                 CLNRC01 \\n                 0.46 \\n                 0.42 \\n               \\n               \\n                 CLNRS53 \\n                 0.54 \\n                 1.26 \\n               \\n               \\n                 CLNSG27 \\n                 0.61 \\n                 0.65 \\n               \\n               \\n                 CLNTX01 \\n                 1.62 \\n                 0.59 \\n               \\n               \\n                 CVXKS52 \\n                 3.54 \\n                 4.98 \\n               \\n               \\n                 CVXNKS55 \\n                 7.35 \\n                 4.40 \\n               \\n               \\n                 CVXNKS25 \\n                 4.23 \\n                 4.81 \\n               \\n               \\n                 CVXNKS18 \\n                 1.26 \\n                 3.88 \\n               \\n               \\n                 CVXNKS54 \\n                 3.00 \\n                 1.47 \\n               \\n               \\n                 ENDO10479 \\n                 3.07 \\n                 0.37 \\n               \\n               \\n                 ENDO28XA \\n                 4.24 \\n                 0.69 \\n               \\n               \\n                 ENDO8XA \\n                 0.31 \\n                 3.57 \\n               \\n               \\n                 KID106XD \\n                 0.11 \\n                 0.33 \\n               \\n               \\n                 KID12XD \\n                 0.27 \\n                 2.13 \\n               \\n               \\n                 KID10XD \\n                 0.10 \\n                 0.21 \\n               \\n               \\n                 KID22K \\n                 0.60 \\n                 0.28 \\n               \\n               \\n                 KID107XD \\n                 0.16 \\n                 0.44 \\n               \\n               \\n                 LNG205L \\n                 0.81 \\n                 1.09 \\n               \\n               \\n                 LNG315L \\n                 0.89 \\n                 2.02 \\n               \\n               \\n                 LNG507L \\n                 1.16 \\n                 1.68 \\n               \\n               \\n                 LNG528L \\n                 9.15 \\n                 1.43 \\n               \\n               \\n                 LNG8837L \\n                 1.46 \\n                 1.65 \\n               \\n               \\n                 LNGAC11 \\n                 0.86 \\n                 1.78 \\n               \\n               \\n                 LNGAC39 \\n                 6.93 \\n                 1.66 \\n               \\n               \\n                 LNGSQ80 \\n                 1.13 \\n                 0.32 \\n               \\n               \\n                 LNGSQ81 \\n                 1.95 \\n                 1.13 \\n               \\n               \\n                 LVR15XA \\n                 0.01 \\n                 0.03 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 2.35 \\n               \\n               \\n                 MAM19DN \\n                 3.52 \\n                 3.45 \\n               \\n               \\n                 MAM42DN \\n                 0.83 \\n                 1.62 \\n               \\n               \\n                 MAM517 \\n                 10.39 \\n                 3.02 \\n               \\n               \\n                 MAM781M \\n                 1.80 \\n                 0.34 \\n               \\n               \\n                 MAM869M \\n                 1.85 \\n                 0.13 \\n               \\n               \\n                 MAM976M \\n                 4.08 \\n                 0.67 \\n               \\n               \\n                 MAMS570 \\n                 2.43 \\n                 4.41 \\n               \\n               \\n                 MAMS699 \\n                 1.16 \\n                 1.50 \\n               \\n               \\n                 MAMS997 \\n                 1.20 \\n                 1.39 \\n               \\n               \\n                 PAN71XL \\n                 1.91 \\n                 1.83 \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 PAN92X \\n                 3.25 \\n                 0.25 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 2.41 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 1.07 \\n               \\n               \\n                 PRO23B \\n                 1.32 \\n                 1.17 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 1.30 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.88 \\n               \\n               \\n                 PRO65XB \\n                 0.87 \\n                 1.60 \\n               \\n               \\n                 PRO675P \\n                 1.50 \\n                 0.69 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 4.10 \\n               \\n               \\n                 PRO84XB \\n                 1.41 \\n                 1.13 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 1.16 \\n               \\n               \\n                 PRO958P \\n                 2.49 \\n                 2.56 \\n               \\n               \\n                 SKN287S \\n                 0.76 \\n                 0.57 \\n               \\n               \\n                 SKN39A \\n                 0.25 \\n                 0.20 \\n               \\n               \\n                 SKN669S \\n                 0.60 \\n                 1.12 \\n               \\n               \\n                 SMINT171S \\n                 1.06 \\n                 2.38 \\n               \\n               \\n                 SMINT20SM \\n                 3.20 \\n                 1.14 \\n               \\n               \\n                 SMINTH89 \\n                 1.92 \\n                 1.80 \\n               \\n               \\n                 STO261S \\n                 3.86 \\n                 0.75 \\n               \\n               \\n                 STO288S \\n                 1.00 \\n                 0.23 \\n               \\n               \\n                 STOAC93 \\n                 0.66 \\n                 2.01 \\n               \\n               \\n                 STO88S \\n                 2.57 \\n                 0.20 \\n               \\n               \\n                 THRD143N \\n                 1.77 \\n                 1.15 \\n               \\n               \\n                 THRD270T \\n                 2.23 \\n                 2.56 \\n               \\n               \\n                 THRD56T \\n                 3.02 \\n                 0.40 \\n               \\n               \\n                 TST39X \\n                 0.80 \\n                 0.77 \\n               \\n               \\n                 TST647T \\n                 1.15 \\n                 0.43 \\n               \\n               \\n                 TST663T \\n                 0.55 \\n                 0.05 \\n               \\n               \\n                 UTR135XO \\n                 0.58 \\n                 0.52 \\n               \\n               \\n                 UTR85XU \\n                 2.70 \\n                 1.49 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.19 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.93 \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.06 \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.13 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.15 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 1.03 \\n               \\n               \\n                 CVX06CV \\n                   \\n                   \\n                 0.48 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 3.34 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 HUMREF00HR \\n                 0.08 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.27 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 3.99 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 2.38 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.13 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 3.27 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 2.01 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.55 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.46 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.98 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.54 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 3.04 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.43 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr223v2 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr223v2 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr223v2 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, Up vs. NAT \\n                 22% \\n                 33% \\n                 44% \\n                 0% \\n                 20% \\n               \\n               \\n                 Sensitivity, Down vs. NAT \\n                 11% \\n                 22% \\n                 \\u20020% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Sensitivity, Up vs. NRM \\n                 11% \\n                 11% \\n                 11% \\n                 85%\\u2002 \\n                 \\u20020% \\n               \\n               \\n                 Sensitivity, Down vs. NRM \\n                 11% \\n                 78% \\n                 22% \\n                 0% \\n                 80% \\n               \\n               \\n                 Specificity \\n                 8.06%\\u2002\\u2009 \\n                 12.9%\\u2002\\u2009 \\n                 16.67%\\u2003\\u2009 \\n                 19.77%\\u2003\\u2002\\u2009 \\n                 14.89%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr223v2 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr223v2 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003305)                     (Ovr223v2_forward):   TCCAGATGGCTCAGCTTCTTC                         (SEQ\\u2003ID\\u2003NO:\\u2003306)                     (Ovr223v2_reverse):   GAAGGTGTTCGGAGAATGAGTGA                         (SEQ\\u2003ID\\u2003NO:\\u2003307)                     (Ovr223v2_probe):   TTTCTTCTGTGGCTCTGTGTTTTCCAGGC            \\nDEX0455 — 037.nt.6 (Ovr229)\\n \\n     The relative expression level of Ovr229 in various tissue samples is included below. Tissue samples include 74 pairs of matching samples, 10 non matched cancer samples, and 40 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal prostate sample PRO09PR (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.01 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.36 \\n                 0.00 \\n               \\n               \\n                 OVRG021 \\n                 0.39 \\n                 0.09 \\n               \\n               \\n                 OVR1157 \\n                 0.00 \\n               \\n               \\n                 OVR773O \\n                 0.31 \\n               \\n               \\n                 OVR988Z \\n                 1.25 \\n               \\n               \\n                 OVRC360 \\n                 1.64 \\n               \\n               \\n                 OVR1005O \\n                 0.47 \\n               \\n               \\n                 OVR1040O \\n                 1.49 \\n               \\n               \\n                 OVR105O \\n                 0.33 \\n               \\n               \\n                 OVR130X \\n                 0.00 \\n               \\n               \\n                 OVR718O \\n                 0.42 \\n               \\n               \\n                 OVRA1B \\n                 0.27 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.05 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.40 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.12 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.42 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.22 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR451O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.04 \\n                 0.14 \\n               \\n               \\n                 BLD520B \\n                 0.00 \\n                 0.23 \\n               \\n               \\n                 BLDTR17 \\n                 0.37 \\n                 0.19 \\n               \\n               \\n                 CLN401C \\n                 0.04 \\n                 0.04 \\n               \\n               \\n                 CLNAS43 \\n                 0.10 \\n                 0.14 \\n               \\n               \\n                 CLNAS98 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNCM12 \\n                 0.11 \\n                 0.12 \\n               \\n               \\n                 CLNDC19 \\n                 0.00 \\n                 0.09 \\n               \\n               \\n                 CLNRC01 \\n                 0.01 \\n                 0.02 \\n               \\n               \\n                 CLNRS53 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNSG27 \\n                 0.08 \\n                 0.31 \\n               \\n               \\n                 CLNTX01 \\n                 0.00 \\n                 0.24 \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.35 \\n               \\n               \\n                 CVXNKS55 \\n                 0.03 \\n                 0.25 \\n               \\n               \\n                 CVXNKS25 \\n                 1.68 \\n                 0.25 \\n               \\n               \\n                 CVXNKS18 \\n                 0.00 \\n                 0.06 \\n               \\n               \\n                 CVXNKS54 \\n                 0.00 \\n                 1.22 \\n               \\n               \\n                 ENDO10479 \\n                 0.13 \\n                 0.35 \\n               \\n               \\n                 ENDO28XA \\n                 0.18 \\n                 0.54 \\n               \\n               \\n                 ENDO8XA \\n                 0.00 \\n                 0.05 \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 KID12XD \\n                 0.01 \\n                 0.37 \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 KID22K \\n                 0.02 \\n                 0.06 \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 LNG205L \\n                 0.01 \\n                 1.04 \\n               \\n               \\n                 LNG315L \\n                 0.14 \\n                 1.69 \\n               \\n               \\n                 LNG507L \\n                 0.48 \\n                 3.36 \\n               \\n               \\n                 LNG528L \\n                 0.00 \\n                 0.71 \\n               \\n               \\n                 LNG8837L \\n                 0.12 \\n                 1.08 \\n               \\n               \\n                 LNGAC11 \\n                 0.10 \\n                 0.20 \\n               \\n               \\n                 LNGAC39 \\n                 0.52 \\n                 2.65 \\n               \\n               \\n                 LNGSQ80 \\n                 0.16 \\n                 2.29 \\n               \\n               \\n                 LNGSQ81 \\n                 0.23 \\n                 2.01 \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.03 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM19DN \\n                 0.00 \\n                 0.28 \\n               \\n               \\n                 MAM42DN \\n                 0.17 \\n                 0.00 \\n               \\n               \\n                 MAM517 \\n                 2.59 \\n                 0.00 \\n               \\n               \\n                 MAM781M \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM869M \\n                 0.05 \\n                 0.74 \\n               \\n               \\n                 MAM976M \\n                 0.26 \\n                 0.00 \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS699 \\n                 0.28 \\n                 0.89 \\n               \\n               \\n                 MAMS997 \\n                 0.13 \\n                 0.23 \\n               \\n               \\n                 PAN71XL \\n                 0.06 \\n                 0.09 \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                 0.05 \\n               \\n               \\n                 PAN92X \\n                 0.27 \\n                 0.00 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 8.84 \\n               \\n               \\n                 PRO23B \\n                 1.11 \\n                 1.14 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 1.16 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.93 \\n               \\n               \\n                 PRO65XB \\n                 0.14 \\n                 0.85 \\n               \\n               \\n                 PRO675P \\n                 0.42 \\n                 0.51 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.88 \\n               \\n               \\n                 PRO84XB \\n                 0.15 \\n                 3.51 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 2.76 \\n               \\n               \\n                 PRO958P \\n                 0.76 \\n                 2.69 \\n               \\n               \\n                 SKN287S \\n                 0.22 \\n                 2.01 \\n               \\n               \\n                 SKN39A \\n                 0.16 \\n                 0.00 \\n               \\n               \\n                 SKN669S \\n                 0.40 \\n                 0.00 \\n               \\n               \\n                 SMINT171S \\n                 0.02 \\n                 0.04 \\n               \\n               \\n                 SMINT20SM \\n                 0.07 \\n                 0.15 \\n               \\n               \\n                 SMINTH89 \\n                 0.05 \\n                 0.00 \\n               \\n               \\n                 STO261S \\n                 0.00 \\n                 0.11 \\n               \\n               \\n                 STO288S \\n                 0.02 \\n                 0.12 \\n               \\n               \\n                 STOAC93 \\n                 0.23 \\n                 0.05 \\n               \\n               \\n                 THRD143N \\n                 0.00 \\n                 0.27 \\n               \\n               \\n                 THRD270T \\n                 0.09 \\n                 0.07 \\n               \\n               \\n                 THRD56T \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 TST39X \\n                 0.00 \\n                 8.21 \\n               \\n               \\n                 TST647T \\n                 0.19 \\n                 9.27 \\n               \\n               \\n                 TST663T \\n                 0.14 \\n                 10.16 \\n               \\n               \\n                 UTR135XO \\n                 0.58 \\n                 0.35 \\n               \\n               \\n                 UTR85XU \\n                 0.00 \\n                 0.99 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.17 \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 2.25 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 CVX06CV \\n                   \\n                   \\n                 2.46 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.88 \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.08 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.29 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 1.46 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.67 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.04 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.80 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.46 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.15 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 12.18 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 1.54 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr229 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr229 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr229 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 \\u20020% \\n                 33% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NAT \\n               \\n               \\n                 Sensitivity, \\n                 44% \\n                 100%\\u2002 \\n                 33% \\n                 \\u20020% \\n                 60% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NAT \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 78% \\n                 67% \\n                 85% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NRM \\n               \\n               \\n                 Sensitivity, \\n                 67% \\n                 22% \\n                 33% \\n                 \\u20020% \\n                 60% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NRM \\n               \\n               \\n                 Specificity \\n                 26.06%\\u2003\\u2009 \\n                 31.38%\\u2003\\u2009 \\n                 28.19%\\u2003\\u2009 \\n                 35.96%\\u2003\\u2009 \\n                 42.11%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr229 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr229 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003308)                             (Ovr229_forward):   CCTGCCGCGGAGATCCAT                         (SEQ\\u2003ID\\u2003NO:\\u2003309)                             (Ovr229_reverse):   GCAGCGCGTACTGGTCGTA                         (SEQ\\u2003ID\\u2003NO:\\u2003310)                             (Ovr229_probe):   CCTACTCCGTGTCAGTGGTGGAG            \\nDEX0455 — 037.nt.7 (Ovr227)\\n \\n     The relative expression level of Ovr227 in various tissue samples is included below. Tissue samples include 74 pairs of matching samples, 10 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to prostate normal sample PRO09PR (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 1.31 \\n                 0.24 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 1.65 \\n                 0.75 \\n               \\n               \\n                 OVRG021 \\n                 0.87 \\n                 0.00 \\n               \\n               \\n                 OVR1157 \\n                 0.85 \\n               \\n               \\n                 OVR773O \\n                 0.21 \\n               \\n               \\n                 OVR814O \\n                 0.24 \\n               \\n               \\n                 OVRC360 \\n                 0.58 \\n               \\n               \\n                 OVR1005O \\n                 0.33 \\n               \\n               \\n                 OVR1040O \\n                 0.11 \\n               \\n               \\n                 OVR105O \\n                 0.12 \\n               \\n               \\n                 OVR130X \\n                 0.15 \\n               \\n               \\n                 OVR718O \\n                 0.32 \\n               \\n               \\n                 OVRA1B \\n                 0.11 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.40 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.06 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.16 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.08 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.63 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.02 \\n                 0.00 \\n               \\n               \\n                 BLD520B \\n                 0.00 \\n                 0.06 \\n               \\n               \\n                 BLDTR17 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLN401C \\n                 0.02 \\n                 0.04 \\n               \\n               \\n                 CLNAS43 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNAS98 \\n                 0.00 \\n                 0.09 \\n               \\n               \\n                 CLNCM12 \\n                 0.06 \\n                 0.05 \\n               \\n               \\n                 CLNDC19 \\n                 0.04 \\n                 0.10 \\n               \\n               \\n                 CLNRC01 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CLNRS53 \\n                 0.18 \\n                 0.40 \\n               \\n               \\n                 CLNSG27 \\n                 0.00 \\n                 0.28 \\n               \\n               \\n                 CLNTX01 \\n                 0.58 \\n                 0.00 \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.49 \\n               \\n               \\n                 CVXNK23 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CVXNKS54 \\n                 1.12 \\n                 2.58 \\n               \\n               \\n                 CVXNKS55 \\n                 0.01 \\n                 0.00 \\n               \\n               \\n                 CVXNKS81 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 ENDO10479 \\n                 0.00 \\n                 2.93 \\n               \\n               \\n                 ENDO28XA \\n                 0.76 \\n                 0.52 \\n               \\n               \\n                 ENDO8XA \\n                 0.03 \\n                 0.00 \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.04 \\n               \\n               \\n                 KID109XD \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 KID10XD \\n                 0.21 \\n                 0.02 \\n               \\n               \\n                 KID22K \\n                 0.01 \\n                 0.02 \\n               \\n               \\n                 LNG205L \\n                 0.00 \\n                 0.35 \\n               \\n               \\n                 LNG315L \\n                 0.33 \\n                 1.50 \\n               \\n               \\n                 LNG507L \\n                 0.24 \\n                 2.81 \\n               \\n               \\n                 LNG528L \\n                 0.00 \\n                 0.42 \\n               \\n               \\n                 LNG8837L \\n                 0.18 \\n                 1.12 \\n               \\n               \\n                 LNGAC11 \\n                 0.20 \\n                 0.04 \\n               \\n               \\n                 LNGAC39 \\n                 0.59 \\n                 1.37 \\n               \\n               \\n                 LNGSQ80 \\n                 1.38 \\n                 1.09 \\n               \\n               \\n                 LNGSQ81 \\n                 0.65 \\n                 1.59 \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.02 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.09 \\n               \\n               \\n                 MAM19DN \\n                 0.00 \\n                 0.07 \\n               \\n               \\n                 MAM42DN \\n                 0.16 \\n                 0.00 \\n               \\n               \\n                 MAM517 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM781M \\n                 0.00 \\n                 0.24 \\n               \\n               \\n                 MAM869M \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAM976M \\n                 0.12 \\n                 0.00 \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS699 \\n                 0.53 \\n                 0.00 \\n               \\n               \\n                 MAMS997 \\n                 0.20 \\n                 0.11 \\n               \\n               \\n                 PAN71XL \\n                 0.00 \\n                 0.03 \\n               \\n               \\n                 PAN82XP \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 PAN92X \\n                 0.10 \\n                 0.78 \\n               \\n               \\n                 PRO23B \\n                 0.35 \\n                 0.20 \\n               \\n               \\n                 PRO65XB \\n                 0.05 \\n                 0.61 \\n               \\n               \\n                 PRO675P \\n                 0.22 \\n                 0.40 \\n               \\n               \\n                 PRO84XB \\n                 0.12 \\n                 1.68 \\n               \\n               \\n                 PRO958P \\n                 0.18 \\n                 0.31 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.32 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.21 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.69 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.29 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.38 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 1.35 \\n               \\n               \\n                 SKN287S \\n                 0.00 \\n                 2.19 \\n               \\n               \\n                 SKN39A \\n                 0.17 \\n                 0.00 \\n               \\n               \\n                 SKN669S \\n                 0.14 \\n                 0.12 \\n               \\n               \\n                 SMINT171S \\n                 0.39 \\n                 0.15 \\n               \\n               \\n                 SMINT20SM \\n                 0.06 \\n                 0.07 \\n               \\n               \\n                 SMINTH89 \\n                 0.01 \\n                 0.00 \\n               \\n               \\n                 STO261S \\n                 0.60 \\n                 0.18 \\n               \\n               \\n                 STO288S \\n                 0.03 \\n                 0.04 \\n               \\n               \\n                 STO88S \\n                 0.00 \\n                 0.07 \\n               \\n               \\n                 THRD143N \\n                 0.01 \\n                 0.04 \\n               \\n               \\n                 THRD270T \\n                 0.03 \\n                 0.03 \\n               \\n               \\n                 THRD56T \\n                 0.00 \\n                 0.14 \\n               \\n               \\n                 TST39X \\n                 0.00 \\n                 1.74 \\n               \\n               \\n                 TST647T \\n                 0.02 \\n                 3.30 \\n               \\n               \\n                 TST663T \\n                 0.05 \\n                 0.68 \\n               \\n               \\n                 UTR135XO \\n                 0.22 \\n                 0.17 \\n               \\n               \\n                 UTR85XU \\n                 0.12 \\n                 0.19 \\n               \\n               \\n                 BLOB1 \\n                   \\n                   \\n                 7.89 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.07 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 2.25 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 1.02 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.25 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.11 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.07 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.20 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.42 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.28 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 1.33 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.38 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 2.47 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.43 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr227 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr227 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr227 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, \\n                 11% \\n                 11% \\n                 33% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NAT \\n               \\n               \\n                 Sensitivity, \\n                 67% \\n                 78% \\n                 22% \\n                 \\u20020% \\n                 40% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NAT \\n               \\n               \\n                 Sensitivity, \\n                 56% \\n                 56% \\n                 44% \\n                 100%\\u2002 \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NRM \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 22% \\n                 \\u20020% \\n                 \\u20020% \\n                 100%\\u2002 \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NRM \\n               \\n               \\n                 Specificity \\n                 28.11%\\u2003\\u2009 \\n                 40.54%\\u2003\\u2009 \\n                 25.41%\\u2003\\u2009 \\n                 42.86%\\u2003\\u2009 \\n                 39.04%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr227 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr227 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003311)                             (Ovr227_forward):   AGAGGCGCCCCCGCAGGTA                         (SEQ\\u2003ID\\u2003NO:\\u2003312)                             (Ovr227_reverse):   CCCGGAGCCAGCTCGAGTT                         (SEQ\\u2003ID\\u2003NO:\\u2003313)                             (Ovr227_probe):   CAGGAACTGCGGCGAGCGACCC            \\nDEX0455 — 040.nt.2 (Ovr218)\\n \\n     The relative expression level of Ovr218 in various tissue samples is included below. Tissue samples include 75 pairs of matching samples, 10 non matched cancer samples, and 41 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 6 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to cancer pool reference HUMREF00HR (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.46 \\n                 0.17 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 1.55 \\n                 3.95 \\n               \\n               \\n                 OVRG021 \\n                 6.74 \\n                 6.08 \\n               \\n               \\n                 OVR1157 \\n                 4.90 \\n               \\n               \\n                 OVR773O \\n                 8.80 \\n               \\n               \\n                 OVR814O \\n                 3.90 \\n               \\n               \\n                 OVRC360 \\n                 1.37 \\n               \\n               \\n                 OVR1005O \\n                 19.92 \\n               \\n               \\n                 OVR1040O \\n                 20.35 \\n               \\n               \\n                 OVR105O \\n                 5.63 \\n               \\n               \\n                 OVR130X \\n                 19.60 \\n               \\n               \\n                 OVR718O \\n                 55.03 \\n               \\n               \\n                 OVRA1B \\n                 34.34 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 1.35 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 2.50 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.98 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.23 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 3.37 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 1.42 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 1.49 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 3.54 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 3.29 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 3.49 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 1.62 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 9.36 \\n               \\n               \\n                 BLD030B \\n                 3.36 \\n                 0.72 \\n               \\n               \\n                 BLD520B \\n                 3.23 \\n                 2.25 \\n               \\n               \\n                 BLDTR17 \\n                 1.08 \\n                 1.89 \\n               \\n               \\n                 CLN401C \\n                 3.90 \\n                 3.01 \\n               \\n               \\n                 CLNAS43 \\n                 4.55 \\n                 1.92 \\n               \\n               \\n                 CLNAS98 \\n                 3.44 \\n                 2.33 \\n               \\n               \\n                 CLNCM12 \\n                 3.07 \\n                 3.22 \\n               \\n               \\n                 CLNDC19 \\n                 7.72 \\n                 2.05 \\n               \\n               \\n                 CLNRC01 \\n                 1.80 \\n                 2.17 \\n               \\n               \\n                 CLNRS53 \\n                 2.59 \\n                 3.02 \\n               \\n               \\n                 CLNSG27 \\n                 2.69 \\n                 4.49 \\n               \\n               \\n                 CLNTX01 \\n                 5.68 \\n                 5.10 \\n               \\n               \\n                 CVXKS52 \\n                 9.10 \\n                 10.59 \\n               \\n               \\n                 CVXNK23 \\n                 9.81 \\n                 41.11 \\n               \\n               \\n                 CVXNKS54 \\n                 20.97 \\n                 12.22 \\n               \\n               \\n                 CVXNKS55 \\n                 37.01 \\n                 21.89 \\n               \\n               \\n                 CVXNKS81 \\n                 17.75 \\n                 35.18 \\n               \\n               \\n                 ENDO10479 \\n                 13.27 \\n                 1.33 \\n               \\n               \\n                 ENDO28XA \\n                 13.53 \\n                 4.98 \\n               \\n               \\n                 ENDO8XA \\n                 0.34 \\n                 0.58 \\n               \\n               \\n                 KID106XD \\n                 0.28 \\n                 0.70 \\n               \\n               \\n                 KID107XD \\n                 5.01 \\n                 2.27 \\n               \\n               \\n                 KID109XD \\n                 7.16 \\n                 4.83 \\n               \\n               \\n                 KID10XD \\n                 1.34 \\n                 0.46 \\n               \\n               \\n                 KID22K \\n                 2.79 \\n                 0.65 \\n               \\n               \\n                 LNG205L \\n                 1.40 \\n                 4.10 \\n               \\n               \\n                 LNG315L \\n                 8.68 \\n                 8.32 \\n               \\n               \\n                 LNG507L \\n                 6.50 \\n                 4.85 \\n               \\n               \\n                 LNG528L \\n                 9.26 \\n                 4.03 \\n               \\n               \\n                 LNG8837L \\n                 4.36 \\n                 5.37 \\n               \\n               \\n                 LNGAC11 \\n                 2.50 \\n                 4.70 \\n               \\n               \\n                 LNGAC39 \\n                 16.03 \\n                 4.63 \\n               \\n               \\n                 LNGSQ80 \\n                 3.70 \\n                 0.84 \\n               \\n               \\n                 LNGSQ81 \\n                 14.10 \\n                 7.33 \\n               \\n               \\n                 LVR15XA \\n                 0.05 \\n                 0.03 \\n               \\n               \\n                 LVR174L \\n                 0.15 \\n                 0.12 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 9.89 \\n               \\n               \\n                 MAM19DN \\n                 17.32 \\n                 17.15 \\n               \\n               \\n                 MAM42DN \\n                 15.00 \\n                 9.52 \\n               \\n               \\n                 MAM517 \\n                 66.52 \\n                 6.34 \\n               \\n               \\n                 MAM781M \\n                 4.45 \\n                 3.02 \\n               \\n               \\n                 MAM869M \\n                 9.21 \\n                 1.73 \\n               \\n               \\n                 MAM976M \\n                 28.64 \\n                 3.82 \\n               \\n               \\n                 MAMS570 \\n                 22.00 \\n                 25.62 \\n               \\n               \\n                 MAMS699 \\n                 5.42 \\n                 5.54 \\n               \\n               \\n                 MAMS997 \\n                 10.63 \\n                 7.95 \\n               \\n               \\n                 PAN71XL \\n                 5.56 \\n                 5.74 \\n               \\n               \\n                 PAN82XP \\n                 2.41 \\n                 26.35 \\n               \\n               \\n                 PAN92X \\n                 52.91 \\n                 6.82 \\n               \\n               \\n                 PRO23B \\n                 7.13 \\n                 7.97 \\n               \\n               \\n                 PRO65XB \\n                 5.61 \\n                 6.99 \\n               \\n               \\n                 PRO675P \\n                 7.00 \\n                 4.30 \\n               \\n               \\n                 PRO84XB \\n                 7.18 \\n                 6.80 \\n               \\n               \\n                 PRO958P \\n                 6.32 \\n                 4.35 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 6.28 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 4.78 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 10.75 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 5.51 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 9.97 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 8.32 \\n               \\n               \\n                 SKN287S \\n                 6.30 \\n                 6.42 \\n               \\n               \\n                 SKN39A \\n                 4.04 \\n                 1.83 \\n               \\n               \\n                 SKN669S \\n                 6.16 \\n                 19.67 \\n               \\n               \\n                 SMINT171S \\n                 11.57 \\n                 8.96 \\n               \\n               \\n                 SMINT20SM \\n                 10.72 \\n                 4.23 \\n               \\n               \\n                 SMINTH89 \\n                 5.77 \\n                 4.77 \\n               \\n               \\n                 STO261S \\n                 8.85 \\n                 2.39 \\n               \\n               \\n                 STO288S \\n                 2.33 \\n                 1.18 \\n               \\n               \\n                 STO509L \\n                 5.78 \\n                 10.86 \\n               \\n               \\n                 STO88S \\n                 4.07 \\n                 1.01 \\n               \\n               \\n                 THRD143N \\n                 8.25 \\n                 15.21 \\n               \\n               \\n                 THRD270T \\n                 10.97 \\n                 7.35 \\n               \\n               \\n                 THRD56T \\n                 9.88 \\n                 11.23 \\n               \\n               \\n                 TST39X \\n                 9.41 \\n                 4.59 \\n               \\n               \\n                 TST647T \\n                 11.31 \\n                 1.05 \\n               \\n               \\n                 TST663T \\n                 7.35 \\n                 2.94 \\n               \\n               \\n                 UTR135XO \\n                 2.34 \\n                 5.62 \\n               \\n               \\n                 UTR85XU \\n                 17.13 \\n                 6.68 \\n               \\n               \\n                 BLOB1 \\n                   \\n                   \\n                 7.23 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 3.50 \\n               \\n               \\n                 BLOB5 \\n                   \\n                   \\n                 122.49 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 9.34 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 5.44 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 14.78 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.61 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.99 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.51 \\n               \\n               \\n                 CVX1ACV \\n                   \\n                   \\n                 14.89 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 5.63 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 1.00 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.29 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.05 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 2.25 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.93 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 2.42 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 3.63 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 3.03 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 2.74 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 2.21 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 1.19 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.87 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 5.68 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 8.67 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 9.06 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 1.93 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr218 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr218 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr218 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, Up vs. \\n                 22% \\n                 33% \\n                 33% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 NAT \\n               \\n               \\n                 Sensitivity, Down \\n                 \\u20020% \\n                 11% \\n                 \\u20020% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 vs. NAT \\n               \\n               \\n                 Sensitivity, Up \\n                 100%\\u2002 \\n                 56% \\n                 100%\\u2002 \\n                 77% \\n                 80% \\n               \\n               \\n                 vs. NRM \\n               \\n               \\n                 Sensitivity, Down \\n                 \\u20020% \\n                 \\u20020% \\n                 \\u20020% \\n                 \\u20028% \\n                 \\u20020% \\n               \\n               \\n                 vs. NRM \\n               \\n               \\n                 Specificity \\n                 6.88%\\u2002\\u2009 \\n                 9.52%\\u2002\\u2009 \\n                 20.63%\\u2003\\u2009 \\n                 8.94%\\u2002\\u2009 \\n                 9.95%\\u2002\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr218 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Additionally, the tissue specificity, plus the mRNA differential expression in the samples tested may make Ovr218 a good marker for diagnosing, monitoring, staging, imaging and/or treating breast cancer. \\n     Primers used for QPCR Expression Analysis of Ovr218 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003314)                             (Ovr218_forward):   TGCCCAGCTGTGGTTTACATTA                         (SEQ\\u2003ID\\u2003NO:\\u2003315)                             (Ovr218_reverse):   CACCACCTCGCCATTCTCA                         (SEQ\\u2003ID\\u2003NO:\\u2003316)                             (Ovr218_probe):   TTCACTGTGAACATCATCTTGGCA            \\nDEX0455 — 049.nt.1 (Ovr232)\\n \\n     The relative expression level of Ovr232 in various tissue samples is included below. Tissue samples include 73 pairs of matching samples, 10 non matched cancer samples, and 36 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 4 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to ovarian cancer sample OVRAO84 (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 1.00 \\n                 0.10 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.01 \\n                 0.31 \\n               \\n               \\n                 OVRG021 \\n                 0.39 \\n                 0.02 \\n               \\n               \\n                 OVR1157 \\n                 2.79 \\n               \\n               \\n                 OVR773O \\n                 1.14 \\n               \\n               \\n                 OVR814O \\n                 0.37 \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n               \\n               \\n                 OVR1005O \\n                 5.91 \\n               \\n               \\n                 OVR1040O \\n                 5.77 \\n               \\n               \\n                 OVR105O \\n                 1.68 \\n               \\n               \\n                 OVR130X \\n                 1.08 \\n               \\n               \\n                 OVR718O \\n                 0.55 \\n               \\n               \\n                 OVRA1B \\n                 4.48 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.26 \\n                 0.00 \\n               \\n               \\n                 BLD520B \\n                 0.13 \\n                 0.02 \\n               \\n               \\n                 BLDTR17 \\n                 0.24 \\n                 0.25 \\n               \\n               \\n                 CLN401C \\n                 3.46 \\n                 2.62 \\n               \\n               \\n                 CLNAS43 \\n                 4.08 \\n                 1.49 \\n               \\n               \\n                 CLNAS98 \\n                 1.19 \\n                 5.27 \\n               \\n               \\n                 CLNCM12 \\n                 2.46 \\n                 7.45 \\n               \\n               \\n                 CLNDC19 \\n                 9.09 \\n                 1.85 \\n               \\n               \\n                 CLNRC01 \\n                 2.55 \\n                 3.52 \\n               \\n               \\n                 CLNRS53 \\n                 1.38 \\n                 9.36 \\n               \\n               \\n                 CLNSG27 \\n                 4.28 \\n                 3.65 \\n               \\n               \\n                 CLNTX01 \\n                 3.83 \\n                 4.54 \\n               \\n               \\n                 CVXKS52 \\n                 0.15 \\n                 0.12 \\n               \\n               \\n                 CVXNK23 \\n                 0.13 \\n                 0.00 \\n               \\n               \\n                 CVXNKS54 \\n                 0.59 \\n                 0.54 \\n               \\n               \\n                 CVXNKS55 \\n                 0.58 \\n                 0.15 \\n               \\n               \\n                 CVXNKS81 \\n                 0.25 \\n                 0.61 \\n               \\n               \\n                 ENDO10479 \\n                 6.19 \\n                 1.01 \\n               \\n               \\n                 ENDO28XA \\n                 6.03 \\n                 0.82 \\n               \\n               \\n                 ENDO8XA \\n                 0.40 \\n                 1.67 \\n               \\n               \\n                 KID106XD \\n                 0.02 \\n                 0.24 \\n               \\n               \\n                 KID107XD \\n                 0.10 \\n                 0.34 \\n               \\n               \\n                 KID109XD \\n                 0.07 \\n                 0.59 \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.15 \\n               \\n               \\n                 KID22K \\n                 0.05 \\n                 0.14 \\n               \\n               \\n                 LNG205L \\n                 0.08 \\n                 1.91 \\n               \\n               \\n                 LNG315L \\n                 1.42 \\n                 0.43 \\n               \\n               \\n                 LNG507L \\n                 0.96 \\n                 0.87 \\n               \\n               \\n                 LNG528L \\n                 9.39 \\n                 0.92 \\n               \\n               \\n                 LNG8837L \\n                 1.08 \\n                 0.45 \\n               \\n               \\n                 LNGAC11 \\n                 0.28 \\n                 1.23 \\n               \\n               \\n                 LNGAC39 \\n                 13.19 \\n                 0.76 \\n               \\n               \\n                 LNGSQ80 \\n                 2.02 \\n                 0.86 \\n               \\n               \\n                 LNGSQ81 \\n                 2.19 \\n                 0.67 \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.01 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 10.06 \\n               \\n               \\n                 MAM19DN \\n                 0.46 \\n                 0.85 \\n               \\n               \\n                 MAM42DN \\n                 0.71 \\n                 0.74 \\n               \\n               \\n                 MAM517 \\n                 3.27 \\n                 0.33 \\n               \\n               \\n                 MAM781M \\n                 1.52 \\n                 0.34 \\n               \\n               \\n                 MAM976M \\n                 0.83 \\n                 0.37 \\n               \\n               \\n                 MAMS570 \\n                 0.35 \\n                 1.02 \\n               \\n               \\n                 MAMS699 \\n                 0.28 \\n                 0.39 \\n               \\n               \\n                 MAMS997 \\n                 1.23 \\n                 0.52 \\n               \\n               \\n                 PAN71XL \\n                 6.96 \\n                 4.45 \\n               \\n               \\n                 PAN82XP \\n                 0.15 \\n                 2.74 \\n               \\n               \\n                 PAN92X \\n                 2.89 \\n                 0.00 \\n               \\n               \\n                 PRO23B \\n                 0.23 \\n                 0.12 \\n               \\n               \\n                 PRO65XB \\n                 0.24 \\n                 0.50 \\n               \\n               \\n                 PRO675P \\n                 0.40 \\n                 0.21 \\n               \\n               \\n                 PRO84XB \\n                 0.45 \\n                 0.30 \\n               \\n               \\n                 PRO958P \\n                 0.22 \\n                 0.21 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.27 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.12 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.24 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.21 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.18 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.44 \\n               \\n               \\n                 SKN287S \\n                 0.38 \\n                 0.11 \\n               \\n               \\n                 SKN39A \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 SKN669S \\n                 0.03 \\n                 0.08 \\n               \\n               \\n                 SMINT171S \\n                 3.18 \\n                 4.30 \\n               \\n               \\n                 SMINT20SM \\n                 8.08 \\n                 5.63 \\n               \\n               \\n                 SMINTH89 \\n                 8.24 \\n                 3.50 \\n               \\n               \\n                 STO261S \\n                 6.10 \\n                 2.42 \\n               \\n               \\n                 STO288S \\n                 5.52 \\n                 0.23 \\n               \\n               \\n                 STO88S \\n                 2.64 \\n                 0.14 \\n               \\n               \\n                 THRD143N \\n                 1.00 \\n                 5.56 \\n               \\n               \\n                 THRD270T \\n                 8.64 \\n                 11.30 \\n               \\n               \\n                 THRD56T \\n                 3.91 \\n                 1.96 \\n               \\n               \\n                 TST39X \\n                 0.42 \\n                 0.56 \\n               \\n               \\n                 TST647T \\n                 4.38 \\n                 0.11 \\n               \\n               \\n                 TST663T \\n                 2.81 \\n                 0.13 \\n               \\n               \\n                 UTR135XO \\n                 0.40 \\n                 0.48 \\n               \\n               \\n                 UTR85XU \\n                 3.06 \\n                 1.79 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.31 \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.07 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.57 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 0.17 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.05 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 2.56 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.13 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.09 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.30 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 4.27 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.97 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.09 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.04 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.43 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.11 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.07 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr232 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr232 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr232 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, \\n                 22% \\n                 67% \\n                 44% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NAT \\n               \\n               \\n                 Sensitivity, \\n                 33% \\n                 22% \\n                 11% \\n                 \\u20020% \\n                 20% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NAT \\n               \\n               \\n                 Sensitivity, \\n                 100%\\u2002 \\n                 22% \\n                 100%\\u2002 \\n                 92% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NRM \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 44% \\n                 \\u20020% \\n                 \\u20028% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NRM \\n               \\n               \\n                 Specificity \\n                 50.82%\\u2003\\u2009 \\n                 33.88%\\u2003\\u2009 \\n                 22.95%\\u2003\\u2009 \\n                 21.84%\\u2003\\u2009 \\n                 19.46%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr232 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr232 are as follows: \\n                            (Ovr232_forward):                 (SEQ\\u2003ID\\u2003NO:\\u2003317)                         GCTCAAAGCGTGAGTAAAATATCCT                   (Ovr232_reverse):                 (SEQ\\u2003ID\\u2003NO:\\u2003318)                         CCACACTTACTTTGTAACATGATTCAGA                   (Ovr232_probe):                 (SEQ\\u2003ID\\u2003NO:\\u2003319)                         TTTGACTTAATACTTCTTTAATTGATGTGCCTTGAGTTGG            \\nDEX0455 — 049.nt.2 (Ovr232v1)\\n \\n     The relative expression level of Ovr232v1 in various tissue samples is included below. Tissue samples include 75 pairs of matching samples, 10 non matched cancer samples, and 40 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal colon sample CLN01CL (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 0.02 \\n                 0.67 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 OVRG021 \\n                 0.10 \\n                 0.00 \\n               \\n               \\n                 OVR1157 \\n                 0.00 \\n               \\n               \\n                 OVR773O \\n                 0.00 \\n               \\n               \\n                 OVR988Z \\n                 0.00 \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n               \\n               \\n                 OVR1005O \\n                 0.42 \\n               \\n               \\n                 OVR1040O \\n                 0.53 \\n               \\n               \\n                 OVR105O \\n                 0.09 \\n               \\n               \\n                 OVR130X \\n                 0.53 \\n               \\n               \\n                 OVR718O \\n                 0.25 \\n               \\n               \\n                 OVRA1B \\n                 0.26 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.05 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.38 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.01 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR451O \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLD030B \\n                 0.06 \\n                 0.09 \\n               \\n               \\n                 BLD520B \\n                 0.20 \\n                 0.03 \\n               \\n               \\n                 BLDTR17 \\n                 0.46 \\n                 0.01 \\n               \\n               \\n                 CLN401C \\n                 0.18 \\n                 0.22 \\n               \\n               \\n                 CLNAS43 \\n                 0.39 \\n                 0.21 \\n               \\n               \\n                 CLNAS98 \\n                 0.31 \\n                 0.47 \\n               \\n               \\n                 CLNCM12 \\n                 0.10 \\n                 0.20 \\n               \\n               \\n                 CLNDC19 \\n                 0.40 \\n                 0.07 \\n               \\n               \\n                 CLNRC01 \\n                 0.27 \\n                 0.13 \\n               \\n               \\n                 CLNRS53 \\n                 0.15 \\n                 0.33 \\n               \\n               \\n                 CLNSG27 \\n                 0.17 \\n                 0.25 \\n               \\n               \\n                 CLNTX01 \\n                 0.13 \\n                 0.20 \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CVXNKS55 \\n                 0.00 \\n                 0.12 \\n               \\n               \\n                 CVXNKS25 \\n                 0.85 \\n                 0.00 \\n               \\n               \\n                 CVXNKS18 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 CVXNKS54 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 ENDO10479 \\n                 0.13 \\n                 0.00 \\n               \\n               \\n                 ENDO28XA \\n                 0.32 \\n                 0.12 \\n               \\n               \\n                 ENDO8XA \\n                 0.07 \\n                 0.40 \\n               \\n               \\n                 KID106XD \\n                 0.05 \\n                 0.10 \\n               \\n               \\n                 KID12XD \\n                 0.04 \\n                 0.12 \\n               \\n               \\n                 KID10XD \\n                 0.05 \\n                 0.05 \\n               \\n               \\n                 KID22K \\n                 0.02 \\n                 0.03 \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.04 \\n               \\n               \\n                 LNG205L \\n                 0.00 \\n                 0.26 \\n               \\n               \\n                 LNG315L \\n                 0.38 \\n                 0.00 \\n               \\n               \\n                 LNG507L \\n                 0.20 \\n                 0.00 \\n               \\n               \\n                 LNG528L \\n                 0.37 \\n                 0.37 \\n               \\n               \\n                 LNG8837L \\n                 0.10 \\n                 0.06 \\n               \\n               \\n                 LNGAC11 \\n                 0.03 \\n                 0.06 \\n               \\n               \\n                 LNGAC39 \\n                 0.58 \\n                 0.63 \\n               \\n               \\n                 LNGSQ80 \\n                 0.21 \\n                 0.19 \\n               \\n               \\n                 LNGSQ81 \\n                 0.15 \\n                 0.00 \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.37 \\n               \\n               \\n                 MAM19DN \\n                 0.12 \\n                 0.25 \\n               \\n               \\n                 MAM42DN \\n                 0.44 \\n                 0.64 \\n               \\n               \\n                 MAM517 \\n                 0.25 \\n                 0.00 \\n               \\n               \\n                 MAM781M \\n                 0.24 \\n                 0.67 \\n               \\n               \\n                 MAM869M \\n                 0.04 \\n                 0.00 \\n               \\n               \\n                 MAM976M \\n                 0.22 \\n                 0.00 \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 0.47 \\n               \\n               \\n                 MAMS699 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 MAMS997 \\n                 0.11 \\n                 0.04 \\n               \\n               \\n                 PAN71XL \\n                 1.10 \\n                 0.31 \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 PAN92X \\n                 0.19 \\n                 0.00 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.20 \\n               \\n               \\n                 PRO23B \\n                 0.17 \\n                 0.10 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.54 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.27 \\n               \\n               \\n                 PRO65XB \\n                 0.17 \\n                 0.11 \\n               \\n               \\n                 PRO675P \\n                 0.47 \\n                 0.85 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 PRO84XB \\n                 0.12 \\n                 0.13 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.08 \\n               \\n               \\n                 PRO958P \\n                 0.15 \\n                 0.12 \\n               \\n               \\n                 SKN287S \\n                 0.10 \\n                 0.00 \\n               \\n               \\n                 SKN39A \\n                 0.06 \\n                 0.00 \\n               \\n               \\n                 SKN669S \\n                 0.00 \\n                 0.51 \\n               \\n               \\n                 SMINT171S \\n                 0.38 \\n                 0.67 \\n               \\n               \\n                 SMINT20SM \\n                 0.23 \\n                 0.40 \\n               \\n               \\n                 SMINTH89 \\n                 0.14 \\n                 0.31 \\n               \\n               \\n                 STO261S \\n                 0.69 \\n                 0.24 \\n               \\n               \\n                 STO288S \\n                 0.36 \\n                 0.17 \\n               \\n               \\n                 STOAC93 \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 STO88S \\n                 0.00 \\n                 0.17 \\n               \\n               \\n                 THRD143N \\n                 0.15 \\n                 0.25 \\n               \\n               \\n                 THRD270T \\n                 0.37 \\n                 0.28 \\n               \\n               \\n                 THRD56T \\n                 0.34 \\n                 0.45 \\n               \\n               \\n                 TST39X \\n                 0.20 \\n                 0.43 \\n               \\n               \\n                 TST647T \\n                 0.59 \\n                 0.41 \\n               \\n               \\n                 TST663T \\n                 0.33 \\n                 0.25 \\n               \\n               \\n                 UTR135XO \\n                 0.19 \\n                 0.13 \\n               \\n               \\n                 UTR85XU \\n                 1.42 \\n                 0.14 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.12 \\n               \\n               \\n                 CVX06CV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 0.08 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.10 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.03 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.17 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.36 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.13 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.09 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.39 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.09 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.50 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.15 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr232v1 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr232v1 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr232v1 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Sensitivity, \\n                 22% \\n                 33% \\n                 44% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NAT \\n               \\n               \\n                 Sensitivity, \\n                 22% \\n                 22% \\n                 33% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. NAT \\n               \\n               \\n                 Sensitivity, \\n                 44% \\n                 \\u20020% \\n                 44% \\n                 62% \\n                 100%\\u2002 \\n               \\n               \\n                 Up vs. NRM \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 89% \\n                 33% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. \\n               \\n               \\n                 NRM \\n               \\n               \\n                 Specificity \\n                 36.7%\\u2002\\u2009 \\n                 34.57%\\u2003\\u2009 \\n                 32.45%\\u2003\\u2009 \\n                 28.65%\\u2003\\u2009 \\n                 35.26%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr232v1 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr232v1 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003320)                     (Ovr232v1_forward):   GGCGGTGACTCATCAACGA                         (SEQ\\u2003ID\\u2003NO:\\u2003321)                     (Ovr232v1_reverse):   CATTGACGATTATTATTCACAAAGCA                         (SEQ\\u2003ID\\u2003NO:\\u2003322)                     (Ovr232v1_probe):   GCGGCCAGAGAATGTGTCTGTGAAAACT            \\nDEX0455 — 049.nt.3 (Ovr232v2)\\n \\n     The relative expression level of Ovr232v2 in various tissue samples is included below. Tissue samples include 72 pairs of matching samples, 12 non matched cancer samples, and 37 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal spleen sample SPL7GSP (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 20.82 \\n                 1.47 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 7.06 \\n                 0.00 \\n               \\n               \\n                 OVRG021 \\n                 2.01 \\n                 0.55 \\n               \\n               \\n                 OVR1157 \\n                 17.09 \\n               \\n               \\n                 OVR773O \\n                 31.56 \\n               \\n               \\n                 OVR988Z \\n                 17.68 \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n               \\n               \\n                 OVR1005O \\n                 20.28 \\n               \\n               \\n                 OVR1040O \\n                 29.36 \\n               \\n               \\n                 OVR105O \\n                 15.24 \\n               \\n               \\n                 OVR130X \\n                 10.08 \\n               \\n               \\n                 OVR718O \\n                 12.73 \\n               \\n               \\n                 OVRA1B \\n                 34.60 \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.11 \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.43 \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 1.11 \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 3.47 \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.08 \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.06 \\n               \\n               \\n                 BLD030B \\n                 6.81 \\n                 0.00 \\n               \\n               \\n                 BLD520B \\n                 4.04 \\n                 0.57 \\n               \\n               \\n                 BLDTR17 \\n                 3.89 \\n                 2.17 \\n               \\n               \\n                 CLN401C \\n                 22.89 \\n                 17.80 \\n               \\n               \\n                 CLNAS43 \\n                 72.65 \\n                 16.04 \\n               \\n               \\n                 CLNAS98 \\n                 15.32 \\n                 35.15 \\n               \\n               \\n                 CLNCM12 \\n                 17.48 \\n                 29.75 \\n               \\n               \\n                 CLNDC19 \\n                 81.83 \\n                 20.01 \\n               \\n               \\n                 CLNRC01 \\n                 20.30 \\n                 18.70 \\n               \\n               \\n                 CLNRS53 \\n                 17.98 \\n                 55.34 \\n               \\n               \\n                 CLNSG27 \\n                 59.40 \\n                 41.80 \\n               \\n               \\n                 CLNTX01 \\n                 30.45 \\n                 37.83 \\n               \\n               \\n                 CVXKS52 \\n                 3.47 \\n                 2.77 \\n               \\n               \\n                 CVXNKS55 \\n                 12.43 \\n                 2.43 \\n               \\n               \\n                 CVXNKS18 \\n                 0.00 \\n                 0.54 \\n               \\n               \\n                 CVXNKS54 \\n                 13.64 \\n                 2.13 \\n               \\n               \\n                 ENDO10479 \\n                 95.97 \\n                 4.22 \\n               \\n               \\n                 ENDO28XA \\n                 39.72 \\n                 8.50 \\n               \\n               \\n                 ENDO8XA \\n                 3.02 \\n                 11.79 \\n               \\n               \\n                 KID106XD \\n                 0.18 \\n                 1.97 \\n               \\n               \\n                 KID12XD \\n                 1.46 \\n                 10.05 \\n               \\n               \\n                 KID10XD \\n                 0.35 \\n                 1.92 \\n               \\n               \\n                 KID22K \\n                 0.65 \\n                 1.57 \\n               \\n               \\n                 KID107XD \\n                 4.13 \\n                 2.74 \\n               \\n               \\n                 LNG205L \\n                 3.09 \\n                 13.46 \\n               \\n               \\n                 LNG315L \\n                 18.48 \\n                 9.39 \\n               \\n               \\n                 LNG507L \\n                 15.67 \\n                 4.96 \\n               \\n               \\n                 LNG528L \\n                 78.28 \\n                 10.67 \\n               \\n               \\n                 LNG8837L \\n                 14.25 \\n                 6.13 \\n               \\n               \\n                 LNGAC11 \\n                 7.45 \\n                 16.04 \\n               \\n               \\n                 LNGAC39 \\n                 151.52 \\n                 5.87 \\n               \\n               \\n                 LNGSQ80 \\n                 27.78 \\n                 24.91 \\n               \\n               \\n                 LNGSQ81 \\n                 9.10 \\n                 5.92 \\n               \\n               \\n                 LVR15XA \\n                 0.27 \\n                 0.09 \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.23 \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 85.59 \\n               \\n               \\n                 MAM19DN \\n                 7.21 \\n                 18.30 \\n               \\n               \\n                 MAM42DN \\n                 29.31 \\n                 5.38 \\n               \\n               \\n                 MAM517 \\n                 13.24 \\n                 1.54 \\n               \\n               \\n                 MAM781M \\n                 26.05 \\n                 0.95 \\n               \\n               \\n                 MAM869M \\n                 4.02 \\n                 0.00 \\n               \\n               \\n                 MAM976M \\n                 13.42 \\n                 2.33 \\n               \\n               \\n                 MAMS570 \\n                 4.31 \\n                 5.78 \\n               \\n               \\n                 MAMS699 \\n                 1.12 \\n                 4.34 \\n               \\n               \\n                 MAMS997 \\n                 13.01 \\n                 5.21 \\n               \\n               \\n                 PAN71XL \\n                 64.87 \\n                 58.75 \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                 0.00 \\n               \\n               \\n                 PAN92X \\n                 26.90 \\n                 0.00 \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 2.57 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 5.10 \\n               \\n               \\n                 PRO23B \\n                 3.74 \\n                 3.66 \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 3.92 \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 1.99 \\n               \\n               \\n                 PRO65XB \\n                 3.35 \\n                 4.51 \\n               \\n               \\n                 PRO675P \\n                 8.17 \\n                 1.15 \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 10.45 \\n               \\n               \\n                 PRO84XB \\n                 5.75 \\n                 3.97 \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 3.29 \\n               \\n               \\n                 PRO958P \\n                 2.91 \\n                 5.35 \\n               \\n               \\n                 SKN287S \\n                 5.73 \\n                 0.91 \\n               \\n               \\n                 SKN39A \\n                 0.00 \\n               \\n               \\n                 SKN669S \\n                 0.13 \\n                 2.14 \\n               \\n               \\n                 SMINT171S \\n                 56.03 \\n                 62.72 \\n               \\n               \\n                 SMINT20SM \\n                 106.47 \\n                 33.80 \\n               \\n               \\n                 SMINTH89 \\n                 96.97 \\n                 40.02 \\n               \\n               \\n                 STO261S \\n                 118.64 \\n                 19.05 \\n               \\n               \\n                 STO288S \\n                 47.55 \\n                 4.07 \\n               \\n               \\n                 STOAC93 \\n                 67.18 \\n                 64.23 \\n               \\n               \\n                 STO88S \\n                 79.32 \\n               \\n               \\n                 THRD143N \\n                 14.71 \\n                 30.26 \\n               \\n               \\n                 THRD270T \\n                 43.65 \\n                 40.86 \\n               \\n               \\n                 THRD56T \\n                 23.82 \\n                 8.72 \\n               \\n               \\n                 TST39X \\n                 6.89 \\n                 5.65 \\n               \\n               \\n                 TST647T \\n                 30.28 \\n                 3.55 \\n               \\n               \\n                 TST663T \\n                 23.55 \\n                 1.69 \\n               \\n               \\n                 UTR135XO \\n                 2.75 \\n                 5.63 \\n               \\n               \\n                 UTR85XU \\n                 32.07 \\n                 28.53 \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 2.60 \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.34 \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.47 \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 24.82 \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 HUMREF00HR \\n                 4.31 \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 2.28 \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 10.08 \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 1.17 \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.02 \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.61 \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 8.47 \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 95.94 \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 16.37 \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 1.00 \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 2.19 \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.83 \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 6.78 \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 1.57 \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 2.24 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detcted \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr232v2 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr232v2 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr232v2 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, Up vs. NAT \\n                 22% \\n                 44% \\n                 67% \\n                 0% \\n                 20% \\n               \\n               \\n                 Sensitivity, Down vs. NAT \\n                 22% \\n                 22% \\n                 22% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Sensitivity, Up vs. NRM \\n                 33% \\n                 33% \\n                 89% \\n                 92%\\u2002 \\n                 \\u20020% \\n               \\n               \\n                 Sensitivity, Down vs. NRM \\n                 \\u20020% \\n                 11% \\n                 \\u20020% \\n                 8% \\n                 60% \\n               \\n               \\n                 Specificity \\n                 36.46%\\u2003\\u2009 \\n                 29.28%\\u2003\\u2009 \\n                 25.41%\\u2003\\u2009 \\n                 24.28%\\u2003\\u2002\\u2009 \\n                 19.13%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr232v2 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr232v2 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003323)                     (Ovr232v2_forward):   CCTTTTTATCCACTTACAGATCAACCA                         (SEQ\\u2003ID\\u2003NO:\\u2003324)                     (Ovr232v2_reverse):   ACAAGCAAGATGCATGTGAGTGA                         (SEQ\\u2003ID\\u2003NO:\\u2003325)                     (Ovr232v2_probe):   ATGGTTCGCTGCTGCCGTT            \\nDEX0455 — 049.nt.4 (Ovr232v3)\\n \\n     The relative expression level of Ovr232v3 in various tissue samples is included below. Tissue samples include 75 pairs of matching samples, 10 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to normal lung sample LNG90LN (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 OVRA084 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.07 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG021 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1157 \\n                 0.01 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR773O \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR988Z \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1005O \\n                 0.52 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1040O \\n                 0.55 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR105O \\n                 0.25 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR130X \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR718O \\n                 0.29 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRA1B \\n                 0.22 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR451O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLD030B \\n                 0.12 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLD520B \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLDTR17 \\n                 0.00 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLN401C \\n                 0.57 \\n                 0.24 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS43 \\n                 1.60 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS98 \\n                 0.86 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNCM12 \\n                 0.06 \\n                 0.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNDC19 \\n                 0.47 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRC01 \\n                 0.12 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRS53 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNSG27 \\n                 1.08 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNTX01 \\n                 0.00 \\n                 0.41 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS55 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS25 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS18 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS54 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO10479 \\n                 0.30 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO28XA \\n                 0.19 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO8XA \\n                 0.00 \\n                 0.46 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID12XD \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID22K \\n                 0.00 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG205L \\n                 0.00 \\n                 0.68 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG315L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG507L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG528L \\n                 1.50 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG8837L \\n                 0.96 \\n                 0.81 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC11 \\n                 0.03 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC39 \\n                 0.35 \\n                 1.20 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ80 \\n                 0.87 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ81 \\n                 0.65 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR15XA \\n                 0.10 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.38 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM19DN \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM42DN \\n                 0.00 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM517 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM781M \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM869M \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM976M \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS699 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS997 \\n                 0.05 \\n                 0.21 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN71XL \\n                 0.00 \\n                 0.64 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN92X \\n                 0.19 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO23B \\n                 0.04 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.00 \\n                   \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.00 \\n                   \\n               \\n               \\n                 PRO65XB \\n                 0.09 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO675P \\n                 0.68 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.09 \\n                   \\n               \\n               \\n                 PRO84XB \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.01 \\n                   \\n               \\n               \\n                 PRO958P \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN287S \\n                 0.06 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN39A \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN669S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT171S \\n                 0.03 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT20SM \\n                 0.55 \\n                 0.24 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINTH89 \\n                 0.00 \\n                 0.47 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO261S \\n                 1.03 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO288S \\n                 0.54 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STOAC93 \\n                 0.00 \\n                 2.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO88S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD143N \\n                 0.51 \\n                 2.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD270T \\n                 0.49 \\n                 0.97 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD56T \\n                 0.79 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST39X \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST647T \\n                 0.52 \\n                 0.59 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST663T \\n                 0.40 \\n                 0.46 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR135XO \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR85XU \\n                 0.77 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 CVX06CV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 1.00 \\n                   \\n                   \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.06 \\n                   \\n                   \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 1.27 \\n                   \\n                   \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 1.21 \\n                   \\n                   \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr232v3 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr232v3 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr232v3 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, \\n                 56% \\n                 44% \\n                 \\u20020% \\n                 0% \\n                 60% \\n               \\n               \\n                 Up vs. NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 11% \\n                 22% \\n                 22% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 78% \\n                 \\u20020% \\n                 \\u20020% \\n                 62%\\u2002 \\n                 60% \\n               \\n               \\n                 Up vs. NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 22% \\n                 56% \\n                 89% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Specificity \\n                 72.73%\\u2003\\u2009 \\n                 70.59%\\u2003\\u2009 \\n                 61.5%\\u2002\\u2009 \\n                 62.36%\\u2003\\u2002\\u2009 \\n                 63.49%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr232v3 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr232v3 are as follows: \\n                            (Ovr232v3_forward):                 (SEQ\\u2003ID\\u2003NO:\\u2003326)                         CCTCACTTCGCAGCTTTGCT                   (Ovr232v3_reverse):                 (SEQ\\u2003ID\\u2003NO:\\u2003327)                         CTGGCATTGACGATTATTATTCACA                   (Ovr232v3probe):                 (SEQ\\u2003ID\\u2003NO:\\u2003328)                         CTGTGAAAACTACAAGCTGGCCGTAAACTGCT            \\nDEX0455 — 052.nt.2 (Ovr107v1)\\n \\n     The relative expression level of Ovr107v1 in various tissue samples is included below. Tissue samples include 69 pairs of matching samples, 14 non matched cancer samples, and 33 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 2 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to prostate normal sample PRO09PR (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 OVRA084 \\n                 1.11 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.00 \\n                 6.59 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG021 \\n                 0.36 \\n                 0.35 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1157 \\n                 3.79 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR773O \\n                 7.68 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR814O \\n                 1.90 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1005O \\n                 4.09 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1040O \\n                 3.29 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR105O \\n                 4.05 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR130X \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR718O \\n                 0.84 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRA1B \\n                 3.99 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.12 \\n                   \\n                   \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.14 \\n                   \\n                   \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.06 \\n                   \\n                   \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.22 \\n                   \\n                   \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.42 \\n                   \\n                   \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.02 \\n                   \\n                   \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLD030B \\n                 0.79 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLD520B \\n                 0.10 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLDTR17 \\n                 2.53 \\n                 1.19 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLN401C \\n                 0.26 \\n                 0.44 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS43 \\n                 4.02 \\n                 1.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS98 \\n                 1.42 \\n                 0.50 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNCM12 \\n                 1.48 \\n                 0.45 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNDC19 \\n                 2.32 \\n                 0.79 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRC01 \\n                 0.33 \\n                 0.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRS53 \\n                 0.31 \\n                 0.88 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNSG27 \\n                 2.00 \\n                 1.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNTX01 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXKS52 \\n                 1.77 \\n                 3.80 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNK23 \\n                 1.76 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS54 \\n                 2.77 \\n                 3.22 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS55 \\n                 6.45 \\n                 9.73 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS81 \\n                 2.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO10479 \\n                 5.01 \\n                 1.45 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO28XA \\n                 5.66 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO8XA \\n                 0.85 \\n                 0.18 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.61 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID107XD \\n                 0.44 \\n                 1.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID109XD \\n                 2.85 \\n                 0.99 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.09 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID22K \\n                 0.32 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG205L \\n                 0.26 \\n                 1.68 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG315L \\n                 0.44 \\n                 0.44 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG507L \\n                 0.24 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG528L \\n                 0.19 \\n                 0.17 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG8837L \\n                 1.07 \\n                 0.62 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC11 \\n                 0.63 \\n                 0.30 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC39 \\n                 1.29 \\n                 1.24 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ80 \\n                 1.39 \\n                 0.25 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ81 \\n                 1.23 \\n                 0.56 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR187L \\n                 0.25 \\n                 0.86 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM19DN \\n                 1.91 \\n                 1.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM42DN \\n                 0.36 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM517 \\n                   \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM781M \\n                 0.00 \\n                 0.53 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM869M \\n                 1.40 \\n                 1.23 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM976M \\n                 2.55 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS570 \\n                 0.00 \\n                 1.69 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS699 \\n                 1.35 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS997 \\n                 2.41 \\n                 1.23 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN71XL \\n                 0.72 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN82XP \\n                 0.71 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN92X \\n                 5.33 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO23B \\n                 1.06 \\n                 0.93 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO65XB \\n                 0.61 \\n                 0.70 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO675P \\n                 0.57 \\n                 0.48 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO84XB \\n                 0.62 \\n                 0.75 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO958P \\n                 1.10 \\n                 1.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 1.38 \\n                   \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.66 \\n                   \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 2.26 \\n                   \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.76 \\n                   \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.26 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.36 \\n               \\n               \\n                 SKN287S \\n                 2.27 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN39A \\n                 0.54 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN669S \\n                 0.52 \\n                 6.42 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT171S \\n                 1.91 \\n                 0.09 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT20SM \\n                 3.08 \\n                 1.13 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINTH89 \\n                 1.92 \\n                 1.28 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO261S \\n                 1.20 \\n                 0.35 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO288S \\n                 0.14 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO88S \\n                 0.58 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD143N \\n                 1.09 \\n                 6.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD270T \\n                 5.60 \\n                 6.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD56T \\n                 2.63 \\n                 2.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST39X \\n                 0.58 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST647T \\n                 0.41 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST663T \\n                 0.95 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR135XO \\n                 0.63 \\n                 1.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR85XU \\n                 0.00 \\n                 0.19 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.35 \\n                   \\n                   \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.79 \\n                   \\n                   \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 1.69 \\n                   \\n                   \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HUMREF00HR \\n                 0.67 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.17 \\n                   \\n                   \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.36 \\n                   \\n                   \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.55 \\n                   \\n                   \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.33 \\n                   \\n                   \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.24 \\n                   \\n                   \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 5.67 \\n                   \\n                   \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 1.00 \\n                   \\n                   \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.51 \\n                   \\n                   \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.12 \\n                   \\n                   \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.08 \\n                   \\n                   \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 2.33 \\n                   \\n                   \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.20 \\n                   \\n                   \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 2.37 \\n                   \\n                   \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.33 \\n                   \\n                   \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.32 \\n               \\n               \\n                   \\n               \\n               \\n                 Note: \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr107v1 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr107v1 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr107v1 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, Up vs. NAT \\n                 56% \\n                 44% \\n                 38% \\n                 \\u20020% \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NAT \\n                 11% \\n                 11% \\n                 25% \\n                 \\u20020% \\n                 0% \\n               \\n               \\n                 Sensitivity, Up vs. NRM \\n                 33% \\n                 44% \\n                 63% \\n                 77% \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NRM \\n                 44% \\n                 \\u20020% \\n                 25% \\n                 23% \\n                 0% \\n               \\n               \\n                 Specificity \\n                 25.86%\\u2003\\u2009 \\n                 21.26%\\u2003\\u2009 \\n                 22.86%\\u2003\\u2009 \\n                 25.15%\\u2003\\u2009 \\n                 21.59%\\u2003\\u2002\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr107v1 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr107v1 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003329)                             (Ovr107v1_forward):   CGCCTGACCCGACTGTCTTA                         (SEQ\\u2003ID\\u2003NO:\\u2003330)                             (Ovr107v1_reverse):   GCTCAGATTCTGGCTCCAAGTCT                         (SEQ\\u2003ID\\u2003NO:\\u2003331)                             (Ovr107v1_probe):   CCTACAGCAAAGCGCCCCCCA            \\nDEX0455 — 052.nt.4 (Ovr107v3)\\n \\n     The relative expression level of Ovr107v3 in various tissue samples is included below. Tissue samples include 73 pairs of matching samples, 11 non matched cancer samples, and 37 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 4 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to ovarian cancer sample OVR8140 (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 OVRA084 \\n                 0.32 \\n                 0.10 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.01 \\n                 4.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG021 \\n                 0.17 \\n                 0.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1157 \\n                 2.88 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR773O \\n                 6.48 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR814O \\n                 1.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRC360 \\n                 0.11 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1005O \\n                 1.56 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1040O \\n                 1.09 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR105O \\n                 0.68 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR130X \\n                 1.17 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR718O \\n                 0.79 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRA1B \\n                 1.43 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.06 \\n                   \\n                   \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.08 \\n                   \\n                   \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.09 \\n                   \\n                   \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.04 \\n                   \\n                   \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.05 \\n                   \\n                   \\n               \\n               \\n                 BLD030B \\n                 0.16 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLD520B \\n                 0.09 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLDTR17 \\n                 0.06 \\n                 0.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLN401C \\n                 0.09 \\n                 0.10 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS43 \\n                 0.24 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS98 \\n                 0.14 \\n                 0.11 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNCM12 \\n                 0.05 \\n                 0.11 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNDC19 \\n                 0.40 \\n                 0.14 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRC01 \\n                 0.05 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRS53 \\n                 0.06 \\n                 0.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNSG27 \\n                 0.11 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNTX01 \\n                 0.07 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXKS52 \\n                 0.50 \\n                 1.56 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNK23 \\n                 0.51 \\n                 2.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS54 \\n                 0.56 \\n                 0.93 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS55 \\n                 1.32 \\n                 3.28 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS81 \\n                 0.55 \\n                 1.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO10479 \\n                 1.12 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO28XA \\n                 1.33 \\n                 0.11 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO8XA \\n                 0.30 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID106XD \\n                 0.01 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID107XD \\n                 0.03 \\n                 0.13 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID109XD \\n                 0.25 \\n                 0.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID10XD \\n                 0.02 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID22K \\n                 0.07 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG205L \\n                 0.03 \\n                 0.05 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG315L \\n                 0.03 \\n                 0.08 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG507L \\n                 0.58 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG528L \\n                 0.29 \\n                 0.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG8837L \\n                 0.09 \\n                 0.17 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC11 \\n                 0.14 \\n                 0.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC39 \\n                 0.50 \\n                 0.08 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ80 \\n                 0.18 \\n                 0.22 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ81 \\n                 0.07 \\n                 0.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR174L \\n                 0.01 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR187L \\n                 0.01 \\n                 0.19 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM19DN \\n                 0.62 \\n                 0.28 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM42DN \\n                 0.57 \\n                 0.37 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM517 \\n                 2.06 \\n                 0.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM781M \\n                 0.07 \\n                 0.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM869M \\n                 0.67 \\n                 0.11 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM976M \\n                 0.60 \\n                 0.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS570 \\n                 0.72 \\n                 0.76 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS699 \\n                 0.10 \\n                 0.46 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS997 \\n                 0.18 \\n                 0.34 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN71XL \\n                 0.09 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN82XP \\n                 0.12 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN92X \\n                 2.58 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO23B \\n                 0.23 \\n                 0.27 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO65XB \\n                 0.22 \\n                 0.25 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO675P \\n                 0.40 \\n                 0.19 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO84XB \\n                 0.34 \\n                 0.42 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO958P \\n                 0.38 \\n                 0.22 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.30 \\n                   \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.24 \\n                   \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.93 \\n                   \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.44 \\n                   \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.32 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.19 \\n               \\n               \\n                 SKN287S \\n                 0.86 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN39A \\n                 0.03 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN669S \\n                 0.12 \\n                 0.40 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT171S \\n                 0.21 \\n                 0.04 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT20SM \\n                 2.32 \\n                 0.40 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINTH89 \\n                 0.51 \\n                 0.05 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO261S \\n                 0.65 \\n                 0.05 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO288S \\n                 0.08 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO88S \\n                 0.15 \\n                 0.07 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD143N \\n                 0.09 \\n                 0.73 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD270T \\n                 1.23 \\n                 1.14 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD56T \\n                 0.64 \\n                 0.18 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST39X \\n                 0.07 \\n                 0.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST647T \\n                 0.11 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST663T \\n                 0.12 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR135XO \\n                 0.13 \\n                 0.27 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR85XU \\n                 0.15 \\n                 0.09 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLOB6 \\n                   \\n                   \\n                 0.69 \\n                   \\n                   \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.02 \\n                   \\n                   \\n               \\n               \\n                 BLO982B \\n                   \\n                   \\n                 0.10 \\n                   \\n                   \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.02 \\n                   \\n                   \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.05 \\n                   \\n                   \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.93 \\n                   \\n                   \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HUMREF00HR \\n                 0.09 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.05 \\n                   \\n                   \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.04 \\n                   \\n                   \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 0.15 \\n                   \\n                   \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.07 \\n                   \\n                   \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.10 \\n                   \\n                   \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.82 \\n                   \\n                   \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.50 \\n                   \\n                   \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.26 \\n                   \\n                   \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 1.10 \\n                   \\n                   \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.02 \\n                   \\n                   \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.32 \\n                   \\n                   \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.01 \\n                   \\n                   \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.08 \\n               \\n               \\n                   \\n               \\n               \\n                 Note: \\n               \\n               \\n                 0.00 = Negative or Not Detected \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr107v3 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr107v3 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr107v3 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, \\n                 33% \\n                 33% \\n                 44% \\n                 0% \\n                 20% \\n               \\n               \\n                 Up vs. NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 22% \\n                 22% \\n                 11% \\n                 0% \\n                 \\u20020% \\n               \\n               \\n                 Down vs. \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 44% \\n                 67% \\n                 67% \\n                 92%\\u2002 \\n                 \\u20020% \\n               \\n               \\n                 Up vs. NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, \\n                 \\u20020% \\n                 \\u20020% \\n                 11% \\n                 8% \\n                 40% \\n               \\n               \\n                 Down vs. \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Specificity \\n                 8.79%\\u2002\\u2009 \\n                 10.44%\\u2003\\u2009 \\n                 33.52%\\u2003\\u2009 \\n                 43.35%\\u2003\\u2002\\u2009 \\n                 21.74%\\u2003\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr107v3 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr107v3 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003332)                             (Ovr107v3_forward):   CCTGCAGCCCAGAGCAAT                         (SEQ\\u2003ID\\u2003NO:\\u2003333)                             (Ovr107v3_reverse):   GCTCAGATTCTGGCTCCAAGTC                         (SEQ\\u2003ID\\u2003NO:\\u2003334)                             (Ovr107v3_probe):   ATCTCCAACCCTCCCGCTTCT            \\nDEX0455 — 051.nt.6 (Ovr107v4)\\n \\n     The relative expression level of Ovr107v4 in various tissue samples is included below. Tissue samples include 69 pairs of matching samples, 15 non matched cancer samples, and 34 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 2 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to breast normal sample MAM01 MA (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 OVRA084 \\n                 1.03 \\n                 0.60 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.43 \\n                 1.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG021 \\n                 0.72 \\n                 1.63 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1157 \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR773O \\n                 2.63 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR814O \\n                 1.26 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRC360 \\n                 0.46 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1005O \\n                 13.92 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1040O \\n                 6.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR105O \\n                 6.04 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR718O \\n                 4.15 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRA1B \\n                 3.67 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR247A \\n                   \\n                   \\n                 0.55 \\n                   \\n                   \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 1.06 \\n                   \\n                   \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.35 \\n                   \\n                   \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.43 \\n                   \\n                   \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.93 \\n                   \\n                   \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 2.17 \\n                   \\n                   \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 1.17 \\n                   \\n                   \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.68 \\n                   \\n                   \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 1.89 \\n                   \\n                   \\n               \\n               \\n                 OVRC004 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLD030B \\n                 0.77 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLD520B \\n                 2.75 \\n                 0.88 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLDTR17 \\n                 0.70 \\n                 2.67 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLN401C \\n                 0.78 \\n                 1.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS43 \\n                 2.36 \\n                 0.77 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS98 \\n                 1.73 \\n                 1.27 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNCM12 \\n                 0.67 \\n                 0.61 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNDC19 \\n                 1.46 \\n                 0.43 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRC01 \\n                 0.12 \\n                 0.36 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRS53 \\n                 0.36 \\n                 2.08 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNSG27 \\n                 0.48 \\n                 2.08 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNTX01 \\n                 0.72 \\n                 0.56 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXKS52 \\n                 1.32 \\n                 10.88 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNK23 \\n                 2.75 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS54 \\n                 1.33 \\n                 10.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS55 \\n                 9.56 \\n                 20.77 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS81 \\n                 3.27 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO10479 \\n                 3.77 \\n                 4.17 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO28XA \\n                 5.41 \\n                 4.55 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO8XA \\n                 1.21 \\n                 1.31 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID106XD \\n                 0.27 \\n                 0.12 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID107XD \\n                 0.60 \\n                 0.45 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID109XD \\n                 2.94 \\n                 0.76 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID10XD \\n                 0.18 \\n                 0.28 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID22K \\n                 0.80 \\n                 0.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG205L \\n                 0.46 \\n                 1.80 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG315L \\n                 0.37 \\n                 2.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG507L \\n                 1.43 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG528L \\n                 1.26 \\n                 0.85 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG8837L \\n                 0.86 \\n                 1.74 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC11 \\n                 0.77 \\n                 1.37 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC39 \\n                 1.28 \\n                 1.22 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ80 \\n                 1.34 \\n                 2.91 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ81 \\n                 0.95 \\n                 1.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR15XA \\n                 0.05 \\n                 0.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR174L \\n                 0.10 \\n                 0.05 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.86 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM19DN \\n                 1.31 \\n                 3.79 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM42DN \\n                 1.98 \\n                 3.48 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM517 \\n                 3.35 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM781M \\n                 0.57 \\n                 0.51 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM869M \\n                 2.29 \\n                 1.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM976M \\n                 3.78 \\n                 2.13 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS570 \\n                 2.14 \\n                 3.13 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS699 \\n                 0.58 \\n                 4.99 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS997 \\n                 2.72 \\n                 1.84 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN71XL \\n                 0.76 \\n                 0.24 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN82XP \\n                 1.49 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN92X \\n                 4.92 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO23B \\n                 0.87 \\n                 1.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO65XB \\n                 0.62 \\n                 0.72 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO675P \\n                 1.19 \\n                 2.30 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO84XB \\n                 0.99 \\n                 2.38 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO958P \\n                 1.31 \\n                 1.39 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 1.64 \\n                   \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.60 \\n                   \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 3.10 \\n                   \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.92 \\n                   \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 1.33 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 2.41 \\n               \\n               \\n                 SKN287S \\n                 5.46 \\n                 0.65 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN39A \\n                 2.56 \\n                 0.22 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN669S \\n                 6.12 \\n                 9.44 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT171S \\n                 1.39 \\n                 0.62 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT20SM \\n                 7.46 \\n                 2.59 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINTH89 \\n                 0.97 \\n                 0.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO261S \\n                 4.97 \\n                 3.16 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO288S \\n                 0.23 \\n                 0.40 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO88S \\n                 3.10 \\n                 0.38 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD143N \\n                 0.70 \\n                 5.66 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD270T \\n                 11.59 \\n                 12.76 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD56T \\n                 4.61 \\n                 1.92 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST39X \\n                 0.91 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST647T \\n                 1.42 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST663T \\n                 1.42 \\n                 0.37 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR135XO \\n                 3.28 \\n                 4.02 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR85XU \\n                 1.51 \\n                 2.11 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.25 \\n                   \\n                   \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.92 \\n                   \\n                   \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.22 \\n                   \\n                   \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 7.88 \\n                   \\n                   \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.06 \\n                   \\n                   \\n               \\n               \\n                 HUMREF00HR \\n                 0.49 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.10 \\n                   \\n                   \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.03 \\n                   \\n                   \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.26 \\n                   \\n                   \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 1.00 \\n                   \\n                   \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.06 \\n                   \\n                   \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 1.02 \\n                   \\n                   \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.14 \\n                   \\n                   \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 2.01 \\n                   \\n                   \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.57 \\n                   \\n                   \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 1.21 \\n                   \\n                   \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.11 \\n                   \\n                   \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.26 \\n                   \\n                   \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 1.56 \\n                   \\n                   \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.20 \\n                   \\n                   \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 1.45 \\n                   \\n                   \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.19 \\n                   \\n                   \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 1.28 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or no expression \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr107v4 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr107v4 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr107v4 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, Up vs. \\n                 22% \\n                 0% \\n                 22% \\n                 \\u20020% \\n                 \\u20020% \\n               \\n               \\n                 NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, Down vs. \\n                 33% \\n                 50%\\u2002 \\n                 22% \\n                 \\u20020% \\n                 20% \\n               \\n               \\n                 NAT \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, Up vs. \\n                 78% \\n                 78%\\u2002 \\n                 56% \\n                 50% \\n                 40% \\n               \\n               \\n                 NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Sensitivity, Down vs. \\n                 \\u20020% \\n                 0% \\n                 \\u20020% \\n                 25% \\n                 \\u20020% \\n               \\n               \\n                 NRM \\n                   \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Specificity \\n                 8.05%\\u2002\\u2009 \\n                 8% \\n                 13.79%\\u2003\\u2009 \\n                 13.17%\\u2003\\u2009 \\n                 7.95%\\u2002\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr107v4 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Primers used for QPCR Expression Analysis of Ovr107v4 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003335)                             (Ovr107v4_forward):   GGAGCCCTGAGCATTGTAATATG                         (SEQ\\u2003ID\\u2003NO:\\u2003336)                             (Ovr107v4_reverse):   CCCTGGTAGCCGGGTAGAG                         (SEQ\\u2003ID\\u2003NO:\\u2003337)                             (Ovr107v4_probe):   CAGATGGTGTGCCAACTGCTGT            \\nDEX0455 — 053.nt.2 (Ovr110v1)\\n \\n     The relative expression level of Ovr110v1 in various tissue samples is included below. Tissue samples include 74 pairs of matching samples, 111 non matched cancer samples, and 39 normal samples, all from various tissues annotated in the table. A matching pair is formed by mRNA from the cancer sample for a particular tissue and mRNA from the normal adjacent sample for that same tissue from the same individual. Of the normal samples 5 were blood samples which measured the expression levels in blood cells. Additionally, 2 prostatitis, and 4 Benign Prostatic Hyperplasia (BPH) samples are included. All the values are compared to breast normal sample MAM01MA (calibrator). \\n     The table below contains the relative expression level values for the sample as compared to the calibrator. The table includes the Sample ID, and expression level values for the following samples: Cancer (CAN), Normal Adjacent Tissue (NAT), Normal Tissue (NRM), Benign Prostatic Hyperplasia (BPH), and Prostatitis (PROST). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                 Sample ID \\n                 CAN \\n                 NAT \\n                 NRM \\n                 BPH \\n                 PROST \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 OVRA084 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG010 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRG021 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1157 \\n                 4.92 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR773O \\n                 4.23 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRC360 \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1005O \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR1040O \\n                 0.11 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR105O \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR130X \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR718O \\n                 0.33 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVRA1B \\n                 0.07 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 OVR35GA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC087 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC109 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR206I \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR515O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR18GA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR337O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR123O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVRC177 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR40G \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR451O \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLD030B \\n                 0.00 \\n                 0.53 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLD520B \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLDTR17 \\n                 0.00 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLN401C \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS43 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNAS98 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNCM12 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNDC19 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRC01 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNRS53 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNSG27 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CLNTX01 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXKS52 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS55 \\n                 0.03 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS25 \\n                 0.00 \\n                 0.29 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS18 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 CVXNKS54 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO10479 \\n                 0.10 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO28XA \\n                 0.78 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 ENDO8XA \\n                 0.00 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID106XD \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID12XD \\n                 0.01 \\n                 0.15 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID10XD \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID22K \\n                 0.00 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID107XD \\n                 0.00 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG205L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG315L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG507L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG528L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNG8837L \\n                 0.21 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC11 \\n                 0.01 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGAC39 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ80 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LNGSQ81 \\n                 0.08 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR15XA \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR174L \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 LVR187L \\n                 0.00 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM19DN \\n                 0.36 \\n                 1.23 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM42DN \\n                 0.09 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM517 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM781M \\n                 0.47 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM869M \\n                 0.46 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAM976M \\n                 0.22 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS570 \\n                 0.55 \\n                 0.45 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS699 \\n                 0.22 \\n                 1.06 \\n                   \\n                   \\n                   \\n               \\n               \\n                 MAMS997 \\n                 0.73 \\n                 0.21 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN71XL \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN77X \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 PAN92X \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO10R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO20R \\n                   \\n                   \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                 PRO23B \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO263C \\n                   \\n                   \\n                   \\n                 0.00 \\n                   \\n               \\n               \\n                 PRO276P \\n                   \\n                   \\n                   \\n                 0.01 \\n                   \\n               \\n               \\n                 PRO65XB \\n                 0.01 \\n                 0.01 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO675P \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO767B \\n                   \\n                   \\n                   \\n                 0.35 \\n                   \\n               \\n               \\n                 PRO84XB \\n                 0.00 \\n                 0.08 \\n                   \\n                   \\n                   \\n               \\n               \\n                 PRO855P \\n                   \\n                   \\n                   \\n                 0.00 \\n                   \\n               \\n               \\n                 PRO958P \\n                 0.03 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN287S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN39A \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SKN669S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT171S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINT20SM \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 SMINTH89 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO261S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO288S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STOAC93 \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 STO88S \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD143N \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD270T \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 THRD56T \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST39X \\n                 0.84 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST647T \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 TST663T \\n                 0.04 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR135XO \\n                 0.00 \\n                 0.00 \\n                   \\n                   \\n                   \\n               \\n               \\n                 UTR85XU \\n                 0.00 \\n                 0.03 \\n                   \\n                   \\n                   \\n               \\n               \\n                 BLOB3 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLOB11 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO69 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO72 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BLO73 \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 ADR48AD \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 BRN10BR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 CLN01CL \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 CVX06CV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 ESO01ES \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HRT46HR \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 HUMREF00HR \\n                 0.00 \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 KID55KD \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 LVR89LV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 LNG90LN \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 MAM01MA \\n                   \\n                   \\n                 1.00 \\n                   \\n                   \\n               \\n               \\n                 MSL84MU \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 OVR3APV \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 PAN04PA \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 PLA59PL \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 PRO09PR \\n                   \\n                   \\n                 0.51 \\n                   \\n                   \\n               \\n               \\n                 REC21RC \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 SMINT59SM \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 SPL7GSP \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 STO09ST \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 THYM99TM \\n                   \\n                   \\n                 0.00 \\n                   \\n                   \\n               \\n               \\n                 TRA16TR \\n                   \\n                   \\n                 0.15 \\n                   \\n                   \\n               \\n               \\n                 TST4GTS \\n                   \\n                   \\n                 0.15 \\n                   \\n                   \\n               \\n               \\n                 UTR57UT \\n                   \\n                   \\n                 0.00 \\n               \\n               \\n                   \\n               \\n               \\n                 0.00 = Negative or no expression \\n               \\n            \\n           \\n         \\n       \\n     \\n     The sensitivity for Ovr110v1 expression was calculated for the cancer samples versus normal samples. The sensitivity value indicates the percentage of cancer samples that show levels of Ovr110v1 at least 2 fold higher than the normal tissue or the corresponding normal adjacent form the same patient. \\n     This specificity is an indication of the level of ovary tissue specific expression of the transcript compared to all the other tissue types tested in our assay. Thus, these experiments indicate Ovr110v1 being useful as an ovarian cancer diagnostic marker and/or therapeutic target. \\n     Sensitivity and specificity data is reported in the table below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                   \\n               \\n               \\n                   \\n                 CLN \\n                 LNG \\n                 MAM \\n                 OVR \\n                 PRO \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Sensitivity, Up vs. NAT \\n                 0% \\n                 33% \\n                 56% \\n                 0% \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NAT \\n                 0% \\n                 \\u20020% \\n                 22% \\n                 0% \\n                 20%\\u2002 \\n               \\n               \\n                 Sensitivity, Up vs. NRM \\n                 0% \\n                 33% \\n                 \\u20020% \\n                 42%\\u2002 \\n                 0% \\n               \\n               \\n                 Sensitivity, Down vs. NRM \\n                 0% \\n                 \\u20020% \\n                 78% \\n                 0% \\n                 100%\\u2003 \\n               \\n               \\n                 Specificity \\n                 74.73%\\u2003\\u2009 \\n                 76.34%\\u2003\\u2009 \\n                 89.78%\\u2003\\u2009 \\n                 76.27%\\u2003\\u2002\\u2009 \\n                 79.26%\\u2003\\u2002\\u2009 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Altogether, the tissue specificity, plus the mRNA differential expression in the samples tested are believed to make Ovr110v1 a good marker for diagnosing, monitoring, staging, imaging and/or treating ovarian cancer. \\n     Additionally, the tissue specificity, plus the mRNA differential expression in the samples tested may make Ovr110v1 a good marker for diagnosing, monitoring, staging, imaging and/or treating lung cancer. \\n     Primers used for QPCR Expression Analysis of Ovr110v1 are as follows: \\n                    (SEQ\\u2003ID\\u2003NO:\\u2003338)                     (Ovr110v1_forward):   TCATTGGCTTTGGTATTTCAGAAG                         (SEQ\\u2003ID\\u2003NO:\\u2003339)                     (Ovr110v1_reverse):   GTTCAGGAAGCAAAGATCAATGC                         (SEQ\\u2003ID\\u2003NO:\\u2003340)                     (Ovr110v1_probe):   AGCAATGAAGGGTTTGGTTGTAGAAG            \\nConclusions\\n \\n     Altogether, the high level of tissue specificity, plus the mRNA overexpression in matched samples tested are indicative of SEQ ID NO: 1-128 being a diagnostic marker and/or a therapeutic target for cancer. \\n     Example 3 \\n     Protein Expression \\n     The OSNA is amplified by polymerase chain reaction (PCR) and the amplified DNA fragment encoding the OSNA is subcloned in pET-21d for expression in  E. coli . In addition to the OSNA coding sequence, codons for two amino acids, Met-Ala, flanking the NH 2 -terminus of the coding sequence of OSNA, and six histidines, flanking the COOH-terminus of the coding sequence of OSNA, are incorporated to serve as initiating Met/restriction site and purification tag, respectively. \\n     An over-expressed protein band of the appropriate molecular weight may be observed on a Coomassie blue stained polyacrylamide gel. This protein band is confirmed by Western blot analysis using monoclonal antibody against 6× Histidine tag. \\n     Large-scale purification of OSP is achieved using cell paste generated from 6-liter bacterial cultures, and purified using immobilized metal affinity chromatography (IMAC). Soluble fractions that are separated from total cell lysate were incubated with a nickel chelating resin. The column is packed and washed with five column volumes of wash buffer. OSP is eluted stepwise with various concentration imidazole buffers. \\n     Example 4 \\n     Fusion Proteins \\n     The human Fc portion of the IgG molecule can be PCR amplified, using primers that span the 5′ and 3′ ends of the sequence described below. These primers also should have convenient restriction enzyme sites that will facilitate cloning into an expression vector, preferably a mammalian expression vector. For example, if pC4 (Accession No. 209646) is used, the human Fc portion can be ligated into the BamHI cloning site. Note that the 3′ BamHI site should be destroyed. Next, the vector containing the human Fc portion is re-restricted with BamHI, linearizing the vector, and a polynucleotide of the present invention, isolated by the PCR protocol described in Example 2, is ligated into this BamHI site. Note that the polynucleotide is cloned without a stop codon, otherwise a fusion protein will not be produced. If the naturally occurring signal sequence is used to produce the secreted protein, pC4 does not need a second signal peptide. Alternatively, if the naturally occurring signal sequence is not used, the vector can be modified to include a heterologous signal sequence. See, e.g., WO 96/34891. \\n     Example 5 \\n     Production of an Antibody from a Polypeptide \\n     In general, such procedures involve immunizing an animal (preferably a mouse) with polypeptide or, more preferably, with a secreted polypeptide-expressing cell. Such cells may be cultured in any suitable tissue culture medium; however, it is preferable to culture cells in Earle&#39;s modified Eagle&#39;s medium supplemented with 10% fetal bovine serum (inactivated at about 56° C.), and supplemented with about 10 g/l of nonessential amino acids, about 1,000 U/ml of penicillin, and about 100, μg/ml of streptomycin. The splenocytes of such mice are extracted and fused with a suitable myeloma cell line. Any suitable myeloma cell line may be employed in accordance with the present invention; however, it is preferable to employ the parent myeloma cell line (SP20), available from the ATCC. After fusion, the resulting hybridoma cells are selectively maintained in HAT medium, and then cloned by limiting dilution as described by Wands et al.,  Gastroenterology  80: 225-232 (1981). \\n     The hybridoma cells obtained through such a selection are then assayed to identify clones which secrete antibodies capable of binding the polypeptide. Alternatively, additional antibodies capable of binding to the polypeptide can be produced in a two-step procedure using anti-idiotypic antibodies. Such a method makes use of the fact that antibodies are themselves antigens, and therefore, it is possible to obtain an antibody which binds to a second antibody. In accordance with this method, protein specific antibodies are used to immunize an animal, preferably a mouse. The splenocytes of such an animal are then used to produce hybridoma cells, and the hybridoma cells are screened to identify clones which produce an antibody whose ability to bind to the protein-specific antibody can be blocked by the polypeptide. Such antibodies comprise anti-idiotypic antibodies to the protein specific antibody and can be used to immunize an animal to induce formation of further protein-specific antibodies. \\n     Example 6 \\n     Method of Determining Alterations in a Gene Corresponding to a Polynucleotide \\n     RNA is isolated from individual patients or from a family of individuals that have a phenotype of interest. cDNA is then generated from these RNA samples using protocols known in the art. See, Sambrook (2001), supra. The cDNA is then used as a template for PCR, employing primers surrounding regions of interest in SEQ ID NO: 1-128. Suggested PCR conditions consist of 35 cycles at 95° C. for 30 seconds; 60-120 seconds at 52-58° C.; and 60-120 seconds at 70° C., using buffer solutions described in Sidransky et al.,  Science  252(5006): 706-9 (1991). See also Sidransky et al.,  Science  278(5340): 1054-9 (1997). \\n     PCR products are then sequenced using primers labeled at their 5′ end with T4 polynucleotide kinase, employing SequiTherm Polymerase. (Epicentre Technologies). The intron-exon borders of selected exons are also determined and genomic PCR products analyzed to confirm the results. PCR products harboring suspected mutations are then cloned and sequenced to validate the results of the direct sequencing. PCR products is cloned into T-tailed vectors as described in Holton et al.,  Nucleic Acids Res.,  19: 1156 (1991) and sequenced with T7 polymerase (United States Biochemical). Affected individuals are identified by mutations not present in unaffected individuals. \\n     Genomic rearrangements may also be determined. Genomic clones are nick-translated with digoxigenin deoxyuridine 5′ triphosphate (Boehringer Manheim), and FISH is performed as described in Johnson et al.,  Methods Cell Biol.  35: 73-99 (1991). Hybridization with the labeled probe is carried out using a vast excess of human cot-1 DNA for specific hybridization to the corresponding genomic locus. \\n     Chromosomes are counterstained with 4,6-diamino-2-phenylidole and propidium iodide, producing a combination of C- and R-bands. Aligned images for precise mapping are obtained using a triple-band filter set (Chroma Technology, Brattleboro, Vt.) in combination with a cooled charge-coupled device camera (Photometrics, Tucson, Ariz.) and variable excitation wavelength filters. Johnson (1991). Image collection, analysis and chromosomal fractional length measurements are performed using the ISee Graphical Program System. (Inovision Corporation, Durham, N.C.) Chromosome alterations of the genomic region hybridized by the probe are identified as insertions, deletions, and translocations. These alterations are used as a diagnostic marker for an associated disease. \\n     Example 7 \\n     Method of Detecting Abnormal Levels of a Polypeptide in a Biological Sample \\n     Antibody-sandwich ELISAs are used to detect polypeptides in a sample, preferably a biological sample. Wells of a microtiter plate are coated with specific antibodies, at a final concentration of 0.2 to 10 ug/ml. The antibodies are either monoclonal or polyclonal and are produced by the method described above. The wells are blocked so that non-specific binding of the polypeptide to the well is reduced. The coated wells are then incubated for &gt;2 hours at RT with a sample containing the polypeptide. Preferably, serial dilutions of the sample should be used to validate results. The plates are then washed three times with deionized or distilled water to remove unbound polypeptide. Next, 50 μl of specific antibody-alkaline phosphatase conjugate, at a concentration of 25-400 ng, is added and incubated for 2 hours at room temperature. The plates are again washed three times with deionized or distilled water to remove unbound conjugate. 75 μl of 4-methylumbelliferyl phosphate (MUP) or p-nitrophenyl phosphate (NPP) substrate solution are added to each well and incubated 1 hour at room temperature. \\n     The reaction is measured by a microtiter plate reader. A standard curve is prepared, using serial dilutions of a control sample, and polypeptide concentrations are plotted on the X-axis (log scale) and fluorescence or absorbance on the Y-axis (linear scale). The concentration of the polypeptide in the sample is calculated using the standard curve. \\n     Example 8 \\n     Formulating a Polypeptide \\n     The secreted polypeptide composition will be formulated and dosed in a fashion consistent with good medical practice, taking into account the clinical condition of the individual patient (especially the side effects of treatment with the secreted polypeptide alone), the site of delivery, the method of administration, the scheduling of administration, and other factors known to practitioners. The “effective amount” for purposes herein is thus determined by such considerations. \\n     As a general proposition, the total pharmaceutically effective amount of secreted polypeptide administered parenterally per dose will be in the range of about 1, μg/kg/day to 10 mg/kg/day of patient body weight, although, as noted above, this will be subject to therapeutic discretion. More preferably, this dose is at least 0.01 mg/kg/day, and most preferably for humans between about 0.01 and 1 mg/kg/day for the hormone. If given continuously, the secreted polypeptide is typically administered at a dose rate of about 1 dg/kg/hour to about 50 mg/kg/hour, either by 1-4 injections per day or by continuous subcutaneous infusions, for example, using a mini-pump. An intravenous bag solution may also be employed. The length of treatment needed to observe changes and the interval following treatment for responses to occur appears to vary depending on the desired effect. \\n     Pharmaceutical compositions containing the secreted protein of the invention are administered orally, rectally, parenterally, intracistemally, intravaginally, intraperitoneally, topically (as by powders, ointments, gels, drops or transdermal patch), bucally, or as an oral or nasal spray. “Pharmaceutically acceptable carrier” refers to a non-toxic solid, semisolid or liquid filler, diluent, encapsulating material or formulation auxiliary of any type. The term “parenteral” as used herein refers to modes of administration which include intravenous, intramuscular, intraperitoneal, intrasternal, subcutaneous and intraarticular injection and infusion. \\n     The secreted polypeptide is also suitably administered by sustained-release systems. Suitable examples of sustained-release compositions include semipermeable polymer matrices in the form of shaped articles, e.g., films, or microcapsules. Sustained-release matrices include polylactides (U.S. Pat. No. 3,773,919, EP 58,481, the contents of which are hereby incorporated by reference herein in their entirety), copolymers of L-glutamic acid and gamma-ethyl-L-glutamate (Sidman, U. et al., Biopolymers 22: 547-556 (1983)), poly (2-hydroxyethyl methacrylate) (R. Langer et al., J. Biomed. Mater. Res. 15: 167-277 (1981), and R. Langer, Chem. Tech. 12: 98-105 (1982)), ethylene vinyl acetate (R. Langer et al.) or poly-D-(−)-3-hydroxybutyric acid (EP 133,988). Sustained-release compositions also include liposomally entrapped polypeptides. Liposomes containing the secreted polypeptide are prepared by methods known per se: D E Epstein et al., Proc. Natl. Acad. Sci. USA 82: 3688-3692 (1985); Hwang et al., Proc. Natl. Acad. Sci. USA 77: 4030-4034 (1980); EP 52,322; EP 36,676; EP 88,046; EP 143,949; EP 142,641; Japanese Pat. Appl. 83-118008; U.S. Pat. Nos. 4,485,045 and 4,544,545; and EP 102,324, the contents of which are hereby incorporated by reference herein in their entirety. Ordinarily, the liposomes are of the small (about 200-800 Angstroms) unilamellar type in which the lipid content is greater than about 30 mol. percent cholesterol, the selected proportion being adjusted for the optimal secreted polypeptide therapy. \\n     For parenteral administration, in one embodiment, the secreted polypeptide is formulated generally by mixing it at the desired degree of purity, in a unit dosage injectable form (solution, suspension, or emulsion), with a pharmaceutically acceptable carrier, i.e., one that is non-toxic to recipients at the dosages and concentrations employed and is compatible with other ingredients of the formulation. \\n     For example, the formulation preferably does not include oxidizing agents and other compounds that are known to be deleterious to polypeptides. Generally, the formulations are prepared by contacting the polypeptide uniformly and intimately with liquid carriers or finely divided solid carriers or both. Then, if necessary, the product is shaped into the desired formulation. Preferably, the carrier is a parenteral carrier, more preferably, a solution that is isotonic with the blood of the recipient. Examples of such carrier vehicles include water, saline, Ringer&#39;s solution, and dextrose solution. Non-aqueous vehicles such as fixed oils and ethyl oleate are also useful herein, as well as liposomes. \\n     The carrier suitably contains minor amounts of additives such as substances that enhance isotonicity and chemical stability. Such materials are non-toxic to recipients at the dosages and concentrations employed, and include buffers such as phosphate, citrate, succinate, acetic acid, and other organic acids or their salts; antioxidants such as ascorbic acid; low molecular weight (less than about ten residues) polypeptides, e.g., polyarginine or tripeptides; proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidone; amino acids, such as glycine, glutamic acid, aspartic acid, or arginine; monosaccharides, disaccharides, and other carbohydrates including cellulose or its derivatives, glucose, manose, or dextrins; chelating agents such as EDTA; sugar alcohols such as mannitol or sorbitol; counterions such as sodium; and/or nonionic surfactants such as polysorbates, poloxamers, or PEG. \\n     The secreted polypeptide is typically formulated in such vehicles at a concentration of about 0.1 mg/ml to 100 mg/ml, preferably 1-10 mg/ml, at a pH of about 3 to 8. It will be understood that the use of certain of the foregoing excipients, carriers, or stabilizers will result in the formation of polypeptide salts. \\n     Any polypeptide to be used for therapeutic administration can be sterile. Sterility is readily accomplished by filtration through sterile filtration membranes (e.g., 0.2 micron membranes). Therapeutic polypeptide compositions generally are placed into a container having a sterile access port, for example, an intravenous solution bag or vial having a stopper pierceable by a hypodermic injection needle. \\n     Polypeptides ordinarily will be stored in unit or multi-dose containers, for example, sealed ampules or vials, as an aqueous solution or as a lyophilized formulation for reconstitution. As an example of a lyophilized formulation, 10-ml vials are filled with 5 ml of sterile-filtered 1% (w/v) aqueous polypeptide solution, and the resulting mixture is lyophilized. The infusion solution is prepared by reconstituting the lyophilized polypeptide using bacteriostatic Water-for-Injection. \\n     The invention also provides a pharmaceutical pack or kit comprising one or more containers filled with one or more of the ingredients of the pharmaceutical compositions of the invention. Associated with such container (s) can be a notice in the form prescribed by a governmental agency regulating the manufacture, use or sale of pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human administration. In addition, the polypeptides of the present invention may be employed in conjunction with other therapeutic compounds. \\n     Example 9 \\n     Method of Treating Decreased Levels of the Polypeptide \\n     It will be appreciated that conditions caused by a decrease in the standard or normal expression level of a secreted protein in an individual can be treated by administering the polypeptide of the present invention, preferably in the secreted form. Thus, the invention also provides a method of treatment of an individual in need of an increased level of the polypeptide comprising administering to such an individual a pharmaceutical composition comprising an amount of the polypeptide to increase the activity level of the polypeptide in such an individual. \\n     For example, a patient with decreased levels of a polypeptide receives a daily dose 0.1-100 ug/kg of the polypeptide for six consecutive days. Preferably, the polypeptide is in the secreted form. The exact details of the dosing scheme, based on administration and formulation, are provided above. \\n     Example 10 \\n     Method of Treating Increased Levels of the Polypeptide \\n     Antisense or RNAi technology are used to inhibit production of a polypeptide of the present invention. This technology is one example of a method of decreasing levels of a polypeptide, preferably a secreted form, due to a variety of etiologies, such as cancer. \\n     For example, a patient diagnosed with abnormally increased levels of a polypeptide is administered intravenously antisense polynucleotides at 0.5, 1.0, 1.5, 2.0 and 3.0 mg/kg day for 21 days. This treatment is repeated after a 7-day rest period if the treatment was well tolerated. The formulation of the antisense polynucleotide is provided above. \\n     Example 11 \\n     Method of Treatment Using Gene Therapy \\n     One method of gene therapy transplants fibroblasts, which are capable of expressing a polypeptide, onto a patient. Generally, fibroblasts are obtained from a subject by skin biopsy. The resulting tissue is placed in tissue-culture medium and separated into small pieces. Small chunks of the tissue are placed on a wet surface of a tissue culture flask, approximately ten pieces are placed in each flask. The flask is turned upside down, closed tight and left at room temperature over night. After 24 hours at room temperature, the flask is inverted and the chunks of tissue remain fixed to the bottom of the flask and fresh media (e.g., Ham&#39;s F12 media, with 10% FBS, penicillin and streptomycin) is added. The flasks are then incubated at 37° C. for approximately one week. \\n     At this time, fresh media is added and subsequently changed every several days. After an additional two weeks in culture, a monolayer of fibroblasts emerge. The monolayer is trypsinized and scaled into larger flasks. pMV-7 (Kirschmeier, P. T. et al., DNA, 7: 219-25 (1988)), flanked by the long terminal repeats of the Moloney murine sarcoma virus, is digested with EcoRI and HindIII and subsequently treated with calf intestinal phosphatase. The linear vector is fractionated on agarose gel and purified, using glass beads. \\n     The cDNA encoding a polypeptide of the present invention can be amplified using PCR primers which correspond to the 5′ and 3′ end sequences respectively as set forth in Example 3. Preferably, the 5′ primer contains an EcoRI site and the 3′ primer includes a HindIII site. Equal quantities of the Moloney murine sarcoma virus linear backbone and the amplified EcoRI and HindIII fragment are added together, in the presence of T4 DNA ligase. The resulting mixture is maintained under conditions appropriate for ligation of the two fragments. The ligation mixture is then used to transform bacteria HB 101, which are then plated onto agar containing kanamycin for the purpose of confirming that the vector has the gene of interest properly inserted. \\n     The amphotropic pA317 or GP+aml2 packaging cells are grown in tissue culture to confluent density in Dulbecco&#39;s Modified Eagles Medium (DMEM) with 10% calf serum (CS), penicillin and streptomycin. The MSV vector containing the gene is then added to the media and the packaging cells transduced with the vector. The packaging cells now produce infectious viral particles containing the gene (the packaging cells are now referred to as producer cells). \\n     Fresh media is added to the transduced producer cells, and subsequently, the media is harvested from a 10 cm plate of confluent producer cells. The spent media, containing the infectious viral particles, is filtered through a millipore filter to remove detached producer cells and this media is then used to infect fibroblast cells. Media is removed from a sub-confluent plate of fibroblasts and quickly replaced with the media from the producer cells. This media is removed and replaced with fresh media. \\n     If the titer of virus is high, then virtually all fibroblasts will be infected and no selection is required. If the titer is very low, then it is necessary to use a retroviral vector that has a selectable marker, such as neo or his. Once the fibroblasts have been efficiently infected, the fibroblasts are analyzed to determine whether protein is produced. \\n     The engineered fibroblasts are then transplanted onto the host, either alone or after having been grown to confluence on cytodex 3 microcarrier beads. \\n     Example 12 \\n     Method of Treatment Using Gene Therapy-In Vivo \\n     Another aspect of the present invention is using in vivo gene therapy methods to treat disorders, diseases and conditions. The gene therapy method relates to the introduction of naked nucleic acid (DNA, RNA, and antisense DNA or RNA) sequences into an animal to increase or decrease the expression of the polypeptide. \\n     The polynucleotide of the present invention may be operatively linked to a promoter or any other genetic elements necessary for the expression of the polypeptide by the target tissue. Such gene therapy and delivery techniques and methods are known in the art, see, for example, Tabata H. et al.  Cardiovasc. Res.  35 (3): 470-479 (1997); Chao J et al. Pharmacol. Res. 35 (6): 517-522 (1997); Wolff J. A.  Neuromuscul. Disord.  7 (5): 314-318 (1997), Schwartz B. et al.  Gene Ther.  3 (5): 405-411 (1996); and Tsurumi Y. et al.  Circulation  94 (12): 3281-3290 (1996); WO 90/11092, WO 98/11779; U.S. Pat. Nos. 5,693,622; 5,705,151; 5,580,859, the contents of which are hereby incorporated by reference herein in their entirety. \\n     The polynucleotide constructs may be delivered by any method that delivers injectable materials to the cells of an animal, such as, injection into the interstitial space of tissues (heart, muscle, skin, ovarian, liver, intestine and the like). The polynucleotide constructs can be delivered in a pharmaceutically acceptable liquid or aqueous carrier. \\n     The term “naked” polynucleotide, DNA or RNA, refers to sequences that are free from any delivery vehicle that acts to assist, promote, or facilitate entry into the cell, including viral sequences, viral particles, liposome formulations, lipofectin or precipitating agents and the like. However, the polynucleotides of the present invention may also be delivered in liposome formulations (such as those taught in Felgner P. L. et al.  Ann. NY Acad. Sci.  772: 126-139 (1995) and Abdallah B. et al.  Biol. Cell  85 (1): 1-7 (1995)) which can be prepared by methods well known to those skilled in the art. \\n     The polynucleotide vector constructs used in the gene therapy method are preferably constructs that will not integrate into the host genome nor will they contain sequences that allow for replication. Any strong promoter known to those skilled in the art can be used for driving the expression of DNA. Unlike other gene therapies techniques, one major advantage of introducing naked nucleic acid sequences into target cells is the transitory nature of the polynucleotide synthesis in the cells. Studies have shown that non-replicating DNA sequences can be introduced into cells to provide production of the desired polypeptide for periods of up to six months. \\n     The polynucleotide construct can be delivered to the interstitial space of tissues within the an animal, including of muscle, skin, brain, ovarian, liver, spleen, bone marrow, thymus, heart, lymph, blood, bone, cartilage, pancreas, kidney, gall bladder, stomach, intestine, testis, ovary, uterus, rectum, nervous system, eye, gland, and connective tissue. Interstitial space of the tissues comprises the intercellular fluid, mucopolysaccharide matrix among the reticular fibers of organ tissues, elastic fibers in the walls of vessels or chambers, collagen fibers of fibrous tissues, or that same matrix within connective tissue ensheathing muscle cells or in the lacunae of bone. It is similarly the space occupied by the plasma of the circulation and the lymph fluid of the lymphatic channels. Delivery to the interstitial space of muscle tissue is preferred for the reasons discussed below. They may be conveniently delivered by injection into the tissues comprising these cells. They are preferably delivered to and expressed in persistent, non-dividing cells which are differentiated, although delivery and expression may be achieved in non-differentiated or less completely differentiated cells, such as, for example, stem cells of blood or skin fibroblasts. In vivo muscle cells are particularly competent in their ability to take up and express polynucleotides. \\n     For the naked polynucleotide injection, an effective dosage amount of DNA or RNA will be in the range of from about 0.05 μg/kg body weight to about 50 mg/kg body weight. Preferably the dosage will be from about 0.005 mg/kg to about 20 mg/kg and more preferably from about 0.05 mg/kg to about 5 mg/kg. Of course, as the artisan of ordinary skill will appreciate, this dosage will vary according to the tissue site of injection. The appropriate and effective dosage of nucleic acid sequence can readily be determined by those of ordinary skill in the art and may depend on the condition being treated and the route of administration. The preferred route of administration is by the parenteral route of injection into the interstitial space of tissues. However, other parenteral routes may also be used, such as, inhalation of an aerosol formulation particularly for delivery to ovarians or bronchial tissues, throat or mucous membranes of the nose. In addition, naked polynucleotide constructs can be delivered to arteries during angioplasty by the catheter used in the procedure. \\n     The dose response effects of injected polynucleotide in muscle in vivo is determined as follows. Suitable template DNA for production of mRNA coding for polypeptide of the present invention is prepared in accordance with a standard recombinant DNA methodology. The template DNA, which may be either circular or linear, is either used as naked DNA or complexed with liposomes. The quadriceps muscles of mice are then injected with various amounts of the template DNA. \\n     Five to six week old female and male Balb/C mice are anesthetized by intraperitoneal injection with 0.3 ml of 2.5% Avertin. A 1.5 cm incision is made on the anterior thigh, and the quadriceps muscle is directly visualized. The template DNA is injected in 0.1 ml of carrier in a 1 cc syringe through a 27 gauge needle over one minute, approximately 0.5 cm from the distal insertion site of the muscle into the knee and about 0.2 cm deep. A suture is placed over the injection site for future localization, and the skin is closed with stainless steel clips. \\n     After an appropriate incubation time (e.g., 7 days) muscle extracts are prepared by excising the entire quadriceps. Every fifth 15 um cross-section of the individual quadriceps muscles is histochemically stained for protein expression. A time course for protein expression may be done in a similar fashion except that quadriceps from different mice are harvested at different times. Persistence of DNA in muscle following injection may be determined by Southern blot analysis after preparing total cellular DNA and HIRT supernatants from injected and control mice. \\n     The results of the above experimentation in mice can be use to extrapolate proper dosages and other treatment parameters in humans and other animals using naked DNA. \\n     Example 13 \\n     Transgenic Animals \\n     The polypeptides of the invention can also be expressed in transgenic animals. Animals of any species, including, but not limited to, mice, rats, rabbits, hamsters, guinea pigs, pigs, micro-pigs, goats, sheep, cows and non-human primates, e.g., baboons, monkeys, and chimpanzees may be used to generate transgenic animals. In a specific embodiment, techniques described herein or otherwise known in the art, are used to express polypeptides of the invention in humans, as part of a gene therapy protocol. \\n     Any technique known in the art may be used to introduce the transgene (I. e., polynucleotides of the invention) into animals to produce the founder lines of transgenic animals. Such techniques include, but are not limited to, pronuclear microinjection (Paterson et al.,  Appl. Microbiol. Biotechnol.  40: 691-698 (1994); Carver et al.,  Biotechnology  11: 1263-1270 (1993); Wright et al.,  Biotechnology  9: 830-834 (1991); and U.S. Pat. No. 4,873,191, the contents of which is hereby incorporated by reference herein in its entirety); retrovirus mediated gene transfer into germ lines (Van der Putten et al.,  Proc. Natl. Acad. Sci., USA  82: 6148-6152 (1985)), blastocysts or embryos; gene targeting in embryonic stem cells (Thompson et al.,  Cell  56: 313-321 (1989)); electroporation of cells or embryos (Lo, 1983 , Mol. Cell. Biol.  3: 1803-1814 (1983)); introduction of the polynucleotides of the invention using a gene gun (see, e.g., Ulmer et al.,  Science  259: 1745 (1993); introducing nucleic acid constructs into embryonic pleuripotent stem cells and transferring the stem cells back into the blastocyst; and sperm mediated gene transfer (Lavitrano et al.,  Cell  57: 717-723 (1989). For a review of such techniques, see Gordon, “Transgenic Animals,”  Intl. Rev. Cytol.  115: 171-229 (1989). \\n     Any technique known in the art may be used to produce transgenic clones containing polynucleotides of the invention, for example, nuclear transfer into enucleated oocytes of nuclei from cultured embryonic, fetal, or adult cells induced to quiescence (Campell et al.,  Nature  380: 64-66 (1996); Wilmut et al.,  Nature  385: 810813 (1997)). \\n     The present invention provides for transgenic animals that carry the transgene in all their cells, as well as animals which carry the transgene in some, but not all their cells, I. e., mosaic animals or chimeric. The transgene may be integrated as a single transgene or as multiple copies such as in concatamers, e.g., head-to-head tandems or head-to-tail tandems. The transgene may also be selectively introduced into and activated in a particular cell type by following, for example, the teaching of Lasko et al. (Lasko et al.,  Proc. Natl. Acad. Sci. USA  89: 6232-6236 (1992)). The regulatory sequences required for such a cell-type specific activation will depend upon the particular cell type of interest, and will be apparent to those of skill in the art. When it is desired that the polynucleotide transgene be integrated into the chromosomal site of the endogenous gene, gene targeting is preferred. Briefly, when such a technique is to be utilized, vectors containing some nucleotide sequences homologous to the endogenous gene are designed for the purpose of integrating, via homologous recombination with chromosomal sequences, into and disrupting the function of the nucleotide sequence of the endogenous gene. The transgene may also be selectively introduced into a particular cell type, thus inactivating the endogenous gene in only that cell type, by following, for example, the teaching of Gu et al. (Gu et al.,  Science  265: 103-106 (1994)). The regulatory sequences required for such a cell-type specific inactivation will depend upon the particular cell type of interest, and will be apparent to those of skill in the art. \\n     Once transgenic animals have been generated, the expression of the recombinant gene may be assayed utilizing standard techniques. Initial screening may be accomplished by Southern blot analysis or PCR techniques to analyze animal tissues to verify that integration of the transgene has taken place. The level of mRNA expression of the transgene in the tissues of the transgenic animals may also be assessed using techniques which include, but are not limited to, Northern blot analysis of tissue samples obtained from the animal, in situ hybridization analysis, and reverse transcriptase-PCR (rt-PCR). Samples of transgenic gene-expressing tissue may also be evaluated immunocytochemically or immunohistochemically using antibodies specific for the transgene product. \\n     Once the founder animals are produced, they may be bred, inbred, outbred, or crossbred to produce colonies of the particular animal. Examples of such breeding strategies include, but are not limited to: outbreeding of founder animals with more than one integration site in order to establish separate lines; inbreeding of separate lines in order to produce compound transgenics that express the transgene at higher levels because of the effects of additive expression of each transgene; crossing of heterozygous transgenic animals to produce animals homozygous for a given integration site in order to both augment expression and eliminate the need for screening of animals by DNA analysis; crossing of separate homozygous lines to produce compound heterozygous or homozygous lines; and breeding to place the transgene on a distinct background that is appropriate for an experimental model of interest. \\n     Transgenic animals of the invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying conditions and/or disorders associated with aberrant expression, and in screening for compounds effective in ameliorating such conditions and/or disorders. \\n     Example 14 \\n     Knock-Out Animals \\n     Endogenous gene expression can also be reduced by inactivating or “knocking out” the gene and/or its promoter using targeted homologous recombination. (E. g., see Smithies et al., Nature 317: 230-234 (1985); Thomas &amp; Capecchi, Cell 51: 503512 (1987); Thompson et al., Cell 5: 313-321 (1989)) Alternatively, RNAi technology may be used. For example, a mutant, non-functional polynucleotide of the invention (or a completely unrelated DNA sequence) flanked by DNA homologous to the endogenous polynucleotide sequence (either the coding regions or regulatory regions of the gene) can be used, with or without a selectable marker and/or a negative selectable marker, to transfect cells that express polypeptides of the invention in vivo. In another embodiment, techniques known in the art are used to generate knockouts in cells that contain, but do not express the gene of interest. Insertion of the DNA construct, via targeted homologous recombination, results in inactivation of the targeted gene. Such approaches are particularly suited in research and agricultural fields where modifications to embryonic stem cells can be used to generate animal offspring with an inactive targeted gene (e.g., see Thomas &amp; Capecchi 1987 and Thompson 1989, supra). However, this approach can be routinely adapted for use in humans provided the recombinant DNA constructs are directly administered or targeted to the required site in vivo using appropriate viral vectors that will be apparent to those of skill in the art. \\n     In further embodiments of the invention, cells that are genetically engineered to express the polypeptides of the invention, or alternatively, that are genetically engineered not to express the polypeptides of the invention (e.g., knockouts) are administered to a patient in vivo. Such cells may be obtained from the patient (i.e., animal, including human) or an MHC compatible donor and can include, but are not limited to fibroblasts, bone marrow cells, blood cells (e.g., lymphocytes), adipocytes, muscle cells, endothelial cells etc. The cells are genetically engineered in vitro using recombinant DNA techniques to introduce the coding sequence of polypeptides of the invention into the cells, or alternatively, to disrupt the coding sequence and/or endogenous regulatory sequence associated with the polypeptides of the invention, e.g., by transduction (using viral vectors, and preferably vectors that integrate the transgene into the cell genome) or transfection procedures, including, but not limited to, the use of plasmids, cosmids, YACs, naked DNA, electroporation, liposomes, etc. \\n     The coding sequence of the polypeptides of the invention can be placed under the control of a strong constitutive or inducible promoter or promoter/enhancer to achieve expression, and preferably secretion, of the polypeptides of the invention. The engineered cells which express and preferably secrete the polypeptides of the invention can be introduced into the patient systemically, e.g., in the circulation, or intraperitoneally. \\n     Alternatively, the cells can be incorporated into a matrix and implanted in the body, e.g., genetically engineered fibroblasts can be implanted as part of a skin graft; genetically engineered endothelial cells can be implanted as part of a lymphatic or vascular graft. (See, for example, Anderson et al. U.S. Pat. No. 5,399,349; and Mulligan &amp; Wilson, U.S. Pat. No. 5,460,959, the contents of which are hereby incorporated by reference herein in their entirety). \\n     When the cells to be administered are non-autologous or non-MHC compatible cells, they can be administered using well known techniques which prevent the development of a host immune response against the introduced cells. For example, the cells may be introduced in an encapsulated form which, while allowing for an exchange of components with the immediate extracellular environment, does not allow the introduced cells to be recognized by the host immune system. \\n     Transgenic and “knock-out” animals of the invention have uses which include, but are not limited to, animal model systems useful in elaborating the biological function of polypeptides of the present invention, studying conditions and/or disorders associated with aberrant expression, and in screening for compounds effective in ameliorating such conditions and/or disorders. \\n     While preferred illustrative embodiments of the present invention are described, one skilled in the art will appreciate that the present invention can be practiced by other than the described embodiments, which are presented for purposes of illustration only and not by way of limitation. The present invention is limited only by the claims that follow.', 'id': 'US-8460880-B2', 'ti': 'Compositions, splice variants and methods relating to ovarian specific genes and proteins'}>,\n",
       " <Hit {'ab': 'Compositions and methods for inducing pluripotent stem cells into retinal ganglion cells for administration to a subject for treating progressive optic neuropathies, thereby alleviating symptoms of such disorders including glaucoma., Compositions and methods for inducing pluripotent stem cells into retinal ganglion cells for administration to a subject for treating progressive optic neuropathies, thereby alleviating symptoms of such disorders including glaucoma.', 'clm': 'What is claimed is: \\n     \\n         1 . A method of preparing retinal progenitor cells and retinal ganglion cells by directed differentiation of induced pluripotent stem cells (iPSCs) in a defined chemical medium, comprising,\\n a) culturing said iPSC in iPSC initiation medium;   b) transferring said iPSC of step a) to retinal progenitor cell (RPC) culture medium for transdifferentiation into RPCs;   c) culturing said RPCs of step b) in retinal ganglion cell (RGC) culture medium for transdifferentiation into RGCs; thereby providing a population of RGCs suitable for in vitro studies and for transplantation into the eye.   \\n     \\n     \\n         2 . The method of  claim 1 , wherein said iPSCs are cultured in a 37° C., 5% CO 2  and 5% O 2  incubator and the iPSC initiation medium comprises one of:\\n i) 80% HES/20% MEF-CM+20 ng/ml bFGF+4 μM Y27632; \\n ii) 100% HES+20 ng/ml bFGF+4 μM Y27632; or \\n iii) 100% StemMACS™ iPS Brew-Xenofree (XF)+20 ng/ml bFGF+4 μM Y27632. \\n \\n     \\n     \\n         3 . The method of  claim 1 , wherein the RPC culture medium comprises RPC induction media, with 0.1 μM LDN193189, 10 μM SB431542, 2 μM XAV939, 10 mM Nicotinamide, 10 ng/ml IGF1, 1.5 μM CHIR99021 and 10 ng/ml bFGF reagents. \\n     \\n     \\n         4 . The method of  claim 1 , wherein the RGC culture medium comprises RGC induction media, 3 μM DAPT, 250 ng/ml sonic hedgehog (SHH) and 100 ng/ml FGF8 or smoothened agonist (SAG). \\n     \\n     \\n         5 . The method of  claim 1 , further comprising isolating said RGCs, wherein said RGCs express BRN3A, BRN3B, TUBB3, CD90, MAP2, TUJ1, RBPMS and TUJ1 markers. \\n     \\n     \\n         6 . The method of  claim 1  wherein said RGC population is obtained after between 30 to 45 days in culture from iPSCs. \\n     \\n     \\n         7 . A method for the production of retinal progenitor cells (RPCs), comprising;\\n a) culturing iPSCs on 0.1% gelatinized plates containing irradiated MEFs until cells achieved approximately 75% confluence;   b) removing said MEFs and plating remaining iPSCs onto plates containing 1:100 diluted growth factor reduced Matrigel and iPSC: MEF-conditioned medium (80:20)+20 ng/mL of bFGF and 5 ng/ml of stable bFGF until reaching 100% confluence;   c) replacing iPSC: MEF-conditioned media with RPC induction media comprising DMEM/F12 (50:50), 1% P/S, 1% Glut, 1% NEAA, 0.1 mM 2-ME, 2% B27 supplement (w/o vitamin A), 1% N2 supplement, containing effective amounts of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 mM nicotinamide, and 10 ng/mL IGF1 and culturing cells for 4 days with daily media changes;   d) replacing the culture media of step c) with RPC induction media containing an effective amount of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 ng/mL IGF1, and 10 ng/mL bFGF on day 5 and culturing said cells for 12 days with daily media changes;   e) replacing RPC induction media with RGC differentiation media comprising 0.1 μM LDN193198, 10 μM SB431542, 2 μM XAV939, 1.5 μM CHIR99021, 10 ng/ml IGF1 and 10 ng/ml bFGF on day 16 of differentiation culturing said cells until Day 23, upon which RGC differentiation occurred thereby producing a population of RGCs suitable for transplantation into the eye.   \\n     \\n     \\n         8 . The method of  claim 7 , wherein the Wnt signaling pathway activator is one or more selected from the group consisting of Wnt1, Wnt2, Wnt2b, Wnt3, Wnt3a, Wnt4, Wnt5a, Wnt5b, Wnt6, Wnt7a, Wnt7b, Wnt8a, Wnt8b, Wnt9a, Wnt9b, Wnt10a, Wnt10b, Wnt11, and Wnt16b, substance increasing p3-catenin levels; lithium, LiCl, bivalent zinc, BIO (6-bromoindirubin-3′-oxime), SB216763, SB415286, CHIR99021, QS 11 hydrate, TWS 119, Kenpaullone, alsterpaullone, indirubin-3′-oxime, TDZD-8 and Ro 318220 methanesulfonate salt; Axin inhibitors; APC (adenomatous polyposis coli) inhibitors; norrin and R-spondin 2. \\n     \\n     \\n         9 . The method of  claim 7 , further comprising the step of determining whether the retinal progenitor cells are differentiated into the mature retinal ganglion cells. \\n     \\n     \\n         10 . The method of  claim 1 , wherein the mature retinal ganglion cells are present at from about 60% to about 95% or more of total cells after the culturing of step \\n     \\n     \\n         11 . The method of  claim 1 , where in iPSCs are differentiated into RGCs from normal and disease patients with ocular pathologies selected from glaucoma, age-related macular degeneration (AMD) or any type of retinal degeneration, a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     \\n     \\n         12 . A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RGCs as claimed in  claim 1 , in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     \\n     \\n         13 . A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RPCs as claimed in  claim 7 , in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     \\n     \\n         14 . The method of  claim 12 , wherein the cells are administered via intravitreal, intrastromal, intracameral, sub-tenon, sub-retinal, retro-bulbar, peribulbar, suprachoroidal, choroidal, subchoroidal, conjunctival, subconjunctival, epi scleral, posterior juxtascleral, circumcorneal, optic nerve or tear duct injection. \\n     \\n     \\n         15 . The method of  claim 13 , wherein the cells are administered via intravitreal, intrastromal, intracameral, sub-tenon, sub-retinal, retro-bulbar, peribulbar, suprachoroidal, choroidal, subchoroidal, conjunctival, subconjunctival, epi scleral, posterior juxtascleral, circumcorneal, optic nerve or tear duct injection. \\n     \\n     \\n         16 . The method of  claim 1 , wherein the RGC maturation medium comprises 30 ng/ml to 50 ng/ml BDNF. \\n     \\n     \\n         17 . A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RGCs using immunopanning, flowcytometry and cell sorting with RGC specific surface antibodies, in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     \\n     \\n         18 . The method of  claim 17 , wherein CD90+RGCs are immunopurified via MACS sorting with CD90.2 magnetic beads. \\n     \\n     \\n         19 . An isolated population of RGC prepared by the method of  claim 1 . \\n     \\n     \\n         20 . An isolated population of RPC prepared by the method of  claim 7 .', 'cpc': 'A61N5/1017', 'detd': 'CROSS REFERENCE TO RELATED APPLICATION \\n     This application claims priority to U.S. Provisional Application No. 63/222,789, filed on Jul. 16, 2021, the entire disclosure of which is incorporated herein by reference as though set forth in full. \\n    \\n    \\n     GRANT STATEMENT \\n     This invention was made with government support under EY023557-01 awarded by the National Institutes of Health. The government has certain rights in the invention. \\n    \\n    \\n     REFERENCE TO AN ELECTRONIC SEQUENCE LISTING \\n     The contents of the electronic sequence listing (UPNK-110-US01_SEQLIST.xml, Size: 2,955 bytes; and Date of Creation: Oct. 3, 2022) is herein incorporated by reference in its entirety. \\n     FIELD OF THE INVENTION \\n     This invention relates to fields of progressive optic neuropathies and generation of desired cell types via directed differentiation of iPSCs. More specifically, the invention provides compositions and methods for the treatment of eye-related disorders, such as Glaucoma, Optic Neuritis and retinal degenerations via the administration of retinal ganglion cells (RGCs) and/or retinal progenitor cells (RPCs), thereby alleviating symptoms of such disorders. \\n     Background of the Invention \\n     Several publications and patent documents are cited throughout the specification in order to describe the state of the art to which this invention pertains. Each of these citations is incorporated herein by reference as though set forth in full. \\n     Glaucoma encompasses a heterogenous group of optic neuropathies, characterized by their progressive structural and functional deterioration of the optic nerve resulting in permanent visual field loss. This degeneration is due to the death of RGCs, with subsequent visual field loss. Primary open-angle glaucoma (POAG), the most common form of glaucoma, is characterized by chronic and progressive optic nerve degeneration and corresponding visual field deficits in the presence of an open and normal iridocorneal chamber angle. In 2020, about 52.68 million individuals, aged 40-80, were diagnosed with POAG. By 2040, that number is expected to increase to 79.76 million (3, 4), making POAG the leading cause of irreversible blindness worldwide (5, 6). In the United States alone, 3 million Americans are projected to have glaucoma and this number is estimated to double by the year 2050. Furthermore, glaucoma is responsible for 16.2 billion dollars of total direct medical costs (7). POAG is disproportionately more prevalent among senior individuals of African descent (&gt;5%) compared to other groups of the population such as Hispanic or Latino (2.7%), Asian (˜2%), and Caucasian (1.5%) (8-10). \\n     Despite the prevalence of POAG, its pathogenesis remains poorly understood. Disruption of the aqueous humor outflow pathway is known to increase the intraocular pressure (IOP), which preferentially and progressively damages the retinal ganglion cells (RGCs) (11). RGCs are involved in the transmission of visual signals from the photoreceptors to the brain (12). They are the first set of neuronal cells to be differentiated in the developing retina, and since RGCs do not regenerate their death results in permanent vision loss. RGC death causes optic nerve atrophy and results in a severe reduction of visual function (2). Elevated IOP is a primary risk factor and the only treatable clinical characteristic of glaucoma (13), however, increased IOP alone is neither sufficient nor necessary to cause RGC death leading to glaucoma (14). According to the Baltimore Eye Survey, approximately 50% of all glaucoma patients have normal-tension glaucoma where the eye pressure is below 22 mm Hg (14). The complexity of glaucoma certainly makes it possible, if not probable, that RGCs become inherently susceptible to this disease process. \\n     Existing treatments for glaucoma target lowering the IOP and include medications such as anti-hypertensive drops, laser trabeculoplasty, and surgical interventions. Surgeries for glaucoma are plagued with complications that result in the need for additional surgery and frequent follow-up. Anti-hypertensive drops have very high rates of non-compliance stemming from the multiple daily drug regimens of eye drops, which can account for 35,000 doses throughout one&#39;s lifetime. Importantly, anti-hypertensive drops should ideally be administered to patients before or during early vision loss. But, due to the asymptomatic progression of glaucoma, patients often present with late-stage disease that is irreversible and there are no treatments available once the RGC death has occurred (15). Consequently, there is an unmet need for the development of new strategies for the treatment of glaucoma. \\n     Cell replacement strategies are an innovative and promising alternative to treat glaucoma. One such method involves transferring healthy RGCs into the eyes of afflicted individuals to replace the function of the degenerated and dead ganglion cells. Prior RGC transplantation studies have produced variable outcomes and success (16-25), but thus far have resulted in limited clinical use. The inconsistency in transplantation efficiency may be attributed to the availability and purity of a reliable donor source, differences in graft and host species, and route of delivery. \\n     RGCs are found in the ganglion cell layer of the retina and serve as the projection neurons of the retina, utilizing long axons to effectively connect the eye to the brain. They transmit both image-forming and non-image-forming visual information, processed by retinal cells such as photoreceptors and bipolar cells, to higher visual centers in the lateral geniculate body through the optic nerve. Currently, there are many treatments that slow disease, but no precision treatment exists for glaucoma or RGC degeneration. As mature mammalian RGCs are a terminally differentiated lineage, they do not regenerate after succumbing to disease, consequently leading to irreparable blindness. Understandably, there is a great desire for an application devised to rejuvenate or replace injured RGCs. \\n     In view of the foregoing, it is clear that a need exists for an efficient, reproducible, and safe differentiation protocol for generation of RGCs for use in therapeutic approaches for the treatment of ocular disorders. \\n     SUMMARY OF THE INVENTION \\n     In accordance with the present invention, a method of preparing retinal ganglion cells by directed differentiation of induced pluripotent stem cells (iPSCs) in a defined chemical medium using 2D-culture system is disclosed. An exemplary method entails culturing iPSC in iPSC initiation medium, transferring iPSCs into retinal progenitor cell (RPC) culture medium for transdifferentiation into RPCs, culturing RPCs in retinal ganglion cell (RGC) culture medium for transdifferentiation into RGCs; thereby providing a population of RGCs suitable for in vitro studies and for transplantation into the eye. In certain embodiments, the iPSCs are cultured in a 37° C., 5% CO 2  and 5% O 2  incubator and the iPSC initiation medium comprises one of: i) 80% HES/20% MEF-CM+20 ng/ml bFGF+4 μM Y27632; ii) 100% HES+20 ng/ml bFGF+4 μM Y27632; or iii) 100% StemMACS™ iPS Brew-Xenofree (XF)+20 ng/ml bFGF+4 μM Y27632. \\n     The RPC culture medium comprises RPC induction media, having 0.1 μM LDN193189, 10 μM SB431542, 2 μM XAV939, 10 mM Nicotinamide, 10 ng/ml IGF1, 1.5 μM CHIR99021 and 10 ng/ml bFGF reagents. The RGC culture medium comprises of RGC induction media, 3 μM DAPT, 250 ng/ml sonic hedgehog (SHH) and 100 ng/ml FGF8 or smoothened agonist (SAG). In certain embodiments, the RGCs produced are isolated. In preferred embodiments, the differentiated RGCs express BRN3A, BRN3B, TUBB3, CD90, MAP2, TUJ1, RBPMS and TUJ1 biomarkers. In some embodiments of the method, the RGC population is obtained after between 30 to 45 days in culture from iPSCs. \\n     The invention also provides a method for the production of retinal progenitor cells (RPCs). An exemplary method comprises culturing iPSCs on 0.1% gelatinized plates containing irradiated MEFs until cells achieve approximately 75% confluence and removing said MEFs and plating remaining iPSCs onto plates containing 1:100 diluted growth factor reduced Matrigel and iPSC:MEF-conditioned medium (80:20)+20 ng/mL of bFGF and 5 ng/ml of stable bFGF until reaching 100% confluence. The iPSC:MEF-conditioned media is then replaced with RPC induction media comprising DMEM/F12 (50:50), 1% Penicillin/Streptomycin, 1% Glut, 1% NEAA, 0.1 mM 2-ME, 2% B27 supplement (w/o vitamin A), 1% N2 supplement, containing effective amounts of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 mM nicotinamide, and 10 ng/mL IGF1 and cells cultured for 4 days with daily media changes. On day 5, the culture media is replaced with media containing an effective amount of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 ng/mL IGF1, and 10 ng/mL bFGF on day 5 and cells are cultured for 12 days with daily media changes. The RPC induction media is then replaced with RGC differentiation media comprising 0.1 μM LDN193198, 10 μM SB431542, 2 μM XAV939, 1.5 μM CHIR99021, 10 ng/ml IGF1 and 10 ng/ml bFGF on day 16 of differentiation culturing said cells until Day 23, upon which RGC differentiation occurred, thereby producing a population of RGCs suitable for transplantation into the eye. \\n     In preferred embodiments, the Wnt inhibitor is XAV939, the TGF-β inhibitor is SB431542 and the BMP inhibitor is LDN193189. Stem cells for production of iPSCs, include but are not limited to bone marrow stem cells (BMS), cord blood stem cells, amniotic fluid stem cells, fat stem cells, retinal stem cells (RSCs), keratinocyte stem cells, intraretinal Müller glial cells, embryonic stem cells (ESCs), corneal endothelial cells and somatic cell nuclear transfer cells (SCNTCs). Other Wnt activators useful in the methods of the present invention include for example, Wnt1, Wnt2, Wnt2b, Wnt3, Wnt3a, Wnt4, Wnt5a, Wnt5b, Wnt6, Wnt7a, Wnt7b, Wnt8a, Wnt8b, Wnt9a, Wnt9b, Wnt10a, Wnt10b, Wnt11, and Wnt16b, substance increasing p3-catenin levels; lithium, LiCl, bivalent zinc, BIO (6-bromoindirubin-3′-oxime), SB216763, SB415286, CHIR99021, QS11 hydrate, TWS119, Kenpaullone, alsterpaullone, indirubin-3′-oxime, TDZD-8 and Ro 318220 methanesulfonate salt; Axin inhibitors; APC (adenomatous polyposis coli) inhibitors; norrin and R-spondin 2. \\n     The methods can further comprise the step(s) of determining whether the retinal progenitor cells are differentiated into the mature retinal ganglion cells. In certain embodiments, the step of determining whether the retinal progenitor cells are differentiated into the mature retinal ganglion cells is performed by measuring mRNA or protein expression levels of one or more genes selected from the group consisting of SOX2, RAX, PAX6, SIX6, SIX3, VSX2 and LHX2. In other preferred embodiments, the retinal ganglion cells make up about 60% to about 95% or more of total cells after the RGC differentiation. \\n     In certain approaches, the RGCs comprise an AAV vector for expressing a transgene of interest. In other embodiments, the RGCs are differentiated from iPSCs without genetic manipulation using CRISPR gene editing methodologies. \\n     The invention also provides methods where the iPSCs are differentiated into RGCs from normal and disease patients with ocular pathologies selected from glaucoma, optic neuritis, age-related macular degeneration (AMD) or any type of retinal degeneration, a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     A method for the treatment of an ocular disorder is also encompassed by the present invention. An exemplary method comprises administering a therapeutically effective amount of a population of purified RGCs or RPCs described herein either directly, or in a pharmaceutically carrier or along with a neuroprotective agent into a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, a disorder related to ganglion cell loss, a disorder linked to retinal degeneration or photoreceptor loss or diabetic retinopathy. In preferred embodiments, the host is a human. The cells of the invention may be administered via route selected from intravitreal, intrastromal, intracameral, sub-tenon, sub-retinal, retro-bulbar, peribulbar, suprachoroidal, choroidal, subchoroidal, conjunctival, subconjunctival, epi scleral, posterior juxtascleral, circumcorneal, optic nerve or tear duct injection. \\n     In certain approaches, the RPC or RGC cells are administered through the routes listed above either before optic nerve injury through optic nerve crush or via delivery to the optic nerve. In certain cases, the RGC maturation medium comprises 30 ng/ml to 50 ng/ml BDNF, CNTF or NGF. \\n     In yet another embodiment, a method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RGCs using immunopanning, flowcytometry and cell sorting with RGC specific surface antibodies, fluorescent tag labeling with RGC specific promoter with or without a lentiviral or adenoviral vector, and in a pharmaceutically carrier to a host in need thereof is provided. Disorders to be treated, include without limitation, glaucoma, age-related macular degeneration (AMD), retinal degeneration, a disorder related to an increase in intraocular pressure (IOP), a disorder linked to optic nerve loss due to trauma or injury, a disorder related to neuroprotection, or diabetic retinopathy. In other embodiments, CD90+ RGCs are immunopurified via MACS sorting with CD90.2 magnetic beads. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE DRAWINGS \\n         FIGS.  1 A- 1 D : Differentiation of iPSCs into neural retina and retinal ganglion cells (RGCs). A) Schematic showing iPSCs differentiation in neural retinal differentiation media (DMEM/F12+2% B27+1% N2+10 ng/mL IGF1), including inhibitors of BMP (bone morphogenic protein), WNT and TGF-β for 15 days, bFGF2 was included in media from days 4-14. At 16 days in vitro, cells were treated with RGC induction and maturation media. Differentiation using seven different conditions, L, X, LX, LSB, CHIR, LSB and LSBX are shown. B) Characterization of RPCs using immunocytochemistry with SOX2, RAX and PAX6 along with secondary only control antibodies is shown. C) FACS analysis demonstrated that 96.7% of RPC cells were positive for RAX, and 98.9% of cells were positive for LHX2, when compared to isotype controls. To normalize for flow cytometry cell counts data is represented as % Max. D) Gene expression profiles showing RPC markers in three different iPSC lines, showing a high enrichment. Mean±SEM represented from at least 3 independent experiments per individual. \\n         FIGS.  2 A- 2 E : Efficient generation of iPSC-RGCs require inhibition of TGF-β, Wnt, and BMP signaling. A) Phase contrast images of different stages of iPSC-RGC differentiation at DIV7, DIV16 and DIV35 is shown. B) Differentiation of retinal progenitor cells (RPCs) to retinal ganglion cells (RGCs) using defined inhibition or activation of developmental pathways, C) Percentage of Thy-1 (CD90) positive iPSC-RGCs in our studies in different differentiation conditions, D) Proportions of CD90 positive RGC cells across four differentiation conditions is shown. Each symbol represents an individual experiment. E) Percentage of CD90 positive cells obtained across three control iPSC lines across each differentiation condition. Mean±SEM represented from at least 3 independent experiments per individual. \\n         FIG.  3   : Flow cytometry analysis showing percentage of Brn3b positive and RBPMS positive iPSC-RGCs. \\n         FIG.  4   : Characterization of iPSC-RGCs after MACS sorting using ICC. The iPSC-RGCs stained positively to RGC markers, Brn3a, TUJ1, MAP2, RBPMS. Positive staining with GFAP showed &lt;5% astrocytes like cells. No cells stained positive with CRALBP. \\n         FIGS.  5 A and  5 B : Transcriptional Profiling of iPSC-RPCs. Quantitative-RT-PCR profiles showing relative expression of retinal ganglion cell and retinal subtypes (photoreceptors, amacrine, horizontal, and RPE) at Day 23 and Day 35 stages of development from iPSCs to RGCs. \\n         FIGS.  6 A- 6 D : Functional Characteristics of Induced Retinal Ganglion Cells. A) RGCs were transduced with AAV7m8-SNCG-eGFP and immunofluorescence RGC markers (BRN3a, THY1, TUBB3, MAP2, TUJ1, RBPMS). B) Density of RGCs expressing GFP and other axonal processes increases between DIV35 ( 5 B, cell c) and maturation DIV75 ( 5 B, cells a and b). Blue arrows point to cells selected for patch clamp recordings and that fired action potential upon depolarization. C) Action potential firing in response to depolarizing voltage step recorded from cell a and cell b (DIV75) and from cell c (DIV35). D) This trace demonstrates action potential firing in response to the two different depolarizing steps of current recorded from cell a from  FIG.  5 B . We also observe a spontaneous spike firing at the start of the trace. Same scale bar applies to all images. \\n         FIGS.  7 A- 7 G :  FIG.  7 A . Sample timeline for protocol for differentiation of iPSC-RGCs using small molecules and proteins.  FIG.  7 B  shows that by day 3, cells continued to expand, with cells reaching more than 100% confluence (Mag-10×).  FIG.  7 C  shows that by day 19, more rosette like cell clusters formed (starting around Day 15), (Mag-10×).  FIG.  7 D  shows that on day 24 before crosshatching, abundant cell clusters with crowded packed cells forming multi layers (light yellow parts).  FIG.  7 E  shows that on day 24, about 3 hours after crosshatching, neuron axon cell bodies start to form (Mag-20×).  FIG.  7 F  shows that on day 27, numerous ganglion cell bodies with extended axons (Mag-10×).  FIG.  7 G  shows Day 37, matured RGC with long axons, RGC cell bodies migrate forming clusters (Mag-10×). \\n         FIGS.  8 A- 8 B : Design and in vitro characterization of AAV2-CAG and AAV7m8 SNCG vectors.  FIG.  8 A  Outline of proviral expression cassettes used in the study. A1 and A2 illustrates comparative vector using AAV2 and CAG promoter with the cDNAs encoding eGFP or SIRT1 respectively. A3 and A4 Illustrates plasmid using the ganglion cell specific promoter, SNCG, driving cDNAs eGFP or SIRT1 respectively.  FIG.  8 B  Fluorescent micrographs of human SIRT1 protein expression in iPSC-RGCs transduced with AAV7m8. SNCG.hSIRT1-3×FLAG. Blue-DAPI nuclear stain, Red-BRN3A nuclear stain, green-3×Flag Tag epitope showing cytoplasmic and nuclear expression from the vector and blue/red/green merged images. \\n         FIGS.  9 A- 9 C : AAV7m8 transduction profile and RGC transduction efficiency following intravitreal delivery.  FIG.  9 A  Representative micrograph of retinal flat mount following intravitreal injection of AAV7m8-SNCG.eGFP. RGCs are labeled with BRN3A (red)(inlet). Representative retinal flat mount used for calculating RGC transduction efficiency with AAV7m8.  FIG.  9 B  Quantification of RGC transduction (n=10, experiments performed in triplicate; retinal whole mounts) comparing AAV2-CAG.SIRT1 vector.  FIG.  9 C  Representative cross-section of mouse retina following intravitreal injection of AAV2-CAG.SIRT1. RGCs are labeled with Brn3a (red) and localized to the ganglion cell layer (GCL). Cells expressing the SIRT1 transgene are labeled green and largely localized to the GCL. Data represented as mean±SEM. \\n         FIGS.  10 A- 10 C : Effect of AAV2 gene transfer on visual acuity and RGC in ONC. OKR recordings demonstrate significantly decreased visual acuity in eyes of ONC mice treated with AAV2-CAG.eGFP (n=10; experiments performed in triplicate). Treatment with AAV2-Mice treated with AAV2-CAG.SIRT1 (n=12; experiments performed in triplicate) show no significant effect on ( FIG.  10 A ) visual acuity or ( FIG.  10 B ) RGC survival by day 7.  FIG.  10 C  Representative RGC counts by nuclear BRN3A staining. Data represented as mean±SEM. *P&lt;0.05, **P&lt;0.01 by 1-way ANOVA with Tukey&#39;s HSD post-test. \\n         FIG.  11 A- 11 C : Effect of AAV7m8 gene transfer on visual acuity and RGC in ONC. OKR recordings demonstrate significantly decreased visual acuity in eyes of ONC mice treated with AAV2-eGFP (n=10; experiments performed in triplicate) compared with Sham injured mice (n=10; experiments performed in triplicate ***p=0.002).  FIG.  11 A ) Treatment with AAV7m8-SIRT1 (n=15; experiments performed in triplicate) showed a significant delay in loss of visual function in ONC (n=10; experiments performed in triplicate) compared with Sham injured mice (n=10; experiments performed in triplicate) (*p=0.03) on visual acuity seen in peach area outlined in the graph.  FIG.  11 B ) RGC flat mount counts demonstrate significantly decreased numbers in eyes of ONC mice treated with AAV2-eGFP (n=10; experiments performed in triplicate) compared with Sham injured mice (n=10; experiments performed in triplicate; ***p=0.001). Treatment with AAV7m8-SIRT1 (n=10; experiments performed in triplicate) showed a significant increase in retinal ganglion cell counted per flat month function in ONC (n=7) compared with control ONC mice (n=10; experiments performed in triplicate) (*p=0.03).  FIG.  11 C ) Representative RGC counts by nuclear BRN3A staining Data represented as mean±SEM. *P&lt;0.05, **P&lt;0.01 by 1-way ANOVA with Tukey&#39;s HSD post-test. \\n         FIGS.  12 A- 12 C : Functional validation for the association between PPP1R13L and primary open angle glaucoma.  FIG.  12 A ) Differential expression profile of Ppp1r131 transcript in mouse optic nerve head (ONH) with varying stages of intraocular pressure (IOP)-induced glaucoma. Data represent the fold change in Ppp1r131 expression between different stages of D2 mice (glaucoma) and D2 Gpnmb+ (control) samples. * represents q (FDR)&lt;0.05.  FIG.  12 B ) Localization of PPP1R13L protein in the human retina. Shown is the distribution of PPP1R13L by immunohistochemical localization in the retina from normal 68-year-old donor eyes. Overlay of images from DAPI (blue; nuclei) and PPP1R13L (red) in adult human retinal layers are shown on the right. The left represents primary antibody control. Scale bars are shown in each image. ONL, outer nuclear layer; OPL, outer plexiform layer; IPL, inner plexiform layer; GCL, ganglion cell layer.  FIG.  12 C ) Relative expression of PPP1R13L transcript in response to oxidative stress in induced pluripotent stem cell-derived retinal ganglion cells (iPSC-RGCs). Expression of PPP1R13L in iPSC-RGCs is shown under increasing concentrations of H 2 O 2  treatment. Plotted are the fold changes in comparison to no H 2 O 2 . Error bars represent standard error of the mean (SEM). *** denotes p&lt;0.0001, ** denotes p&lt;0.001, and * denotes p&lt;0.01. \\n         FIGS.  13 A- 13 H :  FIGS.  13 A- 13 B ) Mature iPSC-RGCs were differentiated according to Schematic.  FIG.  13 C ) RPCs characterized using qRT-PCR using gene specific primers and ICC using RPC specified antibodies.  FIG.  13 D ) Detailed description of iPSC-RGC characterization with MACS with CD90 antibody.  FIG.  13 E ) FACS sorting of iPSC-RGCs with different markers. Characterization using ICC markers of RGCs.  FIG.  13 F ) and other neuronal cells  FIG.  13 G ) obtained using our method.  FIG.  13 H  Electrophysiological responses of mature iPSC-RGCs response to light stimuli. \\n         FIGS.  14 A- 14 K :  FIG.  14 A . The schematic of iPSC-RGCs used for transplantation in a mouse retina.  FIG.  14 B- 14 C . Fundus Imaging and SD-OCT of a control mouse which received 1×PBS injection.  FIG.  14 D- 14 E . Fundus Imaging and SD-OCT of a mouse which received iPSC-RGC injection.  FIG.  14 F- 14 G . Fundus Imaging and follow-up studies for 10 mice (both sexes) to show the reproducibility of integrations post injections.  FIG.  14 H  cryosections of mouse showing integration of eGFP labeled iPSC-RGCs in the retinal ganglion cell layer (left image) and Flatmount retinal imaging (right image) showing transplanted iPSC-RGCs integrated into normal mouse retina.  FIG.  14 I . Light dependent change in firing when recorded from transplanted iPSC-RGCs (n=6).  FIG.  14 J- 14 K . Immunohistochemistry showing staining of transplanted iPSC-RGCs in mouse optic nerve sections two months post-transplantation. iPSC-RGCs are labelled with AAV2.7m8.eGFP and they colocalize with Neurofilament. \\n         FIGS.  15 A- 15 C : Images of hiPSC-RGCs expressing SNCG-eGFP after intravitreal injection.  FIG.  15 A ) Fluorescent images of the hiPSC-RGCs transduced with AAV2.7m8 SNCG-eGFP (4× and 10× magnification).  FIGS.  15 B,  15 C ) hiPSC-RGCs transduced with SNCG-eGFP vector were injected into the vitreous space of wild type C57BL/6J mice. Fundus photography of the transplanted hiPSC-RGCs was taken using BAF-cSLO and SD-OCT at 2-, 4-, and 6-weeks post-transplantation. \\n         FIGS.  16 A- 16 J : Images of transplanted hiPSC-RGCs in vivo.  FIGS.  16 A,  16 B,  16 C ) Fluorescent images of the flattened whole-mount retina of saline-injected left eye and hiPSC-RGCs injected right eye (B=4×, C=10×).  FIGS.  16 D,  16 E, and  16 F ) Neurite outgrowth of transplanted hiPSC-RGCs taken as two-photon images acquired from live retinal samples. Cryosection images of transplanted hiPSC-RGCs were detected in the GCL and INL layer of the murine retina and co-stained for  FIG.  16 G- 16 H ) Brn3,  FIG.  161   ) MAP2, and  FIG.  16 J ) Synapsin I. \\n         FIGS.  17 A- 17 H : Immuno-staining of the transplanted hiPSC-RGCs in mouse retina with RGC-specific markers. Flattened whole-mount images of transplanted hiPSC-RGCs were co-stained with RGC-specific markers A) BRN3, B) RBPMS, C) TUJ1, D) MAP2, E) VGLUT2, and F) PSD95. Donor hiPSC-RGCs were also co-stained with G) human nuclear antigen and H) Ku80 to confirm the donor origin of SNCG-eGFP+ cells.* Since the human nuclear antigen antibody was raised in mice, some cross-reactivity with the murine retina was observed. Z-1: Zoom-1, Z-2: Zoom-2, Z-4: Zoom-4, and Z-5: Zoom-5. \\n         FIGS.  18 A- 18 F : Injected hiPSCs integrate into the host retina and generate light responses. Two-photon images at the top row show targeted hiPSC-RGCs (centered) before, during, and after recording its light responses.  FIG.  18 A ) Image shows eGFP fluorescence observed from hiPSC-RGCs in the retinal sample before any attempt to approach it with the recording pipettes. The image is a projection of the Z-stack consisting of 234 optical slices acquired with a 0.5 μm step.  FIG.  18 B ) The red fluorescence from the CF 633 filled pipette and targeted cell is shown. This image comes from the single optical slice acquired at the end of the patch-clamp recording session. The patch-pipette can be seen on the right side of the cell. A slight repositioning of the targeted cell was due to its re-centering for the IR video system used to control pipette manipulations.  FIG.  18 C ) The Z-stack projection (151 optical slices with 0.5 μm step) shows combined eGFP and CF 633 fluorescence after completion of the pipette recording and withdrawal of the patch-pipette. The targeted cell has a bright yellow color. Note that eGFP+ hiPSC-RGC processes observed before the patch-clamp recording ( FIG.  18 A ) can still be readily identified after its completion ( FIG.  18 C ).  FIGS.  18 D and  18 E ) Light responses of the targeted cell recorded in the cell-attached configuration (voltage-clamp mode) and whole-cell configuration (voltage-clamp mode) are illustrated. Here upper graphs give the time course of light stimuli and indicate flash intensities, the middle graphs plot current vs time and membrane voltage vs time traces ( FIG.  18 D  and  FIG.  18 E  respectively), and the bottom graphs plot firing rate vs time traces. The current vs time trace from  FIG.  18 D  has been high pass filtered, and the voltage vs time trace from  FIG.  18 E  has not been corrected for the UP, the correction can be done by subtracting 15 mV from the reported values. The firing rate traces were calculated using a 200 ms time bin. The colored light bars indicate light stimulation events.  FIG.  18 F ) Detection of transplanted hiPSC-RGCs in the optic nerve of the mouse and its expression colocalized with the neurofilament marker. \\n         FIGS.  19 A- 19 F : Transplanted hiPSCs increase firing in response to depolarization induced by light or current injection. Two-photon images obtained from the single optical slice acquired at the end of the patch-clamp recording session are shown on the left of the figure. EGFP fluorescence from hiPSCs is illustrated on  FIG.  19 A , red fluorescence from the CF 633 filled pipette and targeted cell on  FIG.  19 B , and combined EGFP and CF 633 fluorescence on  FIG.  19 C . The pipette can be seen on the right side of the cell on  FIG.  19 B  and  FIG.  19 C . Targeted cell has a bright yellow color on  FIG.  19 C , the yellowish cell above it was stained with CF 633 during the preceding patching attempt. The scale bar is 50 μm. Traces from  FIG.  19 D  and  FIG.  19 E  show an increase in firing upon light stimulation similar to the corresponding traces from  FIG.  18   , but with a much lower baseline firing. Firing rates were calculated using a 200 ms time bin. Traces on  FIG.  19 F  illustrate membrane depolarization and action potential firing caused by an injection of depolarizing currents (two current steps of 30 and 40 pA, and the current ramp from 0 to 40 pA). For simplicity, current vs time traces on  FIG.  19 F  (1 st  and 3 rd  graphs from the top) show command current, not the actual membrane current measured by the amplifier (the latter will have the same amplitude and time course but will also include electrical noise and spiking activity). The membrane voltage from  FIG.  19 E  and  FIG.  19 F  wasn&#39;t corrected for UP, correction can be done by subtracting 15 mV from the reported values. \\n         FIGS.  20 A- 20 E : Example of hiPSC firing triggered by large and mostly spontaneous depolarizations. Two-photon images (single optical slice) on  FIGS.  20 A,  20 B , and  20 C show EGFP, CF 633, and combined EGFP+CF 633 fluorescence respectively at the end of the patch-clamp recording. The scale bar is 50 μm. Graphs from  FIG.  20 D  show firing (bottom trace, firing rate calculated using 200 ms time bin) triggered by two depolarizations, one aligned with the flash and a spontaneous one (middle trace). The trace at the top shows the time course of the stimulation, colored bars indicate light stimulation. Traces on  FIG.  20 E  show membrane depolarization and action potential firing in response to current injections (two steps of 20 and 30 pA and a ramp from 0 to 40 pA). To correct for the UP 15 mV should be subtracted from the reported membrane voltages. All traces were recorded in the current-clamp mode. \\n         FIGS.  21 A- 21 C : Larger membrane depolarizations caused by repeated stimuli can lead to smaller increases in the firing rate. All traces were recorded in the current-clamp mode from the same cell as that presented in  FIG.  18   .  FIGS.  21 A and  21 B  show depolarization and a related increase in the firing rate caused by light exposures. The flash from  FIG.  21 B  was delivered 30 s after that from  FIG.  21 A . Traces on  FIG.  21 C  were recorded following repeated stimulation with two flashes like that and including that illustrated on  FIG.  21 B , and two 2× brighter flashes. The traces at the top of each panel illustrate the time course and intensity of the stimulation ( FIG.  21 C  trace shows command current, not the actual membrane current recorded by the amplifier). Stimulation events are also indicated by the colored (light stimulation) and gray (current injection) bars. The firing rates were calculated using a 200 milliseconds time bin. For each stimulation time bins were aligned with stimulation onset (stimulation onset is between two-time bins). The membrane voltages were not corrected for the UP, correction can be done by subtracting 15 mV from the reported values. \\n         FIGS.  22 A- 22 B : Fundus image of a murine eye was taken following Pronase E injection in BAF and IR mode in the cSLO. 0.0001% of Pronase E was intravitreally injected into the vitreous cavity of C57BL/6 mice to degrade the ILM/NFL layer of the retina. The use of Pronase E  FIG.  22 A ) caused cataract formation and  FIG.  22 B ) induced inflammation as observed in the fundus image obtained in the BAF and IR modes of the cSLO. \\n         FIG.  23   : Sample Transplantation of iPSC-RGCs and RPC in human eyes. \\n     \\n    \\n    \\n     DETAILED DESCRIPTION \\n     Glaucoma is a group of progressive optic neuropathies that share common biological and clinical characteristics including irreversible changes to the optic nerve and visual field loss caused by death of retinal ganglion cells (RGCs). The loss of RGCs manifests as characteristic cupping or optic nerve degeneration, resulting in visual field loss in patients with Glaucoma. Published studies on in vitro RGC differentiation from stem cells utilized classical RGC signaling pathways mimicking retinal development in vivo. Although many strategies allowed for the generation of RGCs, increased variability between experiments and lower yield hampered the cross comparison between individual lines and between experiments. To address this critical need, we developed a reproducible, chemically defined in vitro methodology for generating retinal progenitor cell (RPC) populations from iPSCs, that are efficiently directed towards RGC lineage. Using this method, we reproducibly differentiated iPSCs into RGCs with greater than 80% purity, without any genetic modifications. We used small molecules and peptide modulators to inhibit BMP, Transforming growth factor beta (TGF-β or SMAD), and canonical Wnt pathways that reduced variability between iPSC lines and yielded functional and mature iPSC-RGCs. Using CD90.2 antibody and Magnetic Activated Cell Sorter (MACS) technique, we successfully purified Thy-1 positive RGCs with nearly 95% purity. \\n     The RGCs developed in accordance with the invention can be used to express transgenes of interest to assess their effects on RGC viability and function. For example, SIRT1 prevents retinal ganglion cell (RGC) loss in models of optic neuropathy following pharmacologic activation or genetic overexpression. The exact mechanism of loss is not known, although prior evidence suggests this due to oxidative stress to either neighboring cells or RGC specifically. In Example 2, the neuroprotective potential of RGC-selective SIRT1 gene therapy in the optic nerve crush (ONC) model was assessed. We hypothesized that AAV-mediated overexpression of SIRT1 in RGCs reduces RGC loss, thereby preserving visual function. Cohorts of C57Bl/6J mice received intravitreal injection of experimental or control AAVs using either a ganglion cell promoter or a constitutive promoter and ONC was performed. Visual function was examined by optokinetic response (OKR) for 7 days following ONC. Retina and optic nerves were harvested to investigate RGC survival by immunolabeling. The AAV7m8-SNCG.SIRT1 vector showed 44% transduction efficiency for RGCs compared with 25% (P&gt;0.05) by AAV2-CAG.SIRT1, and AAV7m8-SNCG.SIRT1 drives expression selectively in RGCs in vivo. Animal modeling of ONC demonstrated reduced visual acuity compared to controls. Intravitreal delivery of AAV7m8-SNCG.SIRT1 mediated significant preservation of the OKR and RGC survival compared to AAV7m8-SNCG.eGFP controls, an effect not seen with the AAV2 vector. RGC-selective expression of SIRT1 offers a targeted therapy in an animal model with significant ganglion cell loss. Over-expression of SIRT1 through AAV-mediated gene transduction suggests a RGC selective component of neuro-protection using the ONC model. This study expands our understanding of SIRT1 mediated neuroprotection in the context of compressive or traumatic optic neuropathy, making it a strong therapeutic candidate for testing in all optic neuropathies. Notably, application of resveratrol has been observed to preserve visual function and RGC survival by activating the SIRT1 pathway. \\n     Leveraging 10,900 whole exomes linked to EHR data in the Penn Medicine Biobank PMBB) for discovery, we addressed the association of the cumulative effect of rare predicted loss-of-function (pLOF) variants per gene on an exome-wide scale in iPSC-RGCs. After discovering several significant gene-disease associations (p&lt;10-6), we robustly replicated and tested these gene-disease associations in RGCs. This analysis revealed that variants in the PPPIR13L gene are associated with primary open angle glaucoma and the RGCs can be used as ideal in vitro models to study RGC pathobiology, evaluate stress conditions using chemicals and reagents to mimic Glaucoma or optic neuritis. \\n     We further transplanted mature human iPSC-derived retinal ganglion cells (hiPSCs-RGCs) intravitreally in wild type C57BL/6J mice. Our protocol for generating hiPSC-RGCs represents one of the most versatile, robust, and highly reproducible methods for the large-scale production of pure RGC populations without any gene modification (26). Following intravitreal injections, eGFP+hiPSC-RGCs were detected in 15 of the 16 wild type mice, with an average successful transplantation rate of about 94%. Overall, an average of 672 donor hiPSC-RGCs were visible per explanted host retina. The transplanted RGCs integrated within the ganglion cell layer of host retinas and survived at least 5-months post-transplantation. Furthermore, the transplanted hiPSC-RGCs were functional and showed electrophysiologic profiles similar to native mouse RGCs. The data disclosed herein indicate that hiPSC-RGCs are a necessary source of donor cells that may aid in the development of patient-tailored cell therapies designed to improve visual function in glaucoma. \\n     Disclosed herein are methods for generating RPCs and RGCs from iPSCs and methods of using said RGCs for treatment of ocular disorders and in screening assays to identify new biomarkers of disease and efficacious agents for the treatment of the same. \\n     Definitions \\n     The present subject matter may be understood more readily by reference to the following detailed description which forms part of this disclosure. It is to be understood that this invention is not limited to the specific products, methods, conditions or parameters described and/or shown herein, and that the terminology used herein for the purpose of describing particular embodiments by way of example only and is not intended to be limiting of the claimed invention. \\n     Unless otherwise defined herein, scientific, and technical terms used in connection with the present application shall have the meanings that are commonly understood by those of ordinary skill in the art. In addition to definitions included in this sub-section, further definitions of terms are interspersed throughout the text. \\n     In this invention, “a” or “an” means “at least one” or “one or more,” etc., unless clearly indicated otherwise by context. The term “or” means “and/or” unless stated otherwise. In the case of a multiple-dependent claim, however, use of the term “or” refers to more than one preceding claim in the alternative only. \\n     A “sample” refers to a sample from a subject that may be tested. The sample may comprise cells, and it may comprise body fluids, such as blood, serum, plasma, cerebral spinal fluid, urine, saliva, tears, pleural fluid, and the like. \\n     As used herein, the terms “component,” “composition,” “composition of compounds,” “compound,” “drug,” “pharmacologically active agent,” “active agent,” “therapeutic,” “therapy,” “treatment,” or “medicament” are used interchangeably herein to refer to a compound or compounds or composition of matter which, when administered to a subject (human or animal) induces a desired pharmacological and/or physiologic effect by local and/or systemic action. The terms “agent” and “test compound” denote a chemical compound, a mixture of chemical compounds, a biological macromolecule, or an extract made from biological materials such as bacteria, plants, fungi, or animal (particularly mammalian) cells or tissues. \\n     As used herein the term “wild type” is a term of the art understood by skilled persons and means the typical form of an organism, strain, gene or characteristic as it occurs in nature as distinguished from mutant or variant forms. As used herein the term “variant” should be taken to mean the exhibition of qualities that have a pattern that deviates from the wild type or a comprises non naturally occurring components. \\n     The terms “non-naturally occurring” or “engineered” are used interchangeably and indicate the involvement of the hand of man. The terms, when referring to nucleic acid molecules or polypeptides mean that the nucleic acid molecule or the polypeptide is at least substantially free from at least one other component with which they are naturally associated in nature and as found in nature. \\n     The term “effective amount” or “therapeutically effective amount” refers to the amount of an agent that is sufficient to effect beneficial or desired results. The therapeutically effective amount may vary depending upon one or more of: the subject and disease condition being treated, the weight and age of the subject, the severity of the disease condition, the manner of administration and the like, which can readily be determined by one of ordinary skill in the art. The term also applies to a dose that will provide an image for detection by any one of the imaging methods described herein. The specific dose may vary depending on one or more of: the particular agent chosen, the dosing regimen to be followed, whether it is administered in combination with other compounds, timing of administration, the tissue to be imaged, and the physical delivery system in which it is carried. \\n     The phrase “consisting essentially of” when referring to a particular nucleotide sequence or amino acid sequence means a sequence having the properties of a given SEQ ID NO. For example, when used in reference to an amino acid sequence, the phrase includes the sequence per se and molecular modifications that would not affect the basic and novel characteristics of the sequence. \\n     A “clonal cell population” refers to a group of identical cells that are derived from the same cell. \\n     “Multipotent” implies that a cell is capable, through its progeny, of giving rise to several different cell types found in the adult animal. \\n     “Pluripotent” implies that a cell is capable, through its progeny, of giving rise to all the cell types which comprise the adult animal including the germ cells. Both embryonic stem and embryonic germ cells are pluripotent cells under this definition. \\n     The term “cell line” as used herein can refer to cultured cells that can be passaged at least one time without terminating. The invention relates to cell lines that can be passaged at least 1, 2, 5, 10, 15, 20, 30, 40, 50, 60, 80, 100, and 200 times. Cell passaging is defined hereafter. \\n     The term “suspension” as used herein can refer to cell culture conditions in which cells are not attached to a solid support. Cells proliferating in suspension can be stirred while proliferating using apparatus well known to those skilled in the art. \\n     The term “monolayer” as used herein can refer to cells that are attached to a solid support while proliferating in suitable culture conditions. A small portion of cells proliferating in a monolayer under suitable growth conditions may be attached to cells in the monolayer but not to the solid support. Preferably less than 15% of these cells are not attached to the solid support, more preferably less than 10% of these cells are not attached to the solid support, and most preferably less than 5% of these cells are not attached to the solid support. \\n     The term “plated” or “plating” as used herein in reference to cells can refer to establishing cell cultures in vitro. For example, cells can be diluted in cell culture media and then added to a cell culture plate, dish, or flask. Cell culture plates are commonly known to a person of ordinary skill in the art. Cells may be plated at a variety of concentrations and/or cell densities. \\n     The term “cell plating” can also extend to the term “cell passaging.” Cells of the invention can be passaged using cell culture techniques well known to those skilled in the art. The term “cell passaging” can refer to a technique that involves the steps of (1) releasing cells from a solid support or substrate and disassociation of these cells, and (2) diluting the cells in media suitable for further cell proliferation. Cell passaging may also refer to removing a portion of liquid medium containing cultured cells and adding liquid medium to the original culture vessel to dilute the cells and allow further cell proliferation. In addition, cells may also be added to a new culture vessel which has been supplemented with medium suitable for further cell proliferation. \\n     The term “proliferation” as used herein in reference to cells can refer to a group of cells that can increase in number over a period of time. \\n     The term “reprogramming” or “reprogrammed” as used herein can refer to materials and methods that can convert a cell into another cell having at least one differing characteristic. Also, such materials and methods may reprogram or convert a cell into another cell type that is not typically expressed during the life cycle of the former cell. For example, (1) a non-totipotent cell can be reprogrammed into a totipotent cell or (2) a precursor cell can be reprogrammed into a totipotent cell. \\n     The term “differentiated cell” as used herein can refer to a precursor cell that has developed from an unspecialized phenotype to a specialized phenotype. For example, embryonic cells can differentiate into an epithelial cell lining the intestine. Materials and methods of the invention can reprogram differentiated cells into totipotent cells. Differentiated cells can be isolated from a fetus or a live born animal, for example. \\n     The term “undifferentiated cell” as used herein can refer to a precursor cell that has an unspecialized phenotype and is capable of differentiating. An example of an undifferentiated cell is a stem cell. \\n     The term “differentiated cell” means any cell that does not have stem cell capacity, where stem cell capacity is the ability to divide in a manner that renews a baseline state of relative undifferentiation (stem cell phenotype) will simultaneously producing cells of different and developmentally more restricted state of differentiation (i.e. differentiated non-stem cells). In practice, the terms differentiated and undifferentiated always require a developmental reference, whether explicit or implicit. Tissue stem cells are undifferentiated relative to their differentiated, non-stem progeny cells. However, relative to iPSCs, embryonic stem cells, and embryonic precursor cells, tissue stem cells are more differentiated. It should be noted that placing many primary cells in culture can lead to some loss of fully differentiated characteristics. Thus, simply culturing such cells are included in the term differentiated cells and do not render these cells tissue stem cells or pluripotent cells. The transition of a differentiated cell to pluripotency requires a reprogramming stimulus beyond the factors that lead to partial loss of differentiated character in culture. Reprogrammed cells also have the characteristic of the capacity of extended passaging without loss of proliferative potential, relative to their primary cell parents, which generally have capacity for only a limited number of divisions in culture. \\n     As used herein, the term “somatic cell” refers to any cell other than a germ cell, a cell present in or obtained from a pre-implantation embryo, or a cell resulting from proliferation of such a cell in vitro. Stated another way, a somatic cell refers to any cells forming the body of an organism, as opposed to germ line cells. In mammals, germ line cells (also known as “gametes”) are the spermatozoa and ova, which fuse during fertilization to produce a cell called a zygote, from which the entire mammalian embryo develops. Every other cell type in the mammalian body (aside from the sperm and ova), is a somatic cell: internal organs, skin, bones, blood, and connective tissue are all made up of somatic cells. In some embodiments the somatic cell is a “non-embryonic somatic cell”, by which is meant a somatic cell that is not present in or obtained from an embryo and does not result from proliferation of such a cell in vitro. In some embodiments, the somatic cell is an “adult somatic cell”, by which is meant a cell that is present in or obtained from an organism other than an embryo or a fetus or results from proliferation of such a cell in vitro. Unless otherwise indicated the methods for reprogramming a differentiated cell can be performed both in vivo and in vitro (where in vivo is practiced when a differentiated cell is present within a subject, and where in vitro is practiced using isolated differentiated cell maintained in culture). In some embodiments, where a differentiated cell or population of differentiated cells are cultured in vitro, the differentiated cell can be cultured in an organotypic slice culture, such as described in, e.g., Meneghel-Rozzo et al., 2004, Cell Tissue Res, 316:295-303, which is incorporated herein in its entirety by reference. \\n     As used herein, the term “adult cell” refers to a cell found throughout the body after embryonic development. \\n     As used herein, the terms “iPSC”, “iPS cell” and “induced pluripotent stem cell” are used interchangeably and refers to a pluripotent cell technically derived (e.g., induced by complete or partial reversal) from a differentiated cell (e.g. a non-pluripotent cell), typically an adult differentiated cell. \\n     The term “progenitor cell” is used herein to refer to cells that have a cellular phenotype that is at an earlier step along a developmental pathway or progression than is a later differentiated cell relative to a cell to which it can give rise by differentiation. Often, progenitor cells also have significant or very high proliferative potential. Progenitor cells can give rise to multiple distinct differentiated cell types or to a single differentiated cell type, depending on the developmental pathway and on the environment in which the cells develop and differentiate. Progenitor cells are distinct from tissue stem cells in that they lack asymmetric self-renewal. In the absence of their own producer stem cell, progenitor cells&#39; populations are rapidly exhausted because of their inability to simultaneously preserve their own initial degree of differentiation. \\n     The phrase “Retinal Progenitor Cell” or “RPC” refers to multipotent progenitor cells that can give rise to all the various cell types of the retina. The cells retinal ganglion cells, amacrine cells, bipolar cells, horizontal cells, rod photoreceptors, cone photoreceptors, and Müller glia cells. The differentiation of retinal precursor cells into the mature cell types found in the retina is coordinated in time and space by factors within the cell as well as factors in the environment of the cell. \\n     In the context of cell ontogeny, the term “differentiate”, or “differentiating” is a relative term meaning a “differentiated cell” which has progressed further down the developmental pathway than its precursor cell. Thus, in some embodiments, a reprogrammed cell as this term is defined herein, can differentiate to lineage-restricted precursor cells (such as a mesodermal stem cell or a endodermal stem cell), which in turn can differentiate into other types of precursor cells further down the pathway (such as a tissue specific precursor, for example, a retinal precursor, an eye-specific cell type precursor, a cardiomyocyte precursor, or a pancreatic precursor), and then to an end-stage differentiated cell, which plays a characteristic role in a certain tissue type, and may or may not retain the capacity to proliferate further. \\n     The term “exogenous” refers to a substance present in a cell that was introduced from outside the cell by either a natural process or via genetic recombination. The terms “exogenous” when used herein refers to a nucleic acid (e.g., a nucleic acid encoding a reprogramming transcription factor) or a protein (e.g., a transcription factor polypeptide) that has been introduced by a process involving the hand of man into a biological system such as a cell or organism in which it is not normally found or in which it is found (17) in lower amounts. A substance (e.g. a nucleic acid or a protein) will be considered exogenous if it is introduced into a cell or an ancestor of the cell that inherits the substance. In contrast, the term “endogenous” refers to a substance that is produced within the cell by natural processes. \\n     The terms “increased”, “increase” or “enhance” or “activate” are all used herein to generally mean an increase by a statically significant amount; for the avoidance of any doubt, the terms “increased”, “increase” or “enhance” or “activate” means an increase of at least 10% as compared to a reference level, for example an increase of at least about 20%, or at least about 30%, or at least about 40%, or at least about 50%, or at least about 60%, or at least about 70%, or at least about 80%, or at least about 90% or up to and including a 100% increase or any increase between 10-100% as compared to a reference level, or at least about a 2-fold, or at least about a 3-fold, or at least about a 4-fold, or at least about a 5-fold or at least about a 10-fold increase, or any increase between 2-fold and 10-fold or greater as compared to a reference level. \\n     The term “contacting” or “contact” as used herein as in connection with contacting a differentiated cell (e.g. tissue stem cell) with a compound as disclosed herein (e.g., an expression vector), includes subjecting the cell to a culture media, which comprises the compound. Where the differentiated cell is in vivo, contacting the differentiated cell with a compound includes administering the compound in a composition to a subject via an appropriate administration route such that the compound contacts the differentiated cell in vivo. \\n     As used herein, the term “expanding” refers to increasing the number of like cells through symmetrical cell division (mitosis). The term “proliferating” and “expanding” are used interchangeably. \\n     The phrase “optic neuropathy” refers to optic nerve abnormalities or damage. This damage could be from blocked blood flow, certain medical conditions or toxic exposure. A “progressive optic neuropathy” refers to an optic neuropathy that causes progressive vision loss. “Glaucoma” is a group of progressive optic neuropathies that share common biological and clinical characteristics including irreversible changes to the optic nerve and visual field loss caused by the death of retinal ganglion cells (RGCs). The loss of RGCs manifests as characteristic cupping or optic nerve degeneration, resulting in visual field loss in patients. \\n     The phrase “retinal ganglion cells” or “RGCs” refers to a type of neuron located near the inner surface (the ganglion cell layer) of the retina of the eye. Retinal ganglion cells collectively transmit image-forming and non-image forming visual information from the retina to several regions in the thalamus, hypothalamus, and mesencephalon, or midbrain. \\n     With reference to nucleic acids of the invention, the term “isolated nucleic acid” is sometimes used. This term, when applied to DNA, refers to a DNA molecule that is separated from sequences with which it is immediately contiguous (in the 5′ and 3′ direction) in the naturally occurring genome of the organism from which it originates. For example, the “isolated nucleic acid” may comprise a DNA or cDNA molecule inserted into a vector, such as a plasmid or virus vector, or integrated into the DNA of a prokaryote or eukaryote. \\n     The term “promoter region or expression control sequence” refers to the transcriptional regulatory regions of a gene, which may be found at the 5′ or 3′ side of the coding region, or within the coding region, or within introns. Such sequences regulate expression of a polypeptide coded for by a polynucleotide to which it is functionally (“operably”) linked. Expression can be regulated at the level of the mRNA or polypeptide. Thus, the term expression control sequence includes mRNA-related elements and protein-related elements. Such elements include promoters, domains within promoters, upstream elements, enhancers, elements that confer tissue or cell specificity, response elements, ribosome binding sequences, transcriptional terminators, etc. \\n     Those skilled in the art will recognize that a nucleic acid vector can contain nucleic acid elements other than the promoter element and the nucleic acid molecule of interest. These other nucleic acid elements include, but are not limited to, origins of replication, ribosomal binding sites, nucleic acid sequences encoding drug resistance enzymes or amino acid metabolic enzymes, and nucleic acid sequences encoding secretion signals, localization signals, or signals useful for polypeptide purification. \\n     In some embodiments, the expression control sequence comprises a tissue- or organ-specific promoter. Many such expression control sequences will be evident to the skilled worker. \\n     The term “vector” refers to a small carrier DNA molecule into which a DNA sequence can be inserted for introduction into a host cell where it will be replicated. An “expression vector” is a specialized vector that contains a gene or nucleic acid sequence with the necessary regulatory regions needed for expression in a host cell. \\n     Many techniques are available to those skilled in the art to facilitate transformation, transfection, or transduction of the expression construct into a prokaryotic or eukaryotic organism. The terms “transformation”, “transfection”, and “transduction” refer to methods of inserting a nucleic acid and/or expression construct into a cell or host organism. These methods involve a variety of techniques, such as treating the cells with high concentrations of salt, an electric field, or detergent, to render the host cell outer membrane or wall permeable to nucleic acid molecules of interest, microinjection, PEG-fusion, a viral vector, a naked plasmid and the like. \\n     As used herein, the terms “reporter,” “reporter system”, “reporter gene,” or “reporter gene product” shall mean an operative genetic system in which a nucleic acid comprises a gene that encodes a product that when expressed produces a reporter signal that is a readily measurable, e.g., by biological assay, immunoassay, radio immunoassay, or by colorimetric, fluorogenic, chemiluminescent or other methods. The nucleic acid may be either RNA or DNA, linear or circular, single or double stranded, antisense or sense polarity, and is operatively linked to the necessary control elements for the expression of the reporter gene product. The required control elements will vary according to the nature of the reporter system and whether the reporter gene is in the form of DNA or RNA, but may include, but not be limited to, such elements as promoters, enhancers, translational control sequences, poly A addition signals, transcriptional termination signals and the like. \\n     The introduced nucleic acid may or may not be integrated (covalently linked) into nucleic acid of the recipient cell or organism. In mammalian cells, for example, the introduced nucleic acid may be maintained as an episomal element or independent replicon such as a plasmid. Alternatively, the introduced nucleic acid may become integrated into the nucleic acid of the recipient cell or organism and be stably maintained in that cell or organism and further passed on or inherited to progeny cells or organisms of the recipient cell or organism. Finally, the introduced nucleic acid may exist in the recipient cell or host organism only transiently. \\n     The term “selectable marker gene” refers to a gene that when expressed confers a selectable phenotype, such as antibiotic resistance, on a transformed cell. \\n     The term “operably linked” means that the regulatory sequences necessary for expression of a coding sequence are placed in the DNA molecule in the appropriate positions relative to the coding sequence so as to effect expression of the coding sequence. This same definition is sometimes applied to the arrangement of coding sequences and transcription control elements (e.g. promoters, enhancers, and termination elements) in an expression vector. This definition is also sometimes applied to the arrangement of nucleic acid sequences of a first and a second nucleic acid molecule wherein a hybrid nucleic acid molecule is generated. \\n     The term “primer” as used herein refers to an oligonucleotide, either RNA or DNA, either single-stranded or double-stranded, either derived from a biological system, generated by restriction enzyme digestion, or produced synthetically which, when placed in the proper environment, is able to functionally act as an initiator of template-dependent nucleic acid synthesis. When presented with an appropriate nucleic acid template, suitable nucleoside triphosphate precursors of nucleic acids, a polymerase enzyme, suitable cofactors and conditions such as a suitable temperature and pH, the primer may be extended at its 3′ terminus by the addition of nucleotides by the action of a polymerase or similar activity to yield a primer extension product. The primer may vary in length depending on the particular conditions and requirement of the application. For example, in diagnostic applications, the oligonucleotide primer is typically 15-25 or more nucleotides in length. The primer must be of sufficient complementarity to the desired template to prime the synthesis of the desired extension product, that is, to be able anneal with the desired template strand in a manner sufficient to provide the 3′ hydroxyl moiety of the primer in appropriate juxtaposition for use in the initiation of synthesis by a polymerase or similar enzyme. It is not required that the primer sequence represent an exact complement of the desired template. For example, a non-complementary nucleotide sequence may be attached to the 5′ end of an otherwise complementary primer. Alternatively, non-complementary bases may be interspersed within the oligonucleotide primer sequence, provided that the primer sequence has sufficient complementarity with the sequence of the desired template strand to functionally provide a template-primer complex for the synthesis of the extension product. \\n     As used herein, the term “treatment” refers to the application or administration of a therapeutic agent to a patient, or application or administration of a drug to an isolated tissue or cell line from a patient, who has a medical condition, e.g., a disease or disorder, a symptom of disease, or a predisposition toward a disease, with the purpose to cure, heal, alleviate, relieve, alter, remedy, ameliorate, improve, or affect the disease, the symptoms of disease, or the predisposition toward disease. In an alternative embodiment, tissues or cells or cell lines from a normal donor may also be “treated”. \\n     As used herein, the terms “modulate”, “modulating” or “modulation” refer to changing the rate at which a particular process occurs, inhibiting a particular process, reversing a particular process, and/or preventing the initiation of a particular process. Accordingly, if the particular process is tumor growth or metastasis, the term “modulation” includes, without limitation, decreasing the rate at which tumor growth and/or metastasis occurs; inhibiting tumor growth and/or metastasis; reversing tumor growth and/or metastasis (including tumor shrinkage and/or eradication) and/or preventing tumor growth and/or metastasis. \\n     A “pharmaceutical composition” comprises a pharmacologically effective amount of a therapeutic agent, optionally other drug(s), and a pharmaceutically acceptable carrier. As used herein, “pharmacologically effective amount,” “therapeutically effective amount” or simply “effective amount” refers to that amount of an agent effective to produce a commercially viable pharmacological, therapeutic, preventive or other commercial result. \\n     “Pharmaceutically acceptable carrier” refers to a carrier or diluent for administration of a therapeutic agent. Pharmaceutically acceptable carriers for therapeutic use are well known in the pharmaceutical art, and are described, for example, in Remington&#39;s Pharmaceutical Sciences, A R Gennaro (editor), 18.sup.th edition, 1990, Mack Publishing, which is hereby incorporated by reference herein. Such carriers include, but are not limited to, saline, buffered saline, dextrose, water, glycerol, ethanol, and combinations thereof. The term specifically excludes cell culture medium. For drugs administered orally, pharmaceutically acceptable carriers include, but are not limited to pharmaceutically acceptable excipients such as inert diluents, disintegrating agents, binding agents, lubricating agents, sweetening agents, flavoring agents, coloring agents and preservatives. Suitable inert diluents include sodium and calcium carbonate, sodium and calcium phosphate, and lactose, while corn starch and alginic acid are suitable disintegrating agents. Binding agents may include starch and gelatin, while the lubricating agent, if present, will generally be magnesium stearate, stearic acid or talc. If desired, the tablets may be coated with a material such as glyceryl monostearate or glyceryl distearate, to delay absorption in the gastrointestinal tract. The compounds of the invention may also be admixed, encapsulated, conjugated or otherwise associated with other molecules, molecule structures or mixtures of compounds, as for example, liposomes, receptor targeted molecules, oral, rectal, topical or other formulations, for assisting in uptake, distribution and/or absorption. Representative United States patents that teach the preparation of such uptake, distribution and/or absorption assisting formulations include, but are not limited to, U.S. Pat. Nos. 5,108,921; 5,354,844; 5,416,016; 5,459,127; 5,521,291; 5,543,158; 5,547,932; 5,583,020; 5,591,721; 4,426,330; 4,534,899; 5,013,556; 5,108,921; 5,213,804; 5,227,170; 5,264,221; 5,356,633; 5,395,619; 5,416,016; 5,417,978; 5,462,854; 5,469,854; 5,512,295; 5,527,528; 5,534,259; 5,543,152; 5,556,948; 5,580,575; and 5,595,756. The single or double stranded oligos of the present invention encompass any pharmaceutically acceptable salts, esters, or salts of such esters, or any other compound which, upon administration to an animal including a human, is capable of providing (directly or indirectly) the biologically active metabolite or residue thereof. Accordingly, for example, prodrugs and pharmaceutically acceptable salts of the compounds of the invention, pharmaceutically acceptable salts of such prodrugs, and other bioequivalents are also encompassed by the present invention. In addition, single stranded oligos may be formulated for oral delivery (Tillman et al., J Pharm Sci 97: 225, 2008; Raoof et al., J Pharm Sci 93: 1431, 2004; Raoof et al., Eur J Pharm Sci 17: 131, 2002; U.S. Pat. No. 6,747,014; U.S. 2003/0040497; U.S. 2003/0083286; U.S. 2003/0124196; U.S. 2003/0176379; U.S. 2004/0229831; US 2005/0196443; U.S. 2007/0004668; U.S. 2007/0249551; WO O 2/092616 ; WO 03/017940; WO 03/018134; WO 99/60012). Such formulations may incorporate one or more permeability enhancers such as sodium caprate that may be incorporated into an enteric-coated dosage form with the oligo. \\n     The term “inhibitor” refers to compounds and/or molecules that reduce the expression, amount, or activity of another biological compound. The phrase “bone morphogenic protein inhibitor” or “BMP inhibitor” refers to an agent that inhibits BMP activity and/or BMP nucleic acid expression. \\n     The phrase “TGF-β inhibitor” or “transforming growth factor beta inhibitor” refers to an agent that inhibits TGF-β activity and/or TGF-β nucleic acid expression. \\n     The phrase “Wnt signaling pathway” refers to an evolutionarily conserved pathway that regulates crucial aspects of cell fate determination, cell migration, cell polarity, neural patterning and organogenesis during embryonic development. The Wnts are secreted glycoproteins that act as short-range ligands to activate receptor-mediated signaling pathways and comprise a large family of proteins in humans. To date major signaling branches downstream of the Fz receptor have been identified including a canonical or Wnt/β-catenin dependent pathway and the non-canonical or β-catenin-independent pathway. The phrase “canonical Wnt pathway” or “canonical Wnt signaling pathway” refers to Wnt signaling that inhibits the degradation of β-catenin, which can regulate transcription of a number of genes. Canonical WNTs control the β-catenin dynamics as the cytoplasmic level of β-catenin is tightly regulated via phosphorylation by the ‘destruction complex’, consisting of glycogen synthase kinase 3β (GSK3β), casein kinase 1α (CK1α), the scaffold protein AXIN, and the tumor suppressor adenomatous polyposis coli (APC). An “inhibitor of the canonical WNT pathway” refers to any agent or compound that inhibits this pathway. \\n     The phrase “optokinetic response” refers to a combination of a slow and fast phase eye movements. It is observed seen when a mouse/individual tracks (pursuit movement) a moving object with their eyes, which then moves out of the field of vision at which point their eye moves back to the position it was in (saccade movement) when it first saw the object. The reflex/response is monitored and tracked as a measure to assess visual function. \\n     The following examples are provided to illustrate certain embodiments of the invention. They are not intended to limit the invention in any way. \\n     Example 1 \\n     Methods for Reproducible Differentiation of iPSCs into RPCs and RGCs \\n     We detail a chemically defined, in vitro technique for cultivating RPC populations from iPSCs that are then directed toward the RGC lineage. True to their in vivo development, we directed control lines of iPSCs to differentiate in a stepwise manner using a temporal induction method centered on small molecule and peptide modulator treatment to inhibit BMP and TGF-β (SMAD), and canonical Wnt, yielding a robust population of iPSC-RGCs after Day 36. Analysis at the molecular and physiologic levels using flow cytometry, immunolabeling, gene expression, and electrophysiology yielded results in accordance with the RGC lineage and allowed specific subtype identification. \\n     The following materials and methods are provided to facilitate the practice of the Example 1. \\n     Human iPSC Culture \\n     Undifferentiated iPS cells, Control 1 (CHOPWT8), Control 2 (CHOPWT9), and Control 3 (CHOPWT10) were derived and characterized as previously published showing a complete analysis of iPSC characteristics. 34-36  All human sample collection protocols were approved by the University of Pennsylvania and Children&#39;s Hospital of Philadelphia Human Subjects Research Institutional Review Board following the Declaration of Helsinki. All methods were performed in accordance with the relevant research guidelines and regulations of University of Pennsylvania. Written informed consent was obtained from all human cell donors. The iPSC cells were maintained in iPSC medium (Dulbecco&#39;s modified essential medium/Ham&#39;s F12 nutrient media; DMEM/F12 (50:50; Corning)) containing 1% Glutamax, 1% penicillin/streptomycin (PS), 15% Knockout serum replacement (KSR), 1% non-essential amino acids (NEAA), 0.1 mM β-Mercaptoethanol (2-ME) (Life Technologies, CA), and 5 ng/mL of basic fibroblast growth factor (bFGF; R&amp;D Systems) on 0.1% gelatin coated dishes with irradiated mouse embryonic fibroblast (iMEFs). \\n     Retinal Progenitor Cell Generation and Conditions \\n     iPSCs were cultured on 0.1% gelatinized plates containing iMEFs in 37° C. 5% O 2  and 5% CO 2  conditions. Cells were maintained until approximately 75% confluence, then feeder cells were depleted and approximately 1.5×10 6  iPSCs were seeded in one well of 6-well tissue culture dish (Corning) coated with 1:100 diluted growth factor reduced Matrigel. iPSCs were maintained in iPSC: MEF-conditioned medium (80:20)+20 ng/mL of bFGF and 5 ng/ml of stable bFGF. The MEF-conditioned media was prepared by plating iMEFs onto 0.1% gelatin at a density of 20,000 cells/cm 2  in iPSC media. Two days post-plating, media was collected, filtered, and either used directly or cryopreserved for later use. iPSCs were maintained in 37° C. at 5% O 2  and 5% CO 2  until reaching 100% confluence then transferred to 37° C., 5% CO 2  overnight prior to induction. On day 0, iPSC: MEF-conditioned media was changed to RPC induction media: DMEM/F12 (50:50), 1% P/S, 1% Glut, 1% NEAA, 0.1 mm 2-ME, 2% B27 supplement (w/o vitamin A), 1% N2 supplement, containing 2 μM XAV939 (X) (Wnt inhibitor; SelleckChem), 10 μM SB431542 (SB) (TGF-β inhibitor; SelleckChem), 100 nM LDN193189 (L) (BMP inhibitor; Stemgent), 10 mM nicotinamide (Sigma-Aldrich), and 10 ng/mL IGF1 (R&amp;D Systems). Cultures were fed daily for 4 days. On day 4, culture media was exchanged with RPC induction media containing: 2 μM XAV939, 10 μM SB431542, 100 nM LDN193189, 10 ng/mL IGF1, and 10 ng/mL bFGF. Media was changed daily for 12 days. On day 16 of differentiation, seven conditions—1) L, 2) X, 3) 1.5 μM CHIR99021 (CHIR) (Wnt agonist; SelleckChem), 4) LX, 5) LSB, 6) LSB-CHIR and 7) LSBX were evaluated for RPC and RGC markers on day 23. Cultures from the following four conditions—1) LSBX, 2) LSB, 3) LX, and 4) 1.5 μM CHIR99021 (CHIR) (Wnt agonist; SelleckChem) were further evaluated at 35. An additional five conditions—1) L only, 2) X only, 3) LSB, 4) LSBX, and 5) LSB-CHIR were evaluated for RPC markers on day 23 to evaluate for transcript expression differences across different conditions in RPCs. Cultures were treated for 6 days before transcript analysis. \\n     Retinal Ganglion Cell (RGC) Differentiation. \\n     Prior to RGC differentiation, medium was changed daily using RGC induction media containing SHH (sonic hedgehog) (250 ng/mL; R&amp;D systems) or SAG (100 nM; Tocris), SHH was replaced with SAG after establishing equal competence, FGF8 (100 ng/mL, R&amp;D systems) for two days. For RGC differentiation (see  FIGS.  1 D and  2 B  for kinetics), on day 24, cells were manually crossed into small clusters with Leibovitz&#39;s medium containing 34 uM Glucose using the crosshatching technique as previously described (Espuny-Camacho et al., 2013), then replated at a density of 1.0×10 5  cells/well of 6-well plate coated with 1:100 diluted growth factor reduced Matrigel in RGCs induction media containing: Follistatin 300 (100 ng/ml), cyclopamine (0.5 uM, Tocris), DAPT (3 uM, Tocris), and 4.2 μM Rock inhibitor (Y27632). Media was changed 24 h post-plating with RGC induction media containing: Follistatin 300 (100 ng/ml) and DAPT (3 uM), daily for two days. From day 27, media was changed to RGC induction media containing: DAPT (3 uM, Tocris), 10 μM Rock inhibitor (Y27632), forskolin (5 uM; SelleckChem), cAMP (400 uM; Sigma-Aldrich), BDNF (40 ng/mL; R&amp;D systems), NT4 (5 ng/mL; R&amp;D systems), and CNTF (10 ng/mL; R&amp;D systems). Media was changed every 2-3 days until day 36. After maturation (D36), medium was exchanged every 3-4 days with RGC induction media containing: DAPT (3 uM, Tocris), 10 μM Rock inhibitor (Y27632). Detailed step-wise methodology of iPSC-RGC generation, reagents and concentrations used in our study are provided below. In certain embodiments, activators of the SIRT1 pathway, such as resveratrol, may be included in the differentiation media during RGC differentiation. \\n     Protocol for Differentiation of iPSC-RGCs Using Small Molecules and Proteins \\n     A timeline showing the differentiation of iPSC-RGCs is shown in  FIG.  7 A . \\n     iPSC Initiation: Day-3 to Day-1 \\n     Day1: MEFs cell depletion: When iPSC cells reach 80% confluent, 1 ml of TrypLE solution was applied and the cells incubated at 37° C. for 4-5 minutes. The TrypLE medium was removed after incubation and iPSCs replated into a 100 mm dish precoated with matrigel, in medium containing: 80% HES/20% MEF-CM+20 ng/ml bFGF+4 uM Y27632. The plate was incubated in a 37° C., 5% CO 2  and 5% O 2  incubator. In an alternate embodiment, when iPSC cells reach 60% confluence, the iPSC cells are plated on 100 mm dish precoated with Matrigel (without MEFs) and in medium containing either: \\n     i) 80% HES/20% MEF-CM+20 ng/ml bFGF+4 uM Y27632. Cells incubated in 37° C., 5% CO 2  and 5% O 2  incubator. \\n     ii) 100% HES+20 ng/ml bFGF+4 uM Y27632. Cells incubated in 37° C., 5% CO2 and 5% O 2  incubator. \\n     iii) 100% STEMACS iPS Brew XF (Miltenyi Biotec)+20 ng/ml bFGF+4 uM Y27632. Cells incubated in 37° C., 5% CO 2  and 5% O 2  incubator. \\n     In certain embodiments Vitronectin (without MEFs) may replace Matrigel as a matrix for iPSC growth. \\n     Day2: Second MEFs cell depletion step: After iPSC cells reach 80% or more confluency, 2 ml of TrypLE solution was added to 100 mm dish and incubated at 37° C. for 4-5 minutes. After TrypLE treatment, iPSC cells were replated into one well of a 6 well plate, precoated with Matrigel (seeding density of 1.5×10 6  cells per well), in medium containing: 80% HES/20% MEF-CM+20 ng/ml bFGF+5 ng/ml stable bFGF+4 uM Y27632 and incubated in 37° C., 5% CO 2  and 5% O 2  incubator. \\n     In an alternative embodiment, iPSCs at 80% confluence are cultured on 100 mm dishes precoated with Matrigel/Vitronectin in the absence of MEFs. Media is then replaced with any of the above three conditions without passaging. The Day1 culture will yield highly pure iPSC-RGCs following the differentiation protocol and avoid using MEF cells during the process of differentiation. \\n     Day3: The spent medium is replaced with a fresh medium without Y27632. (selected from any of i), ii), or iii) listed above). The plate is moved from the 5% O 2  incubator to the regular O 2  incubator and maintained in 37° C., 5% CO 2  incubator (the night before) to continue with RPC induction and differentiation. \\n     RPC Induction and Differentiation: Day 0-21 \\n     Day 0-Day 3: medium is changed daily with RPC induction media containing: 0.1 μM LDN, 10 μM SB, 2 μM XAV, 10 mM NIC, 10 ng/ml IGF1. All small molecule and proteins added into each day&#39;s medium are freshly aliquoted (just before medium change). (See  FIG.  7 B ). \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 1 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 RPC induction Media \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                 Final \\n               \\n               \\n                   \\n                 Reagent \\n                 Concentration \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 1 \\n                 Regular DMEM/F12 \\n                   \\n               \\n               \\n                 2 \\n                 1 x Penn/Strep (P/S; 100X) \\n                 1% \\n               \\n               \\n                 3 \\n                 1 x Glutamine MAX (100X) \\n                 1% \\n               \\n               \\n                 4 \\n                 NEAA (MEM Non Essential Amino acid) \\n                 1% \\n               \\n               \\n                 5 \\n                 2-Mercaptoethanol (55 mM stock) \\n                 0.1 mM \\n               \\n               \\n                 6 \\n                 N2 supplement (1%) \\n                 1% \\n               \\n               \\n                 7 \\n                 B27 Supplement without Vitamin A (2%) \\n                 2% \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Day 4-Day 21: Medium is changed daily with RPC induction media containing: 0.1 μM LDN, 10 μM SB, 2 μM XAV, 1.5 μM. CHIR, 10 ng/ml IGF1, and 10 ng/ml bFGF. All small molecules and proteins are into each day&#39;s medium after freshly aliquoting just before medium change. ( FIG.  7 C ) \\n     RGC Initiation and Differentiation: Day 22-36 \\n     Day 22-Day 23: Medium is changed daily with RGC induction media containing: 3 μM DAPT, 250 ng/ml SHH, and 100 ng/ml FGF8. ( FIG.  7 D ) \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 2 \\n               \\n               \\n                   \\n               \\n               \\n                 RGC Induction Media \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 1 \\n                 Regular DMEM/F12 \\n                 50%\\u2002 \\n               \\n               \\n                 2 \\n                 Neuraobasal media \\n                 50%\\u2002 \\n               \\n               \\n                 3 \\n                 1 x Penn/Strep (P/S; 100X) \\n                 1% \\n               \\n               \\n                 4 \\n                 1 x Glutamine MAX (100X) \\n                 1% \\n               \\n               \\n                 7 \\n                 B27 Supplement without Vitamin A (2%) \\n                 2% \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     On Day 24:\\n         1. 30 ml of Leibovitz&#39;s (LM) medium L15+1 M Glucose is added into 50 ml tube stored in an ice bucket.   2. The 6 well cell plate is removed from the incubator, and the wells washed once with cold LM+Gluc. (LMG) media (2 mL/well).   3. 2 mL of LMG media is added to each well of the 6-well dish and the cells firmly scraped in horizontal and vertical lines across the bottom surface of the well using a p1000 tip with the pipette, to dislodge cells into small clusters. Rinse plate surface well with a pipette to ensure all cells are being removed (&gt;90%). This can be achieved by drawing medium up and down several times, each time to rinse off cells from different bottom area. Pipetting should be done until any cell clusters are broken down. Cells are then checked under a microscope, if cell clusters remain large, pipette them again vigorously to obtain single cells.   4. Cells are removed with LM+Gluc from above into 20 mL of LMG media in a 50 ml tube, and centrifuged at 3000 rpm for 3 min.   5. The supernatant is removed and cell pellets resuspended in 8 ml Day24 medium+4 uM Y27632), and the cells pipetted up and down about 4-6 times to break up clumps or clusters. Day24 medium+Y27632 is then added for plating. Cell seeding density should be about 1×10 5 /well of 6 well plates. ( FIG.  7 E )       \\n\\n     Day 25-26: Change medium daily with RGC induction media containing: 3 μM DAPT and 100 ng/ml Follistatin \\n     Day 27-36: Change medium every 2-3 days with RGC induction media containing: 3 μM DAPT, 10 μM Y27632, 400 μM cAMP, 5 μM Forskolin, 40 ng/ml BDNF, 5 ng/ml NT4 and 10 ng/ml CNTF. ( FIG.  7 F ). In certain embodiments, the RGC induction media may contain concentrations of BDNF may be selected from 10 ng/μl, 20 ng/μl, 30 ng/μl, 40 ng/μl, and 50 ng/μl. \\n     Mature RGC Maintenance (Day 37 on) \\n     From Day 37 on: Medium can be changed twice a week with RGC induction media containing: 3 μM DAPT, 10 μM Y27632. ( FIG.  7 G ) \\n     Removal of Neuronal Progenitor Cells (NPCs) \\n     In certain embodiments, neuronal progenitor cells may contaminate mature iPSC-RGC cultures either in RGCs isolated from 3D-retinal organoids or using 2D-RGC cultures. To maintain long term cultures and use these cultures in in vitro studies, growth of neuronal progenitor cells should be minimized or eliminated. \\n     We optimized compounds to enrich long term cultures of iPSC-RGCs by treatment with mitotic inhibitors like AraC-Cytosine β-D-arabinofuranoside, Sigma (0.05 μM to 10 μM) and Aphidicolin (Aphi) (0.05 μM-2 μM), treatment with polyIC peptide and/or transduction with AAV virus with eGFP in culture to prevent the growth of neuronal progenitor cells. Treatment of these compounds at Days 26-37 and Days 36-44 improved RGC purity and assisted long term invitro iPSC-RGC cultures. \\n     Flow Cytometry Analysis of RGCs \\n     iPSC-RGC cultures were lifted using TrypLE (Invitrogen) and collected by centrifugation at 1600 rpm for 5 minutes at 4° C. The pelleted RGCs were resuspended in 1×HBSS and run through a 100 μm filter to ensure single cell suspension. A portion of cells were incubated with various antibodies anti-CD90-PE-Cy7 (Thy1), anti-Brn-3b Alexa Fluor 647, and anti-RBPMS Alexa Fluor 647 in 1% PBS supplemented with 0.5% bovine serum albumin plus 0.1% Na-Azide (FACS buffer) and incubated at room temperature (RT) for 40 minutes on ice at 4° C. as previously described. 41  We analyzed stained cells using AccuriC6 and data analyzed using FlowJo Version 10.0.8 software (TreeStar). \\n     Immunohistochemistry \\n     Immunohistochemical analysis was performed as previously published (Duong et al. 2018). For immunolocalization of antigens, RGCs plated on 14 mm coverslips were fixed in 4% paraformaldehyde followed by permeabilization using 0.1% Triton X-100 for 5 minutes. Cells were incubated for 1 hour at RT with a blocking buffer (1% BSA, 5% normal goat serum, 0.1% Triton X-100 in 1% PBS). RGCs were incubated with primary antibodies diluted in 1% BSA, 0.1% Triton X-100 in 1% PBS buffer for overnight at 4° C. After washing thrice with 1×PBS, cells were incubated with secondary antibodies diluted in 1% BSA, 0.1% Triton X-100 in 1% PBS buffer for 1 hour at RT. The coverslips were then mounted on slides using prolong Gold mounting medium with DAPI (4′,6-diamidino-2-phenylindole) (Invitrogen). RGCs immunohistochemistry was performed as described earlier 36-39 using SOX2, PAX6, RAX, Tuj1, Thy1/CD90, MAP2, GFAP, CRALBP, Bm-3a, and RBPMS antibodies with dilution as provided in Tables 3-5. Slides were observed under an Olympus FV1000 Confocal microscope and images were captured with the use of appropriate filters and lasers. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 3 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Antibodies, their source and concentration used in Example 1 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Protein \\n                 Antibody \\n                 Clone/ \\n                   \\n                 Concentration/ \\n                   \\n                   \\n               \\n               \\n                 species \\n                 Source \\n                 Catalog No \\n                 Species \\n                 Volume \\n                 Dilution \\n                 Application \\n               \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 SOX2 \\n                 Cell \\n                 3579S \\n                 Rabbit \\n                   \\n                   \\n                 1/100 \\n                 ICC \\n               \\n               \\n                   \\n                 Signaling \\n               \\n               \\n                   \\n                 Technology \\n               \\n               \\n                 PAX6 \\n                 Biolegend \\n                 PRB-278P100 \\n                 Rabbit \\n                 2 \\n                 mg/ml \\n                 1/100 \\n                 ICC \\n               \\n               \\n                 RAX \\n                 Abcam \\n                 ab23340 \\n                 Rabbit \\n                   \\n                   \\n                 1/100 \\n                 ICC \\n               \\n               \\n                 THY1/CD90 \\n                 R&amp;D \\n                 AF2067 \\n                 Sheep \\n                 25 \\n                 μg \\n                 1/100 \\n                 ICC, How \\n               \\n               \\n                   \\n                 Systems \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 cytometry \\n               \\n               \\n                 BRN3b \\n                 Santa cruz \\n                 sc-514474 \\n                 Mouse \\n                 200 \\n                 μg \\n                   \\u2003 1 μg/10 6  cells \\n                 Flow \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 cytometry \\n               \\n               \\n                 RBPMS \\n                 Novus \\n                 NBP2- \\n                 Mouse \\n                 0.64 \\n                 mg/ml \\n                 \\u2002 \\u2009 1 μL/10 6  cells \\n                 Flow \\n               \\n               \\n                   \\n                 Biologicals \\n                 73835AF647 \\n                   \\n                   \\n                   \\n                   \\n                 cytometry \\n               \\n               \\n                 TUJ1 \\n                 Biolegend \\n                 MRB-435P \\n                 Rabbit \\n                 1 \\n                 mg/ml \\n                 1/250 \\n                 ICC \\n               \\n               \\n                 RBPMS \\n                 Sigma- \\n                 ABN1362 \\n                 Rabbit \\n                 100 \\n                 μg \\n                 \\u20021/1500 \\n                 ICC \\n               \\n               \\n                   \\n                 Aldrich \\n               \\n               \\n                 MAP2 \\n                 Sigma \\n                 M1406 \\n                 Mouse \\n                 2 \\n                 mg/ml \\n                 1/500 \\n                 ICC \\n               \\n               \\n                 BRN3a \\n                 Santa cruz \\n                 sc-8429 \\n                 Mouse \\n                 200 \\n                 μg/ml \\n                 1/100 \\n                 ICC \\n               \\n               \\n                 BRN3b \\n                 Santa cruz \\n                 sc-6026 \\n                 Goat \\n                 100 \\n                 μg/ml \\n                 1/100 \\n                 ICC \\n               \\n               \\n                 TUBB3 \\n                 Biolegend \\n                 PRB-435P \\n                 Rabbit \\n                 1.0 \\n                 mg/ml \\n                 1/100 \\n                 ICC \\n               \\n               \\n                 CRALBP \\n                 Abcam \\n                 ab15051 \\n                 Mouse \\n                 1.0 \\n                 mg/ml \\n                 \\u20021/1000 \\n                 ICC \\n               \\n               \\n                 GFAP \\n                 Abcam \\n                 ab7260 \\n                 Rabbit \\n                   \\n                   \\n                 \\u20021/1000 \\n                 ICC \\n               \\n               \\n                 SSEA4-APC \\n                 Biolegend \\n                 MC-813-70 \\n                 Mouse \\n                 25 \\n                 μg/ml \\n                 1/100 \\n                 Flow \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 cytometry \\n               \\n               \\n                 BRN3a \\n                 Santa cruz \\n                 sc-31984 \\n                 Goat \\n                 100 \\n                 μg \\n                 1/100 \\n                 ICC (after \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 MACS) \\n               \\n               \\n                 MAP2 \\n                 Santa cruz \\n                 sc-74421 \\n                 Mouse \\n                 200 \\n                 μg \\n                 1/100 \\n                 ICC (after \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 MACS) \\n               \\n               \\n                 GFAP \\n                 Abcam \\n                 ab7260 \\n                 Rabbit \\n                 50 \\n                 μl \\n                 \\u20021/1000 \\n                 ICC (after \\n               \\n               \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                   \\n                 MACS) \\n               \\n               \\n                 Alexa Fluor \\n                 Invitrogen \\n                 A11020 \\n                 Goat \\n                 500 \\n                 μg \\n                 1/500 \\n                 ICC \\n               \\n               \\n                 594 Goat \\n               \\n               \\n                 Anti-Mouse \\n               \\n               \\n                 Alexa Fluor \\n                 Invitrogen \\n                 A21467 \\n                 Chicken \\n                 2 \\n                 mg/mL \\n                 1/500 \\n                 ICC \\n               \\n               \\n                 488 Chicken \\n               \\n               \\n                 anti- Goat \\n               \\n               \\n                 Alexa Fluor \\n                 Invitrogen \\n                 A11034 \\n                 Goat \\n                 1 \\n                 mg \\n                 1/500 \\n                 ICC \\n               \\n               \\n                 488 Goat \\n               \\n               \\n                 anti- Rabbit \\n               \\n               \\n                   \\n               \\n               \\n                 *ICC—Immunocytochemistry \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 4 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Small molecules and protein source information \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Small molecules/recombinant \\n                   \\n                   \\n                   \\n               \\n               \\n                 protein \\n                 Company \\n                 Cat No. \\n                 Reconstitution \\n               \\n               \\n                   \\n               \\n               \\n                 Recombinant Human FGF \\n                 R &amp; D Systems \\n                 233-FB \\n                 0.1% BSA in PBS \\n               \\n               \\n                 basic (146 aa) Protein \\n               \\n               \\n                 Recombinant Human \\n                 R &amp; D Systems \\n                 8908-SH \\n                 0.1% BSA in PBS \\n               \\n               \\n                 Sonic Hedgehog/Shh \\n               \\n               \\n                 Protein \\n               \\n               \\n                 Recombinant Human/Mouse \\n                 R &amp; D Systems \\n                 423-F8 \\n                 0.1% BSA in PBS \\n               \\n               \\n                 FGF-8b Protein \\n               \\n               \\n                 DAPT \\n                 Stemgent \\n                 04-0041 \\n                 DMSO \\n               \\n               \\n                 Recombinant Human \\n                 R &amp; D Systems \\n                 669-FO \\n                 0.1% BSA in PBS \\n               \\n               \\n                 Follistatin 300 Protein \\n               \\n               \\n                 Cyclopamine \\n                 R &amp; D Systems \\n                 1623/1 \\n                 DMSO \\n               \\n               \\n                 0BRecombinant Human BDNF \\n                 R &amp; D Systems \\n                 248-BDB \\n                 0.1% BSA in PBS \\n               \\n               \\n                 Protein \\n               \\n               \\n                 Forskolin \\n                 Selleckchem \\n                 S2449 \\n               \\n               \\n                 Recombinant Human \\n                 R &amp; D Systems \\n                 268-N4 \\n                 0.1% BSA in PBS \\n               \\n               \\n                 NT-4 Protein \\n               \\n               \\n                 Recombinant Human \\n                 R &amp; D Systems \\n                 257-NT \\n                 0.1% BSA in PBS \\n               \\n               \\n                 CNTF Protein \\n               \\n               \\n                 cAMP \\n                 Sigma-aldrich \\n                 D0627 \\n                 DMSO \\n               \\n               \\n                 N 6 ,2′-O- \\n               \\n               \\n                 Dibutyryladenosine 3′,5′- \\n               \\n               \\n                 cyclic monophosphate \\n               \\n               \\n                 sodium salt \\n               \\n               \\n                 Y-27632 \\n                 R &amp; D Systems \\n                 1254/1 \\n                 DMSO \\n               \\n               \\n                 dihydrochloride \\n               \\n               \\n                 LDN 193189 \\n                 R &amp; D Systems \\n                 6053 \\n                 DMSO \\n               \\n               \\n                 dihydrochloride \\n               \\n               \\n                 SB 431542 \\n                 R &amp; D Systems \\n                 1614 \\n                 DMSO \\n               \\n               \\n                 CHIR 99021 \\n                 Selleckchem \\n                 S2924 \\n                 DMSO \\n               \\n               \\n                 XAV 939 \\n                 R &amp; D Systems \\n                 3748 \\n                 DMSO \\n               \\n               \\n                 Nicotinamide \\n                 Sigma-aldrich \\n                 N0636 \\n                 DMSO \\n               \\n               \\n                 IGF \\n                 R &amp; D Systems \\n                 291-G1 \\n                 0.1% BSA in PBS \\n               \\n               \\n                 CHIR99021 \\n                 Selleckchem \\n                 S2924 \\n                 DMSO \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 5 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Media and reagents used in Example 1 \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                 Media/Reagent \\n                 Product \\n                 Catalog no. \\n               \\n               \\n                   \\n               \\n               \\n                 DMEM/F12 + Glutamine + Hepes \\n                 Corning \\n                 10-092-cm \\n               \\n               \\n                 1 x Penn/Strep \\n                 Life Technologies \\n                 10378-016 \\n               \\n               \\n                 1 x Glutamine MAX \\n                 Life Technologies \\n                 35050-061 \\n               \\n               \\n                 NEAA \\n                 Life Technologies \\n                 11140-050 \\n               \\n               \\n                 2ME \\n                 Life Technologies \\n                 21985-023 \\n               \\n               \\n                 N2 supplement \\n                 Life Technologies \\n                 17502-048 \\n               \\n               \\n                 B27 Supplement without Vitamin A \\n                 Life Technologies \\n                 12587-010 \\n               \\n               \\n                 Neurolbasal \\n                 Life Technologies \\n                 21103049 \\n               \\n               \\n                 TrypLE \\n                 Life Technologies \\n                 12605-010 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Voltage Recordings \\n     The circular cover glass coated with 1:100 growth factor reduced matrigel containing adherent cells was transferred to the recording chamber filled with Neurobasal media supplemented with 2% B27 without vitamin-A and 1% Glutamax. The chamber was placed on the microscope stage (Olympus BX-61 microscope) and perfused with oxygenated Ames&#39;s solution (Sigma-Aldrich) resulting in gradual replacement of the growth medium with the Ames medium. The temperature of the solution inside the chamber was gradually increased to 37° C. prior to imaging and recording session using Warner Instruments TC 344B. Confocal images of the cells were acquired with Olympus Fluoview 1000 MPE system and cells demonstrating strong GFP expression were selected for patch-clamp recording. Whole cell configuration was achieved in the voltage-clamp mode at −60 mV holding potential. After achieving whole-cell configuration, cells were either maintained in the current clamp mode at zero holding current and depolarized with calibrated current steps, or in the voltage-clamp mode at −60 mV holding potential from which they were depolarized with voltage steps. Warner Instruments PC 505B amplifier and Molecular Devices Digidata 1440 digitizer under the control of the Clampex software were used for recording of the membrane voltage and current. Patch pipettes (1.2/0.69 mm) were filled with (in mM) 110 K-gluconate, 12 NaCl, 10 HEPES, 1 EGTA solution. Pipettes were mounted on Sutter MPC-200 micromanipulators; MTI CCD 72 camera system was used to provide video control over pipette and cell positioning in the chamber. \\n     Quantitative Real-Time PCR \\n     Total RNA was extracted using Purelink RNA isolation kit (invitrogen). For each sample, 1 μg of total RNA was reverse transcribed using the SuperScript III first stand cDNA synthesis kit with random primers (ThermoFisher). Amplified cDNA was used to quantify Taqman probes (target-FAM and housekeeping-TET) and TaqMan Fast advanced Master Mix (ThermoFisher) on a 7900 Fast Real-time PCR system (Applied Biosystems). All results were normalized to a B2M housekeeping control and were from 3 technical replicates of 3 independent biological samples for each time-point and experimental condition. \\n     Magnetic Activated Cell Sorting (MACS) to Purify CD90+ve RGCs \\n     RGC cells were lifted using TrypLE (Invitrogen), pelleted by centrifugation at 350 g for 5 minutes, and total cell number was determined. Cell pellet was resuspended in 90 μL buffer (1×PBS pH 7.2, 0.5% BSA, and 2 mM EDTA) and 10 μL of CD90.2 microbeads (catalog #130-121-278, Miltenyi Biotec) per 107 total cells. Cell suspension was mixed well and incubated at RT for 15 min in a tube rotator. In the meantime, MS column was placed onto a MACS separator and the column was prepped. Following the 15 min incubation, the cell suspension was applied onto the column. Flow-through from the column represented the unlabeled or CD90.2− cell fraction. The column was washed with appropriate volume of buffer for at least twice. The column was then removed from the separator and placed on a suitable collection tube. Appropriate volume of buffer was added to the column and magnetically labeled CD90.2+ cells were immediately flushed out by firmly pushing the plunger into the column. The cells were plated using RGCs induction media containing 3 μM DAPT and 10 μM ROCK inhibitor. \\n     Statistical Analysis \\n     Quantitative data were obtained from three independent experiments per cell line in triplicate. Statistical analysis was performed with Student T-test in Prism. *p-values of &lt;0.05 were considered statistically significant. \\n     Results for Example 1 \\n     Combined Inhibition of Wnt, BMP, TGF-β and Nicotinamide Contributes to Efficient Generation of Early RPCs. \\n     hiPSC retinal differentiation methods can differ between 2D and 3D culture conditions, and the use of proteins and small molecules to recapitulate processes responsible for vertebrate retina development have been studied. 26,40  Previously, the generation of RGCs from hPSCs required that cells be maintained on matrigel as an adherent support to culture cells obtained from 3D cell aggregates and several additive factors like Noggin, DKK-1, IGF-1, bFGF2, and proneural supplements. 41,42,43  Teotia et al, successfully used a modified version of the “Lamba protocol” by differentiating hiPSCs initially into neural rosettes, manual picking, and then induction to functional RGCs, by recapitulating retinal developmental pathways using chemical regulators. 26,41  These conditions require mid-differentiation enrichment steps that involves the establishment of retinal rosettes followed by manual isolation and maturation to generate retinal subtypes. CRISPR engineered iPSC lines containing fluorescent RFP reporter tag in the Brn3B locus, greatly assisted in evaluation of pathways necessary for RGC differentiation and characterization. 44  This methodology provided a protocol which utilized a monolayer cultures with defined factor supplementations; however, the evaluation were only performed using human embryonic stem cells (hESCs) and resulted in proportions of RGCs between 20-30% of the overall retinal differentiation. \\n     A major challenge in the regenerative medicine and disease modeling field are the reproducibility between experiments, and variation between individual to individual. Therefore, we set out to develop and characterize a modified two-stage protocol that differentiates hiPSCs into an enriched population of retinal progenitor cell (RPC) cultures followed by targeted differentiation to RGCs that is reproducible, efficient, and requires minimal personnel interpretation in RGCs generation and maintenance. 26  To accomplish this, hiPSCs were grown to confluence and subsequently treated with a RPC induction media containing: DMEM/F12 plus N2, B27, XAV939 (WNT inhibitor), SB431542 (TGF-β inhibitor), LDN193189, (BMP inhibitor), nicotinamide, and IGF1 for 4 days ( FIG.  1 A ). The inhibition of Wnt and BMP signaling has been documented to enhance the expression of eye field transcription factors (EFTFs) during retinal differentiations of hiPSC. We observed that addition of TGF-β inhibition induced greater EFTFs expression during early retinal differentiation. Nicotinamide was added to the differentiation media (DO-D3) to promote the expression of early eye field markers LHX2 and RAX. Nicotinamide has been shown to promote cell expansion and adaptation to a radial/rosette morphology. Differentiation factors such as IGF-1 and bFGF2 aid in the specification of eye field identity to differentiating retinal progenitors. 27  From Day 4-21, nicotinamide was removed and bFGF was added to RPC induction media. Analysis at day 7 showed a uniform population of SOX2, RAX and PAX6 positive cells ( FIG.  1 A ). The expression of early retinal progenitor markers, LHX2 and RAX, were identified in over 95% of day 7 cultures ( FIG.  1 B ) indicating an efficient and robust generation of RPCs. Quantification of EFTFs, LHX2, RAX, PAX6, SIX6, SIX3, and VSX2, showed significant expression relative to pluripotent stem cells in all three individual cell lines ( FIG.  1 C ). \\n     The efficient induction of retinal progenitors is largely dependent on the inhibition of pathways responsible for patterning of diencephalon; however, it was previously unclear if modification of these inhibitory pathways play a role on retinal ganglion cell generation. Here, we tested whether pharmacological inhibition of Wnt, TGF-β, and BMP can improve RGC induction and subsequently enable the controlled generation of mature iPSC-RGC neurons. We evaluated seven different treatment conditions to determine pathways that enhance or restrict RPC generation from iPSCs. The following 7 conditions were tested from d16-d21: 1) BMP (L), 2) Wnt inhibition (X), 3) CHIR (Wnt agonist), 4) LX, 5) BMP and TGF-β (LSB), 6) LSB-CHIR and LSBX (Schematic in  FIG.  1 A ). \\n     Extended Exposures of BMP, Wnt, and TGF-β Inhibition Leads to Efficient Generation of RGC Neurons. \\n     RGC differentiation proceeded through two phases: 1) uniform RPC cultures (up to D21), and 2) late stage RGC differentiation (D22-36) ( FIG.  2 A ,B). At day 22-23, cells were treated with RGC induction conditions that included the activation of sonic hedgehog, by augmenting with SHH (250 ng/mL). Comparative studies in our lab showed the use of SAG (100 nM, smoothened agonist) could replace SHH with no difference in RGC induction changes. Fibroblast growth factor 8 (FGF8) and notch inhibitor (DAPT) were added along with SHH or SAG. During iPSC generation, SHH pathway signaling was shown to enhance the specification of RPCs to RGCs during this 48 h time frame. On day 24, cultures were expanded using a cross-hatching technique known separate cells into small clusters and SHH, TGF-β, and notch signaling were inhibited to promote RGC maturation using cyclopamine, follistatin, and DAPT, cyclopamine was removed for the following 2 days. To improve survival of RGCs during the expansion, cAMP, BDNF, neurotrophin 4 (NT4) and CNTF, were added to the media to promote the development and survival of cells with neuronal fate. Forskolin and Y-27632 were included as compounds shown to promote RGC neurite growth, survival and lineage commitment. After 35 days in vitro, cultures were disassociated and flow cytometric analysis were performed on single cell suspension obtained from three independent iPSC lines, and data showed that highest proportions of RGCs were generated from the LSBX conditions, with a range between 40-50% Thy-1 (CD90) ( FIG.  2 C , E), 82-84% Brn-3b, and 11-12% RBPMS ( FIG.  3   ). The LX, CHIR, and LSB conditions resulted in 26.7% (±3.2), 22.5% (±4.7), and 27.2% Thy-1 positive cells (±6.2), respectively ( FIG.  2 C ). We observed consistent results across individuals and experiments ( FIG.  2 D ). \\n     Additionally, on Day 35 we further concentrated the number of RGCs in our culture by employing the MACS technique and CD90.2 microbeads. Since RPCs are multipotent cells, they have the potential to differentiate into neuronal cell types and one glial cell type called the Müller glial cells 61,62 . Therefore, we utilized the CD90.2 microbeads for the positive selection of RGCs expressing Thy1 cell surface marker and the removal of Müller glial cells from our culture. Following the purification of RGCs using MACS, the cells were analyzed using immunocytochemistry by detecting for the presence of Müller glial cells, astrocytes, and RGCs using CRALBP, GFAP, MAP2 TUJ1, RBPMS and Brn-3a markers, respectively ( FIG.  4   ). Our results show that roughly 95% of the cells in our culture were positive for Brn-3a with the presence of extended synaptic connections between RGCs. Whereas, about 5% of the cells were positive for astrocytes and we detected no presence of Müller glial cells in our system ( FIG.  4   ). However, the presence of astrocytes is significant since they are essential for the functional activity of the neuronal cell types by ensuring proper synaptic maturation and signaling 63 . \\n     For long-term cultures, we initially used RGC induction media supplemented with 1% N 2  and 1% B27 for the maintenance of RGC cultures after DIV35; however, this resulted in significant overgrowth of proliferative late RPCs. The removal of N 2  supplementation reduced the number of dividing cells in our cultures, providing a culture with predominantly RGCs. We have also cultured RGCs in 1% CultureOne supplement (Life Technologies) in the RGC maturation media, which reduced the number of non-RGC cell growth in day 45 (only in two weeks, from Day 27-40) cultures and beyond. After Day 35 in culture, the iPSC derived RGCs produce appropriate morphological and physiological features of mature RGCs. \\n     Gene Expression Profiles During Differentiation of iPSC to RPCs and RGCs: \\n     During eye formation in vertebrates, cell intrinsic signals, extrinsic signals and/or transcription factors control the differentiation and fate determination of retinal cells. We evaluated the gene expression profile of three EFTs, RAX, PAX6, and SIX3 that play a role in the anterior neural plate ( FIG.  5   ). The expression of Rx (encoded by RAX gene) was maximum in the RPC inhibited by BMP and Wnt inhibition when compared to the other conditions at DIV23 ( FIG.  5   , Table 6). \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 6 \\n               \\n               \\n                   \\n               \\n               \\n                 Detector \\n                   \\n                   \\n                   \\n                   \\n               \\n               \\n                 Name \\n                 Conditions \\n                 ΔΔCt \\n                 p value \\n                 Fold change \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 ATOH7 \\n                 L vs LSB \\n                 0.962 \\n                 0.012 \\n                 0.513 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 0.409 \\n                 0.185 \\n                 0.753 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 0.435 \\n                 0.008 \\n                 0.740 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.308 \\n                 0.094 \\n                 0.808 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.246 \\n                 0.352 \\n                 1.186 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.220 \\n                 0.034 \\n                 1.165 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.553 \\n                 0.133 \\n                 1.467 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.527 \\n                 0.034 \\n                 1.441 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 0.026 \\n                 0.910 \\n                 0.982 \\n               \\n               \\n                 BRN3a \\n                 L vs LSB \\n                 −0.417 \\n                 0.067 \\n                 1.335 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.471 \\n                 0.003 \\n                 2.772 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.866 \\n                 0.008 \\n                 3.645 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.026 \\n                 0.888 \\n                 0.982 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −1.029 \\n                 0.018 \\n                 2.040 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.423 \\n                 0.016 \\n                 2.682 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −1.054 \\n                 0.010 \\n                 2.077 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −1.449 \\n                 0.016 \\n                 2.730 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.395 \\n                 0.141 \\n                 1.315 \\n               \\n               \\n                 CALB2 \\n                 L vs LSB \\n                 −0.849 \\n                 0.018 \\n                 1.801 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.554 \\n                 0.000 \\n                 2.937 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.477 \\n                 0.002 \\n                 2.783 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 1.000 \\n                 0.027 \\n                 0.500 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 0.295 \\n                 0.149 \\n                 0.815 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 0.372 \\n                 0.036 \\n                 0.773 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.705 \\n                 0.025 \\n                 1.630 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.628 \\n                 0.036 \\n                 1.545 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 0.078 \\n                 0.304 \\n                 0.948 \\n               \\n               \\n                 CARTPT \\n                 L vs LSB \\n                 0.860 \\n                 0.006 \\n                 0.551 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 0.733 \\n                 0.010 \\n                 0.602 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −0.864 \\n                 0.005 \\n                 1.820 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.942 \\n                 0.040 \\n                 0.520 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 0.815 \\n                 0.053 \\n                 0.568 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.782 \\n                 0.000 \\n                 1.719 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.127 \\n                 0.075 \\n                 1.092 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −1.724 \\n                 0.000 \\n                 3.303 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −1.597 \\n                 0.000 \\n                 3.025 \\n               \\n               \\n                 CDH6 \\n                 L vs LSB \\n                 −0.215 \\n                 0.371 \\n                 1.160 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.672 \\n                 0.090 \\n                 1.593 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.911 \\n                 0.012 \\n                 3.761 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.049 \\n                 0.859 \\n                 0.967 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.408 \\n                 0.264 \\n                 1.327 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.648 \\n                 0.004 \\n                 3.133 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.457 \\n                 0.064 \\n                 1.373 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −1.697 \\n                 0.004 \\n                 3.241 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −1.240 \\n                 0.015 \\n                 2.361 \\n               \\n               \\n                 CRX \\n                 L vs LSB \\n                 −2.150 \\n                 0.003 \\n                 4.437 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.928 \\n                 0.020 \\n                 3.806 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −2.775 \\n                 0.005 \\n                 6.843 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −2.550 \\n                 0.002 \\n                 5.855 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −2.329 \\n                 0.014 \\n                 5.023 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −3.175 \\n                 0.081 \\n                 9.031 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.221 \\n                 0.501 \\n                 0.858 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.625 \\n                 0.081 \\n                 1.542 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.846 \\n                 0.118 \\n                 1.798 \\n               \\n               \\n                 FSTL4 \\n                 L vs LSB \\n                 −0.591 \\n                 0.011 \\n                 1.506 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.665 \\n                 0.012 \\n                 1.585 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.344 \\n                 0.024 \\n                 2.539 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.642 \\n                 0.081 \\n                 0.641 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 0.568 \\n                 0.104 \\n                 0.675 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.112 \\n                 0.074 \\n                 1.081 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.074 \\n                 0.491 \\n                 1.053 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.754 \\n                 0.074 \\n                 1.686 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.680 \\n                 0.091 \\n                 1.602 \\n               \\n               \\n                 Gli1 \\n                 L vs LSB \\n                 −1.790 \\n                 0.006 \\n                 3.459 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.195 \\n                 0.002 \\n                 2.289 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −2.035 \\n                 0.002 \\n                 4.098 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −0.345 \\n                 0.139 \\n                 1.270 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 0.250 \\n                 0.069 \\n                 0.841 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.590 \\n                 0.278 \\n                 1.505 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.595 \\n                 0.054 \\n                 0.662 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.245 \\n                 0.278 \\n                 1.185 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.840 \\n                 0.016 \\n                 1.790 \\n               \\n               \\n                 Gli3 \\n                 L vs LSB \\n                 −0.597 \\n                 0.004 \\n                 1.513 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.980 \\n                 0.025 \\n                 1.972 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.562 \\n                 0.005 \\n                 2.953 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.118 \\n                 0.234 \\n                 0.922 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.265 \\n                 0.255 \\n                 1.201 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.847 \\n                 0.014 \\n                 1.799 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.383 \\n                 0.136 \\n                 1.304 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.965 \\n                 0.014 \\n                 1.952 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.582 \\n                 0.094 \\n                 1.497 \\n               \\n               \\n                 ISL1 \\n                 L vs LSB \\n                 −0.095 \\n                 0.417 \\n                 1.068 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.264 \\n                 0.063 \\n                 1.201 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.012 \\n                 0.002 \\n                 2.016 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −0.426 \\n                 0.065 \\n                 1.343 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.595 \\n                 0.025 \\n                 1.511 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.343 \\n                 0.008 \\n                 2.537 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.169 \\n                 0.225 \\n                 1.125 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.917 \\n                 0.008 \\n                 1.888 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.748 \\n                 0.005 \\n                 1.679 \\n               \\n               \\n                 LHX2 \\n                 L vs LSB \\n                 −2.009 \\n                 0.009 \\n                 4.025 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.911 \\n                 0.033 \\n                 1.880 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.243 \\n                 0.018 \\n                 2.368 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −1.826 \\n                 0.024 \\n                 3.546 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.728 \\n                 0.118 \\n                 1.657 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.061 \\n                 0.016 \\n                 2.086 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 1.098 \\n                 0.008 \\n                 0.467 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 0.765 \\n                 0.016 \\n                 0.588 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.333 \\n                 0.022 \\n                 1.259 \\n               \\n               \\n                 MMP17 \\n                 L vs LSB \\n                 −1.903 \\n                 0.013 \\n                 3.741 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.883 \\n                 0.014 \\n                 3.687 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −2.266 \\n                 0.009 \\n                 4.811 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −1.373 \\n                 0.025 \\n                 2.590 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −1.352 \\n                 0.027 \\n                 2.553 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.736 \\n                 0.043 \\n                 3.331 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.021 \\n                 0.842 \\n                 0.986 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.363 \\n                 0.043 \\n                 1.286 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.384 \\n                 0.054 \\n                 1.305 \\n               \\n               \\n                 RCVRN \\n                 L vs LSB \\n                 −0.764 \\n                 0.223 \\n                 1.698 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −2.338 \\n                 0.017 \\n                 5.055 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −4.277 \\n                 0.041 \\n                 19.388 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −1.798 \\n                 0.079 \\n                 3.477 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −3.372 \\n                 0.016 \\n                 10.351 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −5.311 \\n                 0.072 \\n                 39.704 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −1.574 \\n                 0.097 \\n                 2.977 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −3.513 \\n                 0.072 \\n                 11.419 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −1.939 \\n                 0.177 \\n                 3.836 \\n               \\n               \\n                 SIX3 \\n                 L vs LSB \\n                 −0.163 \\n                 0.079 \\n                 1.120 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.639 \\n                 0.006 \\n                 1.557 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −0.990 \\n                 0.003 \\n                 1.986 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.215 \\n                 0.243 \\n                 0.862 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.260 \\n                 0.184 \\n                 1.198 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.612 \\n                 0.001 \\n                 1.528 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.475 \\n                 0.001 \\n                 1.390 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.827 \\n                 0.001 \\n                 1.774 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.351 \\n                 0.004 \\n                 1.276 \\n               \\n               \\n                 SNCG \\n                 L vs LSB \\n                 −0.351 \\n                 0.235 \\n                 1.276 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.354 \\n                 0.036 \\n                 1.278 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −0.411 \\n                 0.025 \\n                 1.330 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 0.102 \\n                 0.737 \\n                 0.932 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 0.099 \\n                 0.629 \\n                 0.933 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 0.042 \\n                 0.794 \\n                 0.971 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 −0.003 \\n                 0.991 \\n                 1.002 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.060 \\n                 0.794 \\n                 1.042 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.057 \\n                 0.212 \\n                 1.040 \\n               \\n               \\n                 Sox11 \\n                 L vs LSB \\n                 −2.023 \\n                 0.005 \\n                 4.063 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.797 \\n                 0.003 \\n                 3.474 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.833 \\n                 0.013 \\n                 3.564 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −2.054 \\n                 0.002 \\n                 4.152 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −1.828 \\n                 0.000 \\n                 3.550 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.865 \\n                 0.458 \\n                 3.642 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.226 \\n                 0.157 \\n                 0.855 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 0.189 \\n                 0.458 \\n                 0.877 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.037 \\n                 0.861 \\n                 1.026 \\n               \\n               \\n                 SPP1 \\n                 L vs LSB \\n                 −0.986 \\n                 0.094 \\n                 1.981 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 0.086 \\n                 0.816 \\n                 0.942 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.270 \\n                 0.076 \\n                 2.412 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −1.127 \\n                 0.007 \\n                 2.183 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.054 \\n                 0.628 \\n                 1.038 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −1.411 \\n                 0.255 \\n                 2.659 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 1.072 \\n                 0.001 \\n                 0.475 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.284 \\n                 0.255 \\n                 1.218 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −1.357 \\n                 0.017 \\n                 2.561 \\n               \\n               \\n                 RAX \\n                 L vs LSB \\n                 −3.264 \\n                 0.000 \\n                 9.607 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.326 \\n                 0.027 \\n                 2.507 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.526 \\n                 0.002 \\n                 2.880 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −2.416 \\n                 0.000 \\n                 5.336 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.478 \\n                 0.167 \\n                 1.392 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.678 \\n                 0.002 \\n                 1.600 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 1.938 \\n                 0.013 \\n                 0.261 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 1.738 \\n                 0.002 \\n                 0.300 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.200 \\n                 0.473 \\n                 1.149 \\n               \\n               \\n                 MITF \\n                 L vs LSB \\n                 −1.349 \\n                 0.001 \\n                 2.548 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −1.335 \\n                 0.002 \\n                 2.523 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.567 \\n                 0.002 \\n                 2.963 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −0.355 \\n                 0.014 \\n                 1.279 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.340 \\n                 0.018 \\n                 1.266 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.572 \\n                 0.107 \\n                 1.487 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.014 \\n                 0.837 \\n                 0.990 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.218 \\n                 0.107 \\n                 1.163 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.232 \\n                 0.099 \\n                 1.175 \\n               \\n               \\n                 PAX6 \\n                 L vs LSB \\n                 −1.015 \\n                 0.003 \\n                 2.021 \\n               \\n               \\n                   \\n                 L vs LSBX \\n                 −0.960 \\n                 0.003 \\n                 1.945 \\n               \\n               \\n                   \\n                 L vs LSB-CHIR \\n                 −1.303 \\n                 0.010 \\n                 2.467 \\n               \\n               \\n                   \\n                 X vs LSB \\n                 −0.291 \\n                 0.028 \\n                 1.224 \\n               \\n               \\n                   \\n                 X vs LSBX \\n                 −0.236 \\n                 0.031 \\n                 1.178 \\n               \\n               \\n                   \\n                 X vs LSB-CHIR \\n                 −0.579 \\n                 0.174 \\n                 1.493 \\n               \\n               \\n                   \\n                 LSB vs LSBX \\n                 0.055 \\n                 0.488 \\n                 0.962 \\n               \\n               \\n                   \\n                 LSB vs LSB-CHIR \\n                 −0.287 \\n                 0.174 \\n                 1.220 \\n               \\n               \\n                   \\n                 LSBX vs LSB-CHIR \\n                 −0.343 \\n                 0.129 \\n                 1.268 \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     The RAX expression at DIV35 RGCs was minimal suggesting a commitment to a more differentiated retinal fate, a consequence of retinal progenitor cell (RPC) expansion. PAX6 is expressed in the cornea, lens, ciliary body, and retina through development and plays a role in determining their cell fate. The PAX6 transcript expression was observed in all experimental conditions in RPCs and RGCs; however, predominant expression of PAX6 is detected at DIV23 and DIV35 in the CHIR condition ( FIG.  5   ), which stimulates the canonical Wnt signaling. Our results indicate that prolonged stimulation of RPCs with Wnt restricts their differentiation potential and maintains majority of the cells as multipotent progenitors. \\n     Among other gene transcripts analyzed, we observed an increase in SIX3 expression in both LX and LSBX conditions at DIV23 days, when compared to RPCs stimulated with LSB or CHIR. The expression of SIX3 decreased in RGCs at DIV35 indicating that neurospecification was reaching completion at this stage. \\n     The SOX11 transcript is heavily expressed in developing retina during embryonic stages 50 . It is required for the maintenance of hedgehog signaling and is critical for axonal growth, extension and driving adult neurogenesis 51-53 . The expression of SOX11 significantly increased in LSBX and LX conditions when compared to LSB and CHIR in RPCs at DIV23. The expression of SOX11 was significantly reduced in RGCs at DIV35 ( FIG.  5   ). GLI3 has a dual function as a transcriptional activator and a repressor of the sonic hedgehog (Shh) pathway. GLI1 is a simple transcriptional activator encoded by a target gene of Shh signaling. We observed increased expression of GLI3 in RPCs at DIV23 when compared to RGCs at DIV35. Expression of both GLI3 and GLI1 genes demonstrated that sonic hedgehog signaling is important for development of RGCs. \\n     Direction selective RGC (DS-RGC) are subtypes of RGCs that respond to motion of light in different directions can be identified by expression of specific molecular markers, such as CART, CDH6, and FSTL4, among other genes. 54-57  The CDH6 mediates axon-target matching and promotes wiring specificity that does not lead to image formation in the mammalian visual system. Cadherin mediated cell-cell adhesion ensures precise connectivity of neurons in the eye to target nuclei in the brain. Increased expression of CDH6 was observed in LSBX condition in DIV35 RGCs demonstrating their maturation towards increased specificity for axonal wiring between RGCs. The CARTPT (cocaine- and amphetamine-regulated transcript) is expressed by a major subtype of RGCs, ooDSGCs. Here, the expression of CARTPT was seen in RGCs matured only in LSB, CHIR and LSBX conditions at DIV35 indicating that these conditions develop specific subtypes of mature RGCs that are known to be markers for ON-OFF direction-selective RGCs. \\n     FSTL4 is a gene expressed in ON DS-RGCs and is colocalized with BRN3B in few RGCs. 43  We have seen increased expression of FSTL4 gene transcript in CHIR induced RGCs when compared with other conditions at DIV35 indicating the development of various subtypes of mature RGCs in our culture conditions. We did not observe expression of CARTPT or FSTL4 transcripts in RPCs at DIV23 ( FIG.  5   ). \\n     The RGCs at DIV35 expressed BRN3A transcript predominantly in LSBX growth condition when compared to other conditions. Interestingly, ATOH7 expression was observed early in RPC differentiation at DIV23 and decreased in RGCs by DIV35. We detected low CRX expression levels in DIV35 RGC cultures across all culture conditions indicating that the RGC differentiation conditions at this stage restrict the photoreceptor precursor cell populations. We observed low expression of RCVRN (expressed by photoreceptors) and MITF in our cultures at DIV23 and 35 indicating that our cultures are differentiated predominantly towards RGC fate, with minimal retinal pigment epithelium cell identity ( FIG.  5   ). Development of few interneurons in mature RGCs (amacrine or horizontal cells) was also observed, as evidenced by expression of CALB2 especially in RGCs with Wnt activation at DIV35. \\n     To understand the importance and requirement of BMP, Wnt, TGF-β inhibition in differentiation and maintenance of RPCs, we performed pairwise gene expression analysis of the RPC cultures at DIV23 for the four conditions (L, SB, X and CHIR) in different combinations (Table 6). We observed increased expression of RAX, PAX6, LHX2 and SOX11 at DIV23 in L and X when compared to LSB, LSBX and LSB-CHIR indicating that BMP and Wnt inhibition is required for maintaining majority of cells in retinal progenitor state. High expression of CRX, and RCVRN at DIV23 in L, X and LSB when compared to LSB-CHIR indicates that Wnt signaling may play role in preventing the differentiation of RPCs toward photoreceptors lineage. Decreased expression of CALB2 at DIV23 in X (Wnt inhibitor) condition when compared to LSB, LSBX and LSB-CHIR; while increased expression at DIV35 under CHIR (Wnt agonist) condition indicates that Wnt signaling is crucial for promoting RPC differentiation to RGC. Increased expression of GLI1 and GLI3 transcripts at DIV23 in L (BMP inhibitor) condition when compared to LSB, LSBX and LSB-CHIR indicates that BMP inhibition is important for promoting SHH signaling (Table 6). \\n     Functional Analysis of iPSC-RGCs \\n     Confocal imaging of iPSC-RGC cultures at DIV35 show the presence of the GFP-expressing neurons with neurite projections, and an increase in the density and complexity of the projections over the period of one month. The iPSC-derived RGCs exhibited morphological features, with large somas connected by elongated axonal processes. Immunocytochemical analysis shows the expression of transcription factors like BRN3A, BRN3B, SNCG, RNA-binding proteins like RBPMS, CD90/THY1 and other cytoskeletal markers like MAP2 and TUJ1; signifying the characteristic of RGCs and RGC expression ( FIG.  6 A ). \\n     Two out of the six cells tested demonstrated action potential firing in the current clamp mode in response to the depolarizing currents. The responses of one of these cells (cell a) are illustrated in  FIG.  6 B . Although cells are not expected to fire action potentials under ideal voltage-clamp conditions, we believe that imperfect clamping in distal processes can lead to such firing observed as illustrated in  FIG.  5 C  for the same cell a from  FIG.  6 D . Interestingly, all but one cell (6 out of 6 and 4 out of 5 DIV35 old cells) demonstrated reliable firing when depolarized with voltage steps in the voltage-clamp mode. In  FIG.  6 B , cell b illustrates firing of the older cells and cell c illustrates firing of the younger cells in response to the depolarizing voltage steps. As expected, later stage iPSC-RGCs cultures (D75) produced higher frequency, sustained firing and generated larger spikes when compared to early born iPSC-RGCs. \\n     Cells firing under current-clamp conditions had resting membrane potentials around −50 mV. In contrast, cells that fired under voltage-clamp but not current-clamp conditions had resting potentials around or above −30 mV. In voltage clamp mode membrane potential was maintained at −60 mV allowing more effective recovery of sodium channels from inactivation after depolarizing steps, thus enabling action potential firing in response to depolarization. \\n     Discussion of Example 1 \\n     We employed a two-step/stage differentiation to induce RGC differentiation from iPSCs. The first stage involved differentiation of iPSCs to RPCs. The RPCs were matured in a stage-specific manner using small molecules and recombinant proteins to modulate SHH pathway, Wnt pathway and Notch signaling to reliably produce abundant RGCs, which stained positive for RGC markers and emulated action potential. \\n     This method is a quick and efficient RGC generation protocol without the need for 3D aggregate formation or manual enrichment to initiate RGC differentiation. In our method, the entire hiPSCs monolayer was differentiated to RPCs using a chemically defined medium in 2D cultures. We employed crosshatching technique to generate clumps of cells that underwent stage-specific differentiation to produce functionally mature RGCs by DIV28. This provided an accelerated timeline considering other published methods to date. Our methodology involved a chemically defined media of RGC differentiation by pairing with our novel RPC generation protocol to produce RGCs. The RPCs generated showed immunoreactivity to RAX and LHX2 in the RPC lines for over 97% of cells in iPSC culture indicating that our protocol committed cells towards RPC lineage. \\n     Using our method, we reliably differentiated six normal iPSC lines and multiple clones from those lines (data not shown) to generate mature RGCs that stain positive for Thy1/Tuj1, BRN3A, BRN3B, TUBB3, RBPMS, MAP2 and SNCG markers. Using different chemical conditions, we generated over 80% of pure iPSC-RGC cultures. Using FACS sorting analysis, we quantified the presence of Thy1/CD90.1 positive RGCs (˜91%), Brn3b positive (˜94%), SNCG positive (˜95%) and RBPMS positive (˜35%) iPSC-RGCs in our differentiated cultures. \\n     In addition to the matured RGCs, we also generated fewer other retinal cell types that express pan-retinal markers and appear to be astrocytes, amacrine and/or bipolar cells. We did not produce RPE and photoreceptor outer or inner segments in our differentiations. \\n     Here, we also show valuable data representing a heterogenous population of RGCs that can be characterized by cell type specific gene expression. This is extremely useful in providing a runway for identification of new surface markers specific to RGC subtypes, allowing for selection using magnetic bead isolation, fluorescent activated cell sorting or immunopanning of iPSC derived RGCs. \\n     Furthermore, the RGCs differentiated using the LSBX condition exhibited electrophysiological function, with the ability to conduct sodium and potassium through voltage-dependent channels and fire action potentials. When comparing action potentials obtained from RGCs at day 35, day 75 and day 110, we observed that RGCs were able to fire continuously producing larger spikes with higher frequency, as they matured and aged in culture when compared to younger cells. Based on the nature and type of physiological responses, several RGC subtypes like ON- OFF- and alpha-RGCs were observed. Therefore, our validated methodology can reliably harness iPSC technology as a renewable source of RPCs to efficiently produce highly enriched populations of RGCs for in vitro studies of glaucoma and potential therapeutic modalities for incurable RGC-related diseases. \\n     The present method is a reproducible and efficient chemically defined in vitro approach for generating unprecedented yields of RPC populations from multiple iPSCs, that are then directed toward the RGC lineage. We differentiated multiple iPSC lines into RGCs in a step-wise manner using small molecules and peptide modulators by inhibiting bone morphogenetic protein, TGF-β, and canonical Wnt pathways. Purified populations of these mature iPSC-RGCs can be used to advantage for in vitro studies of glaucoma and for therapeutic purposes for many RGC-related ocular diseases. \\n     REFERENCES FOR EXAMPLE 1 \\n     \\n         \\n         1 Janssen, S. F. et al. The vast complexity of primary open angle glaucoma: disease genes, risks, molecular mechanisms and pathobiology. Progress in retinal and eye research 37, 31-67 (2013). \\n         2 Tham, Y. C. et al. Global prevalence of glaucoma and projections of glaucoma burden through 2040: a systematic review and meta-analysis. Ophthalmology 121, 2081-2090, doi:10.1016/j.ophtha.2014.05.013 [doi] (2014). \\n         3 Weinreb, R. N. et al. Primary open-angle glaucoma. Nat Rev Dis Primers 2, 16067, doi:10.1038/nrdp.2016.67 (2016). \\n         4 Kapetanakis, V. V. et al. Global variations and time trends in the prevalence of primary open angle glaucoma (POAG): a systematic review and meta-analysis. Br J Ophthalmol 100, 86-93, doi:10.1136/bjophthalmol-2015-307223 (2016). \\n         5 Quigley, H. A. &amp; Broman, A. T. The number of people with glaucoma worldwide in 2010 and 2020. Br J Ophthalmol 90, 262-267, doi:10.1136/bjo.2005.081224 (2006). \\n         6 Yu, D. Y. et al. Retinal ganglion cells: Energetics, compartmentation, axonal transport, cytoskeletons and vulnerability. Prog Retin Eye Res 36, 217-246, doi:10.1016/j.preteyeres.2013.07.001 (2013). \\n         7 Dahlmann-Noor, A. H., Vijay, S., Limb, G. A. &amp; Khaw, P. T. Strategies for optic nerve rescue and regeneration in glaucoma and other optic neuropathies. Drug Discov Today 15, 287-299, doi:10.1016/j.drudis.2010.02.007 (2010). \\n         8 Danesh-Meyer, H. V. &amp; Levin, L. A. Neuroprotection: extrapolating from neurologic diseases to the eye. Am J Ophthalmol 148, 186-191 e182, doi:10.1016/j.ajo.2009.03.029 (2009). \\n         9 Sluch, V. M. &amp; Zack, D. J. Stem cells, retinal ganglion cells and glaucoma. Dev Ophthalmol 53, 111-121, doi:10.1159/000358409 (2014). \\n         10 Meyer, J. S. et al. Optic vesicle-like structures derived from human pluripotent stem cells facilitate a customized approach to retinal disease treatment. Stem Cells 29, 1206-1218, doi:10.1002/stem.674 (2011). \\n         11 Singh, R. et al. iPS cell modeling of Best disease: insights into the pathophysiology of an inherited macular degeneration. Hum Mol Genet 22, 593-607, doi:10.1093/hmg/dds469 (2013). \\n         12 Takahashi, K. et al. Induction of pluripotent stem cells from adult human fibroblasts by defined factors. Cell 131, 861-872, doi:10.1016/j.cell.2007.11.019 (2007). \\n         13 Takahashi, K. &amp; Yamanaka, S. Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. Cell 126, 663-676, doi:10.1016/j.cell.2006.07.024 (2006). \\n         14 Chamling, X., Sluch, V. M. &amp; Zack, D. J. The Potential of Human Stem Cells for the Study and Treatment of Glaucoma. Invest Ophthalmol Vis Sci 57, ORSFil-6, doi:10.1167/iovs.15-18590 (2016). \\n         15 Tang, Z. et al. Progress of stem/progenitor cell-based therapy for retinal degeneration. J Transl Med 15, 99, doi:10.1186/s12967-017-1183-y (2017). \\n         16 Wright, L. S., Phillips, M. J., Pinilla, I., Hei, D. &amp; Gamm, D. M. Induced pluripotent stem cells as custom therapeutics for retinal repair: progress and rationale. Exp Eye Res 123, 161-172, doi:10.1016/j.exer.2013.12.001 (2014). \\n         17 Eiraku, M. et al. Self-organizing optic-cup morphogenesis in three-dimensional culture. Nature 472, 51-56, doi:10.1038/nature09941 (2011). \\n         18 Nakano, T. et al. Self-formation of optic cups and storable stratified neural retina from human ESCs. Cell Stem Cell 10, 771-785, doi:10.1016/j.stem.2012.05.009 (2012). \\n         19 Gill, K. P., Hewitt, A. W., Davidson, K. C., Pebay, A. &amp; Wong, R. C. Methods of Retinal Ganglion Cell Differentiation From Pluripotent Stem Cells. Transl Vis Sci Technol 3, 7, doi:10.1167/tvst.3.3.7 (2014). \\n         20 Chuang, J. H. et al. Expression profiling of cell-intrinsic regulators in the process of differentiation of human iPSCs into retinal lineages. Stem Cell Res Ther 9, 140, doi:10.1186/s13287-018-0848-7 (2018). \\n         21 Marquardt, T. &amp; Gruss, P. Generating neuronal diversity in the retina: one for nearly all. Trends Neurosci 25, 32-38 (2002). \\n         22 Tanaka, T. et al. Generation of retinal ganglion cells with functional axons from human induced pluripotent stem cells. Sci Rep 5, 8344, doi:10.1038/srep08344 (2015). \\n         23 Maekawa, Y. et al. Optimized Culture System to Induce Neurite Outgrowth From Retinal Ganglion Cells in Three-Dimensional Retinal Aggregates Differentiated From Mouse and Human Embryonic Stem Cells. Curr Eye Res 41, 558-568, doi:10.3109/02713683.2015.1038359 (2016). \\n         24 Ohlemacher, S. K. et al. Stepwise Differentiation of Retinal Ganglion Cells from Human Pluripotent Stem Cells Enables Analysis of Glaucomatous Neurodegeneration. Stem Cells 34, 1553-1562, doi:10.1002/stem.2356 (2016). \\n         25 Deng, F. et al. Stage-specific differentiation of iPSCs toward retinal ganglion cell lineage. Mol Vis 22, 536-547 (2016). \\n         26 Teotia, P. et al. Generation of Functional Human Retinal Ganglion Cells with Target Specificity from Pluripotent Stem Cells by Chemically Defined Recapitulation of Developmental Mechanism. Stem Cells 35, 572-585, doi:10.1002/stem.2513 (2017). \\n         27 Lee, J. et al. Defined Conditions for Differentiation of Functional Retinal Ganglion Cells From Human Pluripotent Stem Cells. Invest Ophthalmol Vis Sci 59, 3531-3542, doi:10.1167/iovs.17-23439 (2018). \\n         28 Zhou, L. et al. Differentiation of induced pluripotent stem cells of swine into rod photoreceptors and their integration into the retina. Stem Cells 29, 972-980, doi:10.1002/stem.637 (2011). \\n         29 Cavodeassi, F. et al. Early stages of zebrafish eye formation require the coordinated activity of Wnt11, Fz5, and the Wnt/beta-catenin pathway. Neuron 47, 43-56, doi:10.1016/j.neuron.2005.05.026 (2005). \\n         30 Hirami, Y. et al. Generation of retinal cells from mouse and human induced pluripotent stem cells. Neurosci Lett 458, 126-131, doi:10.1016/j.neulet.2009.04.035 (2009). \\n         31 Yang, Z., Ding, K., Pan, L., Deng, M. &amp; Gan, L. Math5 determines the competence state of retinal ganglion cell progenitors. Dev Biol 264, 240-254 (2003). \\n         32 Sun, Y. et al. A central role for Islet1 in sensory neuron development linking sensory and spinal gene regulatory programs. Nat Neurosci 11, 1283-1293, doi:10.1038/nn.2209 (2008). \\n         33 Lipton, S. A. &amp; Tauck, D. L. Voltage-dependent conductances of solitary ganglion cells dissociated from the rat retina. J Physiol 385, 361-391 (1987). \\n         34 Sullivan, S. K. et al. High-level transgene expression in induced pluripotent stem cell-derived megakaryocytes: correction of Glanzmann thrombasthenia. Blood 123, 753-757, doi:10.1182/blood-2013-10-530725 (2014). \\n         35 Duong, T. T. et al. Use of induced pluripotent stem cell models to probe the pathogenesis of Choroideremia and to develop a potential treatment. Stem Cell Res 27, 140-150, doi:10.1016/j.scr.2018.01.009 (2018). \\n         36 Maguire, J. A., Gagne, A., Mills, J. A., Gadue, P. &amp; French, D. L. Generation of human control iPS cell line CHOPWT9 from healthy adult peripheral blood mononuclear cells. Stem Cell Res 16, 14-16, doi:10.1016/j.scr.2015.11.005 (2016). \\n         37 Lyssenko, N. N. et al. Directional ABCA1-mediated cholesterol efflux and apoB-lipoprotein secretion in the retinal pigment epithelium. J Lipid Res 59, 1927-1939, doi:10.1194/jlr.M087361 (2018). \\n         38 Chavali, V. R. et al. A CTRP5 gene S163R mutation knock-in mouse model for late-onset retinal degeneration. Hum Mol Genet 20, 2000-2014, doi:10.1093/hmg/ddr080 (2011). \\n         39 Chavali, V. R., Sommer, J. R., Petters, R. M. &amp; Ayyagari, R. Identification of a promoter for the human C1Q-tumor necrosis factor-related protein-5 gene associated with late-onset retinal degeneration. Invest Ophthalmol Vis Sci 51, 5499-5507, doi:10.1167/iovs.10-5543 (2010). \\n         40 Sluch, V. M. et al. Differentiation of human ESCs to retinal ganglion cells using a CRISPR engineered reporter cell line. Sci Rep 5, 16595, doi:10.1038/srep16595 (2015). \\n         41 Lamba, D. A. et al. Generation, purification and transplantation of photoreceptors derived from human induced pluripotent stem cells. PLoS One 5, e8763, doi:10.1371/journal.pone.0008763 (2010). \\n         42 Fligor, C. M. et al. Three-Dimensional Retinal Organoids Facilitate the Investigation of Retinal Ganglion Cell Development, Organization and Neurite Outgrowth from Human Pluripotent Stem Cells. Sci Rep 8, 14520, doi:10.1038/s41598-018-32871-8 (2018). \\n         43 Langer, K. B. et al. Retinal Ganglion Cell Diversity and Subtype Specification from Human Pluripotent Stem Cells. Stem Cell Reports 10, 1282-1293, doi:10.1016/j.stemcr.2018.02.010 (2018). \\n         44 Sluch, V. M. et al. Enhanced Stem Cell Differentiation and Immunopurification of Genome Engineered Human Retinal Ganglion Cells. Stem Cells Transl Med 6, 1972-1986, doi:10.1002/sctm.17-0059 (2017). \\n         45 Meng, Y. et al. Nicotinamide Promotes Cell Survival and Differentiation as Kinase Inhibitor in Human Pluripotent Stem Cells. Stem Cell Reports 11, 1347-1356, doi:10.1016/j.stemcr.2018.10.023 (2018). \\n         46 Buchholz, D. E. et al. Rapid and efficient directed differentiation of human pluripotent stem cells into retinal pigmented epithelium. Stem Cells Transl Med 2, 384-393, doi:10.5966/sctm.2012-0163 (2013). \\n         47 Gallardo, V. &amp; Bovolenta, P. Positive and negative regulation of Shh signalling in vertebrate retinal development. F1000Res 7, doi:10.12688/f1000research.16190.1 (2018). \\n         48 Espuny-Camacho, I. et al. Pyramidal neurons derived from human pluripotent stem cells integrate efficiently into mouse brain circuits in vivo. Neuron 77, 440-456, doi:10.1016/j.neuron.2012.12.011 (2013). \\n         49 Proenca, C. C., Song, M. &amp; Lee, F. S. Differential effects of BDNF and neurotrophin 4 (NT4) on endocytic sorting of TrkB receptors. J Neurochem 138, 397-406, doi:10.1111/jnc.13676 (2016). \\n         50 Jiang, Y. et al. Transcription factors SOX4 and SOX11 function redundantly to regulate the development of mouse retinal ganglion cells. J Biol Chem 288, 18429-18438, doi:10.1074/jbc.M113.478503 (2013). \\n         51 Bergsland, M., Werme, M., Malewicz, M., Perlmann, T. &amp; Muhr, J. The establishment of neuronal properties is controlled by Sox4 and Sox11. Genes Dev 20, 3475-3486, doi:10.1101/gad.403406 (2006). \\n         52 Wen, W., Pillai-Kastoori, L., Wilson, S. G. &amp; Morris, A. C. Sox4 regulates choroid fissure closure by limiting Hedgehog signaling during ocular morphogenesis. Dev Biol 399, 139-153, doi:10.1016/j.ydbio.2014.12.026 (2015). \\n         53 Pillai-Kastoori, L. et al. Sox11 is required to maintain proper levels of Hedgehog signaling during vertebrate ocular morphogenesis. PLoS Genet 10, e1004491, doi:10.1371/journal.pgen.1004491 (2014). \\n         54 De la Huerta, I., Kim, I. J., Voinescu, P. E. &amp; Sanes, J. R. Direction-selective retinal ganglion cells arise from molecularly specified multipotential progenitors. Proc Natl Acad Sci USA 109, 17663-17668, doi:10.1073/pnas.1215806109 (2012). \\n         55 Kay, J. N. et al. Retinal ganglion cells with distinct directional preferences differ in molecular identity, structure, and central projections. J Neurosci 31, 7753-7762, doi:10.1523/JNEUROSCI.0907-11.2011 (2011). \\n         56 Huberman, A. D. et al. Architecture and activity-mediated refinement of axonal projections from a mosaic of genetically identified retinal ganglion cells. Neuron 59, 425-438, doi:10.1016/j.neuron.2008.07.018 (2008). \\n         57 Yonehara, K. et al. Expression of SPIG1 reveals development of a retinal ganglion cell subtype projecting to the medial terminal nucleus in the mouse. PLoS One 3, e1533, doi:10.1371/journal.pone.0001533 (2008). \\n         58 Trakhtenberg, E. F. et al. Cell types differ in global coordination of splicing and proportion of highly expressed genes. Sci Rep 6, 32249, doi:10.1038/srep32249 (2016). \\n         59 Meyer, J. S. et al. Modeling early retinal development with human embryonic and induced pluripotent stem cells. Proc Natl Acad Sci USA 106, 16698-16703, doi:10.1073/pnas.0905245106 (2009). \\n         60 Riazifar, H., Jia, Y., Chen, J., Lynch, G. &amp; Huang, T. Chemically induced specification of retinal ganglion cells from human embryonic and induced pluripotent stem cells. Stem Cells Transl Med 3, 424-432, doi:10.5966/sctm.2013-0147 (2014). \\n         61 Zaghloul, N. A., Yan, B. &amp; Moody, S. A. Step-wise specification of retinal stem cells during normal embryogenesis. Biol Cell 97, 321-337, doi:10.1042/BC20040521 (2005). \\n         62 Klassen, H., Sakaguchi, D. S. &amp; Young, M. J. Stem cells and retinal repair. Prog Retin Eye Res 23, 149-181, doi:10.1016/j.preteyeres.2004.01.002 (2004). \\n         63 Nedergaard, M., Ransom, B. &amp; Goldman, S. A. New roles for astrocytes: redefining the functional architecture of the brain. Trends Neurosci 26, 523-530, doi:10.1016/j.tins.2003.08.008 (2003). \\n         64 Rodriguez, A. R., de Sevilla Müller, L. P. &amp; Brecha, N. C. The RNA binding protein RBPMS is a selective marker of ganglion cells in the mammalian retina. J Comp Neurol 522, 1411-1443, doi:10.1002/cne.23521 (2014). \\n         65 Chavali V R M, Haider N, Rathi S, et al. Dual SMAD inhibition and Wnt inhibition enable efficient and reproducible differentiations of induced pluripotent stem cells into retinal ganglion cells. Sci Rep. 2020; 10(1):11828. Published 2020 Jul. 16. doi:10.1038/s41598-O 20 -68811-8 \\n       \\n    \\n     Example 2 \\n     Method for Production of Recombinant RGCS \\n     Here, we compared the effects of SIRT1 overexpression in experimental optic nerve crush (ONC) experiments via AAV gene transfer to RGCs. We developed and characterized AAV7m8 vectors that drive RGC-specific expression of human SIRT1 in vitro in iPSC-RGCs and in the mouse retina. This vector can be used to drive RGC-specific expression of any gene of interest in a natural RGC or a RGC produced through the methods provided herein. In certain embodiments, a gene of interest is any gene that is neuroprotective of RGC, preserves vision and/or suppress RGC death. We examined the neuroprotective contribution of SIRT1 gene augmentation in suppressing RGC death and vision loss in mice that have undergone ONC using iPSC-RGCs. \\n     Methods and Materials for Example 2 \\n     Animals \\n     C57Bl/6J mice were obtained from the Jackson Laboratory and raised in a 12-h light/dark cycle. Animals were housed at the University of Pennsylvania in compliance with ARVO Statement for the Use of Animals in Ophthalmic and Vision Research as well as with institutional and federal regulations. \\n     Cell Culture \\n     The human iPSCs were generated from keratinocytes or blood cells via polycistronic lentiviral transduction (Human STEMCCA Cre-Excisable constitutive polycistronic [OKS/Myc] Lentivirus Reprogramming Kit, Millipore) by University of Pennsylvania iPSC Core facility and characterized with a hES/iPS cell pluripotency RT-PCR kit [11]. The induced pluripotent stem cell-derived retinal ganglion cells (iPSC-RGCs) were derived using the protocol described above. The iPSC-RGCs cells with structural and functional features characteristic of native RGCs are used herein. \\n     We seeded iPSC-RGCs cells at a density of 350,000 cells and transduced with AAV2 vectors at a multiplicity of infection (MOI) of 100,000 vector genomes (vg) per cell. iPSC-RGCs were harvested 48 h post-transduction for immunocytochemistry analysis. Cells were rinsed with 1×PBS and fixed in 4% paraformaldehyde (PFA) for 15 min at room temperature. Afterwards, cells were blocked in 0.1% Triton X-100 and 1% bovine serum albumin (BSA) for 30 min at room temperature. Cells were incubated with primary antibody solution in 1% BSA and rabbit anti-FLAG antibody (CST #14793; 1:200), SIRT1 antibody (sc-74465; 1:200), or BRN3A (EP1972Y, 1:2000) for 1 h at room temperature. Cells were washed with 1×PBS and incubated in secondary antibody solution containing 1% BSA and goat anti-rabbit AlexaFluor-594 antibodies (1:500) for 1 h at room temperature. Cells were removed from secondary incubation, washed in 1×PBS, and mounted with (Fluoromount-G; Southern Biotech; Birmingham, Ala., USA) containing DAPI. \\n     AAV Vector Design and Production \\n     Human SIRT1 (transcript variant 1) cDNA clones were obtained from Origene. Sequences were amplified with Q5 DNA polymerase (NEB) and cloned into an AAV expression plasmid using a commercial cloning kit (In-Fusion HD; Clontech Laboratories, Mountain View, Calif., USA). Transgene expression was driven by either the CAG promoter derived from pDRIVE-CAG (InvivoGen, San Diego, Calif., USA) [13] or the codon optimized SNCG (gamma-synuclein promoter) [13]. Both cDNA sequences contained a C-terminal 3×FLAG epitope tag that terminates into a bovine growth hormone polyadenylation sequence. AAV expression cassettes were flanked by the AAV2 inverted terminal repeats. A pro-viral plasmid driving expression of enhanced green fluorescent protein (eGFP) was used [8] and contains identical cis regulatory elements. AAV2-CAG.SIRT1, AAV2-CAG.eGFP, AAV7.m8-SNCG.SIRT1, AAV7m8-SNCG.eGFP vectors were generated using previously described methods and purified with CsCl gradient by the CAROT research vector core at the University of Pennsylvania [14]. The AAV7m8 capsid plasmid was a kind gift from Dr. John Flannery (UC-Berkeley). (Dalkara D, Byrne L C, Klimczak R R, et al. In vivo-directed evolution of a new adeno-associated virus for therapeutic outer retinal gene delivery from the vitreous. Sci Transl Med. 2013; 5(189):189ra76. doi:10.1126/scitranslmed.3005708) \\n     Intravitreal Injections \\n     4-week-old mice were anesthetized by isoflurane inhalation. A 33½ gauge needle was used to create a small incision at the limbus. Afterward, a 10-uL Hamilton syringe (701 RN; Hamilton Company, Reno, Nev., USA) attached to a 33-gauge blunt-end needle was inserted into the vitreous cavity with the needle tip placed directly above the optic nerve head. Two μL of AAV preparation containing approximately 1×10 10  vector genomes was dispensed into each eye bilaterally. Vehicle treated eyes were injected with an equivalent volume of vector dilution buffer (0.001% Pluronic F68 in PBS). The two eyes of each mouse received different injections (vehicle, AAV2-CAG.SIRT1, AAV7m8-SNCG.SIRT1, AAV2-CAG.eGFP, or AAV7m8-ANCG.eGFP) allowing each eye to serve as an independent experimental end point. \\n     Optic Nerve Crush \\n     Optic nerve crush (ONC) was performed on 12-week-old C57Bl/6J wild-type mice as in our prior studies [4]. Mice were anesthetized systemically with xylazine and ketamine and topically with 0.5% proparacaine eye drops. Under a dissecting microscope, the conjunctiva was lifted with fine forceps and cut using scissors, exposing the sclera. Forceps were used to manipulate and retract orbital fat and muscles, allowing for exposure of the optic nerve. Injury to the optic nerve was induced using curved fine-tip forceps to induce a focal crush injury to the optic nerve ˜1 to 2 mm behind the globe. Maximal pressure was used to close the forceps for 1 s. Use of fine-tip forceps facilitated avoidance of orbital vessels to avoid ocular ischemia. Bleeding during the procedure was categorized as none, minimal, moderate, and large. Mice were excluded for more than moderate bleeding. In each experiment, the ONC was performed in one eye only, allowing the contralateral uninjured eye to serve as a control. The surgeon performing ONC was masked to eyes that received the AAV constructs. \\n     Optokinetic Response Recordings (OKRs) \\n     Visual function was assessed by measuring the OKR using commercial software and apparatus (OptoMetry; CerebralMechanics, Inc., Medicine Hat, AB, Canada) as previously described [15]. OKR was determined as the highest spatial frequency where mice track a 100% contrast grating that is projected at different spatial frequencies. Measurements were performed by an investigator masked to the experimental treatments. Each datapoint represents ten animals in experiments performed in triplicate. \\n     Retinal Histology and RGC Quantification \\n     Eyes were harvested and placed in 4% PFA overnight at 48° C. Eyes were washed in PBS followed by dissection of retinal cups. Tissues were permeabilized and blocked in 2% Triton X-100, 10% normal bovine serum, and PBS and then incubated with goat anti-Brn3a antibody (Novus Biologics) diluted 1:100 at 48° C. Retinal cups were washed and then incubated in secondary antibody solution containing 2% Triton X-100, 10% normal bovine serum, and donkey anti-goat AlexaFluor 594 antibody (1:500 dilution). After washing, samples were prepared as flat mounts and mounted onto glass slides with an aqueous mounting medium (SouthernBiotech) containing 4′,6-diamidino-2-phenylindole (DAPI). RGCs were quantified as previously described [5, 6, 16, 17]. Briefly, retinal micrographs were recorded at ×40 magnification in 12 standard fields (from the center of the retina in each quadrant). Total RGC counts from the 12 fields per retinal sample covering a total area of 0.45 mm 2 /retina were recorded by an investigator masked to the experimental conditions using Photometrics cell counting software. Retinal cross-sections were incubated in blocking buffer containing PBS, 2% Triton X-100, and 10% normal donkey serum for 1 h at room temperature. Next, sections were incubated in primary antibody solution containing the previously described components and a rabbit anti-FLAG antibody (CST #14793) at 1:100 dilutions overnight in a humidified chamber at room temperature. Sections were washed in PBS three times and incubated in secondary antibody solution containing donkey anti-rabbit AlexaFluor 488 antibody diluted at 1:200 for 2 h at room temperature. Slides were then washed in PBS three times and mounted with aqueous mounting medium (SouthernBiotech) containing DAPI. Retinal whole mount photography and counting of RGCs was performed by a masked investigator. Each experiment represents ten animals per group with experiments performed in triplicate. \\n     Axon Analysis \\n     Neurofilament staining was performed and quantified in longitudinal paraffin embedded sections. Briefly, optic nerves were isolated, processed and embedded in paraffin. 5 μm longitudinal paraffin sections were deparaffinized, rehydrated, and nonspecific binding was reduced using Blocking reagent (Vector Laboratories, Burlingame, Calif., USA). Specimens were then incubated in rabbit anti-neurofilament antibody 1:500 (Abcam, Cambridge, Mass., USA) at 4° C. overnight. Sections were washed three times with PBS, then incubated with anti-rabbit secondary antibody (Vectastain Elite ABC Rabbit kit) for 30 min at 37° C. Avidin/Biotin Complex detection was performed by incubating with Vectastain Elite ABC reagent at 37° C. for 30 min and DAB (diaminobenzidine, Vector labs) substrate for 3 min at RT followed by washing in running water for 5 min. Dehydrated slides were mounted using Refrax mounting medium. Photographs of three fields/nerve (one each at the distal, central, and proximal regions of the longitudinal optic nerve section) at ×40 magnification were taken by a masked investigator. Neurofilament staining optical density was quantified by using ImageJ software (http://nih.gov). \\n     Bisected optic nerves were incubated in 2% osmium tetroxide and dehydrated in graded ethanol immersions. The nerves were then embedded in epoxy resin Embed 812 (Electron Microscopy Sciences, Hatfield, Pa.), 0.75 μm thick cross sections were generated from a section of the nerve 1.5 mm posterior to the globe and stained with 1% toluidine blue. Each optic nerve cross section was analyzed at five standardized photomicrographs (75×75 μm) obtained at ×100 magnification: one in the center and in four quadrants. Axon counts were obtained by a masked operator using the AxonJ ImageAnalysis algorithm plugin for ImageJ (http://imagej.nih.gov/ij/plugins/axonj/) [18]. Mean±standard error of the mean (SEM). \\n     Statistics \\n     All data are represented as mean±SEM. Each experiment was repeated 3 times for statistical analysis. Differences between treatment groups with respect to OKR responses, RGC quantification, and optic nerve histopathology were compared using a 1-way ANOVA followed by Tukey&#39;s honest significant difference test using statistical software (GraphPad Prism 5.0; GraphPad Software, Inc., La Jolla, Calif., USA). Differences were considered statistically significant at P&lt;0.05. Data meet the assumption of normal distribution of tests with variances between GFP and SIRT1 groups. No randomization was performed; however, investigators were masked to disease and therapeutic intervention to quantify endpoints. \\n     Results of Example 2 \\n     Design and Characterization of AAV7m8 Vectors \\n     We generated AAV2 and AAV7m8 vectors expressing eGFP and a target gene human SIRT1 driven by the CAG promoter or a RGC-selective gamma synuclein promotor [13] ( FIG.  8 A ). Using induced pluripotent stem cell derived RGCs (IPS-RGCs) developed in a manner previously described and collected from a healthy visually unaffected male, age 32, vector expression was examined in vitro using immunofluorescent labeling on iPSC-RGCs ( FIG.  8 B ). This revealed robust cell number ( FIG.  8 B ) in cultures of viable iPS-RGCs ( FIG.  8 B ), with notable levels of gene expression with cytoplasmic and nuclear distribution of the tagged protein ( FIG.  8 B ). \\n     Next, we compared the retinal transduction profile of AAV2-CAG.eGFP to AAV7m8-SNCG.eGFP following intravitreal delivery with a vector expressing enhanced green fluorescent protein in a cohort of wild-type mice as previously described [20]. This vector was compared to a previously described vector developed with a ubiquitous CAG promoter expressing the same human and codon optimized target gene SIRT1 but packaged in an AAV2 capsid. The AAV2.7m8-eGFP vector displayed a higher transduction efficiency of the ganglion cell layer than previously published [8]. Compared with the 25% RGC transduction displayed by the AAV2-CAG.eGFP vector, AAV7m8-SNCG.eGFP achieved ˜44% RGC transduction by quantifying the number of eGFP positive RGCs labeled with BRN3A antibody in retinal whole mounts ( FIG.  9 A ). AAV7m8 vectors driving expression of SIRT1 were injected in to the right and left eyes, respectively, all wild-type mice displayed similar transduction profiles in vivo ( FIG.  9 B ). \\n     Constitutively Expressed SIRT1 Gene Transfer Using AAV2 Vector does not Rescue RGC Protection after ONC \\n     C57B16/J mice received intravitreal injections of experimental or control AAV2 vectors at postnatal week 4 followed by optic nerve crush (ONC) induction at postnatal week 12 ( FIG.  10 A ). Following ONC [4, 21] we measured visual function by OKR daily. Control treated animals treated post-intravitreal injections of AAV2-CAG.eGFP exhibit normal OKR scores ( FIG.  10 A ) and retained RGCs ( FIG.  10 B,  10 C ) throughout the experimental timeline ( FIG.  10 A ) which suggests no adverse effects associated with delivery or overexpression of the control transgene. Similarly, animals injected with AAV2-CAG.SIRT1 displayed strong responses prior to induction. Following optic nerve trauma, ONC animals exhibit a decline in OKR scores beginning at day 1 post trauma, which is sustained through the experimental timeline. AAV2-CAG.SIRT1 transfection demonstrated no significant protection of visual acuity throughout the experimental timeline ( FIG.  10 A ). \\n     Permanent visual decline observed in ONC uniformly results in loss of RGC numbers [4, 21]. Retinas from each treatment group were stained with antibodies directed against Brn3a to determine whether SIRT1 augmentation conferred a protective advantage upon RGCs during ONC ( FIG.  10 C ). Intravitreal injection of AAV2 was well tolerated as indicated by comparative total RGC counts in sham-induced animals treated with eGFP. In mice traumatized with ONC, RGC numbers were significantly reduced in all treatment groups compared to sham-induced controls. This demonstrated no significant effect of SIRT1 overexpression using a ubiquitous CAG driven promoter on RGC survival that transfects only 24% of RGCs ( FIG.  9   ). \\n     Ganglion Cell-Selective Expression of SIRT1 Mediated by AAV7m8 Vector with the SNCG Delays Loss of Visual Function and Protects Against RGC Loss after ONC \\n     C57B16/J mice received intravitreal injections of AAV7m8 experimental and control vectors at postnatal week 4 followed by ONC at postnatal week 12. Sham Intravitreal injections of AAV2.7m8-eGFP did not interfere with OKR scores prior to ONC ( FIG.  11 A ). Similarly, animals injected with AAV2-SNCG.SIRT1 displayed strong responses prior to induction. Following optic nerve trauma, untreated or AAV7m8-SNCG.eGFP-treated ONC animals exhibit a decline in OKR scores beginning at day 1 post trauma, which is sustained through the experimental timeline. However, eyes treated with AAV7m8-SIRT1 demonstrate a delay in reduction of functional responses by day 2 ( FIG.  11 A ), (AAV2-CAG.SIRT1=0.282±0.015; AAV2-CAG.eGFP=0.127±0.028; P=0.002). This effect was able to delay vision loss for 3 days. \\n     Retinas of each treatment group were stained with antibodies directed against Brn3a to determine whether SIRT1 augmentation conferred a protective advantage upon RGCs during ONC ( FIG.  11 B ). Intravitreal injection of AAV7m8-SNCG was well tolerated as indicated by comparative total RGC counts in control AAV2.eGFP-injected animals. Treatment with AAV7m8-SNCG.SIRT1 resulted in a statistically significant increase in RGC survival compared to control eyes treated with AAV7m8-SNCG.eGFP ( FIG.  11 B,  11 C ) by day 6 after ONC (AAV7m8-SNCG.eGFP=711.6±513.52; AAV7m8-SNCG.SIRT1=1159.14±400.1; P=0.0156). \\n     Discussion of Example 2 \\n     Adeno-associated virus (AAV) vectors have become the standard for achieving stable gene transfer with a safe clinical profile, especially when targeted to neurons, when taking into account numerous factors including dose, capsid, cassette, and manufacturing process. AAV2-vectors encoding RPE65 demonstrated a robust safety profile following subretinal delivery in human clinical trials for Leber congenital amaurosis type 2 [22, 23]. Our study compared the neuroprotective effects of RGC specific to a non-cell specific gene transfer in an experimental mouse model of ONC. Our results show that SIRT1 driven by a RGC specific promoter delayed loss of visual acuity and enhanced RGC survival in ONC. Interestingly, even with notable loss of visual function ( FIG.  10 A ) there are still significant RGCs present ( FIG.  10 B,  10 C ) suggesting that these cell bodies, although present, might be too damaged to convey visual signals. While using different transgene cassettes, this change, and the cell selectivity, resulted in a greater transduction efficiency as well as a more impressive therapeutic effect ( FIG.  11   ). Results show that SIRT1 overexpression specifically in RGCs plays a significant role in delaying loss of RGC function and reducing RGC death from optic nerve injury. The RGC layer, after intravitreal inject of the construct, should receive the highest concentration of the virus. Current results with the AAV7m8 construct indicate that it is the level expression of SIRT1 in RGCs that is critical and sufficient to promote their survival. These effects are similar to neuroprotective effects promoted by pharmacologic and small molecular modulators of the SIRT1 pathway, as seen in mice treated with resveratrol and ST266 during experimental optic neuritis [21, 24]. Together, these results suggest that modulating SIRT1 can promote RGC survival in multiple forms of optic neuropathy. \\n     Under conditions of oxidative stress, SIRT1 is translocated to the nucleus and modulates activity of protein targets primarily involved in oxidative phosphorylation and mitochondrial biogenesis [25, 26]. This function is induced by activating PGC1-α, a transcription regulator of mitochondrial function and antioxidant metabolism [27]. In addition to these known functions, active SIRT1 deacetylates and inhibits the transcription factor, p53, thereby downregulating apoptotic gene expression and thus improving cell viability [28]. With ONC, we observed a statistically significant visual decline at day 1 post trauma with all AAV treated animals subjected to optic nerve insult, and this effect was significantly delayed in AAV7m8-SNCG.SIRT1 treated animals. The RGC survival found in the current study illustrate a key role of SIRT1 signaling specifically within RGCs in SIRT1-mediated neuroprotection. \\n     Treatment with small molecular activators of the SIRT1 pathway, such as resveratrol, preserves visual function and RGC survival in mouse models of optic neuritis and optic nerve trauma [4, 5, 6, 7, 17, 21, 24]. The current results show that the AAV2.7m8-SIRT1 vector has utility in multiple optic neuropathies such as more chronic injury to the optic nerve, including glaucoma, or acute injury, such as ischemic optic neuropathy. Our data demonstrate that with cell specificity using the SNGC promoter, protection of visual function and RGCs is better achieved, suggesting the importance of cell specificity in the design of potential neuroprotective gene therapies for optic neuropathies. \\n     REFERENCES FOR EXAMPLE 2 \\n     \\n         \\n         1. Jang S Y. Traumatic optic neuropathy. Korean J Neurotrauma. 2018; 14:1-5. \\n         2. Kumaran A M, Sundar G, Chye L T. Traumatic optic neuropathy: a review. Craniomaxillofac Trauma Reconstr. 2015; 8:31-41. \\n         3. Tang Z, Zhang S, Lee C, Kumar A, Arjunan P, Li Y, et al. An optic nerve crush injury murine model to study retinal ganglion cell survival. J Vis Exp. 2011:2685. https://doi.org/10.3791/2685. \\n         4. Zuo L, Khan R S, Lee V, Dine K, Wu W, Shindler K S. SIRT1 promotes RGC survival and delays loss of function following optic nerve crush. Invest Ophthalmol Vis Sci. 2013; 54:5097-102. \\n         5. Shindler K S, Ventura E, Dutt M, Elliott P, Fitzgerald D C, Rostami A. Oral resveratrol reduces neuronal damage in a model of multiple sclerosis. J Neuroophthalmol. 2010; 30:328-39. \\n         6. Shindler K S, Ventura E, Rex T S, Elliott P, Rostami A. SIRT1 activation confers neuroprotection in experimental optic neuritis. Invest Ophthalmol Vis Sci. 2007; 48:3602-9. \\n         7. Khan R S, Dine K, Wessel H, Brown L, Shindler K S. Effects of varying intranasal treatment regimens in ST266-mediated retinal ganglion cell neuroprotection. J Neuroophthalmol. 2019; 39:191-9. \\n         8. McDougald D S, Dine K E, Zezulin A U, Bennett J, Shindler K S. SIRT1 and NRF2 gene transfer mediate distinct neuroprotective effects upon retinal ganglion cell survival and function in experi-mental optic neuritis. Invest Ophthalmol Vis Sci. 2018; 59:1212-20. \\n         9. Chan L, Mahajan V B, Tsang S H. Genome surgery and gene therapy in retinal disorders. Yale J Biol Med. 2017; 90:523-32. \\n         10. DeBusk A, Moster M L. Gene therapy in optic nerve disease. Curr Opin Ophthalmol. 2018; 29:234-8. \\n         11. Yang W, Liu Y, Slovik K J, Wu J C, Duncan S A, Rader D J, et al. Generation of iPSCs as a pooled culture using magnetic activated cell sorting of newly reprogrammed cells. PLoS ONE. 2015; 10: e0134995. \\n         12. Teotia P, Chopra D A, Dravid S M, Van Hook M J, Qiu F, Morrison J, et al. Generation of functional human retinal ganglion cells with target specificity from pluripotent stem cells by chemically defined recapitulation of developmental mechanism. Stem Cells. 2017; 35:572-85. \\n         13. Chaffiol A, Caplette R, Jaillard C, Brazhnikova E, Desrosiers M, Dubus E, et al. A new promoter allows optogenetic vision restoration with enhanced sensitivity in Macaque retina. Mol Ther. 2017; 25:2546-60. \\n         14. Costello F, Coupland S, Hodge W, Lorello G R, Koroluk J, Pan Y I, et al. Quantifying axonal loss after optic neuritis with optical coherence tomography. Ann Neurol. 2006; 59:963-9. \\n         15. Prusky G T, Alam N M, Beekman S, Douglas R M. Rapid quantifi-cation of adult and developing mouse spatial vision using a virtual optomotor system. Invest Ophthalmol Vis Sci. 2004; 45:4611-6. \\n         16. Quinn T A, Dutt M, Shindler K S. Optic neuritis and retinal ganglion cell loss in a chronic murine model of multiple sclerosis. Front Neurol. 2011; 2:50. \\n         17. Fonseca-Kelly Z, Nassrallah M, Uribe J, Khan R S, Dine K, Dutt M, et al. Resveratrol neuroprotection in a chronic mouse model of multiple sclerosis. Front Neurol. 2012; 3:84. \\n         18. Zarei K, Scheetz T E, Christopher M, Miller K, Hedberg-Buenz A, Tandon A, et al. Automated axon counting in rodent optic nerve sections with AxonJ. Sci Rep. 2016; 6:26559. \\n         19. Chavali V R M, Haider N, Rathi S, Alapati T, He J, Gill K, et al. Dual SMAD inhibition and Wnt inhibition enable efficient and reproducible differentiations of induced pluripotent stem cells into retinal ganglion cells. Sci Rep. 2020; 10:11828. \\n         20. Dudus L, Anand V, Acland G M, Chen S J, Wilson J M, Fisher K J, et al. Persistent transgene product in retina, optic nerve and brain after intraocular injection of rAAV. Vision Res. 1999; 39:2545-53. \\n         21. Grinblat G A, Khan R S, Dine K, Wessel H, Brown L, Shindler K S. RGC neuroprotection following optic nerve trauma mediated by intranasal delivery of amnion cell secretome. Investig Ophthalmol Vis Sci. 2018; 59:2470-7. \\n         22. Bennett J, Wellman J, Marshall K A, McCague S, Ashtari M, DiStefano-Pappas J, et al. Safety and durability of effect of contralateral-eye administration of AAV2 gene therapy in patients with childhood-onset blindness caused by RPE65 mutations: a follow-on phase 1 trial. Lancet. 2016; 388:661-72. \\n         23. Maguire A M, Simonelli F, Pierce E A, Pugh E N Jr., Mingozzi F, Bennicelli J, et al. Safety and efficacy of gene transfer for Leber&#39;s congenital amaurosis. N Engl J Med. 2008; 358:2240-8. \\n         24. Khan R S, Dine K, Bauman B, Lorentsen M, Lin L, Brown H, et al. Intranasal delivery of a novel amnion cell secretome prevents neuronal damage and preserves function in a mouse multiple sclerosis model. Sci Rep. 2017; 7:41768. \\n         25. Balaiya S, Abu-Amero K K, Kondkar A A, Chalam K V. Sirtuins expression and their role in retinal diseases. Oxid Med Cell Longev. 2017; 2017:3187594. \\n         26. Balaiya S, Ferguson L R, Chalam K V. Evaluation of sirtuin role in neuroprotection of retinal ganglion cells in hypoxia. Invest Oph-thalmol Vis Sci. 2012; 53:4315-22. \\n         27. Nemoto S, Fergusson M M, Finkel T. SIRT1 functionally interacts with the metabolic regulator and transcriptional coactivator PGC-1 {alpha}. J Biol Chem. 2005; 280:16456-60. \\n         28. Luo J, Nikolaev A Y, Imai S, Chen D, Su F, Shiloh A, et al. Negative control of p53 by Sir2alpha promotes cell survival under stress. Cell. 2001; 107:137-48. \\n         29. Khan R S, Fonseca-Kelly Z, Callinan C, Zuo L, Sachdeva M M, Shindler K S. SIRT1 activating compounds reduce oxidative stress and prevent cell death in neuronal cells. Front Cell Neurosci. 2012; 6:63. \\n         30. Ross A G, McDougald D S, Khan R S, et al. Rescue of retinal ganglion cells in optic nerve injury using cell-selective AAV mediated delivery of SIRT1. Gene Ther. 2021; 28(5):256-264. doi:10.1038/s41434-O 21 -00219-z. \\n         31. Park J, Lucas A M, Zhang X, et al. Exome-wide evaluation of rare coding variants using electronic health records identifies new gene-phenotype associations. Nat Med. 2021; 27(1):66-72. doi:10.1038/s41591-O 20 -1133-8 \\n       \\n    \\n     Example 3 \\n     Evaluation of Rare Coding Variants in RGCS \\n     After interrogating a dataset of 10,900 individuals with WES data in PMBB for carriers of rare loss of function (pLOF) variants, we found a novel rare pLOV variant in PPP1R13L, which is associated with primary open angle glaucoma—a disease of the optic nerve head (ONH) that causes progressive vision loss. \\n     In Silico Analysis for PPP1R13L Expression in Ocular Tissues \\n     To understand the functional relevance of PPP1R13L in the eye, we evaluated its expression in human ocular tissues using the publicly available Ocular Tissue Database (OTDB; on the world wide web at https://genome.uiowa.edu/otdb/). The OTDB consists of gene expression data for eye tissues from 20 normal human donors, generated using Affymetrix Human Exon 1.0 ST arrays and described as Probe Logarithmic Intensity Error (PLIER) values, where individual gene expression values are normalized with its expression in other tissues. \\n     Gene Expression in DBA/2J Mouse Ocular Tissues \\n     We assessed the gene expression of PPP1R13L in mouse ocular tissues using the publicly available Glaucoma Discovery Platform (http://glaucomadb.jax.org/glaucoma). This platform provides an interactive way to analyze RNA sequencing data obtained from retinal ganglion cells (RGCs) isolated from retina and optic nerve head of a 9-month-old female D2 mouse, which is an age-dependent model of ocular hypertension/glaucoma, and D2-Gpnmb+ mouse that do not develop high IOP/glaucoma. For transcriptomic studies, four distinct groups were compared based on axonal degeneration and gene expression patterns. The transcriptome of D2 group 1 is identical to the control strain (D2-Gpnmb+), while D2 groups 2-4 exhibit increasing levels of molecular changes relevant to axonal degeneration when compared to control group. We used the Datgan software to assess the differential expression of PPP1R13L in the retina. \\n     Immunolocalization of PPP1R13L in Human Retina \\n     To study the localization of PPP1R13L protein in different retinal layers of the human eye, we performed immunofluorescence on formalin-fixed paraffin-embedded section (N=3) obtained from normal 68-year old donor&#39;s cadaver eyes with a commercially available antibody, anti-PPP1R13L (Cat #51141-1-AP, Proteintech, IL, USA). Antigen retrieval was performed in 1× citrate buffer (Life Technologies) warmed to 95° C. for 30 minutes. Sections were allowed to cool to room temperature and subsequently blocked in 10% normal goat serum with 1% bovine serum albumin in 1×TBS buffer for one hour. The retinal distribution of PPP1R13L protein was visualized by incubating the retinal section with rabbit polyclonal anti-PPP1R13L antibody at 1:300 dilution overnight at 4° C., followed by chicken anti-rabbit IgG conjugated with Alexa Fluor 594 (Life Technologies, A21442). Nuclei were stained with the use of Vectashield DAPI in the mounting media. The images were captured using a Zeiss Imager Z1 fluorescence microscope equipped with AxioVS40 software version 4.8.1.0. \\n     Human iPSC-RGC Cultures \\n     The human iPSCs were generated from keratinocytes or blood cells via polycistronic lentiviral transduction (Human STEMCCA Cre-Excisable constitutive polycistronic [OKS/L-Myc] Lentivirus Reprogramming Kit, Millipore) and characterized with a hES/iPS cell pluripotency RT-PCR kit. The induced pluripotent stem cell-derived retinal ganglion cells (iPSC-RGCs) for our studies were derived using the methods described above in Example 1, which includes incubation with small molecules to inhibit BMP, TGF-beta (SMAD) and Wnt signaling thereby differentiating retinal ganglion cells (RGCs) from iPSCs. The iPSCs were differentiated into pure iPSC-RGCs cells with structural and functional features characteristic of native RGC cells as described herein. \\n     Evaluating Oxidative Stress in iPSC-RGCs \\n     Induced pluripotent stem cell-derived retinal ganglion cells (iPSC-RGCs) were incubated with increasing amounts of H 2 O 2  overnight before replacing the cultures with complete media. The cells were collected 24 hours after the H 2 O 2  treatment, and levels of PPP1R13L transcripts were assessed using quantitative RT-PCR and gene expression primers, Fwd-5′-TGCCCCAATTCTGGAGTAGG-3′ (SEQ ID NO: 1) and Rev-5′-CGGCACGTGGACACAGATT-3′ (SEQ ID NO: 2) following previously established protocols. Mean expression levels (±standard error of mean) were calculated by analyzing at least three independent samples with replica reactions and presented on an arbitrary scale that represents the expression over the housekeeping gene ACTB. Relative gene expression was quantified using the comparative Ct method. The relative gene expression was compared against no treatment control to obtain normalized gene expression. \\n     Results for Example 3 \\n     PPP1R13L is highly expressed in ocular tissues, with optic nerve and the ONH among the highest (Table 7). Table 7 shows expression of PPP1R13L in human ocular tissues per the Ocular Tissue Database (OTDB) ranked from highest to lowest expression per ocular tissue. Expression values are represented as Probe Logarithmic Intensity Error (PLIER) values, where individual gene expression values are normalized to its expression in other tissues \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 7 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Expression of PPP1R13L transcript in human ocular tissues \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                 PPP1R13L \\n               \\n               \\n                   \\n                 Eye Tissue \\n                 Probe ID: 3865344 \\n               \\n               \\n                   \\n                 Expression \\n                 Probe Logarithmic Intensity Error (PLIER) \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 Choroid RPE \\n                 36.0174 \\n               \\n               \\n                   \\n                 Ciliary Body \\n                 34.7373 \\n               \\n               \\n                   \\n                 Cornea \\n                 43.0693 \\n               \\n               \\n                   \\n                 Iris \\n                 29.1438 \\n               \\n               \\n                   \\n                 Lens \\n                 39.9613 \\n               \\n               \\n                   \\n                 Optic Nerve \\n                 46.2664 \\n               \\n               \\n                   \\n                 Optic Nerve Head \\n                 41.1548 \\n               \\n               \\n                   \\n                 Retina \\n                 35.4081 \\n               \\n               \\n                   \\n                 Sclera \\n                 49.7938 \\n               \\n               \\n                   \\n                 Trabecular \\n                 39.1355 \\n               \\n               \\n                   \\n                 Meshwork \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Retinal ganglion cells (RGCs) are the primary cells affected by glaucoma, and cells in the ONH such as astroglia, microglia, and endothelial cells mediate RGC degeneration in response to stress such as increased intraocular pressure. We investigated whether PPP1R13L is differentially expressed in the mouse ONH in glaucoma by comparing microarray gene expression datasets of the ONH. We found PPP1R13L expression to be highest during late-early to moderate stages of glaucoma ( FIG.  12 A ). Additionally, inhibition of PPP1R13L has been shown to exacerbate retinal ganglion cell (RGC) death following axonal injury. We found that the PPP1R13L protein is predominantly localized to the ganglion cell layer in the adult human retina with some expression in the outer and inner plexiform layers, confirming its role in RGC function ( FIG.  12 B ). Using human induced pluripotent stem cell-derived RGCs (iPSC-RGCs), we found that oxidative stress markedly upregulated PPP1R13L expression ( FIG.  12 C ) to a much greater extent than even superoxide dismutase 1 (SOD1), which is known to be transcriptionally upregulated in response to oxidative stress. Thus, PPP1R13L is expressed in RGCs, is significantly upregulated by oxidative stress, and may help to prevent RGC death from p53 activation and p53-mediated apoptosis in primary open angle glaucoma. Haploinsufficiency of PPP1R13L in RGCs increases the visual consequences of primary open angle glaucoma. \\n     Example 4 \\n     Identification of RGC Specific Markers \\n     Mature iPSC-RGCs at Day 35-37 express RGC specific markers. iPSC-RGCs stain positive and can be segregated and sorted using MACS sorting using CD90.2 antibody. They express RGC specific markers, including Brn3b (82-89%), RBPMS (33-35%), SNCG (84-95%) and CD90 (88-91%). The percentage of cells expressing these markers are the numbers/percentages of cells expressing these markers when iPSCs are differentiated into iPSC-RGCs (with MACS sorting or any physical separation methods). \\n     Mature iPSC-RGCs were differentiated according to Schematic described in  FIGS.  13 A and  13 B  using a two-step differentiation approach initially to RPCs and then into iPSC-RGCs. RPCs were characterized using qRT-PCR using gene specific primers and ICC using RPC specific antibodies  FIG.  13 C . Additional immunocytochemistry analysis ( FIG.  4   ) using RGC specific antibodies and purification with MACS with CD90.1 antibody yielded over 96% pure iPSC-RGCs ( FIGS.  4 ,  13 D, and  13 E ). \\n       FIG.  13 E  shows the results of FACS sorting and provides the percentage of RGC specific markers expressed by iPSC-RGCs from our protocol.  FIG.  13 E .  FIGS.  13 F and  13 G  show RGC characterization using ICC markers and other neuronal cells we obtained using our method. Electrophysiological responses using patch clamp method demonstrates that the mature iPSC-RGCs respond to light stimuli and displays characteristic Ganglion cell responses  FIG.  13 H . \\n     Example 5 \\n     In Vivo Application of RGCs in a Mouse Model \\n     The information herein above has been applied clinically to mice as a model for therapeutic intervention as part of a cell replacement therapy. The schematic of iPSC-RGCs used for transplantation in a mouse retina is shown in  FIG.  14 A . Donor iPSCs were obtained from a donor and induced using the method discussed above to produce iPSC-RGCs. The cells were purified and, at day 36, the iPSC-RGCs were transduced with an AAV2.7m8-SNCG-GFP virus to fluorescently label the iPSC-RGCs. 2 μl of saline containing 1 million cells was then intravitrealously injected into the mouse retina of one eye in 10 mouse subjects. In each subject, the other eye was injected with saline as a control. Mice were then assessed at 2, 4, and 6 weeks following injection using Fundus imaging, SD-OCT and cSLO. ( FIGS.  14 B- 14 G ) Integration was then determined using IHC on Flat mounts and sections of electrophysiology using whole cell recordings. \\n     At six weeks, cryosections of the mice were taken to assess retinal integration after injection.  FIG.  14 H  demonstrates that integration of the eGFP labeled iPSC-RGCs in the retinal ganglion cell layer was achieved. Flatmount retinal imaging (right image) shows transplanted iPSC-RGCs integrated into normal mouse retina, showing neuronal projections and dendrites. The transplanted iPSC-RGCs also showed light dependent change in firing. ( FIG.  14 I ) The mice were continuously monitored for long term follow-up studies. At two months, Immunohistochemistry showed staining of transplanted iPSC-RGCs in mouse optic nerve sections two months post-transplantation.  FIG.  14 J- 14 K . \\n     Example 6 \\n     Transplanted Human iPSC-Derived Retinal Ganglion Cells Integrate into Mouse Retinas and are Electrophysiologically Functional \\n     Due to the asymptomatic progression of glaucoma, the majority of patients are unaware of the disease onset until it is severe, making it the leading cause of irreversible blindness worldwide. Consequently, there is an unmet need for the development of new strategies for the treatment of glaucoma. We investigated RGC replacement therapy as a treatment for ganglion cell loss. Human induced pluripotent stem cells (hiPSCs) were differentiated into mature, functional RGCs in vitro, labeled with AAV7m8-SNCG-eGFP, and transplanted intravitreally in wild type 4-month-old C57BL/6J mice. Survival of the transplanted hiPSC-RGCs was assessed by confocal microscopy and color fundus photography at 2-, 4-, and 6-weeks post-transplantation. Histological studies performed 2- and 5-months post-transplant confirmed the localization of the transplanted hiPSC-RGCs within the ganglion cell layer (GCL) of the retina. Two-photon live imaging of retinal explants and electrophysiological studies confirmed that the morphology and function of the transplanted hiPSC-RGCs were similar to native RGCs. The data provided herein disclose key strategies for enhancing the efficiency of stem cell replacement therapy and thereby advance potential treatments for neurodegenerative diseases including glaucoma and optic neuritis. \\n     Materials and Methods \\n     Human iPSC Culture \\n     Undifferentiated human induced pluripotent stein cells (hiPSCs) were derived and characterized as previously published showing a complete analysis of iPSC characteristics (68-70). The iPSCs were maintained in an iPSC medium: StemMACs iPSC-Brew XF (Miltenyi Biotec, catalog #: 130-107-086, North Rhine-Westphalia, Germany) containing 50× supplement (Miltenyi Biotec, catalog #: 130-107-087, North Rhine-Westphalia, Germany), and 5 ng/mL of basic fibroblast growth factor (bFGF; R&amp;D Systems, catalog #: 233-FB, Minneapolis, Minn., USA) on 0.1% gelatin-coated dishes with irradiated mouse embryonic fibroblast (iMEFs). The components of StemMACs iPSC-Brew XF are provided in Table 6 below. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n               \\n             \\n               \\n                 TABLE 6 \\n               \\n               \\n                   \\n               \\n               \\n                 Media (500ml) \\n                 Stock conc \\n                 Final conc \\n                 Volume \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                   \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 DMEM/F12 \\n                   \\n                   \\n                 500 \\n                 ml \\n               \\n               \\n                 Sodium Bicarbonate 7.5% \\n                   \\n                   \\n                 3.6 \\n                 ml \\n               \\n               \\n                 solution \\n               \\n               \\n                 L-Ascorbic Acid 2-Phosphate \\n                 100x \\n                 1% \\n                 5 \\n                 ml \\n               \\n               \\n                 Sesquimagnes** \\n               \\n               \\n                 Insulin-Transferrin- Selenium \\n                 100x \\n                 2% \\n                 10 \\n                 ml \\n               \\n               \\n                 bFGF solution \\n                 4 ug/ml \\n                 15 ng/ml \\n                 1875 \\n                 ul \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     Retinal Progenitor Cell (RPC) Generation and Conditions \\n     hiPSCs were cultured on 0.1% gelatinized plates containing iMEFs in 37° C., 5% O 2 , and 5% CO 2  conditions. Cells were maintained until approximately 75% confluence, then feeder cells were depleted and approximately 1.5×10 6  iPSCs were seeded in one well of a 6-well tissue culture dish (Corning, catalog #229106. Corning, N.Y., USA) coated with 1:100 diluted growth factor reduced Matrigel Growth Factor (GFR; Corning, catalog #: 354230, Corning, N.Y., USA). iPSCs were maintained in an iPSC: MEF-conditioned medium (80:20)+20 ng/mL of bFGF and 5 ng/mL of stable bFGF. The MEF-conditioned media was prepared by plating iMEFs onto 0.1% gelatin at a density of 20,000 cells/cm 2  in iPSC media. Two days post-plating, media was collected, filtered, and either used directly or cryopreserved for later use. iPSCs were maintained at 37° C. at 5% O 2  and 5% CO 2  until reaching 100% confluence then transferred to 37° C., 5% CO 2  overnight prior to induction. \\n     On day 0. iPSC: MEF-conditioned media was changed to RPC induction media: DMEM/F12 (50:50: Corning, catalog #: 10-092-cm, Corning, N.Y., USA), 1% Penn/Strep (Life Technologies, catalog #: 10378-016, Carlsbad, Calif., USA), 1% Glutamine MAX (Life Technologies, catalog #: 35050-061, Carlsbad, Calif., USA), 1% NEAA (Life Technologies, catalog #: 11140-050, Carlsbad, Calif., USA), 0.1 mM 2-ME (Life Technologies, catalog #: 21985-O 23 , Carlsbad, Calif., USA), 2% B27 supplement (w/o vitamin A; Life Technologies, catalog #: 12587-010, Carlsbad, Calif., USA), 1% N2 supplement (Life Technologies, catalog #: 17502-048, Carlsbad, Calif., USA), containing 2 μM XAV939 (X) (Wint inhibitor; R&amp;D Systems, catalog #: 3748, Minneapolis, Minn., USA), 10 μM SB431542 (SB) (TGFβ inhibitor; R&amp;D Systems, catalog #: 1614, Minneapolis, Minn., USA), 100 nM LDN193189 (L) (BMP inhibitor; R&amp;D Systems, catalog #: 6053, Minneapolis, Minn., USA), 10 mM nicotinamide (Sigma-Aldrich, catalog #: N0636, St. Louis, Mo., USA), and 10 ng/mL IGF1 (R&amp;D Systems, catalog #: 291-G1, Minneapolis, Minn., USA). \\n     Cultures were fed daily for 4 days. On day 4, culture media was exchanged with RPC induction media containing: 2 μM XAV939, 10 μM SB431542, 100 nM LDN1931.89, 10 ng/mL IGF1, and 10 ng/mL bFGF (26). \\n     Retinal Ganglion Cell (RGC) Differentiation \\n     Before RGC differentiation, the medium was changed daily using RGC induction media containing 250 ng/mL Shh (R&amp;D systems, catalog #: 8908-SH, Minneapolis, Minn., USA). 100 ng/mL FGF8 (R&amp;D Systems, catalog #: 423-F8, Minneapolis, Minn., USA) for 2 days. \\n     For RGC differentiation on day 24, cells were manually crossed into small clusters with Leibovitz&#39;s medium containing 34 LM D-Glucose (Research Products International, catalog #: G32045-500, Mt. Prospect, Ill., USA) using the crosshatching technique as previously described (71), then re-plated at a density of 1.0×10 5  cells/well of 6-well plate coated with 1:100 diluted growth factor reduced Matrigel Growth Factor in RGCs induction media containing: 100 ng/mL Follistatin 300 (R&amp;D Systems, catalog #: 669-FO, Minneapolis, Minn., USA), 0.5 μM Cyclopamine (R&amp;D Systems, catalog #: 1623/1, Minneapolis, Minn., USA), 3 μM DAPT (Stemgent, catalog #: 04-0041, Cambridge, Mass., USA), and 4.2 μM Y-27632 dihydrochloride (Rock inhibitor; R&amp;D Systems, catalog #: 1254/1, Minneapolis, Minn., USA). \\n     Media was changed 24 hours post-plating with RGC induction media containing: 100 ng/mL Follistatin 300 and 3 μM DAPT daily for 2 days. From day 27, media was changed to RGC induction media containing: 3 μM DAPT, 10 μM Rock inhibitor (Y27632), 5 μM Forskolin (Selleckchem, catalog #: S2449. Houston, Tex., USA), 400 μM N 6 ,2′-O-Dibutyryladenosine 3′,5′-cyclic monophosphate sodium salt (cAMP; Sigma-Aldrich, catalog #: D0627, St. Louis, Mo., USA), 40 ng/mL BDNF (R&amp;D Systems, catalog #: 248-BDB, Minneapolis, Minn., USA), 5 ng/mL NT4 (R&amp;D systems, catalog #: 268-N4, Minneapolis, Minn., USA), and 10 ng/mL CNTF (R&amp;D systems, catalog #: 257-NT, Minneapolis, Minn., USA). Media was changed every 2-3 days until day 36. After maturation (D36), the medium was exchanged every 3-4 days with RGC induction media containing: 3 μM DAPT and 10 μM Rock inhibitor (Y27632) (26). \\n     AAV Vector Design and Production \\n     AAV expression cassettes were flanked by the AAV2 inverted terminal repeats. A pro-viral plasmid driving expression of enhanced green fluorescent protein (eGFP) (72) was driven by codon optimized SNCG (gamma-synuclein promoter) (73) and contains identical cis regulatory elements. AAV2.7m8-SNCG.eGFP vector was generated using previously described methods and purified with CsCl gradient by the CAROT research vector core at the University of Pennsylvania (74). \\n     hiPSC-RGCs (Day 40) were transduced with AAV2.7m8 virus-containing SNCG-eGFP vector at a concentration of 1.92×10 10  vg/ml. hiPSC-RGCs expressed green fluorescence as early as 48 hours post-transduction, the cells were washed with 1×PBS 3 days after the virus was applied and media was changed at least two times before being intravitreally injected (Day 50) into the eye of C57BL/6J mice. \\n     Animal Husbandry \\n     Wild type C57BL/6J mice were obtained from the Jackson Laboratory (Bar Harbor, Me., USA) and raised in a 12-h light/dark cycle. Animals were housed at the University of Pennsylvania in compliance with the ARVO Statement for the Use of Animals in Ophthalmic and Vision Research as well as with institutional and federal regulations. \\n     Flow Cytometry Analysis of hiPSC-RPCs and hiPSC-RGCs \\n     hiPSC-RPCs and hiPSC-RGCs cultures were lifted using TrypLE Express (Invitrogen, catalog #: 12605-010, Waltham, Mass., USA) and collected by centrifugation at 1,600 rpm for 5 minutes at 4° C. The pelleted cells were resuspended in 1×PBS supplemented with 0.5% bovine serum albumin and 0.1% Na-Azide (FACS buffer). Cells were fixed in 4% paraformaldehyde (PFA) (v/v) for 15 minutes at room temperature (RT) followed by permeabilization using 0.5% Tween-20 (v/v) for 10 minutes at RT. Cells were incubated with various antibodies: anti-Ki67/MK167 (Novus Biologicals, catalog #: NB500-170SS, Littleton, Colo., USA), anti-Chx10 (Millipore, catalog #: AB9016, Burlington, Mass., USA), anti-CD90 (Thy1; Novus Biologicals, catalog #: AF2067, Littleton, Colo., USA), anti-sheep Alexa Fluor 405 (Abcam, catalog #: ab175676, Cambridge, UK), anti-BRN3 Alexa Fluor 594 (Santa Cruz, catalog #: sc-390780, Dallas, Tex., USA), anti-SNCG Alexa Fluor 488 (Santa Cruz, catalog #: sc-65979, Dallas, Tex., USA), and anti-RBPMS Alexa Fluor 647 (Novus Biologicals, catalog #: NBP273835AF647, Littleton, Colo., USA). Stained cells were analyzed using LSR B and LSRFortessa B at Penn Cytomics and Cell Sorting Resource Laboratory. Data were further analyzed using the FCS Express software. \\n     Intravitreal Injections of hiPSC-RGC \\n     4-month-old wild type C57BL/6J mice were anesthetized by isoflurane inhalation to effect (2-2.5%). For our injections, a 30-gauge needle was used to create a small incision at the limbus of the eye. Followed by 2 μL of 5×10 5  live hiPSC-RGCs transduced with AAV2.7m8 virus-containing SNCG-eGFP vector was injected into the vitreous space of C57BL/6 mouse&#39;s right eye (n=16) using a 33-gauge 0.5″ blunt needle attached to a 5 μL Hamilton syringe (Hamilton Company, catalog #: 7803-05, Reno, Nev., USA). The contralateral eye injected with 2 μL of 1×PBS served as the experimental control. Following the injections, Prednisolone 1%/Gentamicin 0.3% ophthalmic ointment was applied topically on the eyes. The animals were placed on a heating pad until awake and the mice were monitored periodically for any signs of distress or complications. \\n     In-Vivo Ocular Fundus Imaging \\n     Non-invasive retinal imaging was performed using Confocal Scanning Laser Ophthalmoscopy (“cSLO”, model Spectralis HRA, Heidelberg Engineering, Inc., Heidelberg, Germany), Color Fundus Imaging (“CFI”, model Micron III, Phoenix Instruments, Inc., Naperville, Ill.) and Spectral-Domain Optical Coherence Tomography (“SD-OCT”, model Envisu R2200 UHR, Bioptigen, Inc., Morrisville, N.C., USA) at 2-, 4-, and 6-weeks post-injection. Mice were anesthetized using Ketamine (100 mg/kg)/Xylazine (8 mg/kg), pupils dilated with a combination of 2.5% Phenylephrine/1% Tropicamide, followed by topical anesthesia to the cornea using 0.5% Proparacaine. Ocular protection against evaporative corneal desiccation during the imaging session was accomplished by using various combinations of artificial tears (Refresh, Genteal, and Balanced Salt Solution) and ocular eye shields. First, Wide-field (WF-55° FOV) and Ultra-wide field (UWF-105° FOV) cSLO images were collected in each eye with the plane of focus trained on the retina ganglion cell layer/nerve fiber layer region. BluePeak Autofluorescence (BAF-cSLO, 486 nm excitation/500-680 nm emission) images were collected with the optic nerve centrally positioned within the image field of view (FOV) frame. \\n     This procedure was repeated with the Micron III system using the blue excitation and green auto-fluorescence emission channel. SD-OCT was performed last to assess the posterior pole retina and vitreous body structure. Posterior pole images included the posterior lens, vitreous body, retina, and choroid from a 50° FOV with the optic nerve central positioned. Following the imaging assessment, mouse eyes were covered with unmedicated ophthalmic ointment (Puralube Vet Ointment) and placed on a warmed heating pad (model 50-7220-F, Harvard Apparatus, Inc.) until recovered. \\n     Retinal Histology and RGC Quantification \\n     A) Flattened whole mount: Eyes were harvested, fixed in 4% PFA at RT for 1 hour, and washed with 1×PBS, followed by isolation and dissection of the retina. Retinas were prepared as flattened whole mounts, washed with 1×PBS three times, and permeabilized in 0.5% Triton X-100 in PBS by freezing at −80° C. for 15 minutes. Retinas were incubated overnight at 4° C. in a humidified chamber with either BRN3 (Santa Cruz, catalog #: sc-6026, Dallas, Tex., USA), RBPMS (Sigma-Aldrich, catalog #: ABN1362, St. Louis, Mo., USA), Tubulin β 3 (TUJ1; BioLegend, catalog #: 801201, San Diego, Calif., USA), PSD95 (Cell Signaling Technology, catalog #: 3409S, Danvers, Mass., USA), VGLUT2 (Cell Signaling Technology, catalog #: 14487S, Danvers, Mass., USA), Ku80 (Cell Signaling Technology, catalog #: 2180S, Danvers, Mass., USA), or Human Nuclear Antigen (Novus Biologicals, catalog #: NBP2-34342, Littleton, Colo., USA). \\n     The next day, the primary antibody was removed, and the retinas were washed with 1×PBS five times for 5 minutes each, followed by incubation with appropriate secondary antibodies: anti-goat Alexa Fluor-594 (ThermoFisher Scientific, catalog #: A11080, Waltham, Mass., USA), anti-mouse Alexa Fluor-594 (ThermoFisher Scientific, catalog #: A11020, Waltham, Mass., USA), and anti-rabbit Alexa Fluor-594 (ThermoFisher Scientific, catalog #: A21442, Waltham, Mass., USA) for 1 hour at RT. The secondary antibody was discarded, and the retinas were washed with 1×PBS seven times for 5 minutes each. The coverslips were then mounted on slides using proLong Gold antifade reagent with DAPI (4′,6-diamidino-2-phenylindole) (Invitrogen, catalog #: P36935, Waltham, Mass., USA). Slides were observed under an Invitrogen EVOS M5000 fluorescent microscope and Olympus FV1000 Confocal laser scanning microscope and images were captured using 4×, 10×, and 40× objectives with the use of appropriate filters and lasers. Images were analyzed using ImageJ software. \\n     B) Cryosections: Enucleated eyes were fixed in 4% PFA overnight at 4° C. For cryo-sectioning, eyecups were dissected, and the lens was extracted. The retinal cups were first transferred to 15% sucrose solution for 1 hour at 4° C. and were then again transferred to 30% sucrose solution and incubated overnight at 4° C. Eyes were embedded in optimal cutting temperature (OCT) compound and were frozen in place on dry ice blocks and stored at −20° C. for long-term storage. Using a cryostat, 14 μm sections of the retinal cups were generated. For immunohistochemistry, the cryo-sections were washed with 1×PBS three times for 5 minutes each. The slides were incubated with blocking buffer (1% BSA, 0.1% Triton X-100, and 1% normal donkey serum in 1×PBS) in a humidified chamber for 1 hour at RT. The slides were then incubated overnight at 4° C. using the following antibodies: BRN3 (Santa Cruz, catalog #: sc-6026, Dallas, Tex., USA), MAP2 (Santa Cruz, catalog #: sc-74421, Dallas, Tex., USA), and Synapsin I (Sigma-Aldrich, catalog #: 574777, St. Louis, Mo., USA). \\n     The next day, samples were washed with 1×PBS three times for 5 minutes each, followed by incubation with appropriate secondary antibodies: anti-goat Alexa Fluor 594 (ThermoFisher Scientific, catalog #: A11080, Waltham, Mass., USA), anti-mouse Alexa Fluor 594 (ThermoFisher Scientific, catalog #: A 11020, Waltham, Mass., USA), and anti-rabbit Alexa Fluor-594 (ThermoFisher Scientific, catalog #: A21442, Waltham, Mass., USA) for 1 hour at RT. The coverslips were then mounted on slides using proLong Gold antifade reagent with DAPI (4′,6-diamidino-2-phenylindole) (Invitrogen, catalog #: P36935, Waltham, Mass., USA). Slides were observed under an Olympus FV1000 Confocal laser scanning microscope and images were captured using 40× objective with the use of appropriate filters and lasers. Images were analyzed using ImageJ software. \\n     Voltage Reading of Transplanted hiPSC-RGCs \\n     All animal procedures conformed to National Institutes of Health guidelines for animals in research and of the University of Pennsylvania Committee for the Care and Use of Animals. Mice were sacrificed by the injection of a lethal dose of ketamine/xylazine mixture (100 μg/g ketamine and 10 μg/g xylazine) followed by cervical dislocation, and eyes were enucleated and dissected in a Petri dish filled with Ames solution (Sigma-Aldrich Inc., Burlington, Mass., USA) maintained at room temperature and continuously oxygenated with a mixture of 95% O 2  and 5% CO 2 . Retinas were flat-mounted ganglion cells side up in the recording chamber perfused with oxygenated Ames solution maintained at 35-37° C. using TC-344C two-channel temperature controller (Warner Instruments, Holliston, Mass., USA). All procedures were done under dim red-light illumination. \\n     Two-photon imaging using Olympus FV1000 MPE Multiphoton Laser Scanning Microscope (Olympus Corporation of the Americas, Center Valley, Pa., USA) was employed to identify, image, and target eGFP-expressing hiPSCs in a mouse retina. The wavelength for two-photon excitation was set at 920 nm. All pipette manipulations were done using a different IR viewing system providing regular video frame rates. It included a Dage-MTI CCD 72 IR camera and controller (Dage-MTI Inc., Michigan City, Ind., USA), a Sony Trinitron TV monitor (Sony Corporation of America, New York, N.Y., USA), and a #87 IR filter (Lee Filters, Andover, England) installed in the microscope&#39;s light path for a regularly transmitted light illumination. If needed imaging could be briefly switched back to two-photon mode right before touching the cell membrane with the pipette tip to confirm that the red die CF 633 filled pipette is indeed positioned next to the targeted EGFP expressing hiPSC. At the end of the electrophysiological recording two-photon imaging was used once again to confirm that the targeted cell was filled with the CF 633 die. Images were processed using Olympus Fluoview software and ImageJ (Rasband, W. S., ImageJ, U. S. National Institutes of Health, Bethesda, Md., USA, available on the world wide web at imagej.nih.gov/ij/, 1997-2018). \\n     The hiPSCs were recorded in the cell-attached and whole-cell modes. First, an Ames-filled pipette was used to break through the inner limiting membrane and clean a small area around a cell membrane region to be patched. After that, it was replaced with another Ames-filled pipette used to record spiking activity from the targeted cell in the cell-attached (also known as the loose patch) configuration in the voltage-clamp mode. In this configuration, a seal is formed between the pipette and cell membrane although it is not required to be as tight as a seal used in patch-clamp recording, and a cell membrane is not ruptured. Finally, a pipette filled with intracellular solution (112 mM K-Gluconate, 12 mM NaCl, 2 mM MgCl 2 , 1 mM EGTA, 10 mM HEPES, Ph adjusted to 7.4 using 26 mM of KOH) was used to achieve a tight seal (typically above 2-3 GΩ) with the cellular membrane, a whole-cell configuration established by rupturing a piece of the membrane inside the pipette with a brief pulse of suction (in the voltage-clamp mode), and cellar responses to depolarizing inputs caused by light stimulation or current injection were recorded in the current-clamp mode. All pipettes were fabricated using a Sutter P-87 puller (Sutter Instrument Company, Novato, Calif., USA). Resistance of the Ames-filled pipettes was around 5-7 MΩ, and the resistance of patch-pipettes was around 7-9 MΩ. \\n     Light stimulation included calibrated full-field flashes of 500 nm light delivered from below the stage using a custom LED-based system. Cellular responses were amplified using Warner Instruments PC-505B patch-clamp amplifier, Axon Digidata 1400 digitizer, and Clampex software (Molecular Devices, San Jose, Calif., USA) was used to record data on the computer hard disk and to control light and current stimulation. Custom Matlab-based code was used for data analysis (Math Works, Natick, Mass., USA). Because of fast action potential spike kinetics (spike duration typically is less than 3 ms) and a real-world limitation of the current-clamp system, spikes can be reliably detected from current vs time traces recorded both in voltage and current clamp modes. To detect spikes these traces were high pass filtered (4 pole Butterworth filter, cutoff frequency 250 Hz), and a 4 STD detection threshold was used. Reported voltage traces recorded in the current-clamp mode were not corrected for the liquid junction potential (UP), estimated to be around 15 mV (to correct for UP 15 mV should be subtracted from the reported values). \\n     Avon Analysis \\n     Optic nerves were isolated, processed, and embedded in paraffin. 5 μm longitudinal paraffin sections of the optic nerve were deparaffinized and rehydrated, followed by incubation with blocking buffer (1% BSA, 0.1% Triton X-100, and 1% normal donkey serum in 1×PBS) in a humidified chamber for 1 hour at RT. The slides were then incubated overnight at 4° C. with Neurofilament antibody (Sigma-Aldrich, catalog #: MAB5266, St. Louis, Mo., USA). The next day, samples were washed with 1×PBS three times for 5 minutes each, followed by incubation with anti-mouse Alexa Fluor-594 (ThermoFisher Scientific, catalog #: A11020, Waltham, Mass., USA), staining was performed for 1 hour at RT. The coverslips were then mounted on slides using proLong Gold antifade reagent with DAPI (4′,6-diamidino-2-phenylindole) (Invitrogen, catalog #: P36935, Waltham, Mass., USA). Slides were observed under an Olympus FV1000 Confocal laser scanning microscope and images were captured using 60× objective with the use of appropriate filters and lasers. \\n     Statistical Analysis \\n     The total number of eGFP+hiPSC-RGCs detected in mouse retinas were quantified using Ilastik cell density counting software (Heidelberg, Germany) (75) and validated by manual counting in ImageJ software (NIH, Bethesda, Md., USA). \\n     Results: \\n     In Vitro Human iPSC Differentiation Generates Pure Populations of RPCs and RGCs \\n     Clinical translation of hiPSC-RGC transplantation in an in vivo model is highly dependent on the availability of a scalable differentiation protocol to efficiently generate mature and functional RGCs in an expedited and less labor-intensive manner. Described herein is a two-step/stage differentiation technique, in which hiPSCs are initially differentiated to retinal progenitor cells (RPCs) via the step-wise addition of small molecules and peptide modulators in a chemically defined media. This is followed by step 2, a cross-hatching technique on day 24 that allows for the expansion and differentiation of RPCs into functionally mature iPSC-RGCs. The detailed and stepwise description of the hiPSC-RGC differentiation protocol is outlined above. \\n     The hiPSC lines obtained from de-identified patient-derived normal Caucasian and African American donors were differentiated into hiPSC-RPCs and hiPSC-RGCs (28)). Prior to transplantation, the purity of hiPSC-RPC and hiPSC-RGC populations were assessed using fluorescence-activated cell sorting (FACS) on day 15 and day 35, respectively. Ki67 and Chx10 were used for the characterization of hiPSC-RPCs. BRN3/POU4F, gamma Synuclein (SNCG), CD90/Thy1, and RNA-binding protein with multiple splicing (RBPMS) were used for the identification of hiPSC-RGCs. \\n     hiPSC-RPCs generated using our method stain positive for Ki67 (95.5%) and Chx10 (82%) markers (29). Similarly, hiPSC-RGCs stain positive for BRN3 (87%), SNCG (93%), CD90 (85.5%), and RBPMS (22.5%) (29). Our differentiation protocol consistently generates RPCs and RGCs that are highly positive for the respective differentiation markers identified in Table 7A-7B. The hiPSC-RGCs can be maintained in culture for over 3 months with minimal cell loss, and therefore, serve as a readily available source of RGCs for transplantation studies. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 7A \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Flow-cytometric analysis of patient-derived iPSC for RPCs \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 PennID for iPSC \\n                 RPC Markers \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n               \\n            \\n               \\n                 Line \\n                 Race \\n                 Gender \\n                 Ki67 \\n                 Chx10 \\n               \\n               \\n                   \\n               \\n               \\n                 1) Penn123i-SV20 \\n                 European \\n                 Male \\n                   \\u2003 90% \\n                 * \\n               \\n               \\n                   \\n                 American \\n               \\n               \\n                 2) Penn087i-38-1 \\n                 European \\n                 Male \\n                 \\u2009100% \\n                 73.5% \\u2003   \\n               \\n               \\n                   \\n                 American \\n               \\n               \\n                 3) Penn019i-136-6 \\n                 European \\n                 Male \\n                 77.5% \\n                 80% \\n               \\n               \\n                   \\n                 American \\n               \\n               \\n                 4) Penn024i-370-3 \\n                 African \\n                 Female \\n                 95.5% \\n                 82% \\n               \\n               \\n                   \\n                 American \\n               \\n               \\n                   \\n               \\n               \\n                 hiPSC-RPCs were analyzed by FACS on day 15 for ki67 and Chx10 markers \\n               \\n               \\n                 *Indicative of negligible/low presence of markers \\n               \\n            \\n           \\n         \\n       \\n     \\n                             TABLE 7B                          RGC Markers                                         PennID for iPSC Line   Race   Gender   Brn3   RBPMS   SNCG   CD90               1) Penn123i-SV20   European American   Male   71%   46%   *   *       2) Penn087i-38-1   European American   Male   49.5% \\u2003     \\u20024%   25%   87%       3) Penn019i-136-6   European American   Male   87%   22.5% \\u2003     93%   85%       4) Penn039i-63-1   African American   Female   85%   2.3%\\u2009   78%   58%       5) Penn059i-555-1   African American   Female   83%   34%   84%   91%               hiPSC-RGCs were analyzed by FACS on day 35 for RGC-specific markers such as Brn3, RBPMS, SNCG, and CD90       *Indicative of negligible/low presence of markers            \\nTransplanted hiPSC-RGCs were Detected in the Murine Retina\\n \\n     For transplantation into the murine retina, we fluorescently labeled hiPSC-RGCs by transduction with AAV2.7m8-SNCG-eGFP, which expresses eGFP under the control of an RGC-specific gamma Synuclein (SNCG) promoter (30). Labeled hiPSC-RGCs expressed green fluorescence as early as 48 hours post-transduction ( FIG.  15 A ). \\n     Owing to the high purity of transduced hiPSC-RGCs, 5×10 5  viable cells (˜Day 50) were intravitreally injected into the eye of 4-month-old wild type C57BL/6J mice (n=16. The contralateral eye was injected with 1×PBS as an experimental control. The mice were subsequently monitored for any signs of distress or complications. In vivo fundus images were taken using wide-field (WF) and ultra-widefield (UWF) blue autofluorescence-confocal scanning laser ophthalmology (BAF-cSLO) and spectral-domain optical coherence tomography (SD-OCT) to assess the presence of eGFP+ cells at 2-, 4-, and 6-weeks post-injection ( FIG.  15 B- 15 C ). \\n     Following intravitreal delivery, eGFP+hiPSC-RGCs were detected within the ganglion cell layer of the murine retina as early as 2-weeks post-injection seen as punctate hyperfluorescent foci in the BAF-cSLO and color fundus imaging (CFI) ( FIG.  15 B ). We did not observe any gross abnormalities in the fundus images of any saline- or hiPSC-RGCs injected eyes. SNCG-eGFP+ donor cells were uniformly distributed across the pan-field of view of the mouse retina, however, they were predominantly localized adjacent to retinal veins than retinal arteries. The nearest neighbor index (NNI) (the ratio of mean nearest observed distance between eGFP+ cells (Nnd=˜3 μm) to mean random expected distance) was used to quantify the donor cell spatial distribution pattern. If the index is greater than 1, the cells are determined to be well dispersed, whereas if the index is less than 1, the cells are considered to be clustered. In our study, NNI was determined to be 0.511 (n=3), hence the transplanted hiPSC-RGCs were partially clustered within the mouse retina (31-33). \\n     On average there were 672 (range 230 to 1535) hiPSC-RGCs detected following intravitreal injections (Table 8). The average transplantation efficiency was 0.134%. Furthermore, based on recent studies, transplantation is considered successful if 0.1% of the total transplanted donor RGCs survive in vivo (21, 24). In that case, the transplantation experiments were successful in n=9/16 mice. \\n     \\n       \\n         \\n           \\n               \\n             \\n               \\n                 TABLE 8 \\n               \\n             \\n            \\n               \\n                   \\n               \\n               \\n                 Number of eGFP+ donor hiPSC-RGCs detected per mouse retina \\n               \\n            \\n           \\n           \\n               \\n               \\n               \\n               \\n            \\n               \\n                   \\n                   \\n                   \\n                 Number of eGFP+ hiPSC- \\n               \\n               \\n                   \\n                 PennID for \\n                   \\n                 RGCs Detected per Mouse \\n               \\n               \\n                 Mouse ID \\n                 iPSC Line \\n                 Gender \\n                 Retina \\n               \\n               \\n                   \\n               \\n               \\n                 1) \\n                 Penn123i-SV20 \\n                 Male \\n                 1098*\\u2002 \\n               \\n               \\n                 2) \\n                 Penn123i-SV20 \\n                 Male \\n                 1535*\\u2002 \\n               \\n               \\n                 3) \\n                 Penn123i-SV20 \\n                 Male \\n                 816* \\n               \\n               \\n                 4) \\n                 Penn087i-38-1 \\n                 Female \\n                 702* \\n               \\n               \\n                 5) \\n                 Penn087i-38-1 \\n                 Female \\n                 403\\u2002 \\n               \\n               \\n                 6) \\n                 Penn087i-38-1 \\n                 Female \\n                 787* \\n               \\n               \\n                 7) \\n                 Penn087i-38-1 \\n                 Female \\n                 375\\u2002 \\n               \\n               \\n                 8) \\n                 Penn087i-38-1 \\n                 Female \\n                 753* \\n               \\n               \\n                 9) \\n                 Penn087i-38-1 \\n                 Male \\n                 428\\u2002 \\n               \\n               \\n                 10)\\u2002 \\n                 Penn087i-38-1 \\n                 Male \\n                 1166*\\u2002 \\n               \\n               \\n                 11)\\u2002 \\n                 Penn087i-38-1 \\n                 Male \\n                 282\\u2002 \\n               \\n               \\n                 12)\\u2002 \\n                 Penn087i-38-1 \\n                 Male \\n                 445\\u2002 \\n               \\n               \\n                 13)\\u2002 \\n                 Penn087i-38-1 \\n                 Male \\n                 230\\u2002 \\n               \\n               \\n                 14)\\u2002 \\n                 Penn059i-555-1 \\n                 Female \\n                 537* \\n               \\n               \\n                 15)\\u2002 \\n                 Penn059i-555-1 \\n                 Female \\n                 525* \\n               \\n            \\n           \\n           \\n               \\n               \\n            \\n               \\n                 Average \\n                 672\\u2002 \\n               \\n               \\n                   \\n               \\n               \\n                 *Indicative of hiPSC-RGCs detected in a mouse retina that were more than 0.1% of the total donor cells injected and hence considered as successful transplantation. \\n               \\n            \\n           \\n         \\n       \\n     \\n     SD-OCT images confirm the structural integrity of the mouse retina following intravitreal injections ( FIG.  15 C ). We evaluated the vitreous cavity for the presence of abnormal scattering that evidences donor hiPSC-RGC remnants and/or the presence of immune cells at 2-, 4-, and 6-weeks following intravitreal injections. \\n     Detection of the Transplanted hiPSC-RGCs within the Murine Retina \\n     The eyes of wild type mice injected with hiPSC-RGCs were enucleated 2- and 5-months post-transplantation. Immunohistochemistry (IHC) analysis of retinal sections (n=3) was performed to determine the localization of the transplanted hiPSC-RGCs within the host retina. Retinal cryosections were co-stained with RGC-specific markers including BRN3, MAP2, and Synapsin I to confirm the identity of the transplanted cells. \\n     BRN3, MAP2, and Synapsin I \\n     The majority of the transplanted hiPSC-RGCs were found within the ganglion cell layer (GCL) of the mouse retina interweaved with host ganglion cells ( FIG.  16 B- 16 C ), while a few displaced RGCs (dRGCs) migrated into the inner nuclear layer (34) ( FIG.  16 G- 16 J ). Transplanted hiPSC-RGCs exhibited morphological characteristics similar to endogenous RGCs, with long complex dendritic stratification and some with short-range neurite outgrowth ( FIG.  16 D- 16 F ). Furthermore, the donor cells had dendritic extensions into the inner plexiform layer as seen in the side projection of two-photon Z-stack images acquired from the live murine retinal samples, thus indicating that the hiPSC-RGCs are forming projections to the appropriate cell layer within the host retinal circuit. \\n     Characterization of the Transplanted hiPSC-RGCs within the Murine Retina \\n     Whole-mount immunohistochemistry (n=8) was performed on the retinas isolated from enucleated eyes 2- and 5-months after injection with hiPSC-RGCs. Flattened retinas were counter-stained for RGC-specific markers such as BRN3, RBPMS, MAP2, and TUJ1, including synaptic markers like VGLUT2 and PSD95. \\n     All the SNCG-eGFP+hiPSC-RGCs detected in the mouse retinas expressed BRN3 ( FIG.  17 A ) and RBPMS ( FIG.  17 B ). MAP2 and TUJ1. Synaptic markers (VGLUT2 and PSD95). \\n     Human nuclear antigen was used to label the donor hiPSC-RGCs to confirm the donor origin of the transplanted cells; all SNCG-eGFP+ cells were also human nuclear antigen and Ku80-positive, suggesting that there was no material exchange between transplanted hiPSC-RGCs and the host RGCs ( FIG.  17 G- 17 H ). \\n     Function of the Transplanted hiPSC-RGCs \\n     Transplanted retinas from mice aged 9-months (n=4) were used for live two-photon imaging and electrophysiological studies. Two-photon live images revealed an elaborate dendritic arborization of transplanted hiPSC-RGCs and synaptic connections resembling host RGCs suggesting successful integration in the host retina ( FIG.  18 A- 18 C ). Functional integration was assessed by electrophysiologic responses to full-field photopic stimuli. Light responses were first recorded in the cell-attached (loose patch) configuration in the voltage-clamp mode ( FIG.  18 D ) before rupturing the membrane and establishing the whole-cell configuration, and in the current-clamp mode in the whole-cell configuration ( FIG.  18 E ). Of the six recorded hiPSC-RGCs, two cells produced light responses with elevated baseline firing ( FIG.  18   ), two cells generated light responses with very low baseline firing ( FIGS.  19 D and  19 E ), and two cells produced increased firing over large, often spontaneous depolarizations. For one of the last two cells, the probability of synchronization between flash and spike firing was around 60%, well above 10% which would be expected if spike firing was completely random. Depolarizations and spike firing observed for the other cell appear to be mostly spontaneous ( FIGS.  19 F and  20 E ). \\n     The hiPSC-RGCs firing rate increase in response to depolarizing stimuli appears to diminish upon repeated stimuli. The data indicates that such a decrease in responsivity was not caused by the dilution of the intracellular content by the pipette solution because it was detected in the cell-attached as well as in the whole-cell configurations. It was also observed that a brighter flash delivered later on could cause a larger membrane depolarization but a smaller increase in the firing rate compared to the preceding dim flash ( FIGS.  21 A and  21 B ). Moreover, a small depolarization in response to the first light flash could produce a larger increase in the firing rate than much larger depolarizations caused by current injections, when current injections were done after repeated light stimuli ( FIG.  20 A- 20 C ).  FIGS.  18  and  20    present data from the same cell and a comparison of Z-stack projections before and after pipette recording.  FIG.  18 C  does not show any broken/missing cellular processes, indicating that the observed decrease in responsivity is not due to any physical damage caused to the cell by pipette manipulations. Progressively smaller increases in the firing rate in response to depolarizations caused by light exposures or current injections were observed for all recorded cells. Additionally, the responsivity at least partially recovered when cells were rested without stimulation for 30 s or longer. \\n     To improve vision, axons of functionally integrated donor hiPSC-RGCs need to extend all the way to synapse with targets in the visual cortex of the brain. Herein, we processed optic nerves of mouse eyes intravitreally injected with hiPSC-RGCs (n=16), and detected graft axonal projections of eGFP+hiPSC-RGCs in the optic nerve head (ONH). The cells are also shown herein to express neurofilament marker ( FIG.  18 F ). Neurofilament proteins are essential for the maintenance of the structure and shape of the neurons, and they are involved in the regulation of signal transduction and synaptic plasticity of the axons (35). Furthermore, this indicates that the donor hiPSC-RGCs were aligned with and followed along with host RGC axons. \\n     DISCUSSION \\n     Transplantation experiments of immature retinal sheets into the eye can be traced as far back as 1964 (36). However, the success of ganglion cell transplantation is highly dependent on the purity of the donor retinal cells. The efficiency of current retinal differentiation protocols across different cell lines such as mESC, hESC, miPSC, and hiPSC is highly variable and often results in low yield and/or low purity of retinal cell populations (37-39). Early attempts included co-culturing mESCs with extracted mouse retinal tissue to promote the differentiation towards retinal cell fate, but this resulted in few cells that expressed RGC-like markers (40-44). RGCs derived from cultured embryoid bodies that formed optic vesicle-appearing neurospheres were detected from days 20-50 and were reported to express BRN3 and Calretinin. This technique was reported in both hiPSCs and ESCs (45, 46). This method was then expanded to develop optic cup-like structures from mESCs and hESCs and has been reported to recapitulate in vivo retinal development (47, 48). RGCs have also been differentiated from miPSC- and mESC-derived 3D-retinal tissues but resulted in low cell purity (8% Brn3a+ and 5% RBPMS+) and thus required further enrichment with Thy1.2 based magnetic associated cell sorting (24). \\n     In recent years, the use of iPSCs for cell therapies has garnered a lot of attention, as iPSCs are an excellent source of stem cells that can be directed to generate every key cell type in the eye and can be used to treat neurodegenerative diseases such as glaucoma (49). Furthermore, using human iPSCs for the treatment of diseases can overcome some of the ethical concerns associated with the use of human embryonic stem cells (50). Patient-derived iPSCs are an ideal source of donor cells as they circumvent host vs. donor transplant immunological rejection. The differentiation protocol presented herein consistently produces large populations of hiPSC-derived neuronal cells that express over 70% of RGC-specific markers such as BRN3, Thy1, and SNCG to date. \\n     However, it should be noted that although these markers are specific to RGCs within the retina, they are expressed by a variety of different non-retinal neuronal cells such as CNS neurons and motor neurons. Therefore, if necessary, different techniques to confirm in vitro cell identity by RNA sequencing analysis, DNA methylation patterns of RGC-specific genes, microarrays, or transcriptomic and proteomic approaches can be employed. For translational studies, the purity of donor RGCs can be determined to reduce the risk of teratoma formation from undifferentiated or heterogeneous cell populations. To achieve successful transplantation, different RGC subtypes can be identified, as each of the 20 different RGC subtype identified to date has a unique pre- and post-synaptic function. \\n     We performed in vivo studies in a wild type adult C57BL/6J mouse model. The use of murine models for transplantation studies considerably reduces the risk, time, and costs of advancing the therapy into human clinical trials. hiPSC-RGC transplantation into the murine eye was administered via intravitreal injection as it is the most targeted route of cell delivery to the retina without disrupting the blood-retinal barrier and it bypasses systemic exposure (51). However, since the vitreous space is a large cavity, it is difficult to control the dispersion of the injected cells to a specific area of the retina and in some cases, the donor cells adhere to the posterior lens capsule and ciliary body (52). \\n     In our current transplantation methodology, we recovered about 50% of viable cells in the single cell dissociation step. Optimization of this step can be done to lessen cell loss and efficiently break up the cell clusters into single cells to perform injections with a higher number of live and healthy RGCs. Furthermore, cell sorting by FACS to remove dead cells after cell dissociation will improve hiPSC-RGC survival post-transplantation and their integration in the mouse retina. However, numerous cues influence the survival rate of the donor cells such as the innate and adaptive immune system of the recipient animal, and factors in the ocular environment within the mouse such as the blood flow, oxygen, and nutrient supply which all have a direct and indirect role in the eventual homing and integration of the transplanted cells. \\n     The inner limiting membrane/neural fiber layer (ILM/NFL) of the eye acts as a physical barrier to the successful engraftment of donor RGCs into the GCL layer of the retina (53). Previously, the use of collagenase to digest ILM (54) or mechanical peeling of the ILM (55, 56) has been shown to be deleterious to the retinal structure. We performed intravitreal injections of 0.0001% Pronase E (n=5) 4-weeks prior to cell transplantation to digest the ILM and enhance cell engraftment. However, the use of Pronase E induced cataract formation and inflammation as observed by cSLO imaging ( FIG.  22   ) and did not increase the efficiency of transplantation. We observed that C57BL/6J mice that received hiPSC-RGCs without Pronase E had a significant number of successfully transplanted cells. Furthermore, there is a greater probability that transplanted RGCs will extend their dendrites into the IPL and towards the optic nerve when the ILM layer of the host is intact (21, 25). This may be due to the presence of laminin in ILM that functions as a neuritogenic signal and growth substrate (57, 58). \\n     Another factor that may influence the success of the transplantation is the immunological response mediated by the host. In terms of transplantation sites, the vitreous cavity of the eye is considered to be relatively immune-privileged. However, since we did not immune suppress our mice before injection and the hiPSC-RGCs are xenografts in mice, the eventual number of donor cells that survive the transplantation may depend on the response facilitated by the resident innate neuroinflammatory cells such as retinal macroglia (astrocytes and Müller glia) and microglia. To this end, we stained for the microglia-specific marker (Iba1) in the retinal sections transplanted with hiPSC-RGCs and did not detect the presence of microglia (Data not shown). Additionally, in a human clinical trial, the chances of immunologic rejection would be considerably reduced if autologous patient-derived iPSC-RGCs were used for transplantation. \\n     In recent years, intercellular material transfer between donor and host cells has confounded the interpretation of transplantation studies (59-63). \\n     Transplanted hiPSCs successfully integrated, produced axonal and elaborate dendritic connections in the host retinas, and were capable of generating light responses. Responses could be detected at moderate intensities (1e5 photon s −1  μm −2 ). However, most of the responses were observed at 10-100 times brighter intensities. One of the reasons for the lower light sensitivity appears to be the reduced ability of these cells to fire upon repeated stimuli. To some degree, this can be compensated by increasing flash intensity and light-dependent membrane depolarization. The observed decrease in light sensitivity cannot be explained by photoreceptor bleaching alone; brighter flashes delivered later during the experiment produced larger depolarizations (suggesting a more powerful signal originating from photoreceptors) but a smaller increase in the firing rate compared to earlier, dimmer flashes. Accordingly, large depolarizations caused by current injections through patch-pipettes resulted in smaller than expected increases in the firing rate when performed after repeated light stimuli. It appears that hiPSCs may require a longer resting time to recover their ability to increase their firing rate in response to depolarizing stimuli. Partial restoration of this ability was observed when cells were rested in the dark for at least 30 seconds. \\n     When designing a cell replacement strategy, the unique microenvironment present within aged glaucomatous host retinas must also be considered. hiPSC-RGC injections in mouse models of N-Methyl-D-Aspartate (NMDA) by following published protocols that recapitulate critical aspects of the glaucomatous disease model, induced RGC loss and optic nerve crush model of optic neuropathy (64-67). Thus far our intravitreal injection technique results in the successful detection of eGFP+hiPSC-RGCs in NMDA—(n=10/10) and optic nerve crush (n=7/8) models of glaucoma (Data not shown). The data indicates that glaucomatous retinas exhibit an increased ability to readily accept donor hiPSC-RGCs compared to wild type adult retinas. \\n     In summary, we have shown that the two-step hiPSC differentiation protocol disclosed herein consistently generates a large population of pure RGCs reported to date. Furthermore, this method does not employ any additional purification step or gene modification as commonly required by other differentiation protocols. Intravitreal injections of hiPSC-RGCs into the vitreous cavity of adult wild type C57BL/6J mice resulted in a successful transplantation rate of about 94% and an average number of donor hiPSC-RGCs that survived was calculated to be 672 (range 230 to 1535). In relation to other groups that are also trying to regenerate RGCs via transplantation, their successful transplantation rate was reported as 10% (16, 21) and &gt;65% (24). The transplantation efficiency in these studies ranged from 50 to &gt;2,000 GFP+RGCs (21) and 0.5-5% (24), respectively. \\n     The transplanted cells disclosed herein were localized within the ganglion cell layer and expressed RGC-specific markers within the host retinas. Furthermore, the transplanted hiPSC-RGCs exhibited similar morphology and functional activities as the endogenous murine RGCs. This data helps provide the necessary framework required for extending induced pluripotent stem cell studies beyond the pre-clinical stages of development and into human clinical trials. This novel strategy will help alleviate a significant emotional and financial burden endured by glaucoma patients and society. \\n     REFERENCES FOR EXAMPLE 6 \\n     \\n         \\n         1. Janssen S F, Gorgels T G, Ramdas W D, Klaver C C, van Duijn C M, Jansonius N M, et al. The vast complexity of primary open angle glaucoma: disease genes, risks, molecular mechanisms and pathobiology. Prog Retin Eye Res. 2013; 37:31-67. \\n         2. Weinreb R N, Leung C K, Crowston J G, Medeiros F A, Friedman D S, Wiggs J L, et al. Primary open-angle glaucoma. Nature Reviews Disease Primers. 2016; 2:16067. \\n         3. Tham Y, Li X, Wong T Y, Quigley H A, Aung T, Cheng C. Global prevalence of glaucoma and projections of glaucoma burden through 2040: a systematic review and meta-analysis. Ophthalmology. 2014; 121(11):2081-90. \\n         4. Zhang N, Wang J, Li Y, Jiang B. Prevalence of primary open angle glaucoma in the last 20 years: a meta-analysis and systematic review. Scientific Reports. 2021; 11(1):13762. \\n         5. Cook C, Foster P. Epidemiology of glaucoma: what&#39;s new? Canadian Journal of Ophthalmology. 2012; 47(3):223-6. \\n         6. Quigley H A, Broman A T. The number of people with glaucoma worldwide in 2010 and 2020. Br J Ophthalmol. 2006; 90(3):262-7. \\n         7. Rein D B, Zhang P, Wirth K E, Lee P P, Hoerger T J, McCall N, et al. The economic burden of major adult visual disorders in the United States. Arch Ophthalmol. 2006; 124(12):1754-60. \\n         8. Charlson E S, Sankar P S, Miller-Ellis E, Regina M, Fertig R, Salinas J, et al. The Primary Open-Angle African American Glaucoma Genetics Study: Baseline Demographics. Ophthalmology. 2015; 122(4):711-20. \\n         9. Salowe R, Salinas J, Farbman N H, Mohammed A, Warren J Z, Rhodes A, et al. Primary Open-Angle Glaucoma in Individuals of African Descent: A Review of Risk Factors. J Clin Exp Ophthalmol. 2015 August; 6(4):450. doi: 10.4172/2155,9570.1000450. Epub 2015 Jul. 31. \\n         10. Kapetanakis V V, Chan M P, Foster P J, Cook D G, Owen C G, Rudnicka A R. Global variations and time trends in the prevalence of primary open angle glaucoma (POAG): a systematic review and meta-analysis. Br J Ophthalmol. 2016; 100(1):86-93. \\n         11. Danford I D, Verkuil L D, Choi D J, Collins D W, Gudiseva H V, Uyhazi K E, et al. Characterizing the “POAGome”: A bioinformatics-driven approach to primary open-angle glaucoma. Prog Retin Eye Res. 2017; 58:89-114. \\n         12. Heavner W, Pevny L. Eye development and retinogenesis. Cold Spring Harbor perspectives in biology. 2012; 4 (12):a008391. \\n         13. Liu Y, Allingham R R. Major review: Molecular genetics of primary open-angle glaucoma. Exp Eye Res. 2017; 160:62-84. \\n         14. Sommer A, Tielsch J M, Katz J, Quigley H A, Gottsch J D, Javitt J, et al. Relationship between intraocular pressure and primary open angle glaucoma among white and black Americans: the Baltimore Eye Survey. Arch Ophthalmol. 1991; 109(8):1090-5. \\n         15. Cohen L P, Pasquale L R. Clinical characteristics and current treatment of glaucoma. Cold Spring Harbor perspectives in medicine. 2014; 4 (6):a017236. \\n         16. Hertz J, Qu B, Hu Y, Patel R D, Valenzuela D A, Goldberg J L. Survival and integration of developing and progenitor-derived retinal ganglion cells following transplantation. Cell Transplant. 2014; 23(7):855-72. \\n         17. Jagatha B, Divya M S, Sanalkumar R, Indulekha C L, Vidyanand S, Divya T S, et al. In vitro differentiation of retinal ganglion-like cells from embryonic stem cell derived neural progenitors. Biochem Biophys Res Commun. 2009; 380(2):230-5. \\n         18. Divya M S, Rasheed V A, Schmidt T, Lalitha S, Hattar S, James J. Intraocular injection of ES cell-derived neural progenitors improve visual function in retinal ganglion cell-depleted mouse models. Frontiers in cellular neuroscience. 2017; 11:295. \\n         19. Cho J, Mao C, Klein W H. Adult mice transplanted with embryonic retinal progenitor cells: new approach for repairing damaged optic nerves. Molecular vision. 2012; 18:2658. \\n         20. Scruggs B A, Jiao C, Cranston C M, Kaalberg E, Wang K, Russell S R, et al. Optimizing Donor Cellular Dissociation and Subretinal Injection Parameters for Stem Cell-Based Treatments. Stem cells translational medicine. \\n         21. Venugopalan P, Wang Y, Nguyen T, Huang A, Müller K J, Goldberg J L. Transplanted neurons integrate into adult retinas and respond to light. Nature communications. 2016; 7:10472. \\n         22. Miltner A M, La Torre A. Retinal Ganglion Cell Replacement: Current Status and Challenges Ahead. Developmental Dynamics. 2019; 248(1):118-28. \\n         23. Suen H C, Qian Y, Liao J, Luk C S, Lee W T, Ng J K W, et al. Transplantation of Retinal Ganglion Cells Derived from Male Germline Stem Cell as a Potential Treatment to Glaucoma. Stem Cells and Development. 2019; 28(20):1365-75. \\n         24. Oswald J, Kegeles E, Minelli T, Volchkov P, Baranov P. Transplantation of miPSC/mESC-derived retinal ganglion cells into healthy and glaucomatous retinas. Mol Ther Methods Clin Dev. 2021 Mar. 10; 21:180-98. \\n         25. Zhang K Y, Aguzzi E A, Johnson T V. Retinal Ganglion Cell Transplantation: Approaches for Overcoming Challenges to Functional Integration. Cells. 2021 Jun. 8; 10(6):1426. doi: 10.3390/cells10061426. \\n         26. Chavali V R M, Haider N, Rathi S, Vrathasha V, Alapati T, He J, et al. Dual SMAD inhibition and Wnt inhibition enable efficient and reproducible differentiations of induced pluripotent stem cells into retinal ganglion cells. Scientific Reports. 2020; 10(1):11828. \\n         27. Teotia P, Chopra D A, Dravid S M, Van Hook M J, Qiu F, Morrison J, et al. Generation of functional human retinal ganglion cells with target specificity from pluripotent stem cells by chemically defined recapitulation of developmental mechanism. Stem Cells. 2017; 35(3):572-85. \\n         28. Pashos E E, Park Y, Wang X, Raghavan A, Yang W, Abbey D, et al. Large, Diverse Population Cohorts of hiPSCs and Derived Hepatocyte-like Cells Reveal Functional Genetic Variation at Blood Lipid-Associated Loci. Cell Stem Cell. 2017 Apr. 6; 20(4):558,570.e10. \\n         29. Gudiseva H V, Vrathasha V, He J, Bungatavula D, O&#39;Brien J M, Chavali V R M. Single Cell Sequencing of Induced Pluripotent Stem Cell Derived Retinal Ganglion Cells (iPSC-RGC) Reveals Distinct Molecular Signatures and RGC Subtypes. Genes. 2021; 12 (12). \\n         30. Ross A G, McDougald D S, Khan R S, Duong T T, Dine K E, Aravand P, et al. Rescue of retinal ganglion cells in optic nerve injury using cell-selective AAV mediated delivery of SIRT1. Gene Ther. 2021; 28(5):256-64. \\n         31. Clark P J, Evans F C. Distance to Nearest Neighbor as a Measure of Spatial Relationships in Populations. Ecology. 1954; 35(4):445-53. \\n         32. Keeley P W, Eglen S J, Reese B E. From random to regular: Variation in the patterning of retinal mosaics*. J Comp Neurol. 2020; 528(13):2135-60. \\n         33. Cressie N. Statistics for spatial data. John Wiley &amp; Sons; 2015. \\n         34. Drager U C, Olsen J F. Ganglion cell distribution in the retina of the mouse. Invest Ophthalmol Vis Sci. 1981; 20(3):285-93. \\n         35. Didonna A, Opal P. The role of neurofilament aggregation in neurodegeneration: lessons from rare inherited neurological disorders. Molecular Neurodegeneration. 2019; 14(1):19. \\n         36. Tansley K. The gecko retina. Vision Res. 1964; 4 (1-2):33-IN14. \\n         37. Banin E, Obolensky A, Idelson M, Hemo I, Reinhardtz E, Pikarsky E, et al. Retinal incorporation and differentiation of neural precursors derived from human embryonic stem cells. Stem Cells. 2006 February; 24(2):246-57. \\n         38. Osakada F, Ikeda H, Mandai M, Wataya T, Watanabe K, Yoshimura N, et al. Toward the generation of rod and cone photoreceptors from mouse, monkey and human embryonic stem cells. Nat Biotechnol. 2008 February; 26(2):215-24. \\n         39. Lamba D A, Karl M O, Ware C B, Reh T A. Efficient generation of retinal progenitor cells from human embryonic stem cells. Proc Natl Acad Sci USA. 2006 Aug. 22; 103(34):12769-74. \\n         40. Ikeda H, Osakada F, Watanabe K, Mizuseki K, Haraguchi T, Miyoshi H, et al. Generation of Rx+/Pax6+ neural retinal precursors from embryonic stem cells. Proc Natl Acad Sci USA. 2005 Aug. 9; 102(32):11331-6. \\n         41. Meno C, Ito Y, Saijoh Y, Matsuda Y, Tashiro K, Kuhara S, et al. Two closely-related left-right asymmetrically expressed genes, lefty-1 and lefty-2: their distinct expression domains, chromosomal linkage and direct neuralizing activity in  Xenopus  embryos. Genes Cells. 1997 August; 2(8):513-24. \\n         42. del Barco Barrantes I, Davidson G, Gröne H J, Westphal H, Niehrs C. Dkk1 and noggin cooperate in mammalian head induction. Genes Dev. 2003 Sep. 15; 17(18):2239-44. \\n         43. Davis A A, Matzuk M M, Reh T A. Activin A promotes progenitor differentiation into photoreceptors in rodent retina. Mol Cell Neurosci. 2000 January; 15(1):11-21. \\n         44. Aoki H, Hara A, Niwa M, Motohashi T, Suzuki T, Kunisada T. An in vitro mouse model for retinal ganglion cell replacement therapy using eye-like structures differentiated from E S cells. Exp Eye Res. 2007 May; 84(5):868-75. \\n         45. Meyer J S, Shearer R L, Capowski E E, Wright L S, Wallace K A, McMillan E L, et al. Modeling early retinal development with human embryonic and induced pluripotent stem cells. Proc Natl Acad Sci USA. 2009 Sep. 29; 106(39):16698-703. \\n         46. Meyer J S, Howden S E, Wallace K A, Verhoeven A D, Wright L S, Capowski E E, et al. Optic vesicle-like structures derived from human pluripotent stem cells facilitate a customized approach to retinal disease treatment. Stem Cells. 2011; 29(8):1206-18. \\n         47. Nakano T, Ando S, Takata N, Kawada M, Muguruma K, Sekiguchi K, et al. Self-formation of optic cups and storable stratified neural retina from human ESCs. Cell Stem Cell. 2012 Jun. 14; 10(6):771-85. \\n         48. Eiraku M, Takata N, Ishibashi H, Kawada M, Sakakura E, Okuda S, et al. Self-organizing optic-cup morphogenesis in three-dimensional culture. Nature. 2011 Apr. 7; 472(7341):51-6. \\n         49. Singh R, Shen W, Kuai D, Martin J M, Guo X, Smith M A, et al. iPS cell modeling of Best disease: insights into the pathophysiology of an inherited macular degeneration. Hum Mol Genet. 2012; 22(3):593-607. \\n         50. Chamling X, Sluch V M, Zack D J. The potential of human stem cells for the study and treatment of glaucoma. Invest Ophthalmol Vis Sci. 2016; 57 (5):ORSFil-6. \\n         51. Varela-Fernández R, Díaz-Tomé V, Luaces-Rodríguez A, Conde-Penedo A, García-Otero X, Luzardo-Álvarez A, et al. Drug Delivery to the Posterior Segment of the Eye: Biopharmaceutic and Pharmacokinetic Considerations. Pharmaceutics. 2020; 12(3):269. \\n         52. Johnson T V, Martin K R. Development and characterization of an adult retinal explant organotypic tissue culture system as an in vitro intraocular stem cell transplantation model. Invest Ophthalmol Vis Sci. 2008 August; 49(8):3503-12. \\n         53. Zhang K Y, Tuffy C, Mertz J L, Quillen S, Wechsler L, Quigley H A, et al. Role of the Internal Limiting Membrane in Structural Engraftment and Topographic Spacing of Transplanted Human Stem Cell-Derived Retinal Ganglion Cells. Stem Cell Reports. 2021 Jan. 12; 16(1):149-67. \\n         54. Dalkara D, Kolstad K D, Caporale N, Visel M, Klimczak R R, Schaffer D V, et al. Inner Limiting Membrane Barriers to AAV-mediated Retinal Transduction From the Vitreous. Molecular Therapy. 2009; 17(12):2096-102. \\n         55. Takahashi K, Igarashi T, Miyake K, Kobayashi M, Yaguchi C, Iijima O, et al. Improved Intravitreal AAV-Mediated Inner Retinal Gene Transduction after Surgical Internal Limiting Membrane Peeling in Cynomolgus Monkeys. Molecular Therapy. 2017; 25(1):296-302. \\n         56. Teo K Y C, Lee S Y, Barathi A V, Tun S B B, Tan L, Constable I J. Surgical Removal of Internal Limiting Membrane and Layering of AAV Vector on the Retina Under Air Enhances Gene Transfection in a Nonhuman Primate. Invest Ophthalmol Vis Sci. 2018; 59(8):3574-83. \\n         57. Randlett O, Poggi L, Zolessi F R, Harris W A. The oriented emergence of axons from retinal ganglion cells is directed by laminin contact in vivo. Neuron. 2011 Apr. 28; 70(2):266-80. \\n         58. Riccomagno M M, Sun L O, Brady C M, Alexandropoulos K, Seo S, Kurokawa M, et al. Cas adaptor proteins organize the retinal ganglion cell layer downstream of integrin signaling. Neuron. 2014 Feb. 19; 81(4):779-86. \\n         59. Nickerson P E B, Ortin-Martinez A, Wallace V A. Material exchange in photoreceptor transplantation: Updating our understanding of donor/host communication and the future of cell engraftment science. Frontiers in Neural Circuits. 2018; 12. \\n         60. Pearson R A, Gonzalez-Cordero A, West E L, Ribeiro J R, Aghaizu N, Goh D, et al. Donor and host photoreceptors engage in material transfer following transplantation of post-mitotic photoreceptor precursors. Nature Communications. 2016; 7. \\n         61. Waldron P V, Di Marco F, Kruczek K, Ribeiro J, Graca A B, Hippert C, et al. Transplanted Donor- or Stem Cell-Derived Cone Photoreceptors Can Both Integrate and Undergo Material Transfer in an Environment-Dependent Manner. Stem Cell Reports. 2018; 10(2):406-21. \\n         62. Singh M S, Balmer J, Barnard A R, Aslam S A, Moralli D, Green C M, et al. Transplanted photoreceptor precursors transfer proteins to host photoreceptors by a mechanism of cytoplasmic fusion. Nature Communications. 2016; 7. \\n         63. Santos-Ferreira T, Llonch S, Borsch O, Postel K, Haas J, Ader M. Retinal transplantation of photoreceptors results in donor-host cytoplasmic exchange. Nature Communications. 2016; 7(1):13028. \\n         64. Lam T T, Abler A S, Kwong J M, Tso M O. N-methyl-D-aspartate (NMDA)â€ “induced apoptosis in rat retina. Invest Ophthalmol Vis Sci. 1999; 40(10):2391-7. \\n         65. Honda S, Namekata K, Kimura A, Guo X, Harada C, Murakami A, et al. Survival of Alpha and Intrinsically Photosensitive Retinal Ganglion Cells in NMDA-Induced Neurotoxicity and a Mouse Model of Normal Tension Glaucoma. Invest Ophthalmol Vis Sci. 2019; 60(12):3696-707. \\n         66. Khan R, Chen Y, Cwanger A, Song Y, Dunaief J, Kim S, et al. Dexras1 Mediates Retinal Ganglion Cell Loss Induced by NMDA Excitotoxicity. Invest Ophthalmol Vis Sci. 2013; 54(15):1414. \\n         67. Chen Y, Khan R S, Cwanger A, Song Y, Steenstra C, Bang S, et al. Dexras1, a Small GTPase, Is Required for Glutamate-NMDA Neurotoxicity. J Neurosci. 2013; 33(8):3582-7. \\n         68. Duong T T, Vasireddy V, Ramachandran P, Herrera P S, Leo L, Merkel C, et al. Use of induced pluripotent stem cell models to probe the pathogenesis of Choroideremia and to develop a potential treatment. Stem cell research. 2018; 27:140-50. \\n         69. Sullivan S K, Mills J A, Koukouritaki S B, Vo K K, Lyde R B, Paluru P, et al. High-level transgene expression in induced pluripotent stem cell-derived megakaryocytes: correction of Glanzmann thrombasthenia. Blood. 2014; 123(5):753-7. \\n         70. Maguire J A, Gagne A, Mills J A, Gadue P, French D L. Generation of human control iPS cell line CHOPWT9 from healthy adult peripheral blood mononuclear cells. Stem Cell Research. 2016; 16(1):14-6. \\n         71. Espuny-Camacho I, Michelsen K A, Gall D, Linaro D, Hasche A, Bonnefont J, et al. Pyramidal neurons derived from human pluripotent stem cells integrate efficiently into mouse brain circuits in vivo. Neuron. 2013; 77(3):440-56. \\n         72. McDougald D S, Dine K E, Zezulin A U, Bennett J, Shindler K S. SIRT1 and NRF2 Gene Transfer Mediate Distinct Neuroprotective Effects Upon Retinal Ganglion Cell Survival and Function in Experimental Optic Neuritis. Invest Ophthalmol Vis Sci. 2018; 59(3):1212-20. \\n         73. Chaffiol A, Caplette R, Jaillard C, Brazhnikova E, Desrosiers M, Dubus E, et al. A New Promoter Allows Optogenetic Vision Restoration with Enhanced Sensitivity in Macaque Retina. Mol Ther. 2017 Nov. 1; 25(11):2546-60. \\n         74. Costello F, Coupland S, Hodge W, Lorello G R, Koroluk J, Pan Y I, et al. Quantifying axonal loss after optic neuritis with optical coherence tomography. Ann Neurol. 2006 June; 59(6):963-9. \\n         75. Berg S, Kutra D, Kroeger T, Straehle C N, Kausler B X, Haubold C, et al. Ilastik: Interactive Machine Learning for (Bio)image Analysis. Nat Methods. 2019 December; 16(12):1226-32. \\n       \\n    \\n     Example 7 \\n     RGC Cell Replacement Therapy for the Human Eye \\n     The information herein above can be applied clinically to patients for therapeutic intervention as part of a cell replacement therapy. One embodiment of the cell replacement therapy is shown in  FIG.  23   . A preferred embodiment of the invention comprises clinical application of the RGCs recited herein to a patient. This can occur after a patient arrives in the clinic and presents with symptoms of a progressive optic neuropathy. In certain embodiments, the progressive optic neuropathy is glaucoma. Purified iPSC-RGCs can be injected intravitreally. In certain embodiments RGCs are harvested at Day 32-35 of culture. In other embodiments, the RGCs are induced from stem cells obtained from the patient. In certain embodiments, a 30-34 gauge needle is used to administer the iPSC-RGCs. Integration of iPSC-RGCs into human retina can be evaluated using ERGs and Optokinetic Response (OKR). In certain embodiments, administration of the iPSC-RGCs results in a reduction in progressive optic neuropathy symptoms. After administration, patients may be assessed using Fundus imaging, SD-OCT, and/or cSLO every 2 weeks, every 4 weeks, and/or every two month. This assessment may continue for 1 year or longer. In certain embodiments, visual field testing and OCT analysis will be done for positive functional outcomes. Such testing may occur every 2 months for 1 year or longer. Testing for immune rejection markers can also be done to assess transplantation success. \\n     In certain embodiments, personalized treatments are administered by deriving the iPSCs from the glaucoma patient. These iPSCs are then treated and the purified iPSC-RGCs are then administered to the back to the patient. In certain embodiments the iPSCs are derived from the patients blood using peripheral blood mononuclear cells (PBMNCs) or using skin biopsies to obtain and culture keratinocytes. Other sources of stem cells, such as fat cells, can be used. \\n     In a different approach, a control iPSC is obtained from a source other than the patient. In certain embodiments, the control iPSC has no SNPs that are associated with primary open angle glaucoma, age related macular degeneration or retinal degenerations. These control cells may be cultured in batches, thereby generating retinal progenitor cell (RPC) libraries. These RPC libraries may be frozen. In certain embodiments the RPC libraries are frozen at Day 10, 12, 15 or 18. These RPCs are unfrozen prior to differentiation into RGCs using the protocol described above. Mature iPSC-RGCs may then be administered via intravitreal injection into a patient&#39;s retina for cell replacement therapy. \\n     RPC may be generated from the protocol described above. These RPCs can used as retinal precursors. The RPCs may be injected intravitreally to differentiate into photoreceptors to treat their loss in patients with retinal degeneration as cell-based therapy. \\n     The iPSC-RGCs can optionally be labeled prior to administration, e.g., with a fluorescent label or channel rhodopsin proteins for optogenetic treatment approaches. iPSC-RGCs can be labeled by transduction with an AAV virus as described in Example 2. \\n     The compositions and methods disclosed herein provide the means to generate and produce homogeneous populations of retinal ganglion cells that can be used to advantage for the treatment of various ocular disorders. \\n     While certain features of the invention have been described herein, many modifications, substitutions, changes, and equivalents will now occur to those of ordinary skill in the art. It is, therefore, to be understood that the appended claims are intended to cover all such modifications and changes as fall within the true spirit of the invention. \\n     Embodiments \\n     1. A method of preparing retinal progenitor cells and retinal ganglion cells by directed differentiation of induced pluripotent stem cells (iPSCs) in a defined chemical medium, comprising,\\n         a) culturing said iPSC in iPSC initiation medium;   b) transferring said iPSC of step a) to retinal progenitor cell (RPC) culture medium for transdifferentiation into RPCs;   c) culturing said RPCs of step b) in retinal ganglion cell (RGC) culture medium for transdifferentiation into RGCs; thereby providing a population of RGCs suitable for in vitro studies and for transplantation into the eye.       \\n\\n     2. The method of embodiment 1, wherein said iPSCs are cultured in a 37° C., 5% CO 2  and 5% O 2  incubator and the iPSC initiation medium comprises one of:\\n         i) 80% HES/20% MEF-CM+20 ng/ml bFGF+4 μM Y27632;   ii) 100% HES+20 ng/ml bFGF+4 μM Y27632; or   iii) 100% StemMACS™ iPS Brew-Xenofree (XF)+20 ng/ml bFGF+4 μM Y27632.       \\n\\n     3. The method of any of the preceding embodiments, wherein the RPC culture medium comprises RPC induction media, with 0.1 μM LDN193189, 10 μM SB431542, 2 μM XAV939, 10 mM Nicotinamide, 10 ng/ml IGF1, 1.5 μM CHIR99021 and 10 ng/ml bFGF reagents. \\n     4. The method of any one of the preceding embodiments, wherein the RGC culture medium comprises RGC induction media, 3 μM DAPT, 250 ng/ml sonic hedgehog (SHH) and 100 ng/ml FGF8 or smoothened agonist (SAG). \\n     5. The method of any of the preceding embodiments comprising isolating said RGCs. \\n     6. The method of embodiment 5, wherein said RGCs express BRN3A, BRN3B, TUBB3, CD90, MAP2, TUJ1, RBPMS and TUJ1 markers. \\n     7. The method of any of the preceding embodiments wherein said RGC population is obtained after between 30 to 45 days in culture from iPSCs. \\n     8. A method for the production of retinal progenitor cells (RPCs), comprising;\\n         a) culturing iPSCs on 0.1% gelatinized plates containing irradiated MEFs until cells achieved approximately 75% confluence;   b) removing said MEFs and plating remaining iPSCs onto plates containing 1:100 diluted growth factor reduced Matrigel and iPSC: MEF-conditioned medium (80:20)+20 ng/mL of bFGF and 5 ng/ml of stable bFGF until reaching 100% confluence;   c) replacing iPSC: MEF-conditioned media with RPC induction media comprising DMEM/F12 (50:50), 1% P/S, 1% Glut, 1% NEAA, 0.1 mM 2-ME, 2% B27 supplement (w/o vitamin A), 1% N2 supplement, containing effective amounts of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 mM nicotinamide, and 10 ng/mL IGF1 and culturing cells for 4 days with daily media changes;   d) replacing the culture media of step c) with RPC induction media containing an effective amount of a Wnt inhibitor, a TGFβ inhibitor and a BMP inhibitor, 10 ng/mL IGF1, and 10 ng/mL bFGF on day 5 and culturing said cells for 12 days with daily media changes;   e) replacing RPC induction media with RGC differentiation media comprising 0.1 μM LDN193198, 10 μM SB431542, 2 μM XAV939, 1.5 μM CHIR99021, 10 ng/ml IGF1 and 10 ng/ml bFGF on day 16 of differentiation culturing said cells until Day 23, upon which RGC differentiation occurred. thereby producing a population of RGCs suitable for transplantation into the eye.       \\n\\n     9. The method of any of the preceding embodiments wherein said Wnt inhibitor is XAV939, said TGF-β inhibitor is SB431542 and said BMP inhibitor is LDN193189. \\n     10. The method of any of the preceding embodiments, wherein the iPSCs are from stem cells selected from the group consisting of bone marrow stem cells (BMS), cord blood stem cells, amniotic fluid stem cells, fat stem cells, retinal stem cells (RSCs), keratinocyte stem cells, intraretinal Müller glial cells, embryonic stem cells (ESCs), corneal endothelial cells and somatic cell nuclear transfer cells (SCNTCs). \\n     11. The method of embodiment 8, wherein the Wnt signaling pathway activator is one or more selected from the group consisting of Wnt1, Wnt2, Wnt2b, Wnt3, Wnt3a, Wnt4, Wnt5a, Wnt5b, Wnt6, Wnt7a, Wnt7b, Wnt8a, Wnt8b, Wnt9a, Wnt9b, Wnt10a, Wnt10b, Wnt11, and Wnt16b, substance increasing p3-catenin levels; lithium, LiCl, bivalent zinc, BIO (6-bromoindirubin-3′-oxime), SB216763, SB415286, CHIR99021, QS11 hydrate, TWS119, Kenpaullone, alsterpaullone, indirubin-3′-oxime, TDZD-8 and Ro 318220 methanesulfonate salt; Axin inhibitors; APC (adenomatous polyposis coli) inhibitors; norrin and R-spondin 2. \\n     12. The method of embodiment 8, further comprising the step of determining whether the retinal progenitor cells are differentiated into the mature retinal ganglion cells. \\n     13. The method of embodiment 12, wherein the step of determining whether the retinal progenitor cells are differentiated into the mature retinal ganglion cells is performed by measuring mRNA or protein expression levels of one or more genes selected from the group consisting of SOX2, RAX, PAX6, SIX6, SIX3, VSX2 and LHX2. \\n     14. The method of embodiment 1, wherein the mature retinal ganglion cells are present at from about 60% to about 95% or more of total cells after the culturing of step \\n     15. The method of embodiment 1, wherein said RGCs comprise an AAV vector expressing a transgene of interest. \\n     16. The method of embodiment 1, where in RGCs are differentiated from iPSCs without genetic manipulation using CRISPR gene editing methodologies. \\n     17. The method of embodiment 1, where in iPSCs are differentiated into RGCs from normal and disease patients with ocular pathologies selected from glaucoma, age-related macular degeneration (AMD) or any type of retinal degeneration, a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     18. A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RGCs as embodied in embodiment 1, in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     19. A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RPCs as embodied in embodiment 8, in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     20. The method according to embodiment 18 or embodiment 19, wherein the disorder is glaucoma. \\n     21. The method of embodiment 18 or embodiment 19, wherein the disorder is related to an increase in intraocular pressure (IOP). \\n     22. The method of embodiment 18 or embodiment 19, wherein the age-related macular degeneration is wet age-related macular degeneration. \\n     23. The method of embodiment 18 or embodiment 19, wherein the host is a human. \\n     24 The method of embodiment 18 or embodiment 19, wherein the cells are administered via intravitreal, intrastromal, intracameral, sub-tenon, sub-retinal, retro-bulbar, peribulbar, suprachoroidal, choroidal, subchoroidal, conjunctival, subconjunctival, epi scleral, posterior juxtascleral, circumcorneal, optic nerve or tear duct injection. \\n     25. The method of embodiment 22, wherein the cells are administered via intravitreal injection. \\n     26. The method of embodiment 22, wherein the compound is administered via subretinal injection. \\n     27. The method of embodiment 22, wherein the cells described in embodiment 18 and embodiment 19 are administered through routes in embodiment 24 either before optic nerve injury through optic nerve crush or via delivery to the optic nerve. \\n     28. The method of any one of the preceding embodiments, wherein the RGC maturation medium comprises 30 ng/ml to 50 ng/ml BDNF. \\n     29. A method for the treatment of an ocular disorder, comprising administering a therapeutically effective amount of a population of purified RGCs using immunopanning, flowcytometry and cell sorting with RGC specific surface antibodies, in a pharmaceutically carrier to a host in need thereof wherein the disorder is selected from glaucoma, age-related macular degeneration (AMD), a disorder related to an increase in intraocular pressure (IOP), a disorder related to neuroprotection, or diabetic retinopathy. \\n     30. The method of embodiment 29, wherein CD90+RGCs are immunopurified via MACS sorting with CD90.2 magnetic beads. \\n     31. An isolated population of RGC prepared by the method of embodiment 1. \\n     32. An isolated population of RPC prepared by the method of embodiment 8.', 'id': 'US-2023042830-A1', 'ti': 'Compositions and methods for generation of retinal ganglion cells from inducible pluripotent stem cells for the treatment of progressive optic neuropathies, including glaucoma'}>]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query = \"USA\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))\n",
    "results = whoosh_utils.execute_query(query, qp, searcher)[:5]\n",
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "(ti:visited OR ab:visited OR clm:visited OR detd:visited OR cpc:visited OR id:visited)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[<Hit {'ab': 'A wireless router is provided which is configured to support a first subnet and a second subnet. The wireless router comprises a plurality of virtual wireless switches. Each virtual wireless switch comprises a plurality of access ports. The first subnet comprises a group of the access ports belonging to the first virtual wireless switch, and the second subnet comprises a second group of the access ports belonging to the second virtual wireless switch. The wireless router is configured to support layer 3 mobility when a client, having a client IP address from within the first subnet, roams from the first subnet to the second subnet, from the first to the second virtual wireless switch. The wireless router stores registration information associated with the client to allow a client to roam between the first subnet and the second subnet while keeping the client IP address. The wireless router uses the registration information to send packets to the client when the client has roamed to the second subnet.', 'clm': '1 . A wireless local area network, comprising: \\n a router; and    a wireless router coupled to the router, comprising: \\n a home virtual wireless switch module configured to support a first subnet; and  \\n a visited virtual wireless switch module configured to support a second subnet.  \\n   \\n     \\n     \\n         2 . The wireless local area network of  claim 1 , further comprising: \\n a connection between the home virtual wireless switch module and the visited virtual wireless switch module.    \\n     \\n     \\n         3 . The wireless local area network of  claim 2 , further comprising: \\n a client, initially associated with the home virtual wireless switch module and having a client IP address from within the first subnet,    wherein the client maintains the client IP address when the client roams from the first subnet to the second subnet.    \\n     \\n     \\n         4 . The wireless local area network of  claim 3 , wherein the visited virtual wireless switch module is configured to receive an IP packet from the client and forwards the IP packet through the connection to the home virtual wireless switch module.  \\n     \\n     \\n         5 . The wireless local area network of  claim 4 , wherein the home virtual wireless switch module is configured to receive a second IP packet for the client and forward the second IP packet through the connection to the visited virtual wireless switch module, wherein the visited virtual wireless switch module sends the second IP packet to the client.  \\n     \\n     \\n         6 . The wireless local area network of  claim 5 , wherein the home virtual wireless switch module creates a first active client list using the registration information from each client in the first subnet, wherein the visited virtual wireless switch module creates a second active client list using the registration information from each client in the second subnet, and further comprising: \\n a virtual registration server module, coupled to home virtual wireless switch module and the visited virtual wireless switch module, configured to create a master active client list using the first and second active client lists, and send a copy of the active client list to each wireless switch;    wherein the registration information from each client comprises a MAC address of the client, the client IP address of the client, the home virtual wireless switch module of the client, the visited virtual wireless switch module of the client, inactivity timers for the home virtual wireless switch module and the visited virtual wireless switch module.    \\n     \\n     \\n         7 . The wireless local area network of  claim 6 , wherein the visited virtual wireless switch module uses the master active client list to determine that the client IP address belongs to the first subnet and that the client was originally associated with the home virtual wireless switch module.  \\n     \\n     \\n         8 . The wireless local area network of  claim 7 , wherein the visited virtual wireless switch module uses the master active client list to obtain the client IP address from registration information associated with the client.  \\n     \\n     \\n         9 . The wireless local area network of  claim 8 , when the client roams from the home virtual wireless switch module to the visited virtual wireless switch module, the client 802.11 authenticates with the visited virtual wireless switch module, 802.11 associates with the visited virtual wireless switch module, 802.1x authenticates with the visited virtual wireless switch module and issues a Dynamic Host Configuration Protocol (DHCP) request, and wherein the visited virtual wireless switch module relays the DHCP request to the home virtual wireless switch module through the connection.  \\n     \\n     \\n         10 . The wireless local area network of  claim 9 , wherein the master active client list comprises a record for the client comprising a MAC address of the client, the client IP address of the client, the home virtual wireless switch module of the client, the visited virtual wireless switch module of the client, inactivity timers for the home virtual wireless switch module and the visited virtual wireless switch module, and wherein the client 802.11 re-authenticates and searches the record to get the home virtual wireless switch module from the MAC address of the client.  \\n     \\n     \\n         11 . The wireless local area network of  claim 10 , further comprising: \\n a first Dynamic Host Configuration Protocol (DHCP) server coupled to the wireless router, wherein the home virtual wireless switch module passes the DHCP request to the first DHCP server, and wherein the first DHCP server re-assigns the client IP address to the client and forwards a DHCP response to the visited virtual wireless switch module and the client    \\n     \\n     \\n         12 . The wireless local area network of  claim 10 , wherein the client 802.11 reautheticates, 802.1x authenticates, generates new transient keys and continues with an existing TCP session.  \\n     \\n     \\n         13 . The wireless local area network of  claim 10 , wherein the client 802.11 reautheticates, 802.1x authenticates, generates new transient keys and continues with an existing UDP session.  \\n     \\n     \\n         14 . The wireless local area network of  claim 6 , wherein if the inactivity timers of the client indicate that the client is inactive on its home switch and its visited switch, then the client is forced to reauthenticate and get a new client A address on a new wireless switch  \\n     \\n     \\n         15 . In a wireless local area network a method for allowing a client, initially associated with a home virtual wireless switch module and having a client IP address from within a first subnet, to roam from the first subnet to a second subnet supported by a visited virtual wireless switch module configured to support a second subnet, the method comprising: \\n creating a connection between the home virtual wireless switch module and the visited virtual wireless switch module; and    maintaining the client IP address on the client when the client roams from the first subnet to the second subnet;    receiving an IP packet from the client at the visited Dual wireless switch module; and    forwarding the IP packet through the connection to the home virtual wireless switch module.    \\n     \\n     \\n         16 . The method of  claim 15 , further comprising: \\n receiving a second IP packet for the client at the home virtual wireless switch module;    forwarding the second IP packet through the connection to the visited virtual wireless switch module; and    sending the second IP packet from the visited virtual wireless switch module to the client.    \\n     \\n     \\n         17 . The method of  claim 16 , further comprising: \\n sending registration information associated with each client in the first subnet to the home virtual wireless switch module;    creating a first active client list using the registration information from each client in the first subnet;    sending registration information associated with each client in the second subnet to the visited virtual wireless switch module;    creating a second active client list using the registration information from each client in the second subnet;    creating an master active client list using the first active client list and the second active client list; and    sending a copy of the master active client list to each wireless switch in the wireless local area network.    \\n     \\n     \\n         18 . The method of  claim 17 , wherein the registration information from each client comprises a MAC address of the client, the client IP address of the client, the home virtual wireless switch module of the client, the visited virtual wireless switch module of the client, inactivity timers for the home virtual wireless switch module and the visited virtual wireless switch module.  \\n     \\n     \\n         19 . The method of  claim 18 , further comprising: \\n using the master active client list at the visited virtual wireless switch module to determine that the client IP address belongs to the first subnet and that the client was originally associated with the home virtual wireless switch module.    \\n     \\n     \\n         20 . The method of  claim 19 , wherein using the master active client list at the visited virtual wireless switch module to determine that the client IP address belongs to the first subnet and that the client was originally associated with the home virtual wireless switch module, comprises: \\n obtaining the client IP address from a portion of the master active client list including the registration information associated with the home virtual wireless switch module.    \\n     \\n     \\n         21 . The method of  claim 18 , wherein maintaining the client IP address at the client when the client roams from the home virtual wireless switch module and the visited virtual wireless switch module, comprises: \\n 802.11 authenticating the client with the visited virtual wireless switch module;    802.11 associating the client with the visited virtual wireless switch module;    802.1x authenticating the client with the visited virtual wireless switch module; and    issuing a Dynamic Host Configuration Protocol (DHCP) request from the client to the visited virtual wireless switch module;    relaying the DHCP request from the visited virtual wireless switch module to the home virtual wireless switch module through the connection;    passing the DHCP request from the home virtual wireless switch module to a first Dynamic Host Configuration Protocol (DHCP) server, wherein the first DHCP server re-assigns the client IP address to the client; and    forwarding a DHCP response from the first DHCP server to the visited virtual wireless switch module and the client.    \\n     \\n     \\n         22 . The method of  claim 20 , wherein 802.11 authenticating the client with the visited virtual wireless switch module, comprises: \\n 802.11 authenticating the client with the visited virtual wireless switch module; and    using the record of the client to obtain the home virtual wireless switch module from the MAC address of the client.    \\n     \\n     \\n         23 . The method of  claim 20 , further comprising: \\n generating new transient keys and continuing with an existing TCP session.    \\n     \\n     \\n         24 . The method of  claim 2 , further comprising: \\n generating new transient keys and continuing with an existing UDP session.    \\n     \\n     \\n         25 . The method of  claim 17 , wherein if the inactivity timers of the client indicate that the client is inactive on its home switch and its visited switch, further comprising: \\n making the client reauthenticate and get a new client IP address on a new wireless switch.    \\n     \\n     \\n         26 . A wireless router configured to support a first subnet and a second subnet.  \\n     \\n     \\n         27 . The wireless router of  claim 26 , wherein the wireless router comprises a plurality of access ports, wherein the first subnet comprises a first group of the access ports, and wherein the second subnet comprises a second group of the access ports.  \\n     \\n     \\n         28 . The wireless router of  claim 27 , wherein the wireless router is configured to support layer 3 mobility when a client, having a client IP address from within the first subnet, roams from the first subnet to the second subnet.  \\n     \\n     \\n         29 . The wireless router of  claim 28 , wherein the wireless router stores registration information associated with the client to allow a client to roam between the first subnet and the second subnet while keeping the client IP address.  \\n     \\n     \\n         30 . The wireless router of  claim 29 , wherein the wireless router uses the registration information to send packets to the client when the client has roamed to the second subnet.  \\n     \\n     \\n         31 . The wireless local area network of  claim 1 , wherein the wireless router is a wireless switch.  \\n     \\n     \\n         32 . The wireless local area network of  claim 2 , wherein the connection comprises a UDP/IP tunnel.  \\n     \\n     \\n         33 . The method of  claim 15 , wherein the connection comprises a UDP/IP tunnel.  \\n     \\n     \\n         34 . The wireless router of  claim 26 , wherein the wireless router is a wireless switch.', 'cpc': 'H04L2101/668', 'detd': 'TECHNICAL FIELD OF THE INVENTION  \\n      The present invention generally relates to computer networks and, more particularly, to methods, systems and apparatus for layer 3 roaming in the context of Wireless Local Area Networks (WLANs).  \\n     BACKGROUND OF THE INVENTION  \\n      WLANs, based on the IEEE 802.11 standards, have conventionally been used for ordinary Internet services such as web browsing, file transfers and electronic mail. However, with the emerging usage of real time multimedia applications such as voice over IP (VoIP) telephony, these same WLAN networks can also be used as infrastructure for enabling such applications. WLANs can give clients the ability to “roam” or physically move from place to place without being connected by wires. In the context of WLANs the term “roaming” describes the act of physically moving between access ports (APs). One issue in the area of WLANs relates to the ability to maintain an IP-connection while roaming.  \\n       FIG. 1  is a block diagram of a conventional wireless local area network (WLAN). The WLAN  1  of  FIG. 1  includes wireless clients  2 ,  4 , a first subnet (A)  10 , a wireless switch  12 , access ports (APs)  14 ,  16 , a second subnet (B)  20 , a wireless switch  22 , access ports (APs)  24 ,  26  and layer 3 routers  34 ,  36 . The router  34  is coupled to the wireless switch  12 . The wireless switch  12  supports the first subnet (A)  10  and is coupled to the access ports (APs)  14 ,  16 . The access ports (APs)  14 ,  16  have IP addresses within the first subnet (A)  10 . The router  36  is coupled to the wireless switch  22 . The wireless switch  22  supports the second subnet (B)  20  and is coupled to the access ports (APs)  24 ,  26 . The access ports (APs)  24 ,  26  have IP addresses within the second subnet (B)  20 . The clients  2 ,  4  are wireless devices which physically move around the WLAN  1 , and communicate with an IP network via the access ports (APs)  14 ,  16  and access ports (APs)  24 ,  26 , respectively.  \\n       FIG. 1  illustrates the concept of layer 2 roaming and the concept of layer 3 roaming in the WLAN. A layer 2 network is defined as a single IP subnet and broadcast domain, such as the first subnet (A)  10 , while a layer 3 network is defined as the combination of multiple IP subnets and broadcast domains, such as the first subnet (A)  10  and the second subnet (B)  20 .  \\n      Layer 2 refers to the data link layer of the Open Systems Interconnection (OSI) communication model. The data link layer is concerned with moving data across the physical links in the network. In a network, the switch is a device that redirects data messages at the layer 2 level, using the destination Media Access Control (MAC) address to determine where to direct the message. In the context of the IEEE-802 LAN standards, the data link layer contains two sublayers called the Media Access Control (MAC) sublayer and the Logical Link Control (LLC) sublayer. The data link layer ensures that an initial connection has been set up, divides output data into data frames, and handles the acknowledgements from a receiver that the data arrived successfully. The data link layer also ensures that incoming data has been received successfully by analyzing bit patterns at special places in the frames.  \\n      Layer 2 roaming occurs when a client moves far enough away from its AP such that its radio associates with a different AP in the same subnet. The client disconnects from one Access port (AP) and re-connects to another AP in the same subnet (broadcast domain) where several APs use the same Service Set Identifier (SSID). A client continuously listens to nearby APs and can decide to roam if it finds an AP with the same SSID and a stronger signal or is experiencing too much loss with the current AP. To initiate a layer 2 roam, the client sends an associate (or reassociate) request to the new AP. It may disassociate from the old AP, or the old AP may notice the client is no longer there.  \\n      IEEE&#39;s 802.11f Inter Access port Protocol (IAPP) addresses roaming between Access ports (APs) inside client&#39;s home subnet and assures constant IP-connectivity in this case. With layer 2 roaming, APs inside a given subnet share the same Extended Service Set (ESS), and although the physical point of attachment (the AP) changes, the client is still served by the same Access Router. Because the original and the new AP offer coverage for the same IP subnet, the device&#39;s IP address is still valid after the roam and can remain unchanged. For example, when the roams within the first subnet (A)  10 , the IP address of the client will remain the same.  \\n      After the client successfully roams, LAN traffic for the client can be relayed through the new AP. However, because the scalability of subnets is limited by the number of APs and clients that can be supported within a given subnet, in some situations the client roams to a new AP in a different or foreign subnet supported by another wireless switch. Because the client cannot be identified by its original home IP address anymore, a new IP address is required for the routing the client&#39;s IP data. Consequently, any on-going connections can be disrupted and IP connectivity can be lost. For applications like wireless VoIP phones or streaming applications, this is not acceptable.  \\n      Layer 3 refers to the network layer of the Open Systems Interconnection (OSI) multilayered communication model. The network layer is concerned with knowing the address of the neighboring nodes in the network, selecting routes and quality of service, and recognizing and forwarding to the transport layer incoming messages for local host domains.  \\n      Layer 3 roaming occurs when a client moves from an AP within its home IP subnet, such as the first subnet (A)  10 , to a new AP within a foreign IP subnet, such as the second subnet (B)  20 . This foreign IP subnet has a different Basic Service Set (BSS) than the home IP subnet. The client disconnects from one AP and reconnects or re-associates with another foreign AP in a foreign IP subnet outside its home IP subnet. In this re-association, the client is supposed to be served by a different access router (through the foreign AP), which bares a different IP address, while the client itself preserves its original IP address. At that point, the client would no longer have an IP address and default gateway that are valid within the foreign IP subnet. Therefore, if no other protocol is implemented to address an L3 roam, the client will not able to send/receive IP packets from/to its current location. As a result, active IP sessions can be dropped because IP-connectivity is lost.  \\n      To prevent existing data sessions or voice calls from failing because the remote client can no longer reach the local client, processes called “IP handoff” or “L3 handover” can be used to preserve the IP traffic to/from the client after such re-association with the foreign AP. Because this process is not addressed by current IEEE nor Wi-Fi standards, important functions, such as preservation of the client&#39;s IP connectivity upon a layer 3 handover, have yet to be standardized.  \\n      Nevertheless, some vendors of WLANs have developed solutions which can allow layer 3 roaming to occur by providing mechanisms for a client to obtain a new IP address. For instance, if the client roams across a boundary between the first subnet (A)  10  and the second subnet (B)  20  and a Dynamic Host Configuration Protocol (DHCP) is enabled on the client, then the client can use DHCP to obtain a new IP address of the second subnet (B)  20 .  \\n      However, layer 3 traffic re-routing requires more than updating MAC address tables and ARP caches. Many applications require persistent connections and drop their sessions as a result of inter-subnet roaming. Network layer devices such as routers and layer  3  switches must somehow be told to forward IP packets to the client&#39;s new subnet. To provide session persistence, mechanisms are need to allow a client to maintain the same Layer 3 address while roaming throughout a multi-subnet network. Otherwise, many applications will timeout trying to reach the client&#39;s old IP and must be reconnect with the client&#39;s new IP.  \\n      One way to support layer 3 roaming in WLANs is via an open IETF standard called Mobile IP. Mobile IP provides one solution for handling the L3 movements of clients regardless of the underlying layer 2 technology.  \\n      In the context of Mobile IP, the client is referred to as a mobile node (MN). In the description that follows, these terms are used interchangeably. Mobile IP uses a Home Agent (HA) to forward IP packets to a Foreign Agent (FA) in the client&#39;s new subnet. The HA and FA advertise themselves using the ICMP Router Discovery Protocol (IRDP). The Foreign Agent periodically advertises its presence wirelessly and waits for a solicitation message from a roaming mobile node. When a Mobile IP-enabled client roams to a new subnet, it must discover and register itself with a nearby FA. The registration process for such a node is triggered by a wireless registration request (after the 802.11 association is completed) issued by the MN. The FA forwards that request to that client&#39;s original HA. Wired messages can then be exchanged between the HA and the FA as well as with binding table updates. An acknowledgment can then be sent wirelessly to the MN.  \\n      If the request is accepted, a tunnel is established between the HA and FA to relay incoming packets sent to the client&#39;s original IP address. The HA serves as the anchor point for communication with the wireless client. It tunnels packets from Corresponding Nodes (CNs) towards the current address of the MN and vise versa. Outbound packets are routed back through the tunnel from the FA to HA, and then on to their destination.  \\n      Although Mobile IP preserves subnet connectivity for roaming clients, it can result in sub-optimal routing and longer roaming delay. As noted above, the wireless client must first regain over the air connectivity with its new FA before the Agent Discovery Phase is launched. This can result in considerable reconnection time which increases latency. Furthermore, the registration process involves wire line and wireless communication. The amount of packet loss and the significant delay introduced during these procedures make the method unsuitable for many WLAN application, such as VoIP over 802.11 or streaming over 802.11.  \\n      Notwithstanding these advances, as new applications emerge and are implemented, such as VoIP over 802.11, changes to the WLAN deployment are required. For example, coverage-oriented deployments must move to capacity-oriented deployments characterized by low user to AP ratio and more APs in a given coverage area. The move to capacity-oriented deployments emphasizes the need for techniques that allow clients to roam across subnets and roaming domains.  \\n      There is a need for layer 3 roaming techniques which can allow a client to roam across different IP subnets of a WLAN while preserving the client&#39;s original IP-connection and original IP address. It would be desirable if such techniques could allow the client to perform a seamless and smooth L3 handoff between APs of different IP subnets, while maintaining an active session without losing IP connectivity. It would be desirable if such techniques could enable routing of IP data to/from the client&#39;s current foreign subnet to their original IP address and home subnet even though the client is currently in a foreign subnet. It would also be desirable to provide layer 3 roaming techniques which can eliminate the need to re-key during re-authentication. Other desirable features and characteristics of the present invention will become apparent from the subsequent detailed description and the appended claims, taken in conjunction with the accompanying drawings and the foregoing technical field and background.  \\n     SUMMARY OF THE INVENTION  \\n      According to one embodiment, a wireless router is provided which is configured to support a first subnet and a second subnet. The wireless router comprises a plurality of wireless switches. Each wireless switch comprises a plurality of access ports. The first subnet comprises a group of the access ports belonging to the first virtual wireless switch, and the second subnet comprises a second group of the access ports belonging to the second virtual wireless switch, The wireless router is configured to support layer 3 mobility when a client, having a client IP address from within the first subnet, roams from the first subnet to the second subnet, from the first to the second virtual wireless switch. The wireless router stores registration information associated with the client to allow a client to roam between the first subnet and the second subnet while keeping the client IP address. The wireless router uses the registration information to send packets to the client when the client has roamed to the second subnet.  \\n      According to one implementation of this embodiment, techniques are provided for allowing a client to layer 3 roam within a single wireless router. The client is initially associated with a home virtual wireless switch module and has a client IP address from within a first subnet. The client roams from the first subnet to a second subnet supported by a visited virtual wireless switch module configured to support a second subnet. A connection or interprocess communication can be used to communicate between the home virtual wireless switch module and the visited virtual wireless switch module.  \\n      Registration information associated with each client in the first subnet is sent to the home virtual wireless switch module. A first active client list is created using the registration information from each client in the first subnet. Registration information associated with each client in the second subnet is sent to the visited virtual wireless switch module. A second active client list is created using the registration information from each client in the second subnet. A master active client list is generated using the first active client list and the second active client list. A copy of the master active client list is sent to each wireless switch in the wireless local area network.  \\n      Techniques are provided for maintaining the client IP address on the client when the client roams from the first subnet to the second subnet. For example, the client IP address can be maintained at the client when the client roams from the home virtual wireless switch module to the visited virtual wireless switch module by 802.11 authenticating the client with the visited virtual wireless switch module, 802.11 associating the client with the visited virtual wireless switch module, 802.1x authenticating the client with the visited virtual wireless switch module, issuing a Dynamic Host Configuration Protocol (DHCP) request from the client to the visited virtual wireless switch module, relaying the DHCP request from the visited virtual wireless switch module to the home virtual wireless switch module through the interprocess communication, passing the DHCP request from the home virtual wireless switch module to a first Dynamic Host Configuration Protocol (DHCP) server, wherein the first DHCP server re-assigns the client IP address to the client, and forwarding a DHCP response from the first DHCP server to the visited virtual wireless switch module and the client.  \\n      When the client roams from the home virtual wireless switch module to the visited virtual wireless switch module, The visited virtual wireless switch module can use the master active client list to determine that the client IP address belongs to the first subnet and that the client was originally associated with the home virtual wireless switch module. The visited virtual wireless switch module can obtain the client IP address from a portion of the master active client list including the registration information associated with the home virtual wireless switch module. The record of the client can be used to obtain the home virtual wireless switch module from the MAC address of the client. When an IP packet from the client is received at the visited virtual wireless switch module, the IP packet can be forwarded through the interprocess communication to the home virtual wireless switch module. When a second IP packet for the client is received at the home virtual wireless switch module, the second IP packet can be forwarded through the interprocess communication to the visited virtual wireless switch module. The second IP packet can then be sent from the visited virtual wireless switch module to the client.  \\n      According to one implementation, the wireless router can be implemented in wireless local area network in which the wireless router is coupled to another core L3 router. The wireless router comprises a home virtual wireless switch module configured to support a first subnet, and a visited virtual wireless switch module configured to support a second subnet. A interprocess communication coupled the home virtual wireless switch module to the visited virtual wireless switch module. This configuration of the wireless switch can allow a client, initially associated with the home virtual wireless switch module and having a client IP address from within the first subnet, to maintain its client IP address when the client roams from the first subnet to the second subnet.  \\n      When the client roams from the home virtual wireless switch module to the visited virtual wireless switch module, the client 802.11 authenticates with the visited virtual wireless switch module, 802.11 associates with the visited virtual wireless switch module, 802.1x authenticates with the visited virtual wireless switch module and issues a Dynamic Host Configuration Protocol (DHCP) request. The visited virtual wireless switch module relays the DHCP request to the home virtual wireless switch module through the interprocess communication. A first Dynamic Host Configuration Protocol (DHCP) server is coupled to the wireless switch. The home virtual wireless switch module passes the DHCP request to the first DHCP server. The first DHCP server then re-assigns the client IP address to the client and forwards a DHCP response to the visited virtual wireless switch module and the client.  \\n      The home virtual wireless switch module creates a first active client list using the registration information from each client in the first subnet, and the visited virtual wireless switch module creates a second active client list using the registration information from each client in the second subnet. The virtual registration server module, coupled to home virtual wireless switch module and the visited virtual wireless switch module, can then create a master active client list using the first and second active client lists, and send a copy of the active client list to each wireless switch. The master active client list comprises a record for the client which can include, for example, a MAC address of the client, the client IP address of the client, the home virtual wireless switch module of the client, the visited virtual wireless switch module of the client, inactivity timers for the home virtual wireless switch module and the visited virtual wireless switch module.  \\n      The visited virtual wireless switch module uses the master active client list to determine that the client IP address belongs to the first subnet and that the client was originally associated with the home virtual wireless switch module. The visited virtual wireless switch module uses the master active client list to obtain the client IP address from registration information associated with the client.  \\n      The visited virtual wireless switch module is configured to receive an IP packet from the client and forwards the IP packet through the interprocess communication to the home virtual wireless switch module, and the home virtual wireless switch module is configured to receive a second IP packet for the client and forward the second IP packet through the interprocess communication to the visited virtual wireless switch module. The visited virtual wireless switch module can then sends the second IP packet to the client. \\n    \\n    \\n     BRIEF DESCRIPTION OF THE DRAWINGS  \\n      The present invention will be described in conjunction with the following drawing figures, wherein like numerals denote like elements, and  \\n       FIG. 1  is a block diagram of a conventional wireless local area network (WLAN) which illustrates the concept of layer 2 roaming and the concept of layer 3 roaming in the WLAN;  \\n       FIG. 2  is a block diagram of a WLAN according to one exemplary embodiment which implements a registration server and a plurality of wireless switches;  \\n       FIG. 3  is a block diagram of a registration server according to one exemplary embodiment;  \\n       FIG. 4  is a block diagram of a wireless switch according to one exemplary embodiment;  \\n       FIG. 5  is a flow chart showing an exemplary method creating a mesh network of wireless switches according to one exemplary embodiment;  \\n       FIG. 6  is a flow chart showing an exemplary method for providing an active client list to a plurality of wireless switches according to one exemplary embodiment;  \\n       FIG. 7  is a block diagram of a WLAN according to one exemplary embodiment which implements a registration server and a home wireless switch supporting a first subnet and a visited wireless switch supporting a second subnet;  \\n       FIG. 8  is a flow chart showing an exemplary method for allowing a client, initially associated with a home wireless switch and having a client IP address from within a first subnet, to roam from the home wireless switch to a visited wireless switch configured to support a second subnet according to one exemplary embodiment;  \\n       FIG. 9  is a flow chart showing exemplary message exchanges between the home wireless switch which supports a first subnet and the visited wireless switch which supports a second subnet to allow the client to maintain a client IP address when the client roams to the second subnet according to one exemplary embodiment;  \\n       FIG. 10  is a block diagram of a WLAN according to one exemplary embodiment which implements a wireless router supporting a first subnet and a second subnet; and  \\n       FIG. 11  is a flow chart showing an exemplary method layer 3 roaming inside a wireless router according to one exemplary embodiment. \\n    \\n    \\n     DETAILED DESCRIPTION OF THE INVENTION  \\n      The following detailed description is merely exemplary in nature and is not intended to limit the invention or the application and uses of the invention. Furthermore, there is no intention to be bound by any expressed or implied theory presented in the preceding technical field, background, brief summary or the following detailed description. As used herein, the word “exemplary” means “serving as an example, instance, or illustration.” Any embodiment described herein as “exemplary” is not necessarily to be construed as preferred or advantageous over other embodiments. All of the embodiments described in this Detailed Description are exemplary embodiments provided to enable persons skilled in the art to make or use the invention and not to limit the scope of the invention which is defined by the claims.  \\n      Embodiments of the present invention provide methods and apparatus that allow for a client to roam from a first subnet to a second subnet while keeping the same IP address used in the home subnet. As used herein, a “client” is a mobile device in a WLAN. The term “mobile device” can generally refer to a wireless communication device or other hardware with which an access network communicates. At any given time a mobile device may be mobile or stationary and can include devices that communicate through a wireless channel or through a wired channel. A mobile device may further be any of a number of types of mobile computing devices including but not limited to a laptop computer, a PC card, compact flash, external or internal modem, wireless or wireline phone, personal digital assistant (PDA) or mobile telephone handset.  \\n       FIG. 2  is a block diagram of a WLAN according to one exemplary embodiment which implements a registration server  130  and wireless switches  112 ,  122 ,  132 ,  142 . As used herein, the term “WLAN” refers to a network in which a mobile user can connect to a local area network (LAN) through a wireless (radio) connection. The IEEE 802.11 standard specifies some features of exemplary wireless LANs.  \\n      As used herein, the term “packet” refers to a unit of data that is routed between an origin and a destination on a packet-switched network such as the Internet. When any file is sent from one place to another on the Internet, the Transmission Control Protocol (TCP) layer divides the file into “chunks” of an efficient size for routing. Each of these packets is separately numbered and includes the Internet address of the destination. The individual packets for a given file may travel different routes through the Internet. When they have all arrived, they are reassembled into the original file by the TCP layer at the receiving end. In the context of the User Datagram Protocol (UDP), it should be appreciated that the term “datagram” has a similar meaning to the term “packet.” \\n      As used herein, the term “switch” refers to a device that channels incoming data from any of multiple input ports to the specific output port that will take the data toward its intended destination. A switch typically performs the data-link or layer 2 functions and determines, from an IP address in each packet, which output port to use for the next part of its trip to the intended destination. The destination address generally requires a look-up in a routing table by a device known as a router. In some embodiments, the switch can function as an IP switch which may also perform network or layer 3 routing functions.  \\n      The registration server  130  and wireless switches  112 ,  122 ,  132 ,  142  can be coupled to each other via IP sockets or tunnels which the wireless switches  112 ,  122 ,  132 ,  142  create to the registration server  130 . The wireless switches  112 ,  122 ,  132 ,  142  are coupled to each other by a mesh network of IP sockets or tunnels. As used herein, the term “tunneling” refers to the process of allowing two disparate networks to connect directly to one another when they normally would not or when they are physically disjointed. Tunneling is synonymous with encapsulation, and is generally done by encapsulating private network data and protocol information within public network transmission units so that the private network protocol information appears to the public network as data. A tunnel requires an entry point and an exit point. The entry point encapsulates the tunneled packets within another IP header. The new IP header might include some other parameters, but the basic function of the encapsulation header is to direct the packet to the tunnel endpoint. A packet received by the tunnel endpoint is stripped of the encapsulation header and forwarded to the client.  \\n      The registration server  130  is a network entity that can be implemented as dedicated hardware on an external high availability platform. For example, the registration server  130  might be implemented in a blade server. Alternatively, the registration server  130  can be implemented as a module hosted on two wireless switches.  \\n      The registration server  130  is used for registering wireless switches in the WLAN when the wireless switches join the WLAN. The registration server  130  has a first Internet Protocol (IP) address which is configured on every wireless switch in the WLAN. As used herein, the term “Internet Protocol (IP) address” refers to a layer 3 address. Each communication from a user on the Internet carries an IP address of the source and destination networks and the particular machine within the network associated with the user or host computer at each end. In one implementation, the IP address is a 32-bit address comprising one part identifies the network with a network number and another part which identifies the specific machine or host within the network with a host number. Some of the bits in the machine or host part of the address can be used to identify a specific subnet. In this case, the IP address then contains three parts: the network number, the subnet number, and the machine number.  \\n      Each of the wireless switches 112 ,  122 ,  132 ,  142  has configuration information associated with it which can include, for example, an IP address and a list of subnets (IP domains) which the particular wireless switch supports. As used herein, the term sub-network or “subnet” refers to an identifiably separate part of a network. Typically, a subnet may represent all the machines at one geographic location, in one building, or on the same wireless local area network (WLAN). One standard procedure for creating and identifying subnets is described in Internet Request for Comments (RFC)  950 .  \\n      Each of the wireless switches  112 ,  122 ,  132 ,  142  registers with the registration server  130  by communicating its configuration information to the registration server  130  and uses the IP address of the registration server  130  to create or open a first IP socket (tunnel) to the registration server  130 . The wireless switches  112 ,  122 ,  132 ,  142  can periodically send update messages to each other. These update messages can include, for example, changes to the configuration information associated with each wireless switch.  \\n      The registration server  130  can use the configuration information to create an AWSL which includes a listing of each of the wireless switches  112 ,  122 ,  132 ,  142  in the WLAN. The registration server  130  sends the AWSL to each of the wireless switches  112 ,  122 ,  132 ,  142 . Each of the wireless switches  112 ,  122 ,  132 ,  142  uses the AWSL to open a UDP/IP socket to each of the other wireless switches  112 ,  122 ,  132 ,  142 . Once all of the wireless switches  112 ,  122 ,  132 ,  142  are coupled together via UDP/IP sockets and are coupled to the registration server  130  via IP sockets, the mesh network is complete. This mesh network changes dynamically as new switches are added (e.g., register with the registration server  130 ) or removed from the WLAN.  \\n      In one implementation, each of the wireless switches  112 ,  122 ,  132 ,  142  can send configuration information to each of the other wireless switches  112 ,  122 ,  132 ,  142 . Alternatively, the registration server  130  can send the configuration information for each of the wireless switches  112 ,  122 ,  132 ,  142  to each of the other wireless switches  112 ,  122 ,  132 ,  142 .  \\n      The wireless switches  112 ,  122 ,  132 ,  142  can also periodically send update messages to each other. If a certain amount of time passes and one of the wireless switches do not send update messages, then the other wireless switches can assume that wireless switch is no longer in the WLAN.  \\n      Typically, any communications between the registration server  130  and the wireless switches  112 ,  122 ,  132 ,  142  over the IP sockets are unencrypted. However, in another embodiment, if security is a concern, the IP sockets (tunnels) can go over a security protocol, such as Internet Protocol Security (IPSec), and the communications can be encrypted using IPSec. “Internet Protocol Security (IPSec)” refers to a framework for a set of security protocols at the network or packet processing layer of network communication. IPsec can allow security arrangements to be handled without requiring changes to individual user computers. IPsec provides two choices of security service: Authentication Header (AH), which essentially allows authentication of the sender of data, and Encapsulating Security Payload (ESP), which supports both authentication of the sender and encryption of data as well. The specific information associated with each of these services is inserted into the packet in a header that follows the IP packet header. Separate key protocols can be selected, such as the ISAKMP/Oakley protocol. As will be described below, the client  202  can use IPSec terminated on the home wireless switch  212 .  \\n      In another embodiment, it may be desirable to deploy redundant registration servers. When multiple registration servers  130  are implemented the configuration of the active or master registration server  130  can be synchronized with the configuration of a standby or slave registration server. That way, in the event the active or master registration server  130  fails, the standby or slave registration server can take over since it includes the same information (e.g., wireless switch list, active client list) as the active or master registration server  130 .  \\n      In addition to the functionality related to the L3 roaming in WLANs, other functionality can be implemented in the registration server  130  that is typically implemented in other external servers. For example, the registration server  130  can host wireless intrusion detection system (WIDS) functionality, location server functionality, billing functionality, etc. Because the registration server  130  has information about each wireless switch (e.g., wireless switch list) and each client (e.g., active client list) in the WLAN, the registration server  130  can leverage this information with other functions provided by the additional functionality.  \\n       FIG. 3  is a block diagram of a registration server  130  according to one exemplary embodiment. The registration server  130  can include, for example, a transceiver  131  which includes a transmitter  132  and a receiver  134 , a database  133 , a processor  135  and a number of ports  137 .  \\n      The receiver  134  of the registration server  130  can communicate the IP address of the registration server  130  to each of the wireless switches. Each of the wireless switches can use the IP address to open an IP socket to one of the ports. The receiver  134  receives configuration information from each wireless switch that includes attributes and parameters associated with each of the wireless switches  112 ,  122 ,  132 ,  142 . This configuration information is communicated over a set of first IP sockets or tunnels between each of the wireless switches  112 ,  122 ,  132 ,  142  and the registration server  130 . The configuration information for each wireless switch  112 ,  122 ,  132 ,  142  comprises a switch IP address and a list of subnets (IP domains) which the wireless switch supports. The processor  135  registers each of the wireless switches  112 ,  122 ,  132 ,  142  with the registration server  130  using the configuration information received from the wireless switches  112 ,  122 ,  132 ,  142  during registration and, optionally, updates received from the wireless switches  112 ,  122 ,  132 ,  142 . The processor  135  can use the configuration information communicated received from the wireless switches  112 ,  122 ,  132 ,  142  to create an active wireless switch list (AWSL). The AWSL includes a listing of each of the wireless the switches in the WLAN. The transmitter  132  subsystem can communicate the configuration information for each of the wireless switches and the AWSL to each of the wireless switches. Each of the wireless switches can use the configuration information and the AWSL to open a UDP/IP socket to each of the other wireless switches. The database  135  can store the configuration information for each of the plurality of wireless switches and the AWSL.  \\n       FIG. 4  is a block diagram of a wireless switch  140  according to one exemplary embodiment. The wireless switch  140  could be implemented as any or all of the wireless switches  112 ,  122 ,  132 ,  142  described above. The wireless switch  140  can include, for example, a transceiver  141  which includes a transmitter  142  and a receiver  144 , a database  143 , a processor  145  and a number of ports  147 .  \\n      The transmitter  142  can communicate configuration information about the wireless switch  140  to a registration server over an IP socket to the registration server  130 . The transmitter  142  can also send configuration information for the wireless switch  140  to each of the other wireless switches.  \\n      The receiver  144  can receive configuration information for each of the other wireless switches and a copy of the AWSL which includes a listing of each of the other wireless switches in the WLAN.  \\n      The processor  145  can use the configuration information and the AWSL to open a UDP/IP sockets from the ports  147  to each of the other wireless switches.  \\n      The transmitter  142  can send the update messages for the wireless switch to each of the other wireless switches. The receiver  144  can also receive update messages from each of the other wireless switches. These update messages comprise changes to configuration information for each of the other wireless switches.  \\n       FIG. 5  is a flow chart showing an exemplary method creating a mesh network of wireless switches in a WLAN comprising a wireless switches  112 ,  122 ,  132 ,  142  and a registration server  130 . An IP address of the registration server  130  can be configured on each of the wireless switches  112 ,  122 ,  132 ,  142 .  \\n      At step  502 , the IP address of the registration server  130  can be used to create or open an IP socket from each of the wireless switches  112 ,  122 ,  132 ,  142  to the registration server  130 . Each of the wireless switches 112 ,  122 ,  132 ,  142  can register with the registration server  130  by communicating configuration information about each of the wireless switches  112 ,  122 ,  132 ,  142  to the registration server  130 . In one implementation, the configuration information for each switch  112 ,  122 ,  132 ,  142  comprises a switch IP address and a list of subnets the switch supports.  \\n      At step  504 , the registration server  130  can use the configuration information to create an active wireless switch list (AWSL) which includes a listing of each of the wireless switches  112 ,  122 ,  132 ,  142  in the WLAN.  \\n      At step  506 , the AWSL and the configuration information for each of the wireless switches  112 ,  122 ,  132 ,  142  can then be communicated to each of the wireless switches 112 ,  122 ,  132 ,  142 .  \\n      At step  508 , each of the wireless switches  112 ,  122 ,  132 ,  142  can use the configuration information and the AWSL to open a UDP/IP socket to each of the other wireless switches  112 ,  122 ,  132 ,  142 . Each wireless switch is then connected to each of the other wireless switches  112 ,  122 ,  132 ,  142  and a mesh network of wireless switches  112 ,  122 ,  132 ,  142  is created.  \\n      In other implementations, each of the wireless switches  112 ,  122 ,  132 ,  142  can send configuration information to each of the other wireless switches  112 ,  122 ,  132 ,  142 . Alternatively, the registration server  130  can send the configuration information and the AWSL for each of the wireless switches  112 ,  122 ,  132 ,  142  to each of the other wireless switches  112 ,  122 ,  132 ,  142 . Each wireless switch  112 ,  122 ,  132 ,  142  can also send update messages to each of the other wireless switches  112 ,  122 ,  132 ,  142 . These update messages can include, for example, changes to configuration information for each wireless switch  112 ,  122 ,  132 ,  142 .  \\n       FIG. 6  is a flow chart showing an exemplary method for providing an active client list (ACL) to a plurality of wireless switches  112 ,  122 ,  132 ,  142  according to one exemplary embodiment. The wireless switches  112 ,  122 ,  132 ,  142  can be located, for instance, in a WLAN such as the WLAN of  FIG. 2  comprising a registration server  130  and a plurality of active clients (not shown) supported by the wireless switches  112 ,  122 ,  132 ,  142 .  \\n      At step  602 , registration information associated with each of the active clients is communicated to the wireless switches  112 ,  122 ,  132 ,  142  that support those active clients. At step  604 , the registration information associated with each of the active clients is communicated from the wireless switches  112 ,  122 ,  132 ,  142 , over an IP tunnel, to the registration server  130 . At step  606 , an active client list can be created using the registration information for each active client. The active client list comprises a record for each active client in the WLAN. The record of each client comprises a MAC address of the client, a client IP address of the client, a home switch of the client, a visited switch of the client, inactivity timers for the home switch and the visited switch and location information. At step  608 , the active client list and the registration information for each active client is communicated to each wireless switch  112 ,  122 ,  132 ,  142 . At step  610 , registration information updates are communicated from each wireless switch  112 ,  122 ,  132 ,  142  to the registration server  130 . The registration server  130  can use the registration information updates received from the wireless switches  112 ,  122 ,  132 ,  142  to update the active client list. At step  612 , the registration information updates are communicated to each of the other wireless switches  112 ,  122 ,  132 ,  142  in the WLAN. Alternatively, the registration server  130  can communicate an updated active client list including the registration information updates to the active client list to each wireless switch  112 ,  122 ,  132 ,  142 .  \\n      Referring again to  FIG. 3 , the registration server  130  can include ports  137 , a transceiver  131  comprising a transmitter  132  and a receiver  134 , a processor  135 , a database  133 . Selected ports couple the registration server  130  to the wireless switches 112 ,  122 ,  132 ,  142  via IP sockets. The receiver  134  can receive registration information for each active client from the wireless switch that supports each active client. The processor  135  can create an ACL using the registration information for each active client. The database  135  can store the ACL and registration information for each active client, and the transmitter  132  can communicate the ACL and registration information for each active client to each wireless switch. In one implementation, the wireless switches send registration information updates. The receiver  134  can receive registration information updates from the wireless switches, and the processor  135  can use the registration information updates to create an updated ACL. The transmitter  132  can then send the registration information updates to each of the wireless switches. In another implementation, the wireless switches send registration information updates to the receiver  134 , and the processor  135  can use the registration information updates to update the ACL. The transmitter  132  can send the registration information updates to the ACL to each wireless switch  112 ,  122 ,  132 ,  142  as the registration information updates are received from the wireless switches  112 ,  122 ,  132 ,  142 .  \\n      Referring again to  FIG. 4 , each of the wireless switches  112 ,  122 ,  132 ,  142  can include, for example, a number of ports  147 , a transceiver  141  including a transmitter  142  and a receiver  144 , a processor  145  and a database  143 . The receiver  144  can receive registration information from each of the active clients the wireless switch supports. The ports  247  couple the wireless switches  112 ,  122 ,  132 ,  142  to the registration server  130  via IP sockets. The transmitter  142  transmits the registration information to the registration server  130 . The receiver  144  can receive the ACL from the registration server  130 . The ACL comprises a record for each of the active clients in the WLAN. The receiver  144  can also receive registration information updates from each of the active clients the wireless switch supports, and the transmitter  142  can send the registration information updates to the registration server  130 . The transmitter  142  can also send the registration information updates to each of the other wireless switches in the WLAN. The receiver  144  can receive an updated ACL from the registration server  130  which includes the registration information updates received from each of the wireless switches.  \\n      If a client moves away from its home switch, the home switch needs a way to determine that it no longer needs to support that client. Otherwise, the client would tunnel to their home switch indefinitely regardless of the client&#39;s location. Techniques are needed to allow the home switch to determine that it is no longer the best switch for that particular client. In another embodiment, the registration server  230  or the switches can monitor the inactivity timers. If the inactivity timers of the client  202  indicate that the client  202  is inactive on its home switch (and the visited switch) for a given period of time, then the registration server  230  forces the client  202  to reauthenticate, reassociate and get a new client IP address on a new wireless switch. This allows the WLAN to avoid transmitting unnecessary overhead and cleans up unnecessary traffic in the tunnels between switches. [ \\n       FIG. 7  is a block diagram of a WLAN according to one exemplary embodiment which implements a registration server  230  and a home wireless switch  212  supporting a first subnet  210  and a visited wireless switch  222  supporting a second subnet  220 . Although  FIG. 7  shows two wireless switches  212 ,  222  and two subnets  210 ,  220 , it should be appreciated that more than two switches and subnets can be implemented in the WLAN. It should also be appreciated that while  FIG. 7  shows a single client  202 , more than one client is typically present in the WLAN. Typically, in a given WLAN there are a number of active clients. In this example, the first subnet  210  would typically support a group of the active clients having client IP addresses within the first subnet  210 , and the second subnet  220  would typically supports another group of the active clients having client IP addresses within the second subnet  220 . In addition, in  FIG. 7 , each subnet  210 ,  220  is shown as comprising three access ports (APs)  215 - 217  and  225 - 227 , however, any number of APs could be implemented within a subnet.  \\n      As used herein, the terms “access point (AP)” or “access port (AP)” refer to a station that transmits and receives data (sometimes referred to as a transceiver). Throughout this document the terms “access point (AP)” or “access port (AP)” can be used interchangeably. An access point connects users to other users within the network and also can serve as the point of interconnection between the WLAN and a fixed wire network. Each access point can serve multiple users within a defined network area. As a client moves beyond the range of one access point, the client can be automatically handed over to the next AP. A WLAN may only require a single access point. The number of APs in a given subnet generally increases with the number of network users and the physical size of the network.  \\n      The home wireless switch  212  supports a first VLAN comprising a first subnet  210  which includes access ports (AP 1 )  215 , (AP 2 )  216 , and (AP 3 )  217 . All clients on the first VLAN have IP addresses in the first subnet  210 . Tunnels couple the access ports (AP  1 )  215 , (AP 2 )  216 , and (AP 3 )  217  to the home wireless switch  212 . The home wireless switch  212  has first configuration information comprising a first IP address and a list of first subnets (IP domains) supported by the home wireless switch  212 . The home wireless switch  212  registers with the registration server  230  by communicating the first configuration information to the registration server  230  over the first IP socket  214 . The client  202  is initially associated with first subnet  210  communicating with the home wireless switch  212  through the AP 3   217 . The client  202  has a client IP address from within the first subnet  210 . The client  202  eventually roams into the second subnet  220  where it communicates with the visited virtual wireless switch  222  through the access port (AP 4 )  225 .  \\n      Similarly, the visited wireless switch  222  supports a second VLAN comprising a second subnet  220  which includes access ports (AP 4 )  225 , (AP 5 )  226 , and (AP 6 )  227 . All clients on the second VLAN have IP addresses in the second subnet  220 . Tunnels couple the access ports (AP 4 )  225 , (AP 5 )  226 , and (AP 6 )  227 to the visited wireless switch  222 . The visited wireless switch  222  has second configuration information comprising a second IP address and a list of second subnets (IP domains) supported by the visited wireless switch  222 . The visited wireless switch  222  registers with the registration server  230  by communicating the second configuration information to the registration server  230  over the second IP socket  224 .  \\n      Because the IP address of the registration server is configured on each of the wireless switches, each of the wireless switches can use the IP address during registration to open an IP socket to the registration server. In this example, a first IP socket  214  can be provided which couples the home wireless switch  212  and the registration server  230 , and a second IP socket  224  between the visited wireless switch  222  and the registration server  230 .  \\n      A database  137  in the registration server  230  stores the associated configuration information for each of the plurality of wireless switches.  \\n      Each of the wireless switches also communicates registration information for each active client to the registration server  230 . The registration server  230  can use the registration information to create an active client list (ACL). The active client list comprises a record for each active client  202  in the WLAN. The record of each client  202  comprises a number of attributes, for instance, a MAC address of the client, a client IP address of the client, a home switch of the client, a visited switch of the client, inactivity timers for the home switch and the visited switch and location information. The registration server  230  can send a copy of the active client list (or a portion of the active client list) to each wireless switch in the WLAN.  \\n      In one embodiment, the registered wireless switches can periodically send updates regarding registration information for each active client to the registration server  230 . The registration server  230  can use these updates to create an updated active client list. Whenever the registration server  230  receives updated registration information (or new registration information from a new switch joining the network), the registration server  230  can then send the updates of the active client list to each wireless switch as the updates are received from the wireless switches.  \\n      As will be described below, when the client roams from its original home subnet to a visited subnet supported by a visited wireless switch, the active client list can be used by each of the wireless switches to allow a client to keep its original TCP/IP or UDP/IP connection and its original client IP address assigned by its home wireless switch.  \\n      The active client list includes a record for the client  202  which is based on the first configuration information. This record of comprises a MAC address of the client  202 , the client IP address of the client, the home wireless switch  212  of the client, the visited wireless switch  222  of the client  202 , inactivity timers for the home wireless switch  212  and the visited wireless switch  222 . This record can be periodically updated using updates received from the wireless switch. A database  133  in the registration server  230  can store the first configuration information, second configuration information, and the active client list.  \\n      Because the home wireless switch  212  and the visited wireless switch  222  are registered on the registration server  230 , a UDP/IP tunnel  215  can be created which couples the home wireless switch  212  and the visited wireless switch  222 . Each of the wireless switches can use configuration information from the wireless switch list to open a UDP/IP tunnel or socket to the other wireless switch. As will be explained in greater detail below, this tunnel allows the client  202  to maintain the client&#39;s IP address from its home wireless switch  212  when the client  202  roams from the home wireless switch  212  and the visited wireless switch  222 .  \\n      A protocol can be implemented which allows a DHCP server  211  to assign the original client IP address to the client even when the client  202  roams from the home wireless switch  212  to the visited wireless switch  222 . As used herein, the “Dynamic Host Configuration Protocol (DHCP)” refers to a protocol for assigning dynamic IP addresses to devices on a network. DHCP typically sends a new IP address when a computer is plugged into a different place in the network. This protocol allows a device to have a different IP address every time it connects to the network, and the device&#39;s IP address can even change while it is still connected. DHCP can also support a mix of static and dynamic IP addresses. DHCP uses the concept of a “lease” or amount of time that a given IP address will be valid for a computer. Using very short leases, DHCP can dynamically reconfigure networks in which there are more computers than there are available IP addresses.  \\n      When the client  202  begins to roam to the visited wireless switch  222 , as the client  202  approaches the visited wireless switch  222 , the client  202  hears a new beacon sent out by an access port (AP)  225  connected to the visited wireless switch  222 . The new beacon has a new BSSID (MAC address) different from the one used by access port (AP 3 )  217  connected to the home wireless switch  212 . As such, the client  202  802.11 authenticates with the visited wireless switch  222 , 802.11 associates with the visited wireless switch  222 , 802.1x authenticates with the visited wireless switch  222  and issues a Dynamic Host Configuration Protocol (DHCP) request. Once the client  202  802.11 authenticates and 802.11 associates with the visited wireless switch  222 , the client  202  can continue its existing TCP/IP connection.  \\n      To allow for layer 3 roaming between the home wireless switch  212  and the visited wireless switch  222 , it would be desirable to send a Dynamic Host Configuration Protocol (DHCP) request to the client&#39;s home wireless switch  212  since this can allow the client  202  to keep its original client IP address. Because the ACL is sent to each wireless switch, each switch has information about all active clients in the network. The visited wireless switch  222  can obtain the client IP address from the registration information that was sent to the registration server  230  when the home wireless switch  212  registered with the registration server  230 . For example, the visited wireless switch  222  can search the record of the client  202  to get the MAC address of the client  202 . The visited wireless switch  222  can use the MAC address of the client  202  to determine that the client IP address belongs to the first subnet  210  and that the client  202  was originally associated with the home wireless switch  212 . Thus, the visited wireless switch  222  knows that the client  202  was initially associated with the home wireless switch  212  and that it had a client IP address belonging to the first subnet  210 .  \\n      The visited wireless switch  222  can then relays the DHCP request to the home wireless switch  212  through the tunnel  215 , and the home wireless switch  212  passes the DHCP request to the DHCP server  211 . The DHCP server  211  re-assigns the same original client IP address to the client  202 . Because the client  202  maintains its original client IP address from the home switch, the client  202  does not need to re-establish its connection. This can prevent the session from dropping. The home wireless switch  212  forwards a Dynamic Host Configuration Protocol (DHCP) response to the visited wireless switch  222  and the client  202 .  \\n      When the client  202  sends IP packets to the network, the IP packets will go to the visited wireless switch  222 . The visited wireless switch  222  can then forward any IP packets it receives through the tunnel  215  to the home wireless switch  212  which can forward the IP packets to a router. Likewise, for outbound packets destined to the client IP address, the home wireless switch  212  can forward the outbound IP packets it receives to the client  202  through the visited wireless switch  222 .  \\n      In one embodiment, if the client  202  comprises a WPA2 client, then the WPA2 client  202  is pre-authenticated with the visited wireless switch  222  to achieve layer 3 mobility with low latency. If the client uses IPSec, terminated on the home switch and no 802.11 encryptions, then the client could 802.11 re-authenticate and search the ACL to get the home wireless switch  212  from client&#39;s MAC address. This can allow all packets from the client  202  to be forwarded to the home wireless switch  212 . Otherwise the client will 802.11 re-authenticate, go through dot1.x authentication, four way and two handshake to generate new transient keys and then continue with existing TCP or UDP sessions. The dot1.x authentication involves a RADIUS server and the latency can depend on type of the inter-authentication method (PEAP, TTLS, TLS).  \\n       FIG. 8  is a flow chart showing an exemplary method for allowing a client  202 , initially associated with a home wireless switch  212  and having a client IP address from within a first subnet  210 , to roam from the home wireless switch  212  to a visited wireless switch  222  configured to support a second subnet  220  according to one exemplary embodiment. This method can be used, for example, in a WLAN to allow a client  202  to keep its client IP address and maintain IP connectivity while roaming between the first subnet  210  and the second subnet  220 .  \\n      At step  802 , a tunnel is created or opened between the home wireless switch  212  to the visited wireless switch  222  by using the AWSL and configuration information for the home wireless switch  212  to the visited wireless switch  222 . At step  804 , the client  202  roams from the home wireless switch  212  to the visited wireless switch  222 . The client  202  can keep its original client IP address and maintain IP connectivity while roaming from the first subnet  210  to the second subnet  220  using techniques which will now be described with reference to  FIG. 9 .  \\n       FIG. 9  is a flow chart showing exemplary message exchanges between the home wireless switch  212 , which supports a first subnet  210 , and the visited wireless switch  222 , which supports a second subnet  220 , to allow the client  202  to maintain its original client IP address when the client  202  roams to the second subnet  220 . At step  902 , the client  202  is 802.11 authenticated and associated with the visited wireless switch  222 , and at step  904 , 802.1x authenticated with the visited wireless switch  222 . At step  906 , the client  202  issues a Dynamic Host Configuration Protocol (DHCP) request which is relayed, at step  908 , from the visited wireless switch  222  to the home wireless switch  212  through the tunnel  215 . At step  910 , the DHCP request can then be passed from the home wireless switch  212  to the DHCP server  211 . At step  912 , the DHCP server  211  re-assigns the client IP address to the client, and at step  914 , a Dynamic Host Configuration Protocol (DHCP) response can be forwarded from the home wireless switch  212  to the visited wireless switch  222  and the client  202 .  \\n      Referring again to  FIG. 8 , at step  806 , the active client list can be used to determine that the client IP address belongs to the first subnet  210  and that the client  202  was originally associated with the home wireless switch  212 . The visited wireless switch  222  can obtain the client IP address from the registration information sent to the visited wireless switch  222  by registration server  230  when the visited wireless switch  222  registered with the registration server  230 . At step  808 , any IP packet sent from the client  202  and received by the visited wireless switch  222  can be forwarded to the home wireless switch  212  through the tunnel  215 , and, at step  810 , any IP packet received by the home wireless switch  212  can be forwarded through the tunnel  215  to the visited wireless switch  222  which forwards the IP packet to the client  202 .  \\n      Referring again to  FIG. 3 , some of the ports  137  can couple the registration server  130  to the home wireless switch  212  and the visited wireless switch  222 . The receiver  134  can receive registration information associated with each client from each of the wireless switches. The processor  135  can create an active client list (ACL) using the registration information from each client. The transmitter  132  can send a copy of the ACL to each wireless switch in the WLAN.  \\n      Referring again to  FIG. 4 , one of the ports  147  of the home wireless switch  212  can be coupled to one of the ports  137  of the visited wireless switch  212  via the UDP/IP tunnel. To enable the client to maintain the client IP address when the client roams from the home wireless switch  212  and the visited wireless switch  222 , the client  202  802.11 authenticates with the visited wireless switch  222 , 802.11 associates with the visited wireless switch  222 , 802.1x authenticates with the visited wireless switch  222  and issues a Dynamic Host Configuration Protocol (DHCP) request to the visited wireless switch  222 . The receiver  144  of the home wireless switch can receive the DHCP request from the visited wireless switch  22  through the tunnel, and the transmitter  142  of the home wireless switch  212  can send the DHCP request to a Dynamic Host Configuration Protocol (DHCP) server  211  which re-assigns the client IP address to the client  202 . The transmitter  142  of the home wireless switch can send a DHCP response to the visited wireless switch and the client. The receiver  144  of the visited wireless switch  222  can receive the DHCP response from the home wireless switch  212 .  \\n      The receiver  144  of the visited wireless switch  222  can receive an active client list from the registration server  230 , and the processor  145  of the visited wireless switch  222  can use the active client list to determine that the client IP address belongs to the first subnet  210  and that the client  202  was originally associated with the home wireless switch  212 . The processor  145  of the visited wireless switch  222  obtains the client IP address from the registration information sent to the registration server  230  by the client  202  when the home wireless switch  212  registered with the registration server  230 . The processor  145  of the visited wireless switch  222  can search the record associated with the client  202  to get the home wireless switch  212  from the MAC address of the client  202 . The transmitter  142  of the visited wireless switch  222  can send registration information for each client in the second subnet to the registration server. The receiver  144  of the visited wireless switch  222  can receive, after the client  202  has roamed from the home wireless switch  212  to the visited wireless switch  222 , an IP packet sent from the client  202 . The transmitter  142  of the visited wireless switch  222  can then send the IP packet through the UDP/IP tunnel to the home wireless switch  212 . The receiver  144  of the home wireless switch can be coupled to the first port and can receive, after the client has roamed from the home wireless switch to the visited wireless switch, an IP packet sent from the visited wireless switch through the UDP/IP tunnel. This IP packet originates at the client.  \\n      Another one of the ports  147  can be coupled to the registration server. The receiver  144  of the home wireless switch can receive an active client list from the registration server. The processor  145  of the home wireless switch can use the active client list to determine that the client is now associated with the home wireless switch. The receiver  144  of the home wireless switch can receive a second IP packet addressed to the client. The transmitter  142  of the home wireless switch, which is coupled to the port, can send the second IP packet to the visited wireless switch through the UDP/IP tunnel. The visited wireless switch sends the second IP packet to the client. The receiver  144  of the visited wireless switch  222  can receive a second IP packet for the client  202  sent from the home wireless switch  212  through the UDP/IP tunnel.  \\n       FIG. 10  is a block diagram of a WLAN according to one exemplary embodiment which implements a wireless router  312  supporting a first subnet  310  and a second subnet  320 .  \\n      The WLAN comprises a core layer 3 router  318 , a wireless router  312 , a first subnet  310 , a second subnet  320 , a first DHCP server  311 , a second DHCP server  313 , and a client  302 .  \\n      The term “routing” typically refers to a function associated with the network layer or layer 3 of the Open Systems Interconnection (OSI) model. As used herein, a “router” can be either a device or software in a computer which determines the next network point to which a packet should be forwarded toward its destination. The router is connected to at least two networks and decides which way to send each information packet based on its current understanding of the state of the networks it is connected to. A router can be located at any gateway where one network meets another, including each point-of-presence on the Internet. In some embodiments, the router can be included as part of a network switch. A router may create or maintain a table of the available routes and their conditions and use this information along with distance and cost algorithms to determine the best route for a given packet. As used herein, a “gateway” refers to a network point that acts as an entrance to another network. On the Internet, a node or stopping point can be either a gateway node or a host (end-point) node. A gateway can be associated with both a router, which knows where to direct a given packet of data that arrives at the gateway, and a switch, which furnishes the actual path in and out of the gateway for a given packet.  \\n      The core router  318  comprises a plurality of layer 3 (L3) interfaces  352 ,  354 ,  356 ,  358  and is configured to decide which way to send each IP packet.  \\n      The wireless router  312  comprises a home virtual wireless switch, a visited virtual wireless switch module, a virtual registration server module, a layer 3 router module, a number of ports (not shown) and layer 3 (L3) interfaces  358 ,  360 ,  362 ,  364 ,  366 .  \\n      One port of the wireless router  312  is connected to the backbone or to a core router  318  by layer 3 (L3) interface  358  and has an IP address which belongs to a third IP subnet. Other ports are coupled to layer 3 (L3) interfaces  360 ,  362 ,  364  and  366 . Layer 3 (L3) interface  364  couples the wireless router  312  to the first subnet  310  which can be configured to support multiple clients connected to any one of three access ports AP 1   315 , AP 2   316 , AP 3   317 . Layer 3 (L3) interface  366  couples the wireless router  312  to the second subnet  320  which can be configured to support multiple clients connected to any one of three access ports AP 4   325 , AP 5   326 , AP 6   327 . Each of the access ports AP 1   315 , AP 2   316 , AP 3   317 , AP 4   325 , AP 5   326 , and AP 6   327  are coupled to a port of the wireless router  312 , respectively, via tunnels to layer 3 (L3) interfaces  364 ,  366 . Other ports couple the first DHCP server  311  and the second DHCP server  313  to the wireless router  312 .  \\n      Although the wireless router  312  is coupled to the core layer 3 router  318 , the wireless router  312  also includes a virtual layer 3 router and performs layer 3 functions. The wireless router  312  is configured as a layer 3 router having configured static routes or running a Routing Information Protocol (RIP) or Open Shortest Path First (OSFP) protocol. The “Routing Information Protocol (RIP)” is an internal gateway protocol for managing router information within a self-contained network or an interconnected group of such LANs. To enable network convergence using RIP, a gateway host with a router can periodically send its entire routing table (which lists all the other hosts it knows about) to its closest neighbor host. The neighbor host in turn will pass the information on to its next neighbor and so on until all hosts within the network have the same knowledge of routing paths. RIP typically uses a hop count as a way to determine network distance. Each host with a router in the network uses the routing table information to determine the next host to route a packet to for a specified destination. By contrast, the “Open Shortest Path First (OSFP) protocol” is an internal gateway protocol for which allows a host to immediately multicast changes in network information or routing tables to all other hosts in the network so that all hosts will have the same routing table information. The host using OSPF sends only the part that has changed. OSPF typically multicasts the updated information only when a change has taken place. OSPF bases its path descriptions on “link states” that take into account additional network information. OSPF also lets the user assign cost metrics to a given host router so that some paths are given preference. OSPF supports a variable network subnet mask so that a network can be subdivided. RIP is supported within OSPF for router-to-end station communication.  \\n      The wireless router  312  can route traffic between the virtual wireless switches and the other layer 3 interfaces. Since the wireless router  312  comprises multiple instances of wireless switches these switches can be referred to as virtual wireless switches. Each virtual wireless switch can switch wireless traffic on one IP subnet, such as home virtual wireless switch  310 , and can be brought up and configured on one Ethernet interface or multiple Ethernet interfaces configured as a Switch Virtual Interface (SVI). If the destination IP address of a packet does not belonging to the subnet controlled by the virtual wireless switch, such as home virtual wireless switch  310 , then a L3 routing component inside the wireless router  312  will properly route the packet to some other virtual switch, such as visited virtual wireless switch  320 , in the wireless router  312  or to the core L3 router  318 . Because the virtual wireless switch modules, internal L3 router  312  and the virtual registration server are running on the same platform, the home virtual wireless switch module and visited virtual wireless switch module communicate by an interprocess communication such as a UDP/IP tunnel, unix-domain sockets, named pipes, message queues, shared memory, or other such abstractions found in operating systems.  \\n      To accomplish this, in one implementation, the wireless router  312  may comprise a home virtual wireless switch configured to support a first subnet  310 , a visited virtual wireless switch configured to support a second subnet  320 , a virtual tunnel which couples the home virtual wireless switch to the visited virtual wireless switch, a virtual registration server, a first virtual IP socket which couples the home virtual wireless switch to the virtual registration server, and a second virtual IP socket which couples the visited virtual wireless switch to the virtual registration server. Because the home virtual wireless switch, the visited virtual wireless switch, and the virtual registration server are virtual modules implemented within the wireless router  312 , these modules are not marked with reference numerals. Similarly, connections such as the virtual tunnel, the first virtual IP socket, and the second virtual IP socket are also virtual, and therefore are not marked with reference numerals.  \\n      Typically, in a given WLAN there are a number of active clients. In this example, the first subnet  310  would typically support a group of the active clients having client IP addresses within the first subnet  310 , and the second subnet  320  would typically supports another group of the active clients having client IP addresses within the second subnet  320 . The first DHCP server  311  will assign the IP addresses from the first subnet  310  to the clients connected to access ports AP 1   315 , AP 2   316 , AP 3   317 . The second DHCP server  313  will assign the IP addresses from the second subnet  320  to the clients connected to access ports AP 4   325 , AP 5   326 , AP 6   327 .  \\n      The client  302  is initially in the first subnet  310  and is associated with the home virtual wireless switch and has a client IP address from the within the first subnet  310 . The client  302  initially communicates with the home virtual wireless switch through the AP 3   317 , but eventually roams into the second subnet  320  where it communicates with the visited virtual wireless switch through the AP 4   325 .  \\n      The home virtual wireless switch supports the first subnet  310  and has first configuration information associated with it comprising a first IP address and a list of first subnets  310  or IP domains which the home virtual wireless switch supports. The visited virtual wireless switch supports the second subnet  320  and has second configuration information associated with it comprising a second IP address and a list of second subnets  320  (which the visited virtual wireless switch supports. The home virtual wireless switch and the visited virtual wireless switch can create a first active client list for the first subnet  310  and a second active client list for the second subnet  320 , respectively.  \\n      For example, the ACL created by the home virtual wireless switch comprises a record for each active client in the first subnet  310 . The record of each client  302  comprises, for instance, a MAC address of the client, a client IP address of the client, the home virtual wireless switch of the client, the visited virtual wireless switch of the client, inactivity timers for the home virtual wireless switch and location information. The home virtual wireless switch can send a copy of the active client list to the virtual registration server and the other virtual wireless switches in the WLAN. The visited virtual wireless switch can also create an ACL for each active client in the second subnet  320  in a similar manner.  \\n      The virtual registration server has a server IP address. The home virtual wireless switch registers with the virtual registration server by communicating its configuration information to the virtual registration server over the first IP socket. Likewise, the visited virtual wireless switch registers with the virtual registration server by communicating its configuration information to the virtual registration server over the second IP socket.  \\n      A database in the virtual registration server can store the first active client list, the second active client list, and any other active client lists from other virtual wireless switches. The virtual registration server is configured to maintain a master active client list (MACL) by combining all of the ACLs into a MACL. In this example, the virtual registration server combines the first ACL and second ACL to create the MACL. This MACL comprises a record for each active client  302  in the WLAN. The virtual registration server can send a copy of the MACL to each virtual wireless switch in the wireless router  312 .  \\n      The virtual wireless switches can send updates to the virtual registration server and the virtual registration server can update the MACL and send the updates of the MACL to each wireless switch in the wireless router  312  as the updates are received from the virtual wireless switches. Alternatively, the virtual wireless switches can send the updates to the virtual registration server and then send the updates to each of the other virtual wireless switches in the WLAN.  \\n      When the client  302  roams from the home virtual wireless switch to the visited virtual wireless switch, the client  302  802.11 authenticates with the visited virtual wireless switch, 802.11 associates with the visited virtual wireless switch, 802.1x authenticates with the visited virtual wireless switch and issues a Dynamic Host Configuration Protocol (DHCP) request. The visited virtual wireless switch relays the DHCP request to the home virtual wireless switch through the virtual tunnel. The visited virtual wireless switch uses the MACL to determine that the client IP address belongs to the first subnet  310  and that the client  302  was originally associated with the home virtual wireless switch. The visited virtual wireless switch obtains the client IP address from the first configuration information sent to the virtual registration server when the home virtual wireless switch registered with the virtual registration server. For example, after the client  302  802.11 re-authenticates and the visited virtual wireless switch can search the record to get the home virtual wireless switch from the MAC address of the client  302 .  \\n      The home virtual wireless switch passes the DHCP request to the DHCP server  311 , and the DHCP server  311  re-assigns the client IP address to the client. The home virtual wireless switch forwards a Dynamic Host Configuration Protocol (DHCP) response to the visited virtual wireless switch and the client. The visited virtual wireless switch can then forward any IP packets it receives from the client  302  through the virtual tunnel to the home virtual wireless switch. The home virtual wireless switch uses the MACL to determine that the client is now associated with the visited virtual wireless switch. Likewise, the home virtual wireless switch can forward any IP packets it receives to the client  302  through the visited virtual wireless switch.  \\n       FIG. 11  is a flow chart showing an exemplary method for layer 3 roaming inside a wireless switch or wireless router according to one exemplary embodiment.  \\n      This method can be used, for example, in the WLAN described in  FIG. 10  to allow a client  302  to keep its client IP address and maintain IP connectivity while roaming between a first subnet  310  and a second subnet  320  supported by a single wireless router  312 .  \\n      At step  1104 , a first ACL is created using registration information from each client  302  in the first subnet  310  and a second active client  302  list is created using the registration information from each client  302  in the second subnet  320 .  \\n      At step  1106 , a MACL is generated using the first ACL and the second ACL. A copy of the MACL is sent to each wireless switch in the WLAN.  \\n      At step  1108 , the client  302 , initially associated with a home virtual wireless switch module and having a client IP address from within a first subnet  310 , roams from the home virtual wireless switch or first subnet  310  to the a second subnet  320  supported by a visited virtual wireless switch module. Because the virtual wireless switch modules, internal L3 router and the virtual registration server are running on the same platform, the home virtual wireless switch module and visited virtual wireless switch module communicate by an interprocess communication such as a UDP/IP tunnel, unix-domain sockets, named pipes, message queues, shared memory, or other such abstractions found in operating systems. At this step, techniques are provided for maintaining the client IP address on the client  302  when the client  302  roams from the first subnet  310  to the second subnet  320 . For example, the client IP address can be maintained at the client  302  when the client  302  roams from the home virtual wireless switch module and the visited virtual wireless switch module by 802.11 authenticating the client  302  with the visited virtual wireless switch module, 802.11 associating the client  302  with the visited virtual wireless switch module, 802.1x authenticating the client  302  with the visited virtual wireless switch module, issuing a DHCP request from the client  302  to the visited virtual wireless switch module, relaying the DHCP request from the visited virtual wireless switch module to the home virtual wireless switch module via the interprocess communication, passing the DHCP request from the home virtual wireless switch module to a first DHCP server  311 , which re-assigns the client IP address to the client  302 , and forwarding a DHCP response from the first DHCP server  311  to the visited virtual wireless switch module and the client  302 .  \\n      At step  1110 , the visited virtual wireless switch module receives IP packets addressed to the client.  \\n      At step  1112 , the visited virtual wireless switch module can use the MACL to determine that the client IP address belongs to the first subnet  310  and that the client  302  was originally associated with the home virtual wireless switch module. The visited virtual wireless switch module can obtain the client IP address from a portion of the MACL including the registration information associated with the home virtual wireless switch module. The record of the client  302  can be used to obtain the home virtual wireless switch module from the MAC address of the client  302 .  \\n      At step  1114 , when an IP packet from the client  302  is received at the visited virtual wireless switch module, the IP packet can be forwarded via the interprocess communication to the home virtual wireless switch module.  \\n      At step  1116 , when a second IP packet for the client  302  is received at the home virtual wireless switch module, the second IP packet can be forwarded via the interprocess communication to the visited virtual wireless switch module. The second IP packet can then be sent from the visited virtual wireless switch module to the client  302 .  \\n      Thus, numerous embodiments have been disclosed above which can provide techniques which support layer 3 IP roaming and allow a client to keep its original, pre-roam IP address and TCP/IP connection from its home subnet when the client undergoes a layer 3 roam to a new subnet. These techniques can help reduce the likelihood of dropped calls or sessions without requiring modification to the client software.  \\n      The sequence of the text in any of the claims does not imply that process steps must be performed in a temporal or logical order according to such sequence unless it is specifically defined by the language of the claim. The process steps may be interchanged in any order without departing from the scope of the invention as long as such an interchange does not contradict the claim language and is not logically nonsensical. Furthermore, numerical ordinals such as “first,” “second,” “third,” etc. simply denote different singles of a plurality and do not imply any order or sequence unless specifically defined by the claim language.  \\n      Furthermore, words such as “connect” or “coupled to” used in describing a relationship between different elements do not imply that a direct physical connection must be made between these elements. For example, two elements may be connected to each other physically, electronically, logically, or in any other manner, through one or more additional elements, without departing from the scope of the invention. Thus, to the extent the description refers to certain features being “connected” or “coupled” together, unless expressly stated otherwise, “connected” or “coupled” means that one feature is directly or indirectly connected or coupled to another feature, and not necessarily mechanically. Although drawings depict exemplary arrangements of elements, additional intervening elements, devices, features, or components may be present in an actual embodiment assuming that the functionality of the circuit is not adversely affected. The connecting lines shown in the various figures represent example functional relationships and/or physical couplings between the various elements. Many alternative or additional functional relationships or physical connections may be present in a practical embodiment or implementation.  \\n      Those of skill in the art would understand that information and signals may be represented using any of a variety of different technologies and techniques. For example, data, instructions, commands, information, signals, bits, symbols, and chips that may be referenced throughout the above description may be represented by voltages, currents, electromagnetic waves, magnetic fields or particles, optical fields or particles, or any combination thereof.  \\n      Those of skill would further appreciate that the various illustrative logical blocks, modules, circuits, and algorithm steps described in connection with the embodiments disclosed herein may be implemented as electronic hardware, computer software, or combinations of both. To clearly illustrate this interchangeability of hardware and software, various illustrative components, blocks, modules, circuits, and steps have been described above generally in terms of their functionality. Whether such functionality is implemented as hardware or software depends upon the particular application and design constraints imposed on the overall system. Skilled artisans may implement the described functionality in varying ways for each particular application, but such implementation decisions should not be interpreted as causing a departure from the scope of the present invention.  \\n      The various illustrative logical blocks, modules, and circuits described in connection with the embodiments disclosed herein may be implemented or performed with a general purpose processor, a digital signal processor (DSP), an application specific integrated circuit (ASIC), a field programmable gate array (FPGA) or other programmable logic device, discrete gate or transistor logic, discrete hardware components, or any combination thereof designed to perform the functions described herein. A general-purpose processor may be a microprocessor, but in the alternative, the processor may be any conventional processor, controller, microcontroller, or state machine. A processor may also be implemented as a combination of computing devices, e.g., a combination of a DSP and a microprocessor, a plurality of microprocessors, one or more microprocessors in conjunction with a DSP core, or any other such configuration.  \\n      The steps of a method or algorithm described in connection with the embodiments disclosed herein may be embodied directly in hardware, in a software module executed by a processor, or in a combination of the two. A software module may reside in RAM memory, flash memory, ROM memory, EPROM memory, EEPROM memory, registers, hard disk, a removable disk, a CD-ROM, or any other form of storage medium known in the art. An exemplary storage medium is coupled to the processor such the processor can read information from, and write information to, the storage medium. In the alternative, the storage medium may be integral to the processor. The processor and the storage medium may reside in an ASIC. The ASIC may reside in a user terminal. In the alternative, the processor and the storage medium may reside as discrete components in a user terminal.  \\n      The previous description of the disclosed embodiments is provided to enable any person skilled in the art to make or use the present invention. Various modifications to these embodiments will be readily apparent to those skilled in the art, and the generic principles defined herein may be applied to other embodiments without departing from the spirit or scope of the invention. While at least one exemplary embodiment has been presented in the foregoing detailed description, it should be appreciated that a vast number of variations exist. It should also be appreciated that the exemplary embodiment or exemplary embodiments are only examples, and are not intended to limit the scope, applicability, or configuration of the invention in any way. Rather, the foregoing detailed description will provide those skilled in the art with a convenient road map for implementing the exemplary embodiment or exemplary embodiments. It should also be understood that various changes can be made in the function and arrangement of elements without departing from the scope of the invention as set forth in the appended claims and the legal equivalents thereof. Thus, the present invention is not intended to be limited to the embodiments shown herein but is to be accorded the widest scope consistent with the principles and novel features disclosed herein.', 'id': 'US-2006268834-A1', 'ti': 'Method, system and wireless router apparatus supporting multiple subnets for layer 3 roaming in wireless local area networks (WLANs)'}>,\n",
       " <Hit {'ab': 'A method for improved handling of emergency calls in a roaming scenario includes: a voice call being initiated by the user equipment via the user equipment transmitting a first message directed to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number; wherein a second message is transmitted to the network node of the home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call; and a third message being generated by the node of the home telecommunications network, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment to trigger an emergency call procedure., A method for improved handling of emergency calls in a roaming scenario includes: a voice call being initiated by the user equipment via the user equipment transmitting a first message directed to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number; wherein a second message is transmitted to the network node of the home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call; and a third message being generated by the node of the home telecommunications network, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment to trigger an emergency call procedure.', 'clm': 'The invention claimed is: \\n     \\n       1. A method for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein a proxy functionality of the home telecommunications network is used in the roaming scenario, wherein a voice call is initiated by the user equipment using a specific emergency call telephone number, wherein the specific emergency call telephone number is a valid emergency call telephone number in the visited telecommunications network, wherein the user equipment initially handles the voice call as a normal voice call, wherein the method comprises:\\n in a first step, transmitting, by the user equipment so as to initiate the voice call, a first message directed via the visited telecommunications network to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number; \\n in a second step, subsequent to the first step, receiving, by the telephone number detecting network node of the visited telecommunications network, the first message, detecting, by the telephone number detecting network node, the specific emergency call telephone number, and transmitting, by the telephone number detecting network node to the network node of the home telecommunications network, a second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call; and \\n in a third step, subsequent to the second step, a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment, the third and fourth messages comprising an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message. \\n \\n     \\n     \\n       2. The method according to  claim 1 , wherein the normal manner is a non-emergency call manner. \\n     \\n     \\n       3. The method according to  claim 1 , wherein the emergency call information comprises:\\n a uniform resource name; and/or \\n an information related to an emergency type; and/or \\n the specific emergency call telephone number; and/or \\n a further specific emergency call telephone number. \\n \\n     \\n     \\n       4. The method according to  claim 1 , wherein the third message and/or the fourth message comprises a further emergency call information, wherein the further emergency call information comprises:\\n a uniform resource name; and/or \\n an information related to the emergency type; and/or \\n the specific emergency call telephone number; and/or \\n a further specific emergency call telephone number. \\n \\n     \\n     \\n       5. The method according to  claim 1 , wherein the third message is verified by the visited telecommunications network, wherein the third message is transmitted to the telephone number detecting network node of the visited telecommunications network. \\n     \\n     \\n       6. The method according to  claim 1 , wherein the telephone number detecting network node of the visited telecommunications network is:\\n a session border controller; or \\n a network node realizing an interworking border control function; or \\n an application layer gateway; or \\n a network node realizing a transit and roaming function (TRF); or \\n a standalone network node. \\n \\n     \\n     \\n       7. The method according to  claim 1 , wherein the network node of the home telecommunications network is a network node realizing a proxy call session control function, and wherein the network node of the home telecommunications network is configured for analyzing and/or detecting the emergency call information. \\n     \\n     \\n       8. The method according to  claim 1 , wherein the user equipment detects the initiated voice call being an emergency call upon receiving the fourth message. \\n     \\n     \\n       9. A system for improved handling of emergency calls in a roaming scenario, the system comprising:\\n a home telecommunications network comprising a network node; \\n a visited telecommunications network including a telephone number detecting network node; and \\n a user equipment assigned to the home telecommunications network but connected to the visited telecommunications network, wherein a proxy functionality of the home telecommunications network is used in a roaming scenario of the user equipment, \\n wherein the user equipment is configured to transmit, so as to initiate a voice call using a specific emergency call telephone number that is a valid emergency call telephone number in the visited telecommunications network, a first message directed via the visited telecommunications network to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number, \\n wherein the user equipment is further configured to initially handle the voice call as a normal voice call, \\n wherein the telephone number detecting network node of the visited telecommunications network is configured to receive the first message, to detect the specific emergency call telephone number, and to transmit a second message directed to the network node of the home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation to the voice call, \\n wherein the network node of the home telecommunications network is configured to receive the second message, and to generate a third message directed to the user equipment, \\n wherein a network node of the visited telecommunications network is configured to receive the third message and to forward the third message, as a fourth message, to the user equipment, and \\n wherein the third and fourth messages comprise an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message. \\n \\n     \\n     \\n       10. The system according to  claim 9 , wherein the emergency call information comprises:\\n a uniform resource name; and/or \\n an information related to the emergency type; and/or \\n the specific emergency call telephone number; and/or \\n a further specific emergency call telephone number. \\n \\n     \\n     \\n       11. The system according to  claim 9 , wherein the third message and/or the fourth message comprises a further emergency call information, wherein the further emergency call information comprises:\\n a uniform resource name; and/or \\n an category information related to the emergency type; and/or \\n the specific emergency call telephone number; and/or \\n a further specific emergency call telephone number. \\n \\n     \\n     \\n       12. One or more non-transitory computer-readable mediums having processor-executable instructions stored thereon for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein a proxy functionality of the home telecommunications network is used in the roaming scenario, wherein a voice call is initiated by the user equipment using a specific emergency call telephone number, wherein the specific emergency call telephone number is a valid emergency call telephone number in the visited telecommunications network, wherein the user equipment initially handles the voice call as a normal voice call, wherein the processor-executable instructions, when executed, facilitate:\\n in a first step, transmitting, by the user equipment so as to initiate the voice call, a first message directed via the visited telecommunications network to a network node oldie home telecommunications network, the first message comprising the specific emergency call telephone number; \\n in a second step, subsequent to the first step, receiving, by the telephone number detecting network node of the visited telecommunications network, the first message, detecting, by the telephone number detecting network node, the specific emergency call telephone number, and transmitting, by the telephone number detecting network node to the network node of the home telecommunications network, a second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call; and \\n in a third step, subsequent to the second step, a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment, the third and fourth messages comprising an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message.', 'cpc': 'H04L2101/385', 'detd': 'CROSS-REFERENCE TO PRIOR APPLICATIONS \\n     This application is a U.S. National Phase application under 35 U.S.C. § 371 of International Application No. PCT/EP2015/077911, filed on Nov. 27, 2015, and claims benefit to European Patent Application No. EP 15151293.6, filed on Jan. 15, 2015. The International Application was published in English on Jul. 21, 2016 as WO 2016/113028 A1 under PCT Article 21(2). \\n     FIELD \\n     The present invention relates a method for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein the proxy functionality of the home telecommunications network is used in the roaming scenario, wherein a voice call is initiated by the user equipment using a specific emergency call telephone number being a valid emergency call telephone number in the visited telecommunications network but wherein the user equipment in unaware of the specific emergency call telephone number being a valid emergency call telephone number in the visited telecommunications network (corresponding to the so-called non-UE detected emergency calls). \\n     Furthermore, the present invention relates to a system, a telecommunications network and/or a network node for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein the proxy functionality of the home telecommunications network is used in the roaming scenario. \\n     Additionally, the present invention relates to a program and to a computer program product for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein the proxy functionality of the home telecommunications network is used in the roaming scenario. \\n     The present invention relates to the area of both wireless and nomadic or fixed communications. \\n     BACKGROUND \\n     To support large numbers of (roaming) subscribers—e.g. mobile subscribers or nomadic, i.e. temporarily fixed subscribers—a number of telecommunications networks such as public land mobile networks and/or stationary (fixed line) networks have been established and are operative. Telecommunications networks such as public land mobile networks (PLMNs) may communicatively connect to other mobile telecommunications networks and/or to fixed, land-wired public switched telephone networks (PSTNs) and/or to data networks such as the internet (or IP-networks, internet protocol networks). User equipments (e.g. a mobile or nomadic terminal) are typically assigned to a home telecommunications network, such as a home public land mobile network. In a roaming scenario, the user equipment is typically not located in reach of the home telecommunications network (or home public land mobile network) but is connected to a visited telecommunications network (or visited public land mobile network). Most modern telecommunications network (such as public land mobile networks or also data networks) are packet-switched networks, in which data are routed in autonomous units called data packets. Packet-switched networks are characterized by high speed, low latency, and high network resource utilization efficiency. Two examples of packet-switched technologies are the General Packet Radio Service (GPRS) of the Global System for Mobile communications (GSM), and the Evolved Packet System (EPS), also commercially known as the Long Term Evolution (LTE), of the Universal Mobile Telecommunications System (UMTS), wireless local area network (WLAN). The Internet Protocol (IP) is a widely-deployed protocol for data communications in packet-switched networks. The IP Multimedia System (IMS) is an industry standard framework for delivering voice and multimedia communications over IP networks. Communications between nodes within an IMS network utilize the Session Initiation Protocol (SIP). SIP is a signaling protocol for Internet conferencing, telephony, presence, events notification, instant messaging, and the like. SIP signaling uses a long-term stable identifier, the SIP Universal Resource Indicator (URI). SIP signaling data packets in an IMS network are processed by network node of the telecommunications network, usually also called SIP servers or proxies. Together, these network nodes provide a call session control function (CSCF). Different types of such CSCF network nodes perform specific functions: A proxy call session control function (P-CSCF) is a SIP proxy that is the first point of contact for a user equipment (or an IMS terminal). Typically, the proxy call session control function that a user equipment uses is located in the visited telecommunications network in case that local breakout is used. In case local breakout is not used, the proxy call session control function a user equipment normally uses is located in the home telecommunications network (i.e. the proxy call session control function is part of or is assigned to the home telecommunications network). \\n     Also IMS-enabled user equipments need to be able to place emergency calls even when located in (and connected to) a visited telecommunications network. Non-UE-detected emergency calls (i.e. emergency calls that are placed (or initiated) by the user equipment which are emergency calls in the network environment of the user equipment (i.e. the visited telecommunications network) that the user equipment is unaware of, e.g. due to the use of an emergency call telephone number (valid in the visited telecommunications network) that is not maintained in a list within the user equipment) are relatively easy to handle in case of the use of local breakout: In this case, the local proxy call session control function (i.e. the proxy call session control function of the visited telecommunications network) has a list of the local emergency numbers (of the visited telecommunications network or valid with in the respective context of the user equipment in the visited telecommunications network), and is hence able to detect the call type (i.e. the emergency type of an emergency call) and apply the needed procedures to guarantee successful handling of the requested emergency service. \\n     In scenarios where no local breakout is to be used, and each call is routed to (a respective proxy call session control function within) the home telecommunications network, there is only the possibility to maintain a (normally huge) list of local emergency numbers (that are applicable or valid within the different telecommunications networks of all the possible roaming partners of the home telecommunications network) with a mapping to the related uniform resource names of the emergency call telephone numbers (also called sos-uniform resource names) in the local network node or the home telecommunications network (typically a proxy call session control function of the home telecommunications network). The maintenance of such a database for all roaming partners worldwide requires a considerable effort and—hence—is very costly. \\n     SUMMARY \\n     In an embodiment, the present invention provides a method for improved handling of emergency calls in a roaming scenario. A user equipment is assigned to a home telecommunications network and the user equipment is connected to a visited telecommunications network. A proxy functionality of the home telecommunications network is used in the roaming scenario. A voice call is initiated by the user equipment using a specific emergency call telephone number. The specific emergency call telephone number is a valid emergency call telephone number in the visited telecommunications network. The user equipment initially handles the voice call as a normal voice call. The method includes: in a first step, the voice call is initiated by the user equipment via the user equipment transmitting a first message directed to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number; in a second step, subsequent to the first step, the specific emergency call telephone number is detected by a telephone number detecting network node of the visited telecommunications network, wherein a second message is transmitted to the network node of the home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call; and in a third step, subsequent to the second step, a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment, the third and fourth messages comprising an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE DRAWINGS \\n       The present invention will be described in even greater detail below based on the exemplary figures. The invention is not limited to the exemplary embodiments. All features described and/or illustrated herein can be used alone or combined in different combinations in embodiments of the invention. The features and advantages of various embodiments of the present invention will become apparent by reading the following detailed description with reference to the attached drawings which illustrate the following: \\n         FIG. 1  schematically illustrates a system according to the present invention, comprising a first telecommunications network (or home telecommunications network) and a second telecommunications network (or visited telecommunications network), and a user equipment connected to the visited telecommunications network. \\n         FIG. 2  schematically illustrates a communication diagram between the user equipment, the home telecommunications network and the visited telecommunications network, wherein the exchange of different messages is shown in view of initiating an emergency call. \\n     \\n    \\n    \\n     DETAILED DESCRIPTION \\n     Exemplary embodiments of the present invention provide an effective and simple solution (especially reducing operational costs (OPEX costs)) to identify local emergency calls in roaming situations (without local breakout) without the need to maintain an important database for all roaming partners worldwide within the home telecommunications network of the user equipment, and without the need to rely on user equipment implementation when providing numbers to the user equipment. \\n     In an exemplary embodiment, the present invention provides a method for improved handling of emergency calls in a roaming scenario, wherein a user equipment is assigned to a home telecommunications network and wherein the user equipment is connected to a visited telecommunications network, wherein the proxy functionality of the home telecommunications network is used in the roaming scenario, \\n     wherein a voice call is initiated by the user equipment using a specific emergency call telephone number being a valid emergency call telephone number in the visited telecommunications network, \\n     wherein the user equipment initially handles the voice call—involving the specific emergency call telephone number—as a normal voice call (non-UE detected emergency call), \\n     wherein the method comprises the steps of: \\n     in a first step, the voice call is initiated by the user equipment via the user equipment transmitting a first message directed to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number, wherein the specific emergency call telephone number is detected by a telephone number detecting network node of the visited telecommunications network, \\n     in a second step, subsequent to the first step, a second message is transmitted to the network node home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call, \\n     in a third step, subsequent to the second step, a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment, the third and fourth messages comprising an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message. \\n     It is thereby advantageously possible according to the present invention to identify local emergency calls in roaming scenarios without the need to rely on user equipment implementation when providing numbers to the user equipment, and avoid maintaining a huge list of local emergency numbers of all roaming partners in the home telecommunications network. According to the present invention, it is hence advantageously possible that that network operators are enabled to deploy a simple, cost efficient, and network based method with minor provisioning effort to handle non-user equipment detected emergency calls when roaming without local breakout. \\n     National regulation defines emergency services by the local numbering plan. If an emergency number (i.e. an emergency call telephone number) is not provisioned to a user equipment (or a terminal equipment such as an IMS UE or generic SIP UE, etc.) and the user dials such an unknown emergency number (i.e. a telephone number for which the user equipment is not aware that it is an emergency number), the user equipment will initiate the call setup as a normal call using the dialed number. Therefore, it is the task of the telecommunications network to detect that the initiated call is dedicated to an emergency service, identify the service and proceed with the right procedures related to this service as assigning an sos-URN (uniform resource name of the sos top level domain, especially as defined according to the 3GPP (Third Generation Partnership Project) and/or IETF (Internet Engineering Task Force standard) to guarantee the routing to the responsible public safety answering point (PSAP). \\n     According to the present invention, improved handling of emergency calls in a roaming scenario is possible in cases where the proxy functionality of the home telecommunications network is used in the roaming scenario, i.e. in cases without local breakout. It is therefore advantageously possible according to the present invention that the responsibility to comply (both regarding the detection which type of emergency is to apply, and regarding the definition of the routing to proceed with the call handling) with the national regulations (especially regarding the numbering plan) relies on the network operator of the visited telecommunications network. \\n     According to the present invention, a voice call is initiated by the user equipment using a telephone number that is a valid emergency call telephone number in the context of the visited telecommunications network. The valid emergency call telephone numbers can vary from country to country and also their number and the associated or assigned emergency types of the emergency call telephone numbers. The telephone number used by the user equipment to initiate the voice call is also called the specific emergency call telephone number. The assumption is that the user equipment is not aware of the fact of using a valid emergency call telephone number, i.e. the initiated voice call is a non-user equipment detected emergency call: The user equipment initially handles the voice call—involving the specific emergency call telephone number—as a normal voice call (as at least part of the locally (in the visited telecommunications network) valid emergency call telephone numbers might not be provided or provisioned to the user equipment). According to the present invention, an exemplary embodiment of the inventive method the comprises the steps of initiating, by the user equipment, of a voice call (like for any other voice call) in a first step via the user equipment transmitting a first message directed to a network node of the home telecommunications network. The first message comprises the specific emergency call telephone number. \\n     In a second step, subsequent to the first step, the specific emergency call telephone number is detected by a telephone number detecting network node of the visited telecommunications network, and a second message is transmitted to the network node of the home telecommunications network. The second message comprises an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call. \\n     Hence, according to the present invention, the telephone number detecting network node of the visited telecommunications network has an updated database information regarding the current emergency call telephone numbers applicable in the visited telecommunications network. However, this database can be restricted to the emergency telephone numbers applicable in the home telecommunications network, thus comparably small. \\n     In a third step, subsequent to the second step, a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded—as a fourth message—to the user equipment. The third and fourth messages comprise an indication, to the user equipment, to trigger an emergency call procedure towards the visited telecommunications network via a fifth message. \\n     According to the present invention it is preferred that the normal manner is a non-emergency call manner. \\n     According to a further embodiment of the present invention, it is also possible and preferred that the emergency call information comprises \\n     a uniform resource name, especially related to an emergency type and/or \\n     an information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     According to the present invention, it is thereby advantageously possible—according to one variant of the present invention—that the emergency call information does only comprise one bit (i.e. an indication that the initiated voice call relates to an emergency call number). According to further variants of the present invention, the emergency call information comprises, e.g., a uniform resource name (such as a uniform resource name that is not at all related to a specific emergency type or such as a uniform resource name that is related to a specific emergency type). According to still further (alternative or cumulative) variants of the present invention, the emergency call information comprises an information related to the emergency type. Furthermore, (likewise alternatively or cumulatively to the other possible parts of the emergency call information) the emergency call information comprises the specific emergency call telephone number (i.e. the one used by the user equipment when initiating the voice call (or emergency call)). Furthermore (likewise alternatively or cumulatively to the other possible parts of the emergency call information), the emergency call information comprises a further specific emergency call telephone number (i.e. a different emergency call telephone number when compared to the specific emergency call telephone number). \\n     According to a further embodiment of the present invention, it is also possible and preferred that the third message and/or the fourth message comprises a further emergency call information, wherein the further emergency call information comprises \\n     a uniform resource name, especially related to an emergency type and/or \\n     an information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     In a manner analogous to the emergency call information, it is advantageously possible according to the present invention that—according to one variant of the present invention—the further emergency call information does only comprise one bit (i.e. an indication that the initiated voice call relates to an emergency call number). However, this variant at most applies to the third message (and not to the fourth message as this information is not sufficient for the user equipment to have the correct behavior with respect to the fifth message: the user equipment at least needs to know the uniform resource name to direct the emergency call to in case of initiating the emergency call using packet switched procedures). According to further variants of the present invention, the further emergency call information comprises (either in the third message or in the fourth message or in both the third and the fourth message), e.g., a uniform resource name (such as a uniform resource name that is not at all related to a specific emergency type or such as a uniform resource name that is related to a specific emergency type). According to still further (alternative or cumulative) variants of the present invention, the further emergency call information comprises (either in the third message or in the fourth message or in both the third and the fourth message) an information related to the emergency type. Furthermore, (likewise alternatively or cumulatively to the other possible parts of the emergency call information) the further emergency call information comprises (either in the third message or in the fourth message or in both the third and the fourth message) the specific emergency call telephone number (i.e. the one used by the user equipment when initiating the voice call (or emergency call)). Furthermore (likewise alternatively or cumulatively to the other possible parts of the emergency call information), the further emergency call information comprises (either in the third message or in the fourth message or in both the third and the fourth message) a further specific emergency call telephone number (i.e. a different emergency call telephone number when compared to the specific emergency call telephone number). \\n     According to the present invention it is furthermore preferred that the third message is verified by the visited telecommunications network, wherein the third message is especially transmitted to the telephone number detecting network node of the visited telecommunications network. \\n     It is thereby advantageously possible according to the present invention that the verification process can be used to protect the visited telecommunications network from emergency call procedures that are inappropriate. With or without such a verification process, it is advantageously possible and preferred that the third message is especially transmitted to the telephone number detecting network node of the visited telecommunications network (i.e. the network node of the visited telecommunications network that also received the first message and transmitted the second message to the home telecommunications network). \\n     According to a further preferred embodiment of the present invention, the telephone number detecting network node of the visited telecommunications network is \\n     a session border controller, or \\n     a network node realizing an interworking border control function, or \\n     an application layer gateway, or \\n     a network node realizing a transit and roaming function (TRF), or \\n     a standalone network node. \\n     Thereby, it is advantageously possible to provide a high degree of flexibility in the realization of the telephone number detecting network node of the visited telecommunications network. The telephone number detecting network node of the visited telecommunications network can be implemented in the form of a new network node of the visited telecommunications network or in the form of an additional functionality of an existing network node, especially one of the mentioned network nodes and especially a network node that is capable to access the SIP signaling traffic. \\n     According to all embodiments of the present invention, the telephone number detecting network node of the visited telecommunications network is to be understood as a telephone number detecting functionality and the manner of implementation of this functionality can be chosen according to the architecture of the visited telecommunications network or the associated technical needs. \\n     According to the present invention it is furthermore preferred that the network node of the home telecommunications network is a network node realizing a proxy call session control function, wherein the network node of the home telecommunications network is especially configured for analyzing and/or detecting the emergency call information. \\n     It is thereby advantageously possible that the network node of the home telecommunications network, especially a network node realizing a proxy call session control function, triggers or invokes the user equipment to initiate an emergency call procedure. \\n     According to a further preferred embodiment of the present invention, the user equipment detects the initiated voice call being an emergency call upon receiving the fourth message. \\n     It is thereby advantageously possible that finally the user equipment is aware of the having initiated an emergency call by originally initiating the voice call via the first message. \\n     Furthermore, the present invention relates to a system for improved handling of emergency calls in a roaming scenario, the system comprising a user equipment, a home telecommunications network and a visited telecommunications network, wherein the user equipment is assigned to the home telecommunications network and wherein the user equipment is connected to the visited telecommunications network, wherein the proxy functionality of the home telecommunications network is used in the roaming scenario, \\n     wherein a voice call is initiated by the user equipment using a specific emergency call telephone number being a valid emergency call telephone number in the visited telecommunications network, \\n     wherein the user equipment is configured to initially handle the voice call—involving the specific emergency call telephone number—as a normal voice call (non-UE detected emergency call), \\n     wherein the system is configured such that: \\n     the voice call is initiated by the user equipment via the user equipment transmitting a first message directed to a network node of the home telecommunications network, the first message comprising the specific emergency call telephone number, wherein the specific emergency call telephone number is detected by a telephone number detecting network node of the visited telecommunications network, \\n     a second message is transmitted to the network node of the home telecommunications network, the second message comprising an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call, \\n     a third message is generated by the node of the home telecommunications network, the third message being directed to the user equipment, wherein the third message is transmitted to a network node of the visited telecommunications network, and forwarded, as a fourth message, to the user equipment, the third and fourth messages comprising an indication, to the user equipment, to trigger an emergency call procedure in accordance to the configuration of the visited telecommunications network via a fifth message. \\n     Thereby, it is advantageously possible to provide a system that is capable of identifying local emergency calls in roaming scenarios without local breakout and without the need to rely on user equipment implementation when providing numbers to the user equipment. Consequently, it is advantageously possible to avoid maintaining a huge list of local emergency numbers of all roaming partners in the home telecommunications network. \\n     According to the present invention it is furthermore preferred—especially with respect to a further exemplary embodiment of the inventive system—that the system is configured such that the emergency call information comprises \\n     a uniform resource name, especially related to an emergency type and/or \\n     an information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     According to a further embodiment of the present invention, it is also possible and preferred that the system is configured such that the third message and/or the fourth message comprises a further emergency call information, wherein the further emergency call information comprises \\n     a uniform resource name, especially related to an emergency type and/or \\n     a category information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     Furthermore, the present invention relates to a telecommunications network or a network node, wherein the telecommunications network and/or the network node is either part of the home telecommunications network of the system or of the visited telecommunications network of the system. \\n     Thereby, it is advantageously possible to provide a telecommunications network and/or a network node such that local emergency calls in roaming scenarios that do not use local breakout can be handled easily and without the need to rely on user equipment implementation when providing numbers to the user equipment. \\n     Additionally, the present invention relates to a program comprising a computer readable program code which, when executed on a computer or on a network node of a home telecommunications network or on a network node of a visited telecommunications network, or in part on the network node of the home telecommunications network and in part on the network node of the visited telecommunications network, causes the computer or the network node of the home telecommunications network or the network node of the visited telecommunications network to perform an exemplary embodiment of the inventive method. \\n     Still additionally, the present invention relates to a computer program product for a home telecommunications network and/or a visited telecommunications network, the computer program product comprising a computer program stored on a storage medium, the computer program comprising program code which, when executed on a computer or on a network node of a home telecommunications network or on a network node of a visited telecommunications network, or in part on the network node of the home telecommunications network and in part on the network node of the visited telecommunications network, causes the computer or the network node of the home telecommunications network or the network node of the visited telecommunications network to perform an exemplary embodiment of the inventive method. \\n     These and other characteristics, features and advantages of the present invention will become apparent from the following detailed description, taken in conjunction with the accompanying drawings, which illustrate, by way of example, the principles of the invention. The description is given for the sake of example only, without limiting the scope of the invention. The reference figures quoted below refer to the attached drawings. \\n     The present invention will be described with respect to particular embodiments and with reference to certain drawings but the invention is not limited thereto but only by the claims. The drawings described are only schematic and are non-limiting. In the drawings, the size of some of the elements may be exaggerated and not drawn on scale for illustrative purposes. \\n     Where an indefinite or definite article is used when referring to a singular noun, e.g. “a”, “an”, “the”, this includes a plural of that noun unless something else is specifically stated. \\n     Furthermore, the terms first, second, third and the like in the description and in the claims are used for distinguishing between similar elements and not necessarily for describing a sequential or chronological order. It is to be understood that the terms so used are interchangeable under appropriate circumstances and that the embodiments of the invention described herein are capable of operation in other sequences than described or illustrated herein. \\n     In  FIG. 1 , a first telecommunications network, e.g. a mobile communication network such as a public land mobile network, and a second telecommunications network, likewise e.g. a mobile communication network such as a public land mobile network, are schematically shown. The first telecommunications network is associated to (or assigned to) a user equipment  20  (or the user equipment  20  is assigned to the first telecommunications network), resulting in the first telecommunications network being the home telecommunications network  100  of the user equipment  20 . The present invention relates to the situation of the user equipment  20  being in a roaming situation, i.e. the user equipment  20  being connected not to the home telecommunications network  100  but to the second telecommunications network, which is referred to by the term visited telecommunications network  200 . \\n     The home telecommunications network  100  comprises a first access network  110  or access part and a first core network  120 . The home telecommunications network  100  is, e.g., a cellular telecommunications network comprising typically a plurality of network cells (not depicted in  FIG. 1 ). The first access network  110  comprises at least one first access entity  111  such as (especially in case of mobile communication networks) a base station entity enabled to serve a user equipment  20 . In case of the home telecommunications network  100  being a mobile communication network, the first access entity  111  typically serves a radio cell, i.e. a radio coverage are of the access entity (or base station entity) or associated to the first access entity  111 , and the user equipment  20  is able to be connected to the first access entity  111  via a radio link or a wireless communication link. In case of the home telecommunications network  100  being a fixed communication network, the first access entity  111  is typically a network access point to which the user equipment  20  is able to be connected by a wireline connection, or the first access entity  111  is connected to a local wireless access point, providing the possibility for the user equipment  20  to be connected to the first access entity  111  via a wireless communication link. The visited telecommunications network  200  comprises a second access network  210  or access part and a second core network  220 . The visited telecommunications network  200  is, e.g., a cellular telecommunications network comprising typically a plurality of network cells (not depicted in  FIG. 1 ). The second access network  210  comprises at least one second access entity  211  such as (especially in case of mobile communication networks) a base station entity enabled to serve the user equipment  20 . In case of the visited telecommunications network  200  being a mobile communication network, the second access entity  211  typically serves a radio cell, i.e. a radio coverage are of the second access entity  211  (or base station entity) or associated to the second access entity  211 , and the user equipment  20  is able to be connected to the second access entity  211  via a radio link or wireless communication link. In case of the visited telecommunications network  200  being a fixed communication network, the second access entity  211  is typically a network access point to which the user equipment  20  is able to be connected by a wireline connection, or the second access entity  211  is connected to a local wireless access point, providing the possibility for the user equipment  20  to be connected to the first access entity  111  via a wireless communication link. \\n     In the exemplary representation of a roaming situation depicted in  FIG. 1 , the user equipment  20  is connected—while being assigned (or associated) to the home telecommunications network  100 —to the second access entity  211  of the visited telecommunications network  200 . Furthermore, a further access entity  212  is schematically shown in  FIG. 1  as being part of the visited telecommunications network  200 . \\n     In such a roaming situation of the user equipment  20 , in case that a type of call handling involving local breakout is used (i.e. the proxy functionality of the visited telecommunications network  200  (and not of the home telecommunications network  100 ) is used), a network node of the visited telecommunications network  200  is primarily used for call handling, such as a proxy call session control function of the visited telecommunications network  200 . In such a situation, such a network node of the visited telecommunications network  200  should know any valid emergency call telephone numbers that can be used within the visited telecommunications network  200 . \\n     However, in case that a type of call handling avoiding local breakout is used (i.e., the proxy functionality of the home telecommunications network  100  is used), either the user equipment  20  would normally need to be provisioned with the valid emergency call telephone numbers that can be used within the visited telecommunications network  200 , or a network node of the home telecommunications network  100  would need to have (or at least be able to easily access) this information, i.e. the valid emergency call telephone numbers that can be used within this visited telecommunications network  200  but also the valid emergency call telephone numbers of any other possible visited telecommunications network of any roaming partner of the home telecommunications network  100 . The maintenance efforts of such a database are quite important, especially for network operators having a comparatively important number of roaming partners. \\n     The present invention solves this problem by removing the need both to provision the user equipment  20  or a network node of the home telecommunications network  100  with these potentially changing data: Upon the user equipment  20  initiating a voice call using an emergency call telephone number (among the (possibly numerous) valid emergency call telephone numbers accessible in the visited telecommunications network  200 ), hereinafter also called specific emergency call telephone number, the user equipment  20  is not necessarily aware of the fact that this voice call relates to a valid emergency call telephone number within the visited telecommunications network  200  (non-UE detected emergency call). As a consequence, the user equipment  20  handles the voice call as a normal voice call. \\n     According to the present invention, the fact that the voice call initiated by the user equipment  20  involves a valid emergency call telephone number within the visited telecommunications network  200  is first detected by the visited telecommunications network  200  (despite the fact that the proxy functionality of the home telecommunications network  100  is used for processing the voice call), and this fact communicated via—an emergency call information—to the home telecommunications network  100 , after which the home telecommunications network  100  in turn requests the user equipment  20  to trigger an emergency call procedure towards the visited telecommunications network  200 . Hence, according to the present invention, local breakout is not used and local emergency numbers are not provided to the user equipment  20 . A node in the visited network is detecting a call setup as an emergency call by analyzing the requested number (of the voice call). This node in the visited network typically assigns a corresponding sos-URN (uniform resource name of the sos top level domain) which can be added, together with an indication of emergency call request, into the signaling of the call setup to the home network. The node in the home network can use this information to apply the procedures to handle non-UE-detected emergency calls. \\n     An exemplary embodiment of the inventive method is represented schematically in greater detail in  FIG. 2 . \\n     In a first step, the voice call  310  is initiated by the user equipment  20  via the user equipment  20  transmitting a first message  301 . The first message  301  is directed to a network node of the home telecommunications network  100 , which will typically by the proxy call session control function of the home telecommunications network  100 . This proxy call session control function as an example of a network node of the home telecommunications network  100  is schematically represented in  FIG. 1  by means of a box and reference sign  140 . As for any normal voice call, the first message  301  comprises the telephone number dialed by the user equipment  20 , hence in the situation considered here, the specific emergency call telephone number. \\n     In a second step, subsequent to the first step, the first message  301  is received by a network node  230  of the visited telecommunications network  200  prior to being transmitted to the home telecommunications network  100 . In the second step, the specific emergency call telephone number is detected by the visited telecommunications network  200  (based on a database or other information storage component relating to the valid emergency call telephone numbers of only the concerned network, i.e. in this case typically of the visited telecommunications network  200 ); the corresponding network node of the visited telecommunications network  200  is consequently also called telephone number detecting network node  230  of the visited telecommunications network  200 . The telephone number detecting network node  230  of the visited telecommunications network  200  transmits a second message  302  to the network node of the home telecommunications network  100 , and the second message  302  comprises an emergency call information, the emergency call information being indicative of the use of a valid emergency call telephone number in relation of the voice call  310 . Via the emergency call information, the home telecommunications network  100  is informed at least that a valid emergency call telephone number has been used by the user equipment  20  when initiating the voice call  310 . \\n     The emergency call information transmitted by the second message  302  can, e.g., correspond—according to a variant of the present invention—to only one bit (or a flag information), indicating that a valid emergency call telephone number has been used by the user equipment  20  when initiating the voice call  310 . According other variants of the present invention the emergency call information comprises (either additional to the flag information or instead): \\n     a uniform resource name, especially related to an emergency type and/or \\n     an information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     In a third step, subsequent to the second step, a third message  303  is generated by the node of the home telecommunications network  100 —typically a network node realizing a proxy call session control function  140  within the home telecommunications network  100 . The third message  303  is directed to the user equipment  20 , and it is transmitted to a network node of the visited telecommunications network  200 —either the telephone number detecting network node  230  of the visited telecommunications network  200  or another network node of the visited telecommunications network  200 ; in  FIG. 2 , the situation of the third message  303  being transmitted generally “to the visited telecommunications network  200 ” (i.e. not necessarily to the telephone number detecting network node  230 ) is represented however, as this telephone number detecting network node  230  is also part of the visited telecommunications network  200 , this also refers to the case that the third message  303  is received by the telephone number detecting network node  230 . The third message  303  is forwarded, as a fourth message  304 , to the user equipment  20 , the third and fourth messages  303 ,  304  comprising an indication (e.g. in the form of a “380 Alternative Service”-message), to the user equipment  20 , to trigger an emergency call procedure towards the visited telecommunications network  200  via a fifth message  305 . \\n     According to variants of the present invention, the third message  303  and/or the fourth message  304  comprise a further emergency call information. Either the further emergency call information of the third message  303  is identical to the further emergency call information of the fourth message  304  (and hence called “further emergency call information”), or the further emergency call information of the third message  303  is not identical to the further emergency call information of the fourth message  304 ; in this latter case the further emergency call information of the third message  303  is also called first further emergency call information and the further emergency call information of the fourth message  304  is also called second further emergency call information. According to variants of the present invention, the further emergency call information (and, if applicable, the first further emergency call information and/or the second further emergency call information) comprises: \\n     a uniform resource name, especially related to an emergency type and/or \\n     an information related to the emergency type and/or \\n     the specific emergency call telephone number and/or \\n     a further specific emergency call telephone number. \\n     In short, in an exemplary embodiment, the inventive method enables a node in the visited telecommunications network  200  (i.e. the telephone number detecting network node  230 ; examples thereof are: a Session border controller (SBC), an Interworking Border Control Function (IBCF)) to detect the emergency call by analyzing the dialed number (of the voice call  310 ) and to put an indication into the call setup signaling to indicate to the proxy of the home telecommunications network  100  (P-CSCF or SIP proxy, reference sign  140 ) that the roaming user equipment  20  is initiating an emergency call. Additionally the telephone number detecting network node  230  in the visited telecommunications network  200  assigns an sos-URN related to the type of emergency call deduced from the requested number and put this sos-URN together with an optional additional indication into the call setup signaling to the proxy of the home telecommunications network  100  (P-CSCF or SIP Proxy). The node in the home network (P-CSCF) can use this information to apply the procedures to handle non-UE-detected emergency calls. This enables the network operators to implement a simple mechanism to handle non-UE detected emergency calls in roaming scenarios where local breakout is not used. \\n     In an IMS roaming scenario for mobile users or nomadic fixed network users when no P-CSCF (proxy call session control function) is in visited telecommunications network  200  and all calls are home routed (i.e. the proxy call session control function of the home telecommunications network  100  is used). Additionally, all or some local emergency numbers (i.e. of the visited telecommunications network  200 ) are not provisioned to the user equipment  20 . An SBC or IBCF in visited network detect an emergency call setup by analyzing the dialed number. The telephone number detecting network node  230  of the visited telecommunications network  200  (especially a session border controller (SBC) or an interworking border control function (IBCF)) identifies the type of emergency service and assigns a corresponding sos-URN. The telephone number detecting network node  230  of the visited telecommunications network  200  (typically SBC or IBCF) sends back—especially in a session setup signaling, e.g., as a Tel-URI parameter or sip-feature-tag or new sip header or a new SIP message body—an indication that this is a request for an emergency call. Instead or together to this indication in session setup signaling (e.g. as a Tel-URI parameter or sip-feature-tag or new sip header or new SIP message body), the assigned sos-URN is included. The proxy call session control function  140  (P-CSCF) in the home telecommunications network  100  can read the related information and trigger the procedures for non-UE detected emergency calls. \\n     While the invention has been illustrated and described in detail in the drawings and foregoing description, such illustration and description are to be considered illustrative or exemplary and not restrictive. It will be understood that changes and modifications may be made by those of ordinary skill within the scope of the following claims. In particular, the present invention covers further embodiments with any combination of features from different embodiments described above and below. Additionally, statements made herein characterizing the invention refer to an embodiment of the invention and not necessarily all embodiments. \\n     The terms used in the claims should be construed to have the broadest reasonable interpretation consistent with the foregoing description. For example, the use of the article “a” or “the” in introducing an element should not be interpreted as being exclusive of a plurality of elements. Likewise, the recitation of “or” should be interpreted as being inclusive, such that the recitation of “A or B” is not exclusive of “A and B,” unless it is clear from the context or the foregoing description that only one of A and B is intended. Further, the recitation of “at least one of A, B and C” should be interpreted as one or more of a group of elements consisting of A, B and C, and should not be interpreted as requiring at least one of each of the listed elements A, B and C, regardless of whether A, B and C are related as categories or otherwise. Moreover, the recitation of “A, B and/or C” or “at least one of A, B or C” should be interpreted as including any singular entity from the listed elements, e.g., A, any subset from the listed elements, e.g., A and B, or the entire list of elements A, B and C.', 'id': 'US-10231109-B2', 'ti': 'Handling of emergency calls in a roaming scenario'}>,\n",
       " <Hit {'ab': 'A method for reducing inter-PLMN forwarding of messages relating to Nnrf service operations includes receiving, at a first NRF, a first request for an Nnrf service operation, determining that the first request identifies a PLMN different from a PLMN of the first NRF, determining that the first request cannot be processed locally at the first NRF, and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request. The method further includes receiving , from the second NRF, a response to the first request, caching information from the response, receiving a second request for an Nnrf service operation, determining that the second request identifies a PLMN different from the PLMN of the first NRF, determining that the second request can be processed locally at the first NRF, and processing the second request locally at the first NRF using the cached information from the first response., A method for reducing inter-PLMN forwarding of messages relating to Nnrf service operations includes receiving, at a first NRF, a first request for an Nnrf service operation, determining that the first request identifies a PLMN different from a PLMN of the first NRF, determining that the first request cannot be processed locally at the first NRF, and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request. The method further includes receiving , from the second NRF, a response to the first request, caching information from the response, receiving a second request for an Nnrf service operation, determining that the second request identifies a PLMN different from the PLMN of the first NRF, determining that the second request can be processed locally at the first NRF, and processing the second request locally at the first NRF using the cached information from the first response.', 'clm': 'What is claimed is: \\n     \\n         1 . A method for reducing inter-public land mobile network (PLMN) forwarding of messages relating to Nnrf service operations, the method comprising:\\n at a first network function (NF) repository function (NRF):\\n receiving a first request for an Nnrf service operation; \\n determining that the first request identifies a PLMN different from a PLMN of the first NRF; \\n determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request; \\n receiving , from the second NRF, a response to the first request; \\n caching information from the response; \\n receiving a second request for an Nnrf service operation; \\n determining that the second request identifies a PLMN different from the PLMN of the first NRF; \\n determining that the second request can be processed locally at the first NRF; and \\n processing the second request locally at the first NRF using the cached information from the first response. \\n   \\n     \\n     \\n         2 . The method of  claim 1  wherein receiving a first request for an Nnrf service operation includes receiving a first request for an Nnrf_NFDiscovery service operation and caching information from the response includes caching NF profiles from the response. \\n     \\n     \\n         3 . The method of  claim 2  wherein receiving a second request for an Nnrf service operation includes receiving a second request for the Nnrf_NFDiscovery service operation and processing the second request locally at the first NRF includes responding to the second request for the Nnrf_NFDiscovery service operation using the NF profiles cached from the response to the first request for the Nnrf_NFDiscovery service operation. \\n     \\n     \\n         4 . The method of  claim 2  comprising reading, by the first NRF and from the response to the first request for the Nnrf_NFDiscovery service operation, a validity period, detecting expiration of the validity period, and, in response to detecting expiration of the validity period, sending, to the second NRF, a third request for the Nnrf_NFDiscovery service operation, receiving a response to the third request for the Nnrf_NFDiscovery service operation, and updating NF profiles cached by the first NRF based on the response. \\n     \\n     \\n         5 . The method of  claim 2  comprising transmitting, by the first NRF, a request for an NFStatusSubscribe service operation requesting to be notified of changes in status of the NF profiles obtained from the response to the first request for the Nnrf_NFDiscovery service operation, receiving a response to the request for the NFStatusSubscribe service operation including updated information for the NF profiles, and caching the updated information for the NF profiles locally at the first NRF. \\n     \\n     \\n         6 . The method of  claim 1  wherein receiving a first request for an Nnrf service operation includes receiving a first request for an Nnrf_AccessToken service operation and caching information from the response includes caching an access token from the response. \\n     \\n     \\n         7 . The method of  claim 6  wherein receiving a second request for an Nnrf service operation includes receiving a second request for the Nnrf_AccessToken service operation, and processing the second request locally at the first NRF includes responding to the second request for the Nnrf_AccessToken service operation using the access token cached from the response to the first request for the Nnrf_AccessToken service operation. \\n     \\n     \\n         8 . The method of  claim 1  wherein receiving a first request for an Nnrf service operation includes receiving a first request for an NFStatusSubscribe service operation and caching information from the response includes caching an NF profile from the response. \\n     \\n     \\n         9 . The method of  claim 8  wherein receiving a second request for an Nnrf service operation includes receiving a second request for the NFStatusSubscribe service operation, and processing the second request locally at the first NRF includes responding to the second request for the NFStatusSubscribe service operation using the NF profile cached from the response to the first request for the NFStatusSubscribe service operation. \\n     \\n     \\n         10 . The method of  claim 1  processing the second request locally at the first NRF using the cached information from the first response includes processing the second request at the visited NRF without forwarding the second request to the second NRF. \\n     \\n     \\n         11 . A system for reducing inter-public land mobile network (PLMN) forwarding of messages relating to Nnrf service operations, the system comprising:\\n a first network function (NF) repository function (NRF) including at least one processor and a memory; and   an Nnrf service operation message handler implemented by the at least one processor for receiving a first request for an Nnrf service operation, determining that the first request identifies a PLMN different from a PLMN of the first NRF, determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request, receiving, from the second NRF, a response to the first request, caching information from the response, receiving a second request for an Nnrf service operation, determining that the second request identifies a PLMN different from a PLMN of the first NRF, determining that the second request can be processed locally at the first NRF, and processing the second request locally at the first NRF using the cached information from the first response.   \\n     \\n     \\n         12 . The system of  claim 11  wherein the first request for an Nnrf service operation includes a first request for an Nnrf_NFDiscovery service operation and the information cached from the response includes NF profiles from the response. \\n     \\n     \\n         13 . The system of  claim 12  wherein the second request for an Nnrf service operation includes a second request for the Nnrf_NFDiscovery service operation and the Nnrf service operation message handler is configured to process the second request locally at the first NRF by responding to the second request for the Nnrf_NFDiscovery service operation using the NF profiles cached from the response to the first request for the Nnrf_NFDiscovery service operation. \\n     \\n     \\n         14 . The system of  claim 12  wherein the Nnrf service operation message handler is configured to read, from the response to the first request for the Nnrf_NFDiscovery service operation, a validity period, detect expiration of the validity period, and in response to detecting expiration of the validity period, send, to the second NRF, a third request for the Nnrf_NFDiscovery service operation, receive a response to the third request for the Nnrf_NFDiscovery service operation, and update NF profiles cached by the first NRF based on the response. \\n     \\n     \\n         15 . The system of  claim 12  wherein the Nnrf service operation message handler is configured to transmit, to the second NRF, a request for an NFStatusSubscribe service operation requesting to be notified of changes in status of the NF profiles obtained from the response to the first request for the Nnrf_NFDiscovery service operation, receive a response to the request for the NFStatusSubscribe service operation including updated information for the NF profiles, and cache the updated information for the NF profiles locally at the first NRF. \\n     \\n     \\n         16 . The system of  claim 11  wherein the first request for an Nnrf service operation includes a first request for an Nnrf_AccessToken service operation and the Nnrf service operation message handler is configured to cache an access token from the response. \\n     \\n     \\n         17 . The system of  claim 16  wherein the second request for an Nnrf service operation includes a second request for the Nnrf_AccessToken service operation and the Nnrf service operation message handler is configured to respond to the second request for the Nnrf_AccessToken service operation using the access token cached from the response to the first request for the Nnrf_AccessToken service operation. \\n     \\n     \\n         18 . The system of  claim 11  wherein the first request for an Nnrf service operation includes a first request for an NFStatusSubscribe service operation and the Nnrf service operation message handler is configured to cache an NF profile from the response. \\n     \\n     \\n         19 . The system of  claim 18  wherein the second request for an Nnrf service operation includes a second request for the NFStatusSubscribe service operation and the Nnrf service operation message handler is configured to process the second request locally at the first NRF by responding to the second request for the NFStatusSubscribe service operation using the NF profile cached from the response to the first request for the NFStatusSubscribe service operation. \\n     \\n     \\n         20 . A non-transitory computer readable medium having stored thereon executable instructions that when executed by a processor of a computer control the computer to perform steps comprising:\\n at a first network function (NF) repository function (NRF):\\n receiving a first request for an Nnrf service operation; \\n determining that the first request identifies a public land mobile network (PLMN) different from a PLMN of the first NRF; \\n determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request; \\n receiving , from the second NRF, a response to the first request; \\n caching information from the response; \\n receiving a second request for an Nnrf service operation; \\n determining that the second request identifies a PLMN different from the PLMN of the first NRF; \\n determining that the second request can be processed locally at the first NRF; and \\n processing the second request locally at the first NRF using the cached information from the first response.', 'cpc': 'H04L67/16', 'detd': 'TECHNICAL FIELD \\n     The subject matter described herein relates to processing of messages relating to Nnrf service operations. More particularly, the subject matter described herein relates to methods, systems, and computer readable media for reducing inter-PLMN forwarding of messages relating to Nnrf service operations. \\n     BACKGROUND \\n     In 5G telecommunications networks, a network function that provides service is referred to as a producer NF or NF service producer. A network function that consumes services is referred to as a consumer NF or NF service consumer. A network function can be a producer NF, a consumer NF, or both, depending on whether the network function is consuming, producing, or consuming and producing services. The terms “producer NF” and “NF service producer” are used interchangeably herein. Similarly, the terms “consumer NF” and “NF service consumer” are used interchangeably herein. \\n     A given producer NF may have many service endpoints, where a service endpoint is the point of contact for one or more NF instances hosted by the producer NF. The service endpoint is identified by a combination of Internet protocol (IP) address and port number or a fully qualified domain name that resolves to an IP address and port number on a network node that hosts a producer NF. An NF instance is an instance of a producer NF that provides a service. A given producer NF may include more than one NF instance. It should also be noted that multiple NF instances can share the same service endpoint. \\n     Producer NFs register with a network function repository function (NRF). The NRF maintains service profiles of available NF instances identifying the services supported by each NF instance. The terms “service profiles” and “NF profiles” are used interchangeably herein. Consumer NFs can obtain information about producer NF instances that have registered with the NRF through the NF service discovery procedure. According to the NF service discovery procedure, a consumer NF sends an NF discovery request to the NRF. The NF discovery request includes query parameters that the NRF uses to locate NF profiles of producer NFs capable of providing the service identified by the query parameters. NF profiles are data structures that define the type of service provided by a producer NF instance and well as contact and capacity information regarding the producer NF instance. \\n     In addition to consumer NFs, another type of network node that can invoke the NF service discovery procedure to obtain information about NF service instances is a service communications proxy (SCP). The case where the SCP uses the NF service discovery procedure to obtain information about producer NF instances on behalf of consumer NFs is referred to as delegated discovery. Consumer NFs connect to the service communications proxy, and the service communications proxy load balances traffic among producer NF service instances that provide the required services or directly routes the traffic to the destination producer NF instances. The communications model where consumer NFs communicate with producer NFs via the SCP is referred to as the indirect communications model. \\n     In addition to the SCP, another example of an intermediate proxy that forwards traffic between producer and consumer NFs is the security edge protection proxy (SEPP). The SEPP is the network function used to protect control plane traffic that is exchanged between different 5G public land mobile networks (PLMNs). As such, the SEPP performs message filtering, policing and topology hiding for all application programming interface (API) messages that are transmitted between PLMNs. \\n     One problem that can occur in 5G and other communications networks is that requests for Nnrf service operations are routed from an NRF in a network serving a subscriber (i.e., a visited or serving network) to an NRF in a home network of the subscriber, and responses to requests for service operations are routed from the NRF in the home network to the NRF in the visited network. Nnrf is the 3GPP-defined name of the interface provided by the NRF. Nnrf service operations are service provided by the NRF, and include NFDiscovery, NFStatusSubscribe, and AccessToken service operations. Requiring each request for an Nnrf service operation to travel from the NRF in the visited network to the NRF in the home network of the subscriber and each response to travel from the NRF In the home network to the NRF in the visited network increases latency for services required by mobile subscribers. \\n     Accordingly, there exists a need for improved methods, systems and computer readable media for handling messages relating to Nnrf service operations. \\n     SUMMARY \\n     A method for reducing inter-public land mobile network (PLMN) forwarding of messages relating to Nnrf service operations includes, at a first network function (NF) repository function (NRF), receiving a first request for an Nnrf service operation. The method further includes determining that the first request identifies a PLMN different from a PLMN of the first NRF. The method further includes determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request. The method further includes receiving , from the second NRF, a response to the first request. The method further includes caching information from the response. The method further includes receiving a second request for an Nnrf service operation. The method further includes determining that the second request identifies a PLMN different from the PLMN of the first NRF. The method further includes determining that the second request can be processed locally at the first NRF. The method further includes processing the second request locally at the first NRF using the cached information from the first response. \\n     According to another aspect of the subject matter described herein, receiving a first request for an Nnrf service operation includes receiving a first request for an Nnrf_NFDiscovery service operation and caching information from the response includes caching NF profiles from the response. \\n     According to another aspect of the subject matter described herein, receiving a second request for the Nnrf service operation includes receiving a second request for the Nnrf_NFDiscovery service operation and processing the second request locally at the first NRF includes responding to the second request for the Nnrf_NFDiscovery service operation using the NF profiles cached from the response to the first request for the Nnrf_NFDiscovery service operation. \\n     According to another aspect of the subject matter described herein, the method for reducing inter-PLMN forwarding of messages relating to Nnrf service operations includes reading, by the first NRF and from the response to the first request for the Nnrf_NFDiscovery service operation, a validity period, detecting expiration of the validity period, and in response to detecting expiration of the validity period, sending, to the second NRF, a third request for the Nnrf_NFDiscovery service operation, receiving a response to the third request for the Nnrf_NFDiscovery service operation, and updating NF profiles cached by the first NRF based on the response. \\n     According to another aspect of the subject matter described herein, the method for reducing inter-PLMN forwarding of messages relating to Nnrf service operations includes transmitting, by the first NRF, a request for an NFStatusSubscribe service operation requesting to be notified of changes in status of the NF profiles obtained from the response to the first request for the Nnrf_NFDiscovery service operation, receiving a response to the request for the NFStatusSubscribe service operation including updated information for the NF profiles, and caching the updated information for the NF profiles locally at the first NRF. \\n     According to another aspect of the subject matter described herein, receiving a first request for an Nnrf service operation includes receiving a first request for an Nnrf_AccessToken service operation and caching information from the response includes caching an access token from the response. \\n     According to another aspect of the subject matter described herein, receiving a second request for an Nnrf service operation includes receiving a second request for an Nnrf_AccessToken service operation and processing the second request locally at the first NRF includes responding to the second request for the Nnrf_AccessToken service operation using the access token cached from the response to the first request for the Nnrf_AccessToken service operation. \\n     According to another aspect of the subject matter described herein, receiving a first request for an Nnrf service operation includes receiving a first request for an NFStatusSubscribe service operation and caching information from the response includes caching an NF profile from the response. \\n     According to another aspect of the subject matter described herein, receiving a second request for the Nnrf service operation includes receiving a second request for the NFStatusSubscribe service operation and processing the second request locally at the first NRF includes responding to the second request for the NFStatusSubscribe service operation using the NF profile cached from the response to the first request for the NFStatusSubscribe service operation. \\n     According to another aspect of the subject matter described herein, processing the second request locally at the first NRF using the cached information from the first response includes processing the second request at the visited NRF without forwarding the second request to the second NRF. \\n     According to another aspect of the subject matter described herein, a system for reducing inter-public land mobile network (PLMN) forwarding of messages relating to Nnrf service operations is provided. The system includes a first network function (NF) repository function (NRF) including at least one processor and a memory. The system further includes an Nnrf service operation message handler implemented by the at least one processor for receiving a first request for an Nnrf service operation, determining that the first request identifies a PLMN different from a PLMN of the first NRF, determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request, receiving , from the second NRF, a response to the first request, caching information from the response, receiving a second request for an Nnrf service operation, determining that the second request identifies a PLMN different from the PLMN of the first NRF, determining that the second request can be processed locally at the first NRF, and processing the second request locally at the first NRF using the cached information from the first response. \\n     According to another aspect of the subject matter described herein, the first request for an Nnrf service operation includes a first request for an Nnrf_NFDiscovery service operation and the information cached from the response includes NF profiles from the response. \\n     According to another aspect of the subject matter described herein, the second request for an Nnrf service operation includes a second request for the Nnrf_NFDiscovery service operation and the Nnrf service operation message handler is configured to process the second request locally at the first NRF by responding to the second request for the Nnrf_NFDiscovery service operation using the NF profiles cached from the response to the first request for the Nnrf_NFDiscovery service operation. \\n     According to another aspect of the subject matter described herein, the Nnrf service operation message handler is configured to read, from the response to the first request for the Nnrf_NFDiscovery service operation, a validity period, detect expiration of the validity period, and in response to detecting expiration of the validity period, send, to the second NRF, a third request for the Nnrf_NFDiscovery service operation, receive a response to the third request for the Nnrf_NFDiscovery service operation, and update NF profiles cached by the first NRF based on the response. \\n     According to another aspect of the subject matter described herein, the Nnrf service operation message handler is configured to transmit, to the second NRF, a request for an NFStatusSubscribe service operation requesting to be notified of changes in status of the NF profiles obtained from the response to the first request for the Nnrf_NFDiscovery service operation, receive a response to the request for the NFStatusSubscribe service operation including updated information for the NF profiles, and cache the updated information for the NF profiles locally at the first NRF. \\n     According to another aspect of the subject matter described herein, the first request for an Nnrf service operation includes a first request for an Nnrf_AccessToken service operation and the Nnrf service operation message handler is configured to cache an access token from the response. \\n     According to another aspect of the subject matter described herein, the second request for an Nnrf service operation includes a second request for the Nnrf_AccessToken service operation, the Nnrf service operation message handler is configured to respond to the second request for the Nnrf_AccessToken service operation using the access token cached from the response to the first request for the Nnrf_AccessToken service operation. \\n     According to another aspect of the subject matter described herein, the first request for an Nnrf service operation includes a first request for an NFStatusSubscribe service operation and the Nnrf service operation message handler is configured to cache an NF profile from the response. \\n     According to another aspect of the subject matter described herein, the second request for the Nnrf service operation includes a second request for the NFStatusSubscribe service operation, and the Nnrf service operation message handler is configured to process the second request locally at the first NRF by responding to the second request for the NFStatusSubscribe service operation using the NF profile cached from the response to the first request for the NFStatusSubscribe service operation. \\n     According to another aspect of the subject matter described herein, a non-transitory computer readable medium having stored thereon executable instructions that when executed by a processor of a computer control the computer to perform steps is provided. The steps are performed at a first network function (NF) repository function (NRF). The steps include receiving a first request for an Nnrf service operation. The steps further include determining that the first request identifies a public land mobile network (PLMN) different from a PLMN of the first NRF. The steps further include \\n     determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request. The steps further include receiving , from the second NRF, a response to the first request. The steps further include caching information from the response. The steps further include receiving a second request for an Nnrf service operation. The steps further include determining that the second request identifies a PLMN different from the PLMN of the first NRF. The steps further include determining that the second request can be processed locally at the first NRF. The steps further include processing the second request locally at the first NRF using the cached information from the first response. \\n     The subject matter described herein can be implemented in software in combination with hardware and/or firmware. For example, the subject matter described herein can be implemented in software executed by a processor. In one exemplary implementation, the subject matter described herein can be implemented using a non-transitory computer readable medium having stored thereon computer executable instructions that when executed by the processor of a computer control the computer to perform steps. Exemplary computer readable media suitable for implementing the subject matter described herein include non-transitory computer-readable media, such as disk memory devices, chip memory devices, programmable logic devices, and application specific integrated circuits. In addition, a computer readable medium that implements the subject matter described herein may be located on a single device or computing platform or may be distributed across multiple devices or computing platforms. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE DRAWINGS \\n       Exemplary implementations of the subject matter described herein will now be explained with reference to the accompanying drawings, of which: \\n         FIG.  1    is a network diagram illustrating an exemplary 5G system network architecture; \\n         FIG.  2    is a message flow diagram illustrating exemplary messages exchanged in an NFDiscovery service operation involving NRFs located in different PLMNs; \\n         FIG.  3    is a network diagram illustrating a visited PLMN, a home PLMN, and messaging between the visited and home PLMN for an Nnrf service operation; \\n         FIG.  4    is a network diagram illustrating a visited PLMN, a home PLMN, and the construction of an inter-PLMN Nnrf service operation message processing database at a visited NRF; \\n         FIG.  5    is a network diagram illustrating a visited PLMN, a home PLMN, and the use of the inter-PLMN Nnrf service operation message processing database to process messages relating to Nnrf service operations in the visited PLMN; \\n         FIG.  6    is a network diagram illustrating a visited PLMN, a home PLMN, and the synchronizing of the inter-PLMN Nnrf service operation message processing database with the NF profiles database at the NRF in the home PLMN; \\n         FIG.  7    is a flow chart illustrating an exemplary process for handling an initial message relating to an Nnrf service operation at an NRF located in a visited or serving network; \\n         FIG.  8    is a flow chart illustrating an exemplary process for handling a message relating to an Nnrf service operation at an NRF located in a visited or serving network where the message is processed locally using the inter-PLMN Nnrf service operation message processing database at the visited NRF; \\n         FIGS.  9 A and  9 B  are a flow chart illustrating an exemplary process for synchronizing the inter-PLMN Nnrf service operation message processing database with the NF profiles database in the home PLMN; \\n         FIG.  10    is a message flow diagram illustrating exemplary messages exchanged in an NFStatusSubscribe service operation; \\n         FIG.  11    is a message flow diagram illustrating exemplary messages exchanged in an AccessToken service operation; \\n         FIG.  12    is a block diagram illustrating an exemplary architecture for an NRF configured to reduce inter-PLMN forwarding of messages relating to Nnrf service operations; and \\n         FIG.  13    is a flow chart illustrating an exemplary process for reducing inter-PLMN forwarding of messages relating to Nnrf service operations. \\n     \\n    \\n    \\n     DETAILED DESCRIPTION \\n       FIG.  1    is a block diagram illustrating an exemplary 5G system network architecture. The architecture in  FIG.  1    includes NRF  100  and SCP  101 , which may be located in the same home public land mobile network (HPLMN). As described above, NRF  100  may maintain profiles of available producer NF service instances and their supported services and allow consumer NFs or SCPs to subscribe to and be notified of the registration of new/updated producer NF service instances. SCP  101  may also support service discovery and selection of producer NF instances. SCP  101  may perform load balancing of connections between consumer and producer NFs. \\n     NRF  100  is a repository for NF or service profiles of producer NF instances. In order to communicate with a producer NF instance, a consumer NF or an SCP must obtain the NF or service profile of the producer NF instance from NRF  100 . The NF or service profile is a JavaScript object notation (JSON) data structure defined in 3GPP TS 29.510. The NF or service profile includes attributes that indicate the type of service provided, capacity of the NF instance, and information for contacting the NF instance. \\n     In  FIG.  1   , any of the network functions can be consumer NFs, producer NFs, or both, depending on whether they are requesting, providing, or requesting and providing services. In the illustrated example, the NFs include a policy control function (PCF)  102  that performs policy related operations in a network, a user defined management (UDM)  104  that manages user data, and an application function (AF)  106  that provides application services. \\n     The NFs illustrated in  FIG.  1    further include a session management function (SMF)  108  that manages sessions between access and mobility management function (AMF)  110  and PCF  102 . AMF  110  performs mobility management operations similar to those performed by a mobility management entity (MME) in 4G networks. An authentication server function (AUSF)  112  performs authentication services for user equipment (UEs), such as user equipment (UE)  114 , seeking access to the network. \\n     A network slice selection function (NSSF)  116  provides network slicing services for devices seeking to access specific network capabilities and characteristics associated with a network slice. A network exposure function (NEF)  118  provides application programming interfaces (APIs) for application functions seeking to obtain information about Internet of things (IoT) devices and other UEs attached to the network. NEF  118  performs similar functions to the service capability exposure function (SCEF) in 4G networks. \\n     A radio access network (RAN)  120  connects user equipment (UE)  114  to the network via a wireless link. Radio access network  120  may be accessed using a g-Node B (gNB) (not shown in  FIG.  1   ) or other wireless access point. A user plane function (UPF)  122  can support various proxy functionality for user plane services. One example of such proxy functionality is multipath transmission control protocol (MPTCP) proxy functionality. UPF  122  may also support performance measurement functionality, which may be used by UE  114  to obtain network performance measurements. Also illustrated in  FIG.  1    is a data network (DN)  124  through which UEs access data network services, such as Internet services. \\n     SEPP  126  filters incoming traffic from another PLMN and performs topology hiding for traffic exiting the home PLMN. SEPP  126  may communicate with a SEPP in a foreign PLMN which manages security for the foreign PLMN. Thus, traffic between NFs in different PLMNs may traverse two SEPP functions, one for the home PLMN and the other for the foreign PLMN. \\n     As stated above, one problem that can occur in 5G and other networks is that requests for Nnrf service operations are routed from the NRF in the visited network to the NRF in the home network for processing, and responses to the requests are routed from the NRF in the home network to the NRF in the visited network, resulting in latency in performing Nnrf service operations. 3GPP TS 29.510 defines an inter-PLMN routing mechanism for 5G when a subscriber is roaming in a non-home network, which becomes a visited network or serving network with respect to the subscriber. In order to serve subscribers, consumer NFs send requests for 3GPP-defined Nnrf service operations towards the visited NRF, and the visited NRF forwards the requests to the home NRF through the visited SEPP and the home SEPP. In the current mechanism, the visited NRF sends every inter-PLMN Nnrf service request coming from multiple consumer NFs serving multiple roaming subscribers towards the home NRF, which leads to consumption of a significant amount of network bandwidth and resources, causing a multifold increase in service latency. As used herein, the phrase “inter-PLMN request” refers to a request for a service operation that identifies a PLMN that is different from the PLMN of the NRF that receives the request. \\n     The subject matter described herein includes a mechanism in which, rather than forwarding every inter-PLMN Nnrf service request towards the home NRF, the visited NRF forwards only required/selected Nnrf service request towards the home NRF, which significantly increases throughput and reduces service latency. \\n     In order to serve subscribers in a network in which a subscriber is roaming, consumer network functions send requests for 3GPP-defined Nnrf service operations towards a visited/serving NRF, and the visited/serving NRF forwards the requests to the home NRF through the visited SEPP and the home SEPP. 3GPP TS 29.510 Section 5.3.2.2.3 defines how a request for the NFDiscovery service operation is routed from an NRF in a serving PLMN towards the home PLMN.  FIG.  2    illustrates exemplary messages exchanged in sending a request for an NFDiscovery service operation to the NRF in the home PLMN. Referring to  FIG.  2   , in line 1, visited NRF  100 A sends an NFDiscovery request to home NRF  100 B in the home PLMN of a subscriber. Home NRF  100 B responds in line  2   a  with a  200  OK message containing the NF profiles matching the query parameters in the NFDiscovery request if the processing of the request is successful. If the processing of the request is not successful or the request is redirected, NRF  100 B responds as indicated in line  2   b  with a 4xx or 5xx message including problem details or a 3xx message indicating redirection. \\n     Similar to the NFDiscovery service operations, other service operations regarding roaming subscribers are covered in the following sections of 3GPP TS 29.510:\\n     5.2.2.5.3 Subscription to NF Instances in a different PLMN   5.2.2.7.3 Subscription removal in a different PLMN   5.2.2.6.3 Notification from an NRF in a different PLMN   \\n It should be noted that for these service operations, the NRF in visited/serving PLMN remains in the path between the consumer NF and the home PLMN.\\n       FIG.  3    is a network diagram illustrating inter-PLMN routing of messages relating to an inter-PLMN Nnrf NFDiscovery service operation. Referring to  FIG.  3   , a visited or serving PLMN is assumed to be the network serving a mobile subscriber. The visited PLMN includes a visited NRF (vNRF)  100 A, a visited SEPP (vSEPP)  126 A, and one or more consumer NFs  300 . The home PLMN includes a home SEPP (hSEPP)  126 B, home NRF (hNRF)  100 B, and one or more producer NFs  302 . The steps for providing inter-PLMN Nnrf_NFDiscovery service illustrated in  FIG.  3    are as follows: \\n     1. One of consumer NFs  200  sends to visited NRF  100 A an Nnrf_NFDiscovery request to discover NF profiles of producer NFs  302  so that producer NFs  302  can serve subscribers in the visited/serving PLMN. \\n     2. vNRF  100 A forwards the request for Nnrf_NFDiscovery service towards hNRF  100 B, which is in the home PLMN. The request for Nnrf_NFDiscovery service is routed through vSEPP  126 A and hSEPP  126 B. \\n     3. hNRF  100 B processes the request for Nnrf_NFDiscovery service and responds to vNRF  100 B with producer NF profiles. The response is routed through hSEPP  126 B and vSEPP  126 A. \\n     4. vNRF  100 A sends producer NF profiles towards consumer NFs  300 . \\n     As illustrated by the example in  FIG.  3   , visited/serving NRF  100 A forwards every inter-PLMN Nnrf service request towards the home NRF  100 B through visited SEPP  126 A and home SEPP  126 B. Other types of Nnrf service requests, such as requests for NFStatusSubscribe and AccessToken service, require visited/serving NRF  100 A to forward the service requests towards the home NRF  100 B. The forwarding of service requests to the home network consumes a significant amount of network bandwidth and resources. High latency can be observed because of various network nodes, including the WAN network, while processing the inter-PLMN routing messages. The inter-PLMN forwarding of Nnrf service requests and responses can adversely impact the service experience of a 5G subscriber. \\n     In order to address these issues, the subject matter described herein includes using the visited NRF to identify requests for Nnrf service operations that can be processed locally at the visited NRF and processing the identified requests at the visited NRF without forwarding the requests to the home NRF. The subject matter described herein also includes caching data from the home NRF at the visited NRF and using the data to process requests for inter-PLMN Nnrf service operations, such as the Nnrf NFDiscovery, AcessToken, and StatusSubscribe service operations. For example, during inter-PLMN NFDiscovery service operation handling, the visited/serving NRF receives NF profiles from the home NRF. The visited/serving NRF stores the resulting NF profiles into a database local to the visited NRF. When the same consumer NF or other consumer NFs of the same or different NF type as the original requestor of the NFDiscovery service operation requests an Nnrf service operation, NF profiles which are saved locally at the visited NRF are first checked to see whether they can be used to respond to the request. In the case where the visited/serving NRF can process the service request locally, the visited/serving NRF sends a response including the NF profiles to the consumer NF. If the visited/serving NRF cannot process the request locally, the visited/serving NRF can fall back to the existing home NRF communication based mechanism. The visited/serving NRF can perform similar operations to process requests for inter-PLMN AccessToken and NFStatusSubscribe service operations. Once the visited/serving NRF has built up its local data store, it can use the 3GPP-defined NFStatusSubscribe to subscribe to receive notification of updates regarding NF profiles saved in its local database to keep them updated with respect to the home NRF. The visited/serving NRF may also proactively refresh its local database using the 3GPP-defined NFDiscovery service operation when the last searched NFDiscovery validity time has elapsed. \\n       FIG.  4    is a network diagram illustrating exemplary operations performed by vNRF  100 A in processing an initial request for an Nnrf service operation and building a database local to the vNRF  100 A to process subsequent requests for Nnrf service operations. Referring to  FIG.  4   , the following operations are performed: \\n     1. One of consumer NFs  300  generates a request for an inter-PLMN NFDiscovery service operation and sends the request to vNRF  100 A.   2. vNRF  100 A determines that it cannot process the request locally because it does not have the requested NF profile information. Accordingly, vNRF  100 A forwards the request towards hNRF  100 B, which is in the home PLMN.   3. hNRF  100 B processes the service request and sends producer NF profiles in a response towards vNRF  100 A.   4. vNRF  100 A saves the returned NF profiles in an inter-PLMN Nnrf service operation message processing database  400  local to vNRF  100 A.   5. vNRF  100 A sends producer NF profiles towards consumer NFs  300 .   \\n\\n       FIG.  5    is a network diagram illustrating processing of a request for an inter-PLMN Nnrf_NFDiscovery service operation using data cached locally at Nnrf  100 A. Referring to  FIG.  5   , the following operations are performed: \\n     1. One of consumer NFs  300  generates and sends a request for an NFDiscovery service operation and sends the request to vNRF  100 A.   2. vNRF  100 A checks inter-PLMN Nnrf service operation message processing database  400  and determines that it has the NF profiles requested by the request for the NFDiscovery service operation.   3. vNRF  100 A generates and sends a response to the request for the NFDiscovery service operation to the requesting consumer NF and includes the NF profiles extracted from inter-PLMN Nnrf service operation message processing database  400  local to vNRF  100 A in the NF discovery response.   \\n\\n     In addition to building inter-PLMN Nnrf service operation message processing database  400  from responses to requests for Nnrf service operations and using the local data to respond to requests for Nnrf service operations, vNRF  100 A may also perform operations to keep inter-PLMN Nnrf service operation message processing database  400  synchronized with changes in NF profiles at hNRF  100 B.  FIG.  6    is a network diagram illustrating exemplary operations performed by vNRF  100 A in keeping inter-PLMN Nnrf service operation message processing database  400  synchronized with changes in NF profiles maintained by hNRF  100 B. In  FIG.  6   , vNRF  100 A performs the following operations: \\n     1. Once vNRF  100 A has added NF profiles to inter-PLMN Nnrf service operation message processing database  400 , vNRF  100 A can use 3GPP-defined NFStatusSubscribe service operation to subscribe with hNRF  100 B for NF profiles saved in inter-PLMN Nnrf service operation message processing database  400  to keep them updated with respect to hNRF  100 B.   2. vNRF  100 A may also proactively refresh inter-PLMN Nnrf service operation message processing database  400  using the 3GPP-defined NFDiscovery service operation when the last searched NF discovery validity time is elapsed.   \\n\\n       FIG.  7    is a flow chart illustrating an exemplary process for generating a local database at a visited NRF for processing inter-PLMN Nnrf service requests. Referring to  FIG.  7   , in step  700 , a consumer NF determines that it needs NF profiles to serve a subscriber in a visited or serving PLMN. For example, an AMF in a serving or visited PLMN may determine that it needs NF profiles of UDMs in the subscriber’s home network to provide communications service to the subscriber. \\n     In step  702 , the consumer NF generates and sends a request for an NFDiscovery service operation to the NRF local to the consumer NF. \\n     In step  704 , the serving NRF determines that the request is a request concerning a roaming subscriber and that the serving NRF does not have the NF profiles corresponding to the query parameters in the request for the NFDiscovery service operation. Accordingly, the serving NRF forwards the request for the NFDiscovery service operation to the home NRF. \\n     In step  706 , the home NRF processes the request for the NF discovery service operation, extracts the NF profiles from its NF profiles database, and generates and sends a response including the NF profiles to the visited or serving NRF. \\n     In step  708 , the visited or serving NRF sends the NF profiles to the requesting consumer NF. The visited NRF also stores the returned NF profiles in its local NF profiles database. \\n     In step  710 , the consumer NF serves the mobile subscriber according to 3GPP-defined operations. Continuing with the AMF example, the AMF serving the subscriber may select a UDM from the UDM profiles received from the visited NRF, contact the UDM to obtain subscription information regarding a roaming mobile subscriber, and use the subscription information to provide communications services to the mobile subscriber. \\n       FIG.  8    is a flow chart illustrating an exemplary process for using a local database at a visited NRF to process inter-PLMN Nnrf service requests. Referring to  FIG.  8   , in step  800 , a consumer NF determines that it needs NF profiles to serve a subscriber in a visited or serving PLMN. For example, an AMF in a serving or visited PLMN may determine that it needs NF profiles of UDMs in the subscriber’s home network to provide communications service to the subscriber. \\n     In step  802 , the consumer NF generates and sends a request for an NFDiscovery service operation to the NRF local to the consumer NF. The request includes query parameters, such as NF type, that the NRF can use to identify NF profiles of producer NFs to provide in response to the service request. \\n     In step  804 , the serving NRF determines that the request is a request concerning a roaming subscriber. For example, the serving NRF may read the PLMN specified in the request and determine that the PLMN does not match the PLMN of the serving NRF. \\n     In step  806 , the serving NRF checks whether the serving NRF has the NF profiles corresponding to the query parameters in the request for the NFDiscovery service operation. If the serving NRF does not have the NF profiles, control proceeds to step  808  where the serving NRF sends the NFDiscovery request to the home NRF. If the serving NRF has the NF profiles to satisfy the NFDiscovery request, control proceeds to step  810  where the serving NRF obtains the NF profiles from its local database and generates and sends a response to the NFDiscovery request, including the NF profiles, to the consumer NF. In step  812 , the consumer NF uses the NF profiles to initiate 3GPP service operations. \\n     Once the local database is created at the serving or visited NRF, the serving or visited NRF may perform active operations to maintain the database, i.e., make sure that the database is synchronized with the NF profiles or other database maintained by the home NRF.  FIGS.  9 A and  9 B  are a flow chart illustrating an exemplary process for synchronizing the local database at the visited NRF with the database at the home NRF. Referring to  FIG.  9 A , in step  900 , the visited NRF uses the 3GPP-defined NFStatusSubscribe service operation to subscribe to receive notification of changes in NF profiles maintained by the home NRF.  FIG.  10    is a message flow diagram illustrating exemplary messages exchanged in the NFStatusSubscribe service operation. Referring to  FIG.  10   , in line 1, visited NRF  100 A sends an HTTP POST message to NRF  100 B located in the home PLMN. The HTTP POST message includes subscription data, which in this example, includes the NF instance IDs of the NF instances whose NF profiles are stored in the local database maintained by visited NRF  100 A. In line  2   a , NRF  100 B located in the home PLMN responds with a  201  Created message indicating that the subscription was successfully created and includes subscription data, which will include any updates to the NF profiles identified by the subscription data in the subscription request. If creation of the subscription is not successful or if the request for creating the subscription is redirected, NRF  100 B located in the home PLMN responds as indicated in line  2   b  with a 4xx or 5xx message with problem details or a 3xx message in the redirection case. \\n     Returning to  FIG.  9 A , in step  902 , the home NRF informs the visited NRF of any changes in the NF profiles identified in the subscription request. Such changes can include deregistration or any change to the NF profile database. Home NRF  100 B may inform visited NRF  100 A of changes in the NF profiles by transmitting an NFStatusNotify message to visited NRF  100 A. \\n     In step  904 , visited NRF  100 A updates inter-PLMN Nnrf service operation message processing database  400  using the information obtained from home NRF  100 B. Such changes may include deregistration of NF profiles or change in NF profile parameter values of the NF profiles in inter-PLMN Nnrf service operation message processing database  400  maintained by visited NRF  100 A. Visited NRF  100 A may used the updated NF profile parameters to respond locally to NFDiscovery and other requests for inter-PLMN Nnrf service. \\n     In addition to using the NFStatusSubscribe service operation to maintain synchronization between inter-PLMN Nnrf service operation message processing database  400  and the database maintained by home NRF  100 B, visited NRF  100 A may also use the validity timer specified for the NFDiscovery service operation to obtain updated NF profile data from the home NRF. According to Section 6.2.2.2.3 of 3GPP TS 29.510, a response to a request for an NFDiscovery service operation includes a cache control header which includes a max-age value, which is an amount of time in seconds after which the received response is considered stale. Visited NRF  100 A may use this value to determine when to update inter-PLMN Nnrf service operation message processing database  400 . Referring to  FIG.  9 B , in step  906 , visited NRF  100 A detects expiration of a validity period for an NFDiscovery service operation and transmits and NFDiscovery request to the home NRF. Detecting expiration of the validity period may include determining that a response received from home NRF  100 B is stale because the current time is past the validity period. The NFDiscovery request may include the same query parameters as the NFDiscovery request for which the validity timer expired. \\n     In step  908 , the home NRF responds to the visited NRF with updated NF profile information. For example, home NRF  100 B may respond to the NRF discovery request with NF profiles matching query parameters in the NF discovery request. \\n     In step  910 , visited NRF  100 A updates inter-PLMN Nnrf service operation message processing database  400  with the updated NF profile information received from home NRF  100 A. Visited NRF may use the updated NF profile information to respond to inter-PLMN requests for Nnrf service. \\n     As indicated above, in addition to using its local database to respond to inter-PLMN NFDiscovery requests, visited NRF  100 A may also use its local database to respond to a request for an inter-PLMN NFStatusSubscribe service operation and an AccessToken service operation. In using inter-PLMN Nnrf service operation message processing database  400  to respond to a request for an inter-PLMN NFStatusSubscribe service operation, visited NRF  100 A may perform the steps illustrated in  FIGS.  8 ,  9 A, and  9 B , which include determining if the request for the inter-PLMN NFStatusSubscribe service operation can be handled locally by visited NRF  100 A. If the request can be handled locally by visited NRF  100 A, visited NRF  100 A creates the subscription to the requested resource in its local database and subscribes with home NRF  100 B to be notified of changes in the requested resource. The messaging exchanged between visited NRF  100 A and home NRF  100 B is the same as that described above with respect to  FIG.  10   . When visited NRF  100 A is informed of a change in a resource corresponding to a subscription, visited NRF  100 A will inform the consumer NFs subscribed to be notified of the changes. \\n     In processing requests for inter-PLMN Nnrf_AccessToken service using inter-PLMN Nnrf service operation message processing database  400  , visited NRF  100 A may perform the steps described above with regarding  FIGS.  8 ,  9 A, and  9 B , except the local database and the responses will include OAuth2 access tokens instead of NF profiles. Processing requests for Nnrf_AccessToken service includes determining if a request for an inter-PLMN Nnrf_AccessToken service operation can be handled locally by visited NRF  100 A. If the request can be handled locally by visited NRF  100 A, visited NRF  100 A responds with an access token response, which includes the OAuth2 access token requested by the consumer NF. The messaging exchanged between visited NRF  100 A and a consumer NF  300  for the case where visited NRF  100 A has the requested access token of the remote producer NF is illustrated in  FIG.  11   . Referring to  FIG.  11   , in line 1, consumer NF  300  sends an access token request to visited NRF  100 A. In line  2   a , after retrieving the requested access token from inter-PLMN Nnrf service operation message processing database  400 , visited NRF  100 A responds with a  200  OK message including the requested access token. If an error occurs or the request is redirected, visited NRF  100 A responds as indicated in step  2   b  with a  400  Bad Request or a 3xx message indicating that the request has been redirected. If the access token request is redirected to home NRF  100 B, home NRF  100 B may respond to the access token request with the requested access token. Visited NRF  100 A may forward the access token response to consumer NF  300 . Visited NRF  100 A may also store the access token in inter-PLMN Nnrf service operation message processing database  400  so that visited NRF  100 A can respond to a subsequent access token request for the same access token from the same or a different consumer NF. \\n       FIG.  12    is a block diagram illustrating an exemplary system for reducing inter-PLMN forwarding of messages relating to Nnrf service operations. In  FIG.  12   , the system includes a first network function (NF) repository function (NRF)  100 A including at least one processor  1200  and a memory  1202 . NRF  100 A further includes an inter-PLMN Nnrf service operation message handler  1204  implemented by processor  1200  for receiving a first request for an Nnrf service operation, determining that the first request identifies a PLMN different from a PLMN of the first NRF, determining that the first request cannot be processed locally at the first NRF and, in response, forwarding the first request to a second NRF in the PLMN identified in the first request, receiving, from the second NRF, a response to the first request, caching information from the response, receiving a second request for an Nnrf service operation, determining that the second request identifies a PLMN different from a PLMN of the first NRF, determining that the second request can be processed locally at the first NRF, and processing the second request locally at the first NRF using the cached information from the first response. NRF  100 A further includes local inter-PLMN NF service operation message processing database  400  stored in memory  1202  and including the locally cached information (NF profile information, access tokens, etc.) for processing requests for inter-PLMN Nnrf service operations. \\n       FIG.  13    is a flow chart illustrating an exemplary process for reducing inter-PLMN forwarding of messages relating to Nnrf service operations. Referring to  FIG.  13   , in step  1300 , the process includes receiving, at a first NRF, a first request for an Nnrf service operation. For example, visited NRF  100 A may receive a request for an Nnrf service operation, such as an NFDiscovery service operation, an NFStatusSubscribe service operation, or an AccessToken service operation. \\n     In step  1302 , the process includes determining that the first request identifies a PLMN different from a PLMN of the first NRF. For example, visited NRF  100 A may read the PLMN from a query parameter of the first request and determine that the PLMN is not the same as the PLMN of NRF  100 A. \\n     In step  1304 , the process includes determining that the first request cannot be processed locally at the first NRF. For example, if the first request is an NFDiscovery or NFStatusSubscribe request, NRF  100 A may determine that the request cannot be processed locally by NRF  100 A because NRF  100 A does not have NF profiles identified by query parameters in the NFDiscovery or NFStatusSubscribe request. If the first request is an AccessToken request, NRF  100 A may determine that the first request cannot be processed locally by NRF  100 A because NRF  100 A does not have the requested access token. \\n     In step  1306 , the process includes forwarding the first request to a second NRF in the PLMN identified in the first request. For example, visited NRF  100 A may, in response to determining that it cannot process the first request for an Nnrf service operation, forward the request to the PLMN identified in the first request. \\n     In step  1308 , the process includes receiving, from a second NRF, a response to the first request and caching information from the response. For example, visited NRF  100 A may receive a response to the first request from home NRF  100 B and store NF profile or access token information in local inter-PLMN Nnrf service operation message processing database  400 . \\n     In step  1310 , the process includes receiving a second request for an Nnrf service operation. For example, visited NRF  100 A may receive a request for the same or a different Nnrf service from that requested by the first request. \\n     In step  1312 , the process includes determining that the second request identifies a PLMN other than the PLMN of the first NRF. For example, visited NRF  100 A may determine that the PLMN identified in the query parameters of the second request does not match the configured PLMN of visited NRF  100 A \\n     In step  1314 , the process includes determining that the second request can be processed locally at the first NRF. For example, visited NRF  100 A may access local inter-PLMN Nnrf service operation message processing database  400  using query parameters from the second request, determine that database  400  includes the requested NF profile or access token information, and, as a result, determine that the second request can be processed locally by visited NRF  100 A. \\n     In step  1316 , the process includes processing the second request locally at the first NRF using the cached information from the first response. For example, visited NRF  100 A may generate and send a response to the second request using the information from local inter-PLMN Nnrf service operation message processing database  400  without forwarding the second request to the PLMN identified in the second request. \\n     The disclosure of each of the following references is incorporated herein by reference in its entirety. \\n     References \\n     1. 3 rd  Generation Partnership Project; Technical Specification Group Core Network and Terminals; 5G System; Network Function Repository Services; Stage 3 (Release 17) 3GPP TS 29.510 V17.3.0 (2021-09) \\n     It will be understood that various details of the subject matter described herein may be changed without departing from the scope of the subject matter described herein. Furthermore, the foregoing description is for the purpose of illustration only, and not for the purpose of limitation, as the subject matter described herein is defined by the claims as set forth hereinafter.', 'id': 'US-2023188963-A1', 'ti': 'METHODS, SYSTEMS, AND COMPUTER READABLE MEDIA FOR REDUCING INTER-PUBLIC LAND MOBILE NETWORK (PLMN) FORWARDING OF MESSAGES RELATING TO Nnrf SERVICE OPERATIONS'}>,\n",
       " <Hit {'ab': 'A non-transitory computer-readable medium including a circulation control program for causing a server device controlling a progress of making each of the plurality of users circulate a plurality of destinations to implement the following functions including an update function of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user., A non-transitory computer-readable medium including a circulation control program for causing a server device controlling a progress of making each of the plurality of users circulate a plurality of destinations to implement the following functions including an update function of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user.', 'clm': 'What is claimed is: \\n     \\n         1 . A non-transitory computer-readable medium comprising a circulation control program causing a server device comprising a storage medium to execute functions, comprising:\\n connecting to a plurality of terminal devices through a communication network; and   controlling a progress of the plurality of terminal devices guiding a plurality of corresponding users circulate a plurality of destinations, including:\\n updating user information of each user of the plurality of users registered in the storage medium, the user information including history information specifying one or more already visited destinations and designation information specifying a designated destination to be visited; \\n determining the designated destination for each user of the plurality of users that each user has not visited yet based on the history information, and further according to a congestion status of the plurality of destinations; and \\n transmitting information on the designated destination to each terminal device of each user, \\n   wherein updating the user information comprises:\\n updating the history information of each user when information indicating visiting the designated destination is received from each terminal device of each user; and \\n updating the designation information of each user when the designated destination of each user is determined. \\n   \\n     \\n     \\n         2 . The non-transitory computer-readable medium according to  claim 1 , wherein determining the designated destination for each user includes determining one or more candidate destinations from among the one or more destinations that one user has not visited yet according to the congestion status of the plurality of destinations, and determining the designated destination selected by each user from among the candidate destinations. \\n     \\n     \\n         3 . The non-transitory computer-readable medium according to  claim 2 , wherein\\n the candidate destination of each user is a designated destination of a smallest number of users among the destinations that the one user has not visited yet.   \\n     \\n     \\n         4 . The non transitory computer-readable medium according to  claim 1 , wherein\\n the designated destination of each user is a designated destination of a smallest number of users among the destinations that each user has not visited yet.   \\n     \\n     \\n         5 . The non-transitory computer-readable medium according to  claim 1 , wherein\\n the information on the designated destination includes information for notifying a user of the terminal device of a question having an answer indicative of the designated destination.   \\n     \\n     \\n         6 . The non-transitory computer-readable medium according to  claim 1 , wherein\\n the terminal device is configured to receive position information by performing short-range wireless communication with each communication device at each destination of a plurality of destinations, each communication device storing the position information of each destination, and   wherein the information indicating visiting the designated destination includes the position information of the designated destination.   \\n     \\n     \\n         7 . The non-transitory computer-readable medium according to  claim 1 , the functions further comprises:\\n determining a new designated destination of each user upon a receipt of the information indicating visiting the designated destination from the terminal device of each user.   \\n     \\n     \\n         8 . A circulation control system comprising:\\n a plurality of terminal devices configured to guide a plurality of users of the plurality of corresponding terminal devices circulate a plurality of destinations;   a server device configured to connect to the plurality of terminal devices through a communication network and further configured to control a progress of the plurality of terminal devices guiding the plurality of corresponding users circulate the destinations;   a storage medium configured to register user information of each user of the plurality of users;   an updater configured to update the user information of each user of the plurality of users, including history information specifying one or more already visited destinations and designation information specifying a designated destination to be visited; and   a designator configured to determine the designated destination for each user of the plurality of users that each user has not visited yet based on the history information of each user and further according to a congestion status of the plurality of destinations, and further configured to transmit information on the designated destination to each terminal device of each user,   wherein the updater is configured to update the history information of each user when information indicating visiting the designated destination is received from each terminal device of each user, and further configured to update the designation information of each user when the designated destination of the each user is determined by the designator.   \\n     \\n     \\n         9 . A non-transitory computer-readable medium comprising:\\n a communication control program causing a terminal device to execute functions, comprising:   communicating with a server device that is configured to control a progress of the terminal device guiding a user of the terminal device;   receiving information on a designated destination from the server device, the server being configured to:   update user information of each of a plurality of users registered in a storage medium, the user information including history information specifying one or more already visited destinations and designation information specifying a designated destination to be visited,   determine the designated destination for each user of the plurality of users that each user has not visited yet based on the history information, and further according to a congestion status of the plurality of destinations, and   transmit information on the designated destination to each terminal device of each user, wherein   the server is further configured to:\\n update the history information of each user when information indicating visiting the designated destination is received from each terminal device of each user, and \\n update the designation information of each user when the designated destination of each user is determined.', 'cpc': 'A63F13/216', 'detd': 'CROSS-REFERENCE TO RELATED APPLICATION(S) \\n     This application claims priority to and the benefit of Japanese Patent Application No. 2020-131447, filed on Aug. 3, 2020, the disclosure of which is expressly incorporated herein by reference in its entirety for any purpose. \\n     BACKGROUND \\n     At least one of the embodiments of the present invention relates to a non-transitory computer-readable medium including a circulation control program, and a non-transitory computer-readable medium including a circulation control system and a communication control program. \\n     In recent years, smart devices such as smartphones and tablet terminals that can be carried by users have become widespread. Events such as a treasure hunt game using such smart devices are also actively held. The treasure hunt game is an event in which participants solve a plurality of mysteries (questions) and search for treasure hidden in the real space. The treasure is disposed at a predetermined position in the real space. The participants can specify where the treasure is disposed by solving a plurality of mysteries. In the treasure hunt game, the participants need to go around a plurality of places on foot or the like to acquire all the hidden treasures. The above-described event is a circulating type (migratory type) event in which the participants go around a plurality of places (destination). \\n     Since the participants go around a plurality of places, the above-described events are held at commercial facilities such as a shopping street for the purpose of sales promotion. Accordingly, the place (destination) where the treasure or the like is disposed may also be a part of a region of a certain store. An example of related art may be found in JP-A-2004-126689. \\n     SUMMARY \\n     In a case where participants go around a plurality of places (destinations) as described above, or in a case where the participants go around each destination in a predetermined order, when the participants all visit the same destination at the same time, there is a possibility that the destination may be congested. There is a demand to avoid congestion as much as possible since a large space is not always secured in a place set as a destination. \\n     An object of at least one embodiment of the present disclosure is to solve a shortage of related technology. \\n     According to a non-limiting viewpoint, a non-transitory computer-readable medium according to an embodiment of the present disclosure includes a circulation control program for causing a server device that is connected to a terminal device carried by each of a plurality of users through a communication network, the server device controlling a progress of making each of the plurality of users circulate a plurality of destinations, to implement the following functions including an update function updating user information of each plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the update function may include a function of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and a function of updating the designation information of one user when the designated destination of the one user is determined by the designation function, and the designation function may include a function of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     According to a non-limiting viewpoint, a circulation control system according to an embodiment of the present disclosure is a circulation control system for controlling a progress of making each of a plurality of users circulate a plurality of destinations, the circulation control system including a terminal device carried by each of the plurality of users, and a server device connected to the terminal device through a communication network, the circulation control system including an updater configured to update user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designator configured to determine the designated destination for each of a plurality of users, the designator determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the updater updates the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and updates the designation information of one user when the designated destination of the one user is determined by the designator, and the designator determines the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     According to a non-limiting viewpoint, a non-transitory computer-readable medium according to an embodiment of the present disclosure includes a communication control program for causing a terminal device carried by each of a plurality of users to implement the following functions, for communicating with a server device having a function of controlling a progress of making each of the plurality of users circulate a plurality of destinations, the program causing the terminal device to implement a function of receiving information on a designated destination from the server device having the following functions including an update function of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the update function may include a function of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and a function of updating the designation information of one user when the designated destination of the one user is determined by the designation function, and the designation function may include a function of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     Each embodiment of the present application solves one or more shortages. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF THE DRAWINGS \\n         FIG. 1  is a block diagram showing an example of a configuration of a circulation control system corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 2  is a functional block diagram showing a configuration of a server device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 3  is a flowchart showing an example of circulation processing corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 4  is a functional block diagram showing a configuration of a server device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 5  is a flowchart showing an example of circulation processing corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 6  is a block diagram showing an example of a configuration of a circulation control system corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 7  is a diagram showing a flow of actions of one user corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 8  is a functional block diagram showing a configuration of a server device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 9  is a diagram showing an example of registration data corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 10  is a diagram showing an example of a total result corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 11  is a functional block diagram showing a configuration of a terminal device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 12  is a flowchart showing an example of circulation processing corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 13  is a block diagram showing an example of a configuration of a circulation control system corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 14  is a diagram showing a flow of actions of one user corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 15  is a functional block diagram showing a configuration of a server device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 16  is a diagram showing an example of registration data corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 17  is a functional block diagram showing a configuration of a terminal device corresponding to at least one of the embodiments of the present disclosure. \\n         FIG. 18  is a flowchart showing an example of circulation processing corresponding to at least one of the embodiments of the present disclosure. \\n     \\n    \\n    \\n     DESCRIPTION OF EMBODIMENTS \\n     Hereinafter, examples of embodiments of the present disclosure will be described with reference to the drawings. The various components in the examples of each embodiment described below can be appropriately combined as long as there is no contradiction or the like. In addition, the contents described as an example of a certain embodiment may be omitted in the other embodiments. The contents of operations and processing not related to the characteristic parts of each embodiment may be omitted. Further, an order of the various processing constituting the various flows described below is in no particular order as long as there is no contradiction or the like in the processing contents. \\n     First Embodiment \\n       FIG. 1  is a block diagram showing an example of a configuration of a circulation control system  100  according to the embodiment of the present disclosure. As shown in  FIG. 1 , the circulation control system  100  includes a server device  10  and a plurality of terminal devices  20  and  201  to  20 N (N is a random integer). The server device  10  and the plurality of terminal devices  20  and  201  to  20 N are each connected to a communication network  15  such as the Internet. A configuration of the circulation control system  100  is not limited to this, and may be, for example, a configuration including a plurality of server devices. \\n     The circulation control system  100  has various functions for controlling a progress of making each of a plurality of users circulate a plurality of destinations. A plurality of destinations mean places in the real space that the user should visit. In addition to walking, public transportation may be used for movement. \\n     The circulation control system  100  of the example of the present embodiment is applied to, for example, a circulating type (migratory type) event such as a treasure hunt game. The treasure hunt game is an event in which a plurality of participants (users) solve a plurality of mysteries (questions) to search for treasure hidden in the real space. The treasure is a virtual treasure in the game which is hidden (disposed) in a plurality of places in the real space. The plurality of users can specify a place where the treasure is disposed by solving the plurality of questions. In the treasure hunt game, the plurality of users need to move on foot or the like to go around a plurality of places (destination) where the treasure is hidden and acquire all the treasures. \\n     In the example of the present embodiment, a circulation order of the plurality of destinations of each user is not fixed, and the circulation control system  100  determines (designates). Each time one user visits one destination, the circulation control system  100  determines a destination to which the one user should visit next. Hereinafter, with respect to one user, among a plurality of destinations, the destination to which the one user should visit may be referred to as a designated destination at present. In the case of the treasure hunt game described above, each time one user visits one place and acquires a treasure, the circulation control system  100  determines a place (designated destination) of the next treasure that the one user should search for. In the example of the present embodiment, the server device  10  determines the designated destination. In addition, the server device  10  transmits information on the determined designated destination to the terminal devices  20  and  201  to  20 N carried by each user. For example, in the case of the above-described treasure hunt game, information for notifying the user of a question indicating the place of the treasure corresponds to the information on the designated destination. Specifically, text data of the question sentence corresponds to the information on the designated destination. That is, in the case of the above-described treasure hunt game, each time one user visits one place and acquires a treasure, a question indicating a place (designated destination) of the next treasure that the one user should search for is delivered to a terminal device of the one user. \\n     For example, a user P 1  carrying the terminal device  20  moves toward a designated destination among a plurality of destinations. Then, the user transmits information indicating visiting the designated destination from the terminal device  20  to the server device  10 . For example, in the case of the above-described treasure hunt game, identification information of the treasure disposed at the designated destination is transmitted as the information indicating visiting the designated destination. \\n     In the example of the present embodiment, the server device  10  determines the designated destination to prevent many users from being temporarily crowded at one destination. Specifically, the server device  10  determines a designated destination of each user according to the congestion status of the plurality of destinations. The congestion status also includes a status that is expected to be congested. The congestion status of each destination is a status that is specified at least based on the information (designation information) on the destination (designated destination) to which each user should visit. \\n     For example, in the above-described treasure hunt game, a case where a plurality of users circulate three destinations A 1 , A 2 , and A 3  to acquire the treasure will be described. At present, the number of users who should visit the destination A 1  is 10, the number of users who should visit the destination A 2  is 5, and the number of users who should visit the destination A 3  is 3. In this case, the destination A 1  may be temporarily congested by gathering 10 users. Therefore, in a case of one user who circulates the three destinations A 1 , A 2 , and A 3  from now on, the server device  10  may determine the destination A 3 , which has the smallest number of users, as the first destination (designated destination) of the one user. \\n     The server device  10  is managed by an administrator of the game system, and has various functions for providing information on the progress of the circulation to the terminal devices  20  and  201  to  20 N. The server device  10  provides, for example, the information on the destination (designated destination) to which the user should visit. In the example of the present embodiment, the server device  10  is configured by an information processing device such as a WWW server to provide the information on the progress of the circulation, and includes one or more storage media (storage units) for storing various information. In addition, the server device  10  also includes a control unit and the like composed of a central processing unit (CPU) and the like. Since the above-described configuration is a general configuration, detailed description thereof will be omitted. The storage unit may have a configuration built in the server device  10  or may be a storage device such as a database device separate from the server device  10 . \\n       FIG. 2  is a functional block diagram showing a configuration of a server device  10 A, which is an example of the configuration of the server device  10 . The server device  10 A includes at least an update unit  11  and a designation unit  12  by executing software (a circulation control program) for controlling the progress of the circulation stored in the storage unit by the control unit. The circulation control program also includes control data used to control the progress of the circulation. \\n     The update unit (update function)  11  updates the user information of each of the plurality of users registered in the storage unit of the server device  10 A. The user information is information on the circulation of a plurality of destinations of the user (refer to, for example, the registration data shown in  FIG. 9 ). The user information includes history information specifying an already visited destination and designation information specifying a destination (designated destination) to be visited. The history information is, for example, identification information of the already visited destination. The designation information is, for example, identification information of the destination to be visited. \\n     When receiving the information indicating visiting the designated destination from the terminal devices  20  and  201  to  20 N of any user, the update unit  11  updates the received history information of the user of the terminal device. When the designated destination of a certain user is determined by the designation unit  12 , the update unit  11  updates the designation information of the user. \\n     The designation unit (designation function)  12  determines a designated destination for each of a plurality of users. Specifically, the designation unit  12  determines, based on history information of one user, one destination that the one user has not visited yet among the plurality of destinations as a designated destination of the one user. However, the designation unit  12  determines a designated destination of each user according to the congestion status of the plurality of destinations. That is, the designation unit  12  determines a designated destination of one user according to the congestion status of a plurality of destinations among the destinations that the one user has not visited yet. As described above, the congestion status of each destination is a status that is specified at least based on the information (designation information) on the destination (designated destination) to which each user should visit. As in the above example, the designation unit  12  may determine a destination with the smallest number of users to be visited as a designated destination. \\n     In addition, the designation unit  12  transmits information on the designated destination determined for one user to the terminal device of the one user. In the case of the above-described treasure hunt game, the information on the designated destination corresponds to the information for notifying the user of a question whose answer indicates the designated destination (treasure place), such as the text data of the sentence of the question. In addition, a name (text data) of the designated destination may be used as information on the designated destination. The information on the designated destination may be included in the control data in association with the identification information of the destination, for example. \\n     Next, the terminal devices  20  and  201  to  20 N will be described. Each of the terminal devices  20  and  201  to  20 N is constituted by a user-portable terminal device such as a personal computer, a mobile phone terminal such as a smartphone, a personal digital assistants (PDA), or a tablet terminal. Accordingly, each of the terminal devices  20  and  201  to  20 N moves with the movement of the user carrying the terminal device. \\n     Each terminal device  20  and  201  to  20 N includes the operation device (operation unit), the storage medium (storage unit), the control unit composed of the CPU, the display unit, the communication control unit, and the like. Since the above-described configuration is a general configuration, detailed description thereof will be omitted. In addition, software (application programs) and the like for performing various communications with the server device  10 A regarding the circulation of the destination are stored in the storage units of the terminal devices  20  and  201  to  20 N. By executing the above-described program, the terminal devices  20  and  201  to  20 N receive, for example, the information on the designated destination from the server device  10 A. In this case, for example, the information on the designated destination is displayed on the display units of the terminal devices  20  and  201  to  20 N. In the case of the above-described treasure hunt game, a question sentence is displayed. \\n     In addition, the terminal devices  20  and  201  to  20 N transmit the information indicating visiting the designated destination to the server device  10 A. As for the information indicating visiting the designated destination, any information can be specified as long as it is information that can specify that the user has visited the designated destination. For example, in the case of the above-described treasure hunt game, the identification information of the treasure acquired at the designated destination may be used as the information indicating visiting the designated destination. \\n     Next, as an example of a configuration of the circulation control system  100 , an operation of a circulation control system  100 A including the server device  10 A and the plurality of terminal devices  20  and  201  to  20 N will be described. \\n       FIG. 3  is a flowchart showing an example of circulation processing executed by the circulation control system  100 A. In the circulation processing in the example of the present embodiment, processing for determining a designated destination is performed for each of a plurality of users.  FIG. 3  mainly describes processing of determining a designated destination and processing of updating user information, and some other processing may be omitted. Hereinafter, a case where the server device  10 A and the terminal device  20  carried by the user P 1  execute the circulation processing will be described as an example. \\n     The circulation processing of the example of the present embodiment is executed, for example, when the server device  10 A receives predetermined information from any of the terminal devices  20  and  201  to  20 N. \\n     The predetermined information corresponds to, for example, information on a designated destination visited by the user. In addition, the predetermined information corresponds to, for example, information of a circulation start request for the user to start the circulation of a plurality of destinations from now on. The terminal device  20  transmits the predetermined information together with the identification information of the user P 1 , for example. \\n     The server device  10 A executes update processing A (step S 10 ). In the update processing A, the user information registered in the storage unit of the server device  10 A is updated based on the received predetermined information. For example, when receiving information on the designated destination visited by the user P 1  from the terminal device  20 , the server device  10 A updates the history information among the user information of the user P 1 . For example, identification information of the designated destination visited by the user P 1  is added to the history information of the user P 1 . Then, the designation information of the user P 1  is reset (set to an initial value). In addition, for example, when the user P 1  receives the circulation start request from the terminal device  20 , the server device  10 A newly registers the user information of the user P 1  in the storage unit of the server device  10 A. For example, user information of user P 1  including the identification information of user P 1 , history information in which the initial value is set, and designation information is generated. \\n     Next, the server device  10 A executes extraction processing (step S 11 ). In the extraction processing, a candidate destination as a candidate of the designated destination is extracted. Specifically, based on the history information of the user P 1 , destinations that the user P 1  has not visited yet are extracted as a candidate destination. \\n     Thereafter, the server device  10 A executes determination processing (step S 12 ). In the determination processing, as described above, one candidate destination selected from among the above-described candidate destinations is determined as a designated destination according to the congestion status. That is, the destination to which the user P 1  should visit is determined as a designated destination. \\n     In addition, in the determination processing, the server device  10 A transmits information on the determined designated destination to the terminal device  20 . For example, in the case of the above-described treasure hunt game, the text data of the sentence of the question is transmitted. The terminal device  20  outputs (displays on the display unit) the received information on the designated destination (step S 30 ). For example, in the case of the above-described treasure hunt game, a question sentence is displayed on the display unit of the terminal device  20 . \\n     After that, the server device  10 A executes update processing B (step S 13 ). In the update processing B, the designation information of the user information is updated. For example, the identification information of the designated destination determined in the determination processing of step S 12  is registered as the designation information. \\n     After that, the server device  10 A ends the circulation processing. Regarding the circulation processing shown in  FIG. 3 , the same processing as that of the terminal device  20  is performed on the other terminal devices  201  to  20 N. \\n     As described above, as one aspect of the first embodiment, since the server device  10 A is configured to include the update unit  11  and the designation unit  12 , the destination to which the user should visit is set according to the congestion status. Accordingly, it is possible to prevent many users from being crowded and the destination being congested. Therefore, it is effective as an infection control of viruses and the like. \\n     The circulation control system of the example of the above-described embodiment is not limited to application to a circulating type (migratory type) event such as a treasure hunt game. The circulation control system can be applied to any configuration as long as it is configured to allow each of a plurality of users to circulate a plurality of destinations. For example, in an attraction facility such as an amusement park, the circulation control system may be applied when a plurality of users ride a plurality of attractions (for example, a Ferris wheel). \\n     In the example of the above-described embodiment, the server device executes the circulation control program to control the progress of the circulation, but the present disclosure is not particularly limited to this. The terminal device may execute a part of the processing executed by the server device shown in  FIG. 3 . \\n     In the above-described example of the embodiment, the history information and the designation information are included in the user information and registered in the storage unit, but the present disclosure is not particularly limited to this. The history information and the designation information may be stored separately. \\n     In the example of the above-described embodiment, the congestion status is specified based on the designation information of the user information, but other information (for example, the size of the destination, the number of people that can be accommodated) may be additionally referred to. \\n     In the example of the above-described embodiment, an intention of the user is not reflected in the configuration in which the server device (designation unit) determines the designated destination, but a configuration that reflects the intention of the user may be adopted. For example, when determining the designated destination of one user, the designated destination may be selected by the one user. Specifically, the server device determines one or more candidate destinations from among one or more destinations that the one user has not visited yet, depending on the congestion status of a plurality of destinations. Then, the server device may determine the candidate destination selected by the one user using the operation unit from among the determined candidate destinations as the designated destination. For example, in the above-described treasure hunt game, in a case of one user who circulates three destinations A 1 , A 2 , and A 3  from now on, the destination A 3  with the smallest number of users and the destination A 2  with the second smallest number of users are set as candidate destinations. Then, the one user selects any one of the candidate destinations (destination A 2  and A 3 ). The server device determines the selected destination as the designated destination. A destination other than the destination with the largest number of users may be set as a candidate destination. Further, the candidate destination may include at least the destination with the smallest number of users among the destinations that one user has not visited yet. \\n     Second Embodiment \\n     In the example of the present embodiment, a circulation control system  100 B, which is an example of the circulation control system  100 , will be described. The circulation control system  100 B includes a server device  10 B and a plurality of terminal devices  20  and  201  to  20 N.  FIG. 4  is a block diagram showing a configuration of the server device  10 B, which is an example of the server device  10 . In the example of the present embodiment, the server device  10 B includes at least the update unit  11  and a designation unit  12 B. \\n     The update unit  11  updates the user information of each of a plurality of users registered in the storage unit of the server device  10 B. The user information includes history information specifying an already visited destination and designation information specifying a destination (designated destination) to be visited. In addition, when receiving the information indicating visiting the designated destination from the terminal devices  20  and  201  to  20 N of any user, the update unit  11  updates the received history information of the user of the terminal device. When the designated destination of a certain user is determined by the designation unit  12 B, the update unit  11  updates the designation information of the user. \\n     The designation unit  12 B determines a designated destination for each of a plurality of users. Specifically, the designation unit  12 B determines, based on history information of one user, one destination that the one user has not visited yet among the plurality of destinations as a designated destination of the one user. However, the designation unit  12 B determines a designated destination of each user according to congestion status of the plurality of destinations. That is, the designation unit  12 B determines a designated destination of one user according to the congestion status of a plurality of destinations among the destinations that the one user has not visited yet. The congestion status of each destination is a status that is specified at least based on the information (designation information) on the destination (designated destination) to which each user should visit. In the example of the present embodiment, when the designation unit  12 B determines the designated destination of one user, the designation unit  12 B determines, as the designated destination of the one user, the destination with at least the smallest number of users as the designated destination among the destinations that the one user has not visited yet. The designation unit  12 B may specify the number of users based on the designation information of each user. \\n     In addition, the designation unit  12 B transmits information on the designated destination determined for one user to the terminal device of the one user. The information on the designated destination may be included in the control data in association with the identification information of the destination, for example. \\n     Next, an operation of the circulation control system  100 B will be described. \\n       FIG. 5  is a flowchart showing an example of circulation processing executed by the circulation control system  100 B. In the circulation processing in the example of the present embodiment, processing for determining a designated destination is performed for each of a plurality of users.  FIG. 5  mainly describes processing of determining a designated destination and processing of updating user information, and some other processings may be omitted. Hereinafter, a case where the server device  10 B and the terminal device  20  carried by the user P 1  execute the circulation processing will be described as an example. \\n     The circulation processing of the example of the present embodiment is executed, for example, when the server device  10 B receives predetermined information from any of the terminal devices  20  and  201  to  20 N. \\n     The predetermined information corresponds to, for example, information on a designated destination visited by the user. In addition, the predetermined information corresponds to, for example, information of a circulation start request for the user to start the circulation of a plurality of destinations from now on. The terminal device  20  transmits the predetermined information together with the identification information of the user P 1 , for example. \\n     The server device  10 B executes update processing A (step S 10 ). In the update processing A, the user information registered in the storage unit of the server device  10 B is updated based on the received predetermined information. For example, when receiving information on the designated destination visited by the user P 1  from the terminal device  20 , the server device  10 B updates the history information among the user information of the user P 1 . For example, identification information of the designated destination visited by the user P 1  is added to the history information of the user P 1 . Then, the designation information of the user P 1  is reset (set to an initial value). In addition, for example, when the user P 1  receives the circulation start request from the terminal device  20 , the server device  10 B newly registers the user information of the user P 1  in the storage unit of the server device  10 B. For example, user information of user P 1  including the identification information of user P 1 , history information in which the initial value is set, and designation information is generated. \\n     Next, the server device  10 B executes extraction processing (step S 11 ). In the extraction processing, a candidate destination as a candidate of the designated destination is extracted. Specifically, based on the history information of the user P 1 , destinations that the user P 1  has not visited yet are extracted as a candidate destination. \\n     Thereafter, the server device  10 B executes determination processing (step S 12 B). In the determination processing, one candidate destination selected from among the above-described candidate destinations is determined as a designated destination according to the congestion status. That is, the destination to which the user P 1  should visit is determined as a designated destination. In the determination processing of the example of the present embodiment, the destination with the smallest number of users as the designated destination is determined as the designated destination of the user P 1  at present. \\n     In addition, in the determination processing, the server device  10 B transmits information on the determined designated destination to the terminal device  20 . For example, in the case of the above-described treasure hunt game, the text data of the sentence of the question is transmitted. The terminal device  20  outputs (displays on the display unit) the received information on the designated destination (step S 30 ). For example, in the case of the above-described treasure hunt game, a question sentence is displayed on the display unit of the terminal device  20 . \\n     After that, the server device  10 B executes update processing B (step S 13 ). In the update processing B, the designation information of the user information is updated. For example, the identification information of the designated destination determined in the determination processing of step S 12 B is registered as the designation information. \\n     After that, the server device  10 B ends the circulation processing. Regarding the circulation processing shown in  FIG. 5 , the same processing as that of the terminal device  20  is performed on the other terminal devices  201  to  20 N. \\n     As described above, as one aspect of the second embodiment, since the server device  10 B is configured to include the update unit  11  and the designation unit  12 B, the destination to which the user should visit is set according to the congestion status. Accordingly, it is possible to prevent many users from being crowded and the destination being congested. Therefore, it is effective as an infection control of viruses and the like. \\n     In the example of the above-described embodiment, an intention of the user is not reflected in the configuration in which the server device (designation unit) determines the designated destination, but a configuration that reflects the intention of the user may be adopted as in the first embodiment. \\n     Third Embodiment \\n       FIG. 6  is a block diagram showing a configuration of a circulation control system  100 C, which is an example of the configuration of the circulation control system  100 . The circulation control system  100 C of the example of the present embodiment is applied to a treasure hunt game which is a circulating type event. The circulation control system  100 C includes a server device  10 C and terminal devices  20 C and  201 C to  20 NC carried by a plurality of users. Each of the server device  10 C and a plurality of terminal devices  20 C and  201 C to  20 NC are wirelessly connected to a communication network  15  such as the Internet. \\n     The treasure hunt game is an event in which a plurality of participants (users) solve a plurality of mysteries (questions) to search for treasure hidden in the real space. In the treasure hunt game, a plurality of users need to go around a plurality of places (destination) to acquire all the hidden treasures. In the example of the present embodiment, three treasures are hidden (disposed) in a certain area  400  in the real space. Specifically, treasures T 1  to T 3  are disposed at places (destinations) of check points  91 C to  93 C. \\n     The check points  91 C to  93 C can be provided in, for example, a park, a plaza, a meeting place, a store, or the like. The area  400  may be any region in the real space that is randomly determined. For example, the area  400  corresponds to facilities such as attraction facilities and shopping malls. \\n     Each user is given a total of three questions Q 1  to Q 3 . An answer to question Q 1  indicates the place of the check point  91 C. An answer to question Q 2  indicates the place of the check point  92 C. An answer to question Q 3  indicates the place of the check point  93 C. Accordingly, each user can specify a place where all the treasures T 1  to T 3  are disposed by solving all the questions. \\n     The treasures T 1  to T 3  are virtual treasures in the game and are boxes that imitate treasure boxes. Near filed communication (NFC) tags TG 1  to TG 3  are disposed in the treasures T 1  to T 3 . When each user holds the user&#39;s terminal devices  20 C and  201 C to  20 NC over the NFC tags TG 1  to TG 3 , the user has acquired the treasures T 1  to T 3 . That is, the terminal devices  20 C and  201 C to  20 NC acquire the identification information of the treasures T 1  to T 3  from the NFC tags TG 1  to TG 3  by performing short-range wireless communication with the NFC tags TG 1  to TG 3 . The identification information of the treasure T 1  is recorded in the NFC tag TG 1 . The identification information of the treasure T 2  is recorded in the NFC tag TG 2 . The identification information of the treasure T 3  is recorded on the NFC tag TG 3 . \\n     For example, when the terminal device  20 C of the user P 1  acquires the identification information of the treasure T 1 , the terminal device  20 C transmits the identification information of the treasure T 1  to the server device  10 C. The server device  10 C registers the received identification information of the treasure T 1  in the user information (acquisition history) of the user P 1 . That is, the user P 1  has acquired the treasure T 1  by registering in the user information. The other terminal devices  201 C to  20 NC also have the same configuration as the terminal devices  20 C. \\n       FIG. 7  is a diagram showing the flow of actions of one user P 1  in the treasure hunt game. One question is delivered to the user P 1  from the server device  10 C to the user&#39;s terminal device  20 C. The user P 1  first sees one question displayed on the terminal device  20 C. The user P 1  searches a place (check point) where the treasure is disposed by solving the question. Then, the user P 1  moves to the place where the treasure is disposed and acquires the treasure. Specifically, the user P 1  acquires the identification information of the treasure by holding the terminal device  20 C over the NFC tag disposed on the treasure. Then, the terminal device  20 C transmits the acquired identification information of the treasure to the server device  10 C. \\n     After that, next one question is delivered from the server device  10 C to the terminal device  20 C of the user P 1 . The user P 1  will also act in the same manner as described above for the next question. Finally, when the user P 1  solves the three questions Q 1  to Q 3  and acquires the three treasures T 1  to T 3 , the treasure hunt game ends. \\n     In the present embodiment, as described above, the server device  10 C delivers the questions Q 1  to Q 3  to the terminal devices  20 C and  201 C to  20 NC. More specifically, the server device  10 C transmits the text data of the sentences of the questions Q 1  to Q 3 . The delivery of questions Q 1  to Q 3  is one question at a time in no particular order. As shown in  FIG. 7 , each time one question is solved and one treasure is acquired, another new question is delivered. The acquisition of one treasure means a state in which the identification information of the one treasure is registered in the user information (acquisition history). \\n     The treasure hunt game is started by transmitting a game start request (circulation start request) from the terminal devices  20 C and  201 C to  20 NC carried by each user to the server device  10 C. The game start request includes, for example, information such as identification information of the user and a user name. Each user may, for example, input each information such as identification information of the user and a user name in the user&#39;s terminal device, and then transmit the game start request from the user&#39;s terminal device to the server device  10 C. The server device  10 C that has received the game start request additionally registers new user information in the storage unit, and delivers one question to the terminal device that has transmitted the game start request. As a result, the treasure hunt game is started. \\n     The circulation control system  100 C (server device  10 C) determines a destination (designated destination) to which each user should visit in order to prevent many users from being temporarily crowded at one destination (place where the treasure is disposed). In the example of the present embodiment, the server device  10 C determines the designated destination by determining a question to be delivered from the plurality of questions Q 1  to Q 3 . The answers to questions Q 1  to Q 3  indicate the check points  91 C to  93 C (places where the treasures T 1  to T 3  are disposed), which are the destinations to which the user should visit. Accordingly, the determination of the question to be delivered is the same as determining the designated destination. The server device  10 C determines, among the questions Q 1  to Q 3 , a question that is currently the least delivered to each user as a question to be delivered. For example, when the total result is as shown in  FIG. 10 , the question Q 3  with a small number of users is determined as the question to be delivered to one user. In this case, this is because it is estimated that the check point  93 C corresponding to the question Q 3  is least likely to be crowded. The total result shown in  FIG. 10  does not include the delivered number of questions that have already been cleared (the treasure has been acquired). \\n       FIG. 8  is a block diagram showing a configuration of the server device  10 C, which is an example of the server device  10 . In the example of the present embodiment, the server device  10 C includes at least an update unit  11 C and a designation unit  12 C. \\n     The update unit  11 C updates the user information of each of a plurality of users registered in the storage unit of the server device  10 C. The user information is information on the circulation of a plurality of destinations of the user in the treasure hunt game. The user information of a plurality of users is registered as registration data in the storage unit of the server device  10 C.  FIG. 9  is a diagram showing an example of registration data. \\n     The registration data is composed of fields such as a user ID, a user name, an acquisition history, and a delivery question. The user information of one user is registered in one record. The identification information of the user is registered in a user ID field. The identification information of the user corresponds to, for example, an e-mail address owned by the user. Text data of a user name is registered in a user name field. \\n     The identification information of the treasure that the user has already acquired is registered in an acquisition history field. That is, in the acquisition history field, information specifying the destination that the user has already visited is registered. This is because the treasures T 1  to T 3  are disposed in different places, and the identification information of the treasures T 1  to T 3  functions as the identification information of the place (destination) where the treasures T 1  to T 3  are disposed. The acquisition history corresponds to the history information. \\n     In a delivery question field, the identification information of the question currently being delivered to the user is registered. That is, in the delivery question field, the information specifying the designated destination, which is the destination to which the user should visit, is registered. This is because the answers to the questions Q 1  to Q 3  indicate the places (check points  91 C to  93 C) where the treasures T 1  to T 3  are disposed, and the identification information of the questions Q 1  to Q 3  functions as the identification information of the designated destination to which the user should visit. In a case of the user who has acquired all the treasures T 1  to T 3 , the identification information indicating that the treasure hunt game has ended is registered in the delivery question field. In  FIG. 9 , for convenience of explanation, the characters “FINISH” are displayed instead of the identification information indicating that the treasure hunt game has ended. The information registered in the delivery question field corresponds to the designation information. \\n     For example, in a case of the user P 2  in  FIG. 9 , it is specified from the acquisition history that the treasures T 1  and T 2  have already been acquired. That is, it is specified that the user P 1  has already visited the check points  91 C and  92 C (destinations). Further, it is specified from the delivery question that the user P 2  is currently in a state where the question Q 3  is being delivered and is searching for the treasure T 3 . That is, it is specified that the user P 2  is currently in a state where the user should visit the check point  93 C (designated destination). \\n     In addition, for example, in a case of a user P 3  in  FIG. 9 , it is specified from the acquisition history that the treasures T 1  to T 3  have already been acquired. That is, it is specified that the user P 3  has already visited the check points  91 C to  93 C (destinations). Further, it is specified from the delivery question that the user P 3  is in a state where the treasure hunt game is ended. \\n     When receiving the information indicating visiting the designated destination (the identification information of the treasure) from the terminal devices  20 C and  201 C to  20 NC of any user, the update unit  11 C updates the received acquisition history of the user of the terminal device. Specifically, the update unit  11 C additionally registers the received identification information of the treasure in the received acquisition history of the user of the terminal device. In addition, when the designated destination (question to be delivered) of one user is determined by the designation unit  12 C, the update unit  11 C updates the delivery question of the one user. Specifically, the update unit  11 C registers the identification information of the question to be delivered in the delivery question of the one user. \\n     The designation unit  12 C determines a designated destination for each of a plurality of users. In the example of the present embodiment, the designation unit  12 C determines a designated destination for each of the plurality of users by determining a question to be delivered for each of the plurality of users. Specifically, the designation unit  12 C determines, based on the history information (acquisition history) of one user, as a question to be delivered to the one user, a question corresponding to one destination that the one user has not visited yet among the questions Q 1  to Q 3  corresponding to a plurality of destinations. That is, the designation unit  12 C determines, based on the acquisition history of one user, a question that has not been delivered yet to the one user as a question to be delivered to the one user. \\n     However, the designation unit  12 C determines the question to be delivered to each user according to the congestion status of the plurality of destinations. The congestion status of each destination is a status that is specified at least based on the designation information (delivery question) of each user. In the example of the present embodiment, with respect to one user, the designation unit  12 C determines, as a question to be delivered to the one user, a question corresponding to the destination with the smallest number of users as the designated destination among the destinations that the one user has not visited yet. Specifically, the designation unit  12 C refers to the delivery question of the registration data, and totals the number of users to be delivered for each of the questions Q 1  to Q 3 . That is, the designation unit  12 C totals the number of users for each destination (check points  91 C to  93 C) to which each user should visit. From the total result, the designation unit  12 C determines a question with the smallest number of users as a question to be delivered to the one user. For example, when the total result is as shown in  FIG. 10 , the designation unit  12 C determines the delivered question Q 3  with the smallest number of users as a question to be delivered to the one user. \\n     In addition, the designation unit  12 C transmits determined information on the designated destination of one user to the terminal device of the one user. In the example of the present embodiment, the information on the designated destination is, for example, text data of the question sentence of the question corresponding to the designated destination. The text data of the question sentence may be included in the control data. \\n       FIG. 11  is a block diagram showing a configuration of the terminal device  20 C, which is an example of the terminal device  20 . In the example of the present embodiment, the terminal device  20 C is a smartphone. The terminal device  20 C includes at least an acquisition unit  21 C, a communication control unit  22 C, and a display control unit  23 C. The terminal devices  201 C to  20 NC have the same configuration as the terminal devices  20 C. \\n     The acquisition unit  21 C acquires the identification information of the treasures T 1  to T 3  from the NFC tags TG 1  to TG 3  by short-range wireless communication. The communication control unit  22 C receives the information and the like of the question delivered from the server device  10 C. In addition, the communication control unit  22 C transmits the game start request, the identification information of the acquired treasure, and the like to the server device  10 C. The display control unit  23 C causes the display unit to display information on the treasure hunt game such as the sentence of question. \\n     Next, an operation of the circulation control system  100 C will be described. \\n       FIG. 12  is a flowchart showing an example of circulation processing executed by the circulation control system  100 C. In the circulation processing in the example of the present embodiment, processing for determining a designated destination is performed for each of a plurality of users. Specifically, processing for determining a question corresponding to a designated destination is performed for each of a plurality of users.  FIG. 12  mainly describes processing of determining a question and processing of updating user information, and some other processings may be omitted. Hereinafter, a case where the server device  10 C and the terminal device  20 C carried by the user P 1  execute the circulation processing will be described as an example. \\n     The circulation processing of the example of the present embodiment is executed, for example, when the server device  10 C receives predetermined information from any of the terminal devices  20 C and  201 C to  20 NC. \\n     The predetermined information corresponds to the identification information (information on the designated destination) the treasure acquired by the user and the information on the game start request (circulation start request). The terminal device  20 C transmits the updated information together with the identification information of the user P 1 . \\n     The server device  10 C executes update processing A (step S 10 C). In the update processing A, the user information registered in the storage unit of the server device  10 C is updated based on the received predetermined information. For example, when receiving information (the identification information of the treasure) on the designated destination visited by the user P 1  from the terminal device  20 C, the server device  10 C additionally registers the identification information of the treasure in the history information (acquisition history) of the user P 1 . Then, the server device  10 C resets (sets to an initial value) the designation information (delivery question) of the user P 1 . When the identification information of all the treasures T 1  to T 3  is registered in the acquisition history, the identification information of the end of the game is registered in the delivery question of the user P 1 . \\n     In addition, when receiving the game start request from the terminal device  20 C, the server device  10 C newly registers the user information of the user PI in the storage unit of the server device  10 C. More specifically, the identification information of the user P 1 , the acquisition history in which the initial value is set, and the user information of the user P 1  including the delivery question are additionally registered in the registration data shown in  FIG. 9 . \\n     Next, the server device  10 C executes extraction processing (step S 11 C). In the extraction processing, a candidate destination as a candidate of the designated destination is extracted. In the example of the present embodiment, destinations that the user P 1  has not visited yet are extracted as a candidate destination based on the acquisition history of the user P 1 . \\n     After that, the server device  10 C executes determination processing (step S 12 C). In the determination processing, one candidate destination selected from among the above-described candidate destinations is determined as a designated destination according to the congestion status. In the example of the present embodiment, the server device  10 C determines the question to be delivered to the user P 1 . As a result, the designated destination is determined. As described above, the server device  10 C refers to the designation information (delivery question) of the registration data, and totals the number of users delivered for each of the questions Q 1  to Q 3 . Then, based on the total result, the server device  10 C determines the delivered question with the smallest number of users among the questions that have not been solved at present as the question to be delivered to the user P 1 . \\n     In addition, in the determination processing, the server device  10 C transmits information on the determined designated destination to the terminal device  20 C. In the example of the present embodiment, the text data of the question sentence of the determined question is transmitted. The terminal device  20 C displays the received information on the designated destination on the display unit (step S 30 C). As described above, the question sentence is displayed on the display unit of the terminal device  20 C. \\n     When all the identification information of the treasures T 1  to T 3  is registered in the acquisition history, the above-described extraction processing and determination processing are not executed. \\n     After that, the server device  10 C executes update processing B (step S 13 C). In the update processing B, the designation information (delivery question) of the user information is updated. Specifically, the identification information of the question determined in the determination processing of step S 12 C is registered as the delivery question of the user P 1  in the registration data. When the above-described determination processing is not executed, the update processing B is not executed. \\n     After that, the server device  10 C ends the circulation processing. Regarding the circulation processing shown in  FIG. 12 , the same processing as that of the terminal device  20 C is performed on the other terminal devices  201 C to  20 NC. \\n     As described above, as one aspect of the third embodiment, since the server device  10 C is configured to include the update unit  11 C and the designation unit  12 C, the destination (treasure to be searched) to which the user should visit is set according to the congestion status. Accordingly, it is possible to prevent many users from being crowded and the destination being congested. Therefore, it is effective as an infection control of viruses and the like. \\n     In the example of the above-described embodiment, a configuration for acquiring the identification information of the treasure using the NFC tag is adopted, but the configuration is not particularly limited to this configuration. For example, a GPS function of the terminal device carried by the user may be used to acquire position information of a place where the treasure is disposed as the history information specifying the destination that the user has already visited. \\n     In the example of the above-described embodiment, the designation unit directly determines the question to be delivered to each user, but it may be configured to determine the question to be delivered after determining the designated destination of each user. \\n     In the example of the above-described embodiment, the text data which is the sentence of question is transmitted to the terminal device, but the text data is not limited to the text data. For example, in a case of a graphic question, graphic data (image data) may be transmitted to the terminal device. \\n     In the example of the above-described embodiment, an intention of the user is not reflected in the configuration in which the server device (designation unit) determines the designated destination, but a configuration that reflects the intention of the user may be adopted as in the first embodiment. For example, when determining the designated destination of one user, the designated destination may be selected by the one user. Specifically, the server device determines one or more candidate destinations from among one or more destinations that the one user has not visited yet, depending on the congestion status of a plurality of destinations. Then, the server device may determine the candidate destination selected by the one user using the operation unit from among the determined candidate destinations as the designated destination. For example, in the above-described treasure hunt game, in a case of one user who circulates three destinations (check points  91  C to  93  C) from now on, a destination with the smallest number of users (check point  93 C: question Q 3 ) and a destination with the second smallest number of users (check point  92 C: question Q 2 ) are set as candidate destinations. Then, the one user selects any one of the candidate destinations (questions Q 2  and Q 3 ). The server device determines the selected destination (question) as the designated destination. A destination other than the destination with the largest number of users may be set as a candidate destination. Further, the candidate destination may include at least the destination with the smallest number of users among the destinations that one user has not visited yet. \\n     Fourth Embodiment \\n       FIG. 13  is a block diagram showing a configuration of a circulation control system  100 D, which is an example of the configuration of the circulation control system  100 . The circulation control system  100 D of the example of the present embodiment is applied to control the progress of the circulation of a plurality of attractions  80 C to  84 C in the amusement park (attraction facility)  500  in the real space. The circulation control system  100 D includes a server device  10 D and terminal devices  20 D and  201 D to  20 ND carried by a plurality of users. Each of the server device  10 D and a plurality of terminal devices  20 D and  201 D to  20 ND are wirelessly connected to a communication network  15  such as the Internet. \\n     In an amusement park  500 , a plurality of attractions  80 C to  84 C such as a Ferris wheel and a roller coaster are disposed. The amusement park  500  is surrounded by fences. The user (visitor) can enter the facility from a gate G, which is the entrance and exit. \\n     A plurality of users can ride a plurality of attractions  80 D to  84 D. The circulation control system  100 D of the example of the present embodiment instructs the attractions to be ridden (to be visited) with respect to a plurality of attractions selected by each user. That is, the circulation control system  100 D provides each user with a guidance about attractions to be ridden. Specifically, the circulation control system  100 D instructs each user of one attraction to be ridden next each time the riding of the attraction is completed. .As a result, the circulation control system  100 D causes each user to circulate a plurality of destinations (attractions). \\n     The NFC tags TG 10  to TG 14  are disposed at an exit of the plurality of attractions  80 D to  84 D. The identification information of the disposed attractions is recorded in the NFC tags TG 10  to TG 14 . Each user can acquire the attraction identification information by holding the user&#39;s terminal devices  20 D and  201 D to  20 ND over the NFC tags TG 10  to TG 14 . The identification information of the attraction  80 D is recorded on the NFC tag TG 10 . The identification information of the attraction  81 D is recorded on the NFC tag TG 11 . The identification information of the attraction  82 D is recorded on the NFC tag TG 12 . The identification information of the attraction  83 D is recorded on the NFC tag TG 13 . The identification information of the attraction  84 D is recorded on the NFC tag TG 14 . \\n     For example, when the terminal device  20 D of the user P 1  acquires the identification information of the attraction  80 D, the terminal device  20 D transmits the identification information of the attraction  80 D to the server device  10 D. The server device  10 D registers the received identification information of the attraction  80 D in the user information (riding history) of the user P 1 . That is, by registering in the user information, the user P 1  has completed riding at the attraction  80 D. The other terminal devices  201 D to  20 ND also have the same configuration as the terminal devices  20 D. \\n     The guidance about attractions is started by transmitting a guidance start request (circulation start request) from the terminal devices  20 D and  201 D to  20 ND carried by each user to the server device  10 D. The guidance start request includes, for example, information such as identification information of the user, a user name, and an attraction. The attraction information is the attraction information (identification information) selected by the user. Each user may, for example, input each information such as the identification information of the user, the user name, and the attraction in the user&#39;s terminal device, and then transmit the guidance start request from the user&#39;s terminal device to the server device  10 D. The server device  10 D which has received the guidance start request additionally registers the new user information in the storage unit, and transmits information on the attraction to be ridden (to be visited) first to the terminal device which has transmitted the guidance start request. \\n       FIG. 14  is a diagram showing the flow of actions of one user P 1  in the guidance about attractions. After the user P 1  (terminal device  20 D) makes the guidance start request, one attraction is delivered to the user P 1  from the server device  10 D to the user&#39;s terminal device  20 D. The user P 1  sees the guidance (attraction name) displayed on the terminal device  20 D. The user P 1  moves to the guided attraction. Then, the user P 1  rides the attraction. After riding, the terminal device  20 D is held over the NFC tag at an exit of the attraction to acquire the identification information of the attraction. Then, the terminal device  20 D transmits the acquired attraction identification information to the server device  10 D. \\n     After that, the terminal device  20 D of the user P 1  is guided (instructed) by the server device  10 D to one attraction to be ridden next. The user P 1  also acts in the same manner as described above for the next attraction. Finally, when the user P 1  has ridden all the selected attractions, the guidance about attractions ends. \\n     The circulation control system  100 D (server device  10 D) determines a destination (designated destination) to which each user should visit in order to prevent many users from being temporarily crowded at one destination (attraction). That is, the server device  10 D instructs each user about the attraction to be ridden. \\n     Each user can select an attraction in a range of 2 or more and 5 or less, for example. In addition, the selectable number of attractions may be a fixed value. \\n       FIG. 15  is a block diagram showing a configuration of the server device  10 D, which is an example of the server device  10 . In the example of the present embodiment, the server device  10 D includes at least an update unit  11 D and a designation unit  12 D. \\n     The update unit  11 D updates the user information of each of a plurality of users registered in the storage unit of the server device  10 D. The user information is information on the circulation of a plurality of destinations of the user in the guidance about attractions. The user information of a plurality of users is registered as registration data in the storage unit of the server device  10 D.  FIG. 16  is a diagram showing an example of registration data. \\n     The registration data is composed of fields such as a user ID, a user name, a selected attraction, a riding history, and a guidance attraction. The user information of one user is registered in one record. The identification information of the user is registered in a user ID field. The identification information of the user corresponds to, for example, an e-mail address owned by the user. Text data of a user name is registered in a user name field. \\n     The identification information of the attraction selected by the user is registered in a selected attraction field. The circulation control system  100 D provides the guidance for a plurality of attractions registered as selected attractions. \\n     In a riding history field, the identification information of the attraction that the user has already ridden is registered. That is, in the riding history field, the information specifying the destination that the user has already visited is registered. This is because the identification information of the attractions  80 D to  84 D functions as the identification information of the place where the attractions  80 D to  84 D are disposed. The riding history corresponds to the history information. \\n     In a guidance attraction field, the identification information of the attraction currently being guided to the user is registered. That is, in the guidance attraction field, the information specifying the designated destination, which is the destination to which the user should visit, is registered. In a case of a user who has ridden all the selected attractions, the identification information indicating that the guidance has been ended is registered in the guidance attraction field. In  FIG. 16 , for convenience of explanation, the characters “FINISH” are displayed instead of the identification information indicating that the guidance has ended. The guidance attraction corresponds to the designation information. \\n     For example, in the case of the user P 2  in  FIG. 16 , the three attractions  80 D,  83 D, and  84 D are the attractions that the circulation control system  100 D guides the user P 2 . That is, the three attractions  80 D,  83 D, and  84 D are the attractions selected by the user P 2 . In addition, it is specified from the riding history that the user P 2  has already ridden the attraction  83 D. Further, it is specified from the guidance attraction that the user P 2  is currently in a state of being instructed to visit the attraction  80 D. That is, it is specified that the user P 2  is currently in a state where the user should visit the attraction  80 D (designated destination). \\n     In addition, for example, in the case of the user P 3  in  FIG. 16 , the two attractions  80 D and  81 D are the attractions that the circulation control system  100 D guides the user P 3 . Then, it is specified from the riding history that the user P 3  has already ridden the attractions  80 D and  81 D. Further, as the designation information of the user P 3 , it is specified from the guidance attraction that the guidance has been ended. \\n     When receiving the information indicating visiting the designated destination (the identification information of the attraction) from the terminal devices  20 D and  201 D to  20 ND of any user, the update unit  11 D updates the received riding history of the user of the terminal device. Specifically, the update unit  11 D additionally registers the received identification information of the attraction in the received riding history of the user of the terminal device. In addition, when the designated destination (attraction) of one user is determined by the designation unit  12 D, the update unit  11 D updates the guidance attraction of the one user. Specifically, the update unit  11 D registers the identification information of the determined designated destination (attraction) in the guidance attraction of the one user. \\n     The designation unit  12 D determines a designated destination for each of a plurality of users. In the example of the present embodiment, the designation unit  12 D determines one attraction to be ridden for each of a plurality of users. Specifically, the designation unit  12 D determines, based on the history information (riding history) of one user, one attraction which the one user has not visited yet among a plurality of attractions (selected attraction) selected by the one user as the attraction on which the one user should ride. \\n     However, the designation unit  12 D determines an attraction on which each user should ride according to the congestion status of a plurality of destinations (attractions). The congestion status of each destination is a status that is specified at least based on the designation information (guidance attraction) of each user. In the example of the present embodiment, with respect to one user, the designation unit  12 D determines an attraction with the smallest number of users as an attraction on which the one user should ride among the attractions that the one user has not visited yet. Specifically, the designation unit  12 D refers to the guidance attraction of the registration data, and totals the number of users being guided for each attraction  80 D to  84 D. That is, the designation unit  12 D totals the number of users for each destination (attractions  80 D to  84 D) that each user is heading for riding. From the total result, the designation unit  12 D determines the attraction with the smallest number of users as the attraction to be ridden by the one user. \\n     In addition, the designation unit  12 D transmits determined information on the designated destination of one user to the terminal device of the one user. In the example of the present embodiment, the information on the designated destination is, for example, text data of the name of the designated destination (attraction). The text data of the attraction may be included in the control data. \\n       FIG. 17  is a block diagram showing a configuration of the terminal device  20 D, which is an example of the terminal device  20 . In the example of the present embodiment, the terminal device  20 D is a smartphone. The terminal device  20 D includes at least an acquisition unit  21 D, a communication control unit  22 D, and a display control unit  23 D. The terminal devices  201 D to  20 ND have the same configuration as the terminal devices  20 D. \\n     The acquisition unit  21 D acquires identification information of attractions from the NFC tags TG 10  to TG 14  by short-range wireless communication. The communication control unit  22 D receives information and the like regarding the attractions transmitted from the server device  10 D. In addition, the communication control unit  22 D transmits the guidance start request, the acquired identification information of the attraction, and the like to the server device  10 D. The display control unit  23 D causes the display unit to display the name of the attraction and the like. \\n     Next, an operation of the circulation control system  100 D will be described. \\n       FIG. 18  is a flowchart showing an example of circulation processing executed by the circulation control system  100 D. In the circulation processing in the example of the present embodiment, processing for determining a designated destination (attraction) is performed for each of a plurality of users.  FIG. 15  mainly describes processing of determining an attraction and processing of updating user information, and some other processings may be omitted. Hereinafter, a case where the server device  10 D and the terminal device  20 D carried by the user P 1  execute the circulation processing will be described as an example. \\n     The circulation processing of the example of the present embodiment is executed, for example, when the server device  10 D receives predetermined information from any of the terminal devices  20 D and  201 D to  20 ND. \\n     The predetermined information corresponds to identification information (information of the designated destination) of the attraction in which the user has appeared and information of the guidance start request (circulation start request). The terminal device  20 D transmits the predetermined information together with the identification information of the user P 1 . \\n     The server device  10 D executes update processing A (step S 10 D). In the update processing A, the user information registered in the storage unit of the server device  10 D is updated based on the received predetermined information. For example, when receiving information (identification information of an attraction) on the designated destination visited by the user P 1  from the terminal device  20 D, the server device  10 D additionally registers the identification information of the attraction in the history information (riding history) of the user P 1 . Then, the server device  10 D resets (sets to an initial value) the designation information (guidance attraction) of the user P 1 . When the identification information of all the attractions is registered in the riding history, identification information of the guidance end is registered in the guidance attraction of the user P 1 . \\n     In addition, when receiving the guidance start request from the terminal device  20 D, the server device  10 D newly registers the user information of the user P 1  in the storage unit of the server device  10 D. Specifically, the identification information of the user P 1 , the selected attraction, the riding history in which the initial value is set, and the user information of the user P 1  including the guidance attraction are additionally registered in the registration data shown in  FIG. 16 . \\n     Next, the server device  10 D executes extraction processing (step S 11 D). In the extraction processing, a candidate destination as a candidate of the designated destination is extracted. In the example of the present embodiment, a destination (attraction) that the user P 1  has not visited yet (not ridden) is extracted as a candidate destination based on the selected attraction and the riding history of the user P 1 . \\n     After that, the server device  10 D executes determination processing (step S 12 D). In the determination processing, one candidate destination selected from among the above-described candidate destinations is determined as a designated destination according to the congestion status. In the example of the present embodiment, a destination (attraction) to which the user P 1  should visit is determined. As described above, the server device  10 D refers to the designation information (guidance attraction) of the registration data, and totals the number of users for each destination (attractions  80 D to  84 D) that each user heading for riding. Then, the server device  10 D determines the attraction with the smallest number of users as the attraction on which the user P 1  should ride at present from the total result. \\n     In addition, in the determination processing, the server device  10 D transmits information on the determined designated destination to the terminal device  20 D. In the example of the present embodiment, the text data of the name of the determined attraction is transmitted. The terminal device  20 D displays the received information on the designated destination on the display unit (step S 30 D). As described above, the name of the attraction is displayed on the display unit of the terminal device  20 D. \\n     When the identification information of all the attractions selected by the user P 1  is registered in the riding history, the above-described extraction processing and determination processing are not executed. \\n     After that, the server device  10 D executes update processing B (step S 13 D). In the update processing B, the designation information (guidance attraction) of the user information is updated. The identification information of the designated destination (attraction) determined in the determination processing of step S 12 D is registered in the guidance attraction. When the above-described determination processing is not executed, the update processing B is not executed. \\n     After that, the server device  10 D ends the circulation processing. Regarding the circulation processing shown in  FIG. 15 , the same processing as that of the terminal device  20 D is performed on the other terminal devices  201 D to  20 ND. \\n     As described above, as one aspect of the fourth embodiment, since the server device  10 D is configured to include the update unit  11 D and the designation unit  12 D, the destination (the attraction on which the user should ride) to which the user should visit is set according to the congestion status. Accordingly, it is possible to prevent many users from being crowded and the destination being congested. Therefore, it is effective as an infection control of viruses and the like. \\n     In the example of the above-described embodiment, a configuration for acquiring the identification information of the attraction using the NFC tag is adopted, but the configuration is not particularly limited to this configuration. For example, a GPS function of the terminal device carried by the user may be used to acquire position information of a place where the attraction is disposed as the history information specifying the destination that the user has already visited. \\n     In the example of the above-described embodiment, a configuration in which the user selects a plurality of attractions is used, but a configuration in which the user does not select the attractions may be used. In this case, for example, all attractions may be targeted to be guided to the user. \\n     In the example of the above-described embodiment, the congestion status is specified based on the guidance attraction of the user information, but other information may be additionally referred to. For example, waiting time information until riding for each attraction may be additionally referred to. \\n     In the example of the above-described embodiment, an intention of the user is not reflected in the configuration in which the server device (designation unit) determines the designated destination, but a configuration that reflects the intention of the user may be adopted as in the first embodiment. For example, when determining the designated destination of one user, the designated destination may be selected by the one user. Specifically, the server device determines one or more candidate destinations from among one or more destinations that the one user has not visited yet, depending on the congestion status of a plurality of destinations. Then, the server device may determine the candidate destination selected by the one user using the operation unit from among the determined candidate destinations as the designated destination. For example, in the guidance about attractions described above, in a case of one user who circulates three destinations (attractions  80 D to  83 D) from now on, a destination (attraction) with the smallest number of users and a destination (attraction) with the second smallest number of users are set as candidate destinations. Then, the one user selects one of the two candidate destinations (attractions). The server device determines the selected attraction as the designated destination. A destination other than the destination with the lamest number of users may be set as a candidate destination. Further, the candidate destination may include at least the destination with the smallest number of users among the destinations that one user has not visited yet. \\n     Appendix \\n     The description of the above-described embodiment describes at least the following disclosure such that a person having ordinary knowledge in the field to which the invention belongs can carry out the invention. \\n     [1] \\n     A circulation control program for causing a server device that is connected to a terminal device carried by each of a plurality of users through a communication network to control a progress of making each of the plurality of users circulate a plurality of destinations, the circulation control program causes the server device to implement the following functions including an update function of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the update function includes a function of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and a function of updating the designation information of one user when the designated destination of the one user is determined by the designation function, and the designation function includes a function of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     [2] \\n     The circulation control program according to [1], in which the designation function includes a function of determining, as a designated destination of one user, a destination having the smallest number of users as the designated destination specified based on at least the designation information of each user among the destinations that the one user has not visited yet, when determining the designated destination of the one user. \\n     [3] \\n     The circulation control program according to [1] or [2], in which the information on the designated destination is information for notifying a user of the terminal device of a question whose answer indicates the designated destination. \\n     [4] \\n     The circulation control program according to any one of [1] to [3], in which the terminal device acquires position information by performing short-range wireless communication with a communication device disposed at each of a plurality of destinations, the communication device storing the position information specifying the destination at which the communication device is disposed, and the update function includes a function of updating the history information of a user when the position information is received from the terminal device of the user as the information indicating visiting the designated destination. \\n     [5] \\n     The circulation control program according to any one of [1] to [4], in which the designation function includes a function of determining a new designated destination of a user each time the information indicating visiting the designated. destination is received from the terminal device of the user. \\n     [6] \\n     The server device on which the circulation control program according to any one of [1] to [5] is installed. \\n     [7] \\n     A circulation control system for controlling a progress of making each of a plurality of users circulate a plurality of destinations, the circulation control system including a terminal device carried by each of the plurality of users, and a server device connected to the terminal device through a communication network, the circulation control system including an updater configured to update user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designator configured to determine the designated destination for each of a plurality of users, the designator determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the updater updates the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and updates the designation information of one user when the designated destination of the one user is determined by the designator, and the designator determines the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     [8] \\n     The circulation control system according to [7], in which the server device includes the updater and the designator, and the terminal device includes a communication controller configured to receive information on the designated destination from the server device and transmit information indicating visiting the designated destination to the server device. \\n     [9] \\n     A circulation control program for causing a computer of a terminal device carried by each of a plurality of users to implement a function of communicating with a server device having a function of controlling a progress of making each of the plurality of users circulate a plurality of destinations, the circulation control program causes the terminal device to implement a function of receiving information on a designated destination from the server device having the following functions including an update function of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and a designation function of determining, the designated destination for each of a plurality of users, the designation function determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, in which the update function including a function of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and a function of updating the designation information of one user when the designated destination of the one user is determined by the designation function, and the designation function including a function of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     [10] \\n     The terminal device of a user on which the circulation control program according to [9] is installed. \\n     [11] \\n     A circulation control method for controlling a progress in which a terminal device carried by each of a plurality of users and a server device connected through a communication network make each of the plurality of users circulate a plurality of destinations, the circulation control method includes update processing of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and designation processing of determining the designated destination for each of a plurality of users, the designation processing determining, based on the history information of one user, one destination that the one user has not visited yet as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, the update processing is processing of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and processing of updating the designation information of one user when the designated destination of the one user is determined by the designation processing, and the designation processing is processing of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     [12] \\n     A circulation control method executed by a circulation control system for controlling a progress of making each of the plurality of users circulate a plurality of destinations, the circulation control system including a terminal device carried by each of a plurality of users, and a server device connected to the terminal device through a communication network, the circulation control method includes update processing of updating user information of each of a plurality of users registered in a storage unit, the user information including history information specifying an already visited destination and designation information specifying a designated destination to be visited, and designation processing of determining the designated destination for each of a plurality of users, the designation processing determining, based on the history information of one user, one destination that the one user has not visited ye as the designated destination of the one user, and transmitting information on the designated destination to a terminal device of the one user, the update processing is processing of updating the history information of one user when information indicating visiting the designated destination is received from the terminal device of the one user, and processing of updating the designation information of one user when the designated destination of the one user is determined by the designation processing, and the designation processing is processing of determining the designated destination according to a congestion status of the plurality of destinations based on at least the designation information of each user. \\n     According to one of the embodiments of the present disclosure, in a configuration in which a user circulates a plurality of destinations, it is useful to prevent many users from being crowded and the destinations being congested.', 'id': 'US-2022032175-A1', 'ti': 'Non-transitory computer-readable medium including circulation control program, and non-transitory computer-readable medium including circulation control system and communication control program'}>,\n",
       " <Hit {'ab': 'The present invention is directed towards a method and system for redirecting roaming network traffic in an LTE network. The method includes observing a registration process of a subscriber in a visited network. The method further includes sending one or more messages to the visited network to induce a re-registration attempt by the subscriber to another Radio Access Technology (RAT) network of the same visited network. The method further includes applying one or more legacy steering techniques to move the subscriber to a target visited network., The present invention is directed towards a method and system for redirecting roaming network traffic in an LTE network. The method includes observing a registration process of a subscriber in a visited network. The method further includes sending one or more messages to the visited network to induce a re-registration attempt by the subscriber to another Radio Access Technology (RAT) network of the same visited network. The method further includes applying one or more legacy steering techniques to move the subscriber to a target visited network.', 'clm': 'The invention claimed is: \\n     \\n       1. A method for redirecting roaming network traffic in a Long Term Evolution (LTE) network, the method comprising:\\n observing, by a home public mobile network associated with a subscriber, a registration process of the subscriber in a first Radio Access Technology (RAT) network within a non-preferred visited public mobile network; \\n before the subscriber registers in the first RAT network, repeatedly sending, by the home public mobile network, a plurality of messages to the non-preferred visited public mobile network that induces a re-registration attempt by the subscriber to a second RAT network within the non-preferred visited public mobile network, the second RAT network being a different type of network than the first RAT network; \\n while the subscriber attempts to register to the second RAT network within the non-preferred visited public mobile network, sending, by the home public mobile network, one or more messages to the non-preferred visited public mobile network to prevent the subscriber from registering in the second RAT network, such that the home public mobile network can steer the subscriber to a target visited public mobile network while the home public mobile network is preventing the subscriber from registering in the second RAT network; and \\n applying, by the home public mobile network, one or more steering techniques to force the subscriber to register with the target visited public mobile network that is different than the non-preferred visited public mobile network. \\n \\n     \\n     \\n       2. The method of  claim 1 , wherein the plurality of messages comprise at least one Diameter message selected from a group consisting of an InsertsubscriberDataRequest (IDR) message and a Cancel Location Request (CLR) message. \\n     \\n     \\n       3. The method of  claim 1 , wherein the plurality of messages comprise at least one Mobile Application Part (MAP) message selected from a group consisting of a MAP Process Unstructured Supplementary Services (SS) Request and a MAP Provide Subscriber Information message, followed by a MAP Cancel Location message. \\n     \\n     \\n       4. The method of  claim 1 , wherein sending of the one or more messages further comprises: sending a message to a card of the subscriber modifying current location information stored with identifiers of the target visited public mobile network to force a registration attempt by the subscriber to the target visited public mobile network. \\n     \\n     \\n       5. The method of  claim 4 , wherein information about the target visited public mobile network is sent in response to an applet on a handset of the subscriber being able to detect an item selected from a group consisting of roaming, network registration change, handset start, and activation timer, and being able to open a communication channel to obtain a list of networks. \\n     \\n     \\n       6. The method of  claim 1 , wherein the registration process is between a Mobility Management Entity (MME) in the target visited public mobile network and a Home Subscriber Server (HSS) of a home network of the subscriber. \\n     \\n     \\n       7. The method of  claim 1 , wherein the plurality of messages are sent to a Mobility Management Entity (MME) or a Mobile Switching Center (MSC)/Visited Location Register (VLR) corresponding to the non-preferred visited public mobile network. \\n     \\n     \\n       8. The method of  claim 1 , further comprising:\\n identifying, prior to the sending the plurality of messages, a location of the subscriber in the non-preferred visited public mobile network; and \\n determining, based on the location, whether the subscriber is in a region where a radio coverage of a home network overlaps a radio coverage of the non-preferred visited public mobile network. \\n \\n     \\n     \\n       9. The method of  claim 1 , further comprising:\\n assessing a compatibility of supported frequencies of a device of the subscriber and frequencies broadcast by the target visited public mobile network. \\n \\n     \\n     \\n       10. A non-transitory computer-readable medium storing computer executable code for redirecting roaming network traffic in a Long Term Evolution (LTE) network, comprising:\\n code for causing a computer to:\\n observe, by a home public mobile network associated with a subscriber, a registration process of the subscriber in a first Radio Access Technology (RAT) network within a non-preferred visited public mobile network; \\n before the subscriber registers in the first RAT network, repeatedly send, by the home public mobile network, a plurality of messages to the non-preferred visited public mobile network that induces a re-registration attempt by the subscriber to a second RAT network within the non-preferred visited public mobile network, the second RAT network being a different type of network than the first RAT network; \\n while the subscriber attempts to register to the second RAT network within the non-preferred visited public mobile network, send, by the home public mobile network, one or more messages to the non-preferred visited public mobile network to prevent the subscriber from registering in the second RAT network, such that the home public mobile network can steer the subscriber to a target visited public mobile network while the home public mobile network is preventing the subscriber from registering in the second RAT network; and \\n apply, by the home public mobile network, one or more steering techniques to force the subscriber to register with the target visited public mobile network that is different than the non-preferred visited public mobile network. \\n \\n \\n     \\n     \\n       11. The non-transitory computer-readable medium of  claim 10 , wherein at least one of the plurality of messages is selected from a group consisting of an InsertsubscriberDataRequest (IDR) message, a Cancel Location Request (CLR) message, a Mobile Application Part (MAP) Process Unstructured Supplementary Services (SS) Request, and MAP Provide Subscriber Information messages followed by a MAP Cancel Location message. \\n     \\n     \\n       12. The non-transitory computer-readable medium of  claim 10 , wherein at least one of the one or more messages is sent to a card of the subscriber modifying current location information stored with identifiers of the target visited public mobile network to force a registration attempt by the subscriber to the target visited public mobile network. \\n     \\n     \\n       13. The non-transitory computer-readable medium of  claim 12 , wherein information about the target visited public mobile network is sent in response to an applet on a handset of the subscriber being able to detect an item selected from a group consisting of roaming, network registration change, handset start, and activation timer, and being able to open a communication channel to obtain a list of networks. \\n     \\n     \\n       14. The non-transitory computer-readable medium of  claim 10 , wherein the registration process is between a Mobility Management Entity (MME) in the target visited public mobile network and a Home Subscriber Server (HSS) of a home network of the subscriber. \\n     \\n     \\n       15. The non-transitory computer-readable medium of  claim 10 , wherein the plurality of messages are sent to a Mobility Management Entity (MME) or a Mobile Switching Center (MSC)/Visited Location Register (VLR) corresponding to the non-preferred visited public mobile network. \\n     \\n     \\n       16. The non-transitory computer-readable medium of  claim 10 , further comprising:\\n code for causing the computer to:\\n identify, prior to sending the plurality of messages, a location of the subscriber in the non-preferred visited public mobile network, and \\n determine, based on the location, determines whether the subscriber is in a region where a radio coverage of the home network overlaps a radio coverage of the non-preferred visited public mobile network. \\n \\n \\n     \\n     \\n       17. The non-transitory computer-readable medium of  claim 10 , further comprising:\\n code for causing the computer to: \\n assess a compatibility of supported frequencies of a device of the subscriber and frequencies broadcast by the target visited public mobile network. \\n \\n     \\n     \\n       18. A method for redirecting roaming network traffic in a Long Term Evolution (LTE) network, the method comprising:\\n observing, by a home public mobile network associated with a subscriber, a registration process of the subscriber in a first Radio Access Technology (RAT) network within a non-preferred visited public mobile network; \\n before the subscriber registers in the first RAT network, repeatedly sending, by the home public mobile network, a plurality of messages to the non-preferred visited public mobile network that induces a re-registration attempt by the subscriber to a second RAT network within the non-preferred visited public mobile network, the second RAT network being a different type of network than the first RAT network; \\n while the subscriber attempts to register to the second RAT network within the non-preferred visited public mobile network, sending, by the home public mobile network, one or more messages to the non-preferred visited public mobile network to prevent the subscriber from registering in the second RAT network, such that the home public mobile network can steer the subscriber to a target visited public mobile network; and \\n applying, by the home public mobile network, one or more steering techniques to force the subscriber to register with the target visited public mobile network that is different than the non-preferred visited public mobile network. \\n \\n     \\n     \\n       19. The method of  claim 18 , further comprising steering, by the home public mobile network, the subscriber to the target visited public mobile network before the subscriber registers in the first and the second RAT networks.', 'cpc': 'H04W36/0022', 'detd': 'CROSS-REFERENCE TO RELATED APPLICATION \\n     This application is a National Stage entry of International Application No. PCT/US2013/057090, filed Aug. 28, 2013, which claims priority to U.S. Provisional Patent Application No. 61/694,095 entitled STEERING OF ROAMING IN LTE AND LEGACY NETWORK ENVIRONMENT, filed on Aug. 28, 2012, and U.S. Provisional Patent Application No. 61/708,499, filed on Oct. 1, 2012. This application is also a continuation of PCT Application no. PCT/US2012/045780 entitled “NETWORK TRAFFIC REDIRECTION (NTR) IN LONG TERM EVOLUTION (LTE)” filed on Jul. 12, 2012. The previous patent applications are incorporated by reference herein in their entirety. \\n    \\n    \\n     FIELD OF THE INVENTION \\n     The present invention generally relates to telecommunication. More specifically, the invention relates to method and system for redirecting roaming traffic to preferred operators across multiple telecommunication networks. \\n     BACKGROUND OF THE INVENTION \\n     As telecommunication technology has progressed, numerous mobile communication standards have been developed. These standards are broadly categorized into second generation (2G), third generation (3G) and the future, fourth generation (4G) technologies. Examples of 2G/3G technologies include Global System for Mobile communications (GSM), General Packet Radio Service (GPRS), Enhanced Data rates for GSM Evolution (EDGE), Universal Mobile Telecommunications System (UMTS), and the like. The UMTS standard further evolved to LTE technology under the Third Generation Partnership Project (3GPP). LTE technology offers a wireless broadband system with higher data rates, lower latency, and higher spectrum efficiency. \\n     To keep up with the competition, more and more operators are adapting the newer LTE technology standard. These network operators provide voice and data services to their own subscribers and to subscribers from other networks. When the network operator provides service to a subscriber from a foreign country, it is referred to as “international roaming”. When the network operator provides service to a subscriber from another network in the same country, it is referred to as “domestic roaming.” \\n     An operator always has a preference for one or more operators over another set of operators. Various network operators have partnership agreements with each other that include more favorable roaming charges than non-partners receive. Partner networks are “preferred” networks for the network operator&#39;s subscriber to register with when roaming. Non-partner networks are “non-preferred” networks to the subscriber. Network operators can maximize their margins and the roamers can get more attractive roaming rates and services if roamers roam on their home mobile operator&#39;s preferred partner networks. \\n     Over the last few years, the revenues to network operators from home subscribers have consistently declined due to increased competition and resulting pricing pressures. On the other hand, revenues from roamers have consistently grown in the same period due to increased mobile penetration in local markets and an increase in travel. Hence, protecting the existing roaming revenues and growing them further has become an important priority for the network operators worldwide. \\n     Some operators also own networks in various countries. These operators would like to make sure their out-roamers stay within the group properties, or stay on preferred networks, in order to gain the best service experience even when roaming. They will also be able to offer geography based price plans (for example, a single rate all across Western Europe or South East Asia) to users who have subscribed to such plans. In addition, as new technologies like LTE keep rolling out, operators can control the rollout schedule across their own properties and also make sure interoperability issues are taken care of. Keeping roaming subscribers on preferred networks gives the best service experience to the subscriber. However, current methods of controlling which network a subscriber registers on when roaming have disadvantages. \\n     One previous patent from the inventors of this application was directed towards a method and system for facilitating redirection of network traffic towards a preferred network. However, that patent was focusing on GSM network. \\n     Another patent from the inventors was directed to a system and method facilitating redirection of network traffic across multiple networks including LTE and GSM. However, since in some cases the redirection to LTE network is not always successful but redirection is more successful within legacy Radio Access Technology network like 2G or 3G. Hence, there is a need in the art to have a system and method for facilitating redirection of network traffic in both LTE and legacy network environments. \\n     SUMMARY \\n     The present invention is directed towards a method and system for redirecting roaming network traffic in an LTE network. The method includes observing a registration process of a subscriber in a visited network. The method further includes sending one or more messages to the visited network to induce a re-registration attempt by the subscriber to another Radio Access Technology (RAT) network of the same visited network. The method further includes applying one or more legacy steering techniques to move the subscriber to a target visited network. \\n     The present invention in another embodiment also provides a method for redirecting roaming network traffic in an LTE network. The method includes intercepting a Diameter registration request of a subscriber in a visited network. The method further includes sending InsertsubscriberDataRequest (IDR) message or Cancel Location Request (CLR) message network to the visited to induce a re-registration attempt by the subscriber to another Radio Access Technology (RAT) network of the same visited network. The method further includes sending MAP Process Unstructured SS Request and MAP Provide Subscriber Information messages followed by a MAP Cancel Location message to move the subscriber to a target visited network. \\n     The present invention in another embodiment also provides a system for redirecting roaming network traffic in a telecommunication network. The system includes a Network Traffic Redirection (NTR) module that observes a registration process of a subscriber in a visited network. The NTR module further sends one or more messages to the visited network to induce a re-registration attempt by the subscriber to another Radio Access Technology (RAT) network of the same visited network. The NTR module further applies one or more legacy steering techniques to move the subscriber to a target visited network. \\n     In accordance with various embodiments of the present invention, the NTR module is capable of steering roamers across LTE and non-LTE (GSM) networks. This solution of implementing NTR is hereinafter, interchangeably, referred to as NTR LTE or NTR solution. \\n    \\n    \\n     \\n       BRIEF DESCRIPTION OF DRAWINGS \\n       In the drawings, the same or similar reference numbers identify similar elements or acts. \\n         FIG. 1  illustrates a system for implementing the NTR in LTE solution, in accordance with an embodiment of the present invention; \\n         FIG. 2  represents a flowchart for redirecting roaming network traffic in a telecommunication network, in accordance with an embodiment of the present invention; \\n         FIG. 3  represents a flow diagram for implementing NTR solution using an IDR message, in accordance with an embodiment of the present invention; \\n         FIG. 4  represents a flow diagram for implementing NTR solution using a Cancel Location message, in accordance with an embodiment of the present invention; \\n         FIG. 5  represents a flow diagram for implementing NTR solution using a Process Unstructured SS Request (PUSSR) message, in accordance with an embodiment of the present invention; and \\n         FIG. 6  represents a flow diagram for implementing NTR solution using a Provide Subscriber Information (PSI) message, in accordance with an embodiment of the present invention. \\n     \\n    \\n    \\n     DETAILED DESCRIPTION \\n     In the following description, for purposes of explanation, specific numbers, materials and configurations are set forth in order to provide a thorough understanding of the present invention. It will be apparent, however, to one having ordinary skill in the art that the present invention may be practiced without these specific details. In some instances, well-known features may be omitted or simplified, so as not to obscure the present invention. Furthermore, reference in the specification to “one embodiment” or “an embodiment” means that a particular feature, structure or characteristic, described in connection with the embodiment, is included in at least one embodiment of the present invention. The appearance of the phrase “in an embodiment”, in various places in the specification, does not necessarily refer to the same embodiment. \\n     The present invention provides a system and a method for redirecting roaming traffic across multiple telecommunication networks. The general idea of this invention is to allow an operator to apply NTR solution across LTE and non-LTE (GSM) networks (hereinafter interchangeably referred to as “legacy networks”). The HPMN operator deploys an NTR module to implement the solution as described in the present invention. This module enables the operator to steer away its subscribers from a non-preferred VPMN to a preferred VPMN. \\n     In accordance with another embodiment of the present invention, a group of operators may deploy the NTR solution to steer their subscribers to a set of preferred operators. In such an embodiment, the NTR module is deployed at a centralized location that is connected to a host network. In its various implementation of the present invention, the NTR solution is implemented in monitoring mode or in-signaling mode \\n       FIG. 1  illustrates a system  100  that implements the NTR in LTE solution, in accordance with an embodiment of the present invention. NTR module  102  is deployed in HPMN  104  to apply traffic steering on its subscriber  106 , who is roaming in a VPMN  108 . This enables the operator in HPMN to be able to steer its subscribers to preferred operators in VPMN  108 . For sake of representation, system  100  represents network elements from both LTE and GSM networks. HPMN  104  includes an HSS  110  that connects via a DEA/DRA  112  in HPMN  104  and a DEA/DRA  114  in VPMN  108  to an MME  116  in VPMN  108 . The MME  116  is further connected to an MSC-R/VLR-R  118  in VPMN  108  via SGsAP protocol. \\n     HPMN  105  further includes an MSC-H/VLR-H  120 , an SGSN-H  122 , a GGSN-H  124 , an SMSC-H  126  and an STP-H  128 . These network elements communicate with each other over a Signaling System 7 (SS7) link. System  100  further includes in VPMN  108  an SGSN-R  130 , a GMSC-R  132 , and an STP-R  134  that connects to STP-H  128  over SS7 link. \\n     It will also be apparent to a person skilled in the art that HPMN  104  and VPMN  108  may also include various other network components (not shown in  FIG. 1 ), depending on the architecture under consideration. \\n     NTR module  102  connects with various components within HPMN  104  and VPMN  108  to implement the NTR solution for its roaming subscriber, like roamer  106 . The primary objective of NTR module  102  is to be able to do SoR across multiple telecommunication networks and move roamer  106  to target visited network VPMN  140 &#39;s MME  142 . \\n       FIG. 2  represents a flowchart for redirecting roaming network traffic in a telecommunication network, in accordance with an embodiment of the present invention. It will be apparent to a person skilled in the art that various steps of the flowchart would be executed by the NTR module  102 . At step  202 , the NTR module  102  checks the compatibility of the frequency bands supported by the device and the frequency broadcast by target VPMN  140 . At step  206 , the NTR module  102  checks the actual location subscriber  106  and assesses whether it is located in an overlapping region (HPMN  104  and VPMN  108 ) in case a border control is applied. Subsequently, at step  208 , the NTR module  102  sends one or more messages to the VPMN  108  to induce a re-registration attempt by the subscriber  106  in the same network under another RAT legacy network, e.g., 2G or 3G. Finally, at step  210 , the NTR module  106  applies one or more legacy steering techniques to move the subscriber to target VPMN  140 . \\n     In one embodiment of the present invention, roamer  106  registers to an LTE network and the messages are Diameter messages. In one embodiment of the present invention, NTR module  102  sends these messages, while being deployed in monitoring mode. In another embodiment of the present invention, NTR module  102  sends these messages, while being deployed in in-signaling mode. In another embodiment of the present invention, at step  208 , NTR module  102  sends the target visited network as current location information to the roamer  106  while being in VPMN  108  to induce a re-registration attempt by the roamer  106  to target visited network  140 . These messages are sent directly to roamer  106 &#39;s SIM card modifying the current location information stored with the identifiers of the target VPMN  140 , to induce a re-registration attempt by the subscriber to the target VPMN  140 . \\n       FIG. 3  represents a flow diagram for implementing NTR solution using an IDR message, in accordance with an embodiment of the present invention. In order to initiate registration to VPMN  108 , roamer  106  sends an EMM Attach request to MME  116  in VPMN  108 . MME  116  further sends a Diameter-Update Location Request towards HSS  110  of HPMN  104 . In response, HSS  110  sends a Diameter-Update Location Accept to MME  116 . Now before MME  116  confirms the registration roamer  106  by sending EMM Attach accept message to roamer  106 , NTR module  102  attempts SoR to redirect the roamer  106  to another preferred LTE operator by sending Diameter IDR message to MME  116 . The MME  116  acknowledges the message and returns Diameter ISD accept to NTR module  102 . Thereafter, MME  116  sends EMM Combined attach Reject message (cause #15) to roamer  106  for ESM failure. \\n     The roamer  106  now attempts to register to another RAT legacy network 2G or 3G within VPMN  108 . Now, while the roamer  106  attempts to register to the legacy network, NTR module  102  uses one or more legacy steering techniques to move the roamer  106  to the target VPMN  140 &#39;s MME  142 . These legacy steering techniques involve NTR module sending one or more MAP messages and rejecting roamer  106 &#39;s four GSM LUP messages and forcing the roamer  106  to register to target VPMN  140 . \\n       FIG. 4  represents a flow diagram for implementing NTR solution using a Cancel Location Request (CLR) message, in accordance with an embodiment of the present invention. The call flow for this embodiment, is identical to  FIG. 3  embodiment, except that NTR module  102  sends a Diameter Cancel Location Request to MME  116  with cancellation type as subscription withdrawn to force the roamer  106  to steer away from VPMN  106 . Also, in this case when MME  116  sends an EMM Detach request it sends with an error code #7 for EPS not allowed. This rejection is repeated 5 times to force the roamer  106  to fall back on legacy network on another RAT like 2G or 3G. Subsequently, the NTR module  102  sends one or more MAP messages and rejects roamer  106 &#39;s four GSM LUP message or five GPRS LU to force the roamer  106  to register to target VPMN  140 . \\n       FIG. 5  represents a flow diagram for implementing NTR solution using a Process Unstructured SS Request (PUSSR) message, in accordance with an embodiment of the present invention. In this case, when roamer  106  sends the EMM Attach request to MME  116  and subsequently the MME  116  sends a Diameter UL request, the subscriber roamer  106  is successfully registered to the LTE network in VPMN  108 . Now, the NTR module sends MAP Process Unstructured SS Request message followed by MAP Cancel Location method after configurable timer to ensure that roamer  106  falls back to legacy network like 2G/3G. Moreover, the MSC  118  rejects the service request and the roamer  106  is forced to perform a LUP. The NTR module  102  then apply legacy SoR techniques using LUP reject until the roamer  106  is registered with target VPMN  140 . \\n       FIG. 6  represents a flow diagram for implementing NTR solution using a Provide Subscriber Information (PSI) message, in accordance with an embodiment of the present invention. Similar to  FIG. 5 , in this embodiment, NTR module  102  sends MAP PSI request message followed by MAP Cancel Location method after configurable timer to ensure that roamer  106  falls back to legacy network like 2G/3G. Moreover, the MSC  118  rejects the service request and the roamer  106  is forced to perform a LUP. The NTR module  102  then applies legacy SoR techniques using LUP reject until the roamer  106  is registered with target VPMN  140 . \\n     The Diameter LTE steering enables to control the radio access technology. In other words, it enables the deploying operator of favoring preferred partner or controlling the traffic distribution between networks but also between Radio Access Technologies (RAT). \\n     The rejection of the Diameter Registration procedures results in various error codes in the radio interface of the mobile of roamer  106 . In accordance with various embodiments of the present invention, the NTR module  102  is able to induce the following new error codes on the radio interfaces:\\n         1. Cause #15 (no suitable cells in Tracking Area), #12 “tracking area not allowed”, #13 “roaming not allowed in this tracking area and it forces the UE to reselect another RAT in the same PMN. The location area is added to the list of “forbidden LAs for roaming” which is stored in the mobile.   2. Cause #14 “EPS services not allowed in this PLMN” is received by the UE, the chosen PLMN identity shall be stored in the “forbidden PLMNs for GPRS service”.   3. Cause #19, “ESM failure”, the UE may set the attach attempt counter to 5 and directly attempt to register on another network.       \\n\\n     The NTR module  102  also follows the usual rejection principles available in SS7 steering. For example,\\n         Cause #11 (PMN Not Allowed)   It forces the mobile UE (User Equipment) to perform a PMN reselection. The mobile UE shall store the PMN identity in the “forbidden PMN list” in the card and the UE shall no more reselect this PMN.   Cause #17 (Network Failure)—leading to additional registration re-attempts (up to 5 attempts) from the mobile before changing network.       \\n\\n     Since the NTR solution intercepts the DIAMETER Update Location Request, it can respond with the correct Diameter Update Location Answer (Reject) leading to the expected radio error code and handset behavior. \\n     The mapping table to be applied is Table 1, shown below. \\n     \\n       \\n         \\n           \\n               \\n               \\n             \\n               \\n                 TABLE 1 \\n               \\n               \\n                   \\n               \\n               \\n                 Diameter Reject Cause \\n                 Radio Error cause \\n               \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 DIAMETER_ERROR_UNKNOWN —   \\n                 #15 “No suitable cells in tracking area” \\n               \\n               \\n                 EPS_SUBSCRIPTION \\n               \\n               \\n                 DIAMETER_AUTHORIZATION —   \\n                 #15 “No suitable cells in tracking area” \\n               \\n               \\n                 REJECTED (5003) \\n               \\n               \\n                 DIAMETER_ERROR_RAT_NOT —   \\n                 #15 “No suitable cells in tracking area”, or \\n               \\n               \\n                 ALLOWED (5421) \\n                 #13 “Roaming not allowed in this tracking area”, or \\n               \\n               \\n                   \\n                 #12 “Tracking area not allowed” \\n               \\n               \\n                   \\n                 Note - the cause selection is an operator&#39;s choice \\n               \\n               \\n                 DIAMETER_ERROR_ROAMING —   \\n                 #11 “PLMN not allowed” \\n               \\n               \\n                 NOT_ALLOWED (5004) \\n               \\n               \\n                 DIAMETER_UNABLE_TO_COMPLY \\n                 #17 “Network failure” \\n               \\n               \\n                 (5012), \\n               \\n               \\n                 DIAMETER_INVALID_AVP_VALUE \\n               \\n               \\n                 (5004) \\n               \\n               \\n                 DIAMETER - Roaming restricted in MME \\n                 #14 “EPS services not allowed in this PLMN” \\n               \\n               \\n                 due to unsupported feature, in of the \\n               \\n               \\n                 Subscription-Data, part of the Update \\n               \\n               \\n                 Location Answer \\n               \\n               \\n                 OPERATOR_DETERMINED_BARRING \\n                 #19 “ESM failure” \\n               \\n               \\n                 is received in the Subscriber-Status \\n               \\n               \\n                 AVP, part of the Update Location \\n               \\n               \\n                 Answer \\n               \\n               \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     In yet another embodiment of the present invention, the NTR module  102  performs OTA based steering. There are some mobile devices which do not maintain an SMS stack in LTE networks. Hence, they need an applet to open an IP channel for specific events. For example, the SIM card in the handset may detect the entry in a new PMN or handset may restart. In such a scenario, the SIM card may open an IP channel with OTA server. The OTA server may then request for a dynamic preference list. This request is intercepted by the NTR module  102 . At this stage, the NTR module  102  sends the list of preferred networks where it wants to steer its subscriber. The NTR module  102  sends the current location information stored with the identifiers of the target VPMN  140 , to induce a re-registration attempt by the roamer  106  to target VPMN  140 . Upon receiving the list, the OTA server sends relevant EF files to the handset. \\n     In accordance with various other embodiment of the present invention, the NTR module  102  is also able to redirect network traffic across multiple networks from different RAT. E.g. NTR solution can steer a roamer from LTE to GSM or vice versa. \\n     In the context of coexistence of LTE with pre-LTE technology, the mobility management of outbound roamers may happen in different fashions, from a core network perspective:\\n         EPS only registration i.e. the registration is entirely happening between the MME and HSS.   EPS and non-EPS registration for CSFB i.e. the registration happens between the MME-HSS and MSC/VLR-HLR.   non-EPS registration i.e. the registration happens between the MSC/VLR-HLR and the SGSN-HLR.       \\n\\n     One challenge of the previous steering solution is to identify the current registration status of the subscriber. In other words, the NTR solution determines whether the subscriber is attached to an E-UTRAN or UTRAN/GERAN network access. \\n     This information is required to correctly apply the rejection principles linked to the observed GSM UL. In some cases, GSM UL may happen for combined EPS/Non-EPS registration but also as a standalone scenario, in case of MO/MT call occurring under a new coverage area. It is therefore required to identify in the scenario of the subscriber as a GSM UL does not provided any information. \\n     In specific scenario of ‘Circuit-Switched Fallback’ (CSFB), the legacy (GSM) and evolved core network protocol (LTE), MAP and Diameter, respectively coexist. When a registration process including CSFB happens, the order of messages observed at HSS/HLR is fully determined. The GSM UL registration only happens after a successful Diameter UL procedure (ULR/ULA). \\n     In other words, observed messages will follow the sequence:\\n         1. Diameter ULR   2. Diameter ULA (assumed successful)   3. GSM UL   4. ISD/ISD ACK   5. GSM UL ACK       \\n\\n     Now if standard SoR is applied on MAP it would mark the Diameter registration as successful but with failure on non-EPS domain. This would lead to ineffective SoR or new steering attempts but with counter rules associated with the EPS domain and not the GSM domain. This misleads the NTR module that would assume a possible manual registration of the subscriber. The observed behavior will vary on handset type (data- or voice-centric), but would be likely to end up in an ineffective steering of roaming action. \\n     It should be obvious for a person skilled in the art that an independent MAP based NTR platform is likely to interfere with the MAP message as it would be unaware of a previous Diameter registration. This would affect negatively the user experience at a time of a call activity. Hence, the NTR solution of the present invention is enhanced with Diameter-handling capabilities like the NTR solution must have information about the two sets of registration procedures (Diameter and MAP). \\n     It will be apparent to a person skilled in the art, that the present invention can also be applied to Code Division Multiple Access (CDMA)/American National Standards Institute #41D (ANSI-41D), and various other technologies such as, but not limited to, VoIP, WiFi, 3GSM and inter-standard roaming. In one exemplary case, a CDMA outbound roamer travels with an HPMN CDMA handset. In another exemplary case, the CDMA outbound roamer travels with an HPMN GSM SIM and a GSM handset. In yet another exemplary case, GSM outbound roamer travels with an HPMN CDMA RUIM and a CDMA handset. To support these variations, system  100  will have a separate SS7 and network interfaces, corresponding to both the HPMN and VPMN networks. It will also be apparent to a person skilled in the art that these two interfaces in different directions may not have to be the same technologies. Moreover, there could be multiple types of interface in both directions. \\n     An exemplary list of the mapping between GSM MAP and ANSI-41D is described in Table 2 below as a reference. \\n     \\n       \\n         \\n           \\n               \\n               \\n               \\n             \\n               \\n                   \\n                 TABLE 2 \\n               \\n               \\n                   \\n                   \\n               \\n               \\n                   \\n                 GSM MAP \\n                 ANSI-41D \\n               \\n               \\n                   \\n                   \\n               \\n             \\n            \\n               \\n                   \\n                 Location Update/ISD \\n                 REGNOT \\n               \\n               \\n                   \\n                 Cancel Location \\n                 REGCAN \\n               \\n               \\n                   \\n                 RegisterSS \\n                 FEATUREREQUEST \\n               \\n               \\n                   \\n                 InterrogateSS \\n                 FEATUREREQUEST \\n               \\n               \\n                   \\n                 SRI-SM \\n                 SMSREQ \\n               \\n               \\n                   \\n                 SRI \\n                 LOCATION REQUEST \\n               \\n               \\n                   \\n                 ForwardSMS \\n                 SMSDPP \\n               \\n               \\n                   \\n                 ReadyForSMS \\n                 SMSNOTIFICATION \\n               \\n               \\n                   \\n                 AlertServiceCenter \\n                 SMSNOTIFICATION \\n               \\n               \\n                   \\n                 ReportSMSDelivery \\n                 SMDPP \\n               \\n               \\n                   \\n                 ProvideRoamingNumber \\n                 ROUTING REQUEST \\n               \\n               \\n                   \\n                   \\n               \\n            \\n           \\n         \\n       \\n     \\n     The present invention can take the form of an entirely hardware embodiment, an entirely software embodiment, or an embodiment containing both hardware and software elements. In accordance with an embodiment of the present invention, software, including but not limited to, firmware, resident software, and microcode, implements the invention. \\n     Furthermore, the invention can take the form of a computer program product, accessible from a computer-usable or computer-readable medium providing program code for use by, or in connection with, a computer or any instruction execution system. For the purposes of this description, a computer-usable or computer readable medium can be any apparatus that can contain, store, communicate, propagate, or transport the program for use by or in connection with the instruction execution system, apparatus, or device. \\n     The medium can be an electronic, magnetic, optical, electromagnetic, infrared, or semiconductor system (or apparatus or device) or a propagation medium. Examples of a computer-readable medium include a semiconductor or solid state memory, magnetic tape, a removable computer diskette, a random access memory (RAM), a read-only memory (ROM), a rigid magnetic disk and an optical disk. Current examples of optical disks include compact disk-read only memory (CDROM), compact disk-read/write (CD-R/W) and Digital Versatile Disk (DVD). \\n     The components of present system described above include any combination of computing components and devices operating together. The components of the present system can also be components or subsystems within a larger computer system or network. The present system components can also be coupled with any number of other components (not shown), such as other buses, controllers, memory devices, and data input/output devices, in any number of combinations. In addition, any number or combination of other processor-based components may be carrying out the functions of the present system. \\n     It should be noted that the various components disclosed herein may be described using computer aided design tools and/or expressed (or represented), as data and/or instructions embodied in various computer-readable media, in terms of their behavioral, register transfer, logic component, transistor, layout geometries, and/or other characteristics. Computer-readable media in which such formatted data and/or instructions may be embodied include, but are not limited to, non-volatile storage media in various forms (e.g., optical, magnetic or semiconductor storage media) and carrier waves that may be used to transfer such formatted data and/or instructions through wireless, optical, or wired signaling media or any combination thereof. \\n     Unless the context clearly requires otherwise, throughout the description and the claims, the words “comprise,” “comprising,” and the like are to be construed in an inclusive sense as opposed to an exclusive or exhaustive sense; that is to say, in a sense of “including, but may not be limited to.” Words using the singular or plural number also include the plural or singular number respectively. Additionally, the words “herein,” “hereunder,” “above,” “below,” and words of similar import refer to this application as a whole and not to any particular portions of this application. When the word “or” is used in reference to a list of two or more items, it covers all of the following interpretations: any of the items in the list, all of the items in the list and any combination of the items in the list. \\n     The above description of illustrated embodiments of the present system is not intended to be exhaustive or to limit the present system to the precise form disclosed. While specific embodiments of, and examples for, the present system are described herein for illustrative purposes, various equivalent modifications are possible within the scope of the present system, as those skilled in the art will recognize. The teachings of the present system provided herein can be applied to other processing systems and methods. They may not be limited to the systems and methods described above. \\n     The elements and acts of the various embodiments described above can be combined to provide further embodiments. These and other changes can be made in light of the above detailed description. \\n     Other Variations \\n     Provided above for the edification of those of ordinary skill in the art, and not as a limitation on the scope of the invention, are detailed illustrations of a scheme for proactive roaming tests, discoveries of roaming partner services and discoveries of frauds in roaming using simulated roaming traffic. Numerous variations and modifications within the spirit of the present invention will of course occur to those of ordinary skill in the art in view of the embodiments that have been disclosed. For example, the present invention is implemented primarily from the point of view of GSM mobile networks as described in the embodiments. However, the present invention may also be effectively implemented on GPRS, 3G, CDMA, WCDMA, WiMax etc., or any other network of common carrier telecommunications in which end users are normally configured to operate within a “home” network to which they normally subscribe, but have the capability of also operating on other neighboring networks, which may even be across international borders. \\n     The examples under the system of present invention detailed in the illustrative examples contained herein are described using terms and constructs drawn largely from GSM mobile telephony infrastructure. However, use of these examples should not be interpreted as limiting the invention to those media. The system and method can be of use and provided through any type of telecommunications medium, including without limitation: (i) any mobile telephony network including without limitation GSM, 3GSM, 3G, CDMA, WCDMA or GPRS, satellite phones or other mobile telephone networks or systems; (ii) any so-called WiFi apparatus normally used in a home or subscribed network, but also configured for use on a visited or non-home or non-accustomed network, including apparatus not dedicated to telecommunications such as personal computers, Palm-type or Windows Mobile devices; (iii) an entertainment console platform such as Sony Playstation, PSP or other apparatus that are capable of sending and receiving telecommunications over home or non-home networks, or even (iv) fixed-line devices made for receiving communications, but capable of deployment in numerous locations while preserving a persistent subscriber id such as the eye2eye devices from Dlink; or telecommunications equipment meant for voice over IP communications such as those provided by Vonage or Packet8. \\n     In describing certain embodiments of the system under the present invention, this specification follows the path of a telecommunications call, from a calling party to a called party. For the avoidance of doubt, such a call can be a normal voice call, in which the subscriber telecommunications equipment is also capable of visual, audiovisual or motion-picture display. Alternatively, those devices or calls can be for text, video, pictures or other communicated data. \\n     In the foregoing specification, specific embodiments of the present invention have been described. However, one of ordinary skill in the art will appreciate that various modifications and changes can be made without departing from the scope of the present invention as set forth in the claims below. Accordingly, the specification and the figures are to be regarded in an illustrative rather than a restrictive sense, and all such modifications are intended to be included within the scope of present invention. The benefits, advantages, solutions to problems, and any element(s) that may cause any benefit, advantage, or solution to occur, or to become more pronounced, are not to be construed as a critical, required, or essential feature or element of any or all of the claims. \\n     APPENDIX \\n                                 Acronym   Description                  3G   Third generation of mobile       ACM   ISUP Address Completion Message       ANM   ISUP Answer Message       ANSI-41   American National Standards Institute #41       ATI   Any Time Interrogation       BCSM   Basic Call State Model       BSC   Base Station Controller       BOIC   Barring Outgoing International Calls       BOIC-EX-   Barring Outgoing International Calls except to home       Home   country       CAMEL   Customized Application for Mobile Enhanced Logic       CAP   Camel Application Part       CB   Call Barring       CC   Country Code       CDMA   Code Division Multiplexed Access       CdPA   Called Party Address       CDR   Call Detail Record       CF   Call Forwarding       CgPA   Calling Party Address       CIC   Circuit Identification Code       CLI   Calling Line Identification       CSD   Circuit Switched Data       CSI   Camel Subscription Information       DPC   Destination Point Code       DSD   Delete Subscriber Data       DEA   Diameter Edge Agent       DRA   Diameter Routing Agent       DTMF   Dual Tone Multi-Frequency       ERB   CAP Event Report Basic call state model       EU   European Union       FPMN   Friendly Public Mobile Network       FTN   Forward-To-Number       GLR   Gateway Location Register       GGSN   Gateway GPRS Support Node       GMSC   Gateway MSC       GMSC-F   GMSC in FPMN       GMSC-H   GMSC in HPMN       GPRS   General Packet Radio System       GSM   Global System for Mobile       GSMA   GSM Association       GSM SSF   GSM Service Switching Function       GsmSCF   GSM Service Control Function       GT   Global Title       GTP   GPRS Tunnel Protocol       HLR   Home Location Register       HPMN   Home Public Mobile Network       IN   Intelligent Network       IOT   Inter-Operator Tariff       GTT   Global Title Translation       IAM   Initial Address Message       IDP   Initial DP IN/CAP message       IDD   International Direct Dial       IMSI   International Mobile Subscriber Identity       IMSI-H   HPMN IMSI       IN   Intelligent Network       INAP   Intelligent Network Application Part       INE   Interrogating Network Entity       IP   Internet Protocol       IREG   International Roaming Expert Group       IRS   International Revenue Share       ISC   International Service Carrier       ISD   MAP Insert Subscriber Data       ISG   International Signal Gateway       IST   Immediate Service Termination       ISTP   International STP       ISTP-F   ISTP connected to FPMN STP       ISTP-H   ISTP connected to HPMN STP       ISUP   ISDN User Part       ITPT   Inbound Test Profile Initiation       ITR   Inbound Traffic Redirection       IVR   Interactive Voice Response       LU   Location Update       LUP   MAP Location Update       MAP   Mobile Application Part       MCC   Mobile Country Code       MCC   Mobile Country Code       MD   Missing Data       ME   Mobile Equipment       MGT   Mobile Global Title       MMS   Multimedia Message Service       MMSC   Multimedia Message Service Center       MMSC-F   FPMN MMSC       MMSC-H   HPMN MMSC       MNC   Mobile Network Code       MNP   Mobile Number Portability       MO   Mobile Originated       MOS   Mean Opinion Score       MS   Mobile Station       MSC   Mobile Switching Center       MSISDN   Mobile Station International Subscriber Directory Number       MSISDN-F   FPMN MSISDN       MSISDN-H   HPMN MSISDN       MSRN   Mobile Station Roaming Number       MSRN-F   FPMN MSRN       MSRN-H   HPMN MSRN       MT   Mobile Terminated       MTP   Message Transfer Part       NDC   National Dialing Code       NP   Numbering Plan       NPI   Numbering Plan Indicator       NRTRDE   Near Real Time Roaming Data Exchange       O-CSI   Originating CAMEL Subscription Information       OCN   Original Called Number       ODB   Operator Determined Barring       OPC   Origination Point Code       OR   Optimal Routing       ORLCF   Optimal Routing for Late Call Forwarding       OTA   Over The Air       OTPI   Outbound Test Profile Initiation       PDP   Protocol Data Packet       PDN   Packet Data Network       PDU   Packet Data Unit       PRN   MAP Provide Roaming Number       PSI   MAP Provide Subscriber Information       QoS   Quality of Service       RAEX   Roaming Agreement EXchange       RI   Routing Indicator       RIS   Roaming Intelligence System       RDN   Redirecting Number       RNA   Roaming Not Allowed       RR   Roaming Restricted due to unsupported feature       RRB   CAP Request Report Basic call state model       RSD   Restore Data       RTP   Real-Time Transport Protocol       SAI   Send Authentication Info       SC   Short Code       SCA   Smart Call Assistant       SCCP   Signal Connection Control part       SCP   Signaling Control Point       SF   System Failure       SG   Signaling Gateway       SGSN   Serving GPRS Support Node       SGSN-F   FPMN SGSN       SIM   Subscriber Identity Module       SIGTRAN   Signaling Transport Protocol       SME   Short Message Entity       SM-RP-UI   Short Message Relay Protocol User Information       SMS   Short Message Service       SMSC   Short Message Service Center       SMSC-F   FPMN SMSC       SMSC-H   HPMN SMSC       SoR   Steering of Roaming       SPC   Signal Point Code       SRI   MAP Send Routing Information       SRI-SM   MAP Send Routing Information For Short Message       SS   Supplementary Services       SS7   Signaling System #7       SSN   Sub System Number       SSP   Service Switch Point       STK   SIM Tool Kit Application       STP   Signal Transfer Point       STP-F   FPMN STP       STP-H   HPMN STP       TADIG   Transferred Account Data Interchange Group       TAP   Transferred Account Procedure       TCAP   Transaction Capabilities Application Part       VT-CSI   Visited Terminating CAMEL Service Information       TP   SMS Transport Protocol       TR   Traffic Redirection       TS   Traffic Steering       TT   Translation Type       UD   User Data       UDH   User Data Header       UDHI   User Data Header Indicator       USSD   Unstructured Supplementary Service Data       VAS   Value Added Service       VIP   Very Important Person       VLR   Visited Location Register       VLR-F   FPMN VLR       VLR-H   HPMN VLR       VLR-V   VPMN VLR       VMSC   Visited Mobile Switching Center       VoIP   Voice over IP       VPMN   Visited Public Mobile Network       ATI   Access Transport Information       UDV   Unexpected Data Value       USI   User Service Information       WAP   Wireless Access Protocol                    \\nTechnical References\\n \\n     The entirety of each of the technical references listed below is incorporated by reference herein. \\n     \\n       \\n         \\n           \\n               \\n               \\n             \\n               \\n                   \\n               \\n             \\n            \\n               \\n                 Between AS and SLF \\n                 3GGP TS 29.328 &amp; TS \\n               \\n               \\n                   \\n                 29.329 \\n               \\n               \\n                 Between AS and OFCS \\n                 RFC 4006, 3GGP TS \\n               \\n               \\n                   \\n                 32.225 &amp; TS 32.299 \\n               \\n               \\n                 Between AS and OCS \\n                 RFC 4006, 3GGP TS \\n               \\n               \\n                   \\n                 32.225 &amp; TS 32.299 \\n               \\n               \\n                 Between OCF and Rating function \\n                 3GPP TS 32.296 \\n               \\n               \\n                 Between CSCF and HSS \\n                 3GPP TS 29.228 &amp; \\n               \\n               \\n                   \\n                 TS29.229 \\n               \\n               \\n                 Between CSCF and SLF \\n                 3GPP TS 29.228 &amp; \\n               \\n               \\n                   \\n                 TS29.229 \\n               \\n               \\n                 Between PCRF and SPR \\n                 3GPP TS 23.203, TS \\n               \\n               \\n                   \\n                 29.328 &amp; TS 29.329 \\n               \\n               \\n                 Between AF and the PCRF \\n                 3GPP TS 23.203 &amp; TS \\n               \\n               \\n                   \\n                 29.214 \\n               \\n               \\n                 Rx reference point for EPC \\n                 3GPP TS 32820, 3GPP \\n               \\n               \\n                   \\n                 TS 23.203 &amp; TS 29.214 \\n               \\n               \\n                 Between PCEF and the PCRF \\n                 3GPP TS 29.212 &amp; TS \\n               \\n               \\n                   \\n                 23.203 \\n               \\n               \\n                 Between OCS and PCEF \\n                 3GPP TS 32.29, TS \\n               \\n               \\n                   \\n                 32.251 &amp; RFC 4006 \\n               \\n               \\n                 Between PCEF and OFCS \\n                 3GPP TS 32.240, TS \\n               \\n               \\n                   \\n                 32.295 \\n               \\n               \\n                 Between AF and PDF \\n                 3GPP TS 29.209 \\n               \\n               \\n                 Between Packet Domain and an external \\n                 3GPP TS 29.061 \\n               \\n               \\n                 packet data network \\n               \\n               \\n                 between the EPC based PLMN and the packet \\n                 3GPP TS 29.061 \\n               \\n               \\n                 data network \\n               \\n               \\n                 Between BSF and HSS \\n                 3GPP TS 29.109 &amp; TS \\n               \\n               \\n                   \\n                 33.220 \\n               \\n               \\n                 Between BSF and SLF \\n                 3GPP TS 29.109 &amp; TS \\n               \\n               \\n                   \\n                 33.220 \\n               \\n               \\n                 Between BSF and NAF \\n                 3GPP TS 29.109 &amp; TS \\n               \\n               \\n                   \\n                 33.220 \\n               \\n               \\n                 Between BSF and Zn Proxy \\n                 3GPP TS 29.109 &amp; TS \\n               \\n               \\n                   \\n                 33.220 \\n               \\n               \\n                 Between the 3GPP AAA Server and an SLF \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Between the WLAN AN and the 3GPP AAA \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Proxy \\n               \\n               \\n                 Between the 3GPP AAA Proxy and 3GPP \\n                 3GPP TS 29.234 \\n               \\n               \\n                 AAA Server \\n               \\n               \\n                 Between the 3GPP AAA Server and the HSS \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Between the 3GPP AAA Server and the PDG \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Between the 3GPP AAA Server/Proxy and the \\n                 3GPP TS 29.234 \\n               \\n               \\n                 WAG \\n               \\n               \\n                 Between the 3GPP AAA Server and the PNA \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Between the 3GPP AAA Server and the PDG \\n                 3GPP TS 29.234 \\n               \\n               \\n                 Between GGSN and BM-SC \\n                 3GPP TS 29.061 \\n               \\n               \\n                 Mz is the roaming variant of the Gmb \\n                 3GPP TS 29.061 \\n               \\n               \\n                 reference point with the same functionality \\n               \\n               \\n                 CCF to BS \\n                 3GPP TS TS 32.225 \\n               \\n               \\n                 Multimedia Messaging Service (MMS) \\n                 OMA MM10 interface \\n               \\n               \\n                 Between AGW and PCRF \\n                 3GPP2 TSG-X X.S0013- \\n               \\n               \\n                   \\n                 014 \\n               \\n               \\n                 Between AF and PCRF \\n                 3GPP2 TSG-X X.S0013- \\n               \\n               \\n                   \\n                 013 \\n               \\n               \\n                 Between MME and HSS \\n                 3GPP TS 29.272 \\n               \\n               \\n                 Between the 3GPP AAA Server/Proxy and the \\n                 3GPP TS 23402 \\n               \\n               \\n                 PDN GW \\n               \\n               \\n                 Between MME and HSS \\n                 3GPP TS 29.272 \\n               \\n               \\n                 Transfer of (QoS) policy information from \\n                 3GPP TS 32820, TS \\n               \\n               \\n                 PCRF to the S-GW. \\n                 23402 \\n               \\n               \\n                 Between PCRF in the HPLMN (H PCRF) and a \\n                 3GPP TS 23.203 &amp; TS \\n               \\n               \\n                 PCRF in the VPLMN (V PCRF) \\n                 29.215 \\n               \\n               \\n                 Between MME and EIR \\n                 3GPP TS 29.272 \\n               \\n               \\n                 PCRF and the BBERF \\n                 3GPP TS 23.203 \\n               \\n               \\n                 Between ePDG and vPCRF \\n                 3GPP TS 23.203 \\n               \\n               \\n                 PCRF and the BBERF \\n                 3GPP TS 23.203 \\n               \\n               \\n                 Between an un-trusted non-3GPP IP access \\n                 3GPP TS 23402 \\n               \\n               \\n                 and the 3GPP AAA Server/Proxy \\n               \\n               \\n                 Between the 3GPP AAA Proxy and 3GPP \\n                 3GPP TS 23402 \\n               \\n               \\n                 AAA Server \\n               \\n               \\n                 Between Untrusted Non-3GPP IP Access and \\n                 3GPP TS 23402 \\n               \\n               \\n                 ePDG \\n               \\n               \\n                 Between the 3GPP AAA Server/Proxy and the \\n                 3GPP TS 23402 \\n               \\n               \\n                 ePDG \\n               \\n               \\n                 Between the 3GPP AAA Server and the HSS \\n                 3GPP TS 23402 \\n               \\n               \\n                 Between a trusted non-3GPP IP access and \\n                 3GPP TS 23402 \\n               \\n               \\n                 the 3GPP AAA Server/Proxy \\n               \\n               \\n                 Between the 3GPP AAA Server and the HA \\n                 3GPP TS 23402', 'id': 'US-10028174-B2', 'ti': 'Steering of roaming in LTE and legacy network environment'}>]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mThe Kernel crashed while executing code in the current cell or a previous cell. \n",
      "\u001b[1;31mPlease review the code in the cell(s) to identify a possible cause of the failure. \n",
      "\u001b[1;31mClick <a href='https://aka.ms/vscodeJupyterKernelCrash'>here</a> for more info. \n",
      "\u001b[1;31mView Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
     ]
    }
   ],
   "source": [
    "query = \"visited\"\n",
    "print(whoosh_utils.count_query_tokens(query=query))\n",
    "print(qp.parse(query))\n",
    "results = whoosh_utils.execute_query(query, qp, searcher)[:5]\n",
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.0196078431372549"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test = pl.read_csv(\"/kaggle/input/uspto-boolean-search-optimization/test.csv\")\n",
    "test_patents = set(sum(test.to_numpy().tolist(), []))\n",
    "\n",
    "exist = []\n",
    "for patent in test_patents:\n",
    "    query = f\"id:{patent}\"\n",
    "    results = whoosh_utils.execute_query(query, qp, searcher)\n",
    "    exist.append(len(results) > 0)\n",
    "sum(exist) / len(exist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 200000/200000 [00:40<00:00, 4973.34it/s]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "1.0"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "with open(\"/kaggle/input/uspto-boolean-search-optimization/train_index_patent_ids.json\", \"r\") as f:\n",
    "    train_index_patent_ids = set(json.load(f))\n",
    "\n",
    "exist = []\n",
    "for patent in tqdm(train_index_patent_ids):\n",
    "    query = f\"id:{patent}\"\n",
    "    results = whoosh_utils.execute_query(query, qp, searcher)\n",
    "    exist.append(len(results) > 0)\n",
    "sum(exist) / len(exist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 133077/133077 [00:16<00:00, 7860.05it/s]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0.015336985354343725"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "with open(\"/kaggle/input/patent2parquet/patent2parquet.json\", \"r\") as f:\n",
    "    patent2parquet = json.load(f)\n",
    "\n",
    "patent2parquet = pl.DataFrame({\"publication_number\": list(patent2parquet.keys())})\n",
    "\n",
    "exist = []\n",
    "for patent in tqdm(patent2parquet[\"publication_number\"][::100]):\n",
    "    query = f\"id:{patent}\"\n",
    "    results = whoosh_utils.execute_query(query, qp, searcher)\n",
    "    exist.append(len(results) > 0)\n",
    "sum(exist) / len(exist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
